{
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "TODO: \n",
        "- integrate colab fold\n",
        "- PIP freeze has to be added/check optimum setup\n",
        "- run tests\n",
        "- add option to provide mutations as a list\n",
        "- Getting NotImplementedError: A UTF-8 locale is required. Got ANSI_X3.4-1968 ; added work around to set UTF-8 encoding\n",
        "- pka anii generates _prep.pdb files, do we need to use them?\n",
        "- bash scripts will be finished successfully even if it went wrong. Need to check if file exists after it."
      ],
      "metadata": {
        "id": "WKuDjvoKOuEi"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VotFxKloOQk8"
      },
      "source": [
        "# <b><font color='#009e74'>End-to-end molecular dynamics simulation with AlphaFold and Gromacs </font></b>\n",
        "\n",
        "Preprint pipeline version for running MD simulations using **AlphaFold and GROMACS**. The program, using as input a protein sequence returns MD trajectories and all the associated files to run it.\n",
        "More details can be found in: **Manuscript...:** [\"Manuscript title\"](https://ref). Source code is available on the project [Github](https://github.com/ref) page."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-Q-Y3BeROQk_"
      },
      "source": [
        "##  <b><font color='#009e74'> Reminders and Important informations:</font></b>\n",
        "- This notebook  <b><font color='#d55c00'> can </font></b> be run in a Colab GPU session (go to page menu: `Runtime`->  `Change runtime type` -> select `GPU` and confirm\n",
        "- Cells named as  <b><font color='#56b4e9'>PRELIMINARY OPERATIONS </font></b> have to be run <b><font color='#d55c00'>ONCE only at the start</font></b>  and  skipped for new runnings.\n",
        "- <b><font color='#d55c00'>ONE</font></b> single sequence at the time can be processed by the pipeline. \n",
        "- A  <b><font color='#d55c00'>new run</font></b> can be perform input direcly the new structure in the pdb upload cell and run the pipeline cells again\n",
        "\n",
        "****"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "N98LOBXdOQlA"
      },
      "source": [
        "## <b><font color='#009e74'>PRELIMINARY OPERATIONS </font></b>\n",
        "These cells should be run once at the start of the notebook\n",
        "****"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-mdLQUV-OQlA",
        "outputId": "0434afd4-ef9f-4bc3-e7fa-38389f4eee7b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting condacolab\n",
            "  Downloading condacolab-0.1.5-py3-none-any.whl (6.8 kB)\n",
            "Installing collected packages: condacolab\n",
            "Successfully installed condacolab-0.1.5\n",
            "⏬ Downloading https://github.com/jaimergp/miniforge/releases/latest/download/Mambaforge-colab-Linux-x86_64.sh...\n",
            "📦 Installing...\n",
            "📌 Adjusting configuration...\n",
            "🩹 Patching environment...\n",
            "⏲ Done in 0:00:14\n",
            "🔁 Restarting kernel...\n"
          ]
        }
      ],
      "source": [
        "#@title Install condacolab\n",
        "#@markdown Run this cell to install condacolab. After running this cell the kernel will be automatically restarted (wait ~1min before run the next cell)\n",
        "\n",
        "#@markdown **N.B: This cell should be run only ONCE at the START of the notebook.**\n",
        "try:\n",
        "    import google.colab\n",
        "    !pip install condacolab\n",
        "    import condacolab\n",
        "    condacolab.install()\n",
        "except ModuleNotFoundError:\n",
        "    pass"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5iYQ0-TnspXN",
        "outputId": "83e01766-8450-486a-d496-45625cb57e48"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Get:1 https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/ InRelease [3,622 B]\n",
            "Get:2 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal InRelease [18.1 kB]\n",
            "Hit:3 http://archive.ubuntu.com/ubuntu focal InRelease\n",
            "Get:4 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]\n",
            "Ign:5 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64  InRelease\n",
            "Hit:6 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64  InRelease\n",
            "Hit:7 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64  Release\n",
            "Get:8 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]\n",
            "Hit:10 http://ppa.launchpad.net/cran/libgit2/ubuntu focal InRelease\n",
            "Hit:11 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal InRelease\n",
            "Get:12 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1,013 kB]\n",
            "Get:13 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]\n",
            "Hit:14 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu focal InRelease\n",
            "Get:15 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2,134 kB]\n",
            "Hit:16 http://ppa.launchpad.net/ubuntugis/ppa/ubuntu focal InRelease\n",
            "Get:17 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal/main Sources [2,396 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,307 kB]\n",
            "Get:19 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [31.2 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3,014 kB]\n",
            "Get:21 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal/main amd64 Packages [1,136 kB]\n",
            "Fetched 11.4 MB in 5s (2,241 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "23 packages can be upgraded. Run 'apt list --upgradable' to see them.\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "g++-9 is already the newest version (9.4.0-1ubuntu1~20.04.1).\n",
            "g++-9 set to manually installed.\n",
            "gcc-9 is already the newest version (9.4.0-1ubuntu1~20.04.1).\n",
            "gcc-9 set to manually installed.\n",
            "0 upgraded, 0 newly installed, 0 to remove and 23 not upgraded.\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n"
          ]
        }
      ],
      "source": [
        "#@title Update gcc\n",
        "#@markdown Run this cell to install gcc-9. This is required for GROMACS\n",
        "!sudo echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal main universe' >> /etc/apt/sources.list\n",
        "!sudo apt update\n",
        "!sudo apt install -y gcc-9 g++-9\n",
        "!ln -snf /usr/bin/gcc-9 /usr/bin/gcc\n",
        "!ln -snf /usr/bin/g++-9 /usr/bin/g++"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "TSFKHvOjtZaH",
        "outputId": "5608527c-8bc7-4b33-cca8-6b9b0057eb35"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting biopython\n",
            "  Downloading biopython-1.81-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m55.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.8/site-packages (from biopython) (1.24.2)\n",
            "Installing collected packages: biopython\n",
            "Successfully installed biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Installing biopython\n",
        "!pip install biopython\n",
        "from Bio.PDB import *\n",
        "import Bio.SeqUtils\n",
        "import Bio.SeqIO\n",
        "from Bio.SeqRecord import SeqRecord\n",
        "from Bio.Seq import Seq"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pm9AAM1YtsT3",
        "outputId": "2e39620d-1691-4d25-be4b-a241f468c8c2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): ...working... done\n",
            "Solving environment: ...working... done\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - openmm\n",
            "    - pdbfixer\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ocl-icd-2.3.1              |       h7f98852_0         119 KB  conda-forge\n",
            "    ocl-icd-system-1.0.0       |                1           4 KB  conda-forge\n",
            "    openmm-8.0.0               |   py38he9472fe_0        10.7 MB  conda-forge\n",
            "    pdbfixer-1.8.1             |     pyh6c4a22f_0         498 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        11.3 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  ocl-icd            conda-forge/linux-64::ocl-icd-2.3.1-h7f98852_0 \n",
            "  ocl-icd-system     conda-forge/linux-64::ocl-icd-system-1.0.0-1 \n",
            "  openmm             conda-forge/linux-64::openmm-8.0.0-py38he9472fe_0 \n",
            "  pdbfixer           conda-forge/noarch::pdbfixer-1.8.1-pyh6c4a22f_0 \n",
            "\n",
            "The following packages will be SUPERSEDED by a higher-priority channel:\n",
            "\n",
            "  ca-certificates    anaconda::ca-certificates-2023.01.10-~ --> conda-forge::ca-certificates-2022.12.7-ha878542_0 \n",
            "  certifi            anaconda/linux-64::certifi-2022.12.7-~ --> conda-forge/noarch::certifi-2022.12.7-pyhd8ed1ab_0 \n",
            "  conda               anaconda::conda-23.1.0-py38h06a4308_0 --> conda-forge::conda-23.1.0-py38h578d9bd_0 \n",
            "\n",
            "\n",
            "Preparing transaction: ...working... done\n",
            "Verifying transaction: ...working... done\n",
            "Executing transaction: ...working... done\n"
          ]
        }
      ],
      "source": [
        "#@title Installing PDBfixer\n",
        "#@markdown The repair of PDB is now done with PDBfixer\n",
        "\n",
        "#@markdown We can also add other options, like modeler, later on\n",
        "!conda install -q -y -c conda-forge pdbfixer openmm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ubtGWa9ctyN3",
        "outputId": "8094e16a-8a4d-4b94-c854-c21c146a5bf1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting py3Dmol\n",
            "  Downloading py3Dmol-2.0.1.post1-py2.py3-none-any.whl (12 kB)\n",
            "Installing collected packages: py3Dmol\n",
            "Successfully installed py3Dmol-2.0.1.post1\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mLooking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting pdb-tools\n",
            "  Downloading pdb_tools-2.5.0-py3-none-any.whl (207 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m207.3/207.3 kB\u001b[0m \u001b[31m4.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: pdb-tools\n",
            "Successfully installed pdb-tools-2.5.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Install py3Dmol and pdb-tools\n",
        "!pip install py3Dmol\n",
        "import py3Dmol\n",
        "!pip install pdb-tools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dahUiTcwstzr",
        "outputId": "e0013345-e53d-4d00-a9e1-b70215844562"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting wget\n",
            "  Downloading wget-3.2.zip (10 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Building wheels for collected packages: wget\n",
            "  Building wheel for wget (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for wget: filename=wget-3.2-py3-none-any.whl size=9656 sha256=f2541d248e9012b224822179ab604253c2add2baf94be252f700fe289189a658\n",
            "  Stored in directory: /root/.cache/pip/wheels/46/4a/43/6e71c9584e8b20b326931deaf89f11bf4db1bda170ac38e7b4\n",
            "Successfully built wget\n",
            "Installing collected packages: wget\n",
            "Successfully installed wget-3.2\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title python libs\n",
        "!pip install wget\n",
        "import os\n",
        "import subprocess\n",
        "import shutil\n",
        "import pdbfixer\n",
        "from openmm.app import PDBFile\n",
        "import wget\n",
        "import tarfile\n",
        "import pandas as pd\n",
        "import plotly.graph_objects as go\n",
        "import plotly.express as px\n",
        "import numpy as np\n",
        "from google.colab import files"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WFX55eW2ulgc",
        "outputId": "881db0c8-85de-4732-87a1-c9ab8ffe2cd2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - pytorch\n",
            "    - torchvision\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    _openmp_mutex-4.5          |       2_kmp_llvm           6 KB  conda-forge\n",
            "    aom-3.5.0                  |       h27087fc_0         2.7 MB  conda-forge\n",
            "    cudnn-8.4.1.50             |       hed8a83a_0       618.4 MB  conda-forge\n",
            "    expat-2.5.0                |       h27087fc_0         189 KB  conda-forge\n",
            "    ffmpeg-5.1.2               | gpl_h8dda1f0_106         9.2 MB  conda-forge\n",
            "    font-ttf-dejavu-sans-mono-2.37|       hab24e00_0         388 KB  conda-forge\n",
            "    font-ttf-inconsolata-3.000 |       h77eed37_0          94 KB  conda-forge\n",
            "    font-ttf-source-code-pro-2.038|       h77eed37_0         684 KB  conda-forge\n",
            "    font-ttf-ubuntu-0.83       |       hab24e00_0         1.9 MB  conda-forge\n",
            "    fontconfig-2.14.2          |       h14ed4e7_0         266 KB  conda-forge\n",
            "    fonts-conda-ecosystem-1    |                0           4 KB  conda-forge\n",
            "    fonts-conda-forge-1        |                0           4 KB  conda-forge\n",
            "    gettext-0.21.1             |       h27087fc_0         4.1 MB  conda-forge\n",
            "    gmp-6.2.1                  |       h58526e2_0         806 KB  conda-forge\n",
            "    gnutls-3.7.8               |       hf3e180e_0         2.2 MB  conda-forge\n",
            "    lame-3.100                 |    h166bdaf_1003         496 KB  conda-forge\n",
            "    libdrm-2.4.114             |       h166bdaf_0         298 KB  conda-forge\n",
            "    libhwloc-2.9.0             |       hd6dc26d_0         2.4 MB  conda-forge\n",
            "    libidn2-2.3.4              |       h166bdaf_0         157 KB  conda-forge\n",
            "    libopus-1.3.1              |       h7f98852_1         255 KB  conda-forge\n",
            "    libpciaccess-0.17          |       h166bdaf_0          39 KB  conda-forge\n",
            "    libprotobuf-3.21.12        |       h3eb15da_0         2.1 MB  conda-forge\n",
            "    libtasn1-4.19.0            |       h166bdaf_0         114 KB  conda-forge\n",
            "    libunistring-0.9.10        |       h7f98852_0         1.4 MB  conda-forge\n",
            "    libva-2.17.0               |       h0b41bf4_0         183 KB  conda-forge\n",
            "    libvpx-1.11.0              |       h9c3ff4c_3         1.1 MB  conda-forge\n",
            "    llvm-openmp-15.0.7         |       h0cdce71_0         3.1 MB  conda-forge\n",
            "    magma-2.6.2                |       hc72dce7_0       231.5 MB  conda-forge\n",
            "    mkl-2022.2.1               |   h84fe81f_16997       157.3 MB  conda-forge\n",
            "    nccl-2.14.3.1              |       h0800d71_0       145.2 MB  conda-forge\n",
            "    nettle-3.8.1               |       hc379101_1         1.1 MB  conda-forge\n",
            "    ninja-1.11.1               |       h924138e_0         2.1 MB  conda-forge\n",
            "    openh264-2.3.1             |       hcb278e6_2         702 KB  conda-forge\n",
            "    p11-kit-0.24.1             |       hc5aa10d_0         4.5 MB  conda-forge\n",
            "    pytorch-1.13.1             |cuda112py38hd94e077_200       362.3 MB  conda-forge\n",
            "    pytorch-mutex-1.0          |              cpu           3 KB  pytorch\n",
            "    sleef-3.5.1                |       h9b69904_2         1.5 MB  conda-forge\n",
            "    svt-av1-1.4.1              |       hcb278e6_0         2.4 MB  conda-forge\n",
            "    tbb-2021.8.0               |       hf52228f_0         1.4 MB  conda-forge\n",
            "    torchvision-0.14.1         |         py38_cpu        25.3 MB  pytorch\n",
            "    x264-1!164.3095            |       h166bdaf_2         877 KB  conda-forge\n",
            "    x265-3.5                   |       h924138e_3         3.2 MB  conda-forge\n",
            "    xorg-fixesproto-5.0        |    h7f98852_1002           9 KB  conda-forge\n",
            "    xorg-kbproto-1.0.7         |    h7f98852_1002          27 KB  conda-forge\n",
            "    xorg-libx11-1.8.4          |       h0b41bf4_0         810 KB  conda-forge\n",
            "    xorg-libxext-1.3.4         |       h0b41bf4_2          49 KB  conda-forge\n",
            "    xorg-libxfixes-5.0.3       |    h7f98852_1004          18 KB  conda-forge\n",
            "    xorg-xextproto-7.3.0       |    h0b41bf4_1003          30 KB  conda-forge\n",
            "    xorg-xproto-7.0.31         |    h7f98852_1007          73 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        1.56 GB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  aom                conda-forge/linux-64::aom-3.5.0-h27087fc_0 \n",
            "  cudnn              conda-forge/linux-64::cudnn-8.4.1.50-hed8a83a_0 \n",
            "  expat              conda-forge/linux-64::expat-2.5.0-h27087fc_0 \n",
            "  ffmpeg             conda-forge/linux-64::ffmpeg-5.1.2-gpl_h8dda1f0_106 \n",
            "  font-ttf-dejavu-s~ conda-forge/noarch::font-ttf-dejavu-sans-mono-2.37-hab24e00_0 \n",
            "  font-ttf-inconsol~ conda-forge/noarch::font-ttf-inconsolata-3.000-h77eed37_0 \n",
            "  font-ttf-source-c~ conda-forge/noarch::font-ttf-source-code-pro-2.038-h77eed37_0 \n",
            "  font-ttf-ubuntu    conda-forge/noarch::font-ttf-ubuntu-0.83-hab24e00_0 \n",
            "  fontconfig         conda-forge/linux-64::fontconfig-2.14.2-h14ed4e7_0 \n",
            "  fonts-conda-ecosy~ conda-forge/noarch::fonts-conda-ecosystem-1-0 \n",
            "  fonts-conda-forge  conda-forge/noarch::fonts-conda-forge-1-0 \n",
            "  gettext            conda-forge/linux-64::gettext-0.21.1-h27087fc_0 \n",
            "  gmp                conda-forge/linux-64::gmp-6.2.1-h58526e2_0 \n",
            "  gnutls             conda-forge/linux-64::gnutls-3.7.8-hf3e180e_0 \n",
            "  lame               conda-forge/linux-64::lame-3.100-h166bdaf_1003 \n",
            "  libdrm             conda-forge/linux-64::libdrm-2.4.114-h166bdaf_0 \n",
            "  libhwloc           conda-forge/linux-64::libhwloc-2.9.0-hd6dc26d_0 \n",
            "  libidn2            conda-forge/linux-64::libidn2-2.3.4-h166bdaf_0 \n",
            "  libopus            conda-forge/linux-64::libopus-1.3.1-h7f98852_1 \n",
            "  libpciaccess       conda-forge/linux-64::libpciaccess-0.17-h166bdaf_0 \n",
            "  libprotobuf        conda-forge/linux-64::libprotobuf-3.21.12-h3eb15da_0 \n",
            "  libtasn1           conda-forge/linux-64::libtasn1-4.19.0-h166bdaf_0 \n",
            "  libunistring       conda-forge/linux-64::libunistring-0.9.10-h7f98852_0 \n",
            "  libva              conda-forge/linux-64::libva-2.17.0-h0b41bf4_0 \n",
            "  libvpx             conda-forge/linux-64::libvpx-1.11.0-h9c3ff4c_3 \n",
            "  llvm-openmp        conda-forge/linux-64::llvm-openmp-15.0.7-h0cdce71_0 \n",
            "  magma              conda-forge/linux-64::magma-2.6.2-hc72dce7_0 \n",
            "  mkl                conda-forge/linux-64::mkl-2022.2.1-h84fe81f_16997 \n",
            "  nccl               conda-forge/linux-64::nccl-2.14.3.1-h0800d71_0 \n",
            "  nettle             conda-forge/linux-64::nettle-3.8.1-hc379101_1 \n",
            "  ninja              conda-forge/linux-64::ninja-1.11.1-h924138e_0 \n",
            "  openh264           conda-forge/linux-64::openh264-2.3.1-hcb278e6_2 \n",
            "  p11-kit            conda-forge/linux-64::p11-kit-0.24.1-hc5aa10d_0 \n",
            "  pytorch            conda-forge/linux-64::pytorch-1.13.1-cuda112py38hd94e077_200 \n",
            "  pytorch-mutex      pytorch/noarch::pytorch-mutex-1.0-cpu \n",
            "  sleef              conda-forge/linux-64::sleef-3.5.1-h9b69904_2 \n",
            "  svt-av1            conda-forge/linux-64::svt-av1-1.4.1-hcb278e6_0 \n",
            "  tbb                conda-forge/linux-64::tbb-2021.8.0-hf52228f_0 \n",
            "  torchvision        pytorch/linux-64::torchvision-0.14.1-py38_cpu \n",
            "  x264               conda-forge/linux-64::x264-1!164.3095-h166bdaf_2 \n",
            "  x265               conda-forge/linux-64::x265-3.5-h924138e_3 \n",
            "  xorg-fixesproto    conda-forge/linux-64::xorg-fixesproto-5.0-h7f98852_1002 \n",
            "  xorg-kbproto       conda-forge/linux-64::xorg-kbproto-1.0.7-h7f98852_1002 \n",
            "  xorg-libx11        conda-forge/linux-64::xorg-libx11-1.8.4-h0b41bf4_0 \n",
            "  xorg-libxext       conda-forge/linux-64::xorg-libxext-1.3.4-h0b41bf4_2 \n",
            "  xorg-libxfixes     conda-forge/linux-64::xorg-libxfixes-5.0.3-h7f98852_1004 \n",
            "  xorg-xextproto     conda-forge/linux-64::xorg-xextproto-7.3.0-h0b41bf4_1003 \n",
            "  xorg-xproto        conda-forge/linux-64::xorg-xproto-7.0.31-h7f98852_1007 \n",
            "\n",
            "The following packages will be DOWNGRADED:\n",
            "\n",
            "  _openmp_mutex                                   4.5-2_gnu --> 4.5-2_kmp_llvm \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "libtasn1-4.19.0      | 114 KB    | :   0% 0/1 [00:00<?, ?it/s]\n",
            "fonts-conda-forge-1  | 4 KB      | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "fontconfig-2.14.2    | 266 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libva-2.17.0         | 183 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "libunistring-0.9.10  | 1.4 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "expat-2.5.0          | 189 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libidn2-2.3.4        | 157 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xproto-7.0.31   | 73 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xextproto-7.3.0 | 30 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-mutex-1.0    | 3 KB      | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libhwloc-2.9.0       | 2.4 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.8.4    | 810 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.8.0         | 1.4 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "fonts-conda-forge-1  | 4 KB      | : 100% 1.0/1 [00:00<00:00,  5.98it/s]\u001b[A\n",
            "\n",
            "libtasn1-4.19.0      | 114 KB    | :  14% 0.14018035900682763/1 [00:00<00:01,  1.31s/it]\n",
            "fonts-conda-forge-1  | 4 KB      | : 100% 1.0/1 [00:00<00:00,  5.98it/s]\u001b[A\n",
            "\n",
            "\n",
            "libva-2.17.0         | 183 KB    | :   9% 0.08751388770190582/1 [00:00<00:01,  2.12s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :   1% 0.014286486609034739/1 [00:00<00:11, 12.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "libunistring-0.9.10  | 1.4 MB    | :   1% 0.011429878976110548/1 [00:00<00:17, 17.60s/it]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :   0% 0.0017028885600582624/1 [00:00<01:49, 109.84s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "expat-2.5.0          | 189 KB    | :   8% 0.0844427264527767/1 [00:00<00:02,  2.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libidn2-2.3.4        | 157 KB    | :  10% 0.10213890741791296/1 [00:00<00:01,  2.10s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xproto-7.0.31   | 73 KB     | :  22% 0.21868076132511144/1 [00:00<00:00,  1.07s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   0% 0.0006183991486443362/1 [00:00<06:24, 384.44s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | :   2% 0.018254176935383958/1 [00:00<00:13, 13.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | : 100% 1.0/1 [00:00<00:00,  3.93it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xextproto-7.3.0 | 30 KB     | :  54% 0.5412619755533532/1 [00:00<00:00,  1.95it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :  18% 0.1771004102460593/1 [00:00<00:01,  1.32s/it]    \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-mutex-1.0    | 3 KB      | : 100% 1.0/1 [00:00<00:00,  3.58it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   0% 4.313075232618002e-05/1 [00:00<1:55:55, 6955.47s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libhwloc-2.9.0       | 2.4 MB    | :   1% 0.0063839528373015484/1 [00:00<00:48, 48.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.8.4    | 810 KB    | :   2% 0.01974280370948773/1 [00:00<00:15, 15.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   7% 0.07111590209409865/1 [00:00<00:03,  3.78s/it]   \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :  32% 0.32014304929095333/1 [00:00<00:00,  1.01it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.8.0         | 1.4 MB    | :   1% 0.010792658304727915/1 [00:00<00:35, 35.64s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   0% 0.004140552223313282/1 [00:00<01:15, 75.75s/it]     \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libtasn1-4.19.0      | 114 KB    | : 100% 1.0/1 [00:00<00:00,  2.30it/s]                \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libtasn1-4.19.0      | 114 KB    | : 100% 1.0/1 [00:00<00:00,  2.30it/s]\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :  50% 0.5006492366571291/1 [00:00<00:00,  1.29it/s] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   1% 0.009186850245476344/1 [00:00<00:39, 40.20s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "fontconfig-2.14.2    | 266 KB    | : 100% 1.0/1 [00:00<00:00,  1.97it/s]                 \u001b[A\u001b[A\n",
            "\n",
            "fontconfig-2.14.2    | 266 KB    | : 100% 1.0/1 [00:00<00:00,  1.97it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  23% 0.23004448329569305/1 [00:00<00:01,  1.80s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :  72% 0.7220247494647033/1 [00:00<00:00,  1.60it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   1% 0.014448802029270307/1 [00:00<00:29, 30.16s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libva-2.17.0         | 183 KB    | : 100% 1.0/1 [00:00<00:00,  1.65it/s]                \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libva-2.17.0         | 183 KB    | : 100% 1.0/1 [00:00<00:00,  1.65it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  31% 0.30919957432216805/1 [00:00<00:01,  1.62s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | :  91% 0.9127482681912287/1 [00:00<00:00,  1.69it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "expat-2.5.0          | 189 KB    | : 100% 1.0/1 [00:00<00:00,  1.50it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "expat-2.5.0          | 189 KB    | : 100% 1.0/1 [00:00<00:00,  1.50it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   2% 0.02065963036424023/1 [00:00<00:23, 23.85s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libidn2-2.3.4        | 157 KB    | : 100% 1.0/1 [00:00<00:00,  1.41it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libidn2-2.3.4        | 157 KB    | : 100% 1.0/1 [00:00<00:00,  1.41it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  38% 0.3778418798216894/1 [00:00<00:00,  1.60s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   3% 0.026525412680600713/1 [00:00<00:20, 21.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  46% 0.45761536999680874/1 [00:00<00:00,  1.48s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   3% 0.031787364464394674/1 [00:00<00:22, 22.84s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xproto-7.0.31   | 73 KB     | : 100% 1.0/1 [00:00<00:00,  1.06it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xproto-7.0.31   | 73 KB     | : 100% 1.0/1 [00:00<00:00,  1.06it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  53% 0.5336784652800621/1 [00:00<00:00,  1.43s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   4% 0.03795506204703842/1 [00:01<00:19, 20.73s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  61% 0.6066495648200937/1 [00:01<00:00,  1.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   4% 0.04313075232618002/1 [00:01<00:19, 20.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  71% 0.7117774200896309/1 [00:01<00:00,  1.33s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | : 100% 1.0/1 [00:01<00:00,  1.23s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | : 100% 1.0/1 [00:01<00:00,  1.23s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   5% 0.048220181100669265/1 [00:01<00:20, 21.27s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | : 100% 1.0/1 [00:01<00:00,  3.93it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  82% 0.818760472805101/1 [00:01<00:00,  1.19s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   6% 0.05550927824379369/1 [00:01<00:17, 18.45s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  92% 0.9189411348854835/1 [00:01<00:00,  1.13s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   6% 0.06111627604619709/1 [00:01<00:20, 21.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   7% 0.06926798823584511/1 [00:01<00:16, 17.96s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   8% 0.07539255506616267/1 [00:01<00:16, 17.54s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | : 100% 1.0/1 [00:01<00:00,  1.73s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | : 100% 1.0/1 [00:01<00:00,  1.73s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   8% 0.08229347543835148/1 [00:01<00:15, 16.59s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   9% 0.08854743452564759/1 [00:01<00:15, 16.47s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xextproto-7.3.0 | 30 KB     | : 100% 1.0/1 [00:01<00:00,  2.22s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-xextproto-7.3.0 | 30 KB     | : 100% 1.0/1 [00:01<00:00,  2.22s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "libunistring-0.9.10  | 1.4 MB    | : 100% 1.0/1 [00:01<00:00,  1.93s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "libunistring-0.9.10  | 1.4 MB    | : 100% 1.0/1 [00:01<00:00,  1.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-mutex-1.0    | 3 KB      | : 100% 1.0/1 [00:01<00:00,  3.58it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :   9% 0.09480139361294368/1 [00:02<00:15, 17.53s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  10% 0.10066717592930417/1 [00:02<00:16, 18.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  11% 0.1062310429793814/1 [00:02<00:17, 19.05s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  11% 0.11209682529574187/1 [00:02<00:16, 18.50s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  12% 0.11757443084116674/1 [00:02<00:16, 18.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  12% 0.1230520363865916/1 [00:02<00:16, 19.04s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  13% 0.12835711892271173/1 [00:02<00:19, 22.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.8.4    | 810 KB    | : 100% 1.0/1 [00:02<00:00,  2.70s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.8.4    | 810 KB    | : 100% 1.0/1 [00:02<00:00,  2.70s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  13% 0.13305837092626535/1 [00:02<00:19, 22.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  14% 0.13806153819610226/1 [00:02<00:19, 22.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.8.0         | 1.4 MB    | : 100% 1.0/1 [00:03<00:00,  2.92s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.8.0         | 1.4 MB    | : 100% 1.0/1 [00:03<00:00,  2.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  14% 0.1426765286950035/1 [00:03<00:19, 23.05s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  15% 0.1474640422032095/1 [00:03<00:19, 22.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  15% 0.1519496404451322/1 [00:03<00:19, 22.78s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  16% 0.15639210793472877/1 [00:03<00:19, 22.77s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  16% 0.16122275219526092/1 [00:03<00:18, 22.65s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  17% 0.16609652720811927/1 [00:03<00:18, 22.16s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libhwloc-2.9.0       | 2.4 MB    | : 100% 1.0/1 [00:03<00:00,  2.84it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  17% 0.1710134329733038/1 [00:03<00:17, 21.64s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  18% 0.17584407723383594/1 [00:03<00:17, 21.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  18% 0.18054532923738956/1 [00:03<00:17, 21.83s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  19% 0.18516031973629082/1 [00:04<00:19, 23.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  19% 0.18947339496890883/1 [00:04<00:22, 27.33s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  19% 0.1937002086968745/1 [00:04<00:21, 26.60s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  20% 0.1990484219853208/1 [00:04<00:19, 24.04s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  20% 0.20383593549352677/1 [00:04<00:18, 23.12s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  21% 0.2100898945808229/1 [00:04<00:17, 21.63s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  21% 0.2147911465843765/1 [00:04<00:20, 26.31s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  22% 0.2188885680553636/1 [00:04<00:21, 27.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  22% 0.2227703357647198/1 [00:05<00:21, 27.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  23% 0.22656584196942364/1 [00:05<00:23, 30.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  23% 0.22997317140319187/1 [00:05<00:24, 31.78s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  23% 0.23368241610324336/1 [00:05<00:24, 31.79s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  24% 0.23838366810679698/1 [00:05<00:21, 28.23s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  24% 0.24204978205452227/1 [00:05<00:21, 28.65s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  25% 0.24562963449759523/1 [00:05<00:25, 33.21s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  25% 0.24882131016973255/1 [00:05<00:25, 34.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ffmpeg-5.1.2         | 9.2 MB    | : 100% 1.0/1 [00:05<00:00,  1.69it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  25% 0.2518404628325651/1 [00:06<00:30, 40.17s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  25% 0.2549027462477239/1 [00:06<00:29, 39.91s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  26% 0.2577493759012518/1 [00:06<00:28, 38.79s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  26% 0.26072539781175824/1 [00:06<00:28, 38.75s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  26% 0.2640464657408741/1 [00:06<00:26, 36.07s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  27% 0.268100756459535/1 [00:06<00:24, 32.80s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  27% 0.27124930137934616/1 [00:06<00:23, 32.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  28% 0.27616620714453066/1 [00:06<00:20, 27.89s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  28% 0.28185946645158644/1 [00:06<00:17, 23.88s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  29% 0.28811342553888253/1 [00:07<00:15, 21.18s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  29% 0.2929009390470885/1 [00:07<00:17, 25.18s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  30% 0.29712775277505415/1 [00:07<00:30, 42.86s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  30% 0.30040568995184386/1 [00:07<00:29, 41.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  30% 0.30346797336700265/1 [00:07<00:30, 44.43s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  31% 0.3075222640856636/1 [00:08<00:26, 38.47s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  31% 0.3114902932996721/1 [00:08<00:24, 35.01s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  31% 0.3149838842380927/1 [00:08<00:22, 33.28s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  32% 0.3187793904427965/1 [00:08<00:21, 31.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  32% 0.3222729813812171/1 [00:08<00:21, 31.39s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  33% 0.32568031081498533/1 [00:08<00:25, 37.58s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  33% 0.32861320197316557/1 [00:08<00:25, 37.53s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  33% 0.3321499236639123/1 [00:08<00:23, 35.07s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  34% 0.3354709915930282/1 [00:08<00:23, 35.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  34% 0.33926649779773205/1 [00:09<00:21, 33.10s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  34% 0.34293261174545736/1 [00:09<00:20, 31.55s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  35% 0.3462968104268994/1 [00:09<00:20, 31.38s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  35% 0.34957474760368906/1 [00:09<00:22, 33.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  35% 0.35259390026652165/1 [00:09<00:26, 40.64s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  36% 0.35522487615841863/1 [00:09<00:25, 40.12s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  36% 0.3587615978491654/1 [00:09<00:23, 37.42s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  36% 0.36156509675036713/1 [00:09<00:23, 36.98s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  36% 0.364886164679483/1 [00:09<00:23, 36.43s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  37% 0.3678190558376632/1 [00:10<00:23, 36.51s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  37% 0.3706225547388649/1 [00:10<00:24, 39.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  37% 0.37321039987843574/1 [00:10<00:24, 39.56s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  38% 0.3757982450180065/1 [00:10<00:26, 42.87s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  38% 0.3786880054238606/1 [00:10<00:26, 41.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  38% 0.3811033275541267/1 [00:10<00:31, 50.60s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  38% 0.3832167344181095/1 [00:10<00:32, 52.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  39% 0.38636527933792064/1 [00:10<00:28, 45.76s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  39% 0.38890999372516527/1 [00:11<00:27, 44.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  39% 0.39326619971010945/1 [00:11<00:22, 37.09s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  40% 0.39606969861131114/1 [00:11<00:23, 39.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  40% 0.39947702804507934/1 [00:11<00:22, 37.84s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  40% 0.4034881880114141/1 [00:11<00:20, 34.23s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  41% 0.4070249097021609/1 [00:11<00:19, 32.77s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  41% 0.41094980816384324/1 [00:11<00:18, 31.40s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  41% 0.41418461458830674/1 [00:11<00:18, 32.42s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  42% 0.4173331595081179/1 [00:12<00:19, 32.71s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  42% 0.42048170442792904/1 [00:12<00:19, 34.09s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  42% 0.4237596416047187/1 [00:12<00:19, 33.41s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  43% 0.42677879426755133/1 [00:12<00:20, 35.98s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  43% 0.42962542392107916/1 [00:12<00:20, 36.77s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  43% 0.43307588410717357/1 [00:12<00:20, 35.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  44% 0.43592251376070146/1 [00:12<00:24, 43.76s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  44% 0.43876914341422935/1 [00:12<00:23, 42.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  44% 0.44127072704914777/1 [00:13<00:25, 45.25s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  45% 0.44523875626315634/1 [00:13<00:21, 38.18s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  45% 0.4485598241922722/1 [00:13<00:19, 35.70s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  45% 0.45149271535045243/1 [00:13<00:19, 35.99s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  45% 0.45455499876561123/1 [00:13<00:19, 35.70s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  46% 0.45778980519007473/1 [00:13<00:18, 34.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  46% 0.461197134623843/1 [00:13<00:17, 33.16s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  46% 0.4642594180390017/1 [00:13<00:18, 34.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  47% 0.46814118574835795/1 [00:13<00:17, 33.54s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  47% 0.47154851518212615/1 [00:14<00:18, 35.15s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  47% 0.47443827558798024/1 [00:14<00:19, 36.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  48% 0.4772417744891819/1 [00:14<00:20, 40.02s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  48% 0.47978648887642655/1 [00:14<00:21, 40.39s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  48% 0.4823312032636712/1 [00:14<00:21, 41.77s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  48% 0.48474652539393726/1 [00:14<00:23, 45.11s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  49% 0.4871187167718772/1 [00:14<00:22, 44.36s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  49% 0.4894046466451647/1 [00:14<00:29, 57.58s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  49% 0.49238066855567114/1 [00:15<00:26, 51.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  49% 0.49449407541965396/1 [00:15<00:26, 51.82s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  50% 0.4967800052929415/1 [00:15<00:25, 51.39s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  50% 0.4999716809650788/1 [00:15<00:22, 44.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  50% 0.5023438723430187/1 [00:15<00:23, 46.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  51% 0.505276763501199/1 [00:15<00:21, 43.40s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  51% 0.5076489548791389/1 [00:15<00:22, 45.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  51% 0.5099348847524264/1 [00:15<00:22, 45.08s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  51% 0.5122208146257139/1 [00:15<00:22, 45.90s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  52% 0.5152399672885465/1 [00:16<00:20, 41.47s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  52% 0.5188198197316195/1 [00:16<00:18, 38.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  52% 0.5218389723944521/1 [00:16<00:17, 37.02s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  53% 0.5264539628933533/1 [00:16<00:14, 31.11s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  53% 0.5305513843643405/1 [00:16<00:13, 28.85s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  54% 0.5361152514144176/1 [00:16<00:11, 24.55s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  54% 0.540255803637731/1 [00:16<00:11, 25.68s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  54% 0.5442238328517395/1 [00:16<00:13, 30.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  55% 0.5477174237901601/1 [00:16<00:14, 31.06s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  55% 0.5510816224716021/1 [00:17<00:17, 38.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  56% 0.5551790439425892/1 [00:17<00:15, 34.34s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  56% 0.5600096882031214/1 [00:17<00:13, 29.80s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  56% 0.5649697247206321/1 [00:17<00:11, 26.84s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  57% 0.5701885457520999/1 [00:17<00:10, 24.63s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  58% 0.5758818050591556/1 [00:17<00:09, 23.11s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  58% 0.5817907181278423/1 [00:17<00:08, 21.36s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  59% 0.587225192920941/1 [00:17<00:08, 20.96s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  59% 0.5928321907233444/1 [00:18<00:08, 20.04s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  60% 0.5988704960490095/1 [00:18<00:07, 19.07s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  60% 0.6042187093374559/1 [00:18<00:07, 19.14s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  61% 0.6095237918735761/1 [00:18<00:07, 20.36s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  61% 0.6149582666666747/1 [00:18<00:07, 20.06s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  62% 0.6209965719923399/1 [00:18<00:07, 19.47s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  63% 0.6261722622714816/1 [00:18<00:08, 23.86s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  63% 0.6307872527703828/1 [00:18<00:08, 23.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  64% 0.6363079890681338/1 [00:18<00:07, 21.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  64% 0.641138633328666/1 [00:19<00:08, 22.84s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  65% 0.6479101614438763/1 [00:19<00:07, 20.10s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  65% 0.6546385588067604/1 [00:19<00:06, 18.34s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  66% 0.6618845251975586/1 [00:19<00:05, 16.79s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  67% 0.6683541380464856/1 [00:19<00:05, 16.50s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  68% 0.6750825354093697/1 [00:19<00:05, 15.99s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  68% 0.6823716325524941/1 [00:19<00:04, 15.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  69% 0.6893156836770091/1 [00:19<00:05, 16.31s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  70% 0.6976399188759619/1 [00:19<00:04, 14.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  70% 0.7044545777434983/1 [00:20<00:04, 15.94s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  71% 0.7114848903726656/1 [00:20<00:04, 15.46s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  72% 0.7186877260111377/1 [00:20<00:04, 15.05s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  73% 0.7261924769158931/1 [00:20<00:04, 14.70s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  73% 0.7330933972880819/1 [00:20<00:04, 15.05s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  74% 0.7398217946509659/1 [00:20<00:04, 15.38s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  75% 0.7463776690045453/1 [00:20<00:03, 15.55s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  75% 0.7534511123860388/1 [00:20<00:03, 15.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  76% 0.7600069867396182/1 [00:20<00:03, 15.31s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  77% 0.7665628610931975/1 [00:20<00:03, 15.92s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  77% 0.772903081685146/1 [00:21<00:03, 17.39s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  78% 0.7793295637817468/1 [00:21<00:03, 16.86s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  79% 0.7856266536213691/1 [00:21<00:03, 16.74s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  79% 0.7923550509842532/1 [00:21<00:03, 16.25s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  80% 0.8002911094122703/1 [00:21<00:03, 15.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  81% 0.8071488990321329/1 [00:21<00:02, 15.51s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  81% 0.814136080908974/1 [00:21<00:02, 15.20s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  83% 0.825867645541695/1 [00:21<00:02, 12.58s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  84% 0.8352701495488023/1 [00:21<00:02, 12.17s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  85% 0.8461390991349996/1 [00:22<00:01, 11.13s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  86% 0.857482486996785/1 [00:22<00:01, 10.34s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  87% 0.8675750830411111/1 [00:22<00:01, 10.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  88% 0.8773226330668278/1 [00:22<00:01, 10.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  89% 0.8870701830925445/1 [00:22<00:01, 10.50s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  90% 0.8966452101089565/1 [00:22<00:01, 12.36s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  91% 0.905400752831171/1 [00:22<00:01, 12.10s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  92% 0.9152776951138663/1 [00:22<00:00, 11.57s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  92% 0.9241626300930593/1 [00:22<00:00, 11.84s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  93% 0.9327887805582953/1 [00:23<00:00, 12.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  94% 0.9412424080142267/1 [00:23<00:00, 11.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  95% 0.951162481049248/1 [00:23<00:00, 11.38s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  96% 0.9601336775330934/1 [00:23<00:00, 11.31s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  97% 0.9691911355215913/1 [00:23<00:00, 11.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  98% 0.9794131238228959/1 [00:23<00:00, 10.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | :  99% 0.9885999740683723/1 [00:23<00:00, 13.18s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | : 100% 0.9989082238743293/1 [00:23<00:00, 12.09s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | : 100% 1.0/1 [00:27<00:00,  1.13s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pytorch-1.13.1       | 362.3 MB  | : 100% 1.0/1 [01:17<00:00, 12.09s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "Preparing transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Verifying transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Executing transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ By downloading and using the cuDNN conda packages, you accept the terms and conditions of the NVIDIA cuDNN EULA -\n",
            "  https://docs.nvidia.com/deeplearning/cudnn/sla/index.html\n",
            "\n",
            "\b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Solving environment: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - cudatoolkit\n",
            "    - torchani=2.2.0\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    lark-parser-0.12.0         |     pyhd8ed1ab_0          78 KB  conda-forge\n",
            "    torchani-2.2               |     pyh9f0ad1d_0        10.1 MB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        10.2 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  lark-parser        conda-forge/noarch::lark-parser-0.12.0-pyhd8ed1ab_0 \n",
            "  torchani           conda-forge/noarch::torchani-2.2-pyh9f0ad1d_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "lark-parser-0.12.0   | 78 KB     | :   0% 0/1 [00:00<?, ?it/s]\n",
            "torchani-2.2         | 10.1 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   0% 0.001547510674555596/1 [00:00<05:31, 332.27s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   1% 0.009285064047333575/1 [00:00<00:55, 56.06s/it] \u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   2% 0.01857012809466715/1 [00:00<00:30, 31.56s/it] \u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   3% 0.027855192142000728/1 [00:00<00:22, 22.88s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   5% 0.04642532023666788/1 [00:01<00:12, 12.96s/it] \u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   6% 0.06499544833133503/1 [00:01<00:09,  9.86s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :   9% 0.08975561912422457/1 [00:01<00:06,  7.32s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :  13% 0.12689587531355886/1 [00:01<00:04,  5.04s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :  17% 0.17332119555022676/1 [00:01<00:03,  3.73s/it]\u001b[A\n",
            "lark-parser-0.12.0   | 78 KB     | :  21% 0.20642300084413703/1 [00:01<00:06,  7.85s/it]\n",
            "lark-parser-0.12.0   | 78 KB     | : 100% 1.0/1 [00:01<00:00,  7.85s/it]                \n",
            "torchani-2.2         | 10.1 MB   | :  43% 0.4302079675264557/1 [00:01<00:00,  1.48s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :  58% 0.5756739709346818/1 [00:01<00:00,  1.12s/it]\u001b[A\n",
            "torchani-2.2         | 10.1 MB   | :  77% 0.773755337277798/1 [00:01<00:00,  1.21it/s] \u001b[A\n",
            "torchani-2.2         | 10.1 MB   | : 100% 1.0/1 [00:03<00:00,  3.17s/it]              \u001b[A\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "Preparing transaction: / \b\b- \b\bdone\n",
            "Verifying transaction: | \b\b/ \b\bdone\n",
            "Executing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - scikit-learn\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    scikit-learn-1.2.1         |   py38h1e1a916_0         7.3 MB  conda-forge\n",
            "    threadpoolctl-3.1.0        |     pyh8a188c0_0          18 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:         7.4 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  scikit-learn       conda-forge/linux-64::scikit-learn-1.2.1-py38h1e1a916_0 \n",
            "  threadpoolctl      conda-forge/noarch::threadpoolctl-3.1.0-pyh8a188c0_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "threadpoolctl-3.1.0  | 18 KB     | :   0% 0/1 [00:00<?, ?it/s]\n",
            "threadpoolctl-3.1.0  | 18 KB     | :  90% 0.8973109151651241/1 [00:00<00:00,  5.12it/s]\n",
            "threadpoolctl-3.1.0  | 18 KB     | : 100% 1.0/1 [00:00<00:00,  5.12it/s]               \n",
            "scikit-learn-1.2.1   | 7.3 MB    | : 100% 1.0/1 [00:00<00:00,  1.49it/s]                 \u001b[A\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "Preparing transaction: / \b\bdone\n",
            "Verifying transaction: \\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install Torch and TorchAni\n",
        "!conda install pytorch torchvision -c pytorch\n",
        "!conda install -c conda-forge torchani=2.2.0 cudatoolkit\n",
        "!conda install scikit-learn"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "u3P301beutJd",
        "outputId": "49d78be3-6e39-45d9-d3f2-ccca81883f87"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "# All requested packages already installed.\n",
            "\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - joblib\n",
            "\n",
            "\n",
            "The following packages will be UPDATED:\n",
            "\n",
            "  ca-certificates    conda-forge::ca-certificates-2022.12.~ --> anaconda::ca-certificates-2023.01.10-h06a4308_0 \n",
            "\n",
            "The following packages will be SUPERSEDED by a higher-priority channel:\n",
            "\n",
            "  certifi            conda-forge/noarch::certifi-2022.12.7~ --> anaconda/linux-64::certifi-2022.12.7-py38h06a4308_0 \n",
            "  conda              conda-forge::conda-23.1.0-py38h578d9b~ --> anaconda::conda-23.1.0-py38h06a4308_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "\n",
            "Preparing transaction: / \b\bdone\n",
            "Verifying transaction: \\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install ASE and Joblib\n",
        "!conda install -c conda-forge ase\n",
        "!conda install -c anaconda joblib"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "f4ZunoI4uUcF",
        "outputId": "89c3cc92-e986-4b4e-b49c-f0b7ca3be2f7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'pKa-ANI'...\n",
            "remote: Enumerating objects: 206, done.\u001b[K\n",
            "remote: Counting objects: 100% (137/137), done.\u001b[K\n",
            "remote: Compressing objects: 100% (95/95), done.\u001b[K\n",
            "remote: Total 206 (delta 80), reused 78 (delta 42), pack-reused 69\u001b[K\n",
            "Receiving objects: 100% (206/206), 39.70 MiB | 13.18 MiB/s, done.\n",
            "Resolving deltas: 100% (108/108), done.\n"
          ]
        }
      ],
      "source": [
        "#@title Get pKa-ANI\n",
        "!git clone https://github.com/isayevlab/pKa-ANI.git"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Lba7CzcVvJVt",
        "outputId": "721e9767-9c0d-42e9-b51e-f30a4889944d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "running install\n",
            "running bdist_egg\n",
            "running egg_info\n",
            "creating pkaani.egg-info\n",
            "writing pkaani.egg-info/PKG-INFO\n",
            "writing dependency_links to pkaani.egg-info/dependency_links.txt\n",
            "writing entry points to pkaani.egg-info/entry_points.txt\n",
            "writing top-level names to pkaani.egg-info/top_level.txt\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest template 'MANIFEST.in'\n",
            "adding license file 'LICENSE'\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "installing library code to build/bdist.linux-x86_64/egg\n",
            "running install_lib\n",
            "running build_py\n",
            "creating build\n",
            "creating build/lib\n",
            "creating build/lib/pkaani\n",
            "copying pkaani/pkaani.py -> build/lib/pkaani\n",
            "copying pkaani/__init__.py -> build/lib/pkaani\n",
            "copying pkaani/prep_pdb.py -> build/lib/pkaani\n",
            "copying pkaani/ani_descriptors.py -> build/lib/pkaani\n",
            "copying pkaani/ase_io_proteindatabank_mod.py -> build/lib/pkaani\n",
            "copying pkaani/run.py -> build/lib/pkaani\n",
            "copying pkaani/__main__.py -> build/lib/pkaani\n",
            "creating build/lib/pkaani/models\n",
            "copying pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FASP.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FGLU.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FHIS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FLYS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FTYR.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "creating build/bdist.linux-x86_64\n",
            "creating build/bdist.linux-x86_64/egg\n",
            "creating build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/pkaani.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__init__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/prep_pdb.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "creating build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FGLU.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FASP.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FLYS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FTYR.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FHIS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/ani_descriptors.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/ase_io_proteindatabank_mod.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/run.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__main__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/pkaani.py to pkaani.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__init__.py to __init__.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/prep_pdb.py to prep_pdb.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ani_descriptors.py to ani_descriptors.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ase_io_proteindatabank_mod.py to ase_io_proteindatabank_mod.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/run.py to run.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__main__.py to __main__.cpython-38.pyc\n",
            "creating build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "creating dist\n",
            "creating 'dist/pkaani-0.1.0-py3.8.egg' and adding 'build/bdist.linux-x86_64/egg' to it\n",
            "removing 'build/bdist.linux-x86_64/egg' (and everything under it)\n",
            "Processing pkaani-0.1.0-py3.8.egg\n",
            "creating /usr/local/lib/python3.8/site-packages/pkaani-0.1.0-py3.8.egg\n",
            "Extracting pkaani-0.1.0-py3.8.egg to /usr/local/lib/python3.8/site-packages\n",
            "Adding pkaani 0.1.0 to easy-install.pth file\n",
            "Installing pkaani script to /usr/local/bin\n",
            "\n",
            "Installed /usr/local/lib/python3.8/site-packages/pkaani-0.1.0-py3.8.egg\n",
            "Processing dependencies for pkaani==0.1.0\n",
            "Finished processing dependencies for pkaani==0.1.0\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'pkaani.models' as data is deprecated, please list it in `packages`.\n",
            "    !!\n",
            "\n",
            "\n",
            "    ############################\n",
            "    # Package would be ignored #\n",
            "    ############################\n",
            "    Python recognizes 'pkaani.models' as an importable package,\n",
            "    but it is not listed in the `packages` configuration of setuptools.\n",
            "\n",
            "    'pkaani.models' has been automatically added to the distribution only\n",
            "    because it may contain data files, but this behavior is likely to change\n",
            "    in future versions of setuptools (and therefore is considered deprecated).\n",
            "\n",
            "    Please make sure that 'pkaani.models' is included as a package by using\n",
            "    the `packages` configuration field or the proper discovery methods\n",
            "    (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
            "    instead of `find_packages(...)`/`find:`).\n",
            "\n",
            "    You can read more about \"package discovery\" and \"data files\" on setuptools\n",
            "    documentation page.\n",
            "\n",
            "\n",
            "!!\n",
            "\n",
            "  check.warn(importable)\n",
            "zip_safe flag not set; analyzing archive contents...\n",
            "pkaani.__pycache__.pkaani.cpython-38: module references __file__\n"
          ]
        }
      ],
      "source": [
        "#@title Install pKa-ANI\n",
        "%%bash\n",
        "cd pKa-ANI\n",
        "python setup.py install"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Z500vZchxNUV",
        "outputId": "52b2590a-1498-4661-a0fa-b51c83ca851c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Solving environment: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - modeller\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    hdf5-1107-1.10.7           |                0         3.1 MB  salilab\n",
            "    modeller-10.4              |   py38h9bf148f_0        16.8 MB  salilab\n",
            "    zlib-1.2.13                |       h166bdaf_4          92 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        19.9 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  hdf5-1107          salilab/linux-64::hdf5-1107-1.10.7-0 \n",
            "  modeller           salilab/linux-64::modeller-10.4-py38h9bf148f_0 \n",
            "  zlib               conda-forge/linux-64::zlib-1.2.13-h166bdaf_4 \n",
            "\n",
            "The following packages will be SUPERSEDED by a higher-priority channel:\n",
            "\n",
            "  conda               anaconda::conda-23.1.0-py38h06a4308_0 --> conda-forge::conda-23.1.0-py38h578d9bd_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "hdf5-1107-1.10.7     | 3.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\n",
            "modeller-10.4        | 16.8 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "zlib-1.2.13          | 92 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "zlib-1.2.13          | 92 KB     | :  17% 0.1741144964346061/1 [00:00<00:00,  1.34it/s]\u001b[A\u001b[A\n",
            "\n",
            "hdf5-1107-1.10.7     | 3.1 MB    | : 100% 1.0/1 [00:03<00:00,  3.82s/it]\n",
            "modeller-10.4        | 16.8 MB   | : 100% 1.0/1 [00:08<00:00,  8.35s/it]\u001b[A\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "Preparing transaction: | \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Executing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install Modeller\n",
        "#@markdown You will need to obtain modeller Licence Key from \n",
        "#@markdown https://salilab.org/modeller/. \n",
        "modellerkey = 'MODELIRANJE' #@param {type:\"string\"}\n",
        "os.environ['KEY_MODELLER']=modellerkey\n",
        "!conda config --add channels salilab\n",
        "!conda install modeller\n",
        "from modeller import *\n",
        "from modeller.automodel import *"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YN2XoO6K0xcI",
        "outputId": "c9a585aa-aa7a-461b-cba6-9a44f48387bb"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting MDAnalysis\n",
            "  Downloading MDAnalysis-2.4.2-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (8.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m8.1/8.1 MB\u001b[0m \u001b[31m39.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: tqdm>=4.43.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (4.65.0)\n",
            "Requirement already satisfied: threadpoolctl in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (3.1.0)\n",
            "Collecting fasteners\n",
            "  Downloading fasteners-0.18-py3-none-any.whl (18 kB)\n",
            "Requirement already satisfied: matplotlib>=1.5.1 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (3.7.0)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (23.0)\n",
            "Requirement already satisfied: biopython>=1.80 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.81)\n",
            "Collecting mmtf-python>=1.0.0\n",
            "  Downloading mmtf_python-1.1.3-py2.py3-none-any.whl (25 kB)\n",
            "Collecting networkx>=2.0\n",
            "  Downloading networkx-3.0-py3-none-any.whl (2.0 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.0/2.0 MB\u001b[0m \u001b[31m70.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numpy>=1.20.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.24.2)\n",
            "Collecting gsd>=1.9.3\n",
            "  Downloading gsd-2.8.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (408 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m408.2/408.2 kB\u001b[0m \u001b[31m39.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: scipy>=1.5.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.10.1)\n",
            "Requirement already satisfied: joblib>=0.12 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.1.1)\n",
            "Collecting GridDataFormats>=0.4.0\n",
            "  Downloading GridDataFormats-1.0.1-py3-none-any.whl (2.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.1/2.1 MB\u001b[0m \u001b[31m73.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting mrcfile\n",
            "  Downloading mrcfile-1.4.3-py2.py3-none-any.whl (43 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m43.9/43.9 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (4.38.0)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (0.11.0)\n",
            "Requirement already satisfied: contourpy>=1.0.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (1.0.7)\n",
            "Requirement already satisfied: pyparsing>=2.3.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (3.0.9)\n",
            "Requirement already satisfied: importlib-resources>=3.2.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (5.12.0)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (9.4.0)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (2.8.2)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (1.4.4)\n",
            "Collecting msgpack>=1.0.0\n",
            "  Downloading msgpack-1.0.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (322 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m322.5/322.5 kB\u001b[0m \u001b[31m36.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: zipp>=3.1.0 in /usr/local/lib/python3.8/site-packages (from importlib-resources>=3.2.0->matplotlib>=1.5.1->MDAnalysis) (3.15.0)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/site-packages (from python-dateutil>=2.7->matplotlib>=1.5.1->MDAnalysis) (1.16.0)\n",
            "Installing collected packages: msgpack, networkx, mrcfile, mmtf-python, gsd, fasteners, GridDataFormats, MDAnalysis\n",
            "Successfully installed GridDataFormats-1.0.1 MDAnalysis-2.4.2 fasteners-0.18 gsd-2.8.0 mmtf-python-1.1.3 mrcfile-1.4.3 msgpack-1.0.4 networkx-3.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Install MDAnalysis\n",
        "!pip install --upgrade MDAnalysis\n",
        "import MDAnalysis as mda\n",
        "from MDAnalysis.analysis import rms, align"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "K8CDGsuuvPyP"
      },
      "source": [
        "# Set up parameters"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Gromacs setup\n",
        "#@markdown We expect GROMACS to be preinstalled with one of our scripts and\n",
        "#@markdown stored on Google Drive\n",
        "from google.colab import drive\n",
        "drive.mount('/content/gdrive')\n",
        "\n",
        "#@markdown Select GROMACS version from dropdown menu\n",
        "gmx_version = '2022' #@param [\"2021\",\"2022\"]\n",
        "#@markdown - Currently versions 2021(2021.5) and 2022(2022.3) are availablle\n",
        "gdrive_path = str('/content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-'+gmx_version+'.zip')\n",
        "local_path = str('/content/gromacs-'+gmx_version)\n",
        "os.environ['GMX_LOCAL_PATH']=local_path\n",
        "os.environ['GMX_ZIP_PATH']=gdrive_path\n",
        "\n",
        "##@title Extract GROMACS binaries\n",
        "!unzip $GMX_ZIP_PATH"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4jNQ29kq5s7P",
        "outputId": "b196e28c-9e8f-41b9-9d8f-7c3a374f995d"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/gdrive\n",
            "Archive:  /content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-2022.zip\n",
            "   creating: gromacs-2022/\n",
            "   creating: gromacs-2022/lib/\n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0.3.1  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2.3.2  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0.1.0  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7.0.0  \n",
            "   creating: gromacs-2022/lib/pkgconfig/\n",
            "  inflating: gromacs-2022/lib/pkgconfig/muparser.pc  \n",
            "  inflating: gromacs-2022/lib/pkgconfig/libgromacs.pc  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0  \n",
            "   creating: gromacs-2022/include/\n",
            "  inflating: gromacs-2022/include/muParserToken.h  \n",
            "  inflating: gromacs-2022/include/muParserTemplateMagic.h  \n",
            "   creating: gromacs-2022/include/nblib/\n",
            "  inflating: gromacs-2022/include/nblib/nbnxmsetuphelpers.h  \n",
            "  inflating: gromacs-2022/include/nblib/integrator.h  \n",
            "   creating: gromacs-2022/include/nblib/listed_forces/\n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/definitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/calculator.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/bondtypes.h  \n",
            "  inflating: gromacs-2022/include/nblib/topology.h  \n",
            "   creating: gromacs-2022/include/nblib/util/\n",
            "  inflating: gromacs-2022/include/nblib/util/traits.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/util/setup.h  \n",
            "  inflating: gromacs-2022/include/nblib/util/util.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/gmxcalculatorcpu.h  \n",
            "  inflating: gromacs-2022/include/nblib/particletype.h  \n",
            "  inflating: gromacs-2022/include/nblib/interactions.h  \n",
            "  inflating: gromacs-2022/include/nblib/tpr.h  \n",
            "  inflating: gromacs-2022/include/nblib/exception.h  \n",
            "  inflating: gromacs-2022/include/nblib/basicdefinitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/nblib.h  \n",
            "  inflating: gromacs-2022/include/nblib/kerneloptions.h  \n",
            "  inflating: gromacs-2022/include/nblib/particlesequencer.h  \n",
            "  inflating: gromacs-2022/include/nblib/box.h  \n",
            "  inflating: gromacs-2022/include/nblib/simulationstate.h  \n",
            "  inflating: gromacs-2022/include/nblib/molecules.h  \n",
            "  inflating: gromacs-2022/include/nblib/vector.h  \n",
            "  inflating: gromacs-2022/include/muParserInt.h  \n",
            "   creating: gromacs-2022/include/gmxapi/\n",
            "  inflating: gromacs-2022/include/gmxapi/status.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/session.h  \n",
            "   creating: gromacs-2022/include/gmxapi/md/\n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdmodule.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdsignals.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/version.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gromacsfwd.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapicompat.h  \n",
            "   creating: gromacs-2022/include/gmxapi/session/\n",
            "  inflating: gromacs-2022/include/gmxapi/session/resources.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/exceptions.h  \n",
            "   creating: gromacs-2022/include/gmxapi/compat/\n",
            "  inflating: gromacs-2022/include/gmxapi/compat/tpr.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/compat/mdparams.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/system.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/context.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md.h  \n",
            "   creating: gromacs-2022/include/gmxapi/mpi/\n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/gmxapi_mpi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/resourceassignment.h  \n",
            "  inflating: gromacs-2022/include/muParserTokenReader.h  \n",
            "  inflating: gromacs-2022/include/muParserBytecode.h  \n",
            "   creating: gromacs-2022/include/gromacs/\n",
            "   creating: gromacs-2022/include/gromacs/mdrun/\n",
            "  inflating: gromacs-2022/include/gromacs/mdrun/simulationinputhandle.h  \n",
            "  inflating: gromacs-2022/include/muParserFixes.h  \n",
            "  inflating: gromacs-2022/include/muParserError.h  \n",
            "  inflating: gromacs-2022/include/muParserDef.h  \n",
            "  inflating: gromacs-2022/include/muParserBase.h  \n",
            "  inflating: gromacs-2022/include/muParser.h  \n",
            "  inflating: gromacs-2022/include/muParserDLL.h  \n",
            "  inflating: gromacs-2022/include/muParserTest.h  \n",
            "  inflating: gromacs-2022/include/muParserCallback.h  \n",
            "   creating: gromacs-2022/bin/\n",
            "  inflating: gromacs-2022/bin/GMXRC.csh  \n",
            "  inflating: gromacs-2022/bin/GMXRC  \n",
            "  inflating: gromacs-2022/bin/xplor2gmx.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.zsh  \n",
            "  inflating: gromacs-2022/bin/gmx    \n",
            "  inflating: gromacs-2022/bin/demux.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion-gmx.bash  \n",
            "   creating: gromacs-2022/share/\n",
            "   creating: gromacs-2022/share/cmake/\n",
            "   creating: gromacs-2022/share/cmake/gmxapi/\n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config.cmake  \n",
            "   creating: gromacs-2022/share/cmake/gromacs/\n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-hints.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config.cmake  \n",
            "   creating: gromacs-2022/share/gromacs/\n",
            "  inflating: gromacs-2022/share/gromacs/README_FreeEnergyModifications.txt  \n",
            "   creating: gromacs-2022/share/gromacs/top/\n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a1.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a5.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-9.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a2.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/oplsaa.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ethanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/1propanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5pe.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residuetypes.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa-n.tst  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atommass.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/export.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/edissoc.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/nsfactor.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-12.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/dgsolv.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip5p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/defselection.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/bonds.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/surface.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/electroneg.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/elements.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a6.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ps.m2p  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sfactor.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/charmm27.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/cmap.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tips3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.arn  \n",
            " extracting: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnanonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnabonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/specbond.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/co-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdwradii.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber03.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber96.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a5.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG45a3.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/phbres.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.xml  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a1.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-8.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atom_nom.tbl  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ha-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ss.map  \n",
            "   creating: gromacs-2022/share/gromacs/top/amberGS.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos54a7.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tmcl.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/dppc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/popc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/random.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.dtd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-11.xvg  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos45a3.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdw-msms.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber94.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ca-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sw.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-10.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/xlateat.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a6.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a2.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexwat-ferguson.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/README  \n",
            "  inflating: gromacs-2022/share/gromacs/top/refi_aa.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/cb-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/COPYING  \n",
            "  inflating: gromacs-2022/share/gromacs/README.tutor  \n",
            "   creating: gromacs-2022/share/man/\n",
            "   creating: gromacs-2022/share/man/man1/\n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-lie.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-density.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mk_angndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-grompp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spol.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spatial.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sasa.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densmap.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dipoles.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmens.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-xpm2ps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pdb2gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wheel.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-insert-molecules.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-tpr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dielectric.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotacf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dos.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-energy.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-clustsize.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-report-methods.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-filter.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gyrate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-editconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-do_dssp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-velacc.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hbond.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-confrms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-view.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-analyze.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_edi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-awh.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tune_pme.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nonbonded-benchmark.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-distance.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-trj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmtraj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-principal.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tcaf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genrestr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-vanhove.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pairdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bundle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_ndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-potential.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dyecoupl.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-help.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-h2order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-disre.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdrun.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mindist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-eneconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-anaeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dump.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helix.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-check.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genion.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sans.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-chi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-msd.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-solvate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rama.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sigeps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-extract-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-angle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-enemat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-current.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gangle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saxs.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-freevolume.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helixorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-x2top.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-select.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pme_error.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saltbr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hydorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trajectory.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-polystat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjcat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-covar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rdf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-traj.1  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Test GROMACS installation\n",
        "%%bash\n",
        "chmod -R 755 $GMX_LOCAL_PATH\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "gmx grompp -h"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cFUZK5OvLzSF",
        "outputId": "91987c2f-bb5e-4119-d4bc-8c9f96680499"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "SYNOPSIS\n",
            "\n",
            "gmx grompp [-f [<.mdp>]] [-c [<.gro/.g96/...>]] [-r [<.gro/.g96/...>]]\n",
            "           [-rb [<.gro/.g96/...>]] [-n [<.ndx>]] [-p [<.top>]]\n",
            "           [-t [<.trr/.cpt/...>]] [-e [<.edr>]] [-qmi [<.inp>]]\n",
            "           [-ref [<.trr/.cpt/...>]] [-po [<.mdp>]] [-pp [<.top>]]\n",
            "           [-o [<.tpr>]] [-imd [<.gro>]] [-[no]v] [-time <real>]\n",
            "           [-[no]rmvsbds] [-maxwarn <int>] [-[no]zero] [-[no]renum]\n",
            "\n",
            "DESCRIPTION\n",
            "\n",
            "gmx grompp (the gromacs preprocessor) reads a molecular topology file, checks\n",
            "the validity of the file, expands the topology from a molecular description to\n",
            "an atomic description. The topology file contains information about molecule\n",
            "types and the number of molecules, the preprocessor copies each molecule as\n",
            "needed. There is no limitation on the number of molecule types. Bonds and\n",
            "bond-angles can be converted into constraints, separately for hydrogens and\n",
            "heavy atoms. Then a coordinate file is read and velocities can be generated\n",
            "from a Maxwellian distribution if requested. gmx grompp also reads parameters\n",
            "for gmx mdrun (eg. number of MD steps, time step, cut-off). Eventually a\n",
            "binary file is produced that can serve as the sole input file for the MD\n",
            "program.\n",
            "\n",
            "gmx grompp uses the atom names from the topology file. The atom names in the\n",
            "coordinate file (option -c) are only read to generate warnings when they do\n",
            "not match the atom names in the topology. Note that the atom names are\n",
            "irrelevant for the simulation as only the atom types are used for generating\n",
            "interaction parameters.\n",
            "\n",
            "gmx grompp uses a built-in preprocessor to resolve includes, macros, etc. The\n",
            "preprocessor supports the following keywords:\n",
            "\n",
            "    #ifdef VARIABLE\n",
            "    #ifndef VARIABLE\n",
            "    #else\n",
            "    #endif\n",
            "    #define VARIABLE\n",
            "    #undef VARIABLE\n",
            "    #include \"filename\"\n",
            "    #include <filename>\n",
            "\n",
            "The functioning of these statements in your topology may be modulated by using\n",
            "the following two flags in your .mdp file:\n",
            "\n",
            "    define = -DVARIABLE1 -DVARIABLE2\n",
            "    include = -I/home/john/doe\n",
            "\n",
            "For further information a C-programming textbook may help you out. Specifying\n",
            "the -pp flag will get the pre-processed topology file written out so that you\n",
            "can verify its contents.\n",
            "\n",
            "When using position restraints, a file with restraint coordinates must be\n",
            "supplied with -r (can be the same file as supplied for -c). For free energy\n",
            "calculations, separate reference coordinates for the B topology can be\n",
            "supplied with -rb, otherwise they will be equal to those of the A topology.\n",
            "\n",
            "Starting coordinates can be read from trajectory with -t. The last frame with\n",
            "coordinates and velocities will be read, unless the -time option is used. Only\n",
            "if this information is absent will the coordinates in the -c file be used.\n",
            "Note that these velocities will not be used when gen_vel = yes in your .mdp\n",
            "file. An energy file can be supplied with -e to read Nose-Hoover and/or\n",
            "Parrinello-Rahman coupling variables.\n",
            "\n",
            "gmx grompp can be used to restart simulations (preserving continuity) by\n",
            "supplying just a checkpoint file with -t. However, for simply changing the\n",
            "number of run steps to extend a run, using gmx convert-tpr is more convenient\n",
            "than gmx grompp. You then supply the old checkpoint file directly to gmx mdrun\n",
            "with -cpi. If you wish to change the ensemble or things like output frequency,\n",
            "then supplying the checkpoint file to gmx grompp with -t along with a new .mdp\n",
            "file with -f is the recommended procedure. Actually preserving the ensemble\n",
            "(if possible) still requires passing the checkpoint file to gmx mdrun -cpi.\n",
            "\n",
            "By default, all bonded interactions which have constant energy due to virtual\n",
            "site constructions will be removed. If this constant energy is not zero, this\n",
            "will result in a shift in the total energy. All bonded interactions can be\n",
            "kept by turning off -rmvsbds. Additionally, all constraints for distances\n",
            "which will be constant anyway because of virtual site constructions will be\n",
            "removed. If any constraints remain which involve virtual sites, a fatal error\n",
            "will result.\n",
            "\n",
            "To verify your run input file, please take note of all warnings on the screen,\n",
            "and correct where necessary. Do also look at the contents of the mdout.mdp\n",
            "file; this contains comment lines, as well as the input that gmx grompp has\n",
            "read. If in doubt, you can start gmx grompp with the -debug option which will\n",
            "give you more information in a file called grompp.log (along with real debug\n",
            "info). You can see the contents of the run input file with the gmx dump\n",
            "program. gmx check can be used to compare the contents of two run input files.\n",
            "\n",
            "The -maxwarn option can be used to override warnings printed by gmx grompp\n",
            "that otherwise halt output. In some cases, warnings are harmless, but usually\n",
            "they are not. The user is advised to carefully interpret the output messages\n",
            "before attempting to bypass them with this option.\n",
            "\n",
            "OPTIONS\n",
            "\n",
            "Options to specify input files:\n",
            "\n",
            " -f      [<.mdp>]           (grompp.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -c      [<.gro/.g96/...>]  (conf.gro)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -r      [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -rb     [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -n      [<.ndx>]           (index.ndx)      (Opt.)\n",
            "           Index file\n",
            " -p      [<.top>]           (topol.top)\n",
            "           Topology file\n",
            " -t      [<.trr/.cpt/...>]  (traj.trr)       (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            " -e      [<.edr>]           (ener.edr)       (Opt.)\n",
            "           Energy file\n",
            " -qmi    [<.inp>]           (topol-qmmm.inp) (Opt.)\n",
            "           Input file for QM program\n",
            "\n",
            "Options to specify input/output files:\n",
            "\n",
            " -ref    [<.trr/.cpt/...>]  (rotref.trr)     (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            "\n",
            "Options to specify output files:\n",
            "\n",
            " -po     [<.mdp>]           (mdout.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -pp     [<.top>]           (processed.top)  (Opt.)\n",
            "           Topology file\n",
            " -o      [<.tpr>]           (topol.tpr)\n",
            "           Portable xdr run input file\n",
            " -imd    [<.gro>]           (imdgroup.gro)   (Opt.)\n",
            "           Coordinate file in Gromos-87 format\n",
            "\n",
            "Other options:\n",
            "\n",
            " -[no]v                     (no)\n",
            "           Be loud and noisy\n",
            " -time   <real>             (-1)\n",
            "           Take frame at or first after this time.\n",
            " -[no]rmvsbds               (yes)\n",
            "           Remove constant bonded interactions with virtual sites\n",
            " -maxwarn <int>             (0)\n",
            "           Number of allowed warnings during input processing. Not for normal\n",
            "           use and may generate unstable systems\n",
            " -[no]zero                  (no)\n",
            "           Set parameters for bonded interactions without defaults to zero\n",
            "           instead of generating an error\n",
            " -[no]renum                 (yes)\n",
            "           Renumber atomtypes and minimize number of atomtypes\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content\n",
            "Command line:\n",
            "  gmx grompp -h\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"In mathematics you don't understand things, you just get used to them\" (John von Neumann)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#PP this is work around the strange bug related to the encoding\n",
        "# It should be UTF-8, but sometimes it becomes ANSI and nothing work\n",
        "import locale\n",
        "print(locale.getpreferredencoding())\n",
        "def getpreferredencoding(do_setlocale = True):\n",
        "    return \"UTF-8\"\n",
        "locale.getpreferredencoding = getpreferredencoding\n",
        "print(locale.getpreferredencoding())"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cZHEYoXpNCbT",
        "outputId": "a91e663f-84f8-4ff9-b441-b0ab14feffed"
      },
      "execution_count": 44,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "UTF-8\n",
            "UTF-8\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title <b><font color='#56b4e9'> PDB Input</font></b>\n",
        "\n",
        "#@markdown Choose between <b><font color='#d55c00'> ONE</font></b> of the possible input sources for the target pdb and <ins>leave the other cells empty or unmarked</ins>\n",
        "#@markdown - AlphaFold2 PDB via Uniprot ID:\n",
        "uniprot_id =''#@param {type:\"string\"}\n",
        "#@markdown - PDB ID (imported from RCSB PDB):\n",
        "pdb_id =''#@param {type:\"string\"}\n",
        "#@markdown - Upload PDB from local drive (in this case you will be prompt to the file upload later)\n",
        "pdb_local =False#@param {type:\"boolean\"}\n",
        "#@markdown - Use PDB from google drive (insert path to pdb file in the google drive)\n",
        "# '/content/gdrive/MyDrive/NESP/data/targetA_relaxed.pdb'\n",
        "\n",
        "#pdb_gdrive ='/content/gdrive/MyDrive/NESP/data/targetA_relaxed.pdb'#@param {type:\"string\"}\n",
        "pdb_gdrive = ''#@param {type:\"string\"}\n",
        "\n",
        "# sequence to test: VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQRVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGTNAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKALGSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK\n",
        "\n",
        "#@markdown - Insert protein sequence to model the target structure using ESMFold\n",
        "run_esmfold = 'VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQRVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGTNAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKALGSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK'#@param {type:\"string\"}\n",
        "\n",
        "\n",
        "#@markdown - Insert protein sequence to model the target structure using AlphaFold\n",
        "run_alphafold =''#@param {type:\"string\"}\n",
        "\n",
        "#@markdown\n",
        "\n",
        "#@markdown Select target chain (default is A)\n",
        "chain_id='A' #@param {type:'string'}\n",
        "\n",
        "project_dir = \"/content/alphagmx_project/\"\n",
        "pdb_path = os.path.join(project_dir, 'target.pdb')\n",
        "pdb_preprocessed_path = os.path.join(project_dir, 'target_preprocessed.pdb')\n",
        "\n",
        "if os.path.exists(project_dir):\n",
        "    print(f'Warning! {project_dir} exists, all files in this directory will be deleted')\n",
        "    #os.remove(project_dir)\n",
        "    shutil.rmtree(project_dir) \n",
        "    os.mkdir(project_dir)\n",
        "else:\n",
        "    os.mkdir(project_dir)\n",
        "\n",
        "if (uniprot_id !='') and (len(uniprot_id)==6) : \n",
        "    print('Loading PDB by Uniprot_ID')\n",
        "    subprocess.call(['curl','-s','-f',f'https://alphafold.ebi.ac.uk/files/AF-{uniprot_id}-F1-model_v2.pdb','-o', pdb_path])\n",
        "elif (pdb_id !='') and (len(pdb_id)==4):\n",
        "    print('Loading PDB by PDB_ID')\n",
        "    subprocess.call(['curl','-s','-f',f'https://files.rcsb.org/download/{pdb_id}.pdb','-o', pdb_path])\n",
        "elif pdb_local:\n",
        "    print('Upload PDB file:')\n",
        "    uploaded_pdb = files.upload()\n",
        "    for fn in uploaded_pdb.keys():\n",
        "        shutil.copy(fn, pdb_path)\n",
        "elif pdb_gdrive!='' and os.path.exists(pdb_gdrive):\n",
        "    print('Using PDB file from gdrive:')\n",
        "    shutil.copy(pdb_gdrive, pdb_path)\n",
        "elif run_esmfold!='':\n",
        "    sequence = run_esmfold\n",
        "    print('Using ESMFold API to model the structure')\n",
        "    os.system(f'curl -X POST --data \"{sequence}\" https://api.esmatlas.com/foldSequence/v1/pdb/ > {pdb_path}')\n",
        "elif run_alphafold!='':\n",
        "    sequence = run_alphafold\n",
        "    print('RUNNING ALPHAFOLD TODO')\n",
        "else:\n",
        "    print(f'ERROR: unrecognized pdb input')\n",
        "\n",
        "\n",
        "#@markdown N.B. This cell will also perform preliminary operations to correcly format the uploaded PDB\n",
        "\n",
        "#@markdown - mark to preprocess pdb (keep only single chain, delete hetatms)\n",
        "flag_preprocess =True#@param {type:\"boolean\"}\n",
        "\n",
        "if flag_preprocess:\n",
        "    print('Preprocessing PDBs...')\n",
        "    #!pdb_selchain -\"$chain_id\" \"$pdb_path\" | pdb_delhetatm | pdb_delres --999:0:1 | pdb_fixinsert | pdb_tidy  > \"$pdb_preprocessed_path\"\n",
        "    !pdb_selchain -\"$chain_id\" \"$pdb_path\" | pdb_delhetatm | pdb_fixinsert | pdb_tidy  > \"$pdb_preprocessed_path\"\n",
        "    print(f'Kept only chain {chain_id}, deleted hetatms, fixed insertion code, sorted residue ids, formatted pdb file...')\n",
        "    parser = PDBParser()\n",
        "    io=PDBIO()\n",
        "    structure = parser.get_structure('X', pdb_preprocessed_path)\n",
        "\n",
        "    #And here we set the residue conformation we want to keep\n",
        "    keepAltID = \"A\"\n",
        "\n",
        "    class KeepOneConfOnly(Select):  # Inherit methods from Select class\n",
        "        def accept_atom(self, atom):\n",
        "            if (not atom.is_disordered()) or atom.get_altloc() == keepAltID:\n",
        "                atom.set_altloc(\" \")  # Eliminate alt location ID before output.\n",
        "                return True\n",
        "            else:  # Alt location was not one to be output.\n",
        "                return False\n",
        "            # end of accept_atom()\n",
        "\n",
        "    #This will keep only conformation for each residue\n",
        "    io.set_structure(structure)\n",
        "\n",
        "    io.save(os.path.join(pdb_preprocessed_path), select=KeepOneConfOnly())\n",
        "    print('Removed alternative side chain conformations...')\n",
        "\n",
        "else:\n",
        "    pdb_preprocessed_path = pdb_path\n",
        "\n",
        "if os.path.exists(pdb_preprocessed_path):\n",
        "  print(f\"Pre-processing PDBs correctly ended\")\n",
        "else:\n",
        "  print(f\"Pre-processing PDB didn't end correcly, please check the inputs\")\n",
        "\n",
        "#@markdown ****"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yN_gGaa_RfB1",
        "outputId": "87f7e272-4a6e-495d-d6d3-d42ea063f9dd"
      },
      "execution_count": 84,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Warning! /content/alphagmx_project/ exists, all files in this directory will be deleted\n",
            "Using ESMFold API to model the structure\n",
            "Preprocessing PDBs...\n",
            "Kept only chain A, deleted hetatms, fixed insertion code, sorted residue ids, formatted pdb file...\n",
            "Removed alternative side chain conformations...\n",
            "Pre-processing PDBs correctly ended\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 86,
      "metadata": {
        "id": "SlmykXYo6HYG"
      },
      "outputs": [],
      "source": [
        "#@title Fix missing residues\n",
        "#@markdown Select the software to deal with missing residues. Currently, only\n",
        "#@markdown `PDBFixer` is available.\n",
        "missing_residues_soft = 'pdbfixer' #@param [\"pdbfixer\",\"modeller\",\"-\"]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 87,
      "metadata": {
        "id": "OVMgDV4OJM_L",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0fff8644-a269-4557-81c5-7567947aa438"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/coordinates/PDB.py:753: UserWarning: Unit cell dimensions not found. CRYST1 record set to unitary values.\n",
            "  warnings.warn(\"Unit cell dimensions not found. \"\n",
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/coordinates/PDB.py:1129: UserWarning: Found no information for attr: 'formalcharges' Using default value of '0'\n",
            "  warnings.warn(\"Found no information for attr: '{}'\"\n"
          ]
        }
      ],
      "source": [
        "#@title Run pdbfixer\n",
        "\n",
        "def prepare_protein(pdb_file, ignore_missing_residues=False, ignore_terminal_missing_residues=True, pdb_out=None):\n",
        "    \"\"\"Use pdbfixer to prepare the protein from a PDB file. Hetero atoms such as ligands are \n",
        "    removed and non-standard residues replaced. Missing atoms to existing residues are added. \n",
        "    Missing residues are ignored by default, but can be included.\n",
        "    \n",
        "    Parameters\n",
        "    ----------\n",
        "    pdb_file: pathlib.Path or str\n",
        "        PDB file containing the system to prepare.\n",
        "    ignore_missing_residues: bool\n",
        "        If missing residues should be ignored or built.\n",
        "    ignore_terminal_missing_residues: bool\n",
        "        If missing residues at the beginning and the end of a chain should be ignored or built.\n",
        "    pdb_out: pathlib.Path or str\n",
        "        Output PDB file containing the system to simulate.\n",
        "    \n",
        "    Returns\n",
        "    -------\n",
        "    fixer: pdbfixer.pdbfixer.PDBFixer\n",
        "        Prepared protein system.\n",
        "    \"\"\"\n",
        "    fixer = pdbfixer.PDBFixer(str(pdb_file))\n",
        "    fixer.removeHeterogens()  # co-crystallized ligands are unknown to PDBFixer\n",
        "    fixer.findMissingResidues()  # identify missing residues, needed for identification of missing atoms\n",
        "    \n",
        "    # if missing terminal residues shall be ignored, remove them from the dictionary\n",
        "    if ignore_terminal_missing_residues:  \n",
        "        chains = list(fixer.topology.chains())\n",
        "        keys = fixer.missingResidues.keys()\n",
        "        for key in list(keys):\n",
        "            chain = chains[key[0]]\n",
        "            if key[1] == 0 or key[1] == len(list(chain.residues())):\n",
        "                del fixer.missingResidues[key]\n",
        "                \n",
        "    # if all missing residues shall be ignored ignored, clear the dictionary\n",
        "    if ignore_missing_residues:  \n",
        "        fixer.missingResidues = {}\n",
        "        \n",
        "    fixer.findNonstandardResidues()  # find non-standard residue\n",
        "    fixer.replaceNonstandardResidues()  # replace non-standard residues with standard one\n",
        "    fixer.findMissingAtoms()  # find missing heavy atoms\n",
        "    fixer.addMissingAtoms()  # add missing atoms\n",
        "    PDBFile.writeFile(fixer.topology, fixer.positions, open(pdb_out, 'w'))\n",
        "\n",
        "if missing_residues_soft == '-':\n",
        "    pass\n",
        "elif missing_residues_soft == \"pdbfixer\":\n",
        "    prepare_protein(pdb_preprocessed_path, pdb_out=pdb_preprocessed_path)\n",
        "else:\n",
        "    print('Warning! Currently only pdbfixer option is available for missing residues')\n",
        "\n",
        "u = mda.Universe(pdb_preprocessed_path)\n",
        "#u.select_atoms(\"not resname HOH\").write(os.path.splitext(pdb_preprocessed_path)[0]+\"_no_water.pdb\")\n",
        "u.select_atoms(\"not resname HOH\").write(os.path.splitext(pdb_preprocessed_path)[0]+\".pdb\") #PP: why do we need a separate file for _no_water.pdb?"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "vKOCRWjtJ70X"
      },
      "source": [
        "# Display structure"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 88,
      "metadata": {
        "id": "9T3rn1_OJa2l"
      },
      "outputs": [],
      "source": [
        "def show_pdb(pdb_file, show_sidechains=False, show_mainchains=False):\n",
        "  model_name = \"Parsed structure\"\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "  view.addModel(open(pdb_file,'r').read(),'pdb')\n",
        "\n",
        "  view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  \n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})  \n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "  view.zoomTo()\n",
        "  return view"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 89,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 515
        },
        "id": "GoXENrlQKrRr",
        "outputId": "4d34b4a5-6e47-49e2-b073-dd44f723883f"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1678119207039042\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_1678119207039042\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_1678119207039042 = null;\nvar warn = document.getElementById(\"3dmolwarning_1678119207039042\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1678119207039042 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1678119207039042\"),{backgroundColor:\"white\"});\nviewer_1678119207039042.zoomTo();\n\tviewer_1678119207039042.addModel(\"HEADER    \\nTITLE     MDANALYSIS FRAME 0: Created by PDBWriter\\nREMARK     1 CREATED WITH OPENMM 8.0, 2023-03-06\\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1\\nREMARK     285 UNITARY VALUES FOR THE UNIT CELL AUTOMATICALLY SET\\nREMARK     285 BY MDANALYSIS PDBWRITER BECAUSE UNIT CELL INFORMATION\\nREMARK     285 WAS MISSING.\\nREMARK     285 PROTEIN DATA BANK CONVENTIONS REQUIRE THAT\\nREMARK     285 CRYST1 RECORD IS INCLUDED, BUT THE VALUES ON\\nREMARK     285 THIS RECORD ARE MEANINGLESS.\\nATOM      1  N   VAL A   1       6.677 -18.759 -21.233  1.00  0.00      A    N  \\nATOM      2  CA  VAL A   1       7.160 -17.735 -20.313  1.00  0.00      A    C  \\nATOM      3  C   VAL A   1       6.887 -18.164 -18.873  1.00  0.00      A    C  \\nATOM      4  CB  VAL A   1       6.506 -16.364 -20.594  1.00  0.00      A    C  \\nATOM      5  O   VAL A   1       5.747 -18.466 -18.514  1.00  0.00      A    O  \\nATOM      6  CG1 VAL A   1       7.193 -15.262 -19.790  1.00  0.00      A    C  \\nATOM      7  CG2 VAL A   1       6.552 -16.048 -22.088  1.00  0.00      A    C  \\nATOM      8  N   PRO A   2       7.887 -18.889 -18.155  1.00  0.00      A    N  \\nATOM      9  CA  PRO A   2       7.602 -19.467 -16.839  1.00  0.00      A    C  \\nATOM     10  C   PRO A   2       6.430 -18.785 -16.137  1.00  0.00      A    C  \\nATOM     11  CB  PRO A   2       8.905 -19.245 -16.067  1.00  0.00      A    C  \\nATOM     12  O   PRO A   2       6.338 -17.555 -16.133  1.00  0.00      A    O  \\nATOM     13  CG  PRO A   2       9.713 -18.335 -16.935  1.00  0.00      A    C  \\nATOM     14  CD  PRO A   2       9.016 -18.193 -18.258  1.00  0.00      A    C  \\nATOM     15  N   VAL A   3       5.124 -19.345 -16.235  1.00  0.00      A    N  \\nATOM     16  CA  VAL A   3       4.067 -19.145 -15.250  1.00  0.00      A    C  \\nATOM     17  C   VAL A   3       4.665 -18.606 -13.952  1.00  0.00      A    C  \\nATOM     18  CB  VAL A   3       3.292 -20.453 -14.976  1.00  0.00      A    C  \\nATOM     19  O   VAL A   3       5.626 -19.170 -13.423  1.00  0.00      A    O  \\nATOM     20  CG1 VAL A   3       1.798 -20.174 -14.823  1.00  0.00      A    C  \\nATOM     21  CG2 VAL A   3       3.538 -21.463 -16.096  1.00  0.00      A    C  \\nATOM     22  N   ASN A   4       4.985 -17.260 -13.753  1.00  0.00      A    N  \\nATOM     23  CA  ASN A   4       5.411 -16.807 -12.433  1.00  0.00      A    C  \\nATOM     24  C   ASN A   4       4.919 -17.743 -11.333  1.00  0.00      A    C  \\nATOM     25  CB  ASN A   4       4.924 -15.380 -12.173  1.00  0.00      A    C  \\nATOM     26  O   ASN A   4       3.740 -18.100 -11.297  1.00  0.00      A    O  \\nATOM     27  CG  ASN A   4       6.058 -14.374 -12.133  1.00  0.00      A    C  \\nATOM     28  ND2 ASN A   4       5.719 -13.094 -12.218  1.00  0.00      A    N  \\nATOM     29  OD1 ASN A   4       7.230 -14.746 -12.027  1.00  0.00      A    O  \\nATOM     30  N   PRO A   5       5.604 -18.831 -10.877  1.00  0.00      A    N  \\nATOM     31  CA  PRO A   5       4.976 -19.577  -9.785  1.00  0.00      A    C  \\nATOM     32  C   PRO A   5       4.015 -18.722  -8.962  1.00  0.00      A    C  \\nATOM     33  CB  PRO A   5       6.167 -20.031  -8.937  1.00  0.00      A    C  \\nATOM     34  O   PRO A   5       4.305 -17.556  -8.684  1.00  0.00      A    O  \\nATOM     35  CG  PRO A   5       7.317 -19.209  -9.423  1.00  0.00      A    C  \\nATOM     36  CD  PRO A   5       6.924 -18.557 -10.718  1.00  0.00      A    C  \\nATOM     37  N   GLU A   6       2.648 -18.911  -9.065  1.00  0.00      A    N  \\nATOM     38  CA  GLU A   6       1.752 -18.443  -8.012  1.00  0.00      A    C  \\nATOM     39  C   GLU A   6       2.488 -18.308  -6.682  1.00  0.00      A    C  \\nATOM     40  CB  GLU A   6       0.560 -19.390  -7.860  1.00  0.00      A    C  \\nATOM     41  O   GLU A   6       3.291 -19.170  -6.320  1.00  0.00      A    O  \\nATOM     42  CG  GLU A   6      -0.504 -19.218  -8.935  1.00  0.00      A    C  \\nATOM     43  CD  GLU A   6      -1.851 -19.804  -8.545  1.00  0.00      A    C  \\nATOM     44  OE1 GLU A   6      -1.902 -20.647  -7.621  1.00  0.00      A    O  \\nATOM     45  OE2 GLU A   6      -2.865 -19.418  -9.169  1.00  0.00      A    O  \\nATOM     46  N   PRO A   7       2.901 -17.078  -6.204  1.00  0.00      A    N  \\nATOM     47  CA  PRO A   7       3.474 -17.018  -4.857  1.00  0.00      A    C  \\nATOM     48  C   PRO A   7       3.026 -18.180  -3.973  1.00  0.00      A    C  \\nATOM     49  CB  PRO A   7       2.953 -15.686  -4.312  1.00  0.00      A    C  \\nATOM     50  O   PRO A   7       1.846 -18.540  -3.971  1.00  0.00      A    O  \\nATOM     51  CG  PRO A   7       2.022 -15.180  -5.366  1.00  0.00      A    C  \\nATOM     52  CD  PRO A   7       2.035 -16.144  -6.517  1.00  0.00      A    C  \\nATOM     53  N   ASP A   8       3.786 -19.312  -3.767  1.00  0.00      A    N  \\nATOM     54  CA  ASP A   8       3.546 -20.366  -2.786  1.00  0.00      A    C  \\nATOM     55  C   ASP A   8       2.798 -19.825  -1.570  1.00  0.00      A    C  \\nATOM     56  CB  ASP A   8       4.867 -21.005  -2.350  1.00  0.00      A    C  \\nATOM     57  O   ASP A   8       3.302 -18.951  -0.863  1.00  0.00      A    O  \\nATOM     58  CG  ASP A   8       4.931 -22.493  -2.642  1.00  0.00      A    C  \\nATOM     59  OD1 ASP A   8       3.887 -23.095  -2.975  1.00  0.00      A    O  \\nATOM     60  OD2 ASP A   8       6.034 -23.071  -2.534  1.00  0.00      A    O  \\nATOM     61  N   ALA A   9       1.543 -19.409  -1.662  1.00  0.00      A    N  \\nATOM     62  CA  ALA A   9       0.564 -19.236  -0.592  1.00  0.00      A    C  \\nATOM     63  C   ALA A   9       0.739 -20.299   0.489  1.00  0.00      A    C  \\nATOM     64  CB  ALA A   9      -0.854 -19.285  -1.157  1.00  0.00      A    C  \\nATOM     65  O   ALA A   9      -0.006 -20.323   1.471  1.00  0.00      A    O  \\nATOM     66  N   THR A  10       1.773 -21.205   0.502  1.00  0.00      A    N  \\nATOM     67  CA  THR A  10       1.715 -22.406   1.328  1.00  0.00      A    C  \\nATOM     68  C   THR A  10       2.573 -22.244   2.579  1.00  0.00      A    C  \\nATOM     69  CB  THR A  10       2.179 -23.647   0.542  1.00  0.00      A    C  \\nATOM     70  O   THR A  10       3.792 -22.087   2.487  1.00  0.00      A    O  \\nATOM     71  CG2 THR A  10       1.002 -24.338  -0.140  1.00  0.00      A    C  \\nATOM     72  OG1 THR A  10       3.127 -23.247  -0.455  1.00  0.00      A    O  \\nATOM     73  N   SER A  11       2.463 -21.104   3.386  1.00  0.00      A    N  \\nATOM     74  CA  SER A  11       2.650 -21.248   4.826  1.00  0.00      A    C  \\nATOM     75  C   SER A  11       2.516 -19.905   5.538  1.00  0.00      A    C  \\nATOM     76  CB  SER A  11       4.018 -21.860   5.131  1.00  0.00      A    C  \\nATOM     77  O   SER A  11       3.420 -19.070   5.476  1.00  0.00      A    O  \\nATOM     78  OG  SER A  11       5.059 -20.960   4.792  1.00  0.00      A    O  \\nATOM     79  N   VAL A  12       1.416 -19.239   5.242  1.00  0.00      A    N  \\nATOM     80  CA  VAL A  12       1.291 -18.070   6.105  1.00  0.00      A    C  \\nATOM     81  C   VAL A  12       0.733 -18.488   7.464  1.00  0.00      A    C  \\nATOM     82  CB  VAL A  12       0.389 -16.988   5.469  1.00  0.00      A    C  \\nATOM     83  O   VAL A  12      -0.481 -18.628   7.627  1.00  0.00      A    O  \\nATOM     84  CG1 VAL A  12       0.581 -15.644   6.169  1.00  0.00      A    C  \\nATOM     85  CG2 VAL A  12       0.681 -16.863   3.975  1.00  0.00      A    C  \\nATOM     86  N   GLU A  13       1.308 -19.626   8.009  1.00  0.00      A    N  \\nATOM     87  CA  GLU A  13       1.113 -20.129   9.366  1.00  0.00      A    C  \\nATOM     88  C   GLU A  13       1.807 -19.237  10.391  1.00  0.00      A    C  \\nATOM     89  CB  GLU A  13       1.626 -21.566   9.486  1.00  0.00      A    C  \\nATOM     90  O   GLU A  13       3.027 -19.067  10.349  1.00  0.00      A    O  \\nATOM     91  CG  GLU A  13       0.617 -22.619   9.052  1.00  0.00      A    C  \\nATOM     92  CD  GLU A  13       1.124 -24.042   9.220  1.00  0.00      A    C  \\nATOM     93  OE1 GLU A  13       2.327 -24.227   9.511  1.00  0.00      A    O  \\nATOM     94  OE2 GLU A  13       0.311 -24.980   9.058  1.00  0.00      A    O  \\nATOM     95  N   ASN A  14       1.125 -17.939  10.595  1.00  0.00      A    N  \\nATOM     96  CA  ASN A  14       0.880 -17.228  11.846  1.00  0.00      A    C  \\nATOM     97  C   ASN A  14       0.325 -15.829  11.596  1.00  0.00      A    C  \\nATOM     98  CB  ASN A  14       2.162 -17.150  12.678  1.00  0.00      A    C  \\nATOM     99  O   ASN A  14       1.086 -14.870  11.455  1.00  0.00      A    O  \\nATOM    100  CG  ASN A  14       2.444 -18.431  13.437  1.00  0.00      A    C  \\nATOM    101  ND2 ASN A  14       3.718 -18.688  13.713  1.00  0.00      A    N  \\nATOM    102  OD1 ASN A  14       1.526 -19.184  13.772  1.00  0.00      A    O  \\nATOM    103  N   VAL A  15      -0.813 -15.747  10.890  1.00  0.00      A    N  \\nATOM    104  CA  VAL A  15      -1.725 -14.617  11.033  1.00  0.00      A    C  \\nATOM    105  C   VAL A  15      -2.295 -14.588  12.450  1.00  0.00      A    C  \\nATOM    106  CB  VAL A  15      -2.872 -14.679   9.999  1.00  0.00      A    C  \\nATOM    107  O   VAL A  15      -2.965 -15.532  12.875  1.00  0.00      A    O  \\nATOM    108  CG1 VAL A  15      -3.739 -13.424  10.079  1.00  0.00      A    C  \\nATOM    109  CG2 VAL A  15      -2.310 -14.856   8.590  1.00  0.00      A    C  \\nATOM    110  N   ALA A  16      -1.421 -14.155  13.540  1.00  0.00      A    N  \\nATOM    111  CA  ALA A  16      -1.832 -13.876  14.913  1.00  0.00      A    C  \\nATOM    112  C   ALA A  16      -3.349 -13.737  15.015  1.00  0.00      A    C  \\nATOM    113  CB  ALA A  16      -1.150 -12.610  15.426  1.00  0.00      A    C  \\nATOM    114  O   ALA A  16      -4.014 -13.394  14.034  1.00  0.00      A    O  \\nATOM    115  N   LEU A  17      -3.938 -14.397  16.181  1.00  0.00      A    N  \\nATOM    116  CA  LEU A  17      -5.269 -14.381  16.778  1.00  0.00      A    C  \\nATOM    117  C   LEU A  17      -6.032 -13.124  16.372  1.00  0.00      A    C  \\nATOM    118  CB  LEU A  17      -5.174 -14.464  18.303  1.00  0.00      A    C  \\nATOM    119  O   LEU A  17      -5.527 -12.009  16.528  1.00  0.00      A    O  \\nATOM    120  CG  LEU A  17      -4.900 -15.849  18.891  1.00  0.00      A    C  \\nATOM    121  CD1 LEU A  17      -4.151 -15.725  20.214  1.00  0.00      A    C  \\nATOM    122  CD2 LEU A  17      -6.204 -16.618  19.079  1.00  0.00      A    C  \\nATOM    123  N   LYS A  18      -6.849 -13.306  15.469  1.00  0.00      A    N  \\nATOM    124  CA  LYS A  18      -7.897 -12.358  15.101  1.00  0.00      A    C  \\nATOM    125  C   LYS A  18      -8.666 -11.884  16.331  1.00  0.00      A    C  \\nATOM    126  CB  LYS A  18      -8.859 -12.986  14.091  1.00  0.00      A    C  \\nATOM    127  O   LYS A  18      -9.026 -12.689  17.193  1.00  0.00      A    O  \\nATOM    128  CG  LYS A  18      -8.188 -13.461  12.811  1.00  0.00      A    C  \\nATOM    129  CD  LYS A  18      -9.203 -14.022  11.823  1.00  0.00      A    C  \\nATOM    130  CE  LYS A  18      -8.530 -14.523  10.552  1.00  0.00      A    C  \\nATOM    131  NZ  LYS A  18      -9.515 -15.122   9.603  1.00  0.00      A    N  \\nATOM    132  N   THR A  19      -8.580 -10.539  16.522  1.00  0.00      A    N  \\nATOM    133  CA  THR A  19      -9.567  -9.875  17.366  1.00  0.00      A    C  \\nATOM    134  C   THR A  19     -10.477  -8.978  16.532  1.00  0.00      A    C  \\nATOM    135  CB  THR A  19      -8.887  -9.039  18.467  1.00  0.00      A    C  \\nATOM    136  O   THR A  19     -10.058  -8.445  15.503  1.00  0.00      A    O  \\nATOM    137  CG2 THR A  19      -7.966  -9.902  19.323  1.00  0.00      A    C  \\nATOM    138  OG1 THR A  19      -8.118  -7.994  17.860  1.00  0.00      A    O  \\nATOM    139  N   GLY A  20     -11.714  -9.163  16.753  1.00  0.00      A    N  \\nATOM    140  CA  GLY A  20     -12.765  -8.414  16.083  1.00  0.00      A    C  \\nATOM    141  C   GLY A  20     -13.643  -9.280  15.199  1.00  0.00      A    C  \\nATOM    142  O   GLY A  20     -13.266 -10.399  14.845  1.00  0.00      A    O  \\nATOM    143  N   SER A  21     -14.824  -8.725  14.874  1.00  0.00      A    N  \\nATOM    144  CA  SER A  21     -15.775  -9.449  14.036  1.00  0.00      A    C  \\nATOM    145  C   SER A  21     -15.572  -9.122  12.561  1.00  0.00      A    C  \\nATOM    146  CB  SER A  21     -17.211  -9.121  14.447  1.00  0.00      A    C  \\nATOM    147  O   SER A  21     -15.950  -9.907  11.688  1.00  0.00      A    O  \\nATOM    148  OG  SER A  21     -17.506  -7.757  14.197  1.00  0.00      A    O  \\nATOM    149  N   GLY A  22     -14.927  -7.923  12.309  1.00  0.00      A    N  \\nATOM    150  CA  GLY A  22     -14.801  -7.471  10.932  1.00  0.00      A    C  \\nATOM    151  C   GLY A  22     -16.100  -6.935  10.359  1.00  0.00      A    C  \\nATOM    152  O   GLY A  22     -16.209  -6.721   9.150  1.00  0.00      A    O  \\nATOM    153  N   ASP A  23     -17.174  -6.705  11.156  1.00  0.00      A    N  \\nATOM    154  CA  ASP A  23     -18.513  -6.372  10.680  1.00  0.00      A    C  \\nATOM    155  C   ASP A  23     -18.656  -4.868  10.452  1.00  0.00      A    C  \\nATOM    156  CB  ASP A  23     -19.572  -6.856  11.672  1.00  0.00      A    C  \\nATOM    157  O   ASP A  23     -19.568  -4.425   9.752  1.00  0.00      A    O  \\nATOM    158  CG  ASP A  23     -19.741  -8.366  11.669  1.00  0.00      A    C  \\nATOM    159  OD1 ASP A  23     -19.349  -9.021  10.679  1.00  0.00      A    O  \\nATOM    160  OD2 ASP A  23     -20.272  -8.904  12.664  1.00  0.00      A    O  \\nATOM    161  N   SER A  24     -17.767  -4.143  11.076  1.00  0.00      A    N  \\nATOM    162  CA  SER A  24     -17.849  -2.689  10.983  1.00  0.00      A    C  \\nATOM    163  C   SER A  24     -16.464  -2.053  11.038  1.00  0.00      A    C  \\nATOM    164  CB  SER A  24     -18.721  -2.128  12.106  1.00  0.00      A    C  \\nATOM    165  O   SER A  24     -15.502  -2.685  11.479  1.00  0.00      A    O  \\nATOM    166  OG  SER A  24     -18.082  -2.275  13.362  1.00  0.00      A    O  \\nATOM    167  N   GLN A  25     -16.431  -0.819  10.559  1.00  0.00      A    N  \\nATOM    168  CA  GLN A  25     -15.195  -0.045  10.609  1.00  0.00      A    C  \\nATOM    169  C   GLN A  25     -14.693   0.098  12.043  1.00  0.00      A    C  \\nATOM    170  CB  GLN A  25     -15.399   1.336   9.984  1.00  0.00      A    C  \\nATOM    171  O   GLN A  25     -13.485   0.108  12.286  1.00  0.00      A    O  \\nATOM    172  CG  GLN A  25     -14.127   2.169   9.904  1.00  0.00      A    C  \\nATOM    173  CD  GLN A  25     -14.336   3.494   9.195  1.00  0.00      A    C  \\nATOM    174  NE2 GLN A  25     -13.261   4.258   9.037  1.00  0.00      A    N  \\nATOM    175  OE1 GLN A  25     -15.455   3.827   8.791  1.00  0.00      A    O  \\nATOM    176  N   SER A  26     -15.610   0.109  13.017  1.00  0.00      A    N  \\nATOM    177  CA  SER A  26     -15.251   0.301  14.418  1.00  0.00      A    C  \\nATOM    178  C   SER A  26     -14.862  -1.020  15.075  1.00  0.00      A    C  \\nATOM    179  CB  SER A  26     -16.408   0.940  15.186  1.00  0.00      A    C  \\nATOM    180  O   SER A  26     -14.355  -1.035  16.199  1.00  0.00      A    O  \\nATOM    181  OG  SER A  26     -17.574   0.139  15.098  1.00  0.00      A    O  \\nATOM    182  N   ASP A  27     -15.188  -2.142  14.440  1.00  0.00      A    N  \\nATOM    183  CA  ASP A  27     -14.835  -3.452  14.979  1.00  0.00      A    C  \\nATOM    184  C   ASP A  27     -14.157  -4.317  13.920  1.00  0.00      A    C  \\nATOM    185  CB  ASP A  27     -16.078  -4.161  15.521  1.00  0.00      A    C  \\nATOM    186  O   ASP A  27     -14.639  -5.406  13.598  1.00  0.00      A    O  \\nATOM    187  CG  ASP A  27     -15.751  -5.445  16.264  1.00  0.00      A    C  \\nATOM    188  OD1 ASP A  27     -14.648  -5.551  16.842  1.00  0.00      A    O  \\nATOM    189  OD2 ASP A  27     -16.604  -6.359  16.269  1.00  0.00      A    O  \\nATOM    190  N   PRO A  28     -12.994  -3.883  13.405  1.00  0.00      A    N  \\nATOM    191  CA  PRO A  28     -12.300  -4.632  12.355  1.00  0.00      A    C  \\nATOM    192  C   PRO A  28     -11.565  -5.858  12.891  1.00  0.00      A    C  \\nATOM    193  CB  PRO A  28     -11.313  -3.609  11.786  1.00  0.00      A    C  \\nATOM    194  O   PRO A  28     -11.270  -5.931  14.087  1.00  0.00      A    O  \\nATOM    195  CG  PRO A  28     -10.974  -2.723  12.941  1.00  0.00      A    C  \\nATOM    196  CD  PRO A  28     -12.199  -2.561  13.794  1.00  0.00      A    C  \\nATOM    197  N   ILE A  29     -11.365  -6.795  12.005  1.00  0.00      A    N  \\nATOM    198  CA  ILE A  29     -10.402  -7.845  12.315  1.00  0.00      A    C  \\nATOM    199  C   ILE A  29      -9.010  -7.238  12.479  1.00  0.00      A    C  \\nATOM    200  CB  ILE A  29     -10.385  -8.938  11.222  1.00  0.00      A    C  \\nATOM    201  O   ILE A  29      -8.504  -6.576  11.569  1.00  0.00      A    O  \\nATOM    202  CG1 ILE A  29     -11.700  -9.725  11.234  1.00  0.00      A    C  \\nATOM    203  CG2 ILE A  29      -9.186  -9.871  11.411  1.00  0.00      A    C  \\nATOM    204  CD1 ILE A  29     -11.846 -10.703  10.077  1.00  0.00      A    C  \\nATOM    205  N   LYS A  30      -8.452  -7.409  13.630  1.00  0.00      A    N  \\nATOM    206  CA  LYS A  30      -7.108  -6.910  13.906  1.00  0.00      A    C  \\nATOM    207  C   LYS A  30      -6.087  -8.044  13.898  1.00  0.00      A    C  \\nATOM    208  CB  LYS A  30      -7.071  -6.181  15.250  1.00  0.00      A    C  \\nATOM    209  O   LYS A  30      -6.301  -9.083  14.526  1.00  0.00      A    O  \\nATOM    210  CG  LYS A  30      -7.932  -4.928  15.299  1.00  0.00      A    C  \\nATOM    211  CD  LYS A  30      -7.845  -4.241  16.656  1.00  0.00      A    C  \\nATOM    212  CE  LYS A  30      -8.744  -3.014  16.721  1.00  0.00      A    C  \\nATOM    213  NZ  LYS A  30      -8.718  -2.380  18.073  1.00  0.00      A    N  \\nATOM    214  N   ALA A  31      -5.108  -7.861  13.045  1.00  0.00      A    N  \\nATOM    215  CA  ALA A  31      -4.114  -8.924  12.922  1.00  0.00      A    C  \\nATOM    216  C   ALA A  31      -2.708  -8.347  12.781  1.00  0.00      A    C  \\nATOM    217  CB  ALA A  31      -4.440  -9.821  11.730  1.00  0.00      A    C  \\nATOM    218  O   ALA A  31      -2.537  -7.222  12.306  1.00  0.00      A    O  \\nATOM    219  N   ASP A  32      -1.752  -9.183  13.284  1.00  0.00      A    N  \\nATOM    220  CA  ASP A  32      -0.340  -8.956  12.995  1.00  0.00      A    C  \\nATOM    221  C   ASP A  32       0.076  -9.653  11.702  1.00  0.00      A    C  \\nATOM    222  CB  ASP A  32       0.530  -9.442  14.157  1.00  0.00      A    C  \\nATOM    223  O   ASP A  32      -0.346 -10.780  11.435  1.00  0.00      A    O  \\nATOM    224  CG  ASP A  32       0.288  -8.669  15.442  1.00  0.00      A    C  \\nATOM    225  OD1 ASP A  32      -0.348  -7.594  15.397  1.00  0.00      A    O  \\nATOM    226  OD2 ASP A  32       0.741  -9.138  16.509  1.00  0.00      A    O  \\nATOM    227  N   LEU A  33       0.884  -8.920  10.867  1.00  0.00      A    N  \\nATOM    228  CA  LEU A  33       1.394  -9.515   9.636  1.00  0.00      A    C  \\nATOM    229  C   LEU A  33       2.884  -9.814   9.754  1.00  0.00      A    C  \\nATOM    230  CB  LEU A  33       1.141  -8.586   8.446  1.00  0.00      A    C  \\nATOM    231  O   LEU A  33       3.665  -8.950  10.158  1.00  0.00      A    O  \\nATOM    232  CG  LEU A  33      -0.295  -8.529   7.923  1.00  0.00      A    C  \\nATOM    233  CD1 LEU A  33      -0.398  -7.546   6.762  1.00  0.00      A    C  \\nATOM    234  CD2 LEU A  33      -0.765  -9.917   7.498  1.00  0.00      A    C  \\nATOM    235  N   GLU A  34       3.227 -11.009   9.491  1.00  0.00      A    N  \\nATOM    236  CA  GLU A  34       4.643 -11.365   9.477  1.00  0.00      A    C  \\nATOM    237  C   GLU A  34       5.286 -11.021   8.137  1.00  0.00      A    C  \\nATOM    238  CB  GLU A  34       4.826 -12.854   9.782  1.00  0.00      A    C  \\nATOM    239  O   GLU A  34       4.843 -11.497   7.090  1.00  0.00      A    O  \\nATOM    240  CG  GLU A  34       6.279 -13.274   9.945  1.00  0.00      A    C  \\nATOM    241  CD  GLU A  34       6.446 -14.758  10.235  1.00  0.00      A    C  \\nATOM    242  OE1 GLU A  34       5.425 -15.474  10.339  1.00  0.00      A    O  \\nATOM    243  OE2 GLU A  34       7.607 -15.206  10.361  1.00  0.00      A    O  \\nATOM    244  N   VAL A  35       6.237 -10.098   8.183  1.00  0.00      A    N  \\nATOM    245  CA  VAL A  35       7.013  -9.712   7.009  1.00  0.00      A    C  \\nATOM    246  C   VAL A  35       8.035 -10.799   6.684  1.00  0.00      A    C  \\nATOM    247  CB  VAL A  35       7.724  -8.357   7.218  1.00  0.00      A    C  \\nATOM    248  O   VAL A  35       8.828 -11.191   7.544  1.00  0.00      A    O  \\nATOM    249  CG1 VAL A  35       8.544  -7.981   5.985  1.00  0.00      A    C  \\nATOM    250  CG2 VAL A  35       6.707  -7.265   7.541  1.00  0.00      A    C  \\nATOM    251  N   LYS A  36       7.965 -11.277   5.524  1.00  0.00      A    N  \\nATOM    252  CA  LYS A  36       8.834 -12.347   5.040  1.00  0.00      A    C  \\nATOM    253  C   LYS A  36       9.431 -11.998   3.680  1.00  0.00      A    C  \\nATOM    254  CB  LYS A  36       8.064 -13.665   4.952  1.00  0.00      A    C  \\nATOM    255  O   LYS A  36       8.852 -11.215   2.924  1.00  0.00      A    O  \\nATOM    256  CG  LYS A  36       7.528 -14.160   6.287  1.00  0.00      A    C  \\nATOM    257  CD  LYS A  36       6.806 -15.493   6.140  1.00  0.00      A    C  \\nATOM    258  CE  LYS A  36       6.246 -15.976   7.471  1.00  0.00      A    C  \\nATOM    259  NZ  LYS A  36       5.523 -17.275   7.329  1.00  0.00      A    N  \\nATOM    260  N   GLY A  37      10.556 -12.709   3.510  1.00  0.00      A    N  \\nATOM    261  CA  GLY A  37      11.170 -12.512   2.206  1.00  0.00      A    C  \\nATOM    262  C   GLY A  37      11.474 -11.057   1.904  1.00  0.00      A    C  \\nATOM    263  O   GLY A  37      12.025 -10.344   2.745  1.00  0.00      A    O  \\nATOM    264  N   GLN A  38      11.155 -10.654   0.753  1.00  0.00      A    N  \\nATOM    265  CA  GLN A  38      11.490  -9.311   0.291  1.00  0.00      A    C  \\nATOM    266  C   GLN A  38      10.305  -8.363   0.449  1.00  0.00      A    C  \\nATOM    267  CB  GLN A  38      11.946  -9.343  -1.169  1.00  0.00      A    C  \\nATOM    268  O   GLN A  38      10.309  -7.258  -0.098  1.00  0.00      A    O  \\nATOM    269  CG  GLN A  38      13.248 -10.100  -1.390  1.00  0.00      A    C  \\nATOM    270  CD  GLN A  38      13.750 -10.000  -2.818  1.00  0.00      A    C  \\nATOM    271  NE2 GLN A  38      14.839 -10.702  -3.115  1.00  0.00      A    N  \\nATOM    272  OE1 GLN A  38      13.164  -9.298  -3.648  1.00  0.00      A    O  \\nATOM    273  N   SER A  39       9.252  -8.827   1.173  1.00  0.00      A    N  \\nATOM    274  CA  SER A  39       8.037  -8.029   1.299  1.00  0.00      A    C  \\nATOM    275  C   SER A  39       8.262  -6.816   2.195  1.00  0.00      A    C  \\nATOM    276  CB  SER A  39       6.893  -8.878   1.856  1.00  0.00      A    C  \\nATOM    277  O   SER A  39       7.391  -5.950   2.308  1.00  0.00      A    O  \\nATOM    278  OG  SER A  39       7.224  -9.392   3.135  1.00  0.00      A    O  \\nATOM    279  N   ALA A  40       9.374  -6.736   2.790  1.00  0.00      A    N  \\nATOM    280  CA  ALA A  40       9.659  -5.664   3.740  1.00  0.00      A    C  \\nATOM    281  C   ALA A  40       9.560  -4.296   3.070  1.00  0.00      A    C  \\nATOM    282  CB  ALA A  40      11.043  -5.852   4.356  1.00  0.00      A    C  \\nATOM    283  O   ALA A  40       9.012  -3.354   3.646  1.00  0.00      A    O  \\nATOM    284  N   LEU A  41      10.073  -4.129   1.901  1.00  0.00      A    N  \\nATOM    285  CA  LEU A  41      10.115  -2.835   1.228  1.00  0.00      A    C  \\nATOM    286  C   LEU A  41       8.706  -2.326   0.941  1.00  0.00      A    C  \\nATOM    287  CB  LEU A  41      10.910  -2.935  -0.077  1.00  0.00      A    C  \\nATOM    288  O   LEU A  41       8.359  -1.204   1.318  1.00  0.00      A    O  \\nATOM    289  CG  LEU A  41      10.857  -1.715  -0.998  1.00  0.00      A    C  \\nATOM    290  CD1 LEU A  41      11.340  -0.472  -0.259  1.00  0.00      A    C  \\nATOM    291  CD2 LEU A  41      11.690  -1.956  -2.252  1.00  0.00      A    C  \\nATOM    292  N   PRO A  42       7.806  -3.115   0.237  1.00  0.00      A    N  \\nATOM    293  CA  PRO A  42       6.470  -2.560   0.007  1.00  0.00      A    C  \\nATOM    294  C   PRO A  42       5.690  -2.343   1.301  1.00  0.00      A    C  \\nATOM    295  CB  PRO A  42       5.794  -3.620  -0.867  1.00  0.00      A    C  \\nATOM    296  O   PRO A  42       4.984  -1.341   1.441  1.00  0.00      A    O  \\nATOM    297  CG  PRO A  42       6.614  -4.854  -0.673  1.00  0.00      A    C  \\nATOM    298  CD  PRO A  42       8.030  -4.447  -0.381  1.00  0.00      A    C  \\nATOM    299  N   PHE A  43       5.830  -3.174   2.315  1.00  0.00      A    N  \\nATOM    300  CA  PHE A  43       5.140  -2.936   3.577  1.00  0.00      A    C  \\nATOM    301  C   PHE A  43       5.674  -1.681   4.257  1.00  0.00      A    C  \\nATOM    302  CB  PHE A  43       5.292  -4.143   4.509  1.00  0.00      A    C  \\nATOM    303  O   PHE A  43       4.927  -0.969   4.932  1.00  0.00      A    O  \\nATOM    304  CG  PHE A  43       4.346  -5.271   4.200  1.00  0.00      A    C  \\nATOM    305  CD1 PHE A  43       2.977  -5.047   4.118  1.00  0.00      A    C  \\nATOM    306  CD2 PHE A  43       4.826  -6.558   3.991  1.00  0.00      A    C  \\nATOM    307  CE1 PHE A  43       2.099  -6.090   3.831  1.00  0.00      A    C  \\nATOM    308  CE2 PHE A  43       3.955  -7.605   3.705  1.00  0.00      A    C  \\nATOM    309  CZ  PHE A  43       2.592  -7.369   3.626  1.00  0.00      A    C  \\nATOM    310  N   ASP A  44       6.977  -1.440   4.090  1.00  0.00      A    N  \\nATOM    311  CA  ASP A  44       7.543  -0.252   4.722  1.00  0.00      A    C  \\nATOM    312  C   ASP A  44       7.003   1.023   4.077  1.00  0.00      A    C  \\nATOM    313  CB  ASP A  44       9.071  -0.276   4.640  1.00  0.00      A    C  \\nATOM    314  O   ASP A  44       6.843   2.044   4.748  1.00  0.00      A    O  \\nATOM    315  CG  ASP A  44       9.736   0.533   5.740  1.00  0.00      A    C  \\nATOM    316  OD1 ASP A  44       9.326   0.418   6.915  1.00  0.00      A    O  \\nATOM    317  OD2 ASP A  44      10.677   1.294   5.428  1.00  0.00      A    O  \\nATOM    318  N   VAL A  45       6.687   0.999   2.780  1.00  0.00      A    N  \\nATOM    319  CA  VAL A  45       5.982   2.098   2.128  1.00  0.00      A    C  \\nATOM    320  C   VAL A  45       4.615   2.295   2.780  1.00  0.00      A    C  \\nATOM    321  CB  VAL A  45       5.819   1.848   0.612  1.00  0.00      A    C  \\nATOM    322  O   VAL A  45       4.219   3.424   3.077  1.00  0.00      A    O  \\nATOM    323  CG1 VAL A  45       4.932   2.919  -0.020  1.00  0.00      A    C  \\nATOM    324  CG2 VAL A  45       7.184   1.809  -0.072  1.00  0.00      A    C  \\nATOM    325  N   ASP A  46       3.912   1.156   3.075  1.00  0.00      A    N  \\nATOM    326  CA  ASP A  46       2.592   1.207   3.698  1.00  0.00      A    C  \\nATOM    327  C   ASP A  46       2.666   1.827   5.091  1.00  0.00      A    C  \\nATOM    328  CB  ASP A  46       1.982  -0.194   3.777  1.00  0.00      A    C  \\nATOM    329  O   ASP A  46       1.862   2.697   5.432  1.00  0.00      A    O  \\nATOM    330  CG  ASP A  46       1.872  -0.871   2.422  1.00  0.00      A    C  \\nATOM    331  OD1 ASP A  46       1.847  -0.168   1.389  1.00  0.00      A    O  \\nATOM    332  OD2 ASP A  46       1.807  -2.119   2.388  1.00  0.00      A    O  \\nATOM    333  N   CYS A  47       3.654   1.382   5.852  1.00  0.00      A    N  \\nATOM    334  CA  CYS A  47       3.841   1.882   7.210  1.00  0.00      A    C  \\nATOM    335  C   CYS A  47       4.181   3.368   7.202  1.00  0.00      A    C  \\nATOM    336  CB  CYS A  47       4.946   1.101   7.921  1.00  0.00      A    C  \\nATOM    337  O   CYS A  47       3.702   4.123   8.049  1.00  0.00      A    O  \\nATOM    338  SG  CYS A  47       4.519  -0.622   8.257  1.00  0.00      A    S  \\nATOM    339  N   TRP A  48       5.020   3.786   6.265  1.00  0.00      A    N  \\nATOM    340  CA  TRP A  48       5.326   5.206   6.127  1.00  0.00      A    C  \\nATOM    341  C   TRP A  48       4.067   6.006   5.812  1.00  0.00      A    C  \\nATOM    342  CB  TRP A  48       6.374   5.427   5.032  1.00  0.00      A    C  \\nATOM    343  O   TRP A  48       3.828   7.059   6.409  1.00  0.00      A    O  \\nATOM    344  CG  TRP A  48       6.658   6.870   4.743  1.00  0.00      A    C  \\nATOM    345  CD1 TRP A  48       7.431   7.722   5.482  1.00  0.00      A    C  \\nATOM    346  CD2 TRP A  48       6.166   7.633   3.637  1.00  0.00      A    C  \\nATOM    347  CE2 TRP A  48       6.682   8.941   3.768  1.00  0.00      A    C  \\nATOM    348  CE3 TRP A  48       5.338   7.336   2.546  1.00  0.00      A    C  \\nATOM    349  NE1 TRP A  48       7.450   8.968   4.901  1.00  0.00      A    N  \\nATOM    350  CH2 TRP A  48       5.585   9.635   1.789  1.00  0.00      A    C  \\nATOM    351  CZ2 TRP A  48       6.397   9.952   2.847  1.00  0.00      A    C  \\nATOM    352  CZ3 TRP A  48       5.055   8.344   1.631  1.00  0.00      A    C  \\nATOM    353  N   ALA A  49       3.246   5.525   4.864  1.00  0.00      A    N  \\nATOM    354  CA  ALA A  49       2.019   6.218   4.482  1.00  0.00      A    C  \\nATOM    355  C   ALA A  49       1.073   6.355   5.671  1.00  0.00      A    C  \\nATOM    356  CB  ALA A  49       1.327   5.481   3.337  1.00  0.00      A    C  \\nATOM    357  O   ALA A  49       0.488   7.419   5.887  1.00  0.00      A    O  \\nATOM    358  N   ILE A  50       0.948   5.353   6.482  1.00  0.00      A    N  \\nATOM    359  CA  ILE A  50       0.072   5.360   7.649  1.00  0.00      A    C  \\nATOM    360  C   ILE A  50       0.638   6.297   8.713  1.00  0.00      A    C  \\nATOM    361  CB  ILE A  50      -0.110   3.939   8.228  1.00  0.00      A    C  \\nATOM    362  O   ILE A  50      -0.057   7.198   9.190  1.00  0.00      A    O  \\nATOM    363  CG1 ILE A  50      -0.873   3.052   7.237  1.00  0.00      A    C  \\nATOM    364  CG2 ILE A  50      -0.830   3.995   9.579  1.00  0.00      A    C  \\nATOM    365  CD1 ILE A  50      -0.820   1.567   7.568  1.00  0.00      A    C  \\nATOM    366  N   LEU A  51       1.914   6.084   9.041  1.00  0.00      A    N  \\nATOM    367  CA  LEU A  51       2.529   6.770  10.172  1.00  0.00      A    C  \\nATOM    368  C   LEU A  51       2.778   8.239   9.848  1.00  0.00      A    C  \\nATOM    369  CB  LEU A  51       3.846   6.091  10.560  1.00  0.00      A    C  \\nATOM    370  O   LEU A  51       2.533   9.114  10.682  1.00  0.00      A    O  \\nATOM    371  CG  LEU A  51       4.585   6.681  11.762  1.00  0.00      A    C  \\nATOM    372  CD1 LEU A  51       3.709   6.611  13.008  1.00  0.00      A    C  \\nATOM    373  CD2 LEU A  51       5.905   5.952  11.989  1.00  0.00      A    C  \\nATOM    374  N   CYS A  52       3.181   8.557   8.598  1.00  0.00      A    N  \\nATOM    375  CA  CYS A  52       3.726   9.879   8.312  1.00  0.00      A    C  \\nATOM    376  C   CYS A  52       2.756  10.698   7.469  1.00  0.00      A    C  \\nATOM    377  CB  CYS A  52       5.069   9.762   7.592  1.00  0.00      A    C  \\nATOM    378  O   CYS A  52       2.836  11.927   7.441  1.00  0.00      A    O  \\nATOM    379  SG  CYS A  52       6.326   8.871   8.535  1.00  0.00      A    S  \\nATOM    380  N   LYS A  53       1.786  10.014   6.837  1.00  0.00      A    N  \\nATOM    381  CA  LYS A  53       0.891  10.772   5.968  1.00  0.00      A    C  \\nATOM    382  C   LYS A  53      -0.560  10.638   6.423  1.00  0.00      A    C  \\nATOM    383  CB  LYS A  53       1.030  10.309   4.517  1.00  0.00      A    C  \\nATOM    384  O   LYS A  53      -1.458  11.249   5.840  1.00  0.00      A    O  \\nATOM    385  CG  LYS A  53       2.412  10.541   3.924  1.00  0.00      A    C  \\nATOM    386  CD  LYS A  53       2.760  12.023   3.879  1.00  0.00      A    C  \\nATOM    387  CE  LYS A  53       1.982  12.748   2.789  1.00  0.00      A    C  \\nATOM    388  NZ  LYS A  53       2.512  14.123   2.550  1.00  0.00      A    N  \\nATOM    389  N   GLY A  54      -0.765   9.824   7.483  1.00  0.00      A    N  \\nATOM    390  CA  GLY A  54      -2.109   9.679   8.019  1.00  0.00      A    C  \\nATOM    391  C   GLY A  54      -3.029   8.879   7.116  1.00  0.00      A    C  \\nATOM    392  O   GLY A  54      -4.247   9.065   7.141  1.00  0.00      A    O  \\nATOM    393  N   ALA A  55      -2.422   7.992   6.282  1.00  0.00      A    N  \\nATOM    394  CA  ALA A  55      -3.235   7.116   5.443  1.00  0.00      A    C  \\nATOM    395  C   ALA A  55      -4.107   6.196   6.293  1.00  0.00      A    C  \\nATOM    396  CB  ALA A  55      -2.346   6.292   4.515  1.00  0.00      A    C  \\nATOM    397  O   ALA A  55      -3.777   5.909   7.446  1.00  0.00      A    O  \\nATOM    398  N   PRO A  56      -5.256   5.726   5.699  1.00  0.00      A    N  \\nATOM    399  CA  PRO A  56      -6.143   4.850   6.468  1.00  0.00      A    C  \\nATOM    400  C   PRO A  56      -5.471   3.541   6.875  1.00  0.00      A    C  \\nATOM    401  CB  PRO A  56      -7.305   4.592   5.506  1.00  0.00      A    C  \\nATOM    402  O   PRO A  56      -4.717   2.960   6.090  1.00  0.00      A    O  \\nATOM    403  CG  PRO A  56      -7.244   5.716   4.523  1.00  0.00      A    C  \\nATOM    404  CD  PRO A  56      -5.818   6.171   4.406  1.00  0.00      A    C  \\nATOM    405  N   ASN A  57      -5.773   3.117   8.135  1.00  0.00      A    N  \\nATOM    406  CA  ASN A  57      -5.206   1.854   8.595  1.00  0.00      A    C  \\nATOM    407  C   ASN A  57      -6.296   0.839   8.929  1.00  0.00      A    C  \\nATOM    408  CB  ASN A  57      -4.302   2.081   9.808  1.00  0.00      A    C  \\nATOM    409  O   ASN A  57      -6.004  -0.249   9.430  1.00  0.00      A    O  \\nATOM    410  CG  ASN A  57      -5.059   2.612  11.009  1.00  0.00      A    C  \\nATOM    411  ND2 ASN A  57      -4.378   2.705  12.145  1.00  0.00      A    N  \\nATOM    412  OD1 ASN A  57      -6.246   2.935  10.917  1.00  0.00      A    O  \\nATOM    413  N   VAL A  58      -7.538   1.288   8.804  1.00  0.00      A    N  \\nATOM    414  CA  VAL A  58      -8.687   0.388   8.812  1.00  0.00      A    C  \\nATOM    415  C   VAL A  58      -9.273   0.286   7.405  1.00  0.00      A    C  \\nATOM    416  CB  VAL A  58      -9.770   0.858   9.808  1.00  0.00      A    C  \\nATOM    417  O   VAL A  58      -9.647   1.297   6.807  1.00  0.00      A    O  \\nATOM    418  CG1 VAL A  58     -10.952  -0.109   9.817  1.00  0.00      A    C  \\nATOM    419  CG2 VAL A  58      -9.180   0.999  11.210  1.00  0.00      A    C  \\nATOM    420  N   LEU A  59      -9.281  -0.928   6.901  1.00  0.00      A    N  \\nATOM    421  CA  LEU A  59      -9.577  -1.138   5.488  1.00  0.00      A    C  \\nATOM    422  C   LEU A  59     -10.710  -2.144   5.313  1.00  0.00      A    C  \\nATOM    423  CB  LEU A  59      -8.329  -1.624   4.745  1.00  0.00      A    C  \\nATOM    424  O   LEU A  59     -11.020  -2.902   6.235  1.00  0.00      A    O  \\nATOM    425  CG  LEU A  59      -7.077  -0.757   4.886  1.00  0.00      A    C  \\nATOM    426  CD1 LEU A  59      -5.879  -1.448   4.242  1.00  0.00      A    C  \\nATOM    427  CD2 LEU A  59      -7.305   0.617   4.266  1.00  0.00      A    C  \\nATOM    428  N   GLN A  60     -11.340  -2.083   4.183  1.00  0.00      A    N  \\nATOM    429  CA  GLN A  60     -12.347  -3.050   3.760  1.00  0.00      A    C  \\nATOM    430  C   GLN A  60     -12.002  -3.644   2.398  1.00  0.00      A    C  \\nATOM    431  CB  GLN A  60     -13.731  -2.399   3.715  1.00  0.00      A    C  \\nATOM    432  O   GLN A  60     -11.774  -2.909   1.434  1.00  0.00      A    O  \\nATOM    433  CG  GLN A  60     -14.871  -3.392   3.540  1.00  0.00      A    C  \\nATOM    434  CD  GLN A  60     -16.237  -2.734   3.600  1.00  0.00      A    C  \\nATOM    435  NE2 GLN A  60     -17.211  -3.439   4.166  1.00  0.00      A    N  \\nATOM    436  OE1 GLN A  60     -16.415  -1.601   3.143  1.00  0.00      A    O  \\nATOM    437  N   ARG A  61     -11.954  -4.994   2.364  1.00  0.00      A    N  \\nATOM    438  CA  ARG A  61     -11.592  -5.678   1.127  1.00  0.00      A    C  \\nATOM    439  C   ARG A  61     -12.687  -5.525   0.076  1.00  0.00      A    C  \\nATOM    440  CB  ARG A  61     -11.322  -7.161   1.391  1.00  0.00      A    C  \\nATOM    441  O   ARG A  61     -13.875  -5.595   0.396  1.00  0.00      A    O  \\nATOM    442  CG  ARG A  61     -10.771  -7.910   0.189  1.00  0.00      A    C  \\nATOM    443  CD  ARG A  61     -10.439  -9.356   0.529  1.00  0.00      A    C  \\nATOM    444  NE  ARG A  61      -9.976 -10.093  -0.643  1.00  0.00      A    N  \\nATOM    445  NH1 ARG A  61     -10.054 -12.167   0.373  1.00  0.00      A    N  \\nATOM    446  NH2 ARG A  61      -9.378 -11.979  -1.810  1.00  0.00      A    N  \\nATOM    447  CZ  ARG A  61      -9.804 -11.411  -0.691  1.00  0.00      A    C  \\nATOM    448  N   VAL A  62     -12.235  -5.340  -1.252  1.00  0.00      A    N  \\nATOM    449  CA  VAL A  62     -13.207  -5.149  -2.323  1.00  0.00      A    C  \\nATOM    450  C   VAL A  62     -13.038  -6.242  -3.376  1.00  0.00      A    C  \\nATOM    451  CB  VAL A  62     -13.067  -3.755  -2.975  1.00  0.00      A    C  \\nATOM    452  O   VAL A  62     -12.035  -6.960  -3.379  1.00  0.00      A    O  \\nATOM    453  CG1 VAL A  62     -13.467  -2.657  -1.992  1.00  0.00      A    C  \\nATOM    454  CG2 VAL A  62     -11.637  -3.539  -3.469  1.00  0.00      A    C  \\nATOM    455  N   ASN A  63     -14.089  -6.351  -4.175  1.00  0.00      A    N  \\nATOM    456  CA  ASN A  63     -14.095  -7.327  -5.259  1.00  0.00      A    C  \\nATOM    457  C   ASN A  63     -13.105  -6.952  -6.358  1.00  0.00      A    C  \\nATOM    458  CB  ASN A  63     -15.503  -7.475  -5.841  1.00  0.00      A    C  \\nATOM    459  O   ASN A  63     -12.549  -5.852  -6.349  1.00  0.00      A    O  \\nATOM    460  CG  ASN A  63     -16.010  -6.193  -6.473  1.00  0.00      A    C  \\nATOM    461  ND2 ASN A  63     -17.292  -6.169  -6.818  1.00  0.00      A    N  \\nATOM    462  OD1 ASN A  63     -15.256  -5.233  -6.651  1.00  0.00      A    O  \\nATOM    463  N   GLU A  64     -12.816  -7.854  -7.216  1.00  0.00      A    N  \\nATOM    464  CA  GLU A  64     -11.789  -7.731  -8.246  1.00  0.00      A    C  \\nATOM    465  C   GLU A  64     -12.083  -6.563  -9.183  1.00  0.00      A    C  \\nATOM    466  CB  GLU A  64     -11.671  -9.031  -9.046  1.00  0.00      A    C  \\nATOM    467  O   GLU A  64     -11.168  -5.854  -9.606  1.00  0.00      A    O  \\nATOM    468  CG  GLU A  64     -10.499  -9.051 -10.017  1.00  0.00      A    C  \\nATOM    469  CD  GLU A  64      -9.147  -8.973  -9.326  1.00  0.00      A    C  \\nATOM    470  OE1 GLU A  64      -9.074  -9.233  -8.103  1.00  0.00      A    O  \\nATOM    471  OE2 GLU A  64      -8.153  -8.648 -10.013  1.00  0.00      A    O  \\nATOM    472  N   LYS A  65     -13.300  -6.406  -9.527  1.00  0.00      A    N  \\nATOM    473  CA  LYS A  65     -13.680  -5.297 -10.397  1.00  0.00      A    C  \\nATOM    474  C   LYS A  65     -13.283  -3.956  -9.784  1.00  0.00      A    C  \\nATOM    475  CB  LYS A  65     -15.184  -5.322 -10.674  1.00  0.00      A    C  \\nATOM    476  O   LYS A  65     -12.699  -3.107 -10.460  1.00  0.00      A    O  \\nATOM    477  CG  LYS A  65     -15.661  -4.214 -11.601  1.00  0.00      A    C  \\nATOM    478  CD  LYS A  65     -17.171  -4.258 -11.794  1.00  0.00      A    C  \\nATOM    479  CE  LYS A  65     -17.663  -3.088 -12.636  1.00  0.00      A    C  \\nATOM    480  NZ  LYS A  65     -19.151  -3.088 -12.769  1.00  0.00      A    N  \\nATOM    481  N   THR A  66     -13.613  -3.718  -8.521  1.00  0.00      A    N  \\nATOM    482  CA  THR A  66     -13.273  -2.485  -7.820  1.00  0.00      A    C  \\nATOM    483  C   THR A  66     -11.764  -2.368  -7.631  1.00  0.00      A    C  \\nATOM    484  CB  THR A  66     -13.972  -2.411  -6.449  1.00  0.00      A    C  \\nATOM    485  O   THR A  66     -11.200  -1.279  -7.756  1.00  0.00      A    O  \\nATOM    486  CG2 THR A  66     -13.631  -1.112  -5.728  1.00  0.00      A    C  \\nATOM    487  OG1 THR A  66     -15.391  -2.481  -6.639  1.00  0.00      A    O  \\nATOM    488  N   LYS A  67     -11.143  -3.482  -7.328  1.00  0.00      A    N  \\nATOM    489  CA  LYS A  67      -9.688  -3.508  -7.205  1.00  0.00      A    C  \\nATOM    490  C   LYS A  67      -9.019  -2.983  -8.472  1.00  0.00      A    C  \\nATOM    491  CB  LYS A  67      -9.200  -4.926  -6.904  1.00  0.00      A    C  \\nATOM    492  O   LYS A  67      -8.089  -2.177  -8.401  1.00  0.00      A    O  \\nATOM    493  CG  LYS A  67      -7.686  -5.053  -6.811  1.00  0.00      A    C  \\nATOM    494  CD  LYS A  67      -7.258  -6.499  -6.600  1.00  0.00      A    C  \\nATOM    495  CE  LYS A  67      -5.754  -6.668  -6.763  1.00  0.00      A    C  \\nATOM    496  NZ  LYS A  67      -5.343  -8.101  -6.671  1.00  0.00      A    N  \\nATOM    497  N   ASN A  68      -9.474  -3.369  -9.647  1.00  0.00      A    N  \\nATOM    498  CA  ASN A  68      -8.915  -2.915 -10.916  1.00  0.00      A    C  \\nATOM    499  C   ASN A  68      -9.186  -1.432 -11.149  1.00  0.00      A    C  \\nATOM    500  CB  ASN A  68      -9.470  -3.745 -12.076  1.00  0.00      A    C  \\nATOM    501  O   ASN A  68      -8.317  -0.707 -11.638  1.00  0.00      A    O  \\nATOM    502  CG  ASN A  68      -8.944  -5.167 -12.080  1.00  0.00      A    C  \\nATOM    503  ND2 ASN A  68      -9.693  -6.074 -12.697  1.00  0.00      A    N  \\nATOM    504  OD1 ASN A  68      -7.875  -5.448 -11.532  1.00  0.00      A    O  \\nATOM    505  N   SER A  69     -10.384  -1.052 -10.773  1.00  0.00      A    N  \\nATOM    506  CA  SER A  69     -10.713   0.368 -10.852  1.00  0.00      A    C  \\nATOM    507  C   SER A  69      -9.819   1.194  -9.933  1.00  0.00      A    C  \\nATOM    508  CB  SER A  69     -12.181   0.599 -10.491  1.00  0.00      A    C  \\nATOM    509  O   SER A  69      -9.351   2.268 -10.315  1.00  0.00      A    O  \\nATOM    510  OG  SER A  69     -12.512   1.974 -10.585  1.00  0.00      A    O  \\nATOM    511  N   ASN A  70      -9.558   0.678  -8.737  1.00  0.00      A    N  \\nATOM    512  CA  ASN A  70      -8.673   1.360  -7.799  1.00  0.00      A    C  \\nATOM    513  C   ASN A  70      -7.262   1.505  -8.364  1.00  0.00      A    C  \\nATOM    514  CB  ASN A  70      -8.635   0.620  -6.461  1.00  0.00      A    C  \\nATOM    515  O   ASN A  70      -6.625   2.546  -8.196  1.00  0.00      A    O  \\nATOM    516  CG  ASN A  70      -9.892   0.833  -5.641  1.00  0.00      A    C  \\nATOM    517  ND2 ASN A  70     -10.070   0.020  -4.605  1.00  0.00      A    N  \\nATOM    518  OD1 ASN A  70     -10.698   1.719  -5.935  1.00  0.00      A    O  \\nATOM    519  N   ARG A  71      -6.772   0.550  -9.081  1.00  0.00      A    N  \\nATOM    520  CA  ARG A  71      -5.455   0.626  -9.705  1.00  0.00      A    C  \\nATOM    521  C   ARG A  71      -5.407   1.739 -10.747  1.00  0.00      A    C  \\nATOM    522  CB  ARG A  71      -5.087  -0.712 -10.349  1.00  0.00      A    C  \\nATOM    523  O   ARG A  71      -4.415   2.464 -10.843  1.00  0.00      A    O  \\nATOM    524  CG  ARG A  71      -4.660  -1.778  -9.354  1.00  0.00      A    C  \\nATOM    525  CD  ARG A  71      -4.226  -3.060 -10.051  1.00  0.00      A    C  \\nATOM    526  NE  ARG A  71      -3.564  -3.978  -9.129  1.00  0.00      A    N  \\nATOM    527  NH1 ARG A  71      -4.975  -5.755  -9.558  1.00  0.00      A    N  \\nATOM    528  NH2 ARG A  71      -3.260  -5.985  -8.054  1.00  0.00      A    N  \\nATOM    529  CZ  ARG A  71      -3.935  -5.237  -8.916  1.00  0.00      A    C  \\nATOM    530  N   ASP A  72      -6.462   1.844 -11.491  1.00  0.00      A    N  \\nATOM    531  CA  ASP A  72      -6.534   2.911 -12.485  1.00  0.00      A    C  \\nATOM    532  C   ASP A  72      -6.559   4.284 -11.815  1.00  0.00      A    C  \\nATOM    533  CB  ASP A  72      -7.768   2.737 -13.372  1.00  0.00      A    C  \\nATOM    534  O   ASP A  72      -5.819   5.187 -12.211  1.00  0.00      A    O  \\nATOM    535  CG  ASP A  72      -7.646   1.570 -14.336  1.00  0.00      A    C  \\nATOM    536  OD1 ASP A  72      -6.514   1.099 -14.583  1.00  0.00      A    O  \\nATOM    537  OD2 ASP A  72      -8.689   1.118 -14.854  1.00  0.00      A    O  \\nATOM    538  N   ARG A  73      -7.292   4.417 -10.759  1.00  0.00      A    N  \\nATOM    539  CA  ARG A  73      -7.513   5.695 -10.089  1.00  0.00      A    C  \\nATOM    540  C   ARG A  73      -6.289   6.105  -9.276  1.00  0.00      A    C  \\nATOM    541  CB  ARG A  73      -8.744   5.621  -9.182  1.00  0.00      A    C  \\nATOM    542  O   ARG A  73      -6.045   7.296  -9.071  1.00  0.00      A    O  \\nATOM    543  CG  ARG A  73     -10.063   5.576  -9.937  1.00  0.00      A    C  \\nATOM    544  CD  ARG A  73     -11.254   5.701  -8.998  1.00  0.00      A    C  \\nATOM    545  NE  ARG A  73     -11.338   4.570  -8.079  1.00  0.00      A    N  \\nATOM    546  NH1 ARG A  73     -12.415   5.743  -6.405  1.00  0.00      A    N  \\nATOM    547  NH2 ARG A  73     -11.909   3.526  -6.115  1.00  0.00      A    N  \\nATOM    548  CZ  ARG A  73     -11.887   4.615  -6.868  1.00  0.00      A    C  \\nATOM    549  N   SER A  74      -5.547   5.112  -8.783  1.00  0.00      A    N  \\nATOM    550  CA  SER A  74      -4.327   5.413  -8.043  1.00  0.00      A    C  \\nATOM    551  C   SER A  74      -3.199   5.828  -8.982  1.00  0.00      A    C  \\nATOM    552  CB  SER A  74      -3.892   4.206  -7.212  1.00  0.00      A    C  \\nATOM    553  O   SER A  74      -2.187   6.376  -8.541  1.00  0.00      A    O  \\nATOM    554  OG  SER A  74      -3.459   3.148  -8.049  1.00  0.00      A    O  \\nATOM    555  N   GLY A  75      -3.346   5.579 -10.327  1.00  0.00      A    N  \\nATOM    556  CA  GLY A  75      -2.319   5.882 -11.311  1.00  0.00      A    C  \\nATOM    557  C   GLY A  75      -1.271   4.792 -11.435  1.00  0.00      A    C  \\nATOM    558  O   GLY A  75      -0.332   4.914 -12.224  1.00  0.00      A    O  \\nATOM    559  N   ALA A  76      -1.412   3.690 -10.716  1.00  0.00      A    N  \\nATOM    560  CA  ALA A  76      -0.428   2.610 -10.740  1.00  0.00      A    C  \\nATOM    561  C   ALA A  76      -0.320   1.998 -12.133  1.00  0.00      A    C  \\nATOM    562  CB  ALA A  76      -0.793   1.536  -9.717  1.00  0.00      A    C  \\nATOM    563  O   ALA A  76       0.773   1.635 -12.575  1.00  0.00      A    O  \\nATOM    564  N   ASN A  77      -1.438   1.950 -12.875  1.00  0.00      A    N  \\nATOM    565  CA  ASN A  77      -1.491   1.323 -14.191  1.00  0.00      A    C  \\nATOM    566  C   ASN A  77      -0.873   2.216 -15.263  1.00  0.00      A    C  \\nATOM    567  CB  ASN A  77      -2.934   0.971 -14.560  1.00  0.00      A    C  \\nATOM    568  O   ASN A  77      -0.655   1.776 -16.393  1.00  0.00      A    O  \\nATOM    569  CG  ASN A  77      -3.431  -0.274 -13.851  1.00  0.00      A    C  \\nATOM    570  ND2 ASN A  77      -4.742  -0.480 -13.866  1.00  0.00      A    N  \\nATOM    571  OD1 ASN A  77      -2.643  -1.042 -13.294  1.00  0.00      A    O  \\nATOM    572  N   LYS A  78      -0.615   3.462 -14.879  1.00  0.00      A    N  \\nATOM    573  CA  LYS A  78       0.004   4.384 -15.828  1.00  0.00      A    C  \\nATOM    574  C   LYS A  78       1.500   4.114 -15.960  1.00  0.00      A    C  \\nATOM    575  CB  LYS A  78      -0.231   5.833 -15.400  1.00  0.00      A    C  \\nATOM    576  O   LYS A  78       2.144   4.602 -16.891  1.00  0.00      A    O  \\nATOM    577  CG  LYS A  78      -1.698   6.237 -15.361  1.00  0.00      A    C  \\nATOM    578  CD  LYS A  78      -1.860   7.735 -15.135  1.00  0.00      A    C  \\nATOM    579  CE  LYS A  78      -3.325   8.148 -15.151  1.00  0.00      A    C  \\nATOM    580  NZ  LYS A  78      -3.874   8.191 -16.539  1.00  0.00      A    N  \\nATOM    581  N   GLY A  79       2.052   3.348 -15.086  1.00  0.00      A    N  \\nATOM    582  CA  GLY A  79       3.464   3.008 -15.156  1.00  0.00      A    C  \\nATOM    583  C   GLY A  79       4.376   4.179 -14.843  1.00  0.00      A    C  \\nATOM    584  O   GLY A  79       5.307   4.466 -15.598  1.00  0.00      A    O  \\nATOM    585  N   PRO A  80       4.146   4.890 -13.710  1.00  0.00      A    N  \\nATOM    586  CA  PRO A  80       4.890   6.126 -13.457  1.00  0.00      A    C  \\nATOM    587  C   PRO A  80       6.389   5.889 -13.286  1.00  0.00      A    C  \\nATOM    588  CB  PRO A  80       4.270   6.652 -12.160  1.00  0.00      A    C  \\nATOM    589  O   PRO A  80       7.200   6.737 -13.668  1.00  0.00      A    O  \\nATOM    590  CG  PRO A  80       3.656   5.451 -11.516  1.00  0.00      A    C  \\nATOM    591  CD  PRO A  80       3.157   4.534 -12.595  1.00  0.00      A    C  \\nATOM    592  N   PHE A  81       6.779   4.700 -12.870  1.00  0.00      A    N  \\nATOM    593  CA  PHE A  81       8.182   4.495 -12.528  1.00  0.00      A    C  \\nATOM    594  C   PHE A  81       8.910   3.760 -13.647  1.00  0.00      A    C  \\nATOM    595  CB  PHE A  81       8.308   3.712 -11.217  1.00  0.00      A    C  \\nATOM    596  O   PHE A  81      10.131   3.599 -13.600  1.00  0.00      A    O  \\nATOM    597  CG  PHE A  81       7.663   4.393 -10.040  1.00  0.00      A    C  \\nATOM    598  CD1 PHE A  81       8.013   5.692  -9.693  1.00  0.00      A    C  \\nATOM    599  CD2 PHE A  81       6.708   3.732  -9.279  1.00  0.00      A    C  \\nATOM    600  CE1 PHE A  81       7.418   6.325  -8.604  1.00  0.00      A    C  \\nATOM    601  CE2 PHE A  81       6.109   4.358  -8.189  1.00  0.00      A    C  \\nATOM    602  CZ  PHE A  81       6.466   5.653  -7.853  1.00  0.00      A    C  \\nATOM    603  N   LYS A  82       8.082   3.300 -14.664  1.00  0.00      A    N  \\nATOM    604  CA  LYS A  82       8.680   2.724 -15.865  1.00  0.00      A    C  \\nATOM    605  C   LYS A  82       9.199   3.814 -16.799  1.00  0.00      A    C  \\nATOM    606  CB  LYS A  82       7.668   1.843 -16.599  1.00  0.00      A    C  \\nATOM    607  O   LYS A  82      10.185   3.611 -17.511  1.00  0.00      A    O  \\nATOM    608  CG  LYS A  82       7.288   0.577 -15.845  1.00  0.00      A    C  \\nATOM    609  CD  LYS A  82       6.364  -0.309 -16.671  1.00  0.00      A    C  \\nATOM    610  CE  LYS A  82       5.976  -1.572 -15.914  1.00  0.00      A    C  \\nATOM    611  NZ  LYS A  82       5.019  -2.413 -16.694  1.00  0.00      A    N  \\nATOM    612  N   ASP A  83       8.503   4.873 -16.706  1.00  0.00      A    N  \\nATOM    613  CA  ASP A  83       8.897   6.042 -17.486  1.00  0.00      A    C  \\nATOM    614  C   ASP A  83       8.860   7.309 -16.635  1.00  0.00      A    C  \\nATOM    615  CB  ASP A  83       7.989   6.203 -18.708  1.00  0.00      A    C  \\nATOM    616  O   ASP A  83       7.994   8.166 -16.824  1.00  0.00      A    O  \\nATOM    617  CG  ASP A  83       8.554   7.159 -19.744  1.00  0.00      A    C  \\nATOM    618  OD1 ASP A  83       9.550   7.855 -19.453  1.00  0.00      A    O  \\nATOM    619  OD2 ASP A  83       7.996   7.218 -20.861  1.00  0.00      A    O  \\nATOM    620  N   PRO A  84       9.858   7.444 -15.754  1.00  0.00      A    N  \\nATOM    621  CA  PRO A  84       9.841   8.565 -14.812  1.00  0.00      A    C  \\nATOM    622  C   PRO A  84       9.872   9.923 -15.509  1.00  0.00      A    C  \\nATOM    623  CB  PRO A  84      11.109   8.347 -13.982  1.00  0.00      A    C  \\nATOM    624  O   PRO A  84       9.205  10.862 -15.069  1.00  0.00      A    O  \\nATOM    625  CG  PRO A  84      11.393   6.884 -14.094  1.00  0.00      A    C  \\nATOM    626  CD  PRO A  84      10.854   6.393 -15.407  1.00  0.00      A    C  \\nATOM    627  N   GLN A  85      10.591  10.085 -16.543  1.00  0.00      A    N  \\nATOM    628  CA  GLN A  85      10.711  11.345 -17.269  1.00  0.00      A    C  \\nATOM    629  C   GLN A  85       9.364  11.784 -17.835  1.00  0.00      A    C  \\nATOM    630  CB  GLN A  85      11.738  11.219 -18.396  1.00  0.00      A    C  \\nATOM    631  O   GLN A  85       9.006  12.962 -17.757  1.00  0.00      A    O  \\nATOM    632  CG  GLN A  85      13.179  11.146 -17.910  1.00  0.00      A    C  \\nATOM    633  CD  GLN A  85      14.183  11.141 -19.047  1.00  0.00      A    C  \\nATOM    634  NE2 GLN A  85      15.463  11.027 -18.708  1.00  0.00      A    N  \\nATOM    635  OE1 GLN A  85      13.812  11.239 -20.221  1.00  0.00      A    O  \\nATOM    636  N   LYS A  86       8.635  10.846 -18.352  1.00  0.00      A    N  \\nATOM    637  CA  LYS A  86       7.312  11.132 -18.901  1.00  0.00      A    C  \\nATOM    638  C   LYS A  86       6.401  11.749 -17.844  1.00  0.00      A    C  \\nATOM    639  CB  LYS A  86       6.677   9.858 -19.461  1.00  0.00      A    C  \\nATOM    640  O   LYS A  86       5.596  12.631 -18.150  1.00  0.00      A    O  \\nATOM    641  CG  LYS A  86       5.250  10.044 -19.954  1.00  0.00      A    C  \\nATOM    642  CD  LYS A  86       4.667   8.740 -20.484  1.00  0.00      A    C  \\nATOM    643  CE  LYS A  86       3.200   8.895 -20.863  1.00  0.00      A    C  \\nATOM    644  NZ  LYS A  86       2.616   7.611 -21.353  1.00  0.00      A    N  \\nATOM    645  N   TRP A  87       6.583  11.351 -16.636  1.00  0.00      A    N  \\nATOM    646  CA  TRP A  87       5.617  11.688 -15.595  1.00  0.00      A    C  \\nATOM    647  C   TRP A  87       6.184  12.737 -14.644  1.00  0.00      A    C  \\nATOM    648  CB  TRP A  87       5.211  10.437 -14.811  1.00  0.00      A    C  \\nATOM    649  O   TRP A  87       5.578  13.043 -13.615  1.00  0.00      A    O  \\nATOM    650  CG  TRP A  87       4.417   9.448 -15.611  1.00  0.00      A    C  \\nATOM    651  CD1 TRP A  87       4.803   8.190 -15.983  1.00  0.00      A    C  \\nATOM    652  CD2 TRP A  87       3.102   9.639 -16.142  1.00  0.00      A    C  \\nATOM    653  CE2 TRP A  87       2.752   8.453 -16.825  1.00  0.00      A    C  \\nATOM    654  CE3 TRP A  87       2.185  10.697 -16.105  1.00  0.00      A    C  \\nATOM    655  NE1 TRP A  87       3.805   7.586 -16.714  1.00  0.00      A    N  \\nATOM    656  CH2 TRP A  87       0.642   9.348 -17.416  1.00  0.00      A    C  \\nATOM    657  CZ2 TRP A  87       1.521   8.297 -17.467  1.00  0.00      A    C  \\nATOM    658  CZ3 TRP A  87       0.960  10.540 -16.744  1.00  0.00      A    C  \\nATOM    659  N   GLY A  88       7.311  13.294 -14.962  1.00  0.00      A    N  \\nATOM    660  CA  GLY A  88       7.918  14.320 -14.131  1.00  0.00      A    C  \\nATOM    661  C   GLY A  88       8.402  13.795 -12.792  1.00  0.00      A    C  \\nATOM    662  O   GLY A  88       8.460  14.540 -11.811  1.00  0.00      A    O  \\nATOM    663  N   ILE A  89       8.639  12.499 -12.707  1.00  0.00      A    N  \\nATOM    664  CA  ILE A  89       9.127  11.823 -11.510  1.00  0.00      A    C  \\nATOM    665  C   ILE A  89      10.654  11.807 -11.513  1.00  0.00      A    C  \\nATOM    666  CB  ILE A  89       8.576  10.383 -11.411  1.00  0.00      A    C  \\nATOM    667  O   ILE A  89      11.277  11.607 -12.558  1.00  0.00      A    O  \\nATOM    668  CG1 ILE A  89       7.059  10.405 -11.194  1.00  0.00      A    C  \\nATOM    669  CG2 ILE A  89       9.279   9.612 -10.289  1.00  0.00      A    C  \\nATOM    670  CD1 ILE A  89       6.396   9.040 -11.313  1.00  0.00      A    C  \\nATOM    671  N   LYS A  90      11.275  12.165 -10.393  1.00  0.00      A    N  \\nATOM    672  CA  LYS A  90      12.720  12.117 -10.193  1.00  0.00      A    C  \\nATOM    673  C   LYS A  90      13.280  10.744 -10.552  1.00  0.00      A    C  \\nATOM    674  CB  LYS A  90      13.074  12.465  -8.746  1.00  0.00      A    C  \\nATOM    675  O   LYS A  90      12.529   9.773 -10.671  1.00  0.00      A    O  \\nATOM    676  CG  LYS A  90      12.777  13.908  -8.366  1.00  0.00      A    C  \\nATOM    677  CD  LYS A  90      13.192  14.204  -6.930  1.00  0.00      A    C  \\nATOM    678  CE  LYS A  90      12.883  15.645  -6.544  1.00  0.00      A    C  \\nATOM    679  NZ  LYS A  90      13.255  15.930  -5.126  1.00  0.00      A    N  \\nATOM    680  N   ALA A  91      14.556  10.655 -10.665  1.00  0.00      A    N  \\nATOM    681  CA  ALA A  91      15.218   9.382 -10.941  1.00  0.00      A    C  \\nATOM    682  C   ALA A  91      14.873   8.343  -9.878  1.00  0.00      A    C  \\nATOM    683  CB  ALA A  91      16.730   9.576 -11.021  1.00  0.00      A    C  \\nATOM    684  O   ALA A  91      14.688   8.681  -8.707  1.00  0.00      A    O  \\nATOM    685  N   LEU A  92      14.604   7.186 -10.289  1.00  0.00      A    N  \\nATOM    686  CA  LEU A  92      14.309   6.016  -9.469  1.00  0.00      A    C  \\nATOM    687  C   LEU A  92      15.407   5.786  -8.436  1.00  0.00      A    C  \\nATOM    688  CB  LEU A  92      14.149   4.773 -10.347  1.00  0.00      A    C  \\nATOM    689  O   LEU A  92      16.513   6.312  -8.572  1.00  0.00      A    O  \\nATOM    690  CG  LEU A  92      12.984   4.788 -11.339  1.00  0.00      A    C  \\nATOM    691  CD1 LEU A  92      13.023   3.545 -12.221  1.00  0.00      A    C  \\nATOM    692  CD2 LEU A  92      11.654   4.884 -10.600  1.00  0.00      A    C  \\nATOM    693  N   PRO A  93      15.053   5.111  -7.350  1.00  0.00      A    N  \\nATOM    694  CA  PRO A  93      16.085   4.802  -6.358  1.00  0.00      A    C  \\nATOM    695  C   PRO A  93      17.286   4.078  -6.961  1.00  0.00      A    C  \\nATOM    696  CB  PRO A  93      15.353   3.905  -5.357  1.00  0.00      A    C  \\nATOM    697  O   PRO A  93      17.165   3.445  -8.014  1.00  0.00      A    O  \\nATOM    698  CG  PRO A  93      13.903   4.080  -5.675  1.00  0.00      A    C  \\nATOM    699  CD  PRO A  93      13.780   4.540  -7.099  1.00  0.00      A    C  \\nATOM    700  N   PRO A  94      18.433   4.336  -6.269  1.00  0.00      A    N  \\nATOM    701  CA  PRO A  94      19.607   3.611  -6.760  1.00  0.00      A    C  \\nATOM    702  C   PRO A  94      19.441   2.095  -6.680  1.00  0.00      A    C  \\nATOM    703  CB  PRO A  94      20.729   4.087  -5.833  1.00  0.00      A    C  \\nATOM    704  O   PRO A  94      18.736   1.594  -5.801  1.00  0.00      A    O  \\nATOM    705  CG  PRO A  94      20.033   4.531  -4.588  1.00  0.00      A    C  \\nATOM    706  CD  PRO A  94      18.651   4.994  -4.953  1.00  0.00      A    C  \\nATOM    707  N   LYS A  95      20.011   1.448  -7.670  1.00  0.00      A    N  \\nATOM    708  CA  LYS A  95      20.030  -0.010  -7.610  1.00  0.00      A    C  \\nATOM    709  C   LYS A  95      20.801  -0.500  -6.388  1.00  0.00      A    C  \\nATOM    710  CB  LYS A  95      20.642  -0.592  -8.884  1.00  0.00      A    C  \\nATOM    711  O   LYS A  95      21.808   0.097  -6.002  1.00  0.00      A    O  \\nATOM    712  CG  LYS A  95      19.796  -0.380 -10.131  1.00  0.00      A    C  \\nATOM    713  CD  LYS A  95      20.378  -1.112 -11.333  1.00  0.00      A    C  \\nATOM    714  CE  LYS A  95      19.524  -0.912 -12.578  1.00  0.00      A    C  \\nATOM    715  NZ  LYS A  95      20.115  -1.590 -13.770  1.00  0.00      A    N  \\nATOM    716  N   ASN A  96      20.321  -1.405  -5.666  1.00  0.00      A    N  \\nATOM    717  CA  ASN A  96      21.028  -2.143  -4.624  1.00  0.00      A    C  \\nATOM    718  C   ASN A  96      21.382  -3.556  -5.080  1.00  0.00      A    C  \\nATOM    719  CB  ASN A  96      20.195  -2.192  -3.342  1.00  0.00      A    C  \\nATOM    720  O   ASN A  96      20.498  -4.397  -5.253  1.00  0.00      A    O  \\nATOM    721  CG  ASN A  96      20.956  -2.784  -2.173  1.00  0.00      A    C  \\nATOM    722  ND2 ASN A  96      20.568  -2.409  -0.960  1.00  0.00      A    N  \\nATOM    723  OD1 ASN A  96      21.887  -3.573  -2.358  1.00  0.00      A    O  \\nATOM    724  N   PRO A  97      22.673  -3.794  -5.297  1.00  0.00      A    N  \\nATOM    725  CA  PRO A  97      23.060  -5.090  -5.858  1.00  0.00      A    C  \\nATOM    726  C   PRO A  97      22.683  -6.262  -4.954  1.00  0.00      A    C  \\nATOM    727  CB  PRO A  97      24.580  -4.974  -5.997  1.00  0.00      A    C  \\nATOM    728  O   PRO A  97      22.582  -7.399  -5.421  1.00  0.00      A    O  \\nATOM    729  CG  PRO A  97      24.948  -3.779  -5.177  1.00  0.00      A    C  \\nATOM    730  CD  PRO A  97      23.715  -2.949  -4.964  1.00  0.00      A    C  \\nATOM    731  N   SER A  98      22.623  -5.964  -3.543  1.00  0.00      A    N  \\nATOM    732  CA  SER A  98      22.325  -7.058  -2.625  1.00  0.00      A    C  \\nATOM    733  C   SER A  98      20.856  -7.462  -2.705  1.00  0.00      A    C  \\nATOM    734  CB  SER A  98      22.677  -6.666  -1.190  1.00  0.00      A    C  \\nATOM    735  O   SER A  98      20.471  -8.522  -2.209  1.00  0.00      A    O  \\nATOM    736  OG  SER A  98      24.045  -6.311  -1.087  1.00  0.00      A    O  \\nATOM    737  N   TRP A  99      20.051  -6.692  -3.233  1.00  0.00      A    N  \\nATOM    738  CA  TRP A  99      18.640  -6.986  -3.462  1.00  0.00      A    C  \\nATOM    739  C   TRP A  99      18.394  -7.384  -4.913  1.00  0.00      A    C  \\nATOM    740  CB  TRP A  99      17.773  -5.777  -3.097  1.00  0.00      A    C  \\nATOM    741  O   TRP A  99      18.622  -6.589  -5.828  1.00  0.00      A    O  \\nATOM    742  CG  TRP A  99      17.399  -5.711  -1.646  1.00  0.00      A    C  \\nATOM    743  CD1 TRP A  99      18.246  -5.569  -0.583  1.00  0.00      A    C  \\nATOM    744  CD2 TRP A  99      16.079  -5.791  -1.100  1.00  0.00      A    C  \\nATOM    745  CE2 TRP A  99      16.201  -5.689   0.303  1.00  0.00      A    C  \\nATOM    746  CE3 TRP A  99      14.804  -5.938  -1.662  1.00  0.00      A    C  \\nATOM    747  NE1 TRP A  99      17.532  -5.554   0.593  1.00  0.00      A    N  \\nATOM    748  CH2 TRP A  99      13.857  -5.874   0.578  1.00  0.00      A    C  \\nATOM    749  CZ2 TRP A  99      15.093  -5.729   1.154  1.00  0.00      A    C  \\nATOM    750  CZ3 TRP A  99      13.703  -5.978  -0.814  1.00  0.00      A    C  \\nATOM    751  N   SER A 100      18.741  -8.547  -5.307  1.00  0.00      A    N  \\nATOM    752  CA  SER A 100      19.147  -9.333  -6.468  1.00  0.00      A    C  \\nATOM    753  C   SER A 100      18.123  -9.227  -7.593  1.00  0.00      A    C  \\nATOM    754  CB  SER A 100      19.341 -10.800  -6.082  1.00  0.00      A    C  \\nATOM    755  O   SER A 100      18.435  -9.511  -8.752  1.00  0.00      A    O  \\nATOM    756  OG  SER A 100      18.094 -11.422  -5.826  1.00  0.00      A    O  \\nATOM    757  N   ALA A 101      17.211  -8.387  -7.763  1.00  0.00      A    N  \\nATOM    758  CA  ALA A 101      16.338  -8.649  -8.903  1.00  0.00      A    C  \\nATOM    759  C   ALA A 101      15.308  -7.535  -9.072  1.00  0.00      A    C  \\nATOM    760  CB  ALA A 101      15.638  -9.996  -8.740  1.00  0.00      A    C  \\nATOM    761  O   ALA A 101      14.858  -6.943  -8.089  1.00  0.00      A    O  \\nATOM    762  N   GLN A 102      15.437  -6.893 -10.584  1.00  0.00      A    N  \\nATOM    763  CA  GLN A 102      14.505  -6.355 -11.568  1.00  0.00      A    C  \\nATOM    764  C   GLN A 102      13.862  -5.064 -11.069  1.00  0.00      A    C  \\nATOM    765  CB  GLN A 102      13.424  -7.384 -11.904  1.00  0.00      A    C  \\nATOM    766  O   GLN A 102      13.409  -4.992  -9.925  1.00  0.00      A    O  \\nATOM    767  CG  GLN A 102      13.945  -8.600 -12.657  1.00  0.00      A    C  \\nATOM    768  CD  GLN A 102      12.841  -9.562 -13.054  1.00  0.00      A    C  \\nATOM    769  NE2 GLN A 102      13.222 -10.690 -13.645  1.00  0.00      A    N  \\nATOM    770  OE1 GLN A 102      11.657  -9.295 -12.829  1.00  0.00      A    O  \\nATOM    771  N   ASP A 103      14.542  -3.947 -11.177  1.00  0.00      A    N  \\nATOM    772  CA  ASP A 103      14.260  -2.525 -11.346  1.00  0.00      A    C  \\nATOM    773  C   ASP A 103      13.120  -2.079 -10.433  1.00  0.00      A    C  \\nATOM    774  CB  ASP A 103      13.916  -2.216 -12.805  1.00  0.00      A    C  \\nATOM    775  O   ASP A 103      12.161  -2.824 -10.220  1.00  0.00      A    O  \\nATOM    776  CG  ASP A 103      15.077  -2.463 -13.753  1.00  0.00      A    C  \\nATOM    777  OD1 ASP A 103      16.235  -2.550 -13.290  1.00  0.00      A    O  \\nATOM    778  OD2 ASP A 103      14.831  -2.570 -14.974  1.00  0.00      A    O  \\nATOM    779  N   PHE A 104      13.379  -1.221  -9.454  1.00  0.00      A    N  \\nATOM    780  CA  PHE A 104      12.415  -0.509  -8.623  1.00  0.00      A    C  \\nATOM    781  C   PHE A 104      11.468   0.321  -9.483  1.00  0.00      A    C  \\nATOM    782  CB  PHE A 104      13.135   0.392  -7.615  1.00  0.00      A    C  \\nATOM    783  O   PHE A 104      11.416   1.546  -9.353  1.00  0.00      A    O  \\nATOM    784  CG  PHE A 104      14.042  -0.353  -6.674  1.00  0.00      A    C  \\nATOM    785  CD1 PHE A 104      13.516  -1.169  -5.680  1.00  0.00      A    C  \\nATOM    786  CD2 PHE A 104      15.422  -0.238  -6.784  1.00  0.00      A    C  \\nATOM    787  CE1 PHE A 104      14.354  -1.861  -4.807  1.00  0.00      A    C  \\nATOM    788  CE2 PHE A 104      16.265  -0.926  -5.916  1.00  0.00      A    C  \\nATOM    789  CZ  PHE A 104      15.729  -1.736  -4.928  1.00  0.00      A    C  \\nATOM    790  N   LYS A 105      10.668  -0.508 -10.364  1.00  0.00      A    N  \\nATOM    791  CA  LYS A 105       9.779   0.228 -11.259  1.00  0.00      A    C  \\nATOM    792  C   LYS A 105       8.317  -0.114 -10.985  1.00  0.00      A    C  \\nATOM    793  CB  LYS A 105      10.122  -0.070 -12.720  1.00  0.00      A    C  \\nATOM    794  O   LYS A 105       7.414   0.486 -11.573  1.00  0.00      A    O  \\nATOM    795  CG  LYS A 105      11.478   0.463 -13.158  1.00  0.00      A    C  \\nATOM    796  CD  LYS A 105      11.757   0.146 -14.622  1.00  0.00      A    C  \\nATOM    797  CE  LYS A 105      13.120   0.666 -15.058  1.00  0.00      A    C  \\nATOM    798  NZ  LYS A 105      13.409   0.334 -16.486  1.00  0.00      A    N  \\nATOM    799  N   SER A 106       8.142  -1.126 -10.198  1.00  0.00      A    N  \\nATOM    800  CA  SER A 106       6.765  -1.467  -9.854  1.00  0.00      A    C  \\nATOM    801  C   SER A 106       6.195  -0.492  -8.829  1.00  0.00      A    C  \\nATOM    802  CB  SER A 106       6.685  -2.894  -9.312  1.00  0.00      A    C  \\nATOM    803  O   SER A 106       6.855  -0.166  -7.840  1.00  0.00      A    O  \\nATOM    804  OG  SER A 106       7.023  -3.834 -10.318  1.00  0.00      A    O  \\nATOM    805  N   PRO A 107       4.998  -0.028  -9.073  1.00  0.00      A    N  \\nATOM    806  CA  PRO A 107       4.400   0.891  -8.101  1.00  0.00      A    C  \\nATOM    807  C   PRO A 107       3.693   0.164  -6.959  1.00  0.00      A    C  \\nATOM    808  CB  PRO A 107       3.401   1.691  -8.941  1.00  0.00      A    C  \\nATOM    809  O   PRO A 107       2.709  -0.543  -7.189  1.00  0.00      A    O  \\nATOM    810  CG  PRO A 107       3.007   0.768 -10.048  1.00  0.00      A    C  \\nATOM    811  CD  PRO A 107       4.163  -0.143 -10.344  1.00  0.00      A    C  \\nATOM    812  N   GLU A 108       4.324   0.206  -5.719  1.00  0.00      A    N  \\nATOM    813  CA  GLU A 108       3.566  -0.113  -4.513  1.00  0.00      A    C  \\nATOM    814  C   GLU A 108       2.543   0.975  -4.200  1.00  0.00      A    C  \\nATOM    815  CB  GLU A 108       4.508  -0.307  -3.322  1.00  0.00      A    C  \\nATOM    816  O   GLU A 108       2.864   2.164  -4.231  1.00  0.00      A    O  \\nATOM    817  CG  GLU A 108       3.791  -0.614  -2.015  1.00  0.00      A    C  \\nATOM    818  CD  GLU A 108       2.886  -1.833  -2.099  1.00  0.00      A    C  \\nATOM    819  OE1 GLU A 108       3.151  -2.729  -2.931  1.00  0.00      A    O  \\nATOM    820  OE2 GLU A 108       1.904  -1.891  -1.326  1.00  0.00      A    O  \\nATOM    821  N   GLU A 109       1.364   0.549  -3.942  1.00  0.00      A    N  \\nATOM    822  CA  GLU A 109       0.241   1.455  -3.723  1.00  0.00      A    C  \\nATOM    823  C   GLU A 109      -0.373   1.250  -2.341  1.00  0.00      A    C  \\nATOM    824  CB  GLU A 109      -0.824   1.262  -4.805  1.00  0.00      A    C  \\nATOM    825  O   GLU A 109      -0.709   0.124  -1.966  1.00  0.00      A    O  \\nATOM    826  CG  GLU A 109      -2.062   2.126  -4.611  1.00  0.00      A    C  \\nATOM    827  CD  GLU A 109      -3.151   1.853  -5.636  1.00  0.00      A    C  \\nATOM    828  OE1 GLU A 109      -3.097   0.802  -6.314  1.00  0.00      A    O  \\nATOM    829  OE2 GLU A 109      -4.065   2.699  -5.764  1.00  0.00      A    O  \\nATOM    830  N   TYR A 110      -0.426   2.404  -1.607  1.00  0.00      A    N  \\nATOM    831  CA  TYR A 110      -1.193   2.394  -0.366  1.00  0.00      A    C  \\nATOM    832  C   TYR A 110      -2.153   3.576  -0.309  1.00  0.00      A    C  \\nATOM    833  CB  TYR A 110      -0.257   2.423   0.845  1.00  0.00      A    C  \\nATOM    834  O   TYR A 110      -1.758   4.717  -0.559  1.00  0.00      A    O  \\nATOM    835  CG  TYR A 110      -0.946   2.117   2.153  1.00  0.00      A    C  \\nATOM    836  CD1 TYR A 110      -1.117   0.802   2.581  1.00  0.00      A    C  \\nATOM    837  CD2 TYR A 110      -1.428   3.140   2.962  1.00  0.00      A    C  \\nATOM    838  CE1 TYR A 110      -1.754   0.515   3.784  1.00  0.00      A    C  \\nATOM    839  CE2 TYR A 110      -2.066   2.865   4.167  1.00  0.00      A    C  \\nATOM    840  OH  TYR A 110      -2.854   1.273   5.761  1.00  0.00      A    O  \\nATOM    841  CZ  TYR A 110      -2.223   1.551   4.569  1.00  0.00      A    C  \\nATOM    842  N   ALA A 111      -3.421   3.423   0.191  1.00  0.00      A    N  \\nATOM    843  CA  ALA A 111      -4.019   2.175   0.658  1.00  0.00      A    C  \\nATOM    844  C   ALA A 111      -4.091   1.147  -0.467  1.00  0.00      A    C  \\nATOM    845  CB  ALA A 111      -5.411   2.432   1.229  1.00  0.00      A    C  \\nATOM    846  O   ALA A 111      -4.005   1.500  -1.646  1.00  0.00      A    O  \\nATOM    847  N   PHE A 112      -4.261  -0.134  -0.088  1.00  0.00      A    N  \\nATOM    848  CA  PHE A 112      -4.212  -1.247  -1.028  1.00  0.00      A    C  \\nATOM    849  C   PHE A 112      -5.304  -1.114  -2.083  1.00  0.00      A    C  \\nATOM    850  CB  PHE A 112      -4.354  -2.583  -0.291  1.00  0.00      A    C  \\nATOM    851  O   PHE A 112      -6.442  -0.762  -1.765  1.00  0.00      A    O  \\nATOM    852  CG  PHE A 112      -3.289  -2.820   0.745  1.00  0.00      A    C  \\nATOM    853  CD1 PHE A 112      -1.943  -2.716   0.416  1.00  0.00      A    C  \\nATOM    854  CD2 PHE A 112      -3.634  -3.148   2.050  1.00  0.00      A    C  \\nATOM    855  CE1 PHE A 112      -0.955  -2.935   1.373  1.00  0.00      A    C  \\nATOM    856  CE2 PHE A 112      -2.653  -3.368   3.012  1.00  0.00      A    C  \\nATOM    857  CZ  PHE A 112      -1.314  -3.263   2.671  1.00  0.00      A    C  \\nATOM    858  N   ALA A 113      -4.940  -1.444  -3.341  1.00  0.00      A    N  \\nATOM    859  CA  ALA A 113      -5.949  -1.477  -4.396  1.00  0.00      A    C  \\nATOM    860  C   ALA A 113      -7.088  -2.426  -4.037  1.00  0.00      A    C  \\nATOM    861  CB  ALA A 113      -5.318  -1.886  -5.725  1.00  0.00      A    C  \\nATOM    862  O   ALA A 113      -8.234  -2.213  -4.442  1.00  0.00      A    O  \\nATOM    863  N   SER A 114      -6.831  -3.400  -3.232  1.00  0.00      A    N  \\nATOM    864  CA  SER A 114      -7.807  -4.421  -2.865  1.00  0.00      A    C  \\nATOM    865  C   SER A 114      -8.740  -3.925  -1.765  1.00  0.00      A    C  \\nATOM    866  CB  SER A 114      -7.100  -5.698  -2.407  1.00  0.00      A    C  \\nATOM    867  O   SER A 114      -9.564  -4.685  -1.253  1.00  0.00      A    O  \\nATOM    868  OG  SER A 114      -6.254  -5.436  -1.300  1.00  0.00      A    O  \\nATOM    869  N   SER A 115      -8.705  -2.657  -1.400  1.00  0.00      A    N  \\nATOM    870  CA  SER A 115      -9.528  -2.126  -0.318  1.00  0.00      A    C  \\nATOM    871  C   SER A 115     -10.412  -0.984  -0.806  1.00  0.00      A    C  \\nATOM    872  CB  SER A 115      -8.649  -1.645   0.837  1.00  0.00      A    C  \\nATOM    873  O   SER A 115     -10.135  -0.376  -1.843  1.00  0.00      A    O  \\nATOM    874  OG  SER A 115      -7.940  -0.472   0.475  1.00  0.00      A    O  \\nATOM    875  N   LEU A 116     -11.446  -0.746  -0.055  1.00  0.00      A    N  \\nATOM    876  CA  LEU A 116     -12.382   0.341  -0.318  1.00  0.00      A    C  \\nATOM    877  C   LEU A 116     -11.679   1.693  -0.247  1.00  0.00      A    C  \\nATOM    878  CB  LEU A 116     -13.542   0.303   0.680  1.00  0.00      A    C  \\nATOM    879  O   LEU A 116     -12.026   2.618  -0.985  1.00  0.00      A    O  \\nATOM    880  CG  LEU A 116     -14.649   1.339   0.474  1.00  0.00      A    C  \\nATOM    881  CD1 LEU A 116     -15.319   1.135  -0.881  1.00  0.00      A    C  \\nATOM    882  CD2 LEU A 116     -15.673   1.260   1.601  1.00  0.00      A    C  \\nATOM    883  N   GLN A 117     -10.656   1.784   0.529  1.00  0.00      A    N  \\nATOM    884  CA  GLN A 117      -9.907   3.016   0.759  1.00  0.00      A    C  \\nATOM    885  C   GLN A 117      -8.907   3.269  -0.365  1.00  0.00      A    C  \\nATOM    886  CB  GLN A 117      -9.182   2.960   2.105  1.00  0.00      A    C  \\nATOM    887  O   GLN A 117      -8.295   4.337  -0.430  1.00  0.00      A    O  \\nATOM    888  CG  GLN A 117     -10.116   2.989   3.308  1.00  0.00      A    C  \\nATOM    889  CD  GLN A 117     -10.771   1.647   3.575  1.00  0.00      A    C  \\nATOM    890  NE2 GLN A 117     -11.790   1.644   4.427  1.00  0.00      A    N  \\nATOM    891  OE1 GLN A 117     -10.364   0.622   3.019  1.00  0.00      A    O  \\nATOM    892  N   GLY A 118      -8.709   2.212  -1.232  1.00  0.00      A    N  \\nATOM    893  CA  GLY A 118      -7.748   2.333  -2.317  1.00  0.00      A    C  \\nATOM    894  C   GLY A 118      -8.236   3.217  -3.450  1.00  0.00      A    C  \\nATOM    895  O   GLY A 118      -9.401   3.621  -3.471  1.00  0.00      A    O  \\nATOM    896  N   GLY A 119      -7.366   3.468  -4.358  1.00  0.00      A    N  \\nATOM    897  CA  GLY A 119      -7.707   4.281  -5.514  1.00  0.00      A    C  \\nATOM    898  C   GLY A 119      -7.199   5.707  -5.411  1.00  0.00      A    C  \\nATOM    899  O   GLY A 119      -6.009   5.933  -5.185  1.00  0.00      A    O  \\nATOM    900  N   THR A 120      -8.182   6.653  -5.354  1.00  0.00      A    N  \\nATOM    901  CA  THR A 120      -7.807   8.063  -5.341  1.00  0.00      A    C  \\nATOM    902  C   THR A 120      -7.020   8.401  -4.078  1.00  0.00      A    C  \\nATOM    903  CB  THR A 120      -9.046   8.972  -5.436  1.00  0.00      A    C  \\nATOM    904  O   THR A 120      -7.311   7.876  -3.000  1.00  0.00      A    O  \\nATOM    905  CG2 THR A 120      -8.645  10.442  -5.508  1.00  0.00      A    C  \\nATOM    906  OG1 THR A 120      -9.790   8.633  -6.613  1.00  0.00      A    O  \\nATOM    907  N   ASN A 121      -5.951   9.115  -4.149  1.00  0.00      A    N  \\nATOM    908  CA  ASN A 121      -5.114   9.615  -3.063  1.00  0.00      A    C  \\nATOM    909  C   ASN A 121      -4.183   8.531  -2.528  1.00  0.00      A    C  \\nATOM    910  CB  ASN A 121      -5.980  10.177  -1.933  1.00  0.00      A    C  \\nATOM    911  O   ASN A 121      -3.550   8.710  -1.486  1.00  0.00      A    O  \\nATOM    912  CG  ASN A 121      -6.766  11.403  -2.354  1.00  0.00      A    C  \\nATOM    913  ND2 ASN A 121      -7.985  11.531  -1.843  1.00  0.00      A    N  \\nATOM    914  OD1 ASN A 121      -6.282  12.229  -3.133  1.00  0.00      A    O  \\nATOM    915  N   ALA A 122      -4.192   7.385  -3.252  1.00  0.00      A    N  \\nATOM    916  CA  ALA A 122      -3.197   6.391  -2.857  1.00  0.00      A    C  \\nATOM    917  C   ALA A 122      -1.781   6.937  -3.019  1.00  0.00      A    C  \\nATOM    918  CB  ALA A 122      -3.367   5.113  -3.675  1.00  0.00      A    C  \\nATOM    919  O   ALA A 122      -1.525   7.765  -3.896  1.00  0.00      A    O  \\nATOM    920  N   ILE A 123      -0.950   6.553  -2.112  1.00  0.00      A    N  \\nATOM    921  CA  ILE A 123       0.473   6.863  -2.195  1.00  0.00      A    C  \\nATOM    922  C   ILE A 123       1.201   5.751  -2.946  1.00  0.00      A    C  \\nATOM    923  CB  ILE A 123       1.092   7.057  -0.792  1.00  0.00      A    C  \\nATOM    924  O   ILE A 123       1.038   4.570  -2.629  1.00  0.00      A    O  \\nATOM    925  CG1 ILE A 123       0.486   8.287  -0.107  1.00  0.00      A    C  \\nATOM    926  CG2 ILE A 123       2.616   7.177  -0.887  1.00  0.00      A    C  \\nATOM    927  CD1 ILE A 123       0.951   8.486   1.330  1.00  0.00      A    C  \\nATOM    928  N   LEU A 124       1.945   6.199  -3.930  1.00  0.00      A    N  \\nATOM    929  CA  LEU A 124       2.731   5.263  -4.727  1.00  0.00      A    C  \\nATOM    930  C   LEU A 124       4.222   5.441  -4.462  1.00  0.00      A    C  \\nATOM    931  CB  LEU A 124       2.439   5.451  -6.218  1.00  0.00      A    C  \\nATOM    932  O   LEU A 124       4.703   6.569  -4.333  1.00  0.00      A    O  \\nATOM    933  CG  LEU A 124       1.063   4.992  -6.703  1.00  0.00      A    C  \\nATOM    934  CD1 LEU A 124       0.060   6.138  -6.615  1.00  0.00      A    C  \\nATOM    935  CD2 LEU A 124       1.148   4.458  -8.128  1.00  0.00      A    C  \\nATOM    936  N   ALA A 125       4.919   4.360  -4.348  1.00  0.00      A    N  \\nATOM    937  CA  ALA A 125       6.377   4.353  -4.271  1.00  0.00      A    C  \\nATOM    938  C   ALA A 125       6.963   3.195  -5.074  1.00  0.00      A    C  \\nATOM    939  CB  ALA A 125       6.833   4.270  -2.816  1.00  0.00      A    C  \\nATOM    940  O   ALA A 125       6.339   2.139  -5.199  1.00  0.00      A    O  \\nATOM    941  N   PRO A 126       8.136   3.451  -5.652  1.00  0.00      A    N  \\nATOM    942  CA  PRO A 126       8.739   2.384  -6.454  1.00  0.00      A    C  \\nATOM    943  C   PRO A 126       9.328   1.264  -5.598  1.00  0.00      A    C  \\nATOM    944  CB  PRO A 126       9.837   3.108  -7.237  1.00  0.00      A    C  \\nATOM    945  O   PRO A 126      10.011   1.534  -4.607  1.00  0.00      A    O  \\nATOM    946  CG  PRO A 126      10.242   4.249  -6.360  1.00  0.00      A    C  \\nATOM    947  CD  PRO A 126       9.037   4.713  -5.593  1.00  0.00      A    C  \\nATOM    948  N   VAL A 127       8.957   0.066  -5.981  1.00  0.00      A    N  \\nATOM    949  CA  VAL A 127       9.513  -1.137  -5.372  1.00  0.00      A    C  \\nATOM    950  C   VAL A 127       9.923  -2.125  -6.462  1.00  0.00      A    C  \\nATOM    951  CB  VAL A 127       8.509  -1.800  -4.403  1.00  0.00      A    C  \\nATOM    952  O   VAL A 127       9.720  -1.867  -7.651  1.00  0.00      A    O  \\nATOM    953  CG1 VAL A 127       8.069  -0.813  -3.323  1.00  0.00      A    C  \\nATOM    954  CG2 VAL A 127       7.300  -2.332  -5.169  1.00  0.00      A    C  \\nATOM    955  N   ASN A 128      10.604  -3.192  -6.147  1.00  0.00      A    N  \\nATOM    956  CA  ASN A 128      10.880  -4.232  -7.132  1.00  0.00      A    C  \\nATOM    957  C   ASN A 128       9.740  -5.243  -7.214  1.00  0.00      A    C  \\nATOM    958  CB  ASN A 128      12.197  -4.941  -6.810  1.00  0.00      A    C  \\nATOM    959  O   ASN A 128       8.991  -5.419  -6.251  1.00  0.00      A    O  \\nATOM    960  CG  ASN A 128      12.125  -5.753  -5.531  1.00  0.00      A    C  \\nATOM    961  ND2 ASN A 128      12.957  -6.783  -5.435  1.00  0.00      A    N  \\nATOM    962  OD1 ASN A 128      11.328  -5.457  -4.637  1.00  0.00      A    O  \\nATOM    963  N   LEU A 129       9.626  -5.902  -8.400  1.00  0.00      A    N  \\nATOM    964  CA  LEU A 129       8.505  -6.784  -8.707  1.00  0.00      A    C  \\nATOM    965  C   LEU A 129       8.478  -7.979  -7.759  1.00  0.00      A    C  \\nATOM    966  CB  LEU A 129       8.587  -7.271 -10.156  1.00  0.00      A    C  \\nATOM    967  O   LEU A 129       7.409  -8.391  -7.303  1.00  0.00      A    O  \\nATOM    968  CG  LEU A 129       7.464  -8.197 -10.625  1.00  0.00      A    C  \\nATOM    969  CD1 LEU A 129       6.112  -7.507 -10.480  1.00  0.00      A    C  \\nATOM    970  CD2 LEU A 129       7.697  -8.632 -12.068  1.00  0.00      A    C  \\nATOM    971  N   ALA A 130       9.624  -8.485  -7.391  1.00  0.00      A    N  \\nATOM    972  CA  ALA A 130       9.695  -9.628  -6.484  1.00  0.00      A    C  \\nATOM    973  C   ALA A 130       9.147  -9.269  -5.106  1.00  0.00      A    C  \\nATOM    974  CB  ALA A 130      11.133 -10.126  -6.368  1.00  0.00      A    C  \\nATOM    975  O   ALA A 130       8.409 -10.052  -4.502  1.00  0.00      A    O  \\nATOM    976  N   SER A 131       9.508  -8.097  -4.634  1.00  0.00      A    N  \\nATOM    977  CA  SER A 131       9.026  -7.656  -3.329  1.00  0.00      A    C  \\nATOM    978  C   SER A 131       7.517  -7.434  -3.343  1.00  0.00      A    C  \\nATOM    979  CB  SER A 131       9.734  -6.371  -2.901  1.00  0.00      A    C  \\nATOM    980  O   SER A 131       6.829  -7.748  -2.370  1.00  0.00      A    O  \\nATOM    981  OG  SER A 131       9.342  -5.283  -3.721  1.00  0.00      A    O  \\nATOM    982  N   GLN A 132       7.098  -6.938  -4.452  1.00  0.00      A    N  \\nATOM    983  CA  GLN A 132       5.661  -6.738  -4.611  1.00  0.00      A    C  \\nATOM    984  C   GLN A 132       4.917  -8.070  -4.610  1.00  0.00      A    C  \\nATOM    985  CB  GLN A 132       5.365  -5.972  -5.901  1.00  0.00      A    C  \\nATOM    986  O   GLN A 132       3.869  -8.201  -3.973  1.00  0.00      A    O  \\nATOM    987  CG  GLN A 132       3.964  -5.378  -5.956  1.00  0.00      A    C  \\nATOM    988  CD  GLN A 132       3.747  -4.496  -7.171  1.00  0.00      A    C  \\nATOM    989  NE2 GLN A 132       2.622  -3.789  -7.198  1.00  0.00      A    N  \\nATOM    990  OE1 GLN A 132       4.584  -4.449  -8.078  1.00  0.00      A    O  \\nATOM    991  N   ASN A 133       5.428  -9.054  -5.292  1.00  0.00      A    N  \\nATOM    992  CA  ASN A 133       4.818 -10.379  -5.338  1.00  0.00      A    C  \\nATOM    993  C   ASN A 133       4.813 -11.042  -3.963  1.00  0.00      A    C  \\nATOM    994  CB  ASN A 133       5.541 -11.268  -6.353  1.00  0.00      A    C  \\nATOM    995  O   ASN A 133       3.832 -11.683  -3.582  1.00  0.00      A    O  \\nATOM    996  CG  ASN A 133       5.255 -10.868  -7.787  1.00  0.00      A    C  \\nATOM    997  ND2 ASN A 133       6.083 -11.342  -8.711  1.00  0.00      A    N  \\nATOM    998  OD1 ASN A 133       4.298 -10.139  -8.062  1.00  0.00      A    O  \\nATOM    999  N   SER A 134       5.942 -10.856  -3.233  1.00  0.00      A    N  \\nATOM   1000  CA  SER A 134       6.035 -11.376  -1.873  1.00  0.00      A    C  \\nATOM   1001  C   SER A 134       4.985 -10.743  -0.966  1.00  0.00      A    C  \\nATOM   1002  CB  SER A 134       7.431 -11.132  -1.298  1.00  0.00      A    C  \\nATOM   1003  O   SER A 134       4.318 -11.440  -0.199  1.00  0.00      A    O  \\nATOM   1004  OG  SER A 134       7.564 -11.737  -0.024  1.00  0.00      A    O  \\nATOM   1005  N   GLN A 135       4.801  -9.427  -1.078  1.00  0.00      A    N  \\nATOM   1006  CA  GLN A 135       3.784  -8.719  -0.307  1.00  0.00      A    C  \\nATOM   1007  C   GLN A 135       2.385  -9.229  -0.642  1.00  0.00      A    C  \\nATOM   1008  CB  GLN A 135       3.868  -7.213  -0.560  1.00  0.00      A    C  \\nATOM   1009  O   GLN A 135       1.572  -9.464   0.254  1.00  0.00      A    O  \\nATOM   1010  CG  GLN A 135       2.849  -6.400   0.226  1.00  0.00      A    C  \\nATOM   1011  CD  GLN A 135       2.602  -5.029  -0.376  1.00  0.00      A    C  \\nATOM   1012  NE2 GLN A 135       2.604  -4.002   0.467  1.00  0.00      A    N  \\nATOM   1013  OE1 GLN A 135       2.411  -4.895  -1.589  1.00  0.00      A    O  \\nATOM   1014  N   GLY A 136       2.071  -9.397  -1.922  1.00  0.00      A    N  \\nATOM   1015  CA  GLY A 136       0.785  -9.936  -2.338  1.00  0.00      A    C  \\nATOM   1016  C   GLY A 136       0.493 -11.303  -1.750  1.00  0.00      A    C  \\nATOM   1017  O   GLY A 136      -0.643 -11.589  -1.365  1.00  0.00      A    O  \\nATOM   1018  N   GLY A 137       1.538 -12.133  -1.709  1.00  0.00      A    N  \\nATOM   1019  CA  GLY A 137       1.387 -13.439  -1.087  1.00  0.00      A    C  \\nATOM   1020  C   GLY A 137       1.035 -13.361   0.387  1.00  0.00      A    C  \\nATOM   1021  O   GLY A 137       0.178 -14.106   0.866  1.00  0.00      A    O  \\nATOM   1022  N   VAL A 138       1.726 -12.488   1.128  1.00  0.00      A    N  \\nATOM   1023  CA  VAL A 138       1.468 -12.322   2.554  1.00  0.00      A    C  \\nATOM   1024  C   VAL A 138       0.039 -11.826   2.767  1.00  0.00      A    C  \\nATOM   1025  CB  VAL A 138       2.473 -11.345   3.204  1.00  0.00      A    C  \\nATOM   1026  O   VAL A 138      -0.680 -12.336   3.628  1.00  0.00      A    O  \\nATOM   1027  CG1 VAL A 138       2.054 -11.009   4.634  1.00  0.00      A    C  \\nATOM   1028  CG2 VAL A 138       3.881 -11.937   3.181  1.00  0.00      A    C  \\nATOM   1029  N   LEU A 139      -0.439 -10.907   1.983  1.00  0.00      A    N  \\nATOM   1030  CA  LEU A 139      -1.774 -10.338   2.136  1.00  0.00      A    C  \\nATOM   1031  C   LEU A 139      -2.846 -11.361   1.775  1.00  0.00      A    C  \\nATOM   1032  CB  LEU A 139      -1.928  -9.091   1.262  1.00  0.00      A    C  \\nATOM   1033  O   LEU A 139      -3.851 -11.488   2.479  1.00  0.00      A    O  \\nATOM   1034  CG  LEU A 139      -1.284  -7.808   1.792  1.00  0.00      A    C  \\nATOM   1035  CD1 LEU A 139      -1.365  -6.703   0.744  1.00  0.00      A    C  \\nATOM   1036  CD2 LEU A 139      -1.952  -7.371   3.091  1.00  0.00      A    C  \\nATOM   1037  N   ASN A 140      -2.629 -12.076   0.698  1.00  0.00      A    N  \\nATOM   1038  CA  ASN A 140      -3.588 -13.105   0.310  1.00  0.00      A    C  \\nATOM   1039  C   ASN A 140      -3.694 -14.199   1.368  1.00  0.00      A    C  \\nATOM   1040  CB  ASN A 140      -3.209 -13.709  -1.043  1.00  0.00      A    C  \\nATOM   1041  O   ASN A 140      -4.783 -14.718   1.624  1.00  0.00      A    O  \\nATOM   1042  CG  ASN A 140      -3.523 -12.787  -2.204  1.00  0.00      A    C  \\nATOM   1043  ND2 ASN A 140      -2.825 -12.975  -3.318  1.00  0.00      A    N  \\nATOM   1044  OD1 ASN A 140      -4.386 -11.911  -2.101  1.00  0.00      A    O  \\nATOM   1045  N   GLY A 141      -2.556 -14.550   1.928  1.00  0.00      A    N  \\nATOM   1046  CA  GLY A 141      -2.578 -15.478   3.048  1.00  0.00      A    C  \\nATOM   1047  C   GLY A 141      -3.394 -14.975   4.223  1.00  0.00      A    C  \\nATOM   1048  O   GLY A 141      -4.170 -15.729   4.815  1.00  0.00      A    O  \\nATOM   1049  N   PHE A 142      -3.220 -13.690   4.514  1.00  0.00      A    N  \\nATOM   1050  CA  PHE A 142      -3.996 -13.060   5.577  1.00  0.00      A    C  \\nATOM   1051  C   PHE A 142      -5.484 -13.083   5.247  1.00  0.00      A    C  \\nATOM   1052  CB  PHE A 142      -3.530 -11.618   5.801  1.00  0.00      A    C  \\nATOM   1053  O   PHE A 142      -6.306 -13.443   6.091  1.00  0.00      A    O  \\nATOM   1054  CG  PHE A 142      -4.479 -10.796   6.630  1.00  0.00      A    C  \\nATOM   1055  CD1 PHE A 142      -4.584 -11.001   8.001  1.00  0.00      A    C  \\nATOM   1056  CD2 PHE A 142      -5.267  -9.817   6.039  1.00  0.00      A    C  \\nATOM   1057  CE1 PHE A 142      -5.462 -10.241   8.770  1.00  0.00      A    C  \\nATOM   1058  CE2 PHE A 142      -6.147  -9.054   6.801  1.00  0.00      A    C  \\nATOM   1059  CZ  PHE A 142      -6.243  -9.268   8.167  1.00  0.00      A    C  \\nATOM   1060  N   TYR A 143      -5.881 -12.725   4.046  1.00  0.00      A    N  \\nATOM   1061  CA  TYR A 143      -7.281 -12.705   3.637  1.00  0.00      A    C  \\nATOM   1062  C   TYR A 143      -7.908 -14.087   3.773  1.00  0.00      A    C  \\nATOM   1063  CB  TYR A 143      -7.412 -12.211   2.193  1.00  0.00      A    C  \\nATOM   1064  O   TYR A 143      -9.029 -14.220   4.271  1.00  0.00      A    O  \\nATOM   1065  CG  TYR A 143      -7.078 -10.750   2.018  1.00  0.00      A    C  \\nATOM   1066  CD1 TYR A 143      -7.272  -9.842   3.057  1.00  0.00      A    C  \\nATOM   1067  CD2 TYR A 143      -6.570 -10.273   0.814  1.00  0.00      A    C  \\nATOM   1068  CE1 TYR A 143      -6.967  -8.494   2.900  1.00  0.00      A    C  \\nATOM   1069  CE2 TYR A 143      -6.262  -8.927   0.646  1.00  0.00      A    C  \\nATOM   1070  OH  TYR A 143      -6.161  -6.713   1.532  1.00  0.00      A    O  \\nATOM   1071  CZ  TYR A 143      -6.464  -8.047   1.693  1.00  0.00      A    C  \\nATOM   1072  N   SER A 144      -7.198 -15.112   3.343  1.00  0.00      A    N  \\nATOM   1073  CA  SER A 144      -7.711 -16.478   3.377  1.00  0.00      A    C  \\nATOM   1074  C   SER A 144      -7.803 -16.998   4.808  1.00  0.00      A    C  \\nATOM   1075  CB  SER A 144      -6.824 -17.405   2.545  1.00  0.00      A    C  \\nATOM   1076  O   SER A 144      -8.844 -17.513   5.221  1.00  0.00      A    O  \\nATOM   1077  OG  SER A 144      -7.320 -18.733   2.572  1.00  0.00      A    O  \\nATOM   1078  N   ALA A 145      -6.764 -16.844   5.590  1.00  0.00      A    N  \\nATOM   1079  CA  ALA A 145      -6.682 -17.369   6.951  1.00  0.00      A    C  \\nATOM   1080  C   ALA A 145      -7.736 -16.728   7.850  1.00  0.00      A    C  \\nATOM   1081  CB  ALA A 145      -5.287 -17.141   7.526  1.00  0.00      A    C  \\nATOM   1082  O   ALA A 145      -8.218 -17.355   8.796  1.00  0.00      A    O  \\nATOM   1083  N   ASN A 146      -8.131 -15.428   7.488  1.00  0.00      A    N  \\nATOM   1084  CA  ASN A 146      -9.052 -14.691   8.346  1.00  0.00      A    C  \\nATOM   1085  C   ASN A 146     -10.420 -14.525   7.690  1.00  0.00      A    C  \\nATOM   1086  CB  ASN A 146      -8.469 -13.324   8.711  1.00  0.00      A    C  \\nATOM   1087  O   ASN A 146     -11.249 -13.746   8.161  1.00  0.00      A    O  \\nATOM   1088  CG  ASN A 146      -7.276 -13.427   9.641  1.00  0.00      A    C  \\nATOM   1089  ND2 ASN A 146      -6.077 -13.355   9.077  1.00  0.00      A    N  \\nATOM   1090  OD1 ASN A 146      -7.432 -13.572  10.856  1.00  0.00      A    O  \\nATOM   1091  N   LYS A 147     -10.597 -15.252   6.562  1.00  0.00      A    N  \\nATOM   1092  CA  LYS A 147     -11.862 -15.262   5.833  1.00  0.00      A    C  \\nATOM   1093  C   LYS A 147     -12.368 -13.842   5.593  1.00  0.00      A    C  \\nATOM   1094  CB  LYS A 147     -12.913 -16.072   6.592  1.00  0.00      A    C  \\nATOM   1095  O   LYS A 147     -13.535 -13.542   5.854  1.00  0.00      A    O  \\nATOM   1096  CG  LYS A 147     -12.535 -17.530   6.806  1.00  0.00      A    C  \\nATOM   1097  CD  LYS A 147     -13.625 -18.287   7.554  1.00  0.00      A    C  \\nATOM   1098  CE  LYS A 147     -13.210 -19.722   7.849  1.00  0.00      A    C  \\nATOM   1099  NZ  LYS A 147     -14.231 -20.436   8.673  1.00  0.00      A    N  \\nATOM   1100  N   VAL A 148     -11.471 -12.991   5.179  1.00  0.00      A    N  \\nATOM   1101  CA  VAL A 148     -11.786 -11.597   4.880  1.00  0.00      A    C  \\nATOM   1102  C   VAL A 148     -12.640 -11.520   3.617  1.00  0.00      A    C  \\nATOM   1103  CB  VAL A 148     -10.506 -10.750   4.711  1.00  0.00      A    C  \\nATOM   1104  O   VAL A 148     -12.145 -11.747   2.510  1.00  0.00      A    O  \\nATOM   1105  CG1 VAL A 148     -10.857  -9.305   4.360  1.00  0.00      A    C  \\nATOM   1106  CG2 VAL A 148      -9.658 -10.804   5.980  1.00  0.00      A    C  \\nATOM   1107  N   ALA A 149     -13.919 -11.250   3.769  1.00  0.00      A    N  \\nATOM   1108  CA  ALA A 149     -14.884 -11.220   2.674  1.00  0.00      A    C  \\nATOM   1109  C   ALA A 149     -14.751  -9.936   1.860  1.00  0.00      A    C  \\nATOM   1110  CB  ALA A 149     -16.306 -11.360   3.212  1.00  0.00      A    C  \\nATOM   1111  O   ALA A 149     -14.367  -8.892   2.394  1.00  0.00      A    O  \\nATOM   1112  N   GLN A 150     -15.105 -10.064   0.554  1.00  0.00      A    N  \\nATOM   1113  CA  GLN A 150     -15.017  -8.931  -0.361  1.00  0.00      A    C  \\nATOM   1114  C   GLN A 150     -16.319  -8.135  -0.377  1.00  0.00      A    C  \\nATOM   1115  CB  GLN A 150     -14.676  -9.407  -1.774  1.00  0.00      A    C  \\nATOM   1116  O   GLN A 150     -17.407  -8.713  -0.347  1.00  0.00      A    O  \\nATOM   1117  CG  GLN A 150     -13.315 -10.083  -1.883  1.00  0.00      A    C  \\nATOM   1118  CD  GLN A 150     -13.071 -10.692  -3.251  1.00  0.00      A    C  \\nATOM   1119  NE2 GLN A 150     -11.897 -11.286  -3.435  1.00  0.00      A    N  \\nATOM   1120  OE1 GLN A 150     -13.931 -10.630  -4.136  1.00  0.00      A    O  \\nATOM   1121  N   PHE A 151     -16.214  -6.897  -0.297  1.00  0.00      A    N  \\nATOM   1122  CA  PHE A 151     -17.286  -5.924  -0.470  1.00  0.00      A    C  \\nATOM   1123  C   PHE A 151     -17.416  -5.517  -1.933  1.00  0.00      A    C  \\nATOM   1124  CB  PHE A 151     -17.038  -4.688   0.399  1.00  0.00      A    C  \\nATOM   1125  O   PHE A 151     -16.412  -5.295  -2.612  1.00  0.00      A    O  \\nATOM   1126  CG  PHE A 151     -18.066  -3.603   0.220  1.00  0.00      A    C  \\nATOM   1127  CD1 PHE A 151     -19.376  -3.791   0.643  1.00  0.00      A    C  \\nATOM   1128  CD2 PHE A 151     -17.723  -2.395  -0.373  1.00  0.00      A    C  \\nATOM   1129  CE1 PHE A 151     -20.330  -2.790   0.478  1.00  0.00      A    C  \\nATOM   1130  CE2 PHE A 151     -18.670  -1.389  -0.541  1.00  0.00      A    C  \\nATOM   1131  CZ  PHE A 151     -19.973  -1.589  -0.116  1.00  0.00      A    C  \\nATOM   1132  N   ASP A 152     -18.623  -5.518  -2.423  1.00  0.00      A    N  \\nATOM   1133  CA  ASP A 152     -18.936  -4.972  -3.740  1.00  0.00      A    C  \\nATOM   1134  C   ASP A 152     -19.596  -3.600  -3.623  1.00  0.00      A    C  \\nATOM   1135  CB  ASP A 152     -19.846  -5.928  -4.515  1.00  0.00      A    C  \\nATOM   1136  O   ASP A 152     -20.774  -3.499  -3.276  1.00  0.00      A    O  \\nATOM   1137  CG  ASP A 152     -19.997  -5.549  -5.977  1.00  0.00      A    C  \\nATOM   1138  OD1 ASP A 152     -19.697  -4.391  -6.341  1.00  0.00      A    O  \\nATOM   1139  OD2 ASP A 152     -20.422  -6.414  -6.773  1.00  0.00      A    O  \\nATOM   1140  N   PRO A 153     -18.827  -2.512  -3.857  1.00  0.00      A    N  \\nATOM   1141  CA  PRO A 153     -19.387  -1.170  -3.680  1.00  0.00      A    C  \\nATOM   1142  C   PRO A 153     -20.583  -0.904  -4.591  1.00  0.00      A    C  \\nATOM   1143  CB  PRO A 153     -18.215  -0.250  -4.034  1.00  0.00      A    C  \\nATOM   1144  O   PRO A 153     -21.389  -0.013  -4.313  1.00  0.00      A    O  \\nATOM   1145  CG  PRO A 153     -16.996  -1.092  -3.837  1.00  0.00      A    C  \\nATOM   1146  CD  PRO A 153     -17.354  -2.523  -4.120  1.00  0.00      A    C  \\nATOM   1147  N   SER A 154     -20.733  -1.653  -5.694  1.00  0.00      A    N  \\nATOM   1148  CA  SER A 154     -21.859  -1.448  -6.600  1.00  0.00      A    C  \\nATOM   1149  C   SER A 154     -23.132  -2.085  -6.053  1.00  0.00      A    C  \\nATOM   1150  CB  SER A 154     -21.545  -2.020  -7.982  1.00  0.00      A    C  \\nATOM   1151  O   SER A 154     -24.230  -1.808  -6.540  1.00  0.00      A    O  \\nATOM   1152  OG  SER A 154     -21.513  -3.437  -7.945  1.00  0.00      A    O  \\nATOM   1153  N   LYS A 155     -22.940  -2.896  -5.020  1.00  0.00      A    N  \\nATOM   1154  CA  LYS A 155     -24.069  -3.569  -4.385  1.00  0.00      A    C  \\nATOM   1155  C   LYS A 155     -23.942  -3.540  -2.865  1.00  0.00      A    C  \\nATOM   1156  CB  LYS A 155     -24.177  -5.014  -4.875  1.00  0.00      A    C  \\nATOM   1157  O   LYS A 155     -23.912  -4.590  -2.219  1.00  0.00      A    O  \\nATOM   1158  CG  LYS A 155     -24.371  -5.146  -6.378  1.00  0.00      A    C  \\nATOM   1159  CD  LYS A 155     -24.433  -6.606  -6.808  1.00  0.00      A    C  \\nATOM   1160  CE  LYS A 155     -24.488  -6.741  -8.324  1.00  0.00      A    C  \\nATOM   1161  NZ  LYS A 155     -24.471  -8.172  -8.753  1.00  0.00      A    N  \\nATOM   1162  N   PRO A 156     -23.986  -2.346  -2.288  1.00  0.00      A    N  \\nATOM   1163  CA  PRO A 156     -23.656  -2.200  -0.868  1.00  0.00      A    C  \\nATOM   1164  C   PRO A 156     -24.673  -2.879   0.047  1.00  0.00      A    C  \\nATOM   1165  CB  PRO A 156     -23.661  -0.683  -0.659  1.00  0.00      A    C  \\nATOM   1166  O   PRO A 156     -24.344  -3.243   1.178  1.00  0.00      A    O  \\nATOM   1167  CG  PRO A 156     -24.513  -0.148  -1.765  1.00  0.00      A    C  \\nATOM   1168  CD  PRO A 156     -24.337  -1.026  -2.971  1.00  0.00      A    C  \\nATOM   1169  N   GLN A 157     -25.894  -3.010  -0.340  1.00  0.00      A    N  \\nATOM   1170  CA  GLN A 157     -26.946  -3.608   0.475  1.00  0.00      A    C  \\nATOM   1171  C   GLN A 157     -26.906  -5.132   0.392  1.00  0.00      A    C  \\nATOM   1172  CB  GLN A 157     -28.319  -3.095   0.041  1.00  0.00      A    C  \\nATOM   1173  O   GLN A 157     -27.508  -5.819   1.219  1.00  0.00      A    O  \\nATOM   1174  CG  GLN A 157     -28.513  -1.600   0.256  1.00  0.00      A    C  \\nATOM   1175  CD  GLN A 157     -29.922  -1.138  -0.067  1.00  0.00      A    C  \\nATOM   1176  NE2 GLN A 157     -30.142   0.171  -0.018  1.00  0.00      A    N  \\nATOM   1177  OE1 GLN A 157     -30.804  -1.951  -0.358  1.00  0.00      A    O  \\nATOM   1178  N   GLN A 158     -26.176  -5.650  -0.617  1.00  0.00      A    N  \\nATOM   1179  CA  GLN A 158     -26.192  -7.085  -0.878  1.00  0.00      A    C  \\nATOM   1180  C   GLN A 158     -24.902  -7.745  -0.398  1.00  0.00      A    C  \\nATOM   1181  CB  GLN A 158     -26.399  -7.359  -2.368  1.00  0.00      A    C  \\nATOM   1182  O   GLN A 158     -24.855  -8.963  -0.209  1.00  0.00      A    O  \\nATOM   1183  CG  GLN A 158     -27.741  -6.877  -2.901  1.00  0.00      A    C  \\nATOM   1184  CD  GLN A 158     -27.893  -7.096  -4.395  1.00  0.00      A    C  \\nATOM   1185  NE2 GLN A 158     -28.991  -6.604  -4.958  1.00  0.00      A    N  \\nATOM   1186  OE1 GLN A 158     -27.029  -7.701  -5.038  1.00  0.00      A    O  \\nATOM   1187  N   THR A 159     -23.909  -7.036  -0.297  1.00  0.00      A    N  \\nATOM   1188  CA  THR A 159     -22.619  -7.620   0.049  1.00  0.00      A    C  \\nATOM   1189  C   THR A 159     -22.075  -7.006   1.336  1.00  0.00      A    C  \\nATOM   1190  CB  THR A 159     -21.596  -7.428  -1.086  1.00  0.00      A    C  \\nATOM   1191  O   THR A 159     -21.991  -5.782   1.459  1.00  0.00      A    O  \\nATOM   1192  CG2 THR A 159     -22.008  -8.201  -2.334  1.00  0.00      A    C  \\nATOM   1193  OG1 THR A 159     -21.504  -6.035  -1.408  1.00  0.00      A    O  \\nATOM   1194  N   LYS A 160     -21.816  -7.875   2.280  1.00  0.00      A    N  \\nATOM   1195  CA  LYS A 160     -21.142  -7.440   3.499  1.00  0.00      A    C  \\nATOM   1196  C   LYS A 160     -19.644  -7.726   3.432  1.00  0.00      A    C  \\nATOM   1197  CB  LYS A 160     -21.751  -8.125   4.723  1.00  0.00      A    C  \\nATOM   1198  O   LYS A 160     -19.234  -8.856   3.159  1.00  0.00      A    O  \\nATOM   1199  CG  LYS A 160     -21.392  -7.463   6.046  1.00  0.00      A    C  \\nATOM   1200  CD  LYS A 160     -22.069  -8.157   7.220  1.00  0.00      A    C  \\nATOM   1201  CE  LYS A 160     -21.649  -7.546   8.550  1.00  0.00      A    C  \\nATOM   1202  NZ  LYS A 160     -22.329  -8.208   9.704  1.00  0.00      A    N  \\nATOM   1203  N   GLY A 161     -18.808  -6.744   3.312  1.00  0.00      A    N  \\nATOM   1204  CA  GLY A 161     -17.360  -6.881   3.323  1.00  0.00      A    C  \\nATOM   1205  C   GLY A 161     -16.774  -6.915   4.722  1.00  0.00      A    C  \\nATOM   1206  O   GLY A 161     -17.408  -6.463   5.678  1.00  0.00      A    O  \\nATOM   1207  N   THR A 162     -15.644  -7.617   4.881  1.00  0.00      A    N  \\nATOM   1208  CA  THR A 162     -14.948  -7.661   6.162  1.00  0.00      A    C  \\nATOM   1209  C   THR A 162     -13.988  -6.483   6.296  1.00  0.00      A    C  \\nATOM   1210  CB  THR A 162     -14.172  -8.980   6.332  1.00  0.00      A    C  \\nATOM   1211  O   THR A 162     -13.245  -6.171   5.363  1.00  0.00      A    O  \\nATOM   1212  CG2 THR A 162     -13.460  -9.030   7.680  1.00  0.00      A    C  \\nATOM   1213  OG1 THR A 162     -15.087 -10.080   6.246  1.00  0.00      A    O  \\nATOM   1214  N   TRP A 163     -14.179  -5.804   7.467  1.00  0.00      A    N  \\nATOM   1215  CA  TRP A 163     -13.217  -4.766   7.826  1.00  0.00      A    C  \\nATOM   1216  C   TRP A 163     -12.001  -5.369   8.522  1.00  0.00      A    C  \\nATOM   1217  CB  TRP A 163     -13.870  -3.717   8.729  1.00  0.00      A    C  \\nATOM   1218  O   TRP A 163     -12.131  -6.310   9.308  1.00  0.00      A    O  \\nATOM   1219  CG  TRP A 163     -14.935  -2.908   8.051  1.00  0.00      A    C  \\nATOM   1220  CD1 TRP A 163     -16.258  -3.230   7.924  1.00  0.00      A    C  \\nATOM   1221  CD2 TRP A 163     -14.766  -1.643   7.404  1.00  0.00      A    C  \\nATOM   1222  CE2 TRP A 163     -16.029  -1.256   6.905  1.00  0.00      A    C  \\nATOM   1223  CE3 TRP A 163     -13.667  -0.799   7.197  1.00  0.00      A    C  \\nATOM   1224  NE1 TRP A 163     -16.922  -2.240   7.236  1.00  0.00      A    N  \\nATOM   1225  CH2 TRP A 163     -15.132   0.749   6.024  1.00  0.00      A    C  \\nATOM   1226  CZ2 TRP A 163     -16.224  -0.059   6.212  1.00  0.00      A    C  \\nATOM   1227  CZ3 TRP A 163     -13.862   0.392   6.507  1.00  0.00      A    C  \\nATOM   1228  N   PHE A 164     -10.859  -4.843   8.223  1.00  0.00      A    N  \\nATOM   1229  CA  PHE A 164      -9.649  -5.364   8.846  1.00  0.00      A    C  \\nATOM   1230  C   PHE A 164      -8.642  -4.247   9.091  1.00  0.00      A    C  \\nATOM   1231  CB  PHE A 164      -9.021  -6.456   7.975  1.00  0.00      A    C  \\nATOM   1232  O   PHE A 164      -8.722  -3.184   8.471  1.00  0.00      A    O  \\nATOM   1233  CG  PHE A 164      -8.481  -5.950   6.664  1.00  0.00      A    C  \\nATOM   1234  CD1 PHE A 164      -9.315  -5.812   5.560  1.00  0.00      A    C  \\nATOM   1235  CD2 PHE A 164      -7.140  -5.614   6.535  1.00  0.00      A    C  \\nATOM   1236  CE1 PHE A 164      -8.818  -5.344   4.346  1.00  0.00      A    C  \\nATOM   1237  CE2 PHE A 164      -6.637  -5.146   5.325  1.00  0.00      A    C  \\nATOM   1238  CZ  PHE A 164      -7.477  -5.013   4.231  1.00  0.00      A    C  \\nATOM   1239  N   GLN A 165      -7.778  -4.430  10.044  1.00  0.00      A    N  \\nATOM   1240  CA  GLN A 165      -6.692  -3.531  10.419  1.00  0.00      A    C  \\nATOM   1241  C   GLN A 165      -5.408  -4.306  10.698  1.00  0.00      A    C  \\nATOM   1242  CB  GLN A 165      -7.080  -2.700  11.642  1.00  0.00      A    C  \\nATOM   1243  O   GLN A 165      -5.407  -5.252  11.488  1.00  0.00      A    O  \\nATOM   1244  CG  GLN A 165      -6.013  -1.702  12.072  1.00  0.00      A    C  \\nATOM   1245  CD  GLN A 165      -6.438  -0.865  13.264  1.00  0.00      A    C  \\nATOM   1246  NE2 GLN A 165      -5.746   0.248  13.483  1.00  0.00      A    N  \\nATOM   1247  OE1 GLN A 165      -7.381  -1.214  13.980  1.00  0.00      A    O  \\nATOM   1248  N   ILE A 166      -4.396  -3.942   9.941  1.00  0.00      A    N  \\nATOM   1249  CA  ILE A 166      -3.081  -4.475  10.281  1.00  0.00      A    C  \\nATOM   1250  C   ILE A 166      -2.472  -3.659  11.419  1.00  0.00      A    C  \\nATOM   1251  CB  ILE A 166      -2.138  -4.476   9.057  1.00  0.00      A    C  \\nATOM   1252  O   ILE A 166      -2.240  -2.457  11.274  1.00  0.00      A    O  \\nATOM   1253  CG1 ILE A 166      -2.753  -5.286   7.910  1.00  0.00      A    C  \\nATOM   1254  CG2 ILE A 166      -0.760  -5.024   9.438  1.00  0.00      A    C  \\nATOM   1255  CD1 ILE A 166      -2.102  -5.034   6.556  1.00  0.00      A    C  \\nATOM   1256  N   THR A 167      -2.238  -4.292  12.599  1.00  0.00      A    N  \\nATOM   1257  CA  THR A 167      -1.866  -3.578  13.815  1.00  0.00      A    C  \\nATOM   1258  C   THR A 167      -0.351  -3.569  13.995  1.00  0.00      A    C  \\nATOM   1259  CB  THR A 167      -2.529  -4.204  15.056  1.00  0.00      A    C  \\nATOM   1260  O   THR A 167       0.210  -2.613  14.534  1.00  0.00      A    O  \\nATOM   1261  CG2 THR A 167      -4.040  -3.998  15.035  1.00  0.00      A    C  \\nATOM   1262  OG1 THR A 167      -2.251  -5.610  15.081  1.00  0.00      A    O  \\nATOM   1263  N   LYS A 168       0.247  -4.597  13.631  1.00  0.00      A    N  \\nATOM   1264  CA  LYS A 168       1.696  -4.703  13.775  1.00  0.00      A    C  \\nATOM   1265  C   LYS A 168       2.301  -5.543  12.654  1.00  0.00      A    C  \\nATOM   1266  CB  LYS A 168       2.058  -5.305  15.134  1.00  0.00      A    C  \\nATOM   1267  O   LYS A 168       1.629  -6.406  12.087  1.00  0.00      A    O  \\nATOM   1268  CG  LYS A 168       3.550  -5.311  15.431  1.00  0.00      A    C  \\nATOM   1269  CD  LYS A 168       3.840  -5.852  16.825  1.00  0.00      A    C  \\nATOM   1270  CE  LYS A 168       5.337  -5.950  17.087  1.00  0.00      A    C  \\nATOM   1271  NZ  LYS A 168       5.627  -6.555  18.421  1.00  0.00      A    N  \\nATOM   1272  N   PHE A 169       3.544  -5.189  12.284  1.00  0.00      A    N  \\nATOM   1273  CA  PHE A 169       4.357  -5.982  11.369  1.00  0.00      A    C  \\nATOM   1274  C   PHE A 169       5.502  -6.661  12.112  1.00  0.00      A    C  \\nATOM   1275  CB  PHE A 169       4.910  -5.104  10.242  1.00  0.00      A    C  \\nATOM   1276  O   PHE A 169       6.274  -6.001  12.811  1.00  0.00      A    O  \\nATOM   1277  CG  PHE A 169       3.853  -4.570   9.314  1.00  0.00      A    C  \\nATOM   1278  CD1 PHE A 169       3.541  -5.237   8.136  1.00  0.00      A    C  \\nATOM   1279  CD2 PHE A 169       3.170  -3.400   9.621  1.00  0.00      A    C  \\nATOM   1280  CE1 PHE A 169       2.563  -4.745   7.275  1.00  0.00      A    C  \\nATOM   1281  CE2 PHE A 169       2.191  -2.902   8.766  1.00  0.00      A    C  \\nATOM   1282  CZ  PHE A 169       1.890  -3.575   7.593  1.00  0.00      A    C  \\nATOM   1283  N   THR A 170       5.505  -7.960  12.009  1.00  0.00      A    N  \\nATOM   1284  CA  THR A 170       6.505  -8.738  12.733  1.00  0.00      A    C  \\nATOM   1285  C   THR A 170       7.529  -9.332  11.770  1.00  0.00      A    C  \\nATOM   1286  CB  THR A 170       5.849  -9.867  13.549  1.00  0.00      A    C  \\nATOM   1287  O   THR A 170       7.409  -9.174  10.553  1.00  0.00      A    O  \\nATOM   1288  CG2 THR A 170       4.818  -9.313  14.526  1.00  0.00      A    C  \\nATOM   1289  OG1 THR A 170       5.201 -10.781  12.656  1.00  0.00      A    O  \\nATOM   1290  N   GLY A 171       8.599  -9.907  12.354  1.00  0.00      A    N  \\nATOM   1291  CA  GLY A 171       9.647 -10.446  11.502  1.00  0.00      A    C  \\nATOM   1292  C   GLY A 171      10.603  -9.386  10.988  1.00  0.00      A    C  \\nATOM   1293  O   GLY A 171      11.003  -8.490  11.734  1.00  0.00      A    O  \\nATOM   1294  N   ALA A 172      11.054  -9.538   9.627  1.00  0.00      A    N  \\nATOM   1295  CA  ALA A 172      12.052  -8.643   9.047  1.00  0.00      A    C  \\nATOM   1296  C   ALA A 172      11.415  -7.337   8.584  1.00  0.00      A    C  \\nATOM   1297  CB  ALA A 172      12.766  -9.326   7.882  1.00  0.00      A    C  \\nATOM   1298  O   ALA A 172      11.557  -6.946   7.422  1.00  0.00      A    O  \\nATOM   1299  N   ALA A 173      10.702  -6.665   9.515  1.00  0.00      A    N  \\nATOM   1300  CA  ALA A 173      10.030  -5.404   9.212  1.00  0.00      A    C  \\nATOM   1301  C   ALA A 173      11.043  -4.288   8.969  1.00  0.00      A    C  \\nATOM   1302  CB  ALA A 173       9.082  -5.020  10.345  1.00  0.00      A    C  \\nATOM   1303  O   ALA A 173      12.100  -4.252   9.603  1.00  0.00      A    O  \\nATOM   1304  N   GLY A 174      10.789  -3.428   8.019  1.00  0.00      A    N  \\nATOM   1305  CA  GLY A 174      11.609  -2.256   7.758  1.00  0.00      A    C  \\nATOM   1306  C   GLY A 174      11.479  -1.187   8.826  1.00  0.00      A    C  \\nATOM   1307  O   GLY A 174      10.682  -1.325   9.756  1.00  0.00      A    O  \\nATOM   1308  N   PRO A 175      12.233  -0.181   8.755  1.00  0.00      A    N  \\nATOM   1309  CA  PRO A 175      12.292   0.820   9.823  1.00  0.00      A    C  \\nATOM   1310  C   PRO A 175      10.961   1.538  10.034  1.00  0.00      A    C  \\nATOM   1311  CB  PRO A 175      13.366   1.795   9.333  1.00  0.00      A    C  \\nATOM   1312  O   PRO A 175      10.603   1.861  11.169  1.00  0.00      A    O  \\nATOM   1313  CG  PRO A 175      13.474   1.538   7.865  1.00  0.00      A    C  \\nATOM   1314  CD  PRO A 175      13.127   0.100   7.605  1.00  0.00      A    C  \\nATOM   1315  N   TYR A 176      10.147   1.855   9.035  1.00  0.00      A    N  \\nATOM   1316  CA  TYR A 176       8.871   2.530   9.246  1.00  0.00      A    C  \\nATOM   1317  C   TYR A 176       7.868   1.603   9.920  1.00  0.00      A    C  \\nATOM   1318  CB  TYR A 176       8.304   3.035   7.915  1.00  0.00      A    C  \\nATOM   1319  O   TYR A 176       7.076   2.040  10.759  1.00  0.00      A    O  \\nATOM   1320  CG  TYR A 176       9.064   4.202   7.334  1.00  0.00      A    C  \\nATOM   1321  CD1 TYR A 176       8.998   5.463   7.922  1.00  0.00      A    C  \\nATOM   1322  CD2 TYR A 176       9.850   4.046   6.197  1.00  0.00      A    C  \\nATOM   1323  CE1 TYR A 176       9.695   6.543   7.389  1.00  0.00      A    C  \\nATOM   1324  CE2 TYR A 176      10.551   5.118   5.656  1.00  0.00      A    C  \\nATOM   1325  OH  TYR A 176      11.161   7.425   5.726  1.00  0.00      A    O  \\nATOM   1326  CZ  TYR A 176      10.468   6.360   6.258  1.00  0.00      A    C  \\nATOM   1327  N   CYS A 177       7.922   0.317   9.531  1.00  0.00      A    N  \\nATOM   1328  CA  CYS A 177       7.015  -0.630  10.170  1.00  0.00      A    C  \\nATOM   1329  C   CYS A 177       7.385  -0.837  11.634  1.00  0.00      A    C  \\nATOM   1330  CB  CYS A 177       7.035  -1.971   9.436  1.00  0.00      A    C  \\nATOM   1331  O   CYS A 177       6.507  -0.985  12.486  1.00  0.00      A    O  \\nATOM   1332  SG  CYS A 177       6.211  -1.932   7.828  1.00  0.00      A    S  \\nATOM   1333  N   LYS A 178       8.696  -0.813  11.930  1.00  0.00      A    N  \\nATOM   1334  CA  LYS A 178       9.120  -0.859  13.326  1.00  0.00      A    C  \\nATOM   1335  C   LYS A 178       8.643   0.376  14.086  1.00  0.00      A    C  \\nATOM   1336  CB  LYS A 178      10.642  -0.977  13.422  1.00  0.00      A    C  \\nATOM   1337  O   LYS A 178       8.153   0.267  15.212  1.00  0.00      A    O  \\nATOM   1338  CG  LYS A 178      11.187  -2.328  12.982  1.00  0.00      A    C  \\nATOM   1339  CD  LYS A 178      12.709  -2.356  13.020  1.00  0.00      A    C  \\nATOM   1340  CE  LYS A 178      13.258  -3.680  12.508  1.00  0.00      A    C  \\nATOM   1341  NZ  LYS A 178      14.745  -3.648  12.372  1.00  0.00      A    N  \\nATOM   1342  N   ALA A 179       8.692   1.511  13.477  1.00  0.00      A    N  \\nATOM   1343  CA  ALA A 179       8.271   2.770  14.086  1.00  0.00      A    C  \\nATOM   1344  C   ALA A 179       6.756   2.813  14.264  1.00  0.00      A    C  \\nATOM   1345  CB  ALA A 179       8.738   3.952  13.240  1.00  0.00      A    C  \\nATOM   1346  O   ALA A 179       6.255   3.384  15.235  1.00  0.00      A    O  \\nATOM   1347  N   LEU A 180       6.023   2.240  13.294  1.00  0.00      A    N  \\nATOM   1348  CA  LEU A 180       4.569   2.196  13.408  1.00  0.00      A    C  \\nATOM   1349  C   LEU A 180       4.145   1.410  14.645  1.00  0.00      A    C  \\nATOM   1350  CB  LEU A 180       3.952   1.570  12.155  1.00  0.00      A    C  \\nATOM   1351  O   LEU A 180       3.226   1.818  15.359  1.00  0.00      A    O  \\nATOM   1352  CG  LEU A 180       2.425   1.486  12.120  1.00  0.00      A    C  \\nATOM   1353  CD1 LEU A 180       1.817   2.885  12.147  1.00  0.00      A    C  \\nATOM   1354  CD2 LEU A 180       1.959   0.719  10.888  1.00  0.00      A    C  \\nATOM   1355  N   GLY A 181       4.878   0.316  14.970  1.00  0.00      A    N  \\nATOM   1356  CA  GLY A 181       4.586  -0.507  16.133  1.00  0.00      A    C  \\nATOM   1357  C   GLY A 181       4.732   0.240  17.445  1.00  0.00      A    C  \\nATOM   1358  O   GLY A 181       4.045  -0.069  18.421  1.00  0.00      A    O  \\nATOM   1359  N   SER A 182       5.544   1.277  17.442  1.00  0.00      A    N  \\nATOM   1360  CA  SER A 182       5.797   2.042  18.659  1.00  0.00      A    C  \\nATOM   1361  C   SER A 182       5.301   3.478  18.521  1.00  0.00      A    C  \\nATOM   1362  CB  SER A 182       7.289   2.039  18.994  1.00  0.00      A    C  \\nATOM   1363  O   SER A 182       5.569   4.318  19.383  1.00  0.00      A    O  \\nATOM   1364  OG  SER A 182       8.047   2.571  17.921  1.00  0.00      A    O  \\nATOM   1365  N   ASN A 183       4.565   3.740  17.438  1.00  0.00      A    N  \\nATOM   1366  CA  ASN A 183       4.075   5.087  17.164  1.00  0.00      A    C  \\nATOM   1367  C   ASN A 183       5.191   6.122  17.266  1.00  0.00      A    C  \\nATOM   1368  CB  ASN A 183       2.930   5.445  18.114  1.00  0.00      A    C  \\nATOM   1369  O   ASN A 183       4.980   7.219  17.784  1.00  0.00      A    O  \\nATOM   1370  CG  ASN A 183       2.117   6.629  17.628  1.00  0.00      A    C  \\nATOM   1371  ND2 ASN A 183       1.579   7.403  18.564  1.00  0.00      A    N  \\nATOM   1372  OD1 ASN A 183       1.972   6.845  16.422  1.00  0.00      A    O  \\nATOM   1373  N   ASP A 184       6.302   5.776  16.793  1.00  0.00      A    N  \\nATOM   1374  CA  ASP A 184       7.460   6.666  16.767  1.00  0.00      A    C  \\nATOM   1375  C   ASP A 184       7.458   7.531  15.508  1.00  0.00      A    C  \\nATOM   1376  CB  ASP A 184       8.758   5.860  16.850  1.00  0.00      A    C  \\nATOM   1377  O   ASP A 184       7.907   7.092  14.447  1.00  0.00      A    O  \\nATOM   1378  CG  ASP A 184       9.999   6.734  16.845  1.00  0.00      A    C  \\nATOM   1379  OD1 ASP A 184       9.873   7.974  16.944  1.00  0.00      A    O  \\nATOM   1380  OD2 ASP A 184      11.114   6.179  16.738  1.00  0.00      A    O  \\nATOM   1381  N   LYS A 185       7.086   8.724  15.650  1.00  0.00      A    N  \\nATOM   1382  CA  LYS A 185       6.919   9.620  14.510  1.00  0.00      A    C  \\nATOM   1383  C   LYS A 185       8.215  10.364  14.200  1.00  0.00      A    C  \\nATOM   1384  CB  LYS A 185       5.792  10.620  14.773  1.00  0.00      A    C  \\nATOM   1385  O   LYS A 185       8.264  11.179  13.277  1.00  0.00      A    O  \\nATOM   1386  CG  LYS A 185       4.404   9.996  14.790  1.00  0.00      A    C  \\nATOM   1387  CD  LYS A 185       3.315  11.059  14.865  1.00  0.00      A    C  \\nATOM   1388  CE  LYS A 185       1.924  10.439  14.828  1.00  0.00      A    C  \\nATOM   1389  NZ  LYS A 185       0.853  11.479  14.860  1.00  0.00      A    N  \\nATOM   1390  N   SER A 186       9.242  10.142  14.956  1.00  0.00      A    N  \\nATOM   1391  CA  SER A 186      10.520  10.794  14.691  1.00  0.00      A    C  \\nATOM   1392  C   SER A 186      11.077  10.387  13.331  1.00  0.00      A    C  \\nATOM   1393  CB  SER A 186      11.531  10.458  15.787  1.00  0.00      A    C  \\nATOM   1394  O   SER A 186      11.830  11.140  12.711  1.00  0.00      A    O  \\nATOM   1395  OG  SER A 186      11.937   9.103  15.698  1.00  0.00      A    O  \\nATOM   1396  N   VAL A 187      10.747   9.211  12.831  1.00  0.00      A    N  \\nATOM   1397  CA  VAL A 187      11.242   8.718  11.550  1.00  0.00      A    C  \\nATOM   1398  C   VAL A 187      10.630   9.531  10.412  1.00  0.00      A    C  \\nATOM   1399  CB  VAL A 187      10.929   7.216  11.361  1.00  0.00      A    C  \\nATOM   1400  O   VAL A 187      11.071   9.433   9.264  1.00  0.00      A    O  \\nATOM   1401  CG1 VAL A 187      11.672   6.375  12.398  1.00  0.00      A    C  \\nATOM   1402  CG2 VAL A 187       9.423   6.971  11.449  1.00  0.00      A    C  \\nATOM   1403  N   CYS A 188       9.584  10.310  10.696  1.00  0.00      A    N  \\nATOM   1404  CA  CYS A 188       8.885  11.075   9.670  1.00  0.00      A    C  \\nATOM   1405  C   CYS A 188       9.582  12.406   9.413  1.00  0.00      A    C  \\nATOM   1406  CB  CYS A 188       7.433  11.321  10.080  1.00  0.00      A    C  \\nATOM   1407  O   CYS A 188       9.155  13.179   8.553  1.00  0.00      A    O  \\nATOM   1408  SG  CYS A 188       6.484   9.806  10.338  1.00  0.00      A    S  \\nATOM   1409  N   ASP A 189      10.643  12.658  10.185  1.00  0.00      A    N  \\nATOM   1410  CA  ASP A 189      11.438  13.870  10.008  1.00  0.00      A    C  \\nATOM   1411  C   ASP A 189      12.072  13.913   8.620  1.00  0.00      A    C  \\nATOM   1412  CB  ASP A 189      12.522  13.961  11.084  1.00  0.00      A    C  \\nATOM   1413  O   ASP A 189      12.747  12.966   8.210  1.00  0.00      A    O  \\nATOM   1414  CG  ASP A 189      13.294  15.268  11.038  1.00  0.00      A    C  \\nATOM   1415  OD1 ASP A 189      13.254  15.964  10.001  1.00  0.00      A    O  \\nATOM   1416  OD2 ASP A 189      13.952  15.603  12.047  1.00  0.00      A    O  \\nATOM   1417  N   LYS A 190      11.829  15.019   7.882  1.00  0.00      A    N  \\nATOM   1418  CA  LYS A 190      12.277  15.135   6.497  1.00  0.00      A    C  \\nATOM   1419  C   LYS A 190      13.798  15.061   6.402  1.00  0.00      A    C  \\nATOM   1420  CB  LYS A 190      11.777  16.442   5.879  1.00  0.00      A    C  \\nATOM   1421  O   LYS A 190      14.344  14.727   5.349  1.00  0.00      A    O  \\nATOM   1422  CG  LYS A 190      12.321  17.693   6.552  1.00  0.00      A    C  \\nATOM   1423  CD  LYS A 190      11.780  18.959   5.899  1.00  0.00      A    C  \\nATOM   1424  CE  LYS A 190      12.349  20.212   6.552  1.00  0.00      A    C  \\nATOM   1425  NZ  LYS A 190      11.921  21.452   5.837  1.00  0.00      A    N  \\nATOM   1426  N   ASN A 191      14.531  15.307   7.501  1.00  0.00      A    N  \\nATOM   1427  CA  ASN A 191      15.989  15.339   7.476  1.00  0.00      A    C  \\nATOM   1428  C   ASN A 191      16.586  13.985   7.848  1.00  0.00      A    C  \\nATOM   1429  CB  ASN A 191      16.518  16.429   8.412  1.00  0.00      A    C  \\nATOM   1430  O   ASN A 191      17.803  13.801   7.788  1.00  0.00      A    O  \\nATOM   1431  CG  ASN A 191      16.097  17.821   7.984  1.00  0.00      A    C  \\nATOM   1432  ND2 ASN A 191      15.674  18.634   8.945  1.00  0.00      A    N  \\nATOM   1433  OD1 ASN A 191      16.152  18.163   6.800  1.00  0.00      A    O  \\nATOM   1434  N   LYS A 192      15.776  13.083   8.186  1.00  0.00      A    N  \\nATOM   1435  CA  LYS A 192      16.264  11.762   8.573  1.00  0.00      A    C  \\nATOM   1436  C   LYS A 192      16.327  10.825   7.370  1.00  0.00      A    C  \\nATOM   1437  CB  LYS A 192      15.375  11.159   9.661  1.00  0.00      A    C  \\nATOM   1438  O   LYS A 192      15.433  10.834   6.522  1.00  0.00      A    O  \\nATOM   1439  CG  LYS A 192      15.555  11.795  11.032  1.00  0.00      A    C  \\nATOM   1440  CD  LYS A 192      14.844  10.997  12.117  1.00  0.00      A    C  \\nATOM   1441  CE  LYS A 192      15.012  11.640  13.486  1.00  0.00      A    C  \\nATOM   1442  NZ  LYS A 192      14.302  10.870  14.551  1.00  0.00      A    N  \\nATOM   1443  N   ASN A 193      17.399  10.127   7.224  1.00  0.00      A    N  \\nATOM   1444  CA  ASN A 193      17.519   9.046   6.251  1.00  0.00      A    C  \\nATOM   1445  C   ASN A 193      17.341   7.679   6.907  1.00  0.00      A    C  \\nATOM   1446  CB  ASN A 193      18.868   9.118   5.532  1.00  0.00      A    C  \\nATOM   1447  O   ASN A 193      18.143   7.283   7.754  1.00  0.00      A    O  \\nATOM   1448  CG  ASN A 193      18.926   8.223   4.310  1.00  0.00      A    C  \\nATOM   1449  ND2 ASN A 193      20.136   7.917   3.857  1.00  0.00      A    N  \\nATOM   1450  OD1 ASN A 193      17.893   7.811   3.777  1.00  0.00      A    O  \\nATOM   1451  N   ILE A 194      16.287   7.107   6.571  1.00  0.00      A    N  \\nATOM   1452  CA  ILE A 194      15.952   5.828   7.187  1.00  0.00      A    C  \\nATOM   1453  C   ILE A 194      15.942   4.730   6.126  1.00  0.00      A    C  \\nATOM   1454  CB  ILE A 194      14.586   5.889   7.908  1.00  0.00      A    C  \\nATOM   1455  O   ILE A 194      14.877   4.298   5.681  1.00  0.00      A    O  \\nATOM   1456  CG1 ILE A 194      14.529   7.104   8.842  1.00  0.00      A    C  \\nATOM   1457  CG2 ILE A 194      14.325   4.592   8.680  1.00  0.00      A    C  \\nATOM   1458  CD1 ILE A 194      13.279   7.159   9.709  1.00  0.00      A    C  \\nATOM   1459  N   ALA A 195      17.153   4.353   5.787  1.00  0.00      A    N  \\nATOM   1460  CA  ALA A 195      17.270   3.338   4.744  1.00  0.00      A    C  \\nATOM   1461  C   ALA A 195      17.044   1.940   5.312  1.00  0.00      A    C  \\nATOM   1462  CB  ALA A 195      18.638   3.421   4.072  1.00  0.00      A    C  \\nATOM   1463  O   ALA A 195      17.471   1.640   6.429  1.00  0.00      A    O  \\nATOM   1464  N   GLY A 196      16.251   1.122   4.627  1.00  0.00      A    N  \\nATOM   1465  CA  GLY A 196      16.156  -0.288   4.971  1.00  0.00      A    C  \\nATOM   1466  C   GLY A 196      17.224  -1.137   4.308  1.00  0.00      A    C  \\nATOM   1467  O   GLY A 196      18.208  -0.609   3.785  1.00  0.00      A    O  \\nATOM   1468  N   ASP A 197      17.096  -2.440   4.410  1.00  0.00      A    N  \\nATOM   1469  CA  ASP A 197      18.039  -3.404   3.852  1.00  0.00      A    C  \\nATOM   1470  C   ASP A 197      18.171  -3.232   2.341  1.00  0.00      A    C  \\nATOM   1471  CB  ASP A 197      17.604  -4.834   4.182  1.00  0.00      A    C  \\nATOM   1472  O   ASP A 197      19.172  -3.642   1.749  1.00  0.00      A    O  \\nATOM   1473  CG  ASP A 197      17.753  -5.174   5.655  1.00  0.00      A    C  \\nATOM   1474  OD1 ASP A 197      18.469  -4.449   6.379  1.00  0.00      A    O  \\nATOM   1475  OD2 ASP A 197      17.152  -6.178   6.095  1.00  0.00      A    O  \\nATOM   1476  N   TRP A 198      17.313  -2.627   1.728  1.00  0.00      A    N  \\nATOM   1477  CA  TRP A 198      17.249  -2.395   0.289  1.00  0.00      A    C  \\nATOM   1478  C   TRP A 198      18.066  -1.167  -0.101  1.00  0.00      A    C  \\nATOM   1479  CB  TRP A 198      15.796  -2.222  -0.164  1.00  0.00      A    C  \\nATOM   1480  O   TRP A 198      18.215  -0.865  -1.287  1.00  0.00      A    O  \\nATOM   1481  CG  TRP A 198      14.996  -1.293   0.699  1.00  0.00      A    C  \\nATOM   1482  CD1 TRP A 198      14.954   0.072   0.623  1.00  0.00      A    C  \\nATOM   1483  CD2 TRP A 198      14.126  -1.662   1.773  1.00  0.00      A    C  \\nATOM   1484  CE2 TRP A 198      13.588  -0.468   2.304  1.00  0.00      A    C  \\nATOM   1485  CE3 TRP A 198      13.747  -2.887   2.338  1.00  0.00      A    C  \\nATOM   1486  NE1 TRP A 198      14.109   0.574   1.585  1.00  0.00      A    N  \\nATOM   1487  CH2 TRP A 198      12.336  -1.677   3.908  1.00  0.00      A    C  \\nATOM   1488  CZ2 TRP A 198      12.690  -0.465   3.374  1.00  0.00      A    C  \\nATOM   1489  CZ3 TRP A 198      12.853  -2.882   3.403  1.00  0.00      A    C  \\nATOM   1490  N   GLY A 199      18.615  -0.404   0.794  1.00  0.00      A    N  \\nATOM   1491  CA  GLY A 199      19.670   0.557   0.515  1.00  0.00      A    C  \\nATOM   1492  C   GLY A 199      19.149   1.962   0.276  1.00  0.00      A    C  \\nATOM   1493  O   GLY A 199      19.913   2.861  -0.079  1.00  0.00      A    O  \\nATOM   1494  N   PHE A 200      17.885   2.201   0.368  1.00  0.00      A    N  \\nATOM   1495  CA  PHE A 200      17.351   3.548   0.210  1.00  0.00      A    C  \\nATOM   1496  C   PHE A 200      16.130   3.754   1.100  1.00  0.00      A    C  \\nATOM   1497  CB  PHE A 200      16.984   3.814  -1.253  1.00  0.00      A    C  \\nATOM   1498  O   PHE A 200      15.595   2.795   1.659  1.00  0.00      A    O  \\nATOM   1499  CG  PHE A 200      15.852   2.963  -1.760  1.00  0.00      A    C  \\nATOM   1500  CD1 PHE A 200      16.099   1.719  -2.327  1.00  0.00      A    C  \\nATOM   1501  CD2 PHE A 200      14.539   3.406  -1.669  1.00  0.00      A    C  \\nATOM   1502  CE1 PHE A 200      15.053   0.928  -2.797  1.00  0.00      A    C  \\nATOM   1503  CE2 PHE A 200      13.488   2.622  -2.136  1.00  0.00      A    C  \\nATOM   1504  CZ  PHE A 200      13.748   1.384  -2.701  1.00  0.00      A    C  \\nATOM   1505  N   ASP A 201      15.806   5.005   1.332  1.00  0.00      A    N  \\nATOM   1506  CA  ASP A 201      14.593   5.430   2.023  1.00  0.00      A    C  \\nATOM   1507  C   ASP A 201      13.419   5.544   1.053  1.00  0.00      A    C  \\nATOM   1508  CB  ASP A 201      14.820   6.767   2.733  1.00  0.00      A    C  \\nATOM   1509  O   ASP A 201      13.371   6.464   0.234  1.00  0.00      A    O  \\nATOM   1510  CG  ASP A 201      13.664   7.164   3.633  1.00  0.00      A    C  \\nATOM   1511  OD1 ASP A 201      12.566   6.578   3.512  1.00  0.00      A    O  \\nATOM   1512  OD2 ASP A 201      13.850   8.074   4.470  1.00  0.00      A    O  \\nATOM   1513  N   PRO A 202      12.465   4.604   1.116  1.00  0.00      A    N  \\nATOM   1514  CA  PRO A 202      11.393   4.627   0.119  1.00  0.00      A    C  \\nATOM   1515  C   PRO A 202      10.559   5.904   0.178  1.00  0.00      A    C  \\nATOM   1516  CB  PRO A 202      10.546   3.405   0.484  1.00  0.00      A    C  \\nATOM   1517  O   PRO A 202       9.993   6.325  -0.835  1.00  0.00      A    O  \\nATOM   1518  CG  PRO A 202      10.846   3.148   1.925  1.00  0.00      A    C  \\nATOM   1519  CD  PRO A 202      12.267   3.553   2.196  1.00  0.00      A    C  \\nATOM   1520  N   ALA A 203      10.458   6.530   1.266  1.00  0.00      A    N  \\nATOM   1521  CA  ALA A 203       9.675   7.747   1.466  1.00  0.00      A    C  \\nATOM   1522  C   ALA A 203      10.165   8.869   0.556  1.00  0.00      A    C  \\nATOM   1523  CB  ALA A 203       9.738   8.185   2.927  1.00  0.00      A    C  \\nATOM   1524  O   ALA A 203       9.421   9.807   0.261  1.00  0.00      A    O  \\nATOM   1525  N   LYS A 204      11.348   8.739   0.029  1.00  0.00      A    N  \\nATOM   1526  CA  LYS A 204      11.932   9.791  -0.798  1.00  0.00      A    C  \\nATOM   1527  C   LYS A 204      11.413   9.716  -2.231  1.00  0.00      A    C  \\nATOM   1528  CB  LYS A 204      13.459   9.697  -0.788  1.00  0.00      A    C  \\nATOM   1529  O   LYS A 204      11.674  10.609  -3.039  1.00  0.00      A    O  \\nATOM   1530  CG  LYS A 204      14.093  10.065   0.545  1.00  0.00      A    C  \\nATOM   1531  CD  LYS A 204      15.612  10.123   0.444  1.00  0.00      A    C  \\nATOM   1532  CE  LYS A 204      16.244  10.559   1.759  1.00  0.00      A    C  \\nATOM   1533  NZ  LYS A 204      17.736  10.534   1.691  1.00  0.00      A    N  \\nATOM   1534  N   TRP A 205      10.622   8.734  -2.573  1.00  0.00      A    N  \\nATOM   1535  CA  TRP A 205      10.138   8.530  -3.934  1.00  0.00      A    C  \\nATOM   1536  C   TRP A 205       8.635   8.270  -3.944  1.00  0.00      A    C  \\nATOM   1537  CB  TRP A 205      10.875   7.364  -4.599  1.00  0.00      A    C  \\nATOM   1538  O   TRP A 205       8.139   7.490  -4.761  1.00  0.00      A    O  \\nATOM   1539  CG  TRP A 205      12.309   7.655  -4.923  1.00  0.00      A    C  \\nATOM   1540  CD1 TRP A 205      12.796   8.218  -6.070  1.00  0.00      A    C  \\nATOM   1541  CD2 TRP A 205      13.443   7.394  -4.091  1.00  0.00      A    C  \\nATOM   1542  CE2 TRP A 205      14.588   7.825  -4.797  1.00  0.00      A    C  \\nATOM   1543  CE3 TRP A 205      13.603   6.838  -2.814  1.00  0.00      A    C  \\nATOM   1544  NE1 TRP A 205      14.166   8.322  -6.001  1.00  0.00      A    N  \\nATOM   1545  CH2 TRP A 205      16.004   7.168  -3.018  1.00  0.00      A    C  \\nATOM   1546  CZ2 TRP A 205      15.876   7.716  -4.269  1.00  0.00      A    C  \\nATOM   1547  CZ3 TRP A 205      14.886   6.731  -2.290  1.00  0.00      A    C  \\nATOM   1548  N   ALA A 206       7.992   8.909  -2.961  1.00  0.00      A    N  \\nATOM   1549  CA  ALA A 206       6.545   8.745  -2.848  1.00  0.00      A    C  \\nATOM   1550  C   ALA A 206       5.807   9.809  -3.656  1.00  0.00      A    C  \\nATOM   1551  CB  ALA A 206       6.117   8.801  -1.383  1.00  0.00      A    C  \\nATOM   1552  O   ALA A 206       6.150  10.992  -3.595  1.00  0.00      A    O  \\nATOM   1553  N   TYR A 207       4.797   9.345  -4.368  1.00  0.00      A    N  \\nATOM   1554  CA  TYR A 207       4.013  10.206  -5.245  1.00  0.00      A    C  \\nATOM   1555  C   TYR A 207       2.523   9.915  -5.107  1.00  0.00      A    C  \\nATOM   1556  CB  TYR A 207       4.446  10.027  -6.703  1.00  0.00      A    C  \\nATOM   1557  O   TYR A 207       2.134   8.828  -4.674  1.00  0.00      A    O  \\nATOM   1558  CG  TYR A 207       5.830  10.555  -6.994  1.00  0.00      A    C  \\nATOM   1559  CD1 TYR A 207       6.013  11.826  -7.534  1.00  0.00      A    C  \\nATOM   1560  CD2 TYR A 207       6.957   9.784  -6.731  1.00  0.00      A    C  \\nATOM   1561  CE1 TYR A 207       7.286  12.316  -7.806  1.00  0.00      A    C  \\nATOM   1562  CE2 TYR A 207       8.235  10.264  -6.999  1.00  0.00      A    C  \\nATOM   1563  OH  TYR A 207       9.652  12.009  -7.802  1.00  0.00      A    O  \\nATOM   1564  CZ  TYR A 207       8.389  11.529  -7.535  1.00  0.00      A    C  \\nATOM   1565  N   GLN A 208       1.764  10.927  -5.455  1.00  0.00      A    N  \\nATOM   1566  CA  GLN A 208       0.315  10.798  -5.576  1.00  0.00      A    C  \\nATOM   1567  C   GLN A 208      -0.171  11.296  -6.933  1.00  0.00      A    C  \\nATOM   1568  CB  GLN A 208      -0.387  11.563  -4.453  1.00  0.00      A    C  \\nATOM   1569  O   GLN A 208       0.303  12.320  -7.431  1.00  0.00      A    O  \\nATOM   1570  CG  GLN A 208      -0.284  10.889  -3.091  1.00  0.00      A    C  \\nATOM   1571  CD  GLN A 208      -1.012  11.653  -2.002  1.00  0.00      A    C  \\nATOM   1572  NE2 GLN A 208      -1.724  10.930  -1.144  1.00  0.00      A    N  \\nATOM   1573  OE1 GLN A 208      -0.935  12.884  -1.931  1.00  0.00      A    O  \\nATOM   1574  N   TYR A 209      -1.025  10.501  -7.452  1.00  0.00      A    N  \\nATOM   1575  CA  TYR A 209      -1.558  10.810  -8.775  1.00  0.00      A    C  \\nATOM   1576  C   TYR A 209      -2.685  11.832  -8.683  1.00  0.00      A    C  \\nATOM   1577  CB  TYR A 209      -2.061   9.538  -9.463  1.00  0.00      A    C  \\nATOM   1578  O   TYR A 209      -3.618  11.667  -7.894  1.00  0.00      A    O  \\nATOM   1579  CG  TYR A 209      -2.598   9.773 -10.854  1.00  0.00      A    C  \\nATOM   1580  CD1 TYR A 209      -1.823  10.404 -11.824  1.00  0.00      A    C  \\nATOM   1581  CD2 TYR A 209      -3.882   9.365 -11.201  1.00  0.00      A    C  \\nATOM   1582  CE1 TYR A 209      -2.314  10.624 -13.107  1.00  0.00      A    C  \\nATOM   1583  CE2 TYR A 209      -4.384   9.580 -12.480  1.00  0.00      A    C  \\nATOM   1584  OH  TYR A 209      -4.085  10.424 -14.693  1.00  0.00      A    O  \\nATOM   1585  CZ  TYR A 209      -3.593  10.209 -13.425  1.00  0.00      A    C  \\nATOM   1586  N   ASP A 210      -2.557  12.933  -9.361  1.00  0.00      A    N  \\nATOM   1587  CA  ASP A 210      -3.592  13.937  -9.581  1.00  0.00      A    C  \\nATOM   1588  C   ASP A 210      -4.383  13.642 -10.854  1.00  0.00      A    C  \\nATOM   1589  CB  ASP A 210      -2.976  15.336  -9.657  1.00  0.00      A    C  \\nATOM   1590  O   ASP A 210      -3.968  14.022 -11.951  1.00  0.00      A    O  \\nATOM   1591  CG  ASP A 210      -4.015  16.435  -9.796  1.00  0.00      A    C  \\nATOM   1592  OD1 ASP A 210      -5.187  16.131 -10.107  1.00  0.00      A    O  \\nATOM   1593  OD2 ASP A 210      -3.658  17.616  -9.595  1.00  0.00      A    O  \\nATOM   1594  N   GLU A 211      -5.518  12.983 -10.630  1.00  0.00      A    N  \\nATOM   1595  CA  GLU A 211      -6.327  12.525 -11.756  1.00  0.00      A    C  \\nATOM   1596  C   GLU A 211      -6.855  13.701 -12.571  1.00  0.00      A    C  \\nATOM   1597  CB  GLU A 211      -7.491  11.660 -11.265  1.00  0.00      A    C  \\nATOM   1598  O   GLU A 211      -6.883  13.647 -13.802  1.00  0.00      A    O  \\nATOM   1599  CG  GLU A 211      -8.268  10.982 -12.385  1.00  0.00      A    C  \\nATOM   1600  CD  GLU A 211      -9.360  10.053 -11.880  1.00  0.00      A    C  \\nATOM   1601  OE1 GLU A 211      -9.709  10.122 -10.680  1.00  0.00      A    O  \\nATOM   1602  OE2 GLU A 211      -9.871   9.249 -12.691  1.00  0.00      A    O  \\nATOM   1603  N   LYS A 212      -7.154  14.790 -11.907  1.00  0.00      A    N  \\nATOM   1604  CA  LYS A 212      -7.693  15.967 -12.582  1.00  0.00      A    C  \\nATOM   1605  C   LYS A 212      -6.679  16.553 -13.560  1.00  0.00      A    C  \\nATOM   1606  CB  LYS A 212      -8.110  17.028 -11.562  1.00  0.00      A    C  \\nATOM   1607  O   LYS A 212      -7.031  16.915 -14.685  1.00  0.00      A    O  \\nATOM   1608  CG  LYS A 212      -8.728  18.273 -12.181  1.00  0.00      A    C  \\nATOM   1609  CD  LYS A 212      -9.120  19.290 -11.117  1.00  0.00      A    C  \\nATOM   1610  CE  LYS A 212      -9.662  20.570 -11.739  1.00  0.00      A    C  \\nATOM   1611  NZ  LYS A 212     -10.056  21.567 -10.699  1.00  0.00      A    N  \\nATOM   1612  N   ASN A 213      -5.410  16.618 -13.138  1.00  0.00      A    N  \\nATOM   1613  CA  ASN A 213      -4.378  17.274 -13.934  1.00  0.00      A    C  \\nATOM   1614  C   ASN A 213      -3.456  16.259 -14.603  1.00  0.00      A    C  \\nATOM   1615  CB  ASN A 213      -3.566  18.240 -13.069  1.00  0.00      A    C  \\nATOM   1616  O   ASN A 213      -2.472  16.634 -15.243  1.00  0.00      A    O  \\nATOM   1617  CG  ASN A 213      -4.402  19.382 -12.527  1.00  0.00      A    C  \\nATOM   1618  ND2 ASN A 213      -4.374  19.566 -11.212  1.00  0.00      A    N  \\nATOM   1619  OD1 ASN A 213      -5.070  20.093 -13.283  1.00  0.00      A    O  \\nATOM   1620  N   ASN A 214      -3.718  15.042 -14.476  1.00  0.00      A    N  \\nATOM   1621  CA  ASN A 214      -2.965  13.950 -15.084  1.00  0.00      A    C  \\nATOM   1622  C   ASN A 214      -1.467  14.096 -14.837  1.00  0.00      A    C  \\nATOM   1623  CB  ASN A 214      -3.251  13.871 -16.585  1.00  0.00      A    C  \\nATOM   1624  O   ASN A 214      -0.682  14.178 -15.783  1.00  0.00      A    O  \\nATOM   1625  CG  ASN A 214      -2.746  12.585 -17.209  1.00  0.00      A    C  \\nATOM   1626  ND2 ASN A 214      -2.445  12.631 -18.501  1.00  0.00      A    N  \\nATOM   1627  OD1 ASN A 214      -2.629  11.558 -16.535  1.00  0.00      A    O  \\nATOM   1628  N   LYS A 215      -1.137  14.186 -13.639  1.00  0.00      A    N  \\nATOM   1629  CA  LYS A 215       0.265  14.289 -13.245  1.00  0.00      A    C  \\nATOM   1630  C   LYS A 215       0.497  13.668 -11.871  1.00  0.00      A    C  \\nATOM   1631  CB  LYS A 215       0.716  15.750 -13.243  1.00  0.00      A    C  \\nATOM   1632  O   LYS A 215      -0.448  13.478 -11.102  1.00  0.00      A    O  \\nATOM   1633  CG  LYS A 215       0.095  16.588 -12.135  1.00  0.00      A    C  \\nATOM   1634  CD  LYS A 215       0.588  18.029 -12.181  1.00  0.00      A    C  \\nATOM   1635  CE  LYS A 215       0.043  18.844 -11.016  1.00  0.00      A    C  \\nATOM   1636  NZ  LYS A 215       0.623  20.220 -10.984  1.00  0.00      A    N  \\nATOM   1637  N   PHE A 216       1.700  13.329 -11.593  1.00  0.00      A    N  \\nATOM   1638  CA  PHE A 216       2.093  12.827 -10.281  1.00  0.00      A    C  \\nATOM   1639  C   PHE A 216       2.762  13.923  -9.460  1.00  0.00      A    C  \\nATOM   1640  CB  PHE A 216       3.035  11.627 -10.423  1.00  0.00      A    C  \\nATOM   1641  O   PHE A 216       3.636  14.633  -9.959  1.00  0.00      A    O  \\nATOM   1642  CG  PHE A 216       2.353  10.372 -10.894  1.00  0.00      A    C  \\nATOM   1643  CD1 PHE A 216       1.762   9.502  -9.986  1.00  0.00      A    C  \\nATOM   1644  CD2 PHE A 216       2.303  10.061 -12.247  1.00  0.00      A    C  \\nATOM   1645  CE1 PHE A 216       1.130   8.339 -10.419  1.00  0.00      A    C  \\nATOM   1646  CE2 PHE A 216       1.673   8.900 -12.688  1.00  0.00      A    C  \\nATOM   1647  CZ  PHE A 216       1.088   8.040 -11.772  1.00  0.00      A    C  \\nATOM   1648  N   ASN A 217       2.284  14.047  -8.231  1.00  0.00      A    N  \\nATOM   1649  CA  ASN A 217       2.847  15.008  -7.290  1.00  0.00      A    C  \\nATOM   1650  C   ASN A 217       3.730  14.323  -6.250  1.00  0.00      A    C  \\nATOM   1651  CB  ASN A 217       1.735  15.802  -6.602  1.00  0.00      A    C  \\nATOM   1652  O   ASN A 217       3.340  13.308  -5.671  1.00  0.00      A    O  \\nATOM   1653  CG  ASN A 217       0.902  16.608  -7.579  1.00  0.00      A    C  \\nATOM   1654  ND2 ASN A 217      -0.416  16.543  -7.430  1.00  0.00      A    N  \\nATOM   1655  OD1 ASN A 217       1.439  17.283  -8.462  1.00  0.00      A    O  \\nATOM   1656  N   TYR A 218       4.807  14.853  -6.154  1.00  0.00      A    N  \\nATOM   1657  CA  TYR A 218       5.713  14.377  -5.115  1.00  0.00      A    C  \\nATOM   1658  C   TYR A 218       5.120  14.601  -3.729  1.00  0.00      A    C  \\nATOM   1659  CB  TYR A 218       7.071  15.078  -5.222  1.00  0.00      A    C  \\nATOM   1660  O   TYR A 218       4.710  15.715  -3.396  1.00  0.00      A    O  \\nATOM   1661  CG  TYR A 218       8.058  14.654  -4.162  1.00  0.00      A    C  \\nATOM   1662  CD1 TYR A 218       8.635  13.386  -4.187  1.00  0.00      A    C  \\nATOM   1663  CD2 TYR A 218       8.415  15.519  -3.133  1.00  0.00      A    C  \\nATOM   1664  CE1 TYR A 218       9.544  12.990  -3.212  1.00  0.00      A    C  \\nATOM   1665  CE2 TYR A 218       9.323  15.133  -2.152  1.00  0.00      A    C  \\nATOM   1666  OH  TYR A 218      10.781  13.483  -1.231  1.00  0.00      A    O  \\nATOM   1667  CZ  TYR A 218       9.882  13.869  -2.200  1.00  0.00      A    C  \\nATOM   1668  N   VAL A 219       5.051  13.533  -2.947  1.00  0.00      A    N  \\nATOM   1669  CA  VAL A 219       4.528  13.647  -1.590  1.00  0.00      A    C  \\nATOM   1670  C   VAL A 219       5.510  13.018  -0.604  1.00  0.00      A    C  \\nATOM   1671  CB  VAL A 219       3.140  12.982  -1.458  1.00  0.00      A    C  \\nATOM   1672  O   VAL A 219       5.145  12.712   0.534  1.00  0.00      A    O  \\nATOM   1673  CG1 VAL A 219       2.089  13.765  -2.243  1.00  0.00      A    C  \\nATOM   1674  CG2 VAL A 219       3.197  11.532  -1.935  1.00  0.00      A    C  \\nATOM   1675  N   GLY A 220       6.713  12.703  -0.999  1.00  0.00      A    N  \\nATOM   1676  CA  GLY A 220       7.729  12.132  -0.130  1.00  0.00      A    C  \\nATOM   1677  C   GLY A 220       8.204  13.092   0.945  1.00  0.00      A    C  \\nATOM   1678  O   GLY A 220       7.593  14.140   1.161  1.00  0.00      A    O  \\nATOM   1679  N   LYS A 221       9.212  12.562   1.749  1.00  0.00      A    N  \\nATOM   1680  CA  LYS A 221       9.723  13.369   2.853  1.00  0.00      A    C  \\nATOM   1681  C   LYS A 221      10.707  14.423   2.353  1.00  0.00      A    C  \\nATOM   1682  CB  LYS A 221      10.395  12.481   3.901  1.00  0.00      A    C  \\nATOM   1683  O   LYS A 221      11.328  14.252   1.302  1.00  0.00      A    O  \\nATOM   1684  CG  LYS A 221       9.419  11.690   4.759  1.00  0.00      A    C  \\nATOM   1685  CD  LYS A 221      10.123  11.016   5.930  1.00  0.00      A    C  \\nATOM   1686  CE  LYS A 221       9.129  10.358   6.877  1.00  0.00      A    C  \\nATOM   1687  NZ  LYS A 221       9.791   9.870   8.124  1.00  0.00      A    N  \\nATOM   1688  OXT LYS A 221      10.423  15.061   3.339  1.00  0.00      A    O  \\nCONECT  338 1332\\nCONECT  379 1408\\nCONECT 1332  338\\nCONECT 1408  379\\nEND\\n\",\"pdb\");\n\tviewer_1678119207039042.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1678119207039042.zoomTo();\nviewer_1678119207039042.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1678119207039042\"  style=\"position: relative; width: 640px; height: 480px\">\n",
              "        <p id=\"3dmolwarning_1678119207039042\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_1678119207039042 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1678119207039042\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1678119207039042 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1678119207039042\"),{backgroundColor:\"white\"});\n",
              "viewer_1678119207039042.zoomTo();\n",
              "\tviewer_1678119207039042.addModel(\"HEADER    \\nTITLE     MDANALYSIS FRAME 0: Created by PDBWriter\\nREMARK     1 CREATED WITH OPENMM 8.0, 2023-03-06\\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1\\nREMARK     285 UNITARY VALUES FOR THE UNIT CELL AUTOMATICALLY SET\\nREMARK     285 BY MDANALYSIS PDBWRITER BECAUSE UNIT CELL INFORMATION\\nREMARK     285 WAS MISSING.\\nREMARK     285 PROTEIN DATA BANK CONVENTIONS REQUIRE THAT\\nREMARK     285 CRYST1 RECORD IS INCLUDED, BUT THE VALUES ON\\nREMARK     285 THIS RECORD ARE MEANINGLESS.\\nATOM      1  N   VAL A   1       6.677 -18.759 -21.233  1.00  0.00      A    N  \\nATOM      2  CA  VAL A   1       7.160 -17.735 -20.313  1.00  0.00      A    C  \\nATOM      3  C   VAL A   1       6.887 -18.164 -18.873  1.00  0.00      A    C  \\nATOM      4  CB  VAL A   1       6.506 -16.364 -20.594  1.00  0.00      A    C  \\nATOM      5  O   VAL A   1       5.747 -18.466 -18.514  1.00  0.00      A    O  \\nATOM      6  CG1 VAL A   1       7.193 -15.262 -19.790  1.00  0.00      A    C  \\nATOM      7  CG2 VAL A   1       6.552 -16.048 -22.088  1.00  0.00      A    C  \\nATOM      8  N   PRO A   2       7.887 -18.889 -18.155  1.00  0.00      A    N  \\nATOM      9  CA  PRO A   2       7.602 -19.467 -16.839  1.00  0.00      A    C  \\nATOM     10  C   PRO A   2       6.430 -18.785 -16.137  1.00  0.00      A    C  \\nATOM     11  CB  PRO A   2       8.905 -19.245 -16.067  1.00  0.00      A    C  \\nATOM     12  O   PRO A   2       6.338 -17.555 -16.133  1.00  0.00      A    O  \\nATOM     13  CG  PRO A   2       9.713 -18.335 -16.935  1.00  0.00      A    C  \\nATOM     14  CD  PRO A   2       9.016 -18.193 -18.258  1.00  0.00      A    C  \\nATOM     15  N   VAL A   3       5.124 -19.345 -16.235  1.00  0.00      A    N  \\nATOM     16  CA  VAL A   3       4.067 -19.145 -15.250  1.00  0.00      A    C  \\nATOM     17  C   VAL A   3       4.665 -18.606 -13.952  1.00  0.00      A    C  \\nATOM     18  CB  VAL A   3       3.292 -20.453 -14.976  1.00  0.00      A    C  \\nATOM     19  O   VAL A   3       5.626 -19.170 -13.423  1.00  0.00      A    O  \\nATOM     20  CG1 VAL A   3       1.798 -20.174 -14.823  1.00  0.00      A    C  \\nATOM     21  CG2 VAL A   3       3.538 -21.463 -16.096  1.00  0.00      A    C  \\nATOM     22  N   ASN A   4       4.985 -17.260 -13.753  1.00  0.00      A    N  \\nATOM     23  CA  ASN A   4       5.411 -16.807 -12.433  1.00  0.00      A    C  \\nATOM     24  C   ASN A   4       4.919 -17.743 -11.333  1.00  0.00      A    C  \\nATOM     25  CB  ASN A   4       4.924 -15.380 -12.173  1.00  0.00      A    C  \\nATOM     26  O   ASN A   4       3.740 -18.100 -11.297  1.00  0.00      A    O  \\nATOM     27  CG  ASN A   4       6.058 -14.374 -12.133  1.00  0.00      A    C  \\nATOM     28  ND2 ASN A   4       5.719 -13.094 -12.218  1.00  0.00      A    N  \\nATOM     29  OD1 ASN A   4       7.230 -14.746 -12.027  1.00  0.00      A    O  \\nATOM     30  N   PRO A   5       5.604 -18.831 -10.877  1.00  0.00      A    N  \\nATOM     31  CA  PRO A   5       4.976 -19.577  -9.785  1.00  0.00      A    C  \\nATOM     32  C   PRO A   5       4.015 -18.722  -8.962  1.00  0.00      A    C  \\nATOM     33  CB  PRO A   5       6.167 -20.031  -8.937  1.00  0.00      A    C  \\nATOM     34  O   PRO A   5       4.305 -17.556  -8.684  1.00  0.00      A    O  \\nATOM     35  CG  PRO A   5       7.317 -19.209  -9.423  1.00  0.00      A    C  \\nATOM     36  CD  PRO A   5       6.924 -18.557 -10.718  1.00  0.00      A    C  \\nATOM     37  N   GLU A   6       2.648 -18.911  -9.065  1.00  0.00      A    N  \\nATOM     38  CA  GLU A   6       1.752 -18.443  -8.012  1.00  0.00      A    C  \\nATOM     39  C   GLU A   6       2.488 -18.308  -6.682  1.00  0.00      A    C  \\nATOM     40  CB  GLU A   6       0.560 -19.390  -7.860  1.00  0.00      A    C  \\nATOM     41  O   GLU A   6       3.291 -19.170  -6.320  1.00  0.00      A    O  \\nATOM     42  CG  GLU A   6      -0.504 -19.218  -8.935  1.00  0.00      A    C  \\nATOM     43  CD  GLU A   6      -1.851 -19.804  -8.545  1.00  0.00      A    C  \\nATOM     44  OE1 GLU A   6      -1.902 -20.647  -7.621  1.00  0.00      A    O  \\nATOM     45  OE2 GLU A   6      -2.865 -19.418  -9.169  1.00  0.00      A    O  \\nATOM     46  N   PRO A   7       2.901 -17.078  -6.204  1.00  0.00      A    N  \\nATOM     47  CA  PRO A   7       3.474 -17.018  -4.857  1.00  0.00      A    C  \\nATOM     48  C   PRO A   7       3.026 -18.180  -3.973  1.00  0.00      A    C  \\nATOM     49  CB  PRO A   7       2.953 -15.686  -4.312  1.00  0.00      A    C  \\nATOM     50  O   PRO A   7       1.846 -18.540  -3.971  1.00  0.00      A    O  \\nATOM     51  CG  PRO A   7       2.022 -15.180  -5.366  1.00  0.00      A    C  \\nATOM     52  CD  PRO A   7       2.035 -16.144  -6.517  1.00  0.00      A    C  \\nATOM     53  N   ASP A   8       3.786 -19.312  -3.767  1.00  0.00      A    N  \\nATOM     54  CA  ASP A   8       3.546 -20.366  -2.786  1.00  0.00      A    C  \\nATOM     55  C   ASP A   8       2.798 -19.825  -1.570  1.00  0.00      A    C  \\nATOM     56  CB  ASP A   8       4.867 -21.005  -2.350  1.00  0.00      A    C  \\nATOM     57  O   ASP A   8       3.302 -18.951  -0.863  1.00  0.00      A    O  \\nATOM     58  CG  ASP A   8       4.931 -22.493  -2.642  1.00  0.00      A    C  \\nATOM     59  OD1 ASP A   8       3.887 -23.095  -2.975  1.00  0.00      A    O  \\nATOM     60  OD2 ASP A   8       6.034 -23.071  -2.534  1.00  0.00      A    O  \\nATOM     61  N   ALA A   9       1.543 -19.409  -1.662  1.00  0.00      A    N  \\nATOM     62  CA  ALA A   9       0.564 -19.236  -0.592  1.00  0.00      A    C  \\nATOM     63  C   ALA A   9       0.739 -20.299   0.489  1.00  0.00      A    C  \\nATOM     64  CB  ALA A   9      -0.854 -19.285  -1.157  1.00  0.00      A    C  \\nATOM     65  O   ALA A   9      -0.006 -20.323   1.471  1.00  0.00      A    O  \\nATOM     66  N   THR A  10       1.773 -21.205   0.502  1.00  0.00      A    N  \\nATOM     67  CA  THR A  10       1.715 -22.406   1.328  1.00  0.00      A    C  \\nATOM     68  C   THR A  10       2.573 -22.244   2.579  1.00  0.00      A    C  \\nATOM     69  CB  THR A  10       2.179 -23.647   0.542  1.00  0.00      A    C  \\nATOM     70  O   THR A  10       3.792 -22.087   2.487  1.00  0.00      A    O  \\nATOM     71  CG2 THR A  10       1.002 -24.338  -0.140  1.00  0.00      A    C  \\nATOM     72  OG1 THR A  10       3.127 -23.247  -0.455  1.00  0.00      A    O  \\nATOM     73  N   SER A  11       2.463 -21.104   3.386  1.00  0.00      A    N  \\nATOM     74  CA  SER A  11       2.650 -21.248   4.826  1.00  0.00      A    C  \\nATOM     75  C   SER A  11       2.516 -19.905   5.538  1.00  0.00      A    C  \\nATOM     76  CB  SER A  11       4.018 -21.860   5.131  1.00  0.00      A    C  \\nATOM     77  O   SER A  11       3.420 -19.070   5.476  1.00  0.00      A    O  \\nATOM     78  OG  SER A  11       5.059 -20.960   4.792  1.00  0.00      A    O  \\nATOM     79  N   VAL A  12       1.416 -19.239   5.242  1.00  0.00      A    N  \\nATOM     80  CA  VAL A  12       1.291 -18.070   6.105  1.00  0.00      A    C  \\nATOM     81  C   VAL A  12       0.733 -18.488   7.464  1.00  0.00      A    C  \\nATOM     82  CB  VAL A  12       0.389 -16.988   5.469  1.00  0.00      A    C  \\nATOM     83  O   VAL A  12      -0.481 -18.628   7.627  1.00  0.00      A    O  \\nATOM     84  CG1 VAL A  12       0.581 -15.644   6.169  1.00  0.00      A    C  \\nATOM     85  CG2 VAL A  12       0.681 -16.863   3.975  1.00  0.00      A    C  \\nATOM     86  N   GLU A  13       1.308 -19.626   8.009  1.00  0.00      A    N  \\nATOM     87  CA  GLU A  13       1.113 -20.129   9.366  1.00  0.00      A    C  \\nATOM     88  C   GLU A  13       1.807 -19.237  10.391  1.00  0.00      A    C  \\nATOM     89  CB  GLU A  13       1.626 -21.566   9.486  1.00  0.00      A    C  \\nATOM     90  O   GLU A  13       3.027 -19.067  10.349  1.00  0.00      A    O  \\nATOM     91  CG  GLU A  13       0.617 -22.619   9.052  1.00  0.00      A    C  \\nATOM     92  CD  GLU A  13       1.124 -24.042   9.220  1.00  0.00      A    C  \\nATOM     93  OE1 GLU A  13       2.327 -24.227   9.511  1.00  0.00      A    O  \\nATOM     94  OE2 GLU A  13       0.311 -24.980   9.058  1.00  0.00      A    O  \\nATOM     95  N   ASN A  14       1.125 -17.939  10.595  1.00  0.00      A    N  \\nATOM     96  CA  ASN A  14       0.880 -17.228  11.846  1.00  0.00      A    C  \\nATOM     97  C   ASN A  14       0.325 -15.829  11.596  1.00  0.00      A    C  \\nATOM     98  CB  ASN A  14       2.162 -17.150  12.678  1.00  0.00      A    C  \\nATOM     99  O   ASN A  14       1.086 -14.870  11.455  1.00  0.00      A    O  \\nATOM    100  CG  ASN A  14       2.444 -18.431  13.437  1.00  0.00      A    C  \\nATOM    101  ND2 ASN A  14       3.718 -18.688  13.713  1.00  0.00      A    N  \\nATOM    102  OD1 ASN A  14       1.526 -19.184  13.772  1.00  0.00      A    O  \\nATOM    103  N   VAL A  15      -0.813 -15.747  10.890  1.00  0.00      A    N  \\nATOM    104  CA  VAL A  15      -1.725 -14.617  11.033  1.00  0.00      A    C  \\nATOM    105  C   VAL A  15      -2.295 -14.588  12.450  1.00  0.00      A    C  \\nATOM    106  CB  VAL A  15      -2.872 -14.679   9.999  1.00  0.00      A    C  \\nATOM    107  O   VAL A  15      -2.965 -15.532  12.875  1.00  0.00      A    O  \\nATOM    108  CG1 VAL A  15      -3.739 -13.424  10.079  1.00  0.00      A    C  \\nATOM    109  CG2 VAL A  15      -2.310 -14.856   8.590  1.00  0.00      A    C  \\nATOM    110  N   ALA A  16      -1.421 -14.155  13.540  1.00  0.00      A    N  \\nATOM    111  CA  ALA A  16      -1.832 -13.876  14.913  1.00  0.00      A    C  \\nATOM    112  C   ALA A  16      -3.349 -13.737  15.015  1.00  0.00      A    C  \\nATOM    113  CB  ALA A  16      -1.150 -12.610  15.426  1.00  0.00      A    C  \\nATOM    114  O   ALA A  16      -4.014 -13.394  14.034  1.00  0.00      A    O  \\nATOM    115  N   LEU A  17      -3.938 -14.397  16.181  1.00  0.00      A    N  \\nATOM    116  CA  LEU A  17      -5.269 -14.381  16.778  1.00  0.00      A    C  \\nATOM    117  C   LEU A  17      -6.032 -13.124  16.372  1.00  0.00      A    C  \\nATOM    118  CB  LEU A  17      -5.174 -14.464  18.303  1.00  0.00      A    C  \\nATOM    119  O   LEU A  17      -5.527 -12.009  16.528  1.00  0.00      A    O  \\nATOM    120  CG  LEU A  17      -4.900 -15.849  18.891  1.00  0.00      A    C  \\nATOM    121  CD1 LEU A  17      -4.151 -15.725  20.214  1.00  0.00      A    C  \\nATOM    122  CD2 LEU A  17      -6.204 -16.618  19.079  1.00  0.00      A    C  \\nATOM    123  N   LYS A  18      -6.849 -13.306  15.469  1.00  0.00      A    N  \\nATOM    124  CA  LYS A  18      -7.897 -12.358  15.101  1.00  0.00      A    C  \\nATOM    125  C   LYS A  18      -8.666 -11.884  16.331  1.00  0.00      A    C  \\nATOM    126  CB  LYS A  18      -8.859 -12.986  14.091  1.00  0.00      A    C  \\nATOM    127  O   LYS A  18      -9.026 -12.689  17.193  1.00  0.00      A    O  \\nATOM    128  CG  LYS A  18      -8.188 -13.461  12.811  1.00  0.00      A    C  \\nATOM    129  CD  LYS A  18      -9.203 -14.022  11.823  1.00  0.00      A    C  \\nATOM    130  CE  LYS A  18      -8.530 -14.523  10.552  1.00  0.00      A    C  \\nATOM    131  NZ  LYS A  18      -9.515 -15.122   9.603  1.00  0.00      A    N  \\nATOM    132  N   THR A  19      -8.580 -10.539  16.522  1.00  0.00      A    N  \\nATOM    133  CA  THR A  19      -9.567  -9.875  17.366  1.00  0.00      A    C  \\nATOM    134  C   THR A  19     -10.477  -8.978  16.532  1.00  0.00      A    C  \\nATOM    135  CB  THR A  19      -8.887  -9.039  18.467  1.00  0.00      A    C  \\nATOM    136  O   THR A  19     -10.058  -8.445  15.503  1.00  0.00      A    O  \\nATOM    137  CG2 THR A  19      -7.966  -9.902  19.323  1.00  0.00      A    C  \\nATOM    138  OG1 THR A  19      -8.118  -7.994  17.860  1.00  0.00      A    O  \\nATOM    139  N   GLY A  20     -11.714  -9.163  16.753  1.00  0.00      A    N  \\nATOM    140  CA  GLY A  20     -12.765  -8.414  16.083  1.00  0.00      A    C  \\nATOM    141  C   GLY A  20     -13.643  -9.280  15.199  1.00  0.00      A    C  \\nATOM    142  O   GLY A  20     -13.266 -10.399  14.845  1.00  0.00      A    O  \\nATOM    143  N   SER A  21     -14.824  -8.725  14.874  1.00  0.00      A    N  \\nATOM    144  CA  SER A  21     -15.775  -9.449  14.036  1.00  0.00      A    C  \\nATOM    145  C   SER A  21     -15.572  -9.122  12.561  1.00  0.00      A    C  \\nATOM    146  CB  SER A  21     -17.211  -9.121  14.447  1.00  0.00      A    C  \\nATOM    147  O   SER A  21     -15.950  -9.907  11.688  1.00  0.00      A    O  \\nATOM    148  OG  SER A  21     -17.506  -7.757  14.197  1.00  0.00      A    O  \\nATOM    149  N   GLY A  22     -14.927  -7.923  12.309  1.00  0.00      A    N  \\nATOM    150  CA  GLY A  22     -14.801  -7.471  10.932  1.00  0.00      A    C  \\nATOM    151  C   GLY A  22     -16.100  -6.935  10.359  1.00  0.00      A    C  \\nATOM    152  O   GLY A  22     -16.209  -6.721   9.150  1.00  0.00      A    O  \\nATOM    153  N   ASP A  23     -17.174  -6.705  11.156  1.00  0.00      A    N  \\nATOM    154  CA  ASP A  23     -18.513  -6.372  10.680  1.00  0.00      A    C  \\nATOM    155  C   ASP A  23     -18.656  -4.868  10.452  1.00  0.00      A    C  \\nATOM    156  CB  ASP A  23     -19.572  -6.856  11.672  1.00  0.00      A    C  \\nATOM    157  O   ASP A  23     -19.568  -4.425   9.752  1.00  0.00      A    O  \\nATOM    158  CG  ASP A  23     -19.741  -8.366  11.669  1.00  0.00      A    C  \\nATOM    159  OD1 ASP A  23     -19.349  -9.021  10.679  1.00  0.00      A    O  \\nATOM    160  OD2 ASP A  23     -20.272  -8.904  12.664  1.00  0.00      A    O  \\nATOM    161  N   SER A  24     -17.767  -4.143  11.076  1.00  0.00      A    N  \\nATOM    162  CA  SER A  24     -17.849  -2.689  10.983  1.00  0.00      A    C  \\nATOM    163  C   SER A  24     -16.464  -2.053  11.038  1.00  0.00      A    C  \\nATOM    164  CB  SER A  24     -18.721  -2.128  12.106  1.00  0.00      A    C  \\nATOM    165  O   SER A  24     -15.502  -2.685  11.479  1.00  0.00      A    O  \\nATOM    166  OG  SER A  24     -18.082  -2.275  13.362  1.00  0.00      A    O  \\nATOM    167  N   GLN A  25     -16.431  -0.819  10.559  1.00  0.00      A    N  \\nATOM    168  CA  GLN A  25     -15.195  -0.045  10.609  1.00  0.00      A    C  \\nATOM    169  C   GLN A  25     -14.693   0.098  12.043  1.00  0.00      A    C  \\nATOM    170  CB  GLN A  25     -15.399   1.336   9.984  1.00  0.00      A    C  \\nATOM    171  O   GLN A  25     -13.485   0.108  12.286  1.00  0.00      A    O  \\nATOM    172  CG  GLN A  25     -14.127   2.169   9.904  1.00  0.00      A    C  \\nATOM    173  CD  GLN A  25     -14.336   3.494   9.195  1.00  0.00      A    C  \\nATOM    174  NE2 GLN A  25     -13.261   4.258   9.037  1.00  0.00      A    N  \\nATOM    175  OE1 GLN A  25     -15.455   3.827   8.791  1.00  0.00      A    O  \\nATOM    176  N   SER A  26     -15.610   0.109  13.017  1.00  0.00      A    N  \\nATOM    177  CA  SER A  26     -15.251   0.301  14.418  1.00  0.00      A    C  \\nATOM    178  C   SER A  26     -14.862  -1.020  15.075  1.00  0.00      A    C  \\nATOM    179  CB  SER A  26     -16.408   0.940  15.186  1.00  0.00      A    C  \\nATOM    180  O   SER A  26     -14.355  -1.035  16.199  1.00  0.00      A    O  \\nATOM    181  OG  SER A  26     -17.574   0.139  15.098  1.00  0.00      A    O  \\nATOM    182  N   ASP A  27     -15.188  -2.142  14.440  1.00  0.00      A    N  \\nATOM    183  CA  ASP A  27     -14.835  -3.452  14.979  1.00  0.00      A    C  \\nATOM    184  C   ASP A  27     -14.157  -4.317  13.920  1.00  0.00      A    C  \\nATOM    185  CB  ASP A  27     -16.078  -4.161  15.521  1.00  0.00      A    C  \\nATOM    186  O   ASP A  27     -14.639  -5.406  13.598  1.00  0.00      A    O  \\nATOM    187  CG  ASP A  27     -15.751  -5.445  16.264  1.00  0.00      A    C  \\nATOM    188  OD1 ASP A  27     -14.648  -5.551  16.842  1.00  0.00      A    O  \\nATOM    189  OD2 ASP A  27     -16.604  -6.359  16.269  1.00  0.00      A    O  \\nATOM    190  N   PRO A  28     -12.994  -3.883  13.405  1.00  0.00      A    N  \\nATOM    191  CA  PRO A  28     -12.300  -4.632  12.355  1.00  0.00      A    C  \\nATOM    192  C   PRO A  28     -11.565  -5.858  12.891  1.00  0.00      A    C  \\nATOM    193  CB  PRO A  28     -11.313  -3.609  11.786  1.00  0.00      A    C  \\nATOM    194  O   PRO A  28     -11.270  -5.931  14.087  1.00  0.00      A    O  \\nATOM    195  CG  PRO A  28     -10.974  -2.723  12.941  1.00  0.00      A    C  \\nATOM    196  CD  PRO A  28     -12.199  -2.561  13.794  1.00  0.00      A    C  \\nATOM    197  N   ILE A  29     -11.365  -6.795  12.005  1.00  0.00      A    N  \\nATOM    198  CA  ILE A  29     -10.402  -7.845  12.315  1.00  0.00      A    C  \\nATOM    199  C   ILE A  29      -9.010  -7.238  12.479  1.00  0.00      A    C  \\nATOM    200  CB  ILE A  29     -10.385  -8.938  11.222  1.00  0.00      A    C  \\nATOM    201  O   ILE A  29      -8.504  -6.576  11.569  1.00  0.00      A    O  \\nATOM    202  CG1 ILE A  29     -11.700  -9.725  11.234  1.00  0.00      A    C  \\nATOM    203  CG2 ILE A  29      -9.186  -9.871  11.411  1.00  0.00      A    C  \\nATOM    204  CD1 ILE A  29     -11.846 -10.703  10.077  1.00  0.00      A    C  \\nATOM    205  N   LYS A  30      -8.452  -7.409  13.630  1.00  0.00      A    N  \\nATOM    206  CA  LYS A  30      -7.108  -6.910  13.906  1.00  0.00      A    C  \\nATOM    207  C   LYS A  30      -6.087  -8.044  13.898  1.00  0.00      A    C  \\nATOM    208  CB  LYS A  30      -7.071  -6.181  15.250  1.00  0.00      A    C  \\nATOM    209  O   LYS A  30      -6.301  -9.083  14.526  1.00  0.00      A    O  \\nATOM    210  CG  LYS A  30      -7.932  -4.928  15.299  1.00  0.00      A    C  \\nATOM    211  CD  LYS A  30      -7.845  -4.241  16.656  1.00  0.00      A    C  \\nATOM    212  CE  LYS A  30      -8.744  -3.014  16.721  1.00  0.00      A    C  \\nATOM    213  NZ  LYS A  30      -8.718  -2.380  18.073  1.00  0.00      A    N  \\nATOM    214  N   ALA A  31      -5.108  -7.861  13.045  1.00  0.00      A    N  \\nATOM    215  CA  ALA A  31      -4.114  -8.924  12.922  1.00  0.00      A    C  \\nATOM    216  C   ALA A  31      -2.708  -8.347  12.781  1.00  0.00      A    C  \\nATOM    217  CB  ALA A  31      -4.440  -9.821  11.730  1.00  0.00      A    C  \\nATOM    218  O   ALA A  31      -2.537  -7.222  12.306  1.00  0.00      A    O  \\nATOM    219  N   ASP A  32      -1.752  -9.183  13.284  1.00  0.00      A    N  \\nATOM    220  CA  ASP A  32      -0.340  -8.956  12.995  1.00  0.00      A    C  \\nATOM    221  C   ASP A  32       0.076  -9.653  11.702  1.00  0.00      A    C  \\nATOM    222  CB  ASP A  32       0.530  -9.442  14.157  1.00  0.00      A    C  \\nATOM    223  O   ASP A  32      -0.346 -10.780  11.435  1.00  0.00      A    O  \\nATOM    224  CG  ASP A  32       0.288  -8.669  15.442  1.00  0.00      A    C  \\nATOM    225  OD1 ASP A  32      -0.348  -7.594  15.397  1.00  0.00      A    O  \\nATOM    226  OD2 ASP A  32       0.741  -9.138  16.509  1.00  0.00      A    O  \\nATOM    227  N   LEU A  33       0.884  -8.920  10.867  1.00  0.00      A    N  \\nATOM    228  CA  LEU A  33       1.394  -9.515   9.636  1.00  0.00      A    C  \\nATOM    229  C   LEU A  33       2.884  -9.814   9.754  1.00  0.00      A    C  \\nATOM    230  CB  LEU A  33       1.141  -8.586   8.446  1.00  0.00      A    C  \\nATOM    231  O   LEU A  33       3.665  -8.950  10.158  1.00  0.00      A    O  \\nATOM    232  CG  LEU A  33      -0.295  -8.529   7.923  1.00  0.00      A    C  \\nATOM    233  CD1 LEU A  33      -0.398  -7.546   6.762  1.00  0.00      A    C  \\nATOM    234  CD2 LEU A  33      -0.765  -9.917   7.498  1.00  0.00      A    C  \\nATOM    235  N   GLU A  34       3.227 -11.009   9.491  1.00  0.00      A    N  \\nATOM    236  CA  GLU A  34       4.643 -11.365   9.477  1.00  0.00      A    C  \\nATOM    237  C   GLU A  34       5.286 -11.021   8.137  1.00  0.00      A    C  \\nATOM    238  CB  GLU A  34       4.826 -12.854   9.782  1.00  0.00      A    C  \\nATOM    239  O   GLU A  34       4.843 -11.497   7.090  1.00  0.00      A    O  \\nATOM    240  CG  GLU A  34       6.279 -13.274   9.945  1.00  0.00      A    C  \\nATOM    241  CD  GLU A  34       6.446 -14.758  10.235  1.00  0.00      A    C  \\nATOM    242  OE1 GLU A  34       5.425 -15.474  10.339  1.00  0.00      A    O  \\nATOM    243  OE2 GLU A  34       7.607 -15.206  10.361  1.00  0.00      A    O  \\nATOM    244  N   VAL A  35       6.237 -10.098   8.183  1.00  0.00      A    N  \\nATOM    245  CA  VAL A  35       7.013  -9.712   7.009  1.00  0.00      A    C  \\nATOM    246  C   VAL A  35       8.035 -10.799   6.684  1.00  0.00      A    C  \\nATOM    247  CB  VAL A  35       7.724  -8.357   7.218  1.00  0.00      A    C  \\nATOM    248  O   VAL A  35       8.828 -11.191   7.544  1.00  0.00      A    O  \\nATOM    249  CG1 VAL A  35       8.544  -7.981   5.985  1.00  0.00      A    C  \\nATOM    250  CG2 VAL A  35       6.707  -7.265   7.541  1.00  0.00      A    C  \\nATOM    251  N   LYS A  36       7.965 -11.277   5.524  1.00  0.00      A    N  \\nATOM    252  CA  LYS A  36       8.834 -12.347   5.040  1.00  0.00      A    C  \\nATOM    253  C   LYS A  36       9.431 -11.998   3.680  1.00  0.00      A    C  \\nATOM    254  CB  LYS A  36       8.064 -13.665   4.952  1.00  0.00      A    C  \\nATOM    255  O   LYS A  36       8.852 -11.215   2.924  1.00  0.00      A    O  \\nATOM    256  CG  LYS A  36       7.528 -14.160   6.287  1.00  0.00      A    C  \\nATOM    257  CD  LYS A  36       6.806 -15.493   6.140  1.00  0.00      A    C  \\nATOM    258  CE  LYS A  36       6.246 -15.976   7.471  1.00  0.00      A    C  \\nATOM    259  NZ  LYS A  36       5.523 -17.275   7.329  1.00  0.00      A    N  \\nATOM    260  N   GLY A  37      10.556 -12.709   3.510  1.00  0.00      A    N  \\nATOM    261  CA  GLY A  37      11.170 -12.512   2.206  1.00  0.00      A    C  \\nATOM    262  C   GLY A  37      11.474 -11.057   1.904  1.00  0.00      A    C  \\nATOM    263  O   GLY A  37      12.025 -10.344   2.745  1.00  0.00      A    O  \\nATOM    264  N   GLN A  38      11.155 -10.654   0.753  1.00  0.00      A    N  \\nATOM    265  CA  GLN A  38      11.490  -9.311   0.291  1.00  0.00      A    C  \\nATOM    266  C   GLN A  38      10.305  -8.363   0.449  1.00  0.00      A    C  \\nATOM    267  CB  GLN A  38      11.946  -9.343  -1.169  1.00  0.00      A    C  \\nATOM    268  O   GLN A  38      10.309  -7.258  -0.098  1.00  0.00      A    O  \\nATOM    269  CG  GLN A  38      13.248 -10.100  -1.390  1.00  0.00      A    C  \\nATOM    270  CD  GLN A  38      13.750 -10.000  -2.818  1.00  0.00      A    C  \\nATOM    271  NE2 GLN A  38      14.839 -10.702  -3.115  1.00  0.00      A    N  \\nATOM    272  OE1 GLN A  38      13.164  -9.298  -3.648  1.00  0.00      A    O  \\nATOM    273  N   SER A  39       9.252  -8.827   1.173  1.00  0.00      A    N  \\nATOM    274  CA  SER A  39       8.037  -8.029   1.299  1.00  0.00      A    C  \\nATOM    275  C   SER A  39       8.262  -6.816   2.195  1.00  0.00      A    C  \\nATOM    276  CB  SER A  39       6.893  -8.878   1.856  1.00  0.00      A    C  \\nATOM    277  O   SER A  39       7.391  -5.950   2.308  1.00  0.00      A    O  \\nATOM    278  OG  SER A  39       7.224  -9.392   3.135  1.00  0.00      A    O  \\nATOM    279  N   ALA A  40       9.374  -6.736   2.790  1.00  0.00      A    N  \\nATOM    280  CA  ALA A  40       9.659  -5.664   3.740  1.00  0.00      A    C  \\nATOM    281  C   ALA A  40       9.560  -4.296   3.070  1.00  0.00      A    C  \\nATOM    282  CB  ALA A  40      11.043  -5.852   4.356  1.00  0.00      A    C  \\nATOM    283  O   ALA A  40       9.012  -3.354   3.646  1.00  0.00      A    O  \\nATOM    284  N   LEU A  41      10.073  -4.129   1.901  1.00  0.00      A    N  \\nATOM    285  CA  LEU A  41      10.115  -2.835   1.228  1.00  0.00      A    C  \\nATOM    286  C   LEU A  41       8.706  -2.326   0.941  1.00  0.00      A    C  \\nATOM    287  CB  LEU A  41      10.910  -2.935  -0.077  1.00  0.00      A    C  \\nATOM    288  O   LEU A  41       8.359  -1.204   1.318  1.00  0.00      A    O  \\nATOM    289  CG  LEU A  41      10.857  -1.715  -0.998  1.00  0.00      A    C  \\nATOM    290  CD1 LEU A  41      11.340  -0.472  -0.259  1.00  0.00      A    C  \\nATOM    291  CD2 LEU A  41      11.690  -1.956  -2.252  1.00  0.00      A    C  \\nATOM    292  N   PRO A  42       7.806  -3.115   0.237  1.00  0.00      A    N  \\nATOM    293  CA  PRO A  42       6.470  -2.560   0.007  1.00  0.00      A    C  \\nATOM    294  C   PRO A  42       5.690  -2.343   1.301  1.00  0.00      A    C  \\nATOM    295  CB  PRO A  42       5.794  -3.620  -0.867  1.00  0.00      A    C  \\nATOM    296  O   PRO A  42       4.984  -1.341   1.441  1.00  0.00      A    O  \\nATOM    297  CG  PRO A  42       6.614  -4.854  -0.673  1.00  0.00      A    C  \\nATOM    298  CD  PRO A  42       8.030  -4.447  -0.381  1.00  0.00      A    C  \\nATOM    299  N   PHE A  43       5.830  -3.174   2.315  1.00  0.00      A    N  \\nATOM    300  CA  PHE A  43       5.140  -2.936   3.577  1.00  0.00      A    C  \\nATOM    301  C   PHE A  43       5.674  -1.681   4.257  1.00  0.00      A    C  \\nATOM    302  CB  PHE A  43       5.292  -4.143   4.509  1.00  0.00      A    C  \\nATOM    303  O   PHE A  43       4.927  -0.969   4.932  1.00  0.00      A    O  \\nATOM    304  CG  PHE A  43       4.346  -5.271   4.200  1.00  0.00      A    C  \\nATOM    305  CD1 PHE A  43       2.977  -5.047   4.118  1.00  0.00      A    C  \\nATOM    306  CD2 PHE A  43       4.826  -6.558   3.991  1.00  0.00      A    C  \\nATOM    307  CE1 PHE A  43       2.099  -6.090   3.831  1.00  0.00      A    C  \\nATOM    308  CE2 PHE A  43       3.955  -7.605   3.705  1.00  0.00      A    C  \\nATOM    309  CZ  PHE A  43       2.592  -7.369   3.626  1.00  0.00      A    C  \\nATOM    310  N   ASP A  44       6.977  -1.440   4.090  1.00  0.00      A    N  \\nATOM    311  CA  ASP A  44       7.543  -0.252   4.722  1.00  0.00      A    C  \\nATOM    312  C   ASP A  44       7.003   1.023   4.077  1.00  0.00      A    C  \\nATOM    313  CB  ASP A  44       9.071  -0.276   4.640  1.00  0.00      A    C  \\nATOM    314  O   ASP A  44       6.843   2.044   4.748  1.00  0.00      A    O  \\nATOM    315  CG  ASP A  44       9.736   0.533   5.740  1.00  0.00      A    C  \\nATOM    316  OD1 ASP A  44       9.326   0.418   6.915  1.00  0.00      A    O  \\nATOM    317  OD2 ASP A  44      10.677   1.294   5.428  1.00  0.00      A    O  \\nATOM    318  N   VAL A  45       6.687   0.999   2.780  1.00  0.00      A    N  \\nATOM    319  CA  VAL A  45       5.982   2.098   2.128  1.00  0.00      A    C  \\nATOM    320  C   VAL A  45       4.615   2.295   2.780  1.00  0.00      A    C  \\nATOM    321  CB  VAL A  45       5.819   1.848   0.612  1.00  0.00      A    C  \\nATOM    322  O   VAL A  45       4.219   3.424   3.077  1.00  0.00      A    O  \\nATOM    323  CG1 VAL A  45       4.932   2.919  -0.020  1.00  0.00      A    C  \\nATOM    324  CG2 VAL A  45       7.184   1.809  -0.072  1.00  0.00      A    C  \\nATOM    325  N   ASP A  46       3.912   1.156   3.075  1.00  0.00      A    N  \\nATOM    326  CA  ASP A  46       2.592   1.207   3.698  1.00  0.00      A    C  \\nATOM    327  C   ASP A  46       2.666   1.827   5.091  1.00  0.00      A    C  \\nATOM    328  CB  ASP A  46       1.982  -0.194   3.777  1.00  0.00      A    C  \\nATOM    329  O   ASP A  46       1.862   2.697   5.432  1.00  0.00      A    O  \\nATOM    330  CG  ASP A  46       1.872  -0.871   2.422  1.00  0.00      A    C  \\nATOM    331  OD1 ASP A  46       1.847  -0.168   1.389  1.00  0.00      A    O  \\nATOM    332  OD2 ASP A  46       1.807  -2.119   2.388  1.00  0.00      A    O  \\nATOM    333  N   CYS A  47       3.654   1.382   5.852  1.00  0.00      A    N  \\nATOM    334  CA  CYS A  47       3.841   1.882   7.210  1.00  0.00      A    C  \\nATOM    335  C   CYS A  47       4.181   3.368   7.202  1.00  0.00      A    C  \\nATOM    336  CB  CYS A  47       4.946   1.101   7.921  1.00  0.00      A    C  \\nATOM    337  O   CYS A  47       3.702   4.123   8.049  1.00  0.00      A    O  \\nATOM    338  SG  CYS A  47       4.519  -0.622   8.257  1.00  0.00      A    S  \\nATOM    339  N   TRP A  48       5.020   3.786   6.265  1.00  0.00      A    N  \\nATOM    340  CA  TRP A  48       5.326   5.206   6.127  1.00  0.00      A    C  \\nATOM    341  C   TRP A  48       4.067   6.006   5.812  1.00  0.00      A    C  \\nATOM    342  CB  TRP A  48       6.374   5.427   5.032  1.00  0.00      A    C  \\nATOM    343  O   TRP A  48       3.828   7.059   6.409  1.00  0.00      A    O  \\nATOM    344  CG  TRP A  48       6.658   6.870   4.743  1.00  0.00      A    C  \\nATOM    345  CD1 TRP A  48       7.431   7.722   5.482  1.00  0.00      A    C  \\nATOM    346  CD2 TRP A  48       6.166   7.633   3.637  1.00  0.00      A    C  \\nATOM    347  CE2 TRP A  48       6.682   8.941   3.768  1.00  0.00      A    C  \\nATOM    348  CE3 TRP A  48       5.338   7.336   2.546  1.00  0.00      A    C  \\nATOM    349  NE1 TRP A  48       7.450   8.968   4.901  1.00  0.00      A    N  \\nATOM    350  CH2 TRP A  48       5.585   9.635   1.789  1.00  0.00      A    C  \\nATOM    351  CZ2 TRP A  48       6.397   9.952   2.847  1.00  0.00      A    C  \\nATOM    352  CZ3 TRP A  48       5.055   8.344   1.631  1.00  0.00      A    C  \\nATOM    353  N   ALA A  49       3.246   5.525   4.864  1.00  0.00      A    N  \\nATOM    354  CA  ALA A  49       2.019   6.218   4.482  1.00  0.00      A    C  \\nATOM    355  C   ALA A  49       1.073   6.355   5.671  1.00  0.00      A    C  \\nATOM    356  CB  ALA A  49       1.327   5.481   3.337  1.00  0.00      A    C  \\nATOM    357  O   ALA A  49       0.488   7.419   5.887  1.00  0.00      A    O  \\nATOM    358  N   ILE A  50       0.948   5.353   6.482  1.00  0.00      A    N  \\nATOM    359  CA  ILE A  50       0.072   5.360   7.649  1.00  0.00      A    C  \\nATOM    360  C   ILE A  50       0.638   6.297   8.713  1.00  0.00      A    C  \\nATOM    361  CB  ILE A  50      -0.110   3.939   8.228  1.00  0.00      A    C  \\nATOM    362  O   ILE A  50      -0.057   7.198   9.190  1.00  0.00      A    O  \\nATOM    363  CG1 ILE A  50      -0.873   3.052   7.237  1.00  0.00      A    C  \\nATOM    364  CG2 ILE A  50      -0.830   3.995   9.579  1.00  0.00      A    C  \\nATOM    365  CD1 ILE A  50      -0.820   1.567   7.568  1.00  0.00      A    C  \\nATOM    366  N   LEU A  51       1.914   6.084   9.041  1.00  0.00      A    N  \\nATOM    367  CA  LEU A  51       2.529   6.770  10.172  1.00  0.00      A    C  \\nATOM    368  C   LEU A  51       2.778   8.239   9.848  1.00  0.00      A    C  \\nATOM    369  CB  LEU A  51       3.846   6.091  10.560  1.00  0.00      A    C  \\nATOM    370  O   LEU A  51       2.533   9.114  10.682  1.00  0.00      A    O  \\nATOM    371  CG  LEU A  51       4.585   6.681  11.762  1.00  0.00      A    C  \\nATOM    372  CD1 LEU A  51       3.709   6.611  13.008  1.00  0.00      A    C  \\nATOM    373  CD2 LEU A  51       5.905   5.952  11.989  1.00  0.00      A    C  \\nATOM    374  N   CYS A  52       3.181   8.557   8.598  1.00  0.00      A    N  \\nATOM    375  CA  CYS A  52       3.726   9.879   8.312  1.00  0.00      A    C  \\nATOM    376  C   CYS A  52       2.756  10.698   7.469  1.00  0.00      A    C  \\nATOM    377  CB  CYS A  52       5.069   9.762   7.592  1.00  0.00      A    C  \\nATOM    378  O   CYS A  52       2.836  11.927   7.441  1.00  0.00      A    O  \\nATOM    379  SG  CYS A  52       6.326   8.871   8.535  1.00  0.00      A    S  \\nATOM    380  N   LYS A  53       1.786  10.014   6.837  1.00  0.00      A    N  \\nATOM    381  CA  LYS A  53       0.891  10.772   5.968  1.00  0.00      A    C  \\nATOM    382  C   LYS A  53      -0.560  10.638   6.423  1.00  0.00      A    C  \\nATOM    383  CB  LYS A  53       1.030  10.309   4.517  1.00  0.00      A    C  \\nATOM    384  O   LYS A  53      -1.458  11.249   5.840  1.00  0.00      A    O  \\nATOM    385  CG  LYS A  53       2.412  10.541   3.924  1.00  0.00      A    C  \\nATOM    386  CD  LYS A  53       2.760  12.023   3.879  1.00  0.00      A    C  \\nATOM    387  CE  LYS A  53       1.982  12.748   2.789  1.00  0.00      A    C  \\nATOM    388  NZ  LYS A  53       2.512  14.123   2.550  1.00  0.00      A    N  \\nATOM    389  N   GLY A  54      -0.765   9.824   7.483  1.00  0.00      A    N  \\nATOM    390  CA  GLY A  54      -2.109   9.679   8.019  1.00  0.00      A    C  \\nATOM    391  C   GLY A  54      -3.029   8.879   7.116  1.00  0.00      A    C  \\nATOM    392  O   GLY A  54      -4.247   9.065   7.141  1.00  0.00      A    O  \\nATOM    393  N   ALA A  55      -2.422   7.992   6.282  1.00  0.00      A    N  \\nATOM    394  CA  ALA A  55      -3.235   7.116   5.443  1.00  0.00      A    C  \\nATOM    395  C   ALA A  55      -4.107   6.196   6.293  1.00  0.00      A    C  \\nATOM    396  CB  ALA A  55      -2.346   6.292   4.515  1.00  0.00      A    C  \\nATOM    397  O   ALA A  55      -3.777   5.909   7.446  1.00  0.00      A    O  \\nATOM    398  N   PRO A  56      -5.256   5.726   5.699  1.00  0.00      A    N  \\nATOM    399  CA  PRO A  56      -6.143   4.850   6.468  1.00  0.00      A    C  \\nATOM    400  C   PRO A  56      -5.471   3.541   6.875  1.00  0.00      A    C  \\nATOM    401  CB  PRO A  56      -7.305   4.592   5.506  1.00  0.00      A    C  \\nATOM    402  O   PRO A  56      -4.717   2.960   6.090  1.00  0.00      A    O  \\nATOM    403  CG  PRO A  56      -7.244   5.716   4.523  1.00  0.00      A    C  \\nATOM    404  CD  PRO A  56      -5.818   6.171   4.406  1.00  0.00      A    C  \\nATOM    405  N   ASN A  57      -5.773   3.117   8.135  1.00  0.00      A    N  \\nATOM    406  CA  ASN A  57      -5.206   1.854   8.595  1.00  0.00      A    C  \\nATOM    407  C   ASN A  57      -6.296   0.839   8.929  1.00  0.00      A    C  \\nATOM    408  CB  ASN A  57      -4.302   2.081   9.808  1.00  0.00      A    C  \\nATOM    409  O   ASN A  57      -6.004  -0.249   9.430  1.00  0.00      A    O  \\nATOM    410  CG  ASN A  57      -5.059   2.612  11.009  1.00  0.00      A    C  \\nATOM    411  ND2 ASN A  57      -4.378   2.705  12.145  1.00  0.00      A    N  \\nATOM    412  OD1 ASN A  57      -6.246   2.935  10.917  1.00  0.00      A    O  \\nATOM    413  N   VAL A  58      -7.538   1.288   8.804  1.00  0.00      A    N  \\nATOM    414  CA  VAL A  58      -8.687   0.388   8.812  1.00  0.00      A    C  \\nATOM    415  C   VAL A  58      -9.273   0.286   7.405  1.00  0.00      A    C  \\nATOM    416  CB  VAL A  58      -9.770   0.858   9.808  1.00  0.00      A    C  \\nATOM    417  O   VAL A  58      -9.647   1.297   6.807  1.00  0.00      A    O  \\nATOM    418  CG1 VAL A  58     -10.952  -0.109   9.817  1.00  0.00      A    C  \\nATOM    419  CG2 VAL A  58      -9.180   0.999  11.210  1.00  0.00      A    C  \\nATOM    420  N   LEU A  59      -9.281  -0.928   6.901  1.00  0.00      A    N  \\nATOM    421  CA  LEU A  59      -9.577  -1.138   5.488  1.00  0.00      A    C  \\nATOM    422  C   LEU A  59     -10.710  -2.144   5.313  1.00  0.00      A    C  \\nATOM    423  CB  LEU A  59      -8.329  -1.624   4.745  1.00  0.00      A    C  \\nATOM    424  O   LEU A  59     -11.020  -2.902   6.235  1.00  0.00      A    O  \\nATOM    425  CG  LEU A  59      -7.077  -0.757   4.886  1.00  0.00      A    C  \\nATOM    426  CD1 LEU A  59      -5.879  -1.448   4.242  1.00  0.00      A    C  \\nATOM    427  CD2 LEU A  59      -7.305   0.617   4.266  1.00  0.00      A    C  \\nATOM    428  N   GLN A  60     -11.340  -2.083   4.183  1.00  0.00      A    N  \\nATOM    429  CA  GLN A  60     -12.347  -3.050   3.760  1.00  0.00      A    C  \\nATOM    430  C   GLN A  60     -12.002  -3.644   2.398  1.00  0.00      A    C  \\nATOM    431  CB  GLN A  60     -13.731  -2.399   3.715  1.00  0.00      A    C  \\nATOM    432  O   GLN A  60     -11.774  -2.909   1.434  1.00  0.00      A    O  \\nATOM    433  CG  GLN A  60     -14.871  -3.392   3.540  1.00  0.00      A    C  \\nATOM    434  CD  GLN A  60     -16.237  -2.734   3.600  1.00  0.00      A    C  \\nATOM    435  NE2 GLN A  60     -17.211  -3.439   4.166  1.00  0.00      A    N  \\nATOM    436  OE1 GLN A  60     -16.415  -1.601   3.143  1.00  0.00      A    O  \\nATOM    437  N   ARG A  61     -11.954  -4.994   2.364  1.00  0.00      A    N  \\nATOM    438  CA  ARG A  61     -11.592  -5.678   1.127  1.00  0.00      A    C  \\nATOM    439  C   ARG A  61     -12.687  -5.525   0.076  1.00  0.00      A    C  \\nATOM    440  CB  ARG A  61     -11.322  -7.161   1.391  1.00  0.00      A    C  \\nATOM    441  O   ARG A  61     -13.875  -5.595   0.396  1.00  0.00      A    O  \\nATOM    442  CG  ARG A  61     -10.771  -7.910   0.189  1.00  0.00      A    C  \\nATOM    443  CD  ARG A  61     -10.439  -9.356   0.529  1.00  0.00      A    C  \\nATOM    444  NE  ARG A  61      -9.976 -10.093  -0.643  1.00  0.00      A    N  \\nATOM    445  NH1 ARG A  61     -10.054 -12.167   0.373  1.00  0.00      A    N  \\nATOM    446  NH2 ARG A  61      -9.378 -11.979  -1.810  1.00  0.00      A    N  \\nATOM    447  CZ  ARG A  61      -9.804 -11.411  -0.691  1.00  0.00      A    C  \\nATOM    448  N   VAL A  62     -12.235  -5.340  -1.252  1.00  0.00      A    N  \\nATOM    449  CA  VAL A  62     -13.207  -5.149  -2.323  1.00  0.00      A    C  \\nATOM    450  C   VAL A  62     -13.038  -6.242  -3.376  1.00  0.00      A    C  \\nATOM    451  CB  VAL A  62     -13.067  -3.755  -2.975  1.00  0.00      A    C  \\nATOM    452  O   VAL A  62     -12.035  -6.960  -3.379  1.00  0.00      A    O  \\nATOM    453  CG1 VAL A  62     -13.467  -2.657  -1.992  1.00  0.00      A    C  \\nATOM    454  CG2 VAL A  62     -11.637  -3.539  -3.469  1.00  0.00      A    C  \\nATOM    455  N   ASN A  63     -14.089  -6.351  -4.175  1.00  0.00      A    N  \\nATOM    456  CA  ASN A  63     -14.095  -7.327  -5.259  1.00  0.00      A    C  \\nATOM    457  C   ASN A  63     -13.105  -6.952  -6.358  1.00  0.00      A    C  \\nATOM    458  CB  ASN A  63     -15.503  -7.475  -5.841  1.00  0.00      A    C  \\nATOM    459  O   ASN A  63     -12.549  -5.852  -6.349  1.00  0.00      A    O  \\nATOM    460  CG  ASN A  63     -16.010  -6.193  -6.473  1.00  0.00      A    C  \\nATOM    461  ND2 ASN A  63     -17.292  -6.169  -6.818  1.00  0.00      A    N  \\nATOM    462  OD1 ASN A  63     -15.256  -5.233  -6.651  1.00  0.00      A    O  \\nATOM    463  N   GLU A  64     -12.816  -7.854  -7.216  1.00  0.00      A    N  \\nATOM    464  CA  GLU A  64     -11.789  -7.731  -8.246  1.00  0.00      A    C  \\nATOM    465  C   GLU A  64     -12.083  -6.563  -9.183  1.00  0.00      A    C  \\nATOM    466  CB  GLU A  64     -11.671  -9.031  -9.046  1.00  0.00      A    C  \\nATOM    467  O   GLU A  64     -11.168  -5.854  -9.606  1.00  0.00      A    O  \\nATOM    468  CG  GLU A  64     -10.499  -9.051 -10.017  1.00  0.00      A    C  \\nATOM    469  CD  GLU A  64      -9.147  -8.973  -9.326  1.00  0.00      A    C  \\nATOM    470  OE1 GLU A  64      -9.074  -9.233  -8.103  1.00  0.00      A    O  \\nATOM    471  OE2 GLU A  64      -8.153  -8.648 -10.013  1.00  0.00      A    O  \\nATOM    472  N   LYS A  65     -13.300  -6.406  -9.527  1.00  0.00      A    N  \\nATOM    473  CA  LYS A  65     -13.680  -5.297 -10.397  1.00  0.00      A    C  \\nATOM    474  C   LYS A  65     -13.283  -3.956  -9.784  1.00  0.00      A    C  \\nATOM    475  CB  LYS A  65     -15.184  -5.322 -10.674  1.00  0.00      A    C  \\nATOM    476  O   LYS A  65     -12.699  -3.107 -10.460  1.00  0.00      A    O  \\nATOM    477  CG  LYS A  65     -15.661  -4.214 -11.601  1.00  0.00      A    C  \\nATOM    478  CD  LYS A  65     -17.171  -4.258 -11.794  1.00  0.00      A    C  \\nATOM    479  CE  LYS A  65     -17.663  -3.088 -12.636  1.00  0.00      A    C  \\nATOM    480  NZ  LYS A  65     -19.151  -3.088 -12.769  1.00  0.00      A    N  \\nATOM    481  N   THR A  66     -13.613  -3.718  -8.521  1.00  0.00      A    N  \\nATOM    482  CA  THR A  66     -13.273  -2.485  -7.820  1.00  0.00      A    C  \\nATOM    483  C   THR A  66     -11.764  -2.368  -7.631  1.00  0.00      A    C  \\nATOM    484  CB  THR A  66     -13.972  -2.411  -6.449  1.00  0.00      A    C  \\nATOM    485  O   THR A  66     -11.200  -1.279  -7.756  1.00  0.00      A    O  \\nATOM    486  CG2 THR A  66     -13.631  -1.112  -5.728  1.00  0.00      A    C  \\nATOM    487  OG1 THR A  66     -15.391  -2.481  -6.639  1.00  0.00      A    O  \\nATOM    488  N   LYS A  67     -11.143  -3.482  -7.328  1.00  0.00      A    N  \\nATOM    489  CA  LYS A  67      -9.688  -3.508  -7.205  1.00  0.00      A    C  \\nATOM    490  C   LYS A  67      -9.019  -2.983  -8.472  1.00  0.00      A    C  \\nATOM    491  CB  LYS A  67      -9.200  -4.926  -6.904  1.00  0.00      A    C  \\nATOM    492  O   LYS A  67      -8.089  -2.177  -8.401  1.00  0.00      A    O  \\nATOM    493  CG  LYS A  67      -7.686  -5.053  -6.811  1.00  0.00      A    C  \\nATOM    494  CD  LYS A  67      -7.258  -6.499  -6.600  1.00  0.00      A    C  \\nATOM    495  CE  LYS A  67      -5.754  -6.668  -6.763  1.00  0.00      A    C  \\nATOM    496  NZ  LYS A  67      -5.343  -8.101  -6.671  1.00  0.00      A    N  \\nATOM    497  N   ASN A  68      -9.474  -3.369  -9.647  1.00  0.00      A    N  \\nATOM    498  CA  ASN A  68      -8.915  -2.915 -10.916  1.00  0.00      A    C  \\nATOM    499  C   ASN A  68      -9.186  -1.432 -11.149  1.00  0.00      A    C  \\nATOM    500  CB  ASN A  68      -9.470  -3.745 -12.076  1.00  0.00      A    C  \\nATOM    501  O   ASN A  68      -8.317  -0.707 -11.638  1.00  0.00      A    O  \\nATOM    502  CG  ASN A  68      -8.944  -5.167 -12.080  1.00  0.00      A    C  \\nATOM    503  ND2 ASN A  68      -9.693  -6.074 -12.697  1.00  0.00      A    N  \\nATOM    504  OD1 ASN A  68      -7.875  -5.448 -11.532  1.00  0.00      A    O  \\nATOM    505  N   SER A  69     -10.384  -1.052 -10.773  1.00  0.00      A    N  \\nATOM    506  CA  SER A  69     -10.713   0.368 -10.852  1.00  0.00      A    C  \\nATOM    507  C   SER A  69      -9.819   1.194  -9.933  1.00  0.00      A    C  \\nATOM    508  CB  SER A  69     -12.181   0.599 -10.491  1.00  0.00      A    C  \\nATOM    509  O   SER A  69      -9.351   2.268 -10.315  1.00  0.00      A    O  \\nATOM    510  OG  SER A  69     -12.512   1.974 -10.585  1.00  0.00      A    O  \\nATOM    511  N   ASN A  70      -9.558   0.678  -8.737  1.00  0.00      A    N  \\nATOM    512  CA  ASN A  70      -8.673   1.360  -7.799  1.00  0.00      A    C  \\nATOM    513  C   ASN A  70      -7.262   1.505  -8.364  1.00  0.00      A    C  \\nATOM    514  CB  ASN A  70      -8.635   0.620  -6.461  1.00  0.00      A    C  \\nATOM    515  O   ASN A  70      -6.625   2.546  -8.196  1.00  0.00      A    O  \\nATOM    516  CG  ASN A  70      -9.892   0.833  -5.641  1.00  0.00      A    C  \\nATOM    517  ND2 ASN A  70     -10.070   0.020  -4.605  1.00  0.00      A    N  \\nATOM    518  OD1 ASN A  70     -10.698   1.719  -5.935  1.00  0.00      A    O  \\nATOM    519  N   ARG A  71      -6.772   0.550  -9.081  1.00  0.00      A    N  \\nATOM    520  CA  ARG A  71      -5.455   0.626  -9.705  1.00  0.00      A    C  \\nATOM    521  C   ARG A  71      -5.407   1.739 -10.747  1.00  0.00      A    C  \\nATOM    522  CB  ARG A  71      -5.087  -0.712 -10.349  1.00  0.00      A    C  \\nATOM    523  O   ARG A  71      -4.415   2.464 -10.843  1.00  0.00      A    O  \\nATOM    524  CG  ARG A  71      -4.660  -1.778  -9.354  1.00  0.00      A    C  \\nATOM    525  CD  ARG A  71      -4.226  -3.060 -10.051  1.00  0.00      A    C  \\nATOM    526  NE  ARG A  71      -3.564  -3.978  -9.129  1.00  0.00      A    N  \\nATOM    527  NH1 ARG A  71      -4.975  -5.755  -9.558  1.00  0.00      A    N  \\nATOM    528  NH2 ARG A  71      -3.260  -5.985  -8.054  1.00  0.00      A    N  \\nATOM    529  CZ  ARG A  71      -3.935  -5.237  -8.916  1.00  0.00      A    C  \\nATOM    530  N   ASP A  72      -6.462   1.844 -11.491  1.00  0.00      A    N  \\nATOM    531  CA  ASP A  72      -6.534   2.911 -12.485  1.00  0.00      A    C  \\nATOM    532  C   ASP A  72      -6.559   4.284 -11.815  1.00  0.00      A    C  \\nATOM    533  CB  ASP A  72      -7.768   2.737 -13.372  1.00  0.00      A    C  \\nATOM    534  O   ASP A  72      -5.819   5.187 -12.211  1.00  0.00      A    O  \\nATOM    535  CG  ASP A  72      -7.646   1.570 -14.336  1.00  0.00      A    C  \\nATOM    536  OD1 ASP A  72      -6.514   1.099 -14.583  1.00  0.00      A    O  \\nATOM    537  OD2 ASP A  72      -8.689   1.118 -14.854  1.00  0.00      A    O  \\nATOM    538  N   ARG A  73      -7.292   4.417 -10.759  1.00  0.00      A    N  \\nATOM    539  CA  ARG A  73      -7.513   5.695 -10.089  1.00  0.00      A    C  \\nATOM    540  C   ARG A  73      -6.289   6.105  -9.276  1.00  0.00      A    C  \\nATOM    541  CB  ARG A  73      -8.744   5.621  -9.182  1.00  0.00      A    C  \\nATOM    542  O   ARG A  73      -6.045   7.296  -9.071  1.00  0.00      A    O  \\nATOM    543  CG  ARG A  73     -10.063   5.576  -9.937  1.00  0.00      A    C  \\nATOM    544  CD  ARG A  73     -11.254   5.701  -8.998  1.00  0.00      A    C  \\nATOM    545  NE  ARG A  73     -11.338   4.570  -8.079  1.00  0.00      A    N  \\nATOM    546  NH1 ARG A  73     -12.415   5.743  -6.405  1.00  0.00      A    N  \\nATOM    547  NH2 ARG A  73     -11.909   3.526  -6.115  1.00  0.00      A    N  \\nATOM    548  CZ  ARG A  73     -11.887   4.615  -6.868  1.00  0.00      A    C  \\nATOM    549  N   SER A  74      -5.547   5.112  -8.783  1.00  0.00      A    N  \\nATOM    550  CA  SER A  74      -4.327   5.413  -8.043  1.00  0.00      A    C  \\nATOM    551  C   SER A  74      -3.199   5.828  -8.982  1.00  0.00      A    C  \\nATOM    552  CB  SER A  74      -3.892   4.206  -7.212  1.00  0.00      A    C  \\nATOM    553  O   SER A  74      -2.187   6.376  -8.541  1.00  0.00      A    O  \\nATOM    554  OG  SER A  74      -3.459   3.148  -8.049  1.00  0.00      A    O  \\nATOM    555  N   GLY A  75      -3.346   5.579 -10.327  1.00  0.00      A    N  \\nATOM    556  CA  GLY A  75      -2.319   5.882 -11.311  1.00  0.00      A    C  \\nATOM    557  C   GLY A  75      -1.271   4.792 -11.435  1.00  0.00      A    C  \\nATOM    558  O   GLY A  75      -0.332   4.914 -12.224  1.00  0.00      A    O  \\nATOM    559  N   ALA A  76      -1.412   3.690 -10.716  1.00  0.00      A    N  \\nATOM    560  CA  ALA A  76      -0.428   2.610 -10.740  1.00  0.00      A    C  \\nATOM    561  C   ALA A  76      -0.320   1.998 -12.133  1.00  0.00      A    C  \\nATOM    562  CB  ALA A  76      -0.793   1.536  -9.717  1.00  0.00      A    C  \\nATOM    563  O   ALA A  76       0.773   1.635 -12.575  1.00  0.00      A    O  \\nATOM    564  N   ASN A  77      -1.438   1.950 -12.875  1.00  0.00      A    N  \\nATOM    565  CA  ASN A  77      -1.491   1.323 -14.191  1.00  0.00      A    C  \\nATOM    566  C   ASN A  77      -0.873   2.216 -15.263  1.00  0.00      A    C  \\nATOM    567  CB  ASN A  77      -2.934   0.971 -14.560  1.00  0.00      A    C  \\nATOM    568  O   ASN A  77      -0.655   1.776 -16.393  1.00  0.00      A    O  \\nATOM    569  CG  ASN A  77      -3.431  -0.274 -13.851  1.00  0.00      A    C  \\nATOM    570  ND2 ASN A  77      -4.742  -0.480 -13.866  1.00  0.00      A    N  \\nATOM    571  OD1 ASN A  77      -2.643  -1.042 -13.294  1.00  0.00      A    O  \\nATOM    572  N   LYS A  78      -0.615   3.462 -14.879  1.00  0.00      A    N  \\nATOM    573  CA  LYS A  78       0.004   4.384 -15.828  1.00  0.00      A    C  \\nATOM    574  C   LYS A  78       1.500   4.114 -15.960  1.00  0.00      A    C  \\nATOM    575  CB  LYS A  78      -0.231   5.833 -15.400  1.00  0.00      A    C  \\nATOM    576  O   LYS A  78       2.144   4.602 -16.891  1.00  0.00      A    O  \\nATOM    577  CG  LYS A  78      -1.698   6.237 -15.361  1.00  0.00      A    C  \\nATOM    578  CD  LYS A  78      -1.860   7.735 -15.135  1.00  0.00      A    C  \\nATOM    579  CE  LYS A  78      -3.325   8.148 -15.151  1.00  0.00      A    C  \\nATOM    580  NZ  LYS A  78      -3.874   8.191 -16.539  1.00  0.00      A    N  \\nATOM    581  N   GLY A  79       2.052   3.348 -15.086  1.00  0.00      A    N  \\nATOM    582  CA  GLY A  79       3.464   3.008 -15.156  1.00  0.00      A    C  \\nATOM    583  C   GLY A  79       4.376   4.179 -14.843  1.00  0.00      A    C  \\nATOM    584  O   GLY A  79       5.307   4.466 -15.598  1.00  0.00      A    O  \\nATOM    585  N   PRO A  80       4.146   4.890 -13.710  1.00  0.00      A    N  \\nATOM    586  CA  PRO A  80       4.890   6.126 -13.457  1.00  0.00      A    C  \\nATOM    587  C   PRO A  80       6.389   5.889 -13.286  1.00  0.00      A    C  \\nATOM    588  CB  PRO A  80       4.270   6.652 -12.160  1.00  0.00      A    C  \\nATOM    589  O   PRO A  80       7.200   6.737 -13.668  1.00  0.00      A    O  \\nATOM    590  CG  PRO A  80       3.656   5.451 -11.516  1.00  0.00      A    C  \\nATOM    591  CD  PRO A  80       3.157   4.534 -12.595  1.00  0.00      A    C  \\nATOM    592  N   PHE A  81       6.779   4.700 -12.870  1.00  0.00      A    N  \\nATOM    593  CA  PHE A  81       8.182   4.495 -12.528  1.00  0.00      A    C  \\nATOM    594  C   PHE A  81       8.910   3.760 -13.647  1.00  0.00      A    C  \\nATOM    595  CB  PHE A  81       8.308   3.712 -11.217  1.00  0.00      A    C  \\nATOM    596  O   PHE A  81      10.131   3.599 -13.600  1.00  0.00      A    O  \\nATOM    597  CG  PHE A  81       7.663   4.393 -10.040  1.00  0.00      A    C  \\nATOM    598  CD1 PHE A  81       8.013   5.692  -9.693  1.00  0.00      A    C  \\nATOM    599  CD2 PHE A  81       6.708   3.732  -9.279  1.00  0.00      A    C  \\nATOM    600  CE1 PHE A  81       7.418   6.325  -8.604  1.00  0.00      A    C  \\nATOM    601  CE2 PHE A  81       6.109   4.358  -8.189  1.00  0.00      A    C  \\nATOM    602  CZ  PHE A  81       6.466   5.653  -7.853  1.00  0.00      A    C  \\nATOM    603  N   LYS A  82       8.082   3.300 -14.664  1.00  0.00      A    N  \\nATOM    604  CA  LYS A  82       8.680   2.724 -15.865  1.00  0.00      A    C  \\nATOM    605  C   LYS A  82       9.199   3.814 -16.799  1.00  0.00      A    C  \\nATOM    606  CB  LYS A  82       7.668   1.843 -16.599  1.00  0.00      A    C  \\nATOM    607  O   LYS A  82      10.185   3.611 -17.511  1.00  0.00      A    O  \\nATOM    608  CG  LYS A  82       7.288   0.577 -15.845  1.00  0.00      A    C  \\nATOM    609  CD  LYS A  82       6.364  -0.309 -16.671  1.00  0.00      A    C  \\nATOM    610  CE  LYS A  82       5.976  -1.572 -15.914  1.00  0.00      A    C  \\nATOM    611  NZ  LYS A  82       5.019  -2.413 -16.694  1.00  0.00      A    N  \\nATOM    612  N   ASP A  83       8.503   4.873 -16.706  1.00  0.00      A    N  \\nATOM    613  CA  ASP A  83       8.897   6.042 -17.486  1.00  0.00      A    C  \\nATOM    614  C   ASP A  83       8.860   7.309 -16.635  1.00  0.00      A    C  \\nATOM    615  CB  ASP A  83       7.989   6.203 -18.708  1.00  0.00      A    C  \\nATOM    616  O   ASP A  83       7.994   8.166 -16.824  1.00  0.00      A    O  \\nATOM    617  CG  ASP A  83       8.554   7.159 -19.744  1.00  0.00      A    C  \\nATOM    618  OD1 ASP A  83       9.550   7.855 -19.453  1.00  0.00      A    O  \\nATOM    619  OD2 ASP A  83       7.996   7.218 -20.861  1.00  0.00      A    O  \\nATOM    620  N   PRO A  84       9.858   7.444 -15.754  1.00  0.00      A    N  \\nATOM    621  CA  PRO A  84       9.841   8.565 -14.812  1.00  0.00      A    C  \\nATOM    622  C   PRO A  84       9.872   9.923 -15.509  1.00  0.00      A    C  \\nATOM    623  CB  PRO A  84      11.109   8.347 -13.982  1.00  0.00      A    C  \\nATOM    624  O   PRO A  84       9.205  10.862 -15.069  1.00  0.00      A    O  \\nATOM    625  CG  PRO A  84      11.393   6.884 -14.094  1.00  0.00      A    C  \\nATOM    626  CD  PRO A  84      10.854   6.393 -15.407  1.00  0.00      A    C  \\nATOM    627  N   GLN A  85      10.591  10.085 -16.543  1.00  0.00      A    N  \\nATOM    628  CA  GLN A  85      10.711  11.345 -17.269  1.00  0.00      A    C  \\nATOM    629  C   GLN A  85       9.364  11.784 -17.835  1.00  0.00      A    C  \\nATOM    630  CB  GLN A  85      11.738  11.219 -18.396  1.00  0.00      A    C  \\nATOM    631  O   GLN A  85       9.006  12.962 -17.757  1.00  0.00      A    O  \\nATOM    632  CG  GLN A  85      13.179  11.146 -17.910  1.00  0.00      A    C  \\nATOM    633  CD  GLN A  85      14.183  11.141 -19.047  1.00  0.00      A    C  \\nATOM    634  NE2 GLN A  85      15.463  11.027 -18.708  1.00  0.00      A    N  \\nATOM    635  OE1 GLN A  85      13.812  11.239 -20.221  1.00  0.00      A    O  \\nATOM    636  N   LYS A  86       8.635  10.846 -18.352  1.00  0.00      A    N  \\nATOM    637  CA  LYS A  86       7.312  11.132 -18.901  1.00  0.00      A    C  \\nATOM    638  C   LYS A  86       6.401  11.749 -17.844  1.00  0.00      A    C  \\nATOM    639  CB  LYS A  86       6.677   9.858 -19.461  1.00  0.00      A    C  \\nATOM    640  O   LYS A  86       5.596  12.631 -18.150  1.00  0.00      A    O  \\nATOM    641  CG  LYS A  86       5.250  10.044 -19.954  1.00  0.00      A    C  \\nATOM    642  CD  LYS A  86       4.667   8.740 -20.484  1.00  0.00      A    C  \\nATOM    643  CE  LYS A  86       3.200   8.895 -20.863  1.00  0.00      A    C  \\nATOM    644  NZ  LYS A  86       2.616   7.611 -21.353  1.00  0.00      A    N  \\nATOM    645  N   TRP A  87       6.583  11.351 -16.636  1.00  0.00      A    N  \\nATOM    646  CA  TRP A  87       5.617  11.688 -15.595  1.00  0.00      A    C  \\nATOM    647  C   TRP A  87       6.184  12.737 -14.644  1.00  0.00      A    C  \\nATOM    648  CB  TRP A  87       5.211  10.437 -14.811  1.00  0.00      A    C  \\nATOM    649  O   TRP A  87       5.578  13.043 -13.615  1.00  0.00      A    O  \\nATOM    650  CG  TRP A  87       4.417   9.448 -15.611  1.00  0.00      A    C  \\nATOM    651  CD1 TRP A  87       4.803   8.190 -15.983  1.00  0.00      A    C  \\nATOM    652  CD2 TRP A  87       3.102   9.639 -16.142  1.00  0.00      A    C  \\nATOM    653  CE2 TRP A  87       2.752   8.453 -16.825  1.00  0.00      A    C  \\nATOM    654  CE3 TRP A  87       2.185  10.697 -16.105  1.00  0.00      A    C  \\nATOM    655  NE1 TRP A  87       3.805   7.586 -16.714  1.00  0.00      A    N  \\nATOM    656  CH2 TRP A  87       0.642   9.348 -17.416  1.00  0.00      A    C  \\nATOM    657  CZ2 TRP A  87       1.521   8.297 -17.467  1.00  0.00      A    C  \\nATOM    658  CZ3 TRP A  87       0.960  10.540 -16.744  1.00  0.00      A    C  \\nATOM    659  N   GLY A  88       7.311  13.294 -14.962  1.00  0.00      A    N  \\nATOM    660  CA  GLY A  88       7.918  14.320 -14.131  1.00  0.00      A    C  \\nATOM    661  C   GLY A  88       8.402  13.795 -12.792  1.00  0.00      A    C  \\nATOM    662  O   GLY A  88       8.460  14.540 -11.811  1.00  0.00      A    O  \\nATOM    663  N   ILE A  89       8.639  12.499 -12.707  1.00  0.00      A    N  \\nATOM    664  CA  ILE A  89       9.127  11.823 -11.510  1.00  0.00      A    C  \\nATOM    665  C   ILE A  89      10.654  11.807 -11.513  1.00  0.00      A    C  \\nATOM    666  CB  ILE A  89       8.576  10.383 -11.411  1.00  0.00      A    C  \\nATOM    667  O   ILE A  89      11.277  11.607 -12.558  1.00  0.00      A    O  \\nATOM    668  CG1 ILE A  89       7.059  10.405 -11.194  1.00  0.00      A    C  \\nATOM    669  CG2 ILE A  89       9.279   9.612 -10.289  1.00  0.00      A    C  \\nATOM    670  CD1 ILE A  89       6.396   9.040 -11.313  1.00  0.00      A    C  \\nATOM    671  N   LYS A  90      11.275  12.165 -10.393  1.00  0.00      A    N  \\nATOM    672  CA  LYS A  90      12.720  12.117 -10.193  1.00  0.00      A    C  \\nATOM    673  C   LYS A  90      13.280  10.744 -10.552  1.00  0.00      A    C  \\nATOM    674  CB  LYS A  90      13.074  12.465  -8.746  1.00  0.00      A    C  \\nATOM    675  O   LYS A  90      12.529   9.773 -10.671  1.00  0.00      A    O  \\nATOM    676  CG  LYS A  90      12.777  13.908  -8.366  1.00  0.00      A    C  \\nATOM    677  CD  LYS A  90      13.192  14.204  -6.930  1.00  0.00      A    C  \\nATOM    678  CE  LYS A  90      12.883  15.645  -6.544  1.00  0.00      A    C  \\nATOM    679  NZ  LYS A  90      13.255  15.930  -5.126  1.00  0.00      A    N  \\nATOM    680  N   ALA A  91      14.556  10.655 -10.665  1.00  0.00      A    N  \\nATOM    681  CA  ALA A  91      15.218   9.382 -10.941  1.00  0.00      A    C  \\nATOM    682  C   ALA A  91      14.873   8.343  -9.878  1.00  0.00      A    C  \\nATOM    683  CB  ALA A  91      16.730   9.576 -11.021  1.00  0.00      A    C  \\nATOM    684  O   ALA A  91      14.688   8.681  -8.707  1.00  0.00      A    O  \\nATOM    685  N   LEU A  92      14.604   7.186 -10.289  1.00  0.00      A    N  \\nATOM    686  CA  LEU A  92      14.309   6.016  -9.469  1.00  0.00      A    C  \\nATOM    687  C   LEU A  92      15.407   5.786  -8.436  1.00  0.00      A    C  \\nATOM    688  CB  LEU A  92      14.149   4.773 -10.347  1.00  0.00      A    C  \\nATOM    689  O   LEU A  92      16.513   6.312  -8.572  1.00  0.00      A    O  \\nATOM    690  CG  LEU A  92      12.984   4.788 -11.339  1.00  0.00      A    C  \\nATOM    691  CD1 LEU A  92      13.023   3.545 -12.221  1.00  0.00      A    C  \\nATOM    692  CD2 LEU A  92      11.654   4.884 -10.600  1.00  0.00      A    C  \\nATOM    693  N   PRO A  93      15.053   5.111  -7.350  1.00  0.00      A    N  \\nATOM    694  CA  PRO A  93      16.085   4.802  -6.358  1.00  0.00      A    C  \\nATOM    695  C   PRO A  93      17.286   4.078  -6.961  1.00  0.00      A    C  \\nATOM    696  CB  PRO A  93      15.353   3.905  -5.357  1.00  0.00      A    C  \\nATOM    697  O   PRO A  93      17.165   3.445  -8.014  1.00  0.00      A    O  \\nATOM    698  CG  PRO A  93      13.903   4.080  -5.675  1.00  0.00      A    C  \\nATOM    699  CD  PRO A  93      13.780   4.540  -7.099  1.00  0.00      A    C  \\nATOM    700  N   PRO A  94      18.433   4.336  -6.269  1.00  0.00      A    N  \\nATOM    701  CA  PRO A  94      19.607   3.611  -6.760  1.00  0.00      A    C  \\nATOM    702  C   PRO A  94      19.441   2.095  -6.680  1.00  0.00      A    C  \\nATOM    703  CB  PRO A  94      20.729   4.087  -5.833  1.00  0.00      A    C  \\nATOM    704  O   PRO A  94      18.736   1.594  -5.801  1.00  0.00      A    O  \\nATOM    705  CG  PRO A  94      20.033   4.531  -4.588  1.00  0.00      A    C  \\nATOM    706  CD  PRO A  94      18.651   4.994  -4.953  1.00  0.00      A    C  \\nATOM    707  N   LYS A  95      20.011   1.448  -7.670  1.00  0.00      A    N  \\nATOM    708  CA  LYS A  95      20.030  -0.010  -7.610  1.00  0.00      A    C  \\nATOM    709  C   LYS A  95      20.801  -0.500  -6.388  1.00  0.00      A    C  \\nATOM    710  CB  LYS A  95      20.642  -0.592  -8.884  1.00  0.00      A    C  \\nATOM    711  O   LYS A  95      21.808   0.097  -6.002  1.00  0.00      A    O  \\nATOM    712  CG  LYS A  95      19.796  -0.380 -10.131  1.00  0.00      A    C  \\nATOM    713  CD  LYS A  95      20.378  -1.112 -11.333  1.00  0.00      A    C  \\nATOM    714  CE  LYS A  95      19.524  -0.912 -12.578  1.00  0.00      A    C  \\nATOM    715  NZ  LYS A  95      20.115  -1.590 -13.770  1.00  0.00      A    N  \\nATOM    716  N   ASN A  96      20.321  -1.405  -5.666  1.00  0.00      A    N  \\nATOM    717  CA  ASN A  96      21.028  -2.143  -4.624  1.00  0.00      A    C  \\nATOM    718  C   ASN A  96      21.382  -3.556  -5.080  1.00  0.00      A    C  \\nATOM    719  CB  ASN A  96      20.195  -2.192  -3.342  1.00  0.00      A    C  \\nATOM    720  O   ASN A  96      20.498  -4.397  -5.253  1.00  0.00      A    O  \\nATOM    721  CG  ASN A  96      20.956  -2.784  -2.173  1.00  0.00      A    C  \\nATOM    722  ND2 ASN A  96      20.568  -2.409  -0.960  1.00  0.00      A    N  \\nATOM    723  OD1 ASN A  96      21.887  -3.573  -2.358  1.00  0.00      A    O  \\nATOM    724  N   PRO A  97      22.673  -3.794  -5.297  1.00  0.00      A    N  \\nATOM    725  CA  PRO A  97      23.060  -5.090  -5.858  1.00  0.00      A    C  \\nATOM    726  C   PRO A  97      22.683  -6.262  -4.954  1.00  0.00      A    C  \\nATOM    727  CB  PRO A  97      24.580  -4.974  -5.997  1.00  0.00      A    C  \\nATOM    728  O   PRO A  97      22.582  -7.399  -5.421  1.00  0.00      A    O  \\nATOM    729  CG  PRO A  97      24.948  -3.779  -5.177  1.00  0.00      A    C  \\nATOM    730  CD  PRO A  97      23.715  -2.949  -4.964  1.00  0.00      A    C  \\nATOM    731  N   SER A  98      22.623  -5.964  -3.543  1.00  0.00      A    N  \\nATOM    732  CA  SER A  98      22.325  -7.058  -2.625  1.00  0.00      A    C  \\nATOM    733  C   SER A  98      20.856  -7.462  -2.705  1.00  0.00      A    C  \\nATOM    734  CB  SER A  98      22.677  -6.666  -1.190  1.00  0.00      A    C  \\nATOM    735  O   SER A  98      20.471  -8.522  -2.209  1.00  0.00      A    O  \\nATOM    736  OG  SER A  98      24.045  -6.311  -1.087  1.00  0.00      A    O  \\nATOM    737  N   TRP A  99      20.051  -6.692  -3.233  1.00  0.00      A    N  \\nATOM    738  CA  TRP A  99      18.640  -6.986  -3.462  1.00  0.00      A    C  \\nATOM    739  C   TRP A  99      18.394  -7.384  -4.913  1.00  0.00      A    C  \\nATOM    740  CB  TRP A  99      17.773  -5.777  -3.097  1.00  0.00      A    C  \\nATOM    741  O   TRP A  99      18.622  -6.589  -5.828  1.00  0.00      A    O  \\nATOM    742  CG  TRP A  99      17.399  -5.711  -1.646  1.00  0.00      A    C  \\nATOM    743  CD1 TRP A  99      18.246  -5.569  -0.583  1.00  0.00      A    C  \\nATOM    744  CD2 TRP A  99      16.079  -5.791  -1.100  1.00  0.00      A    C  \\nATOM    745  CE2 TRP A  99      16.201  -5.689   0.303  1.00  0.00      A    C  \\nATOM    746  CE3 TRP A  99      14.804  -5.938  -1.662  1.00  0.00      A    C  \\nATOM    747  NE1 TRP A  99      17.532  -5.554   0.593  1.00  0.00      A    N  \\nATOM    748  CH2 TRP A  99      13.857  -5.874   0.578  1.00  0.00      A    C  \\nATOM    749  CZ2 TRP A  99      15.093  -5.729   1.154  1.00  0.00      A    C  \\nATOM    750  CZ3 TRP A  99      13.703  -5.978  -0.814  1.00  0.00      A    C  \\nATOM    751  N   SER A 100      18.741  -8.547  -5.307  1.00  0.00      A    N  \\nATOM    752  CA  SER A 100      19.147  -9.333  -6.468  1.00  0.00      A    C  \\nATOM    753  C   SER A 100      18.123  -9.227  -7.593  1.00  0.00      A    C  \\nATOM    754  CB  SER A 100      19.341 -10.800  -6.082  1.00  0.00      A    C  \\nATOM    755  O   SER A 100      18.435  -9.511  -8.752  1.00  0.00      A    O  \\nATOM    756  OG  SER A 100      18.094 -11.422  -5.826  1.00  0.00      A    O  \\nATOM    757  N   ALA A 101      17.211  -8.387  -7.763  1.00  0.00      A    N  \\nATOM    758  CA  ALA A 101      16.338  -8.649  -8.903  1.00  0.00      A    C  \\nATOM    759  C   ALA A 101      15.308  -7.535  -9.072  1.00  0.00      A    C  \\nATOM    760  CB  ALA A 101      15.638  -9.996  -8.740  1.00  0.00      A    C  \\nATOM    761  O   ALA A 101      14.858  -6.943  -8.089  1.00  0.00      A    O  \\nATOM    762  N   GLN A 102      15.437  -6.893 -10.584  1.00  0.00      A    N  \\nATOM    763  CA  GLN A 102      14.505  -6.355 -11.568  1.00  0.00      A    C  \\nATOM    764  C   GLN A 102      13.862  -5.064 -11.069  1.00  0.00      A    C  \\nATOM    765  CB  GLN A 102      13.424  -7.384 -11.904  1.00  0.00      A    C  \\nATOM    766  O   GLN A 102      13.409  -4.992  -9.925  1.00  0.00      A    O  \\nATOM    767  CG  GLN A 102      13.945  -8.600 -12.657  1.00  0.00      A    C  \\nATOM    768  CD  GLN A 102      12.841  -9.562 -13.054  1.00  0.00      A    C  \\nATOM    769  NE2 GLN A 102      13.222 -10.690 -13.645  1.00  0.00      A    N  \\nATOM    770  OE1 GLN A 102      11.657  -9.295 -12.829  1.00  0.00      A    O  \\nATOM    771  N   ASP A 103      14.542  -3.947 -11.177  1.00  0.00      A    N  \\nATOM    772  CA  ASP A 103      14.260  -2.525 -11.346  1.00  0.00      A    C  \\nATOM    773  C   ASP A 103      13.120  -2.079 -10.433  1.00  0.00      A    C  \\nATOM    774  CB  ASP A 103      13.916  -2.216 -12.805  1.00  0.00      A    C  \\nATOM    775  O   ASP A 103      12.161  -2.824 -10.220  1.00  0.00      A    O  \\nATOM    776  CG  ASP A 103      15.077  -2.463 -13.753  1.00  0.00      A    C  \\nATOM    777  OD1 ASP A 103      16.235  -2.550 -13.290  1.00  0.00      A    O  \\nATOM    778  OD2 ASP A 103      14.831  -2.570 -14.974  1.00  0.00      A    O  \\nATOM    779  N   PHE A 104      13.379  -1.221  -9.454  1.00  0.00      A    N  \\nATOM    780  CA  PHE A 104      12.415  -0.509  -8.623  1.00  0.00      A    C  \\nATOM    781  C   PHE A 104      11.468   0.321  -9.483  1.00  0.00      A    C  \\nATOM    782  CB  PHE A 104      13.135   0.392  -7.615  1.00  0.00      A    C  \\nATOM    783  O   PHE A 104      11.416   1.546  -9.353  1.00  0.00      A    O  \\nATOM    784  CG  PHE A 104      14.042  -0.353  -6.674  1.00  0.00      A    C  \\nATOM    785  CD1 PHE A 104      13.516  -1.169  -5.680  1.00  0.00      A    C  \\nATOM    786  CD2 PHE A 104      15.422  -0.238  -6.784  1.00  0.00      A    C  \\nATOM    787  CE1 PHE A 104      14.354  -1.861  -4.807  1.00  0.00      A    C  \\nATOM    788  CE2 PHE A 104      16.265  -0.926  -5.916  1.00  0.00      A    C  \\nATOM    789  CZ  PHE A 104      15.729  -1.736  -4.928  1.00  0.00      A    C  \\nATOM    790  N   LYS A 105      10.668  -0.508 -10.364  1.00  0.00      A    N  \\nATOM    791  CA  LYS A 105       9.779   0.228 -11.259  1.00  0.00      A    C  \\nATOM    792  C   LYS A 105       8.317  -0.114 -10.985  1.00  0.00      A    C  \\nATOM    793  CB  LYS A 105      10.122  -0.070 -12.720  1.00  0.00      A    C  \\nATOM    794  O   LYS A 105       7.414   0.486 -11.573  1.00  0.00      A    O  \\nATOM    795  CG  LYS A 105      11.478   0.463 -13.158  1.00  0.00      A    C  \\nATOM    796  CD  LYS A 105      11.757   0.146 -14.622  1.00  0.00      A    C  \\nATOM    797  CE  LYS A 105      13.120   0.666 -15.058  1.00  0.00      A    C  \\nATOM    798  NZ  LYS A 105      13.409   0.334 -16.486  1.00  0.00      A    N  \\nATOM    799  N   SER A 106       8.142  -1.126 -10.198  1.00  0.00      A    N  \\nATOM    800  CA  SER A 106       6.765  -1.467  -9.854  1.00  0.00      A    C  \\nATOM    801  C   SER A 106       6.195  -0.492  -8.829  1.00  0.00      A    C  \\nATOM    802  CB  SER A 106       6.685  -2.894  -9.312  1.00  0.00      A    C  \\nATOM    803  O   SER A 106       6.855  -0.166  -7.840  1.00  0.00      A    O  \\nATOM    804  OG  SER A 106       7.023  -3.834 -10.318  1.00  0.00      A    O  \\nATOM    805  N   PRO A 107       4.998  -0.028  -9.073  1.00  0.00      A    N  \\nATOM    806  CA  PRO A 107       4.400   0.891  -8.101  1.00  0.00      A    C  \\nATOM    807  C   PRO A 107       3.693   0.164  -6.959  1.00  0.00      A    C  \\nATOM    808  CB  PRO A 107       3.401   1.691  -8.941  1.00  0.00      A    C  \\nATOM    809  O   PRO A 107       2.709  -0.543  -7.189  1.00  0.00      A    O  \\nATOM    810  CG  PRO A 107       3.007   0.768 -10.048  1.00  0.00      A    C  \\nATOM    811  CD  PRO A 107       4.163  -0.143 -10.344  1.00  0.00      A    C  \\nATOM    812  N   GLU A 108       4.324   0.206  -5.719  1.00  0.00      A    N  \\nATOM    813  CA  GLU A 108       3.566  -0.113  -4.513  1.00  0.00      A    C  \\nATOM    814  C   GLU A 108       2.543   0.975  -4.200  1.00  0.00      A    C  \\nATOM    815  CB  GLU A 108       4.508  -0.307  -3.322  1.00  0.00      A    C  \\nATOM    816  O   GLU A 108       2.864   2.164  -4.231  1.00  0.00      A    O  \\nATOM    817  CG  GLU A 108       3.791  -0.614  -2.015  1.00  0.00      A    C  \\nATOM    818  CD  GLU A 108       2.886  -1.833  -2.099  1.00  0.00      A    C  \\nATOM    819  OE1 GLU A 108       3.151  -2.729  -2.931  1.00  0.00      A    O  \\nATOM    820  OE2 GLU A 108       1.904  -1.891  -1.326  1.00  0.00      A    O  \\nATOM    821  N   GLU A 109       1.364   0.549  -3.942  1.00  0.00      A    N  \\nATOM    822  CA  GLU A 109       0.241   1.455  -3.723  1.00  0.00      A    C  \\nATOM    823  C   GLU A 109      -0.373   1.250  -2.341  1.00  0.00      A    C  \\nATOM    824  CB  GLU A 109      -0.824   1.262  -4.805  1.00  0.00      A    C  \\nATOM    825  O   GLU A 109      -0.709   0.124  -1.966  1.00  0.00      A    O  \\nATOM    826  CG  GLU A 109      -2.062   2.126  -4.611  1.00  0.00      A    C  \\nATOM    827  CD  GLU A 109      -3.151   1.853  -5.636  1.00  0.00      A    C  \\nATOM    828  OE1 GLU A 109      -3.097   0.802  -6.314  1.00  0.00      A    O  \\nATOM    829  OE2 GLU A 109      -4.065   2.699  -5.764  1.00  0.00      A    O  \\nATOM    830  N   TYR A 110      -0.426   2.404  -1.607  1.00  0.00      A    N  \\nATOM    831  CA  TYR A 110      -1.193   2.394  -0.366  1.00  0.00      A    C  \\nATOM    832  C   TYR A 110      -2.153   3.576  -0.309  1.00  0.00      A    C  \\nATOM    833  CB  TYR A 110      -0.257   2.423   0.845  1.00  0.00      A    C  \\nATOM    834  O   TYR A 110      -1.758   4.717  -0.559  1.00  0.00      A    O  \\nATOM    835  CG  TYR A 110      -0.946   2.117   2.153  1.00  0.00      A    C  \\nATOM    836  CD1 TYR A 110      -1.117   0.802   2.581  1.00  0.00      A    C  \\nATOM    837  CD2 TYR A 110      -1.428   3.140   2.962  1.00  0.00      A    C  \\nATOM    838  CE1 TYR A 110      -1.754   0.515   3.784  1.00  0.00      A    C  \\nATOM    839  CE2 TYR A 110      -2.066   2.865   4.167  1.00  0.00      A    C  \\nATOM    840  OH  TYR A 110      -2.854   1.273   5.761  1.00  0.00      A    O  \\nATOM    841  CZ  TYR A 110      -2.223   1.551   4.569  1.00  0.00      A    C  \\nATOM    842  N   ALA A 111      -3.421   3.423   0.191  1.00  0.00      A    N  \\nATOM    843  CA  ALA A 111      -4.019   2.175   0.658  1.00  0.00      A    C  \\nATOM    844  C   ALA A 111      -4.091   1.147  -0.467  1.00  0.00      A    C  \\nATOM    845  CB  ALA A 111      -5.411   2.432   1.229  1.00  0.00      A    C  \\nATOM    846  O   ALA A 111      -4.005   1.500  -1.646  1.00  0.00      A    O  \\nATOM    847  N   PHE A 112      -4.261  -0.134  -0.088  1.00  0.00      A    N  \\nATOM    848  CA  PHE A 112      -4.212  -1.247  -1.028  1.00  0.00      A    C  \\nATOM    849  C   PHE A 112      -5.304  -1.114  -2.083  1.00  0.00      A    C  \\nATOM    850  CB  PHE A 112      -4.354  -2.583  -0.291  1.00  0.00      A    C  \\nATOM    851  O   PHE A 112      -6.442  -0.762  -1.765  1.00  0.00      A    O  \\nATOM    852  CG  PHE A 112      -3.289  -2.820   0.745  1.00  0.00      A    C  \\nATOM    853  CD1 PHE A 112      -1.943  -2.716   0.416  1.00  0.00      A    C  \\nATOM    854  CD2 PHE A 112      -3.634  -3.148   2.050  1.00  0.00      A    C  \\nATOM    855  CE1 PHE A 112      -0.955  -2.935   1.373  1.00  0.00      A    C  \\nATOM    856  CE2 PHE A 112      -2.653  -3.368   3.012  1.00  0.00      A    C  \\nATOM    857  CZ  PHE A 112      -1.314  -3.263   2.671  1.00  0.00      A    C  \\nATOM    858  N   ALA A 113      -4.940  -1.444  -3.341  1.00  0.00      A    N  \\nATOM    859  CA  ALA A 113      -5.949  -1.477  -4.396  1.00  0.00      A    C  \\nATOM    860  C   ALA A 113      -7.088  -2.426  -4.037  1.00  0.00      A    C  \\nATOM    861  CB  ALA A 113      -5.318  -1.886  -5.725  1.00  0.00      A    C  \\nATOM    862  O   ALA A 113      -8.234  -2.213  -4.442  1.00  0.00      A    O  \\nATOM    863  N   SER A 114      -6.831  -3.400  -3.232  1.00  0.00      A    N  \\nATOM    864  CA  SER A 114      -7.807  -4.421  -2.865  1.00  0.00      A    C  \\nATOM    865  C   SER A 114      -8.740  -3.925  -1.765  1.00  0.00      A    C  \\nATOM    866  CB  SER A 114      -7.100  -5.698  -2.407  1.00  0.00      A    C  \\nATOM    867  O   SER A 114      -9.564  -4.685  -1.253  1.00  0.00      A    O  \\nATOM    868  OG  SER A 114      -6.254  -5.436  -1.300  1.00  0.00      A    O  \\nATOM    869  N   SER A 115      -8.705  -2.657  -1.400  1.00  0.00      A    N  \\nATOM    870  CA  SER A 115      -9.528  -2.126  -0.318  1.00  0.00      A    C  \\nATOM    871  C   SER A 115     -10.412  -0.984  -0.806  1.00  0.00      A    C  \\nATOM    872  CB  SER A 115      -8.649  -1.645   0.837  1.00  0.00      A    C  \\nATOM    873  O   SER A 115     -10.135  -0.376  -1.843  1.00  0.00      A    O  \\nATOM    874  OG  SER A 115      -7.940  -0.472   0.475  1.00  0.00      A    O  \\nATOM    875  N   LEU A 116     -11.446  -0.746  -0.055  1.00  0.00      A    N  \\nATOM    876  CA  LEU A 116     -12.382   0.341  -0.318  1.00  0.00      A    C  \\nATOM    877  C   LEU A 116     -11.679   1.693  -0.247  1.00  0.00      A    C  \\nATOM    878  CB  LEU A 116     -13.542   0.303   0.680  1.00  0.00      A    C  \\nATOM    879  O   LEU A 116     -12.026   2.618  -0.985  1.00  0.00      A    O  \\nATOM    880  CG  LEU A 116     -14.649   1.339   0.474  1.00  0.00      A    C  \\nATOM    881  CD1 LEU A 116     -15.319   1.135  -0.881  1.00  0.00      A    C  \\nATOM    882  CD2 LEU A 116     -15.673   1.260   1.601  1.00  0.00      A    C  \\nATOM    883  N   GLN A 117     -10.656   1.784   0.529  1.00  0.00      A    N  \\nATOM    884  CA  GLN A 117      -9.907   3.016   0.759  1.00  0.00      A    C  \\nATOM    885  C   GLN A 117      -8.907   3.269  -0.365  1.00  0.00      A    C  \\nATOM    886  CB  GLN A 117      -9.182   2.960   2.105  1.00  0.00      A    C  \\nATOM    887  O   GLN A 117      -8.295   4.337  -0.430  1.00  0.00      A    O  \\nATOM    888  CG  GLN A 117     -10.116   2.989   3.308  1.00  0.00      A    C  \\nATOM    889  CD  GLN A 117     -10.771   1.647   3.575  1.00  0.00      A    C  \\nATOM    890  NE2 GLN A 117     -11.790   1.644   4.427  1.00  0.00      A    N  \\nATOM    891  OE1 GLN A 117     -10.364   0.622   3.019  1.00  0.00      A    O  \\nATOM    892  N   GLY A 118      -8.709   2.212  -1.232  1.00  0.00      A    N  \\nATOM    893  CA  GLY A 118      -7.748   2.333  -2.317  1.00  0.00      A    C  \\nATOM    894  C   GLY A 118      -8.236   3.217  -3.450  1.00  0.00      A    C  \\nATOM    895  O   GLY A 118      -9.401   3.621  -3.471  1.00  0.00      A    O  \\nATOM    896  N   GLY A 119      -7.366   3.468  -4.358  1.00  0.00      A    N  \\nATOM    897  CA  GLY A 119      -7.707   4.281  -5.514  1.00  0.00      A    C  \\nATOM    898  C   GLY A 119      -7.199   5.707  -5.411  1.00  0.00      A    C  \\nATOM    899  O   GLY A 119      -6.009   5.933  -5.185  1.00  0.00      A    O  \\nATOM    900  N   THR A 120      -8.182   6.653  -5.354  1.00  0.00      A    N  \\nATOM    901  CA  THR A 120      -7.807   8.063  -5.341  1.00  0.00      A    C  \\nATOM    902  C   THR A 120      -7.020   8.401  -4.078  1.00  0.00      A    C  \\nATOM    903  CB  THR A 120      -9.046   8.972  -5.436  1.00  0.00      A    C  \\nATOM    904  O   THR A 120      -7.311   7.876  -3.000  1.00  0.00      A    O  \\nATOM    905  CG2 THR A 120      -8.645  10.442  -5.508  1.00  0.00      A    C  \\nATOM    906  OG1 THR A 120      -9.790   8.633  -6.613  1.00  0.00      A    O  \\nATOM    907  N   ASN A 121      -5.951   9.115  -4.149  1.00  0.00      A    N  \\nATOM    908  CA  ASN A 121      -5.114   9.615  -3.063  1.00  0.00      A    C  \\nATOM    909  C   ASN A 121      -4.183   8.531  -2.528  1.00  0.00      A    C  \\nATOM    910  CB  ASN A 121      -5.980  10.177  -1.933  1.00  0.00      A    C  \\nATOM    911  O   ASN A 121      -3.550   8.710  -1.486  1.00  0.00      A    O  \\nATOM    912  CG  ASN A 121      -6.766  11.403  -2.354  1.00  0.00      A    C  \\nATOM    913  ND2 ASN A 121      -7.985  11.531  -1.843  1.00  0.00      A    N  \\nATOM    914  OD1 ASN A 121      -6.282  12.229  -3.133  1.00  0.00      A    O  \\nATOM    915  N   ALA A 122      -4.192   7.385  -3.252  1.00  0.00      A    N  \\nATOM    916  CA  ALA A 122      -3.197   6.391  -2.857  1.00  0.00      A    C  \\nATOM    917  C   ALA A 122      -1.781   6.937  -3.019  1.00  0.00      A    C  \\nATOM    918  CB  ALA A 122      -3.367   5.113  -3.675  1.00  0.00      A    C  \\nATOM    919  O   ALA A 122      -1.525   7.765  -3.896  1.00  0.00      A    O  \\nATOM    920  N   ILE A 123      -0.950   6.553  -2.112  1.00  0.00      A    N  \\nATOM    921  CA  ILE A 123       0.473   6.863  -2.195  1.00  0.00      A    C  \\nATOM    922  C   ILE A 123       1.201   5.751  -2.946  1.00  0.00      A    C  \\nATOM    923  CB  ILE A 123       1.092   7.057  -0.792  1.00  0.00      A    C  \\nATOM    924  O   ILE A 123       1.038   4.570  -2.629  1.00  0.00      A    O  \\nATOM    925  CG1 ILE A 123       0.486   8.287  -0.107  1.00  0.00      A    C  \\nATOM    926  CG2 ILE A 123       2.616   7.177  -0.887  1.00  0.00      A    C  \\nATOM    927  CD1 ILE A 123       0.951   8.486   1.330  1.00  0.00      A    C  \\nATOM    928  N   LEU A 124       1.945   6.199  -3.930  1.00  0.00      A    N  \\nATOM    929  CA  LEU A 124       2.731   5.263  -4.727  1.00  0.00      A    C  \\nATOM    930  C   LEU A 124       4.222   5.441  -4.462  1.00  0.00      A    C  \\nATOM    931  CB  LEU A 124       2.439   5.451  -6.218  1.00  0.00      A    C  \\nATOM    932  O   LEU A 124       4.703   6.569  -4.333  1.00  0.00      A    O  \\nATOM    933  CG  LEU A 124       1.063   4.992  -6.703  1.00  0.00      A    C  \\nATOM    934  CD1 LEU A 124       0.060   6.138  -6.615  1.00  0.00      A    C  \\nATOM    935  CD2 LEU A 124       1.148   4.458  -8.128  1.00  0.00      A    C  \\nATOM    936  N   ALA A 125       4.919   4.360  -4.348  1.00  0.00      A    N  \\nATOM    937  CA  ALA A 125       6.377   4.353  -4.271  1.00  0.00      A    C  \\nATOM    938  C   ALA A 125       6.963   3.195  -5.074  1.00  0.00      A    C  \\nATOM    939  CB  ALA A 125       6.833   4.270  -2.816  1.00  0.00      A    C  \\nATOM    940  O   ALA A 125       6.339   2.139  -5.199  1.00  0.00      A    O  \\nATOM    941  N   PRO A 126       8.136   3.451  -5.652  1.00  0.00      A    N  \\nATOM    942  CA  PRO A 126       8.739   2.384  -6.454  1.00  0.00      A    C  \\nATOM    943  C   PRO A 126       9.328   1.264  -5.598  1.00  0.00      A    C  \\nATOM    944  CB  PRO A 126       9.837   3.108  -7.237  1.00  0.00      A    C  \\nATOM    945  O   PRO A 126      10.011   1.534  -4.607  1.00  0.00      A    O  \\nATOM    946  CG  PRO A 126      10.242   4.249  -6.360  1.00  0.00      A    C  \\nATOM    947  CD  PRO A 126       9.037   4.713  -5.593  1.00  0.00      A    C  \\nATOM    948  N   VAL A 127       8.957   0.066  -5.981  1.00  0.00      A    N  \\nATOM    949  CA  VAL A 127       9.513  -1.137  -5.372  1.00  0.00      A    C  \\nATOM    950  C   VAL A 127       9.923  -2.125  -6.462  1.00  0.00      A    C  \\nATOM    951  CB  VAL A 127       8.509  -1.800  -4.403  1.00  0.00      A    C  \\nATOM    952  O   VAL A 127       9.720  -1.867  -7.651  1.00  0.00      A    O  \\nATOM    953  CG1 VAL A 127       8.069  -0.813  -3.323  1.00  0.00      A    C  \\nATOM    954  CG2 VAL A 127       7.300  -2.332  -5.169  1.00  0.00      A    C  \\nATOM    955  N   ASN A 128      10.604  -3.192  -6.147  1.00  0.00      A    N  \\nATOM    956  CA  ASN A 128      10.880  -4.232  -7.132  1.00  0.00      A    C  \\nATOM    957  C   ASN A 128       9.740  -5.243  -7.214  1.00  0.00      A    C  \\nATOM    958  CB  ASN A 128      12.197  -4.941  -6.810  1.00  0.00      A    C  \\nATOM    959  O   ASN A 128       8.991  -5.419  -6.251  1.00  0.00      A    O  \\nATOM    960  CG  ASN A 128      12.125  -5.753  -5.531  1.00  0.00      A    C  \\nATOM    961  ND2 ASN A 128      12.957  -6.783  -5.435  1.00  0.00      A    N  \\nATOM    962  OD1 ASN A 128      11.328  -5.457  -4.637  1.00  0.00      A    O  \\nATOM    963  N   LEU A 129       9.626  -5.902  -8.400  1.00  0.00      A    N  \\nATOM    964  CA  LEU A 129       8.505  -6.784  -8.707  1.00  0.00      A    C  \\nATOM    965  C   LEU A 129       8.478  -7.979  -7.759  1.00  0.00      A    C  \\nATOM    966  CB  LEU A 129       8.587  -7.271 -10.156  1.00  0.00      A    C  \\nATOM    967  O   LEU A 129       7.409  -8.391  -7.303  1.00  0.00      A    O  \\nATOM    968  CG  LEU A 129       7.464  -8.197 -10.625  1.00  0.00      A    C  \\nATOM    969  CD1 LEU A 129       6.112  -7.507 -10.480  1.00  0.00      A    C  \\nATOM    970  CD2 LEU A 129       7.697  -8.632 -12.068  1.00  0.00      A    C  \\nATOM    971  N   ALA A 130       9.624  -8.485  -7.391  1.00  0.00      A    N  \\nATOM    972  CA  ALA A 130       9.695  -9.628  -6.484  1.00  0.00      A    C  \\nATOM    973  C   ALA A 130       9.147  -9.269  -5.106  1.00  0.00      A    C  \\nATOM    974  CB  ALA A 130      11.133 -10.126  -6.368  1.00  0.00      A    C  \\nATOM    975  O   ALA A 130       8.409 -10.052  -4.502  1.00  0.00      A    O  \\nATOM    976  N   SER A 131       9.508  -8.097  -4.634  1.00  0.00      A    N  \\nATOM    977  CA  SER A 131       9.026  -7.656  -3.329  1.00  0.00      A    C  \\nATOM    978  C   SER A 131       7.517  -7.434  -3.343  1.00  0.00      A    C  \\nATOM    979  CB  SER A 131       9.734  -6.371  -2.901  1.00  0.00      A    C  \\nATOM    980  O   SER A 131       6.829  -7.748  -2.370  1.00  0.00      A    O  \\nATOM    981  OG  SER A 131       9.342  -5.283  -3.721  1.00  0.00      A    O  \\nATOM    982  N   GLN A 132       7.098  -6.938  -4.452  1.00  0.00      A    N  \\nATOM    983  CA  GLN A 132       5.661  -6.738  -4.611  1.00  0.00      A    C  \\nATOM    984  C   GLN A 132       4.917  -8.070  -4.610  1.00  0.00      A    C  \\nATOM    985  CB  GLN A 132       5.365  -5.972  -5.901  1.00  0.00      A    C  \\nATOM    986  O   GLN A 132       3.869  -8.201  -3.973  1.00  0.00      A    O  \\nATOM    987  CG  GLN A 132       3.964  -5.378  -5.956  1.00  0.00      A    C  \\nATOM    988  CD  GLN A 132       3.747  -4.496  -7.171  1.00  0.00      A    C  \\nATOM    989  NE2 GLN A 132       2.622  -3.789  -7.198  1.00  0.00      A    N  \\nATOM    990  OE1 GLN A 132       4.584  -4.449  -8.078  1.00  0.00      A    O  \\nATOM    991  N   ASN A 133       5.428  -9.054  -5.292  1.00  0.00      A    N  \\nATOM    992  CA  ASN A 133       4.818 -10.379  -5.338  1.00  0.00      A    C  \\nATOM    993  C   ASN A 133       4.813 -11.042  -3.963  1.00  0.00      A    C  \\nATOM    994  CB  ASN A 133       5.541 -11.268  -6.353  1.00  0.00      A    C  \\nATOM    995  O   ASN A 133       3.832 -11.683  -3.582  1.00  0.00      A    O  \\nATOM    996  CG  ASN A 133       5.255 -10.868  -7.787  1.00  0.00      A    C  \\nATOM    997  ND2 ASN A 133       6.083 -11.342  -8.711  1.00  0.00      A    N  \\nATOM    998  OD1 ASN A 133       4.298 -10.139  -8.062  1.00  0.00      A    O  \\nATOM    999  N   SER A 134       5.942 -10.856  -3.233  1.00  0.00      A    N  \\nATOM   1000  CA  SER A 134       6.035 -11.376  -1.873  1.00  0.00      A    C  \\nATOM   1001  C   SER A 134       4.985 -10.743  -0.966  1.00  0.00      A    C  \\nATOM   1002  CB  SER A 134       7.431 -11.132  -1.298  1.00  0.00      A    C  \\nATOM   1003  O   SER A 134       4.318 -11.440  -0.199  1.00  0.00      A    O  \\nATOM   1004  OG  SER A 134       7.564 -11.737  -0.024  1.00  0.00      A    O  \\nATOM   1005  N   GLN A 135       4.801  -9.427  -1.078  1.00  0.00      A    N  \\nATOM   1006  CA  GLN A 135       3.784  -8.719  -0.307  1.00  0.00      A    C  \\nATOM   1007  C   GLN A 135       2.385  -9.229  -0.642  1.00  0.00      A    C  \\nATOM   1008  CB  GLN A 135       3.868  -7.213  -0.560  1.00  0.00      A    C  \\nATOM   1009  O   GLN A 135       1.572  -9.464   0.254  1.00  0.00      A    O  \\nATOM   1010  CG  GLN A 135       2.849  -6.400   0.226  1.00  0.00      A    C  \\nATOM   1011  CD  GLN A 135       2.602  -5.029  -0.376  1.00  0.00      A    C  \\nATOM   1012  NE2 GLN A 135       2.604  -4.002   0.467  1.00  0.00      A    N  \\nATOM   1013  OE1 GLN A 135       2.411  -4.895  -1.589  1.00  0.00      A    O  \\nATOM   1014  N   GLY A 136       2.071  -9.397  -1.922  1.00  0.00      A    N  \\nATOM   1015  CA  GLY A 136       0.785  -9.936  -2.338  1.00  0.00      A    C  \\nATOM   1016  C   GLY A 136       0.493 -11.303  -1.750  1.00  0.00      A    C  \\nATOM   1017  O   GLY A 136      -0.643 -11.589  -1.365  1.00  0.00      A    O  \\nATOM   1018  N   GLY A 137       1.538 -12.133  -1.709  1.00  0.00      A    N  \\nATOM   1019  CA  GLY A 137       1.387 -13.439  -1.087  1.00  0.00      A    C  \\nATOM   1020  C   GLY A 137       1.035 -13.361   0.387  1.00  0.00      A    C  \\nATOM   1021  O   GLY A 137       0.178 -14.106   0.866  1.00  0.00      A    O  \\nATOM   1022  N   VAL A 138       1.726 -12.488   1.128  1.00  0.00      A    N  \\nATOM   1023  CA  VAL A 138       1.468 -12.322   2.554  1.00  0.00      A    C  \\nATOM   1024  C   VAL A 138       0.039 -11.826   2.767  1.00  0.00      A    C  \\nATOM   1025  CB  VAL A 138       2.473 -11.345   3.204  1.00  0.00      A    C  \\nATOM   1026  O   VAL A 138      -0.680 -12.336   3.628  1.00  0.00      A    O  \\nATOM   1027  CG1 VAL A 138       2.054 -11.009   4.634  1.00  0.00      A    C  \\nATOM   1028  CG2 VAL A 138       3.881 -11.937   3.181  1.00  0.00      A    C  \\nATOM   1029  N   LEU A 139      -0.439 -10.907   1.983  1.00  0.00      A    N  \\nATOM   1030  CA  LEU A 139      -1.774 -10.338   2.136  1.00  0.00      A    C  \\nATOM   1031  C   LEU A 139      -2.846 -11.361   1.775  1.00  0.00      A    C  \\nATOM   1032  CB  LEU A 139      -1.928  -9.091   1.262  1.00  0.00      A    C  \\nATOM   1033  O   LEU A 139      -3.851 -11.488   2.479  1.00  0.00      A    O  \\nATOM   1034  CG  LEU A 139      -1.284  -7.808   1.792  1.00  0.00      A    C  \\nATOM   1035  CD1 LEU A 139      -1.365  -6.703   0.744  1.00  0.00      A    C  \\nATOM   1036  CD2 LEU A 139      -1.952  -7.371   3.091  1.00  0.00      A    C  \\nATOM   1037  N   ASN A 140      -2.629 -12.076   0.698  1.00  0.00      A    N  \\nATOM   1038  CA  ASN A 140      -3.588 -13.105   0.310  1.00  0.00      A    C  \\nATOM   1039  C   ASN A 140      -3.694 -14.199   1.368  1.00  0.00      A    C  \\nATOM   1040  CB  ASN A 140      -3.209 -13.709  -1.043  1.00  0.00      A    C  \\nATOM   1041  O   ASN A 140      -4.783 -14.718   1.624  1.00  0.00      A    O  \\nATOM   1042  CG  ASN A 140      -3.523 -12.787  -2.204  1.00  0.00      A    C  \\nATOM   1043  ND2 ASN A 140      -2.825 -12.975  -3.318  1.00  0.00      A    N  \\nATOM   1044  OD1 ASN A 140      -4.386 -11.911  -2.101  1.00  0.00      A    O  \\nATOM   1045  N   GLY A 141      -2.556 -14.550   1.928  1.00  0.00      A    N  \\nATOM   1046  CA  GLY A 141      -2.578 -15.478   3.048  1.00  0.00      A    C  \\nATOM   1047  C   GLY A 141      -3.394 -14.975   4.223  1.00  0.00      A    C  \\nATOM   1048  O   GLY A 141      -4.170 -15.729   4.815  1.00  0.00      A    O  \\nATOM   1049  N   PHE A 142      -3.220 -13.690   4.514  1.00  0.00      A    N  \\nATOM   1050  CA  PHE A 142      -3.996 -13.060   5.577  1.00  0.00      A    C  \\nATOM   1051  C   PHE A 142      -5.484 -13.083   5.247  1.00  0.00      A    C  \\nATOM   1052  CB  PHE A 142      -3.530 -11.618   5.801  1.00  0.00      A    C  \\nATOM   1053  O   PHE A 142      -6.306 -13.443   6.091  1.00  0.00      A    O  \\nATOM   1054  CG  PHE A 142      -4.479 -10.796   6.630  1.00  0.00      A    C  \\nATOM   1055  CD1 PHE A 142      -4.584 -11.001   8.001  1.00  0.00      A    C  \\nATOM   1056  CD2 PHE A 142      -5.267  -9.817   6.039  1.00  0.00      A    C  \\nATOM   1057  CE1 PHE A 142      -5.462 -10.241   8.770  1.00  0.00      A    C  \\nATOM   1058  CE2 PHE A 142      -6.147  -9.054   6.801  1.00  0.00      A    C  \\nATOM   1059  CZ  PHE A 142      -6.243  -9.268   8.167  1.00  0.00      A    C  \\nATOM   1060  N   TYR A 143      -5.881 -12.725   4.046  1.00  0.00      A    N  \\nATOM   1061  CA  TYR A 143      -7.281 -12.705   3.637  1.00  0.00      A    C  \\nATOM   1062  C   TYR A 143      -7.908 -14.087   3.773  1.00  0.00      A    C  \\nATOM   1063  CB  TYR A 143      -7.412 -12.211   2.193  1.00  0.00      A    C  \\nATOM   1064  O   TYR A 143      -9.029 -14.220   4.271  1.00  0.00      A    O  \\nATOM   1065  CG  TYR A 143      -7.078 -10.750   2.018  1.00  0.00      A    C  \\nATOM   1066  CD1 TYR A 143      -7.272  -9.842   3.057  1.00  0.00      A    C  \\nATOM   1067  CD2 TYR A 143      -6.570 -10.273   0.814  1.00  0.00      A    C  \\nATOM   1068  CE1 TYR A 143      -6.967  -8.494   2.900  1.00  0.00      A    C  \\nATOM   1069  CE2 TYR A 143      -6.262  -8.927   0.646  1.00  0.00      A    C  \\nATOM   1070  OH  TYR A 143      -6.161  -6.713   1.532  1.00  0.00      A    O  \\nATOM   1071  CZ  TYR A 143      -6.464  -8.047   1.693  1.00  0.00      A    C  \\nATOM   1072  N   SER A 144      -7.198 -15.112   3.343  1.00  0.00      A    N  \\nATOM   1073  CA  SER A 144      -7.711 -16.478   3.377  1.00  0.00      A    C  \\nATOM   1074  C   SER A 144      -7.803 -16.998   4.808  1.00  0.00      A    C  \\nATOM   1075  CB  SER A 144      -6.824 -17.405   2.545  1.00  0.00      A    C  \\nATOM   1076  O   SER A 144      -8.844 -17.513   5.221  1.00  0.00      A    O  \\nATOM   1077  OG  SER A 144      -7.320 -18.733   2.572  1.00  0.00      A    O  \\nATOM   1078  N   ALA A 145      -6.764 -16.844   5.590  1.00  0.00      A    N  \\nATOM   1079  CA  ALA A 145      -6.682 -17.369   6.951  1.00  0.00      A    C  \\nATOM   1080  C   ALA A 145      -7.736 -16.728   7.850  1.00  0.00      A    C  \\nATOM   1081  CB  ALA A 145      -5.287 -17.141   7.526  1.00  0.00      A    C  \\nATOM   1082  O   ALA A 145      -8.218 -17.355   8.796  1.00  0.00      A    O  \\nATOM   1083  N   ASN A 146      -8.131 -15.428   7.488  1.00  0.00      A    N  \\nATOM   1084  CA  ASN A 146      -9.052 -14.691   8.346  1.00  0.00      A    C  \\nATOM   1085  C   ASN A 146     -10.420 -14.525   7.690  1.00  0.00      A    C  \\nATOM   1086  CB  ASN A 146      -8.469 -13.324   8.711  1.00  0.00      A    C  \\nATOM   1087  O   ASN A 146     -11.249 -13.746   8.161  1.00  0.00      A    O  \\nATOM   1088  CG  ASN A 146      -7.276 -13.427   9.641  1.00  0.00      A    C  \\nATOM   1089  ND2 ASN A 146      -6.077 -13.355   9.077  1.00  0.00      A    N  \\nATOM   1090  OD1 ASN A 146      -7.432 -13.572  10.856  1.00  0.00      A    O  \\nATOM   1091  N   LYS A 147     -10.597 -15.252   6.562  1.00  0.00      A    N  \\nATOM   1092  CA  LYS A 147     -11.862 -15.262   5.833  1.00  0.00      A    C  \\nATOM   1093  C   LYS A 147     -12.368 -13.842   5.593  1.00  0.00      A    C  \\nATOM   1094  CB  LYS A 147     -12.913 -16.072   6.592  1.00  0.00      A    C  \\nATOM   1095  O   LYS A 147     -13.535 -13.542   5.854  1.00  0.00      A    O  \\nATOM   1096  CG  LYS A 147     -12.535 -17.530   6.806  1.00  0.00      A    C  \\nATOM   1097  CD  LYS A 147     -13.625 -18.287   7.554  1.00  0.00      A    C  \\nATOM   1098  CE  LYS A 147     -13.210 -19.722   7.849  1.00  0.00      A    C  \\nATOM   1099  NZ  LYS A 147     -14.231 -20.436   8.673  1.00  0.00      A    N  \\nATOM   1100  N   VAL A 148     -11.471 -12.991   5.179  1.00  0.00      A    N  \\nATOM   1101  CA  VAL A 148     -11.786 -11.597   4.880  1.00  0.00      A    C  \\nATOM   1102  C   VAL A 148     -12.640 -11.520   3.617  1.00  0.00      A    C  \\nATOM   1103  CB  VAL A 148     -10.506 -10.750   4.711  1.00  0.00      A    C  \\nATOM   1104  O   VAL A 148     -12.145 -11.747   2.510  1.00  0.00      A    O  \\nATOM   1105  CG1 VAL A 148     -10.857  -9.305   4.360  1.00  0.00      A    C  \\nATOM   1106  CG2 VAL A 148      -9.658 -10.804   5.980  1.00  0.00      A    C  \\nATOM   1107  N   ALA A 149     -13.919 -11.250   3.769  1.00  0.00      A    N  \\nATOM   1108  CA  ALA A 149     -14.884 -11.220   2.674  1.00  0.00      A    C  \\nATOM   1109  C   ALA A 149     -14.751  -9.936   1.860  1.00  0.00      A    C  \\nATOM   1110  CB  ALA A 149     -16.306 -11.360   3.212  1.00  0.00      A    C  \\nATOM   1111  O   ALA A 149     -14.367  -8.892   2.394  1.00  0.00      A    O  \\nATOM   1112  N   GLN A 150     -15.105 -10.064   0.554  1.00  0.00      A    N  \\nATOM   1113  CA  GLN A 150     -15.017  -8.931  -0.361  1.00  0.00      A    C  \\nATOM   1114  C   GLN A 150     -16.319  -8.135  -0.377  1.00  0.00      A    C  \\nATOM   1115  CB  GLN A 150     -14.676  -9.407  -1.774  1.00  0.00      A    C  \\nATOM   1116  O   GLN A 150     -17.407  -8.713  -0.347  1.00  0.00      A    O  \\nATOM   1117  CG  GLN A 150     -13.315 -10.083  -1.883  1.00  0.00      A    C  \\nATOM   1118  CD  GLN A 150     -13.071 -10.692  -3.251  1.00  0.00      A    C  \\nATOM   1119  NE2 GLN A 150     -11.897 -11.286  -3.435  1.00  0.00      A    N  \\nATOM   1120  OE1 GLN A 150     -13.931 -10.630  -4.136  1.00  0.00      A    O  \\nATOM   1121  N   PHE A 151     -16.214  -6.897  -0.297  1.00  0.00      A    N  \\nATOM   1122  CA  PHE A 151     -17.286  -5.924  -0.470  1.00  0.00      A    C  \\nATOM   1123  C   PHE A 151     -17.416  -5.517  -1.933  1.00  0.00      A    C  \\nATOM   1124  CB  PHE A 151     -17.038  -4.688   0.399  1.00  0.00      A    C  \\nATOM   1125  O   PHE A 151     -16.412  -5.295  -2.612  1.00  0.00      A    O  \\nATOM   1126  CG  PHE A 151     -18.066  -3.603   0.220  1.00  0.00      A    C  \\nATOM   1127  CD1 PHE A 151     -19.376  -3.791   0.643  1.00  0.00      A    C  \\nATOM   1128  CD2 PHE A 151     -17.723  -2.395  -0.373  1.00  0.00      A    C  \\nATOM   1129  CE1 PHE A 151     -20.330  -2.790   0.478  1.00  0.00      A    C  \\nATOM   1130  CE2 PHE A 151     -18.670  -1.389  -0.541  1.00  0.00      A    C  \\nATOM   1131  CZ  PHE A 151     -19.973  -1.589  -0.116  1.00  0.00      A    C  \\nATOM   1132  N   ASP A 152     -18.623  -5.518  -2.423  1.00  0.00      A    N  \\nATOM   1133  CA  ASP A 152     -18.936  -4.972  -3.740  1.00  0.00      A    C  \\nATOM   1134  C   ASP A 152     -19.596  -3.600  -3.623  1.00  0.00      A    C  \\nATOM   1135  CB  ASP A 152     -19.846  -5.928  -4.515  1.00  0.00      A    C  \\nATOM   1136  O   ASP A 152     -20.774  -3.499  -3.276  1.00  0.00      A    O  \\nATOM   1137  CG  ASP A 152     -19.997  -5.549  -5.977  1.00  0.00      A    C  \\nATOM   1138  OD1 ASP A 152     -19.697  -4.391  -6.341  1.00  0.00      A    O  \\nATOM   1139  OD2 ASP A 152     -20.422  -6.414  -6.773  1.00  0.00      A    O  \\nATOM   1140  N   PRO A 153     -18.827  -2.512  -3.857  1.00  0.00      A    N  \\nATOM   1141  CA  PRO A 153     -19.387  -1.170  -3.680  1.00  0.00      A    C  \\nATOM   1142  C   PRO A 153     -20.583  -0.904  -4.591  1.00  0.00      A    C  \\nATOM   1143  CB  PRO A 153     -18.215  -0.250  -4.034  1.00  0.00      A    C  \\nATOM   1144  O   PRO A 153     -21.389  -0.013  -4.313  1.00  0.00      A    O  \\nATOM   1145  CG  PRO A 153     -16.996  -1.092  -3.837  1.00  0.00      A    C  \\nATOM   1146  CD  PRO A 153     -17.354  -2.523  -4.120  1.00  0.00      A    C  \\nATOM   1147  N   SER A 154     -20.733  -1.653  -5.694  1.00  0.00      A    N  \\nATOM   1148  CA  SER A 154     -21.859  -1.448  -6.600  1.00  0.00      A    C  \\nATOM   1149  C   SER A 154     -23.132  -2.085  -6.053  1.00  0.00      A    C  \\nATOM   1150  CB  SER A 154     -21.545  -2.020  -7.982  1.00  0.00      A    C  \\nATOM   1151  O   SER A 154     -24.230  -1.808  -6.540  1.00  0.00      A    O  \\nATOM   1152  OG  SER A 154     -21.513  -3.437  -7.945  1.00  0.00      A    O  \\nATOM   1153  N   LYS A 155     -22.940  -2.896  -5.020  1.00  0.00      A    N  \\nATOM   1154  CA  LYS A 155     -24.069  -3.569  -4.385  1.00  0.00      A    C  \\nATOM   1155  C   LYS A 155     -23.942  -3.540  -2.865  1.00  0.00      A    C  \\nATOM   1156  CB  LYS A 155     -24.177  -5.014  -4.875  1.00  0.00      A    C  \\nATOM   1157  O   LYS A 155     -23.912  -4.590  -2.219  1.00  0.00      A    O  \\nATOM   1158  CG  LYS A 155     -24.371  -5.146  -6.378  1.00  0.00      A    C  \\nATOM   1159  CD  LYS A 155     -24.433  -6.606  -6.808  1.00  0.00      A    C  \\nATOM   1160  CE  LYS A 155     -24.488  -6.741  -8.324  1.00  0.00      A    C  \\nATOM   1161  NZ  LYS A 155     -24.471  -8.172  -8.753  1.00  0.00      A    N  \\nATOM   1162  N   PRO A 156     -23.986  -2.346  -2.288  1.00  0.00      A    N  \\nATOM   1163  CA  PRO A 156     -23.656  -2.200  -0.868  1.00  0.00      A    C  \\nATOM   1164  C   PRO A 156     -24.673  -2.879   0.047  1.00  0.00      A    C  \\nATOM   1165  CB  PRO A 156     -23.661  -0.683  -0.659  1.00  0.00      A    C  \\nATOM   1166  O   PRO A 156     -24.344  -3.243   1.178  1.00  0.00      A    O  \\nATOM   1167  CG  PRO A 156     -24.513  -0.148  -1.765  1.00  0.00      A    C  \\nATOM   1168  CD  PRO A 156     -24.337  -1.026  -2.971  1.00  0.00      A    C  \\nATOM   1169  N   GLN A 157     -25.894  -3.010  -0.340  1.00  0.00      A    N  \\nATOM   1170  CA  GLN A 157     -26.946  -3.608   0.475  1.00  0.00      A    C  \\nATOM   1171  C   GLN A 157     -26.906  -5.132   0.392  1.00  0.00      A    C  \\nATOM   1172  CB  GLN A 157     -28.319  -3.095   0.041  1.00  0.00      A    C  \\nATOM   1173  O   GLN A 157     -27.508  -5.819   1.219  1.00  0.00      A    O  \\nATOM   1174  CG  GLN A 157     -28.513  -1.600   0.256  1.00  0.00      A    C  \\nATOM   1175  CD  GLN A 157     -29.922  -1.138  -0.067  1.00  0.00      A    C  \\nATOM   1176  NE2 GLN A 157     -30.142   0.171  -0.018  1.00  0.00      A    N  \\nATOM   1177  OE1 GLN A 157     -30.804  -1.951  -0.358  1.00  0.00      A    O  \\nATOM   1178  N   GLN A 158     -26.176  -5.650  -0.617  1.00  0.00      A    N  \\nATOM   1179  CA  GLN A 158     -26.192  -7.085  -0.878  1.00  0.00      A    C  \\nATOM   1180  C   GLN A 158     -24.902  -7.745  -0.398  1.00  0.00      A    C  \\nATOM   1181  CB  GLN A 158     -26.399  -7.359  -2.368  1.00  0.00      A    C  \\nATOM   1182  O   GLN A 158     -24.855  -8.963  -0.209  1.00  0.00      A    O  \\nATOM   1183  CG  GLN A 158     -27.741  -6.877  -2.901  1.00  0.00      A    C  \\nATOM   1184  CD  GLN A 158     -27.893  -7.096  -4.395  1.00  0.00      A    C  \\nATOM   1185  NE2 GLN A 158     -28.991  -6.604  -4.958  1.00  0.00      A    N  \\nATOM   1186  OE1 GLN A 158     -27.029  -7.701  -5.038  1.00  0.00      A    O  \\nATOM   1187  N   THR A 159     -23.909  -7.036  -0.297  1.00  0.00      A    N  \\nATOM   1188  CA  THR A 159     -22.619  -7.620   0.049  1.00  0.00      A    C  \\nATOM   1189  C   THR A 159     -22.075  -7.006   1.336  1.00  0.00      A    C  \\nATOM   1190  CB  THR A 159     -21.596  -7.428  -1.086  1.00  0.00      A    C  \\nATOM   1191  O   THR A 159     -21.991  -5.782   1.459  1.00  0.00      A    O  \\nATOM   1192  CG2 THR A 159     -22.008  -8.201  -2.334  1.00  0.00      A    C  \\nATOM   1193  OG1 THR A 159     -21.504  -6.035  -1.408  1.00  0.00      A    O  \\nATOM   1194  N   LYS A 160     -21.816  -7.875   2.280  1.00  0.00      A    N  \\nATOM   1195  CA  LYS A 160     -21.142  -7.440   3.499  1.00  0.00      A    C  \\nATOM   1196  C   LYS A 160     -19.644  -7.726   3.432  1.00  0.00      A    C  \\nATOM   1197  CB  LYS A 160     -21.751  -8.125   4.723  1.00  0.00      A    C  \\nATOM   1198  O   LYS A 160     -19.234  -8.856   3.159  1.00  0.00      A    O  \\nATOM   1199  CG  LYS A 160     -21.392  -7.463   6.046  1.00  0.00      A    C  \\nATOM   1200  CD  LYS A 160     -22.069  -8.157   7.220  1.00  0.00      A    C  \\nATOM   1201  CE  LYS A 160     -21.649  -7.546   8.550  1.00  0.00      A    C  \\nATOM   1202  NZ  LYS A 160     -22.329  -8.208   9.704  1.00  0.00      A    N  \\nATOM   1203  N   GLY A 161     -18.808  -6.744   3.312  1.00  0.00      A    N  \\nATOM   1204  CA  GLY A 161     -17.360  -6.881   3.323  1.00  0.00      A    C  \\nATOM   1205  C   GLY A 161     -16.774  -6.915   4.722  1.00  0.00      A    C  \\nATOM   1206  O   GLY A 161     -17.408  -6.463   5.678  1.00  0.00      A    O  \\nATOM   1207  N   THR A 162     -15.644  -7.617   4.881  1.00  0.00      A    N  \\nATOM   1208  CA  THR A 162     -14.948  -7.661   6.162  1.00  0.00      A    C  \\nATOM   1209  C   THR A 162     -13.988  -6.483   6.296  1.00  0.00      A    C  \\nATOM   1210  CB  THR A 162     -14.172  -8.980   6.332  1.00  0.00      A    C  \\nATOM   1211  O   THR A 162     -13.245  -6.171   5.363  1.00  0.00      A    O  \\nATOM   1212  CG2 THR A 162     -13.460  -9.030   7.680  1.00  0.00      A    C  \\nATOM   1213  OG1 THR A 162     -15.087 -10.080   6.246  1.00  0.00      A    O  \\nATOM   1214  N   TRP A 163     -14.179  -5.804   7.467  1.00  0.00      A    N  \\nATOM   1215  CA  TRP A 163     -13.217  -4.766   7.826  1.00  0.00      A    C  \\nATOM   1216  C   TRP A 163     -12.001  -5.369   8.522  1.00  0.00      A    C  \\nATOM   1217  CB  TRP A 163     -13.870  -3.717   8.729  1.00  0.00      A    C  \\nATOM   1218  O   TRP A 163     -12.131  -6.310   9.308  1.00  0.00      A    O  \\nATOM   1219  CG  TRP A 163     -14.935  -2.908   8.051  1.00  0.00      A    C  \\nATOM   1220  CD1 TRP A 163     -16.258  -3.230   7.924  1.00  0.00      A    C  \\nATOM   1221  CD2 TRP A 163     -14.766  -1.643   7.404  1.00  0.00      A    C  \\nATOM   1222  CE2 TRP A 163     -16.029  -1.256   6.905  1.00  0.00      A    C  \\nATOM   1223  CE3 TRP A 163     -13.667  -0.799   7.197  1.00  0.00      A    C  \\nATOM   1224  NE1 TRP A 163     -16.922  -2.240   7.236  1.00  0.00      A    N  \\nATOM   1225  CH2 TRP A 163     -15.132   0.749   6.024  1.00  0.00      A    C  \\nATOM   1226  CZ2 TRP A 163     -16.224  -0.059   6.212  1.00  0.00      A    C  \\nATOM   1227  CZ3 TRP A 163     -13.862   0.392   6.507  1.00  0.00      A    C  \\nATOM   1228  N   PHE A 164     -10.859  -4.843   8.223  1.00  0.00      A    N  \\nATOM   1229  CA  PHE A 164      -9.649  -5.364   8.846  1.00  0.00      A    C  \\nATOM   1230  C   PHE A 164      -8.642  -4.247   9.091  1.00  0.00      A    C  \\nATOM   1231  CB  PHE A 164      -9.021  -6.456   7.975  1.00  0.00      A    C  \\nATOM   1232  O   PHE A 164      -8.722  -3.184   8.471  1.00  0.00      A    O  \\nATOM   1233  CG  PHE A 164      -8.481  -5.950   6.664  1.00  0.00      A    C  \\nATOM   1234  CD1 PHE A 164      -9.315  -5.812   5.560  1.00  0.00      A    C  \\nATOM   1235  CD2 PHE A 164      -7.140  -5.614   6.535  1.00  0.00      A    C  \\nATOM   1236  CE1 PHE A 164      -8.818  -5.344   4.346  1.00  0.00      A    C  \\nATOM   1237  CE2 PHE A 164      -6.637  -5.146   5.325  1.00  0.00      A    C  \\nATOM   1238  CZ  PHE A 164      -7.477  -5.013   4.231  1.00  0.00      A    C  \\nATOM   1239  N   GLN A 165      -7.778  -4.430  10.044  1.00  0.00      A    N  \\nATOM   1240  CA  GLN A 165      -6.692  -3.531  10.419  1.00  0.00      A    C  \\nATOM   1241  C   GLN A 165      -5.408  -4.306  10.698  1.00  0.00      A    C  \\nATOM   1242  CB  GLN A 165      -7.080  -2.700  11.642  1.00  0.00      A    C  \\nATOM   1243  O   GLN A 165      -5.407  -5.252  11.488  1.00  0.00      A    O  \\nATOM   1244  CG  GLN A 165      -6.013  -1.702  12.072  1.00  0.00      A    C  \\nATOM   1245  CD  GLN A 165      -6.438  -0.865  13.264  1.00  0.00      A    C  \\nATOM   1246  NE2 GLN A 165      -5.746   0.248  13.483  1.00  0.00      A    N  \\nATOM   1247  OE1 GLN A 165      -7.381  -1.214  13.980  1.00  0.00      A    O  \\nATOM   1248  N   ILE A 166      -4.396  -3.942   9.941  1.00  0.00      A    N  \\nATOM   1249  CA  ILE A 166      -3.081  -4.475  10.281  1.00  0.00      A    C  \\nATOM   1250  C   ILE A 166      -2.472  -3.659  11.419  1.00  0.00      A    C  \\nATOM   1251  CB  ILE A 166      -2.138  -4.476   9.057  1.00  0.00      A    C  \\nATOM   1252  O   ILE A 166      -2.240  -2.457  11.274  1.00  0.00      A    O  \\nATOM   1253  CG1 ILE A 166      -2.753  -5.286   7.910  1.00  0.00      A    C  \\nATOM   1254  CG2 ILE A 166      -0.760  -5.024   9.438  1.00  0.00      A    C  \\nATOM   1255  CD1 ILE A 166      -2.102  -5.034   6.556  1.00  0.00      A    C  \\nATOM   1256  N   THR A 167      -2.238  -4.292  12.599  1.00  0.00      A    N  \\nATOM   1257  CA  THR A 167      -1.866  -3.578  13.815  1.00  0.00      A    C  \\nATOM   1258  C   THR A 167      -0.351  -3.569  13.995  1.00  0.00      A    C  \\nATOM   1259  CB  THR A 167      -2.529  -4.204  15.056  1.00  0.00      A    C  \\nATOM   1260  O   THR A 167       0.210  -2.613  14.534  1.00  0.00      A    O  \\nATOM   1261  CG2 THR A 167      -4.040  -3.998  15.035  1.00  0.00      A    C  \\nATOM   1262  OG1 THR A 167      -2.251  -5.610  15.081  1.00  0.00      A    O  \\nATOM   1263  N   LYS A 168       0.247  -4.597  13.631  1.00  0.00      A    N  \\nATOM   1264  CA  LYS A 168       1.696  -4.703  13.775  1.00  0.00      A    C  \\nATOM   1265  C   LYS A 168       2.301  -5.543  12.654  1.00  0.00      A    C  \\nATOM   1266  CB  LYS A 168       2.058  -5.305  15.134  1.00  0.00      A    C  \\nATOM   1267  O   LYS A 168       1.629  -6.406  12.087  1.00  0.00      A    O  \\nATOM   1268  CG  LYS A 168       3.550  -5.311  15.431  1.00  0.00      A    C  \\nATOM   1269  CD  LYS A 168       3.840  -5.852  16.825  1.00  0.00      A    C  \\nATOM   1270  CE  LYS A 168       5.337  -5.950  17.087  1.00  0.00      A    C  \\nATOM   1271  NZ  LYS A 168       5.627  -6.555  18.421  1.00  0.00      A    N  \\nATOM   1272  N   PHE A 169       3.544  -5.189  12.284  1.00  0.00      A    N  \\nATOM   1273  CA  PHE A 169       4.357  -5.982  11.369  1.00  0.00      A    C  \\nATOM   1274  C   PHE A 169       5.502  -6.661  12.112  1.00  0.00      A    C  \\nATOM   1275  CB  PHE A 169       4.910  -5.104  10.242  1.00  0.00      A    C  \\nATOM   1276  O   PHE A 169       6.274  -6.001  12.811  1.00  0.00      A    O  \\nATOM   1277  CG  PHE A 169       3.853  -4.570   9.314  1.00  0.00      A    C  \\nATOM   1278  CD1 PHE A 169       3.541  -5.237   8.136  1.00  0.00      A    C  \\nATOM   1279  CD2 PHE A 169       3.170  -3.400   9.621  1.00  0.00      A    C  \\nATOM   1280  CE1 PHE A 169       2.563  -4.745   7.275  1.00  0.00      A    C  \\nATOM   1281  CE2 PHE A 169       2.191  -2.902   8.766  1.00  0.00      A    C  \\nATOM   1282  CZ  PHE A 169       1.890  -3.575   7.593  1.00  0.00      A    C  \\nATOM   1283  N   THR A 170       5.505  -7.960  12.009  1.00  0.00      A    N  \\nATOM   1284  CA  THR A 170       6.505  -8.738  12.733  1.00  0.00      A    C  \\nATOM   1285  C   THR A 170       7.529  -9.332  11.770  1.00  0.00      A    C  \\nATOM   1286  CB  THR A 170       5.849  -9.867  13.549  1.00  0.00      A    C  \\nATOM   1287  O   THR A 170       7.409  -9.174  10.553  1.00  0.00      A    O  \\nATOM   1288  CG2 THR A 170       4.818  -9.313  14.526  1.00  0.00      A    C  \\nATOM   1289  OG1 THR A 170       5.201 -10.781  12.656  1.00  0.00      A    O  \\nATOM   1290  N   GLY A 171       8.599  -9.907  12.354  1.00  0.00      A    N  \\nATOM   1291  CA  GLY A 171       9.647 -10.446  11.502  1.00  0.00      A    C  \\nATOM   1292  C   GLY A 171      10.603  -9.386  10.988  1.00  0.00      A    C  \\nATOM   1293  O   GLY A 171      11.003  -8.490  11.734  1.00  0.00      A    O  \\nATOM   1294  N   ALA A 172      11.054  -9.538   9.627  1.00  0.00      A    N  \\nATOM   1295  CA  ALA A 172      12.052  -8.643   9.047  1.00  0.00      A    C  \\nATOM   1296  C   ALA A 172      11.415  -7.337   8.584  1.00  0.00      A    C  \\nATOM   1297  CB  ALA A 172      12.766  -9.326   7.882  1.00  0.00      A    C  \\nATOM   1298  O   ALA A 172      11.557  -6.946   7.422  1.00  0.00      A    O  \\nATOM   1299  N   ALA A 173      10.702  -6.665   9.515  1.00  0.00      A    N  \\nATOM   1300  CA  ALA A 173      10.030  -5.404   9.212  1.00  0.00      A    C  \\nATOM   1301  C   ALA A 173      11.043  -4.288   8.969  1.00  0.00      A    C  \\nATOM   1302  CB  ALA A 173       9.082  -5.020  10.345  1.00  0.00      A    C  \\nATOM   1303  O   ALA A 173      12.100  -4.252   9.603  1.00  0.00      A    O  \\nATOM   1304  N   GLY A 174      10.789  -3.428   8.019  1.00  0.00      A    N  \\nATOM   1305  CA  GLY A 174      11.609  -2.256   7.758  1.00  0.00      A    C  \\nATOM   1306  C   GLY A 174      11.479  -1.187   8.826  1.00  0.00      A    C  \\nATOM   1307  O   GLY A 174      10.682  -1.325   9.756  1.00  0.00      A    O  \\nATOM   1308  N   PRO A 175      12.233  -0.181   8.755  1.00  0.00      A    N  \\nATOM   1309  CA  PRO A 175      12.292   0.820   9.823  1.00  0.00      A    C  \\nATOM   1310  C   PRO A 175      10.961   1.538  10.034  1.00  0.00      A    C  \\nATOM   1311  CB  PRO A 175      13.366   1.795   9.333  1.00  0.00      A    C  \\nATOM   1312  O   PRO A 175      10.603   1.861  11.169  1.00  0.00      A    O  \\nATOM   1313  CG  PRO A 175      13.474   1.538   7.865  1.00  0.00      A    C  \\nATOM   1314  CD  PRO A 175      13.127   0.100   7.605  1.00  0.00      A    C  \\nATOM   1315  N   TYR A 176      10.147   1.855   9.035  1.00  0.00      A    N  \\nATOM   1316  CA  TYR A 176       8.871   2.530   9.246  1.00  0.00      A    C  \\nATOM   1317  C   TYR A 176       7.868   1.603   9.920  1.00  0.00      A    C  \\nATOM   1318  CB  TYR A 176       8.304   3.035   7.915  1.00  0.00      A    C  \\nATOM   1319  O   TYR A 176       7.076   2.040  10.759  1.00  0.00      A    O  \\nATOM   1320  CG  TYR A 176       9.064   4.202   7.334  1.00  0.00      A    C  \\nATOM   1321  CD1 TYR A 176       8.998   5.463   7.922  1.00  0.00      A    C  \\nATOM   1322  CD2 TYR A 176       9.850   4.046   6.197  1.00  0.00      A    C  \\nATOM   1323  CE1 TYR A 176       9.695   6.543   7.389  1.00  0.00      A    C  \\nATOM   1324  CE2 TYR A 176      10.551   5.118   5.656  1.00  0.00      A    C  \\nATOM   1325  OH  TYR A 176      11.161   7.425   5.726  1.00  0.00      A    O  \\nATOM   1326  CZ  TYR A 176      10.468   6.360   6.258  1.00  0.00      A    C  \\nATOM   1327  N   CYS A 177       7.922   0.317   9.531  1.00  0.00      A    N  \\nATOM   1328  CA  CYS A 177       7.015  -0.630  10.170  1.00  0.00      A    C  \\nATOM   1329  C   CYS A 177       7.385  -0.837  11.634  1.00  0.00      A    C  \\nATOM   1330  CB  CYS A 177       7.035  -1.971   9.436  1.00  0.00      A    C  \\nATOM   1331  O   CYS A 177       6.507  -0.985  12.486  1.00  0.00      A    O  \\nATOM   1332  SG  CYS A 177       6.211  -1.932   7.828  1.00  0.00      A    S  \\nATOM   1333  N   LYS A 178       8.696  -0.813  11.930  1.00  0.00      A    N  \\nATOM   1334  CA  LYS A 178       9.120  -0.859  13.326  1.00  0.00      A    C  \\nATOM   1335  C   LYS A 178       8.643   0.376  14.086  1.00  0.00      A    C  \\nATOM   1336  CB  LYS A 178      10.642  -0.977  13.422  1.00  0.00      A    C  \\nATOM   1337  O   LYS A 178       8.153   0.267  15.212  1.00  0.00      A    O  \\nATOM   1338  CG  LYS A 178      11.187  -2.328  12.982  1.00  0.00      A    C  \\nATOM   1339  CD  LYS A 178      12.709  -2.356  13.020  1.00  0.00      A    C  \\nATOM   1340  CE  LYS A 178      13.258  -3.680  12.508  1.00  0.00      A    C  \\nATOM   1341  NZ  LYS A 178      14.745  -3.648  12.372  1.00  0.00      A    N  \\nATOM   1342  N   ALA A 179       8.692   1.511  13.477  1.00  0.00      A    N  \\nATOM   1343  CA  ALA A 179       8.271   2.770  14.086  1.00  0.00      A    C  \\nATOM   1344  C   ALA A 179       6.756   2.813  14.264  1.00  0.00      A    C  \\nATOM   1345  CB  ALA A 179       8.738   3.952  13.240  1.00  0.00      A    C  \\nATOM   1346  O   ALA A 179       6.255   3.384  15.235  1.00  0.00      A    O  \\nATOM   1347  N   LEU A 180       6.023   2.240  13.294  1.00  0.00      A    N  \\nATOM   1348  CA  LEU A 180       4.569   2.196  13.408  1.00  0.00      A    C  \\nATOM   1349  C   LEU A 180       4.145   1.410  14.645  1.00  0.00      A    C  \\nATOM   1350  CB  LEU A 180       3.952   1.570  12.155  1.00  0.00      A    C  \\nATOM   1351  O   LEU A 180       3.226   1.818  15.359  1.00  0.00      A    O  \\nATOM   1352  CG  LEU A 180       2.425   1.486  12.120  1.00  0.00      A    C  \\nATOM   1353  CD1 LEU A 180       1.817   2.885  12.147  1.00  0.00      A    C  \\nATOM   1354  CD2 LEU A 180       1.959   0.719  10.888  1.00  0.00      A    C  \\nATOM   1355  N   GLY A 181       4.878   0.316  14.970  1.00  0.00      A    N  \\nATOM   1356  CA  GLY A 181       4.586  -0.507  16.133  1.00  0.00      A    C  \\nATOM   1357  C   GLY A 181       4.732   0.240  17.445  1.00  0.00      A    C  \\nATOM   1358  O   GLY A 181       4.045  -0.069  18.421  1.00  0.00      A    O  \\nATOM   1359  N   SER A 182       5.544   1.277  17.442  1.00  0.00      A    N  \\nATOM   1360  CA  SER A 182       5.797   2.042  18.659  1.00  0.00      A    C  \\nATOM   1361  C   SER A 182       5.301   3.478  18.521  1.00  0.00      A    C  \\nATOM   1362  CB  SER A 182       7.289   2.039  18.994  1.00  0.00      A    C  \\nATOM   1363  O   SER A 182       5.569   4.318  19.383  1.00  0.00      A    O  \\nATOM   1364  OG  SER A 182       8.047   2.571  17.921  1.00  0.00      A    O  \\nATOM   1365  N   ASN A 183       4.565   3.740  17.438  1.00  0.00      A    N  \\nATOM   1366  CA  ASN A 183       4.075   5.087  17.164  1.00  0.00      A    C  \\nATOM   1367  C   ASN A 183       5.191   6.122  17.266  1.00  0.00      A    C  \\nATOM   1368  CB  ASN A 183       2.930   5.445  18.114  1.00  0.00      A    C  \\nATOM   1369  O   ASN A 183       4.980   7.219  17.784  1.00  0.00      A    O  \\nATOM   1370  CG  ASN A 183       2.117   6.629  17.628  1.00  0.00      A    C  \\nATOM   1371  ND2 ASN A 183       1.579   7.403  18.564  1.00  0.00      A    N  \\nATOM   1372  OD1 ASN A 183       1.972   6.845  16.422  1.00  0.00      A    O  \\nATOM   1373  N   ASP A 184       6.302   5.776  16.793  1.00  0.00      A    N  \\nATOM   1374  CA  ASP A 184       7.460   6.666  16.767  1.00  0.00      A    C  \\nATOM   1375  C   ASP A 184       7.458   7.531  15.508  1.00  0.00      A    C  \\nATOM   1376  CB  ASP A 184       8.758   5.860  16.850  1.00  0.00      A    C  \\nATOM   1377  O   ASP A 184       7.907   7.092  14.447  1.00  0.00      A    O  \\nATOM   1378  CG  ASP A 184       9.999   6.734  16.845  1.00  0.00      A    C  \\nATOM   1379  OD1 ASP A 184       9.873   7.974  16.944  1.00  0.00      A    O  \\nATOM   1380  OD2 ASP A 184      11.114   6.179  16.738  1.00  0.00      A    O  \\nATOM   1381  N   LYS A 185       7.086   8.724  15.650  1.00  0.00      A    N  \\nATOM   1382  CA  LYS A 185       6.919   9.620  14.510  1.00  0.00      A    C  \\nATOM   1383  C   LYS A 185       8.215  10.364  14.200  1.00  0.00      A    C  \\nATOM   1384  CB  LYS A 185       5.792  10.620  14.773  1.00  0.00      A    C  \\nATOM   1385  O   LYS A 185       8.264  11.179  13.277  1.00  0.00      A    O  \\nATOM   1386  CG  LYS A 185       4.404   9.996  14.790  1.00  0.00      A    C  \\nATOM   1387  CD  LYS A 185       3.315  11.059  14.865  1.00  0.00      A    C  \\nATOM   1388  CE  LYS A 185       1.924  10.439  14.828  1.00  0.00      A    C  \\nATOM   1389  NZ  LYS A 185       0.853  11.479  14.860  1.00  0.00      A    N  \\nATOM   1390  N   SER A 186       9.242  10.142  14.956  1.00  0.00      A    N  \\nATOM   1391  CA  SER A 186      10.520  10.794  14.691  1.00  0.00      A    C  \\nATOM   1392  C   SER A 186      11.077  10.387  13.331  1.00  0.00      A    C  \\nATOM   1393  CB  SER A 186      11.531  10.458  15.787  1.00  0.00      A    C  \\nATOM   1394  O   SER A 186      11.830  11.140  12.711  1.00  0.00      A    O  \\nATOM   1395  OG  SER A 186      11.937   9.103  15.698  1.00  0.00      A    O  \\nATOM   1396  N   VAL A 187      10.747   9.211  12.831  1.00  0.00      A    N  \\nATOM   1397  CA  VAL A 187      11.242   8.718  11.550  1.00  0.00      A    C  \\nATOM   1398  C   VAL A 187      10.630   9.531  10.412  1.00  0.00      A    C  \\nATOM   1399  CB  VAL A 187      10.929   7.216  11.361  1.00  0.00      A    C  \\nATOM   1400  O   VAL A 187      11.071   9.433   9.264  1.00  0.00      A    O  \\nATOM   1401  CG1 VAL A 187      11.672   6.375  12.398  1.00  0.00      A    C  \\nATOM   1402  CG2 VAL A 187       9.423   6.971  11.449  1.00  0.00      A    C  \\nATOM   1403  N   CYS A 188       9.584  10.310  10.696  1.00  0.00      A    N  \\nATOM   1404  CA  CYS A 188       8.885  11.075   9.670  1.00  0.00      A    C  \\nATOM   1405  C   CYS A 188       9.582  12.406   9.413  1.00  0.00      A    C  \\nATOM   1406  CB  CYS A 188       7.433  11.321  10.080  1.00  0.00      A    C  \\nATOM   1407  O   CYS A 188       9.155  13.179   8.553  1.00  0.00      A    O  \\nATOM   1408  SG  CYS A 188       6.484   9.806  10.338  1.00  0.00      A    S  \\nATOM   1409  N   ASP A 189      10.643  12.658  10.185  1.00  0.00      A    N  \\nATOM   1410  CA  ASP A 189      11.438  13.870  10.008  1.00  0.00      A    C  \\nATOM   1411  C   ASP A 189      12.072  13.913   8.620  1.00  0.00      A    C  \\nATOM   1412  CB  ASP A 189      12.522  13.961  11.084  1.00  0.00      A    C  \\nATOM   1413  O   ASP A 189      12.747  12.966   8.210  1.00  0.00      A    O  \\nATOM   1414  CG  ASP A 189      13.294  15.268  11.038  1.00  0.00      A    C  \\nATOM   1415  OD1 ASP A 189      13.254  15.964  10.001  1.00  0.00      A    O  \\nATOM   1416  OD2 ASP A 189      13.952  15.603  12.047  1.00  0.00      A    O  \\nATOM   1417  N   LYS A 190      11.829  15.019   7.882  1.00  0.00      A    N  \\nATOM   1418  CA  LYS A 190      12.277  15.135   6.497  1.00  0.00      A    C  \\nATOM   1419  C   LYS A 190      13.798  15.061   6.402  1.00  0.00      A    C  \\nATOM   1420  CB  LYS A 190      11.777  16.442   5.879  1.00  0.00      A    C  \\nATOM   1421  O   LYS A 190      14.344  14.727   5.349  1.00  0.00      A    O  \\nATOM   1422  CG  LYS A 190      12.321  17.693   6.552  1.00  0.00      A    C  \\nATOM   1423  CD  LYS A 190      11.780  18.959   5.899  1.00  0.00      A    C  \\nATOM   1424  CE  LYS A 190      12.349  20.212   6.552  1.00  0.00      A    C  \\nATOM   1425  NZ  LYS A 190      11.921  21.452   5.837  1.00  0.00      A    N  \\nATOM   1426  N   ASN A 191      14.531  15.307   7.501  1.00  0.00      A    N  \\nATOM   1427  CA  ASN A 191      15.989  15.339   7.476  1.00  0.00      A    C  \\nATOM   1428  C   ASN A 191      16.586  13.985   7.848  1.00  0.00      A    C  \\nATOM   1429  CB  ASN A 191      16.518  16.429   8.412  1.00  0.00      A    C  \\nATOM   1430  O   ASN A 191      17.803  13.801   7.788  1.00  0.00      A    O  \\nATOM   1431  CG  ASN A 191      16.097  17.821   7.984  1.00  0.00      A    C  \\nATOM   1432  ND2 ASN A 191      15.674  18.634   8.945  1.00  0.00      A    N  \\nATOM   1433  OD1 ASN A 191      16.152  18.163   6.800  1.00  0.00      A    O  \\nATOM   1434  N   LYS A 192      15.776  13.083   8.186  1.00  0.00      A    N  \\nATOM   1435  CA  LYS A 192      16.264  11.762   8.573  1.00  0.00      A    C  \\nATOM   1436  C   LYS A 192      16.327  10.825   7.370  1.00  0.00      A    C  \\nATOM   1437  CB  LYS A 192      15.375  11.159   9.661  1.00  0.00      A    C  \\nATOM   1438  O   LYS A 192      15.433  10.834   6.522  1.00  0.00      A    O  \\nATOM   1439  CG  LYS A 192      15.555  11.795  11.032  1.00  0.00      A    C  \\nATOM   1440  CD  LYS A 192      14.844  10.997  12.117  1.00  0.00      A    C  \\nATOM   1441  CE  LYS A 192      15.012  11.640  13.486  1.00  0.00      A    C  \\nATOM   1442  NZ  LYS A 192      14.302  10.870  14.551  1.00  0.00      A    N  \\nATOM   1443  N   ASN A 193      17.399  10.127   7.224  1.00  0.00      A    N  \\nATOM   1444  CA  ASN A 193      17.519   9.046   6.251  1.00  0.00      A    C  \\nATOM   1445  C   ASN A 193      17.341   7.679   6.907  1.00  0.00      A    C  \\nATOM   1446  CB  ASN A 193      18.868   9.118   5.532  1.00  0.00      A    C  \\nATOM   1447  O   ASN A 193      18.143   7.283   7.754  1.00  0.00      A    O  \\nATOM   1448  CG  ASN A 193      18.926   8.223   4.310  1.00  0.00      A    C  \\nATOM   1449  ND2 ASN A 193      20.136   7.917   3.857  1.00  0.00      A    N  \\nATOM   1450  OD1 ASN A 193      17.893   7.811   3.777  1.00  0.00      A    O  \\nATOM   1451  N   ILE A 194      16.287   7.107   6.571  1.00  0.00      A    N  \\nATOM   1452  CA  ILE A 194      15.952   5.828   7.187  1.00  0.00      A    C  \\nATOM   1453  C   ILE A 194      15.942   4.730   6.126  1.00  0.00      A    C  \\nATOM   1454  CB  ILE A 194      14.586   5.889   7.908  1.00  0.00      A    C  \\nATOM   1455  O   ILE A 194      14.877   4.298   5.681  1.00  0.00      A    O  \\nATOM   1456  CG1 ILE A 194      14.529   7.104   8.842  1.00  0.00      A    C  \\nATOM   1457  CG2 ILE A 194      14.325   4.592   8.680  1.00  0.00      A    C  \\nATOM   1458  CD1 ILE A 194      13.279   7.159   9.709  1.00  0.00      A    C  \\nATOM   1459  N   ALA A 195      17.153   4.353   5.787  1.00  0.00      A    N  \\nATOM   1460  CA  ALA A 195      17.270   3.338   4.744  1.00  0.00      A    C  \\nATOM   1461  C   ALA A 195      17.044   1.940   5.312  1.00  0.00      A    C  \\nATOM   1462  CB  ALA A 195      18.638   3.421   4.072  1.00  0.00      A    C  \\nATOM   1463  O   ALA A 195      17.471   1.640   6.429  1.00  0.00      A    O  \\nATOM   1464  N   GLY A 196      16.251   1.122   4.627  1.00  0.00      A    N  \\nATOM   1465  CA  GLY A 196      16.156  -0.288   4.971  1.00  0.00      A    C  \\nATOM   1466  C   GLY A 196      17.224  -1.137   4.308  1.00  0.00      A    C  \\nATOM   1467  O   GLY A 196      18.208  -0.609   3.785  1.00  0.00      A    O  \\nATOM   1468  N   ASP A 197      17.096  -2.440   4.410  1.00  0.00      A    N  \\nATOM   1469  CA  ASP A 197      18.039  -3.404   3.852  1.00  0.00      A    C  \\nATOM   1470  C   ASP A 197      18.171  -3.232   2.341  1.00  0.00      A    C  \\nATOM   1471  CB  ASP A 197      17.604  -4.834   4.182  1.00  0.00      A    C  \\nATOM   1472  O   ASP A 197      19.172  -3.642   1.749  1.00  0.00      A    O  \\nATOM   1473  CG  ASP A 197      17.753  -5.174   5.655  1.00  0.00      A    C  \\nATOM   1474  OD1 ASP A 197      18.469  -4.449   6.379  1.00  0.00      A    O  \\nATOM   1475  OD2 ASP A 197      17.152  -6.178   6.095  1.00  0.00      A    O  \\nATOM   1476  N   TRP A 198      17.313  -2.627   1.728  1.00  0.00      A    N  \\nATOM   1477  CA  TRP A 198      17.249  -2.395   0.289  1.00  0.00      A    C  \\nATOM   1478  C   TRP A 198      18.066  -1.167  -0.101  1.00  0.00      A    C  \\nATOM   1479  CB  TRP A 198      15.796  -2.222  -0.164  1.00  0.00      A    C  \\nATOM   1480  O   TRP A 198      18.215  -0.865  -1.287  1.00  0.00      A    O  \\nATOM   1481  CG  TRP A 198      14.996  -1.293   0.699  1.00  0.00      A    C  \\nATOM   1482  CD1 TRP A 198      14.954   0.072   0.623  1.00  0.00      A    C  \\nATOM   1483  CD2 TRP A 198      14.126  -1.662   1.773  1.00  0.00      A    C  \\nATOM   1484  CE2 TRP A 198      13.588  -0.468   2.304  1.00  0.00      A    C  \\nATOM   1485  CE3 TRP A 198      13.747  -2.887   2.338  1.00  0.00      A    C  \\nATOM   1486  NE1 TRP A 198      14.109   0.574   1.585  1.00  0.00      A    N  \\nATOM   1487  CH2 TRP A 198      12.336  -1.677   3.908  1.00  0.00      A    C  \\nATOM   1488  CZ2 TRP A 198      12.690  -0.465   3.374  1.00  0.00      A    C  \\nATOM   1489  CZ3 TRP A 198      12.853  -2.882   3.403  1.00  0.00      A    C  \\nATOM   1490  N   GLY A 199      18.615  -0.404   0.794  1.00  0.00      A    N  \\nATOM   1491  CA  GLY A 199      19.670   0.557   0.515  1.00  0.00      A    C  \\nATOM   1492  C   GLY A 199      19.149   1.962   0.276  1.00  0.00      A    C  \\nATOM   1493  O   GLY A 199      19.913   2.861  -0.079  1.00  0.00      A    O  \\nATOM   1494  N   PHE A 200      17.885   2.201   0.368  1.00  0.00      A    N  \\nATOM   1495  CA  PHE A 200      17.351   3.548   0.210  1.00  0.00      A    C  \\nATOM   1496  C   PHE A 200      16.130   3.754   1.100  1.00  0.00      A    C  \\nATOM   1497  CB  PHE A 200      16.984   3.814  -1.253  1.00  0.00      A    C  \\nATOM   1498  O   PHE A 200      15.595   2.795   1.659  1.00  0.00      A    O  \\nATOM   1499  CG  PHE A 200      15.852   2.963  -1.760  1.00  0.00      A    C  \\nATOM   1500  CD1 PHE A 200      16.099   1.719  -2.327  1.00  0.00      A    C  \\nATOM   1501  CD2 PHE A 200      14.539   3.406  -1.669  1.00  0.00      A    C  \\nATOM   1502  CE1 PHE A 200      15.053   0.928  -2.797  1.00  0.00      A    C  \\nATOM   1503  CE2 PHE A 200      13.488   2.622  -2.136  1.00  0.00      A    C  \\nATOM   1504  CZ  PHE A 200      13.748   1.384  -2.701  1.00  0.00      A    C  \\nATOM   1505  N   ASP A 201      15.806   5.005   1.332  1.00  0.00      A    N  \\nATOM   1506  CA  ASP A 201      14.593   5.430   2.023  1.00  0.00      A    C  \\nATOM   1507  C   ASP A 201      13.419   5.544   1.053  1.00  0.00      A    C  \\nATOM   1508  CB  ASP A 201      14.820   6.767   2.733  1.00  0.00      A    C  \\nATOM   1509  O   ASP A 201      13.371   6.464   0.234  1.00  0.00      A    O  \\nATOM   1510  CG  ASP A 201      13.664   7.164   3.633  1.00  0.00      A    C  \\nATOM   1511  OD1 ASP A 201      12.566   6.578   3.512  1.00  0.00      A    O  \\nATOM   1512  OD2 ASP A 201      13.850   8.074   4.470  1.00  0.00      A    O  \\nATOM   1513  N   PRO A 202      12.465   4.604   1.116  1.00  0.00      A    N  \\nATOM   1514  CA  PRO A 202      11.393   4.627   0.119  1.00  0.00      A    C  \\nATOM   1515  C   PRO A 202      10.559   5.904   0.178  1.00  0.00      A    C  \\nATOM   1516  CB  PRO A 202      10.546   3.405   0.484  1.00  0.00      A    C  \\nATOM   1517  O   PRO A 202       9.993   6.325  -0.835  1.00  0.00      A    O  \\nATOM   1518  CG  PRO A 202      10.846   3.148   1.925  1.00  0.00      A    C  \\nATOM   1519  CD  PRO A 202      12.267   3.553   2.196  1.00  0.00      A    C  \\nATOM   1520  N   ALA A 203      10.458   6.530   1.266  1.00  0.00      A    N  \\nATOM   1521  CA  ALA A 203       9.675   7.747   1.466  1.00  0.00      A    C  \\nATOM   1522  C   ALA A 203      10.165   8.869   0.556  1.00  0.00      A    C  \\nATOM   1523  CB  ALA A 203       9.738   8.185   2.927  1.00  0.00      A    C  \\nATOM   1524  O   ALA A 203       9.421   9.807   0.261  1.00  0.00      A    O  \\nATOM   1525  N   LYS A 204      11.348   8.739   0.029  1.00  0.00      A    N  \\nATOM   1526  CA  LYS A 204      11.932   9.791  -0.798  1.00  0.00      A    C  \\nATOM   1527  C   LYS A 204      11.413   9.716  -2.231  1.00  0.00      A    C  \\nATOM   1528  CB  LYS A 204      13.459   9.697  -0.788  1.00  0.00      A    C  \\nATOM   1529  O   LYS A 204      11.674  10.609  -3.039  1.00  0.00      A    O  \\nATOM   1530  CG  LYS A 204      14.093  10.065   0.545  1.00  0.00      A    C  \\nATOM   1531  CD  LYS A 204      15.612  10.123   0.444  1.00  0.00      A    C  \\nATOM   1532  CE  LYS A 204      16.244  10.559   1.759  1.00  0.00      A    C  \\nATOM   1533  NZ  LYS A 204      17.736  10.534   1.691  1.00  0.00      A    N  \\nATOM   1534  N   TRP A 205      10.622   8.734  -2.573  1.00  0.00      A    N  \\nATOM   1535  CA  TRP A 205      10.138   8.530  -3.934  1.00  0.00      A    C  \\nATOM   1536  C   TRP A 205       8.635   8.270  -3.944  1.00  0.00      A    C  \\nATOM   1537  CB  TRP A 205      10.875   7.364  -4.599  1.00  0.00      A    C  \\nATOM   1538  O   TRP A 205       8.139   7.490  -4.761  1.00  0.00      A    O  \\nATOM   1539  CG  TRP A 205      12.309   7.655  -4.923  1.00  0.00      A    C  \\nATOM   1540  CD1 TRP A 205      12.796   8.218  -6.070  1.00  0.00      A    C  \\nATOM   1541  CD2 TRP A 205      13.443   7.394  -4.091  1.00  0.00      A    C  \\nATOM   1542  CE2 TRP A 205      14.588   7.825  -4.797  1.00  0.00      A    C  \\nATOM   1543  CE3 TRP A 205      13.603   6.838  -2.814  1.00  0.00      A    C  \\nATOM   1544  NE1 TRP A 205      14.166   8.322  -6.001  1.00  0.00      A    N  \\nATOM   1545  CH2 TRP A 205      16.004   7.168  -3.018  1.00  0.00      A    C  \\nATOM   1546  CZ2 TRP A 205      15.876   7.716  -4.269  1.00  0.00      A    C  \\nATOM   1547  CZ3 TRP A 205      14.886   6.731  -2.290  1.00  0.00      A    C  \\nATOM   1548  N   ALA A 206       7.992   8.909  -2.961  1.00  0.00      A    N  \\nATOM   1549  CA  ALA A 206       6.545   8.745  -2.848  1.00  0.00      A    C  \\nATOM   1550  C   ALA A 206       5.807   9.809  -3.656  1.00  0.00      A    C  \\nATOM   1551  CB  ALA A 206       6.117   8.801  -1.383  1.00  0.00      A    C  \\nATOM   1552  O   ALA A 206       6.150  10.992  -3.595  1.00  0.00      A    O  \\nATOM   1553  N   TYR A 207       4.797   9.345  -4.368  1.00  0.00      A    N  \\nATOM   1554  CA  TYR A 207       4.013  10.206  -5.245  1.00  0.00      A    C  \\nATOM   1555  C   TYR A 207       2.523   9.915  -5.107  1.00  0.00      A    C  \\nATOM   1556  CB  TYR A 207       4.446  10.027  -6.703  1.00  0.00      A    C  \\nATOM   1557  O   TYR A 207       2.134   8.828  -4.674  1.00  0.00      A    O  \\nATOM   1558  CG  TYR A 207       5.830  10.555  -6.994  1.00  0.00      A    C  \\nATOM   1559  CD1 TYR A 207       6.013  11.826  -7.534  1.00  0.00      A    C  \\nATOM   1560  CD2 TYR A 207       6.957   9.784  -6.731  1.00  0.00      A    C  \\nATOM   1561  CE1 TYR A 207       7.286  12.316  -7.806  1.00  0.00      A    C  \\nATOM   1562  CE2 TYR A 207       8.235  10.264  -6.999  1.00  0.00      A    C  \\nATOM   1563  OH  TYR A 207       9.652  12.009  -7.802  1.00  0.00      A    O  \\nATOM   1564  CZ  TYR A 207       8.389  11.529  -7.535  1.00  0.00      A    C  \\nATOM   1565  N   GLN A 208       1.764  10.927  -5.455  1.00  0.00      A    N  \\nATOM   1566  CA  GLN A 208       0.315  10.798  -5.576  1.00  0.00      A    C  \\nATOM   1567  C   GLN A 208      -0.171  11.296  -6.933  1.00  0.00      A    C  \\nATOM   1568  CB  GLN A 208      -0.387  11.563  -4.453  1.00  0.00      A    C  \\nATOM   1569  O   GLN A 208       0.303  12.320  -7.431  1.00  0.00      A    O  \\nATOM   1570  CG  GLN A 208      -0.284  10.889  -3.091  1.00  0.00      A    C  \\nATOM   1571  CD  GLN A 208      -1.012  11.653  -2.002  1.00  0.00      A    C  \\nATOM   1572  NE2 GLN A 208      -1.724  10.930  -1.144  1.00  0.00      A    N  \\nATOM   1573  OE1 GLN A 208      -0.935  12.884  -1.931  1.00  0.00      A    O  \\nATOM   1574  N   TYR A 209      -1.025  10.501  -7.452  1.00  0.00      A    N  \\nATOM   1575  CA  TYR A 209      -1.558  10.810  -8.775  1.00  0.00      A    C  \\nATOM   1576  C   TYR A 209      -2.685  11.832  -8.683  1.00  0.00      A    C  \\nATOM   1577  CB  TYR A 209      -2.061   9.538  -9.463  1.00  0.00      A    C  \\nATOM   1578  O   TYR A 209      -3.618  11.667  -7.894  1.00  0.00      A    O  \\nATOM   1579  CG  TYR A 209      -2.598   9.773 -10.854  1.00  0.00      A    C  \\nATOM   1580  CD1 TYR A 209      -1.823  10.404 -11.824  1.00  0.00      A    C  \\nATOM   1581  CD2 TYR A 209      -3.882   9.365 -11.201  1.00  0.00      A    C  \\nATOM   1582  CE1 TYR A 209      -2.314  10.624 -13.107  1.00  0.00      A    C  \\nATOM   1583  CE2 TYR A 209      -4.384   9.580 -12.480  1.00  0.00      A    C  \\nATOM   1584  OH  TYR A 209      -4.085  10.424 -14.693  1.00  0.00      A    O  \\nATOM   1585  CZ  TYR A 209      -3.593  10.209 -13.425  1.00  0.00      A    C  \\nATOM   1586  N   ASP A 210      -2.557  12.933  -9.361  1.00  0.00      A    N  \\nATOM   1587  CA  ASP A 210      -3.592  13.937  -9.581  1.00  0.00      A    C  \\nATOM   1588  C   ASP A 210      -4.383  13.642 -10.854  1.00  0.00      A    C  \\nATOM   1589  CB  ASP A 210      -2.976  15.336  -9.657  1.00  0.00      A    C  \\nATOM   1590  O   ASP A 210      -3.968  14.022 -11.951  1.00  0.00      A    O  \\nATOM   1591  CG  ASP A 210      -4.015  16.435  -9.796  1.00  0.00      A    C  \\nATOM   1592  OD1 ASP A 210      -5.187  16.131 -10.107  1.00  0.00      A    O  \\nATOM   1593  OD2 ASP A 210      -3.658  17.616  -9.595  1.00  0.00      A    O  \\nATOM   1594  N   GLU A 211      -5.518  12.983 -10.630  1.00  0.00      A    N  \\nATOM   1595  CA  GLU A 211      -6.327  12.525 -11.756  1.00  0.00      A    C  \\nATOM   1596  C   GLU A 211      -6.855  13.701 -12.571  1.00  0.00      A    C  \\nATOM   1597  CB  GLU A 211      -7.491  11.660 -11.265  1.00  0.00      A    C  \\nATOM   1598  O   GLU A 211      -6.883  13.647 -13.802  1.00  0.00      A    O  \\nATOM   1599  CG  GLU A 211      -8.268  10.982 -12.385  1.00  0.00      A    C  \\nATOM   1600  CD  GLU A 211      -9.360  10.053 -11.880  1.00  0.00      A    C  \\nATOM   1601  OE1 GLU A 211      -9.709  10.122 -10.680  1.00  0.00      A    O  \\nATOM   1602  OE2 GLU A 211      -9.871   9.249 -12.691  1.00  0.00      A    O  \\nATOM   1603  N   LYS A 212      -7.154  14.790 -11.907  1.00  0.00      A    N  \\nATOM   1604  CA  LYS A 212      -7.693  15.967 -12.582  1.00  0.00      A    C  \\nATOM   1605  C   LYS A 212      -6.679  16.553 -13.560  1.00  0.00      A    C  \\nATOM   1606  CB  LYS A 212      -8.110  17.028 -11.562  1.00  0.00      A    C  \\nATOM   1607  O   LYS A 212      -7.031  16.915 -14.685  1.00  0.00      A    O  \\nATOM   1608  CG  LYS A 212      -8.728  18.273 -12.181  1.00  0.00      A    C  \\nATOM   1609  CD  LYS A 212      -9.120  19.290 -11.117  1.00  0.00      A    C  \\nATOM   1610  CE  LYS A 212      -9.662  20.570 -11.739  1.00  0.00      A    C  \\nATOM   1611  NZ  LYS A 212     -10.056  21.567 -10.699  1.00  0.00      A    N  \\nATOM   1612  N   ASN A 213      -5.410  16.618 -13.138  1.00  0.00      A    N  \\nATOM   1613  CA  ASN A 213      -4.378  17.274 -13.934  1.00  0.00      A    C  \\nATOM   1614  C   ASN A 213      -3.456  16.259 -14.603  1.00  0.00      A    C  \\nATOM   1615  CB  ASN A 213      -3.566  18.240 -13.069  1.00  0.00      A    C  \\nATOM   1616  O   ASN A 213      -2.472  16.634 -15.243  1.00  0.00      A    O  \\nATOM   1617  CG  ASN A 213      -4.402  19.382 -12.527  1.00  0.00      A    C  \\nATOM   1618  ND2 ASN A 213      -4.374  19.566 -11.212  1.00  0.00      A    N  \\nATOM   1619  OD1 ASN A 213      -5.070  20.093 -13.283  1.00  0.00      A    O  \\nATOM   1620  N   ASN A 214      -3.718  15.042 -14.476  1.00  0.00      A    N  \\nATOM   1621  CA  ASN A 214      -2.965  13.950 -15.084  1.00  0.00      A    C  \\nATOM   1622  C   ASN A 214      -1.467  14.096 -14.837  1.00  0.00      A    C  \\nATOM   1623  CB  ASN A 214      -3.251  13.871 -16.585  1.00  0.00      A    C  \\nATOM   1624  O   ASN A 214      -0.682  14.178 -15.783  1.00  0.00      A    O  \\nATOM   1625  CG  ASN A 214      -2.746  12.585 -17.209  1.00  0.00      A    C  \\nATOM   1626  ND2 ASN A 214      -2.445  12.631 -18.501  1.00  0.00      A    N  \\nATOM   1627  OD1 ASN A 214      -2.629  11.558 -16.535  1.00  0.00      A    O  \\nATOM   1628  N   LYS A 215      -1.137  14.186 -13.639  1.00  0.00      A    N  \\nATOM   1629  CA  LYS A 215       0.265  14.289 -13.245  1.00  0.00      A    C  \\nATOM   1630  C   LYS A 215       0.497  13.668 -11.871  1.00  0.00      A    C  \\nATOM   1631  CB  LYS A 215       0.716  15.750 -13.243  1.00  0.00      A    C  \\nATOM   1632  O   LYS A 215      -0.448  13.478 -11.102  1.00  0.00      A    O  \\nATOM   1633  CG  LYS A 215       0.095  16.588 -12.135  1.00  0.00      A    C  \\nATOM   1634  CD  LYS A 215       0.588  18.029 -12.181  1.00  0.00      A    C  \\nATOM   1635  CE  LYS A 215       0.043  18.844 -11.016  1.00  0.00      A    C  \\nATOM   1636  NZ  LYS A 215       0.623  20.220 -10.984  1.00  0.00      A    N  \\nATOM   1637  N   PHE A 216       1.700  13.329 -11.593  1.00  0.00      A    N  \\nATOM   1638  CA  PHE A 216       2.093  12.827 -10.281  1.00  0.00      A    C  \\nATOM   1639  C   PHE A 216       2.762  13.923  -9.460  1.00  0.00      A    C  \\nATOM   1640  CB  PHE A 216       3.035  11.627 -10.423  1.00  0.00      A    C  \\nATOM   1641  O   PHE A 216       3.636  14.633  -9.959  1.00  0.00      A    O  \\nATOM   1642  CG  PHE A 216       2.353  10.372 -10.894  1.00  0.00      A    C  \\nATOM   1643  CD1 PHE A 216       1.762   9.502  -9.986  1.00  0.00      A    C  \\nATOM   1644  CD2 PHE A 216       2.303  10.061 -12.247  1.00  0.00      A    C  \\nATOM   1645  CE1 PHE A 216       1.130   8.339 -10.419  1.00  0.00      A    C  \\nATOM   1646  CE2 PHE A 216       1.673   8.900 -12.688  1.00  0.00      A    C  \\nATOM   1647  CZ  PHE A 216       1.088   8.040 -11.772  1.00  0.00      A    C  \\nATOM   1648  N   ASN A 217       2.284  14.047  -8.231  1.00  0.00      A    N  \\nATOM   1649  CA  ASN A 217       2.847  15.008  -7.290  1.00  0.00      A    C  \\nATOM   1650  C   ASN A 217       3.730  14.323  -6.250  1.00  0.00      A    C  \\nATOM   1651  CB  ASN A 217       1.735  15.802  -6.602  1.00  0.00      A    C  \\nATOM   1652  O   ASN A 217       3.340  13.308  -5.671  1.00  0.00      A    O  \\nATOM   1653  CG  ASN A 217       0.902  16.608  -7.579  1.00  0.00      A    C  \\nATOM   1654  ND2 ASN A 217      -0.416  16.543  -7.430  1.00  0.00      A    N  \\nATOM   1655  OD1 ASN A 217       1.439  17.283  -8.462  1.00  0.00      A    O  \\nATOM   1656  N   TYR A 218       4.807  14.853  -6.154  1.00  0.00      A    N  \\nATOM   1657  CA  TYR A 218       5.713  14.377  -5.115  1.00  0.00      A    C  \\nATOM   1658  C   TYR A 218       5.120  14.601  -3.729  1.00  0.00      A    C  \\nATOM   1659  CB  TYR A 218       7.071  15.078  -5.222  1.00  0.00      A    C  \\nATOM   1660  O   TYR A 218       4.710  15.715  -3.396  1.00  0.00      A    O  \\nATOM   1661  CG  TYR A 218       8.058  14.654  -4.162  1.00  0.00      A    C  \\nATOM   1662  CD1 TYR A 218       8.635  13.386  -4.187  1.00  0.00      A    C  \\nATOM   1663  CD2 TYR A 218       8.415  15.519  -3.133  1.00  0.00      A    C  \\nATOM   1664  CE1 TYR A 218       9.544  12.990  -3.212  1.00  0.00      A    C  \\nATOM   1665  CE2 TYR A 218       9.323  15.133  -2.152  1.00  0.00      A    C  \\nATOM   1666  OH  TYR A 218      10.781  13.483  -1.231  1.00  0.00      A    O  \\nATOM   1667  CZ  TYR A 218       9.882  13.869  -2.200  1.00  0.00      A    C  \\nATOM   1668  N   VAL A 219       5.051  13.533  -2.947  1.00  0.00      A    N  \\nATOM   1669  CA  VAL A 219       4.528  13.647  -1.590  1.00  0.00      A    C  \\nATOM   1670  C   VAL A 219       5.510  13.018  -0.604  1.00  0.00      A    C  \\nATOM   1671  CB  VAL A 219       3.140  12.982  -1.458  1.00  0.00      A    C  \\nATOM   1672  O   VAL A 219       5.145  12.712   0.534  1.00  0.00      A    O  \\nATOM   1673  CG1 VAL A 219       2.089  13.765  -2.243  1.00  0.00      A    C  \\nATOM   1674  CG2 VAL A 219       3.197  11.532  -1.935  1.00  0.00      A    C  \\nATOM   1675  N   GLY A 220       6.713  12.703  -0.999  1.00  0.00      A    N  \\nATOM   1676  CA  GLY A 220       7.729  12.132  -0.130  1.00  0.00      A    C  \\nATOM   1677  C   GLY A 220       8.204  13.092   0.945  1.00  0.00      A    C  \\nATOM   1678  O   GLY A 220       7.593  14.140   1.161  1.00  0.00      A    O  \\nATOM   1679  N   LYS A 221       9.212  12.562   1.749  1.00  0.00      A    N  \\nATOM   1680  CA  LYS A 221       9.723  13.369   2.853  1.00  0.00      A    C  \\nATOM   1681  C   LYS A 221      10.707  14.423   2.353  1.00  0.00      A    C  \\nATOM   1682  CB  LYS A 221      10.395  12.481   3.901  1.00  0.00      A    C  \\nATOM   1683  O   LYS A 221      11.328  14.252   1.302  1.00  0.00      A    O  \\nATOM   1684  CG  LYS A 221       9.419  11.690   4.759  1.00  0.00      A    C  \\nATOM   1685  CD  LYS A 221      10.123  11.016   5.930  1.00  0.00      A    C  \\nATOM   1686  CE  LYS A 221       9.129  10.358   6.877  1.00  0.00      A    C  \\nATOM   1687  NZ  LYS A 221       9.791   9.870   8.124  1.00  0.00      A    N  \\nATOM   1688  OXT LYS A 221      10.423  15.061   3.339  1.00  0.00      A    O  \\nCONECT  338 1332\\nCONECT  379 1408\\nCONECT 1332  338\\nCONECT 1408  379\\nEND\\n\",\"pdb\");\n",
              "\tviewer_1678119207039042.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
              "\tviewer_1678119207039042.zoomTo();\n",
              "viewer_1678119207039042.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<py3Dmol.view at 0x7f146064b910>"
            ]
          },
          "metadata": {},
          "execution_count": 89
        }
      ],
      "source": [
        "show_pdb(pdb_preprocessed_path)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "q7W8ZdcpFI5X"
      },
      "source": [
        "# Add mutants"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Mutants to simulate\n",
        "#@markdown If you want to simulate mutants upload a file with a list of mutants\n",
        "#@markdown The file should have the following format:\n",
        "\n",
        "#@markdown Commented lines start with \\#.\n",
        "#@markdown Each line corresponds to a single mutant. Each mutation is described\n",
        "#@markdown as XNY, where X is one-letter code of the original amino acid, N\n",
        "#@markdown is the residue id of the mutated amino acid in the PDB file, \n",
        "#@markdown and Y is the mutated amino acid.\n",
        "\n",
        "simulate_mutants = True #@param {type:\"boolean\"}\n",
        "if simulate_mutants:\n",
        "    uploaded = files.upload()\n",
        "    mutations_file = list(uploaded.keys())[-1]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 74
        },
        "id": "OxHFVkKqYI9d",
        "outputId": "da0c17a0-523d-4f59-8877-e72cfda32e91"
      },
      "execution_count": 51,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ],
            "text/html": [
              "\n",
              "     <input type=\"file\" id=\"files-ced8610d-6e37-4fc6-a573-291135788575\" name=\"files[]\" multiple disabled\n",
              "        style=\"border:none\" />\n",
              "     <output id=\"result-ced8610d-6e37-4fc6-a573-291135788575\">\n",
              "      Upload widget is only available when the cell has been executed in the\n",
              "      current browser session. Please rerun this cell to enable.\n",
              "      </output>\n",
              "      <script>// Copyright 2017 Google LLC\n",
              "//\n",
              "// Licensed under the Apache License, Version 2.0 (the \"License\");\n",
              "// you may not use this file except in compliance with the License.\n",
              "// You may obtain a copy of the License at\n",
              "//\n",
              "//      http://www.apache.org/licenses/LICENSE-2.0\n",
              "//\n",
              "// Unless required by applicable law or agreed to in writing, software\n",
              "// distributed under the License is distributed on an \"AS IS\" BASIS,\n",
              "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
              "// See the License for the specific language governing permissions and\n",
              "// limitations under the License.\n",
              "\n",
              "/**\n",
              " * @fileoverview Helpers for google.colab Python module.\n",
              " */\n",
              "(function(scope) {\n",
              "function span(text, styleAttributes = {}) {\n",
              "  const element = document.createElement('span');\n",
              "  element.textContent = text;\n",
              "  for (const key of Object.keys(styleAttributes)) {\n",
              "    element.style[key] = styleAttributes[key];\n",
              "  }\n",
              "  return element;\n",
              "}\n",
              "\n",
              "// Max number of bytes which will be uploaded at a time.\n",
              "const MAX_PAYLOAD_SIZE = 100 * 1024;\n",
              "\n",
              "function _uploadFiles(inputId, outputId) {\n",
              "  const steps = uploadFilesStep(inputId, outputId);\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  // Cache steps on the outputElement to make it available for the next call\n",
              "  // to uploadFilesContinue from Python.\n",
              "  outputElement.steps = steps;\n",
              "\n",
              "  return _uploadFilesContinue(outputId);\n",
              "}\n",
              "\n",
              "// This is roughly an async generator (not supported in the browser yet),\n",
              "// where there are multiple asynchronous steps and the Python side is going\n",
              "// to poll for completion of each step.\n",
              "// This uses a Promise to block the python side on completion of each step,\n",
              "// then passes the result of the previous step as the input to the next step.\n",
              "function _uploadFilesContinue(outputId) {\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  const steps = outputElement.steps;\n",
              "\n",
              "  const next = steps.next(outputElement.lastPromiseValue);\n",
              "  return Promise.resolve(next.value.promise).then((value) => {\n",
              "    // Cache the last promise value to make it available to the next\n",
              "    // step of the generator.\n",
              "    outputElement.lastPromiseValue = value;\n",
              "    return next.value.response;\n",
              "  });\n",
              "}\n",
              "\n",
              "/**\n",
              " * Generator function which is called between each async step of the upload\n",
              " * process.\n",
              " * @param {string} inputId Element ID of the input file picker element.\n",
              " * @param {string} outputId Element ID of the output display.\n",
              " * @return {!Iterable<!Object>} Iterable of next steps.\n",
              " */\n",
              "function* uploadFilesStep(inputId, outputId) {\n",
              "  const inputElement = document.getElementById(inputId);\n",
              "  inputElement.disabled = false;\n",
              "\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  outputElement.innerHTML = '';\n",
              "\n",
              "  const pickedPromise = new Promise((resolve) => {\n",
              "    inputElement.addEventListener('change', (e) => {\n",
              "      resolve(e.target.files);\n",
              "    });\n",
              "  });\n",
              "\n",
              "  const cancel = document.createElement('button');\n",
              "  inputElement.parentElement.appendChild(cancel);\n",
              "  cancel.textContent = 'Cancel upload';\n",
              "  const cancelPromise = new Promise((resolve) => {\n",
              "    cancel.onclick = () => {\n",
              "      resolve(null);\n",
              "    };\n",
              "  });\n",
              "\n",
              "  // Wait for the user to pick the files.\n",
              "  const files = yield {\n",
              "    promise: Promise.race([pickedPromise, cancelPromise]),\n",
              "    response: {\n",
              "      action: 'starting',\n",
              "    }\n",
              "  };\n",
              "\n",
              "  cancel.remove();\n",
              "\n",
              "  // Disable the input element since further picks are not allowed.\n",
              "  inputElement.disabled = true;\n",
              "\n",
              "  if (!files) {\n",
              "    return {\n",
              "      response: {\n",
              "        action: 'complete',\n",
              "      }\n",
              "    };\n",
              "  }\n",
              "\n",
              "  for (const file of files) {\n",
              "    const li = document.createElement('li');\n",
              "    li.append(span(file.name, {fontWeight: 'bold'}));\n",
              "    li.append(span(\n",
              "        `(${file.type || 'n/a'}) - ${file.size} bytes, ` +\n",
              "        `last modified: ${\n",
              "            file.lastModifiedDate ? file.lastModifiedDate.toLocaleDateString() :\n",
              "                                    'n/a'} - `));\n",
              "    const percent = span('0% done');\n",
              "    li.appendChild(percent);\n",
              "\n",
              "    outputElement.appendChild(li);\n",
              "\n",
              "    const fileDataPromise = new Promise((resolve) => {\n",
              "      const reader = new FileReader();\n",
              "      reader.onload = (e) => {\n",
              "        resolve(e.target.result);\n",
              "      };\n",
              "      reader.readAsArrayBuffer(file);\n",
              "    });\n",
              "    // Wait for the data to be ready.\n",
              "    let fileData = yield {\n",
              "      promise: fileDataPromise,\n",
              "      response: {\n",
              "        action: 'continue',\n",
              "      }\n",
              "    };\n",
              "\n",
              "    // Use a chunked sending to avoid message size limits. See b/62115660.\n",
              "    let position = 0;\n",
              "    do {\n",
              "      const length = Math.min(fileData.byteLength - position, MAX_PAYLOAD_SIZE);\n",
              "      const chunk = new Uint8Array(fileData, position, length);\n",
              "      position += length;\n",
              "\n",
              "      const base64 = btoa(String.fromCharCode.apply(null, chunk));\n",
              "      yield {\n",
              "        response: {\n",
              "          action: 'append',\n",
              "          file: file.name,\n",
              "          data: base64,\n",
              "        },\n",
              "      };\n",
              "\n",
              "      let percentDone = fileData.byteLength === 0 ?\n",
              "          100 :\n",
              "          Math.round((position / fileData.byteLength) * 100);\n",
              "      percent.textContent = `${percentDone}% done`;\n",
              "\n",
              "    } while (position < fileData.byteLength);\n",
              "  }\n",
              "\n",
              "  // All done.\n",
              "  yield {\n",
              "    response: {\n",
              "      action: 'complete',\n",
              "    }\n",
              "  };\n",
              "}\n",
              "\n",
              "scope.google = scope.google || {};\n",
              "scope.google.colab = scope.google.colab || {};\n",
              "scope.google.colab._files = {\n",
              "  _uploadFiles,\n",
              "  _uploadFilesContinue,\n",
              "};\n",
              "})(self);\n",
              "</script> "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Saving mutations.txt to mutations (1).txt\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 91,
      "metadata": {
        "id": "VoylKPKIFMgU"
      },
      "outputs": [],
      "source": [
        "#@title Read mutation file and prepare alignments\n",
        "alphabet = list(Bio.SeqUtils.IUPACData.protein_letters_1to3.keys())\n",
        "alphabet.append(\"-\")\n",
        "\n",
        "parser = PDBParser()\n",
        "structure = parser.get_structure('X', pdb_preprocessed_path)\n",
        "\n",
        "residues = \"\"\n",
        "resids = []\n",
        "for model in structure:\n",
        "    for chain in model:\n",
        "        for i, residue in enumerate(chain.get_residues()):\n",
        "            residue_1l = Bio.SeqUtils.seq1(residue.resname)\n",
        "            resid = residue.id[1]\n",
        "            if residue_1l != \"X\":\n",
        "                residues += residue_1l\n",
        "                resids.append(resid)\n",
        "\n",
        "record = SeqRecord(Seq(residues), id=\"0\", name=\"template\", \n",
        "                   description=\"input protein\")\n",
        "\n",
        "mutant_n = 0\n",
        "with open(mutations_file, \"r\") as f:\n",
        "    for line in f:\n",
        "        if line.startswith(\"#\") or line.strip() == \"\":\n",
        "            continue\n",
        "\n",
        "        folder_mask = project_dir+\"/md_mutant_\"\n",
        "\n",
        "        if not os.path.exists(f\"{folder_mask}{mutant_n}\"):\n",
        "            os.mkdir(f\"{folder_mask}{mutant_n}\")\n",
        "        \n",
        "        new_sequence = residues\n",
        "        for mutation in line.split():\n",
        "            original_AA = mutation[0].upper()\n",
        "            if original_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "            \n",
        "            mutant_AA = mutation[-1].upper()\n",
        "            if mutant_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided for in\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "                \n",
        "            try:\n",
        "                resid = int(mutation[1:-1])\n",
        "            except:\n",
        "                print(f\"Can't get resid for mutation {mutation}. Check you input\")\n",
        "                raise\n",
        "            \n",
        "            if residues[resids.index(resid)] != original_AA:\n",
        "                raise Exception(f\"In the PDB file resid {resid} has amino acid\"+\\\n",
        "                                f\" {residues[resids.index(resid)]} while in\"+\\\n",
        "                                f\"mutation file {original_AA} is provide\")\n",
        "            new_sequence = list(new_sequence)\n",
        "            new_sequence[resids.index(resid)] = mutant_AA\n",
        "            new_sequence = \"\".join(new_sequence)\n",
        "\n",
        "        record_mutant = SeqRecord(Seq(new_sequence), id=\"1\", name=\"mutant\", \n",
        "                description=\"mutated protein\")\n",
        "        \n",
        "        handle = open(f\"{folder_mask}{mutant_n}/alignment.ali\", \"w\")\n",
        "        writer = Bio.SeqIO.PirIO.PirWriter(handle)\n",
        "        writer.write_record(record)\n",
        "        writer.write_record(record_mutant)\n",
        "        handle.close()\n",
        "        mutant_n += 1\n",
        "\n",
        "# setting environ variables for modeller \n",
        "os.environ['N_MUTANT'] = str(mutant_n)\n",
        "os.environ['N_RESID'] = str(len(residues))\n",
        "os.environ['PDB'] = os.path.basename(pdb_preprocessed_path).split('.')[0]\n",
        "if not os.path.exists(os.path.join(project_dir, 'md_base')):\n",
        "    os.mkdir(os.path.join(project_dir, 'md_base'))\n",
        "os.environ['MD_PATH'] = os.path.join(project_dir, 'md_base')\n",
        "os.environ['ALPHAGMX'] = project_dir\n",
        "#fn_pdb_md = os.path.join(os.path.dirname(pdb_preprocessed_path), 'md_base', os.path.splitext(os.path.basename(pdb_preprocessed_path))[0]+\"_no_water.pdb\")\n",
        "fn_pdb_md = os.path.join(os.path.dirname(pdb_preprocessed_path), 'md_base', os.path.splitext(os.path.basename(pdb_preprocessed_path))[0]+\".pdb\")\n",
        "#shutil.copy(os.path.splitext(pdb_preprocessed_path)[0]+\"_no_water.pdb\", fn_pdb_md)\n",
        "shutil.copy(os.path.splitext(pdb_preprocessed_path)[0]+\".pdb\", fn_pdb_md)\n",
        "os.environ['PDB4MD'] = fn_pdb_md.split('.')[0]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 95,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "e3QMz2A94hb4",
        "outputId": "52b1e38b-c28d-4e16-9c95-85d03261014d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">P1;0\n",
            "structureX:/content/alphagmx_project/md_base/target_preprocessed:1:A:221:A::::\n",
            "VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQ\n",
            "RVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGT\n",
            "NAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKAL\n",
            "GSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK*\n",
            ">P1;1\n",
            "sequence:mutated protein:1::221:::::\n",
            "APVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQ\n",
            "RVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGT\n",
            "NAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKAL\n",
            "GSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK*\n"
          ]
        }
      ],
      "source": [
        "!head -n 20 alphagmx_project/md_mutant_0/alignment.ali"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 94,
      "metadata": {
        "id": "W1IGNECJ1LyS"
      },
      "outputs": [],
      "source": [
        "#@title Correct naming for alignment file\n",
        "%%bash\n",
        "#echo \"s/template - input protein/structureX:${PDB4MD}:1:A:${N_RESID}:A::::/g\"\n",
        "#echo \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/${PDB}_no_water:1:A:${N_RESID}:A::::/g\"\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "sed -i 's/XX/P1/g' alphagmx_project/md_mutant_${i}/alignment.ali\n",
        "#sed -i \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/${PDB}_no_water:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "#sed -i \"s/template - input protein/structureX:${PDB4MD}:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "#sed -i \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/target_preprocessed_no_water:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/template - input protein/structureX:\\/content\\/alphagmx_project\\/md_base\\/target_preprocessed:1:A:${N_RESID}:A::::/g\" alphagmx_project/md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/mutant - mutated protein/sequence:mutated protein:1::${N_RESID}:::::/g\" alphagmx_project/md_mutant_${i}/alignment.ali\n",
        "done\n",
        "#TODO: A is for chain id? Is it always A here?\n",
        "#TODO: this cell will be completed even if something goes wrong"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 96,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aTlUZ6aqNLwy",
        "outputId": "a6a120ca-72c5-498f-ef39-9e6d3231b732"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "check_ali___> Checking the sequence-structure alignment. \n",
            "\n",
            "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
            "\n",
            "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
            "----------------------------------------------\n",
            "END OF TABLE\n",
            "read_to_681_> topology.submodel read from topology file:        3\n",
            "patch_s_522_> Number of disulfides patched in MODEL:        2\n",
            "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
            "              least one known structure available. MDT, not library, potential is used.\n",
            "0 atoms in HETATM/BLK residues constrained\n",
            "to protein atoms within 2.30 angstroms\n",
            "and protein CA atoms within 10.00 angstroms\n",
            "0 atoms in residues without defined topology\n",
            "constrained to be rigid bodies\n",
            "condens_443_> Restraints marked for deletion were removed.\n",
            "              Total number of restraints before, now:    23449    21841\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            ">> Model assessment by DOPE potential\n",
            "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      221\n",
            "Number of all, selected real atoms                :     1686    1686\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    21841   21841\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   317856\n",
            "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        1    9999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :   15.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :      -20929.4219\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "DOPE score               : -20929.421875\n",
            ">> Model assessment by GA341 potential\n",
            "\n",
            "Surface library          : /usr/local/lib/modeller-10.4/modlib/surf5.de\n",
            "Pair library             : /usr/local/lib/modeller-10.4/modlib/pair9.de\n",
            "Chain identifier         : _\n",
            "% sequence identity      :    99.547997\n",
            "Sequence length          :          221\n",
            "Compactness              :     0.332978\n",
            "Native energy (pair)     :   -83.123132\n",
            "Native energy (surface)  :   -12.060578\n",
            "Native energy (combined) :    -4.870826\n",
            "Z score (pair)           :    -4.571750\n",
            "Z score (surface)        :    -6.283535\n",
            "Z score (combined)       :    -7.113250\n",
            "GA341 score              :     1.000000\n",
            "\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      221\n",
            "Number of all, selected real atoms                :     1686    1686\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    21841   21841\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3640\n",
            "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        0   99999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :    4.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :        1232.9640\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Summary of the restraint violations: \n",
            "\n",
            "   NUM     ... number of restraints.\n",
            "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
            "   RVIOL   ... relative difference from the best value.\n",
            "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
            "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
            "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
            "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
            "\n",
            " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
            "------------------------------------------------------------------------------------------------------\n",
            " 1 Bond length potential              :    1730       0      0   0.005   0.005      13.381       1.000\n",
            " 2 Bond angle potential               :    2348       2      8   2.353   2.353      247.48       1.000\n",
            " 3 Stereochemical cosine torsion poten:    1121       0     43  50.133  50.133      411.47       1.000\n",
            " 4 Stereochemical improper torsion pot:     699       0      0   1.187   1.187      22.743       1.000\n",
            " 5 Soft-sphere overlap restraints     :    3640       0      0   0.001   0.001     0.83723       1.000\n",
            " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 9 Distance restraints 1 (CA-CA)      :    4512       0      0   0.107   0.107      61.011       1.000\n",
            "10 Distance restraints 2 (N-O)        :    4725       0      0   0.171   0.171      129.56       1.000\n",
            "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "13 Mainchain Omega dihedral restraints:     220       0      2   3.934   3.934      40.162       1.000\n",
            "14 Sidechain Chi_1 dihedral restraints:     179       0      2  46.081  46.081      6.9368       1.000\n",
            "15 Sidechain Chi_2 dihedral restraints:     137       0      0  50.393  50.393      29.514       1.000\n",
            "16 Sidechain Chi_3 dihedral restraints:      64       0      1  57.664  57.664      37.378       1.000\n",
            "17 Sidechain Chi_4 dihedral restraints:      27       0      0  68.586  68.586      16.050       1.000\n",
            "18 Disulfide distance restraints      :       2       0      0   0.014   0.014     0.65842E-01   1.000\n",
            "19 Disulfide angle restraints         :       4       0      0   2.439   2.439     0.52532       1.000\n",
            "20 Disulfide dihedral angle restraints:       2       0      0  31.317  31.317      1.8540       1.000\n",
            "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "23 Distance restraints 3 (SDCH-MNCH)  :    3637       0      0   0.325   0.325      67.790       1.000\n",
            "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "25 Phi/Psi pair of dihedral restraints:     219       4     21  23.497  31.428      58.907       1.000\n",
            "26 Distance restraints 4 (SDCH-SDCH)  :    2215       0      0   0.527   0.527      87.291       1.000\n",
            "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "\n",
            "\n",
            "\n",
            "# Heavy relative violation of each residue is written to: 1.V99990001\n",
            "# The profile is NOT normalized by the number of restraints.\n",
            "# The profiles are smoothed over a window of residues:    1\n",
            "# The sum of all numbers in the file:   17339.8457\n",
            "\n",
            "\n",
            "\n",
            "List of the violated restraints:\n",
            "   A restraint is violated when the relative difference\n",
            "   from the best value (RVIOL) is larger than CUTOFF.\n",
            "\n",
            "   ICSR   ... index of a restraint in the current set.\n",
            "   RESNO  ... residue numbers of the first two atoms.\n",
            "   ATM    ... IUPAC atom names of the first two atoms.\n",
            "   FEAT   ... the value of the feature in the model.\n",
            "   restr  ... the mean of the basis restraint with the smallest\n",
            "              difference from the model (local minimum).\n",
            "   viol   ... difference from the local minimum.\n",
            "   rviol  ... relative difference from the local minimum.\n",
            "   RESTR  ... the best value (global minimum).\n",
            "   VIOL   ... difference from the best value.\n",
            "   RVIOL  ... relative difference from the best value.\n",
            "\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature  2                           : Bond angle potential                    \n",
            "List of the RVIOL violations larger than   :       4.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   2905 110Y 111A C   N     838  840  153.19  120.00   33.19    7.54  120.00   33.19    7.54\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
            "List of the RVIOL violations larger than   :       6.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   5912   6E   7P C   N      42   44  -62.56  -58.70   25.54    1.82  -64.50  152.46   11.27\n",
            "    1          7P   7P N   CA     44   45   -5.25  -30.50                  147.20\n",
            "    2   5916  10T  11S C   N      69   71 -120.12 -136.60   72.22    3.28   56.90 -177.48   18.88\n",
            "    2         11S  11S N   CA     71   72   80.88  151.20                   36.40\n",
            "    3   5962  56P  57N C   N     401  403 -119.51  -63.20   70.54    8.27  -63.20   70.54    8.27\n",
            "    3         57N  57N N   CA    403  404    1.39  -41.10                  -41.10\n",
            "    4   6016 110Y 111A C   N     838  840  128.21 -134.00  102.05    4.52  -68.20  165.88   14.93\n",
            "    4        111A 111A N   CA    840  841  117.81  147.00                  145.30\n",
            "\n",
            "\n",
            "report______> Distribution of short non-bonded contacts:\n",
            "\n",
            "\n",
            "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
            "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
            "FREQUENCY:     0    0    0    0    0    8   17   83  102  135  164  205  246  270  274\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "\n",
            ">> Summary of successfully produced models:\n",
            "Filename                          molpdf     DOPE score    GA341 score\n",
            "----------------------------------------------------------------------\n",
            "1.B99990001.pdb               1232.96399   -20929.42188        1.00000\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run modeller\n",
        "for i in range(mutant_n):\n",
        "    os.chdir(project_dir+f\"/md_mutant_{i}\")\n",
        "    env = Environ()\n",
        "    a = AutoModel(env, alnfile='alignment.ali',\n",
        "                  knowns='0', sequence='1',\n",
        "                  assess_methods=(assess.DOPE,\n",
        "                                  #soap_protein_od.Scorer(),\n",
        "                                  assess.GA341))\n",
        "    a.starting_model = 1\n",
        "    a.ending_model = 1\n",
        "    a.make()\n",
        "    for file in os.listdir(\".\"):\n",
        "        if file.endswith(\"pdb\"):\n",
        "            shutil.copy2(file, 'input.pdb')\n",
        "os.chdir(\"/content\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "o4PcAAuQAh5c"
      },
      "source": [
        "# Calculate most probable protonation states "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 97,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L7OhlVxhCvhe",
        "outputId": "e5fb21a4-f6e2-4d31-b6d2-fb9b433c465f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Preparing /content/alphagmx_project/md_base/target_preprocessed for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for /content/alphagmx_project/md_base/target_preprocessed.pdb\n",
            "Preparing input for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for input.pdb\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:299: UserWarning: Trying to unpickle estimator DecisionTreeRegressor from version 1.0.2 when using version 1.2.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:299: UserWarning: Trying to unpickle estimator RandomForestRegressor from version 1.0.2 when using version 1.2.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:299: UserWarning: Trying to unpickle estimator DecisionTreeRegressor from version 1.0.2 when using version 1.2.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:299: UserWarning: Trying to unpickle estimator RandomForestRegressor from version 1.0.2 when using version 1.2.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n"
          ]
        }
      ],
      "source": [
        "#@ Run pkaAni\n",
        "%%bash\n",
        "cd ${MD_PATH}\n",
        "in_pdb=${PDB4MD}.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "cd ../md_mutant_${i}\n",
        "in_pdb=input.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Select the force field\n",
        "#@markdown We currently provide access to three force fields: AMBER99-ILDN, CHARMM36m, and AMBER14SB\n",
        "ff = 'charmm36' #@param {type:\"string\"} [\"amber99\", \"amber14\", \"charmm36\"]\n",
        "os.environ['FF']=ff"
      ],
      "metadata": {
        "id": "UykZO7pon2Er"
      },
      "execution_count": 99,
      "outputs": []
    },
    {
      "cell_type": "code",
      "execution_count": 100,
      "metadata": {
        "id": "YLJhqZARoCKN"
      },
      "outputs": [],
      "source": [
        "#@title Get force fields for pdb2gmx\n",
        "if ff == 'amber14':\n",
        "    wget.download(\"https://github.com/pbuslaev/alphagmx/raw/main/forcefields/amber14sb.ff.tar.gz\")\n",
        "    tar = tarfile.open(\"amber14sb.ff.tar.gz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n",
        "if ff == 'charmm36':\n",
        "    wget.download(\"http://mackerell.umaryland.edu/download.php?filename=CHARMM_ff_params_files/charmm36-jul2021.ff.tgz\")\n",
        "    tar = tarfile.open(\"charmm36-jul2021.ff.tgz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 101,
      "metadata": {
        "id": "0eqhMhZ3NzHU"
      },
      "outputs": [],
      "source": [
        "#@title Change residue names in accordance with pKa calculation\n",
        "pH = 7\n",
        "names              = [\"ASP\",  \"GLU\",  \"HIS\", \"LYS\"]\n",
        "charmmProtonated   = [\"ASPP\", \"GLUP\", \"HSP\", \"LYS\"]\n",
        "charmmDeprotonated = [\"ASP\",  \"GLU\",  \"HIS\", \"LSN\"]\n",
        "amberProtonated    = [\"ASH\",  \"GLH\",  \"HIP\", \"LYS\"]\n",
        "amberDeprotonated  = [\"ASP\",  \"GLU\",  \"HIS\", \"LYN\"]\n",
        "if ff == 'charmm36':\n",
        "    protNames   = charmmProtonated\n",
        "    deprotNames = charmmDeprotonated\n",
        "else:\n",
        "    protNames   = amberProtonated\n",
        "    deprotNames = amberDeprotonated\n",
        "for i in range(-1, mutant_n, 1):\n",
        "    parser = PDBParser()\n",
        "    if i == -1:\n",
        "        #structure = parser.get_structure('X', path+\"pdb\"+pdbid+\".ent\") #PP: why .ent here?\n",
        "        structure = parser.get_structure('X', pdb_preprocessed_path)\n",
        "        #logname = path+pdbid+\"_processed_pka.log\"\n",
        "        logname =  os.environ['PDB4MD']+\"_pka.log\"\n",
        "        #outpath = path+\"input_pka.pdb\"\n",
        "        outpath = os.path.join(os.environ['MD_PATH'], \"input_pka.pdb\")\n",
        "    else:\n",
        "        structure = parser.get_structure('X', os.path.join(os.environ[\"ALPHAGMX\"], f\"md_mutant_{i}/input.pdb\"))\n",
        "        logname = os.path.join(os.environ[\"ALPHAGMX\"], f\"md_mutant_{i}/input_pka.log\")\n",
        "        outpath = os.path.join(os.environ[\"ALPHAGMX\"], f\"md_mutant_{i}/input_pka.pdb\")\n",
        "    residues = []\n",
        "    pkas     = []\n",
        "    with open(logname, \"r\") as f:\n",
        "        for line in f:\n",
        "            if line.startswith(\"Residue\"):\n",
        "                continue\n",
        "            if line.strip() == \"\":\n",
        "                continue\n",
        "            resname, resid, chain, pka = line.split()[0].split(\"-\")[0], \\\n",
        "                                         int(line.split()[0].split(\"-\")[1]), \\\n",
        "                                         line.split()[1], \\\n",
        "                                         float(line.split()[2])\n",
        "            residues.append((resname, resid, chain))\n",
        "            pkas.append(pka)\n",
        "\n",
        "    for model in structure:\n",
        "        for chain in model:\n",
        "            for i, residue in enumerate(chain.get_residues()):\n",
        "                if (residue.resname, residue.id[1], chain.id) in residues:\n",
        "                    id = residues.index((residue.resname, residue.id[1], chain.id))\n",
        "                    if (residue.resname in names) and pH < pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            protNames[names.index(residue.resname)]\n",
        "                    if (residue.resname in names) and pH > pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            deprotNames[names.index(residue.resname)]\n",
        "\n",
        "    pdb_io = PDBIO()\n",
        "    pdb_io.set_structure(structure)\n",
        "    pdb_io.save(outpath)             "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 79,
      "metadata": {
        "id": "SG_Rljap7QnB",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "142d30d7-c839-4e53-f604-f166b94beb15"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/alphagmx/md_base /content/alphagmx//md_mutant_0\n"
          ]
        }
      ],
      "source": [
        "# #PP: this is can be done in the next cell within the cycle\n",
        "# %%bash\n",
        "# # Declare a filename array\n",
        "# arrVar=(${MD_PATH})\n",
        "\n",
        "# # Add mutants\n",
        "# for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "# do\n",
        "# #arrVar+=(\"./md_mutant_${i}\")\n",
        "# arrVar+=(${ALPHAGMX}\"/md_mutant_${i}\")\n",
        "# done\n",
        "\n",
        "# #echo \"${arrVar[@]}\"\n",
        "\n",
        "# for value in \"${arrVar[@]}\"\n",
        "# do\n",
        "\n",
        "# sed -i 's/GLUP /GLUP/g' ${value}/input_pka.pdb\n",
        "# sed -i 's/ASPP /ASPP/g' ${value}/input_pka.pdb\n",
        "\n",
        "# done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 102,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CLvZTTAcZe2-",
        "outputId": "1a19a1c0-f892-45c1-a254-ef5d3ca562ae"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/alphagmx_project/md_base\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 1688 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 221 residues with 1688 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   221   1688  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (1688 atoms, 221 residues)\n",
            "\n",
            "Identified residue VAL1 as a starting terminus.\n",
            "\n",
            "Identified residue LYS221 as a ending terminus.\n",
            "Start terminus VAL-1: NH3+\n",
            "End terminus LYS-221: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 221 residues with 3317 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 3363, now 3363\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  219 cmap torsion pairs\n",
            "\n",
            "There are 8990 dihedrals,  600 impropers, 6093 angles\n",
            "          8809 pairs,     3363 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 23895.800 a.m.u.\n",
            "\n",
            "Total charge 5.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 3317 atoms\n",
            "Volume: 117.081 nm^3, corresponds to roughly 52600 electrons\n",
            "No velocities found\n",
            "    system size :  5.669  4.753  4.345 (nm)\n",
            "    center      :  0.078 -0.108 -0.002 (nm)\n",
            "    box vectors :  5.669  4.753  4.345 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  : 117.08               (nm^3)\n",
            "    shift       :  4.756  4.485  4.174 (nm)\n",
            "new center      :  4.834  4.377  4.173 (nm)\n",
            "new box vectors :  9.669  8.753  8.345 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 706.26               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 21923 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to 1063772079\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 21923      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.453 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x60, spacing 0.134 0.137 0.139\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Will try to add 64 SOD ions and 69 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 133 solute molecules in topology file (topol.top)  by 64 SOD and 69 CLA ions.\n",
            "/content/alphagmx_project//md_mutant_0\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 1686 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 221 residues with 1686 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   221   1686  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (1686 atoms, 221 residues)\n",
            "\n",
            "Identified residue ALA1 as a starting terminus.\n",
            "\n",
            "Identified residue LYS221 as a ending terminus.\n",
            "Start terminus ALA-1: NH3+\n",
            "End terminus LYS-221: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 221 residues with 3311 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 3357, now 3357\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  219 cmap torsion pairs\n",
            "\n",
            "There are 8972 dihedrals,  600 impropers, 6081 angles\n",
            "          8791 pairs,     3357 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 23867.746 a.m.u.\n",
            "\n",
            "Total charge 5.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 3311 atoms\n",
            "Volume: 120.493 nm^3, corresponds to roughly 54200 electrons\n",
            "No velocities found\n",
            "    system size :  5.645  4.606  4.635 (nm)\n",
            "    center      :  0.050  0.067  0.262 (nm)\n",
            "    box vectors :  5.645  4.606  4.634 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  : 120.49               (nm^3)\n",
            "    shift       :  4.772  4.236  4.055 (nm)\n",
            "new center      :  4.823  4.303  4.318 (nm)\n",
            "new box vectors :  9.645  8.606  8.635 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 716.75               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 22333 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to 2147435007\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 22333      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.434 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x64, spacing 0.134 0.134 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Will try to add 65 SOD ions and 70 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 135 solute molecules in topology file (topol.top)  by 65 SOD and 70 CLA ions.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Cloning into 'alphagmx'...\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                   CYS47   CYS52  CYS177\n",
            "                   SG338   SG379  SG1332\n",
            "   CYS52   SG379   0.967\n",
            "  CYS177  SG1332   0.218   1.083\n",
            "  CYS188  SG1408   1.081   0.204   1.201\n",
            "Linking CYS-47 SG-338 and CYS-177 SG-1332...\n",
            "Linking CYS-52 SG-379 and CYS-188 SG-1408...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 8875 pairs\n",
            "Before cleaning: 8990 dihedrals\n",
            "\n",
            "GROMACS reminds you: \"When doing HPC, don't communica\" (Jim Demmel)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"When doing HPC, don't communica\" (Jim Demmel)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 6x5x5 boxes\n",
            "Solvent box contains 78435 atoms in 26145 residues\n",
            "Removed 9543 solvent atoms due to solvent-solvent overlap\n",
            "Removed 3123 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 21923 residues\n",
            "Generated solvent containing 65769 atoms in 21923 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Output configuration contains 69086 atoms in 22144 residues\n",
            "Volume                 :      706.26 (nm^3)\n",
            "Density                :     988.545 (g/l)\n",
            "Number of solvent molecules:  21923   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.1#\n",
            "\n",
            "GROMACS reminds you: \"I Need Love, Not Games\" (Iggy Pop & Kate Pierson)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 31685]:\n",
            "  System has non-zero total charge: 5.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 31685]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 141486.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "GROMACS reminds you: \"Those who cannot remember the past are condemned to compute it.\" (Steve Pinker)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 69086 elements\n",
            "Group     1 (        Protein) has  3317 elements\n",
            "Group     2 (      Protein-H) has  1688 elements\n",
            "Group     3 (        C-alpha) has   221 elements\n",
            "Group     4 (       Backbone) has   663 elements\n",
            "Group     5 (      MainChain) has   883 elements\n",
            "Group     6 (   MainChain+Cb) has  1085 elements\n",
            "Group     7 (    MainChain+H) has  1089 elements\n",
            "Group     8 (      SideChain) has  2228 elements\n",
            "Group     9 (    SideChain-H) has   805 elements\n",
            "Group    10 (    Prot-Masses) has  3317 elements\n",
            "Group    11 (    non-Protein) has 65769 elements\n",
            "Group    12 (          Water) has 65769 elements\n",
            "Group    13 (            SOL) has 65769 elements\n",
            "Group    14 (      non-Water) has  3317 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 21923\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.2#\n",
            "Using random seed -1073743124.\n",
            "Replacing solvent molecule 16607 (atom 53138) with SOD\n",
            "Replacing solvent molecule 5372 (atom 19433) with SOD\n",
            "Replacing solvent molecule 2306 (atom 10235) with SOD\n",
            "Replacing solvent molecule 1135 (atom 6722) with SOD\n",
            "Replacing solvent molecule 18857 (atom 59888) with SOD\n",
            "Replacing solvent molecule 13920 (atom 45077) with SOD\n",
            "Replacing solvent molecule 8399 (atom 28514) with SOD\n",
            "Replacing solvent molecule 17351 (atom 55370) with SOD\n",
            "Replacing solvent molecule 14520 (atom 46877) with SOD\n",
            "Replacing solvent molecule 9039 (atom 30434) with SOD\n",
            "Replacing solvent molecule 6171 (atom 21830) with SOD\n",
            "Replacing solvent molecule 5859 (atom 20894) with SOD\n",
            "Replacing solvent molecule 10161 (atom 33800) with SOD\n",
            "Replacing solvent molecule 11821 (atom 38780) with SOD\n",
            "Replacing solvent molecule 5383 (atom 19466) with SOD\n",
            "Replacing solvent molecule 17733 (atom 56516) with SOD\n",
            "Replacing solvent molecule 611 (atom 5150) with SOD\n",
            "Replacing solvent molecule 7980 (atom 27257) with SOD\n",
            "Replacing solvent molecule 16570 (atom 53027) with SOD\n",
            "Replacing solvent molecule 19637 (atom 62228) with SOD\n",
            "Replacing solvent molecule 12802 (atom 41723) with SOD\n",
            "Replacing solvent molecule 19952 (atom 63173) with SOD\n",
            "Replacing solvent molecule 21852 (atom 68873) with SOD\n",
            "Replacing solvent molecule 21333 (atom 67316) with SOD\n",
            "Replacing solvent molecule 1524 (atom 7889) with SOD\n",
            "Replacing solvent molecule 17207 (atom 54938) with SOD\n",
            "Replacing solvent molecule 19010 (atom 60347) with SOD\n",
            "Replacing solvent molecule 19349 (atom 61364) with SOD\n",
            "Replacing solvent molecule 8471 (atom 28730) with SOD\n",
            "Replacing solvent molecule 11600 (atom 38117) with SOD\n",
            "Replacing solvent molecule 9102 (atom 30623) with SOD\n",
            "Replacing solvent molecule 17839 (atom 56834) with SOD\n",
            "Replacing solvent molecule 18295 (atom 58202) with SOD\n",
            "Replacing solvent molecule 12776 (atom 41645) with SOD\n",
            "Replacing solvent molecule 18465 (atom 58712) with SOD\n",
            "Replacing solvent molecule 18859 (atom 59894) with SOD\n",
            "Replacing solvent molecule 10013 (atom 33356) with SOD\n",
            "Replacing solvent molecule 2782 (atom 11663) with SOD\n",
            "Replacing solvent molecule 2280 (atom 10157) with SOD\n",
            "Replacing solvent molecule 9067 (atom 30518) with SOD\n",
            "Replacing solvent molecule 16549 (atom 52964) with SOD\n",
            "Replacing solvent molecule 21468 (atom 67721) with SOD\n",
            "Replacing solvent molecule 12992 (atom 42293) with SOD\n",
            "Replacing solvent molecule 14094 (atom 45599) with SOD\n",
            "Replacing solvent molecule 5017 (atom 18368) with SOD\n",
            "Replacing solvent molecule 11585 (atom 38072) with SOD\n",
            "Replacing solvent molecule 2089 (atom 9584) with SOD\n",
            "Replacing solvent molecule 20409 (atom 64544) with SOD\n",
            "Replacing solvent molecule 15459 (atom 49694) with SOD\n",
            "Replacing solvent molecule 4314 (atom 16259) with SOD\n",
            "Replacing solvent molecule 2685 (atom 11372) with SOD\n",
            "Replacing solvent molecule 18594 (atom 59099) with SOD\n",
            "Replacing solvent molecule 18463 (atom 58706) with SOD\n",
            "Replacing solvent molecule 20439 (atom 64634) with SOD\n",
            "Replacing solvent molecule 20402 (atom 64523) with SOD\n",
            "Replacing solvent molecule 8010 (atom 27347) with SOD\n",
            "Replacing solvent molecule 16155 (atom 51782) with SOD\n",
            "Replacing solvent molecule 5037 (atom 18428) with SOD\n",
            "Replacing solvent molecule 2373 (atom 10436) with SOD\n",
            "Replacing solvent molecule 2656 (atom 11285) with SOD\n",
            "Replacing solvent molecule 11870 (atom 38927) with SOD\n",
            "Replacing solvent molecule 20976 (atom 66245) with SOD\n",
            "Replacing solvent molecule 20809 (atom 65744) with SOD\n",
            "Replacing solvent molecule 6591 (atom 23090) with SOD\n",
            "Replacing solvent molecule 2838 (atom 11831) with CLA\n",
            "Replacing solvent molecule 14365 (atom 46412) with CLA\n",
            "Replacing solvent molecule 9339 (atom 31334) with CLA\n",
            "Replacing solvent molecule 4423 (atom 16586) with CLA\n",
            "Replacing solvent molecule 20579 (atom 65054) with CLA\n",
            "Replacing solvent molecule 706 (atom 5435) with CLA\n",
            "Replacing solvent molecule 4547 (atom 16958) with CLA\n",
            "Replacing solvent molecule 20258 (atom 64091) with CLA\n",
            "Replacing solvent molecule 20274 (atom 64139) with CLA\n",
            "Replacing solvent molecule 17835 (atom 56822) with CLA\n",
            "Replacing solvent molecule 19615 (atom 62162) with CLA\n",
            "Replacing solvent molecule 5159 (atom 18794) with CLA\n",
            "Replacing solvent molecule 15579 (atom 50054) with CLA\n",
            "Replacing solvent molecule 18918 (atom 60071) with CLA\n",
            "Replacing solvent molecule 1359 (atom 7394) with CLA\n",
            "Replacing solvent molecule 20232 (atom 64013) with CLA\n",
            "Replacing solvent molecule 14410 (atom 46547) with CLA\n",
            "Replacing solvent molecule 12570 (atom 41027) with CLA\n",
            "Replacing solvent molecule 12209 (atom 39944) with CLA\n",
            "Replacing solvent molecule 20653 (atom 65276) with CLA\n",
            "Replacing solvent molecule 14723 (atom 47486) with CLA\n",
            "Replacing solvent molecule 4691 (atom 17390) with CLA\n",
            "Replacing solvent molecule 2418 (atom 10571) with CLA\n",
            "Replacing solvent molecule 460 (atom 4697) with CLA\n",
            "Replacing solvent molecule 7249 (atom 25064) with CLA\n",
            "Replacing solvent molecule 18473 (atom 58736) with CLA\n",
            "Replacing solvent molecule 5749 (atom 20564) with CLA\n",
            "Replacing solvent molecule 13258 (atom 43091) with CLA\n",
            "Replacing solvent molecule 18069 (atom 57524) with CLA\n",
            "Replacing solvent molecule 1675 (atom 8342) with CLA\n",
            "Replacing solvent molecule 7734 (atom 26519) with CLA\n",
            "Replacing solvent molecule 4029 (atom 15404) with CLA\n",
            "Replacing solvent molecule 21022 (atom 66383) with CLA\n",
            "Replacing solvent molecule 4721 (atom 17480) with CLA\n",
            "Replacing solvent molecule 13738 (atom 44531) with CLA\n",
            "Replacing solvent molecule 3162 (atom 12803) with CLA\n",
            "Replacing solvent molecule 6754 (atom 23579) with CLA\n",
            "Replacing solvent molecule 11838 (atom 38831) with CLA\n",
            "Replacing solvent molecule 1282 (atom 7163) with CLA\n",
            "Replacing solvent molecule 5150 (atom 18767) with CLA\n",
            "Replacing solvent molecule 17154 (atom 54779) with CLA\n",
            "Replacing solvent molecule 624 (atom 5189) with CLA\n",
            "Replacing solvent molecule 1869 (atom 8924) with CLA\n",
            "Replacing solvent molecule 21701 (atom 68420) with CLA\n",
            "Replacing solvent molecule 21557 (atom 67988) with CLA\n",
            "Replacing solvent molecule 21440 (atom 67637) with CLA\n",
            "Replacing solvent molecule 17922 (atom 57083) with CLA\n",
            "Replacing solvent molecule 17600 (atom 56117) with CLA\n",
            "Replacing solvent molecule 10933 (atom 36116) with CLA\n",
            "Replacing solvent molecule 17454 (atom 55679) with CLA\n",
            "Replacing solvent molecule 6980 (atom 24257) with CLA\n",
            "Replacing solvent molecule 21253 (atom 67076) with CLA\n",
            "Replacing solvent molecule 1993 (atom 9296) with CLA\n",
            "Replacing solvent molecule 16976 (atom 54245) with CLA\n",
            "Replacing solvent molecule 4556 (atom 16985) with CLA\n",
            "Replacing solvent molecule 4435 (atom 16622) with CLA\n",
            "Replacing solvent molecule 8627 (atom 29198) with CLA\n",
            "Replacing solvent molecule 5273 (atom 19136) with CLA\n",
            "Replacing solvent molecule 14318 (atom 46271) with CLA\n",
            "Replacing solvent molecule 3252 (atom 13073) with CLA\n",
            "Replacing solvent molecule 14544 (atom 46949) with CLA\n",
            "Replacing solvent molecule 8178 (atom 27851) with CLA\n",
            "Replacing solvent molecule 5781 (atom 20660) with CLA\n",
            "Replacing solvent molecule 8448 (atom 28661) with CLA\n",
            "Replacing solvent molecule 14181 (atom 45860) with CLA\n",
            "Replacing solvent molecule 20514 (atom 64859) with CLA\n",
            "Replacing solvent molecule 9336 (atom 31325) with CLA\n",
            "Replacing solvent molecule 2535 (atom 10922) with CLA\n",
            "Replacing solvent molecule 4325 (atom 16292) with CLA\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"Those who cannot remember the past are condemned to compute it.\" (Steve Pinker)\n",
            "\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                   CYS47   CYS52  CYS177\n",
            "                   SG334   SG375  SG1328\n",
            "   CYS52   SG375   0.917\n",
            "  CYS177  SG1328   0.205   1.111\n",
            "  CYS188  SG1404   1.032   0.204   1.212\n",
            "Linking CYS-47 SG-334 and CYS-177 SG-1328...\n",
            "Linking CYS-52 SG-375 and CYS-188 SG-1404...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 8857 pairs\n",
            "Before cleaning: 8972 dihedrals\n",
            "\n",
            "GROMACS reminds you: \"The Path Of the Righteous Man is Beset On All Sides With the Iniquities Of the Selfish and the Tyranny Of Evil Men.\" (Pulp Fiction)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"In practice, throwing traditional norms and values overboard results not in perfect freedom and relationships based on reason, but in chaos and fear\" (Paul Verhaeghe)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 6x5x5 boxes\n",
            "Solvent box contains 79455 atoms in 26485 residues\n",
            "Removed 9426 solvent atoms due to solvent-solvent overlap\n",
            "Removed 3030 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 22333 residues\n",
            "Generated solvent containing 66999 atoms in 22333 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Output configuration contains 70310 atoms in 22554 residues\n",
            "Volume                 :     716.747 (nm^3)\n",
            "Density                :     991.119 (g/l)\n",
            "Number of solvent molecules:  22333   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.1#\n",
            "\n",
            "GROMACS reminds you: \"For a research worker the unforgotten moments of his life are those rare ones which come after years of plodding work, when the veil over natures secret seems suddenly to lift & when what was dark & chaotic appears in a clear & beautiful light & pattern.\" (Gerty Cori)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 31625]:\n",
            "  System has non-zero total charge: 5.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 31625]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 143928.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "GROMACS reminds you: \"How Do You Like Your Vacation So Far ?\" (Speed 2 - Cruise Control)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 70310 elements\n",
            "Group     1 (        Protein) has  3311 elements\n",
            "Group     2 (      Protein-H) has  1686 elements\n",
            "Group     3 (        C-alpha) has   221 elements\n",
            "Group     4 (       Backbone) has   663 elements\n",
            "Group     5 (      MainChain) has   883 elements\n",
            "Group     6 (   MainChain+Cb) has  1085 elements\n",
            "Group     7 (    MainChain+H) has  1089 elements\n",
            "Group     8 (      SideChain) has  2222 elements\n",
            "Group     9 (    SideChain-H) has   803 elements\n",
            "Group    10 (    Prot-Masses) has  3311 elements\n",
            "Group    11 (    non-Protein) has 66999 elements\n",
            "Group    12 (          Water) has 66999 elements\n",
            "Group    13 (            SOL) has 66999 elements\n",
            "Group    14 (      non-Water) has  3311 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 22333\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.2#\n",
            "Using random seed -344463316.\n",
            "Replacing solvent molecule 4066 (atom 15509) with SOD\n",
            "Replacing solvent molecule 18804 (atom 59723) with SOD\n",
            "Replacing solvent molecule 15423 (atom 49580) with SOD\n",
            "Replacing solvent molecule 9875 (atom 32936) with SOD\n",
            "Replacing solvent molecule 17576 (atom 56039) with SOD\n",
            "Replacing solvent molecule 15379 (atom 49448) with SOD\n",
            "Replacing solvent molecule 6923 (atom 24080) with SOD\n",
            "Replacing solvent molecule 18940 (atom 60131) with SOD\n",
            "Replacing solvent molecule 2886 (atom 11969) with SOD\n",
            "Replacing solvent molecule 15798 (atom 50705) with SOD\n",
            "Replacing solvent molecule 5829 (atom 20798) with SOD\n",
            "Replacing solvent molecule 12682 (atom 41357) with SOD\n",
            "Replacing solvent molecule 7911 (atom 27044) with SOD\n",
            "Replacing solvent molecule 21808 (atom 68735) with SOD\n",
            "Replacing solvent molecule 21839 (atom 68828) with SOD\n",
            "Replacing solvent molecule 10621 (atom 35174) with SOD\n",
            "Replacing solvent molecule 15974 (atom 51233) with SOD\n",
            "Replacing solvent molecule 8077 (atom 27542) with SOD\n",
            "Replacing solvent molecule 12019 (atom 39368) with SOD\n",
            "Replacing solvent molecule 20396 (atom 64499) with SOD\n",
            "Replacing solvent molecule 13454 (atom 43673) with SOD\n",
            "Replacing solvent molecule 13133 (atom 42710) with SOD\n",
            "Replacing solvent molecule 3482 (atom 13757) with SOD\n",
            "Replacing solvent molecule 14634 (atom 47213) with SOD\n",
            "Replacing solvent molecule 18612 (atom 59147) with SOD\n",
            "Replacing solvent molecule 2463 (atom 10700) with SOD\n",
            "Replacing solvent molecule 448 (atom 4655) with SOD\n",
            "Replacing solvent molecule 21172 (atom 66827) with SOD\n",
            "Replacing solvent molecule 724 (atom 5483) with SOD\n",
            "Replacing solvent molecule 239 (atom 4028) with SOD\n",
            "Replacing solvent molecule 11141 (atom 36734) with SOD\n",
            "Replacing solvent molecule 1298 (atom 7205) with SOD\n",
            "Replacing solvent molecule 14349 (atom 46358) with SOD\n",
            "Replacing solvent molecule 3854 (atom 14873) with SOD\n",
            "Replacing solvent molecule 2404 (atom 10523) with SOD\n",
            "Replacing solvent molecule 14584 (atom 47063) with SOD\n",
            "Replacing solvent molecule 19940 (atom 63131) with SOD\n",
            "Replacing solvent molecule 10401 (atom 34514) with SOD\n",
            "Replacing solvent molecule 9545 (atom 31946) with SOD\n",
            "Replacing solvent molecule 19248 (atom 61055) with SOD\n",
            "Replacing solvent molecule 537 (atom 4922) with SOD\n",
            "Replacing solvent molecule 4909 (atom 18038) with SOD\n",
            "Replacing solvent molecule 14070 (atom 45521) with SOD\n",
            "Replacing solvent molecule 16274 (atom 52133) with SOD\n",
            "Replacing solvent molecule 21569 (atom 68018) with SOD\n",
            "Replacing solvent molecule 4646 (atom 17249) with SOD\n",
            "Replacing solvent molecule 10577 (atom 35042) with SOD\n",
            "Replacing solvent molecule 12480 (atom 40751) with SOD\n",
            "Replacing solvent molecule 21523 (atom 67880) with SOD\n",
            "Replacing solvent molecule 21770 (atom 68621) with SOD\n",
            "Replacing solvent molecule 3382 (atom 13457) with SOD\n",
            "Replacing solvent molecule 14614 (atom 47153) with SOD\n",
            "Replacing solvent molecule 19110 (atom 60641) with SOD\n",
            "Replacing solvent molecule 15898 (atom 51005) with SOD\n",
            "Replacing solvent molecule 8552 (atom 28967) with SOD\n",
            "Replacing solvent molecule 2076 (atom 9539) with SOD\n",
            "Replacing solvent molecule 16633 (atom 53210) with SOD\n",
            "Replacing solvent molecule 14550 (atom 46961) with SOD\n",
            "Replacing solvent molecule 19556 (atom 61979) with SOD\n",
            "Replacing solvent molecule 12382 (atom 40457) with SOD\n",
            "Replacing solvent molecule 18386 (atom 58469) with SOD\n",
            "Replacing solvent molecule 13054 (atom 42473) with SOD\n",
            "Replacing solvent molecule 1061 (atom 6494) with SOD\n",
            "Replacing solvent molecule 22017 (atom 69362) with SOD\n",
            "Replacing solvent molecule 703 (atom 5420) with SOD\n",
            "Replacing solvent molecule 17689 (atom 56378) with CLA\n",
            "Replacing solvent molecule 5684 (atom 20363) with CLA\n",
            "Replacing solvent molecule 5942 (atom 21137) with CLA\n",
            "Replacing solvent molecule 10776 (atom 35639) with CLA\n",
            "Replacing solvent molecule 13732 (atom 44507) with CLA\n",
            "Replacing solvent molecule 21676 (atom 68339) with CLA\n",
            "Replacing solvent molecule 16123 (atom 51680) with CLA\n",
            "Replacing solvent molecule 17542 (atom 55937) with CLA\n",
            "Replacing solvent molecule 20204 (atom 63923) with CLA\n",
            "Replacing solvent molecule 14769 (atom 47618) with CLA\n",
            "Replacing solvent molecule 4397 (atom 16502) with CLA\n",
            "Replacing solvent molecule 1638 (atom 8225) with CLA\n",
            "Replacing solvent molecule 3231 (atom 13004) with CLA\n",
            "Replacing solvent molecule 21121 (atom 66674) with CLA\n",
            "Replacing solvent molecule 8912 (atom 30047) with CLA\n",
            "Replacing solvent molecule 20412 (atom 64547) with CLA\n",
            "Replacing solvent molecule 17904 (atom 57023) with CLA\n",
            "Replacing solvent molecule 13736 (atom 44519) with CLA\n",
            "Replacing solvent molecule 220 (atom 3971) with CLA\n",
            "Replacing solvent molecule 18867 (atom 59912) with CLA\n",
            "Replacing solvent molecule 20614 (atom 65153) with CLA\n",
            "Replacing solvent molecule 3948 (atom 15155) with CLA\n",
            "Replacing solvent molecule 6441 (atom 22634) with CLA\n",
            "Replacing solvent molecule 8309 (atom 28238) with CLA\n",
            "Replacing solvent molecule 20684 (atom 65363) with CLA\n",
            "Replacing solvent molecule 19516 (atom 61859) with CLA\n",
            "Replacing solvent molecule 13581 (atom 44054) with CLA\n",
            "Replacing solvent molecule 19754 (atom 62573) with CLA\n",
            "Replacing solvent molecule 16636 (atom 53219) with CLA\n",
            "Replacing solvent molecule 17250 (atom 55061) with CLA\n",
            "Replacing solvent molecule 8481 (atom 28754) with CLA\n",
            "Replacing solvent molecule 1427 (atom 7592) with CLA\n",
            "Replacing solvent molecule 762 (atom 5597) with CLA\n",
            "Replacing solvent molecule 1431 (atom 7604) with CLA\n",
            "Replacing solvent molecule 8752 (atom 29567) with CLA\n",
            "Replacing solvent molecule 13117 (atom 42662) with CLA\n",
            "Replacing solvent molecule 7349 (atom 25358) with CLA\n",
            "Replacing solvent molecule 9448 (atom 31655) with CLA\n",
            "Replacing solvent molecule 13447 (atom 43652) with CLA\n",
            "Replacing solvent molecule 11508 (atom 37835) with CLA\n",
            "Replacing solvent molecule 1874 (atom 8933) with CLA\n",
            "Replacing solvent molecule 20486 (atom 64769) with CLA\n",
            "Replacing solvent molecule 5633 (atom 20210) with CLA\n",
            "Replacing solvent molecule 11953 (atom 39170) with CLA\n",
            "Replacing solvent molecule 16972 (atom 54227) with CLA\n",
            "Replacing solvent molecule 2029 (atom 9398) with CLA\n",
            "Replacing solvent molecule 18924 (atom 60083) with CLA\n",
            "Replacing solvent molecule 19738 (atom 62525) with CLA\n",
            "Replacing solvent molecule 6210 (atom 21941) with CLA\n",
            "Replacing solvent molecule 19808 (atom 62735) with CLA\n",
            "Replacing solvent molecule 18624 (atom 59183) with CLA\n",
            "Replacing solvent molecule 7673 (atom 26330) with CLA\n",
            "Replacing solvent molecule 5232 (atom 19007) with CLA\n",
            "Replacing solvent molecule 21779 (atom 68648) with CLA\n",
            "Replacing solvent molecule 20798 (atom 65705) with CLA\n",
            "Replacing solvent molecule 12720 (atom 41471) with CLA\n",
            "Replacing solvent molecule 3842 (atom 14837) with CLA\n",
            "Replacing solvent molecule 14736 (atom 47519) with CLA\n",
            "Replacing solvent molecule 6309 (atom 22238) with CLA\n",
            "Replacing solvent molecule 5404 (atom 19523) with CLA\n",
            "Replacing solvent molecule 4123 (atom 15680) with CLA\n",
            "Replacing solvent molecule 16560 (atom 52991) with CLA\n",
            "Replacing solvent molecule 11646 (atom 38249) with CLA\n",
            "Replacing solvent molecule 18154 (atom 57773) with CLA\n",
            "Replacing solvent molecule 3478 (atom 13745) with CLA\n",
            "Replacing solvent molecule 16164 (atom 51803) with CLA\n",
            "Replacing solvent molecule 8545 (atom 28946) with CLA\n",
            "Replacing solvent molecule 17769 (atom 56618) with CLA\n",
            "Replacing solvent molecule 13807 (atom 44732) with CLA\n",
            "Replacing solvent molecule 8013 (atom 27350) with CLA\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"There's Still Time to Change the Road You're On\" (Led Zeppelin)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run pdb2gmx, editconf, solvate and genion\n",
        "%%bash\n",
        "git clone https://github.com/pbuslaev/alphagmx.git\n",
        "\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "#arrVar+=(\"./md_mutant_${i}\")\n",
        "arrVar+=(${ALPHAGMX}\"/md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "#echo \"${arrVar[@]}\"\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "\n",
        "echo ${value}\n",
        "sed -i 's/GLUP /GLUP/g' ${value}/input_pka.pdb\n",
        "sed -i 's/ASPP /ASPP/g' ${value}/input_pka.pdb\n",
        "\n",
        "cd /content/\n",
        "in_pdb=input_pka.pdb\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "cp -r amber14sb.ff $value/.\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber14sb\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber99sb-ildn\n",
        "fi\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "cp -r charmm36-jul2021.ff $value/.\n",
        "cp alphagmx/parameters/charmm/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
        "fi\n",
        "gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
        "gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
        "# Here we should apply some tricks Issue !4\n",
        "# https://github.com/pbuslaev/alphagmx/issues/4\n",
        "gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
        "#This is a trick to provide interactive options to gmx\n",
        "echo \"SOL\" > options\n",
        "echo \" \" >> options\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA < options\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 103,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "p-TrrUBhmPVe",
        "outputId": "b0c46850-b11c-403a-df80-38fb85121e8b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Setting the LD random seed to -33555497\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 21790      Water residues\n",
            "There are:   133      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.453 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x60, spacing 0.134 0.137 0.139\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Setting the LD random seed to -641\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 21790      Water residues\n",
            "There are:   133      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.435 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x60, spacing 0.134 0.137 0.139\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Setting the LD random seed to -260177925\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 21790      Water residues\n",
            "There are:   133      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.436 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x60, spacing 0.134 0.137 0.139\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Setting the LD random seed to -1644183173\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 22198      Water residues\n",
            "There are:   135      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.434 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x64, spacing 0.134 0.134 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Setting the LD random seed to -806066216\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 22198      Water residues\n",
            "There are:   135      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.427 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x64, spacing 0.134 0.134 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n",
            "Setting the LD random seed to -74448930\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 22198      Water residues\n",
            "There are:   135      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.447 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x64, spacing 0.134 0.134 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 5 Mb of data\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 141087.00\n",
            "\n",
            "GROMACS reminds you: \"The Path Of the Righteous Man is Beset On All Sides With the Iniquities Of the Selfish and the Tyranny Of Evil Men.\" (Pulp Fiction)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 1080 steps\n",
            "Potential Energy  = -1.1209441e+06\n",
            "Maximum force     =  9.4856525e+02 on atom 1024\n",
            "Norm of force     =  1.6307737e+01\n",
            "\n",
            "GROMACS reminds you: \"Nothing in biology makes sense except in the light of evolution.\" (Theodosius Dobzhansky)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 139458.00\n",
            "\n",
            "GROMACS reminds you: \"For those who want some proof that physicists are human, the proof is in the idiocy of all the different units which they use for measuring energy.\" (Richard Feynman)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.296\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:      107.674       53.837      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       16.050        1.495\n",
            "\n",
            "GROMACS reminds you: \"Humans are allergic to change. They love to say, 'We've always done it this way.' I try to fight that. That's why I have a clock on my wall that runs counter-clockwise.\" (Grace Hopper)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 139458.00\n",
            "\n",
            "GROMACS reminds you: \"In science, truth always wins.\" (Max Perutz)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.342\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       56.661       28.331      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       30.503        0.787\n",
            "\n",
            "GROMACS reminds you: \"Come and play on the hospital roof, I got something that's yours\" (Sherlock)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 143523.00\n",
            "\n",
            "GROMACS reminds you: \"Take Dehydrated Water On Your Desert Trips\" (Space Quest III)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 313 steps\n",
            "Potential Energy  = -1.1004244e+06\n",
            "Maximum force     =  8.6678479e+02 on atom 1009\n",
            "Norm of force     =  3.5247416e+01\n",
            "\n",
            "GROMACS reminds you: \"In this house, we OBEY the laws of thermodynamics!\" (Homer Simpson)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 141898.00\n",
            "\n",
            "GROMACS reminds you: \"To survive science you have to become science.\" (Gerrit Groenhof)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.297\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:      131.257       65.629      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       13.166        1.823\n",
            "\n",
            "GROMACS reminds you: \"The Universe is Somewhere In Here\" (J.G.E.M. Fraaije)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 141898.00\n",
            "\n",
            "GROMACS reminds you: \"I have noticed a large, negative correlation between having a well-defined mission workload and concern for the Top500. It's almost like LINPACK is what you focus on when you don't know what to focus on.\" (Jeff Hammond)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.342\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       85.623       42.812      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       20.185        1.189\n",
            "\n",
            "GROMACS reminds you: \"And I Will Strike Down Upon Thee With Great Vengeance and With Furious Anger Those Who Attempt to Poison and Destroy My Brothers.\" (Pulp Fiction)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run equilibration\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "#arrVar+=(\"./md_mutant_${i}\")\n",
        "arrVar+=(${ALPHAGMX}\"/md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro \n",
        "gmx mdrun -deffnm 5_min\n",
        "gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro \n",
        "gmx mdrun -deffnm 6_nvt\n",
        "gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro \n",
        "gmx mdrun -deffnm 7_npt\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 105,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GQqWYtC4_xvq",
        "outputId": "fef75494-87e1-4e32-886c-0466a418c45f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Setting the LD random seed to -1449533441\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 21790      Water residues\n",
            "There are:   133      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.414 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x60, spacing 0.133 0.135 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 40 Mb of data\n",
            "Setting the LD random seed to -273189122\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   221    Protein residues\n",
            "There are: 22198      Water residues\n",
            "There are:   135      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.434 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 72x64x64, spacing 0.133 0.133 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 41 Mb of data\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 139458.00\n",
            "\n",
            "Back Off! I just backed up run.tpr to ./#run.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"And It Goes a Little Something Like This\" (Tag Team)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 10\n",
            "\n",
            "\n",
            "Back Off! I just backed up run.log to ./#run.log.1#\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 10 steps, 0.02 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Back Off! I just backed up run.xtc to ./#run.xtc.1#\n",
            "\n",
            "Back Off! I just backed up run.edr to ./#run.edr.1#\n",
            "starting mdrun 'Protein in water'\n",
            "10 steps,      0.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "Back Off! I just backed up run.gro to ./#run.gro.1#\n",
            "\n",
            "NOTE: 19 % of the run time was spent in pair search,\n",
            "      you might want to increase nstlist (this has no effect on accuracy)\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:        0.911        0.456      199.9\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:        4.169        5.757\n",
            "\n",
            "GROMACS reminds you: \"Hey Man You Know, I'm Really OK\" (Offspring)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 141898.00\n",
            "\n",
            "Back Off! I just backed up run.tpr to ./#run.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"Would You Like to Be the Monster Tonight ?\" (Captain Beefheart)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 10\n",
            "\n",
            "\n",
            "Back Off! I just backed up run.log to ./#run.log.1#\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 10 steps, 0.02 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Back Off! I just backed up run.xtc to ./#run.xtc.1#\n",
            "\n",
            "Back Off! I just backed up run.edr to ./#run.edr.1#\n",
            "starting mdrun 'Protein in water'\n",
            "10 steps,      0.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "NOTE: 20 % of the run time was spent in pair search,\n",
            "      you might want to increase nstlist (this has no effect on accuracy)\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:        0.627        0.313      199.9\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:        6.064        3.958\n",
            "\n",
            "GROMACS reminds you: \"The Path Of the Righteous Man is Beset On All Sides With the Iniquities Of the Selfish and the Tyranny Of Evil Men.\" (Pulp Fiction)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "#arrVar+=(\"./md_mutant_${i}\")\n",
        "arrVar+=(${ALPHAGMX}\"/md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
        "gmx mdrun -deffnm run -nsteps 1000\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 106,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bzmIt5F6FLri",
        "outputId": "00c9a856-a8ab-4a17-dd35-55a7505b37f6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n",
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_base\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 68820 elements\n",
            "Group     1 (        Protein) has  3317 elements\n",
            "Group     2 (      Protein-H) has  1688 elements\n",
            "Group     3 (        C-alpha) has   221 elements\n",
            "Group     4 (       Backbone) has   663 elements\n",
            "Group     5 (      MainChain) has   883 elements\n",
            "Group     6 (   MainChain+Cb) has  1085 elements\n",
            "Group     7 (    MainChain+H) has  1089 elements\n",
            "Group     8 (      SideChain) has  2228 elements\n",
            "Group     9 (    SideChain-H) has   805 elements\n",
            "Group    10 (    Prot-Masses) has  3317 elements\n",
            "Group    11 (    non-Protein) has 65503 elements\n",
            "Group    12 (          Water) has 65370 elements\n",
            "Group    13 (            SOL) has 65370 elements\n",
            "Group    14 (      non-Water) has  3450 elements\n",
            "Group    15 (          Other) has   133 elements\n",
            "Group    16 (            SOD) has    64 elements\n",
            "Group    17 (            CLA) has    69 elements\n",
            "Select a group: \rReading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\rLast frame          0 time    0.000   \n",
            " ->  frame      0 time    0.000      \r\n",
            "Last written: frame      0 time    0.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"It all works because Avogadro's number is closer to infinity than to 10.\" (Ralph Baierlein)\n",
            "\n",
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/alphagmx_project/md_mutant_0\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 70040 elements\n",
            "Group     1 (        Protein) has  3311 elements\n",
            "Group     2 (      Protein-H) has  1686 elements\n",
            "Group     3 (        C-alpha) has   221 elements\n",
            "Group     4 (       Backbone) has   663 elements\n",
            "Group     5 (      MainChain) has   883 elements\n",
            "Group     6 (   MainChain+Cb) has  1085 elements\n",
            "Group     7 (    MainChain+H) has  1089 elements\n",
            "Group     8 (      SideChain) has  2222 elements\n",
            "Group     9 (    SideChain-H) has   803 elements\n",
            "Group    10 (    Prot-Masses) has  3311 elements\n",
            "Group    11 (    non-Protein) has 66729 elements\n",
            "Group    12 (          Water) has 66594 elements\n",
            "Group    13 (            SOL) has 66594 elements\n",
            "Group    14 (      non-Water) has  3446 elements\n",
            "Group    15 (          Other) has   135 elements\n",
            "Group    16 (            SOD) has    65 elements\n",
            "Group    17 (            CLA) has    70 elements\n",
            "Select a group: \rReading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\rLast frame          0 time    0.000   \n",
            " ->  frame      0 time    0.000      \r\n",
            "Last written: frame      0 time    0.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"It all works because Avogadro's number is closer to infinity than to 10.\" (Ralph Baierlein)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title make trajectories whole\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "#arrVar+=(\"./md_mutant_${i}\")\n",
        "arrVar+=(${ALPHAGMX}\"/md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "echo 0 | gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
        "done"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9WK7EtsiG_Wp"
      },
      "source": [
        "# Analysis"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 114,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 830
        },
        "id": "DBUzmhW5HCpl",
        "outputId": "ac47d3bf-b3a9-4e5f-cb0f-53dd9777b3e2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/analysis/align.py:952: DeprecationWarning:\n",
            "\n",
            "The `universe` attribute was deprecated in MDAnalysis 2.0.0 and will be removed in MDAnalysis 3.0.0. Please use `results.universe` instead.\n",
            "\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/alphagmx_project/md_base    Frame  Time (ns)  RMSD, /content/alphagmx_project/md_base\n",
            "0    0.0        0.0                             6.289054e-07\n",
            "/content/alphagmx_project/md_mutant_0/    Frame  Time (ns)  RMSD, /content/alphagmx_project/md_base  \\\n",
            "0    0.0        0.0                             6.289054e-07   \n",
            "\n",
            "   RMSD, /content/alphagmx_project/md_mutant_0/  \n",
            "0                                  3.630987e-07  \n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/analysis/align.py:952: DeprecationWarning:\n",
            "\n",
            "The `universe` attribute was deprecated in MDAnalysis 2.0.0 and will be removed in MDAnalysis 3.0.0. Please use `results.universe` instead.\n",
            "\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"82ea4cbc-74a5-448c-94d5-c6701dee7d34\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"82ea4cbc-74a5-448c-94d5-c6701dee7d34\")) {                    Plotly.newPlot(                        \"82ea4cbc-74a5-448c-94d5-c6701dee7d34\",                        [{\"hovertemplate\":\"variable=RMSD, /content/alphagmx_project/md_base<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, /content/alphagmx_project/md_base\",\"line\":{\"color\":\"#636efa\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, /content/alphagmx_project/md_base\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0],\"xaxis\":\"x\",\"y\":[6.289054338389221e-07],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"variable=RMSD, /content/alphagmx_project/md_mutant_0/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, /content/alphagmx_project/md_mutant_0/\",\"line\":{\"color\":\"#EF553B\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, /content/alphagmx_project/md_mutant_0/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0],\"xaxis\":\"x\",\"y\":[3.630987215217201e-07],\"yaxis\":\"y\",\"type\":\"scatter\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Time (ns)\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"RMSD (A)\"}},\"legend\":{\"title\":{\"text\":\"variable\"},\"tracegroupgap\":0},\"margin\":{\"t\":60}},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('82ea4cbc-74a5-448c-94d5-c6701dee7d34');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title RMSD\n",
        "paths = [os.environ[\"MD_PATH\"]]\n",
        "for i in range(mutant_n):\n",
        "    paths.append(os.path.join(project_dir, f\"md_mutant_{i}/\"))\n",
        "\n",
        "first = True\n",
        "rmsdList = []\n",
        "for current_path in paths:\n",
        "    mobile = mda.Universe(os.path.join(current_path, \"run.gro\"), os.path.join(current_path, \"run_nj.xtc\"))\n",
        "    averageBB = align.AverageStructure(mobile, \n",
        "                                       mobile, \n",
        "                                       select='protein and backbone',\n",
        "                                       ref_frame=0).run()\n",
        "    refBB = averageBB.universe\n",
        "    R_rmsd = rms.RMSD(mobile,  # universe to align\n",
        "                     refBB,  # reference universe or atomgroup\n",
        "                     select='protein and backbone')  # group to superimpose and calculate RMSD\n",
        "        \n",
        "    R_rmsd.run()\n",
        "    if first:\n",
        "        df_rmsd = pd.DataFrame(R_rmsd.results.rmsd,\n",
        "                  columns=['Frame', 'Time (ns)',f'RMSD, {current_path}'])\n",
        "        df_rmsd['Time (ns)'] /= 1000\n",
        "        rmsdList.append(f'RMSD, {current_path}')\n",
        "        first = False\n",
        "    else:\n",
        "        colname = f'RMSD, {current_path}'\n",
        "        df_rmsd[colname] = R_rmsd.results.rmsd[:,2]\n",
        "        rmsdList.append(colname)\n",
        "\n",
        "df_rmsd_melt = df_rmsd.melt(id_vars='Time (ns)', value_vars=rmsdList)\n",
        "df_rmsd_melt.rename(columns = {'value': 'RMSD (A)'}, inplace = True)\n",
        "fig = px.line(df_rmsd_melt, x=\"Time (ns)\", y=\"RMSD (A)\",\n",
        "              color='variable',\n",
        "              line_shape=\"linear\", render_mode=\"svg\", \n",
        "              labels={ \"Name CA\": \"RMSD(Å)\" })\n",
        "fig.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "QgBYxxDKo_9x"
      },
      "outputs": [],
      "source": []
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "provenance": []
    },
    "gpuClass": "standard",
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.9.14"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}