{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "collapsed_sections": [
        "DTdIs9xeSpaa"
      ]
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU",
    "gpuClass": "standard"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# Install dependencies"
      ],
      "metadata": {
        "id": "0n2eT9Q4sf_F"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "NOTE: PIP freeze has to be added"
      ],
      "metadata": {
        "id": "6GNaY_cA0Cmw"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install conda\n",
        "!pip install -q condacolab\n",
        "import condacolab\n",
        "condacolab.install()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WkVyCCfasla_",
        "outputId": "f95c4aef-91dc-4419-bacf-b0017e299f2c"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m✨🍰✨ Everything looks OK!\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Update gcc\n",
        "!sudo echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal main universe' >> /etc/apt/sources.list\n",
        "!sudo apt update\n",
        "!sudo apt install -y gcc-9 g++-9\n",
        "!ln -snf /usr/bin/gcc-9 /usr/bin/gcc\n",
        "!ln -snf /usr/bin/g++-9 /usr/bin/g++"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5iYQ0-TnspXN",
        "outputId": "826acd85-2a61-4722-ddf2-5dcd640ea13b"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Get:1 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/ InRelease [3,626 B]\n",
            "Get:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/ Packages [98.9 kB]\n",
            "Ign:3 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease\n",
            "Get:4 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease [1,581 B]\n",
            "Hit:5 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release\n",
            "Get:6 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Packages [992 kB]\n",
            "Get:8 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic InRelease [15.9 kB]\n",
            "Get:9 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]\n",
            "Hit:10 http://archive.ubuntu.com/ubuntu bionic InRelease\n",
            "Get:11 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]\n",
            "Hit:12 http://ppa.launchpad.net/cran/libgit2/ubuntu bionic InRelease\n",
            "Get:13 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [1,554 kB]\n",
            "Get:14 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [83.3 kB]\n",
            "Get:15 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic InRelease [15.9 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]\n",
            "Hit:17 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease\n",
            "Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [2,332 kB]\n",
            "Get:19 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [1,226 kB]\n",
            "Get:20 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic/main Sources [2,216 kB]\n",
            "Get:21 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [3,040 kB]\n",
            "Get:22 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [3,472 kB]\n",
            "Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [1,267 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1,275 kB]\n",
            "Get:25 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]\n",
            "Get:26 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic/main amd64 Packages [1,133 kB]\n",
            "Fetched 30.5 MB in 7s (4,214 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "821 packages can be upgraded. Run 'apt list --upgradable' to see them.\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following package was automatically installed and is no longer required:\n",
            "  libnvidia-common-460\n",
            "Use 'sudo apt autoremove' to remove it.\n",
            "The following additional packages will be installed:\n",
            "  binutils binutils-common binutils-x86-64-linux-gnu cpp-9 gcc-10-base\n",
            "  gcc-9-base libasan5 libatomic1 libbinutils libcc1-0 libctf-nobfd0 libctf0\n",
            "  libgcc-9-dev libgcc-s1 libgomp1 libisl22 libitm1 liblsan0 libquadmath0\n",
            "  libstdc++-9-dev libstdc++6 libtsan0 libubsan1\n",
            "Suggested packages:\n",
            "  binutils-doc gcc-9-locales g++-9-multilib gcc-9-doc gcc-9-multilib\n",
            "  libstdc++-9-doc\n",
            "The following NEW packages will be installed:\n",
            "  cpp-9 g++-9 gcc-10-base gcc-9 gcc-9-base libasan5 libctf-nobfd0 libctf0\n",
            "  libgcc-9-dev libgcc-s1 libisl22 libstdc++-9-dev libubsan1\n",
            "The following packages will be upgraded:\n",
            "  binutils binutils-common binutils-x86-64-linux-gnu libatomic1 libbinutils\n",
            "  libcc1-0 libgomp1 libitm1 liblsan0 libquadmath0 libstdc++6 libtsan0\n",
            "12 upgraded, 13 newly installed, 0 to remove and 809 not upgraded.\n",
            "Need to get 33.1 MB of archives.\n",
            "After this operation, 122 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu focal/main amd64 gcc-10-base amd64 10-20200411-0ubuntu1 [19.0 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu focal/main amd64 libgcc-s1 amd64 10-20200411-0ubuntu1 [41.6 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu focal/main amd64 libstdc++6 amd64 10-20200411-0ubuntu1 [503 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu focal/main amd64 binutils-x86-64-linux-gnu amd64 2.34-6ubuntu1 [1,614 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu focal/main amd64 libbinutils amd64 2.34-6ubuntu1 [474 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu focal/main amd64 binutils amd64 2.34-6ubuntu1 [3,376 B]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu focal/main amd64 binutils-common amd64 2.34-6ubuntu1 [207 kB]\n",
            "Get:8 http://archive.ubuntu.com/ubuntu focal/main amd64 libctf-nobfd0 amd64 2.34-6ubuntu1 [47.0 kB]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 libctf0 amd64 2.34-6ubuntu1 [46.6 kB]\n",
            "Get:10 http://archive.ubuntu.com/ubuntu focal/main amd64 gcc-9-base amd64 9.3.0-10ubuntu2 [19.3 kB]\n",
            "Get:11 http://archive.ubuntu.com/ubuntu focal/main amd64 libisl22 amd64 0.22.1-1 [592 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu focal/main amd64 cpp-9 amd64 9.3.0-10ubuntu2 [7,491 kB]\n",
            "Get:13 http://archive.ubuntu.com/ubuntu focal/main amd64 libcc1-0 amd64 10-20200411-0ubuntu1 [41.1 kB]\n",
            "Get:14 http://archive.ubuntu.com/ubuntu focal/main amd64 libgomp1 amd64 10-20200411-0ubuntu1 [101 kB]\n",
            "Get:15 http://archive.ubuntu.com/ubuntu focal/main amd64 libitm1 amd64 10-20200411-0ubuntu1 [26.3 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu focal/main amd64 libatomic1 amd64 10-20200411-0ubuntu1 [9,284 B]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu focal/main amd64 libasan5 amd64 9.3.0-10ubuntu2 [395 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu focal/main amd64 liblsan0 amd64 10-20200411-0ubuntu1 [144 kB]\n",
            "Get:19 http://archive.ubuntu.com/ubuntu focal/main amd64 libtsan0 amd64 10-20200411-0ubuntu1 [319 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu focal/main amd64 libubsan1 amd64 10-20200411-0ubuntu1 [136 kB]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu focal/main amd64 libquadmath0 amd64 10-20200411-0ubuntu1 [146 kB]\n",
            "Get:22 http://archive.ubuntu.com/ubuntu focal/main amd64 libgcc-9-dev amd64 9.3.0-10ubuntu2 [2,359 kB]\n",
            "Get:23 http://archive.ubuntu.com/ubuntu focal/main amd64 gcc-9 amd64 9.3.0-10ubuntu2 [8,234 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu focal/main amd64 libstdc++-9-dev amd64 9.3.0-10ubuntu2 [1,711 kB]\n",
            "Get:25 http://archive.ubuntu.com/ubuntu focal/main amd64 g++-9 amd64 9.3.0-10ubuntu2 [8,404 kB]\n",
            "Fetched 33.1 MB in 6s (5,954 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 25.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package gcc-10-base:amd64.\n",
            "(Reading database ... 123942 files and directories currently installed.)\n",
            "Preparing to unpack .../gcc-10-base_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking gcc-10-base:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up gcc-10-base:amd64 (10-20200411-0ubuntu1) ...\n",
            "Selecting previously unselected package libgcc-s1:amd64.\n",
            "(Reading database ... 123947 files and directories currently installed.)\n",
            "Preparing to unpack .../libgcc-s1_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libgcc-s1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Replacing files in old package libgcc1:amd64 (1:8.4.0-1ubuntu1~18.04) ...\n",
            "Setting up libgcc-s1:amd64 (10-20200411-0ubuntu1) ...\n",
            "(Reading database ... 123949 files and directories currently installed.)\n",
            "Preparing to unpack .../libstdc++6_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libstdc++6:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Setting up libstdc++6:amd64 (10-20200411-0ubuntu1) ...\n",
            "(Reading database ... 123949 files and directories currently installed.)\n",
            "Preparing to unpack .../00-binutils-x86-64-linux-gnu_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking binutils-x86-64-linux-gnu (2.34-6ubuntu1) over (2.30-21ubuntu1~18.04.7) ...\n",
            "Preparing to unpack .../01-libbinutils_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking libbinutils:amd64 (2.34-6ubuntu1) over (2.30-21ubuntu1~18.04.7) ...\n",
            "Preparing to unpack .../02-binutils_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking binutils (2.34-6ubuntu1) over (2.30-21ubuntu1~18.04.7) ...\n",
            "Preparing to unpack .../03-binutils-common_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking binutils-common:amd64 (2.34-6ubuntu1) over (2.30-21ubuntu1~18.04.7) ...\n",
            "Selecting previously unselected package libctf-nobfd0:amd64.\n",
            "Preparing to unpack .../04-libctf-nobfd0_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking libctf-nobfd0:amd64 (2.34-6ubuntu1) ...\n",
            "Selecting previously unselected package libctf0:amd64.\n",
            "Preparing to unpack .../05-libctf0_2.34-6ubuntu1_amd64.deb ...\n",
            "Unpacking libctf0:amd64 (2.34-6ubuntu1) ...\n",
            "Selecting previously unselected package gcc-9-base:amd64.\n",
            "Preparing to unpack .../06-gcc-9-base_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking gcc-9-base:amd64 (9.3.0-10ubuntu2) ...\n",
            "Selecting previously unselected package libisl22:amd64.\n",
            "Preparing to unpack .../07-libisl22_0.22.1-1_amd64.deb ...\n",
            "Unpacking libisl22:amd64 (0.22.1-1) ...\n",
            "Selecting previously unselected package cpp-9.\n",
            "Preparing to unpack .../08-cpp-9_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking cpp-9 (9.3.0-10ubuntu2) ...\n",
            "Preparing to unpack .../09-libcc1-0_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libcc1-0:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Preparing to unpack .../10-libgomp1_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libgomp1:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Preparing to unpack .../11-libitm1_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libitm1:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Preparing to unpack .../12-libatomic1_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libatomic1:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Selecting previously unselected package libasan5:amd64.\n",
            "Preparing to unpack .../13-libasan5_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking libasan5:amd64 (9.3.0-10ubuntu2) ...\n",
            "Preparing to unpack .../14-liblsan0_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking liblsan0:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Preparing to unpack .../15-libtsan0_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libtsan0:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Selecting previously unselected package libubsan1:amd64.\n",
            "Preparing to unpack .../16-libubsan1_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libubsan1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Preparing to unpack .../17-libquadmath0_10-20200411-0ubuntu1_amd64.deb ...\n",
            "Unpacking libquadmath0:amd64 (10-20200411-0ubuntu1) over (8.4.0-1ubuntu1~18.04) ...\n",
            "Selecting previously unselected package libgcc-9-dev:amd64.\n",
            "Preparing to unpack .../18-libgcc-9-dev_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking libgcc-9-dev:amd64 (9.3.0-10ubuntu2) ...\n",
            "Selecting previously unselected package gcc-9.\n",
            "Preparing to unpack .../19-gcc-9_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking gcc-9 (9.3.0-10ubuntu2) ...\n",
            "Selecting previously unselected package libstdc++-9-dev:amd64.\n",
            "Preparing to unpack .../20-libstdc++-9-dev_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking libstdc++-9-dev:amd64 (9.3.0-10ubuntu2) ...\n",
            "Selecting previously unselected package g++-9.\n",
            "Preparing to unpack .../21-g++-9_9.3.0-10ubuntu2_amd64.deb ...\n",
            "Unpacking g++-9 (9.3.0-10ubuntu2) ...\n",
            "Setting up libquadmath0:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up libgomp1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up gcc-9-base:amd64 (9.3.0-10ubuntu2) ...\n",
            "Setting up libatomic1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up libcc1-0:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up libctf-nobfd0:amd64 (2.34-6ubuntu1) ...\n",
            "Setting up libtsan0:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up liblsan0:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up binutils-common:amd64 (2.34-6ubuntu1) ...\n",
            "Setting up libisl22:amd64 (0.22.1-1) ...\n",
            "Setting up libasan5:amd64 (9.3.0-10ubuntu2) ...\n",
            "Setting up libitm1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up libubsan1:amd64 (10-20200411-0ubuntu1) ...\n",
            "Setting up libgcc-9-dev:amd64 (9.3.0-10ubuntu2) ...\n",
            "Setting up libbinutils:amd64 (2.34-6ubuntu1) ...\n",
            "Setting up libstdc++-9-dev:amd64 (9.3.0-10ubuntu2) ...\n",
            "Setting up cpp-9 (9.3.0-10ubuntu2) ...\n",
            "Setting up libctf0:amd64 (2.34-6ubuntu1) ...\n",
            "Setting up binutils-x86-64-linux-gnu (2.34-6ubuntu1) ...\n",
            "Setting up binutils (2.34-6ubuntu1) ...\n",
            "Setting up gcc-9 (9.3.0-10ubuntu2) ...\n",
            "Setting up g++-9 (9.3.0-10ubuntu2) ...\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n",
            "Processing triggers for libc-bin (2.27-3ubuntu1.6) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Installing biopython using pip\n",
        "!pip install biopython\n",
        "from Bio.PDB import *\n",
        "import Bio.SeqUtils\n",
        "import Bio.SeqIO\n",
        "from Bio.SeqRecord import SeqRecord\n",
        "from Bio.Seq import Seq"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "TSFKHvOjtZaH",
        "outputId": "08b0f181-2d72-47cd-f1f7-a671327c4e93"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: biopython in /usr/local/lib/python3.7/site-packages (1.79)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/site-packages (from biopython) (1.21.6)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Installing PDBfixer\n",
        "#@markdown The repair of PDB is now done with PDBfixer\n",
        "\n",
        "#@markdown We can also add other options, like modeler, later on\n",
        "!conda install -q -y -c conda-forge pdbfixer openmm"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pm9AAM1YtsT3",
        "outputId": "c47c8244-3f24-45f8-9cd7-db52731adcac"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): ...working... done\n",
            "Solving environment: ...working... done\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - openmm\n",
            "    - pdbfixer\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    conda-22.9.0               |   py37h89c1867_1         960 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:         960 KB\n",
            "\n",
            "The following packages will be UPDATED:\n",
            "\n",
            "  conda                               4.14.0-py37h89c1867_0 --> 22.9.0-py37h89c1867_1\n",
            "\n",
            "\n",
            "Preparing transaction: ...working... done\n",
            "Verifying transaction: ...working... done\n",
            "Executing transaction: ...working... done\n",
            "Retrieving notices: ...working... done\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install py3Dmol\n",
        "!pip install py3Dmol\n",
        "import py3Dmol"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ubtGWa9ctyN3",
        "outputId": "2712fc5e-fc7a-4994-ff1b-e4a7b81e1cf3"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting py3Dmol\n",
            "  Downloading py3Dmol-1.8.1-py2.py3-none-any.whl (6.5 kB)\n",
            "Installing collected packages: py3Dmol\n",
            "Successfully installed py3Dmol-1.8.1\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title python libs\n",
        "!pip install wget\n",
        "import os\n",
        "import shutil\n",
        "import pdbfixer\n",
        "from openmm.app import PDBFile\n",
        "import wget\n",
        "import tarfile\n",
        "import pandas as pd\n",
        "import plotly.graph_objects as go\n",
        "import plotly.express as px\n",
        "import numpy as np"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dahUiTcwstzr",
        "outputId": "7042cb76-3fe4-4f54-b3fb-2f4e76a75892"
      },
      "execution_count": 103,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: wget in /usr/local/lib/python3.7/site-packages (3.2)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install Torch and TorchAni\n",
        "!conda install pytorch torchvision -c pytorch\n",
        "!conda install -c conda-forge torchani=2.2.0 cudatoolkit\n",
        "!conda install scikit-learn"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WFX55eW2ulgc",
        "outputId": "fe3711db-0967-413a-dd71-60fa78c5249c"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - pytorch\n",
            "    - torchvision\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    _openmp_mutex-4.5          |       2_kmp_llvm           6 KB  conda-forge\n",
            "    cudnn-8.4.1.50             |       hed8a83a_0       618.4 MB  conda-forge\n",
            "    freetype-2.12.1            |       hca18f0e_0         884 KB  conda-forge\n",
            "    jpeg-9e                    |       h166bdaf_2         269 KB  conda-forge\n",
            "    lcms2-2.14                 |       h6ed2654_0         256 KB  conda-forge\n",
            "    lerc-4.0.0                 |       h27087fc_0         275 KB  conda-forge\n",
            "    libdeflate-1.14            |       h166bdaf_0          81 KB  conda-forge\n",
            "    libpng-1.6.38              |       h753d276_0         371 KB  conda-forge\n",
            "    libprotobuf-3.20.1         |       h6239696_4         2.6 MB  conda-forge\n",
            "    libtiff-4.4.0              |       h55922b4_4         651 KB  conda-forge\n",
            "    libwebp-base-1.2.4         |       h166bdaf_0         404 KB  conda-forge\n",
            "    libxcb-1.13                |    h7f98852_1004         391 KB  conda-forge\n",
            "    llvm-openmp-14.0.4         |       he0ac6c6_0         5.8 MB  conda-forge\n",
            "    magma-2.5.4                |       hc72dce7_4       106.8 MB  conda-forge\n",
            "    mkl-2022.1.0               |     h84fe81f_915       199.6 MB  conda-forge\n",
            "    nccl-2.14.3.1              |       h0800d71_0       145.2 MB  conda-forge\n",
            "    ninja-1.11.0               |       h924138e_0         2.8 MB  conda-forge\n",
            "    openjpeg-2.5.0             |       h7d73246_1         533 KB  conda-forge\n",
            "    pillow-9.2.0               |   py37h850a105_2        45.0 MB  conda-forge\n",
            "    pthread-stubs-0.4          |    h36c2ea0_1001           5 KB  conda-forge\n",
            "    pytorch-1.12.1             |cuda112py37hfcfbd4c_200       489.7 MB  conda-forge\n",
            "    sleef-3.5.1                |       h9b69904_2         1.5 MB  conda-forge\n",
            "    tbb-2021.6.0               |       h924138e_1         2.0 MB  conda-forge\n",
            "    torchvision-0.2.2          |             py_3          44 KB  pytorch\n",
            "    typing_extensions-4.4.0    |     pyha770c72_0          29 KB  conda-forge\n",
            "    xorg-libxau-1.0.9          |       h7f98852_0          13 KB  conda-forge\n",
            "    xorg-libxdmcp-1.1.3        |       h7f98852_0          19 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        1.59 GB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  cudnn              conda-forge/linux-64::cudnn-8.4.1.50-hed8a83a_0 None\n",
            "  freetype           conda-forge/linux-64::freetype-2.12.1-hca18f0e_0 None\n",
            "  jpeg               conda-forge/linux-64::jpeg-9e-h166bdaf_2 None\n",
            "  lcms2              conda-forge/linux-64::lcms2-2.14-h6ed2654_0 None\n",
            "  lerc               conda-forge/linux-64::lerc-4.0.0-h27087fc_0 None\n",
            "  libdeflate         conda-forge/linux-64::libdeflate-1.14-h166bdaf_0 None\n",
            "  libpng             conda-forge/linux-64::libpng-1.6.38-h753d276_0 None\n",
            "  libprotobuf        conda-forge/linux-64::libprotobuf-3.20.1-h6239696_4 None\n",
            "  libtiff            conda-forge/linux-64::libtiff-4.4.0-h55922b4_4 None\n",
            "  libwebp-base       conda-forge/linux-64::libwebp-base-1.2.4-h166bdaf_0 None\n",
            "  libxcb             conda-forge/linux-64::libxcb-1.13-h7f98852_1004 None\n",
            "  llvm-openmp        conda-forge/linux-64::llvm-openmp-14.0.4-he0ac6c6_0 None\n",
            "  magma              conda-forge/linux-64::magma-2.5.4-hc72dce7_4 None\n",
            "  mkl                conda-forge/linux-64::mkl-2022.1.0-h84fe81f_915 None\n",
            "  nccl               conda-forge/linux-64::nccl-2.14.3.1-h0800d71_0 None\n",
            "  ninja              conda-forge/linux-64::ninja-1.11.0-h924138e_0 None\n",
            "  openjpeg           conda-forge/linux-64::openjpeg-2.5.0-h7d73246_1 None\n",
            "  pillow             conda-forge/linux-64::pillow-9.2.0-py37h850a105_2 None\n",
            "  pthread-stubs      conda-forge/linux-64::pthread-stubs-0.4-h36c2ea0_1001 None\n",
            "  pytorch            conda-forge/linux-64::pytorch-1.12.1-cuda112py37hfcfbd4c_200 None\n",
            "  sleef              conda-forge/linux-64::sleef-3.5.1-h9b69904_2 None\n",
            "  tbb                conda-forge/linux-64::tbb-2021.6.0-h924138e_1 None\n",
            "  torchvision        pytorch/noarch::torchvision-0.2.2-py_3 None\n",
            "  typing_extensions  conda-forge/noarch::typing_extensions-4.4.0-pyha770c72_0 None\n",
            "  xorg-libxau        conda-forge/linux-64::xorg-libxau-1.0.9-h7f98852_0 None\n",
            "  xorg-libxdmcp      conda-forge/linux-64::xorg-libxdmcp-1.1.3-h7f98852_0 None\n",
            "\n",
            "The following packages will be DOWNGRADED:\n",
            "\n",
            "  _openmp_mutex                                   4.5-2_gnu --> 4.5-2_kmp_llvm None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "magma-2.5.4          | 106.8 MB  | : 100% 1.0/1 [00:12<00:00, 12.73s/it]               \n",
            "ninja-1.11.0         | 2.8 MB    | : 100% 1.0/1 [00:00<00:00,  2.28it/s]\n",
            "tbb-2021.6.0         | 2.0 MB    | : 100% 1.0/1 [00:00<00:00,  3.47it/s]\n",
            "freetype-2.12.1      | 884 KB    | : 100% 1.0/1 [00:00<00:00,  5.26it/s]\n",
            "libxcb-1.13          | 391 KB    | : 100% 1.0/1 [00:00<00:00,  6.44it/s]\n",
            "libdeflate-1.14      | 81 KB     | : 100% 1.0/1 [00:00<00:00, 15.88it/s]\n",
            "pillow-9.2.0         | 45.0 MB   | : 100% 1.0/1 [00:04<00:00,  4.83s/it]               \n",
            "libwebp-base-1.2.4   | 404 KB    | : 100% 1.0/1 [00:00<00:00, 10.87it/s]\n",
            "jpeg-9e              | 269 KB    | : 100% 1.0/1 [00:00<00:00, 10.63it/s]\n",
            "libtiff-4.4.0        | 651 KB    | : 100% 1.0/1 [00:00<00:00,  8.38it/s]\n",
            "mkl-2022.1.0         | 199.6 MB  | : 100% 1.0/1 [00:33<00:00, 33.06s/it]               \n",
            "openjpeg-2.5.0       | 533 KB    | : 100% 1.0/1 [00:00<00:00,  8.43it/s]\n",
            "pthread-stubs-0.4    | 5 KB      | : 100% 1.0/1 [00:00<00:00, 32.11it/s]\n",
            "_openmp_mutex-4.5    | 6 KB      | : 100% 1.0/1 [00:00<00:00, 39.01it/s]\n",
            "nccl-2.14.3.1        | 145.2 MB  | : 100% 1.0/1 [00:17<00:00, 17.98s/it]               \n",
            "typing_extensions-4. | 29 KB     | : 100% 1.0/1 [00:00<00:00, 22.04it/s]\n",
            "lcms2-2.14           | 256 KB    | : 100% 1.0/1 [00:00<00:00, 15.36it/s]\n",
            "libpng-1.6.38        | 371 KB    | : 100% 1.0/1 [00:00<00:00, 11.59it/s]\n",
            "libprotobuf-3.20.1   | 2.6 MB    | : 100% 1.0/1 [00:00<00:00,  2.03it/s]\n",
            "lerc-4.0.0           | 275 KB    | : 100% 1.0/1 [00:00<00:00, 14.07it/s]\n",
            "torchvision-0.2.2    | 44 KB     | : 100% 1.0/1 [00:00<00:00, 19.89it/s]\n",
            "cudnn-8.4.1.50       | 618.4 MB  | : 100% 1.0/1 [01:18<00:00, 78.46s/it]               \n",
            "pytorch-1.12.1       | 489.7 MB  | : 100% 1.0/1 [01:14<00:00, 74.55s/it]  \n",
            "llvm-openmp-14.0.4   | 5.8 MB    | : 100% 1.0/1 [00:01<00:00,  1.09s/it]\n",
            "sleef-3.5.1          | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  3.44it/s]\n",
            "xorg-libxdmcp-1.1.3  | 19 KB     | : 100% 1.0/1 [00:00<00:00, 21.03it/s]\n",
            "xorg-libxau-1.0.9    | 13 KB     | : 100% 1.0/1 [00:00<00:00, 25.51it/s]\n",
            "Preparing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Verifying transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Executing transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ By downloading and using the cuDNN conda packages, you accept the terms and conditions of the NVIDIA cuDNN EULA -\n",
            "  https://docs.nvidia.com/deeplearning/cudnn/sla/index.html\n",
            "\n",
            "\b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Retrieving notices: ...working... done\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - cudatoolkit\n",
            "    - torchani=2.2.0\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    lark-parser-0.12.0         |     pyhd8ed1ab_0          78 KB  conda-forge\n",
            "    torchani-2.2               |     pyh9f0ad1d_0        10.1 MB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        10.2 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  lark-parser        conda-forge/noarch::lark-parser-0.12.0-pyhd8ed1ab_0 None\n",
            "  torchani           conda-forge/noarch::torchani-2.2-pyh9f0ad1d_0 None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "lark-parser-0.12.0   | 78 KB     | : 100% 1.0/1 [00:00<00:00,  2.64it/s]                \n",
            "torchani-2.2         | 10.1 MB   | : 100% 1.0/1 [00:03<00:00,  3.07s/it]               \n",
            "Preparing transaction: | \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Retrieving notices: ...working... done\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - scikit-learn\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    joblib-1.2.0               |     pyhd8ed1ab_0         205 KB  conda-forge\n",
            "    scikit-learn-1.0.2         |   py37hf9e9bfc_0         7.8 MB  conda-forge\n",
            "    scipy-1.7.3                |   py37hf2a6cf1_0        21.8 MB  conda-forge\n",
            "    threadpoolctl-3.1.0        |     pyh8a188c0_0          18 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        29.8 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  joblib             conda-forge/noarch::joblib-1.2.0-pyhd8ed1ab_0 None\n",
            "  scikit-learn       conda-forge/linux-64::scikit-learn-1.0.2-py37hf9e9bfc_0 None\n",
            "  scipy              conda-forge/linux-64::scipy-1.7.3-py37hf2a6cf1_0 None\n",
            "  threadpoolctl      conda-forge/noarch::threadpoolctl-3.1.0-pyh8a188c0_0 None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "scipy-1.7.3          | 21.8 MB   | : 100% 1.0/1 [00:03<00:00,  3.35s/it]               \n",
            "scikit-learn-1.0.2   | 7.8 MB    | : 100% 1.0/1 [00:01<00:00,  1.45s/it]               \n",
            "threadpoolctl-3.1.0  | 18 KB     | : 100% 1.0/1 [00:00<00:00, 21.07it/s]\n",
            "joblib-1.2.0         | 205 KB    | : 100% 1.0/1 [00:00<00:00, 11.98it/s]\n",
            "Preparing transaction: - \b\b\\ \b\bdone\n",
            "Verifying transaction: / \b\b- \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Retrieving notices: ...working... done\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install ASE and Joblib\n",
        "!conda install -c conda-forge ase\n",
        "!conda install -c anaconda joblib"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "u3P301beutJd",
        "outputId": "c0fd1f72-63c8-42cd-e4b7-d1959473f091"
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - ase\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ase-3.22.1                 |     pyhd8ed1ab_1         1.6 MB  conda-forge\n",
            "    click-8.1.3                |   py37h89c1867_0         145 KB  conda-forge\n",
            "    cycler-0.11.0              |     pyhd8ed1ab_0          10 KB  conda-forge\n",
            "    flask-2.2.2                |     pyhd8ed1ab_0          74 KB  conda-forge\n",
            "    importlib-metadata-4.11.4  |   py37h89c1867_0          33 KB  conda-forge\n",
            "    itsdangerous-2.1.2         |     pyhd8ed1ab_0          16 KB  conda-forge\n",
            "    jinja2-3.1.2               |     pyhd8ed1ab_1          99 KB  conda-forge\n",
            "    kiwisolver-1.4.4           |   py37h7cecad7_0          73 KB  conda-forge\n",
            "    markupsafe-2.1.1           |   py37h540881e_1          22 KB  conda-forge\n",
            "    matplotlib-base-3.4.3      |   py37h1058ff1_2         7.3 MB  conda-forge\n",
            "    pyparsing-3.0.9            |     pyhd8ed1ab_0          79 KB  conda-forge\n",
            "    python-dateutil-2.8.2      |     pyhd8ed1ab_0         240 KB  conda-forge\n",
            "    tornado-6.2                |   py37h540881e_0         651 KB  conda-forge\n",
            "    typing-extensions-4.4.0    |       hd8ed1ab_0           8 KB  conda-forge\n",
            "    werkzeug-2.2.2             |     pyhd8ed1ab_0         249 KB  conda-forge\n",
            "    zipp-3.10.0                |     pyhd8ed1ab_0          14 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        10.5 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  ase                conda-forge/noarch::ase-3.22.1-pyhd8ed1ab_1 None\n",
            "  click              conda-forge/linux-64::click-8.1.3-py37h89c1867_0 None\n",
            "  cycler             conda-forge/noarch::cycler-0.11.0-pyhd8ed1ab_0 None\n",
            "  flask              conda-forge/noarch::flask-2.2.2-pyhd8ed1ab_0 None\n",
            "  importlib-metadata conda-forge/linux-64::importlib-metadata-4.11.4-py37h89c1867_0 None\n",
            "  itsdangerous       conda-forge/noarch::itsdangerous-2.1.2-pyhd8ed1ab_0 None\n",
            "  jinja2             conda-forge/noarch::jinja2-3.1.2-pyhd8ed1ab_1 None\n",
            "  kiwisolver         conda-forge/linux-64::kiwisolver-1.4.4-py37h7cecad7_0 None\n",
            "  markupsafe         conda-forge/linux-64::markupsafe-2.1.1-py37h540881e_1 None\n",
            "  matplotlib-base    conda-forge/linux-64::matplotlib-base-3.4.3-py37h1058ff1_2 None\n",
            "  pyparsing          conda-forge/noarch::pyparsing-3.0.9-pyhd8ed1ab_0 None\n",
            "  python-dateutil    conda-forge/noarch::python-dateutil-2.8.2-pyhd8ed1ab_0 None\n",
            "  tornado            conda-forge/linux-64::tornado-6.2-py37h540881e_0 None\n",
            "  typing-extensions  conda-forge/noarch::typing-extensions-4.4.0-hd8ed1ab_0 None\n",
            "  werkzeug           conda-forge/noarch::werkzeug-2.2.2-pyhd8ed1ab_0 None\n",
            "  zipp               conda-forge/noarch::zipp-3.10.0-pyhd8ed1ab_0 None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "itsdangerous-2.1.2   | 16 KB     | : 100% 1.0/1 [00:00<00:00,  9.19it/s]\n",
            "typing-extensions-4. | 8 KB      | : 100% 1.0/1 [00:00<00:00, 21.25it/s]\n",
            "jinja2-3.1.2         | 99 KB     | : 100% 1.0/1 [00:00<00:00, 16.51it/s]\n",
            "zipp-3.10.0          | 14 KB     | : 100% 1.0/1 [00:00<00:00, 28.10it/s]\n",
            "tornado-6.2          | 651 KB    | : 100% 1.0/1 [00:00<00:00,  5.97it/s]\n",
            "ase-3.22.1           | 1.6 MB    | : 100% 1.0/1 [00:00<00:00,  2.67it/s]\n",
            "cycler-0.11.0        | 10 KB     | : 100% 1.0/1 [00:00<00:00, 29.01it/s]\n",
            "flask-2.2.2          | 74 KB     | : 100% 1.0/1 [00:00<00:00, 18.97it/s]\n",
            "importlib-metadata-4 | 33 KB     | : 100% 1.0/1 [00:00<00:00, 27.66it/s]\n",
            "markupsafe-2.1.1     | 22 KB     | : 100% 1.0/1 [00:00<00:00, 26.69it/s]\n",
            "python-dateutil-2.8. | 240 KB    | : 100% 1.0/1 [00:00<00:00, 18.63it/s]\n",
            "pyparsing-3.0.9      | 79 KB     | : 100% 1.0/1 [00:00<00:00, 16.49it/s]\n",
            "click-8.1.3          | 145 KB    | : 100% 1.0/1 [00:00<00:00, 17.30it/s]\n",
            "werkzeug-2.2.2       | 249 KB    | : 100% 1.0/1 [00:00<00:00, 12.27it/s]\n",
            "kiwisolver-1.4.4     | 73 KB     | : 100% 1.0/1 [00:00<00:00, 17.58it/s]\n",
            "matplotlib-base-3.4. | 7.3 MB    | : 100% 1.0/1 [00:01<00:00,  1.15s/it]\n",
            "Preparing transaction: | \b\b/ \b\bdone\n",
            "Verifying transaction: \\ \b\b| \b\b/ \b\bdone\n",
            "Executing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Retrieving notices: ...working... done\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - joblib\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ca-certificates-2022.07.19 |       h06a4308_0         131 KB  anaconda\n",
            "    certifi-2022.6.15          |   py37h06a4308_0         156 KB  anaconda\n",
            "    joblib-1.1.0               |     pyhd3eb1b0_0         211 KB  anaconda\n",
            "    openssl-1.1.1q             |       h7f8727e_0         3.8 MB  anaconda\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:         4.3 MB\n",
            "\n",
            "The following packages will be SUPERSEDED by a higher-priority channel:\n",
            "\n",
            "  ca-certificates    conda-forge::ca-certificates-2022.9.2~ --> anaconda::ca-certificates-2022.07.19-h06a4308_0 None\n",
            "  certifi            conda-forge/noarch::certifi-2022.9.24~ --> anaconda/linux-64::certifi-2022.6.15-py37h06a4308_0 None\n",
            "  joblib             conda-forge::joblib-1.2.0-pyhd8ed1ab_0 --> anaconda::joblib-1.1.0-pyhd3eb1b0_0 None\n",
            "  openssl            conda-forge::openssl-1.1.1s-h166bdaf_0 --> anaconda::openssl-1.1.1q-h7f8727e_0 None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "joblib-1.1.0         | 211 KB    | : 100% 1.0/1 [00:00<00:00,  9.06it/s]\n",
            "ca-certificates-2022 | 131 KB    | : 100% 1.0/1 [00:00<00:00, 17.12it/s]\n",
            "openssl-1.1.1q       | 3.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.70it/s]\n",
            "certifi-2022.6.15    | 156 KB    | : 100% 1.0/1 [00:00<00:00, 19.56it/s]\n",
            "Preparing transaction: | \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Retrieving notices: ...working... done\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Get pKa-ANI\n",
        "!git clone https://github.com/isayevlab/pKa-ANI.git"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "f4ZunoI4uUcF",
        "outputId": "cac481e3-da1b-4bf9-e4ea-de92fd7fbfd2"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'pKa-ANI'...\n",
            "remote: Enumerating objects: 203, done.\u001b[K\n",
            "remote: Counting objects: 100% (151/151), done.\u001b[K\n",
            "remote: Compressing objects: 100% (107/107), done.\u001b[K\n",
            "remote: Total 203 (delta 94), reused 80 (delta 44), pack-reused 52\u001b[K\n",
            "Receiving objects: 100% (203/203), 39.70 MiB | 11.98 MiB/s, done.\n",
            "Resolving deltas: 100% (106/106), done.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install pKa-ANI\n",
        "%%bash\n",
        "cd pKa-ANI\n",
        "python setup.py install"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Lba7CzcVvJVt",
        "outputId": "4bcf0b73-c08a-45b8-df73-c20b4ac9fe51"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "running install\n",
            "running bdist_egg\n",
            "running egg_info\n",
            "creating pkaani.egg-info\n",
            "writing pkaani.egg-info/PKG-INFO\n",
            "writing dependency_links to pkaani.egg-info/dependency_links.txt\n",
            "writing entry points to pkaani.egg-info/entry_points.txt\n",
            "writing top-level names to pkaani.egg-info/top_level.txt\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest template 'MANIFEST.in'\n",
            "adding license file 'LICENSE'\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "installing library code to build/bdist.linux-x86_64/egg\n",
            "running install_lib\n",
            "running build_py\n",
            "creating build\n",
            "creating build/lib\n",
            "creating build/lib/pkaani\n",
            "copying pkaani/pkaani.py -> build/lib/pkaani\n",
            "copying pkaani/run.py -> build/lib/pkaani\n",
            "copying pkaani/__main__.py -> build/lib/pkaani\n",
            "copying pkaani/ase_io_proteindatabank_mod.py -> build/lib/pkaani\n",
            "copying pkaani/prep_pdb.py -> build/lib/pkaani\n",
            "copying pkaani/__init__.py -> build/lib/pkaani\n",
            "copying pkaani/ani_descriptors.py -> build/lib/pkaani\n",
            "creating build/lib/pkaani/models\n",
            "copying pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FASP.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FGLU.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FHIS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FLYS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FTYR.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "creating build/bdist.linux-x86_64\n",
            "creating build/bdist.linux-x86_64/egg\n",
            "creating build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/pkaani.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/run.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__main__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "creating build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FTYR.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FLYS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FASP.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FHIS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FGLU.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/ase_io_proteindatabank_mod.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/prep_pdb.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__init__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/ani_descriptors.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/pkaani.py to pkaani.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/run.py to run.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__main__.py to __main__.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ase_io_proteindatabank_mod.py to ase_io_proteindatabank_mod.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/prep_pdb.py to prep_pdb.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__init__.py to __init__.cpython-37.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ani_descriptors.py to ani_descriptors.cpython-37.pyc\n",
            "creating build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "creating dist\n",
            "creating 'dist/pkaani-0.1.0-py3.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it\n",
            "removing 'build/bdist.linux-x86_64/egg' (and everything under it)\n",
            "Processing pkaani-0.1.0-py3.7.egg\n",
            "creating /usr/local/lib/python3.7/site-packages/pkaani-0.1.0-py3.7.egg\n",
            "Extracting pkaani-0.1.0-py3.7.egg to /usr/local/lib/python3.7/site-packages\n",
            "Adding pkaani 0.1.0 to easy-install.pth file\n",
            "Installing pkaani script to /usr/local/bin\n",
            "\n",
            "Installed /usr/local/lib/python3.7/site-packages/pkaani-0.1.0-py3.7.egg\n",
            "Processing dependencies for pkaani==0.1.0\n",
            "Finished processing dependencies for pkaani==0.1.0\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/site-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.\n",
            "  setuptools.SetuptoolsDeprecationWarning,\n",
            "/usr/local/lib/python3.7/site-packages/setuptools/command/easy_install.py:147: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.\n",
            "  EasyInstallDeprecationWarning,\n",
            "/usr/local/lib/python3.7/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'pkaani.models' as data is deprecated, please list it in `packages`.\n",
            "    !!\n",
            "\n",
            "\n",
            "    ############################\n",
            "    # Package would be ignored #\n",
            "    ############################\n",
            "    Python recognizes 'pkaani.models' as an importable package,\n",
            "    but it is not listed in the `packages` configuration of setuptools.\n",
            "\n",
            "    'pkaani.models' has been automatically added to the distribution only\n",
            "    because it may contain data files, but this behavior is likely to change\n",
            "    in future versions of setuptools (and therefore is considered deprecated).\n",
            "\n",
            "    Please make sure that 'pkaani.models' is included as a package by using\n",
            "    the `packages` configuration field or the proper discovery methods\n",
            "    (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
            "    instead of `find_packages(...)`/`find:`).\n",
            "\n",
            "    You can read more about \"package discovery\" and \"data files\" on setuptools\n",
            "    documentation page.\n",
            "\n",
            "\n",
            "!!\n",
            "\n",
            "  check.warn(importable)\n",
            "zip_safe flag not set; analyzing archive contents...\n",
            "pkaani.__pycache__.pkaani.cpython-37: module references __file__\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install Modeller\n",
        "#@markdown You will need to obtain modeller Licence Key from \n",
        "#@markdown https://salilab.org/modeller/. \n",
        "modellerkey = 'MODELIRANJE' #@param {type:\"string\"}\n",
        "os.environ['KEY_MODELLER']=modellerkey\n",
        "!conda config --add channels salilab\n",
        "!conda install modeller\n",
        "from modeller import *\n",
        "from modeller.automodel import *"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Z500vZchxNUV",
        "outputId": "df243311-3afd-4ce7-9d02-33d7aaab7c78"
      },
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Solving environment: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - modeller\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    hdf5-1107-1.10.7           |                0         3.1 MB  salilab\n",
            "    libzlib-1.2.13             |       h166bdaf_4          64 KB  conda-forge\n",
            "    modeller-10.4              |   py37h9bf148f_0        16.7 MB  salilab\n",
            "    zlib-1.2.13                |       h166bdaf_4          92 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        20.0 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  hdf5-1107          salilab/linux-64::hdf5-1107-1.10.7-0 None\n",
            "  modeller           salilab/linux-64::modeller-10.4-py37h9bf148f_0 None\n",
            "  zlib               conda-forge/linux-64::zlib-1.2.13-h166bdaf_4 None\n",
            "\n",
            "The following packages will be UPDATED:\n",
            "\n",
            "  ca-certificates    anaconda::ca-certificates-2022.07.19-~ --> conda-forge::ca-certificates-2022.9.24-ha878542_0 None\n",
            "  certifi            anaconda/linux-64::certifi-2022.6.15-~ --> conda-forge/noarch::certifi-2022.9.24-pyhd8ed1ab_0 None\n",
            "  libzlib                                 1.2.12-h166bdaf_2 --> 1.2.13-h166bdaf_4 None\n",
            "  openssl               anaconda::openssl-1.1.1q-h7f8727e_0 --> conda-forge::openssl-1.1.1s-h166bdaf_0 None\n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "modeller-10.4        | 16.7 MB   | : 100% 1.0/1 [00:07<00:00,  7.54s/it]\n",
            "hdf5-1107-1.10.7     | 3.1 MB    | : 100% 1.0/1 [00:01<00:00,  1.30s/it]\n",
            "libzlib-1.2.13       | 64 KB     | : 100% 1.0/1 [00:00<00:00, 23.28it/s]\n",
            "zlib-1.2.13          | 92 KB     | : 100% 1.0/1 [00:00<00:00, 24.25it/s]\n",
            "Preparing transaction: | \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Executing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Retrieving notices: ...working... done\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install MDAnalysis\n",
        "!pip install --upgrade MDAnalysis\n",
        "import MDAnalysis as mda\n",
        "from MDAnalysis.analysis import rms, align"
      ],
      "metadata": {
        "id": "YN2XoO6K0xcI",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "72f2a620-c7ef-4198-8d4c-17755dc92462"
      },
      "execution_count": 101,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: MDAnalysis in /usr/local/lib/python3.7/site-packages (2.1.0)\n",
            "Requirement already satisfied: joblib>=0.12 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (1.1.0)\n",
            "Requirement already satisfied: threadpoolctl in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (3.1.0)\n",
            "Requirement already satisfied: scipy>=1.0.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (1.7.3)\n",
            "Requirement already satisfied: networkx>=1.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (2.6.3)\n",
            "Requirement already satisfied: matplotlib>=1.5.1 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (3.4.3)\n",
            "Requirement already satisfied: tqdm>=4.43.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (4.64.0)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (21.3)\n",
            "Requirement already satisfied: numpy>=1.18.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (1.21.6)\n",
            "Requirement already satisfied: biopython>=1.71 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (1.79)\n",
            "Requirement already satisfied: mmtf-python>=1.0.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (1.1.3)\n",
            "Requirement already satisfied: gsd>=1.4.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (2.6.1)\n",
            "Requirement already satisfied: GridDataFormats>=0.4.0 in /usr/local/lib/python3.7/site-packages (from MDAnalysis) (0.7.0)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/site-packages (from GridDataFormats>=0.4.0->MDAnalysis) (1.16.0)\n",
            "Requirement already satisfied: mrcfile in /usr/local/lib/python3.7/site-packages (from GridDataFormats>=0.4.0->MDAnalysis) (1.4.3)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.7/site-packages (from matplotlib>=1.5.1->MDAnalysis) (0.11.0)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.7/site-packages (from matplotlib>=1.5.1->MDAnalysis) (1.4.4)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.7/site-packages (from matplotlib>=1.5.1->MDAnalysis) (9.2.0)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.7/site-packages (from matplotlib>=1.5.1->MDAnalysis) (2.8.2)\n",
            "Requirement already satisfied: pyparsing>=2.2.1 in /usr/local/lib/python3.7/site-packages (from matplotlib>=1.5.1->MDAnalysis) (3.0.9)\n",
            "Requirement already satisfied: msgpack>=1.0.0 in /usr/local/lib/python3.7/site-packages (from mmtf-python>=1.0.0->MDAnalysis) (1.0.4)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/site-packages (from kiwisolver>=1.0.1->matplotlib>=1.5.1->MDAnalysis) (4.4.0)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Set up parameters"
      ],
      "metadata": {
        "id": "K8CDGsuuvPyP"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Select GROMACS version\n",
        "#@markdown Select an option from dropdown menu\n",
        "gmx_version = '2022' #@param [\"2021\",\"2022\"]\n",
        "#@markdown - Currently versions 2021(2021.5) and 2022(2022.3) are availablle\n",
        "gdrive_path = str('/content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-'+gmx_version+'.zip')\n",
        "local_path = str('/content/gromacs-'+gmx_version)\n",
        "os.environ['GMX_LOCAL_PATH']=local_path\n",
        "os.environ['GMX_ZIP_PATH']=gdrive_path"
      ],
      "metadata": {
        "id": "WaEsPhxR1Vd9"
      },
      "execution_count": 10,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Select PDB input\n",
        "#@markdown If you want to upload file from google drive, type `google drive`.\n",
        "#@markdown Next, provide in the the path to the file in your google drive in \n",
        "#@markdown `pdb_gdrive_path` entry\n",
        "#@markdown If you want to upload file from local system, type `upload`. You will\n",
        "#@markdown be asked to upload the file later\n",
        "pdbid = '1aki' #@param {type:\"string\"}\n",
        "pdb_gdrive_path = '' #@param {type:\"string\"}"
      ],
      "metadata": {
        "id": "66HlISLG44RU"
      },
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Fix missing residues\n",
        "#@markdown Select the software to deal with missing residues. Currently, only\n",
        "#@markdown `PDBFixer` is available.\n",
        "missing_residues_soft = 'pdbfixer' #@param [\"pdbfixer\",\"modeller\"]"
      ],
      "metadata": {
        "id": "SlmykXYo6HYG"
      },
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Mutants to simulate\n",
        "#@markdown If you want to simulate mutants upload a file with a list of mutants\n",
        "#@markdown The file should have the following format:\n",
        "\n",
        "#@markdown Commented lines start with \\#.\n",
        "#@markdown Each line corresponds to a single mutant. Each mutation is described\n",
        "#@markdown as XNNNY, where X is one-letter code of the original amino acid, NNN\n",
        "#@markdown is the resid of the mutated amino acid in the template PDB file, \n",
        "#@markdown Y is the mutated amino acid.\n",
        "\n",
        "simulate_mutants = \"True\" #@param [\"True\", \"False\"]\n",
        "from google.colab import files\n",
        "if simulate_mutants:\n",
        "    uploaded = files.upload()\n",
        "    mutations_file = list(uploaded.keys())[-1]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 76
        },
        "id": "wt5Sy2pO6290",
        "outputId": "39bd8fdf-566f-4be8-c1c9-a69d0bea3025"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ],
            "text/html": [
              "\n",
              "     <input type=\"file\" id=\"files-ca25f9b5-975d-458c-99c4-803c33400201\" name=\"files[]\" multiple disabled\n",
              "        style=\"border:none\" />\n",
              "     <output id=\"result-ca25f9b5-975d-458c-99c4-803c33400201\">\n",
              "      Upload widget is only available when the cell has been executed in the\n",
              "      current browser session. Please rerun this cell to enable.\n",
              "      </output>\n",
              "      <script>// Copyright 2017 Google LLC\n",
              "//\n",
              "// Licensed under the Apache License, Version 2.0 (the \"License\");\n",
              "// you may not use this file except in compliance with the License.\n",
              "// You may obtain a copy of the License at\n",
              "//\n",
              "//      http://www.apache.org/licenses/LICENSE-2.0\n",
              "//\n",
              "// Unless required by applicable law or agreed to in writing, software\n",
              "// distributed under the License is distributed on an \"AS IS\" BASIS,\n",
              "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
              "// See the License for the specific language governing permissions and\n",
              "// limitations under the License.\n",
              "\n",
              "/**\n",
              " * @fileoverview Helpers for google.colab Python module.\n",
              " */\n",
              "(function(scope) {\n",
              "function span(text, styleAttributes = {}) {\n",
              "  const element = document.createElement('span');\n",
              "  element.textContent = text;\n",
              "  for (const key of Object.keys(styleAttributes)) {\n",
              "    element.style[key] = styleAttributes[key];\n",
              "  }\n",
              "  return element;\n",
              "}\n",
              "\n",
              "// Max number of bytes which will be uploaded at a time.\n",
              "const MAX_PAYLOAD_SIZE = 100 * 1024;\n",
              "\n",
              "function _uploadFiles(inputId, outputId) {\n",
              "  const steps = uploadFilesStep(inputId, outputId);\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  // Cache steps on the outputElement to make it available for the next call\n",
              "  // to uploadFilesContinue from Python.\n",
              "  outputElement.steps = steps;\n",
              "\n",
              "  return _uploadFilesContinue(outputId);\n",
              "}\n",
              "\n",
              "// This is roughly an async generator (not supported in the browser yet),\n",
              "// where there are multiple asynchronous steps and the Python side is going\n",
              "// to poll for completion of each step.\n",
              "// This uses a Promise to block the python side on completion of each step,\n",
              "// then passes the result of the previous step as the input to the next step.\n",
              "function _uploadFilesContinue(outputId) {\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  const steps = outputElement.steps;\n",
              "\n",
              "  const next = steps.next(outputElement.lastPromiseValue);\n",
              "  return Promise.resolve(next.value.promise).then((value) => {\n",
              "    // Cache the last promise value to make it available to the next\n",
              "    // step of the generator.\n",
              "    outputElement.lastPromiseValue = value;\n",
              "    return next.value.response;\n",
              "  });\n",
              "}\n",
              "\n",
              "/**\n",
              " * Generator function which is called between each async step of the upload\n",
              " * process.\n",
              " * @param {string} inputId Element ID of the input file picker element.\n",
              " * @param {string} outputId Element ID of the output display.\n",
              " * @return {!Iterable<!Object>} Iterable of next steps.\n",
              " */\n",
              "function* uploadFilesStep(inputId, outputId) {\n",
              "  const inputElement = document.getElementById(inputId);\n",
              "  inputElement.disabled = false;\n",
              "\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  outputElement.innerHTML = '';\n",
              "\n",
              "  const pickedPromise = new Promise((resolve) => {\n",
              "    inputElement.addEventListener('change', (e) => {\n",
              "      resolve(e.target.files);\n",
              "    });\n",
              "  });\n",
              "\n",
              "  const cancel = document.createElement('button');\n",
              "  inputElement.parentElement.appendChild(cancel);\n",
              "  cancel.textContent = 'Cancel upload';\n",
              "  const cancelPromise = new Promise((resolve) => {\n",
              "    cancel.onclick = () => {\n",
              "      resolve(null);\n",
              "    };\n",
              "  });\n",
              "\n",
              "  // Wait for the user to pick the files.\n",
              "  const files = yield {\n",
              "    promise: Promise.race([pickedPromise, cancelPromise]),\n",
              "    response: {\n",
              "      action: 'starting',\n",
              "    }\n",
              "  };\n",
              "\n",
              "  cancel.remove();\n",
              "\n",
              "  // Disable the input element since further picks are not allowed.\n",
              "  inputElement.disabled = true;\n",
              "\n",
              "  if (!files) {\n",
              "    return {\n",
              "      response: {\n",
              "        action: 'complete',\n",
              "      }\n",
              "    };\n",
              "  }\n",
              "\n",
              "  for (const file of files) {\n",
              "    const li = document.createElement('li');\n",
              "    li.append(span(file.name, {fontWeight: 'bold'}));\n",
              "    li.append(span(\n",
              "        `(${file.type || 'n/a'}) - ${file.size} bytes, ` +\n",
              "        `last modified: ${\n",
              "            file.lastModifiedDate ? file.lastModifiedDate.toLocaleDateString() :\n",
              "                                    'n/a'} - `));\n",
              "    const percent = span('0% done');\n",
              "    li.appendChild(percent);\n",
              "\n",
              "    outputElement.appendChild(li);\n",
              "\n",
              "    const fileDataPromise = new Promise((resolve) => {\n",
              "      const reader = new FileReader();\n",
              "      reader.onload = (e) => {\n",
              "        resolve(e.target.result);\n",
              "      };\n",
              "      reader.readAsArrayBuffer(file);\n",
              "    });\n",
              "    // Wait for the data to be ready.\n",
              "    let fileData = yield {\n",
              "      promise: fileDataPromise,\n",
              "      response: {\n",
              "        action: 'continue',\n",
              "      }\n",
              "    };\n",
              "\n",
              "    // Use a chunked sending to avoid message size limits. See b/62115660.\n",
              "    let position = 0;\n",
              "    do {\n",
              "      const length = Math.min(fileData.byteLength - position, MAX_PAYLOAD_SIZE);\n",
              "      const chunk = new Uint8Array(fileData, position, length);\n",
              "      position += length;\n",
              "\n",
              "      const base64 = btoa(String.fromCharCode.apply(null, chunk));\n",
              "      yield {\n",
              "        response: {\n",
              "          action: 'append',\n",
              "          file: file.name,\n",
              "          data: base64,\n",
              "        },\n",
              "      };\n",
              "\n",
              "      let percentDone = fileData.byteLength === 0 ?\n",
              "          100 :\n",
              "          Math.round((position / fileData.byteLength) * 100);\n",
              "      percent.textContent = `${percentDone}% done`;\n",
              "\n",
              "    } while (position < fileData.byteLength);\n",
              "  }\n",
              "\n",
              "  // All done.\n",
              "  yield {\n",
              "    response: {\n",
              "      action: 'complete',\n",
              "    }\n",
              "  };\n",
              "}\n",
              "\n",
              "scope.google = scope.google || {};\n",
              "scope.google.colab = scope.google.colab || {};\n",
              "scope.google.colab._files = {\n",
              "  _uploadFiles,\n",
              "  _uploadFilesContinue,\n",
              "};\n",
              "})(self);\n",
              "</script> "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Saving mutations.dat to mutations.dat\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Select the force field\n",
        "#@markdown We currently provide access to three force fields: AMBER99-ILDN, CHARMM36m, and AMBER14SB\n",
        "ff = 'charmm36' #@param {type:\"string\"} [\"amber99\", \"amber14\", \"charmm36\"]\n",
        "os.environ['FF']=ff"
      ],
      "metadata": {
        "id": "kPOBdeg6GPw8"
      },
      "execution_count": 14,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Download GROMACS"
      ],
      "metadata": {
        "id": "DTdIs9xeSpaa"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "m3AnBBAYO3Uz",
        "outputId": "d588dbd6-9a76-4e61-b0dd-5ad714124c62"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/gdrive\n"
          ]
        }
      ],
      "source": [
        "#@title Gromacs upload\n",
        "#@markdown We expect GROMACS to be preinstalled with one of our scripts and\n",
        "#@markdown stored on Google Drive\n",
        "from google.colab import drive\n",
        "drive.mount('/content/gdrive')"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Extract GROMACS binaries\n",
        "!unzip $GMX_ZIP_PATH"
      ],
      "metadata": {
        "id": "5sqIYA4lVSD4",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b3409715-4866-4ca4-83a2-103360e3ed3d"
      },
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-2022.zip\n",
            "   creating: gromacs-2022/\n",
            "   creating: gromacs-2022/lib/\n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0.3.1  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2.3.2  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0.1.0  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7.0.0  \n",
            "   creating: gromacs-2022/lib/pkgconfig/\n",
            "  inflating: gromacs-2022/lib/pkgconfig/muparser.pc  \n",
            "  inflating: gromacs-2022/lib/pkgconfig/libgromacs.pc  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0  \n",
            "   creating: gromacs-2022/include/\n",
            "  inflating: gromacs-2022/include/muParserToken.h  \n",
            "  inflating: gromacs-2022/include/muParserTemplateMagic.h  \n",
            "   creating: gromacs-2022/include/nblib/\n",
            "  inflating: gromacs-2022/include/nblib/nbnxmsetuphelpers.h  \n",
            "  inflating: gromacs-2022/include/nblib/integrator.h  \n",
            "   creating: gromacs-2022/include/nblib/listed_forces/\n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/definitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/calculator.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/bondtypes.h  \n",
            "  inflating: gromacs-2022/include/nblib/topology.h  \n",
            "   creating: gromacs-2022/include/nblib/util/\n",
            "  inflating: gromacs-2022/include/nblib/util/traits.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/util/setup.h  \n",
            "  inflating: gromacs-2022/include/nblib/util/util.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/gmxcalculatorcpu.h  \n",
            "  inflating: gromacs-2022/include/nblib/particletype.h  \n",
            "  inflating: gromacs-2022/include/nblib/interactions.h  \n",
            "  inflating: gromacs-2022/include/nblib/tpr.h  \n",
            "  inflating: gromacs-2022/include/nblib/exception.h  \n",
            "  inflating: gromacs-2022/include/nblib/basicdefinitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/nblib.h  \n",
            "  inflating: gromacs-2022/include/nblib/kerneloptions.h  \n",
            "  inflating: gromacs-2022/include/nblib/particlesequencer.h  \n",
            "  inflating: gromacs-2022/include/nblib/box.h  \n",
            "  inflating: gromacs-2022/include/nblib/simulationstate.h  \n",
            "  inflating: gromacs-2022/include/nblib/molecules.h  \n",
            "  inflating: gromacs-2022/include/nblib/vector.h  \n",
            "  inflating: gromacs-2022/include/muParserInt.h  \n",
            "   creating: gromacs-2022/include/gmxapi/\n",
            "  inflating: gromacs-2022/include/gmxapi/status.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/session.h  \n",
            "   creating: gromacs-2022/include/gmxapi/md/\n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdmodule.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdsignals.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/version.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gromacsfwd.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapicompat.h  \n",
            "   creating: gromacs-2022/include/gmxapi/session/\n",
            "  inflating: gromacs-2022/include/gmxapi/session/resources.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/exceptions.h  \n",
            "   creating: gromacs-2022/include/gmxapi/compat/\n",
            "  inflating: gromacs-2022/include/gmxapi/compat/tpr.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/compat/mdparams.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/system.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/context.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md.h  \n",
            "   creating: gromacs-2022/include/gmxapi/mpi/\n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/gmxapi_mpi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/resourceassignment.h  \n",
            "  inflating: gromacs-2022/include/muParserTokenReader.h  \n",
            "  inflating: gromacs-2022/include/muParserBytecode.h  \n",
            "   creating: gromacs-2022/include/gromacs/\n",
            "   creating: gromacs-2022/include/gromacs/mdrun/\n",
            "  inflating: gromacs-2022/include/gromacs/mdrun/simulationinputhandle.h  \n",
            "  inflating: gromacs-2022/include/muParserFixes.h  \n",
            "  inflating: gromacs-2022/include/muParserError.h  \n",
            "  inflating: gromacs-2022/include/muParserDef.h  \n",
            "  inflating: gromacs-2022/include/muParserBase.h  \n",
            "  inflating: gromacs-2022/include/muParser.h  \n",
            "  inflating: gromacs-2022/include/muParserDLL.h  \n",
            "  inflating: gromacs-2022/include/muParserTest.h  \n",
            "  inflating: gromacs-2022/include/muParserCallback.h  \n",
            "   creating: gromacs-2022/bin/\n",
            "  inflating: gromacs-2022/bin/GMXRC.csh  \n",
            "  inflating: gromacs-2022/bin/GMXRC  \n",
            "  inflating: gromacs-2022/bin/xplor2gmx.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.zsh  \n",
            "  inflating: gromacs-2022/bin/gmx    \n",
            "  inflating: gromacs-2022/bin/demux.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion-gmx.bash  \n",
            "   creating: gromacs-2022/share/\n",
            "   creating: gromacs-2022/share/cmake/\n",
            "   creating: gromacs-2022/share/cmake/gmxapi/\n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config.cmake  \n",
            "   creating: gromacs-2022/share/cmake/gromacs/\n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-hints.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config.cmake  \n",
            "   creating: gromacs-2022/share/gromacs/\n",
            "  inflating: gromacs-2022/share/gromacs/README_FreeEnergyModifications.txt  \n",
            "   creating: gromacs-2022/share/gromacs/top/\n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a1.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a5.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-9.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a2.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/oplsaa.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ethanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/1propanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5pe.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residuetypes.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa-n.tst  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atommass.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/export.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/edissoc.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/nsfactor.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-12.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/dgsolv.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip5p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/defselection.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/bonds.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/surface.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/electroneg.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/elements.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a6.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ps.m2p  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sfactor.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/charmm27.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/cmap.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tips3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.arn  \n",
            " extracting: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnanonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnabonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/specbond.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/co-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdwradii.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber03.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber96.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a5.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG45a3.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/phbres.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.xml  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a1.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-8.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atom_nom.tbl  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ha-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ss.map  \n",
            "   creating: gromacs-2022/share/gromacs/top/amberGS.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos54a7.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tmcl.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/dppc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/popc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/random.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.dtd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-11.xvg  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos45a3.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdw-msms.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber94.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ca-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sw.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-10.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/xlateat.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a6.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a2.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexwat-ferguson.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/README  \n",
            "  inflating: gromacs-2022/share/gromacs/top/refi_aa.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/cb-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/COPYING  \n",
            "  inflating: gromacs-2022/share/gromacs/README.tutor  \n",
            "   creating: gromacs-2022/share/man/\n",
            "   creating: gromacs-2022/share/man/man1/\n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-lie.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-density.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mk_angndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-grompp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spol.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spatial.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sasa.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densmap.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dipoles.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmens.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-xpm2ps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pdb2gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wheel.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-insert-molecules.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-tpr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dielectric.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotacf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dos.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-energy.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-clustsize.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-report-methods.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-filter.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gyrate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-editconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-do_dssp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-velacc.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hbond.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-confrms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-view.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-analyze.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_edi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-awh.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tune_pme.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nonbonded-benchmark.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-distance.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-trj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmtraj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-principal.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tcaf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genrestr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-vanhove.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pairdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bundle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_ndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-potential.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dyecoupl.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-help.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-h2order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-disre.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdrun.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mindist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-eneconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-anaeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dump.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helix.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-check.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genion.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sans.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-chi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-msd.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-solvate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rama.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sigeps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-extract-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-angle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-enemat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-current.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gangle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saxs.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-freevolume.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helixorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-x2top.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-select.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pme_error.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saltbr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hydorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trajectory.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-polystat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjcat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-covar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rdf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-traj.1  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Test GROMACS installation\n",
        "%%bash\n",
        "chmod -R 755 $GMX_LOCAL_PATH\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "gmx grompp -h"
      ],
      "metadata": {
        "id": "0lhkHjz0TGBx",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0fb6a661-aed7-45ca-93a2-1f2a82e06755"
      },
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "SYNOPSIS\n",
            "\n",
            "gmx grompp [-f [<.mdp>]] [-c [<.gro/.g96/...>]] [-r [<.gro/.g96/...>]]\n",
            "           [-rb [<.gro/.g96/...>]] [-n [<.ndx>]] [-p [<.top>]]\n",
            "           [-t [<.trr/.cpt/...>]] [-e [<.edr>]] [-qmi [<.inp>]]\n",
            "           [-ref [<.trr/.cpt/...>]] [-po [<.mdp>]] [-pp [<.top>]]\n",
            "           [-o [<.tpr>]] [-imd [<.gro>]] [-[no]v] [-time <real>]\n",
            "           [-[no]rmvsbds] [-maxwarn <int>] [-[no]zero] [-[no]renum]\n",
            "\n",
            "DESCRIPTION\n",
            "\n",
            "gmx grompp (the gromacs preprocessor) reads a molecular topology file, checks\n",
            "the validity of the file, expands the topology from a molecular description to\n",
            "an atomic description. The topology file contains information about molecule\n",
            "types and the number of molecules, the preprocessor copies each molecule as\n",
            "needed. There is no limitation on the number of molecule types. Bonds and\n",
            "bond-angles can be converted into constraints, separately for hydrogens and\n",
            "heavy atoms. Then a coordinate file is read and velocities can be generated\n",
            "from a Maxwellian distribution if requested. gmx grompp also reads parameters\n",
            "for gmx mdrun (eg. number of MD steps, time step, cut-off). Eventually a\n",
            "binary file is produced that can serve as the sole input file for the MD\n",
            "program.\n",
            "\n",
            "gmx grompp uses the atom names from the topology file. The atom names in the\n",
            "coordinate file (option -c) are only read to generate warnings when they do\n",
            "not match the atom names in the topology. Note that the atom names are\n",
            "irrelevant for the simulation as only the atom types are used for generating\n",
            "interaction parameters.\n",
            "\n",
            "gmx grompp uses a built-in preprocessor to resolve includes, macros, etc. The\n",
            "preprocessor supports the following keywords:\n",
            "\n",
            "    #ifdef VARIABLE\n",
            "    #ifndef VARIABLE\n",
            "    #else\n",
            "    #endif\n",
            "    #define VARIABLE\n",
            "    #undef VARIABLE\n",
            "    #include \"filename\"\n",
            "    #include <filename>\n",
            "\n",
            "The functioning of these statements in your topology may be modulated by using\n",
            "the following two flags in your .mdp file:\n",
            "\n",
            "    define = -DVARIABLE1 -DVARIABLE2\n",
            "    include = -I/home/john/doe\n",
            "\n",
            "For further information a C-programming textbook may help you out. Specifying\n",
            "the -pp flag will get the pre-processed topology file written out so that you\n",
            "can verify its contents.\n",
            "\n",
            "When using position restraints, a file with restraint coordinates must be\n",
            "supplied with -r (can be the same file as supplied for -c). For free energy\n",
            "calculations, separate reference coordinates for the B topology can be\n",
            "supplied with -rb, otherwise they will be equal to those of the A topology.\n",
            "\n",
            "Starting coordinates can be read from trajectory with -t. The last frame with\n",
            "coordinates and velocities will be read, unless the -time option is used. Only\n",
            "if this information is absent will the coordinates in the -c file be used.\n",
            "Note that these velocities will not be used when gen_vel = yes in your .mdp\n",
            "file. An energy file can be supplied with -e to read Nose-Hoover and/or\n",
            "Parrinello-Rahman coupling variables.\n",
            "\n",
            "gmx grompp can be used to restart simulations (preserving continuity) by\n",
            "supplying just a checkpoint file with -t. However, for simply changing the\n",
            "number of run steps to extend a run, using gmx convert-tpr is more convenient\n",
            "than gmx grompp. You then supply the old checkpoint file directly to gmx mdrun\n",
            "with -cpi. If you wish to change the ensemble or things like output frequency,\n",
            "then supplying the checkpoint file to gmx grompp with -t along with a new .mdp\n",
            "file with -f is the recommended procedure. Actually preserving the ensemble\n",
            "(if possible) still requires passing the checkpoint file to gmx mdrun -cpi.\n",
            "\n",
            "By default, all bonded interactions which have constant energy due to virtual\n",
            "site constructions will be removed. If this constant energy is not zero, this\n",
            "will result in a shift in the total energy. All bonded interactions can be\n",
            "kept by turning off -rmvsbds. Additionally, all constraints for distances\n",
            "which will be constant anyway because of virtual site constructions will be\n",
            "removed. If any constraints remain which involve virtual sites, a fatal error\n",
            "will result.\n",
            "\n",
            "To verify your run input file, please take note of all warnings on the screen,\n",
            "and correct where necessary. Do also look at the contents of the mdout.mdp\n",
            "file; this contains comment lines, as well as the input that gmx grompp has\n",
            "read. If in doubt, you can start gmx grompp with the -debug option which will\n",
            "give you more information in a file called grompp.log (along with real debug\n",
            "info). You can see the contents of the run input file with the gmx dump\n",
            "program. gmx check can be used to compare the contents of two run input files.\n",
            "\n",
            "The -maxwarn option can be used to override warnings printed by gmx grompp\n",
            "that otherwise halt output. In some cases, warnings are harmless, but usually\n",
            "they are not. The user is advised to carefully interpret the output messages\n",
            "before attempting to bypass them with this option.\n",
            "\n",
            "OPTIONS\n",
            "\n",
            "Options to specify input files:\n",
            "\n",
            " -f      [<.mdp>]           (grompp.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -c      [<.gro/.g96/...>]  (conf.gro)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -r      [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -rb     [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -n      [<.ndx>]           (index.ndx)      (Opt.)\n",
            "           Index file\n",
            " -p      [<.top>]           (topol.top)\n",
            "           Topology file\n",
            " -t      [<.trr/.cpt/...>]  (traj.trr)       (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            " -e      [<.edr>]           (ener.edr)       (Opt.)\n",
            "           Energy file\n",
            " -qmi    [<.inp>]           (topol-qmmm.inp) (Opt.)\n",
            "           Input file for QM program\n",
            "\n",
            "Options to specify input/output files:\n",
            "\n",
            " -ref    [<.trr/.cpt/...>]  (rotref.trr)     (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            "\n",
            "Options to specify output files:\n",
            "\n",
            " -po     [<.mdp>]           (mdout.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -pp     [<.top>]           (processed.top)  (Opt.)\n",
            "           Topology file\n",
            " -o      [<.tpr>]           (topol.tpr)\n",
            "           Portable xdr run input file\n",
            " -imd    [<.gro>]           (imdgroup.gro)   (Opt.)\n",
            "           Coordinate file in Gromos-87 format\n",
            "\n",
            "Other options:\n",
            "\n",
            " -[no]v                     (no)\n",
            "           Be loud and noisy\n",
            " -time   <real>             (-1)\n",
            "           Take frame at or first after this time.\n",
            " -[no]rmvsbds               (yes)\n",
            "           Remove constant bonded interactions with virtual sites\n",
            " -maxwarn <int>             (0)\n",
            "           Number of allowed warnings during input processing. Not for normal\n",
            "           use and may generate unstable systems\n",
            " -[no]zero                  (no)\n",
            "           Set parameters for bonded interactions without defaults to zero\n",
            "           instead of generating an error\n",
            " -[no]renum                 (yes)\n",
            "           Renumber atomtypes and minimize number of atomtypes\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content\n",
            "Command line:\n",
            "  gmx grompp -h\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"Millions saw the apple fall, Newton was the only one who asked why?\" (Bernard Baruch)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Download PDB structure"
      ],
      "metadata": {
        "id": "PwNRqMHj_X-K"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Prepare pdb\n",
        "#@markdown If you have selected an option for uploading PDB from local drive,\n",
        "#@markdown please provide the PDB structure.\n",
        "if pdbid == \"upload\":\n",
        "    uploaded = files.upload()\n",
        "    pdb_file = list(uploaded.keys())[-1]\n",
        "    path = f\"./md_base/\"\n",
        "    os.environ['MD_PATH']=path\n",
        "    os.environ['PDB']=\"base\"\n",
        "    if not os.path.exists(path):\n",
        "        os.mkdir(path)\n",
        "    shutil.copy2(pdb_file, path+'input.pdb')\n",
        "    input_pdb = path + \"input.pdb\"\n",
        "elif pdbid == \"google drive\":\n",
        "    path = f\"./md_base/\"\n",
        "    os.environ['MD_PATH']=path\n",
        "    os.environ['PDB']=\"base\"\n",
        "    if not os.path.exists(path):\n",
        "        os.mkdir(path)\n",
        "    shutil.copy2('./gdrive/'+pdb_gdrive_path, path+'input.pdb')\n",
        "    input_pdb = path + \"input.pdb\"\n",
        "else:\n",
        "    if not os.path.exists(f\"./md_{pdbid}\"):\n",
        "        os.mkdir(f\"./md_{pdbid}\")\n",
        "    path = f\"./md_{pdbid}/\"\n",
        "    os.environ['MD_PATH']=path\n",
        "    os.environ['PDB']=pdbid\n",
        "    pdbl = PDBList()\n",
        "    filename = pdbl.retrieve_pdb_file(pdbid,file_format=\"pdb\",pdir=path)\n",
        "    input_pdb = path+\"pdb\"+pdbid+\".ent\"\n",
        "\n",
        "if pdbid in [\"upload\",\"google drive\"]:\n",
        "    pdbid = \"input\"\n",
        "#@markdown We need to parse the PDB and remove all alternative conformations\n",
        "#Here we set up a parser for our PDB\n",
        "parser = PDBParser()\n",
        "io=PDBIO()\n",
        "structure = parser.get_structure('X', input_pdb)\n",
        "#And here we set the residue conformation we want to keep\n",
        "keepAltID = \"A\"\n",
        "\n",
        "class KeepOneConfOnly(Select):  # Inherit methods from Select class\n",
        "    def accept_atom(self, atom):\n",
        "        if (not atom.is_disordered()) or atom.get_altloc() == keepAltID:\n",
        "            atom.set_altloc(\" \")  # Eliminate alt location ID before output.\n",
        "            return True\n",
        "        else:  # Alt location was not one to be output.\n",
        "            return False\n",
        "        # end of accept_atom()\n",
        "\n",
        "#This will keep only conformation for each residue\n",
        "io.set_structure(structure)\n",
        "io.save(path+pdbid+\"_parsed.pdb\", select=KeepOneConfOnly())\n",
        "print(\"Your PDB was processed. Alternative side chain conformations removed!\")\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pQ-FVxQo_sDI",
        "outputId": "64a7d564-8418-4e7b-dc2d-20c67b9efe0c"
      },
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloading PDB structure '1aki'...\n",
            "Your PDB was processed. Alternative side chain conformations removed!\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Run pdbfixer\n",
        "\n",
        "def prepare_protein(pdb_file, ignore_missing_residues=False, ignore_terminal_missing_residues=True):\n",
        "    \"\"\"Use pdbfixer to prepare the protein from a PDB file. Hetero atoms such as ligands are \n",
        "    removed and non-standard residues replaced. Missing atoms to existing residues are added. \n",
        "    Missing residues are ignored by default, but can be included.\n",
        "    \n",
        "    Parameters\n",
        "    ----------\n",
        "    pdb_file: pathlib.Path or str\n",
        "        PDB file containing the system to simulate.\n",
        "    ignore_missing_residues: bool\n",
        "        If missing residues should be ignored or built.\n",
        "    ignore_terminal_missing_residues: bool\n",
        "        If missing residues at the beginning and the end of a chain should be ignored or built.\n",
        "    \n",
        "    Returns\n",
        "    -------\n",
        "    fixer: pdbfixer.pdbfixer.PDBFixer\n",
        "        Prepared protein system.\n",
        "    \"\"\"\n",
        "    fixer = pdbfixer.PDBFixer(str(pdb_file))\n",
        "    fixer.removeHeterogens()  # co-crystallized ligands are unknown to PDBFixer\n",
        "    fixer.findMissingResidues()  # identify missing residues, needed for identification of missing atoms\n",
        "    \n",
        "    # if missing terminal residues shall be ignored, remove them from the dictionary\n",
        "    if ignore_terminal_missing_residues:  \n",
        "        chains = list(fixer.topology.chains())\n",
        "        keys = fixer.missingResidues.keys()\n",
        "        for key in list(keys):\n",
        "            chain = chains[key[0]]\n",
        "            if key[1] == 0 or key[1] == len(list(chain.residues())):\n",
        "                del fixer.missingResidues[key]\n",
        "                \n",
        "    # if all missing residues shall be ignored ignored, clear the dictionary\n",
        "    if ignore_missing_residues:  \n",
        "        fixer.missingResidues = {}\n",
        "        \n",
        "    fixer.findNonstandardResidues()  # find non-standard residue\n",
        "    fixer.replaceNonstandardResidues()  # replace non-standard residues with standard one\n",
        "    fixer.findMissingAtoms()  # find missing heavy atoms\n",
        "    fixer.addMissingAtoms()  # add missing atoms\n",
        "    PDBFile.writeFile(fixer.topology, fixer.positions, open(path+pdbid+\"_processed.pdb\", 'w'))\n",
        "\n",
        "if missing_residues_soft == \"pdbfixer\":\n",
        "    prepare_protein(path+pdbid+\"_parsed.pdb\")\n",
        "\n",
        "u = mda.Universe(path+pdbid+\"_processed.pdb\")\n",
        "u.select_atoms(\"not resname HOH\").write(path+pdbid+\"_no_water.pdb\")\n"
      ],
      "metadata": {
        "id": "OVMgDV4OJM_L"
      },
      "execution_count": 26,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Display structure"
      ],
      "metadata": {
        "id": "vKOCRWjtJ70X"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "pdb_file = path+pdbid+\"_processed.pdb\"\n",
        "\n",
        "def show_pdb(show_sidechains=False, show_mainchains=False):\n",
        "  model_name = \"Parsed structure\"\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "  view.addModel(open(pdb_file,'r').read(),'pdb')\n",
        "\n",
        "  view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  \n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})  \n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "  view.zoomTo()\n",
        "  return view"
      ],
      "metadata": {
        "id": "9T3rn1_OJa2l"
      },
      "execution_count": 22,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "show_pdb()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 515
        },
        "id": "GoXENrlQKrRr",
        "outputId": "28a69910-8ef9-4dee-abcb-eb0085d1fd78"
      },
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16678367835488749\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16678367835488749\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16678367835488749 = null;\nvar warn = document.getElementById(\"3dmolwarning_16678367835488749\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16678367835488749 = $3Dmol.createViewer($(\"#3dmolviewer_16678367835488749\"),{backgroundColor:\"white\"});\nviewer_16678367835488749.zoomTo();\n\tviewer_16678367835488749.addModel(\"REMARK   1 CREATED WITH OPENMM 7.7, 2022-11-07\\nATOM      1  N   LYS A   1      35.365  22.342 -11.980  1.00  0.00           N  \\nATOM      2  CA  LYS A   1      35.892  21.073 -11.427  1.00  0.00           C  \\nATOM      3  C   LYS A   1      34.741  20.264 -10.844  1.00  0.00           C  \\nATOM      4  O   LYS A   1      33.945  20.813 -10.081  1.00  0.00           O  \\nATOM      5  CB  LYS A   1      36.872  21.435 -10.306  1.00  0.00           C  \\nATOM      6  CG  LYS A   1      37.453  20.248  -9.565  1.00  0.00           C  \\nATOM      7  CD  LYS A   1      38.688  20.649  -8.775  1.00  0.00           C  \\nATOM      8  CE  LYS A   1      39.057  19.508  -7.837  1.00  0.00           C  \\nATOM      9  NZ  LYS A   1      40.423  19.771  -7.299  1.00  0.00           N  \\nATOM     10  N   VAL A   2      34.739  18.961 -11.042  1.00  0.00           N  \\nATOM     11  CA  VAL A   2      33.903  17.998 -10.333  1.00  0.00           C  \\nATOM     12  C   VAL A   2      34.800  17.312  -9.294  1.00  0.00           C  \\nATOM     13  O   VAL A   2      35.759  16.605  -9.665  1.00  0.00           O  \\nATOM     14  CB  VAL A   2      33.140  17.034 -11.232  1.00  0.00           C  \\nATOM     15  CG1 VAL A   2      32.251  16.084 -10.434  1.00  0.00           C  \\nATOM     16  CG2 VAL A   2      32.294  17.714 -12.290  1.00  0.00           C  \\nATOM     17  N   PHE A   3      34.491  17.546  -8.038  1.00  0.00           N  \\nATOM     18  CA  PHE A   3      35.185  16.903  -6.918  1.00  0.00           C  \\nATOM     19  C   PHE A   3      34.742  15.441  -6.771  1.00  0.00           C  \\nATOM     20  O   PHE A   3      33.525  15.162  -6.862  1.00  0.00           O  \\nATOM     21  CB  PHE A   3      34.967  17.632  -5.594  1.00  0.00           C  \\nATOM     22  CG  PHE A   3      35.944  18.737  -5.375  1.00  0.00           C  \\nATOM     23  CD1 PHE A   3      35.666  20.050  -5.798  1.00  0.00           C  \\nATOM     24  CD2 PHE A   3      37.000  18.557  -4.473  1.00  0.00           C  \\nATOM     25  CE1 PHE A   3      36.577  21.076  -5.568  1.00  0.00           C  \\nATOM     26  CE2 PHE A   3      37.869  19.589  -4.157  1.00  0.00           C  \\nATOM     27  CZ  PHE A   3      37.636  20.873  -4.666  1.00  0.00           C  \\nATOM     28  N   GLY A   4      35.724  14.639  -6.331  1.00  0.00           N  \\nATOM     29  CA  GLY A   4      35.366  13.280  -5.870  1.00  0.00           C  \\nATOM     30  C   GLY A   4      34.924  13.420  -4.415  1.00  0.00           C  \\nATOM     31  O   GLY A   4      35.303  14.403  -3.781  1.00  0.00           O  \\nATOM     32  N   ARG A   5      34.053  12.538  -3.973  1.00  0.00           N  \\nATOM     33  CA  ARG A   5      33.565  12.538  -2.588  1.00  0.00           C  \\nATOM     34  C   ARG A   5      34.665  12.734  -1.556  1.00  0.00           C  \\nATOM     35  O   ARG A   5      34.669  13.651  -0.704  1.00  0.00           O  \\nATOM     36  CB  ARG A   5      32.765  11.262  -2.331  1.00  0.00           C  \\nATOM     37  CG  ARG A   5      32.213  11.203  -0.920  1.00  0.00           C  \\nATOM     38  CD  ARG A   5      31.375  10.001  -0.722  1.00  0.00           C  \\nATOM     39  NE  ARG A   5      32.059   8.749  -0.958  1.00  0.00           N  \\nATOM     40  CZ  ARG A   5      32.733   8.011  -0.097  1.00  0.00           C  \\nATOM     41  NH1 ARG A   5      32.836   8.332   1.187  1.00  0.00           N  \\nATOM     42  NH2 ARG A   5      33.245   6.836  -0.526  1.00  0.00           N  \\nATOM     43  N   CYS A   6      35.674  11.853  -1.612  1.00  0.00           N  \\nATOM     44  CA  CYS A   6      36.781  11.870  -0.654  1.00  0.00           C  \\nATOM     45  C   CYS A   6      37.747  13.050  -0.777  1.00  0.00           C  \\nATOM     46  O   CYS A   6      38.148  13.609   0.264  1.00  0.00           O  \\nATOM     47  CB  CYS A   6      37.491  10.532  -0.621  1.00  0.00           C  \\nATOM     48  SG  CYS A   6      36.540   9.205   0.140  1.00  0.00           S  \\nATOM     49  N   GLU A   7      37.861  13.481  -2.019  1.00  0.00           N  \\nATOM     50  CA  GLU A   7      38.685  14.686  -2.311  1.00  0.00           C  \\nATOM     51  C   GLU A   7      38.049  15.926  -1.658  1.00  0.00           C  \\nATOM     52  O   GLU A   7      38.744  16.729  -1.011  1.00  0.00           O  \\nATOM     53  CB  GLU A   7      38.784  14.846  -3.818  1.00  0.00           C  \\nATOM     54  CG  GLU A   7      39.540  16.051  -4.379  1.00  0.00           C  \\nATOM     55  CD  GLU A   7      39.576  16.242  -5.870  1.00  0.00           C  \\nATOM     56  OE1 GLU A   7      38.672  15.644  -6.491  1.00  0.00           O  \\nATOM     57  OE2 GLU A   7      40.415  16.953  -6.381  1.00  0.00           O  \\nATOM     58  N   LEU A   8      36.743  16.049  -1.819  1.00  0.00           N  \\nATOM     59  CA  LEU A   8      35.964  17.158  -1.255  1.00  0.00           C  \\nATOM     60  C   LEU A   8      36.051  17.132   0.266  1.00  0.00           C  \\nATOM     61  O   LEU A   8      36.159  18.166   0.920  1.00  0.00           O  \\nATOM     62  CB  LEU A   8      34.528  17.172  -1.811  1.00  0.00           C  \\nATOM     63  CG  LEU A   8      33.718  18.354  -1.305  1.00  0.00           C  \\nATOM     64  CD1 LEU A   8      34.297  19.656  -1.841  1.00  0.00           C  \\nATOM     65  CD2 LEU A   8      32.246  18.143  -1.596  1.00  0.00           C  \\nATOM     66  N   ALA A   9      35.754  15.980   0.828  1.00  0.00           N  \\nATOM     67  CA  ALA A   9      35.838  15.757   2.284  1.00  0.00           C  \\nATOM     68  C   ALA A   9      37.144  16.314   2.840  1.00  0.00           C  \\nATOM     69  O   ALA A   9      37.151  16.978   3.897  1.00  0.00           O  \\nATOM     70  CB  ALA A   9      35.656  14.287   2.623  1.00  0.00           C  \\nATOM     71  N   ALA A  10      38.272  15.983   2.204  1.00  0.00           N  \\nATOM     72  CA  ALA A  10      39.610  16.431   2.616  1.00  0.00           C  \\nATOM     73  C   ALA A  10      39.736  17.944   2.459  1.00  0.00           C  \\nATOM     74  O   ALA A  10      40.193  18.499   3.469  1.00  0.00           O  \\nATOM     75  CB  ALA A  10      40.708  15.706   1.842  1.00  0.00           C  \\nATOM     76  N   ALA A  11      39.227  18.519   1.385  1.00  0.00           N  \\nATOM     77  CA  ALA A  11      39.264  19.982   1.223  1.00  0.00           C  \\nATOM     78  C   ALA A  11      38.491  20.702   2.321  1.00  0.00           C  \\nATOM     79  O   ALA A  11      38.946  21.658   2.953  1.00  0.00           O  \\nATOM     80  CB  ALA A  11      38.869  20.421  -0.175  1.00  0.00           C  \\nATOM     81  N   MET A  12      37.288  20.214   2.590  1.00  0.00           N  \\nATOM     82  CA  MET A  12      36.398  20.781   3.612  1.00  0.00           C  \\nATOM     83  C   MET A  12      36.990  20.715   5.007  1.00  0.00           C  \\nATOM     84  O   MET A  12      36.906  21.637   5.840  1.00  0.00           O  \\nATOM     85  CB  MET A  12      34.993  20.213   3.515  1.00  0.00           C  \\nATOM     86  CG  MET A  12      34.320  20.724   2.265  1.00  0.00           C  \\nATOM     87  SD  MET A  12      32.634  19.986   2.235  1.00  0.00           S  \\nATOM     88  CE  MET A  12      31.788  21.135   1.138  1.00  0.00           C  \\nATOM     89  N   LYS A  13      37.688  19.628   5.314  1.00  0.00           N  \\nATOM     90  CA  LYS A  13      38.387  19.385   6.579  1.00  0.00           C  \\nATOM     91  C   LYS A  13      39.460  20.467   6.731  1.00  0.00           C  \\nATOM     92  O   LYS A  13      39.507  21.137   7.776  1.00  0.00           O  \\nATOM     93  CB  LYS A  13      38.934  17.952   6.572  1.00  0.00           C  \\nATOM     94  CG  LYS A  13      39.742  17.555   7.798  1.00  0.00           C  \\nATOM     95  CD  LYS A  13      38.973  16.777   8.834  1.00  0.00           C  \\nATOM     96  CE  LYS A  13      39.293  15.305   8.751  1.00  0.00           C  \\nATOM     97  NZ  LYS A  13      38.077  14.461   8.946  1.00  0.00           N  \\nATOM     98  N   ARG A  14      40.267  20.629   5.688  1.00  0.00           N  \\nATOM     99  CA  ARG A  14      41.387  21.577   5.713  1.00  0.00           C  \\nATOM    100  C   ARG A  14      40.927  23.017   5.881  1.00  0.00           C  \\nATOM    101  O   ARG A  14      41.557  23.834   6.584  1.00  0.00           O  \\nATOM    102  CB  ARG A  14      42.388  21.351   4.601  1.00  0.00           C  \\nATOM    103  CG  ARG A  14      42.173  22.079   3.289  1.00  0.00           C  \\nATOM    104  CD  ARG A  14      43.444  22.075   2.490  1.00  0.00           C  \\nATOM    105  NE  ARG A  14      43.687  20.710   2.012  1.00  0.00           N  \\nATOM    106  CZ  ARG A  14      43.098  20.255   0.892  1.00  0.00           C  \\nATOM    107  NH1 ARG A  14      42.695  21.186   0.018  1.00  0.00           N  \\nATOM    108  NH2 ARG A  14      42.949  18.957   0.689  1.00  0.00           N  \\nATOM    109  N   HIS A  15      39.681  23.247   5.463  1.00  0.00           N  \\nATOM    110  CA  HIS A  15      39.075  24.591   5.526  1.00  0.00           C  \\nATOM    111  C   HIS A  15      38.310  24.861   6.814  1.00  0.00           C  \\nATOM    112  O   HIS A  15      37.608  25.875   6.952  1.00  0.00           O  \\nATOM    113  CB  HIS A  15      38.223  24.918   4.285  1.00  0.00           C  \\nATOM    114  CG  HIS A  15      39.085  25.388   3.171  1.00  0.00           C  \\nATOM    115  ND1 HIS A  15      39.457  24.635   2.091  1.00  0.00           N  \\nATOM    116  CD2 HIS A  15      39.739  26.570   2.993  1.00  0.00           C  \\nATOM    117  CE1 HIS A  15      40.211  25.312   1.258  1.00  0.00           C  \\nATOM    118  NE2 HIS A  15      40.524  26.419   1.889  1.00  0.00           N  \\nATOM    119  N   GLY A  16      38.296  23.927   7.720  1.00  0.00           N  \\nATOM    120  CA  GLY A  16      37.765  24.025   9.072  1.00  0.00           C  \\nATOM    121  C   GLY A  16      36.264  23.799   9.210  1.00  0.00           C  \\nATOM    122  O   GLY A  16      35.646  24.400  10.127  1.00  0.00           O  \\nATOM    123  N   LEU A  17      35.665  23.073   8.274  1.00  0.00           N  \\nATOM    124  CA  LEU A  17      34.238  22.795   8.298  1.00  0.00           C  \\nATOM    125  C   LEU A  17      33.845  21.682   9.266  1.00  0.00           C  \\nATOM    126  O   LEU A  17      32.643  21.594   9.552  1.00  0.00           O  \\nATOM    127  CB  LEU A  17      33.648  22.647   6.901  1.00  0.00           C  \\nATOM    128  CG  LEU A  17      33.451  23.889   6.060  1.00  0.00           C  \\nATOM    129  CD1 LEU A  17      32.933  23.420   4.705  1.00  0.00           C  \\nATOM    130  CD2 LEU A  17      32.556  24.946   6.679  1.00  0.00           C  \\nATOM    131  N   ASP A  18      34.785  20.814   9.605  1.00  0.00           N  \\nATOM    132  CA  ASP A  18      34.492  19.722  10.526  1.00  0.00           C  \\nATOM    133  C   ASP A  18      34.015  20.222  11.901  1.00  0.00           C  \\nATOM    134  O   ASP A  18      34.826  20.778  12.658  1.00  0.00           O  \\nATOM    135  CB  ASP A  18      35.557  18.633  10.598  1.00  0.00           C  \\nATOM    136  CG  ASP A  18      35.017  17.408  11.311  1.00  0.00           C  \\nATOM    137  OD1 ASP A  18      33.805  17.154  11.455  1.00  0.00           O  \\nATOM    138  OD2 ASP A  18      35.925  16.603  11.662  1.00  0.00           O  \\nATOM    139  N   ASN A  19      32.746  19.990  12.205  1.00  0.00           N  \\nATOM    140  CA  ASN A  19      32.123  20.431  13.448  1.00  0.00           C  \\nATOM    141  C   ASN A  19      31.854  21.941  13.497  1.00  0.00           C  \\nATOM    142  O   ASN A  19      31.426  22.398  14.573  1.00  0.00           O  \\nATOM    143  CB  ASN A  19      32.767  20.004  14.770  1.00  0.00           C  \\nATOM    144  CG  ASN A  19      32.162  20.512  16.064  1.00  0.00           C  \\nATOM    145  OD1 ASN A  19      30.967  20.273  16.355  1.00  0.00           O  \\nATOM    146  ND2 ASN A  19      32.847  21.361  16.852  1.00  0.00           N  \\nATOM    147  N   TYR A  20      31.969  22.650  12.406  1.00  0.00           N  \\nATOM    148  CA  TYR A  20      31.707  24.099  12.411  1.00  0.00           C  \\nATOM    149  C   TYR A  20      30.231  24.343  12.759  1.00  0.00           C  \\nATOM    150  O   TYR A  20      29.288  23.874  12.118  1.00  0.00           O  \\nATOM    151  CB  TYR A  20      32.070  24.736  11.066  1.00  0.00           C  \\nATOM    152  CG  TYR A  20      32.061  26.250  11.144  1.00  0.00           C  \\nATOM    153  CD1 TYR A  20      33.141  26.886  11.760  1.00  0.00           C  \\nATOM    154  CD2 TYR A  20      30.979  27.004  10.691  1.00  0.00           C  \\nATOM    155  CE1 TYR A  20      33.206  28.277  11.794  1.00  0.00           C  \\nATOM    156  CE2 TYR A  20      31.018  28.399  10.779  1.00  0.00           C  \\nATOM    157  CZ  TYR A  20      32.102  29.017  11.383  1.00  0.00           C  \\nATOM    158  OH  TYR A  20      32.136  30.371  11.525  1.00  0.00           O  \\nATOM    159  N   ARG A  21      30.088  25.142  13.803  1.00  0.00           N  \\nATOM    160  CA  ARG A  21      28.774  25.553  14.319  1.00  0.00           C  \\nATOM    161  C   ARG A  21      27.964  24.312  14.623  1.00  0.00           C  \\nATOM    162  O   ARG A  21      26.733  24.333  14.606  1.00  0.00           O  \\nATOM    163  CB  ARG A  21      28.014  26.565  13.453  1.00  0.00           C  \\nATOM    164  CG  ARG A  21      28.507  27.995  13.535  1.00  0.00           C  \\nATOM    165  CD  ARG A  21      28.110  28.755  14.765  1.00  0.00           C  \\nATOM    166  NE  ARG A  21      29.319  29.321  15.324  1.00  0.00           N  \\nATOM    167  CZ  ARG A  21      30.215  30.234  14.978  1.00  0.00           C  \\nATOM    168  NH1 ARG A  21      31.501  29.856  14.846  1.00  0.00           N  \\nATOM    169  NH2 ARG A  21      29.938  31.437  14.495  1.00  0.00           N  \\nATOM    170  N   GLY A  22      28.689  23.250  14.998  1.00  0.00           N  \\nATOM    171  CA  GLY A  22      28.103  22.021  15.519  1.00  0.00           C  \\nATOM    172  C   GLY A  22      27.748  21.018  14.436  1.00  0.00           C  \\nATOM    173  O   GLY A  22      27.085  20.022  14.784  1.00  0.00           O  \\nATOM    174  N   TYR A  23      28.209  21.230  13.216  1.00  0.00           N  \\nATOM    175  CA  TYR A  23      27.887  20.318  12.111  1.00  0.00           C  \\nATOM    176  C   TYR A  23      29.124  19.533  11.628  1.00  0.00           C  \\nATOM    177  O   TYR A  23      30.045  20.191  11.139  1.00  0.00           O  \\nATOM    178  CB  TYR A  23      27.351  21.107  10.913  1.00  0.00           C  \\nATOM    179  CG  TYR A  23      26.001  21.745  11.127  1.00  0.00           C  \\nATOM    180  CD1 TYR A  23      24.846  20.962  11.139  1.00  0.00           C  \\nATOM    181  CD2 TYR A  23      25.897  23.096  11.458  1.00  0.00           C  \\nATOM    182  CE1 TYR A  23      23.600  21.518  11.422  1.00  0.00           C  \\nATOM    183  CE2 TYR A  23      24.647  23.673  11.726  1.00  0.00           C  \\nATOM    184  CZ  TYR A  23      23.518  22.881  11.701  1.00  0.00           C  \\nATOM    185  OH  TYR A  23      22.289  23.438  11.912  1.00  0.00           O  \\nATOM    186  N   SER A  24      29.029  18.223  11.810  1.00  0.00           N  \\nATOM    187  CA  SER A  24      30.143  17.347  11.414  1.00  0.00           C  \\nATOM    188  C   SER A  24      30.359  17.379   9.895  1.00  0.00           C  \\nATOM    189  O   SER A  24      29.442  17.687   9.139  1.00  0.00           O  \\nATOM    190  CB  SER A  24      29.922  15.934  11.907  1.00  0.00           C  \\nATOM    191  OG  SER A  24      28.799  15.336  11.308  1.00  0.00           O  \\nATOM    192  N   LEU A  25      31.593  17.028   9.540  1.00  0.00           N  \\nATOM    193  CA  LEU A  25      32.035  17.092   8.138  1.00  0.00           C  \\nATOM    194  C   LEU A  25      31.030  16.437   7.183  1.00  0.00           C  \\nATOM    195  O   LEU A  25      30.874  16.924   6.056  1.00  0.00           O  \\nATOM    196  CB  LEU A  25      33.410  16.409   8.084  1.00  0.00           C  \\nATOM    197  CG  LEU A  25      34.015  16.477   6.689  1.00  0.00           C  \\nATOM    198  CD1 LEU A  25      34.174  17.929   6.289  1.00  0.00           C  \\nATOM    199  CD2 LEU A  25      35.398  15.810   6.752  1.00  0.00           C  \\nATOM    200  N   GLY A  26      30.501  15.280   7.576  1.00  0.00           N  \\nATOM    201  CA  GLY A  26      29.539  14.561   6.756  1.00  0.00           C  \\nATOM    202  C   GLY A  26      28.338  15.378   6.277  1.00  0.00           C  \\nATOM    203  O   GLY A  26      27.886  15.250   5.120  1.00  0.00           O  \\nATOM    204  N   ASN A  27      27.905  16.281   7.161  1.00  0.00           N  \\nATOM    205  CA  ASN A  27      26.827  17.227   6.857  1.00  0.00           C  \\nATOM    206  C   ASN A  27      27.164  18.015   5.597  1.00  0.00           C  \\nATOM    207  O   ASN A  27      26.317  18.208   4.720  1.00  0.00           O  \\nATOM    208  CB  ASN A  27      26.484  18.126   8.054  1.00  0.00           C  \\nATOM    209  CG  ASN A  27      25.681  17.267   9.048  1.00  0.00           C  \\nATOM    210  OD1 ASN A  27      24.511  16.933   8.820  1.00  0.00           O  \\nATOM    211  ND2 ASN A  27      26.348  17.052  10.169  1.00  0.00           N  \\nATOM    212  N   TRP A  28      28.344  18.591   5.583  1.00  0.00           N  \\nATOM    213  CA  TRP A  28      28.831  19.475   4.513  1.00  0.00           C  \\nATOM    214  C   TRP A  28      28.940  18.741   3.183  1.00  0.00           C  \\nATOM    215  O   TRP A  28      28.742  19.321   2.090  1.00  0.00           O  \\nATOM    216  CB  TRP A  28      30.104  20.145   5.023  1.00  0.00           C  \\nATOM    217  CG  TRP A  28      29.941  20.978   6.251  1.00  0.00           C  \\nATOM    218  CD1 TRP A  28      30.176  20.624   7.546  1.00  0.00           C  \\nATOM    219  CD2 TRP A  28      29.319  22.284   6.287  1.00  0.00           C  \\nATOM    220  NE1 TRP A  28      29.924  21.690   8.365  1.00  0.00           N  \\nATOM    221  CE2 TRP A  28      29.337  22.679   7.641  1.00  0.00           C  \\nATOM    222  CE3 TRP A  28      28.894  23.168   5.295  1.00  0.00           C  \\nATOM    223  CZ2 TRP A  28      28.913  23.943   8.038  1.00  0.00           C  \\nATOM    224  CZ3 TRP A  28      28.398  24.404   5.682  1.00  0.00           C  \\nATOM    225  CH2 TRP A  28      28.431  24.766   7.025  1.00  0.00           C  \\nATOM    226  N   VAL A  29      29.572  17.543   3.261  1.00  0.00           N  \\nATOM    227  CA  VAL A  29      29.729  16.711   2.047  1.00  0.00           C  \\nATOM    228  C   VAL A  29      28.379  16.340   1.429  1.00  0.00           C  \\nATOM    229  O   VAL A  29      28.166  16.496   0.228  1.00  0.00           O  \\nATOM    230  CB  VAL A  29      30.649  15.492   2.359  1.00  0.00           C  \\nATOM    231  CG1 VAL A  29      30.782  14.596   1.136  1.00  0.00           C  \\nATOM    232  CG2 VAL A  29      32.010  16.050   2.772  1.00  0.00           C  \\nATOM    233  N   CYS A  30      27.501  15.906   2.299  1.00  0.00           N  \\nATOM    234  CA  CYS A  30      26.115  15.567   1.991  1.00  0.00           C  \\nATOM    235  C   CYS A  30      25.388  16.723   1.302  1.00  0.00           C  \\nATOM    236  O   CYS A  30      24.894  16.516   0.172  1.00  0.00           O  \\nATOM    237  CB  CYS A  30      25.343  15.046   3.172  1.00  0.00           C  \\nATOM    238  SG  CYS A  30      23.719  14.376   2.695  1.00  0.00           S  \\nATOM    239  N   ALA A  31      25.533  17.913   1.870  1.00  0.00           N  \\nATOM    240  CA  ALA A  31      24.949  19.130   1.305  1.00  0.00           C  \\nATOM    241  C   ALA A  31      25.487  19.354  -0.115  1.00  0.00           C  \\nATOM    242  O   ALA A  31      24.675  19.686  -0.998  1.00  0.00           O  \\nATOM    243  CB  ALA A  31      25.167  20.335   2.200  1.00  0.00           C  \\nATOM    244  N   ALA A  32      26.807  19.354  -0.286  1.00  0.00           N  \\nATOM    245  CA  ALA A  32      27.461  19.536  -1.579  1.00  0.00           C  \\nATOM    246  C   ALA A  32      26.943  18.538  -2.620  1.00  0.00           C  \\nATOM    247  O   ALA A  32      26.767  18.857  -3.789  1.00  0.00           O  \\nATOM    248  CB  ALA A  32      28.982  19.476  -1.398  1.00  0.00           C  \\nATOM    249  N   LYS A  33      26.731  17.303  -2.193  1.00  0.00           N  \\nATOM    250  CA  LYS A  33      26.261  16.216  -3.037  1.00  0.00           C  \\nATOM    251  C   LYS A  33      24.903  16.555  -3.658  1.00  0.00           C  \\nATOM    252  O   LYS A  33      24.806  16.511  -4.890  1.00  0.00           O  \\nATOM    253  CB  LYS A  33      26.221  14.860  -2.351  1.00  0.00           C  \\nATOM    254  CG  LYS A  33      25.697  13.696  -3.185  1.00  0.00           C  \\nATOM    255  CD  LYS A  33      26.498  13.400  -4.446  1.00  0.00           C  \\nATOM    256  CE  LYS A  33      25.686  12.464  -5.331  1.00  0.00           C  \\nATOM    257  NZ  LYS A  33      26.423  11.979  -6.525  1.00  0.00           N  \\nATOM    258  N   PHE A  34      23.972  16.946  -2.817  1.00  0.00           N  \\nATOM    259  CA  PHE A  34      22.569  17.125  -3.247  1.00  0.00           C  \\nATOM    260  C   PHE A  34      22.346  18.506  -3.836  1.00  0.00           C  \\nATOM    261  O   PHE A  34      21.504  18.692  -4.759  1.00  0.00           O  \\nATOM    262  CB  PHE A  34      21.626  16.673  -2.130  1.00  0.00           C  \\nATOM    263  CG  PHE A  34      21.644  15.172  -1.965  1.00  0.00           C  \\nATOM    264  CD1 PHE A  34      21.209  14.353  -3.007  1.00  0.00           C  \\nATOM    265  CD2 PHE A  34      22.272  14.627  -0.851  1.00  0.00           C  \\nATOM    266  CE1 PHE A  34      21.372  12.961  -2.910  1.00  0.00           C  \\nATOM    267  CE2 PHE A  34      22.443  13.245  -0.743  1.00  0.00           C  \\nATOM    268  CZ  PHE A  34      21.923  12.406  -1.737  1.00  0.00           C  \\nATOM    269  N   GLU A  35      23.251  19.415  -3.451  1.00  0.00           N  \\nATOM    270  CA  GLU A  35      23.178  20.789  -3.996  1.00  0.00           C  \\nATOM    271  C   GLU A  35      23.661  20.944  -5.423  1.00  0.00           C  \\nATOM    272  O   GLU A  35      23.014  21.514  -6.310  1.00  0.00           O  \\nATOM    273  CB  GLU A  35      23.698  21.892  -3.107  1.00  0.00           C  \\nATOM    274  CG  GLU A  35      22.994  22.212  -1.809  1.00  0.00           C  \\nATOM    275  CD  GLU A  35      21.631  22.846  -1.864  1.00  0.00           C  \\nATOM    276  OE1 GLU A  35      21.408  23.360  -2.981  1.00  0.00           O  \\nATOM    277  OE2 GLU A  35      20.947  23.032  -0.874  1.00  0.00           O  \\nATOM    278  N   SER A  36      24.867  20.483  -5.674  1.00  0.00           N  \\nATOM    279  CA  SER A  36      25.626  20.615  -6.903  1.00  0.00           C  \\nATOM    280  C   SER A  36      26.139  19.341  -7.569  1.00  0.00           C  \\nATOM    281  O   SER A  36      26.750  19.464  -8.642  1.00  0.00           O  \\nATOM    282  CB  SER A  36      26.830  21.528  -6.654  1.00  0.00           C  \\nATOM    283  OG  SER A  36      27.747  20.951  -5.748  1.00  0.00           O  \\nATOM    284  N   ASN A  37      25.957  18.222  -6.927  1.00  0.00           N  \\nATOM    285  CA  ASN A  37      26.628  16.968  -7.297  1.00  0.00           C  \\nATOM    286  C   ASN A  37      28.149  17.091  -7.302  1.00  0.00           C  \\nATOM    287  O   ASN A  37      28.813  16.627  -8.254  1.00  0.00           O  \\nATOM    288  CB  ASN A  37      26.028  16.490  -8.617  1.00  0.00           C  \\nATOM    289  CG  ASN A  37      26.156  14.980  -8.782  1.00  0.00           C  \\nATOM    290  OD1 ASN A  37      26.640  14.266  -7.885  1.00  0.00           O  \\nATOM    291  ND2 ASN A  37      25.867  14.506  -9.990  1.00  0.00           N  \\nATOM    292  N   PHE A  38      28.691  17.882  -6.383  1.00  0.00           N  \\nATOM    293  CA  PHE A  38      30.129  18.115  -6.279  1.00  0.00           C  \\nATOM    294  C   PHE A  38      30.717  18.903  -7.448  1.00  0.00           C  \\nATOM    295  O   PHE A  38      31.923  18.794  -7.702  1.00  0.00           O  \\nATOM    296  CB  PHE A  38      30.914  16.823  -6.047  1.00  0.00           C  \\nATOM    297  CG  PHE A  38      30.487  16.006  -4.863  1.00  0.00           C  \\nATOM    298  CD1 PHE A  38      30.100  16.572  -3.655  1.00  0.00           C  \\nATOM    299  CD2 PHE A  38      30.507  14.603  -4.993  1.00  0.00           C  \\nATOM    300  CE1 PHE A  38      29.766  15.808  -2.558  1.00  0.00           C  \\nATOM    301  CE2 PHE A  38      30.136  13.814  -3.891  1.00  0.00           C  \\nATOM    302  CZ  PHE A  38      29.835  14.410  -2.646  1.00  0.00           C  \\nATOM    303  N   ASN A  39      29.936  19.792  -8.033  1.00  0.00           N  \\nATOM    304  CA  ASN A  39      30.341  20.573  -9.199  1.00  0.00           C  \\nATOM    305  C   ASN A  39      30.543  22.034  -8.823  1.00  0.00           C  \\nATOM    306  O   ASN A  39      29.544  22.649  -8.423  1.00  0.00           O  \\nATOM    307  CB  ASN A  39      29.406  20.300 -10.366  1.00  0.00           C  \\nATOM    308  CG  ASN A  39      29.876  20.771 -11.716  1.00  0.00           C  \\nATOM    309  OD1 ASN A  39      30.579  21.750 -11.956  1.00  0.00           O  \\nATOM    310  ND2 ASN A  39      29.578  19.864 -12.653  1.00  0.00           N  \\nATOM    311  N   THR A  40      31.766  22.492  -8.928  1.00  0.00           N  \\nATOM    312  CA  THR A  40      32.139  23.869  -8.657  1.00  0.00           C  \\nATOM    313  C   THR A  40      31.504  24.890  -9.589  1.00  0.00           C  \\nATOM    314  O   THR A  40      31.316  26.054  -9.213  1.00  0.00           O  \\nATOM    315  CB  THR A  40      33.684  24.103  -8.422  1.00  0.00           C  \\nATOM    316  OG1 THR A  40      34.270  24.156  -9.775  1.00  0.00           O  \\nATOM    317  CG2 THR A  40      34.414  23.119  -7.491  1.00  0.00           C  \\nATOM    318  N   GLN A  41      31.001  24.430 -10.706  1.00  0.00           N  \\nATOM    319  CA  GLN A  41      30.524  25.291 -11.812  1.00  0.00           C  \\nATOM    320  C   GLN A  41      28.993  25.381 -11.874  1.00  0.00           C  \\nATOM    321  O   GLN A  41      28.504  26.019 -12.837  1.00  0.00           O  \\nATOM    322  CB  GLN A  41      31.047  24.817 -13.168  1.00  0.00           C  \\nATOM    323  CG  GLN A  41      32.549  25.004 -13.279  1.00  0.00           C  \\nATOM    324  CD  GLN A  41      32.763  26.236 -14.129  1.00  0.00           C  \\nATOM    325  OE1 GLN A  41      32.356  26.308 -15.291  1.00  0.00           O  \\nATOM    326  NE2 GLN A  41      33.276  27.231 -13.420  1.00  0.00           N  \\nATOM    327  N   ALA A  42      28.329  24.640 -11.012  1.00  0.00           N  \\nATOM    328  CA  ALA A  42      26.859  24.579 -11.061  1.00  0.00           C  \\nATOM    329  C   ALA A  42      26.257  25.969 -10.807  1.00  0.00           C  \\nATOM    330  O   ALA A  42      26.645  26.686  -9.884  1.00  0.00           O  \\nATOM    331  CB  ALA A  42      26.355  23.604  -9.998  1.00  0.00           C  \\nATOM    332  N   THR A  43      25.276  26.293 -11.643  1.00  0.00           N  \\nATOM    333  CA  THR A  43      24.441  27.490 -11.476  1.00  0.00           C  \\nATOM    334  C   THR A  43      22.976  27.112 -11.714  1.00  0.00           C  \\nATOM    335  O   THR A  43      22.715  26.271 -12.594  1.00  0.00           O  \\nATOM    336  CB  THR A  43      24.814  28.728 -12.375  1.00  0.00           C  \\nATOM    337  OG1 THR A  43      24.555  28.321 -13.756  1.00  0.00           O  \\nATOM    338  CG2 THR A  43      26.247  29.213 -12.184  1.00  0.00           C  \\nATOM    339  N   ASN A  44      22.088  27.742 -10.971  1.00  0.00           N  \\nATOM    340  CA  ASN A  44      20.640  27.497 -11.108  1.00  0.00           C  \\nATOM    341  C   ASN A  44      19.903  28.842 -10.963  1.00  0.00           C  \\nATOM    342  O   ASN A  44      20.169  29.600 -10.033  1.00  0.00           O  \\nATOM    343  CB  ASN A  44      20.155  26.480 -10.080  1.00  0.00           C  \\nATOM    344  CG  ASN A  44      18.646  26.315 -10.115  1.00  0.00           C  \\nATOM    345  OD1 ASN A  44      18.128  25.720 -11.078  1.00  0.00           O  \\nATOM    346  ND2 ASN A  44      17.906  26.927  -9.188  1.00  0.00           N  \\nATOM    347  N   ARG A  45      19.058  29.117 -11.924  1.00  0.00           N  \\nATOM    348  CA  ARG A  45      18.303  30.363 -12.015  1.00  0.00           C  \\nATOM    349  C   ARG A  45      16.955  30.163 -11.326  1.00  0.00           C  \\nATOM    350  O   ARG A  45      16.396  29.062 -11.425  1.00  0.00           O  \\nATOM    351  CB  ARG A  45      18.143  30.836 -13.462  1.00  0.00           C  \\nATOM    352  CG  ARG A  45      17.012  31.859 -13.557  1.00  0.00           C  \\nATOM    353  CD  ARG A  45      17.502  33.134 -12.930  1.00  0.00           C  \\nATOM    354  NE  ARG A  45      18.311  33.758 -13.981  1.00  0.00           N  \\nATOM    355  CZ  ARG A  45      17.620  34.271 -15.020  1.00  0.00           C  \\nATOM    356  NH1 ARG A  45      16.287  34.331 -15.098  1.00  0.00           N  \\nATOM    357  NH2 ARG A  45      18.374  34.698 -16.030  1.00  0.00           N  \\nATOM    358  N   ASN A  46      16.553  31.171 -10.554  1.00  0.00           N  \\nATOM    359  CA  ASN A  46      15.304  31.071  -9.782  1.00  0.00           C  \\nATOM    360  C   ASN A  46      14.261  32.063 -10.313  1.00  0.00           C  \\nATOM    361  O   ASN A  46      14.617  33.104 -10.880  1.00  0.00           O  \\nATOM    362  CB  ASN A  46      15.576  31.229  -8.295  1.00  0.00           C  \\nATOM    363  CG  ASN A  46      16.543  30.240  -7.679  1.00  0.00           C  \\nATOM    364  OD1 ASN A  46      17.659  30.661  -7.346  1.00  0.00           O  \\nATOM    365  ND2 ASN A  46      16.125  28.975  -7.600  1.00  0.00           N  \\nATOM    366  N   THR A  47      13.027  31.834  -9.887  1.00  0.00           N  \\nATOM    367  CA  THR A  47      11.871  32.654 -10.246  1.00  0.00           C  \\nATOM    368  C   THR A  47      12.001  34.107  -9.810  1.00  0.00           C  \\nATOM    369  O   THR A  47      11.600  34.969 -10.606  1.00  0.00           O  \\nATOM    370  CB  THR A  47      10.499  32.017  -9.789  1.00  0.00           C  \\nATOM    371  OG1 THR A  47      10.507  32.195  -8.342  1.00  0.00           O  \\nATOM    372  CG2 THR A  47      10.331  30.554 -10.188  1.00  0.00           C  \\nATOM    373  N   ASP A  48      12.625  34.377  -8.683  1.00  0.00           N  \\nATOM    374  CA  ASP A  48      12.885  35.716  -8.168  1.00  0.00           C  \\nATOM    375  C   ASP A  48      14.010  36.505  -8.823  1.00  0.00           C  \\nATOM    376  O   ASP A  48      14.246  37.621  -8.309  1.00  0.00           O  \\nATOM    377  CB  ASP A  48      13.023  35.735  -6.640  1.00  0.00           C  \\nATOM    378  CG  ASP A  48      14.367  35.174  -6.168  1.00  0.00           C  \\nATOM    379  OD1 ASP A  48      15.104  34.602  -6.991  1.00  0.00           O  \\nATOM    380  OD2 ASP A  48      14.750  35.442  -5.013  1.00  0.00           O  \\nATOM    381  N   GLY A  49      14.650  36.018  -9.862  1.00  0.00           N  \\nATOM    382  CA  GLY A  49      15.744  36.702 -10.546  1.00  0.00           C  \\nATOM    383  C   GLY A  49      17.147  36.315 -10.096  1.00  0.00           C  \\nATOM    384  O   GLY A  49      18.168  36.688 -10.694  1.00  0.00           O  \\nATOM    385  N   SER A  50      17.173  35.661  -8.930  1.00  0.00           N  \\nATOM    386  CA  SER A  50      18.435  35.225  -8.279  1.00  0.00           C  \\nATOM    387  C   SER A  50      18.977  33.963  -8.941  1.00  0.00           C  \\nATOM    388  O   SER A  50      18.273  33.255  -9.697  1.00  0.00           O  \\nATOM    389  CB  SER A  50      18.272  35.089  -6.781  1.00  0.00           C  \\nATOM    390  OG  SER A  50      17.530  33.930  -6.463  1.00  0.00           O  \\nATOM    391  N   THR A  51      20.271  33.766  -8.734  1.00  0.00           N  \\nATOM    392  CA  THR A  51      20.970  32.557  -9.202  1.00  0.00           C  \\nATOM    393  C   THR A  51      21.718  31.969  -8.000  1.00  0.00           C  \\nATOM    394  O   THR A  51      22.160  32.712  -7.119  1.00  0.00           O  \\nATOM    395  CB  THR A  51      21.904  32.908 -10.419  1.00  0.00           C  \\nATOM    396  OG1 THR A  51      21.099  33.576 -11.407  1.00  0.00           O  \\nATOM    397  CG2 THR A  51      22.686  31.699 -10.952  1.00  0.00           C  \\nATOM    398  N   ASP A  52      21.708  30.650  -7.927  1.00  0.00           N  \\nATOM    399  CA  ASP A  52      22.528  29.900  -6.959  1.00  0.00           C  \\nATOM    400  C   ASP A  52      23.843  29.488  -7.635  1.00  0.00           C  \\nATOM    401  O   ASP A  52      23.847  29.066  -8.805  1.00  0.00           O  \\nATOM    402  CB  ASP A  52      21.765  28.649  -6.554  1.00  0.00           C  \\nATOM    403  CG  ASP A  52      20.396  28.991  -6.002  1.00  0.00           C  \\nATOM    404  OD1 ASP A  52      20.220  29.928  -5.237  1.00  0.00           O  \\nATOM    405  OD2 ASP A  52      19.517  28.144  -6.222  1.00  0.00           O  \\nATOM    406  N   TYR A  53      24.924  29.734  -6.905  1.00  0.00           N  \\nATOM    407  CA  TYR A  53      26.278  29.604  -7.450  1.00  0.00           C  \\nATOM    408  C   TYR A  53      27.161  28.595  -6.708  1.00  0.00           C  \\nATOM    409  O   TYR A  53      27.289  28.606  -5.486  1.00  0.00           O  \\nATOM    410  CB  TYR A  53      26.993  30.972  -7.489  1.00  0.00           C  \\nATOM    411  CG  TYR A  53      26.437  31.959  -8.487  1.00  0.00           C  \\nATOM    412  CD1 TYR A  53      26.843  32.003  -9.821  1.00  0.00           C  \\nATOM    413  CD2 TYR A  53      25.510  32.907  -8.050  1.00  0.00           C  \\nATOM    414  CE1 TYR A  53      26.291  32.922 -10.717  1.00  0.00           C  \\nATOM    415  CE2 TYR A  53      24.907  33.803  -8.932  1.00  0.00           C  \\nATOM    416  CZ  TYR A  53      25.357  33.847 -10.252  1.00  0.00           C  \\nATOM    417  OH  TYR A  53      24.864  34.804 -11.094  1.00  0.00           O  \\nATOM    418  N   GLY A  54      27.751  27.721  -7.493  1.00  0.00           N  \\nATOM    419  CA  GLY A  54      28.845  26.832  -7.121  1.00  0.00           C  \\nATOM    420  C   GLY A  54      28.499  25.601  -6.290  1.00  0.00           C  \\nATOM    421  O   GLY A  54      27.339  25.155  -6.255  1.00  0.00           O  \\nATOM    422  N   ILE A  55      29.560  25.088  -5.684  1.00  0.00           N  \\nATOM    423  CA  ILE A  55      29.452  23.795  -4.994  1.00  0.00           C  \\nATOM    424  C   ILE A  55      28.424  23.748  -3.872  1.00  0.00           C  \\nATOM    425  O   ILE A  55      27.770  22.697  -3.693  1.00  0.00           O  \\nATOM    426  CB  ILE A  55      30.891  23.340  -4.563  1.00  0.00           C  \\nATOM    427  CG1 ILE A  55      30.785  21.812  -4.319  1.00  0.00           C  \\nATOM    428  CG2 ILE A  55      31.396  24.210  -3.390  1.00  0.00           C  \\nATOM    429  CD1 ILE A  55      32.101  21.018  -4.339  1.00  0.00           C  \\nATOM    430  N   LEU A  56      28.216  24.881  -3.236  1.00  0.00           N  \\nATOM    431  CA  LEU A  56      27.173  24.998  -2.203  1.00  0.00           C  \\nATOM    432  C   LEU A  56      25.981  25.838  -2.659  1.00  0.00           C  \\nATOM    433  O   LEU A  56      25.161  26.111  -1.774  1.00  0.00           O  \\nATOM    434  CB  LEU A  56      27.816  25.424  -0.877  1.00  0.00           C  \\nATOM    435  CG  LEU A  56      28.692  24.348  -0.204  1.00  0.00           C  \\nATOM    436  CD1 LEU A  56      29.331  24.997   1.008  1.00  0.00           C  \\nATOM    437  CD2 LEU A  56      27.808  23.177   0.192  1.00  0.00           C  \\nATOM    438  N   GLN A  57      25.865  26.104  -3.946  1.00  0.00           N  \\nATOM    439  CA  GLN A  57      24.668  26.785  -4.465  1.00  0.00           C  \\nATOM    440  C   GLN A  57      24.277  27.940  -3.558  1.00  0.00           C  \\nATOM    441  O   GLN A  57      23.116  28.031  -3.094  1.00  0.00           O  \\nATOM    442  CB  GLN A  57      23.521  25.765  -4.549  1.00  0.00           C  \\nATOM    443  CG  GLN A  57      23.733  24.790  -5.699  1.00  0.00           C  \\nATOM    444  CD  GLN A  57      23.684  25.440  -7.069  1.00  0.00           C  \\nATOM    445  OE1 GLN A  57      22.574  25.602  -7.591  1.00  0.00           O  \\nATOM    446  NE2 GLN A  57      24.813  25.805  -7.651  1.00  0.00           N  \\nATOM    447  N   ILE A  58      25.164  28.910  -3.428  1.00  0.00           N  \\nATOM    448  CA  ILE A  58      24.930  30.118  -2.649  1.00  0.00           C  \\nATOM    449  C   ILE A  58      24.204  31.163  -3.504  1.00  0.00           C  \\nATOM    450  O   ILE A  58      24.555  31.405  -4.667  1.00  0.00           O  \\nATOM    451  CB  ILE A  58      26.301  30.665  -2.134  1.00  0.00           C  \\nATOM    452  CG1 ILE A  58      26.751  29.724  -0.985  1.00  0.00           C  \\nATOM    453  CG2 ILE A  58      26.178  32.135  -1.693  1.00  0.00           C  \\nATOM    454  CD1 ILE A  58      28.246  29.954  -0.641  1.00  0.00           C  \\nATOM    455  N   ASN A  59      23.145  31.671  -2.905  1.00  0.00           N  \\nATOM    456  CA  ASN A  59      22.146  32.510  -3.590  1.00  0.00           C  \\nATOM    457  C   ASN A  59      22.550  33.974  -3.730  1.00  0.00           C  \\nATOM    458  O   ASN A  59      22.917  34.607  -2.740  1.00  0.00           O  \\nATOM    459  CB  ASN A  59      20.805  32.306  -2.884  1.00  0.00           C  \\nATOM    460  CG  ASN A  59      19.650  32.986  -3.588  1.00  0.00           C  \\nATOM    461  OD1 ASN A  59      19.464  34.171  -3.244  1.00  0.00           O  \\nATOM    462  ND2 ASN A  59      19.155  32.320  -4.612  1.00  0.00           N  \\nATOM    463  N   SER A  60      22.268  34.485  -4.941  1.00  0.00           N  \\nATOM    464  CA  SER A  60      22.570  35.910  -5.255  1.00  0.00           C  \\nATOM    465  C   SER A  60      21.687  36.993  -4.642  1.00  0.00           C  \\nATOM    466  O   SER A  60      22.054  38.177  -4.774  1.00  0.00           O  \\nATOM    467  CB  SER A  60      22.643  36.139  -6.750  1.00  0.00           C  \\nATOM    468  OG  SER A  60      21.432  36.007  -7.444  1.00  0.00           O  \\nATOM    469  N   ARG A  61      20.526  36.660  -4.138  1.00  0.00           N  \\nATOM    470  CA  ARG A  61      19.691  37.653  -3.434  1.00  0.00           C  \\nATOM    471  C   ARG A  61      20.277  38.108  -2.106  1.00  0.00           C  \\nATOM    472  O   ARG A  61      20.282  39.333  -1.876  1.00  0.00           O  \\nATOM    473  CB  ARG A  61      18.267  37.195  -3.250  1.00  0.00           C  \\nATOM    474  CG  ARG A  61      17.315  38.350  -2.959  1.00  0.00           C  \\nATOM    475  CD  ARG A  61      16.063  37.657  -2.503  1.00  0.00           C  \\nATOM    476  NE  ARG A  61      15.101  38.653  -2.063  1.00  0.00           N  \\nATOM    477  CZ  ARG A  61      13.794  38.351  -2.111  1.00  0.00           C  \\nATOM    478  NH1 ARG A  61      13.439  37.257  -2.784  1.00  0.00           N  \\nATOM    479  NH2 ARG A  61      12.925  39.249  -1.646  1.00  0.00           N  \\nATOM    480  N   TRP A  62      20.773  37.202  -1.273  1.00  0.00           N  \\nATOM    481  CA  TRP A  62      21.321  37.563   0.032  1.00  0.00           C  \\nATOM    482  C   TRP A  62      22.848  37.643   0.101  1.00  0.00           C  \\nATOM    483  O   TRP A  62      23.323  38.396   0.968  1.00  0.00           O  \\nATOM    484  CB  TRP A  62      20.833  36.611   1.134  1.00  0.00           C  \\nATOM    485  CG  TRP A  62      19.360  36.361   1.096  1.00  0.00           C  \\nATOM    486  CD1 TRP A  62      18.719  35.247   0.643  1.00  0.00           C  \\nATOM    487  CD2 TRP A  62      18.326  37.305   1.427  1.00  0.00           C  \\nATOM    488  NE1 TRP A  62      17.360  35.457   0.609  1.00  0.00           N  \\nATOM    489  CE2 TRP A  62      17.090  36.696   1.096  1.00  0.00           C  \\nATOM    490  CE3 TRP A  62      18.333  38.584   1.965  1.00  0.00           C  \\nATOM    491  CZ2 TRP A  62      15.875  37.327   1.307  1.00  0.00           C  \\nATOM    492  CZ3 TRP A  62      17.115  39.208   2.186  1.00  0.00           C  \\nATOM    493  CH2 TRP A  62      15.906  38.611   1.814  1.00  0.00           C  \\nATOM    494  N   TRP A  63      23.537  36.731  -0.584  1.00  0.00           N  \\nATOM    495  CA  TRP A  63      24.906  36.398  -0.276  1.00  0.00           C  \\nATOM    496  C   TRP A  63      26.089  36.924  -1.052  1.00  0.00           C  \\nATOM    497  O   TRP A  63      27.142  37.177  -0.432  1.00  0.00           O  \\nATOM    498  CB  TRP A  63      25.068  34.919   0.112  1.00  0.00           C  \\nATOM    499  CG  TRP A  63      24.036  34.428   1.068  1.00  0.00           C  \\nATOM    500  CD1 TRP A  63      22.959  33.620   0.777  1.00  0.00           C  \\nATOM    501  CD2 TRP A  63      23.919  34.728   2.450  1.00  0.00           C  \\nATOM    502  NE1 TRP A  63      22.243  33.344   1.899  1.00  0.00           N  \\nATOM    503  CE2 TRP A  63      22.761  34.067   2.931  1.00  0.00           C  \\nATOM    504  CE3 TRP A  63      24.694  35.480   3.323  1.00  0.00           C  \\nATOM    505  CZ2 TRP A  63      22.393  34.133   4.278  1.00  0.00           C  \\nATOM    506  CZ3 TRP A  63      24.302  35.586   4.648  1.00  0.00           C  \\nATOM    507  CH2 TRP A  63      23.179  34.910   5.117  1.00  0.00           C  \\nATOM    508  N   CYS A  64      25.962  36.914  -2.353  1.00  0.00           N  \\nATOM    509  CA  CYS A  64      26.985  37.433  -3.271  1.00  0.00           C  \\nATOM    510  C   CYS A  64      26.324  38.355  -4.286  1.00  0.00           C  \\nATOM    511  O   CYS A  64      25.102  38.314  -4.475  1.00  0.00           O  \\nATOM    512  CB  CYS A  64      27.638  36.265  -3.988  1.00  0.00           C  \\nATOM    513  SG  CYS A  64      26.562  35.233  -5.007  1.00  0.00           S  \\nATOM    514  N   ASN A  65      27.157  39.165  -4.908  1.00  0.00           N  \\nATOM    515  CA  ASN A  65      26.700  40.134  -5.920  1.00  0.00           C  \\nATOM    516  C   ASN A  65      26.985  39.646  -7.342  1.00  0.00           C  \\nATOM    517  O   ASN A  65      28.130  39.316  -7.647  1.00  0.00           O  \\nATOM    518  CB  ASN A  65      27.381  41.492  -5.712  1.00  0.00           C  \\nATOM    519  CG  ASN A  65      26.910  42.423  -6.824  1.00  0.00           C  \\nATOM    520  OD1 ASN A  65      25.736  42.559  -7.141  1.00  0.00           O  \\nATOM    521  ND2 ASN A  65      27.914  42.938  -7.527  1.00  0.00           N  \\nATOM    522  N   ASP A  66      25.920  39.484  -8.116  1.00  0.00           N  \\nATOM    523  CA  ASP A  66      26.102  39.180  -9.545  1.00  0.00           C  \\nATOM    524  C   ASP A  66      25.664  40.317 -10.460  1.00  0.00           C  \\nATOM    525  O   ASP A  66      25.719  40.143 -11.673  1.00  0.00           O  \\nATOM    526  CB  ASP A  66      25.462  37.858  -9.894  1.00  0.00           C  \\nATOM    527  CG  ASP A  66      23.951  37.903  -9.833  1.00  0.00           C  \\nATOM    528  OD1 ASP A  66      23.288  38.898  -9.542  1.00  0.00           O  \\nATOM    529  OD2 ASP A  66      23.455  36.769 -10.048  1.00  0.00           O  \\nATOM    530  N   GLY A  67      25.234  41.408  -9.860  1.00  0.00           N  \\nATOM    531  CA  GLY A  67      24.817  42.607 -10.577  1.00  0.00           C  \\nATOM    532  C   GLY A  67      23.544  42.553 -11.401  1.00  0.00           C  \\nATOM    533  O   GLY A  67      23.191  43.542 -12.055  1.00  0.00           O  \\nATOM    534  N   ARG A  68      22.822  41.441 -11.348  1.00  0.00           N  \\nATOM    535  CA  ARG A  68      21.560  41.308 -12.092  1.00  0.00           C  \\nATOM    536  C   ARG A  68      20.424  40.765 -11.243  1.00  0.00           C  \\nATOM    537  O   ARG A  68      19.385  40.383 -11.795  1.00  0.00           O  \\nATOM    538  CB  ARG A  68      21.746  40.454 -13.339  1.00  0.00           C  \\nATOM    539  CG  ARG A  68      22.197  39.048 -12.946  1.00  0.00           C  \\nATOM    540  CD  ARG A  68      22.477  38.187 -14.122  1.00  0.00           C  \\nATOM    541  NE  ARG A  68      21.439  37.201 -14.281  1.00  0.00           N  \\nATOM    542  CZ  ARG A  68      20.242  37.255 -14.855  1.00  0.00           C  \\nATOM    543  NH1 ARG A  68      19.503  38.311 -15.145  1.00  0.00           N  \\nATOM    544  NH2 ARG A  68      19.897  36.127 -15.497  1.00  0.00           N  \\nATOM    545  N   THR A  69      20.590  40.896  -9.952  1.00  0.00           N  \\nATOM    546  CA  THR A  69      19.607  40.408  -8.973  1.00  0.00           C  \\nATOM    547  C   THR A  69      18.994  41.597  -8.224  1.00  0.00           C  \\nATOM    548  O   THR A  69      19.683  42.241  -7.425  1.00  0.00           O  \\nATOM    549  CB  THR A  69      20.208  39.384  -7.910  1.00  0.00           C  \\nATOM    550  OG1 THR A  69      20.851  38.334  -8.715  1.00  0.00           O  \\nATOM    551  CG2 THR A  69      19.123  38.863  -6.955  1.00  0.00           C  \\nATOM    552  N   PRO A  70      17.687  41.741  -8.418  1.00  0.00           N  \\nATOM    553  CA  PRO A  70      16.953  42.824  -7.733  1.00  0.00           C  \\nATOM    554  C   PRO A  70      16.955  42.520  -6.242  1.00  0.00           C  \\nATOM    555  O   PRO A  70      16.739  41.356  -5.886  1.00  0.00           O  \\nATOM    556  CB  PRO A  70      15.557  42.749  -8.333  1.00  0.00           C  \\nATOM    557  CG  PRO A  70      15.759  42.092  -9.671  1.00  0.00           C  \\nATOM    558  CD  PRO A  70      16.887  41.095  -9.463  1.00  0.00           C  \\nATOM    559  N   GLY A  71      17.163  43.493  -5.401  1.00  0.00           N  \\nATOM    560  CA  GLY A  71      17.135  43.383  -3.947  1.00  0.00           C  \\nATOM    561  C   GLY A  71      18.328  42.710  -3.287  1.00  0.00           C  \\nATOM    562  O   GLY A  71      18.257  42.257  -2.126  1.00  0.00           O  \\nATOM    563  N   SER A  72      19.418  42.627  -4.018  1.00  0.00           N  \\nATOM    564  CA  SER A  72      20.665  41.976  -3.588  1.00  0.00           C  \\nATOM    565  C   SER A  72      21.139  42.582  -2.276  1.00  0.00           C  \\nATOM    566  O   SER A  72      21.055  43.819  -2.125  1.00  0.00           O  \\nATOM    567  CB  SER A  72      21.659  42.002  -4.737  1.00  0.00           C  \\nATOM    568  OG  SER A  72      23.015  41.835  -4.343  1.00  0.00           O  \\nATOM    569  N   ARG A  73      21.646  41.775  -1.369  1.00  0.00           N  \\nATOM    570  CA  ARG A  73      22.254  42.196  -0.113  1.00  0.00           C  \\nATOM    571  C   ARG A  73      23.774  42.058   0.029  1.00  0.00           C  \\nATOM    572  O   ARG A  73      24.425  42.749   0.849  1.00  0.00           O  \\nATOM    573  CB  ARG A  73      21.615  41.555   1.127  1.00  0.00           C  \\nATOM    574  CG  ARG A  73      20.187  41.983   1.439  1.00  0.00           C  \\nATOM    575  CD  ARG A  73      20.209  43.299   2.123  1.00  0.00           C  \\nATOM    576  NE  ARG A  73      18.928  43.771   2.617  1.00  0.00           N  \\nATOM    577  CZ  ARG A  73      17.980  44.302   1.823  1.00  0.00           C  \\nATOM    578  NH1 ARG A  73      17.841  43.934   0.544  1.00  0.00           N  \\nATOM    579  NH2 ARG A  73      17.571  45.545   2.146  1.00  0.00           N  \\nATOM    580  N   ASN A  74      24.346  41.143  -0.705  1.00  0.00           N  \\nATOM    581  CA  ASN A  74      25.780  40.833  -0.695  1.00  0.00           C  \\nATOM    582  C   ASN A  74      26.315  40.718   0.731  1.00  0.00           C  \\nATOM    583  O   ASN A  74      27.255  41.471   1.060  1.00  0.00           O  \\nATOM    584  CB  ASN A  74      26.565  41.786  -1.589  1.00  0.00           C  \\nATOM    585  CG  ASN A  74      27.982  41.328  -1.909  1.00  0.00           C  \\nATOM    586  OD1 ASN A  74      28.318  40.169  -1.652  1.00  0.00           O  \\nATOM    587  ND2 ASN A  74      28.838  42.192  -2.436  1.00  0.00           N  \\nATOM    588  N   LEU A  75      25.723  39.860   1.544  1.00  0.00           N  \\nATOM    589  CA  LEU A  75      26.153  39.690   2.930  1.00  0.00           C  \\nATOM    590  C   LEU A  75      27.518  39.011   3.085  1.00  0.00           C  \\nATOM    591  O   LEU A  75      28.167  39.197   4.141  1.00  0.00           O  \\nATOM    592  CB  LEU A  75      25.009  39.055   3.733  1.00  0.00           C  \\nATOM    593  CG  LEU A  75      23.815  39.998   3.979  1.00  0.00           C  \\nATOM    594  CD1 LEU A  75      22.574  39.193   4.336  1.00  0.00           C  \\nATOM    595  CD2 LEU A  75      24.156  40.969   5.110  1.00  0.00           C  \\nATOM    596  N   CYS A  76      27.973  38.312   2.061  1.00  0.00           N  \\nATOM    597  CA  CYS A  76      29.299  37.680   2.111  1.00  0.00           C  \\nATOM    598  C   CYS A  76      30.390  38.598   1.589  1.00  0.00           C  \\nATOM    599  O   CYS A  76      31.573  38.229   1.568  1.00  0.00           O  \\nATOM    600  CB  CYS A  76      29.396  36.303   1.477  1.00  0.00           C  \\nATOM    601  SG  CYS A  76      28.595  34.961   2.361  1.00  0.00           S  \\nATOM    602  N   ASN A  77      29.964  39.681   0.971  1.00  0.00           N  \\nATOM    603  CA  ASN A  77      30.827  40.725   0.395  1.00  0.00           C  \\nATOM    604  C   ASN A  77      31.711  40.163  -0.702  1.00  0.00           C  \\nATOM    605  O   ASN A  77      32.947  40.331  -0.689  1.00  0.00           O  \\nATOM    606  CB  ASN A  77      31.589  41.488   1.483  1.00  0.00           C  \\nATOM    607  CG  ASN A  77      32.126  42.812   0.957  1.00  0.00           C  \\nATOM    608  OD1 ASN A  77      31.351  43.589   0.396  1.00  0.00           O  \\nATOM    609  ND2 ASN A  77      33.430  43.012   1.047  1.00  0.00           N  \\nATOM    610  N   ILE A  78      31.149  39.408  -1.616  1.00  0.00           N  \\nATOM    611  CA  ILE A  78      31.904  38.780  -2.713  1.00  0.00           C  \\nATOM    612  C   ILE A  78      31.099  38.850  -4.013  1.00  0.00           C  \\nATOM    613  O   ILE A  78      29.864  38.688  -3.972  1.00  0.00           O  \\nATOM    614  CB  ILE A  78      32.183  37.255  -2.384  1.00  0.00           C  \\nATOM    615  CG1 ILE A  78      30.882  36.639  -1.851  1.00  0.00           C  \\nATOM    616  CG2 ILE A  78      33.437  37.003  -1.524  1.00  0.00           C  \\nATOM    617  CD1 ILE A  78      30.936  35.120  -1.546  1.00  0.00           C  \\nATOM    618  N   PRO A  79      31.828  38.795  -5.108  1.00  0.00           N  \\nATOM    619  CA  PRO A  79      31.195  38.634  -6.421  1.00  0.00           C  \\nATOM    620  C   PRO A  79      30.792  37.146  -6.446  1.00  0.00           C  \\nATOM    621  O   PRO A  79      31.576  36.273  -6.005  1.00  0.00           O  \\nATOM    622  CB  PRO A  79      32.261  38.967  -7.445  1.00  0.00           C  \\nATOM    623  CG  PRO A  79      33.555  39.029  -6.710  1.00  0.00           C  \\nATOM    624  CD  PRO A  79      33.276  39.023  -5.215  1.00  0.00           C  \\nATOM    625  N   CYS A  80      29.629  36.908  -7.033  1.00  0.00           N  \\nATOM    626  CA  CYS A  80      29.167  35.522  -7.236  1.00  0.00           C  \\nATOM    627  C   CYS A  80      30.150  34.716  -8.073  1.00  0.00           C  \\nATOM    628  O   CYS A  80      30.214  33.484  -7.880  1.00  0.00           O  \\nATOM    629  CB  CYS A  80      27.749  35.338  -7.747  1.00  0.00           C  \\nATOM    630  SG  CYS A  80      26.471  36.160  -6.762  1.00  0.00           S  \\nATOM    631  N   SER A  81      30.769  35.294  -9.083  1.00  0.00           N  \\nATOM    632  CA  SER A  81      31.775  34.671  -9.933  1.00  0.00           C  \\nATOM    633  C   SER A  81      32.907  34.027  -9.109  1.00  0.00           C  \\nATOM    634  O   SER A  81      33.338  32.939  -9.561  1.00  0.00           O  \\nATOM    635  CB  SER A  81      32.381  35.643 -10.953  1.00  0.00           C  \\nATOM    636  OG  SER A  81      33.035  36.681 -10.235  1.00  0.00           O  \\nATOM    637  N   ALA A  82      33.226  34.572  -7.950  1.00  0.00           N  \\nATOM    638  CA  ALA A  82      34.272  33.963  -7.107  1.00  0.00           C  \\nATOM    639  C   ALA A  82      33.869  32.604  -6.540  1.00  0.00           C  \\nATOM    640  O   ALA A  82      34.703  31.798  -6.081  1.00  0.00           O  \\nATOM    641  CB  ALA A  82      34.722  34.930  -6.020  1.00  0.00           C  \\nATOM    642  N   LEU A  83      32.571  32.338  -6.519  1.00  0.00           N  \\nATOM    643  CA  LEU A  83      31.973  31.093  -6.042  1.00  0.00           C  \\nATOM    644  C   LEU A  83      32.060  29.962  -7.049  1.00  0.00           C  \\nATOM    645  O   LEU A  83      31.671  28.815  -6.739  1.00  0.00           O  \\nATOM    646  CB  LEU A  83      30.618  31.424  -5.431  1.00  0.00           C  \\nATOM    647  CG  LEU A  83      30.511  32.364  -4.244  1.00  0.00           C  \\nATOM    648  CD1 LEU A  83      29.040  32.573  -3.857  1.00  0.00           C  \\nATOM    649  CD2 LEU A  83      31.277  31.872  -3.020  1.00  0.00           C  \\nATOM    650  N   LEU A  84      32.473  30.271  -8.267  1.00  0.00           N  \\nATOM    651  CA  LEU A  84      32.610  29.295  -9.347  1.00  0.00           C  \\nATOM    652  C   LEU A  84      34.047  28.841  -9.570  1.00  0.00           C  \\nATOM    653  O   LEU A  84      34.334  28.243 -10.622  1.00  0.00           O  \\nATOM    654  CB  LEU A  84      31.914  29.793 -10.614  1.00  0.00           C  \\nATOM    655  CG  LEU A  84      30.446  30.183 -10.573  1.00  0.00           C  \\nATOM    656  CD1 LEU A  84      30.014  30.771 -11.916  1.00  0.00           C  \\nATOM    657  CD2 LEU A  84      29.597  28.942 -10.329  1.00  0.00           C  \\nATOM    658  N   SER A  85      34.942  29.197  -8.676  1.00  0.00           N  \\nATOM    659  CA  SER A  85      36.340  28.754  -8.727  1.00  0.00           C  \\nATOM    660  C   SER A  85      36.474  27.245  -8.497  1.00  0.00           C  \\nATOM    661  O   SER A  85      35.765  26.640  -7.681  1.00  0.00           O  \\nATOM    662  CB  SER A  85      37.109  29.482  -7.633  1.00  0.00           C  \\nATOM    663  OG  SER A  85      38.484  29.212  -7.834  1.00  0.00           O  \\nATOM    664  N   SER A  86      37.619  26.742  -8.943  1.00  0.00           N  \\nATOM    665  CA  SER A  86      38.008  25.343  -8.700  1.00  0.00           C  \\nATOM    666  C   SER A  86      38.388  25.153  -7.233  1.00  0.00           C  \\nATOM    667  O   SER A  86      38.314  24.055  -6.671  1.00  0.00           O  \\nATOM    668  CB  SER A  86      39.107  24.868  -9.650  1.00  0.00           C  \\nATOM    669  OG  SER A  86      38.401  24.215 -10.691  1.00  0.00           O  \\nATOM    670  N   ASP A  87      38.846  26.244  -6.668  1.00  0.00           N  \\nATOM    671  CA  ASP A  87      39.282  26.393  -5.271  1.00  0.00           C  \\nATOM    672  C   ASP A  87      37.999  26.657  -4.471  1.00  0.00           C  \\nATOM    673  O   ASP A  87      37.490  27.762  -4.712  1.00  0.00           O  \\nATOM    674  CB  ASP A  87      40.221  27.607  -5.212  1.00  0.00           C  \\nATOM    675  CG  ASP A  87      40.762  28.041  -3.869  1.00  0.00           C  \\nATOM    676  OD1 ASP A  87      40.335  27.702  -2.742  1.00  0.00           O  \\nATOM    677  OD2 ASP A  87      41.785  28.802  -3.933  1.00  0.00           O  \\nATOM    678  N   ILE A  88      37.732  25.850  -3.461  1.00  0.00           N  \\nATOM    679  CA  ILE A  88      36.515  26.089  -2.658  1.00  0.00           C  \\nATOM    680  C   ILE A  88      36.582  27.142  -1.563  1.00  0.00           C  \\nATOM    681  O   ILE A  88      35.600  27.315  -0.801  1.00  0.00           O  \\nATOM    682  CB  ILE A  88      35.993  24.736  -2.046  1.00  0.00           C  \\nATOM    683  CG1 ILE A  88      36.920  24.200  -0.934  1.00  0.00           C  \\nATOM    684  CG2 ILE A  88      35.587  23.743  -3.163  1.00  0.00           C  \\nATOM    685  CD1 ILE A  88      36.363  23.226   0.137  1.00  0.00           C  \\nATOM    686  N   THR A  89      37.742  27.786  -1.369  1.00  0.00           N  \\nATOM    687  CA  THR A  89      37.913  28.772  -0.306  1.00  0.00           C  \\nATOM    688  C   THR A  89      36.736  29.761  -0.161  1.00  0.00           C  \\nATOM    689  O   THR A  89      36.341  29.983   1.000  1.00  0.00           O  \\nATOM    690  CB  THR A  89      39.267  29.591  -0.423  1.00  0.00           C  \\nATOM    691  OG1 THR A  89      40.339  28.592  -0.514  1.00  0.00           O  \\nATOM    692  CG2 THR A  89      39.419  30.553   0.766  1.00  0.00           C  \\nATOM    693  N   ALA A  90      36.507  30.482  -1.242  1.00  0.00           N  \\nATOM    694  CA  ALA A  90      35.479  31.555  -1.244  1.00  0.00           C  \\nATOM    695  C   ALA A  90      34.125  31.066  -0.735  1.00  0.00           C  \\nATOM    696  O   ALA A  90      33.366  31.671   0.065  1.00  0.00           O  \\nATOM    697  CB  ALA A  90      35.366  32.212  -2.617  1.00  0.00           C  \\nATOM    698  N   SER A  91      33.746  29.919  -1.296  1.00  0.00           N  \\nATOM    699  CA  SER A  91      32.494  29.220  -1.024  1.00  0.00           C  \\nATOM    700  C   SER A  91      32.429  28.817   0.467  1.00  0.00           C  \\nATOM    701  O   SER A  91      31.407  29.057   1.110  1.00  0.00           O  \\nATOM    702  CB  SER A  91      32.282  28.035  -1.917  1.00  0.00           C  \\nATOM    703  OG  SER A  91      32.020  28.335  -3.260  1.00  0.00           O  \\nATOM    704  N   VAL A  92      33.502  28.211   0.953  1.00  0.00           N  \\nATOM    705  CA  VAL A  92      33.595  27.846   2.381  1.00  0.00           C  \\nATOM    706  C   VAL A  92      33.477  29.105   3.261  1.00  0.00           C  \\nATOM    707  O   VAL A  92      32.716  29.020   4.244  1.00  0.00           O  \\nATOM    708  CB  VAL A  92      34.890  27.038   2.650  1.00  0.00           C  \\nATOM    709  CG1 VAL A  92      35.125  26.987   4.151  1.00  0.00           C  \\nATOM    710  CG2 VAL A  92      34.776  25.672   2.013  1.00  0.00           C  \\nATOM    711  N   ASN A  93      34.199  30.163   2.912  1.00  0.00           N  \\nATOM    712  CA  ASN A  93      34.160  31.413   3.685  1.00  0.00           C  \\nATOM    713  C   ASN A  93      32.747  32.034   3.803  1.00  0.00           C  \\nATOM    714  O   ASN A  93      32.361  32.393   4.908  1.00  0.00           O  \\nATOM    715  CB  ASN A  93      35.168  32.472   3.247  1.00  0.00           C  \\nATOM    716  CG  ASN A  93      36.582  32.083   3.642  1.00  0.00           C  \\nATOM    717  OD1 ASN A  93      37.546  32.671   3.120  1.00  0.00           O  \\nATOM    718  ND2 ASN A  93      36.710  31.188   4.631  1.00  0.00           N  \\nATOM    719  N   CYS A  94      32.096  31.996   2.670  1.00  0.00           N  \\nATOM    720  CA  CYS A  94      30.695  32.449   2.633  1.00  0.00           C  \\nATOM    721  C   CYS A  94      29.778  31.527   3.424  1.00  0.00           C  \\nATOM    722  O   CYS A  94      28.973  32.025   4.236  1.00  0.00           O  \\nATOM    723  CB  CYS A  94      30.291  32.704   1.185  1.00  0.00           C  \\nATOM    724  SG  CYS A  94      28.644  33.431   1.046  1.00  0.00           S  \\nATOM    725  N   ALA A  95      29.945  30.225   3.263  1.00  0.00           N  \\nATOM    726  CA  ALA A  95      29.134  29.227   3.969  1.00  0.00           C  \\nATOM    727  C   ALA A  95      29.236  29.330   5.491  1.00  0.00           C  \\nATOM    728  O   ALA A  95      28.222  29.180   6.179  1.00  0.00           O  \\nATOM    729  CB  ALA A  95      29.495  27.794   3.612  1.00  0.00           C  \\nATOM    730  N   LYS A  96      30.424  29.628   5.987  1.00  0.00           N  \\nATOM    731  CA  LYS A  96      30.627  29.912   7.419  1.00  0.00           C  \\nATOM    732  C   LYS A  96      29.802  31.115   7.906  1.00  0.00           C  \\nATOM    733  O   LYS A  96      29.296  31.011   9.027  1.00  0.00           O  \\nATOM    734  CB  LYS A  96      32.100  30.112   7.775  1.00  0.00           C  \\nATOM    735  CG  LYS A  96      32.874  28.792   7.697  1.00  0.00           C  \\nATOM    736  CD  LYS A  96      34.358  29.071   7.879  1.00  0.00           C  \\nATOM    737  CE  LYS A  96      35.205  27.816   7.938  1.00  0.00           C  \\nATOM    738  NZ  LYS A  96      36.610  28.242   8.169  1.00  0.00           N  \\nATOM    739  N   LYS A  97      29.660  32.101   7.049  1.00  0.00           N  \\nATOM    740  CA  LYS A  97      28.811  33.263   7.379  1.00  0.00           C  \\nATOM    741  C   LYS A  97      27.333  32.905   7.386  1.00  0.00           C  \\nATOM    742  O   LYS A  97      26.570  33.318   8.272  1.00  0.00           O  \\nATOM    743  CB  LYS A  97      29.087  34.456   6.470  1.00  0.00           C  \\nATOM    744  CG  LYS A  97      30.537  34.927   6.563  1.00  0.00           C  \\nATOM    745  CD  LYS A  97      30.841  36.073   5.610  1.00  0.00           C  \\nATOM    746  CE  LYS A  97      32.340  36.293   5.510  1.00  0.00           C  \\nATOM    747  NZ  LYS A  97      32.569  37.524   4.708  1.00  0.00           N  \\nATOM    748  N   ILE A  98      26.897  32.127   6.416  1.00  0.00           N  \\nATOM    749  CA  ILE A  98      25.520  31.693   6.266  1.00  0.00           C  \\nATOM    750  C   ILE A  98      25.030  30.871   7.453  1.00  0.00           C  \\nATOM    751  O   ILE A  98      24.059  31.254   8.126  1.00  0.00           O  \\nATOM    752  CB  ILE A  98      25.307  30.997   4.896  1.00  0.00           C  \\nATOM    753  CG1 ILE A  98      25.643  31.910   3.702  1.00  0.00           C  \\nATOM    754  CG2 ILE A  98      23.887  30.386   4.817  1.00  0.00           C  \\nATOM    755  CD1 ILE A  98      25.620  31.204   2.321  1.00  0.00           C  \\nATOM    756  N   VAL A  99      25.875  29.966   7.915  1.00  0.00           N  \\nATOM    757  CA  VAL A  99      25.488  29.009   8.955  1.00  0.00           C  \\nATOM    758  C   VAL A  99      25.431  29.697  10.316  1.00  0.00           C  \\nATOM    759  O   VAL A  99      24.737  29.212  11.233  1.00  0.00           O  \\nATOM    760  CB  VAL A  99      26.398  27.774   8.859  1.00  0.00           C  \\nATOM    761  CG1 VAL A  99      27.811  28.138   9.279  1.00  0.00           C  \\nATOM    762  CG2 VAL A  99      25.834  26.594   9.632  1.00  0.00           C  \\nATOM    763  N   SER A 100      26.205  30.752  10.409  1.00  0.00           N  \\nATOM    764  CA  SER A 100      26.297  31.529  11.650  1.00  0.00           C  \\nATOM    765  C   SER A 100      25.124  32.495  11.765  1.00  0.00           C  \\nATOM    766  O   SER A 100      24.995  33.131  12.820  1.00  0.00           O  \\nATOM    767  CB  SER A 100      27.647  32.194  11.723  1.00  0.00           C  \\nATOM    768  OG  SER A 100      28.714  31.264  11.818  1.00  0.00           O  \\nATOM    769  N   ASP A 101      24.307  32.599  10.750  1.00  0.00           N  \\nATOM    770  CA  ASP A 101      23.162  33.495  10.650  1.00  0.00           C  \\nATOM    771  C   ASP A 101      21.924  33.183  11.481  1.00  0.00           C  \\nATOM    772  O   ASP A 101      21.132  34.143  11.678  1.00  0.00           O  \\nATOM    773  CB  ASP A 101      22.806  33.854   9.207  1.00  0.00           C  \\nATOM    774  CG  ASP A 101      22.426  35.320   9.009  1.00  0.00           C  \\nATOM    775  OD1 ASP A 101      23.248  36.223   9.266  1.00  0.00           O  \\nATOM    776  OD2 ASP A 101      21.276  35.519   8.551  1.00  0.00           O  \\nATOM    777  N   GLY A 102      21.723  31.942  11.887  1.00  0.00           N  \\nATOM    778  CA  GLY A 102      20.622  31.656  12.823  1.00  0.00           C  \\nATOM    779  C   GLY A 102      19.812  30.413  12.505  1.00  0.00           C  \\nATOM    780  O   GLY A 102      19.195  29.898  13.458  1.00  0.00           O  \\nATOM    781  N   ASN A 103      19.805  30.005  11.244  1.00  0.00           N  \\nATOM    782  CA  ASN A 103      19.076  28.799  10.848  1.00  0.00           C  \\nATOM    783  C   ASN A 103      19.990  27.600  10.597  1.00  0.00           C  \\nATOM    784  O   ASN A 103      19.472  26.579  10.117  1.00  0.00           O  \\nATOM    785  CB  ASN A 103      17.976  29.046   9.824  1.00  0.00           C  \\nATOM    786  CG  ASN A 103      16.708  28.245  10.063  1.00  0.00           C  \\nATOM    787  OD1 ASN A 103      16.513  27.577  11.098  1.00  0.00           O  \\nATOM    788  ND2 ASN A 103      15.672  28.428   9.240  1.00  0.00           N  \\nATOM    789  N   GLY A 104      21.240  27.711  10.997  1.00  0.00           N  \\nATOM    790  CA  GLY A 104      22.177  26.567  10.860  1.00  0.00           C  \\nATOM    791  C   GLY A 104      22.173  26.142   9.388  1.00  0.00           C  \\nATOM    792  O   GLY A 104      22.151  27.040   8.541  1.00  0.00           O  \\nATOM    793  N   MET A 105      22.185  24.847   9.115  1.00  0.00           N  \\nATOM    794  CA  MET A 105      22.278  24.370   7.724  1.00  0.00           C  \\nATOM    795  C   MET A 105      20.915  24.309   7.063  1.00  0.00           C  \\nATOM    796  O   MET A 105      20.902  23.921   5.871  1.00  0.00           O  \\nATOM    797  CB  MET A 105      23.036  23.048   7.632  1.00  0.00           C  \\nATOM    798  CG  MET A 105      24.519  23.221   7.871  1.00  0.00           C  \\nATOM    799  SD  MET A 105      25.379  21.660   7.491  1.00  0.00           S  \\nATOM    800  CE  MET A 105      25.369  21.706   5.703  1.00  0.00           C  \\nATOM    801  N   ASN A 106      19.896  24.839   7.735  1.00  0.00           N  \\nATOM    802  CA  ASN A 106      18.557  24.898   7.103  1.00  0.00           C  \\nATOM    803  C   ASN A 106      18.566  25.831   5.892  1.00  0.00           C  \\nATOM    804  O   ASN A 106      17.678  25.748   5.021  1.00  0.00           O  \\nATOM    805  CB  ASN A 106      17.420  25.150   8.084  1.00  0.00           C  \\nATOM    806  CG  ASN A 106      17.207  23.938   8.967  1.00  0.00           C  \\nATOM    807  OD1 ASN A 106      16.836  22.867   8.443  1.00  0.00           O  \\nATOM    808  ND2 ASN A 106      17.542  24.047  10.255  1.00  0.00           N  \\nATOM    809  N   ALA A 107      19.617  26.647   5.800  1.00  0.00           N  \\nATOM    810  CA  ALA A 107      19.824  27.514   4.634  1.00  0.00           C  \\nATOM    811  C   ALA A 107      20.007  26.744   3.329  1.00  0.00           C  \\nATOM    812  O   ALA A 107      19.734  27.292   2.260  1.00  0.00           O  \\nATOM    813  CB  ALA A 107      21.060  28.374   4.877  1.00  0.00           C  \\nATOM    814  N   TRP A 108      20.371  25.465   3.434  1.00  0.00           N  \\nATOM    815  CA  TRP A 108      20.532  24.624   2.241  1.00  0.00           C  \\nATOM    816  C   TRP A 108      19.317  23.688   2.110  1.00  0.00           C  \\nATOM    817  O   TRP A 108      19.240  22.816   2.993  1.00  0.00           O  \\nATOM    818  CB  TRP A 108      21.840  23.826   2.338  1.00  0.00           C  \\nATOM    819  CG  TRP A 108      23.000  24.736   2.026  1.00  0.00           C  \\nATOM    820  CD1 TRP A 108      23.360  25.234   0.803  1.00  0.00           C  \\nATOM    821  CD2 TRP A 108      23.798  25.437   2.991  1.00  0.00           C  \\nATOM    822  NE1 TRP A 108      24.414  26.102   0.952  1.00  0.00           N  \\nATOM    823  CE2 TRP A 108      24.711  26.237   2.272  1.00  0.00           C  \\nATOM    824  CE3 TRP A 108      23.833  25.426   4.379  1.00  0.00           C  \\nATOM    825  CZ2 TRP A 108      25.682  26.985   2.917  1.00  0.00           C  \\nATOM    826  CZ3 TRP A 108      24.773  26.200   5.033  1.00  0.00           C  \\nATOM    827  CH2 TRP A 108      25.691  26.960   4.298  1.00  0.00           C  \\nATOM    828  N   VAL A 109      18.487  23.925   1.101  1.00  0.00           N  \\nATOM    829  CA  VAL A 109      17.265  23.096   1.025  1.00  0.00           C  \\nATOM    830  C   VAL A 109      17.573  21.611   0.884  1.00  0.00           C  \\nATOM    831  O   VAL A 109      16.923  20.811   1.587  1.00  0.00           O  \\nATOM    832  CB  VAL A 109      16.146  23.660   0.145  1.00  0.00           C  \\nATOM    833  CG1 VAL A 109      16.607  23.905  -1.285  1.00  0.00           C  \\nATOM    834  CG2 VAL A 109      14.901  22.783   0.136  1.00  0.00           C  \\nATOM    835  N   ALA A 110      18.562  21.308   0.075  1.00  0.00           N  \\nATOM    836  CA  ALA A 110      18.972  19.935  -0.237  1.00  0.00           C  \\nATOM    837  C   ALA A 110      19.502  19.244   1.006  1.00  0.00           C  \\nATOM    838  O   ALA A 110      19.197  18.046   1.174  1.00  0.00           O  \\nATOM    839  CB  ALA A 110      19.857  19.777  -1.455  1.00  0.00           C  \\nATOM    840  N   TRP A 111      20.121  19.983   1.892  1.00  0.00           N  \\nATOM    841  CA  TRP A 111      20.551  19.500   3.202  1.00  0.00           C  \\nATOM    842  C   TRP A 111      19.343  19.085   4.046  1.00  0.00           C  \\nATOM    843  O   TRP A 111      19.284  17.986   4.643  1.00  0.00           O  \\nATOM    844  CB  TRP A 111      21.486  20.451   3.925  1.00  0.00           C  \\nATOM    845  CG  TRP A 111      21.858  19.904   5.252  1.00  0.00           C  \\nATOM    846  CD1 TRP A 111      22.856  18.975   5.486  1.00  0.00           C  \\nATOM    847  CD2 TRP A 111      21.199  20.093   6.504  1.00  0.00           C  \\nATOM    848  NE1 TRP A 111      22.848  18.592   6.808  1.00  0.00           N  \\nATOM    849  CE2 TRP A 111      21.798  19.221   7.435  1.00  0.00           C  \\nATOM    850  CE3 TRP A 111      20.182  20.959   6.908  1.00  0.00           C  \\nATOM    851  CZ2 TRP A 111      21.492  19.284   8.784  1.00  0.00           C  \\nATOM    852  CZ3 TRP A 111      19.818  20.954   8.240  1.00  0.00           C  \\nATOM    853  CH2 TRP A 111      20.443  20.110   9.162  1.00  0.00           C  \\nATOM    854  N   ARG A 112      18.438  20.038   4.228  1.00  0.00           N  \\nATOM    855  CA  ARG A 112      17.185  19.830   4.955  1.00  0.00           C  \\nATOM    856  C   ARG A 112      16.472  18.593   4.391  1.00  0.00           C  \\nATOM    857  O   ARG A 112      16.000  17.792   5.220  1.00  0.00           O  \\nATOM    858  CB  ARG A 112      16.214  20.990   4.944  1.00  0.00           C  \\nATOM    859  CG  ARG A 112      16.592  22.438   5.091  1.00  0.00           C  \\nATOM    860  CD  ARG A 112      15.368  23.296   5.135  1.00  0.00           C  \\nATOM    861  NE  ARG A 112      14.776  23.555   3.836  1.00  0.00           N  \\nATOM    862  CZ  ARG A 112      14.785  24.695   3.143  1.00  0.00           C  \\nATOM    863  NH1 ARG A 112      15.596  25.721   3.383  1.00  0.00           N  \\nATOM    864  NH2 ARG A 112      13.717  25.008   2.398  1.00  0.00           N  \\nATOM    865  N   ASN A 113      16.307  18.521   3.074  1.00  0.00           N  \\nATOM    866  CA  ASN A 113      15.464  17.463   2.491  1.00  0.00           C  \\nATOM    867  C   ASN A 113      16.111  16.083   2.586  1.00  0.00           C  \\nATOM    868  O   ASN A 113      15.454  15.054   2.808  1.00  0.00           O  \\nATOM    869  CB  ASN A 113      14.966  17.779   1.085  1.00  0.00           C  \\nATOM    870  CG  ASN A 113      14.003  18.961   1.018  1.00  0.00           C  \\nATOM    871  OD1 ASN A 113      13.454  19.344   2.059  1.00  0.00           O  \\nATOM    872  ND2 ASN A 113      13.840  19.555  -0.159  1.00  0.00           N  \\nATOM    873  N   ARG A 114      17.401  16.041   2.355  1.00  0.00           N  \\nATOM    874  CA  ARG A 114      18.156  14.836   2.042  1.00  0.00           C  \\nATOM    875  C   ARG A 114      19.258  14.440   2.991  1.00  0.00           C  \\nATOM    876  O   ARG A 114      19.505  13.217   3.097  1.00  0.00           O  \\nATOM    877  CB  ARG A 114      18.622  14.910   0.576  1.00  0.00           C  \\nATOM    878  CG  ARG A 114      17.395  14.614  -0.300  1.00  0.00           C  \\nATOM    879  CD  ARG A 114      17.729  14.399  -1.731  1.00  0.00           C  \\nATOM    880  NE  ARG A 114      18.153  15.677  -2.301  1.00  0.00           N  \\nATOM    881  CZ  ARG A 114      17.826  16.080  -3.535  1.00  0.00           C  \\nATOM    882  NH1 ARG A 114      17.378  15.225  -4.456  1.00  0.00           N  \\nATOM    883  NH2 ARG A 114      17.735  17.388  -3.796  1.00  0.00           N  \\nATOM    884  N   CYS A 115      19.743  15.372   3.773  1.00  0.00           N  \\nATOM    885  CA  CYS A 115      20.843  15.041   4.708  1.00  0.00           C  \\nATOM    886  C   CYS A 115      20.448  14.933   6.159  1.00  0.00           C  \\nATOM    887  O   CYS A 115      20.972  14.116   6.940  1.00  0.00           O  \\nATOM    888  CB  CYS A 115      21.991  16.018   4.426  1.00  0.00           C  \\nATOM    889  SG  CYS A 115      22.563  16.009   2.739  1.00  0.00           S  \\nATOM    890  N   LYS A 116      19.714  15.918   6.619  1.00  0.00           N  \\nATOM    891  CA  LYS A 116      19.332  16.085   8.025  1.00  0.00           C  \\nATOM    892  C   LYS A 116      18.634  14.821   8.518  1.00  0.00           C  \\nATOM    893  O   LYS A 116      17.819  14.234   7.785  1.00  0.00           O  \\nATOM    894  CB  LYS A 116      18.492  17.363   8.126  1.00  0.00           C  \\nATOM    895  CG  LYS A 116      17.930  17.512   9.547  1.00  0.00           C  \\nATOM    896  CD  LYS A 116      16.745  18.481   9.554  1.00  0.00           C  \\nATOM    897  CE  LYS A 116      16.658  19.147  10.918  1.00  0.00           C  \\nATOM    898  NZ  LYS A 116      15.454  20.010  11.047  1.00  0.00           N  \\nATOM    899  N   GLY A 117      19.152  14.318   9.635  1.00  0.00           N  \\nATOM    900  CA  GLY A 117      18.558  13.126  10.267  1.00  0.00           C  \\nATOM    901  C   GLY A 117      19.018  11.781   9.733  1.00  0.00           C  \\nATOM    902  O   GLY A 117      18.499  10.733  10.164  1.00  0.00           O  \\nATOM    903  N   THR A 118      19.892  11.802   8.740  1.00  0.00           N  \\nATOM    904  CA  THR A 118      20.473  10.578   8.171  1.00  0.00           C  \\nATOM    905  C   THR A 118      21.868  10.375   8.761  1.00  0.00           C  \\nATOM    906  O   THR A 118      22.321  11.119   9.650  1.00  0.00           O  \\nATOM    907  CB  THR A 118      20.440  10.571   6.598  1.00  0.00           C  \\nATOM    908  OG1 THR A 118      21.560  11.404   6.161  1.00  0.00           O  \\nATOM    909  CG2 THR A 118      19.095  11.104   6.074  1.00  0.00           C  \\nATOM    910  N   ASP A 119      22.392   9.213   8.431  1.00  0.00           N  \\nATOM    911  CA  ASP A 119      23.768   8.830   8.756  1.00  0.00           C  \\nATOM    912  C   ASP A 119      24.713   9.543   7.779  1.00  0.00           C  \\nATOM    913  O   ASP A 119      25.178   8.950   6.780  1.00  0.00           O  \\nATOM    914  CB  ASP A 119      23.934   7.313   8.738  1.00  0.00           C  \\nATOM    915  CG  ASP A 119      25.347   6.900   9.121  1.00  0.00           C  \\nATOM    916  OD1 ASP A 119      26.051   7.633   9.830  1.00  0.00           O  \\nATOM    917  OD2 ASP A 119      25.715   5.746   8.804  1.00  0.00           O  \\nATOM    918  N   VAL A 120      24.988  10.793   8.094  1.00  0.00           N  \\nATOM    919  CA  VAL A 120      25.886  11.637   7.306  1.00  0.00           C  \\nATOM    920  C   VAL A 120      27.342  11.199   7.304  1.00  0.00           C  \\nATOM    921  O   VAL A 120      28.099  11.604   6.406  1.00  0.00           O  \\nATOM    922  CB  VAL A 120      25.721  13.134   7.630  1.00  0.00           C  \\nATOM    923  CG1 VAL A 120      24.356  13.655   7.183  1.00  0.00           C  \\nATOM    924  CG2 VAL A 120      26.088  13.478   9.055  1.00  0.00           C  \\nATOM    925  N   GLN A 121      27.701  10.430   8.306  1.00  0.00           N  \\nATOM    926  CA  GLN A 121      29.021   9.783   8.389  1.00  0.00           C  \\nATOM    927  C   GLN A 121      29.317   8.861   7.207  1.00  0.00           C  \\nATOM    928  O   GLN A 121      30.480   8.669   6.820  1.00  0.00           O  \\nATOM    929  CB  GLN A 121      29.088   9.048   9.728  1.00  0.00           C  \\nATOM    930  CG  GLN A 121      30.530   8.965  10.167  1.00  0.00           C  \\nATOM    931  CD  GLN A 121      30.615   8.877  11.674  1.00  0.00           C  \\nATOM    932  OE1 GLN A 121      31.368   9.632  12.283  1.00  0.00           O  \\nATOM    933  NE2 GLN A 121      29.884   7.871  12.151  1.00  0.00           N  \\nATOM    934  N   ALA A 122      28.300   8.271   6.576  1.00  0.00           N  \\nATOM    935  CA  ALA A 122      28.390   7.548   5.311  1.00  0.00           C  \\nATOM    936  C   ALA A 122      29.186   8.290   4.227  1.00  0.00           C  \\nATOM    937  O   ALA A 122      30.031   7.710   3.523  1.00  0.00           O  \\nATOM    938  CB  ALA A 122      26.999   7.244   4.783  1.00  0.00           C  \\nATOM    939  N   TRP A 123      29.021   9.616   4.240  1.00  0.00           N  \\nATOM    940  CA  TRP A 123      29.703  10.502   3.283  1.00  0.00           C  \\nATOM    941  C   TRP A 123      31.218  10.581   3.392  1.00  0.00           C  \\nATOM    942  O   TRP A 123      31.905  10.942   2.412  1.00  0.00           O  \\nATOM    943  CB  TRP A 123      29.027  11.872   3.347  1.00  0.00           C  \\nATOM    944  CG  TRP A 123      27.621  11.735   2.850  1.00  0.00           C  \\nATOM    945  CD1 TRP A 123      26.485  11.608   3.588  1.00  0.00           C  \\nATOM    946  CD2 TRP A 123      27.241  11.547   1.481  1.00  0.00           C  \\nATOM    947  NE1 TRP A 123      25.405  11.458   2.774  1.00  0.00           N  \\nATOM    948  CE2 TRP A 123      25.827  11.391   1.479  1.00  0.00           C  \\nATOM    949  CE3 TRP A 123      27.947  11.468   0.283  1.00  0.00           C  \\nATOM    950  CZ2 TRP A 123      25.111  11.184   0.311  1.00  0.00           C  \\nATOM    951  CZ3 TRP A 123      27.222  11.329  -0.886  1.00  0.00           C  \\nATOM    952  CH2 TRP A 123      25.835  11.127  -0.869  1.00  0.00           C  \\nATOM    953  N   ILE A 124      31.741  10.269   4.554  1.00  0.00           N  \\nATOM    954  CA  ILE A 124      33.186  10.292   4.801  1.00  0.00           C  \\nATOM    955  C   ILE A 124      33.863   8.955   5.024  1.00  0.00           C  \\nATOM    956  O   ILE A 124      35.100   8.892   5.134  1.00  0.00           O  \\nATOM    957  CB  ILE A 124      33.504  11.403   5.863  1.00  0.00           C  \\nATOM    958  CG1 ILE A 124      32.956  10.984   7.234  1.00  0.00           C  \\nATOM    959  CG2 ILE A 124      33.024  12.804   5.387  1.00  0.00           C  \\nATOM    960  CD1 ILE A 124      33.729  11.437   8.488  1.00  0.00           C  \\nATOM    961  N   ARG A 125      33.080   7.898   5.131  1.00  0.00           N  \\nATOM    962  CA  ARG A 125      33.594   6.534   5.320  1.00  0.00           C  \\nATOM    963  C   ARG A 125      34.311   6.113   4.036  1.00  0.00           C  \\nATOM    964  O   ARG A 125      33.843   6.337   2.906  1.00  0.00           O  \\nATOM    965  CB  ARG A 125      32.476   5.551   5.650  1.00  0.00           C  \\nATOM    966  CG  ARG A 125      32.117   5.596   7.145  1.00  0.00           C  \\nATOM    967  CD  ARG A 125      31.277   4.392   7.482  1.00  0.00           C  \\nATOM    968  NE  ARG A 125      30.282   4.754   8.466  1.00  0.00           N  \\nATOM    969  CZ  ARG A 125      28.984   4.993   8.331  1.00  0.00           C  \\nATOM    970  NH1 ARG A 125      28.334   4.690   7.207  1.00  0.00           N  \\nATOM    971  NH2 ARG A 125      28.392   5.549   9.392  1.00  0.00           N  \\nATOM    972  N   GLY A 126      35.497   5.568   4.243  1.00  0.00           N  \\nATOM    973  CA  GLY A 126      36.291   5.058   3.102  1.00  0.00           C  \\nATOM    974  C   GLY A 126      37.334   6.066   2.658  1.00  0.00           C  \\nATOM    975  O   GLY A 126      38.220   5.729   1.855  1.00  0.00           O  \\nATOM    976  N   CYS A 127      37.335   7.221   3.297  1.00  0.00           N  \\nATOM    977  CA  CYS A 127      38.234   8.333   2.961  1.00  0.00           C  \\nATOM    978  C   CYS A 127      39.422   8.382   3.925  1.00  0.00           C  \\nATOM    979  O   CYS A 127      39.206   8.267   5.138  1.00  0.00           O  \\nATOM    980  CB  CYS A 127      37.453   9.628   2.990  1.00  0.00           C  \\nATOM    981  SG  CYS A 127      36.010   9.816   1.936  1.00  0.00           S  \\nATOM    982  N   ARG A 128      40.586   8.695   3.393  1.00  0.00           N  \\nATOM    983  CA  ARG A 128      41.774   8.960   4.217  1.00  0.00           C  \\nATOM    984  C   ARG A 128      41.820  10.438   4.578  1.00  0.00           C  \\nATOM    985  O   ARG A 128      41.976  11.291   3.694  1.00  0.00           O  \\nATOM    986  CB  ARG A 128      43.047   8.304   3.707  1.00  0.00           C  \\nATOM    987  CG  ARG A 128      43.231   6.886   4.280  1.00  0.00           C  \\nATOM    988  CD  ARG A 128      43.833   6.911   5.651  1.00  0.00           C  \\nATOM    989  NE  ARG A 128      45.246   7.263   5.636  1.00  0.00           N  \\nATOM    990  CZ  ARG A 128      45.862   8.258   6.281  1.00  0.00           C  \\nATOM    991  NH1 ARG A 128      45.241   9.069   7.151  1.00  0.00           N  \\nATOM    992  NH2 ARG A 128      47.134   8.554   5.973  1.00  0.00           N  \\nATOM    993  N   LEU A 129      41.289  10.715   5.771  1.00  0.00           N  \\nATOM    994  CA  LEU A 129      41.094  12.084   6.273  1.00  0.00           C  \\nATOM    995  C   LEU A 129      42.119  12.382   7.370  1.00  0.00           C  \\nATOM    996  O   LEU A 129      41.730  12.276   8.559  1.00  0.00           O  \\nATOM    997  CB  LEU A 129      39.635  12.335   6.646  1.00  0.00           C  \\nATOM    998  CG  LEU A 129      38.689  12.917   5.620  1.00  0.00           C  \\nATOM    999  CD1 LEU A 129      39.112  12.657   4.191  1.00  0.00           C  \\nATOM   1000  CD2 LEU A 129      37.310  12.325   5.886  1.00  0.00           C  \\nATOM   1001  OXT LEU A 129      43.232  12.675   6.905  1.00  0.00           O  \\nHETATM 1002  O   HOH A 130      23.434  40.063  -6.661  1.00  0.00           O  \\nHETATM 1003  O   HOH A 131      31.994  26.416  -6.047  1.00  0.00           O  \\nHETATM 1004  O   HOH A 132      30.250  13.337   9.787  1.00  0.00           O  \\nHETATM 1005  O   HOH A 133      22.384  42.331  -8.165  1.00  0.00           O  \\nHETATM 1006  O   HOH A 134      29.239  27.621  -3.670  1.00  0.00           O  \\nHETATM 1007  O   HOH A 135      29.464  37.761 -10.492  1.00  0.00           O  \\nHETATM 1008  O   HOH A 136      20.807  36.305 -11.082  1.00  0.00           O  \\nHETATM 1009  O   HOH A 137      41.318  17.849  -1.378  1.00  0.00           O  \\nHETATM 1010  O   HOH A 138      34.697  29.056  -4.039  1.00  0.00           O  \\nHETATM 1011  O   HOH A 139      26.871  17.298  13.496  1.00  0.00           O  \\nHETATM 1012  O   HOH A 140      32.131  11.050  -5.817  1.00  0.00           O  \\nHETATM 1013  O   HOH A 141      23.468  40.040  -2.372  1.00  0.00           O  \\nHETATM 1014  O   HOH A 142      21.390  45.524 -11.035  1.00  0.00           O  \\nHETATM 1015  O   HOH A 143      34.490  26.578  -5.741  1.00  0.00           O  \\nHETATM 1016  O   HOH A 144      16.422  34.139  -3.527  1.00  0.00           O  \\nHETATM 1017  O   HOH A 145      21.374  29.926   8.946  1.00  0.00           O  \\nHETATM 1018  O   HOH A 146      41.048  12.539  -0.011  1.00  0.00           O  \\nHETATM 1019  O   HOH A 147      32.794  35.686   2.558  1.00  0.00           O  \\nHETATM 1020  O   HOH A 148      49.648   8.964   6.343  1.00  0.00           O  \\nHETATM 1021  O   HOH A 149      14.452  34.901 -13.339  1.00  0.00           O  \\nHETATM 1022  O   HOH A 150      22.930  10.839   4.044  1.00  0.00           O  \\nHETATM 1023  O   HOH A 151      16.012  18.490  -2.200  1.00  0.00           O  \\nHETATM 1024  O   HOH A 152      12.130  21.587   3.044  1.00  0.00           O  \\nHETATM 1025  O   HOH A 153      15.684  38.922  -5.813  1.00  0.00           O  \\nHETATM 1026  O   HOH A 154      10.652  24.228   3.428  1.00  0.00           O  \\nHETATM 1027  O   HOH A 155      44.070  17.975   2.852  1.00  0.00           O  \\nHETATM 1028  O   HOH A 156      32.029  13.080  -8.110  1.00  0.00           O  \\nHETATM 1029  O   HOH A 157      36.425  19.613  15.174  1.00  0.00           O  \\nHETATM 1030  O   HOH A 158      37.941  30.505  -3.686  1.00  0.00           O  \\nHETATM 1031  O   HOH A 159      30.710  42.741  -6.289  1.00  0.00           O  \\nHETATM 1032  O   HOH A 160      23.922  44.367  -7.653  1.00  0.00           O  \\nHETATM 1033  O   HOH A 161      33.829  34.252   0.626  1.00  0.00           O  \\nHETATM 1034  O   HOH A 162      29.613  40.730  -9.602  1.00  0.00           O  \\nHETATM 1035  O   HOH A 163      23.563   7.995   4.406  1.00  0.00           O  \\nHETATM 1036  O   HOH A 164      31.511  42.362  -4.183  1.00  0.00           O  \\nHETATM 1037  O   HOH A 165      21.882  29.536 -15.013  1.00  0.00           O  \\nHETATM 1038  O   HOH A 166      37.763  20.913   9.782  1.00  0.00           O  \\nHETATM 1039  O   HOH A 167      42.338  17.481   5.165  1.00  0.00           O  \\nHETATM 1040  O   HOH A 168      23.344  39.739  -4.358  1.00  0.00           O  \\nHETATM 1041  O   HOH A 169      22.984  29.224  13.124  1.00  0.00           O  \\nHETATM 1042  O   HOH A 170      30.778   7.794  -3.514  1.00  0.00           O  \\nHETATM 1043  O   HOH A 171      42.965  14.657   4.991  1.00  0.00           O  \\nHETATM 1044  O   HOH A 172      36.927  17.948 -13.093  1.00  0.00           O  \\nHETATM 1045  O   HOH A 173      35.412  25.852 -11.575  1.00  0.00           O  \\nHETATM 1046  O   HOH A 174      37.428  32.540  -5.787  1.00  0.00           O  \\nHETATM 1047  O   HOH A 175      37.317   8.592   7.456  1.00  0.00           O  \\nHETATM 1048  O   HOH A 176       9.314  36.705 -11.546  1.00  0.00           O  \\nHETATM 1049  O   HOH A 177      39.972  23.760  -2.655  1.00  0.00           O  \\nHETATM 1050  O   HOH A 178      22.128  30.274  -0.543  1.00  0.00           O  \\nHETATM 1051  O   HOH A 179      22.244  15.813  10.000  1.00  0.00           O  \\nHETATM 1052  O   HOH A 180      40.729   9.223   0.292  1.00  0.00           O  \\nHETATM 1053  O   HOH A 181      12.500  15.267   4.097  1.00  0.00           O  \\nHETATM 1054  O   HOH A 182      20.372  28.618  -2.353  1.00  0.00           O  \\nHETATM 1055  O   HOH A 183      22.793  15.462  -6.673  1.00  0.00           O  \\nHETATM 1056  O   HOH A 184      23.138  31.809  15.121  1.00  0.00           O  \\nHETATM 1057  O   HOH A 185      22.671  38.691   8.245  1.00  0.00           O  \\nHETATM 1058  O   HOH A 186      33.966  33.112   6.837  1.00  0.00           O  \\nHETATM 1059  O   HOH A 187      19.572  25.423  -1.420  1.00  0.00           O  \\nHETATM 1060  O   HOH A 188      14.790  15.672   7.259  1.00  0.00           O  \\nHETATM 1061  O   HOH A 189      19.112  28.022 -14.647  1.00  0.00           O  \\nHETATM 1062  O   HOH A 190      17.302  39.059 -12.453  1.00  0.00           O  \\nHETATM 1063  O   HOH A 191      16.198  14.502   5.577  1.00  0.00           O  \\nHETATM 1064  O   HOH A 192      17.345  46.346  -7.080  1.00  0.00           O  \\nHETATM 1065  O   HOH A 193      14.992  31.300  -4.242  1.00  0.00           O  \\nHETATM 1066  O   HOH A 194      28.196  44.775  -3.148  1.00  0.00           O  \\nHETATM 1067  O   HOH A 195      29.479  13.863  -9.107  1.00  0.00           O  \\nHETATM 1068  O   HOH A 196      23.613  44.811   2.608  1.00  0.00           O  \\nHETATM 1069  O   HOH A 197      40.572  22.184  -6.358  1.00  0.00           O  \\nHETATM 1070  O   HOH A 198      12.475  31.860  -6.226  1.00  0.00           O  \\nHETATM 1071  O   HOH A 199      16.684  13.594  -5.832  1.00  0.00           O  \\nHETATM 1072  O   HOH A 200      27.534  38.059 -12.862  1.00  0.00           O  \\nHETATM 1073  O   HOH A 201      25.892  35.973  11.563  1.00  0.00           O  \\nHETATM 1074  O   HOH A 202      24.790  25.182  16.063  1.00  0.00           O  \\nHETATM 1075  O   HOH A 203      12.580  21.214   5.006  1.00  0.00           O  \\nHETATM 1076  O   HOH A 204      19.687  23.750  -4.851  1.00  0.00           O  \\nHETATM 1077  O   HOH A 205      27.098  35.956 -12.358  1.00  0.00           O  \\nHETATM 1078  O   HOH A 206      37.255   9.634  10.002  1.00  0.00           O  \\nHETATM 1079  O   HOH A 207      43.755  23.843   8.038  1.00  0.00           O  \\nTER    1080      HOH A 207\\nCONECT   48  981\\nCONECT  238  889\\nCONECT  513  630\\nCONECT  601  724\\nCONECT  630  513\\nCONECT  724  601\\nCONECT  889  238\\nCONECT  981   48\\nEND\\n\",\"pdb\");\n\tviewer_16678367835488749.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_16678367835488749.zoomTo();\nviewer_16678367835488749.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16678367835488749\"  style=\"position: relative; width: 640px; height: 480px\">\n",
              "        <p id=\"3dmolwarning_16678367835488749\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16678367835488749 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16678367835488749\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16678367835488749 = $3Dmol.createViewer($(\"#3dmolviewer_16678367835488749\"),{backgroundColor:\"white\"});\n",
              "viewer_16678367835488749.zoomTo();\n",
              "\tviewer_16678367835488749.addModel(\"REMARK   1 CREATED WITH OPENMM 7.7, 2022-11-07\\nATOM      1  N   LYS A   1      35.365  22.342 -11.980  1.00  0.00           N  \\nATOM      2  CA  LYS A   1      35.892  21.073 -11.427  1.00  0.00           C  \\nATOM      3  C   LYS A   1      34.741  20.264 -10.844  1.00  0.00           C  \\nATOM      4  O   LYS A   1      33.945  20.813 -10.081  1.00  0.00           O  \\nATOM      5  CB  LYS A   1      36.872  21.435 -10.306  1.00  0.00           C  \\nATOM      6  CG  LYS A   1      37.453  20.248  -9.565  1.00  0.00           C  \\nATOM      7  CD  LYS A   1      38.688  20.649  -8.775  1.00  0.00           C  \\nATOM      8  CE  LYS A   1      39.057  19.508  -7.837  1.00  0.00           C  \\nATOM      9  NZ  LYS A   1      40.423  19.771  -7.299  1.00  0.00           N  \\nATOM     10  N   VAL A   2      34.739  18.961 -11.042  1.00  0.00           N  \\nATOM     11  CA  VAL A   2      33.903  17.998 -10.333  1.00  0.00           C  \\nATOM     12  C   VAL A   2      34.800  17.312  -9.294  1.00  0.00           C  \\nATOM     13  O   VAL A   2      35.759  16.605  -9.665  1.00  0.00           O  \\nATOM     14  CB  VAL A   2      33.140  17.034 -11.232  1.00  0.00           C  \\nATOM     15  CG1 VAL A   2      32.251  16.084 -10.434  1.00  0.00           C  \\nATOM     16  CG2 VAL A   2      32.294  17.714 -12.290  1.00  0.00           C  \\nATOM     17  N   PHE A   3      34.491  17.546  -8.038  1.00  0.00           N  \\nATOM     18  CA  PHE A   3      35.185  16.903  -6.918  1.00  0.00           C  \\nATOM     19  C   PHE A   3      34.742  15.441  -6.771  1.00  0.00           C  \\nATOM     20  O   PHE A   3      33.525  15.162  -6.862  1.00  0.00           O  \\nATOM     21  CB  PHE A   3      34.967  17.632  -5.594  1.00  0.00           C  \\nATOM     22  CG  PHE A   3      35.944  18.737  -5.375  1.00  0.00           C  \\nATOM     23  CD1 PHE A   3      35.666  20.050  -5.798  1.00  0.00           C  \\nATOM     24  CD2 PHE A   3      37.000  18.557  -4.473  1.00  0.00           C  \\nATOM     25  CE1 PHE A   3      36.577  21.076  -5.568  1.00  0.00           C  \\nATOM     26  CE2 PHE A   3      37.869  19.589  -4.157  1.00  0.00           C  \\nATOM     27  CZ  PHE A   3      37.636  20.873  -4.666  1.00  0.00           C  \\nATOM     28  N   GLY A   4      35.724  14.639  -6.331  1.00  0.00           N  \\nATOM     29  CA  GLY A   4      35.366  13.280  -5.870  1.00  0.00           C  \\nATOM     30  C   GLY A   4      34.924  13.420  -4.415  1.00  0.00           C  \\nATOM     31  O   GLY A   4      35.303  14.403  -3.781  1.00  0.00           O  \\nATOM     32  N   ARG A   5      34.053  12.538  -3.973  1.00  0.00           N  \\nATOM     33  CA  ARG A   5      33.565  12.538  -2.588  1.00  0.00           C  \\nATOM     34  C   ARG A   5      34.665  12.734  -1.556  1.00  0.00           C  \\nATOM     35  O   ARG A   5      34.669  13.651  -0.704  1.00  0.00           O  \\nATOM     36  CB  ARG A   5      32.765  11.262  -2.331  1.00  0.00           C  \\nATOM     37  CG  ARG A   5      32.213  11.203  -0.920  1.00  0.00           C  \\nATOM     38  CD  ARG A   5      31.375  10.001  -0.722  1.00  0.00           C  \\nATOM     39  NE  ARG A   5      32.059   8.749  -0.958  1.00  0.00           N  \\nATOM     40  CZ  ARG A   5      32.733   8.011  -0.097  1.00  0.00           C  \\nATOM     41  NH1 ARG A   5      32.836   8.332   1.187  1.00  0.00           N  \\nATOM     42  NH2 ARG A   5      33.245   6.836  -0.526  1.00  0.00           N  \\nATOM     43  N   CYS A   6      35.674  11.853  -1.612  1.00  0.00           N  \\nATOM     44  CA  CYS A   6      36.781  11.870  -0.654  1.00  0.00           C  \\nATOM     45  C   CYS A   6      37.747  13.050  -0.777  1.00  0.00           C  \\nATOM     46  O   CYS A   6      38.148  13.609   0.264  1.00  0.00           O  \\nATOM     47  CB  CYS A   6      37.491  10.532  -0.621  1.00  0.00           C  \\nATOM     48  SG  CYS A   6      36.540   9.205   0.140  1.00  0.00           S  \\nATOM     49  N   GLU A   7      37.861  13.481  -2.019  1.00  0.00           N  \\nATOM     50  CA  GLU A   7      38.685  14.686  -2.311  1.00  0.00           C  \\nATOM     51  C   GLU A   7      38.049  15.926  -1.658  1.00  0.00           C  \\nATOM     52  O   GLU A   7      38.744  16.729  -1.011  1.00  0.00           O  \\nATOM     53  CB  GLU A   7      38.784  14.846  -3.818  1.00  0.00           C  \\nATOM     54  CG  GLU A   7      39.540  16.051  -4.379  1.00  0.00           C  \\nATOM     55  CD  GLU A   7      39.576  16.242  -5.870  1.00  0.00           C  \\nATOM     56  OE1 GLU A   7      38.672  15.644  -6.491  1.00  0.00           O  \\nATOM     57  OE2 GLU A   7      40.415  16.953  -6.381  1.00  0.00           O  \\nATOM     58  N   LEU A   8      36.743  16.049  -1.819  1.00  0.00           N  \\nATOM     59  CA  LEU A   8      35.964  17.158  -1.255  1.00  0.00           C  \\nATOM     60  C   LEU A   8      36.051  17.132   0.266  1.00  0.00           C  \\nATOM     61  O   LEU A   8      36.159  18.166   0.920  1.00  0.00           O  \\nATOM     62  CB  LEU A   8      34.528  17.172  -1.811  1.00  0.00           C  \\nATOM     63  CG  LEU A   8      33.718  18.354  -1.305  1.00  0.00           C  \\nATOM     64  CD1 LEU A   8      34.297  19.656  -1.841  1.00  0.00           C  \\nATOM     65  CD2 LEU A   8      32.246  18.143  -1.596  1.00  0.00           C  \\nATOM     66  N   ALA A   9      35.754  15.980   0.828  1.00  0.00           N  \\nATOM     67  CA  ALA A   9      35.838  15.757   2.284  1.00  0.00           C  \\nATOM     68  C   ALA A   9      37.144  16.314   2.840  1.00  0.00           C  \\nATOM     69  O   ALA A   9      37.151  16.978   3.897  1.00  0.00           O  \\nATOM     70  CB  ALA A   9      35.656  14.287   2.623  1.00  0.00           C  \\nATOM     71  N   ALA A  10      38.272  15.983   2.204  1.00  0.00           N  \\nATOM     72  CA  ALA A  10      39.610  16.431   2.616  1.00  0.00           C  \\nATOM     73  C   ALA A  10      39.736  17.944   2.459  1.00  0.00           C  \\nATOM     74  O   ALA A  10      40.193  18.499   3.469  1.00  0.00           O  \\nATOM     75  CB  ALA A  10      40.708  15.706   1.842  1.00  0.00           C  \\nATOM     76  N   ALA A  11      39.227  18.519   1.385  1.00  0.00           N  \\nATOM     77  CA  ALA A  11      39.264  19.982   1.223  1.00  0.00           C  \\nATOM     78  C   ALA A  11      38.491  20.702   2.321  1.00  0.00           C  \\nATOM     79  O   ALA A  11      38.946  21.658   2.953  1.00  0.00           O  \\nATOM     80  CB  ALA A  11      38.869  20.421  -0.175  1.00  0.00           C  \\nATOM     81  N   MET A  12      37.288  20.214   2.590  1.00  0.00           N  \\nATOM     82  CA  MET A  12      36.398  20.781   3.612  1.00  0.00           C  \\nATOM     83  C   MET A  12      36.990  20.715   5.007  1.00  0.00           C  \\nATOM     84  O   MET A  12      36.906  21.637   5.840  1.00  0.00           O  \\nATOM     85  CB  MET A  12      34.993  20.213   3.515  1.00  0.00           C  \\nATOM     86  CG  MET A  12      34.320  20.724   2.265  1.00  0.00           C  \\nATOM     87  SD  MET A  12      32.634  19.986   2.235  1.00  0.00           S  \\nATOM     88  CE  MET A  12      31.788  21.135   1.138  1.00  0.00           C  \\nATOM     89  N   LYS A  13      37.688  19.628   5.314  1.00  0.00           N  \\nATOM     90  CA  LYS A  13      38.387  19.385   6.579  1.00  0.00           C  \\nATOM     91  C   LYS A  13      39.460  20.467   6.731  1.00  0.00           C  \\nATOM     92  O   LYS A  13      39.507  21.137   7.776  1.00  0.00           O  \\nATOM     93  CB  LYS A  13      38.934  17.952   6.572  1.00  0.00           C  \\nATOM     94  CG  LYS A  13      39.742  17.555   7.798  1.00  0.00           C  \\nATOM     95  CD  LYS A  13      38.973  16.777   8.834  1.00  0.00           C  \\nATOM     96  CE  LYS A  13      39.293  15.305   8.751  1.00  0.00           C  \\nATOM     97  NZ  LYS A  13      38.077  14.461   8.946  1.00  0.00           N  \\nATOM     98  N   ARG A  14      40.267  20.629   5.688  1.00  0.00           N  \\nATOM     99  CA  ARG A  14      41.387  21.577   5.713  1.00  0.00           C  \\nATOM    100  C   ARG A  14      40.927  23.017   5.881  1.00  0.00           C  \\nATOM    101  O   ARG A  14      41.557  23.834   6.584  1.00  0.00           O  \\nATOM    102  CB  ARG A  14      42.388  21.351   4.601  1.00  0.00           C  \\nATOM    103  CG  ARG A  14      42.173  22.079   3.289  1.00  0.00           C  \\nATOM    104  CD  ARG A  14      43.444  22.075   2.490  1.00  0.00           C  \\nATOM    105  NE  ARG A  14      43.687  20.710   2.012  1.00  0.00           N  \\nATOM    106  CZ  ARG A  14      43.098  20.255   0.892  1.00  0.00           C  \\nATOM    107  NH1 ARG A  14      42.695  21.186   0.018  1.00  0.00           N  \\nATOM    108  NH2 ARG A  14      42.949  18.957   0.689  1.00  0.00           N  \\nATOM    109  N   HIS A  15      39.681  23.247   5.463  1.00  0.00           N  \\nATOM    110  CA  HIS A  15      39.075  24.591   5.526  1.00  0.00           C  \\nATOM    111  C   HIS A  15      38.310  24.861   6.814  1.00  0.00           C  \\nATOM    112  O   HIS A  15      37.608  25.875   6.952  1.00  0.00           O  \\nATOM    113  CB  HIS A  15      38.223  24.918   4.285  1.00  0.00           C  \\nATOM    114  CG  HIS A  15      39.085  25.388   3.171  1.00  0.00           C  \\nATOM    115  ND1 HIS A  15      39.457  24.635   2.091  1.00  0.00           N  \\nATOM    116  CD2 HIS A  15      39.739  26.570   2.993  1.00  0.00           C  \\nATOM    117  CE1 HIS A  15      40.211  25.312   1.258  1.00  0.00           C  \\nATOM    118  NE2 HIS A  15      40.524  26.419   1.889  1.00  0.00           N  \\nATOM    119  N   GLY A  16      38.296  23.927   7.720  1.00  0.00           N  \\nATOM    120  CA  GLY A  16      37.765  24.025   9.072  1.00  0.00           C  \\nATOM    121  C   GLY A  16      36.264  23.799   9.210  1.00  0.00           C  \\nATOM    122  O   GLY A  16      35.646  24.400  10.127  1.00  0.00           O  \\nATOM    123  N   LEU A  17      35.665  23.073   8.274  1.00  0.00           N  \\nATOM    124  CA  LEU A  17      34.238  22.795   8.298  1.00  0.00           C  \\nATOM    125  C   LEU A  17      33.845  21.682   9.266  1.00  0.00           C  \\nATOM    126  O   LEU A  17      32.643  21.594   9.552  1.00  0.00           O  \\nATOM    127  CB  LEU A  17      33.648  22.647   6.901  1.00  0.00           C  \\nATOM    128  CG  LEU A  17      33.451  23.889   6.060  1.00  0.00           C  \\nATOM    129  CD1 LEU A  17      32.933  23.420   4.705  1.00  0.00           C  \\nATOM    130  CD2 LEU A  17      32.556  24.946   6.679  1.00  0.00           C  \\nATOM    131  N   ASP A  18      34.785  20.814   9.605  1.00  0.00           N  \\nATOM    132  CA  ASP A  18      34.492  19.722  10.526  1.00  0.00           C  \\nATOM    133  C   ASP A  18      34.015  20.222  11.901  1.00  0.00           C  \\nATOM    134  O   ASP A  18      34.826  20.778  12.658  1.00  0.00           O  \\nATOM    135  CB  ASP A  18      35.557  18.633  10.598  1.00  0.00           C  \\nATOM    136  CG  ASP A  18      35.017  17.408  11.311  1.00  0.00           C  \\nATOM    137  OD1 ASP A  18      33.805  17.154  11.455  1.00  0.00           O  \\nATOM    138  OD2 ASP A  18      35.925  16.603  11.662  1.00  0.00           O  \\nATOM    139  N   ASN A  19      32.746  19.990  12.205  1.00  0.00           N  \\nATOM    140  CA  ASN A  19      32.123  20.431  13.448  1.00  0.00           C  \\nATOM    141  C   ASN A  19      31.854  21.941  13.497  1.00  0.00           C  \\nATOM    142  O   ASN A  19      31.426  22.398  14.573  1.00  0.00           O  \\nATOM    143  CB  ASN A  19      32.767  20.004  14.770  1.00  0.00           C  \\nATOM    144  CG  ASN A  19      32.162  20.512  16.064  1.00  0.00           C  \\nATOM    145  OD1 ASN A  19      30.967  20.273  16.355  1.00  0.00           O  \\nATOM    146  ND2 ASN A  19      32.847  21.361  16.852  1.00  0.00           N  \\nATOM    147  N   TYR A  20      31.969  22.650  12.406  1.00  0.00           N  \\nATOM    148  CA  TYR A  20      31.707  24.099  12.411  1.00  0.00           C  \\nATOM    149  C   TYR A  20      30.231  24.343  12.759  1.00  0.00           C  \\nATOM    150  O   TYR A  20      29.288  23.874  12.118  1.00  0.00           O  \\nATOM    151  CB  TYR A  20      32.070  24.736  11.066  1.00  0.00           C  \\nATOM    152  CG  TYR A  20      32.061  26.250  11.144  1.00  0.00           C  \\nATOM    153  CD1 TYR A  20      33.141  26.886  11.760  1.00  0.00           C  \\nATOM    154  CD2 TYR A  20      30.979  27.004  10.691  1.00  0.00           C  \\nATOM    155  CE1 TYR A  20      33.206  28.277  11.794  1.00  0.00           C  \\nATOM    156  CE2 TYR A  20      31.018  28.399  10.779  1.00  0.00           C  \\nATOM    157  CZ  TYR A  20      32.102  29.017  11.383  1.00  0.00           C  \\nATOM    158  OH  TYR A  20      32.136  30.371  11.525  1.00  0.00           O  \\nATOM    159  N   ARG A  21      30.088  25.142  13.803  1.00  0.00           N  \\nATOM    160  CA  ARG A  21      28.774  25.553  14.319  1.00  0.00           C  \\nATOM    161  C   ARG A  21      27.964  24.312  14.623  1.00  0.00           C  \\nATOM    162  O   ARG A  21      26.733  24.333  14.606  1.00  0.00           O  \\nATOM    163  CB  ARG A  21      28.014  26.565  13.453  1.00  0.00           C  \\nATOM    164  CG  ARG A  21      28.507  27.995  13.535  1.00  0.00           C  \\nATOM    165  CD  ARG A  21      28.110  28.755  14.765  1.00  0.00           C  \\nATOM    166  NE  ARG A  21      29.319  29.321  15.324  1.00  0.00           N  \\nATOM    167  CZ  ARG A  21      30.215  30.234  14.978  1.00  0.00           C  \\nATOM    168  NH1 ARG A  21      31.501  29.856  14.846  1.00  0.00           N  \\nATOM    169  NH2 ARG A  21      29.938  31.437  14.495  1.00  0.00           N  \\nATOM    170  N   GLY A  22      28.689  23.250  14.998  1.00  0.00           N  \\nATOM    171  CA  GLY A  22      28.103  22.021  15.519  1.00  0.00           C  \\nATOM    172  C   GLY A  22      27.748  21.018  14.436  1.00  0.00           C  \\nATOM    173  O   GLY A  22      27.085  20.022  14.784  1.00  0.00           O  \\nATOM    174  N   TYR A  23      28.209  21.230  13.216  1.00  0.00           N  \\nATOM    175  CA  TYR A  23      27.887  20.318  12.111  1.00  0.00           C  \\nATOM    176  C   TYR A  23      29.124  19.533  11.628  1.00  0.00           C  \\nATOM    177  O   TYR A  23      30.045  20.191  11.139  1.00  0.00           O  \\nATOM    178  CB  TYR A  23      27.351  21.107  10.913  1.00  0.00           C  \\nATOM    179  CG  TYR A  23      26.001  21.745  11.127  1.00  0.00           C  \\nATOM    180  CD1 TYR A  23      24.846  20.962  11.139  1.00  0.00           C  \\nATOM    181  CD2 TYR A  23      25.897  23.096  11.458  1.00  0.00           C  \\nATOM    182  CE1 TYR A  23      23.600  21.518  11.422  1.00  0.00           C  \\nATOM    183  CE2 TYR A  23      24.647  23.673  11.726  1.00  0.00           C  \\nATOM    184  CZ  TYR A  23      23.518  22.881  11.701  1.00  0.00           C  \\nATOM    185  OH  TYR A  23      22.289  23.438  11.912  1.00  0.00           O  \\nATOM    186  N   SER A  24      29.029  18.223  11.810  1.00  0.00           N  \\nATOM    187  CA  SER A  24      30.143  17.347  11.414  1.00  0.00           C  \\nATOM    188  C   SER A  24      30.359  17.379   9.895  1.00  0.00           C  \\nATOM    189  O   SER A  24      29.442  17.687   9.139  1.00  0.00           O  \\nATOM    190  CB  SER A  24      29.922  15.934  11.907  1.00  0.00           C  \\nATOM    191  OG  SER A  24      28.799  15.336  11.308  1.00  0.00           O  \\nATOM    192  N   LEU A  25      31.593  17.028   9.540  1.00  0.00           N  \\nATOM    193  CA  LEU A  25      32.035  17.092   8.138  1.00  0.00           C  \\nATOM    194  C   LEU A  25      31.030  16.437   7.183  1.00  0.00           C  \\nATOM    195  O   LEU A  25      30.874  16.924   6.056  1.00  0.00           O  \\nATOM    196  CB  LEU A  25      33.410  16.409   8.084  1.00  0.00           C  \\nATOM    197  CG  LEU A  25      34.015  16.477   6.689  1.00  0.00           C  \\nATOM    198  CD1 LEU A  25      34.174  17.929   6.289  1.00  0.00           C  \\nATOM    199  CD2 LEU A  25      35.398  15.810   6.752  1.00  0.00           C  \\nATOM    200  N   GLY A  26      30.501  15.280   7.576  1.00  0.00           N  \\nATOM    201  CA  GLY A  26      29.539  14.561   6.756  1.00  0.00           C  \\nATOM    202  C   GLY A  26      28.338  15.378   6.277  1.00  0.00           C  \\nATOM    203  O   GLY A  26      27.886  15.250   5.120  1.00  0.00           O  \\nATOM    204  N   ASN A  27      27.905  16.281   7.161  1.00  0.00           N  \\nATOM    205  CA  ASN A  27      26.827  17.227   6.857  1.00  0.00           C  \\nATOM    206  C   ASN A  27      27.164  18.015   5.597  1.00  0.00           C  \\nATOM    207  O   ASN A  27      26.317  18.208   4.720  1.00  0.00           O  \\nATOM    208  CB  ASN A  27      26.484  18.126   8.054  1.00  0.00           C  \\nATOM    209  CG  ASN A  27      25.681  17.267   9.048  1.00  0.00           C  \\nATOM    210  OD1 ASN A  27      24.511  16.933   8.820  1.00  0.00           O  \\nATOM    211  ND2 ASN A  27      26.348  17.052  10.169  1.00  0.00           N  \\nATOM    212  N   TRP A  28      28.344  18.591   5.583  1.00  0.00           N  \\nATOM    213  CA  TRP A  28      28.831  19.475   4.513  1.00  0.00           C  \\nATOM    214  C   TRP A  28      28.940  18.741   3.183  1.00  0.00           C  \\nATOM    215  O   TRP A  28      28.742  19.321   2.090  1.00  0.00           O  \\nATOM    216  CB  TRP A  28      30.104  20.145   5.023  1.00  0.00           C  \\nATOM    217  CG  TRP A  28      29.941  20.978   6.251  1.00  0.00           C  \\nATOM    218  CD1 TRP A  28      30.176  20.624   7.546  1.00  0.00           C  \\nATOM    219  CD2 TRP A  28      29.319  22.284   6.287  1.00  0.00           C  \\nATOM    220  NE1 TRP A  28      29.924  21.690   8.365  1.00  0.00           N  \\nATOM    221  CE2 TRP A  28      29.337  22.679   7.641  1.00  0.00           C  \\nATOM    222  CE3 TRP A  28      28.894  23.168   5.295  1.00  0.00           C  \\nATOM    223  CZ2 TRP A  28      28.913  23.943   8.038  1.00  0.00           C  \\nATOM    224  CZ3 TRP A  28      28.398  24.404   5.682  1.00  0.00           C  \\nATOM    225  CH2 TRP A  28      28.431  24.766   7.025  1.00  0.00           C  \\nATOM    226  N   VAL A  29      29.572  17.543   3.261  1.00  0.00           N  \\nATOM    227  CA  VAL A  29      29.729  16.711   2.047  1.00  0.00           C  \\nATOM    228  C   VAL A  29      28.379  16.340   1.429  1.00  0.00           C  \\nATOM    229  O   VAL A  29      28.166  16.496   0.228  1.00  0.00           O  \\nATOM    230  CB  VAL A  29      30.649  15.492   2.359  1.00  0.00           C  \\nATOM    231  CG1 VAL A  29      30.782  14.596   1.136  1.00  0.00           C  \\nATOM    232  CG2 VAL A  29      32.010  16.050   2.772  1.00  0.00           C  \\nATOM    233  N   CYS A  30      27.501  15.906   2.299  1.00  0.00           N  \\nATOM    234  CA  CYS A  30      26.115  15.567   1.991  1.00  0.00           C  \\nATOM    235  C   CYS A  30      25.388  16.723   1.302  1.00  0.00           C  \\nATOM    236  O   CYS A  30      24.894  16.516   0.172  1.00  0.00           O  \\nATOM    237  CB  CYS A  30      25.343  15.046   3.172  1.00  0.00           C  \\nATOM    238  SG  CYS A  30      23.719  14.376   2.695  1.00  0.00           S  \\nATOM    239  N   ALA A  31      25.533  17.913   1.870  1.00  0.00           N  \\nATOM    240  CA  ALA A  31      24.949  19.130   1.305  1.00  0.00           C  \\nATOM    241  C   ALA A  31      25.487  19.354  -0.115  1.00  0.00           C  \\nATOM    242  O   ALA A  31      24.675  19.686  -0.998  1.00  0.00           O  \\nATOM    243  CB  ALA A  31      25.167  20.335   2.200  1.00  0.00           C  \\nATOM    244  N   ALA A  32      26.807  19.354  -0.286  1.00  0.00           N  \\nATOM    245  CA  ALA A  32      27.461  19.536  -1.579  1.00  0.00           C  \\nATOM    246  C   ALA A  32      26.943  18.538  -2.620  1.00  0.00           C  \\nATOM    247  O   ALA A  32      26.767  18.857  -3.789  1.00  0.00           O  \\nATOM    248  CB  ALA A  32      28.982  19.476  -1.398  1.00  0.00           C  \\nATOM    249  N   LYS A  33      26.731  17.303  -2.193  1.00  0.00           N  \\nATOM    250  CA  LYS A  33      26.261  16.216  -3.037  1.00  0.00           C  \\nATOM    251  C   LYS A  33      24.903  16.555  -3.658  1.00  0.00           C  \\nATOM    252  O   LYS A  33      24.806  16.511  -4.890  1.00  0.00           O  \\nATOM    253  CB  LYS A  33      26.221  14.860  -2.351  1.00  0.00           C  \\nATOM    254  CG  LYS A  33      25.697  13.696  -3.185  1.00  0.00           C  \\nATOM    255  CD  LYS A  33      26.498  13.400  -4.446  1.00  0.00           C  \\nATOM    256  CE  LYS A  33      25.686  12.464  -5.331  1.00  0.00           C  \\nATOM    257  NZ  LYS A  33      26.423  11.979  -6.525  1.00  0.00           N  \\nATOM    258  N   PHE A  34      23.972  16.946  -2.817  1.00  0.00           N  \\nATOM    259  CA  PHE A  34      22.569  17.125  -3.247  1.00  0.00           C  \\nATOM    260  C   PHE A  34      22.346  18.506  -3.836  1.00  0.00           C  \\nATOM    261  O   PHE A  34      21.504  18.692  -4.759  1.00  0.00           O  \\nATOM    262  CB  PHE A  34      21.626  16.673  -2.130  1.00  0.00           C  \\nATOM    263  CG  PHE A  34      21.644  15.172  -1.965  1.00  0.00           C  \\nATOM    264  CD1 PHE A  34      21.209  14.353  -3.007  1.00  0.00           C  \\nATOM    265  CD2 PHE A  34      22.272  14.627  -0.851  1.00  0.00           C  \\nATOM    266  CE1 PHE A  34      21.372  12.961  -2.910  1.00  0.00           C  \\nATOM    267  CE2 PHE A  34      22.443  13.245  -0.743  1.00  0.00           C  \\nATOM    268  CZ  PHE A  34      21.923  12.406  -1.737  1.00  0.00           C  \\nATOM    269  N   GLU A  35      23.251  19.415  -3.451  1.00  0.00           N  \\nATOM    270  CA  GLU A  35      23.178  20.789  -3.996  1.00  0.00           C  \\nATOM    271  C   GLU A  35      23.661  20.944  -5.423  1.00  0.00           C  \\nATOM    272  O   GLU A  35      23.014  21.514  -6.310  1.00  0.00           O  \\nATOM    273  CB  GLU A  35      23.698  21.892  -3.107  1.00  0.00           C  \\nATOM    274  CG  GLU A  35      22.994  22.212  -1.809  1.00  0.00           C  \\nATOM    275  CD  GLU A  35      21.631  22.846  -1.864  1.00  0.00           C  \\nATOM    276  OE1 GLU A  35      21.408  23.360  -2.981  1.00  0.00           O  \\nATOM    277  OE2 GLU A  35      20.947  23.032  -0.874  1.00  0.00           O  \\nATOM    278  N   SER A  36      24.867  20.483  -5.674  1.00  0.00           N  \\nATOM    279  CA  SER A  36      25.626  20.615  -6.903  1.00  0.00           C  \\nATOM    280  C   SER A  36      26.139  19.341  -7.569  1.00  0.00           C  \\nATOM    281  O   SER A  36      26.750  19.464  -8.642  1.00  0.00           O  \\nATOM    282  CB  SER A  36      26.830  21.528  -6.654  1.00  0.00           C  \\nATOM    283  OG  SER A  36      27.747  20.951  -5.748  1.00  0.00           O  \\nATOM    284  N   ASN A  37      25.957  18.222  -6.927  1.00  0.00           N  \\nATOM    285  CA  ASN A  37      26.628  16.968  -7.297  1.00  0.00           C  \\nATOM    286  C   ASN A  37      28.149  17.091  -7.302  1.00  0.00           C  \\nATOM    287  O   ASN A  37      28.813  16.627  -8.254  1.00  0.00           O  \\nATOM    288  CB  ASN A  37      26.028  16.490  -8.617  1.00  0.00           C  \\nATOM    289  CG  ASN A  37      26.156  14.980  -8.782  1.00  0.00           C  \\nATOM    290  OD1 ASN A  37      26.640  14.266  -7.885  1.00  0.00           O  \\nATOM    291  ND2 ASN A  37      25.867  14.506  -9.990  1.00  0.00           N  \\nATOM    292  N   PHE A  38      28.691  17.882  -6.383  1.00  0.00           N  \\nATOM    293  CA  PHE A  38      30.129  18.115  -6.279  1.00  0.00           C  \\nATOM    294  C   PHE A  38      30.717  18.903  -7.448  1.00  0.00           C  \\nATOM    295  O   PHE A  38      31.923  18.794  -7.702  1.00  0.00           O  \\nATOM    296  CB  PHE A  38      30.914  16.823  -6.047  1.00  0.00           C  \\nATOM    297  CG  PHE A  38      30.487  16.006  -4.863  1.00  0.00           C  \\nATOM    298  CD1 PHE A  38      30.100  16.572  -3.655  1.00  0.00           C  \\nATOM    299  CD2 PHE A  38      30.507  14.603  -4.993  1.00  0.00           C  \\nATOM    300  CE1 PHE A  38      29.766  15.808  -2.558  1.00  0.00           C  \\nATOM    301  CE2 PHE A  38      30.136  13.814  -3.891  1.00  0.00           C  \\nATOM    302  CZ  PHE A  38      29.835  14.410  -2.646  1.00  0.00           C  \\nATOM    303  N   ASN A  39      29.936  19.792  -8.033  1.00  0.00           N  \\nATOM    304  CA  ASN A  39      30.341  20.573  -9.199  1.00  0.00           C  \\nATOM    305  C   ASN A  39      30.543  22.034  -8.823  1.00  0.00           C  \\nATOM    306  O   ASN A  39      29.544  22.649  -8.423  1.00  0.00           O  \\nATOM    307  CB  ASN A  39      29.406  20.300 -10.366  1.00  0.00           C  \\nATOM    308  CG  ASN A  39      29.876  20.771 -11.716  1.00  0.00           C  \\nATOM    309  OD1 ASN A  39      30.579  21.750 -11.956  1.00  0.00           O  \\nATOM    310  ND2 ASN A  39      29.578  19.864 -12.653  1.00  0.00           N  \\nATOM    311  N   THR A  40      31.766  22.492  -8.928  1.00  0.00           N  \\nATOM    312  CA  THR A  40      32.139  23.869  -8.657  1.00  0.00           C  \\nATOM    313  C   THR A  40      31.504  24.890  -9.589  1.00  0.00           C  \\nATOM    314  O   THR A  40      31.316  26.054  -9.213  1.00  0.00           O  \\nATOM    315  CB  THR A  40      33.684  24.103  -8.422  1.00  0.00           C  \\nATOM    316  OG1 THR A  40      34.270  24.156  -9.775  1.00  0.00           O  \\nATOM    317  CG2 THR A  40      34.414  23.119  -7.491  1.00  0.00           C  \\nATOM    318  N   GLN A  41      31.001  24.430 -10.706  1.00  0.00           N  \\nATOM    319  CA  GLN A  41      30.524  25.291 -11.812  1.00  0.00           C  \\nATOM    320  C   GLN A  41      28.993  25.381 -11.874  1.00  0.00           C  \\nATOM    321  O   GLN A  41      28.504  26.019 -12.837  1.00  0.00           O  \\nATOM    322  CB  GLN A  41      31.047  24.817 -13.168  1.00  0.00           C  \\nATOM    323  CG  GLN A  41      32.549  25.004 -13.279  1.00  0.00           C  \\nATOM    324  CD  GLN A  41      32.763  26.236 -14.129  1.00  0.00           C  \\nATOM    325  OE1 GLN A  41      32.356  26.308 -15.291  1.00  0.00           O  \\nATOM    326  NE2 GLN A  41      33.276  27.231 -13.420  1.00  0.00           N  \\nATOM    327  N   ALA A  42      28.329  24.640 -11.012  1.00  0.00           N  \\nATOM    328  CA  ALA A  42      26.859  24.579 -11.061  1.00  0.00           C  \\nATOM    329  C   ALA A  42      26.257  25.969 -10.807  1.00  0.00           C  \\nATOM    330  O   ALA A  42      26.645  26.686  -9.884  1.00  0.00           O  \\nATOM    331  CB  ALA A  42      26.355  23.604  -9.998  1.00  0.00           C  \\nATOM    332  N   THR A  43      25.276  26.293 -11.643  1.00  0.00           N  \\nATOM    333  CA  THR A  43      24.441  27.490 -11.476  1.00  0.00           C  \\nATOM    334  C   THR A  43      22.976  27.112 -11.714  1.00  0.00           C  \\nATOM    335  O   THR A  43      22.715  26.271 -12.594  1.00  0.00           O  \\nATOM    336  CB  THR A  43      24.814  28.728 -12.375  1.00  0.00           C  \\nATOM    337  OG1 THR A  43      24.555  28.321 -13.756  1.00  0.00           O  \\nATOM    338  CG2 THR A  43      26.247  29.213 -12.184  1.00  0.00           C  \\nATOM    339  N   ASN A  44      22.088  27.742 -10.971  1.00  0.00           N  \\nATOM    340  CA  ASN A  44      20.640  27.497 -11.108  1.00  0.00           C  \\nATOM    341  C   ASN A  44      19.903  28.842 -10.963  1.00  0.00           C  \\nATOM    342  O   ASN A  44      20.169  29.600 -10.033  1.00  0.00           O  \\nATOM    343  CB  ASN A  44      20.155  26.480 -10.080  1.00  0.00           C  \\nATOM    344  CG  ASN A  44      18.646  26.315 -10.115  1.00  0.00           C  \\nATOM    345  OD1 ASN A  44      18.128  25.720 -11.078  1.00  0.00           O  \\nATOM    346  ND2 ASN A  44      17.906  26.927  -9.188  1.00  0.00           N  \\nATOM    347  N   ARG A  45      19.058  29.117 -11.924  1.00  0.00           N  \\nATOM    348  CA  ARG A  45      18.303  30.363 -12.015  1.00  0.00           C  \\nATOM    349  C   ARG A  45      16.955  30.163 -11.326  1.00  0.00           C  \\nATOM    350  O   ARG A  45      16.396  29.062 -11.425  1.00  0.00           O  \\nATOM    351  CB  ARG A  45      18.143  30.836 -13.462  1.00  0.00           C  \\nATOM    352  CG  ARG A  45      17.012  31.859 -13.557  1.00  0.00           C  \\nATOM    353  CD  ARG A  45      17.502  33.134 -12.930  1.00  0.00           C  \\nATOM    354  NE  ARG A  45      18.311  33.758 -13.981  1.00  0.00           N  \\nATOM    355  CZ  ARG A  45      17.620  34.271 -15.020  1.00  0.00           C  \\nATOM    356  NH1 ARG A  45      16.287  34.331 -15.098  1.00  0.00           N  \\nATOM    357  NH2 ARG A  45      18.374  34.698 -16.030  1.00  0.00           N  \\nATOM    358  N   ASN A  46      16.553  31.171 -10.554  1.00  0.00           N  \\nATOM    359  CA  ASN A  46      15.304  31.071  -9.782  1.00  0.00           C  \\nATOM    360  C   ASN A  46      14.261  32.063 -10.313  1.00  0.00           C  \\nATOM    361  O   ASN A  46      14.617  33.104 -10.880  1.00  0.00           O  \\nATOM    362  CB  ASN A  46      15.576  31.229  -8.295  1.00  0.00           C  \\nATOM    363  CG  ASN A  46      16.543  30.240  -7.679  1.00  0.00           C  \\nATOM    364  OD1 ASN A  46      17.659  30.661  -7.346  1.00  0.00           O  \\nATOM    365  ND2 ASN A  46      16.125  28.975  -7.600  1.00  0.00           N  \\nATOM    366  N   THR A  47      13.027  31.834  -9.887  1.00  0.00           N  \\nATOM    367  CA  THR A  47      11.871  32.654 -10.246  1.00  0.00           C  \\nATOM    368  C   THR A  47      12.001  34.107  -9.810  1.00  0.00           C  \\nATOM    369  O   THR A  47      11.600  34.969 -10.606  1.00  0.00           O  \\nATOM    370  CB  THR A  47      10.499  32.017  -9.789  1.00  0.00           C  \\nATOM    371  OG1 THR A  47      10.507  32.195  -8.342  1.00  0.00           O  \\nATOM    372  CG2 THR A  47      10.331  30.554 -10.188  1.00  0.00           C  \\nATOM    373  N   ASP A  48      12.625  34.377  -8.683  1.00  0.00           N  \\nATOM    374  CA  ASP A  48      12.885  35.716  -8.168  1.00  0.00           C  \\nATOM    375  C   ASP A  48      14.010  36.505  -8.823  1.00  0.00           C  \\nATOM    376  O   ASP A  48      14.246  37.621  -8.309  1.00  0.00           O  \\nATOM    377  CB  ASP A  48      13.023  35.735  -6.640  1.00  0.00           C  \\nATOM    378  CG  ASP A  48      14.367  35.174  -6.168  1.00  0.00           C  \\nATOM    379  OD1 ASP A  48      15.104  34.602  -6.991  1.00  0.00           O  \\nATOM    380  OD2 ASP A  48      14.750  35.442  -5.013  1.00  0.00           O  \\nATOM    381  N   GLY A  49      14.650  36.018  -9.862  1.00  0.00           N  \\nATOM    382  CA  GLY A  49      15.744  36.702 -10.546  1.00  0.00           C  \\nATOM    383  C   GLY A  49      17.147  36.315 -10.096  1.00  0.00           C  \\nATOM    384  O   GLY A  49      18.168  36.688 -10.694  1.00  0.00           O  \\nATOM    385  N   SER A  50      17.173  35.661  -8.930  1.00  0.00           N  \\nATOM    386  CA  SER A  50      18.435  35.225  -8.279  1.00  0.00           C  \\nATOM    387  C   SER A  50      18.977  33.963  -8.941  1.00  0.00           C  \\nATOM    388  O   SER A  50      18.273  33.255  -9.697  1.00  0.00           O  \\nATOM    389  CB  SER A  50      18.272  35.089  -6.781  1.00  0.00           C  \\nATOM    390  OG  SER A  50      17.530  33.930  -6.463  1.00  0.00           O  \\nATOM    391  N   THR A  51      20.271  33.766  -8.734  1.00  0.00           N  \\nATOM    392  CA  THR A  51      20.970  32.557  -9.202  1.00  0.00           C  \\nATOM    393  C   THR A  51      21.718  31.969  -8.000  1.00  0.00           C  \\nATOM    394  O   THR A  51      22.160  32.712  -7.119  1.00  0.00           O  \\nATOM    395  CB  THR A  51      21.904  32.908 -10.419  1.00  0.00           C  \\nATOM    396  OG1 THR A  51      21.099  33.576 -11.407  1.00  0.00           O  \\nATOM    397  CG2 THR A  51      22.686  31.699 -10.952  1.00  0.00           C  \\nATOM    398  N   ASP A  52      21.708  30.650  -7.927  1.00  0.00           N  \\nATOM    399  CA  ASP A  52      22.528  29.900  -6.959  1.00  0.00           C  \\nATOM    400  C   ASP A  52      23.843  29.488  -7.635  1.00  0.00           C  \\nATOM    401  O   ASP A  52      23.847  29.066  -8.805  1.00  0.00           O  \\nATOM    402  CB  ASP A  52      21.765  28.649  -6.554  1.00  0.00           C  \\nATOM    403  CG  ASP A  52      20.396  28.991  -6.002  1.00  0.00           C  \\nATOM    404  OD1 ASP A  52      20.220  29.928  -5.237  1.00  0.00           O  \\nATOM    405  OD2 ASP A  52      19.517  28.144  -6.222  1.00  0.00           O  \\nATOM    406  N   TYR A  53      24.924  29.734  -6.905  1.00  0.00           N  \\nATOM    407  CA  TYR A  53      26.278  29.604  -7.450  1.00  0.00           C  \\nATOM    408  C   TYR A  53      27.161  28.595  -6.708  1.00  0.00           C  \\nATOM    409  O   TYR A  53      27.289  28.606  -5.486  1.00  0.00           O  \\nATOM    410  CB  TYR A  53      26.993  30.972  -7.489  1.00  0.00           C  \\nATOM    411  CG  TYR A  53      26.437  31.959  -8.487  1.00  0.00           C  \\nATOM    412  CD1 TYR A  53      26.843  32.003  -9.821  1.00  0.00           C  \\nATOM    413  CD2 TYR A  53      25.510  32.907  -8.050  1.00  0.00           C  \\nATOM    414  CE1 TYR A  53      26.291  32.922 -10.717  1.00  0.00           C  \\nATOM    415  CE2 TYR A  53      24.907  33.803  -8.932  1.00  0.00           C  \\nATOM    416  CZ  TYR A  53      25.357  33.847 -10.252  1.00  0.00           C  \\nATOM    417  OH  TYR A  53      24.864  34.804 -11.094  1.00  0.00           O  \\nATOM    418  N   GLY A  54      27.751  27.721  -7.493  1.00  0.00           N  \\nATOM    419  CA  GLY A  54      28.845  26.832  -7.121  1.00  0.00           C  \\nATOM    420  C   GLY A  54      28.499  25.601  -6.290  1.00  0.00           C  \\nATOM    421  O   GLY A  54      27.339  25.155  -6.255  1.00  0.00           O  \\nATOM    422  N   ILE A  55      29.560  25.088  -5.684  1.00  0.00           N  \\nATOM    423  CA  ILE A  55      29.452  23.795  -4.994  1.00  0.00           C  \\nATOM    424  C   ILE A  55      28.424  23.748  -3.872  1.00  0.00           C  \\nATOM    425  O   ILE A  55      27.770  22.697  -3.693  1.00  0.00           O  \\nATOM    426  CB  ILE A  55      30.891  23.340  -4.563  1.00  0.00           C  \\nATOM    427  CG1 ILE A  55      30.785  21.812  -4.319  1.00  0.00           C  \\nATOM    428  CG2 ILE A  55      31.396  24.210  -3.390  1.00  0.00           C  \\nATOM    429  CD1 ILE A  55      32.101  21.018  -4.339  1.00  0.00           C  \\nATOM    430  N   LEU A  56      28.216  24.881  -3.236  1.00  0.00           N  \\nATOM    431  CA  LEU A  56      27.173  24.998  -2.203  1.00  0.00           C  \\nATOM    432  C   LEU A  56      25.981  25.838  -2.659  1.00  0.00           C  \\nATOM    433  O   LEU A  56      25.161  26.111  -1.774  1.00  0.00           O  \\nATOM    434  CB  LEU A  56      27.816  25.424  -0.877  1.00  0.00           C  \\nATOM    435  CG  LEU A  56      28.692  24.348  -0.204  1.00  0.00           C  \\nATOM    436  CD1 LEU A  56      29.331  24.997   1.008  1.00  0.00           C  \\nATOM    437  CD2 LEU A  56      27.808  23.177   0.192  1.00  0.00           C  \\nATOM    438  N   GLN A  57      25.865  26.104  -3.946  1.00  0.00           N  \\nATOM    439  CA  GLN A  57      24.668  26.785  -4.465  1.00  0.00           C  \\nATOM    440  C   GLN A  57      24.277  27.940  -3.558  1.00  0.00           C  \\nATOM    441  O   GLN A  57      23.116  28.031  -3.094  1.00  0.00           O  \\nATOM    442  CB  GLN A  57      23.521  25.765  -4.549  1.00  0.00           C  \\nATOM    443  CG  GLN A  57      23.733  24.790  -5.699  1.00  0.00           C  \\nATOM    444  CD  GLN A  57      23.684  25.440  -7.069  1.00  0.00           C  \\nATOM    445  OE1 GLN A  57      22.574  25.602  -7.591  1.00  0.00           O  \\nATOM    446  NE2 GLN A  57      24.813  25.805  -7.651  1.00  0.00           N  \\nATOM    447  N   ILE A  58      25.164  28.910  -3.428  1.00  0.00           N  \\nATOM    448  CA  ILE A  58      24.930  30.118  -2.649  1.00  0.00           C  \\nATOM    449  C   ILE A  58      24.204  31.163  -3.504  1.00  0.00           C  \\nATOM    450  O   ILE A  58      24.555  31.405  -4.667  1.00  0.00           O  \\nATOM    451  CB  ILE A  58      26.301  30.665  -2.134  1.00  0.00           C  \\nATOM    452  CG1 ILE A  58      26.751  29.724  -0.985  1.00  0.00           C  \\nATOM    453  CG2 ILE A  58      26.178  32.135  -1.693  1.00  0.00           C  \\nATOM    454  CD1 ILE A  58      28.246  29.954  -0.641  1.00  0.00           C  \\nATOM    455  N   ASN A  59      23.145  31.671  -2.905  1.00  0.00           N  \\nATOM    456  CA  ASN A  59      22.146  32.510  -3.590  1.00  0.00           C  \\nATOM    457  C   ASN A  59      22.550  33.974  -3.730  1.00  0.00           C  \\nATOM    458  O   ASN A  59      22.917  34.607  -2.740  1.00  0.00           O  \\nATOM    459  CB  ASN A  59      20.805  32.306  -2.884  1.00  0.00           C  \\nATOM    460  CG  ASN A  59      19.650  32.986  -3.588  1.00  0.00           C  \\nATOM    461  OD1 ASN A  59      19.464  34.171  -3.244  1.00  0.00           O  \\nATOM    462  ND2 ASN A  59      19.155  32.320  -4.612  1.00  0.00           N  \\nATOM    463  N   SER A  60      22.268  34.485  -4.941  1.00  0.00           N  \\nATOM    464  CA  SER A  60      22.570  35.910  -5.255  1.00  0.00           C  \\nATOM    465  C   SER A  60      21.687  36.993  -4.642  1.00  0.00           C  \\nATOM    466  O   SER A  60      22.054  38.177  -4.774  1.00  0.00           O  \\nATOM    467  CB  SER A  60      22.643  36.139  -6.750  1.00  0.00           C  \\nATOM    468  OG  SER A  60      21.432  36.007  -7.444  1.00  0.00           O  \\nATOM    469  N   ARG A  61      20.526  36.660  -4.138  1.00  0.00           N  \\nATOM    470  CA  ARG A  61      19.691  37.653  -3.434  1.00  0.00           C  \\nATOM    471  C   ARG A  61      20.277  38.108  -2.106  1.00  0.00           C  \\nATOM    472  O   ARG A  61      20.282  39.333  -1.876  1.00  0.00           O  \\nATOM    473  CB  ARG A  61      18.267  37.195  -3.250  1.00  0.00           C  \\nATOM    474  CG  ARG A  61      17.315  38.350  -2.959  1.00  0.00           C  \\nATOM    475  CD  ARG A  61      16.063  37.657  -2.503  1.00  0.00           C  \\nATOM    476  NE  ARG A  61      15.101  38.653  -2.063  1.00  0.00           N  \\nATOM    477  CZ  ARG A  61      13.794  38.351  -2.111  1.00  0.00           C  \\nATOM    478  NH1 ARG A  61      13.439  37.257  -2.784  1.00  0.00           N  \\nATOM    479  NH2 ARG A  61      12.925  39.249  -1.646  1.00  0.00           N  \\nATOM    480  N   TRP A  62      20.773  37.202  -1.273  1.00  0.00           N  \\nATOM    481  CA  TRP A  62      21.321  37.563   0.032  1.00  0.00           C  \\nATOM    482  C   TRP A  62      22.848  37.643   0.101  1.00  0.00           C  \\nATOM    483  O   TRP A  62      23.323  38.396   0.968  1.00  0.00           O  \\nATOM    484  CB  TRP A  62      20.833  36.611   1.134  1.00  0.00           C  \\nATOM    485  CG  TRP A  62      19.360  36.361   1.096  1.00  0.00           C  \\nATOM    486  CD1 TRP A  62      18.719  35.247   0.643  1.00  0.00           C  \\nATOM    487  CD2 TRP A  62      18.326  37.305   1.427  1.00  0.00           C  \\nATOM    488  NE1 TRP A  62      17.360  35.457   0.609  1.00  0.00           N  \\nATOM    489  CE2 TRP A  62      17.090  36.696   1.096  1.00  0.00           C  \\nATOM    490  CE3 TRP A  62      18.333  38.584   1.965  1.00  0.00           C  \\nATOM    491  CZ2 TRP A  62      15.875  37.327   1.307  1.00  0.00           C  \\nATOM    492  CZ3 TRP A  62      17.115  39.208   2.186  1.00  0.00           C  \\nATOM    493  CH2 TRP A  62      15.906  38.611   1.814  1.00  0.00           C  \\nATOM    494  N   TRP A  63      23.537  36.731  -0.584  1.00  0.00           N  \\nATOM    495  CA  TRP A  63      24.906  36.398  -0.276  1.00  0.00           C  \\nATOM    496  C   TRP A  63      26.089  36.924  -1.052  1.00  0.00           C  \\nATOM    497  O   TRP A  63      27.142  37.177  -0.432  1.00  0.00           O  \\nATOM    498  CB  TRP A  63      25.068  34.919   0.112  1.00  0.00           C  \\nATOM    499  CG  TRP A  63      24.036  34.428   1.068  1.00  0.00           C  \\nATOM    500  CD1 TRP A  63      22.959  33.620   0.777  1.00  0.00           C  \\nATOM    501  CD2 TRP A  63      23.919  34.728   2.450  1.00  0.00           C  \\nATOM    502  NE1 TRP A  63      22.243  33.344   1.899  1.00  0.00           N  \\nATOM    503  CE2 TRP A  63      22.761  34.067   2.931  1.00  0.00           C  \\nATOM    504  CE3 TRP A  63      24.694  35.480   3.323  1.00  0.00           C  \\nATOM    505  CZ2 TRP A  63      22.393  34.133   4.278  1.00  0.00           C  \\nATOM    506  CZ3 TRP A  63      24.302  35.586   4.648  1.00  0.00           C  \\nATOM    507  CH2 TRP A  63      23.179  34.910   5.117  1.00  0.00           C  \\nATOM    508  N   CYS A  64      25.962  36.914  -2.353  1.00  0.00           N  \\nATOM    509  CA  CYS A  64      26.985  37.433  -3.271  1.00  0.00           C  \\nATOM    510  C   CYS A  64      26.324  38.355  -4.286  1.00  0.00           C  \\nATOM    511  O   CYS A  64      25.102  38.314  -4.475  1.00  0.00           O  \\nATOM    512  CB  CYS A  64      27.638  36.265  -3.988  1.00  0.00           C  \\nATOM    513  SG  CYS A  64      26.562  35.233  -5.007  1.00  0.00           S  \\nATOM    514  N   ASN A  65      27.157  39.165  -4.908  1.00  0.00           N  \\nATOM    515  CA  ASN A  65      26.700  40.134  -5.920  1.00  0.00           C  \\nATOM    516  C   ASN A  65      26.985  39.646  -7.342  1.00  0.00           C  \\nATOM    517  O   ASN A  65      28.130  39.316  -7.647  1.00  0.00           O  \\nATOM    518  CB  ASN A  65      27.381  41.492  -5.712  1.00  0.00           C  \\nATOM    519  CG  ASN A  65      26.910  42.423  -6.824  1.00  0.00           C  \\nATOM    520  OD1 ASN A  65      25.736  42.559  -7.141  1.00  0.00           O  \\nATOM    521  ND2 ASN A  65      27.914  42.938  -7.527  1.00  0.00           N  \\nATOM    522  N   ASP A  66      25.920  39.484  -8.116  1.00  0.00           N  \\nATOM    523  CA  ASP A  66      26.102  39.180  -9.545  1.00  0.00           C  \\nATOM    524  C   ASP A  66      25.664  40.317 -10.460  1.00  0.00           C  \\nATOM    525  O   ASP A  66      25.719  40.143 -11.673  1.00  0.00           O  \\nATOM    526  CB  ASP A  66      25.462  37.858  -9.894  1.00  0.00           C  \\nATOM    527  CG  ASP A  66      23.951  37.903  -9.833  1.00  0.00           C  \\nATOM    528  OD1 ASP A  66      23.288  38.898  -9.542  1.00  0.00           O  \\nATOM    529  OD2 ASP A  66      23.455  36.769 -10.048  1.00  0.00           O  \\nATOM    530  N   GLY A  67      25.234  41.408  -9.860  1.00  0.00           N  \\nATOM    531  CA  GLY A  67      24.817  42.607 -10.577  1.00  0.00           C  \\nATOM    532  C   GLY A  67      23.544  42.553 -11.401  1.00  0.00           C  \\nATOM    533  O   GLY A  67      23.191  43.542 -12.055  1.00  0.00           O  \\nATOM    534  N   ARG A  68      22.822  41.441 -11.348  1.00  0.00           N  \\nATOM    535  CA  ARG A  68      21.560  41.308 -12.092  1.00  0.00           C  \\nATOM    536  C   ARG A  68      20.424  40.765 -11.243  1.00  0.00           C  \\nATOM    537  O   ARG A  68      19.385  40.383 -11.795  1.00  0.00           O  \\nATOM    538  CB  ARG A  68      21.746  40.454 -13.339  1.00  0.00           C  \\nATOM    539  CG  ARG A  68      22.197  39.048 -12.946  1.00  0.00           C  \\nATOM    540  CD  ARG A  68      22.477  38.187 -14.122  1.00  0.00           C  \\nATOM    541  NE  ARG A  68      21.439  37.201 -14.281  1.00  0.00           N  \\nATOM    542  CZ  ARG A  68      20.242  37.255 -14.855  1.00  0.00           C  \\nATOM    543  NH1 ARG A  68      19.503  38.311 -15.145  1.00  0.00           N  \\nATOM    544  NH2 ARG A  68      19.897  36.127 -15.497  1.00  0.00           N  \\nATOM    545  N   THR A  69      20.590  40.896  -9.952  1.00  0.00           N  \\nATOM    546  CA  THR A  69      19.607  40.408  -8.973  1.00  0.00           C  \\nATOM    547  C   THR A  69      18.994  41.597  -8.224  1.00  0.00           C  \\nATOM    548  O   THR A  69      19.683  42.241  -7.425  1.00  0.00           O  \\nATOM    549  CB  THR A  69      20.208  39.384  -7.910  1.00  0.00           C  \\nATOM    550  OG1 THR A  69      20.851  38.334  -8.715  1.00  0.00           O  \\nATOM    551  CG2 THR A  69      19.123  38.863  -6.955  1.00  0.00           C  \\nATOM    552  N   PRO A  70      17.687  41.741  -8.418  1.00  0.00           N  \\nATOM    553  CA  PRO A  70      16.953  42.824  -7.733  1.00  0.00           C  \\nATOM    554  C   PRO A  70      16.955  42.520  -6.242  1.00  0.00           C  \\nATOM    555  O   PRO A  70      16.739  41.356  -5.886  1.00  0.00           O  \\nATOM    556  CB  PRO A  70      15.557  42.749  -8.333  1.00  0.00           C  \\nATOM    557  CG  PRO A  70      15.759  42.092  -9.671  1.00  0.00           C  \\nATOM    558  CD  PRO A  70      16.887  41.095  -9.463  1.00  0.00           C  \\nATOM    559  N   GLY A  71      17.163  43.493  -5.401  1.00  0.00           N  \\nATOM    560  CA  GLY A  71      17.135  43.383  -3.947  1.00  0.00           C  \\nATOM    561  C   GLY A  71      18.328  42.710  -3.287  1.00  0.00           C  \\nATOM    562  O   GLY A  71      18.257  42.257  -2.126  1.00  0.00           O  \\nATOM    563  N   SER A  72      19.418  42.627  -4.018  1.00  0.00           N  \\nATOM    564  CA  SER A  72      20.665  41.976  -3.588  1.00  0.00           C  \\nATOM    565  C   SER A  72      21.139  42.582  -2.276  1.00  0.00           C  \\nATOM    566  O   SER A  72      21.055  43.819  -2.125  1.00  0.00           O  \\nATOM    567  CB  SER A  72      21.659  42.002  -4.737  1.00  0.00           C  \\nATOM    568  OG  SER A  72      23.015  41.835  -4.343  1.00  0.00           O  \\nATOM    569  N   ARG A  73      21.646  41.775  -1.369  1.00  0.00           N  \\nATOM    570  CA  ARG A  73      22.254  42.196  -0.113  1.00  0.00           C  \\nATOM    571  C   ARG A  73      23.774  42.058   0.029  1.00  0.00           C  \\nATOM    572  O   ARG A  73      24.425  42.749   0.849  1.00  0.00           O  \\nATOM    573  CB  ARG A  73      21.615  41.555   1.127  1.00  0.00           C  \\nATOM    574  CG  ARG A  73      20.187  41.983   1.439  1.00  0.00           C  \\nATOM    575  CD  ARG A  73      20.209  43.299   2.123  1.00  0.00           C  \\nATOM    576  NE  ARG A  73      18.928  43.771   2.617  1.00  0.00           N  \\nATOM    577  CZ  ARG A  73      17.980  44.302   1.823  1.00  0.00           C  \\nATOM    578  NH1 ARG A  73      17.841  43.934   0.544  1.00  0.00           N  \\nATOM    579  NH2 ARG A  73      17.571  45.545   2.146  1.00  0.00           N  \\nATOM    580  N   ASN A  74      24.346  41.143  -0.705  1.00  0.00           N  \\nATOM    581  CA  ASN A  74      25.780  40.833  -0.695  1.00  0.00           C  \\nATOM    582  C   ASN A  74      26.315  40.718   0.731  1.00  0.00           C  \\nATOM    583  O   ASN A  74      27.255  41.471   1.060  1.00  0.00           O  \\nATOM    584  CB  ASN A  74      26.565  41.786  -1.589  1.00  0.00           C  \\nATOM    585  CG  ASN A  74      27.982  41.328  -1.909  1.00  0.00           C  \\nATOM    586  OD1 ASN A  74      28.318  40.169  -1.652  1.00  0.00           O  \\nATOM    587  ND2 ASN A  74      28.838  42.192  -2.436  1.00  0.00           N  \\nATOM    588  N   LEU A  75      25.723  39.860   1.544  1.00  0.00           N  \\nATOM    589  CA  LEU A  75      26.153  39.690   2.930  1.00  0.00           C  \\nATOM    590  C   LEU A  75      27.518  39.011   3.085  1.00  0.00           C  \\nATOM    591  O   LEU A  75      28.167  39.197   4.141  1.00  0.00           O  \\nATOM    592  CB  LEU A  75      25.009  39.055   3.733  1.00  0.00           C  \\nATOM    593  CG  LEU A  75      23.815  39.998   3.979  1.00  0.00           C  \\nATOM    594  CD1 LEU A  75      22.574  39.193   4.336  1.00  0.00           C  \\nATOM    595  CD2 LEU A  75      24.156  40.969   5.110  1.00  0.00           C  \\nATOM    596  N   CYS A  76      27.973  38.312   2.061  1.00  0.00           N  \\nATOM    597  CA  CYS A  76      29.299  37.680   2.111  1.00  0.00           C  \\nATOM    598  C   CYS A  76      30.390  38.598   1.589  1.00  0.00           C  \\nATOM    599  O   CYS A  76      31.573  38.229   1.568  1.00  0.00           O  \\nATOM    600  CB  CYS A  76      29.396  36.303   1.477  1.00  0.00           C  \\nATOM    601  SG  CYS A  76      28.595  34.961   2.361  1.00  0.00           S  \\nATOM    602  N   ASN A  77      29.964  39.681   0.971  1.00  0.00           N  \\nATOM    603  CA  ASN A  77      30.827  40.725   0.395  1.00  0.00           C  \\nATOM    604  C   ASN A  77      31.711  40.163  -0.702  1.00  0.00           C  \\nATOM    605  O   ASN A  77      32.947  40.331  -0.689  1.00  0.00           O  \\nATOM    606  CB  ASN A  77      31.589  41.488   1.483  1.00  0.00           C  \\nATOM    607  CG  ASN A  77      32.126  42.812   0.957  1.00  0.00           C  \\nATOM    608  OD1 ASN A  77      31.351  43.589   0.396  1.00  0.00           O  \\nATOM    609  ND2 ASN A  77      33.430  43.012   1.047  1.00  0.00           N  \\nATOM    610  N   ILE A  78      31.149  39.408  -1.616  1.00  0.00           N  \\nATOM    611  CA  ILE A  78      31.904  38.780  -2.713  1.00  0.00           C  \\nATOM    612  C   ILE A  78      31.099  38.850  -4.013  1.00  0.00           C  \\nATOM    613  O   ILE A  78      29.864  38.688  -3.972  1.00  0.00           O  \\nATOM    614  CB  ILE A  78      32.183  37.255  -2.384  1.00  0.00           C  \\nATOM    615  CG1 ILE A  78      30.882  36.639  -1.851  1.00  0.00           C  \\nATOM    616  CG2 ILE A  78      33.437  37.003  -1.524  1.00  0.00           C  \\nATOM    617  CD1 ILE A  78      30.936  35.120  -1.546  1.00  0.00           C  \\nATOM    618  N   PRO A  79      31.828  38.795  -5.108  1.00  0.00           N  \\nATOM    619  CA  PRO A  79      31.195  38.634  -6.421  1.00  0.00           C  \\nATOM    620  C   PRO A  79      30.792  37.146  -6.446  1.00  0.00           C  \\nATOM    621  O   PRO A  79      31.576  36.273  -6.005  1.00  0.00           O  \\nATOM    622  CB  PRO A  79      32.261  38.967  -7.445  1.00  0.00           C  \\nATOM    623  CG  PRO A  79      33.555  39.029  -6.710  1.00  0.00           C  \\nATOM    624  CD  PRO A  79      33.276  39.023  -5.215  1.00  0.00           C  \\nATOM    625  N   CYS A  80      29.629  36.908  -7.033  1.00  0.00           N  \\nATOM    626  CA  CYS A  80      29.167  35.522  -7.236  1.00  0.00           C  \\nATOM    627  C   CYS A  80      30.150  34.716  -8.073  1.00  0.00           C  \\nATOM    628  O   CYS A  80      30.214  33.484  -7.880  1.00  0.00           O  \\nATOM    629  CB  CYS A  80      27.749  35.338  -7.747  1.00  0.00           C  \\nATOM    630  SG  CYS A  80      26.471  36.160  -6.762  1.00  0.00           S  \\nATOM    631  N   SER A  81      30.769  35.294  -9.083  1.00  0.00           N  \\nATOM    632  CA  SER A  81      31.775  34.671  -9.933  1.00  0.00           C  \\nATOM    633  C   SER A  81      32.907  34.027  -9.109  1.00  0.00           C  \\nATOM    634  O   SER A  81      33.338  32.939  -9.561  1.00  0.00           O  \\nATOM    635  CB  SER A  81      32.381  35.643 -10.953  1.00  0.00           C  \\nATOM    636  OG  SER A  81      33.035  36.681 -10.235  1.00  0.00           O  \\nATOM    637  N   ALA A  82      33.226  34.572  -7.950  1.00  0.00           N  \\nATOM    638  CA  ALA A  82      34.272  33.963  -7.107  1.00  0.00           C  \\nATOM    639  C   ALA A  82      33.869  32.604  -6.540  1.00  0.00           C  \\nATOM    640  O   ALA A  82      34.703  31.798  -6.081  1.00  0.00           O  \\nATOM    641  CB  ALA A  82      34.722  34.930  -6.020  1.00  0.00           C  \\nATOM    642  N   LEU A  83      32.571  32.338  -6.519  1.00  0.00           N  \\nATOM    643  CA  LEU A  83      31.973  31.093  -6.042  1.00  0.00           C  \\nATOM    644  C   LEU A  83      32.060  29.962  -7.049  1.00  0.00           C  \\nATOM    645  O   LEU A  83      31.671  28.815  -6.739  1.00  0.00           O  \\nATOM    646  CB  LEU A  83      30.618  31.424  -5.431  1.00  0.00           C  \\nATOM    647  CG  LEU A  83      30.511  32.364  -4.244  1.00  0.00           C  \\nATOM    648  CD1 LEU A  83      29.040  32.573  -3.857  1.00  0.00           C  \\nATOM    649  CD2 LEU A  83      31.277  31.872  -3.020  1.00  0.00           C  \\nATOM    650  N   LEU A  84      32.473  30.271  -8.267  1.00  0.00           N  \\nATOM    651  CA  LEU A  84      32.610  29.295  -9.347  1.00  0.00           C  \\nATOM    652  C   LEU A  84      34.047  28.841  -9.570  1.00  0.00           C  \\nATOM    653  O   LEU A  84      34.334  28.243 -10.622  1.00  0.00           O  \\nATOM    654  CB  LEU A  84      31.914  29.793 -10.614  1.00  0.00           C  \\nATOM    655  CG  LEU A  84      30.446  30.183 -10.573  1.00  0.00           C  \\nATOM    656  CD1 LEU A  84      30.014  30.771 -11.916  1.00  0.00           C  \\nATOM    657  CD2 LEU A  84      29.597  28.942 -10.329  1.00  0.00           C  \\nATOM    658  N   SER A  85      34.942  29.197  -8.676  1.00  0.00           N  \\nATOM    659  CA  SER A  85      36.340  28.754  -8.727  1.00  0.00           C  \\nATOM    660  C   SER A  85      36.474  27.245  -8.497  1.00  0.00           C  \\nATOM    661  O   SER A  85      35.765  26.640  -7.681  1.00  0.00           O  \\nATOM    662  CB  SER A  85      37.109  29.482  -7.633  1.00  0.00           C  \\nATOM    663  OG  SER A  85      38.484  29.212  -7.834  1.00  0.00           O  \\nATOM    664  N   SER A  86      37.619  26.742  -8.943  1.00  0.00           N  \\nATOM    665  CA  SER A  86      38.008  25.343  -8.700  1.00  0.00           C  \\nATOM    666  C   SER A  86      38.388  25.153  -7.233  1.00  0.00           C  \\nATOM    667  O   SER A  86      38.314  24.055  -6.671  1.00  0.00           O  \\nATOM    668  CB  SER A  86      39.107  24.868  -9.650  1.00  0.00           C  \\nATOM    669  OG  SER A  86      38.401  24.215 -10.691  1.00  0.00           O  \\nATOM    670  N   ASP A  87      38.846  26.244  -6.668  1.00  0.00           N  \\nATOM    671  CA  ASP A  87      39.282  26.393  -5.271  1.00  0.00           C  \\nATOM    672  C   ASP A  87      37.999  26.657  -4.471  1.00  0.00           C  \\nATOM    673  O   ASP A  87      37.490  27.762  -4.712  1.00  0.00           O  \\nATOM    674  CB  ASP A  87      40.221  27.607  -5.212  1.00  0.00           C  \\nATOM    675  CG  ASP A  87      40.762  28.041  -3.869  1.00  0.00           C  \\nATOM    676  OD1 ASP A  87      40.335  27.702  -2.742  1.00  0.00           O  \\nATOM    677  OD2 ASP A  87      41.785  28.802  -3.933  1.00  0.00           O  \\nATOM    678  N   ILE A  88      37.732  25.850  -3.461  1.00  0.00           N  \\nATOM    679  CA  ILE A  88      36.515  26.089  -2.658  1.00  0.00           C  \\nATOM    680  C   ILE A  88      36.582  27.142  -1.563  1.00  0.00           C  \\nATOM    681  O   ILE A  88      35.600  27.315  -0.801  1.00  0.00           O  \\nATOM    682  CB  ILE A  88      35.993  24.736  -2.046  1.00  0.00           C  \\nATOM    683  CG1 ILE A  88      36.920  24.200  -0.934  1.00  0.00           C  \\nATOM    684  CG2 ILE A  88      35.587  23.743  -3.163  1.00  0.00           C  \\nATOM    685  CD1 ILE A  88      36.363  23.226   0.137  1.00  0.00           C  \\nATOM    686  N   THR A  89      37.742  27.786  -1.369  1.00  0.00           N  \\nATOM    687  CA  THR A  89      37.913  28.772  -0.306  1.00  0.00           C  \\nATOM    688  C   THR A  89      36.736  29.761  -0.161  1.00  0.00           C  \\nATOM    689  O   THR A  89      36.341  29.983   1.000  1.00  0.00           O  \\nATOM    690  CB  THR A  89      39.267  29.591  -0.423  1.00  0.00           C  \\nATOM    691  OG1 THR A  89      40.339  28.592  -0.514  1.00  0.00           O  \\nATOM    692  CG2 THR A  89      39.419  30.553   0.766  1.00  0.00           C  \\nATOM    693  N   ALA A  90      36.507  30.482  -1.242  1.00  0.00           N  \\nATOM    694  CA  ALA A  90      35.479  31.555  -1.244  1.00  0.00           C  \\nATOM    695  C   ALA A  90      34.125  31.066  -0.735  1.00  0.00           C  \\nATOM    696  O   ALA A  90      33.366  31.671   0.065  1.00  0.00           O  \\nATOM    697  CB  ALA A  90      35.366  32.212  -2.617  1.00  0.00           C  \\nATOM    698  N   SER A  91      33.746  29.919  -1.296  1.00  0.00           N  \\nATOM    699  CA  SER A  91      32.494  29.220  -1.024  1.00  0.00           C  \\nATOM    700  C   SER A  91      32.429  28.817   0.467  1.00  0.00           C  \\nATOM    701  O   SER A  91      31.407  29.057   1.110  1.00  0.00           O  \\nATOM    702  CB  SER A  91      32.282  28.035  -1.917  1.00  0.00           C  \\nATOM    703  OG  SER A  91      32.020  28.335  -3.260  1.00  0.00           O  \\nATOM    704  N   VAL A  92      33.502  28.211   0.953  1.00  0.00           N  \\nATOM    705  CA  VAL A  92      33.595  27.846   2.381  1.00  0.00           C  \\nATOM    706  C   VAL A  92      33.477  29.105   3.261  1.00  0.00           C  \\nATOM    707  O   VAL A  92      32.716  29.020   4.244  1.00  0.00           O  \\nATOM    708  CB  VAL A  92      34.890  27.038   2.650  1.00  0.00           C  \\nATOM    709  CG1 VAL A  92      35.125  26.987   4.151  1.00  0.00           C  \\nATOM    710  CG2 VAL A  92      34.776  25.672   2.013  1.00  0.00           C  \\nATOM    711  N   ASN A  93      34.199  30.163   2.912  1.00  0.00           N  \\nATOM    712  CA  ASN A  93      34.160  31.413   3.685  1.00  0.00           C  \\nATOM    713  C   ASN A  93      32.747  32.034   3.803  1.00  0.00           C  \\nATOM    714  O   ASN A  93      32.361  32.393   4.908  1.00  0.00           O  \\nATOM    715  CB  ASN A  93      35.168  32.472   3.247  1.00  0.00           C  \\nATOM    716  CG  ASN A  93      36.582  32.083   3.642  1.00  0.00           C  \\nATOM    717  OD1 ASN A  93      37.546  32.671   3.120  1.00  0.00           O  \\nATOM    718  ND2 ASN A  93      36.710  31.188   4.631  1.00  0.00           N  \\nATOM    719  N   CYS A  94      32.096  31.996   2.670  1.00  0.00           N  \\nATOM    720  CA  CYS A  94      30.695  32.449   2.633  1.00  0.00           C  \\nATOM    721  C   CYS A  94      29.778  31.527   3.424  1.00  0.00           C  \\nATOM    722  O   CYS A  94      28.973  32.025   4.236  1.00  0.00           O  \\nATOM    723  CB  CYS A  94      30.291  32.704   1.185  1.00  0.00           C  \\nATOM    724  SG  CYS A  94      28.644  33.431   1.046  1.00  0.00           S  \\nATOM    725  N   ALA A  95      29.945  30.225   3.263  1.00  0.00           N  \\nATOM    726  CA  ALA A  95      29.134  29.227   3.969  1.00  0.00           C  \\nATOM    727  C   ALA A  95      29.236  29.330   5.491  1.00  0.00           C  \\nATOM    728  O   ALA A  95      28.222  29.180   6.179  1.00  0.00           O  \\nATOM    729  CB  ALA A  95      29.495  27.794   3.612  1.00  0.00           C  \\nATOM    730  N   LYS A  96      30.424  29.628   5.987  1.00  0.00           N  \\nATOM    731  CA  LYS A  96      30.627  29.912   7.419  1.00  0.00           C  \\nATOM    732  C   LYS A  96      29.802  31.115   7.906  1.00  0.00           C  \\nATOM    733  O   LYS A  96      29.296  31.011   9.027  1.00  0.00           O  \\nATOM    734  CB  LYS A  96      32.100  30.112   7.775  1.00  0.00           C  \\nATOM    735  CG  LYS A  96      32.874  28.792   7.697  1.00  0.00           C  \\nATOM    736  CD  LYS A  96      34.358  29.071   7.879  1.00  0.00           C  \\nATOM    737  CE  LYS A  96      35.205  27.816   7.938  1.00  0.00           C  \\nATOM    738  NZ  LYS A  96      36.610  28.242   8.169  1.00  0.00           N  \\nATOM    739  N   LYS A  97      29.660  32.101   7.049  1.00  0.00           N  \\nATOM    740  CA  LYS A  97      28.811  33.263   7.379  1.00  0.00           C  \\nATOM    741  C   LYS A  97      27.333  32.905   7.386  1.00  0.00           C  \\nATOM    742  O   LYS A  97      26.570  33.318   8.272  1.00  0.00           O  \\nATOM    743  CB  LYS A  97      29.087  34.456   6.470  1.00  0.00           C  \\nATOM    744  CG  LYS A  97      30.537  34.927   6.563  1.00  0.00           C  \\nATOM    745  CD  LYS A  97      30.841  36.073   5.610  1.00  0.00           C  \\nATOM    746  CE  LYS A  97      32.340  36.293   5.510  1.00  0.00           C  \\nATOM    747  NZ  LYS A  97      32.569  37.524   4.708  1.00  0.00           N  \\nATOM    748  N   ILE A  98      26.897  32.127   6.416  1.00  0.00           N  \\nATOM    749  CA  ILE A  98      25.520  31.693   6.266  1.00  0.00           C  \\nATOM    750  C   ILE A  98      25.030  30.871   7.453  1.00  0.00           C  \\nATOM    751  O   ILE A  98      24.059  31.254   8.126  1.00  0.00           O  \\nATOM    752  CB  ILE A  98      25.307  30.997   4.896  1.00  0.00           C  \\nATOM    753  CG1 ILE A  98      25.643  31.910   3.702  1.00  0.00           C  \\nATOM    754  CG2 ILE A  98      23.887  30.386   4.817  1.00  0.00           C  \\nATOM    755  CD1 ILE A  98      25.620  31.204   2.321  1.00  0.00           C  \\nATOM    756  N   VAL A  99      25.875  29.966   7.915  1.00  0.00           N  \\nATOM    757  CA  VAL A  99      25.488  29.009   8.955  1.00  0.00           C  \\nATOM    758  C   VAL A  99      25.431  29.697  10.316  1.00  0.00           C  \\nATOM    759  O   VAL A  99      24.737  29.212  11.233  1.00  0.00           O  \\nATOM    760  CB  VAL A  99      26.398  27.774   8.859  1.00  0.00           C  \\nATOM    761  CG1 VAL A  99      27.811  28.138   9.279  1.00  0.00           C  \\nATOM    762  CG2 VAL A  99      25.834  26.594   9.632  1.00  0.00           C  \\nATOM    763  N   SER A 100      26.205  30.752  10.409  1.00  0.00           N  \\nATOM    764  CA  SER A 100      26.297  31.529  11.650  1.00  0.00           C  \\nATOM    765  C   SER A 100      25.124  32.495  11.765  1.00  0.00           C  \\nATOM    766  O   SER A 100      24.995  33.131  12.820  1.00  0.00           O  \\nATOM    767  CB  SER A 100      27.647  32.194  11.723  1.00  0.00           C  \\nATOM    768  OG  SER A 100      28.714  31.264  11.818  1.00  0.00           O  \\nATOM    769  N   ASP A 101      24.307  32.599  10.750  1.00  0.00           N  \\nATOM    770  CA  ASP A 101      23.162  33.495  10.650  1.00  0.00           C  \\nATOM    771  C   ASP A 101      21.924  33.183  11.481  1.00  0.00           C  \\nATOM    772  O   ASP A 101      21.132  34.143  11.678  1.00  0.00           O  \\nATOM    773  CB  ASP A 101      22.806  33.854   9.207  1.00  0.00           C  \\nATOM    774  CG  ASP A 101      22.426  35.320   9.009  1.00  0.00           C  \\nATOM    775  OD1 ASP A 101      23.248  36.223   9.266  1.00  0.00           O  \\nATOM    776  OD2 ASP A 101      21.276  35.519   8.551  1.00  0.00           O  \\nATOM    777  N   GLY A 102      21.723  31.942  11.887  1.00  0.00           N  \\nATOM    778  CA  GLY A 102      20.622  31.656  12.823  1.00  0.00           C  \\nATOM    779  C   GLY A 102      19.812  30.413  12.505  1.00  0.00           C  \\nATOM    780  O   GLY A 102      19.195  29.898  13.458  1.00  0.00           O  \\nATOM    781  N   ASN A 103      19.805  30.005  11.244  1.00  0.00           N  \\nATOM    782  CA  ASN A 103      19.076  28.799  10.848  1.00  0.00           C  \\nATOM    783  C   ASN A 103      19.990  27.600  10.597  1.00  0.00           C  \\nATOM    784  O   ASN A 103      19.472  26.579  10.117  1.00  0.00           O  \\nATOM    785  CB  ASN A 103      17.976  29.046   9.824  1.00  0.00           C  \\nATOM    786  CG  ASN A 103      16.708  28.245  10.063  1.00  0.00           C  \\nATOM    787  OD1 ASN A 103      16.513  27.577  11.098  1.00  0.00           O  \\nATOM    788  ND2 ASN A 103      15.672  28.428   9.240  1.00  0.00           N  \\nATOM    789  N   GLY A 104      21.240  27.711  10.997  1.00  0.00           N  \\nATOM    790  CA  GLY A 104      22.177  26.567  10.860  1.00  0.00           C  \\nATOM    791  C   GLY A 104      22.173  26.142   9.388  1.00  0.00           C  \\nATOM    792  O   GLY A 104      22.151  27.040   8.541  1.00  0.00           O  \\nATOM    793  N   MET A 105      22.185  24.847   9.115  1.00  0.00           N  \\nATOM    794  CA  MET A 105      22.278  24.370   7.724  1.00  0.00           C  \\nATOM    795  C   MET A 105      20.915  24.309   7.063  1.00  0.00           C  \\nATOM    796  O   MET A 105      20.902  23.921   5.871  1.00  0.00           O  \\nATOM    797  CB  MET A 105      23.036  23.048   7.632  1.00  0.00           C  \\nATOM    798  CG  MET A 105      24.519  23.221   7.871  1.00  0.00           C  \\nATOM    799  SD  MET A 105      25.379  21.660   7.491  1.00  0.00           S  \\nATOM    800  CE  MET A 105      25.369  21.706   5.703  1.00  0.00           C  \\nATOM    801  N   ASN A 106      19.896  24.839   7.735  1.00  0.00           N  \\nATOM    802  CA  ASN A 106      18.557  24.898   7.103  1.00  0.00           C  \\nATOM    803  C   ASN A 106      18.566  25.831   5.892  1.00  0.00           C  \\nATOM    804  O   ASN A 106      17.678  25.748   5.021  1.00  0.00           O  \\nATOM    805  CB  ASN A 106      17.420  25.150   8.084  1.00  0.00           C  \\nATOM    806  CG  ASN A 106      17.207  23.938   8.967  1.00  0.00           C  \\nATOM    807  OD1 ASN A 106      16.836  22.867   8.443  1.00  0.00           O  \\nATOM    808  ND2 ASN A 106      17.542  24.047  10.255  1.00  0.00           N  \\nATOM    809  N   ALA A 107      19.617  26.647   5.800  1.00  0.00           N  \\nATOM    810  CA  ALA A 107      19.824  27.514   4.634  1.00  0.00           C  \\nATOM    811  C   ALA A 107      20.007  26.744   3.329  1.00  0.00           C  \\nATOM    812  O   ALA A 107      19.734  27.292   2.260  1.00  0.00           O  \\nATOM    813  CB  ALA A 107      21.060  28.374   4.877  1.00  0.00           C  \\nATOM    814  N   TRP A 108      20.371  25.465   3.434  1.00  0.00           N  \\nATOM    815  CA  TRP A 108      20.532  24.624   2.241  1.00  0.00           C  \\nATOM    816  C   TRP A 108      19.317  23.688   2.110  1.00  0.00           C  \\nATOM    817  O   TRP A 108      19.240  22.816   2.993  1.00  0.00           O  \\nATOM    818  CB  TRP A 108      21.840  23.826   2.338  1.00  0.00           C  \\nATOM    819  CG  TRP A 108      23.000  24.736   2.026  1.00  0.00           C  \\nATOM    820  CD1 TRP A 108      23.360  25.234   0.803  1.00  0.00           C  \\nATOM    821  CD2 TRP A 108      23.798  25.437   2.991  1.00  0.00           C  \\nATOM    822  NE1 TRP A 108      24.414  26.102   0.952  1.00  0.00           N  \\nATOM    823  CE2 TRP A 108      24.711  26.237   2.272  1.00  0.00           C  \\nATOM    824  CE3 TRP A 108      23.833  25.426   4.379  1.00  0.00           C  \\nATOM    825  CZ2 TRP A 108      25.682  26.985   2.917  1.00  0.00           C  \\nATOM    826  CZ3 TRP A 108      24.773  26.200   5.033  1.00  0.00           C  \\nATOM    827  CH2 TRP A 108      25.691  26.960   4.298  1.00  0.00           C  \\nATOM    828  N   VAL A 109      18.487  23.925   1.101  1.00  0.00           N  \\nATOM    829  CA  VAL A 109      17.265  23.096   1.025  1.00  0.00           C  \\nATOM    830  C   VAL A 109      17.573  21.611   0.884  1.00  0.00           C  \\nATOM    831  O   VAL A 109      16.923  20.811   1.587  1.00  0.00           O  \\nATOM    832  CB  VAL A 109      16.146  23.660   0.145  1.00  0.00           C  \\nATOM    833  CG1 VAL A 109      16.607  23.905  -1.285  1.00  0.00           C  \\nATOM    834  CG2 VAL A 109      14.901  22.783   0.136  1.00  0.00           C  \\nATOM    835  N   ALA A 110      18.562  21.308   0.075  1.00  0.00           N  \\nATOM    836  CA  ALA A 110      18.972  19.935  -0.237  1.00  0.00           C  \\nATOM    837  C   ALA A 110      19.502  19.244   1.006  1.00  0.00           C  \\nATOM    838  O   ALA A 110      19.197  18.046   1.174  1.00  0.00           O  \\nATOM    839  CB  ALA A 110      19.857  19.777  -1.455  1.00  0.00           C  \\nATOM    840  N   TRP A 111      20.121  19.983   1.892  1.00  0.00           N  \\nATOM    841  CA  TRP A 111      20.551  19.500   3.202  1.00  0.00           C  \\nATOM    842  C   TRP A 111      19.343  19.085   4.046  1.00  0.00           C  \\nATOM    843  O   TRP A 111      19.284  17.986   4.643  1.00  0.00           O  \\nATOM    844  CB  TRP A 111      21.486  20.451   3.925  1.00  0.00           C  \\nATOM    845  CG  TRP A 111      21.858  19.904   5.252  1.00  0.00           C  \\nATOM    846  CD1 TRP A 111      22.856  18.975   5.486  1.00  0.00           C  \\nATOM    847  CD2 TRP A 111      21.199  20.093   6.504  1.00  0.00           C  \\nATOM    848  NE1 TRP A 111      22.848  18.592   6.808  1.00  0.00           N  \\nATOM    849  CE2 TRP A 111      21.798  19.221   7.435  1.00  0.00           C  \\nATOM    850  CE3 TRP A 111      20.182  20.959   6.908  1.00  0.00           C  \\nATOM    851  CZ2 TRP A 111      21.492  19.284   8.784  1.00  0.00           C  \\nATOM    852  CZ3 TRP A 111      19.818  20.954   8.240  1.00  0.00           C  \\nATOM    853  CH2 TRP A 111      20.443  20.110   9.162  1.00  0.00           C  \\nATOM    854  N   ARG A 112      18.438  20.038   4.228  1.00  0.00           N  \\nATOM    855  CA  ARG A 112      17.185  19.830   4.955  1.00  0.00           C  \\nATOM    856  C   ARG A 112      16.472  18.593   4.391  1.00  0.00           C  \\nATOM    857  O   ARG A 112      16.000  17.792   5.220  1.00  0.00           O  \\nATOM    858  CB  ARG A 112      16.214  20.990   4.944  1.00  0.00           C  \\nATOM    859  CG  ARG A 112      16.592  22.438   5.091  1.00  0.00           C  \\nATOM    860  CD  ARG A 112      15.368  23.296   5.135  1.00  0.00           C  \\nATOM    861  NE  ARG A 112      14.776  23.555   3.836  1.00  0.00           N  \\nATOM    862  CZ  ARG A 112      14.785  24.695   3.143  1.00  0.00           C  \\nATOM    863  NH1 ARG A 112      15.596  25.721   3.383  1.00  0.00           N  \\nATOM    864  NH2 ARG A 112      13.717  25.008   2.398  1.00  0.00           N  \\nATOM    865  N   ASN A 113      16.307  18.521   3.074  1.00  0.00           N  \\nATOM    866  CA  ASN A 113      15.464  17.463   2.491  1.00  0.00           C  \\nATOM    867  C   ASN A 113      16.111  16.083   2.586  1.00  0.00           C  \\nATOM    868  O   ASN A 113      15.454  15.054   2.808  1.00  0.00           O  \\nATOM    869  CB  ASN A 113      14.966  17.779   1.085  1.00  0.00           C  \\nATOM    870  CG  ASN A 113      14.003  18.961   1.018  1.00  0.00           C  \\nATOM    871  OD1 ASN A 113      13.454  19.344   2.059  1.00  0.00           O  \\nATOM    872  ND2 ASN A 113      13.840  19.555  -0.159  1.00  0.00           N  \\nATOM    873  N   ARG A 114      17.401  16.041   2.355  1.00  0.00           N  \\nATOM    874  CA  ARG A 114      18.156  14.836   2.042  1.00  0.00           C  \\nATOM    875  C   ARG A 114      19.258  14.440   2.991  1.00  0.00           C  \\nATOM    876  O   ARG A 114      19.505  13.217   3.097  1.00  0.00           O  \\nATOM    877  CB  ARG A 114      18.622  14.910   0.576  1.00  0.00           C  \\nATOM    878  CG  ARG A 114      17.395  14.614  -0.300  1.00  0.00           C  \\nATOM    879  CD  ARG A 114      17.729  14.399  -1.731  1.00  0.00           C  \\nATOM    880  NE  ARG A 114      18.153  15.677  -2.301  1.00  0.00           N  \\nATOM    881  CZ  ARG A 114      17.826  16.080  -3.535  1.00  0.00           C  \\nATOM    882  NH1 ARG A 114      17.378  15.225  -4.456  1.00  0.00           N  \\nATOM    883  NH2 ARG A 114      17.735  17.388  -3.796  1.00  0.00           N  \\nATOM    884  N   CYS A 115      19.743  15.372   3.773  1.00  0.00           N  \\nATOM    885  CA  CYS A 115      20.843  15.041   4.708  1.00  0.00           C  \\nATOM    886  C   CYS A 115      20.448  14.933   6.159  1.00  0.00           C  \\nATOM    887  O   CYS A 115      20.972  14.116   6.940  1.00  0.00           O  \\nATOM    888  CB  CYS A 115      21.991  16.018   4.426  1.00  0.00           C  \\nATOM    889  SG  CYS A 115      22.563  16.009   2.739  1.00  0.00           S  \\nATOM    890  N   LYS A 116      19.714  15.918   6.619  1.00  0.00           N  \\nATOM    891  CA  LYS A 116      19.332  16.085   8.025  1.00  0.00           C  \\nATOM    892  C   LYS A 116      18.634  14.821   8.518  1.00  0.00           C  \\nATOM    893  O   LYS A 116      17.819  14.234   7.785  1.00  0.00           O  \\nATOM    894  CB  LYS A 116      18.492  17.363   8.126  1.00  0.00           C  \\nATOM    895  CG  LYS A 116      17.930  17.512   9.547  1.00  0.00           C  \\nATOM    896  CD  LYS A 116      16.745  18.481   9.554  1.00  0.00           C  \\nATOM    897  CE  LYS A 116      16.658  19.147  10.918  1.00  0.00           C  \\nATOM    898  NZ  LYS A 116      15.454  20.010  11.047  1.00  0.00           N  \\nATOM    899  N   GLY A 117      19.152  14.318   9.635  1.00  0.00           N  \\nATOM    900  CA  GLY A 117      18.558  13.126  10.267  1.00  0.00           C  \\nATOM    901  C   GLY A 117      19.018  11.781   9.733  1.00  0.00           C  \\nATOM    902  O   GLY A 117      18.499  10.733  10.164  1.00  0.00           O  \\nATOM    903  N   THR A 118      19.892  11.802   8.740  1.00  0.00           N  \\nATOM    904  CA  THR A 118      20.473  10.578   8.171  1.00  0.00           C  \\nATOM    905  C   THR A 118      21.868  10.375   8.761  1.00  0.00           C  \\nATOM    906  O   THR A 118      22.321  11.119   9.650  1.00  0.00           O  \\nATOM    907  CB  THR A 118      20.440  10.571   6.598  1.00  0.00           C  \\nATOM    908  OG1 THR A 118      21.560  11.404   6.161  1.00  0.00           O  \\nATOM    909  CG2 THR A 118      19.095  11.104   6.074  1.00  0.00           C  \\nATOM    910  N   ASP A 119      22.392   9.213   8.431  1.00  0.00           N  \\nATOM    911  CA  ASP A 119      23.768   8.830   8.756  1.00  0.00           C  \\nATOM    912  C   ASP A 119      24.713   9.543   7.779  1.00  0.00           C  \\nATOM    913  O   ASP A 119      25.178   8.950   6.780  1.00  0.00           O  \\nATOM    914  CB  ASP A 119      23.934   7.313   8.738  1.00  0.00           C  \\nATOM    915  CG  ASP A 119      25.347   6.900   9.121  1.00  0.00           C  \\nATOM    916  OD1 ASP A 119      26.051   7.633   9.830  1.00  0.00           O  \\nATOM    917  OD2 ASP A 119      25.715   5.746   8.804  1.00  0.00           O  \\nATOM    918  N   VAL A 120      24.988  10.793   8.094  1.00  0.00           N  \\nATOM    919  CA  VAL A 120      25.886  11.637   7.306  1.00  0.00           C  \\nATOM    920  C   VAL A 120      27.342  11.199   7.304  1.00  0.00           C  \\nATOM    921  O   VAL A 120      28.099  11.604   6.406  1.00  0.00           O  \\nATOM    922  CB  VAL A 120      25.721  13.134   7.630  1.00  0.00           C  \\nATOM    923  CG1 VAL A 120      24.356  13.655   7.183  1.00  0.00           C  \\nATOM    924  CG2 VAL A 120      26.088  13.478   9.055  1.00  0.00           C  \\nATOM    925  N   GLN A 121      27.701  10.430   8.306  1.00  0.00           N  \\nATOM    926  CA  GLN A 121      29.021   9.783   8.389  1.00  0.00           C  \\nATOM    927  C   GLN A 121      29.317   8.861   7.207  1.00  0.00           C  \\nATOM    928  O   GLN A 121      30.480   8.669   6.820  1.00  0.00           O  \\nATOM    929  CB  GLN A 121      29.088   9.048   9.728  1.00  0.00           C  \\nATOM    930  CG  GLN A 121      30.530   8.965  10.167  1.00  0.00           C  \\nATOM    931  CD  GLN A 121      30.615   8.877  11.674  1.00  0.00           C  \\nATOM    932  OE1 GLN A 121      31.368   9.632  12.283  1.00  0.00           O  \\nATOM    933  NE2 GLN A 121      29.884   7.871  12.151  1.00  0.00           N  \\nATOM    934  N   ALA A 122      28.300   8.271   6.576  1.00  0.00           N  \\nATOM    935  CA  ALA A 122      28.390   7.548   5.311  1.00  0.00           C  \\nATOM    936  C   ALA A 122      29.186   8.290   4.227  1.00  0.00           C  \\nATOM    937  O   ALA A 122      30.031   7.710   3.523  1.00  0.00           O  \\nATOM    938  CB  ALA A 122      26.999   7.244   4.783  1.00  0.00           C  \\nATOM    939  N   TRP A 123      29.021   9.616   4.240  1.00  0.00           N  \\nATOM    940  CA  TRP A 123      29.703  10.502   3.283  1.00  0.00           C  \\nATOM    941  C   TRP A 123      31.218  10.581   3.392  1.00  0.00           C  \\nATOM    942  O   TRP A 123      31.905  10.942   2.412  1.00  0.00           O  \\nATOM    943  CB  TRP A 123      29.027  11.872   3.347  1.00  0.00           C  \\nATOM    944  CG  TRP A 123      27.621  11.735   2.850  1.00  0.00           C  \\nATOM    945  CD1 TRP A 123      26.485  11.608   3.588  1.00  0.00           C  \\nATOM    946  CD2 TRP A 123      27.241  11.547   1.481  1.00  0.00           C  \\nATOM    947  NE1 TRP A 123      25.405  11.458   2.774  1.00  0.00           N  \\nATOM    948  CE2 TRP A 123      25.827  11.391   1.479  1.00  0.00           C  \\nATOM    949  CE3 TRP A 123      27.947  11.468   0.283  1.00  0.00           C  \\nATOM    950  CZ2 TRP A 123      25.111  11.184   0.311  1.00  0.00           C  \\nATOM    951  CZ3 TRP A 123      27.222  11.329  -0.886  1.00  0.00           C  \\nATOM    952  CH2 TRP A 123      25.835  11.127  -0.869  1.00  0.00           C  \\nATOM    953  N   ILE A 124      31.741  10.269   4.554  1.00  0.00           N  \\nATOM    954  CA  ILE A 124      33.186  10.292   4.801  1.00  0.00           C  \\nATOM    955  C   ILE A 124      33.863   8.955   5.024  1.00  0.00           C  \\nATOM    956  O   ILE A 124      35.100   8.892   5.134  1.00  0.00           O  \\nATOM    957  CB  ILE A 124      33.504  11.403   5.863  1.00  0.00           C  \\nATOM    958  CG1 ILE A 124      32.956  10.984   7.234  1.00  0.00           C  \\nATOM    959  CG2 ILE A 124      33.024  12.804   5.387  1.00  0.00           C  \\nATOM    960  CD1 ILE A 124      33.729  11.437   8.488  1.00  0.00           C  \\nATOM    961  N   ARG A 125      33.080   7.898   5.131  1.00  0.00           N  \\nATOM    962  CA  ARG A 125      33.594   6.534   5.320  1.00  0.00           C  \\nATOM    963  C   ARG A 125      34.311   6.113   4.036  1.00  0.00           C  \\nATOM    964  O   ARG A 125      33.843   6.337   2.906  1.00  0.00           O  \\nATOM    965  CB  ARG A 125      32.476   5.551   5.650  1.00  0.00           C  \\nATOM    966  CG  ARG A 125      32.117   5.596   7.145  1.00  0.00           C  \\nATOM    967  CD  ARG A 125      31.277   4.392   7.482  1.00  0.00           C  \\nATOM    968  NE  ARG A 125      30.282   4.754   8.466  1.00  0.00           N  \\nATOM    969  CZ  ARG A 125      28.984   4.993   8.331  1.00  0.00           C  \\nATOM    970  NH1 ARG A 125      28.334   4.690   7.207  1.00  0.00           N  \\nATOM    971  NH2 ARG A 125      28.392   5.549   9.392  1.00  0.00           N  \\nATOM    972  N   GLY A 126      35.497   5.568   4.243  1.00  0.00           N  \\nATOM    973  CA  GLY A 126      36.291   5.058   3.102  1.00  0.00           C  \\nATOM    974  C   GLY A 126      37.334   6.066   2.658  1.00  0.00           C  \\nATOM    975  O   GLY A 126      38.220   5.729   1.855  1.00  0.00           O  \\nATOM    976  N   CYS A 127      37.335   7.221   3.297  1.00  0.00           N  \\nATOM    977  CA  CYS A 127      38.234   8.333   2.961  1.00  0.00           C  \\nATOM    978  C   CYS A 127      39.422   8.382   3.925  1.00  0.00           C  \\nATOM    979  O   CYS A 127      39.206   8.267   5.138  1.00  0.00           O  \\nATOM    980  CB  CYS A 127      37.453   9.628   2.990  1.00  0.00           C  \\nATOM    981  SG  CYS A 127      36.010   9.816   1.936  1.00  0.00           S  \\nATOM    982  N   ARG A 128      40.586   8.695   3.393  1.00  0.00           N  \\nATOM    983  CA  ARG A 128      41.774   8.960   4.217  1.00  0.00           C  \\nATOM    984  C   ARG A 128      41.820  10.438   4.578  1.00  0.00           C  \\nATOM    985  O   ARG A 128      41.976  11.291   3.694  1.00  0.00           O  \\nATOM    986  CB  ARG A 128      43.047   8.304   3.707  1.00  0.00           C  \\nATOM    987  CG  ARG A 128      43.231   6.886   4.280  1.00  0.00           C  \\nATOM    988  CD  ARG A 128      43.833   6.911   5.651  1.00  0.00           C  \\nATOM    989  NE  ARG A 128      45.246   7.263   5.636  1.00  0.00           N  \\nATOM    990  CZ  ARG A 128      45.862   8.258   6.281  1.00  0.00           C  \\nATOM    991  NH1 ARG A 128      45.241   9.069   7.151  1.00  0.00           N  \\nATOM    992  NH2 ARG A 128      47.134   8.554   5.973  1.00  0.00           N  \\nATOM    993  N   LEU A 129      41.289  10.715   5.771  1.00  0.00           N  \\nATOM    994  CA  LEU A 129      41.094  12.084   6.273  1.00  0.00           C  \\nATOM    995  C   LEU A 129      42.119  12.382   7.370  1.00  0.00           C  \\nATOM    996  O   LEU A 129      41.730  12.276   8.559  1.00  0.00           O  \\nATOM    997  CB  LEU A 129      39.635  12.335   6.646  1.00  0.00           C  \\nATOM    998  CG  LEU A 129      38.689  12.917   5.620  1.00  0.00           C  \\nATOM    999  CD1 LEU A 129      39.112  12.657   4.191  1.00  0.00           C  \\nATOM   1000  CD2 LEU A 129      37.310  12.325   5.886  1.00  0.00           C  \\nATOM   1001  OXT LEU A 129      43.232  12.675   6.905  1.00  0.00           O  \\nHETATM 1002  O   HOH A 130      23.434  40.063  -6.661  1.00  0.00           O  \\nHETATM 1003  O   HOH A 131      31.994  26.416  -6.047  1.00  0.00           O  \\nHETATM 1004  O   HOH A 132      30.250  13.337   9.787  1.00  0.00           O  \\nHETATM 1005  O   HOH A 133      22.384  42.331  -8.165  1.00  0.00           O  \\nHETATM 1006  O   HOH A 134      29.239  27.621  -3.670  1.00  0.00           O  \\nHETATM 1007  O   HOH A 135      29.464  37.761 -10.492  1.00  0.00           O  \\nHETATM 1008  O   HOH A 136      20.807  36.305 -11.082  1.00  0.00           O  \\nHETATM 1009  O   HOH A 137      41.318  17.849  -1.378  1.00  0.00           O  \\nHETATM 1010  O   HOH A 138      34.697  29.056  -4.039  1.00  0.00           O  \\nHETATM 1011  O   HOH A 139      26.871  17.298  13.496  1.00  0.00           O  \\nHETATM 1012  O   HOH A 140      32.131  11.050  -5.817  1.00  0.00           O  \\nHETATM 1013  O   HOH A 141      23.468  40.040  -2.372  1.00  0.00           O  \\nHETATM 1014  O   HOH A 142      21.390  45.524 -11.035  1.00  0.00           O  \\nHETATM 1015  O   HOH A 143      34.490  26.578  -5.741  1.00  0.00           O  \\nHETATM 1016  O   HOH A 144      16.422  34.139  -3.527  1.00  0.00           O  \\nHETATM 1017  O   HOH A 145      21.374  29.926   8.946  1.00  0.00           O  \\nHETATM 1018  O   HOH A 146      41.048  12.539  -0.011  1.00  0.00           O  \\nHETATM 1019  O   HOH A 147      32.794  35.686   2.558  1.00  0.00           O  \\nHETATM 1020  O   HOH A 148      49.648   8.964   6.343  1.00  0.00           O  \\nHETATM 1021  O   HOH A 149      14.452  34.901 -13.339  1.00  0.00           O  \\nHETATM 1022  O   HOH A 150      22.930  10.839   4.044  1.00  0.00           O  \\nHETATM 1023  O   HOH A 151      16.012  18.490  -2.200  1.00  0.00           O  \\nHETATM 1024  O   HOH A 152      12.130  21.587   3.044  1.00  0.00           O  \\nHETATM 1025  O   HOH A 153      15.684  38.922  -5.813  1.00  0.00           O  \\nHETATM 1026  O   HOH A 154      10.652  24.228   3.428  1.00  0.00           O  \\nHETATM 1027  O   HOH A 155      44.070  17.975   2.852  1.00  0.00           O  \\nHETATM 1028  O   HOH A 156      32.029  13.080  -8.110  1.00  0.00           O  \\nHETATM 1029  O   HOH A 157      36.425  19.613  15.174  1.00  0.00           O  \\nHETATM 1030  O   HOH A 158      37.941  30.505  -3.686  1.00  0.00           O  \\nHETATM 1031  O   HOH A 159      30.710  42.741  -6.289  1.00  0.00           O  \\nHETATM 1032  O   HOH A 160      23.922  44.367  -7.653  1.00  0.00           O  \\nHETATM 1033  O   HOH A 161      33.829  34.252   0.626  1.00  0.00           O  \\nHETATM 1034  O   HOH A 162      29.613  40.730  -9.602  1.00  0.00           O  \\nHETATM 1035  O   HOH A 163      23.563   7.995   4.406  1.00  0.00           O  \\nHETATM 1036  O   HOH A 164      31.511  42.362  -4.183  1.00  0.00           O  \\nHETATM 1037  O   HOH A 165      21.882  29.536 -15.013  1.00  0.00           O  \\nHETATM 1038  O   HOH A 166      37.763  20.913   9.782  1.00  0.00           O  \\nHETATM 1039  O   HOH A 167      42.338  17.481   5.165  1.00  0.00           O  \\nHETATM 1040  O   HOH A 168      23.344  39.739  -4.358  1.00  0.00           O  \\nHETATM 1041  O   HOH A 169      22.984  29.224  13.124  1.00  0.00           O  \\nHETATM 1042  O   HOH A 170      30.778   7.794  -3.514  1.00  0.00           O  \\nHETATM 1043  O   HOH A 171      42.965  14.657   4.991  1.00  0.00           O  \\nHETATM 1044  O   HOH A 172      36.927  17.948 -13.093  1.00  0.00           O  \\nHETATM 1045  O   HOH A 173      35.412  25.852 -11.575  1.00  0.00           O  \\nHETATM 1046  O   HOH A 174      37.428  32.540  -5.787  1.00  0.00           O  \\nHETATM 1047  O   HOH A 175      37.317   8.592   7.456  1.00  0.00           O  \\nHETATM 1048  O   HOH A 176       9.314  36.705 -11.546  1.00  0.00           O  \\nHETATM 1049  O   HOH A 177      39.972  23.760  -2.655  1.00  0.00           O  \\nHETATM 1050  O   HOH A 178      22.128  30.274  -0.543  1.00  0.00           O  \\nHETATM 1051  O   HOH A 179      22.244  15.813  10.000  1.00  0.00           O  \\nHETATM 1052  O   HOH A 180      40.729   9.223   0.292  1.00  0.00           O  \\nHETATM 1053  O   HOH A 181      12.500  15.267   4.097  1.00  0.00           O  \\nHETATM 1054  O   HOH A 182      20.372  28.618  -2.353  1.00  0.00           O  \\nHETATM 1055  O   HOH A 183      22.793  15.462  -6.673  1.00  0.00           O  \\nHETATM 1056  O   HOH A 184      23.138  31.809  15.121  1.00  0.00           O  \\nHETATM 1057  O   HOH A 185      22.671  38.691   8.245  1.00  0.00           O  \\nHETATM 1058  O   HOH A 186      33.966  33.112   6.837  1.00  0.00           O  \\nHETATM 1059  O   HOH A 187      19.572  25.423  -1.420  1.00  0.00           O  \\nHETATM 1060  O   HOH A 188      14.790  15.672   7.259  1.00  0.00           O  \\nHETATM 1061  O   HOH A 189      19.112  28.022 -14.647  1.00  0.00           O  \\nHETATM 1062  O   HOH A 190      17.302  39.059 -12.453  1.00  0.00           O  \\nHETATM 1063  O   HOH A 191      16.198  14.502   5.577  1.00  0.00           O  \\nHETATM 1064  O   HOH A 192      17.345  46.346  -7.080  1.00  0.00           O  \\nHETATM 1065  O   HOH A 193      14.992  31.300  -4.242  1.00  0.00           O  \\nHETATM 1066  O   HOH A 194      28.196  44.775  -3.148  1.00  0.00           O  \\nHETATM 1067  O   HOH A 195      29.479  13.863  -9.107  1.00  0.00           O  \\nHETATM 1068  O   HOH A 196      23.613  44.811   2.608  1.00  0.00           O  \\nHETATM 1069  O   HOH A 197      40.572  22.184  -6.358  1.00  0.00           O  \\nHETATM 1070  O   HOH A 198      12.475  31.860  -6.226  1.00  0.00           O  \\nHETATM 1071  O   HOH A 199      16.684  13.594  -5.832  1.00  0.00           O  \\nHETATM 1072  O   HOH A 200      27.534  38.059 -12.862  1.00  0.00           O  \\nHETATM 1073  O   HOH A 201      25.892  35.973  11.563  1.00  0.00           O  \\nHETATM 1074  O   HOH A 202      24.790  25.182  16.063  1.00  0.00           O  \\nHETATM 1075  O   HOH A 203      12.580  21.214   5.006  1.00  0.00           O  \\nHETATM 1076  O   HOH A 204      19.687  23.750  -4.851  1.00  0.00           O  \\nHETATM 1077  O   HOH A 205      27.098  35.956 -12.358  1.00  0.00           O  \\nHETATM 1078  O   HOH A 206      37.255   9.634  10.002  1.00  0.00           O  \\nHETATM 1079  O   HOH A 207      43.755  23.843   8.038  1.00  0.00           O  \\nTER    1080      HOH A 207\\nCONECT   48  981\\nCONECT  238  889\\nCONECT  513  630\\nCONECT  601  724\\nCONECT  630  513\\nCONECT  724  601\\nCONECT  889  238\\nCONECT  981   48\\nEND\\n\",\"pdb\");\n",
              "\tviewer_16678367835488749.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
              "\tviewer_16678367835488749.zoomTo();\n",
              "viewer_16678367835488749.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<py3Dmol.view at 0x7f14a0a26510>"
            ]
          },
          "metadata": {},
          "execution_count": 23
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Add mutants"
      ],
      "metadata": {
        "id": "q7W8ZdcpFI5X"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Read mutation file and prepare alignments\n",
        "alphabet = list(Bio.SeqUtils.IUPACData.protein_letters_1to3.keys())\n",
        "alphabet.append(\"-\")\n",
        "\n",
        "parser = PDBParser()\n",
        "structure = parser.get_structure('X', path+pdbid+\"_processed.pdb\")\n",
        "\n",
        "residues = \"\"\n",
        "resids = []\n",
        "for model in structure:\n",
        "    for chain in model:\n",
        "        for i, residue in enumerate(chain.get_residues()):\n",
        "            residue_1l = Bio.SeqUtils.seq1(residue.resname)\n",
        "            resid = residue.id[1]\n",
        "            if residue_1l != \"X\":\n",
        "                residues += residue_1l\n",
        "                resids.append(resid)\n",
        "\n",
        "record = SeqRecord(Seq(residues), id=\"0\", name=\"template\", \n",
        "                   description=\"input protein\")\n",
        "\n",
        "mutant_n = 0\n",
        "with open(mutations_file, \"r\") as f:\n",
        "    for line in f:\n",
        "        if line.strip().startswith(\"#\") or line.strip() == \"\":\n",
        "            continue\n",
        "        \n",
        "        if not os.path.exists(f\"md_mutant_{mutant_n}\"):\n",
        "            os.mkdir(f\"md_mutant_{mutant_n}\")\n",
        "        \n",
        "        new_sequence = residues\n",
        "        for mutation in line.split():\n",
        "            original_AA = mutation[0].upper()\n",
        "            if original_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "            \n",
        "            mutant_AA = mutation[-1].upper()\n",
        "            if mutant_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided for in\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "                \n",
        "            try:\n",
        "                resid = int(mutation[1:-1])\n",
        "            except:\n",
        "                print(f\"Can't get resid for mutation {mutation}. Check you input\")\n",
        "                raise\n",
        "            \n",
        "            if residues[resids.index(resid)] != original_AA:\n",
        "                raise Exception(f\"In the PDB file resid {resid} has amino acid\"+\\\n",
        "                                f\" {residues[resids.index(resid)]} while in\"+\\\n",
        "                                f\"mutation file {original_AA} is provide\")\n",
        "            new_sequence = list(new_sequence)\n",
        "            new_sequence[resids.index(resid)] = mutant_AA\n",
        "            new_sequence = \"\".join(new_sequence)\n",
        "\n",
        "        record_mutant = SeqRecord(Seq(new_sequence), id=\"1\", name=\"mutant\", \n",
        "                description=\"mutated protein\")\n",
        "        handle = open(f\"md_mutant_{mutant_n}/alignment.ali\", \"w\")\n",
        "        writer = Bio.SeqIO.PirIO.PirWriter(handle)\n",
        "        writer.write_record(record)\n",
        "        writer.write_record(record_mutant)\n",
        "        handle.close()\n",
        "        mutant_n += 1\n",
        "\n",
        "os.environ['N_MUTANT'] = str(mutant_n)\n",
        "os.environ['N_RESID'] = str(len(residues))"
      ],
      "metadata": {
        "id": "VoylKPKIFMgU"
      },
      "execution_count": 63,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!head md_mutant_0/alignment.ali"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "e3QMz2A94hb4",
        "outputId": "727ef8cc-66f3-4c5a-c787-e760f6aa4eee"
      },
      "execution_count": 65,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">P1;0\n",
            "structureX:/content/md_1aki/1aki_no_water:1:A:129:A::::\n",
            "KVFGRCELAAAMKRHGLDNYRGYSLGNWVCAAKFESNFNTQATNRNTDGSTDYGILQINS\n",
            "RWWCNDGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGNGMNAWVAWRNRCKGTDV\n",
            "QAWIRGCRL*\n",
            ">P1;1\n",
            "sequence:mutated protein:1::129:::::\n",
            "KVFGRCELAAAMKRHGLDNARGYSLGNWVCAAKFESNFNTQATNRNTDGSTDYGILQINS\n",
            "RWWCNDGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGNGMNAWVAWRNRCKGTDV\n",
            "QAWIRGCRL*\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Correct naming for alignment file\n",
        "%%bash\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "sed -i 's/XX/P1/g' md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/template - input protein/structureX:\\/content\\/md_${PDB}\\/${PDB}_no_water:1:A:${N_RESID}:A::::/g\" md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/mutant - mutated protein/sequence:mutated protein:1::${N_RESID}:::::/g\" md_mutant_${i}/alignment.ali\n",
        "done"
      ],
      "metadata": {
        "id": "W1IGNECJ1LyS"
      },
      "execution_count": 64,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Run modeller\n",
        "for i in range(mutant_n):\n",
        "    os.chdir(f\"/content/md_mutant_{i}\")\n",
        "    env = Environ()\n",
        "    a = AutoModel(env, alnfile='alignment.ali',\n",
        "                  knowns='0', sequence='1',\n",
        "                  assess_methods=(assess.DOPE,\n",
        "                                  #soap_protein_od.Scorer(),\n",
        "                                  assess.GA341))\n",
        "    a.starting_model = 1\n",
        "    a.ending_model = 1\n",
        "    a.make()\n",
        "    for file in os.listdir(\".\"):\n",
        "        if file.endswith(\"pdb\"):\n",
        "            shutil.copy2(file, 'input.pdb')\n",
        "os.chdir(\"/content\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aTlUZ6aqNLwy",
        "outputId": "393d3052-29ae-4219-c13f-2710b6f698b5"
      },
      "execution_count": 67,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "check_ali___> Checking the sequence-structure alignment. \n",
            "\n",
            "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
            "\n",
            "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
            "----------------------------------------------\n",
            "END OF TABLE\n",
            "read_to_681_> topology.submodel read from topology file:        3\n",
            "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
            "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
            "              least one known structure available. MDT, not library, potential is used.\n",
            "0 atoms in HETATM/BLK residues constrained\n",
            "to protein atoms within 2.30 angstroms\n",
            "and protein CA atoms within 10.00 angstroms\n",
            "0 atoms in residues without defined topology\n",
            "constrained to be rigid bodies\n",
            "condens_443_> Restraints marked for deletion were removed.\n",
            "              Total number of restraints before, now:    13074    12204\n",
            ">> Model assessment by DOPE potential\n",
            "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      129\n",
            "Number of all, selected real atoms                :      994     994\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    12204   12204\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   171793\n",
            "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        1    9999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :   15.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :      -13474.7998\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "DOPE score               : -13474.799805\n",
            ">> Model assessment by GA341 potential\n",
            "\n",
            "Surface library          : /usr/local/lib/modeller-10.4/modlib/surf5.de\n",
            "Pair library             : /usr/local/lib/modeller-10.4/modlib/pair9.de\n",
            "Chain identifier         : _\n",
            "% sequence identity      :    99.224998\n",
            "Sequence length          :          129\n",
            "Compactness              :     0.302273\n",
            "Native energy (pair)     :  -131.093494\n",
            "Native energy (surface)  :   -10.109922\n",
            "Native energy (combined) :    -8.159022\n",
            "Z score (pair)           :    -7.262293\n",
            "Z score (surface)        :    -5.405777\n",
            "Z score (combined)       :    -8.764831\n",
            "GA341 score              :     1.000000\n",
            "\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      129\n",
            "Number of all, selected real atoms                :      994     994\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    12204   12204\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2087\n",
            "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        0   99999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :    4.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :         713.6249\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Summary of the restraint violations: \n",
            "\n",
            "   NUM     ... number of restraints.\n",
            "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
            "   RVIOL   ... relative difference from the best value.\n",
            "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
            "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
            "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
            "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
            "\n",
            " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
            "------------------------------------------------------------------------------------------------------\n",
            " 1 Bond length potential              :    1013       0      0   0.006   0.006      9.8292       1.000\n",
            " 2 Bond angle potential               :    1370       0      8   2.107   2.107      126.14       1.000\n",
            " 3 Stereochemical cosine torsion poten:     647       0     22  47.311  47.311      224.08       1.000\n",
            " 4 Stereochemical improper torsion pot:     409       0      0   1.245   1.245      15.119       1.000\n",
            " 5 Soft-sphere overlap restraints     :    2087       0      0   0.001   0.001     0.12163       1.000\n",
            " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 9 Distance restraints 1 (CA-CA)      :    2421       0      0   0.109   0.109      31.011       1.000\n",
            "10 Distance restraints 2 (N-O)        :    2599       0      7   0.215   0.215      100.26       1.000\n",
            "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "13 Mainchain Omega dihedral restraints:     128       0      4   5.257   5.257      41.714       1.000\n",
            "14 Sidechain Chi_1 dihedral restraints:     104       0      1  48.530  48.530     0.31545       1.000\n",
            "15 Sidechain Chi_2 dihedral restraints:      73       0      0  52.337  52.337      22.007       1.000\n",
            "16 Sidechain Chi_3 dihedral restraints:      26       0      0  76.623  76.623      15.650       1.000\n",
            "17 Sidechain Chi_4 dihedral restraints:      17       0      0 110.113 110.113      13.403       1.000\n",
            "18 Disulfide distance restraints      :       4       0      0   0.005   0.005     0.17022E-01   1.000\n",
            "19 Disulfide angle restraints         :       8       0      0   2.749   2.749      1.3346       1.000\n",
            "20 Disulfide dihedral angle restraints:       4       0      0  24.463  24.463      2.3859       1.000\n",
            "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "23 Distance restraints 3 (SDCH-MNCH)  :    2020       0      0   0.424   0.424      44.018       1.000\n",
            "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "25 Phi/Psi pair of dihedral restraints:     127       4     12  17.939  33.873      4.5184       1.000\n",
            "26 Distance restraints 4 (SDCH-SDCH)  :    1234       0      0   0.620   0.620      61.699       1.000\n",
            "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "\n",
            "\n",
            "\n",
            "# Heavy relative violation of each residue is written to: 1.V99990001\n",
            "# The profile is NOT normalized by the number of restraints.\n",
            "# The profiles are smoothed over a window of residues:    1\n",
            "# The sum of all numbers in the file:   10608.7627\n",
            "\n",
            "\n",
            "\n",
            "List of the violated restraints:\n",
            "   A restraint is violated when the relative difference\n",
            "   from the best value (RVIOL) is larger than CUTOFF.\n",
            "\n",
            "   ICSR   ... index of a restraint in the current set.\n",
            "   RESNO  ... residue numbers of the first two atoms.\n",
            "   ATM    ... IUPAC atom names of the first two atoms.\n",
            "   FEAT   ... the value of the feature in the model.\n",
            "   restr  ... the mean of the basis restraint with the smallest\n",
            "              difference from the model (local minimum).\n",
            "   viol   ... difference from the local minimum.\n",
            "   rviol  ... relative difference from the local minimum.\n",
            "   RESTR  ... the best value (global minimum).\n",
            "   VIOL   ... difference from the best value.\n",
            "   RVIOL  ... relative difference from the best value.\n",
            "\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
            "List of the RVIOL violations larger than   :       6.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   3570 115C 116K C   N     881  883  -54.05  -62.90   17.46    2.01  -70.20  164.54   12.30\n",
            "    1        116K 116K N   CA    883  884  -55.85  -40.80                  140.40\n",
            "    2   3571 116K 117G C   N     890  892  -74.65  -62.40   15.49    2.26   82.20  161.92   12.06\n",
            "    2        117G 117G N   CA    892  893  -31.71  -41.20                    8.50\n",
            "    3   3579 124I 125R C   N     952  954  -49.21  -63.00   23.66    2.69  -72.10  159.42   12.75\n",
            "    3        125R 125R N   CA    954  955  -60.33  -41.10                  141.90\n",
            "    4   3580 125R 126G C   N     963  965  -59.05  -62.40   12.91    2.10   82.20  146.08   10.90\n",
            "    4        126G 126G N   CA    965  966  -28.74  -41.20                    8.50\n",
            "\n",
            "\n",
            "report______> Distribution of short non-bonded contacts:\n",
            "\n",
            "\n",
            "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
            "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
            "FREQUENCY:     0    0    0    0    0    2   10   28   56   86   94  103  153  175  156\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "\n",
            ">> Summary of successfully produced models:\n",
            "Filename                          molpdf     DOPE score    GA341 score\n",
            "----------------------------------------------------------------------\n",
            "1.B99990001.pdb                713.62494   -13474.79980        1.00000\n",
            "\n",
            "\n",
            "check_ali___> Checking the sequence-structure alignment. \n",
            "\n",
            "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
            "\n",
            "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
            "----------------------------------------------\n",
            "END OF TABLE\n",
            "read_to_681_> topology.submodel read from topology file:        3\n",
            "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
            "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
            "              least one known structure available. MDT, not library, potential is used.\n",
            "0 atoms in HETATM/BLK residues constrained\n",
            "to protein atoms within 2.30 angstroms\n",
            "and protein CA atoms within 10.00 angstroms\n",
            "0 atoms in residues without defined topology\n",
            "constrained to be rigid bodies\n",
            "condens_443_> Restraints marked for deletion were removed.\n",
            "              Total number of restraints before, now:    12965    12100\n",
            ">> Model assessment by DOPE potential\n",
            "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      128\n",
            "Number of all, selected real atoms                :      989     989\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    12100   12100\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   170642\n",
            "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        1    9999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :   15.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :      -13119.2910\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "DOPE score               : -13119.291016\n",
            ">> Model assessment by GA341 potential\n",
            "\n",
            "Surface library          : /usr/local/lib/modeller-10.4/modlib/surf5.de\n",
            "Pair library             : /usr/local/lib/modeller-10.4/modlib/pair9.de\n",
            "Chain identifier         : _\n",
            "% sequence identity      :   100.000000\n",
            "Sequence length          :          128\n",
            "Compactness              :     0.298515\n",
            "Native energy (pair)     :  -135.691452\n",
            "Native energy (surface)  :   -10.743974\n",
            "Native energy (combined) :    -8.483980\n",
            "Z score (pair)           :    -7.393156\n",
            "Z score (surface)        :    -5.545020\n",
            "Z score (combined)       :    -9.361240\n",
            "GA341 score              :     1.000000\n",
            "\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      128\n",
            "Number of all, selected real atoms                :      989     989\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    12100   12100\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2108\n",
            "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        0   99999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :    4.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :         824.6160\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Summary of the restraint violations: \n",
            "\n",
            "   NUM     ... number of restraints.\n",
            "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
            "   RVIOL   ... relative difference from the best value.\n",
            "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
            "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
            "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
            "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
            "\n",
            " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
            "------------------------------------------------------------------------------------------------------\n",
            " 1 Bond length potential              :    1008       0      0   0.007   0.007      12.041       1.000\n",
            " 2 Bond angle potential               :    1363       0      7   2.178   2.178      132.08       1.000\n",
            " 3 Stereochemical cosine torsion poten:     645       0     25  47.087  47.087      225.58       1.000\n",
            " 4 Stereochemical improper torsion pot:     407       0      0   1.448   1.448      19.257       1.000\n",
            " 5 Soft-sphere overlap restraints     :    2108       0      0   0.002   0.002     0.57045       1.000\n",
            " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 9 Distance restraints 1 (CA-CA)      :    2389       0      0   0.193   0.193      63.919       1.000\n",
            "10 Distance restraints 2 (N-O)        :    2561       2      6   0.246   0.246      112.93       1.000\n",
            "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "13 Mainchain Omega dihedral restraints:     127       0      2   4.929   4.929      36.389       1.000\n",
            "14 Sidechain Chi_1 dihedral restraints:     104       0      2  47.322  47.322      8.4720       1.000\n",
            "15 Sidechain Chi_2 dihedral restraints:      73       0      0  55.894  55.894      25.991       1.000\n",
            "16 Sidechain Chi_3 dihedral restraints:      26       0      0  72.125  72.125      18.296       1.000\n",
            "17 Sidechain Chi_4 dihedral restraints:      17       0      0 102.168 102.168      14.164       1.000\n",
            "18 Disulfide distance restraints      :       4       0      0   0.006   0.006     0.24138E-01   1.000\n",
            "19 Disulfide angle restraints         :       8       0      0   1.724   1.724     0.52506       1.000\n",
            "20 Disulfide dihedral angle restraints:       4       0      0  28.924  28.924      3.0814       1.000\n",
            "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "23 Distance restraints 3 (SDCH-MNCH)  :    2009       0      0   0.448   0.448      57.686       1.000\n",
            "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "25 Phi/Psi pair of dihedral restraints:     126       4     16  21.530  29.745      24.558       1.000\n",
            "26 Distance restraints 4 (SDCH-SDCH)  :    1229       0      0   0.690   0.690      69.053       1.000\n",
            "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "\n",
            "\n",
            "\n",
            "# Heavy relative violation of each residue is written to: 1.V99990001\n",
            "# The profile is NOT normalized by the number of restraints.\n",
            "# The profiles are smoothed over a window of residues:    1\n",
            "# The sum of all numbers in the file:   11224.4238\n",
            "\n",
            "\n",
            "\n",
            "List of the violated restraints:\n",
            "   A restraint is violated when the relative difference\n",
            "   from the best value (RVIOL) is larger than CUTOFF.\n",
            "\n",
            "   ICSR   ... index of a restraint in the current set.\n",
            "   RESNO  ... residue numbers of the first two atoms.\n",
            "   ATM    ... IUPAC atom names of the first two atoms.\n",
            "   FEAT   ... the value of the feature in the model.\n",
            "   restr  ... the mean of the basis restraint with the smallest\n",
            "              difference from the model (local minimum).\n",
            "   viol   ... difference from the local minimum.\n",
            "   rviol  ... relative difference from the local minimum.\n",
            "   RESTR  ... the best value (global minimum).\n",
            "   VIOL   ... difference from the best value.\n",
            "   RVIOL  ... relative difference from the best value.\n",
            "\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature 10                           : Distance restraints 2 (N-O)             \n",
            "List of the RVIOL violations larger than   :       4.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   7258  47D  69P N   O     361  546   11.32    8.57    2.75    4.88    8.57    2.75    4.88\n",
            "    2   7268  48G  69P N   O     369  546    9.52    6.98    2.54    4.69    6.98    2.54    4.69\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
            "List of the RVIOL violations larger than   :       6.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   3458  19N  20R C   N     145  147  -93.94  -63.00   94.90   11.10   57.30  151.61   17.16\n",
            "    1         20R  20R N   CA    147  148   48.61  -41.10                   38.00\n",
            "    2   3473  34E  35S C   N     264  266 -135.00  -64.10   78.16    8.27  -64.10   78.16    8.27\n",
            "    2         35S  35S N   CA    266  267   -2.10  -35.00                  -35.00\n",
            "    3   3507  68T  69P C   N     538  540  -53.06  -58.70   49.30    3.64  -64.50  133.81   10.60\n",
            "    3         69P  69P N   CA    540  541  -79.48  -30.50                  147.20\n",
            "    4   3508  69P  70G C   N     545  547  -61.68  -62.40   26.53    4.00   82.20  145.74   10.53\n",
            "    4         70G  70G N   CA    547  548  -14.68  -41.20                    8.50\n",
            "\n",
            "\n",
            "report______> Distribution of short non-bonded contacts:\n",
            "\n",
            "\n",
            "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
            "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
            "FREQUENCY:     0    0    0    0    0    4    8   32   62   91  102  122  135  153  158\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "\n",
            ">> Summary of successfully produced models:\n",
            "Filename                          molpdf     DOPE score    GA341 score\n",
            "----------------------------------------------------------------------\n",
            "1.B99990001.pdb                824.61603   -13119.29102        1.00000\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Calculate most probable protonation states "
      ],
      "metadata": {
        "id": "o4PcAAuQAh5c"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@ Run pkaAni\n",
        "%%bash\n",
        "cd ${MD_PATH}\n",
        "in_pdb=${PDB}_processed.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "cd ../md_mutant_${i}\n",
        "in_pdb=input.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "done"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L7OhlVxhCvhe",
        "outputId": "e71a90cf-9c31-4bbb-b34d-63c296a6ff9f"
      },
      "execution_count": 68,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Preparing 1aki_processed for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Downloading ANI model parameters ...\n",
            "Finished Loading.\n",
            "Calculating pKa for 1aki_processed.pdb\n",
            "Preparing input for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for input.pdb\n",
            "Preparing input for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for input.pdb\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/site-packages/torchani/aev.py:236: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').\n",
            "  pair_sizes = counts * (counts - 1) // 2\n",
            "/usr/local/lib/python3.7/site-packages/torchani/aev.py:236: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').\n",
            "  pair_sizes = counts * (counts - 1) // 2\n",
            "/usr/local/lib/python3.7/site-packages/torchani/aev.py:236: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').\n",
            "  pair_sizes = counts * (counts - 1) // 2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Get force fields for pdb2gmx\n",
        "if ff == 'amber14':\n",
        "    wget.download(\"https://github.com/pbuslaev/alphagmx/raw/main/forcefields/amber14sb.ff.tar.gz\")\n",
        "    tar = tarfile.open(\"amber14sb.ff.tar.gz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n",
        "if ff == 'charmm36':\n",
        "    wget.download(\"http://mackerell.umaryland.edu/download.php?filename=CHARMM_ff_params_files/charmm36-jul2021.ff.tgz\")\n",
        "    tar = tarfile.open(\"charmm36-jul2021.ff.tgz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n"
      ],
      "metadata": {
        "id": "YLJhqZARoCKN"
      },
      "execution_count": 84,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "path"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        },
        "id": "6C2J-2u78nB2",
        "outputId": "a784ccd9-86a1-4dad-fa86-a92705076fc6"
      },
      "execution_count": 70,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'./md_1aki/'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 70
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Change residue names in accordance with pKa calculation\n",
        "pH = 7\n",
        "names              = [\"ASP\",  \"GLU\",  \"HIS\", \"LYS\"]\n",
        "charmmProtonated   = [\"ASPP\", \"GLUP\", \"HSP\", \"LYS\"]\n",
        "charmmDeprotonated = [\"ASP\",  \"GLU\",  \"HIS\", \"LSN\"]\n",
        "amberProtonated    = [\"ASH\",  \"GLH\",  \"HIP\", \"LYS\"]\n",
        "amberDeprotonated  = [\"ASP\",  \"GLU\",  \"HIS\", \"LYN\"]\n",
        "if ff == 'charmm36':\n",
        "    protNames   = charmmProtonated\n",
        "    deprotNames = charmmDeprotonated\n",
        "else:\n",
        "    protNames   = amberProtonated\n",
        "    deprotNames = amberDeprotonated\n",
        "for i in range(-1, mutant_n, 1):\n",
        "    parser = PDBParser()\n",
        "    if i == -1:\n",
        "        structure = parser.get_structure('X', path+\"pdb\"+pdbid+\".ent\")\n",
        "        logname = path+pdbid+\"_processed_pka.log\"\n",
        "        outpath = path+\"input_pka.pdb\"\n",
        "    else:\n",
        "        structure = parser.get_structure('X', f\"./md_mutant_{i}/input.pdb\")\n",
        "        logname = f\"./md_mutant_{i}/input_pka.log\"\n",
        "        outpath = f\"./md_mutant_{i}/input_pka.pdb\"\n",
        "    residues = []\n",
        "    pkas     = []\n",
        "    with open(logname, \"r\") as f:\n",
        "        for line in f:\n",
        "            if line.startswith(\"Residue\"):\n",
        "                continue\n",
        "            if line.strip() == \"\":\n",
        "                continue\n",
        "            resname, resid, chain, pka = line.split()[0].split(\"-\")[0], \\\n",
        "                                         int(line.split()[0].split(\"-\")[1]), \\\n",
        "                                         line.split()[1], \\\n",
        "                                         float(line.split()[2])\n",
        "            residues.append((resname, resid, chain))\n",
        "            pkas.append(pka)\n",
        "\n",
        "    for model in structure:\n",
        "        for chain in model:\n",
        "            for i, residue in enumerate(chain.get_residues()):\n",
        "                if (residue.resname, residue.id[1], chain.id) in residues:\n",
        "                    id = residues.index((residue.resname, residue.id[1], chain.id))\n",
        "                    if (residue.resname in names) and pH < pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            protNames[names.index(residue.resname)]\n",
        "                    if (residue.resname in names) and pH > pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            deprotNames[names.index(residue.resname)]\n",
        "\n",
        "    pdb_io = PDBIO()\n",
        "    pdb_io.set_structure(structure)\n",
        "    pdb_io.save(outpath)             "
      ],
      "metadata": {
        "id": "0eqhMhZ3NzHU"
      },
      "execution_count": 71,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "\n",
        "sed -i 's/GLUP /GLUP/g' ${value}/input_pka.pdb\n",
        "sed -i 's/ASPP /ASPP/g' ${value}/input_pka.pdb\n",
        "\n",
        "done"
      ],
      "metadata": {
        "id": "SG_Rljap7QnB"
      },
      "execution_count": 76,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Run pdb2gmx, editconf, solvate and genion\n",
        "%%bash\n",
        "git clone https://github.com/pbuslaev/alphagmx.git\n",
        "\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "in_pdb=input_pka.pdb\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "cp -r amber14sb.ff $value/.\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber14sb\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber99sb-ildn\n",
        "fi\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "cp -r charmm36-jul2021.ff $value/.\n",
        "cp alphagmx/parameters/charmm/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
        "fi\n",
        "gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
        "gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
        "# Here we should apply some tricks Issue !4\n",
        "# https://github.com/pbuslaev/alphagmx/issues/4\n",
        "gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
        "#This is a trick to provide interactive options to gmx\n",
        "echo \"SOL\" > options\n",
        "echo \" \" >> options\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA < options\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "done"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CLvZTTAcZe2-",
        "outputId": "425a448b-8a9d-4515-a17f-b2149ee8a40c"
      },
      "execution_count": 94,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 1079 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 1 blocks of water and 207 residues with 1079 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   129   1001  \n",
            "\n",
            "  2 ' '    78     78  (only water)\n",
            "\n",
            "there were 0 atoms with zero occupancy and 74 atoms with          occupancy unequal to one (out of 1079 atoms). Check your pdb file.\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (1001 atoms, 129 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU129 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-129: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 129 residues with 1960 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1984, now 1984\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  127 cmap torsion pairs\n",
            "\n",
            "There are 5187 dihedrals,  373 impropers, 3547 angles\n",
            "          5106 pairs,     1984 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14313.255 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Processing chain 2 (78 atoms, 78 residues)\n",
            "\n",
            "Problem with chain definition, or missing terminal residues. This chain does not appear to contain a recognized chain molecule. If this is incorrect, you can edit residuetypes.dat to modify the behavior.\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 78 residues with 234 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 234, now 234\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are    0 dihedrals,    0 impropers,  234 angles\n",
            "             0 pairs,      234 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 1405.201 a.m.u.\n",
            "\n",
            "Total charge 0.000 e\n",
            "\n",
            "Including chain 1 in system: 1960 atoms 129 residues\n",
            "\n",
            "Including chain 2 in system: 234 atoms 78 residues\n",
            "\n",
            "Now there are 2194 atoms and 207 residues\n",
            "\n",
            "Total mass in system 15718.456 a.m.u.\n",
            "\n",
            "Total charge in system 8.000 e\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 2194 atoms\n",
            "Volume: 61.8999 nm^3, corresponds to roughly 27800 electrons\n",
            "No velocities found\n",
            "    system size :  4.196  4.268  3.455 (nm)\n",
            "    center      :  2.772  2.507  0.003 (nm)\n",
            "    box vectors :  4.197  4.268  3.456 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  61.90               (nm^3)\n",
            "    shift       :  1.326  1.627  3.725 (nm)\n",
            "new center      :  4.098  4.134  3.727 (nm)\n",
            "new box vectors :  8.196  8.268  7.455 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 505.18               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 15945 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -545522963\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 16023      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.443 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 46 SOD ions and 54 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 100 solute molecules in topology file (topol.top)  by 46 SOD and 54 CLA ions.\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 994 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 129 residues with 994 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   129    994  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (994 atoms, 129 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU129 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-129: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 129 residues with 1949 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1972, now 1972\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  127 cmap torsion pairs\n",
            "\n",
            "There are 5155 dihedrals,  373 impropers, 3528 angles\n",
            "          5077 pairs,     1972 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14221.157 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 1949 atoms\n",
            "Volume: 55.7625 nm^3, corresponds to roughly 25000 electrons\n",
            "No velocities found\n",
            "    system size :  4.100  4.270  3.185 (nm)\n",
            "    center      :  2.852  3.153  0.238 (nm)\n",
            "    box vectors :  4.100  4.270  3.185 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  55.76               (nm^3)\n",
            "    shift       :  1.198  0.982  3.355 (nm)\n",
            "new center      :  4.050  4.135  3.593 (nm)\n",
            "new box vectors :  8.100  8.270  7.185 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 481.30               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 15296 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -268973061\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15296      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 43 SOD ions and 51 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 94 solute molecules in topology file (topol.top)  by 43 SOD and 51 CLA ions.\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 989 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 128 residues with 989 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   128    989  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (989 atoms, 128 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU128 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-128: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 128 residues with 1939 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1962, now 1962\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  126 cmap torsion pairs\n",
            "\n",
            "There are 5130 dihedrals,  371 impropers, 3510 angles\n",
            "          5052 pairs,     1962 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14150.078 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 1939 atoms\n",
            "Volume: 56.0089 nm^3, corresponds to roughly 25200 electrons\n",
            "No velocities found\n",
            "    system size :  3.619  4.216  3.671 (nm)\n",
            "    center      :  2.515  2.529  0.061 (nm)\n",
            "    box vectors :  3.619  4.216  3.671 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  56.01               (nm^3)\n",
            "    shift       :  1.294  1.579  3.775 (nm)\n",
            "new center      :  3.810  4.108  3.835 (nm)\n",
            "new box vectors :  7.619  8.216  7.671 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 480.19               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 14617 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -3276969\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14617      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 43 SOD ions and 51 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 94 solute molecules in topology file (topol.top)  by 43 SOD and 51 CLA ions.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "fatal: destination path 'alphagmx' already exists and is not an empty directory.\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "there were 0 atoms with zero occupancy and 74 atoms with          occupancy unequal to one (out of 1079 atoms). Check your pdb file.\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.24#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 213 donors and 184 acceptors were found.\n",
            "There are 255 hydrogen bonds\n",
            "Will use HISE for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS30   CYS64   CYS76   CYS80\n",
            "                    SG48    SD87  NE2118   SG238   SG513   SG601   SG630\n",
            "   MET12    SD87   1.166\n",
            "   HIS15  NE2118   1.776   1.019\n",
            "   CYS30   SG238   1.406   1.054   2.069\n",
            "   CYS64   SG513   2.835   1.794   1.789   2.241\n",
            "   CYS76   SG601   2.704   1.551   1.468   2.116   0.765\n",
            "   CYS80   SG630   2.959   1.951   1.916   2.391   0.199   0.944\n",
            "   CYS94   SG724   2.550   1.407   1.382   1.975   0.665   0.202   0.855\n",
            "  MET105   SD799   1.827   0.911   1.683   0.888   1.849   1.461   2.036\n",
            "  CYS115   SG889   1.576   1.084   2.078   0.200   2.111   1.989   2.262\n",
            "  CYS127   SG981   0.197   1.072   1.721   1.313   2.799   2.622   2.934\n",
            "                   CYS94  MET105  CYS115\n",
            "                   SG724   SD799   SG889\n",
            "  MET105   SD799   1.381\n",
            "  CYS115   SG889   1.853   0.790\n",
            "  CYS127   SG981   2.475   1.686   1.483\n",
            "Linking CYS-6 SG-48 and CYS-127 SG-981...\n",
            "Linking CYS-30 SG-238 and CYS-115 SG-889...\n",
            "Linking CYS-64 SG-513 and CYS-80 SG-630...\n",
            "Linking CYS-76 SG-601 and CYS-94 SG-724...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5142 pairs\n",
            "Before cleaning: 5187 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.8#\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"Christianity may be OK between consenting adults in private but should not be taught to young children.\" (Francis Crick)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"Christianity may be OK between consenting adults in private but should not be taught to young children.\" (Francis Crick)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x5 boxes\n",
            "Solvent box contains 56946 atoms in 18982 residues\n",
            "Removed 7026 solvent atoms due to solvent-solvent overlap\n",
            "Removed 2085 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 15945 residues\n",
            "Generated solvent containing 47835 atoms in 15945 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.8#\n",
            "\n",
            "Output configuration contains 50029 atoms in 16152 residues\n",
            "Volume                 :     505.185 (nm^3)\n",
            "Density                :     999.014 (g/l)\n",
            "Number of solvent molecules:  15945   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.25#\n",
            "\n",
            "GROMACS reminds you: \"A computer once beat me at chess, but it was no match for me at kick boxing.\" (Emo Philips)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18487]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18487]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 102015.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.8#\n",
            "\n",
            "GROMACS reminds you: \"You got one part of that wrong. This is not meth.\" (Breaking Bad)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 50029 elements\n",
            "Group     1 (        Protein) has  1960 elements\n",
            "Group     2 (      Protein-H) has  1001 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1316 elements\n",
            "Group     9 (    SideChain-H) has   486 elements\n",
            "Group    10 (    Prot-Masses) has  1960 elements\n",
            "Group    11 (    non-Protein) has 48069 elements\n",
            "Group    12 (          Water) has 48069 elements\n",
            "Group    13 (            SOL) has 48069 elements\n",
            "Group    14 (      non-Water) has  1960 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 16023\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.26#\n",
            "Using random seed -1212158546.\n",
            "Replacing solvent molecule 3120 (atom 11320) with SOD\n",
            "Replacing solvent molecule 15187 (atom 47521) with SOD\n",
            "Replacing solvent molecule 5588 (atom 18724) with SOD\n",
            "Replacing solvent molecule 15367 (atom 48061) with SOD\n",
            "Replacing solvent molecule 3087 (atom 11221) with SOD\n",
            "Replacing solvent molecule 4994 (atom 16942) with SOD\n",
            "Replacing solvent molecule 406 (atom 3178) with SOD\n",
            "Replacing solvent molecule 1538 (atom 6574) with SOD\n",
            "Replacing solvent molecule 2170 (atom 8470) with SOD\n",
            "Replacing solvent molecule 1650 (atom 6910) with SOD\n",
            "Replacing solvent molecule 10192 (atom 32536) with SOD\n",
            "Replacing solvent molecule 15719 (atom 49117) with SOD\n",
            "Replacing solvent molecule 9542 (atom 30586) with SOD\n",
            "Replacing solvent molecule 13236 (atom 41668) with SOD\n",
            "Replacing solvent molecule 828 (atom 4444) with SOD\n",
            "Replacing solvent molecule 3539 (atom 12577) with SOD\n",
            "Replacing solvent molecule 14427 (atom 45241) with SOD\n",
            "Replacing solvent molecule 993 (atom 4939) with SOD\n",
            "Replacing solvent molecule 10348 (atom 33004) with SOD\n",
            "Replacing solvent molecule 2263 (atom 8749) with SOD\n",
            "Replacing solvent molecule 1293 (atom 5839) with SOD\n",
            "Replacing solvent molecule 6767 (atom 22261) with SOD\n",
            "Replacing solvent molecule 2727 (atom 10141) with SOD\n",
            "Replacing solvent molecule 8270 (atom 26770) with SOD\n",
            "Replacing solvent molecule 15069 (atom 47167) with SOD\n",
            "Replacing solvent molecule 3560 (atom 12640) with SOD\n",
            "Replacing solvent molecule 14383 (atom 45109) with SOD\n",
            "Replacing solvent molecule 1063 (atom 5149) with SOD\n",
            "Replacing solvent molecule 5456 (atom 18328) with SOD\n",
            "Replacing solvent molecule 8062 (atom 26146) with SOD\n",
            "Replacing solvent molecule 10622 (atom 33826) with SOD\n",
            "Replacing solvent molecule 15323 (atom 47929) with SOD\n",
            "Replacing solvent molecule 7660 (atom 24940) with SOD\n",
            "Replacing solvent molecule 6755 (atom 22225) with SOD\n",
            "Replacing solvent molecule 5553 (atom 18619) with SOD\n",
            "Replacing solvent molecule 14166 (atom 44458) with SOD\n",
            "Replacing solvent molecule 6414 (atom 21202) with SOD\n",
            "Replacing solvent molecule 7880 (atom 25600) with SOD\n",
            "Replacing solvent molecule 2911 (atom 10693) with SOD\n",
            "Replacing solvent molecule 12962 (atom 40846) with SOD\n",
            "Replacing solvent molecule 13934 (atom 43762) with SOD\n",
            "Replacing solvent molecule 6193 (atom 20539) with SOD\n",
            "Replacing solvent molecule 12724 (atom 40132) with SOD\n",
            "Replacing solvent molecule 14476 (atom 45388) with SOD\n",
            "Replacing solvent molecule 538 (atom 3574) with SOD\n",
            "Replacing solvent molecule 6925 (atom 22735) with SOD\n",
            "Replacing solvent molecule 9089 (atom 29227) with CLA\n",
            "Replacing solvent molecule 4494 (atom 15442) with CLA\n",
            "Replacing solvent molecule 8672 (atom 27976) with CLA\n",
            "Replacing solvent molecule 13468 (atom 42364) with CLA\n",
            "Replacing solvent molecule 11760 (atom 37240) with CLA\n",
            "Replacing solvent molecule 15207 (atom 47581) with CLA\n",
            "Replacing solvent molecule 4502 (atom 15466) with CLA\n",
            "Replacing solvent molecule 943 (atom 4789) with CLA\n",
            "Replacing solvent molecule 1865 (atom 7555) with CLA\n",
            "Replacing solvent molecule 6237 (atom 20671) with CLA\n",
            "Replacing solvent molecule 2773 (atom 10279) with CLA\n",
            "Replacing solvent molecule 12632 (atom 39856) with CLA\n",
            "Replacing solvent molecule 5574 (atom 18682) with CLA\n",
            "Replacing solvent molecule 5601 (atom 18763) with CLA\n",
            "Replacing solvent molecule 10288 (atom 32824) with CLA\n",
            "Replacing solvent molecule 15513 (atom 48499) with CLA\n",
            "Replacing solvent molecule 5876 (atom 19588) with CLA\n",
            "Replacing solvent molecule 5209 (atom 17587) with CLA\n",
            "Replacing solvent molecule 4189 (atom 14527) with CLA\n",
            "Replacing solvent molecule 4752 (atom 16216) with CLA\n",
            "Replacing solvent molecule 5558 (atom 18634) with CLA\n",
            "Replacing solvent molecule 10725 (atom 34135) with CLA\n",
            "Replacing solvent molecule 12096 (atom 38248) with CLA\n",
            "Replacing solvent molecule 2218 (atom 8614) with CLA\n",
            "Replacing solvent molecule 13510 (atom 42490) with CLA\n",
            "Replacing solvent molecule 9932 (atom 31756) with CLA\n",
            "Replacing solvent molecule 3225 (atom 11635) with CLA\n",
            "Replacing solvent molecule 7218 (atom 23614) with CLA\n",
            "Replacing solvent molecule 13737 (atom 43171) with CLA\n",
            "Replacing solvent molecule 4712 (atom 16096) with CLA\n",
            "Replacing solvent molecule 154 (atom 2422) with CLA\n",
            "Replacing solvent molecule 12685 (atom 40015) with CLA\n",
            "Replacing solvent molecule 1505 (atom 6475) with CLA\n",
            "Replacing solvent molecule 15764 (atom 49252) with CLA\n",
            "Replacing solvent molecule 7744 (atom 25192) with CLA\n",
            "Replacing solvent molecule 9720 (atom 31120) with CLA\n",
            "Replacing solvent molecule 907 (atom 4681) with CLA\n",
            "Replacing solvent molecule 12388 (atom 39124) with CLA\n",
            "Replacing solvent molecule 4508 (atom 15484) with CLA\n",
            "Replacing solvent molecule 7423 (atom 24229) with CLA\n",
            "Replacing solvent molecule 4563 (atom 15649) with CLA\n",
            "Replacing solvent molecule 9045 (atom 29095) with CLA\n",
            "Replacing solvent molecule 9449 (atom 30307) with CLA\n",
            "Replacing solvent molecule 120 (atom 2320) with CLA\n",
            "Replacing solvent molecule 9605 (atom 30775) with CLA\n",
            "Replacing solvent molecule 11801 (atom 37363) with CLA\n",
            "Replacing solvent molecule 10387 (atom 33121) with CLA\n",
            "Replacing solvent molecule 1884 (atom 7612) with CLA\n",
            "Replacing solvent molecule 13529 (atom 42547) with CLA\n",
            "Replacing solvent molecule 8905 (atom 28675) with CLA\n",
            "Replacing solvent molecule 10997 (atom 34951) with CLA\n",
            "Replacing solvent molecule 1243 (atom 5689) with CLA\n",
            "Replacing solvent molecule 10303 (atom 32869) with CLA\n",
            "Replacing solvent molecule 7775 (atom 25285) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"You got one part of that wrong. This is not meth.\" (Breaking Bad)\n",
            "\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.3#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 212 donors and 183 acceptors were found.\n",
            "There are 269 hydrogen bonds\n",
            "Will use HISD for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS30   CYS64   CYS76   CYS80\n",
            "                    SG46    SD85  NE2114   SG229   SG504   SG592   SG621\n",
            "   MET12    SD85   1.157\n",
            "   HIS15  NE2114   1.595   0.933\n",
            "   CYS30   SG229   1.573   1.021   1.935\n",
            "   CYS64   SG504   2.839   1.837   1.695   2.220\n",
            "   CYS76   SG592   2.687   1.577   1.402   2.089   0.761\n",
            "   CYS80   SG621   2.966   2.000   1.822   2.379   0.204   0.942\n",
            "   CYS94   SG715   2.527   1.424   1.297   1.933   0.678   0.204   0.871\n",
            "  MET105   SD790   1.969   0.917   1.548   0.977   1.764   1.346   1.961\n",
            "  CYS115   SG880   1.731   1.048   1.933   0.203   2.091   1.953   2.254\n",
            "  CYS127   SG972   0.202   1.038   1.578   1.403   2.779   2.599   2.915\n",
            "                   CYS94  MET105  CYS115\n",
            "                   SG715   SD790   SG880\n",
            "  MET105   SD790   1.255\n",
            "  CYS115   SG880   1.803   0.850\n",
            "  CYS127   SG972   2.441   1.804   1.563\n",
            "Linking CYS-6 SG-46 and CYS-127 SG-972...\n",
            "Linking CYS-30 SG-229 and CYS-115 SG-880...\n",
            "Linking CYS-64 SG-504 and CYS-80 SG-621...\n",
            "Linking CYS-76 SG-592 and CYS-94 SG-715...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5110 pairs\n",
            "Before cleaning: 5155 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.1#\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x4 boxes\n",
            "Solvent box contains 53784 atoms in 17928 residues\n",
            "Removed 6102 solvent atoms due to solvent-solvent overlap\n",
            "Removed 1794 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 15296 residues\n",
            "Generated solvent containing 45888 atoms in 15296 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.1#\n",
            "\n",
            "Output configuration contains 47837 atoms in 15425 residues\n",
            "Volume                 :     481.302 (nm^3)\n",
            "Density                :     1003.07 (g/l)\n",
            "Number of solvent molecules:  15296   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.4#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18383]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18383]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 97620.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"FORTRAN, the infantile disorder, by now nearly 20 years old, is hopelessly inadequate for whatever computer application you have in mind today: it is now too clumsy, too risky, and too expensive to use.\" (Edsger Dijkstra, 1970)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 47837 elements\n",
            "Group     1 (        Protein) has  1949 elements\n",
            "Group     2 (      Protein-H) has   994 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1305 elements\n",
            "Group     9 (    SideChain-H) has   479 elements\n",
            "Group    10 (    Prot-Masses) has  1949 elements\n",
            "Group    11 (    non-Protein) has 45888 elements\n",
            "Group    12 (          Water) has 45888 elements\n",
            "Group    13 (            SOL) has 45888 elements\n",
            "Group    14 (      non-Water) has  1949 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 15296\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.5#\n",
            "Using random seed -18945541.\n",
            "Replacing solvent molecule 4699 (atom 16046) with SOD\n",
            "Replacing solvent molecule 1242 (atom 5675) with SOD\n",
            "Replacing solvent molecule 2979 (atom 10886) with SOD\n",
            "Replacing solvent molecule 3149 (atom 11396) with SOD\n",
            "Replacing solvent molecule 3555 (atom 12614) with SOD\n",
            "Replacing solvent molecule 3072 (atom 11165) with SOD\n",
            "Replacing solvent molecule 15153 (atom 47408) with SOD\n",
            "Replacing solvent molecule 1757 (atom 7220) with SOD\n",
            "Replacing solvent molecule 8199 (atom 26546) with SOD\n",
            "Replacing solvent molecule 3001 (atom 10952) with SOD\n",
            "Replacing solvent molecule 10428 (atom 33233) with SOD\n",
            "Replacing solvent molecule 13187 (atom 41510) with SOD\n",
            "Replacing solvent molecule 11192 (atom 35525) with SOD\n",
            "Replacing solvent molecule 9601 (atom 30752) with SOD\n",
            "Replacing solvent molecule 1491 (atom 6422) with SOD\n",
            "Replacing solvent molecule 4559 (atom 15626) with SOD\n",
            "Replacing solvent molecule 3558 (atom 12623) with SOD\n",
            "Replacing solvent molecule 3574 (atom 12671) with SOD\n",
            "Replacing solvent molecule 11886 (atom 37607) with SOD\n",
            "Replacing solvent molecule 4546 (atom 15587) with SOD\n",
            "Replacing solvent molecule 8377 (atom 27080) with SOD\n",
            "Replacing solvent molecule 10126 (atom 32327) with SOD\n",
            "Replacing solvent molecule 2592 (atom 9725) with SOD\n",
            "Replacing solvent molecule 217 (atom 2600) with SOD\n",
            "Replacing solvent molecule 7275 (atom 23774) with SOD\n",
            "Replacing solvent molecule 13300 (atom 41849) with SOD\n",
            "Replacing solvent molecule 534 (atom 3551) with SOD\n",
            "Replacing solvent molecule 64 (atom 2141) with SOD\n",
            "Replacing solvent molecule 7661 (atom 24932) with SOD\n",
            "Replacing solvent molecule 4731 (atom 16142) with SOD\n",
            "Replacing solvent molecule 14161 (atom 44432) with SOD\n",
            "Replacing solvent molecule 6104 (atom 20261) with SOD\n",
            "Replacing solvent molecule 12082 (atom 38195) with SOD\n",
            "Replacing solvent molecule 12550 (atom 39599) with SOD\n",
            "Replacing solvent molecule 2564 (atom 9641) with SOD\n",
            "Replacing solvent molecule 9175 (atom 29474) with SOD\n",
            "Replacing solvent molecule 2541 (atom 9572) with SOD\n",
            "Replacing solvent molecule 14492 (atom 45425) with SOD\n",
            "Replacing solvent molecule 8720 (atom 28109) with SOD\n",
            "Replacing solvent molecule 11414 (atom 36191) with SOD\n",
            "Replacing solvent molecule 3911 (atom 13682) with SOD\n",
            "Replacing solvent molecule 6298 (atom 20843) with SOD\n",
            "Replacing solvent molecule 12840 (atom 40469) with SOD\n",
            "Replacing solvent molecule 3865 (atom 13544) with CLA\n",
            "Replacing solvent molecule 14254 (atom 44711) with CLA\n",
            "Replacing solvent molecule 10624 (atom 33821) with CLA\n",
            "Replacing solvent molecule 11130 (atom 35339) with CLA\n",
            "Replacing solvent molecule 7957 (atom 25820) with CLA\n",
            "Replacing solvent molecule 12782 (atom 40295) with CLA\n",
            "Replacing solvent molecule 5100 (atom 17249) with CLA\n",
            "Replacing solvent molecule 5103 (atom 17258) with CLA\n",
            "Replacing solvent molecule 1920 (atom 7709) with CLA\n",
            "Replacing solvent molecule 12816 (atom 40397) with CLA\n",
            "Replacing solvent molecule 10080 (atom 32189) with CLA\n",
            "Replacing solvent molecule 9507 (atom 30470) with CLA\n",
            "Replacing solvent molecule 7058 (atom 23123) with CLA\n",
            "Replacing solvent molecule 4847 (atom 16490) with CLA\n",
            "Replacing solvent molecule 6591 (atom 21722) with CLA\n",
            "Replacing solvent molecule 3464 (atom 12341) with CLA\n",
            "Replacing solvent molecule 13938 (atom 43763) with CLA\n",
            "Replacing solvent molecule 2802 (atom 10355) with CLA\n",
            "Replacing solvent molecule 6635 (atom 21854) with CLA\n",
            "Replacing solvent molecule 5393 (atom 18128) with CLA\n",
            "Replacing solvent molecule 2855 (atom 10514) with CLA\n",
            "Replacing solvent molecule 799 (atom 4346) with CLA\n",
            "Replacing solvent molecule 6675 (atom 21974) with CLA\n",
            "Replacing solvent molecule 2638 (atom 9863) with CLA\n",
            "Replacing solvent molecule 14862 (atom 46535) with CLA\n",
            "Replacing solvent molecule 11998 (atom 37943) with CLA\n",
            "Replacing solvent molecule 2005 (atom 7964) with CLA\n",
            "Replacing solvent molecule 2158 (atom 8423) with CLA\n",
            "Replacing solvent molecule 8083 (atom 26198) with CLA\n",
            "Replacing solvent molecule 14962 (atom 46835) with CLA\n",
            "Replacing solvent molecule 15110 (atom 47279) with CLA\n",
            "Replacing solvent molecule 14551 (atom 45602) with CLA\n",
            "Replacing solvent molecule 8461 (atom 27332) with CLA\n",
            "Replacing solvent molecule 7638 (atom 24863) with CLA\n",
            "Replacing solvent molecule 10798 (atom 34343) with CLA\n",
            "Replacing solvent molecule 1297 (atom 5840) with CLA\n",
            "Replacing solvent molecule 4125 (atom 14324) with CLA\n",
            "Replacing solvent molecule 12565 (atom 39644) with CLA\n",
            "Replacing solvent molecule 10199 (atom 32546) with CLA\n",
            "Replacing solvent molecule 1617 (atom 6800) with CLA\n",
            "Replacing solvent molecule 10208 (atom 32573) with CLA\n",
            "Replacing solvent molecule 14757 (atom 46220) with CLA\n",
            "Replacing solvent molecule 4606 (atom 15767) with CLA\n",
            "Replacing solvent molecule 10031 (atom 32042) with CLA\n",
            "Replacing solvent molecule 8085 (atom 26204) with CLA\n",
            "Replacing solvent molecule 14610 (atom 45779) with CLA\n",
            "Replacing solvent molecule 4347 (atom 14990) with CLA\n",
            "Replacing solvent molecule 1144 (atom 5381) with CLA\n",
            "Replacing solvent molecule 1587 (atom 6710) with CLA\n",
            "Replacing solvent molecule 8860 (atom 28529) with CLA\n",
            "Replacing solvent molecule 10000 (atom 31949) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"This is Tense !\" (Star Wars Episode I The Phantom Menace)\n",
            "\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.3#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 211 donors and 182 acceptors were found.\n",
            "There are 263 hydrogen bonds\n",
            "Will use HISE for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS29   CYS63   CYS75   CYS79\n",
            "                    SG46    SD85  NE2114   SG224   SG499   SG587   SG616\n",
            "   MET12    SD85   1.203\n",
            "   HIS15  NE2114   1.736   0.884\n",
            "   CYS29   SG224   1.457   1.021   1.889\n",
            "   CYS63   SG499   2.872   1.804   1.637   2.238\n",
            "   CYS75   SG587   2.744   1.559   1.343   2.095   0.760\n",
            "   CYS79   SG616   2.993   1.962   1.764   2.395   0.204   0.943\n",
            "   CYS93   SG710   2.578   1.404   1.235   1.942   0.677   0.203   0.870\n",
            "  MET104   SD785   1.857   0.903   1.552   0.831   1.894   1.505   2.086\n",
            "  CYS114   SG875   1.622   1.038   1.874   0.203   2.092   1.954   2.252\n",
            "  CYS126   SG967   0.203   1.103   1.686   1.340   2.832   2.659   2.964\n",
            "                   CYS93  MET104  CYS114\n",
            "                   SG710   SD785   SG875\n",
            "  MET104   SD785   1.407\n",
            "  CYS114   SG875   1.805   0.731\n",
            "  CYS126   SG967   2.500   1.703   1.506\n",
            "Linking CYS-6 SG-46 and CYS-126 SG-967...\n",
            "Linking CYS-29 SG-224 and CYS-114 SG-875...\n",
            "Linking CYS-63 SG-499 and CYS-79 SG-616...\n",
            "Linking CYS-75 SG-587 and CYS-93 SG-710...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5085 pairs\n",
            "Before cleaning: 5130 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.1#\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x5 boxes\n",
            "Solvent box contains 54135 atoms in 18045 residues\n",
            "Removed 8490 solvent atoms due to solvent-solvent overlap\n",
            "Removed 1794 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 14617 residues\n",
            "Generated solvent containing 43851 atoms in 14617 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.1#\n",
            "\n",
            "Output configuration contains 45790 atoms in 14745 residues\n",
            "Volume                 :     480.187 (nm^3)\n",
            "Density                :     962.833 (g/l)\n",
            "Number of solvent molecules:  14617   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.4#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18291]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18291]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 93516.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"There is no place like ~\" (Anonymous)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 45790 elements\n",
            "Group     1 (        Protein) has  1939 elements\n",
            "Group     2 (      Protein-H) has   989 elements\n",
            "Group     3 (        C-alpha) has   128 elements\n",
            "Group     4 (       Backbone) has   384 elements\n",
            "Group     5 (      MainChain) has   511 elements\n",
            "Group     6 (   MainChain+Cb) has   627 elements\n",
            "Group     7 (    MainChain+H) has   639 elements\n",
            "Group     8 (      SideChain) has  1300 elements\n",
            "Group     9 (    SideChain-H) has   478 elements\n",
            "Group    10 (    Prot-Masses) has  1939 elements\n",
            "Group    11 (    non-Protein) has 43851 elements\n",
            "Group    12 (          Water) has 43851 elements\n",
            "Group    13 (            SOL) has 43851 elements\n",
            "Group    14 (      non-Water) has  1939 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 14617\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.5#\n",
            "Using random seed -1844226.\n",
            "Replacing solvent molecule 10305 (atom 32854) with SOD\n",
            "Replacing solvent molecule 4693 (atom 16018) with SOD\n",
            "Replacing solvent molecule 11875 (atom 37564) with SOD\n",
            "Replacing solvent molecule 7423 (atom 24208) with SOD\n",
            "Replacing solvent molecule 11092 (atom 35215) with SOD\n",
            "Replacing solvent molecule 3175 (atom 11464) with SOD\n",
            "Replacing solvent molecule 14455 (atom 45304) with SOD\n",
            "Replacing solvent molecule 10744 (atom 34171) with SOD\n",
            "Replacing solvent molecule 14153 (atom 44398) with SOD\n",
            "Replacing solvent molecule 11232 (atom 35635) with SOD\n",
            "Replacing solvent molecule 6065 (atom 20134) with SOD\n",
            "Replacing solvent molecule 2709 (atom 10066) with SOD\n",
            "Replacing solvent molecule 6948 (atom 22783) with SOD\n",
            "Replacing solvent molecule 11038 (atom 35053) with SOD\n",
            "Replacing solvent molecule 1409 (atom 6166) with SOD\n",
            "Replacing solvent molecule 2786 (atom 10297) with SOD\n",
            "Replacing solvent molecule 6377 (atom 21070) with SOD\n",
            "Replacing solvent molecule 11463 (atom 36328) with SOD\n",
            "Replacing solvent molecule 13777 (atom 43270) with SOD\n",
            "Replacing solvent molecule 14205 (atom 44554) with SOD\n",
            "Replacing solvent molecule 9768 (atom 31243) with SOD\n",
            "Replacing solvent molecule 9079 (atom 29176) with SOD\n",
            "Replacing solvent molecule 8721 (atom 28102) with SOD\n",
            "Replacing solvent molecule 13983 (atom 43888) with SOD\n",
            "Replacing solvent molecule 8627 (atom 27820) with SOD\n",
            "Replacing solvent molecule 7954 (atom 25801) with SOD\n",
            "Replacing solvent molecule 11322 (atom 35905) with SOD\n",
            "Replacing solvent molecule 5618 (atom 18793) with SOD\n",
            "Replacing solvent molecule 5388 (atom 18103) with SOD\n",
            "Replacing solvent molecule 11920 (atom 37699) with SOD\n",
            "Replacing solvent molecule 8283 (atom 26788) with SOD\n",
            "Replacing solvent molecule 13695 (atom 43024) with SOD\n",
            "Replacing solvent molecule 1413 (atom 6178) with SOD\n",
            "Replacing solvent molecule 13877 (atom 43570) with SOD\n",
            "Replacing solvent molecule 7340 (atom 23959) with SOD\n",
            "Replacing solvent molecule 11932 (atom 37735) with SOD\n",
            "Replacing solvent molecule 13059 (atom 41116) with SOD\n",
            "Replacing solvent molecule 1107 (atom 5260) with SOD\n",
            "Replacing solvent molecule 42 (atom 2065) with SOD\n",
            "Replacing solvent molecule 11703 (atom 37048) with SOD\n",
            "Replacing solvent molecule 3816 (atom 13387) with SOD\n",
            "Replacing solvent molecule 11476 (atom 36367) with SOD\n",
            "Replacing solvent molecule 10428 (atom 33223) with SOD\n",
            "Replacing solvent molecule 12052 (atom 38095) with CLA\n",
            "Replacing solvent molecule 1534 (atom 6541) with CLA\n",
            "Replacing solvent molecule 10248 (atom 32683) with CLA\n",
            "Replacing solvent molecule 3559 (atom 12616) with CLA\n",
            "Replacing solvent molecule 9675 (atom 30964) with CLA\n",
            "Replacing solvent molecule 7467 (atom 24340) with CLA\n",
            "Replacing solvent molecule 14130 (atom 44329) with CLA\n",
            "Replacing solvent molecule 7711 (atom 25072) with CLA\n",
            "Replacing solvent molecule 8317 (atom 26890) with CLA\n",
            "Replacing solvent molecule 12305 (atom 38854) with CLA\n",
            "Replacing solvent molecule 10160 (atom 32419) with CLA\n",
            "Replacing solvent molecule 2324 (atom 8911) with CLA\n",
            "Replacing solvent molecule 4139 (atom 14356) with CLA\n",
            "Replacing solvent molecule 10557 (atom 33610) with CLA\n",
            "Replacing solvent molecule 5342 (atom 17965) with CLA\n",
            "Replacing solvent molecule 13176 (atom 41467) with CLA\n",
            "Replacing solvent molecule 2464 (atom 9331) with CLA\n",
            "Replacing solvent molecule 913 (atom 4678) with CLA\n",
            "Replacing solvent molecule 9019 (atom 28996) with CLA\n",
            "Replacing solvent molecule 4260 (atom 14719) with CLA\n",
            "Replacing solvent molecule 13406 (atom 42157) with CLA\n",
            "Replacing solvent molecule 13582 (atom 42685) with CLA\n",
            "Replacing solvent molecule 7918 (atom 25693) with CLA\n",
            "Replacing solvent molecule 7732 (atom 25135) with CLA\n",
            "Replacing solvent molecule 8122 (atom 26305) with CLA\n",
            "Replacing solvent molecule 8083 (atom 26188) with CLA\n",
            "Replacing solvent molecule 3498 (atom 12433) with CLA\n",
            "Replacing solvent molecule 6610 (atom 21769) with CLA\n",
            "Replacing solvent molecule 8076 (atom 26167) with CLA\n",
            "Replacing solvent molecule 2922 (atom 10705) with CLA\n",
            "Replacing solvent molecule 8017 (atom 25990) with CLA\n",
            "Replacing solvent molecule 626 (atom 3817) with CLA\n",
            "Replacing solvent molecule 11016 (atom 34987) with CLA\n",
            "Replacing solvent molecule 3660 (atom 12919) with CLA\n",
            "Replacing solvent molecule 6582 (atom 21685) with CLA\n",
            "Replacing solvent molecule 3473 (atom 12358) with CLA\n",
            "Replacing solvent molecule 2282 (atom 8785) with CLA\n",
            "Replacing solvent molecule 10410 (atom 33169) with CLA\n",
            "Replacing solvent molecule 6564 (atom 21631) with CLA\n",
            "Replacing solvent molecule 7750 (atom 25189) with CLA\n",
            "Replacing solvent molecule 1754 (atom 7201) with CLA\n",
            "Replacing solvent molecule 3153 (atom 11398) with CLA\n",
            "Replacing solvent molecule 11838 (atom 37453) with CLA\n",
            "Replacing solvent molecule 7213 (atom 23578) with CLA\n",
            "Replacing solvent molecule 11313 (atom 35878) with CLA\n",
            "Replacing solvent molecule 3648 (atom 12883) with CLA\n",
            "Replacing solvent molecule 2010 (atom 7969) with CLA\n",
            "Replacing solvent molecule 2357 (atom 9010) with CLA\n",
            "Replacing solvent molecule 1510 (atom 6469) with CLA\n",
            "Replacing solvent molecule 12574 (atom 39661) with CLA\n",
            "Replacing solvent molecule 2142 (atom 8365) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"There is no place like ~\" (Anonymous)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Run equilibration\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro \n",
        "gmx mdrun -deffnm 5_min\n",
        "gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro \n",
        "gmx mdrun -deffnm 6_nvt\n",
        "gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro \n",
        "gmx mdrun -deffnm 7_npt\n",
        "done"
      ],
      "metadata": {
        "id": "p-TrrUBhmPVe",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "562691d8-5548-491d-ba8f-52680b4fccd3"
      },
      "execution_count": 95,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Setting the LD random seed to -1622155281\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.443 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -908066849\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.442 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -771825809\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.441 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -272629796\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -1109426757\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.439 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -67150989\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.441 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -46239765\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -25428163\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.446 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -268501761\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.444 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.233 nm, buffer size 0.033 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 101715.00\n",
            "\n",
            "GROMACS reminds you: \"Chemistry: It tends to be a messy science.\" (Gunnar von Heijne, former chair of the Nobel Committee for chemistry)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 182 steps\n",
            "Potential Energy  = -7.8605156e+05\n",
            "Maximum force     =  9.0188641e+02 on atom 1366\n",
            "Norm of force     =  4.8009664e+01\n",
            "\n",
            "GROMACS reminds you: \"I Do It All the Time\" (Magnapop)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"In a talk you have a choice: You can make one point or no points.\" (Paul Sigler)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.298\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       63.801       31.901      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       27.087        0.886\n",
            "\n",
            "GROMACS reminds you: \"I am at two with nature.\" (Woody Allen)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"I cannot think of a single one, not even intelligence.\" (Enrico Fermi, when asked what characteristics physics Nobel laureates had in common.)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.343\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       26.701       13.351      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       64.728        0.371\n",
            "\n",
            "GROMACS reminds you: \"Uh-oh\" (Tinky Winky)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 97338.00\n",
            "\n",
            "GROMACS reminds you: \"I had a polynomial once. My doctor removed it.\" (Michael Grant)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "step 15: One or more water molecules can not be settled.\n",
            "Check for bad contacts and/or reduce the timestep if appropriate.\n",
            "Wrote pdb files with previous and current coordinates\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 364 steps\n",
            "Potential Energy  = -7.6430638e+05\n",
            "Maximum force     =  8.2403162e+02 on atom 164\n",
            "Norm of force     =  3.2836407e+01\n",
            "\n",
            "GROMACS reminds you: \"We'll celebrate a woman for anything, as long as it's not her talent.\" (Colleen McCullough)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"Since the mathematicians have invaded the theory of relativity I do not understand it myself any more.\" (Albert Einstein)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.298\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       70.294       35.147      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       24.585        0.976\n",
            "\n",
            "GROMACS reminds you: \"He's using code that only you and I know\" (Kate Bush)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"The time for theory is over\" (J. Hajdu)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       29.517       14.759      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       58.554        0.410\n",
            "\n",
            "GROMACS reminds you: \"All sorts of things can happen when you're open to new ideas and playing around with things.\" (Stephanie Kwolek, inventor of Kevlar)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 93234.00\n",
            "\n",
            "GROMACS reminds you: \"Garbage Collecting...\" (GNU Emacs)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 638 steps\n",
            "Potential Energy  = -7.4052944e+05\n",
            "Maximum force     =  8.1008386e+02 on atom 1107\n",
            "Norm of force     =  2.2840394e+01\n",
            "\n",
            "GROMACS reminds you: \"Shake Yourself\" (YES)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"You Should Sleep Late Man, It's Much Easier On Your Constitution\" (Beastie Boys)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.296\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       72.276       36.138      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       23.911        1.004\n",
            "\n",
            "GROMACS reminds you: \"All Beauty Must Die\" (Nick Cave)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"One Ripple At a Time\" (Bianca's Smut Shack)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.341\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       29.146       14.573      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       59.299        0.405\n",
            "\n",
            "GROMACS reminds you: \"Remember, being healthy is basically dying as slowly as possible.\" (Ricky Gervais)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
        "gmx mdrun -deffnm run -nsteps 1000000\n",
        "done"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GQqWYtC4_xvq",
        "outputId": "1209da99-439d-4e66-9d8f-19e349992a6b"
      },
      "execution_count": 96,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Setting the LD random seed to -889201811\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.450 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.136 0.137 0.132\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 30 Mb of data\n",
            "Setting the LD random seed to -86184001\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.452 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.134 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 29 Mb of data\n",
            "Setting the LD random seed to -788481\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.450 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.134 0.135 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 27 Mb of data\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"In mathematics you don't understand things, you just get used to them\" (John von Neumann)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.345\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     5581.285     2790.643      200.0\n",
            "                         46:30\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       61.921        0.388\n",
            "\n",
            "GROMACS reminds you: \"A Pretty Village Burning Makes a Pretty Fire\" (David Sandstrom)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"We Don't Bother Anyone\" (LIVE)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.345\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     4993.828     2496.914      200.0\n",
            "                         41:36\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       69.205        0.347\n",
            "\n",
            "GROMACS reminds you: \"Ubiquitin's just a rock\" (Berk Hess)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"Discovery: A couple of months in the laboratory can frequently save a couple of hours in the library.\" (Anonymous)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "NOTE: 10 % of the run time was spent in pair search,\n",
            "      you might want to increase nstlist (this has no effect on accuracy)\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     4349.434     2174.717      200.0\n",
            "                         36:14\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       79.459        0.302\n",
            "\n",
            "GROMACS reminds you: \"California, R.I.P.\" (Red Hot Chili Peppars)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title make trajectories whole\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "echo 0 | gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
        "done"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bzmIt5F6FLri",
        "outputId": "bedd697f-975a-46e3-d818-1f34c3f81936"
      },
      "execution_count": 97,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n",
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n",
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 49829 elements\n",
            "Group     1 (        Protein) has  1960 elements\n",
            "Group     2 (      Protein-H) has  1001 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1316 elements\n",
            "Group     9 (    SideChain-H) has   486 elements\n",
            "Group    10 (    Prot-Masses) has  1960 elements\n",
            "Group    11 (    non-Protein) has 47869 elements\n",
            "Group    12 (          Water) has 47769 elements\n",
            "Group    13 (            SOL) has 47769 elements\n",
            "Group    14 (      non-Water) has  2060 elements\n",
            "Group    15 (          Other) has   100 elements\n",
            "Group    16 (            SOD) has    46 elements\n",
            "Group    17 (            CLA) has    54 elements\n",
            "Select a group: \rReading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\rReading frame       1 time   10.000    ->  frame      0 time    0.000      \r\rReading frame       2 time   20.000    ->  frame      1 time   10.000      \r\rReading frame       3 time   30.000    ->  frame      2 time   20.000      \r\rReading frame       4 time   40.000    ->  frame      3 time   30.000      \r\rReading frame       5 time   50.000    ->  frame      4 time   40.000      \r\rReading frame       6 time   60.000    ->  frame      5 time   50.000      \r\rReading frame       7 time   70.000    ->  frame      6 time   60.000      \r\rReading frame       8 time   80.000    ->  frame      7 time   70.000      \r\rReading frame       9 time   90.000    ->  frame      8 time   80.000      \r\rReading frame      10 time  100.000    ->  frame      9 time   90.000      \r\rReading frame      11 time  110.000    ->  frame     10 time  100.000      \r\rReading frame      12 time  120.000    ->  frame     11 time  110.000      \r\rReading frame      13 time  130.000    ->  frame     12 time  120.000      \r\rReading frame      14 time  140.000    ->  frame     13 time  130.000      \r\rReading frame      15 time  150.000    ->  frame     14 time  140.000      \r\rReading frame      16 time  160.000    ->  frame     15 time  150.000      \r\rReading frame      17 time  170.000    ->  frame     16 time  160.000      \r\rReading frame      18 time  180.000    ->  frame     17 time  170.000      \r\rReading frame      19 time  190.000    ->  frame     18 time  180.000      \r\rReading frame      20 time  200.000    ->  frame     19 time  190.000      \r\rReading frame      30 time  300.000    ->  frame     29 time  290.000      \r\rReading frame      40 time  400.000    ->  frame     39 time  390.000      \r\rReading frame      50 time  500.000    ->  frame     49 time  490.000      \r\rReading frame      60 time  600.000    ->  frame     59 time  590.000      \r\rReading frame      70 time  700.000    ->  frame     69 time  690.000      \r\rReading frame      80 time  800.000    ->  frame     79 time  790.000      \r\rReading frame      90 time  900.000    ->  frame     89 time  890.000      \r\rReading frame     100 time 1000.000    ->  frame     99 time  990.000      \r\rReading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r\rReading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r\rReading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r\rReading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r\rReading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r\rReading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r\rReading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r\rReading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r\rReading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r\rReading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r\rLast frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"Microsecond Here I Come\" (P.J. Van Maaren)\n",
            "\n",
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 47649 elements\n",
            "Group     1 (        Protein) has  1949 elements\n",
            "Group     2 (      Protein-H) has   994 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1305 elements\n",
            "Group     9 (    SideChain-H) has   479 elements\n",
            "Group    10 (    Prot-Masses) has  1949 elements\n",
            "Group    11 (    non-Protein) has 45700 elements\n",
            "Group    12 (          Water) has 45606 elements\n",
            "Group    13 (            SOL) has 45606 elements\n",
            "Group    14 (      non-Water) has  2043 elements\n",
            "Group    15 (          Other) has    94 elements\n",
            "Group    16 (            SOD) has    43 elements\n",
            "Group    17 (            CLA) has    51 elements\n",
            "Select a group: \rReading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\rReading frame       1 time   10.000    ->  frame      0 time    0.000      \r\rReading frame       2 time   20.000    ->  frame      1 time   10.000      \r\rReading frame       3 time   30.000    ->  frame      2 time   20.000      \r\rReading frame       4 time   40.000    ->  frame      3 time   30.000      \r\rReading frame       5 time   50.000    ->  frame      4 time   40.000      \r\rReading frame       6 time   60.000    ->  frame      5 time   50.000      \r\rReading frame       7 time   70.000    ->  frame      6 time   60.000      \r\rReading frame       8 time   80.000    ->  frame      7 time   70.000      \r\rReading frame       9 time   90.000    ->  frame      8 time   80.000      \r\rReading frame      10 time  100.000    ->  frame      9 time   90.000      \r\rReading frame      11 time  110.000    ->  frame     10 time  100.000      \r\rReading frame      12 time  120.000    ->  frame     11 time  110.000      \r\rReading frame      13 time  130.000    ->  frame     12 time  120.000      \r\rReading frame      14 time  140.000    ->  frame     13 time  130.000      \r\rReading frame      15 time  150.000    ->  frame     14 time  140.000      \r\rReading frame      16 time  160.000    ->  frame     15 time  150.000      \r\rReading frame      17 time  170.000    ->  frame     16 time  160.000      \r\rReading frame      18 time  180.000    ->  frame     17 time  170.000      \r\rReading frame      19 time  190.000    ->  frame     18 time  180.000      \r\rReading frame      20 time  200.000    ->  frame     19 time  190.000      \r\rReading frame      30 time  300.000    ->  frame     29 time  290.000      \r\rReading frame      40 time  400.000    ->  frame     39 time  390.000      \r\rReading frame      50 time  500.000    ->  frame     49 time  490.000      \r\rReading frame      60 time  600.000    ->  frame     59 time  590.000      \r\rReading frame      70 time  700.000    ->  frame     69 time  690.000      \r\rReading frame      80 time  800.000    ->  frame     79 time  790.000      \r\rReading frame      90 time  900.000    ->  frame     89 time  890.000      \r\rReading frame     100 time 1000.000    ->  frame     99 time  990.000      \r\rReading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r\rReading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r\rReading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r\rReading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r\rReading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r\rReading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r\rReading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r\rReading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r\rReading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r\rReading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r\rLast frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"If at first you don't succeed, try two more times so that your failure is statistically significant.\" (Dallas Warren)\n",
            "\n",
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 45602 elements\n",
            "Group     1 (        Protein) has  1939 elements\n",
            "Group     2 (      Protein-H) has   989 elements\n",
            "Group     3 (        C-alpha) has   128 elements\n",
            "Group     4 (       Backbone) has   384 elements\n",
            "Group     5 (      MainChain) has   511 elements\n",
            "Group     6 (   MainChain+Cb) has   627 elements\n",
            "Group     7 (    MainChain+H) has   639 elements\n",
            "Group     8 (      SideChain) has  1300 elements\n",
            "Group     9 (    SideChain-H) has   478 elements\n",
            "Group    10 (    Prot-Masses) has  1939 elements\n",
            "Group    11 (    non-Protein) has 43663 elements\n",
            "Group    12 (          Water) has 43569 elements\n",
            "Group    13 (            SOL) has 43569 elements\n",
            "Group    14 (      non-Water) has  2033 elements\n",
            "Group    15 (          Other) has    94 elements\n",
            "Group    16 (            SOD) has    43 elements\n",
            "Group    17 (            CLA) has    51 elements\n",
            "Select a group: \rReading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\rReading frame       1 time   10.000    ->  frame      0 time    0.000      \r\rReading frame       2 time   20.000    ->  frame      1 time   10.000      \r\rReading frame       3 time   30.000    ->  frame      2 time   20.000      \r\rReading frame       4 time   40.000    ->  frame      3 time   30.000      \r\rReading frame       5 time   50.000    ->  frame      4 time   40.000      \r\rReading frame       6 time   60.000    ->  frame      5 time   50.000      \r\rReading frame       7 time   70.000    ->  frame      6 time   60.000      \r\rReading frame       8 time   80.000    ->  frame      7 time   70.000      \r\rReading frame       9 time   90.000    ->  frame      8 time   80.000      \r\rReading frame      10 time  100.000    ->  frame      9 time   90.000      \r\rReading frame      11 time  110.000    ->  frame     10 time  100.000      \r\rReading frame      12 time  120.000    ->  frame     11 time  110.000      \r\rReading frame      13 time  130.000    ->  frame     12 time  120.000      \r\rReading frame      14 time  140.000    ->  frame     13 time  130.000      \r\rReading frame      15 time  150.000    ->  frame     14 time  140.000      \r\rReading frame      16 time  160.000    ->  frame     15 time  150.000      \r\rReading frame      17 time  170.000    ->  frame     16 time  160.000      \r\rReading frame      18 time  180.000    ->  frame     17 time  170.000      \r\rReading frame      19 time  190.000    ->  frame     18 time  180.000      \r\rReading frame      20 time  200.000    ->  frame     19 time  190.000      \r\rReading frame      30 time  300.000    ->  frame     29 time  290.000      \r\rReading frame      40 time  400.000    ->  frame     39 time  390.000      \r\rReading frame      50 time  500.000    ->  frame     49 time  490.000      \r\rReading frame      60 time  600.000    ->  frame     59 time  590.000      \r\rReading frame      70 time  700.000    ->  frame     69 time  690.000      \r\rReading frame      80 time  800.000    ->  frame     79 time  790.000      \r\rReading frame      90 time  900.000    ->  frame     89 time  890.000      \r\rReading frame     100 time 1000.000    ->  frame     99 time  990.000      \r\rReading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r\rReading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r\rReading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r\rReading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r\rReading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r\rReading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r\rReading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r\rReading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r\rReading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r\rReading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r\rLast frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"I'm Only Faking When I Get It Right\" (Soundgarden)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Analysis"
      ],
      "metadata": {
        "id": "9WK7EtsiG_Wp"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title RMSD\n",
        "paths = [path]\n",
        "for i in range(mutant_n):\n",
        "    paths.append(f\"./md_mutant_{i}/\")\n",
        "\n",
        "first = True\n",
        "rmsdList = []\n",
        "for current_path in paths:\n",
        "    mobile = mda.Universe(f\"{current_path}run.gro\", f\"{current_path}run_nj.xtc\")\n",
        "    averageBB = align.AverageStructure(mobile, \n",
        "                                       mobile, \n",
        "                                       select='protein and backbone',\n",
        "                                       ref_frame=0).run()\n",
        "    refBB = averageBB.universe\n",
        "    R_rmsd = rms.RMSD(mobile,  # universe to align\n",
        "                     refBB,  # reference universe or atomgroup\n",
        "                     select='protein and backbone')  # group to superimpose and calculate RMSD\n",
        "        \n",
        "    R_rmsd.run()\n",
        "    if first:\n",
        "        df_rmsd = pd.DataFrame(R_rmsd.results.rmsd,\n",
        "                  columns=['Frame', 'Time (ns)',f'RMSD, {current_path}'])\n",
        "        df_rmsd['Time (ns)'] /= 1000\n",
        "        rmsdList.append(f'RMSD, {current_path}')\n",
        "        first = False\n",
        "    else:\n",
        "        colname = f'RMSD, {current_path}'\n",
        "        df_rmsd[colname] = R_rmsd.results.rmsd[:,2]\n",
        "        rmsdList.append(colname)\n",
        "\n",
        "df_rmsd_melt = df_rmsd.melt(id_vars='Time (ns)', value_vars=rmsdList)\n",
        "df_rmsd_melt.rename(columns = {'value': 'RMSD (A)'}, inplace = True)\n",
        "fig = px.line(df_rmsd_melt, x=\"Time (ns)\", y=\"RMSD (A)\",\n",
        "              color='variable',\n",
        "              line_shape=\"linear\", render_mode=\"svg\", \n",
        "              labels={ \"Name CA\": \"RMSD(Å)\" })\n",
        "fig.show()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 654
        },
        "id": "DBUzmhW5HCpl",
        "outputId": "c31d25b6-c628-4c7c-dc28-dd8239e95624"
      },
      "execution_count": 112,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "./md_1aki/\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/site-packages/MDAnalysis/analysis/align.py:950: DeprecationWarning: The `universe` attribute was deprecated in MDAnalysis 2.0.0 and will be removed in MDAnalysis 3.0.0. Please use `results.universe` instead.\n",
            "  warnings.warn(wmsg, DeprecationWarning)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "./md_mutant_0/\n",
            "./md_mutant_1/\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"854c7745-8e26-48ec-8dae-86a4eb8beb72\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"854c7745-8e26-48ec-8dae-86a4eb8beb72\")) {                    Plotly.newPlot(                        \"854c7745-8e26-48ec-8dae-86a4eb8beb72\",                        [{\"hovertemplate\":\"variable=RMSD, ./md_1aki/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_1aki/\",\"line\":{\"color\":\"#636efa\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_1aki/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[1.032046167612902,1.1185287201515637,0.9053039117305199,1.096550776622478,0.8569471999476059,0.6623549406936307,0.681778306499831,0.6267686443725318,0.6098904116771962,0.5982637032141044,0.6314692802560377,0.693832312370695,0.712689293885094,0.8309735508925363,0.7160732778835688,0.6532374845747646,0.637583591452282,0.6348954905915243,0.6966604526475105,0.6968950078536992,0.5944849709091232,0.7187256625097829,0.7363351007904025,0.6212901030506489,0.6729408208304348,0.5554389212747025,0.5617077111978223,0.56571268657775,0.5545824217700807,0.5680537328321923,0.5867247243876421,0.6693554558872484,0.6446586939657045,0.6363748680586134,0.6747921231735874,0.6071540103303,0.581907182760015,0.6946948248955014,0.677356549696271,0.6035162641964112,0.6589939192242869,0.619854907814415,0.7282623814625158,0.5284372238090012,0.7449222734214063,0.7046463591874618,0.6085383831947432,0.7566785381708997,0.6637487004864056,0.6877972796228419,0.5582861043728765,0.5276558828700373,0.6174995220814532,0.6727017523836123,0.6281796913945263,0.5495968962493648,0.6351747017371681,0.627287437102995,0.6303993692380706,0.5223752449776417,0.5824735785345144,0.6297267860436583,0.6051573215984482,0.6800974643706139,0.49395036454330254,0.7096282415383902,0.6776376846124672,0.6839558851179192,0.503129719082874,0.477509853512699,0.642866351139969,0.5453779865726373,0.7122522772563246,0.5283285702648822,0.4672028176681692,0.5368303436243438,0.4943627453117062,0.49458767344570154,0.4722835768199092,0.6474300248071415,0.5727165406237953,0.5000463494759019,0.5544827918471928,0.5952259778642874,0.6239664479451075,0.5265406343592773,0.47385002909762913,0.5604109630882516,0.7567102715644147,0.6451527482447709,0.5256640719773212,0.5336940938226433,0.5134490500180161,0.5562553109110584,0.596945607466486,0.473013639605088,0.46527565681500954,0.5027812029163647,0.5389167862401805,0.5153942210911894,0.5201799395774083,0.49959258409376667,0.5397946007938622,0.49591918809466995,0.5323664873025759,0.5423094599456173,0.5667855420291555,0.49317051639364506,0.5549037716225255,0.48850835121472896,0.48057737021195485,0.5273444140210441,0.6031341673344883,0.531098828144626,0.548709306447126,0.6570930520056151,0.6112580596105617,0.5886995673336353,0.7032169544519802,0.6725294105946029,0.6416115398351864,0.5384127950433936,0.7768893319773776,0.45030929247685386,0.5318452083340797,0.5730391501522434,0.5307333166507427,0.4482125337847616,0.5766388631623679,0.6344248023151627,0.5671042237740269,0.4974590251630182,0.4703031180631319,0.47456756809704925,0.5329775907912593,0.5097068974120458,0.5360676142061581,0.5308997944201521,0.45687088195467074,0.5905022284130305,0.6829611404740132,0.5872946675500536,0.6091910058607112,0.510219196220267,0.6127036205121118,0.6075800898217022,0.5342136302294059,0.5223155829285429,0.5713120435239806,0.6491385378317993,0.6953940143876028,0.6493919449331373,0.5830900808979392,0.539874618628931,0.5089336679581667,0.5603144172749372,0.4857634968170615,0.5001399144772399,0.5375872767908646,0.5130743726167859,0.4987211097126465,0.5254609627848581,0.5609490931481168,0.5739689187496014,0.5181234023628852,0.4894535931390501,0.5311505706603818,0.6222725105780872,0.6842645585526926,0.6113872402195338,0.5239929648614972,0.5134400748468773,0.5419438074088183,0.5832612267847865,0.6521648959541768,0.5618551705459967,0.6517564364165281,0.5579349614312972,0.6257294629498745,0.6038451978992304,0.5025174422235265,0.5221754448601554,0.706579533796579,0.6945046350765325,0.5565235006334751,0.5510581667068304,0.5540912903837921,0.6119194012037941,0.5621954538931098,0.7043451551236283,0.5400545699449205,0.7175302878048971,0.5483601628092931,0.5517408301534846,0.5781762746694978,0.5911353207522784,0.6152167441704078,0.6068584123042037,0.6381491711222889,0.7469851976454915,0.6399852285441667],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"variable=RMSD, ./md_mutant_0/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_mutant_0/\",\"line\":{\"color\":\"#EF553B\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_mutant_0/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[0.8950691283213248,0.9330506702836896,0.9629492534352861,0.9014369264225996,0.7098989785086244,0.7505119550103851,0.7513122126319364,0.6530786464340292,0.6200255894062668,0.5640816637011241,0.7623003565500289,0.6764313501797952,0.5467265807133349,0.5398768259215796,0.7689001157486994,0.7149990634640426,0.8092735838233949,0.8347750123117836,1.034462286506098,0.7985911626554008,0.7874113701577683,0.719529463281206,0.7230121280539931,0.6931137170986094,0.6926764559225104,0.6381858336002099,0.7750054793586917,0.7408981270547716,0.7701567411220986,0.7347039292206463,0.7138656439356634,0.8267962279336547,0.7190578480662766,0.6813235570945719,0.7496681841796925,0.7106150422507188,0.6959149924416773,0.6080627025461992,0.7475703261295766,0.732083121373128,0.7273982151750275,0.70579346420091,0.6375994271319497,0.7507100050081401,0.5224469929122376,0.5434694670148456,0.6776691456877357,0.6181507678343403,0.62775583755458,0.6587004692333172,0.4781809558165378,0.5425834904913681,0.46823171139941655,0.6312597070779906,0.7224533084644799,0.7676083510864062,0.5876178505554986,0.7459100815976574,0.6344625557504687,0.7193169424373183,0.632230499657909,0.6217559046485365,0.5947592761297589,0.6130246775273163,0.5274825665917566,0.5974213880666223,0.6176548628621945,0.4949588600737308,0.5735533767433332,0.6286903300489566,0.6845973266064451,0.6351886884433041,0.6844237676458131,0.7560516958130707,0.6989574083739538,0.6912306466927657,0.5637417832456513,0.6374796085100722,0.5919329478702928,0.5434883786035933,0.5396041949032179,0.60220613793613,0.5598555626015238,0.5843503989583712,0.6618408944202693,0.5983442354351383,0.6851218622919669,0.9507571609721818,0.6833152511858949,0.5242076175566981,0.5763003002160035,0.6715270480047165,0.8887988878311499,0.8490826794928182,0.643381193705708,0.5888937606064665,0.6350024874655006,0.6443039464340006,0.6712712330469355,0.6504860610204671,0.7368531895112104,0.6647912928283207,0.7904681773235823,0.7802854321050882,0.8387152534052721,0.6590322634432528,0.7238031776111834,0.9432692544191965,0.713904726068527,0.6266258338322745,0.6453869909848863,0.657935857861034,0.6811815875737337,0.6396187487818671,0.6975445941928636,0.7010261141240398,0.6408211138740119,0.6660896311808716,0.5853396187533817,0.7145635150168108,0.6186255438499946,0.5969519118315705,0.6467559794342275,0.6819658827264542,0.63603311119373,0.6297738514893394,0.6225069217673148,0.6078896665089722,0.716574220280723,0.6732675347236662,0.7320402148167336,0.652992152103134,0.8146826711798022,0.7377276342591335,0.8383411670454468,0.7169322636177792,0.6372328068713391,0.6799146634177806,0.6979552894721543,0.6732937528065969,0.5147819087263966,0.7103580934997419,0.6646651577988654,0.7218888097353934,0.647117365251897,0.5866094505636923,0.5193080975838472,0.514588632757893,0.5585843395890192,0.7113798686596811,0.5892110034606793,0.7072381209051167,0.8028877901263968,0.8443576546576514,0.8824911404474756,0.7727269074241718,0.8231731201017233,0.569210109057893,0.667688395627092,0.6469625402601931,0.6121805788388087,0.5800448617207979,0.7531894694816159,0.79164443364741,0.6881048781190203,0.6709236924674382,0.7296252194581856,0.8078610477862908,0.844967898079777,0.7766364638274493,0.8521839105105198,0.6889348272752375,0.7267464809056479,0.7248996009292756,0.690364934492127,0.7574402728071652,0.7358700914273146,0.9050942775961274,0.8762343208476625,0.7971999500506398,0.6703127200060552,0.6729923052959642,0.6667534808855887,0.814564951262505,0.6395621606725868,0.7736264800214666,0.7769443120538794,0.7929281953092027,0.7315605647671329,0.6027703920405673,0.5856677975023528,0.7829390242842506,0.5948421851195214,0.6183969350968722,0.7055657580113154,0.6522341476118848,0.5936605970498846,0.6635194333352338,0.6181620469403598,0.7194649284482415,0.7399138541163629],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"variable=RMSD, ./md_mutant_1/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_mutant_1/\",\"line\":{\"color\":\"#00cc96\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_mutant_1/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[0.9801477763521751,0.7880427152972244,0.855545680442817,0.9852744314907413,1.0131982797764445,0.9438787906356434,1.0493673124661576,0.9178856029428756,0.8613044180099316,0.8140677535784075,0.7914496444308067,0.8375986105687059,0.8617376472662067,0.7717351714654814,0.8606976679147849,0.7952436805567896,0.899346173760099,0.8873194459681459,0.8146739866242194,0.8847531675511334,0.7244472301373095,0.7036565629883436,0.8703620150539823,0.8292691049672817,0.9380888746001486,0.9553133334139056,0.8088296557040754,0.7892944283438644,0.8936410804253999,0.9009203833265458,0.7254004508685998,0.7987462852492363,0.8209855220426773,0.9121004782880905,0.9551512753727368,0.9245663192735264,0.9458905778377926,1.002314978443962,0.8343143755590204,0.7242336562646784,0.8701838923081673,0.771123612713634,0.7946482685502512,0.8413198178136733,0.8106790068479435,0.8046486453146658,0.8133706623098602,0.9363340308969317,0.6968704708690712,0.7198541990940941,0.8382538235233616,0.6862918216652732,0.7316491285935042,1.011422118326304,0.7159748387155725,0.7621289630216386,0.6296779195103905,0.7673039429803535,0.7618417772883694,0.7231138785189741,0.8717399289271559,0.7433296274869668,0.7164751471776556,0.6554624890758449,0.6435088802770018,0.8406378535508745,0.8370171189743946,0.7102029924398566,0.7479607095197128,0.622869490840246,0.7042779384729997,0.833552091528869,0.8297763699612104,0.8326646266419323,0.8316202391681293,0.9772765906967015,0.678053232633059,0.7187827285582736,0.7550707080191973,0.8704363246915319,0.8708078924670594,0.9281098922205859,0.8589361115064311,0.7288356515241313,0.8512855445662045,0.6627894510293071,0.7053927958389011,0.8802814891260673,0.95836367596443,0.8457308532931829,0.7297183625964304,0.7913492740640258,0.8017741500782968,0.7707714603620058,0.8558789074162402,0.7737279464051118,0.6698573052220069,0.7777856160151843,0.9182922682636672,0.8954752658166047,0.782690896457608,0.8927864738789812,0.9696228893456047,0.6137263580929448,0.7499332353206388,0.7757206616740235,0.6754659427031056,0.7308503454809736,0.714478335554908,0.749612319345148,0.7774792800581328,0.693611127375695,0.5876034826083185,0.6533038936717077,0.77711312315606,0.5827270204025208,0.6479627591875162,0.6335492112967199,0.6715950396428507,0.5165550723484915,0.6904951328684757,0.5929053820733184,0.6808357084585306,0.7447913468119552,0.739387751935155,0.5821888007887761,0.6131891933184735,0.6010149372228576,0.757973055263559,0.6586994822060873,0.7868565473909399,0.7829711050051074,0.7964145035476736,0.9043776379826093,0.8793037787018592,0.7471814769082922,0.7639049971985936,0.8374664452964428,0.758754496589933,0.8932435987403324,0.904381263997207,0.8150407213581856,0.7330954546306339,0.9831445040289231,1.0005509222225448,0.9799861045117457,0.9622165390970389,0.8895985450789385,0.987281233514066,0.9363903589063867,1.0090875656183864,0.8925997653681251,0.9154517668635428,1.022422179574082,1.0658641614477469,0.8786290449154478,0.9122093922322302,1.0286609308553567,1.0192859402892167,0.9371928032601478,0.9033347373562319,0.9159066993817582,1.0494739697877378,0.9225100581385169,0.9202461861837705,0.9913820301545369,0.8749338142921257,0.9908411158304186,0.9314751755753814,0.9618641009600561,0.942307683182546,1.0168523146851722,1.0344501103788795,1.0571932747302495,0.9753629064633156,1.1111568171910222,1.1449594056786,0.9896398506685339,1.191565211129711,1.0912233936043214,1.1411857137624648,1.1307602327357458,1.0600781635853889,1.2748604909420511,1.1219382885211775,0.9410162405313741,0.9320139592991049,1.0715946120159583,1.1264391615369993,1.0120337170762779,1.3316205061294446,1.1837662046859199,1.1489589101031874,1.17940553152056,1.1361321031309486,1.1123982978599145,1.1167852429906082,1.0219278207083773,1.205790979448861,1.111168015303999,1.046476108130852],\"yaxis\":\"y\",\"type\":\"scatter\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Time (ns)\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"RMSD (A)\"}},\"legend\":{\"title\":{\"text\":\"variable\"},\"tracegroupgap\":0},\"margin\":{\"t\":60}},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('854c7745-8e26-48ec-8dae-86a4eb8beb72');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "QgBYxxDKo_9x"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}