{
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "TODO: \n",
        "- integrate colab fold\n",
        "- PIP freeze has to be added/check optimum setup\n",
        "- run tests\n",
        "- add option to provide mutations as a list"
      ],
      "metadata": {
        "id": "WKuDjvoKOuEi"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VotFxKloOQk8"
      },
      "source": [
        "# <b><font color='#009e74'>End-to-end molecular dynamics simulation with AlphaFold and Gromacs </font></b>\n",
        "\n",
        "Preprint pipeline version for running MD simulations using **AlphaFold and GROMACS**. The program, using as input a protein sequence returns MD trajectories and all the associated files to run it.\n",
        "More details can be found in: **Manuscript...:** [\"Manuscript title\"](https://ref). Source code is available on the project [Github](https://github.com/ref) page."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-Q-Y3BeROQk_"
      },
      "source": [
        "##  <b><font color='#009e74'> Reminders and Important informations:</font></b>\n",
        "- This notebook  <b><font color='#d55c00'> can </font></b> be run in a Colab GPU session (go to page menu: `Runtime`->  `Change runtime type` -> select `GPU` and confirm\n",
        "- Cells named as  <b><font color='#56b4e9'>PRELIMINARY OPERATIONS </font></b> have to be run <b><font color='#d55c00'>ONCE only at the start</font></b>  and  skipped for new runnings.\n",
        "- <b><font color='#d55c00'>ONE</font></b> single sequence at the time can be processed by the pipeline. \n",
        "- A  <b><font color='#d55c00'>new run</font></b> can be perform input direcly the new structure in the pdb upload cell and run the pipeline cells again\n",
        "\n",
        "****"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "N98LOBXdOQlA"
      },
      "source": [
        "## <b><font color='#009e74'>PRELIMINARY OPERATIONS </font></b>\n",
        "These cells should be run once at the start of the notebook\n",
        "****"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-mdLQUV-OQlA",
        "outputId": "102ee510-d785-44ff-a592-b088a9df3e5b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting condacolab\n",
            "  Downloading condacolab-0.1.5-py3-none-any.whl (6.8 kB)\n",
            "Installing collected packages: condacolab\n",
            "Successfully installed condacolab-0.1.5\n",
            "⏬ Downloading https://github.com/jaimergp/miniforge/releases/latest/download/Mambaforge-colab-Linux-x86_64.sh...\n",
            "📦 Installing...\n",
            "📌 Adjusting configuration...\n",
            "🩹 Patching environment...\n",
            "⏲ Done in 0:00:14\n",
            "🔁 Restarting kernel...\n"
          ]
        }
      ],
      "source": [
        "#@title Install condacolab\n",
        "#@markdown Run this cell to install condacolab. After running this cell the kernel will be automatically restarted (wait ~1min before run the next cell)\n",
        "\n",
        "#@markdown **N.B: This cell should be run only ONCE at the START of the notebook.**\n",
        "try:\n",
        "    import google.colab\n",
        "    !pip install condacolab\n",
        "    import condacolab\n",
        "    condacolab.install()\n",
        "except ModuleNotFoundError:\n",
        "    pass"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5iYQ0-TnspXN",
        "outputId": "c38166ac-a7d3-4f6a-cd4e-b923b4cc7194"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Get:1 https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/ InRelease [3,622 B]\n",
            "Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]\n",
            "Get:3 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal InRelease [18.1 kB]\n",
            "Ign:4 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64  InRelease\n",
            "Hit:5 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64  InRelease\n",
            "Hit:6 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64  Release\n",
            "Hit:7 http://archive.ubuntu.com/ubuntu focal InRelease\n",
            "Get:9 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]\n",
            "Hit:10 http://ppa.launchpad.net/cran/libgit2/ubuntu focal InRelease\n",
            "Hit:11 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal InRelease\n",
            "Get:12 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2,437 kB]\n",
            "Hit:13 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu focal InRelease\n",
            "Get:14 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]\n",
            "Get:15 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal/main Sources [2,377 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2,003 kB]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [2,910 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,286 kB]\n",
            "Get:19 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal/main amd64 Packages [1,126 kB]\n",
            "Get:20 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [984 kB]\n",
            "Fetched 13.5 MB in 6s (2,311 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "28 packages can be upgraded. Run 'apt list --upgradable' to see them.\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "g++-9 is already the newest version (9.4.0-1ubuntu1~20.04.1).\n",
            "g++-9 set to manually installed.\n",
            "gcc-9 is already the newest version (9.4.0-1ubuntu1~20.04.1).\n",
            "gcc-9 set to manually installed.\n",
            "0 upgraded, 0 newly installed, 0 to remove and 28 not upgraded.\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:3 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n",
            "\u001b[1;33mW: \u001b[0mTarget Packages (universe/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:14 and /etc/apt/sources.list:54\u001b[0m\n"
          ]
        }
      ],
      "source": [
        "#@title Update gcc\n",
        "#@markdown Run this cell to install gcc-9. This is required for GROMACS\n",
        "!sudo echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal main universe' >> /etc/apt/sources.list\n",
        "!sudo apt update\n",
        "!sudo apt install -y gcc-9 g++-9\n",
        "!ln -snf /usr/bin/gcc-9 /usr/bin/gcc\n",
        "!ln -snf /usr/bin/g++-9 /usr/bin/g++"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "TSFKHvOjtZaH",
        "outputId": "46f08769-9805-4f0f-ed8e-9c533c647f15"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting biopython\n",
            "  Downloading biopython-1.80-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m53.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting numpy\n",
            "  Downloading numpy-1.24.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m17.3/17.3 MB\u001b[0m \u001b[31m79.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: numpy, biopython\n",
            "Successfully installed biopython-1.80 numpy-1.24.1\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Installing biopython\n",
        "!pip install biopython\n",
        "from Bio.PDB import *\n",
        "import Bio.SeqUtils\n",
        "import Bio.SeqIO\n",
        "from Bio.SeqRecord import SeqRecord\n",
        "from Bio.Seq import Seq"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pm9AAM1YtsT3",
        "outputId": "9dfa32c0-f60c-43b1-9310-77ac2b50d733"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): ...working... done\n",
            "Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.\n",
            "Collecting package metadata (repodata.json): ...working... done\n",
            "Solving environment: ...working... done\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - openmm\n",
            "    - pdbfixer\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ca-certificates-2022.12.7  |       ha878542_0         143 KB  conda-forge\n",
            "    certifi-2022.12.7          |     pyhd8ed1ab_0         147 KB  conda-forge\n",
            "    colorama-0.4.6             |     pyhd8ed1ab_0          25 KB  conda-forge\n",
            "    conda-22.11.1              |   py38h578d9bd_1         905 KB  conda-forge\n",
            "    cudatoolkit-11.2.2         |      hbe64b41_11       560.3 MB  conda-forge\n",
            "    fftw-3.3.10                |nompi_hf0379b8_106         2.1 MB  conda-forge\n",
            "    libblas-3.9.0              |16_linux64_openblas          13 KB  conda-forge\n",
            "    libcblas-3.9.0             |16_linux64_openblas          13 KB  conda-forge\n",
            "    libgfortran-ng-12.2.0      |      h69a702a_19          22 KB  conda-forge\n",
            "    libgfortran5-12.2.0        |      h337968e_19         1.8 MB  conda-forge\n",
            "    liblapack-3.9.0            |16_linux64_openblas          13 KB  conda-forge\n",
            "    libopenblas-0.3.21         |pthreads_h78a6416_3        10.1 MB  conda-forge\n",
            "    numpy-1.24.1               |   py38hab0fcb9_0         6.3 MB  conda-forge\n",
            "    ocl-icd-2.3.1              |       h7f98852_0         119 KB  conda-forge\n",
            "    ocl-icd-system-1.0.0       |                1           4 KB  conda-forge\n",
            "    openmm-7.7.0               |   py38h4b4c9af_1        11.3 MB  conda-forge\n",
            "    pdbfixer-1.8.1             |     pyh6c4a22f_0         498 KB  conda-forge\n",
            "    pluggy-1.0.0               |     pyhd8ed1ab_5          16 KB  conda-forge\n",
            "    ruamel.yaml-0.17.21        |   py38h0a891b7_2         172 KB  conda-forge\n",
            "    ruamel.yaml.clib-0.2.7     |   py38h1de0b5d_1         143 KB  conda-forge\n",
            "    tqdm-4.64.1                |     pyhd8ed1ab_0          82 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:       594.1 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  colorama           conda-forge/noarch::colorama-0.4.6-pyhd8ed1ab_0 None\n",
            "  cudatoolkit        conda-forge/linux-64::cudatoolkit-11.2.2-hbe64b41_11 None\n",
            "  fftw               conda-forge/linux-64::fftw-3.3.10-nompi_hf0379b8_106 None\n",
            "  libblas            conda-forge/linux-64::libblas-3.9.0-16_linux64_openblas None\n",
            "  libcblas           conda-forge/linux-64::libcblas-3.9.0-16_linux64_openblas None\n",
            "  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-12.2.0-h69a702a_19 None\n",
            "  libgfortran5       conda-forge/linux-64::libgfortran5-12.2.0-h337968e_19 None\n",
            "  liblapack          conda-forge/linux-64::liblapack-3.9.0-16_linux64_openblas None\n",
            "  libopenblas        conda-forge/linux-64::libopenblas-0.3.21-pthreads_h78a6416_3 None\n",
            "  numpy              conda-forge/linux-64::numpy-1.24.1-py38hab0fcb9_0 None\n",
            "  ocl-icd            conda-forge/linux-64::ocl-icd-2.3.1-h7f98852_0 None\n",
            "  ocl-icd-system     conda-forge/linux-64::ocl-icd-system-1.0.0-1 None\n",
            "  openmm             conda-forge/linux-64::openmm-7.7.0-py38h4b4c9af_1 None\n",
            "  pdbfixer           conda-forge/noarch::pdbfixer-1.8.1-pyh6c4a22f_0 None\n",
            "  pluggy             conda-forge/noarch::pluggy-1.0.0-pyhd8ed1ab_5 None\n",
            "  ruamel.yaml        conda-forge/linux-64::ruamel.yaml-0.17.21-py38h0a891b7_2 None\n",
            "  ruamel.yaml.clib   conda-forge/linux-64::ruamel.yaml.clib-0.2.7-py38h1de0b5d_1 None\n",
            "  tqdm               conda-forge/noarch::tqdm-4.64.1-pyhd8ed1ab_0 None\n",
            "\n",
            "The following packages will be UPDATED:\n",
            "\n",
            "  ca-certificates                      2022.9.24-ha878542_0 --> 2022.12.7-ha878542_0 None\n",
            "  certifi                            2022.9.24-pyhd8ed1ab_0 --> 2022.12.7-pyhd8ed1ab_0 None\n",
            "  conda                               22.9.0-py38h578d9bd_2 --> 22.11.1-py38h578d9bd_1 None\n",
            "\n",
            "\n",
            "Preparing transaction: ...working... done\n",
            "Verifying transaction: ...working... done\n",
            "Executing transaction: ...working... By downloading and using the CUDA Toolkit conda packages, you accept the terms and conditions of the CUDA End User License Agreement (EULA): https://docs.nvidia.com/cuda/eula/index.html\n",
            "\n",
            "done\n",
            "Retrieving notices: ...working... done\n"
          ]
        }
      ],
      "source": [
        "#@title Installing PDBfixer\n",
        "#@markdown The repair of PDB is now done with PDBfixer\n",
        "\n",
        "#@markdown We can also add other options, like modeler, later on\n",
        "!conda install -q -y -c conda-forge pdbfixer openmm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ubtGWa9ctyN3",
        "outputId": "a14fd457-2bdb-489d-d072-db4b2e802e72"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting py3Dmol\n",
            "  Downloading py3Dmol-2.0.0.post2-py2.py3-none-any.whl (6.7 kB)\n",
            "Installing collected packages: py3Dmol\n",
            "Successfully installed py3Dmol-2.0.0.post2\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mLooking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting pdb-tools\n",
            "  Downloading pdb_tools-2.5.0-py3-none-any.whl (207 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m207.3/207.3 kB\u001b[0m \u001b[31m6.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: pdb-tools\n",
            "Successfully installed pdb-tools-2.5.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Install py3Dmol and pdb-tools\n",
        "!pip install py3Dmol\n",
        "import py3Dmol\n",
        "!pip install pdb-tools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dahUiTcwstzr",
        "outputId": "fc01b0f1-d0ac-4223-89b9-92a91870e833"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting wget\n",
            "  Downloading wget-3.2.zip (10 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Building wheels for collected packages: wget\n",
            "  Building wheel for wget (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for wget: filename=wget-3.2-py3-none-any.whl size=9656 sha256=afcf967912340fb4000f07cd7bf371d83570b8b9b68164c213ccb5c3a1f6303b\n",
            "  Stored in directory: /root/.cache/pip/wheels/46/4a/43/6e71c9584e8b20b326931deaf89f11bf4db1bda170ac38e7b4\n",
            "Successfully built wget\n",
            "Installing collected packages: wget\n",
            "Successfully installed wget-3.2\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title python libs\n",
        "!pip install wget\n",
        "import os\n",
        "import subprocess\n",
        "import shutil\n",
        "import pdbfixer\n",
        "from openmm.app import PDBFile\n",
        "import wget\n",
        "import tarfile\n",
        "import pandas as pd\n",
        "import plotly.graph_objects as go\n",
        "import plotly.express as px\n",
        "import numpy as np\n",
        "from google.colab import files"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WFX55eW2ulgc",
        "outputId": "3ea82719-da40-425f-9576-c59f50081134"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Solving environment: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - pytorch\n",
            "    - torchvision\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    _openmp_mutex-4.5          |       2_kmp_llvm           6 KB  conda-forge\n",
            "    aom-3.5.0                  |       h27087fc_0         2.7 MB  conda-forge\n",
            "    cudnn-8.4.1.50             |       hed8a83a_0       618.4 MB  conda-forge\n",
            "    expat-2.5.0                |       h27087fc_0         189 KB  conda-forge\n",
            "    ffmpeg-5.1.2               | gpl_h8dda1f0_105         9.2 MB  conda-forge\n",
            "    font-ttf-dejavu-sans-mono-2.37|       hab24e00_0         388 KB  conda-forge\n",
            "    font-ttf-inconsolata-3.000 |       h77eed37_0          94 KB  conda-forge\n",
            "    font-ttf-source-code-pro-2.038|       h77eed37_0         684 KB  conda-forge\n",
            "    font-ttf-ubuntu-0.83       |       hab24e00_0         1.9 MB  conda-forge\n",
            "    fontconfig-2.14.1          |       hc2a2eb6_0         365 KB  conda-forge\n",
            "    fonts-conda-ecosystem-1    |                0           4 KB  conda-forge\n",
            "    fonts-conda-forge-1        |                0           4 KB  conda-forge\n",
            "    freetype-2.12.1            |       hca18f0e_1         611 KB  conda-forge\n",
            "    gettext-0.21.1             |       h27087fc_0         4.1 MB  conda-forge\n",
            "    gmp-6.2.1                  |       h58526e2_0         806 KB  conda-forge\n",
            "    gnutls-3.7.8               |       hf3e180e_0         2.2 MB  conda-forge\n",
            "    jpeg-9e                    |       h166bdaf_2         269 KB  conda-forge\n",
            "    lame-3.100                 |    h166bdaf_1003         496 KB  conda-forge\n",
            "    lcms2-2.14                 |       hfd0df8a_1         235 KB  conda-forge\n",
            "    lerc-4.0.0                 |       h27087fc_0         275 KB  conda-forge\n",
            "    libdeflate-1.17            |       h0b41bf4_0          63 KB  conda-forge\n",
            "    libdrm-2.4.114             |       h166bdaf_0         298 KB  conda-forge\n",
            "    libhwloc-2.8.0             |       h32351e8_1         3.0 MB  conda-forge\n",
            "    libidn2-2.3.4              |       h166bdaf_0         157 KB  conda-forge\n",
            "    libjpeg-turbo-2.1.4        |       h166bdaf_0         988 KB  conda-forge\n",
            "    libpciaccess-0.17          |       h166bdaf_0          39 KB  conda-forge\n",
            "    libpng-1.6.39              |       h753d276_0         276 KB  conda-forge\n",
            "    libprotobuf-3.21.12        |       h3eb15da_0         2.1 MB  conda-forge\n",
            "    libtasn1-4.19.0            |       h166bdaf_0         114 KB  conda-forge\n",
            "    libtiff-4.5.0              |       h6adf6a1_2         397 KB  conda-forge\n",
            "    libunistring-0.9.10        |       h7f98852_0         1.4 MB  conda-forge\n",
            "    libva-2.17.0               |       h0b41bf4_0         183 KB  conda-forge\n",
            "    libvpx-1.11.0              |       h9c3ff4c_3         1.1 MB  conda-forge\n",
            "    libwebp-base-1.2.4         |       h166bdaf_0         404 KB  conda-forge\n",
            "    libxcb-1.13                |    h7f98852_1004         391 KB  conda-forge\n",
            "    llvm-openmp-15.0.7         |       h0cdce71_0         3.1 MB  conda-forge\n",
            "    magma-2.6.2                |       hc72dce7_0       231.5 MB  conda-forge\n",
            "    mkl-2022.2.1               |   h84fe81f_16997       157.3 MB  conda-forge\n",
            "    nccl-2.14.3.1              |       h0800d71_0       145.2 MB  conda-forge\n",
            "    nettle-3.8.1               |       hc379101_1         1.1 MB  conda-forge\n",
            "    ninja-1.11.0               |       h924138e_0         2.8 MB  conda-forge\n",
            "    openh264-2.3.1             |       h27087fc_1         1.5 MB  conda-forge\n",
            "    openjpeg-2.5.0             |       hfec8fc6_2         344 KB  conda-forge\n",
            "    p11-kit-0.24.1             |       hc5aa10d_0         4.5 MB  conda-forge\n",
            "    pillow-9.4.0               |   py38hb32c036_0        44.1 MB  conda-forge\n",
            "    pthread-stubs-0.4          |    h36c2ea0_1001           5 KB  conda-forge\n",
            "    pytorch-1.13.1             |cuda112py38hd94e077_200       362.3 MB  conda-forge\n",
            "    pytorch-mutex-1.0          |              cpu           3 KB  pytorch\n",
            "    sleef-3.5.1                |       h9b69904_2         1.5 MB  conda-forge\n",
            "    svt-av1-1.4.1              |       hcb278e6_0         2.4 MB  conda-forge\n",
            "    tbb-2021.7.0               |       h924138e_1         1.5 MB  conda-forge\n",
            "    torchvision-0.14.1         |         py38_cpu        25.3 MB  pytorch\n",
            "    typing_extensions-4.4.0    |     pyha770c72_0          29 KB  conda-forge\n",
            "    x264-1!164.3095            |       h166bdaf_2         877 KB  conda-forge\n",
            "    x265-3.5                   |       h924138e_3         3.2 MB  conda-forge\n",
            "    xorg-fixesproto-5.0        |    h7f98852_1002           9 KB  conda-forge\n",
            "    xorg-kbproto-1.0.7         |    h7f98852_1002          27 KB  conda-forge\n",
            "    xorg-libx11-1.7.2          |       h7f98852_0         941 KB  conda-forge\n",
            "    xorg-libxau-1.0.9          |       h7f98852_0          13 KB  conda-forge\n",
            "    xorg-libxdmcp-1.1.3        |       h7f98852_0          19 KB  conda-forge\n",
            "    xorg-libxext-1.3.4         |       h7f98852_1          54 KB  conda-forge\n",
            "    xorg-libxfixes-5.0.3       |    h7f98852_1004          18 KB  conda-forge\n",
            "    xorg-xextproto-7.3.0       |    h7f98852_1002          28 KB  conda-forge\n",
            "    xorg-xproto-7.0.31         |    h7f98852_1007          73 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        1.60 GB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  aom                conda-forge/linux-64::aom-3.5.0-h27087fc_0 \n",
            "  cudnn              conda-forge/linux-64::cudnn-8.4.1.50-hed8a83a_0 \n",
            "  expat              conda-forge/linux-64::expat-2.5.0-h27087fc_0 \n",
            "  ffmpeg             conda-forge/linux-64::ffmpeg-5.1.2-gpl_h8dda1f0_105 \n",
            "  font-ttf-dejavu-s~ conda-forge/noarch::font-ttf-dejavu-sans-mono-2.37-hab24e00_0 \n",
            "  font-ttf-inconsol~ conda-forge/noarch::font-ttf-inconsolata-3.000-h77eed37_0 \n",
            "  font-ttf-source-c~ conda-forge/noarch::font-ttf-source-code-pro-2.038-h77eed37_0 \n",
            "  font-ttf-ubuntu    conda-forge/noarch::font-ttf-ubuntu-0.83-hab24e00_0 \n",
            "  fontconfig         conda-forge/linux-64::fontconfig-2.14.1-hc2a2eb6_0 \n",
            "  fonts-conda-ecosy~ conda-forge/noarch::fonts-conda-ecosystem-1-0 \n",
            "  fonts-conda-forge  conda-forge/noarch::fonts-conda-forge-1-0 \n",
            "  freetype           conda-forge/linux-64::freetype-2.12.1-hca18f0e_1 \n",
            "  gettext            conda-forge/linux-64::gettext-0.21.1-h27087fc_0 \n",
            "  gmp                conda-forge/linux-64::gmp-6.2.1-h58526e2_0 \n",
            "  gnutls             conda-forge/linux-64::gnutls-3.7.8-hf3e180e_0 \n",
            "  jpeg               conda-forge/linux-64::jpeg-9e-h166bdaf_2 \n",
            "  lame               conda-forge/linux-64::lame-3.100-h166bdaf_1003 \n",
            "  lcms2              conda-forge/linux-64::lcms2-2.14-hfd0df8a_1 \n",
            "  lerc               conda-forge/linux-64::lerc-4.0.0-h27087fc_0 \n",
            "  libdeflate         conda-forge/linux-64::libdeflate-1.17-h0b41bf4_0 \n",
            "  libdrm             conda-forge/linux-64::libdrm-2.4.114-h166bdaf_0 \n",
            "  libhwloc           conda-forge/linux-64::libhwloc-2.8.0-h32351e8_1 \n",
            "  libidn2            conda-forge/linux-64::libidn2-2.3.4-h166bdaf_0 \n",
            "  libjpeg-turbo      conda-forge/linux-64::libjpeg-turbo-2.1.4-h166bdaf_0 \n",
            "  libpciaccess       conda-forge/linux-64::libpciaccess-0.17-h166bdaf_0 \n",
            "  libpng             conda-forge/linux-64::libpng-1.6.39-h753d276_0 \n",
            "  libprotobuf        conda-forge/linux-64::libprotobuf-3.21.12-h3eb15da_0 \n",
            "  libtasn1           conda-forge/linux-64::libtasn1-4.19.0-h166bdaf_0 \n",
            "  libtiff            conda-forge/linux-64::libtiff-4.5.0-h6adf6a1_2 \n",
            "  libunistring       conda-forge/linux-64::libunistring-0.9.10-h7f98852_0 \n",
            "  libva              conda-forge/linux-64::libva-2.17.0-h0b41bf4_0 \n",
            "  libvpx             conda-forge/linux-64::libvpx-1.11.0-h9c3ff4c_3 \n",
            "  libwebp-base       conda-forge/linux-64::libwebp-base-1.2.4-h166bdaf_0 \n",
            "  libxcb             conda-forge/linux-64::libxcb-1.13-h7f98852_1004 \n",
            "  llvm-openmp        conda-forge/linux-64::llvm-openmp-15.0.7-h0cdce71_0 \n",
            "  magma              conda-forge/linux-64::magma-2.6.2-hc72dce7_0 \n",
            "  mkl                conda-forge/linux-64::mkl-2022.2.1-h84fe81f_16997 \n",
            "  nccl               conda-forge/linux-64::nccl-2.14.3.1-h0800d71_0 \n",
            "  nettle             conda-forge/linux-64::nettle-3.8.1-hc379101_1 \n",
            "  ninja              conda-forge/linux-64::ninja-1.11.0-h924138e_0 \n",
            "  openh264           conda-forge/linux-64::openh264-2.3.1-h27087fc_1 \n",
            "  openjpeg           conda-forge/linux-64::openjpeg-2.5.0-hfec8fc6_2 \n",
            "  p11-kit            conda-forge/linux-64::p11-kit-0.24.1-hc5aa10d_0 \n",
            "  pillow             conda-forge/linux-64::pillow-9.4.0-py38hb32c036_0 \n",
            "  pthread-stubs      conda-forge/linux-64::pthread-stubs-0.4-h36c2ea0_1001 \n",
            "  pytorch            conda-forge/linux-64::pytorch-1.13.1-cuda112py38hd94e077_200 \n",
            "  pytorch-mutex      pytorch/noarch::pytorch-mutex-1.0-cpu \n",
            "  sleef              conda-forge/linux-64::sleef-3.5.1-h9b69904_2 \n",
            "  svt-av1            conda-forge/linux-64::svt-av1-1.4.1-hcb278e6_0 \n",
            "  tbb                conda-forge/linux-64::tbb-2021.7.0-h924138e_1 \n",
            "  torchvision        pytorch/linux-64::torchvision-0.14.1-py38_cpu \n",
            "  typing_extensions  conda-forge/noarch::typing_extensions-4.4.0-pyha770c72_0 \n",
            "  x264               conda-forge/linux-64::x264-1!164.3095-h166bdaf_2 \n",
            "  x265               conda-forge/linux-64::x265-3.5-h924138e_3 \n",
            "  xorg-fixesproto    conda-forge/linux-64::xorg-fixesproto-5.0-h7f98852_1002 \n",
            "  xorg-kbproto       conda-forge/linux-64::xorg-kbproto-1.0.7-h7f98852_1002 \n",
            "  xorg-libx11        conda-forge/linux-64::xorg-libx11-1.7.2-h7f98852_0 \n",
            "  xorg-libxau        conda-forge/linux-64::xorg-libxau-1.0.9-h7f98852_0 \n",
            "  xorg-libxdmcp      conda-forge/linux-64::xorg-libxdmcp-1.1.3-h7f98852_0 \n",
            "  xorg-libxext       conda-forge/linux-64::xorg-libxext-1.3.4-h7f98852_1 \n",
            "  xorg-libxfixes     conda-forge/linux-64::xorg-libxfixes-5.0.3-h7f98852_1004 \n",
            "  xorg-xextproto     conda-forge/linux-64::xorg-xextproto-7.3.0-h7f98852_1002 \n",
            "  xorg-xproto        conda-forge/linux-64::xorg-xproto-7.0.31-h7f98852_1007 \n",
            "\n",
            "The following packages will be DOWNGRADED:\n",
            "\n",
            "  _openmp_mutex                                   4.5-2_gnu --> 4.5-2_kmp_llvm \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "libpciaccess-0.17    | 39 KB     | :   0% 0/1 [00:00<?, ?it/s]\n",
            "libxcb-1.13          | 391 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "aom-3.5.0            | 2.7 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.7.2    | 941 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.7.0         | 1.5 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "svt-av1-1.4.1        | 2.4 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "gettext-0.21.1       | 4.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "llvm-openmp-15.0.7   | 3.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-kbproto-1.0.7   | 27 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "lerc-4.0.0           | 275 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "p11-kit-0.24.1       | 4.5 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libpciaccess-0.17    | 39 KB     | :  41% 0.41217610062893084/1 [00:00<00:00,  2.62it/s]\n",
            "\n",
            "\n",
            "\n",
            "libpciaccess-0.17    | 39 KB     | : 100% 1.0/1 [00:00<00:00,  2.62it/s]                \n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   0% 9.935703915635201e-05/1 [00:00<29:42, 1782.64s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.7.0         | 1.5 MB    | :   1% 0.010517220793593123/1 [00:00<00:16, 16.67s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "libxcb-1.13          | 391 KB    | :   4% 0.040970754823141074/1 [00:00<00:04,  4.61s/it]\u001b[A\n",
            "\n",
            "aom-3.5.0            | 2.7 MB    | :   1% 0.005730426522249472/1 [00:00<00:34, 34.46s/it]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | : 100% 1.0/1 [00:00<00:00,  4.37it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   2% 0.01947397967464499/1 [00:00<00:11, 11.53s/it]    \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "svt-av1-1.4.1        | 2.4 MB    | :   1% 0.006584708798246436/1 [00:00<00:42, 42.85s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   4% 0.04262416979807501/1 [00:00<00:06,  6.99s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "gettext-0.21.1       | 4.1 MB    | :   0% 0.003792041342138226/1 [00:00<01:42, 103.20s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   6% 0.06398593321669069/1 [00:00<00:05,  6.02s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   0% 6.748383089018237e-05/1 [00:00<1:57:40, 7060.53s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "gettext-0.21.1       | 4.1 MB    | :  86% 0.8645854260075155/1 [00:00<00:00,  2.30it/s]   \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   0% 0.0006183991486443362/1 [00:00<14:22, 863.20s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | :   2% 0.018254176935383958/1 [00:00<00:29, 30.11s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :   8% 0.08296312769555393/1 [00:00<00:05,  5.75s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   0% 0.0024969017429367474/1 [00:00<02:54, 175.10s/it]   \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :   1% 0.014286486609034739/1 [00:00<00:41, 41.96s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   0% 0.00010762293992949148/1 [00:00<1:31:43, 5503.86s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "libxcb-1.13          | 391 KB    | : 100% 1.0/1 [00:00<00:00,  1.71it/s]                 \u001b[A\n",
            "libxcb-1.13          | 391 KB    | : 100% 1.0/1 [00:00<00:00,  1.71it/s]\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   2% 0.018551974459330083/1 [00:00<00:26, 26.71s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  10% 0.10273517848766797/1 [00:00<00:04,  5.51s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   2% 0.023999915604276598/1 [00:00<00:20, 21.48s/it]      \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :  23% 0.22858378574455582/1 [00:00<00:01,  2.35s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   0% 0.003981546022520759/1 [00:00<02:25, 145.59s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.7.0         | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.43it/s]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tbb-2021.7.0         | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.43it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   4% 0.039577545513237514/1 [00:00<00:12, 13.23s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  12% 0.1217123729665312/1 [00:00<00:04,  5.47s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "_openmp_mutex-4.5    | 6 KB      | : 100% 1.0/1 [00:00<00:00,  4.37it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   4% 0.04444827419087998/1 [00:00<00:11, 12.37s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :  43% 0.4285945982710421/1 [00:00<00:00,  1.32s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.7.2    | 941 KB    | : 100% 1.0/1 [00:00<00:00,  1.27it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "xorg-libx11-1.7.2    | 941 KB    | : 100% 1.0/1 [00:00<00:00,  1.27it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   1% 0.006478447765457507/1 [00:00<01:29, 90.50s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   6% 0.05689272167527892/1 [00:00<00:09, 10.16s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  14% 0.1421799230327397/1 [00:00<00:04,  5.30s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   6% 0.06263655103896404/1 [00:00<00:08,  9.59s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :  63% 0.6286054107975285/1 [00:00<00:00,  1.04it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   1% 0.009042833339284437/1 [00:00<01:07, 68.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :   8% 0.07544469613460901/1 [00:00<00:07,  8.43s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  16% 0.16403847164713717/1 [00:00<00:04,  5.07s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :   8% 0.08157818846655454/1 [00:00<00:07,  7.96s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | :  86% 0.8571891965420843/1 [00:01<00:00,  1.35it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "svt-av1-1.4.1        | 2.4 MB    | : 100% 1.0/1 [00:01<00:00,  1.04it/s]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   1% 0.01255199254557392/1 [00:01<00:49, 50.39s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "svt-av1-1.4.1        | 2.4 MB    | : 100% 1.0/1 [00:01<00:00,  1.04it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  11% 0.10512785526953715/1 [00:01<00:05,  6.07s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  18% 0.18410859355672027/1 [00:01<00:04,  5.12s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "llvm-openmp-15.0.7   | 3.1 MB    | :   1% 0.005012289041185573/1 [00:01<03:35, 216.13s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  10% 0.09955121943477961/1 [00:01<00:06,  7.38s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   2% 0.01673599006076523/1 [00:01<00:38, 39.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  15% 0.15459978716108402/1 [00:01<00:03,  3.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "llvm-openmp-15.0.7   | 3.1 MB    | :  76% 0.7568556452190215/1 [00:01<00:00,  1.14s/it]   \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  12% 0.11687851276342774/1 [00:01<00:06,  7.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   2% 0.020650052252395804/1 [00:01<00:34, 34.78s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  18% 0.1849013454446565/1 [00:01<00:03,  3.79s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  20% 0.20388064434883432/1 [00:01<00:05,  6.73s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   3% 0.028275725142986412/1 [00:01<00:23, 24.04s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  25% 0.25416205009282217/1 [00:01<00:01,  2.60s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  13% 0.13280670787299248/1 [00:01<00:06,  7.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  22% 0.22057262692710145/1 [00:01<00:05,  6.83s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   4% 0.03758849380583158/1 [00:01<00:17, 17.87s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  34% 0.33640913686251883/1 [00:01<00:01,  2.02s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  15% 0.1476586735832623/1 [00:01<00:06,  7.61s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  24% 0.23646975319211777/1 [00:01<00:05,  7.34s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   5% 0.047711068439358933/1 [00:01<00:13, 14.53s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  41% 0.41309063129441653/1 [00:01<00:01,  1.75s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  16% 0.16197252459388467/1 [00:01<00:06,  7.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  25% 0.2509758809089452/1 [00:01<00:05,  7.44s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   6% 0.057901126903776474/1 [00:01<00:12, 12.78s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :   0% 0.004880274801411181/1 [00:01<05:37, 339.04s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  50% 0.5021401086992009/1 [00:01<00:00,  1.51s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  18% 0.183066620820065/1 [00:01<00:05,  6.53s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  27% 0.26637622197817973/1 [00:01<00:05,  7.22s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   7% 0.06761879855196273/1 [00:01<00:11, 11.96s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  58% 0.5775848048338099/1 [00:01<00:00,  1.45s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :   3% 0.029281648808467085/1 [00:01<00:44, 46.25s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  20% 0.19921006180948872/1 [00:01<00:05,  6.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   8% 0.0799683396048661/1 [00:01<00:09, 10.57s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  28% 0.28078299265585077/1 [00:01<00:05,  7.32s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  66% 0.6579766941575736/1 [00:01<00:00,  1.42s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  22% 0.22051940391552802/1 [00:01<00:04,  6.16s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :   6% 0.05856329761693417/1 [00:01<00:20, 21.34s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  30% 0.2953884774118345/1 [00:01<00:05,  7.20s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :   9% 0.08961852742216218/1 [00:01<00:09, 10.79s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  73% 0.7297109954003166/1 [00:01<00:00,  1.42s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  24% 0.23752382842438768/1 [00:02<00:04,  6.44s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :   9% 0.08784494642540126/1 [00:02<00:12, 13.43s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  31% 0.30959653401119286/1 [00:02<00:04,  7.22s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  10% 0.09940368290123863/1 [00:02<00:09, 10.63s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  83% 0.8261812625888331/1 [00:02<00:00,  1.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "aom-3.5.0            | 2.7 MB    | : 100% 1.0/1 [00:02<00:00,  2.13s/it]                 \u001b[A\u001b[A\n",
            "\n",
            "aom-3.5.0            | 2.7 MB    | : 100% 1.0/1 [00:02<00:00,  2.13s/it]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  14% 0.13664769443951308/1 [00:02<00:06,  7.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  11% 0.10891890305675435/1 [00:02<00:10, 11.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  25% 0.2535596464738819/1 [00:02<00:05,  7.37s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | :  90% 0.9047179544666638/1 [00:02<00:00,  1.44s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  32% 0.32370523357139486/1 [00:02<00:05,  8.58s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  20% 0.19521099205644724/1 [00:02<00:03,  4.90s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  12% 0.11802922022692897/1 [00:02<00:09, 11.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  27% 0.26948784158344663/1 [00:02<00:05,  7.06s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  34% 0.33612486346593884/1 [00:02<00:05,  8.48s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-kbproto-1.0.7   | 27 KB     | :  60% 0.5993123125320068/1 [00:02<00:01,  3.91s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  25% 0.2537742896733814/1 [00:02<00:02,  3.65s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "lerc-4.0.0           | 275 KB    | :   6% 0.05814093783490302/1 [00:02<00:38, 41.13s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  28% 0.2843398072937165/1 [00:02<00:05,  7.26s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  13% 0.12700456973532323/1 [00:02<00:10, 12.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "p11-kit-0.24.1       | 4.5 MB    | :   0% 0.0034841064226090947/1 [00:02<11:33, 696.36s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  35% 0.3484451363213265/1 [00:02<00:05,  8.53s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  33% 0.3318586864959603/1 [00:02<00:01,  2.65s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  30% 0.29951464182377474/1 [00:02<00:04,  7.09s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  14% 0.1355075324274862/1 [00:02<00:10, 12.28s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "p11-kit-0.24.1       | 4.5 MB    | :  45% 0.4459656220939641/1 [00:02<00:02,  4.05s/it]    \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  36% 0.36056669509840145/1 [00:02<00:05,  8.51s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | : 100% 1.0/1 [00:02<00:00,  2.43s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x264-1!164.3095      | 877 KB    | : 100% 1.0/1 [00:02<00:00,  2.43s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  46% 0.458745831332651/1 [00:02<00:00,  1.75s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  31% 0.3140437387142561/1 [00:02<00:04,  7.18s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  14% 0.14380804362697863/1 [00:02<00:10, 12.25s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "p11-kit-0.24.1       | 4.5 MB    | :  97% 0.9685815854853284/1 [00:02<00:00,  1.63s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  37% 0.37258889683632/1 [00:02<00:05,  8.51s/it]   \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  33% 0.33158627792276324/1 [00:02<00:04,  6.75s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  15% 0.15318829612071397/1 [00:02<00:10, 11.88s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  39% 0.3876911667880855/1 [00:02<00:04,  7.88s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  35% 0.3507431612302127/1 [00:02<00:04,  6.29s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  40% 0.40458186344466535/1 [00:02<00:04,  7.35s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  16% 0.16169125881287696/1 [00:02<00:12, 14.85s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  37% 0.3712991427567456/1 [00:02<00:03,  5.80s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | :  53% 0.5319499533538188/1 [00:02<00:01,  2.55s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  42% 0.4198828474747436/1 [00:02<00:04,  7.10s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  17% 0.16945189936524793/1 [00:03<00:11, 14.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  39% 0.3887340590253232/1 [00:03<00:03,  6.26s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  44% 0.4357799737397599/1 [00:03<00:03,  7.06s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  18% 0.176875120763168/1 [00:03<00:11, 14.39s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  41% 0.4050927458946059/1 [00:03<00:04,  7.06s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  45% 0.4500873873782746/1 [00:03<00:04,  7.92s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libvpx-1.11.0        | 1.1 MB    | : 100% 1.0/1 [00:03<00:00,  1.35it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  19% 0.18618788942601316/1 [00:03<00:10, 13.24s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  46% 0.4631031595077567/1 [00:03<00:04,  8.03s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  42% 0.41983708866494623/1 [00:03<00:04,  7.41s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  19% 0.1940834976401645/1 [00:03<00:10, 13.19s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  48% 0.47582086051976974/1 [00:03<00:04,  8.29s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  20% 0.20346375013389983/1 [00:03<00:09, 12.40s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  43% 0.4337204479158506/1 [00:03<00:04,  7.66s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  49% 0.4896314889625027/1 [00:03<00:04,  8.05s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  21% 0.2116967775025021/1 [00:03<00:10, 13.06s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  45% 0.4470656924671076/1 [00:03<00:04,  8.53s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  50% 0.5028459751702975/1 [00:03<00:03,  8.02s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  22% 0.22100954616534726/1 [00:03<00:09, 12.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  46% 0.4595499534989286/1 [00:03<00:04,  8.40s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  52% 0.5180476021612194/1 [00:03<00:03,  7.56s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  23% 0.22944502502662006/1 [00:03<00:09, 12.19s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "llvm-openmp-15.0.7   | 3.1 MB    | : 100% 1.0/1 [00:03<00:00,  1.14s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  47% 0.47343331274983297/1 [00:03<00:04,  8.11s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  53% 0.5316595165256396/1 [00:03<00:03,  7.50s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  24% 0.23983753498370813/1 [00:03<00:08, 11.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  49% 0.4883929014000323/1 [00:03<00:03,  7.68s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  55% 0.5451720738509035/1 [00:03<00:03,  8.18s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  25% 0.2488128844921024/1 [00:03<00:09, 12.13s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  50% 0.5017381459512893/1 [00:03<00:03,  7.81s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  56% 0.5576910607846038/1 [00:04<00:03,  8.42s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  26% 0.2572483633533752/1 [00:04<00:09, 12.45s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  52% 0.5154062593223346/1 [00:04<00:03,  7.69s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  57% 0.5698126195616787/1 [00:04<00:03,  8.56s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  27% 0.2662911966926596/1 [00:04<00:09, 12.58s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  53% 0.5286438809336621/1 [00:04<00:03,  8.03s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  27% 0.2743217725685913/1 [00:04<00:09, 12.96s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  58% 0.5816361072212847/1 [00:04<00:04,  9.84s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  54% 0.5412357649054126/1 [00:04<00:04,  8.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  28% 0.2832296382460954/1 [00:04<00:08, 12.47s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  59% 0.5926647385676397/1 [00:04<00:03,  9.67s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  55% 0.5527514194778682/1 [00:04<00:04,  9.10s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  29% 0.2913276979529173/1 [00:04<00:09, 12.94s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  61% 0.6056805106971218/1 [00:04<00:03,  9.06s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  57% 0.5658814181492662/1 [00:04<00:03,  8.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  30% 0.3027999492042483/1 [00:04<00:07, 11.42s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  62% 0.6186962828266039/1 [00:04<00:03,  8.73s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  58% 0.5787961709408052/1 [00:04<00:03,  8.39s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  31% 0.31238265319065417/1 [00:04<00:07, 11.18s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  64% 0.6355869794831838/1 [00:04<00:02,  7.71s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  59% 0.593217644891357/1 [00:04<00:03,  7.94s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  32% 0.32284264697863246/1 [00:04<00:07, 10.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  65% 0.6489008227301349/1 [00:04<00:02,  7.84s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  61% 0.6073162500221204/1 [00:04<00:03,  7.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  33% 0.33235786713414817/1 [00:04<00:07, 11.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  62% 0.6214148551528837/1 [00:04<00:02,  7.50s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  66% 0.6619165948596171/1 [00:04<00:02,  8.11s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  34% 0.3429528285839068/1 [00:05<00:07, 10.73s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  64% 0.6350829685239292/1 [00:05<00:02,  7.45s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  68% 0.6769195077722262/1 [00:05<00:02,  7.65s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  35% 0.35240056490853233/1 [00:05<00:07, 10.86s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  65% 0.648643458955045/1 [00:05<00:02,  7.46s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  69% 0.6902333510191774/1 [00:05<00:02,  7.72s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  66% 0.663280178785456/1 [00:05<00:02,  7.27s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  36% 0.36171333357137747/1 [00:05<00:06, 10.85s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  71% 0.705633692088412/1 [00:05<00:02,  7.33s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  37% 0.37102610223422267/1 [00:05<00:06, 11.03s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  68% 0.6771635380363603/1 [00:05<00:02,  7.48s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  72% 0.7208353190793338/1 [00:05<00:01,  7.11s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  74% 0.7350433756786922/1 [00:05<00:02,  7.65s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  38% 0.3801364194043973/1 [00:05<00:07, 12.23s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  69% 0.6938450937254316/1 [00:05<00:02,  7.68s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  71% 0.7069750923968294/1 [00:05<00:02,  8.02s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  75% 0.7483572189256433/1 [00:05<00:02,  8.33s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  39% 0.3885044144347799/1 [00:05<00:08, 13.40s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  72% 0.71956697636858/1 [00:05<00:02,  8.29s/it]  \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  76% 0.760677491781031/1 [00:05<00:02,  8.39s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  40% 0.3961975711562607/1 [00:05<00:08, 13.70s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  40% 0.404093179370412/1 [00:05<00:07, 13.41s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  73% 0.7317283685806125/1 [00:05<00:02,  8.76s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  77% 0.7727990505581059/1 [00:05<00:02,  9.40s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  41% 0.41171885226100263/1 [00:05<00:07, 13.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  75% 0.745396481951658/1 [00:05<00:02,  8.66s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  78% 0.78412575302193/1 [00:06<00:02,  9.29s/it]  \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  42% 0.42150400774007907/1 [00:06<00:07, 12.38s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  76% 0.757019759464043/1 [00:06<00:02,  8.71s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  80% 0.7951543843682851/1 [00:06<00:01,  9.52s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  43% 0.42973703510868133/1 [00:06<00:07, 12.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  77% 0.7712259875347359/1 [00:06<00:01,  8.23s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  81% 0.8074746572236727/1 [00:06<00:01,  9.19s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  44% 0.4386449007861854/1 [00:06<00:06, 12.44s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  78% 0.7834950026866979/1 [00:06<00:01,  8.48s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  82% 0.8217820708621875/1 [00:06<00:01,  8.46s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  45% 0.44789018561814037/1 [00:06<00:06, 11.94s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  80% 0.7970554931178139/1 [00:06<00:01,  8.14s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  83% 0.833804272600106/1 [00:06<00:01,  9.31s/it] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  82% 0.820947785782161/1 [00:06<00:01,  6.57s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  46% 0.45639314831030336/1 [00:06<00:07, 13.66s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  85% 0.8454290461813992/1 [00:06<00:01,  9.15s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  84% 0.8365531120719372/1 [00:06<00:01,  6.63s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  47% 0.46590836846581907/1 [00:06<00:06, 12.76s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  86% 0.8585441753500377/1 [00:06<00:01,  8.71s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  85% 0.8517279466019955/1 [00:06<00:01,  7.02s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  47% 0.47474875031243297/1 [00:06<00:06, 12.37s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  88% 0.8756335860849302/1 [00:06<00:00,  7.92s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  48% 0.4830492615119254/1 [00:06<00:06, 13.16s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  87% 0.8661494205525474/1 [00:06<00:01,  8.07s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  89% 0.8884506441360996/1 [00:06<00:00,  8.54s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  49% 0.491417256542308/1 [00:06<00:06, 12.98s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  88% 0.8843376974006314/1 [00:07<00:00,  7.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  90% 0.9009696310698/1 [00:07<00:00,  8.52s/it]   \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  50% 0.49931286475645936/1 [00:07<00:06, 13.01s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  90% 0.8998354007504782/1 [00:07<00:00,  7.08s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  92% 0.9151776876691583/1 [00:07<00:00,  8.06s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  51% 0.5078158274486223/1 [00:07<00:06, 12.72s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  91% 0.914687366460748/1 [00:07<00:00,  6.98s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  93% 0.9287896020335785/1 [00:07<00:00,  7.90s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  52% 0.5176684667585889/1 [00:07<00:05, 11.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  93% 0.9312612992098898/1 [00:07<00:00,  6.71s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  94% 0.9416066600847479/1 [00:07<00:00,  8.06s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  53% 0.5268462677596537/1 [00:07<00:05, 11.62s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  95% 0.9541256470184483/1 [00:07<00:00,  8.17s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  95% 0.9464361337399481/1 [00:07<00:00,  7.30s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  54% 0.5355516819444873/1 [00:07<00:05, 11.73s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  97% 0.9674394902653994/1 [00:07<00:00,  8.03s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  96% 0.9612880994502179/1 [00:07<00:00,  7.17s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  55% 0.5452693535926736/1 [00:07<00:05, 11.35s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  98% 0.9799584771990998/1 [00:07<00:00,  8.07s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  98% 0.9764629339802762/1 [00:07<00:00,  7.01s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  55% 0.5541772192701776/1 [00:07<00:05, 11.62s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | :  99% 0.991099653810687/1 [00:07<00:00,  7.09s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | :  99% 0.9924774641328002/1 [00:07<00:00,  8.52s/it]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  56% 0.5628151496241209/1 [00:07<00:05, 12.54s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  57% 0.5726003051031974/1 [00:07<00:05, 11.80s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  59% 0.5858946197885633/1 [00:08<00:04, 10.29s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  60% 0.5983116446723569/1 [00:08<00:03,  9.61s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  61% 0.6122133138357344/1 [00:08<00:03,  8.83s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  62% 0.624292919565077/1 [00:08<00:03,  8.91s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  64% 0.63644000912531/1 [00:08<00:03,  8.73s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  65% 0.6496668399797857/1 [00:08<00:02,  8.36s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  66% 0.6625562516798105/1 [00:08<00:02,  8.17s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  67% 0.6748383089018237/1 [00:08<00:02,  8.79s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "gettext-0.21.1       | 4.1 MB    | : 100% 1.0/1 [00:08<00:00,  2.30it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-kbproto-1.0.7   | 27 KB     | : 100% 1.0/1 [00:08<00:00,  9.98s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  69% 0.6863780439840449/1 [00:08<00:02,  9.33s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-kbproto-1.0.7   | 27 KB     | : 100% 1.0/1 [00:08<00:00,  9.98s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  70% 0.6983901658824974/1 [00:09<00:02,  9.23s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  71% 0.7093900303175971/1 [00:09<00:02, 10.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "lerc-4.0.0           | 275 KB    | : 100% 1.0/1 [00:09<00:00,  8.59s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "lerc-4.0.0           | 275 KB    | : 100% 1.0/1 [00:09<00:00,  8.59s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  72% 0.7216720875396102/1 [00:09<00:02,  9.48s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  73% 0.7329418872982707/1 [00:09<00:02,  9.33s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  75% 0.7469110402925384/1 [00:09<00:02,  8.60s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  76% 0.7587207106983204/1 [00:09<00:02,  8.64s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  77% 0.7732972181705997/1 [00:09<00:01,  8.21s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  79% 0.7871314035030872/1 [00:09<00:01,  7.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  80% 0.8000882990340021/1 [00:09<00:01,  7.88s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  81% 0.8128427430722466/1 [00:10<00:01,  8.95s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  83% 0.8266769284047341/1 [00:10<00:01,  8.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  84% 0.8397687915974293/1 [00:10<00:01,  8.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  85% 0.8534005254372462/1 [00:10<00:01,  8.06s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  87% 0.8660200018137103/1 [00:10<00:01,  8.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  88% 0.8780321237121628/1 [00:10<00:01,  8.72s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  89% 0.8896393426252741/1 [00:10<00:00,  8.78s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  90% 0.9021238513399579/1 [00:10<00:00,  8.65s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  92% 0.917307713290249/1 [00:10<00:00,  7.97s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  93% 0.9299946734976032/1 [00:10<00:00,  8.34s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  94% 0.942141763057836/1 [00:11<00:00,  9.33s/it] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  95% 0.9534790466473867/1 [00:11<00:00,  9.19s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  96% 0.9646138787442667/1 [00:11<00:00,  9.59s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  98% 0.9775707742751818/1 [00:11<00:00,  9.02s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | :  99% 0.9889080578647325/1 [00:11<00:00,  9.57s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "p11-kit-0.24.1       | 4.5 MB    | : 100% 1.0/1 [00:11<00:00,  1.63s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | : 100% 0.9995705031453812/1 [00:11<00:00, 10.00s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | : 100% 1.0/1 [00:13<00:00, 16.80s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "x265-3.5             | 3.2 MB    | : 100% 1.0/1 [00:13<00:00, 16.80s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | : 100% 1.0/1 [00:14<00:00, 45.20s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "torchvision-0.14.1   | 25.3 MB   | : 100% 1.0/1 [00:14<00:00, 45.20s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "mkl-2022.2.1         | 157.3 MB  | : 100% 1.0/1 [00:26<00:00,  8.52s/it]               \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "nccl-2.14.3.1        | 145.2 MB  | : 100% 1.0/1 [00:44<00:00,  7.09s/it]              \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magma-2.6.2          | 231.5 MB  | : 100% 1.0/1 [01:08<00:00, 10.00s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "Preparing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Verifying transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Executing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ By downloading and using the cuDNN conda packages, you accept the terms and conditions of the NVIDIA cuDNN EULA -\n",
            "  https://docs.nvidia.com/deeplearning/cudnn/sla/index.html\n",
            "\n",
            "\b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Solving environment: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - cudatoolkit\n",
            "    - torchani=2.2.0\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    lark-parser-0.12.0         |     pyhd8ed1ab_0          78 KB  conda-forge\n",
            "    torchani-2.2               |     pyh9f0ad1d_0        10.1 MB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        10.2 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  lark-parser        conda-forge/noarch::lark-parser-0.12.0-pyhd8ed1ab_0 \n",
            "  torchani           conda-forge/noarch::torchani-2.2-pyh9f0ad1d_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "torchani-2.2         | 10.1 MB   | :   0% 0/1 [00:00<?, ?it/s]\n",
            "lark-parser-0.12.0   | 78 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "lark-parser-0.12.0   | 78 KB     | :  21% 0.20642300084413703/1 [00:00<00:01,  2.35s/it]\u001b[A\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "Preparing transaction: \\ \b\bdone\n",
            "Verifying transaction: / \b\bdone\n",
            "Executing transaction: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Solving environment: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - scikit-learn\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    appdirs-1.4.4              |     pyh9f0ad1d_0          13 KB  conda-forge\n",
            "    joblib-1.2.0               |     pyhd8ed1ab_0         205 KB  conda-forge\n",
            "    packaging-23.0             |     pyhd8ed1ab_0          40 KB  conda-forge\n",
            "    pooch-1.6.0                |     pyhd8ed1ab_0          44 KB  conda-forge\n",
            "    scikit-learn-1.2.0         |   py38h1e1a916_0         7.2 MB  conda-forge\n",
            "    scipy-1.10.0               |   py38h10c12cc_0        23.5 MB  conda-forge\n",
            "    threadpoolctl-3.1.0        |     pyh8a188c0_0          18 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        31.0 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  appdirs            conda-forge/noarch::appdirs-1.4.4-pyh9f0ad1d_0 \n",
            "  joblib             conda-forge/noarch::joblib-1.2.0-pyhd8ed1ab_0 \n",
            "  packaging          conda-forge/noarch::packaging-23.0-pyhd8ed1ab_0 \n",
            "  pooch              conda-forge/noarch::pooch-1.6.0-pyhd8ed1ab_0 \n",
            "  scikit-learn       conda-forge/linux-64::scikit-learn-1.2.0-py38h1e1a916_0 \n",
            "  scipy              conda-forge/linux-64::scipy-1.10.0-py38h10c12cc_0 \n",
            "  threadpoolctl      conda-forge/noarch::threadpoolctl-3.1.0-pyh8a188c0_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "appdirs-1.4.4        | 13 KB     | :   0% 0/1 [00:00<?, ?it/s]\n",
            "pooch-1.6.0          | 44 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "joblib-1.2.0         | 205 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "scikit-learn-1.2.0   | 7.2 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "threadpoolctl-3.1.0  | 18 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "packaging-23.0       | 40 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "joblib-1.2.0         | 205 KB    | :   8% 0.07801087505118512/1 [00:00<00:03,  3.37s/it]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "appdirs-1.4.4        | 13 KB     | : 100% 1.0/1 [00:00<00:00,  3.69it/s]\n",
            "\n",
            "\n",
            "\n",
            "threadpoolctl-3.1.0  | 18 KB     | :  90% 0.8973109151651241/1 [00:00<00:00,  3.34it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "pooch-1.6.0          | 44 KB     | :  37% 0.36539619527643347/1 [00:00<00:00,  1.29it/s]\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "packaging-23.0       | 40 KB     | :  40% 0.4028918506860768/1 [00:00<00:00,  1.37it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "appdirs-1.4.4        | 13 KB     | : 100% 1.0/1 [00:00<00:00,  3.69it/s]\n",
            "\n",
            "\n",
            "\n",
            "threadpoolctl-3.1.0  | 18 KB     | : 100% 1.0/1 [00:00<00:00,  3.34it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "scikit-learn-1.2.0   | 7.2 MB    | :  44% 0.44041002768954507/1 [00:00<00:00,  1.49it/s]   \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | :  13% 0.1318874802734825/1 [00:00<00:02,  2.35s/it]    \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "pooch-1.6.0          | 44 KB     | : 100% 1.0/1 [00:00<00:00,  2.51it/s]                \u001b[A\n",
            "pooch-1.6.0          | 44 KB     | : 100% 1.0/1 [00:00<00:00,  2.51it/s]\u001b[A\n",
            "\n",
            "\n",
            "scikit-learn-1.2.0   | 7.2 MB    | :  88% 0.8808200553790901/1 [00:00<00:00,  2.44it/s] \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "packaging-23.0       | 40 KB     | : 100% 1.0/1 [00:00<00:00,  2.16it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "packaging-23.0       | 40 KB     | : 100% 1.0/1 [00:00<00:00,  2.16it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | :  26% 0.2631088621617454/1 [00:00<00:01,  1.42s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | :  48% 0.47759254220245934/1 [00:00<00:00,  1.15it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "joblib-1.2.0         | 205 KB    | : 100% 1.0/1 [00:00<00:00,  1.79it/s]                \u001b[A\u001b[A\n",
            "\n",
            "joblib-1.2.0         | 205 KB    | : 100% 1.0/1 [00:00<00:00,  1.79it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | :  71% 0.7100608786441027/1 [00:00<00:00,  1.51it/s] \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "scipy-1.10.0         | 23.5 MB   | : 100% 0.9991475778294129/1 [00:00<00:00,  1.93it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "scikit-learn-1.2.0   | 7.2 MB    | : 100% 1.0/1 [00:01<00:00,  2.44it/s]               \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "Preparing transaction: - \b\b\\ \b\bdone\n",
            "Verifying transaction: / \b\b- \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install Torch and TorchAni\n",
        "!conda install pytorch torchvision -c pytorch\n",
        "!conda install -c conda-forge torchani=2.2.0 cudatoolkit\n",
        "!conda install scikit-learn"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "u3P301beutJd",
        "outputId": "cb6e740d-178b-482e-8d67-f5b21e609400"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - ase\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ase-3.22.1                 |     pyhd8ed1ab_1         1.6 MB  conda-forge\n",
            "    brotli-1.0.9               |       h166bdaf_8          18 KB  conda-forge\n",
            "    brotli-bin-1.0.9           |       h166bdaf_8          20 KB  conda-forge\n",
            "    click-8.1.3                |unix_pyhd8ed1ab_2          74 KB  conda-forge\n",
            "    contourpy-1.0.7            |   py38hfbd4bf9_0         209 KB  conda-forge\n",
            "    cycler-0.11.0              |     pyhd8ed1ab_0          10 KB  conda-forge\n",
            "    flask-2.2.2                |     pyhd8ed1ab_0          74 KB  conda-forge\n",
            "    fonttools-4.38.0           |   py38h0a891b7_1         1.9 MB  conda-forge\n",
            "    importlib-metadata-6.0.0   |     pyha770c72_0          24 KB  conda-forge\n",
            "    itsdangerous-2.1.2         |     pyhd8ed1ab_0          16 KB  conda-forge\n",
            "    jinja2-3.1.2               |     pyhd8ed1ab_1          99 KB  conda-forge\n",
            "    kiwisolver-1.4.4           |   py38h43d8883_1          76 KB  conda-forge\n",
            "    libbrotlicommon-1.0.9      |       h166bdaf_8          66 KB  conda-forge\n",
            "    libbrotlidec-1.0.9         |       h166bdaf_8          33 KB  conda-forge\n",
            "    libbrotlienc-1.0.9         |       h166bdaf_8         288 KB  conda-forge\n",
            "    markupsafe-2.1.2           |   py38h1de0b5d_0          23 KB  conda-forge\n",
            "    matplotlib-base-3.6.3      |   py38hd6c3c57_0         6.3 MB  conda-forge\n",
            "    munkres-1.1.4              |     pyh9f0ad1d_0          12 KB  conda-forge\n",
            "    pyparsing-3.0.9            |     pyhd8ed1ab_0          79 KB  conda-forge\n",
            "    python-dateutil-2.8.2      |     pyhd8ed1ab_0         240 KB  conda-forge\n",
            "    six-1.16.0                 |     pyh6c4a22f_0          14 KB  conda-forge\n",
            "    unicodedata2-15.0.0        |   py38h0a891b7_0         497 KB  conda-forge\n",
            "    werkzeug-2.2.2             |     pyhd8ed1ab_0         249 KB  conda-forge\n",
            "    zipp-3.11.0                |     pyhd8ed1ab_0          15 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        11.8 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  ase                conda-forge/noarch::ase-3.22.1-pyhd8ed1ab_1 \n",
            "  brotli             conda-forge/linux-64::brotli-1.0.9-h166bdaf_8 \n",
            "  brotli-bin         conda-forge/linux-64::brotli-bin-1.0.9-h166bdaf_8 \n",
            "  click              conda-forge/noarch::click-8.1.3-unix_pyhd8ed1ab_2 \n",
            "  contourpy          conda-forge/linux-64::contourpy-1.0.7-py38hfbd4bf9_0 \n",
            "  cycler             conda-forge/noarch::cycler-0.11.0-pyhd8ed1ab_0 \n",
            "  flask              conda-forge/noarch::flask-2.2.2-pyhd8ed1ab_0 \n",
            "  fonttools          conda-forge/linux-64::fonttools-4.38.0-py38h0a891b7_1 \n",
            "  importlib-metadata conda-forge/noarch::importlib-metadata-6.0.0-pyha770c72_0 \n",
            "  itsdangerous       conda-forge/noarch::itsdangerous-2.1.2-pyhd8ed1ab_0 \n",
            "  jinja2             conda-forge/noarch::jinja2-3.1.2-pyhd8ed1ab_1 \n",
            "  kiwisolver         conda-forge/linux-64::kiwisolver-1.4.4-py38h43d8883_1 \n",
            "  libbrotlicommon    conda-forge/linux-64::libbrotlicommon-1.0.9-h166bdaf_8 \n",
            "  libbrotlidec       conda-forge/linux-64::libbrotlidec-1.0.9-h166bdaf_8 \n",
            "  libbrotlienc       conda-forge/linux-64::libbrotlienc-1.0.9-h166bdaf_8 \n",
            "  markupsafe         conda-forge/linux-64::markupsafe-2.1.2-py38h1de0b5d_0 \n",
            "  matplotlib-base    conda-forge/linux-64::matplotlib-base-3.6.3-py38hd6c3c57_0 \n",
            "  munkres            conda-forge/noarch::munkres-1.1.4-pyh9f0ad1d_0 \n",
            "  pyparsing          conda-forge/noarch::pyparsing-3.0.9-pyhd8ed1ab_0 \n",
            "  python-dateutil    conda-forge/noarch::python-dateutil-2.8.2-pyhd8ed1ab_0 \n",
            "  six                conda-forge/noarch::six-1.16.0-pyh6c4a22f_0 \n",
            "  unicodedata2       conda-forge/linux-64::unicodedata2-15.0.0-py38h0a891b7_0 \n",
            "  werkzeug           conda-forge/noarch::werkzeug-2.2.2-pyhd8ed1ab_0 \n",
            "  zipp               conda-forge/noarch::zipp-3.11.0-pyhd8ed1ab_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "brotli-bin-1.0.9     | 20 KB     | :   0% 0/1 [00:00<?, ?it/s]\n",
            "jinja2-3.1.2         | 99 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "pyparsing-3.0.9      | 79 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libbrotlienc-1.0.9   | 288 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "fonttools-4.38.0     | 1.9 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ase-3.22.1           | 1.6 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlicommon-1.0. | 66 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "cycler-0.11.0        | 10 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "six-1.16.0           | 14 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "python-dateutil-2.8. | 240 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "unicodedata2-15.0.0  | 497 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "markupsafe-2.1.2     | 23 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "kiwisolver-1.4.4     | 76 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "click-8.1.3          | 74 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "zipp-3.11.0          | 15 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlidec-1.0.9   | 33 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "contourpy-1.0.7      | 209 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "flask-2.2.2          | 74 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "brotli-bin-1.0.9     | 20 KB     | :  82% 0.8153677714740719/1 [00:00<00:00,  2.43it/s]\n",
            "\n",
            "\n",
            "\n",
            "fonttools-4.38.0     | 1.9 MB    | :   1% 0.0080534247208287/1 [00:00<00:39, 39.59s/it]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | :   0% 0.002499127883076691/1 [00:00<02:06, 126.97s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libbrotlienc-1.0.9   | 288 KB    | :   6% 0.05549364756249979/1 [00:00<00:05,  5.95s/it]\u001b[A\u001b[A\u001b[A\n",
            "jinja2-3.1.2         | 99 KB     | :  16% 0.16150941908263755/1 [00:00<00:01,  2.23s/it]\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ase-3.22.1           | 1.6 MB    | :   1% 0.009950448177077164/1 [00:00<00:33, 33.91s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "brotli-bin-1.0.9     | 20 KB     | : 100% 1.0/1 [00:00<00:00,  2.43it/s]               \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "cycler-0.11.0        | 10 KB     | : 100% 1.0/1 [00:00<00:00,  2.74it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "pyparsing-3.0.9      | 79 KB     | : 100% 1.0/1 [00:00<00:00,  3.02it/s]                \u001b[A\u001b[A\n",
            "\n",
            "pyparsing-3.0.9      | 79 KB     | : 100% 1.0/1 [00:00<00:00,  3.02it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "six-1.16.0           | 14 KB     | : 100% 1.0/1 [00:00<00:00,  2.56it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "python-dateutil-2.8. | 240 KB    | :   7% 0.06660514580038783/1 [00:00<00:05,  6.01s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "fonttools-4.38.0     | 1.9 MB    | :  79% 0.7892356226412126/1 [00:00<00:00,  2.42it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | :  31% 0.3098918575015097/1 [00:00<00:00,  1.04s/it]   \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "unicodedata2-15.0.0  | 497 KB    | :   3% 0.03218025973617844/1 [00:00<00:12, 12.93s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "markupsafe-2.1.2     | 23 KB     | :  70% 0.7008298400205322/1 [00:00<00:00,  1.57it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "kiwisolver-1.4.4     | 76 KB     | :  21% 0.21174524400330852/1 [00:00<00:01,  2.14s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "click-8.1.3          | 74 KB     | :  22% 0.2154372123602893/1 [00:00<00:01,  2.14s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "zipp-3.11.0          | 15 KB     | : 100% 1.0/1 [00:00<00:00,  2.11it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlidec-1.0.9   | 33 KB     | :  48% 0.47856057950695174/1 [00:00<00:00,  1.01it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "contourpy-1.0.7      | 209 KB    | :   8% 0.07654965869430129/1 [00:00<00:05,  6.32s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "flask-2.2.2          | 74 KB     | :  22% 0.21674824712263527/1 [00:00<00:01,  2.24s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "jinja2-3.1.2         | 99 KB     | : 100% 1.0/1 [00:00<00:00,  2.19it/s]                \u001b[A\n",
            "jinja2-3.1.2         | 99 KB     | : 100% 1.0/1 [00:00<00:00,  2.19it/s]\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | :  70% 0.6972566793783969/1 [00:00<00:00,  1.84it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libbrotlienc-1.0.9   | 288 KB    | : 100% 1.0/1 [00:00<00:00,  2.15it/s]                \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "libbrotlienc-1.0.9   | 288 KB    | : 100% 1.0/1 [00:00<00:00,  2.15it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlicommon-1.0. | 66 KB     | : 100% 1.0/1 [00:00<00:00,  1.99it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlicommon-1.0. | 66 KB     | : 100% 1.0/1 [00:00<00:00,  1.99it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "cycler-0.11.0        | 10 KB     | : 100% 1.0/1 [00:00<00:00,  2.74it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "six-1.16.0           | 14 KB     | : 100% 1.0/1 [00:00<00:00,  2.56it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "python-dateutil-2.8. | 240 KB    | : 100% 1.0/1 [00:00<00:00,  1.90it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "python-dateutil-2.8. | 240 KB    | : 100% 1.0/1 [00:00<00:00,  1.90it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "unicodedata2-15.0.0  | 497 KB    | : 100% 1.0/1 [00:00<00:00,  1.57it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "unicodedata2-15.0.0  | 497 KB    | : 100% 1.0/1 [00:00<00:00,  1.57it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ase-3.22.1           | 1.6 MB    | : 100% 1.0/1 [00:01<00:00,  1.39s/it]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "ase-3.22.1           | 1.6 MB    | : 100% 1.0/1 [00:01<00:00,  1.39s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "markupsafe-2.1.2     | 23 KB     | : 100% 1.0/1 [00:01<00:00,  1.73s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "markupsafe-2.1.2     | 23 KB     | : 100% 1.0/1 [00:01<00:00,  1.73s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "kiwisolver-1.4.4     | 76 KB     | : 100% 1.0/1 [00:01<00:00,  1.49s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "kiwisolver-1.4.4     | 76 KB     | : 100% 1.0/1 [00:01<00:00,  1.49s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "click-8.1.3          | 74 KB     | : 100% 1.0/1 [00:01<00:00,  1.52s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "click-8.1.3          | 74 KB     | : 100% 1.0/1 [00:01<00:00,  1.52s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "zipp-3.11.0          | 15 KB     | : 100% 1.0/1 [00:01<00:00,  2.11it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "fonttools-4.38.0     | 1.9 MB    | : 100% 1.0/1 [00:01<00:00,  2.42it/s]               \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlidec-1.0.9   | 33 KB     | : 100% 1.0/1 [00:01<00:00,  1.70s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libbrotlidec-1.0.9   | 33 KB     | : 100% 1.0/1 [00:01<00:00,  1.70s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "contourpy-1.0.7      | 209 KB    | : 100% 1.0/1 [00:01<00:00,  1.52s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "contourpy-1.0.7      | 209 KB    | : 100% 1.0/1 [00:01<00:00,  1.52s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "flask-2.2.2          | 74 KB     | : 100% 1.0/1 [00:01<00:00,  1.60s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "flask-2.2.2          | 74 KB     | : 100% 1.0/1 [00:01<00:00,  1.60s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " ... (more hidden) ...\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | : 100% 1.0/1 [00:02<00:00,  2.58s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "matplotlib-base-3.6. | 6.3 MB    | : 100% 1.0/1 [00:02<00:00,  2.58s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\u001b[A\n",
            "\n",
            "\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "Preparing transaction: \\ \b\b| \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\b| \b\bdone\n",
            "Executing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Solving environment: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - joblib\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    ca-certificates-2022.07.19 |       h06a4308_0         131 KB  anaconda\n",
            "    certifi-2022.6.15          |   py38h06a4308_0         156 KB  anaconda\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:         287 KB\n",
            "\n",
            "The following packages will be SUPERSEDED by a higher-priority channel:\n",
            "\n",
            "  ca-certificates    conda-forge::ca-certificates-2022.12.~ --> anaconda::ca-certificates-2022.07.19-h06a4308_0 \n",
            "  certifi            conda-forge/noarch::certifi-2022.12.7~ --> anaconda/linux-64::certifi-2022.6.15-py38h06a4308_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "ca-certificates-2022 | 131 KB    | :   0% 0/1 [00:00<?, ?it/s]\n",
            "certifi-2022.6.15    | 156 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "ca-certificates-2022 | 131 KB    | :  12% 0.12236088394984279/1 [00:00<00:00,  1.07it/s]\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "Preparing transaction: - \b\bdone\n",
            "Verifying transaction: | \b\bdone\n",
            "Executing transaction: - \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install ASE and Joblib\n",
        "!conda install -c conda-forge ase\n",
        "!conda install -c anaconda joblib"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "f4ZunoI4uUcF",
        "outputId": "8c7c0b7b-91f3-42dd-f3e9-982b5776334c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'pKa-ANI'...\n",
            "remote: Enumerating objects: 203, done.\u001b[K\n",
            "remote: Counting objects: 100% (151/151), done.\u001b[K\n",
            "remote: Compressing objects: 100% (107/107), done.\u001b[K\n",
            "remote: Total 203 (delta 94), reused 80 (delta 44), pack-reused 52\u001b[K\n",
            "Receiving objects: 100% (203/203), 39.70 MiB | 12.17 MiB/s, done.\n",
            "Resolving deltas: 100% (106/106), done.\n"
          ]
        }
      ],
      "source": [
        "#@title Get pKa-ANI\n",
        "!git clone https://github.com/isayevlab/pKa-ANI.git"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Lba7CzcVvJVt",
        "outputId": "e991394c-5e44-42fc-ded3-0ef72cb6cc74"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "running install\n",
            "running bdist_egg\n",
            "running egg_info\n",
            "creating pkaani.egg-info\n",
            "writing pkaani.egg-info/PKG-INFO\n",
            "writing dependency_links to pkaani.egg-info/dependency_links.txt\n",
            "writing entry points to pkaani.egg-info/entry_points.txt\n",
            "writing top-level names to pkaani.egg-info/top_level.txt\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "reading manifest template 'MANIFEST.in'\n",
            "adding license file 'LICENSE'\n",
            "writing manifest file 'pkaani.egg-info/SOURCES.txt'\n",
            "installing library code to build/bdist.linux-x86_64/egg\n",
            "running install_lib\n",
            "running build_py\n",
            "creating build\n",
            "creating build/lib\n",
            "creating build/lib/pkaani\n",
            "copying pkaani/ani_descriptors.py -> build/lib/pkaani\n",
            "copying pkaani/run.py -> build/lib/pkaani\n",
            "copying pkaani/__main__.py -> build/lib/pkaani\n",
            "copying pkaani/__init__.py -> build/lib/pkaani\n",
            "copying pkaani/pkaani.py -> build/lib/pkaani\n",
            "copying pkaani/ase_io_proteindatabank_mod.py -> build/lib/pkaani\n",
            "copying pkaani/prep_pdb.py -> build/lib/pkaani\n",
            "creating build/lib/pkaani/models\n",
            "copying pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FASP.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FGLU.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FHIS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FLYS.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/FTYR.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "copying pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/lib/pkaani/models\n",
            "creating build/bdist.linux-x86_64\n",
            "creating build/bdist.linux-x86_64/egg\n",
            "creating build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/ani_descriptors.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/run.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__main__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/__init__.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "creating build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/GLU_ani2x_FINAL_MODEL_F75.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FASP.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/HIS_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/ASP_ani2x_FINAL_MODEL_F100.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FGLU.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/LYS_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FHIS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/TYR_ani2x_FINAL_MODEL_F25.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FLYS.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/models/FTYR.joblib -> build/bdist.linux-x86_64/egg/pkaani/models\n",
            "copying build/lib/pkaani/pkaani.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/ase_io_proteindatabank_mod.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "copying build/lib/pkaani/prep_pdb.py -> build/bdist.linux-x86_64/egg/pkaani\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ani_descriptors.py to ani_descriptors.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/run.py to run.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__main__.py to __main__.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/__init__.py to __init__.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/pkaani.py to pkaani.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/ase_io_proteindatabank_mod.py to ase_io_proteindatabank_mod.cpython-38.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pkaani/prep_pdb.py to prep_pdb.cpython-38.pyc\n",
            "creating build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pkaani.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "creating dist\n",
            "creating 'dist/pkaani-0.1.0-py3.8.egg' and adding 'build/bdist.linux-x86_64/egg' to it\n",
            "removing 'build/bdist.linux-x86_64/egg' (and everything under it)\n",
            "Processing pkaani-0.1.0-py3.8.egg\n",
            "creating /usr/local/lib/python3.8/site-packages/pkaani-0.1.0-py3.8.egg\n",
            "Extracting pkaani-0.1.0-py3.8.egg to /usr/local/lib/python3.8/site-packages\n",
            "Adding pkaani 0.1.0 to easy-install.pth file\n",
            "Installing pkaani script to /usr/local/bin\n",
            "\n",
            "Installed /usr/local/lib/python3.8/site-packages/pkaani-0.1.0-py3.8.egg\n",
            "Processing dependencies for pkaani==0.1.0\n",
            "Finished processing dependencies for pkaani==0.1.0\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'pkaani.models' as data is deprecated, please list it in `packages`.\n",
            "    !!\n",
            "\n",
            "\n",
            "    ############################\n",
            "    # Package would be ignored #\n",
            "    ############################\n",
            "    Python recognizes 'pkaani.models' as an importable package,\n",
            "    but it is not listed in the `packages` configuration of setuptools.\n",
            "\n",
            "    'pkaani.models' has been automatically added to the distribution only\n",
            "    because it may contain data files, but this behavior is likely to change\n",
            "    in future versions of setuptools (and therefore is considered deprecated).\n",
            "\n",
            "    Please make sure that 'pkaani.models' is included as a package by using\n",
            "    the `packages` configuration field or the proper discovery methods\n",
            "    (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
            "    instead of `find_packages(...)`/`find:`).\n",
            "\n",
            "    You can read more about \"package discovery\" and \"data files\" on setuptools\n",
            "    documentation page.\n",
            "\n",
            "\n",
            "!!\n",
            "\n",
            "  check.warn(importable)\n",
            "zip_safe flag not set; analyzing archive contents...\n",
            "pkaani.__pycache__.pkaani.cpython-38: module references __file__\n"
          ]
        }
      ],
      "source": [
        "#@title Install pKa-ANI\n",
        "%%bash\n",
        "cd pKa-ANI\n",
        "python setup.py install"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Z500vZchxNUV",
        "outputId": "1bff744c-e0d5-4902-f402-5b391ee19b74"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - modeller\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    hdf5-1107-1.10.7           |                0         3.1 MB  salilab\n",
            "    modeller-10.4              |   py38h9bf148f_0        16.8 MB  salilab\n",
            "    zlib-1.2.13                |       h166bdaf_4          92 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        19.9 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  hdf5-1107          salilab/linux-64::hdf5-1107-1.10.7-0 \n",
            "  modeller           salilab/linux-64::modeller-10.4-py38h9bf148f_0 \n",
            "  zlib               conda-forge/linux-64::zlib-1.2.13-h166bdaf_4 \n",
            "\n",
            "The following packages will be UPDATED:\n",
            "\n",
            "  ca-certificates    anaconda::ca-certificates-2022.07.19-~ --> conda-forge::ca-certificates-2022.12.7-ha878542_0 \n",
            "  certifi            anaconda/linux-64::certifi-2022.6.15-~ --> conda-forge/noarch::certifi-2022.12.7-pyhd8ed1ab_0 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "hdf5-1107-1.10.7     | 3.1 MB    | :   0% 0/1 [00:00<?, ?it/s]\n",
            "modeller-10.4        | 16.8 MB   | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "zlib-1.2.13          | 92 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "zlib-1.2.13          | 92 KB     | :  17% 0.1741144964346061/1 [00:00<00:00,  1.33it/s]\u001b[A\u001b[A\n",
            "\n",
            "hdf5-1107-1.10.7     | 3.1 MB    | : 100% 1.0/1 [00:03<00:00,  3.79s/it]\n",
            "modeller-10.4        | 16.8 MB   | : 100% 1.0/1 [06:53<00:00, 413.31s/it]\u001b[A\n",
            "                                                                        \n",
            "                                                                         \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "Preparing transaction: | \b\bdone\n",
            "Verifying transaction: - \b\b\\ \b\b| \b\bdone\n",
            "Executing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n"
          ]
        }
      ],
      "source": [
        "#@title Install Modeller\n",
        "#@markdown You will need to obtain modeller Licence Key from \n",
        "#@markdown https://salilab.org/modeller/. \n",
        "modellerkey = 'MODELIRANJE' #@param {type:\"string\"}\n",
        "os.environ['KEY_MODELLER']=modellerkey\n",
        "!conda config --add channels salilab\n",
        "!conda install modeller\n",
        "from modeller import *\n",
        "from modeller.automodel import *"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YN2XoO6K0xcI",
        "outputId": "b11df2c1-c6d2-41ee-b723-d88d7433c3d1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting MDAnalysis\n",
            "  Downloading MDAnalysis-2.4.2-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (8.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m8.1/8.1 MB\u001b[0m \u001b[31m64.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: scipy>=1.5.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.10.0)\n",
            "Collecting GridDataFormats>=0.4.0\n",
            "  Downloading GridDataFormats-1.0.1-py3-none-any.whl (2.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.1/2.1 MB\u001b[0m \u001b[31m84.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: joblib>=0.12 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.2.0)\n",
            "Requirement already satisfied: numpy>=1.20.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.24.1)\n",
            "Collecting fasteners\n",
            "  Downloading fasteners-0.18-py3-none-any.whl (18 kB)\n",
            "Collecting mmtf-python>=1.0.0\n",
            "  Downloading mmtf_python-1.1.3-py2.py3-none-any.whl (25 kB)\n",
            "Requirement already satisfied: matplotlib>=1.5.1 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (3.6.3)\n",
            "Requirement already satisfied: tqdm>=4.43.0 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (4.64.1)\n",
            "Collecting networkx>=2.0\n",
            "  Downloading networkx-3.0-py3-none-any.whl (2.0 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.0/2.0 MB\u001b[0m \u001b[31m65.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: threadpoolctl in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (3.1.0)\n",
            "Requirement already satisfied: biopython>=1.80 in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (1.80)\n",
            "Collecting gsd>=1.9.3\n",
            "  Downloading gsd-2.7.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (406 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m407.0/407.0 kB\u001b[0m \u001b[31m29.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: packaging in /usr/local/lib/python3.8/site-packages (from MDAnalysis) (23.0)\n",
            "Collecting mrcfile\n",
            "  Downloading mrcfile-1.4.3-py2.py3-none-any.whl (43 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m43.9/43.9 kB\u001b[0m \u001b[31m4.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (9.4.0)\n",
            "Requirement already satisfied: contourpy>=1.0.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (1.0.7)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (2.8.2)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (0.11.0)\n",
            "Requirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (4.38.0)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (1.4.4)\n",
            "Requirement already satisfied: pyparsing>=2.2.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.5.1->MDAnalysis) (3.0.9)\n",
            "Collecting msgpack>=1.0.0\n",
            "  Downloading msgpack-1.0.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (322 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m322.5/322.5 kB\u001b[0m \u001b[31m29.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/site-packages (from python-dateutil>=2.7->matplotlib>=1.5.1->MDAnalysis) (1.16.0)\n",
            "Installing collected packages: msgpack, networkx, mrcfile, mmtf-python, gsd, fasteners, GridDataFormats, MDAnalysis\n",
            "Successfully installed GridDataFormats-1.0.1 MDAnalysis-2.4.2 fasteners-0.18 gsd-2.7.0 mmtf-python-1.1.3 mrcfile-1.4.3 msgpack-1.0.4 networkx-3.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#@title Install MDAnalysis\n",
        "!pip install --upgrade MDAnalysis\n",
        "import MDAnalysis as mda\n",
        "from MDAnalysis.analysis import rms, align"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "K8CDGsuuvPyP"
      },
      "source": [
        "# Set up parameters"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Gromacs setup\n",
        "#@markdown We expect GROMACS to be preinstalled with one of our scripts and\n",
        "#@markdown stored on Google Drive\n",
        "from google.colab import drive\n",
        "drive.mount('/content/gdrive')\n",
        "\n",
        "#@markdown Select GROMACS version from dropdown menu\n",
        "gmx_version = '2022' #@param [\"2021\",\"2022\"]\n",
        "#@markdown - Currently versions 2021(2021.5) and 2022(2022.3) are availablle\n",
        "gdrive_path = str('/content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-'+gmx_version+'.zip')\n",
        "local_path = str('/content/gromacs-'+gmx_version)\n",
        "os.environ['GMX_LOCAL_PATH']=local_path\n",
        "os.environ['GMX_ZIP_PATH']=gdrive_path\n",
        "\n",
        "##@title Extract GROMACS binaries\n",
        "!unzip $GMX_ZIP_PATH"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4jNQ29kq5s7P",
        "outputId": "6fd9ee42-5bd3-45ec-c8fa-cc05bd53db6e"
      },
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/gdrive\n",
            "Archive:  /content/gdrive/MyDrive/GROMACS/GROMACS/gromacs-2022.zip\n",
            "   creating: gromacs-2022/\n",
            "   creating: gromacs-2022/lib/\n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0.3.1  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2.3.2  \n",
            "  inflating: gromacs-2022/lib/libgmxapi.so.0  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0.1.0  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7  \n",
            "  inflating: gromacs-2022/lib/libgromacs.so.7.0.0  \n",
            "   creating: gromacs-2022/lib/pkgconfig/\n",
            "  inflating: gromacs-2022/lib/pkgconfig/muparser.pc  \n",
            "  inflating: gromacs-2022/lib/pkgconfig/libgromacs.pc  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so.2  \n",
            "  inflating: gromacs-2022/lib/libmuparser.so  \n",
            "  inflating: gromacs-2022/lib/libnblib_gmx.so.0  \n",
            "   creating: gromacs-2022/include/\n",
            "  inflating: gromacs-2022/include/muParserToken.h  \n",
            "  inflating: gromacs-2022/include/muParserTemplateMagic.h  \n",
            "   creating: gromacs-2022/include/nblib/\n",
            "  inflating: gromacs-2022/include/nblib/nbnxmsetuphelpers.h  \n",
            "  inflating: gromacs-2022/include/nblib/integrator.h  \n",
            "   creating: gromacs-2022/include/nblib/listed_forces/\n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/definitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/calculator.h  \n",
            "  inflating: gromacs-2022/include/nblib/listed_forces/bondtypes.h  \n",
            "  inflating: gromacs-2022/include/nblib/topology.h  \n",
            "   creating: gromacs-2022/include/nblib/util/\n",
            "  inflating: gromacs-2022/include/nblib/util/traits.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/util/setup.h  \n",
            "  inflating: gromacs-2022/include/nblib/util/util.hpp  \n",
            "  inflating: gromacs-2022/include/nblib/gmxcalculatorcpu.h  \n",
            "  inflating: gromacs-2022/include/nblib/particletype.h  \n",
            "  inflating: gromacs-2022/include/nblib/interactions.h  \n",
            "  inflating: gromacs-2022/include/nblib/tpr.h  \n",
            "  inflating: gromacs-2022/include/nblib/exception.h  \n",
            "  inflating: gromacs-2022/include/nblib/basicdefinitions.h  \n",
            "  inflating: gromacs-2022/include/nblib/nblib.h  \n",
            "  inflating: gromacs-2022/include/nblib/kerneloptions.h  \n",
            "  inflating: gromacs-2022/include/nblib/particlesequencer.h  \n",
            "  inflating: gromacs-2022/include/nblib/box.h  \n",
            "  inflating: gromacs-2022/include/nblib/simulationstate.h  \n",
            "  inflating: gromacs-2022/include/nblib/molecules.h  \n",
            "  inflating: gromacs-2022/include/nblib/vector.h  \n",
            "  inflating: gromacs-2022/include/muParserInt.h  \n",
            "   creating: gromacs-2022/include/gmxapi/\n",
            "  inflating: gromacs-2022/include/gmxapi/status.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/session.h  \n",
            "   creating: gromacs-2022/include/gmxapi/md/\n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdmodule.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md/mdsignals.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/version.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gromacsfwd.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapicompat.h  \n",
            "   creating: gromacs-2022/include/gmxapi/session/\n",
            "  inflating: gromacs-2022/include/gmxapi/session/resources.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/exceptions.h  \n",
            "   creating: gromacs-2022/include/gmxapi/compat/\n",
            "  inflating: gromacs-2022/include/gmxapi/compat/tpr.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/compat/mdparams.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/system.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/context.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/gmxapi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/md.h  \n",
            "   creating: gromacs-2022/include/gmxapi/mpi/\n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/gmxapi_mpi.h  \n",
            "  inflating: gromacs-2022/include/gmxapi/mpi/resourceassignment.h  \n",
            "  inflating: gromacs-2022/include/muParserTokenReader.h  \n",
            "  inflating: gromacs-2022/include/muParserBytecode.h  \n",
            "   creating: gromacs-2022/include/gromacs/\n",
            "   creating: gromacs-2022/include/gromacs/mdrun/\n",
            "  inflating: gromacs-2022/include/gromacs/mdrun/simulationinputhandle.h  \n",
            "  inflating: gromacs-2022/include/muParserFixes.h  \n",
            "  inflating: gromacs-2022/include/muParserError.h  \n",
            "  inflating: gromacs-2022/include/muParserDef.h  \n",
            "  inflating: gromacs-2022/include/muParserBase.h  \n",
            "  inflating: gromacs-2022/include/muParser.h  \n",
            "  inflating: gromacs-2022/include/muParserDLL.h  \n",
            "  inflating: gromacs-2022/include/muParserTest.h  \n",
            "  inflating: gromacs-2022/include/muParserCallback.h  \n",
            "   creating: gromacs-2022/bin/\n",
            "  inflating: gromacs-2022/bin/GMXRC.csh  \n",
            "  inflating: gromacs-2022/bin/GMXRC  \n",
            "  inflating: gromacs-2022/bin/xplor2gmx.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.zsh  \n",
            "  inflating: gromacs-2022/bin/gmx    \n",
            "  inflating: gromacs-2022/bin/demux.pl  \n",
            "  inflating: gromacs-2022/bin/GMXRC.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion.bash  \n",
            "  inflating: gromacs-2022/bin/gmx-completion-gmx.bash  \n",
            "   creating: gromacs-2022/share/\n",
            "   creating: gromacs-2022/share/cmake/\n",
            "   creating: gromacs-2022/share/cmake/gmxapi/\n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gmxapi/gmxapi-config.cmake  \n",
            "   creating: gromacs-2022/share/cmake/gromacs/\n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-hints.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config-version.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/libgromacs-release.cmake  \n",
            "  inflating: gromacs-2022/share/cmake/gromacs/gromacs-config.cmake  \n",
            "   creating: gromacs-2022/share/gromacs/\n",
            "  inflating: gromacs-2022/share/gromacs/README_FreeEnergyModifications.txt  \n",
            "   creating: gromacs-2022/share/gromacs/top/\n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a1.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a5.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-9.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG43a2.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/oplsaa.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ethanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/1propanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip5pe.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/oplsaa.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residuetypes.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffoplsaa-n.tst  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atommass.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/export.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/edissoc.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/nsfactor.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-12.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/dgsolv.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip5p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/defselection.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/bonds.dlg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb-ildn.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/surface.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/electroneg.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexspce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/elements.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG53a6.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ps.m2p  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sfactor.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/charmm27.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/cmap.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tips3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.arn  \n",
            " extracting: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/lipids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnanonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/rna.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/ffnabonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/charmm27.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/specbond.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/co-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdwradii.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber03.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber03.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber03.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber96.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber96.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber96.ff/tip4pew.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a5.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/atomname2type.n2t  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a5.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ffG45a3.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/phbres.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.xml  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a1.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/methanol216.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a1.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-8.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/atom_nom.tbl  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ha-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ss.map  \n",
            "   creating: gromacs-2022/share/gromacs/top/amberGS.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amberGS.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amberGS.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/spc.itp  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos54a7.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tmcl.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/dppc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/popc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos54a7.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/random.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/residues.dtd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-11.xvg  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos45a3.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos45a3.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber99sb.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber99sb.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber99sb.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.gro  \n",
            "  inflating: gromacs-2022/share/gromacs/top/vdw-msms.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/amber94.ff/\n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip5p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/urea.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/rna.rtp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.arn  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/dna.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/forcefield.itp  \n",
            " extracting: gromacs-2022/share/gromacs/top/amber94.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/amber94.ff/tip4pew.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/ca-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/sw.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/table6-10.xvg  \n",
            "  inflating: gromacs-2022/share/gromacs/top/xlateat.dat  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos53a6.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos53a6.ff/aminoacids.c.tdb  \n",
            "   creating: gromacs-2022/share/gromacs/top/gromos43a2.ff/\n",
            " extracting: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.doc  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/atomtypes.atp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spce.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffnonbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ions.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip3p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.vsd  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/watermodels.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.r2b  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.rtp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ffbonded.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/spc.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/ff_dum.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.n.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/tip4p.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.hdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/forcefield.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/gromos43a2.ff/aminoacids.c.tdb  \n",
            "  inflating: gromacs-2022/share/gromacs/top/flexwat-ferguson.itp  \n",
            "  inflating: gromacs-2022/share/gromacs/top/README  \n",
            "  inflating: gromacs-2022/share/gromacs/top/refi_aa.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/top/cb-shift.dat  \n",
            "  inflating: gromacs-2022/share/gromacs/COPYING  \n",
            "  inflating: gromacs-2022/share/gromacs/README.tutor  \n",
            "   creating: gromacs-2022/share/man/\n",
            "   creating: gromacs-2022/share/man/man1/\n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-lie.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-density.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mk_angndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-grompp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spol.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-spatial.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sasa.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densmap.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dipoles.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmens.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-xpm2ps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pdb2gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-wheel.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-insert-molecules.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-tpr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dielectric.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotacf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rotmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dos.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-energy.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-clustsize.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-report-methods.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-filter.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gyrate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-editconf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-do_dssp.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-velacc.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hbond.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-confrms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-view.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-analyze.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_edi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-awh.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tune_pme.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nonbonded-benchmark.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-distance.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-convert-trj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmtraj.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-principal.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rms.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-tcaf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genrestr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-vanhove.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-densorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pairdist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bundle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-make_ndx.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-potential.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dyecoupl.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-help.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-h2order.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-disre.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdrun.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mindist.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-eneconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-bar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-anaeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-dump.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helix.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-check.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-genion.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sans.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjconv.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-chi.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-msd.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-solvate.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rama.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rmsf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sigeps.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-mdmat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-extract-cluster.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-angle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-sham.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-enemat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-current.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-gangle.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saxs.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-freevolume.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-helixorient.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-x2top.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-select.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-pme_error.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-saltbr.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-hydorder.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trajectory.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-polystat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-nmeig.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-trjcat.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-covar.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-rdf.1  \n",
            "  inflating: gromacs-2022/share/man/man1/gmx-traj.1  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Test GROMACS installation\n",
        "%%bash\n",
        "chmod -R 755 $GMX_LOCAL_PATH\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "gmx grompp -h"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cFUZK5OvLzSF",
        "outputId": "f3f29055-a906-400a-b452-2d0c17730a4c"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "SYNOPSIS\n",
            "\n",
            "gmx grompp [-f [<.mdp>]] [-c [<.gro/.g96/...>]] [-r [<.gro/.g96/...>]]\n",
            "           [-rb [<.gro/.g96/...>]] [-n [<.ndx>]] [-p [<.top>]]\n",
            "           [-t [<.trr/.cpt/...>]] [-e [<.edr>]] [-qmi [<.inp>]]\n",
            "           [-ref [<.trr/.cpt/...>]] [-po [<.mdp>]] [-pp [<.top>]]\n",
            "           [-o [<.tpr>]] [-imd [<.gro>]] [-[no]v] [-time <real>]\n",
            "           [-[no]rmvsbds] [-maxwarn <int>] [-[no]zero] [-[no]renum]\n",
            "\n",
            "DESCRIPTION\n",
            "\n",
            "gmx grompp (the gromacs preprocessor) reads a molecular topology file, checks\n",
            "the validity of the file, expands the topology from a molecular description to\n",
            "an atomic description. The topology file contains information about molecule\n",
            "types and the number of molecules, the preprocessor copies each molecule as\n",
            "needed. There is no limitation on the number of molecule types. Bonds and\n",
            "bond-angles can be converted into constraints, separately for hydrogens and\n",
            "heavy atoms. Then a coordinate file is read and velocities can be generated\n",
            "from a Maxwellian distribution if requested. gmx grompp also reads parameters\n",
            "for gmx mdrun (eg. number of MD steps, time step, cut-off). Eventually a\n",
            "binary file is produced that can serve as the sole input file for the MD\n",
            "program.\n",
            "\n",
            "gmx grompp uses the atom names from the topology file. The atom names in the\n",
            "coordinate file (option -c) are only read to generate warnings when they do\n",
            "not match the atom names in the topology. Note that the atom names are\n",
            "irrelevant for the simulation as only the atom types are used for generating\n",
            "interaction parameters.\n",
            "\n",
            "gmx grompp uses a built-in preprocessor to resolve includes, macros, etc. The\n",
            "preprocessor supports the following keywords:\n",
            "\n",
            "    #ifdef VARIABLE\n",
            "    #ifndef VARIABLE\n",
            "    #else\n",
            "    #endif\n",
            "    #define VARIABLE\n",
            "    #undef VARIABLE\n",
            "    #include \"filename\"\n",
            "    #include <filename>\n",
            "\n",
            "The functioning of these statements in your topology may be modulated by using\n",
            "the following two flags in your .mdp file:\n",
            "\n",
            "    define = -DVARIABLE1 -DVARIABLE2\n",
            "    include = -I/home/john/doe\n",
            "\n",
            "For further information a C-programming textbook may help you out. Specifying\n",
            "the -pp flag will get the pre-processed topology file written out so that you\n",
            "can verify its contents.\n",
            "\n",
            "When using position restraints, a file with restraint coordinates must be\n",
            "supplied with -r (can be the same file as supplied for -c). For free energy\n",
            "calculations, separate reference coordinates for the B topology can be\n",
            "supplied with -rb, otherwise they will be equal to those of the A topology.\n",
            "\n",
            "Starting coordinates can be read from trajectory with -t. The last frame with\n",
            "coordinates and velocities will be read, unless the -time option is used. Only\n",
            "if this information is absent will the coordinates in the -c file be used.\n",
            "Note that these velocities will not be used when gen_vel = yes in your .mdp\n",
            "file. An energy file can be supplied with -e to read Nose-Hoover and/or\n",
            "Parrinello-Rahman coupling variables.\n",
            "\n",
            "gmx grompp can be used to restart simulations (preserving continuity) by\n",
            "supplying just a checkpoint file with -t. However, for simply changing the\n",
            "number of run steps to extend a run, using gmx convert-tpr is more convenient\n",
            "than gmx grompp. You then supply the old checkpoint file directly to gmx mdrun\n",
            "with -cpi. If you wish to change the ensemble or things like output frequency,\n",
            "then supplying the checkpoint file to gmx grompp with -t along with a new .mdp\n",
            "file with -f is the recommended procedure. Actually preserving the ensemble\n",
            "(if possible) still requires passing the checkpoint file to gmx mdrun -cpi.\n",
            "\n",
            "By default, all bonded interactions which have constant energy due to virtual\n",
            "site constructions will be removed. If this constant energy is not zero, this\n",
            "will result in a shift in the total energy. All bonded interactions can be\n",
            "kept by turning off -rmvsbds. Additionally, all constraints for distances\n",
            "which will be constant anyway because of virtual site constructions will be\n",
            "removed. If any constraints remain which involve virtual sites, a fatal error\n",
            "will result.\n",
            "\n",
            "To verify your run input file, please take note of all warnings on the screen,\n",
            "and correct where necessary. Do also look at the contents of the mdout.mdp\n",
            "file; this contains comment lines, as well as the input that gmx grompp has\n",
            "read. If in doubt, you can start gmx grompp with the -debug option which will\n",
            "give you more information in a file called grompp.log (along with real debug\n",
            "info). You can see the contents of the run input file with the gmx dump\n",
            "program. gmx check can be used to compare the contents of two run input files.\n",
            "\n",
            "The -maxwarn option can be used to override warnings printed by gmx grompp\n",
            "that otherwise halt output. In some cases, warnings are harmless, but usually\n",
            "they are not. The user is advised to carefully interpret the output messages\n",
            "before attempting to bypass them with this option.\n",
            "\n",
            "OPTIONS\n",
            "\n",
            "Options to specify input files:\n",
            "\n",
            " -f      [<.mdp>]           (grompp.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -c      [<.gro/.g96/...>]  (conf.gro)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -r      [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -rb     [<.gro/.g96/...>]  (restraint.gro)  (Opt.)\n",
            "           Structure file: gro g96 pdb brk ent esp tpr\n",
            " -n      [<.ndx>]           (index.ndx)      (Opt.)\n",
            "           Index file\n",
            " -p      [<.top>]           (topol.top)\n",
            "           Topology file\n",
            " -t      [<.trr/.cpt/...>]  (traj.trr)       (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            " -e      [<.edr>]           (ener.edr)       (Opt.)\n",
            "           Energy file\n",
            " -qmi    [<.inp>]           (topol-qmmm.inp) (Opt.)\n",
            "           Input file for QM program\n",
            "\n",
            "Options to specify input/output files:\n",
            "\n",
            " -ref    [<.trr/.cpt/...>]  (rotref.trr)     (Opt.)\n",
            "           Full precision trajectory: trr cpt tng\n",
            "\n",
            "Options to specify output files:\n",
            "\n",
            " -po     [<.mdp>]           (mdout.mdp)\n",
            "           grompp input file with MD parameters\n",
            " -pp     [<.top>]           (processed.top)  (Opt.)\n",
            "           Topology file\n",
            " -o      [<.tpr>]           (topol.tpr)\n",
            "           Portable xdr run input file\n",
            " -imd    [<.gro>]           (imdgroup.gro)   (Opt.)\n",
            "           Coordinate file in Gromos-87 format\n",
            "\n",
            "Other options:\n",
            "\n",
            " -[no]v                     (no)\n",
            "           Be loud and noisy\n",
            " -time   <real>             (-1)\n",
            "           Take frame at or first after this time.\n",
            " -[no]rmvsbds               (yes)\n",
            "           Remove constant bonded interactions with virtual sites\n",
            " -maxwarn <int>             (0)\n",
            "           Number of allowed warnings during input processing. Not for normal\n",
            "           use and may generate unstable systems\n",
            " -[no]zero                  (no)\n",
            "           Set parameters for bonded interactions without defaults to zero\n",
            "           instead of generating an error\n",
            " -[no]renum                 (yes)\n",
            "           Renumber atomtypes and minimize number of atomtypes\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content\n",
            "Command line:\n",
            "  gmx grompp -h\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"Once Again Let Me Do This\" (Urban Dance Squad)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title <b><font color='#56b4e9'> PDB Input</font></b>\n",
        "\n",
        "#@markdown Choose between <b><font color='#d55c00'> ONE</font></b> of the possible input sources for the target pdb and <ins>leave the other cells empty or unmarked</ins>\n",
        "#@markdown - AlphaFold2 PDB via Uniprot ID:\n",
        "uniprot_id =''#@param {type:\"string\"}\n",
        "#@markdown - PDB ID (imported from RCSB PDB):\n",
        "pdb_id =''#@param {type:\"string\"}\n",
        "#@markdown - Upload PDB from local drive (in this case you will be prompt to the file upload later)\n",
        "pdb_local =False#@param {type:\"boolean\"}\n",
        "#@markdown - Use PDB from google drive (insert path to pdb file in the google drive)\n",
        "# '/content/gdrive/MyDrive/NESP/data/targetA_relaxed.pdb'\n",
        "pdb_gdrive ='/content/gdrive/MyDrive/NESP/data/targetA_relaxed.pdb'#@param {type:\"string\"}\n",
        "#@markdown - Insert protein sequence to model the target structure using AlphaFold\n",
        "run_alphafold =''#@param {type:\"string\"}\n",
        "\n",
        "#@markdown\n",
        "\n",
        "#@markdown Select target chain (default is A)\n",
        "chain_id='A' #@param {type:'string'}\n",
        "\n",
        "project_dir = \"/content/alphagmx/\"\n",
        "pdb_path = os.path.join(project_dir, 'target.pdb')\n",
        "pdb_preprocessed_path = os.path.join(project_dir, 'target_preprocessed.pdb')\n",
        "\n",
        "if os.path.exists(project_dir):\n",
        "    print(f'Warning! {project_dir} exists, all files in this directory will be deleted')\n",
        "    #os.remove(project_dir)\n",
        "    shutil.rmtree(project_dir) \n",
        "    os.mkdir(project_dir)\n",
        "else:\n",
        "    os.mkdir(project_dir)\n",
        "\n",
        "if (uniprot_id !='') and (len(uniprot_id)==6) : \n",
        "    print('Loading PDB by Uniprot_ID')\n",
        "    subprocess.call(['curl','-s','-f',f'https://alphafold.ebi.ac.uk/files/AF-{uniprot_id}-F1-model_v2.pdb','-o', pdb_path])\n",
        "elif (pdb_id !='') and (len(pdb_id)==4):\n",
        "    print('Loading PDB by PDB_ID')\n",
        "    subprocess.call(['curl','-s','-f',f'https://files.rcsb.org/download/{pdb_id}.pdb','-o', pdb_path])\n",
        "elif pdb_local:\n",
        "    print('Upload PDB file:')\n",
        "    uploaded_pdb = files.upload()\n",
        "    for fn in uploaded_pdb.keys():\n",
        "        shutil.copy(fn, pdb_path)\n",
        "elif pdb_gdrive!='' and os.path.exists(pdb_gdrive):\n",
        "    print('Using PDB file from gdrive:')\n",
        "    shutil.copy(pdb_gdrive, pdb_path)\n",
        "elif run_alphafold!='':\n",
        "    sequence = run_alphafold\n",
        "    print('RUNNING ALPHAFOLD TODO')\n",
        "else:\n",
        "    print(f'ERROR: unrecognized pdb input')\n",
        "\n",
        "\n",
        "#@markdown N.B. This cell will also perform preliminary operations to correcly format the uploaded PDB\n",
        "\n",
        "#@markdown - mark to preprocess pdb (keep only single chain, delete hetatms)\n",
        "flag_preprocess =True#@param {type:\"boolean\"}\n",
        "\n",
        "if flag_preprocess:\n",
        "    print('Preprocessing PDBs...')\n",
        "    #!pdb_selchain -\"$chain_id\" \"$pdb_path\" | pdb_delhetatm | pdb_delres --999:0:1 | pdb_fixinsert | pdb_tidy  > \"$pdb_preprocessed_path\"\n",
        "    !pdb_selchain -\"$chain_id\" \"$pdb_path\" | pdb_delhetatm | pdb_fixinsert | pdb_tidy  > \"$pdb_preprocessed_path\"\n",
        "    print(f'Kept only chain {chain_id}, deleted hetatms, fixed insertion code, sorted residue ids, formatted pdb file...')\n",
        "    parser = PDBParser()\n",
        "    io=PDBIO()\n",
        "    structure = parser.get_structure('X', pdb_preprocessed_path)\n",
        "\n",
        "    #And here we set the residue conformation we want to keep\n",
        "    keepAltID = \"A\"\n",
        "\n",
        "    class KeepOneConfOnly(Select):  # Inherit methods from Select class\n",
        "        def accept_atom(self, atom):\n",
        "            if (not atom.is_disordered()) or atom.get_altloc() == keepAltID:\n",
        "                atom.set_altloc(\" \")  # Eliminate alt location ID before output.\n",
        "                return True\n",
        "            else:  # Alt location was not one to be output.\n",
        "                return False\n",
        "            # end of accept_atom()\n",
        "\n",
        "    #This will keep only conformation for each residue\n",
        "    io.set_structure(structure)\n",
        "\n",
        "    io.save(os.path.join(pdb_preprocessed_path), select=KeepOneConfOnly())\n",
        "    print('Removed alternative side chain conformations...')\n",
        "\n",
        "else:\n",
        "    pdb_preprocessed_path = pdb_path\n",
        "\n",
        "if os.path.exists(pdb_preprocessed_path):\n",
        "  print(f\"Pre-processing PDBs correctly ended\")\n",
        "else:\n",
        "  print(f\"Pre-processing PDB didn't end correcly, please check the inputs\")\n",
        "\n",
        "#@markdown ****"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yN_gGaa_RfB1",
        "outputId": "38da2c9b-d10b-4c0c-f5d5-5258667b41d7"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using PDB file from gdrive:\n",
            "Preprocessing PDBs...\n",
            "Kept only chain A, deleted hetatms, fixed insertion code, sorted residue ids, formatted pdb file...\n",
            "Removed alternative side chain conformations...\n",
            "Pre-processing PDBs correctly ended\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record '# All ' at line 3330\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record '#BEGIN' at line 3331\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'label ' at line 3332\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'weight' at line 3333\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'pose -' at line 3334\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL:Nt' at line 3335\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_2 ' at line 3336\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_3 ' at line 3337\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_4 ' at line 3338\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_5 ' at line 3339\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_6 ' at line 3340\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_7 ' at line 3341\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_8 ' at line 3342\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_9 ' at line 3343\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_10' at line 3344\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_11' at line 3345\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_12' at line 3346\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_13' at line 3347\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_14' at line 3348\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_15' at line 3349\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_16' at line 3350\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_17' at line 3351\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_18' at line 3352\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_19' at line 3353\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_20' at line 3354\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_21' at line 3355\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_22' at line 3356\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_23' at line 3357\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_24' at line 3358\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_25' at line 3359\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_26' at line 3360\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_27' at line 3361\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_28' at line 3362\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_29' at line 3363\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_30' at line 3364\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_31' at line 3365\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_32' at line 3366\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_33' at line 3367\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_34' at line 3368\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_35' at line 3369\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_36' at line 3370\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_37' at line 3371\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_38' at line 3372\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_39' at line 3373\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_40' at line 3374\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_41' at line 3375\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_42' at line 3376\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_43' at line 3377\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_44' at line 3378\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_45' at line 3379\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_46' at line 3380\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'CYS:di' at line 3381\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_48' at line 3382\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_49' at line 3383\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_50' at line 3384\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_51' at line 3385\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'CYS:di' at line 3386\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_53' at line 3387\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_54' at line 3388\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_55' at line 3389\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_56' at line 3390\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_57' at line 3391\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_58' at line 3392\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_59' at line 3393\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_60' at line 3394\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ARG_61' at line 3395\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_62' at line 3396\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_63' at line 3397\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_64' at line 3398\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_65' at line 3399\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_66' at line 3400\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_67' at line 3401\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_68' at line 3402\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_69' at line 3403\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_70' at line 3404\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ARG_71' at line 3405\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_72' at line 3406\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ARG_73' at line 3407\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_74' at line 3408\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_75' at line 3409\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_76' at line 3410\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_77' at line 3411\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_78' at line 3412\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_79' at line 3413\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_80' at line 3414\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_81' at line 3415\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_82' at line 3416\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_83' at line 3417\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_84' at line 3418\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_85' at line 3419\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_86' at line 3420\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_87' at line 3421\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_88' at line 3422\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_89' at line 3423\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_90' at line 3424\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_91' at line 3425\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_92' at line 3426\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_93' at line 3427\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_94' at line 3428\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_95' at line 3429\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_96' at line 3430\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_97' at line 3431\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_98' at line 3432\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_99' at line 3433\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_10' at line 3434\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_10' at line 3435\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_10' at line 3436\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_10' at line 3437\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_10' at line 3438\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_10' at line 3439\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_10' at line 3440\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_10' at line 3441\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_10' at line 3442\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_10' at line 3443\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_11' at line 3444\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_11' at line 3445\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_11' at line 3446\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_11' at line 3447\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_11' at line 3448\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_11' at line 3449\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_11' at line 3450\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_11' at line 3451\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_11' at line 3452\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_11' at line 3453\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_12' at line 3454\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_12' at line 3455\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_12' at line 3456\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_12' at line 3457\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_12' at line 3458\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_12' at line 3459\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_12' at line 3460\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_12' at line 3461\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_12' at line 3462\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_12' at line 3463\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_13' at line 3464\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_13' at line 3465\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_13' at line 3466\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_13' at line 3467\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_13' at line 3468\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_13' at line 3469\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_13' at line 3470\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_13' at line 3471\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_13' at line 3472\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_13' at line 3473\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_14' at line 3474\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_14' at line 3475\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_14' at line 3476\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_14' at line 3477\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_14' at line 3478\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_14' at line 3479\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_14' at line 3480\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_14' at line 3481\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_14' at line 3482\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_14' at line 3483\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_15' at line 3484\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_15' at line 3485\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_15' at line 3486\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_15' at line 3487\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_15' at line 3488\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_15' at line 3489\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_15' at line 3490\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_15' at line 3491\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_15' at line 3492\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_15' at line 3493\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_16' at line 3494\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_16' at line 3495\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_16' at line 3496\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_16' at line 3497\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_16' at line 3498\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_16' at line 3499\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_16' at line 3500\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_16' at line 3501\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_16' at line 3502\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_16' at line 3503\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'THR_17' at line 3504\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_17' at line 3505\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_17' at line 3506\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_17' at line 3507\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_17' at line 3508\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_17' at line 3509\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_17' at line 3510\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'CYS:di' at line 3511\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_17' at line 3512\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_17' at line 3513\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LEU_18' at line 3514\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_18' at line 3515\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_18' at line 3516\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_18' at line 3517\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_18' at line 3518\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_18' at line 3519\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'SER_18' at line 3520\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_18' at line 3521\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'CYS:di' at line 3522\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_18' at line 3523\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_19' at line 3524\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_19' at line 3525\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_19' at line 3526\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_19' at line 3527\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ILE_19' at line 3528\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_19' at line 3529\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_19' at line 3530\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_19' at line 3531\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_19' at line 3532\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_19' at line 3533\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_20' at line 3534\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_20' at line 3535\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PRO_20' at line 3536\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_20' at line 3537\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_20' at line 3538\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TRP_20' at line 3539\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ALA_20' at line 3540\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_20' at line 3541\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLN_20' at line 3542\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_20' at line 3543\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASP_21' at line 3544\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLU_21' at line 3545\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_21' at line 3546\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_21' at line 3547\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_21' at line 3548\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS_21' at line 3549\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'PHE_21' at line 3550\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'ASN_21' at line 3551\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'TYR_21' at line 3552\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VAL_21' at line 3553\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'GLY_22' at line 3554\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'LYS:Ct' at line 3555\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record 'VRT_22' at line 3556\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/Bio/PDB/PDBParser.py:395: PDBConstructionWarning: Ignoring unrecognized record '#END_P' at line 3557\n",
            "  warnings.warn(\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {
        "id": "SlmykXYo6HYG"
      },
      "outputs": [],
      "source": [
        "#@title Fix missing residues\n",
        "#@markdown Select the software to deal with missing residues. Currently, only\n",
        "#@markdown `PDBFixer` is available.\n",
        "missing_residues_soft = 'pdbfixer' #@param [\"pdbfixer\",\"modeller\",\"-\"]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {
        "id": "OVMgDV4OJM_L",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3239a576-b088-40ac-c0d0-77c41e90042f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/coordinates/PDB.py:753: UserWarning: Unit cell dimensions not found. CRYST1 record set to unitary values.\n",
            "  warnings.warn(\"Unit cell dimensions not found. \"\n",
            "/usr/local/lib/python3.8/site-packages/MDAnalysis/coordinates/PDB.py:1129: UserWarning: Found no information for attr: 'formalcharges' Using default value of '0'\n",
            "  warnings.warn(\"Found no information for attr: '{}'\"\n"
          ]
        }
      ],
      "source": [
        "#@title Run pdbfixer\n",
        "\n",
        "def prepare_protein(pdb_file, ignore_missing_residues=False, ignore_terminal_missing_residues=True, pdb_out=None):\n",
        "    \"\"\"Use pdbfixer to prepare the protein from a PDB file. Hetero atoms such as ligands are \n",
        "    removed and non-standard residues replaced. Missing atoms to existing residues are added. \n",
        "    Missing residues are ignored by default, but can be included.\n",
        "    \n",
        "    Parameters\n",
        "    ----------\n",
        "    pdb_file: pathlib.Path or str\n",
        "        PDB file containing the system to prepare.\n",
        "    ignore_missing_residues: bool\n",
        "        If missing residues should be ignored or built.\n",
        "    ignore_terminal_missing_residues: bool\n",
        "        If missing residues at the beginning and the end of a chain should be ignored or built.\n",
        "    pdb_out: pathlib.Path or str\n",
        "        Output PDB file containing the system to simulate.\n",
        "    \n",
        "    Returns\n",
        "    -------\n",
        "    fixer: pdbfixer.pdbfixer.PDBFixer\n",
        "        Prepared protein system.\n",
        "    \"\"\"\n",
        "    fixer = pdbfixer.PDBFixer(str(pdb_file))\n",
        "    fixer.removeHeterogens()  # co-crystallized ligands are unknown to PDBFixer\n",
        "    fixer.findMissingResidues()  # identify missing residues, needed for identification of missing atoms\n",
        "    \n",
        "    # if missing terminal residues shall be ignored, remove them from the dictionary\n",
        "    if ignore_terminal_missing_residues:  \n",
        "        chains = list(fixer.topology.chains())\n",
        "        keys = fixer.missingResidues.keys()\n",
        "        for key in list(keys):\n",
        "            chain = chains[key[0]]\n",
        "            if key[1] == 0 or key[1] == len(list(chain.residues())):\n",
        "                del fixer.missingResidues[key]\n",
        "                \n",
        "    # if all missing residues shall be ignored ignored, clear the dictionary\n",
        "    if ignore_missing_residues:  \n",
        "        fixer.missingResidues = {}\n",
        "        \n",
        "    fixer.findNonstandardResidues()  # find non-standard residue\n",
        "    fixer.replaceNonstandardResidues()  # replace non-standard residues with standard one\n",
        "    fixer.findMissingAtoms()  # find missing heavy atoms\n",
        "    fixer.addMissingAtoms()  # add missing atoms\n",
        "    PDBFile.writeFile(fixer.topology, fixer.positions, open(pdb_out, 'w'))\n",
        "\n",
        "if missing_residues_soft == '-':\n",
        "    pass\n",
        "elif missing_residues_soft == \"pdbfixer\":\n",
        "    prepare_protein(pdb_preprocessed_path, pdb_out=pdb_preprocessed_path)\n",
        "else:\n",
        "    print('Warning! Currently only pdbfixer option is available for missing residues')\n",
        "\n",
        "u = mda.Universe(pdb_preprocessed_path)\n",
        "u.select_atoms(\"not resname HOH\").write(os.path.splitext(pdb_preprocessed_path)[0]+\"_no_water.pdb\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "vKOCRWjtJ70X"
      },
      "source": [
        "# Display structure"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {
        "id": "9T3rn1_OJa2l"
      },
      "outputs": [],
      "source": [
        "def show_pdb(pdb_file, show_sidechains=False, show_mainchains=False):\n",
        "  model_name = \"Parsed structure\"\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "  view.addModel(open(pdb_file,'r').read(),'pdb')\n",
        "\n",
        "  view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  \n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})  \n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "  view.zoomTo()\n",
        "  return view"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 515
        },
        "id": "GoXENrlQKrRr",
        "outputId": "c3173685-6911-4ac0-c0b0-098b3ab4115f"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16745495978711402\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16745495978711402\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16745495978711402 = null;\nvar warn = document.getElementById(\"3dmolwarning_16745495978711402\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16745495978711402 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16745495978711402\"),{backgroundColor:\"white\"});\nviewer_16745495978711402.zoomTo();\n\tviewer_16745495978711402.addModel(\"REMARK   1 CREATED WITH OPENMM 7.7, 2023-01-24\\nATOM      1  N   VAL A   1      33.328  -6.825  50.080  1.00  0.00           N  \\nATOM      2  CA  VAL A   1      33.399  -5.965  48.871  1.00  0.00           C  \\nATOM      3  C   VAL A   1      34.509  -6.547  48.012  1.00  0.00           C  \\nATOM      4  O   VAL A   1      35.515  -6.895  48.622  1.00  0.00           O  \\nATOM      5  CB  VAL A   1      33.709  -4.497  49.220  1.00  0.00           C  \\nATOM      6  CG1 VAL A   1      33.860  -3.669  47.953  1.00  0.00           C  \\nATOM      7  CG2 VAL A   1      32.609  -3.931  50.105  1.00  0.00           C  \\nATOM      8  H   VAL A   1      32.609  -6.489  50.688  1.00  0.00           H  \\nATOM      9  H2  VAL A   1      33.121  -7.765  49.809  1.00  0.00           H  \\nATOM     10  H3  VAL A   1      34.206  -6.803  50.558  1.00  0.00           H  \\nATOM     11  HA  VAL A   1      32.459  -6.016  48.320  1.00  0.00           H  \\nATOM     12  HB  VAL A   1      34.661  -4.452  49.749  1.00  0.00           H  \\nATOM     13 HG11 VAL A   1      34.079  -2.634  48.218  1.00  0.00           H  \\nATOM     14 HG12 VAL A   1      34.676  -4.069  47.352  1.00  0.00           H  \\nATOM     15 HG13 VAL A   1      32.933  -3.707  47.380  1.00  0.00           H  \\nATOM     16 HG21 VAL A   1      32.835  -2.894  50.349  1.00  0.00           H  \\nATOM     17 HG22 VAL A   1      31.655  -3.982  49.579  1.00  0.00           H  \\nATOM     18 HG23 VAL A   1      32.546  -4.514  51.025  1.00  0.00           H  \\nATOM     19  N   PRO A   2      34.329  -6.775  46.695  1.00  0.00           N  \\nATOM     20  CA  PRO A   2      33.142  -6.395  45.911  1.00  0.00           C  \\nATOM     21  C   PRO A   2      32.058  -7.511  45.886  1.00  0.00           C  \\nATOM     22  O   PRO A   2      32.207  -8.488  46.604  1.00  0.00           O  \\nATOM     23  CB  PRO A   2      33.832  -6.172  44.568  1.00  0.00           C  \\nATOM     24  CG  PRO A   2      34.837  -7.330  44.496  1.00  0.00           C  \\nATOM     25  CD  PRO A   2      35.385  -7.389  45.922  1.00  0.00           C  \\nATOM     26  HA  PRO A   2      32.674  -5.473  46.256  1.00  0.00           H  \\nATOM     27  HB3 PRO A   2      33.124  -6.124  43.741  1.00  0.00           H  \\nATOM     28  HB2 PRO A   2      34.389  -5.234  44.583  1.00  0.00           H  \\nATOM     29  HG3 PRO A   2      35.625  -7.120  43.772  1.00  0.00           H  \\nATOM     30  HG2 PRO A   2      34.334  -8.266  44.255  1.00  0.00           H  \\nATOM     31  HD3 PRO A   2      35.554  -8.426  46.214  1.00  0.00           H  \\nATOM     32  HD2 PRO A   2      36.334  -6.853  45.964  1.00  0.00           H  \\nATOM     33  N   VAL A   3      30.929  -7.482  45.173  1.00  0.00           N  \\nATOM     34  CA  VAL A   3      30.301  -6.579  44.194  1.00  0.00           C  \\nATOM     35  C   VAL A   3      28.793  -6.693  44.482  1.00  0.00           C  \\nATOM     36  O   VAL A   3      28.309  -7.780  44.797  1.00  0.00           O  \\nATOM     37  CB  VAL A   3      30.620  -6.985  42.742  1.00  0.00           C  \\nATOM     38  CG1 VAL A   3      30.028  -8.351  42.429  1.00  0.00           C  \\nATOM     39  CG2 VAL A   3      30.087  -5.933  41.782  1.00  0.00           C  \\nATOM     40  H   VAL A   3      30.321  -8.253  45.414  1.00  0.00           H  \\nATOM     41  HA  VAL A   3      30.645  -5.555  44.333  1.00  0.00           H  \\nATOM     42  HB  VAL A   3      31.700  -7.069  42.627  1.00  0.00           H  \\nATOM     43 HG11 VAL A   3      30.263  -8.622  41.399  1.00  0.00           H  \\nATOM     44 HG12 VAL A   3      30.450  -9.094  43.105  1.00  0.00           H  \\nATOM     45 HG13 VAL A   3      28.946  -8.316  42.556  1.00  0.00           H  \\nATOM     46 HG21 VAL A   3      30.316  -6.225  40.758  1.00  0.00           H  \\nATOM     47 HG22 VAL A   3      29.007  -5.845  41.901  1.00  0.00           H  \\nATOM     48 HG23 VAL A   3      30.555  -4.972  41.998  1.00  0.00           H  \\nATOM     49  N   ASN A   4      28.060  -5.590  44.381  1.00  0.00           N  \\nATOM     50  CA  ASN A   4      26.599  -5.580  44.406  1.00  0.00           C  \\nATOM     51  C   ASN A   4      26.074  -5.843  42.976  1.00  0.00           C  \\nATOM     52  O   ASN A   4      26.362  -5.014  42.111  1.00  0.00           O  \\nATOM     53  CB  ASN A   4      26.071  -4.269  44.960  1.00  0.00           C  \\nATOM     54  CG  ASN A   4      26.430  -4.067  46.406  1.00  0.00           C  \\nATOM     55  OD1 ASN A   4      26.224  -4.956  47.240  1.00  0.00           O  \\nATOM     56  ND2 ASN A   4      26.964  -2.914  46.719  1.00  0.00           N  \\nATOM     57  H   ASN A   4      28.508  -4.754  44.033  1.00  0.00           H  \\nATOM     58  HA  ASN A   4      26.257  -6.363  45.085  1.00  0.00           H  \\nATOM     59  HB3 ASN A   4      26.475  -3.440  44.378  1.00  0.00           H  \\nATOM     60  HB2 ASN A   4      24.986  -4.242  44.860  1.00  0.00           H  \\nATOM     61 HD21 ASN A   4      27.223  -2.723  47.666  1.00  0.00           H  \\nATOM     62 HD22 ASN A   4      27.113  -2.223  46.012  1.00  0.00           H  \\nATOM     63  N   PRO A   5      25.338  -6.933  42.695  1.00  0.00           N  \\nATOM     64  CA  PRO A   5      24.864  -7.193  41.343  1.00  0.00           C  \\nATOM     65  C   PRO A   5      24.042  -5.992  40.859  1.00  0.00           C  \\nATOM     66  O   PRO A   5      23.241  -5.428  41.607  1.00  0.00           O  \\nATOM     67  CB  PRO A   5      24.008  -8.453  41.503  1.00  0.00           C  \\nATOM     68  CG  PRO A   5      24.599  -9.150  42.680  1.00  0.00           C  \\nATOM     69  CD  PRO A   5      24.984  -8.038  43.619  1.00  0.00           C  \\nATOM     70  HA  PRO A   5      25.712  -7.332  40.672  1.00  0.00           H  \\nATOM     71  HB3 PRO A   5      22.954  -8.175  41.656  1.00  0.00           H  \\nATOM     72  HB2 PRO A   5      24.050  -9.056  40.584  1.00  0.00           H  \\nATOM     73  HG3 PRO A   5      23.865  -9.841  43.120  1.00  0.00           H  \\nATOM     74  HG2 PRO A   5      25.460  -9.758  42.368  1.00  0.00           H  \\nATOM     75  HD3 PRO A   5      24.124  -7.778  44.253  1.00  0.00           H  \\nATOM     76  HD2 PRO A   5      25.838  -8.357  44.235  1.00  0.00           H  \\nATOM     77  N   GLU A   6      24.299  -5.577  39.619  1.00  0.00           N  \\nATOM     78  CA  GLU A   6      23.572  -4.490  38.968  1.00  0.00           C  \\nATOM     79  C   GLU A   6      22.172  -4.979  38.571  1.00  0.00           C  \\nATOM     80  O   GLU A   6      22.040  -6.057  37.985  1.00  0.00           O  \\nATOM     81  CB  GLU A   6      24.332  -3.989  37.738  1.00  0.00           C  \\nATOM     82  CG  GLU A   6      23.709  -2.774  37.066  1.00  0.00           C  \\nATOM     83  CD  GLU A   6      24.515  -2.274  35.899  1.00  0.00           C  \\nATOM     84  OE1 GLU A   6      25.491  -2.901  35.563  1.00  0.00           O  \\nATOM     85  OE2 GLU A   6      24.154  -1.263  35.343  1.00  0.00           O  \\nATOM     86  H   GLU A   6      24.902  -6.135  39.032  1.00  0.00           H  \\nATOM     87  HA  GLU A   6      23.484  -3.670  39.680  1.00  0.00           H  \\nATOM     88  HB3 GLU A   6      25.352  -3.728  38.022  1.00  0.00           H  \\nATOM     89  HB2 GLU A   6      24.392  -4.787  36.997  1.00  0.00           H  \\nATOM     90  HG3 GLU A   6      22.711  -3.036  36.717  1.00  0.00           H  \\nATOM     91  HG2 GLU A   6      23.610  -1.976  37.801  1.00  0.00           H  \\nATOM     92  N   PRO A   7      21.113  -4.225  38.910  1.00  0.00           N  \\nATOM     93  CA  PRO A   7      19.762  -4.617  38.567  1.00  0.00           C  \\nATOM     94  C   PRO A   7      19.674  -4.852  37.061  1.00  0.00           C  \\nATOM     95  O   PRO A   7      20.196  -4.060  36.273  1.00  0.00           O  \\nATOM     96  CB  PRO A   7      18.921  -3.416  39.014  1.00  0.00           C  \\nATOM     97  CG  PRO A   7      19.708  -2.813  40.126  1.00  0.00           C  \\nATOM     98  CD  PRO A   7      21.143  -2.961  39.694  1.00  0.00           C  \\nATOM     99  HA  PRO A   7      19.494  -5.533  39.092  1.00  0.00           H  \\nATOM    100  HB3 PRO A   7      18.775  -2.725  38.171  1.00  0.00           H  \\nATOM    101  HB2 PRO A   7      17.923  -3.753  39.330  1.00  0.00           H  \\nATOM    102  HG3 PRO A   7      19.417  -1.762  40.272  1.00  0.00           H  \\nATOM    103  HG2 PRO A   7      19.494  -3.336  41.070  1.00  0.00           H  \\nATOM    104  HD3 PRO A   7      21.430  -2.101  39.072  1.00  0.00           H  \\nATOM    105  HD2 PRO A   7      21.789  -3.031  40.582  1.00  0.00           H  \\nATOM    106  N   ASP A   8      18.976  -5.925  36.687  1.00  0.00           N  \\nATOM    107  CA  ASP A   8      18.709  -6.285  35.302  1.00  0.00           C  \\nATOM    108  C   ASP A   8      17.950  -5.140  34.630  1.00  0.00           C  \\nATOM    109  O   ASP A   8      16.859  -4.737  35.045  1.00  0.00           O  \\nATOM    110  CB  ASP A   8      17.901  -7.582  35.216  1.00  0.00           C  \\nATOM    111  CG  ASP A   8      17.617  -8.010  33.783  1.00  0.00           C  \\nATOM    112  OD1 ASP A   8      18.518  -8.493  33.138  1.00  0.00           O  \\nATOM    113  OD2 ASP A   8      16.503  -7.849  33.346  1.00  0.00           O  \\nATOM    114  H   ASP A   8      18.589  -6.524  37.401  1.00  0.00           H  \\nATOM    115  HA  ASP A   8      19.656  -6.428  34.780  1.00  0.00           H  \\nATOM    116  HB3 ASP A   8      18.445  -8.383  35.718  1.00  0.00           H  \\nATOM    117  HB2 ASP A   8      16.952  -7.455  35.737  1.00  0.00           H  \\nATOM    118  N   ALA A   9      18.553  -4.603  33.572  1.00  0.00           N  \\nATOM    119  CA  ALA A   9      17.984  -3.536  32.786  1.00  0.00           C  \\nATOM    120  C   ALA A   9      16.924  -4.166  31.877  1.00  0.00           C  \\nATOM    121  O   ALA A   9      17.080  -4.226  30.658  1.00  0.00           O  \\nATOM    122  CB  ALA A   9      19.110  -2.822  32.026  1.00  0.00           C  \\nATOM    123  H   ALA A   9      19.403  -5.042  33.248  1.00  0.00           H  \\nATOM    124  HA  ALA A   9      17.496  -2.821  33.449  1.00  0.00           H  \\nATOM    125  HB1 ALA A   9      18.690  -2.022  31.416  1.00  0.00           H  \\nATOM    126  HB2 ALA A   9      19.631  -3.527  31.379  1.00  0.00           H  \\nATOM    127  HB3 ALA A   9      19.820  -2.397  32.736  1.00  0.00           H  \\nATOM    128  N   THR A  10      15.835  -4.657  32.470  1.00  0.00           N  \\nATOM    129  CA  THR A  10      14.765  -5.293  31.713  1.00  0.00           C  \\nATOM    130  C   THR A  10      14.378  -4.315  30.623  1.00  0.00           C  \\nATOM    131  O   THR A  10      13.975  -3.183  30.904  1.00  0.00           O  \\nATOM    132  CB  THR A  10      13.548  -5.644  32.589  1.00  0.00           C  \\nATOM    133  OG1 THR A  10      13.952  -6.534  33.638  1.00  0.00           O  \\nATOM    134  CG2 THR A  10      12.464  -6.308  31.755  1.00  0.00           C  \\nATOM    135  H   THR A  10      15.706  -4.487  33.458  1.00  0.00           H  \\nATOM    136  HA  THR A  10      15.139  -6.210  31.259  1.00  0.00           H  \\nATOM    137  HB  THR A  10      13.148  -4.735  33.038  1.00  0.00           H  \\nATOM    138  HG1 THR A  10      14.822  -6.888  33.439  1.00  0.00           H  \\nATOM    139 HG21 THR A  10      11.612  -6.548  32.390  1.00  0.00           H  \\nATOM    140 HG22 THR A  10      12.146  -5.628  30.964  1.00  0.00           H  \\nATOM    141 HG23 THR A  10      12.855  -7.223  31.312  1.00  0.00           H  \\nATOM    142  N   SER A  11      14.554  -4.743  29.376  1.00  0.00           N  \\nATOM    143  CA  SER A  11      14.343  -3.933  28.194  1.00  0.00           C  \\nATOM    144  C   SER A  11      12.839  -3.829  27.990  1.00  0.00           C  \\nATOM    145  O   SER A  11      12.246  -4.581  27.218  1.00  0.00           O  \\nATOM    146  CB  SER A  11      15.014  -4.546  26.981  1.00  0.00           C  \\nATOM    147  OG  SER A  11      16.404  -4.602  27.151  1.00  0.00           O  \\nATOM    148  H   SER A  11      14.753  -5.721  29.224  1.00  0.00           H  \\nATOM    149  HA  SER A  11      14.762  -2.938  28.345  1.00  0.00           H  \\nATOM    150  HB3 SER A  11      14.624  -5.551  26.819  1.00  0.00           H  \\nATOM    151  HB2 SER A  11      14.777  -3.956  26.097  1.00  0.00           H  \\nATOM    152  HG  SER A  11      16.573  -4.310  28.050  1.00  0.00           H  \\nATOM    153  N   VAL A  12      12.211  -2.921  28.730  1.00  0.00           N  \\nATOM    154  CA  VAL A  12      10.781  -2.689  28.622  1.00  0.00           C  \\nATOM    155  C   VAL A  12      10.621  -2.056  27.250  1.00  0.00           C  \\nATOM    156  O   VAL A  12      10.745  -0.842  27.090  1.00  0.00           O  \\nATOM    157  CB  VAL A  12      10.255  -1.757  29.730  1.00  0.00           C  \\nATOM    158  CG1 VAL A  12       8.769  -1.493  29.543  1.00  0.00           C  \\nATOM    159  CG2 VAL A  12      10.527  -2.371  31.095  1.00  0.00           C  \\nATOM    160  H   VAL A  12      12.764  -2.312  29.315  1.00  0.00           H  \\nATOM    161  HA  VAL A  12      10.261  -3.647  28.639  1.00  0.00           H  \\nATOM    162  HB  VAL A  12      10.764  -0.796  29.656  1.00  0.00           H  \\nATOM    163 HG11 VAL A  12       8.414  -0.833  30.335  1.00  0.00           H  \\nATOM    164 HG12 VAL A  12       8.602  -1.021  28.576  1.00  0.00           H  \\nATOM    165 HG13 VAL A  12       8.224  -2.436  29.587  1.00  0.00           H  \\nATOM    166 HG21 VAL A  12      10.153  -1.707  31.874  1.00  0.00           H  \\nATOM    167 HG22 VAL A  12      10.022  -3.335  31.167  1.00  0.00           H  \\nATOM    168 HG23 VAL A  12      11.600  -2.513  31.224  1.00  0.00           H  \\nATOM    169  N   GLU A  13      10.465  -2.893  26.223  1.00  0.00           N  \\nATOM    170  CA  GLU A  13      10.227  -2.409  24.874  1.00  0.00           C  \\nATOM    171  C   GLU A  13       8.996  -1.512  24.980  1.00  0.00           C  \\nATOM    172  O   GLU A  13       7.963  -1.910  25.520  1.00  0.00           O  \\nATOM    173  CB  GLU A  13      10.001  -3.557  23.887  1.00  0.00           C  \\nATOM    174  CG  GLU A  13       9.890  -3.122  22.432  1.00  0.00           C  \\nATOM    175  CD  GLU A  13       9.647  -4.272  21.495  1.00  0.00           C  \\nATOM    176  OE1 GLU A  13       9.612  -5.389  21.952  1.00  0.00           O  \\nATOM    177  OE2 GLU A  13       9.497  -4.033  20.320  1.00  0.00           O  \\nATOM    178  H   GLU A  13      10.327  -3.875  26.415  1.00  0.00           H  \\nATOM    179  HA  GLU A  13      11.077  -1.818  24.531  1.00  0.00           H  \\nATOM    180  HB3 GLU A  13      10.823  -4.269  23.961  1.00  0.00           H  \\nATOM    181  HB2 GLU A  13       9.084  -4.086  24.148  1.00  0.00           H  \\nATOM    182  HG3 GLU A  13       9.068  -2.412  22.338  1.00  0.00           H  \\nATOM    183  HG2 GLU A  13      10.809  -2.613  22.145  1.00  0.00           H  \\nATOM    184  N   ASN A  14       9.139  -0.260  24.542  1.00  0.00           N  \\nATOM    185  CA  ASN A  14       8.110   0.762  24.636  1.00  0.00           C  \\nATOM    186  C   ASN A  14       7.114   0.531  23.494  1.00  0.00           C  \\nATOM    187  O   ASN A  14       6.993   1.333  22.569  1.00  0.00           O  \\nATOM    188  CB  ASN A  14       8.708   2.156  24.585  1.00  0.00           C  \\nATOM    189  CG  ASN A  14       9.522   2.480  25.807  1.00  0.00           C  \\nATOM    190  OD1 ASN A  14       9.120   2.173  26.935  1.00  0.00           O  \\nATOM    191  ND2 ASN A  14      10.658   3.097  25.605  1.00  0.00           N  \\nATOM    192  H   ASN A  14      10.027  -0.013  24.127  1.00  0.00           H  \\nATOM    193  HA  ASN A  14       7.565   0.639  25.570  1.00  0.00           H  \\nATOM    194  HB3 ASN A  14       9.345   2.247  23.704  1.00  0.00           H  \\nATOM    195  HB2 ASN A  14       7.909   2.891  24.488  1.00  0.00           H  \\nATOM    196 HD21 ASN A  14      11.241   3.338  26.381  1.00  0.00           H  \\nATOM    197 HD22 ASN A  14      10.944   3.326  24.675  1.00  0.00           H  \\nATOM    198  N   VAL A  15       6.407  -0.595  23.535  1.00  0.00           N  \\nATOM    199  CA  VAL A  15       5.349  -0.901  22.589  1.00  0.00           C  \\nATOM    200  C   VAL A  15       4.192  -0.022  23.034  1.00  0.00           C  \\nATOM    201  O   VAL A  15       3.593  -0.266  24.081  1.00  0.00           O  \\nATOM    202  CB  VAL A  15       4.956  -2.390  22.613  1.00  0.00           C  \\nATOM    203  CG1 VAL A  15       3.827  -2.660  21.630  1.00  0.00           C  \\nATOM    204  CG2 VAL A  15       6.169  -3.250  22.292  1.00  0.00           C  \\nATOM    205  H   VAL A  15       6.481  -1.161  24.370  1.00  0.00           H  \\nATOM    206  HA  VAL A  15       5.665  -0.616  21.585  1.00  0.00           H  \\nATOM    207  HB  VAL A  15       4.584  -2.642  23.607  1.00  0.00           H  \\nATOM    208 HG11 VAL A  15       3.563  -3.717  21.661  1.00  0.00           H  \\nATOM    209 HG12 VAL A  15       2.958  -2.061  21.901  1.00  0.00           H  \\nATOM    210 HG13 VAL A  15       4.151  -2.397  20.623  1.00  0.00           H  \\nATOM    211 HG21 VAL A  15       5.884  -4.302  22.312  1.00  0.00           H  \\nATOM    212 HG22 VAL A  15       6.544  -2.994  21.301  1.00  0.00           H  \\nATOM    213 HG23 VAL A  15       6.949  -3.071  23.033  1.00  0.00           H  \\nATOM    214  N   ALA A  16       3.940   1.054  22.289  1.00  0.00           N  \\nATOM    215  CA  ALA A  16       2.936   2.044  22.644  1.00  0.00           C  \\nATOM    216  C   ALA A  16       1.622   1.311  22.953  1.00  0.00           C  \\nATOM    217  O   ALA A  16       1.146   0.502  22.151  1.00  0.00           O  \\nATOM    218  CB  ALA A  16       2.794   3.044  21.491  1.00  0.00           C  \\nATOM    219  H   ALA A  16       4.450   1.171  21.424  1.00  0.00           H  \\nATOM    220  HA  ALA A  16       3.268   2.577  23.535  1.00  0.00           H  \\nATOM    221  HB1 ALA A  16       2.480   2.528  20.583  1.00  0.00           H  \\nATOM    222  HB2 ALA A  16       3.747   3.541  21.307  1.00  0.00           H  \\nATOM    223  HB3 ALA A  16       2.046   3.795  21.746  1.00  0.00           H  \\nATOM    224  N   LEU A  17       1.102   1.518  24.166  1.00  0.00           N  \\nATOM    225  CA  LEU A  17      -0.101   0.844  24.635  1.00  0.00           C  \\nATOM    226  C   LEU A  17      -1.257   1.173  23.690  1.00  0.00           C  \\nATOM    227  O   LEU A  17      -1.555   2.339  23.429  1.00  0.00           O  \\nATOM    228  CB  LEU A  17      -0.443   1.278  26.066  1.00  0.00           C  \\nATOM    229  CG  LEU A  17      -1.656   0.585  26.699  1.00  0.00           C  \\nATOM    230  CD1 LEU A  17      -1.359  -0.898  26.878  1.00  0.00           C  \\nATOM    231  CD2 LEU A  17      -1.978   1.243  28.033  1.00  0.00           C  \\nATOM    232  H   LEU A  17       1.600   2.120  24.806  1.00  0.00           H  \\nATOM    233  HA  LEU A  17       0.074  -0.232  24.603  1.00  0.00           H  \\nATOM    234  HB3 LEU A  17       0.419   1.083  26.703  1.00  0.00           H  \\nATOM    235  HB2 LEU A  17      -0.635   2.350  26.068  1.00  0.00           H  \\nATOM    236  HG  LEU A  17      -2.515   0.675  26.034  1.00  0.00           H  \\nATOM    237 HD11 LEU A  17      -2.221  -1.390  27.328  1.00  0.00           H  \\nATOM    238 HD12 LEU A  17      -1.151  -1.347  25.907  1.00  0.00           H  \\nATOM    239 HD13 LEU A  17      -0.493  -1.019  27.527  1.00  0.00           H  \\nATOM    240 HD21 LEU A  17      -2.841   0.751  28.483  1.00  0.00           H  \\nATOM    241 HD22 LEU A  17      -1.120   1.152  28.700  1.00  0.00           H  \\nATOM    242 HD23 LEU A  17      -2.204   2.297  27.873  1.00  0.00           H  \\nATOM    243  N   LYS A  18      -1.917   0.124  23.198  1.00  0.00           N  \\nATOM    244  CA  LYS A  18      -3.060   0.257  22.299  1.00  0.00           C  \\nATOM    245  C   LYS A  18      -4.194   0.983  23.001  1.00  0.00           C  \\nATOM    246  O   LYS A  18      -4.485   0.699  24.162  1.00  0.00           O  \\nATOM    247  CB  LYS A  18      -3.531  -1.112  21.806  1.00  0.00           C  \\nATOM    248  CG  LYS A  18      -4.146  -1.992  22.887  1.00  0.00           C  \\nATOM    249  CD  LYS A  18      -4.544  -3.351  22.332  1.00  0.00           C  \\nATOM    250  CE  LYS A  18      -5.170  -4.227  23.408  1.00  0.00           C  \\nATOM    251  NZ  LYS A  18      -5.504  -5.584  22.896  1.00  0.00           N  \\nATOM    252  H   LYS A  18      -1.643  -0.794  23.519  1.00  0.00           H  \\nATOM    253  HA  LYS A  18      -2.758   0.834  21.425  1.00  0.00           H  \\nATOM    254  HB3 LYS A  18      -4.274  -0.980  21.019  1.00  0.00           H  \\nATOM    255  HB2 LYS A  18      -2.688  -1.653  21.374  1.00  0.00           H  \\nATOM    256  HG3 LYS A  18      -3.426  -2.135  23.693  1.00  0.00           H  \\nATOM    257  HG2 LYS A  18      -5.029  -1.502  23.295  1.00  0.00           H  \\nATOM    258  HD3 LYS A  18      -5.261  -3.218  21.521  1.00  0.00           H  \\nATOM    259  HD2 LYS A  18      -3.663  -3.855  21.935  1.00  0.00           H  \\nATOM    260  HE3 LYS A  18      -4.478  -4.327  24.243  1.00  0.00           H  \\nATOM    261  HE2 LYS A  18      -6.082  -3.757  23.775  1.00  0.00           H  \\nATOM    262  HZ1 LYS A  18      -5.917  -6.132  23.637  1.00  0.00           H  \\nATOM    263  HZ2 LYS A  18      -6.161  -5.504  22.132  1.00  0.00           H  \\nATOM    264  HZ3 LYS A  18      -4.663  -6.038  22.570  1.00  0.00           H  \\nATOM    265  N   THR A  19      -4.886   1.837  22.262  1.00  0.00           N  \\nATOM    266  CA  THR A  19      -6.093   2.508  22.752  1.00  0.00           C  \\nATOM    267  C   THR A  19      -7.273   2.188  21.842  1.00  0.00           C  \\nATOM    268  O   THR A  19      -7.079   1.884  20.668  1.00  0.00           O  \\nATOM    269  CB  THR A  19      -5.898   4.033  22.837  1.00  0.00           C  \\nATOM    270  OG1 THR A  19      -5.649   4.559  21.526  1.00  0.00           O  \\nATOM    271  CG2 THR A  19      -4.727   4.372  23.746  1.00  0.00           C  \\nATOM    272  H   THR A  19      -4.669   1.894  21.277  1.00  0.00           H  \\nATOM    273  HA  THR A  19      -6.351   2.111  23.734  1.00  0.00           H  \\nATOM    274  HB  THR A  19      -6.803   4.495  23.232  1.00  0.00           H  \\nATOM    275  HG1 THR A  19      -4.715   4.471  21.319  1.00  0.00           H  \\nATOM    276 HG21 THR A  19      -4.604   5.453  23.794  1.00  0.00           H  \\nATOM    277 HG22 THR A  19      -4.919   3.983  24.746  1.00  0.00           H  \\nATOM    278 HG23 THR A  19      -3.818   3.921  23.350  1.00  0.00           H  \\nATOM    279  N   GLY A  20      -8.482   2.233  22.397  1.00  0.00           N  \\nATOM    280  CA  GLY A  20      -9.726   1.868  21.717  1.00  0.00           C  \\nATOM    281  C   GLY A  20     -10.095   0.386  21.827  1.00  0.00           C  \\nATOM    282  O   GLY A  20      -9.234  -0.488  21.948  1.00  0.00           O  \\nATOM    283  H   GLY A  20      -8.558   2.486  23.373  1.00  0.00           H  \\nATOM    284  HA3 GLY A  20     -10.527   2.441  22.182  1.00  0.00           H  \\nATOM    285  HA2 GLY A  20      -9.686   2.145  20.664  1.00  0.00           H  \\nATOM    286  N   SER A  21     -11.399   0.104  21.832  1.00  0.00           N  \\nATOM    287  CA  SER A  21     -11.942  -1.254  21.984  1.00  0.00           C  \\nATOM    288  C   SER A  21     -11.802  -2.109  20.723  1.00  0.00           C  \\nATOM    289  O   SER A  21     -11.679  -3.329  20.811  1.00  0.00           O  \\nATOM    290  CB  SER A  21     -13.406  -1.177  22.373  1.00  0.00           C  \\nATOM    291  OG  SER A  21     -14.173  -0.630  21.336  1.00  0.00           O  \\nATOM    292  H   SER A  21     -12.055   0.870  21.766  1.00  0.00           H  \\nATOM    293  HA  SER A  21     -11.400  -1.762  22.782  1.00  0.00           H  \\nATOM    294  HB3 SER A  21     -13.772  -2.175  22.613  1.00  0.00           H  \\nATOM    295  HB2 SER A  21     -13.512  -0.566  23.269  1.00  0.00           H  \\nATOM    296  HG  SER A  21     -14.039   0.320  21.381  1.00  0.00           H  \\nATOM    297  N   GLY A  22     -11.832  -1.484  19.544  1.00  0.00           N  \\nATOM    298  CA  GLY A  22     -11.911  -2.176  18.260  1.00  0.00           C  \\nATOM    299  C   GLY A  22     -13.315  -2.641  17.882  1.00  0.00           C  \\nATOM    300  O   GLY A  22     -13.474  -3.295  16.851  1.00  0.00           O  \\nATOM    301  H   GLY A  22     -12.009  -0.490  19.545  1.00  0.00           H  \\nATOM    302  HA3 GLY A  22     -11.275  -3.061  18.292  1.00  0.00           H  \\nATOM    303  HA2 GLY A  22     -11.556  -1.507  17.475  1.00  0.00           H  \\nATOM    304  N   ASP A  23     -14.333  -2.314  18.680  1.00  0.00           N  \\nATOM    305  CA  ASP A  23     -15.706  -2.774  18.459  1.00  0.00           C  \\nATOM    306  C   ASP A  23     -16.466  -1.925  17.438  1.00  0.00           C  \\nATOM    307  O   ASP A  23     -17.547  -2.306  16.988  1.00  0.00           O  \\nATOM    308  CB  ASP A  23     -16.458  -2.780  19.796  1.00  0.00           C  \\nATOM    309  CG  ASP A  23     -15.974  -3.859  20.764  1.00  0.00           C  \\nATOM    310  OD1 ASP A  23     -15.415  -4.882  20.307  1.00  0.00           O  \\nATOM    311  OD2 ASP A  23     -16.467  -3.873  21.909  1.00  0.00           O  \\nATOM    312  H   ASP A  23     -14.137  -1.795  19.523  1.00  0.00           H  \\nATOM    313  HA  ASP A  23     -15.682  -3.786  18.058  1.00  0.00           H  \\nATOM    314  HB3 ASP A  23     -17.509  -2.986  19.594  1.00  0.00           H  \\nATOM    315  HB2 ASP A  23     -16.409  -1.796  20.263  1.00  0.00           H  \\nATOM    316  N   SER A  24     -15.900  -0.782  17.052  1.00  0.00           N  \\nATOM    317  CA  SER A  24     -16.547   0.194  16.182  1.00  0.00           C  \\nATOM    318  C   SER A  24     -15.522   0.974  15.368  1.00  0.00           C  \\nATOM    319  O   SER A  24     -14.424   1.269  15.834  1.00  0.00           O  \\nATOM    320  CB  SER A  24     -17.385   1.153  17.005  1.00  0.00           C  \\nATOM    321  OG  SER A  24     -17.879   2.197  16.212  1.00  0.00           O  \\nATOM    322  H   SER A  24     -14.949  -0.602  17.340  1.00  0.00           H  \\nATOM    323  HA  SER A  24     -17.196  -0.330  15.480  1.00  0.00           H  \\nATOM    324  HB3 SER A  24     -18.216   0.612  17.458  1.00  0.00           H  \\nATOM    325  HB2 SER A  24     -16.781   1.561  17.814  1.00  0.00           H  \\nATOM    326  HG  SER A  24     -18.800   1.986  16.040  1.00  0.00           H  \\nATOM    327  N   GLN A  25     -15.909   1.370  14.158  1.00  0.00           N  \\nATOM    328  CA  GLN A  25     -15.119   2.291  13.345  1.00  0.00           C  \\nATOM    329  C   GLN A  25     -14.810   3.609  14.078  1.00  0.00           C  \\nATOM    330  O   GLN A  25     -13.792   4.236  13.801  1.00  0.00           O  \\nATOM    331  CB  GLN A  25     -15.852   2.587  12.033  1.00  0.00           C  \\nATOM    332  CG  GLN A  25     -15.043   3.402  11.039  1.00  0.00           C  \\nATOM    333  CD  GLN A  25     -15.736   3.530   9.696  1.00  0.00           C  \\nATOM    334  OE1 GLN A  25     -16.756   2.883   9.444  1.00  0.00           O  \\nATOM    335  NE2 GLN A  25     -15.186   4.368   8.825  1.00  0.00           N  \\nATOM    336  H   GLN A  25     -16.780   1.023  13.782  1.00  0.00           H  \\nATOM    337  HA  GLN A  25     -14.169   1.815  13.104  1.00  0.00           H  \\nATOM    338  HB3 GLN A  25     -16.129   1.649  11.552  1.00  0.00           H  \\nATOM    339  HB2 GLN A  25     -16.772   3.131  12.245  1.00  0.00           H  \\nATOM    340  HG3 GLN A  25     -14.892   4.403  11.443  1.00  0.00           H  \\nATOM    341  HG2 GLN A  25     -14.081   2.914  10.882  1.00  0.00           H  \\nATOM    342 HE21 GLN A  25     -15.599   4.494   7.922  1.00  0.00           H  \\nATOM    343 HE22 GLN A  25     -14.358   4.873   9.070  1.00  0.00           H  \\nATOM    344  N   SER A  26     -15.666   4.031  15.016  1.00  0.00           N  \\nATOM    345  CA  SER A  26     -15.465   5.237  15.835  1.00  0.00           C  \\nATOM    346  C   SER A  26     -14.514   5.027  17.017  1.00  0.00           C  \\nATOM    347  O   SER A  26     -14.084   6.006  17.621  1.00  0.00           O  \\nATOM    348  CB  SER A  26     -16.803   5.724  16.355  1.00  0.00           C  \\nATOM    349  OG  SER A  26     -17.345   4.818  17.276  1.00  0.00           O  \\nATOM    350  H   SER A  26     -16.481   3.473  15.227  1.00  0.00           H  \\nATOM    351  HA  SER A  26     -15.024   6.016  15.215  1.00  0.00           H  \\nATOM    352  HB3 SER A  26     -16.676   6.697  16.829  1.00  0.00           H  \\nATOM    353  HB2 SER A  26     -17.491   5.855  15.521  1.00  0.00           H  \\nATOM    354  HG  SER A  26     -17.442   3.987  16.804  1.00  0.00           H  \\nATOM    355  N   ASP A  27     -14.174   3.776  17.334  1.00  0.00           N  \\nATOM    356  CA  ASP A  27     -13.286   3.409  18.439  1.00  0.00           C  \\nATOM    357  C   ASP A  27     -12.361   2.237  18.047  1.00  0.00           C  \\nATOM    358  O   ASP A  27     -12.427   1.159  18.647  1.00  0.00           O  \\nATOM    359  CB  ASP A  27     -14.105   3.036  19.677  1.00  0.00           C  \\nATOM    360  CG  ASP A  27     -13.251   2.883  20.928  1.00  0.00           C  \\nATOM    361  OD1 ASP A  27     -12.333   3.652  21.094  1.00  0.00           O  \\nATOM    362  OD2 ASP A  27     -13.525   2.001  21.706  1.00  0.00           O  \\nATOM    363  H   ASP A  27     -14.502   3.022  16.748  1.00  0.00           H  \\nATOM    364  HA  ASP A  27     -12.636   4.254  18.666  1.00  0.00           H  \\nATOM    365  HB3 ASP A  27     -14.858   3.803  19.860  1.00  0.00           H  \\nATOM    366  HB2 ASP A  27     -14.630   2.097  19.496  1.00  0.00           H  \\nATOM    367  N   PRO A  28     -11.535   2.393  16.993  1.00  0.00           N  \\nATOM    368  CA  PRO A  28     -10.661   1.330  16.516  1.00  0.00           C  \\nATOM    369  C   PRO A  28      -9.482   1.117  17.469  1.00  0.00           C  \\nATOM    370  O   PRO A  28      -9.092   2.034  18.190  1.00  0.00           O  \\nATOM    371  CB  PRO A  28     -10.201   1.854  15.151  1.00  0.00           C  \\nATOM    372  CG  PRO A  28     -10.216   3.336  15.304  1.00  0.00           C  \\nATOM    373  CD  PRO A  28     -11.425   3.613  16.158  1.00  0.00           C  \\nATOM    374  HA  PRO A  28     -11.221   0.401  16.406  1.00  0.00           H  \\nATOM    375  HB3 PRO A  28      -9.202   1.459  14.914  1.00  0.00           H  \\nATOM    376  HB2 PRO A  28     -10.882   1.499  14.363  1.00  0.00           H  \\nATOM    377  HG3 PRO A  28      -9.280   3.679  15.771  1.00  0.00           H  \\nATOM    378  HG2 PRO A  28     -10.274   3.819  14.318  1.00  0.00           H  \\nATOM    379  HD3 PRO A  28     -11.244   4.508  16.770  1.00  0.00           H  \\nATOM    380  HD2 PRO A  28     -12.305   3.754  15.513  1.00  0.00           H  \\nATOM    381  N   ILE A  29      -8.833  -0.048  17.391  1.00  0.00           N  \\nATOM    382  CA  ILE A  29      -7.560  -0.269  18.088  1.00  0.00           C  \\nATOM    383  C   ILE A  29      -6.487   0.587  17.417  1.00  0.00           C  \\nATOM    384  O   ILE A  29      -6.111   0.336  16.272  1.00  0.00           O  \\nATOM    385  CB  ILE A  29      -7.146  -1.752  18.064  1.00  0.00           C  \\nATOM    386  CG1 ILE A  29      -8.176  -2.606  18.807  1.00  0.00           C  \\nATOM    387  CG2 ILE A  29      -5.764  -1.929  18.674  1.00  0.00           C  \\nATOM    388  CD1 ILE A  29      -7.875  -4.087  18.782  1.00  0.00           C  \\nATOM    389  H   ILE A  29      -9.146  -0.739  16.723  1.00  0.00           H  \\nATOM    390  HA  ILE A  29      -7.674   0.056  19.122  1.00  0.00           H  \\nATOM    391  HB  ILE A  29      -7.126  -2.108  17.034  1.00  0.00           H  \\nATOM    392 HG13 ILE A  29      -8.231  -2.287  19.848  1.00  0.00           H  \\nATOM    393 HG12 ILE A  29      -9.162  -2.453  18.367  1.00  0.00           H  \\nATOM    394 HG21 ILE A  29      -5.488  -2.982  18.650  1.00  0.00           H  \\nATOM    395 HG22 ILE A  29      -5.038  -1.350  18.104  1.00  0.00           H  \\nATOM    396 HG23 ILE A  29      -5.775  -1.580  19.707  1.00  0.00           H  \\nATOM    397 HD11 ILE A  29      -8.649  -4.626  19.329  1.00  0.00           H  \\nATOM    398 HD12 ILE A  29      -7.850  -4.436  17.749  1.00  0.00           H  \\nATOM    399 HD13 ILE A  29      -6.908  -4.269  19.250  1.00  0.00           H  \\nATOM    400  N   LYS A  30      -5.967   1.572  18.138  1.00  0.00           N  \\nATOM    401  CA  LYS A  30      -4.924   2.490  17.683  1.00  0.00           C  \\nATOM    402  C   LYS A  30      -3.577   2.010  18.187  1.00  0.00           C  \\nATOM    403  O   LYS A  30      -3.397   1.869  19.397  1.00  0.00           O  \\nATOM    404  CB  LYS A  30      -5.195   3.917  18.162  1.00  0.00           C  \\nATOM    405  CG  LYS A  30      -6.427   4.565  17.544  1.00  0.00           C  \\nATOM    406  CD  LYS A  30      -6.635   5.975  18.076  1.00  0.00           C  \\nATOM    407  CE  LYS A  30      -7.887   6.610  17.489  1.00  0.00           C  \\nATOM    408  NZ  LYS A  30      -8.126   7.974  18.032  1.00  0.00           N  \\nATOM    409  H   LYS A  30      -6.343   1.700  19.067  1.00  0.00           H  \\nATOM    410  HA  LYS A  30      -4.896   2.521  16.593  1.00  0.00           H  \\nATOM    411  HB3 LYS A  30      -5.324   3.919  19.245  1.00  0.00           H  \\nATOM    412  HB2 LYS A  30      -4.335   4.547  17.934  1.00  0.00           H  \\nATOM    413  HG3 LYS A  30      -6.310   4.608  16.460  1.00  0.00           H  \\nATOM    414  HG2 LYS A  30      -7.307   3.966  17.773  1.00  0.00           H  \\nATOM    415  HD3 LYS A  30      -6.728   5.944  19.162  1.00  0.00           H  \\nATOM    416  HD2 LYS A  30      -5.773   6.592  17.820  1.00  0.00           H  \\nATOM    417  HE3 LYS A  30      -7.789   6.675  16.406  1.00  0.00           H  \\nATOM    418  HE2 LYS A  30      -8.752   5.986  17.716  1.00  0.00           H  \\nATOM    419  HZ1 LYS A  30      -8.964   8.360  17.619  1.00  0.00           H  \\nATOM    420  HZ2 LYS A  30      -8.238   7.924  19.035  1.00  0.00           H  \\nATOM    421  HZ3 LYS A  30      -7.340   8.568  17.811  1.00  0.00           H  \\nATOM    422  N   ALA A  31      -2.657   1.752  17.267  1.00  0.00           N  \\nATOM    423  CA  ALA A  31      -1.329   1.257  17.590  1.00  0.00           C  \\nATOM    424  C   ALA A  31      -0.307   1.668  16.529  1.00  0.00           C  \\nATOM    425  O   ALA A  31      -0.647   1.893  15.365  1.00  0.00           O  \\nATOM    426  CB  ALA A  31      -1.391  -0.267  17.735  1.00  0.00           C  \\nATOM    427  H   ALA A  31      -2.868   1.932  16.296  1.00  0.00           H  \\nATOM    428  HA  ALA A  31      -1.007   1.684  18.540  1.00  0.00           H  \\nATOM    429  HB1 ALA A  31      -2.047  -0.526  18.567  1.00  0.00           H  \\nATOM    430  HB2 ALA A  31      -1.785  -0.714  16.823  1.00  0.00           H  \\nATOM    431  HB3 ALA A  31      -0.396  -0.663  17.935  1.00  0.00           H  \\nATOM    432  N   ASP A  32       0.957   1.705  16.932  1.00  0.00           N  \\nATOM    433  CA  ASP A  32       2.069   1.938  16.021  1.00  0.00           C  \\nATOM    434  C   ASP A  32       2.450   0.643  15.287  1.00  0.00           C  \\nATOM    435  O   ASP A  32       2.255  -0.469  15.786  1.00  0.00           O  \\nATOM    436  CB  ASP A  32       3.279   2.485  16.784  1.00  0.00           C  \\nATOM    437  CG  ASP A  32       2.997   3.818  17.465  1.00  0.00           C  \\nATOM    438  OD1 ASP A  32       1.995   4.419  17.158  1.00  0.00           O  \\nATOM    439  OD2 ASP A  32       3.787   4.221  18.284  1.00  0.00           O  \\nATOM    440  H   ASP A  32       1.173   1.490  17.895  1.00  0.00           H  \\nATOM    441  HA  ASP A  32       1.757   2.656  15.262  1.00  0.00           H  \\nATOM    442  HB3 ASP A  32       3.587   1.764  17.541  1.00  0.00           H  \\nATOM    443  HB2 ASP A  32       4.115   2.614  16.095  1.00  0.00           H  \\nATOM    444  N   LEU A  33       3.003   0.794  14.085  1.00  0.00           N  \\nATOM    445  CA  LEU A  33       3.597  -0.288  13.308  1.00  0.00           C  \\nATOM    446  C   LEU A  33       5.084  -0.004  13.091  1.00  0.00           C  \\nATOM    447  O   LEU A  33       5.466   1.054  12.592  1.00  0.00           O  \\nATOM    448  CB  LEU A  33       2.790  -0.471  12.011  1.00  0.00           C  \\nATOM    449  CG  LEU A  33       3.297  -1.623  11.122  1.00  0.00           C  \\nATOM    450  CD1 LEU A  33       2.152  -2.215  10.297  1.00  0.00           C  \\nATOM    451  CD2 LEU A  33       4.361  -1.153  10.129  1.00  0.00           C  \\nATOM    452  H   LEU A  33       3.058   1.727  13.703  1.00  0.00           H  \\nATOM    453  HA  LEU A  33       3.520  -1.216  13.875  1.00  0.00           H  \\nATOM    454  HB3 LEU A  33       1.760  -0.680  12.301  1.00  0.00           H  \\nATOM    455  HB2 LEU A  33       2.794   0.459  11.442  1.00  0.00           H  \\nATOM    456  HG  LEU A  33       3.707  -2.416  11.748  1.00  0.00           H  \\nATOM    457 HD11 LEU A  33       1.336  -2.521  10.953  1.00  0.00           H  \\nATOM    458 HD12 LEU A  33       2.509  -3.088   9.748  1.00  0.00           H  \\nATOM    459 HD13 LEU A  33       1.792  -1.475   9.584  1.00  0.00           H  \\nATOM    460 HD21 LEU A  33       3.959  -0.359   9.499  1.00  0.00           H  \\nATOM    461 HD22 LEU A  33       4.661  -1.981   9.485  1.00  0.00           H  \\nATOM    462 HD23 LEU A  33       5.249  -0.788  10.645  1.00  0.00           H  \\nATOM    463  N   GLU A  34       5.926  -0.973  13.437  1.00  0.00           N  \\nATOM    464  CA  GLU A  34       7.369  -0.880  13.238  1.00  0.00           C  \\nATOM    465  C   GLU A  34       7.754  -1.446  11.864  1.00  0.00           C  \\nATOM    466  O   GLU A  34       7.588  -2.636  11.586  1.00  0.00           O  \\nATOM    467  CB  GLU A  34       8.113  -1.628  14.346  1.00  0.00           C  \\nATOM    468  CG  GLU A  34       9.629  -1.502  14.281  1.00  0.00           C  \\nATOM    469  CD  GLU A  34      10.326  -2.261  15.376  1.00  0.00           C  \\nATOM    470  OE1 GLU A  34       9.796  -3.252  15.817  1.00  0.00           O  \\nATOM    471  OE2 GLU A  34      11.391  -1.849  15.772  1.00  0.00           O  \\nATOM    472  H   GLU A  34       5.557  -1.833  13.817  1.00  0.00           H  \\nATOM    473  HA  GLU A  34       7.665   0.169  13.271  1.00  0.00           H  \\nATOM    474  HB3 GLU A  34       7.788  -1.256  15.318  1.00  0.00           H  \\nATOM    475  HB2 GLU A  34       7.863  -2.688  14.301  1.00  0.00           H  \\nATOM    476  HG3 GLU A  34       9.973  -1.878  13.318  1.00  0.00           H  \\nATOM    477  HG2 GLU A  34       9.899  -0.449  14.347  1.00  0.00           H  \\nATOM    478  N   VAL A  35       8.279  -0.585  10.994  1.00  0.00           N  \\nATOM    479  CA  VAL A  35       8.665  -0.926   9.624  1.00  0.00           C  \\nATOM    480  C   VAL A  35      10.051  -1.566   9.626  1.00  0.00           C  \\nATOM    481  O   VAL A  35      11.004  -1.007  10.172  1.00  0.00           O  \\nATOM    482  CB  VAL A  35       8.637   0.329   8.736  1.00  0.00           C  \\nATOM    483  CG1 VAL A  35       9.114   0.041   7.315  1.00  0.00           C  \\nATOM    484  CG2 VAL A  35       7.224   0.920   8.654  1.00  0.00           C  \\nATOM    485  H   VAL A  35       8.475   0.355  11.307  1.00  0.00           H  \\nATOM    486  HA  VAL A  35       7.948  -1.642   9.223  1.00  0.00           H  \\nATOM    487  HB  VAL A  35       9.307   1.082   9.151  1.00  0.00           H  \\nATOM    488 HG11 VAL A  35      10.155  -0.280   7.310  1.00  0.00           H  \\nATOM    489 HG12 VAL A  35       9.054   0.970   6.749  1.00  0.00           H  \\nATOM    490 HG13 VAL A  35       8.492  -0.724   6.848  1.00  0.00           H  \\nATOM    491 HG21 VAL A  35       7.217   1.786   7.993  1.00  0.00           H  \\nATOM    492 HG22 VAL A  35       6.895   1.241   9.643  1.00  0.00           H  \\nATOM    493 HG23 VAL A  35       6.526   0.173   8.274  1.00  0.00           H  \\nATOM    494  N   LYS A  36      10.181  -2.725   8.969  1.00  0.00           N  \\nATOM    495  CA  LYS A  36      11.411  -3.537   8.972  1.00  0.00           C  \\nATOM    496  C   LYS A  36      11.876  -3.872   7.561  1.00  0.00           C  \\nATOM    497  O   LYS A  36      11.099  -4.374   6.748  1.00  0.00           O  \\nATOM    498  CB  LYS A  36      11.215  -4.817   9.812  1.00  0.00           C  \\nATOM    499  CG  LYS A  36      10.903  -4.466  11.271  1.00  0.00           C  \\nATOM    500  CD  LYS A  36      10.742  -5.669  12.216  1.00  0.00           C  \\nATOM    501  CE  LYS A  36      10.413  -4.992  13.549  1.00  0.00           C  \\nATOM    502  NZ  LYS A  36      10.201  -5.852  14.733  1.00  0.00           N  \\nATOM    503  H   LYS A  36       9.370  -3.075   8.478  1.00  0.00           H  \\nATOM    504  HA  LYS A  36      12.206  -2.961   9.443  1.00  0.00           H  \\nATOM    505  HB3 LYS A  36      10.400  -5.411   9.396  1.00  0.00           H  \\nATOM    506  HB2 LYS A  36      12.131  -5.407   9.780  1.00  0.00           H  \\nATOM    507  HG3 LYS A  36      11.707  -3.834  11.647  1.00  0.00           H  \\nATOM    508  HG2 LYS A  36       9.979  -3.888  11.297  1.00  0.00           H  \\nATOM    509  HD3 LYS A  36      11.676  -6.226  12.283  1.00  0.00           H  \\nATOM    510  HD2 LYS A  36       9.922  -6.308  11.890  1.00  0.00           H  \\nATOM    511  HE3 LYS A  36       9.536  -4.358  13.413  1.00  0.00           H  \\nATOM    512  HE2 LYS A  36      11.232  -4.315  13.785  1.00  0.00           H  \\nATOM    513  HZ1 LYS A  36       9.389  -6.446  14.658  1.00  0.00           H  \\nATOM    514  HZ2 LYS A  36      10.040  -5.195  15.483  1.00  0.00           H  \\nATOM    515  HZ3 LYS A  36      11.037  -6.359  14.988  1.00  0.00           H  \\nATOM    516  N   GLY A  37      13.161  -3.654   7.290  1.00  0.00           N  \\nATOM    517  CA  GLY A  37      13.820  -4.087   6.057  1.00  0.00           C  \\nATOM    518  C   GLY A  37      13.093  -3.641   4.784  1.00  0.00           C  \\nATOM    519  O   GLY A  37      12.685  -2.491   4.648  1.00  0.00           O  \\nATOM    520  H   GLY A  37      13.721  -3.190   7.990  1.00  0.00           H  \\nATOM    521  HA3 GLY A  37      13.898  -5.175   6.065  1.00  0.00           H  \\nATOM    522  HA2 GLY A  37      14.831  -3.677   6.025  1.00  0.00           H  \\nATOM    523  N   GLN A  38      12.902  -4.572   3.848  1.00  0.00           N  \\nATOM    524  CA  GLN A  38      12.286  -4.280   2.550  1.00  0.00           C  \\nATOM    525  C   GLN A  38      10.769  -4.035   2.599  1.00  0.00           C  \\nATOM    526  O   GLN A  38      10.204  -3.676   1.563  1.00  0.00           O  \\nATOM    527  CB  GLN A  38      12.666  -5.370   1.538  1.00  0.00           C  \\nATOM    528  CG  GLN A  38      14.165  -5.303   1.205  1.00  0.00           C  \\nATOM    529  CD  GLN A  38      14.620  -6.481   0.357  1.00  0.00           C  \\nATOM    530  OE1 GLN A  38      14.118  -7.583   0.465  1.00  0.00           O  \\nATOM    531  NE2 GLN A  38      15.593  -6.313  -0.511  1.00  0.00           N  \\nATOM    532  H   GLN A  38      13.244  -5.507   4.014  1.00  0.00           H  \\nATOM    533  HA  GLN A  38      12.708  -3.342   2.190  1.00  0.00           H  \\nATOM    534  HB3 GLN A  38      12.112  -5.210   0.613  1.00  0.00           H  \\nATOM    535  HB2 GLN A  38      12.405  -6.353   1.930  1.00  0.00           H  \\nATOM    536  HG3 GLN A  38      14.756  -5.287   2.121  1.00  0.00           H  \\nATOM    537  HG2 GLN A  38      14.365  -4.384   0.653  1.00  0.00           H  \\nATOM    538 HE21 GLN A  38      15.918  -7.145  -0.983  1.00  0.00           H  \\nATOM    539 HE22 GLN A  38      16.118  -5.451  -0.561  1.00  0.00           H  \\nATOM    540  N   SER A  39      10.114  -4.151   3.764  1.00  0.00           N  \\nATOM    541  CA  SER A  39       8.699  -3.769   3.921  1.00  0.00           C  \\nATOM    542  C   SER A  39       8.469  -2.254   3.837  1.00  0.00           C  \\nATOM    543  O   SER A  39       7.337  -1.825   3.636  1.00  0.00           O  \\nATOM    544  CB  SER A  39       8.099  -4.298   5.227  1.00  0.00           C  \\nATOM    545  OG  SER A  39       8.640  -3.655   6.366  1.00  0.00           O  \\nATOM    546  H   SER A  39      10.623  -4.437   4.588  1.00  0.00           H  \\nATOM    547  HA  SER A  39       8.134  -4.219   3.104  1.00  0.00           H  \\nATOM    548  HB3 SER A  39       7.023  -4.128   5.203  1.00  0.00           H  \\nATOM    549  HB2 SER A  39       8.268  -5.372   5.304  1.00  0.00           H  \\nATOM    550  HG  SER A  39       9.581  -3.843   6.390  1.00  0.00           H  \\nATOM    551  N   ALA A  40       9.529  -1.443   3.934  1.00  0.00           N  \\nATOM    552  CA  ALA A  40       9.429   0.012   3.909  1.00  0.00           C  \\nATOM    553  C   ALA A  40       8.778   0.559   2.636  1.00  0.00           C  \\nATOM    554  O   ALA A  40       7.953   1.459   2.728  1.00  0.00           O  \\nATOM    555  CB  ALA A  40      10.829   0.597   4.121  1.00  0.00           C  \\nATOM    556  H   ALA A  40      10.436  -1.845   4.126  1.00  0.00           H  \\nATOM    557  HA  ALA A  40       8.784   0.331   4.727  1.00  0.00           H  \\nATOM    558  HB1 ALA A  40      11.490   0.315   3.302  1.00  0.00           H  \\nATOM    559  HB2 ALA A  40      11.245   0.229   5.059  1.00  0.00           H  \\nATOM    560  HB3 ALA A  40      10.767   1.684   4.173  1.00  0.00           H  \\nATOM    561  N   LEU A  41       9.104  -0.002   1.468  1.00  0.00           N  \\nATOM    562  CA  LEU A  41       8.523   0.425   0.194  1.00  0.00           C  \\nATOM    563  C   LEU A  41       6.992   0.249   0.140  1.00  0.00           C  \\nATOM    564  O   LEU A  41       6.305   1.248  -0.062  1.00  0.00           O  \\nATOM    565  CB  LEU A  41       9.169  -0.362  -0.953  1.00  0.00           C  \\nATOM    566  CG  LEU A  41       8.475  -0.242  -2.316  1.00  0.00           C  \\nATOM    567  CD1 LEU A  41       8.431   1.221  -2.737  1.00  0.00           C  \\nATOM    568  CD2 LEU A  41       9.219  -1.084  -3.341  1.00  0.00           C  \\nATOM    569  H   LEU A  41       9.772  -0.758   1.473  1.00  0.00           H  \\nATOM    570  HA  LEU A  41       8.699   1.496   0.093  1.00  0.00           H  \\nATOM    571  HB3 LEU A  41      10.196  -0.020  -1.073  1.00  0.00           H  \\nATOM    572  HB2 LEU A  41       9.188  -1.417  -0.683  1.00  0.00           H  \\nATOM    573  HG  LEU A  41       7.447  -0.596  -2.233  1.00  0.00           H  \\nATOM    574 HD11 LEU A  41       7.937   1.306  -3.705  1.00  0.00           H  \\nATOM    575 HD12 LEU A  41       7.876   1.796  -1.996  1.00  0.00           H  \\nATOM    576 HD13 LEU A  41       9.446   1.608  -2.812  1.00  0.00           H  \\nATOM    577 HD21 LEU A  41       8.726  -0.999  -4.309  1.00  0.00           H  \\nATOM    578 HD22 LEU A  41      10.247  -0.730  -3.425  1.00  0.00           H  \\nATOM    579 HD23 LEU A  41       9.220  -2.127  -3.024  1.00  0.00           H  \\nATOM    580  N   PRO A  42       6.427  -0.964   0.302  1.00  0.00           N  \\nATOM    581  CA  PRO A  42       4.981  -1.133   0.257  1.00  0.00           C  \\nATOM    582  C   PRO A  42       4.269  -0.359   1.371  1.00  0.00           C  \\nATOM    583  O   PRO A  42       3.202   0.187   1.124  1.00  0.00           O  \\nATOM    584  CB  PRO A  42       4.745  -2.641   0.309  1.00  0.00           C  \\nATOM    585  CG  PRO A  42       5.979  -3.193   0.999  1.00  0.00           C  \\nATOM    586  CD  PRO A  42       7.076  -2.292   0.447  1.00  0.00           C  \\nATOM    587  HA  PRO A  42       4.602  -0.756  -0.693  1.00  0.00           H  \\nATOM    588  HB3 PRO A  42       4.705  -3.032  -0.708  1.00  0.00           H  \\nATOM    589  HB2 PRO A  42       3.835  -2.894   0.854  1.00  0.00           H  \\nATOM    590  HG3 PRO A  42       6.144  -4.241   0.747  1.00  0.00           H  \\nATOM    591  HG2 PRO A  42       5.879  -3.058   2.076  1.00  0.00           H  \\nATOM    592  HD3 PRO A  42       7.365  -2.630  -0.547  1.00  0.00           H  \\nATOM    593  HD2 PRO A  42       7.936  -2.292   1.118  1.00  0.00           H  \\nATOM    594  N   PHE A  43       4.896  -0.161   2.539  1.00  0.00           N  \\nATOM    595  CA  PHE A  43       4.305   0.725   3.549  1.00  0.00           C  \\nATOM    596  C   PHE A  43       4.338   2.199   3.133  1.00  0.00           C  \\nATOM    597  O   PHE A  43       3.377   2.920   3.379  1.00  0.00           O  \\nATOM    598  CB  PHE A  43       4.972   0.524   4.914  1.00  0.00           C  \\nATOM    599  CG  PHE A  43       4.549  -0.742   5.638  1.00  0.00           C  \\nATOM    600  CD1 PHE A  43       3.187  -1.006   5.872  1.00  0.00           C  \\nATOM    601  CD2 PHE A  43       5.518  -1.644   6.114  1.00  0.00           C  \\nATOM    602  CE1 PHE A  43       2.799  -2.157   6.578  1.00  0.00           C  \\nATOM    603  CE2 PHE A  43       5.132  -2.797   6.813  1.00  0.00           C  \\nATOM    604  CZ  PHE A  43       3.773  -3.053   7.051  1.00  0.00           C  \\nATOM    605  H   PHE A  43       5.788  -0.597   2.723  1.00  0.00           H  \\nATOM    606  HA  PHE A  43       3.248   0.478   3.638  1.00  0.00           H  \\nATOM    607  HB3 PHE A  43       4.701   1.362   5.555  1.00  0.00           H  \\nATOM    608  HB2 PHE A  43       6.055   0.545   4.790  1.00  0.00           H  \\nATOM    609  HD1 PHE A  43       2.429  -0.330   5.506  1.00  0.00           H  \\nATOM    610  HD2 PHE A  43       6.565  -1.446   5.942  1.00  0.00           H  \\nATOM    611  HE1 PHE A  43       1.748  -2.354   6.737  1.00  0.00           H  \\nATOM    612  HE2 PHE A  43       5.886  -3.479   7.177  1.00  0.00           H  \\nATOM    613  HZ  PHE A  43       3.481  -3.939   7.596  1.00  0.00           H  \\nATOM    614  N   ASP A  44       5.389   2.666   2.459  1.00  0.00           N  \\nATOM    615  CA  ASP A  44       5.412   4.030   1.926  1.00  0.00           C  \\nATOM    616  C   ASP A  44       4.335   4.272   0.855  1.00  0.00           C  \\nATOM    617  O   ASP A  44       3.787   5.374   0.776  1.00  0.00           O  \\nATOM    618  CB  ASP A  44       6.792   4.337   1.338  1.00  0.00           C  \\nATOM    619  CG  ASP A  44       7.127   5.822   1.357  1.00  0.00           C  \\nATOM    620  OD1 ASP A  44       7.074   6.410   2.412  1.00  0.00           O  \\nATOM    621  OD2 ASP A  44       7.432   6.355   0.317  1.00  0.00           O  \\nATOM    622  H   ASP A  44       6.198   2.077   2.320  1.00  0.00           H  \\nATOM    623  HA  ASP A  44       5.192   4.710   2.749  1.00  0.00           H  \\nATOM    624  HB3 ASP A  44       7.555   3.800   1.901  1.00  0.00           H  \\nATOM    625  HB2 ASP A  44       6.835   3.983   0.308  1.00  0.00           H  \\nATOM    626  N   VAL A  45       3.964   3.235   0.089  1.00  0.00           N  \\nATOM    627  CA  VAL A  45       2.805   3.270  -0.825  1.00  0.00           C  \\nATOM    628  C   VAL A  45       1.509   3.430  -0.025  1.00  0.00           C  \\nATOM    629  O   VAL A  45       0.703   4.305  -0.349  1.00  0.00           O  \\nATOM    630  CB  VAL A  45       2.734   1.982  -1.666  1.00  0.00           C  \\nATOM    631  CG1 VAL A  45       1.440   1.938  -2.465  1.00  0.00           C  \\nATOM    632  CG2 VAL A  45       3.941   1.898  -2.588  1.00  0.00           C  \\nATOM    633  H   VAL A  45       4.411   2.345   0.261  1.00  0.00           H  \\nATOM    634  HA  VAL A  45       2.899   4.136  -1.479  1.00  0.00           H  \\nATOM    635  HB  VAL A  45       2.727   1.121  -0.996  1.00  0.00           H  \\nATOM    636 HG11 VAL A  45       1.407   1.021  -3.054  1.00  0.00           H  \\nATOM    637 HG12 VAL A  45       0.591   1.963  -1.783  1.00  0.00           H  \\nATOM    638 HG13 VAL A  45       1.396   2.799  -3.133  1.00  0.00           H  \\nATOM    639 HG21 VAL A  45       3.885   0.985  -3.179  1.00  0.00           H  \\nATOM    640 HG22 VAL A  45       3.951   2.762  -3.254  1.00  0.00           H  \\nATOM    641 HG23 VAL A  45       4.854   1.889  -1.993  1.00  0.00           H  \\nATOM    642  N   ASP A  46       1.325   2.637   1.034  1.00  0.00           N  \\nATOM    643  CA  ASP A  46       0.176   2.736   1.946  1.00  0.00           C  \\nATOM    644  C   ASP A  46       0.038   4.158   2.502  1.00  0.00           C  \\nATOM    645  O   ASP A  46      -1.025   4.781   2.446  1.00  0.00           O  \\nATOM    646  CB  ASP A  46       0.317   1.741   3.101  1.00  0.00           C  \\nATOM    647  CG  ASP A  46       0.303   0.290   2.638  1.00  0.00           C  \\nATOM    648  OD1 ASP A  46      -0.159   0.038   1.551  1.00  0.00           O  \\nATOM    649  OD2 ASP A  46       0.755  -0.552   3.377  1.00  0.00           O  \\nATOM    650  H   ASP A  46       1.946   1.850   1.153  1.00  0.00           H  \\nATOM    651  HA  ASP A  46      -0.733   2.480   1.402  1.00  0.00           H  \\nATOM    652  HB3 ASP A  46       1.251   1.929   3.631  1.00  0.00           H  \\nATOM    653  HB2 ASP A  46      -0.498   1.889   3.810  1.00  0.00           H  \\nATOM    654  N   CYS A  47       1.146   4.713   2.990  1.00  0.00           N  \\nATOM    655  CA  CYS A  47       1.184   6.019   3.630  1.00  0.00           C  \\nATOM    656  C   CYS A  47       0.864   7.149   2.647  1.00  0.00           C  \\nATOM    657  O   CYS A  47       0.121   8.066   3.002  1.00  0.00           O  \\nATOM    658  CB  CYS A  47       2.563   6.206   4.256  1.00  0.00           C  \\nATOM    659  SG  CYS A  47       2.895   5.094   5.649  1.00  0.00           S  \\nATOM    660  H   CYS A  47       1.992   4.163   2.960  1.00  0.00           H  \\nATOM    661  HA  CYS A  47       0.434   6.040   4.421  1.00  0.00           H  \\nATOM    662  HB3 CYS A  47       3.319   6.039   3.488  1.00  0.00           H  \\nATOM    663  HB2 CYS A  47       2.648   7.236   4.604  1.00  0.00           H  \\nATOM    664  N   TRP A  48       1.356   7.082   1.405  1.00  0.00           N  \\nATOM    665  CA  TRP A  48       0.961   8.019   0.354  1.00  0.00           C  \\nATOM    666  C   TRP A  48      -0.534   7.922   0.031  1.00  0.00           C  \\nATOM    667  O   TRP A  48      -1.213   8.949  -0.044  1.00  0.00           O  \\nATOM    668  CB  TRP A  48       1.775   7.760  -0.915  1.00  0.00           C  \\nATOM    669  CG  TRP A  48       1.386   8.639  -2.065  1.00  0.00           C  \\nATOM    670  CD1 TRP A  48       1.744   9.940  -2.254  1.00  0.00           C  \\nATOM    671  CD2 TRP A  48       0.559   8.282  -3.199  1.00  0.00           C  \\nATOM    672  NE1 TRP A  48       1.199  10.417  -3.420  1.00  0.00           N  \\nATOM    673  CE2 TRP A  48       0.471   9.415  -4.012  1.00  0.00           C  \\nATOM    674  CE3 TRP A  48      -0.104   7.110  -3.583  1.00  0.00           C  \\nATOM    675  CZ2 TRP A  48      -0.256   9.417  -5.192  1.00  0.00           C  \\nATOM    676  CZ3 TRP A  48      -0.832   7.111  -4.767  1.00  0.00           C  \\nATOM    677  CH2 TRP A  48      -0.905   8.235  -5.551  1.00  0.00           C  \\nATOM    678  H   TRP A  48       1.979   6.323   1.168  1.00  0.00           H  \\nATOM    679  HA  TRP A  48       1.155   9.036   0.695  1.00  0.00           H  \\nATOM    680  HB3 TRP A  48       2.833   7.916  -0.707  1.00  0.00           H  \\nATOM    681  HB2 TRP A  48       1.651   6.721  -1.221  1.00  0.00           H  \\nATOM    682  HD1 TRP A  48       2.372  10.518  -1.577  1.00  0.00           H  \\nATOM    683  HE1 TRP A  48       1.312  11.352  -3.783  1.00  0.00           H  \\nATOM    684  HE3 TRP A  48      -0.047   6.214  -2.966  1.00  0.00           H  \\nATOM    685  HZ2 TRP A  48      -0.326  10.300  -5.828  1.00  0.00           H  \\nATOM    686  HZ3 TRP A  48      -1.345   6.195  -5.059  1.00  0.00           H  \\nATOM    687  HH2 TRP A  48      -1.484   8.202  -6.473  1.00  0.00           H  \\nATOM    688  N   ALA A  49      -1.068   6.709  -0.127  1.00  0.00           N  \\nATOM    689  CA  ALA A  49      -2.484   6.489  -0.396  1.00  0.00           C  \\nATOM    690  C   ALA A  49      -3.367   7.139   0.686  1.00  0.00           C  \\nATOM    691  O   ALA A  49      -4.376   7.779   0.373  1.00  0.00           O  \\nATOM    692  CB  ALA A  49      -2.722   4.978  -0.514  1.00  0.00           C  \\nATOM    693  H   ALA A  49      -0.464   5.902  -0.051  1.00  0.00           H  \\nATOM    694  HA  ALA A  49      -2.734   6.953  -1.351  1.00  0.00           H  \\nATOM    695  HB1 ALA A  49      -3.784   4.774  -0.648  1.00  0.00           H  \\nATOM    696  HB2 ALA A  49      -2.165   4.580  -1.362  1.00  0.00           H  \\nATOM    697  HB3 ALA A  49      -2.390   4.466   0.389  1.00  0.00           H  \\nATOM    698  N   ILE A  50      -2.958   7.058   1.955  1.00  0.00           N  \\nATOM    699  CA  ILE A  50      -3.655   7.699   3.078  1.00  0.00           C  \\nATOM    700  C   ILE A  50      -3.441   9.217   3.045  1.00  0.00           C  \\nATOM    701  O   ILE A  50      -4.397   9.996   3.066  1.00  0.00           O  \\nATOM    702  CB  ILE A  50      -3.171   7.138   4.427  1.00  0.00           C  \\nATOM    703  CG1 ILE A  50      -3.529   5.654   4.547  1.00  0.00           C  \\nATOM    704  CG2 ILE A  50      -3.772   7.929   5.579  1.00  0.00           C  \\nATOM    705  CD1 ILE A  50      -2.830   4.947   5.686  1.00  0.00           C  \\nATOM    706  H   ILE A  50      -2.146   6.490   2.151  1.00  0.00           H  \\nATOM    707  HA  ILE A  50      -4.727   7.515   2.989  1.00  0.00           H  \\nATOM    708  HB  ILE A  50      -2.085   7.205   4.480  1.00  0.00           H  \\nATOM    709 HG13 ILE A  50      -4.605   5.550   4.690  1.00  0.00           H  \\nATOM    710 HG12 ILE A  50      -3.272   5.141   3.620  1.00  0.00           H  \\nATOM    711 HG21 ILE A  50      -3.419   7.518   6.525  1.00  0.00           H  \\nATOM    712 HG22 ILE A  50      -3.468   8.972   5.501  1.00  0.00           H  \\nATOM    713 HG23 ILE A  50      -4.859   7.863   5.537  1.00  0.00           H  \\nATOM    714 HD11 ILE A  50      -3.135   3.901   5.706  1.00  0.00           H  \\nATOM    715 HD12 ILE A  50      -1.751   5.009   5.544  1.00  0.00           H  \\nATOM    716 HD13 ILE A  50      -3.101   5.421   6.628  1.00  0.00           H  \\nATOM    717  N   LEU A  51      -2.181   9.658   2.980  1.00  0.00           N  \\nATOM    718  CA  LEU A  51      -1.821  11.064   3.138  1.00  0.00           C  \\nATOM    719  C   LEU A  51      -2.303  11.923   1.963  1.00  0.00           C  \\nATOM    720  O   LEU A  51      -2.836  13.010   2.179  1.00  0.00           O  \\nATOM    721  CB  LEU A  51      -0.300  11.198   3.283  1.00  0.00           C  \\nATOM    722  CG  LEU A  51       0.225  12.623   3.496  1.00  0.00           C  \\nATOM    723  CD1 LEU A  51      -0.355  13.192   4.783  1.00  0.00           C  \\nATOM    724  CD2 LEU A  51       1.746  12.603   3.543  1.00  0.00           C  \\nATOM    725  H   LEU A  51      -1.437   8.977   2.930  1.00  0.00           H  \\nATOM    726  HA  LEU A  51      -2.315  11.422   4.041  1.00  0.00           H  \\nATOM    727  HB3 LEU A  51       0.023  10.596   4.131  1.00  0.00           H  \\nATOM    728  HB2 LEU A  51       0.171  10.802   2.383  1.00  0.00           H  \\nATOM    729  HG  LEU A  51      -0.103  13.259   2.673  1.00  0.00           H  \\nATOM    730 HD11 LEU A  51       0.018  14.205   4.935  1.00  0.00           H  \\nATOM    731 HD12 LEU A  51      -1.443  13.213   4.714  1.00  0.00           H  \\nATOM    732 HD13 LEU A  51      -0.057  12.566   5.624  1.00  0.00           H  \\nATOM    733 HD21 LEU A  51       2.119  13.616   3.694  1.00  0.00           H  \\nATOM    734 HD22 LEU A  51       2.075  11.968   4.367  1.00  0.00           H  \\nATOM    735 HD23 LEU A  51       2.134  12.209   2.604  1.00  0.00           H  \\nATOM    736  N   CYS A  52      -2.113  11.457   0.732  1.00  0.00           N  \\nATOM    737  CA  CYS A  52      -2.268  12.273  -0.469  1.00  0.00           C  \\nATOM    738  C   CYS A  52      -3.500  11.919  -1.306  1.00  0.00           C  \\nATOM    739  O   CYS A  52      -3.990  12.776  -2.038  1.00  0.00           O  \\nATOM    740  CB  CYS A  52      -1.025  12.139  -1.349  1.00  0.00           C  \\nATOM    741  SG  CYS A  52       0.516  12.608  -0.526  1.00  0.00           S  \\nATOM    742  H   CYS A  52      -1.675  10.553   0.632  1.00  0.00           H  \\nATOM    743  HA  CYS A  52      -2.387  13.318  -0.185  1.00  0.00           H  \\nATOM    744  HB3 CYS A  52      -0.927  11.107  -1.686  1.00  0.00           H  \\nATOM    745  HB2 CYS A  52      -1.138  12.763  -2.235  1.00  0.00           H  \\nATOM    746  N   LYS A  53      -4.013  10.685  -1.213  1.00  0.00           N  \\nATOM    747  CA  LYS A  53      -5.196  10.239  -1.972  1.00  0.00           C  \\nATOM    748  C   LYS A  53      -6.451  10.085  -1.116  1.00  0.00           C  \\nATOM    749  O   LYS A  53      -7.524   9.889  -1.675  1.00  0.00           O  \\nATOM    750  CB  LYS A  53      -4.898   8.914  -2.674  1.00  0.00           C  \\nATOM    751  CG  LYS A  53      -3.748   8.974  -3.670  1.00  0.00           C  \\nATOM    752  CD  LYS A  53      -4.073   9.900  -4.833  1.00  0.00           C  \\nATOM    753  CE  LYS A  53      -5.235   9.367  -5.658  1.00  0.00           C  \\nATOM    754  NZ  LYS A  53      -5.387  10.101  -6.944  1.00  0.00           N  \\nATOM    755  H   LYS A  53      -3.538  10.008  -0.633  1.00  0.00           H  \\nATOM    756  HA  LYS A  53      -5.471  11.011  -2.691  1.00  0.00           H  \\nATOM    757  HB3 LYS A  53      -4.657   8.154  -1.930  1.00  0.00           H  \\nATOM    758  HB2 LYS A  53      -5.787   8.577  -3.208  1.00  0.00           H  \\nATOM    759  HG3 LYS A  53      -2.850   9.336  -3.168  1.00  0.00           H  \\nATOM    760  HG2 LYS A  53      -3.549   7.975  -4.057  1.00  0.00           H  \\nATOM    761  HD3 LYS A  53      -4.333  10.888  -4.450  1.00  0.00           H  \\nATOM    762  HD2 LYS A  53      -3.198   9.997  -5.476  1.00  0.00           H  \\nATOM    763  HE3 LYS A  53      -5.074   8.312  -5.874  1.00  0.00           H  \\nATOM    764  HE2 LYS A  53      -6.159   9.463  -5.089  1.00  0.00           H  \\nATOM    765  HZ1 LYS A  53      -6.166   9.719  -7.460  1.00  0.00           H  \\nATOM    766  HZ2 LYS A  53      -5.555  11.080  -6.756  1.00  0.00           H  \\nATOM    767  HZ3 LYS A  53      -4.543  10.004  -7.490  1.00  0.00           H  \\nATOM    768  N   GLY A  54      -6.335  10.206   0.209  1.00  0.00           N  \\nATOM    769  CA  GLY A  54      -7.479  10.146   1.118  1.00  0.00           C  \\nATOM    770  C   GLY A  54      -7.973   8.722   1.362  1.00  0.00           C  \\nATOM    771  O   GLY A  54      -9.132   8.536   1.728  1.00  0.00           O  \\nATOM    772  H   GLY A  54      -5.409  10.247   0.613  1.00  0.00           H  \\nATOM    773  HA3 GLY A  54      -8.303  10.733   0.713  1.00  0.00           H  \\nATOM    774  HA2 GLY A  54      -7.185  10.565   2.081  1.00  0.00           H  \\nATOM    775  N   ALA A  55      -7.122   7.715   1.130  1.00  0.00           N  \\nATOM    776  CA  ALA A  55      -7.490   6.325   1.356  1.00  0.00           C  \\nATOM    777  C   ALA A  55      -7.851   6.093   2.840  1.00  0.00           C  \\nATOM    778  O   ALA A  55      -7.234   6.698   3.725  1.00  0.00           O  \\nATOM    779  CB  ALA A  55      -6.352   5.417   0.879  1.00  0.00           C  \\nATOM    780  H   ALA A  55      -6.159   7.924   0.907  1.00  0.00           H  \\nATOM    781  HA  ALA A  55      -8.370   6.119   0.747  1.00  0.00           H  \\nATOM    782  HB1 ALA A  55      -6.082   5.668  -0.147  1.00  0.00           H  \\nATOM    783  HB2 ALA A  55      -5.484   5.537   1.528  1.00  0.00           H  \\nATOM    784  HB3 ALA A  55      -6.668   4.374   0.912  1.00  0.00           H  \\nATOM    785  N   PRO A  56      -8.842   5.232   3.133  1.00  0.00           N  \\nATOM    786  CA  PRO A  56      -9.236   4.942   4.506  1.00  0.00           C  \\nATOM    787  C   PRO A  56      -8.031   4.493   5.336  1.00  0.00           C  \\nATOM    788  O   PRO A  56      -7.273   3.638   4.899  1.00  0.00           O  \\nATOM    789  CB  PRO A  56     -10.263   3.817   4.340  1.00  0.00           C  \\nATOM    790  CG  PRO A  56     -10.857   4.060   2.995  1.00  0.00           C  \\nATOM    791  CD  PRO A  56      -9.698   4.516   2.150  1.00  0.00           C  \\nATOM    792  HA  PRO A  56      -9.673   5.833   4.959  1.00  0.00           H  \\nATOM    793  HB3 PRO A  56      -9.765   2.840   4.417  1.00  0.00           H  \\nATOM    794  HB2 PRO A  56     -11.006   3.864   5.150  1.00  0.00           H  \\nATOM    795  HG3 PRO A  56     -11.323   3.140   2.614  1.00  0.00           H  \\nATOM    796  HG2 PRO A  56     -11.655   4.815   3.063  1.00  0.00           H  \\nATOM    797  HD3 PRO A  56      -9.186   3.640   1.726  1.00  0.00           H  \\nATOM    798  HD2 PRO A  56     -10.063   5.177   1.351  1.00  0.00           H  \\nATOM    799  N   ASN A  57      -7.889   5.037   6.546  1.00  0.00           N  \\nATOM    800  CA  ASN A  57      -6.860   4.645   7.518  1.00  0.00           C  \\nATOM    801  C   ASN A  57      -7.417   3.912   8.751  1.00  0.00           C  \\nATOM    802  O   ASN A  57      -6.652   3.518   9.629  1.00  0.00           O  \\nATOM    803  CB  ASN A  57      -6.062   5.894   7.922  1.00  0.00           C  \\nATOM    804  CG  ASN A  57      -6.897   6.964   8.609  1.00  0.00           C  \\nATOM    805  OD1 ASN A  57      -8.115   6.924   8.664  1.00  0.00           O  \\nATOM    806  ND2 ASN A  57      -6.262   7.984   9.134  1.00  0.00           N  \\nATOM    807  H   ASN A  57      -8.482   5.816   6.798  1.00  0.00           H  \\nATOM    808  HA  ASN A  57      -6.168   3.954   7.037  1.00  0.00           H  \\nATOM    809  HB3 ASN A  57      -5.609   6.328   7.030  1.00  0.00           H  \\nATOM    810  HB2 ASN A  57      -5.262   5.592   8.596  1.00  0.00           H  \\nATOM    811 HD21 ASN A  57      -6.842   8.689   9.565  1.00  0.00           H  \\nATOM    812 HD22 ASN A  57      -5.259   8.061   9.044  1.00  0.00           H  \\nATOM    813  N   VAL A  58      -8.737   3.726   8.816  1.00  0.00           N  \\nATOM    814  CA  VAL A  58      -9.416   2.928   9.838  1.00  0.00           C  \\nATOM    815  C   VAL A  58     -10.046   1.731   9.143  1.00  0.00           C  \\nATOM    816  O   VAL A  58     -10.973   1.879   8.343  1.00  0.00           O  \\nATOM    817  CB  VAL A  58     -10.497   3.750  10.565  1.00  0.00           C  \\nATOM    818  CG1 VAL A  58     -11.207   2.895  11.604  1.00  0.00           C  \\nATOM    819  CG2 VAL A  58      -9.869   4.975  11.211  1.00  0.00           C  \\nATOM    820  H   VAL A  58      -9.293   4.069   8.046  1.00  0.00           H  \\nATOM    821  HA  VAL A  58      -8.681   2.564  10.556  1.00  0.00           H  \\nATOM    822  HB  VAL A  58     -11.249   4.065   9.841  1.00  0.00           H  \\nATOM    823 HG11 VAL A  58     -11.968   3.492  12.108  1.00  0.00           H  \\nATOM    824 HG12 VAL A  58     -11.679   2.044  11.114  1.00  0.00           H  \\nATOM    825 HG13 VAL A  58     -10.483   2.538  12.337  1.00  0.00           H  \\nATOM    826 HG21 VAL A  58     -10.640   5.552  11.723  1.00  0.00           H  \\nATOM    827 HG22 VAL A  58      -9.114   4.660  11.931  1.00  0.00           H  \\nATOM    828 HG23 VAL A  58      -9.403   5.593  10.444  1.00  0.00           H  \\nATOM    829  N   LEU A  59      -9.510   0.550   9.424  1.00  0.00           N  \\nATOM    830  CA  LEU A  59      -9.762  -0.671   8.672  1.00  0.00           C  \\nATOM    831  C   LEU A  59     -10.378  -1.748   9.562  1.00  0.00           C  \\nATOM    832  O   LEU A  59     -10.216  -1.730  10.779  1.00  0.00           O  \\nATOM    833  CB  LEU A  59      -8.457  -1.190   8.055  1.00  0.00           C  \\nATOM    834  CG  LEU A  59      -7.689  -0.189   7.182  1.00  0.00           C  \\nATOM    835  CD1 LEU A  59      -6.380  -0.815   6.722  1.00  0.00           C  \\nATOM    836  CD2 LEU A  59      -8.550   0.214   5.994  1.00  0.00           C  \\nATOM    837  H   LEU A  59      -8.823   0.504  10.163  1.00  0.00           H  \\nATOM    838  HA  LEU A  59     -10.470  -0.461   7.871  1.00  0.00           H  \\nATOM    839  HB3 LEU A  59      -7.794  -1.505   8.859  1.00  0.00           H  \\nATOM    840  HB2 LEU A  59      -8.686  -2.060   7.438  1.00  0.00           H  \\nATOM    841  HG  LEU A  59      -7.447   0.696   7.771  1.00  0.00           H  \\nATOM    842 HD11 LEU A  59      -5.834  -0.103   6.102  1.00  0.00           H  \\nATOM    843 HD12 LEU A  59      -5.776  -1.075   7.591  1.00  0.00           H  \\nATOM    844 HD13 LEU A  59      -6.590  -1.713   6.143  1.00  0.00           H  \\nATOM    845 HD21 LEU A  59      -8.005   0.926   5.374  1.00  0.00           H  \\nATOM    846 HD22 LEU A  59      -8.792  -0.670   5.404  1.00  0.00           H  \\nATOM    847 HD23 LEU A  59      -9.471   0.674   6.351  1.00  0.00           H  \\nATOM    848  N   GLN A  60     -11.044  -2.723   8.949  1.00  0.00           N  \\nATOM    849  CA  GLN A  60     -11.561  -3.898   9.653  1.00  0.00           C  \\nATOM    850  C   GLN A  60     -11.051  -5.164   8.972  1.00  0.00           C  \\nATOM    851  O   GLN A  60     -11.257  -5.337   7.774  1.00  0.00           O  \\nATOM    852  CB  GLN A  60     -13.092  -3.844   9.690  1.00  0.00           C  \\nATOM    853  CG  GLN A  60     -13.659  -4.775  10.770  1.00  0.00           C  \\nATOM    854  CD  GLN A  60     -15.182  -4.759  10.813  1.00  0.00           C  \\nATOM    855  OE1 GLN A  60     -15.862  -4.405   9.865  1.00  0.00           O  \\nATOM    856  NE2 GLN A  60     -15.793  -5.155  11.909  1.00  0.00           N  \\nATOM    857  H   GLN A  60     -11.156  -2.667   7.948  1.00  0.00           H  \\nATOM    858  HA  GLN A  60     -11.194  -3.893  10.680  1.00  0.00           H  \\nATOM    859  HB3 GLN A  60     -13.493  -4.107   8.711  1.00  0.00           H  \\nATOM    860  HB2 GLN A  60     -13.412  -2.828   9.926  1.00  0.00           H  \\nATOM    861  HG3 GLN A  60     -13.293  -4.438  11.740  1.00  0.00           H  \\nATOM    862  HG2 GLN A  60     -13.320  -5.798  10.608  1.00  0.00           H  \\nATOM    863 HE21 GLN A  60     -16.801  -5.114  11.873  1.00  0.00           H  \\nATOM    864 HE22 GLN A  60     -15.294  -5.340  12.768  1.00  0.00           H  \\nATOM    865  N   ARG A  61     -10.363  -6.047   9.703  1.00  0.00           N  \\nATOM    866  CA  ARG A  61      -9.668  -7.196   9.100  1.00  0.00           C  \\nATOM    867  C   ARG A  61     -10.657  -8.279   8.666  1.00  0.00           C  \\nATOM    868  O   ARG A  61     -11.500  -8.682   9.455  1.00  0.00           O  \\nATOM    869  CB  ARG A  61      -8.614  -7.737  10.075  1.00  0.00           C  \\nATOM    870  CG  ARG A  61      -7.678  -8.756   9.412  1.00  0.00           C  \\nATOM    871  CD  ARG A  61      -6.748  -9.365  10.462  1.00  0.00           C  \\nATOM    872  NE  ARG A  61      -5.890 -10.404   9.869  1.00  0.00           N  \\nATOM    873  CZ  ARG A  61      -5.146 -11.265  10.540  1.00  0.00           C  \\nATOM    874  NH1 ARG A  61      -5.205 -11.357  11.841  1.00  0.00           N  \\nATOM    875  NH2 ARG A  61      -4.321 -12.053   9.911  1.00  0.00           N  \\nATOM    876  H   ARG A  61     -10.316  -5.911  10.703  1.00  0.00           H  \\nATOM    877  HA  ARG A  61      -9.154  -6.841   8.206  1.00  0.00           H  \\nATOM    878  HB3 ARG A  61      -8.016  -6.904  10.445  1.00  0.00           H  \\nATOM    879  HB2 ARG A  61      -9.118  -8.203  10.922  1.00  0.00           H  \\nATOM    880  HG3 ARG A  61      -8.258  -9.556   8.952  1.00  0.00           H  \\nATOM    881  HG2 ARG A  61      -7.082  -8.257   8.647  1.00  0.00           H  \\nATOM    882  HD3 ARG A  61      -7.356  -9.797  11.257  1.00  0.00           H  \\nATOM    883  HD2 ARG A  61      -6.126  -8.577  10.886  1.00  0.00           H  \\nATOM    884  HE  ARG A  61      -5.848 -10.441   8.861  1.00  0.00           H  \\nATOM    885 HH11 ARG A  61      -5.850 -10.767  12.346  1.00  0.00           H  \\nATOM    886 HH12 ARG A  61      -4.637 -12.026  12.340  1.00  0.00           H  \\nATOM    887 HH21 ARG A  61      -4.237 -12.000   8.906  1.00  0.00           H  \\nATOM    888 HH22 ARG A  61      -3.685 -12.645  10.427  1.00  0.00           H  \\nATOM    889  N   VAL A  62     -10.522  -8.812   7.457  1.00  0.00           N  \\nATOM    890  CA  VAL A  62     -11.429  -9.831   6.905  1.00  0.00           C  \\nATOM    891  C   VAL A  62     -10.750 -11.185   6.725  1.00  0.00           C  \\nATOM    892  O   VAL A  62      -9.524 -11.302   6.760  1.00  0.00           O  \\nATOM    893  CB  VAL A  62     -12.102  -9.368   5.602  1.00  0.00           C  \\nATOM    894  CG1 VAL A  62     -12.973  -8.140   5.868  1.00  0.00           C  \\nATOM    895  CG2 VAL A  62     -11.107  -9.066   4.477  1.00  0.00           C  \\nATOM    896  H   VAL A  62      -9.741  -8.512   6.893  1.00  0.00           H  \\nATOM    897  HA  VAL A  62     -12.242  -9.994   7.613  1.00  0.00           H  \\nATOM    898  HB  VAL A  62     -12.770 -10.163   5.271  1.00  0.00           H  \\nATOM    899 HG11 VAL A  62     -12.364  -7.320   6.247  1.00  0.00           H  \\nATOM    900 HG12 VAL A  62     -13.471  -7.833   4.948  1.00  0.00           H  \\nATOM    901 HG13 VAL A  62     -13.727  -8.384   6.616  1.00  0.00           H  \\nATOM    902 HG21 VAL A  62     -10.469  -9.930   4.286  1.00  0.00           H  \\nATOM    903 HG22 VAL A  62     -10.493  -8.207   4.744  1.00  0.00           H  \\nATOM    904 HG23 VAL A  62     -11.646  -8.820   3.562  1.00  0.00           H  \\nATOM    905  N   ASN A  63     -11.568 -12.219   6.540  1.00  0.00           N  \\nATOM    906  CA  ASN A  63     -11.124 -13.594   6.348  1.00  0.00           C  \\nATOM    907  C   ASN A  63     -10.470 -13.829   4.972  1.00  0.00           C  \\nATOM    908  O   ASN A  63     -10.441 -12.965   4.094  1.00  0.00           O  \\nATOM    909  CB  ASN A  63     -12.291 -14.543   6.550  1.00  0.00           C  \\nATOM    910  CG  ASN A  63     -13.321 -14.432   5.461  1.00  0.00           C  \\nATOM    911  OD1 ASN A  63     -13.125 -13.712   4.475  1.00  0.00           O  \\nATOM    912  ND2 ASN A  63     -14.416 -15.131   5.620  1.00  0.00           N  \\nATOM    913  H   ASN A  63     -12.562 -12.036   6.539  1.00  0.00           H  \\nATOM    914  HA  ASN A  63     -10.357 -13.794   7.096  1.00  0.00           H  \\nATOM    915  HB3 ASN A  63     -11.923 -15.570   6.583  1.00  0.00           H  \\nATOM    916  HB2 ASN A  63     -12.768 -14.335   7.508  1.00  0.00           H  \\nATOM    917 HD21 ASN A  63     -15.137 -15.096   4.927  1.00  0.00           H  \\nATOM    918 HD22 ASN A  63     -14.533 -15.700   6.433  1.00  0.00           H  \\nATOM    919  N   GLU A  64      -9.997 -15.058   4.766  1.00  0.00           N  \\nATOM    920  CA  GLU A  64      -9.261 -15.468   3.566  1.00  0.00           C  \\nATOM    921  C   GLU A  64     -10.100 -15.410   2.280  1.00  0.00           C  \\nATOM    922  O   GLU A  64      -9.631 -14.938   1.246  1.00  0.00           O  \\nATOM    923  CB  GLU A  64      -8.753 -16.900   3.806  1.00  0.00           C  \\nATOM    924  CG  GLU A  64      -7.922 -17.466   2.648  1.00  0.00           C  \\nATOM    925  CD  GLU A  64      -6.577 -16.752   2.489  1.00  0.00           C  \\nATOM    926  OE1 GLU A  64      -5.991 -16.811   1.385  1.00  0.00           O  \\nATOM    927  OE2 GLU A  64      -6.113 -16.083   3.444  1.00  0.00           O  \\nATOM    928  H   GLU A  64     -10.006 -15.704   5.543  1.00  0.00           H  \\nATOM    929  HA  GLU A  64      -8.412 -14.795   3.437  1.00  0.00           H  \\nATOM    930  HB3 GLU A  64      -8.150 -16.919   4.713  1.00  0.00           H  \\nATOM    931  HB2 GLU A  64      -9.608 -17.556   3.972  1.00  0.00           H  \\nATOM    932  HG3 GLU A  64      -8.496 -17.394   1.724  1.00  0.00           H  \\nATOM    933  HG2 GLU A  64      -7.740 -18.523   2.842  1.00  0.00           H  \\nATOM    934  N   LYS A  65     -11.364 -15.847   2.347  1.00  0.00           N  \\nATOM    935  CA  LYS A  65     -12.282 -15.819   1.201  1.00  0.00           C  \\nATOM    936  C   LYS A  65     -12.479 -14.392   0.687  1.00  0.00           C  \\nATOM    937  O   LYS A  65     -12.390 -14.152  -0.515  1.00  0.00           O  \\nATOM    938  CB  LYS A  65     -13.632 -16.431   1.577  1.00  0.00           C  \\nATOM    939  CG  LYS A  65     -14.640 -16.482   0.436  1.00  0.00           C  \\nATOM    940  CD  LYS A  65     -15.928 -17.166   0.866  1.00  0.00           C  \\nATOM    941  CE  LYS A  65     -16.947 -17.191  -0.264  1.00  0.00           C  \\nATOM    942  NZ  LYS A  65     -18.209 -17.869   0.140  1.00  0.00           N  \\nATOM    943  H   LYS A  65     -11.693 -16.195   3.237  1.00  0.00           H  \\nATOM    944  HA  LYS A  65     -11.838 -16.383   0.381  1.00  0.00           H  \\nATOM    945  HB3 LYS A  65     -13.482 -17.450   1.936  1.00  0.00           H  \\nATOM    946  HB2 LYS A  65     -14.077 -15.859   2.391  1.00  0.00           H  \\nATOM    947  HG3 LYS A  65     -14.869 -15.467   0.108  1.00  0.00           H  \\nATOM    948  HG2 LYS A  65     -14.211 -17.028  -0.404  1.00  0.00           H  \\nATOM    949  HD3 LYS A  65     -15.712 -18.190   1.173  1.00  0.00           H  \\nATOM    950  HD2 LYS A  65     -16.356 -16.634   1.716  1.00  0.00           H  \\nATOM    951  HE3 LYS A  65     -17.178 -16.172  -0.568  1.00  0.00           H  \\nATOM    952  HE2 LYS A  65     -16.526 -17.716  -1.121  1.00  0.00           H  \\nATOM    953  HZ1 LYS A  65     -18.857 -17.865  -0.636  1.00  0.00           H  \\nATOM    954  HZ2 LYS A  65     -18.009 -18.823   0.407  1.00  0.00           H  \\nATOM    955  HZ3 LYS A  65     -18.619 -17.380   0.922  1.00  0.00           H  \\nATOM    956  N   THR A  66     -12.720 -13.450   1.598  1.00  0.00           N  \\nATOM    957  CA  THR A  66     -12.908 -12.041   1.242  1.00  0.00           C  \\nATOM    958  C   THR A  66     -11.611 -11.440   0.723  1.00  0.00           C  \\nATOM    959  O   THR A  66     -11.626 -10.795  -0.317  1.00  0.00           O  \\nATOM    960  CB  THR A  66     -13.413 -11.220   2.443  1.00  0.00           C  \\nATOM    961  OG1 THR A  66     -14.621 -11.803   2.949  1.00  0.00           O  \\nATOM    962  CG2 THR A  66     -13.683  -9.781   2.031  1.00  0.00           C  \\nATOM    963  H   THR A  66     -12.751 -13.707   2.574  1.00  0.00           H  \\nATOM    964  HA  THR A  66     -13.655 -11.964   0.452  1.00  0.00           H  \\nATOM    965  HB  THR A  66     -12.662 -11.231   3.233  1.00  0.00           H  \\nATOM    966  HG1 THR A  66     -14.406 -12.576   3.477  1.00  0.00           H  \\nATOM    967 HG21 THR A  66     -14.039  -9.217   2.892  1.00  0.00           H  \\nATOM    968 HG22 THR A  66     -12.764  -9.332   1.656  1.00  0.00           H  \\nATOM    969 HG23 THR A  66     -14.441  -9.763   1.248  1.00  0.00           H  \\nATOM    970  N   LYS A  67     -10.473 -11.719   1.373  1.00  0.00           N  \\nATOM    971  CA  LYS A  67      -9.158 -11.263   0.910  1.00  0.00           C  \\nATOM    972  C   LYS A  67      -8.898 -11.650  -0.543  1.00  0.00           C  \\nATOM    973  O   LYS A  67      -8.531 -10.798  -1.344  1.00  0.00           O  \\nATOM    974  CB  LYS A  67      -8.069 -11.818   1.838  1.00  0.00           C  \\nATOM    975  CG  LYS A  67      -6.653 -11.377   1.426  1.00  0.00           C  \\nATOM    976  CD  LYS A  67      -5.570 -12.250   2.064  1.00  0.00           C  \\nATOM    977  CE  LYS A  67      -5.505 -13.583   1.317  1.00  0.00           C  \\nATOM    978  NZ  LYS A  67      -4.546 -14.528   1.923  1.00  0.00           N  \\nATOM    979  H   LYS A  67     -10.524 -12.252   2.229  1.00  0.00           H  \\nATOM    980  HA  LYS A  67      -9.148 -10.173   0.944  1.00  0.00           H  \\nATOM    981  HB3 LYS A  67      -8.257 -11.478   2.858  1.00  0.00           H  \\nATOM    982  HB2 LYS A  67      -8.124 -12.907   1.830  1.00  0.00           H  \\nATOM    983  HG3 LYS A  67      -6.530 -11.409   0.343  1.00  0.00           H  \\nATOM    984  HG2 LYS A  67      -6.501 -10.346   1.748  1.00  0.00           H  \\nATOM    985  HD3 LYS A  67      -4.610 -11.740   1.979  1.00  0.00           H  \\nATOM    986  HD2 LYS A  67      -5.798 -12.412   3.118  1.00  0.00           H  \\nATOM    987  HE3 LYS A  67      -6.491 -14.048   1.331  1.00  0.00           H  \\nATOM    988  HE2 LYS A  67      -5.260 -13.412   0.269  1.00  0.00           H  \\nATOM    989  HZ1 LYS A  67      -4.678 -15.414   1.455  1.00  0.00           H  \\nATOM    990  HZ2 LYS A  67      -3.580 -14.236   1.941  1.00  0.00           H  \\nATOM    991  HZ3 LYS A  67      -4.895 -14.798   2.831  1.00  0.00           H  \\nATOM    992  N   ASN A  68      -9.117 -12.913  -0.903  1.00  0.00           N  \\nATOM    993  CA  ASN A  68      -8.921 -13.371  -2.278  1.00  0.00           C  \\nATOM    994  C   ASN A  68      -9.882 -12.677  -3.252  1.00  0.00           C  \\nATOM    995  O   ASN A  68      -9.439 -12.218  -4.300  1.00  0.00           O  \\nATOM    996  CB  ASN A  68      -8.999 -14.905  -2.300  1.00  0.00           C  \\nATOM    997  CG  ASN A  68      -7.769 -15.528  -1.655  1.00  0.00           C  \\nATOM    998  OD1 ASN A  68      -6.697 -14.938  -1.598  1.00  0.00           O  \\nATOM    999  ND2 ASN A  68      -7.869 -16.725  -1.133  1.00  0.00           N  \\nATOM   1000  H   ASN A  68      -9.364 -13.583  -0.188  1.00  0.00           H  \\nATOM   1001  HA  ASN A  68      -7.922 -13.081  -2.604  1.00  0.00           H  \\nATOM   1002  HB3 ASN A  68      -9.897 -15.237  -1.778  1.00  0.00           H  \\nATOM   1003  HB2 ASN A  68      -9.048 -15.250  -3.333  1.00  0.00           H  \\nATOM   1004 HD21 ASN A  68      -8.742 -17.232  -1.148  1.00  0.00           H  \\nATOM   1005 HD22 ASN A  68      -7.055 -17.033  -0.621  1.00  0.00           H  \\nATOM   1006  N   SER A  69     -11.144 -12.473  -2.857  1.00  0.00           N  \\nATOM   1007  CA  SER A  69     -12.095 -11.684  -3.653  1.00  0.00           C  \\nATOM   1008  C   SER A  69     -11.657 -10.223  -3.818  1.00  0.00           C  \\nATOM   1009  O   SER A  69     -11.785  -9.657  -4.901  1.00  0.00           O  \\nATOM   1010  CB  SER A  69     -13.466 -11.731  -3.007  1.00  0.00           C  \\nATOM   1011  OG  SER A  69     -13.986 -13.032  -3.024  1.00  0.00           O  \\nATOM   1012  H   SER A  69     -11.440 -12.844  -1.966  1.00  0.00           H  \\nATOM   1013  HA  SER A  69     -12.154 -12.123  -4.650  1.00  0.00           H  \\nATOM   1014  HB3 SER A  69     -13.396 -11.378  -1.978  1.00  0.00           H  \\nATOM   1015  HB2 SER A  69     -14.141 -11.060  -3.537  1.00  0.00           H  \\nATOM   1016  HG  SER A  69     -13.490 -13.524  -2.366  1.00  0.00           H  \\nATOM   1017  N   ASN A  70     -11.092  -9.607  -2.780  1.00  0.00           N  \\nATOM   1018  CA  ASN A  70     -10.568  -8.247  -2.860  1.00  0.00           C  \\nATOM   1019  C   ASN A  70      -9.391  -8.170  -3.838  1.00  0.00           C  \\nATOM   1020  O   ASN A  70      -9.339  -7.236  -4.627  1.00  0.00           O  \\nATOM   1021  CB  ASN A  70     -10.160  -7.751  -1.485  1.00  0.00           C  \\nATOM   1022  CG  ASN A  70     -11.339  -7.369  -0.635  1.00  0.00           C  \\nATOM   1023  OD1 ASN A  70     -12.436  -7.121  -1.149  1.00  0.00           O  \\nATOM   1024  ND2 ASN A  70     -11.136  -7.318   0.657  1.00  0.00           N  \\nATOM   1025  H   ASN A  70     -11.069 -10.081  -1.888  1.00  0.00           H  \\nATOM   1026  HA  ASN A  70     -11.347  -7.601  -3.265  1.00  0.00           H  \\nATOM   1027  HB3 ASN A  70      -9.593  -8.528  -0.972  1.00  0.00           H  \\nATOM   1028  HB2 ASN A  70      -9.507  -6.885  -1.590  1.00  0.00           H  \\nATOM   1029 HD21 ASN A  70     -11.885  -7.070   1.272  1.00  0.00           H  \\nATOM   1030 HD22 ASN A  70     -10.232  -7.528   1.029  1.00  0.00           H  \\nATOM   1031  N   ARG A  71      -8.507  -9.175  -3.871  1.00  0.00           N  \\nATOM   1032  CA  ARG A  71      -7.391  -9.219  -4.837  1.00  0.00           C  \\nATOM   1033  C   ARG A  71      -7.843  -9.309  -6.297  1.00  0.00           C  \\nATOM   1034  O   ARG A  71      -7.131  -8.851  -7.192  1.00  0.00           O  \\nATOM   1035  CB  ARG A  71      -6.487 -10.406  -4.537  1.00  0.00           C  \\nATOM   1036  CG  ARG A  71      -5.660 -10.275  -3.268  1.00  0.00           C  \\nATOM   1037  CD  ARG A  71      -4.765 -11.444  -3.072  1.00  0.00           C  \\nATOM   1038  NE  ARG A  71      -3.814 -11.588  -4.162  1.00  0.00           N  \\nATOM   1039  CZ  ARG A  71      -2.635 -10.940  -4.239  1.00  0.00           C  \\nATOM   1040  NH1 ARG A  71      -2.279 -10.109  -3.284  1.00  0.00           N  \\nATOM   1041  NH2 ARG A  71      -1.837 -11.139  -5.273  1.00  0.00           N  \\nATOM   1042  H   ARG A  71      -8.617  -9.930  -3.210  1.00  0.00           H  \\nATOM   1043  HA  ARG A  71      -6.818  -8.297  -4.755  1.00  0.00           H  \\nATOM   1044  HB3 ARG A  71      -7.090 -11.308  -4.445  1.00  0.00           H  \\nATOM   1045  HB2 ARG A  71      -5.797 -10.557  -5.367  1.00  0.00           H  \\nATOM   1046  HG3 ARG A  71      -5.043  -9.378  -3.327  1.00  0.00           H  \\nATOM   1047  HG2 ARG A  71      -6.324 -10.202  -2.406  1.00  0.00           H  \\nATOM   1048  HD3 ARG A  71      -4.206 -11.323  -2.145  1.00  0.00           H  \\nATOM   1049  HD2 ARG A  71      -5.363 -12.354  -3.019  1.00  0.00           H  \\nATOM   1050  HE  ARG A  71      -4.053 -12.219  -4.915  1.00  0.00           H  \\nATOM   1051 HH11 ARG A  71      -2.888  -9.956  -2.493  1.00  0.00           H  \\nATOM   1052 HH12 ARG A  71      -1.395  -9.623  -3.342  1.00  0.00           H  \\nATOM   1053 HH21 ARG A  71      -2.112 -11.778  -6.007  1.00  0.00           H  \\nATOM   1054 HH22 ARG A  71      -0.955 -10.653  -5.331  1.00  0.00           H  \\nATOM   1055  N   ASP A  72      -9.005  -9.905  -6.550  1.00  0.00           N  \\nATOM   1056  CA  ASP A  72      -9.632  -9.877  -7.874  1.00  0.00           C  \\nATOM   1057  C   ASP A  72     -10.218  -8.492  -8.172  1.00  0.00           C  \\nATOM   1058  O   ASP A  72      -9.948  -7.919  -9.230  1.00  0.00           O  \\nATOM   1059  CB  ASP A  72     -10.731 -10.938  -7.972  1.00  0.00           C  \\nATOM   1060  CG  ASP A  72     -10.180 -12.351  -8.107  1.00  0.00           C  \\nATOM   1061  OD1 ASP A  72      -9.049 -12.491  -8.508  1.00  0.00           O  \\nATOM   1062  OD2 ASP A  72     -10.896 -13.277  -7.808  1.00  0.00           O  \\nATOM   1063  H   ASP A  72      -9.471 -10.392  -5.798  1.00  0.00           H  \\nATOM   1064  HA  ASP A  72      -8.876 -10.075  -8.633  1.00  0.00           H  \\nATOM   1065  HB3 ASP A  72     -11.361 -10.893  -7.083  1.00  0.00           H  \\nATOM   1066  HB2 ASP A  72     -11.365 -10.727  -8.833  1.00  0.00           H  \\nATOM   1067  N   ARG A  73     -10.961  -7.923  -7.212  1.00  0.00           N  \\nATOM   1068  CA  ARG A  73     -11.616  -6.608  -7.339  1.00  0.00           C  \\nATOM   1069  C   ARG A  73     -10.638  -5.446  -7.505  1.00  0.00           C  \\nATOM   1070  O   ARG A  73     -10.949  -4.490  -8.210  1.00  0.00           O  \\nATOM   1071  CB  ARG A  73     -12.484  -6.337  -6.119  1.00  0.00           C  \\nATOM   1072  CG  ARG A  73     -13.739  -7.190  -6.025  1.00  0.00           C  \\nATOM   1073  CD  ARG A  73     -14.452  -6.980  -4.739  1.00  0.00           C  \\nATOM   1074  NE  ARG A  73     -15.646  -7.804  -4.639  1.00  0.00           N  \\nATOM   1075  CZ  ARG A  73     -16.339  -8.011  -3.503  1.00  0.00           C  \\nATOM   1076  NH1 ARG A  73     -15.947  -7.449  -2.381  1.00  0.00           N  \\nATOM   1077  NH2 ARG A  73     -17.415  -8.779  -3.516  1.00  0.00           N  \\nATOM   1078  H   ARG A  73     -11.124  -8.465  -6.374  1.00  0.00           H  \\nATOM   1079  HA  ARG A  73     -12.232  -6.603  -8.238  1.00  0.00           H  \\nATOM   1080  HB3 ARG A  73     -11.903  -6.506  -5.214  1.00  0.00           H  \\nATOM   1081  HB2 ARG A  73     -12.795  -5.292  -6.118  1.00  0.00           H  \\nATOM   1082  HG3 ARG A  73     -14.417  -6.931  -6.839  1.00  0.00           H  \\nATOM   1083  HG2 ARG A  73     -13.469  -8.244  -6.099  1.00  0.00           H  \\nATOM   1084  HD3 ARG A  73     -13.791  -7.236  -3.911  1.00  0.00           H  \\nATOM   1085  HD2 ARG A  73     -14.750  -5.936  -4.655  1.00  0.00           H  \\nATOM   1086  HE  ARG A  73     -15.979  -8.253  -5.482  1.00  0.00           H  \\nATOM   1087 HH11 ARG A  73     -15.124  -6.862  -2.371  1.00  0.00           H  \\nATOM   1088 HH12 ARG A  73     -16.467  -7.604  -1.530  1.00  0.00           H  \\nATOM   1089 HH21 ARG A  73     -17.717  -9.211  -4.378  1.00  0.00           H  \\nATOM   1090 HH22 ARG A  73     -17.935  -8.934  -2.665  1.00  0.00           H  \\nATOM   1091  N   SER A  74      -9.470  -5.507  -6.875  1.00  0.00           N  \\nATOM   1092  CA  SER A  74      -8.403  -4.509  -7.029  1.00  0.00           C  \\nATOM   1093  C   SER A  74      -7.611  -4.681  -8.328  1.00  0.00           C  \\nATOM   1094  O   SER A  74      -6.836  -3.808  -8.710  1.00  0.00           O  \\nATOM   1095  CB  SER A  74      -7.437  -4.620  -5.852  1.00  0.00           C  \\nATOM   1096  OG  SER A  74      -6.713  -5.826  -5.948  1.00  0.00           O  \\nATOM   1097  H   SER A  74      -9.281  -6.299  -6.278  1.00  0.00           H  \\nATOM   1098  HA  SER A  74      -8.835  -3.508  -7.028  1.00  0.00           H  \\nATOM   1099  HB3 SER A  74      -7.984  -4.592  -4.910  1.00  0.00           H  \\nATOM   1100  HB2 SER A  74      -6.741  -3.782  -5.870  1.00  0.00           H  \\nATOM   1101  HG  SER A  74      -6.194  -5.885  -5.142  1.00  0.00           H  \\nATOM   1102  N   GLY A  75      -7.756  -5.823  -9.005  1.00  0.00           N  \\nATOM   1103  CA  GLY A  75      -6.926  -6.200 -10.141  1.00  0.00           C  \\nATOM   1104  C   GLY A  75      -5.553  -6.767  -9.762  1.00  0.00           C  \\nATOM   1105  O   GLY A  75      -4.807  -7.146 -10.665  1.00  0.00           O  \\nATOM   1106  H   GLY A  75      -8.431  -6.496  -8.669  1.00  0.00           H  \\nATOM   1107  HA3 GLY A  75      -6.773  -5.328 -10.777  1.00  0.00           H  \\nATOM   1108  HA2 GLY A  75      -7.460  -6.957 -10.714  1.00  0.00           H  \\nATOM   1109  N   ALA A  76      -5.201  -6.899  -8.478  1.00  0.00           N  \\nATOM   1110  CA  ALA A  76      -3.892  -7.411  -8.058  1.00  0.00           C  \\nATOM   1111  C   ALA A  76      -3.616  -8.827  -8.601  1.00  0.00           C  \\nATOM   1112  O   ALA A  76      -2.510  -9.108  -9.073  1.00  0.00           O  \\nATOM   1113  CB  ALA A  76      -3.809  -7.364  -6.530  1.00  0.00           C  \\nATOM   1114  H   ALA A  76      -5.864  -6.648  -7.758  1.00  0.00           H  \\nATOM   1115  HA  ALA A  76      -3.120  -6.754  -8.459  1.00  0.00           H  \\nATOM   1116  HB1 ALA A  76      -2.846  -7.745  -6.192  1.00  0.00           H  \\nATOM   1117  HB2 ALA A  76      -3.912  -6.333  -6.190  1.00  0.00           H  \\nATOM   1118  HB3 ALA A  76      -4.605  -7.961  -6.086  1.00  0.00           H  \\nATOM   1119  N   ASN A  77      -4.634  -9.695  -8.652  1.00  0.00           N  \\nATOM   1120  CA  ASN A  77      -4.500 -11.059  -9.187  1.00  0.00           C  \\nATOM   1121  C   ASN A  77      -4.252 -11.110 -10.706  1.00  0.00           C  \\nATOM   1122  O   ASN A  77      -3.873 -12.155 -11.235  1.00  0.00           O  \\nATOM   1123  CB  ASN A  77      -5.737 -11.885  -8.790  1.00  0.00           C  \\nATOM   1124  CG  ASN A  77      -5.675 -12.418  -7.368  1.00  0.00           C  \\nATOM   1125  OD1 ASN A  77      -4.635 -12.478  -6.728  1.00  0.00           O  \\nATOM   1126  ND2 ASN A  77      -6.791 -12.843  -6.838  1.00  0.00           N  \\nATOM   1127  H   ASN A  77      -5.518  -9.410  -8.256  1.00  0.00           H  \\nATOM   1128  HA  ASN A  77      -3.621 -11.520  -8.737  1.00  0.00           H  \\nATOM   1129  HB3 ASN A  77      -5.823 -12.754  -9.442  1.00  0.00           H  \\nATOM   1130  HB2 ASN A  77      -6.644 -11.294  -8.909  1.00  0.00           H  \\nATOM   1131 HD21 ASN A  77      -7.654 -12.769  -7.359  1.00  0.00           H  \\nATOM   1132 HD22 ASN A  77      -6.793 -13.193  -5.891  1.00  0.00           H  \\nATOM   1133  N   LYS A  78      -4.376  -9.981 -11.418  1.00  0.00           N  \\nATOM   1134  CA  LYS A  78      -4.076  -9.902 -12.859  1.00  0.00           C  \\nATOM   1135  C   LYS A  78      -2.590  -9.677 -13.135  1.00  0.00           C  \\nATOM   1136  O   LYS A  78      -2.168  -9.760 -14.286  1.00  0.00           O  \\nATOM   1137  CB  LYS A  78      -4.894  -8.786 -13.511  1.00  0.00           C  \\nATOM   1138  CG  LYS A  78      -6.403  -8.955 -13.384  1.00  0.00           C  \\nATOM   1139  CD  LYS A  78      -6.882 -10.201 -14.115  1.00  0.00           C  \\nATOM   1140  CE  LYS A  78      -8.399 -10.316 -14.075  1.00  0.00           C  \\nATOM   1141  NZ  LYS A  78      -8.881 -11.553 -14.749  1.00  0.00           N  \\nATOM   1142  H   LYS A  78      -4.589  -9.126 -10.923  1.00  0.00           H  \\nATOM   1143  HA  LYS A  78      -4.324 -10.857 -13.321  1.00  0.00           H  \\nATOM   1144  HB3 LYS A  78      -4.626  -7.829 -13.063  1.00  0.00           H  \\nATOM   1145  HB2 LYS A  78      -4.652  -8.730 -14.573  1.00  0.00           H  \\nATOM   1146  HG3 LYS A  78      -6.672  -9.036 -12.330  1.00  0.00           H  \\nATOM   1147  HG2 LYS A  78      -6.903  -8.083 -13.803  1.00  0.00           H  \\nATOM   1148  HD3 LYS A  78      -6.556 -10.161 -15.155  1.00  0.00           H  \\nATOM   1149  HD2 LYS A  78      -6.448 -11.085 -13.650  1.00  0.00           H  \\nATOM   1150  HE3 LYS A  78      -8.736 -10.327 -13.040  1.00  0.00           H  \\nATOM   1151  HE2 LYS A  78      -8.843  -9.453 -14.570  1.00  0.00           H  \\nATOM   1152  HZ1 LYS A  78      -9.889 -11.592 -14.701  1.00  0.00           H  \\nATOM   1153  HZ2 LYS A  78      -8.589 -11.546 -15.716  1.00  0.00           H  \\nATOM   1154  HZ3 LYS A  78      -8.491 -12.362 -14.287  1.00  0.00           H  \\nATOM   1155  N   GLY A  79      -1.802  -9.397 -12.093  1.00  0.00           N  \\nATOM   1156  CA  GLY A  79      -0.392  -9.038 -12.195  1.00  0.00           C  \\nATOM   1157  C   GLY A  79      -0.144  -7.863 -13.144  1.00  0.00           C  \\nATOM   1158  O   GLY A  79       0.587  -8.053 -14.117  1.00  0.00           O  \\nATOM   1159  H   GLY A  79      -2.216  -9.421 -11.172  1.00  0.00           H  \\nATOM   1160  HA3 GLY A  79       0.172  -9.899 -12.557  1.00  0.00           H  \\nATOM   1161  HA2 GLY A  79      -0.024  -8.768 -11.205  1.00  0.00           H  \\nATOM   1162  N   PRO A  80      -0.713  -6.664 -12.891  1.00  0.00           N  \\nATOM   1163  CA  PRO A  80      -0.565  -5.508 -13.781  1.00  0.00           C  \\nATOM   1164  C   PRO A  80       0.892  -5.181 -14.116  1.00  0.00           C  \\nATOM   1165  O   PRO A  80       1.208  -4.793 -15.234  1.00  0.00           O  \\nATOM   1166  CB  PRO A  80      -1.213  -4.382 -12.969  1.00  0.00           C  \\nATOM   1167  CG  PRO A  80      -2.269  -5.068 -12.171  1.00  0.00           C  \\nATOM   1168  CD  PRO A  80      -1.638  -6.371 -11.758  1.00  0.00           C  \\nATOM   1169  HA  PRO A  80      -1.096  -5.705 -14.713  1.00  0.00           H  \\nATOM   1170  HB3 PRO A  80      -0.457  -3.887 -12.342  1.00  0.00           H  \\nATOM   1171  HB2 PRO A  80      -1.620  -3.616 -13.646  1.00  0.00           H  \\nATOM   1172  HG3 PRO A  80      -2.558  -4.446 -11.311  1.00  0.00           H  \\nATOM   1173  HG2 PRO A  80      -3.174  -5.209 -12.780  1.00  0.00           H  \\nATOM   1174  HD3 PRO A  80      -1.090  -6.231 -10.815  1.00  0.00           H  \\nATOM   1175  HD2 PRO A  80      -2.420  -7.137 -11.643  1.00  0.00           H  \\nATOM   1176  N   PHE A  81       1.801  -5.421 -13.172  1.00  0.00           N  \\nATOM   1177  CA  PHE A  81       3.216  -5.078 -13.290  1.00  0.00           C  \\nATOM   1178  C   PHE A  81       4.100  -6.222 -13.815  1.00  0.00           C  \\nATOM   1179  O   PHE A  81       5.300  -6.026 -13.992  1.00  0.00           O  \\nATOM   1180  CB  PHE A  81       3.687  -4.551 -11.929  1.00  0.00           C  \\nATOM   1181  CG  PHE A  81       2.851  -3.405 -11.399  1.00  0.00           C  \\nATOM   1182  CD1 PHE A  81       2.887  -2.151 -12.032  1.00  0.00           C  \\nATOM   1183  CD2 PHE A  81       2.017  -3.598 -10.284  1.00  0.00           C  \\nATOM   1184  CE1 PHE A  81       2.072  -1.103 -11.571  1.00  0.00           C  \\nATOM   1185  CE2 PHE A  81       1.214  -2.545  -9.812  1.00  0.00           C  \\nATOM   1186  CZ  PHE A  81       1.235  -1.299 -10.460  1.00  0.00           C  \\nATOM   1187  H   PHE A  81       1.467  -5.726 -12.269  1.00  0.00           H  \\nATOM   1188  HA  PHE A  81       3.316  -4.267 -14.012  1.00  0.00           H  \\nATOM   1189  HB3 PHE A  81       3.655  -5.369 -11.210  1.00  0.00           H  \\nATOM   1190  HB2 PHE A  81       4.721  -4.213 -12.006  1.00  0.00           H  \\nATOM   1191  HD1 PHE A  81       3.548  -1.994 -12.871  1.00  0.00           H  \\nATOM   1192  HD2 PHE A  81       2.004  -4.555  -9.782  1.00  0.00           H  \\nATOM   1193  HE1 PHE A  81       2.096  -0.140 -12.060  1.00  0.00           H  \\nATOM   1194  HE2 PHE A  81       0.588  -2.679  -8.942  1.00  0.00           H  \\nATOM   1195  HZ  PHE A  81       0.624  -0.488 -10.095  1.00  0.00           H  \\nATOM   1196  N   LYS A  82       3.541  -7.412 -14.086  1.00  0.00           N  \\nATOM   1197  CA  LYS A  82       4.293  -8.521 -14.706  1.00  0.00           C  \\nATOM   1198  C   LYS A  82       4.459  -8.326 -16.212  1.00  0.00           C  \\nATOM   1199  O   LYS A  82       5.407  -8.857 -16.778  1.00  0.00           O  \\nATOM   1200  CB  LYS A  82       3.600  -9.857 -14.433  1.00  0.00           C  \\nATOM   1201  CG  LYS A  82       3.619 -10.288 -12.973  1.00  0.00           C  \\nATOM   1202  CD  LYS A  82       2.910 -11.621 -12.781  1.00  0.00           C  \\nATOM   1203  CE  LYS A  82       2.901 -12.038 -11.318  1.00  0.00           C  \\nATOM   1204  NZ  LYS A  82       2.174 -13.320 -11.109  1.00  0.00           N  \\nATOM   1205  H   LYS A  82       2.540  -7.498 -13.998  1.00  0.00           H  \\nATOM   1206  HA  LYS A  82       5.304  -8.546 -14.300  1.00  0.00           H  \\nATOM   1207  HB3 LYS A  82       2.559  -9.798 -14.751  1.00  0.00           H  \\nATOM   1208  HB2 LYS A  82       4.078 -10.641 -15.020  1.00  0.00           H  \\nATOM   1209  HG3 LYS A  82       4.651 -10.383 -12.634  1.00  0.00           H  \\nATOM   1210  HG2 LYS A  82       3.124  -9.532 -12.364  1.00  0.00           H  \\nATOM   1211  HD3 LYS A  82       1.881 -11.540 -13.134  1.00  0.00           H  \\nATOM   1212  HD2 LYS A  82       3.417 -12.389 -13.364  1.00  0.00           H  \\nATOM   1213  HE3 LYS A  82       3.925 -12.153 -10.966  1.00  0.00           H  \\nATOM   1214  HE2 LYS A  82       2.421 -11.262 -10.721  1.00  0.00           H  \\nATOM   1215  HZ1 LYS A  82       2.190 -13.560 -10.128  1.00  0.00           H  \\nATOM   1216  HZ2 LYS A  82       1.216 -13.218 -11.415  1.00  0.00           H  \\nATOM   1217  HZ3 LYS A  82       2.622 -14.051 -11.642  1.00  0.00           H  \\nATOM   1218  N   ASP A  83       3.545  -7.578 -16.823  1.00  0.00           N  \\nATOM   1219  CA  ASP A  83       3.576  -7.198 -18.235  1.00  0.00           C  \\nATOM   1220  C   ASP A  83       2.919  -5.816 -18.405  1.00  0.00           C  \\nATOM   1221  O   ASP A  83       1.740  -5.719 -18.760  1.00  0.00           O  \\nATOM   1222  CB  ASP A  83       2.857  -8.239 -19.096  1.00  0.00           C  \\nATOM   1223  CG  ASP A  83       2.965  -7.951 -20.587  1.00  0.00           C  \\nATOM   1224  OD1 ASP A  83       3.018  -6.799 -20.948  1.00  0.00           O  \\nATOM   1225  OD2 ASP A  83       2.994  -8.886 -21.352  1.00  0.00           O  \\nATOM   1226  H   ASP A  83       2.803  -7.183 -16.264  1.00  0.00           H  \\nATOM   1227  HA  ASP A  83       4.608  -7.128 -18.580  1.00  0.00           H  \\nATOM   1228  HB3 ASP A  83       3.276  -9.226 -18.900  1.00  0.00           H  \\nATOM   1229  HB2 ASP A  83       1.802  -8.271 -18.823  1.00  0.00           H  \\nATOM   1230  N   PRO A  84       3.659  -4.733 -18.096  1.00  0.00           N  \\nATOM   1231  CA  PRO A  84       3.128  -3.378 -18.156  1.00  0.00           C  \\nATOM   1232  C   PRO A  84       2.463  -3.019 -19.491  1.00  0.00           C  \\nATOM   1233  O   PRO A  84       1.463  -2.298 -19.526  1.00  0.00           O  \\nATOM   1234  CB  PRO A  84       4.384  -2.534 -17.919  1.00  0.00           C  \\nATOM   1235  CG  PRO A  84       5.245  -3.393 -17.057  1.00  0.00           C  \\nATOM   1236  CD  PRO A  84       5.045  -4.784 -17.596  1.00  0.00           C  \\nATOM   1237  HA  PRO A  84       2.392  -3.302 -17.356  1.00  0.00           H  \\nATOM   1238  HB3 PRO A  84       4.857  -2.285 -18.880  1.00  0.00           H  \\nATOM   1239  HB2 PRO A  84       4.113  -1.583 -17.438  1.00  0.00           H  \\nATOM   1240  HG3 PRO A  84       6.292  -3.062 -17.115  1.00  0.00           H  \\nATOM   1241  HG2 PRO A  84       4.941  -3.300 -16.004  1.00  0.00           H  \\nATOM   1242  HD3 PRO A  84       5.766  -4.971 -18.406  1.00  0.00           H  \\nATOM   1243  HD2 PRO A  84       5.176  -5.515 -16.784  1.00  0.00           H  \\nATOM   1244  N   GLN A  85       2.996  -3.558 -20.590  1.00  0.00           N  \\nATOM   1245  CA  GLN A  85       2.570  -3.218 -21.946  1.00  0.00           C  \\nATOM   1246  C   GLN A  85       1.144  -3.688 -22.220  1.00  0.00           C  \\nATOM   1247  O   GLN A  85       0.328  -2.891 -22.682  1.00  0.00           O  \\nATOM   1248  CB  GLN A  85       3.541  -3.835 -22.961  1.00  0.00           C  \\nATOM   1249  CG  GLN A  85       4.923  -3.166 -22.919  1.00  0.00           C  \\nATOM   1250  CD  GLN A  85       5.906  -3.796 -23.900  1.00  0.00           C  \\nATOM   1251  OE1 GLN A  85       5.763  -4.911 -24.364  1.00  0.00           O  \\nATOM   1252  NE2 GLN A  85       6.966  -3.106 -24.259  1.00  0.00           N  \\nATOM   1253  H   GLN A  85       3.682  -4.292 -20.480  1.00  0.00           H  \\nATOM   1254  HA  GLN A  85       2.578  -2.135 -22.062  1.00  0.00           H  \\nATOM   1255  HB3 GLN A  85       3.640  -4.902 -22.766  1.00  0.00           H  \\nATOM   1256  HB2 GLN A  85       3.129  -3.705 -23.962  1.00  0.00           H  \\nATOM   1257  HG3 GLN A  85       4.811  -2.109 -23.162  1.00  0.00           H  \\nATOM   1258  HG2 GLN A  85       5.345  -3.247 -21.916  1.00  0.00           H  \\nATOM   1259 HE21 GLN A  85       7.118  -2.164 -23.928  1.00  0.00           H  \\nATOM   1260 HE22 GLN A  85       7.571  -3.569 -24.922  1.00  0.00           H  \\nATOM   1261  N   LYS A  86       0.809  -4.934 -21.856  1.00  0.00           N  \\nATOM   1262  CA  LYS A  86      -0.526  -5.513 -22.087  1.00  0.00           C  \\nATOM   1263  C   LYS A  86      -1.643  -4.743 -21.386  1.00  0.00           C  \\nATOM   1264  O   LYS A  86      -2.784  -4.767 -21.842  1.00  0.00           O  \\nATOM   1265  CB  LYS A  86      -0.552  -6.973 -21.633  1.00  0.00           C  \\nATOM   1266  CG  LYS A  86       0.123  -7.943 -22.595  1.00  0.00           C  \\nATOM   1267  CD  LYS A  86      -0.018  -9.381 -22.119  1.00  0.00           C  \\nATOM   1268  CE  LYS A  86       0.623 -10.354 -23.098  1.00  0.00           C  \\nATOM   1269  NZ  LYS A  86       2.093 -10.151 -23.202  1.00  0.00           N  \\nATOM   1270  H   LYS A  86       1.515  -5.517 -21.428  1.00  0.00           H  \\nATOM   1271  HA  LYS A  86      -0.747  -5.484 -23.154  1.00  0.00           H  \\nATOM   1272  HB3 LYS A  86      -0.056  -7.063 -20.666  1.00  0.00           H  \\nATOM   1273  HB2 LYS A  86      -1.585  -7.296 -21.503  1.00  0.00           H  \\nATOM   1274  HG3 LYS A  86      -0.330  -7.850 -23.583  1.00  0.00           H  \\nATOM   1275  HG2 LYS A  86       1.181  -7.698 -22.675  1.00  0.00           H  \\nATOM   1276  HD3 LYS A  86       0.460  -9.490 -21.145  1.00  0.00           H  \\nATOM   1277  HD2 LYS A  86      -1.075  -9.628 -22.015  1.00  0.00           H  \\nATOM   1278  HE3 LYS A  86       0.432 -11.375 -22.772  1.00  0.00           H  \\nATOM   1279  HE2 LYS A  86       0.180 -10.222 -24.085  1.00  0.00           H  \\nATOM   1280  HZ1 LYS A  86       2.479 -10.813 -23.860  1.00  0.00           H  \\nATOM   1281  HZ2 LYS A  86       2.282  -9.211 -23.522  1.00  0.00           H  \\nATOM   1282  HZ3 LYS A  86       2.518 -10.290 -22.296  1.00  0.00           H  \\nATOM   1283  N   TRP A  87      -1.326  -4.091 -20.273  1.00  0.00           N  \\nATOM   1284  CA  TRP A  87      -2.296  -3.354 -19.465  1.00  0.00           C  \\nATOM   1285  C   TRP A  87      -2.250  -1.838 -19.670  1.00  0.00           C  \\nATOM   1286  O   TRP A  87      -2.998  -1.121 -19.005  1.00  0.00           O  \\nATOM   1287  CB  TRP A  87      -2.069  -3.662 -17.984  1.00  0.00           C  \\nATOM   1288  CG  TRP A  87      -2.309  -5.099 -17.628  1.00  0.00           C  \\nATOM   1289  CD1 TRP A  87      -1.372  -6.012 -17.249  1.00  0.00           C  \\nATOM   1290  CD2 TRP A  87      -3.578  -5.797 -17.616  1.00  0.00           C  \\nATOM   1291  NE1 TRP A  87      -1.966  -7.226 -17.005  1.00  0.00           N  \\nATOM   1292  CE2 TRP A  87      -3.315  -7.112 -17.223  1.00  0.00           C  \\nATOM   1293  CE3 TRP A  87      -4.895  -5.417 -17.903  1.00  0.00           C  \\nATOM   1294  CZ2 TRP A  87      -4.322  -8.058 -17.108  1.00  0.00           C  \\nATOM   1295  CZ3 TRP A  87      -5.904  -6.366 -17.787  1.00  0.00           C  \\nATOM   1296  CH2 TRP A  87      -5.624  -7.652 -17.399  1.00  0.00           C  \\nATOM   1297  H   TRP A  87      -0.374  -4.159 -19.944  1.00  0.00           H  \\nATOM   1298  HA  TRP A  87      -3.301  -3.650 -19.763  1.00  0.00           H  \\nATOM   1299  HB3 TRP A  87      -1.045  -3.409 -17.712  1.00  0.00           H  \\nATOM   1300  HB2 TRP A  87      -2.732  -3.044 -17.378  1.00  0.00           H  \\nATOM   1301  HD1 TRP A  87      -0.307  -5.809 -17.154  1.00  0.00           H  \\nATOM   1302  HE1 TRP A  87      -1.488  -8.066 -16.712  1.00  0.00           H  \\nATOM   1303  HE3 TRP A  87      -5.123  -4.397 -18.209  1.00  0.00           H  \\nATOM   1304  HZ2 TRP A  87      -4.119  -9.085 -16.802  1.00  0.00           H  \\nATOM   1305  HZ3 TRP A  87      -6.927  -6.062 -18.013  1.00  0.00           H  \\nATOM   1306  HH2 TRP A  87      -6.440  -8.371 -17.318  1.00  0.00           H  \\nATOM   1307  N   GLY A  88      -1.390  -1.340 -20.565  1.00  0.00           N  \\nATOM   1308  CA  GLY A  88      -1.339   0.080 -20.918  1.00  0.00           C  \\nATOM   1309  C   GLY A  88      -0.763   0.961 -19.806  1.00  0.00           C  \\nATOM   1310  O   GLY A  88      -1.060   2.153 -19.747  1.00  0.00           O  \\nATOM   1311  H   GLY A  88      -0.727  -1.964 -21.004  1.00  0.00           H  \\nATOM   1312  HA3 GLY A  88      -2.344   0.433 -21.150  1.00  0.00           H  \\nATOM   1313  HA2 GLY A  88      -0.712   0.198 -21.802  1.00  0.00           H  \\nATOM   1314  N   ILE A  89       0.038   0.385 -18.906  1.00  0.00           N  \\nATOM   1315  CA  ILE A  89       0.639   1.120 -17.797  1.00  0.00           C  \\nATOM   1316  C   ILE A  89       2.125   1.348 -18.086  1.00  0.00           C  \\nATOM   1317  O   ILE A  89       2.762   0.630 -18.861  1.00  0.00           O  \\nATOM   1318  CB  ILE A  89       0.469   0.364 -16.467  1.00  0.00           C  \\nATOM   1319  CG1 ILE A  89       1.267  -0.942 -16.490  1.00  0.00           C  \\nATOM   1320  CG2 ILE A  89      -1.002   0.089 -16.195  1.00  0.00           C  \\nATOM   1321  CD1 ILE A  89       1.383  -1.611 -15.139  1.00  0.00           C  \\nATOM   1322  H   ILE A  89       0.395  -0.537 -19.113  1.00  0.00           H  \\nATOM   1323  HA  ILE A  89       0.178   2.106 -17.755  1.00  0.00           H  \\nATOM   1324  HB  ILE A  89       0.873   0.966 -15.653  1.00  0.00           H  \\nATOM   1325 HG13 ILE A  89       0.796  -1.644 -17.178  1.00  0.00           H  \\nATOM   1326 HG12 ILE A  89       2.274  -0.748 -16.860  1.00  0.00           H  \\nATOM   1327 HG21 ILE A  89      -1.103  -0.446 -15.251  1.00  0.00           H  \\nATOM   1328 HG22 ILE A  89      -1.543   1.033 -16.137  1.00  0.00           H  \\nATOM   1329 HG23 ILE A  89      -1.414  -0.517 -17.002  1.00  0.00           H  \\nATOM   1330 HD11 ILE A  89       1.962  -2.529 -15.236  1.00  0.00           H  \\nATOM   1331 HD12 ILE A  89       1.884  -0.938 -14.442  1.00  0.00           H  \\nATOM   1332 HD13 ILE A  89       0.388  -1.847 -14.763  1.00  0.00           H  \\nATOM   1333  N   LYS A  90       2.698   2.376 -17.466  1.00  0.00           N  \\nATOM   1334  CA  LYS A  90       4.059   2.803 -17.786  1.00  0.00           C  \\nATOM   1335  C   LYS A  90       5.081   1.735 -17.381  1.00  0.00           C  \\nATOM   1336  O   LYS A  90       4.807   0.892 -16.526  1.00  0.00           O  \\nATOM   1337  CB  LYS A  90       4.379   4.130 -17.096  1.00  0.00           C  \\nATOM   1338  CG  LYS A  90       4.469   4.043 -15.578  1.00  0.00           C  \\nATOM   1339  CD  LYS A  90       4.843   5.387 -14.969  1.00  0.00           C  \\nATOM   1340  CE  LYS A  90       3.744   6.417 -15.183  1.00  0.00           C  \\nATOM   1341  NZ  LYS A  90       4.031   7.693 -14.472  1.00  0.00           N  \\nATOM   1342  H   LYS A  90       2.204   2.821 -16.705  1.00  0.00           H  \\nATOM   1343  HA  LYS A  90       4.118   2.898 -18.869  1.00  0.00           H  \\nATOM   1344  HB3 LYS A  90       5.330   4.512 -17.466  1.00  0.00           H  \\nATOM   1345  HB2 LYS A  90       3.612   4.863 -17.344  1.00  0.00           H  \\nATOM   1346  HG3 LYS A  90       3.507   3.726 -15.173  1.00  0.00           H  \\nATOM   1347  HG2 LYS A  90       5.221   3.306 -15.300  1.00  0.00           H  \\nATOM   1348  HD3 LYS A  90       5.014   5.266 -13.899  1.00  0.00           H  \\nATOM   1349  HD2 LYS A  90       5.762   5.750 -15.428  1.00  0.00           H  \\nATOM   1350  HE3 LYS A  90       3.641   6.624 -16.247  1.00  0.00           H  \\nATOM   1351  HE2 LYS A  90       2.796   6.019 -14.820  1.00  0.00           H  \\nATOM   1352  HZ1 LYS A  90       3.280   8.348 -14.639  1.00  0.00           H  \\nATOM   1353  HZ2 LYS A  90       4.110   7.516 -13.481  1.00  0.00           H  \\nATOM   1354  HZ3 LYS A  90       4.898   8.081 -14.815  1.00  0.00           H  \\nATOM   1355  N   ALA A  91       6.273   1.806 -17.973  1.00  0.00           N  \\nATOM   1356  CA  ALA A  91       7.367   0.887 -17.667  1.00  0.00           C  \\nATOM   1357  C   ALA A  91       7.739   0.925 -16.176  1.00  0.00           C  \\nATOM   1358  O   ALA A  91       7.567   1.954 -15.516  1.00  0.00           O  \\nATOM   1359  CB  ALA A  91       8.572   1.232 -18.550  1.00  0.00           C  \\nATOM   1360  H   ALA A  91       6.434   2.536 -18.652  1.00  0.00           H  \\nATOM   1361  HA  ALA A  91       7.040  -0.123 -17.909  1.00  0.00           H  \\nATOM   1362  HB1 ALA A  91       8.298   1.163 -19.602  1.00  0.00           H  \\nATOM   1363  HB2 ALA A  91       8.918   2.243 -18.330  1.00  0.00           H  \\nATOM   1364  HB3 ALA A  91       9.385   0.533 -18.351  1.00  0.00           H  \\nATOM   1365  N   LEU A  92       8.244  -0.196 -15.656  1.00  0.00           N  \\nATOM   1366  CA  LEU A  92       8.487  -0.380 -14.224  1.00  0.00           C  \\nATOM   1367  C   LEU A  92       9.609   0.539 -13.717  1.00  0.00           C  \\nATOM   1368  O   LEU A  92      10.442   0.986 -14.511  1.00  0.00           O  \\nATOM   1369  CB  LEU A  92       8.846  -1.843 -13.936  1.00  0.00           C  \\nATOM   1370  CG  LEU A  92       7.770  -2.875 -14.295  1.00  0.00           C  \\nATOM   1371  CD1 LEU A  92       8.278  -4.273 -13.969  1.00  0.00           C  \\nATOM   1372  CD2 LEU A  92       6.493  -2.566 -13.528  1.00  0.00           C  \\nATOM   1373  H   LEU A  92       8.512  -0.943 -16.281  1.00  0.00           H  \\nATOM   1374  HA  LEU A  92       7.575  -0.103 -13.694  1.00  0.00           H  \\nATOM   1375  HB3 LEU A  92       9.747  -2.095 -14.493  1.00  0.00           H  \\nATOM   1376  HB2 LEU A  92       9.062  -1.945 -12.872  1.00  0.00           H  \\nATOM   1377  HG  LEU A  92       7.569  -2.834 -15.365  1.00  0.00           H  \\nATOM   1378 HD11 LEU A  92       7.514  -5.007 -14.225  1.00  0.00           H  \\nATOM   1379 HD12 LEU A  92       9.181  -4.476 -14.546  1.00  0.00           H  \\nATOM   1380 HD13 LEU A  92       8.504  -4.340 -12.906  1.00  0.00           H  \\nATOM   1381 HD21 LEU A  92       5.728  -3.299 -13.783  1.00  0.00           H  \\nATOM   1382 HD22 LEU A  92       6.693  -2.608 -12.457  1.00  0.00           H  \\nATOM   1383 HD23 LEU A  92       6.142  -1.568 -13.793  1.00  0.00           H  \\nATOM   1384  N   PRO A  93       9.643   0.832 -12.406  1.00  0.00           N  \\nATOM   1385  CA  PRO A  93      10.688   1.679 -11.855  1.00  0.00           C  \\nATOM   1386  C   PRO A  93      12.079   1.087 -12.134  1.00  0.00           C  \\nATOM   1387  O   PRO A  93      12.230  -0.140 -12.191  1.00  0.00           O  \\nATOM   1388  CB  PRO A  93      10.361   1.690 -10.358  1.00  0.00           C  \\nATOM   1389  CG  PRO A  93       8.900   1.404 -10.300  1.00  0.00           C  \\nATOM   1390  CD  PRO A  93       8.668   0.415 -11.411  1.00  0.00           C  \\nATOM   1391  HA  PRO A  93      10.622   2.665 -12.315  1.00  0.00           H  \\nATOM   1392  HB3 PRO A  93      10.965   0.934  -9.835  1.00  0.00           H  \\nATOM   1393  HB2 PRO A  93      10.623   2.666  -9.923  1.00  0.00           H  \\nATOM   1394  HG3 PRO A  93       8.630   1.002  -9.312  1.00  0.00           H  \\nATOM   1395  HG2 PRO A  93       8.325   2.333 -10.433  1.00  0.00           H  \\nATOM   1396  HD3 PRO A  93       8.863  -0.603 -11.044  1.00  0.00           H  \\nATOM   1397  HD2 PRO A  93       7.633   0.506 -11.773  1.00  0.00           H  \\nATOM   1398  N   PRO A  94      13.105   1.941 -12.298  1.00  0.00           N  \\nATOM   1399  CA  PRO A  94      14.481   1.476 -12.379  1.00  0.00           C  \\nATOM   1400  C   PRO A  94      14.855   0.712 -11.107  1.00  0.00           C  \\nATOM   1401  O   PRO A  94      14.376   1.019 -10.015  1.00  0.00           O  \\nATOM   1402  CB  PRO A  94      15.281   2.775 -12.522  1.00  0.00           C  \\nATOM   1403  CG  PRO A  94      14.290   3.758 -13.047  1.00  0.00           C  \\nATOM   1404  CD  PRO A  94      13.002   3.396 -12.357  1.00  0.00           C  \\nATOM   1405  HA  PRO A  94      14.602   0.823 -13.243  1.00  0.00           H  \\nATOM   1406  HB3 PRO A  94      15.698   3.069 -11.548  1.00  0.00           H  \\nATOM   1407  HB2 PRO A  94      16.131   2.621 -13.203  1.00  0.00           H  \\nATOM   1408  HG3 PRO A  94      14.617   4.784 -12.822  1.00  0.00           H  \\nATOM   1409  HG2 PRO A  94      14.221   3.680 -14.142  1.00  0.00           H  \\nATOM   1410  HD3 PRO A  94      12.976   3.859 -11.360  1.00  0.00           H  \\nATOM   1411  HD2 PRO A  94      12.151   3.737 -12.966  1.00  0.00           H  \\nATOM   1412  N   LYS A  95      15.741  -0.275 -11.248  1.00  0.00           N  \\nATOM   1413  CA  LYS A  95      16.263  -1.037 -10.109  1.00  0.00           C  \\nATOM   1414  C   LYS A  95      17.185  -0.163  -9.267  1.00  0.00           C  \\nATOM   1415  O   LYS A  95      17.894   0.680  -9.809  1.00  0.00           O  \\nATOM   1416  CB  LYS A  95      17.010  -2.289 -10.576  1.00  0.00           C  \\nATOM   1417  CG  LYS A  95      16.059  -3.316 -11.194  1.00  0.00           C  \\nATOM   1418  CD  LYS A  95      16.831  -4.580 -11.577  1.00  0.00           C  \\nATOM   1419  CE  LYS A  95      15.798  -5.550 -12.122  1.00  0.00           C  \\nATOM   1420  NZ  LYS A  95      16.329  -6.873 -12.518  1.00  0.00           N  \\nATOM   1421  H   LYS A  95      16.161  -0.409 -12.157  1.00  0.00           H  \\nATOM   1422  HA  LYS A  95      15.434  -1.355  -9.477  1.00  0.00           H  \\nATOM   1423  HB3 LYS A  95      17.506  -2.744  -9.718  1.00  0.00           H  \\nATOM   1424  HB2 LYS A  95      17.771  -2.006 -11.303  1.00  0.00           H  \\nATOM   1425  HG3 LYS A  95      15.280  -3.568 -10.474  1.00  0.00           H  \\nATOM   1426  HG2 LYS A  95      15.596  -2.895 -12.087  1.00  0.00           H  \\nATOM   1427  HD3 LYS A  95      17.573  -4.354 -12.343  1.00  0.00           H  \\nATOM   1428  HD2 LYS A  95      17.318  -4.991 -10.692  1.00  0.00           H  \\nATOM   1429  HE3 LYS A  95      15.289  -5.094 -12.972  1.00  0.00           H  \\nATOM   1430  HE2 LYS A  95      15.041  -5.698 -11.352  1.00  0.00           H  \\nATOM   1431  HZ1 LYS A  95      16.932  -6.799 -13.325  1.00  0.00           H  \\nATOM   1432  HZ2 LYS A  95      15.540  -7.452 -12.764  1.00  0.00           H  \\nATOM   1433  HZ3 LYS A  95      16.819  -7.298 -11.744  1.00  0.00           H  \\nATOM   1434  N   ASN A  96      17.260  -0.464  -7.976  1.00  0.00           N  \\nATOM   1435  CA  ASN A  96      18.326   0.030  -7.113  1.00  0.00           C  \\nATOM   1436  C   ASN A  96      19.061  -1.170  -6.493  1.00  0.00           C  \\nATOM   1437  O   ASN A  96      18.666  -1.610  -5.411  1.00  0.00           O  \\nATOM   1438  CB  ASN A  96      17.760   0.990  -6.055  1.00  0.00           C  \\nATOM   1439  CG  ASN A  96      18.879   1.552  -5.192  1.00  0.00           C  \\nATOM   1440  OD1 ASN A  96      20.056   1.325  -5.426  1.00  0.00           O  \\nATOM   1441  ND2 ASN A  96      18.579   2.295  -4.164  1.00  0.00           N  \\nATOM   1442  H   ASN A  96      16.672  -1.202  -7.615  1.00  0.00           H  \\nATOM   1443  HA  ASN A  96      19.041   0.607  -7.697  1.00  0.00           H  \\nATOM   1444  HB3 ASN A  96      17.262   1.823  -6.552  1.00  0.00           H  \\nATOM   1445  HB2 ASN A  96      17.040   0.478  -5.418  1.00  0.00           H  \\nATOM   1446 HD21 ASN A  96      19.349   2.645  -3.610  1.00  0.00           H  \\nATOM   1447 HD22 ASN A  96      17.640   2.356  -3.796  1.00  0.00           H  \\nATOM   1448  N   PRO A  97      20.111  -1.712  -7.143  1.00  0.00           N  \\nATOM   1449  CA  PRO A  97      20.819  -2.902  -6.668  1.00  0.00           C  \\nATOM   1450  C   PRO A  97      21.322  -2.805  -5.223  1.00  0.00           C  \\nATOM   1451  O   PRO A  97      21.407  -3.830  -4.555  1.00  0.00           O  \\nATOM   1452  CB  PRO A  97      21.988  -2.995  -7.653  1.00  0.00           C  \\nATOM   1453  CG  PRO A  97      21.448  -2.421  -8.918  1.00  0.00           C  \\nATOM   1454  CD  PRO A  97      20.589  -1.269  -8.468  1.00  0.00           C  \\nATOM   1455  HA  PRO A  97      20.147  -3.757  -6.742  1.00  0.00           H  \\nATOM   1456  HB3 PRO A  97      22.852  -2.436  -7.265  1.00  0.00           H  \\nATOM   1457  HB2 PRO A  97      22.306  -4.043  -7.762  1.00  0.00           H  \\nATOM   1458  HG3 PRO A  97      22.272  -2.104  -9.573  1.00  0.00           H  \\nATOM   1459  HG2 PRO A  97      20.879  -3.183  -9.469  1.00  0.00           H  \\nATOM   1460  HD3 PRO A  97      21.203  -0.359  -8.394  1.00  0.00           H  \\nATOM   1461  HD2 PRO A  97      19.766  -1.124  -9.183  1.00  0.00           H  \\nATOM   1462  N   SER A  98      21.592  -1.593  -4.724  1.00  0.00           N  \\nATOM   1463  CA  SER A  98      22.043  -1.367  -3.344  1.00  0.00           C  \\nATOM   1464  C   SER A  98      20.947  -1.622  -2.302  1.00  0.00           C  \\nATOM   1465  O   SER A  98      21.248  -1.768  -1.123  1.00  0.00           O  \\nATOM   1466  CB  SER A  98      22.551   0.054  -3.200  1.00  0.00           C  \\nATOM   1467  OG  SER A  98      21.495   0.973  -3.254  1.00  0.00           O  \\nATOM   1468  H   SER A  98      21.493  -0.776  -5.311  1.00  0.00           H  \\nATOM   1469  HA  SER A  98      22.868  -2.046  -3.125  1.00  0.00           H  \\nATOM   1470  HB3 SER A  98      23.079   0.158  -2.252  1.00  0.00           H  \\nATOM   1471  HB2 SER A  98      23.263   0.268  -3.996  1.00  0.00           H  \\nATOM   1472  HG  SER A  98      21.239   1.029  -4.178  1.00  0.00           H  \\nATOM   1473  N   TRP A  99      19.681  -1.681  -2.725  1.00  0.00           N  \\nATOM   1474  CA  TRP A  99      18.532  -1.976  -1.867  1.00  0.00           C  \\nATOM   1475  C   TRP A  99      17.848  -3.305  -2.225  1.00  0.00           C  \\nATOM   1476  O   TRP A  99      17.452  -4.074  -1.345  1.00  0.00           O  \\nATOM   1477  CB  TRP A  99      17.563  -0.795  -1.955  1.00  0.00           C  \\nATOM   1478  CG  TRP A  99      16.372  -0.911  -1.064  1.00  0.00           C  \\nATOM   1479  CD1 TRP A  99      16.340  -0.600   0.249  1.00  0.00           C  \\nATOM   1480  CD2 TRP A  99      15.037  -1.385  -1.392  1.00  0.00           C  \\nATOM   1481  NE1 TRP A  99      15.076  -0.828   0.754  1.00  0.00           N  \\nATOM   1482  CE2 TRP A  99      14.229  -1.312  -0.217  1.00  0.00           C  \\nATOM   1483  CE3 TRP A  99      14.435  -1.869  -2.569  1.00  0.00           C  \\nATOM   1484  CZ2 TRP A  99      12.875  -1.672  -0.216  1.00  0.00           C  \\nATOM   1485  CZ3 TRP A  99      13.087  -2.256  -2.566  1.00  0.00           C  \\nATOM   1486  CH2 TRP A  99      12.303  -2.138  -1.409  1.00  0.00           C  \\nATOM   1487  H   TRP A  99      19.510  -1.529  -3.709  1.00  0.00           H  \\nATOM   1488  HA  TRP A  99      18.858  -2.061  -0.830  1.00  0.00           H  \\nATOM   1489  HB3 TRP A  99      17.222  -0.682  -2.985  1.00  0.00           H  \\nATOM   1490  HB2 TRP A  99      18.100   0.115  -1.686  1.00  0.00           H  \\nATOM   1491  HD1 TRP A  99      17.176  -0.207   0.808  1.00  0.00           H  \\nATOM   1492  HE1 TRP A  99      14.822  -0.600   1.704  1.00  0.00           H  \\nATOM   1493  HE3 TRP A  99      15.024  -1.933  -3.472  1.00  0.00           H  \\nATOM   1494  HZ2 TRP A  99      12.282  -1.574   0.683  1.00  0.00           H  \\nATOM   1495  HZ3 TRP A  99      12.648  -2.649  -3.471  1.00  0.00           H  \\nATOM   1496  HH2 TRP A  99      11.255  -2.397  -1.444  1.00  0.00           H  \\nATOM   1497  N   SER A 100      17.696  -3.600  -3.520  1.00  0.00           N  \\nATOM   1498  CA  SER A 100      17.189  -4.879  -4.023  1.00  0.00           C  \\nATOM   1499  C   SER A 100      17.529  -5.084  -5.498  1.00  0.00           C  \\nATOM   1500  O   SER A 100      17.288  -4.217  -6.338  1.00  0.00           O  \\nATOM   1501  CB  SER A 100      15.687  -4.953  -3.831  1.00  0.00           C  \\nATOM   1502  OG  SER A 100      15.175  -6.159  -4.328  1.00  0.00           O  \\nATOM   1503  H   SER A 100      18.028  -2.930  -4.199  1.00  0.00           H  \\nATOM   1504  HA  SER A 100      17.633  -5.684  -3.436  1.00  0.00           H  \\nATOM   1505  HB3 SER A 100      15.451  -4.864  -2.771  1.00  0.00           H  \\nATOM   1506  HB2 SER A 100      15.213  -4.115  -4.343  1.00  0.00           H  \\nATOM   1507  HG  SER A 100      14.480  -6.422  -3.721  1.00  0.00           H  \\nATOM   1508  N   ALA A 101      17.996  -6.286  -5.842  1.00  0.00           N  \\nATOM   1509  CA  ALA A 101      18.214  -6.696  -7.232  1.00  0.00           C  \\nATOM   1510  C   ALA A 101      16.934  -7.223  -7.928  1.00  0.00           C  \\nATOM   1511  O   ALA A 101      16.959  -7.568  -9.115  1.00  0.00           O  \\nATOM   1512  CB  ALA A 101      19.340  -7.735  -7.240  1.00  0.00           C  \\nATOM   1513  H   ALA A 101      18.202  -6.947  -5.108  1.00  0.00           H  \\nATOM   1514  HA  ALA A 101      18.553  -5.830  -7.802  1.00  0.00           H  \\nATOM   1515  HB1 ALA A 101      19.572  -8.018  -8.267  1.00  0.00           H  \\nATOM   1516  HB2 ALA A 101      19.040  -8.624  -6.685  1.00  0.00           H  \\nATOM   1517  HB3 ALA A 101      20.237  -7.312  -6.786  1.00  0.00           H  \\nATOM   1518  N   GLN A 102      15.813  -7.322  -7.202  1.00  0.00           N  \\nATOM   1519  CA  GLN A 102      14.568  -7.892  -7.720  1.00  0.00           C  \\nATOM   1520  C   GLN A 102      13.799  -6.910  -8.612  1.00  0.00           C  \\nATOM   1521  O   GLN A 102      13.778  -5.708  -8.368  1.00  0.00           O  \\nATOM   1522  CB  GLN A 102      13.675  -8.344  -6.562  1.00  0.00           C  \\nATOM   1523  CG  GLN A 102      14.239  -9.502  -5.757  1.00  0.00           C  \\nATOM   1524  CD  GLN A 102      13.233 -10.066  -4.771  1.00  0.00           C  \\nATOM   1525  OE1 GLN A 102      12.301  -9.375  -4.351  1.00  0.00           O  \\nATOM   1526  NE2 GLN A 102      13.416 -11.327  -4.397  1.00  0.00           N  \\nATOM   1527  H   GLN A 102      15.810  -6.935  -6.269  1.00  0.00           H  \\nATOM   1528  HA  GLN A 102      14.817  -8.761  -8.329  1.00  0.00           H  \\nATOM   1529  HB3 GLN A 102      13.512  -7.509  -5.881  1.00  0.00           H  \\nATOM   1530  HB2 GLN A 102      12.702  -8.645  -6.949  1.00  0.00           H  \\nATOM   1531  HG3 GLN A 102      14.530 -10.299  -6.442  1.00  0.00           H  \\nATOM   1532  HG2 GLN A 102      15.107  -9.154  -5.198  1.00  0.00           H  \\nATOM   1533 HE21 GLN A 102      12.784 -11.753  -3.748  1.00  0.00           H  \\nATOM   1534 HE22 GLN A 102      14.185 -11.851  -4.762  1.00  0.00           H  \\nATOM   1535  N   ASP A 103      13.107  -7.447  -9.618  1.00  0.00           N  \\nATOM   1536  CA  ASP A 103      12.131  -6.695 -10.410  1.00  0.00           C  \\nATOM   1537  C   ASP A 103      10.942  -6.221  -9.568  1.00  0.00           C  \\nATOM   1538  O   ASP A 103      10.395  -6.964  -8.746  1.00  0.00           O  \\nATOM   1539  CB  ASP A 103      11.622  -7.551 -11.573  1.00  0.00           C  \\nATOM   1540  CG  ASP A 103      12.728  -7.956 -12.539  1.00  0.00           C  \\nATOM   1541  OD1 ASP A 103      13.820  -7.457 -12.403  1.00  0.00           O  \\nATOM   1542  OD2 ASP A 103      12.469  -8.759 -13.403  1.00  0.00           O  \\nATOM   1543  H   ASP A 103      13.194  -8.435  -9.804  1.00  0.00           H  \\nATOM   1544  HA  ASP A 103      12.618  -5.813 -10.827  1.00  0.00           H  \\nATOM   1545  HB3 ASP A 103      11.152  -8.453 -11.183  1.00  0.00           H  \\nATOM   1546  HB2 ASP A 103      10.861  -6.998 -12.126  1.00  0.00           H  \\nATOM   1547  N   PHE A 104      10.480  -5.005  -9.853  1.00  0.00           N  \\nATOM   1548  CA  PHE A 104       9.309  -4.389  -9.233  1.00  0.00           C  \\nATOM   1549  C   PHE A 104       8.010  -4.803  -9.940  1.00  0.00           C  \\nATOM   1550  O   PHE A 104       7.320  -3.987 -10.538  1.00  0.00           O  \\nATOM   1551  CB  PHE A 104       9.443  -2.865  -9.250  1.00  0.00           C  \\nATOM   1552  CG  PHE A 104      10.578  -2.347  -8.413  1.00  0.00           C  \\nATOM   1553  CD1 PHE A 104      11.656  -1.704  -9.003  1.00  0.00           C  \\nATOM   1554  CD2 PHE A 104      10.570  -2.502  -7.036  1.00  0.00           C  \\nATOM   1555  CE1 PHE A 104      12.701  -1.227  -8.234  1.00  0.00           C  \\nATOM   1556  CE2 PHE A 104      11.612  -2.025  -6.264  1.00  0.00           C  \\nATOM   1557  CZ  PHE A 104      12.679  -1.387  -6.865  1.00  0.00           C  \\nATOM   1558  H   PHE A 104      10.981  -4.449 -10.530  1.00  0.00           H  \\nATOM   1559  HA  PHE A 104       9.246  -4.742  -8.204  1.00  0.00           H  \\nATOM   1560  HB3 PHE A 104       9.592  -2.525 -10.274  1.00  0.00           H  \\nATOM   1561  HB2 PHE A 104       8.520  -2.415  -8.888  1.00  0.00           H  \\nATOM   1562  HD1 PHE A 104      11.673  -1.577 -10.086  1.00  0.00           H  \\nATOM   1563  HD2 PHE A 104       9.727  -3.006  -6.561  1.00  0.00           H  \\nATOM   1564  HE1 PHE A 104      13.542  -0.724  -8.710  1.00  0.00           H  \\nATOM   1565  HE2 PHE A 104      11.593  -2.153  -5.182  1.00  0.00           H  \\nATOM   1566  HZ  PHE A 104      13.502  -1.013  -6.258  1.00  0.00           H  \\nATOM   1567  N   LYS A 105       7.708  -6.103  -9.939  1.00  0.00           N  \\nATOM   1568  CA  LYS A 105       6.587  -6.690 -10.701  1.00  0.00           C  \\nATOM   1569  C   LYS A 105       5.399  -7.144  -9.846  1.00  0.00           C  \\nATOM   1570  O   LYS A 105       4.397  -7.616 -10.385  1.00  0.00           O  \\nATOM   1571  CB  LYS A 105       7.091  -7.875 -11.525  1.00  0.00           C  \\nATOM   1572  CG  LYS A 105       7.985  -7.491 -12.698  1.00  0.00           C  \\nATOM   1573  CD  LYS A 105       8.659  -8.713 -13.302  1.00  0.00           C  \\nATOM   1574  CE  LYS A 105       9.398  -8.362 -14.585  1.00  0.00           C  \\nATOM   1575  NZ  LYS A 105      10.373  -9.416 -14.972  1.00  0.00           N  \\nATOM   1576  H   LYS A 105       8.334  -6.707  -9.428  1.00  0.00           H  \\nATOM   1577  HA  LYS A 105       6.183  -5.921 -11.360  1.00  0.00           H  \\nATOM   1578  HB3 LYS A 105       7.655  -8.552 -10.883  1.00  0.00           H  \\nATOM   1579  HB2 LYS A 105       6.241  -8.431 -11.921  1.00  0.00           H  \\nATOM   1580  HG3 LYS A 105       7.386  -7.000 -13.466  1.00  0.00           H  \\nATOM   1581  HG2 LYS A 105       8.750  -6.794 -12.358  1.00  0.00           H  \\nATOM   1582  HD3 LYS A 105       9.370  -9.129 -12.585  1.00  0.00           H  \\nATOM   1583  HD2 LYS A 105       7.908  -9.472 -13.523  1.00  0.00           H  \\nATOM   1584  HE3 LYS A 105       8.680  -8.233 -15.394  1.00  0.00           H  \\nATOM   1585  HE2 LYS A 105       9.933  -7.422 -14.450  1.00  0.00           H  \\nATOM   1586  HZ1 LYS A 105      10.841  -9.146 -15.826  1.00  0.00           H  \\nATOM   1587  HZ2 LYS A 105      11.056  -9.531 -14.236  1.00  0.00           H  \\nATOM   1588  HZ3 LYS A 105       9.886 -10.289 -15.119  1.00  0.00           H  \\nATOM   1589  N   SER A 106       5.516  -7.055  -8.525  1.00  0.00           N  \\nATOM   1590  CA  SER A 106       4.486  -7.470  -7.574  1.00  0.00           C  \\nATOM   1591  C   SER A 106       3.534  -6.302  -7.305  1.00  0.00           C  \\nATOM   1592  O   SER A 106       4.014  -5.198  -7.059  1.00  0.00           O  \\nATOM   1593  CB  SER A 106       5.116  -7.943  -6.278  1.00  0.00           C  \\nATOM   1594  OG  SER A 106       5.855  -9.117  -6.478  1.00  0.00           O  \\nATOM   1595  H   SER A 106       6.312  -6.559  -8.147  1.00  0.00           H  \\nATOM   1596  HA  SER A 106       3.937  -8.314  -7.991  1.00  0.00           H  \\nATOM   1597  HB3 SER A 106       5.768  -7.163  -5.885  1.00  0.00           H  \\nATOM   1598  HB2 SER A 106       4.337  -8.122  -5.539  1.00  0.00           H  \\nATOM   1599  HG  SER A 106       5.667  -9.679  -5.723  1.00  0.00           H  \\nATOM   1600  N   PRO A 107       2.207  -6.499  -7.346  1.00  0.00           N  \\nATOM   1601  CA  PRO A 107       1.263  -5.412  -7.114  1.00  0.00           C  \\nATOM   1602  C   PRO A 107       1.073  -5.177  -5.615  1.00  0.00           C  \\nATOM   1603  O   PRO A 107       0.433  -5.986  -4.950  1.00  0.00           O  \\nATOM   1604  CB  PRO A 107      -0.028  -5.855  -7.812  1.00  0.00           C  \\nATOM   1605  CG  PRO A 107      -0.012  -7.377  -7.672  1.00  0.00           C  \\nATOM   1606  CD  PRO A 107       1.469  -7.748  -7.735  1.00  0.00           C  \\nATOM   1607  HA  PRO A 107       1.621  -4.483  -7.557  1.00  0.00           H  \\nATOM   1608  HB3 PRO A 107       0.016  -5.591  -8.868  1.00  0.00           H  \\nATOM   1609  HB2 PRO A 107      -0.912  -5.417  -7.350  1.00  0.00           H  \\nATOM   1610  HG3 PRO A 107      -0.571  -7.851  -8.480  1.00  0.00           H  \\nATOM   1611  HG2 PRO A 107      -0.423  -7.665  -6.705  1.00  0.00           H  \\nATOM   1612  HD3 PRO A 107       1.731  -8.031  -8.755  1.00  0.00           H  \\nATOM   1613  HD2 PRO A 107       1.678  -8.566  -7.047  1.00  0.00           H  \\nATOM   1614  N   GLU A 108       1.630  -4.087  -5.090  1.00  0.00           N  \\nATOM   1615  CA  GLU A 108       1.217  -3.554  -3.786  1.00  0.00           C  \\nATOM   1616  C   GLU A 108      -0.188  -2.983  -3.901  1.00  0.00           C  \\nATOM   1617  O   GLU A 108      -0.502  -2.424  -4.950  1.00  0.00           O  \\nATOM   1618  CB  GLU A 108       2.188  -2.476  -3.300  1.00  0.00           C  \\nATOM   1619  CG  GLU A 108       1.799  -1.832  -1.976  1.00  0.00           C  \\nATOM   1620  CD  GLU A 108       1.656  -2.829  -0.860  1.00  0.00           C  \\nATOM   1621  OE1 GLU A 108       2.138  -3.926  -1.004  1.00  0.00           O  \\nATOM   1622  OE2 GLU A 108       1.063  -2.492   0.138  1.00  0.00           O  \\nATOM   1623  H   GLU A 108       2.060  -3.420  -5.714  1.00  0.00           H  \\nATOM   1624  HA  GLU A 108       1.177  -4.368  -3.063  1.00  0.00           H  \\nATOM   1625  HB3 GLU A 108       3.182  -2.907  -3.183  1.00  0.00           H  \\nATOM   1626  HB2 GLU A 108       2.259  -1.686  -4.048  1.00  0.00           H  \\nATOM   1627  HG3 GLU A 108       2.561  -1.103  -1.701  1.00  0.00           H  \\nATOM   1628  HG2 GLU A 108       0.858  -1.300  -2.106  1.00  0.00           H  \\nATOM   1629  N   GLU A 109      -1.026  -3.130  -2.872  1.00  0.00           N  \\nATOM   1630  CA  GLU A 109      -2.392  -2.611  -2.897  1.00  0.00           C  \\nATOM   1631  C   GLU A 109      -2.815  -1.983  -1.563  1.00  0.00           C  \\nATOM   1632  O   GLU A 109      -2.575  -2.541  -0.490  1.00  0.00           O  \\nATOM   1633  CB  GLU A 109      -3.366  -3.732  -3.268  1.00  0.00           C  \\nATOM   1634  CG  GLU A 109      -3.363  -4.911  -2.306  1.00  0.00           C  \\nATOM   1635  CD  GLU A 109      -4.216  -6.055  -2.779  1.00  0.00           C  \\nATOM   1636  OE1 GLU A 109      -4.982  -5.861  -3.692  1.00  0.00           O  \\nATOM   1637  OE2 GLU A 109      -4.101  -7.124  -2.227  1.00  0.00           O  \\nATOM   1638  H   GLU A 109      -0.674  -3.438  -1.977  1.00  0.00           H  \\nATOM   1639  HA  GLU A 109      -2.417  -1.818  -3.645  1.00  0.00           H  \\nATOM   1640  HB3 GLU A 109      -4.381  -3.334  -3.306  1.00  0.00           H  \\nATOM   1641  HB2 GLU A 109      -3.125  -4.110  -4.261  1.00  0.00           H  \\nATOM   1642  HG3 GLU A 109      -2.339  -5.263  -2.183  1.00  0.00           H  \\nATOM   1643  HG2 GLU A 109      -3.721  -4.575  -1.334  1.00  0.00           H  \\nATOM   1644  N   TYR A 110      -3.531  -0.856  -1.640  1.00  0.00           N  \\nATOM   1645  CA  TYR A 110      -4.091  -0.182  -0.470  1.00  0.00           C  \\nATOM   1646  C   TYR A 110      -5.464   0.456  -0.745  1.00  0.00           C  \\nATOM   1647  O   TYR A 110      -5.625   1.191  -1.713  1.00  0.00           O  \\nATOM   1648  CB  TYR A 110      -3.119   0.855   0.086  1.00  0.00           C  \\nATOM   1649  CG  TYR A 110      -3.523   1.261   1.487  1.00  0.00           C  \\nATOM   1650  CD1 TYR A 110      -4.307   2.411   1.700  1.00  0.00           C  \\nATOM   1651  CD2 TYR A 110      -3.204   0.417   2.568  1.00  0.00           C  \\nATOM   1652  CE1 TYR A 110      -4.756   2.726   2.998  1.00  0.00           C  \\nATOM   1653  CE2 TYR A 110      -3.631   0.740   3.867  1.00  0.00           C  \\nATOM   1654  CZ  TYR A 110      -4.399   1.901   4.085  1.00  0.00           C  \\nATOM   1655  OH  TYR A 110      -4.782   2.218   5.349  1.00  0.00           O  \\nATOM   1656  H   TYR A 110      -3.645  -0.423  -2.546  1.00  0.00           H  \\nATOM   1657  HA  TYR A 110      -4.186  -0.931   0.316  1.00  0.00           H  \\nATOM   1658  HB3 TYR A 110      -2.118   0.422   0.112  1.00  0.00           H  \\nATOM   1659  HB2 TYR A 110      -3.085   1.726  -0.568  1.00  0.00           H  \\nATOM   1660  HD1 TYR A 110      -4.564   3.042   0.863  1.00  0.00           H  \\nATOM   1661  HD2 TYR A 110      -2.601  -0.461   2.393  1.00  0.00           H  \\nATOM   1662  HE1 TYR A 110      -5.356   3.605   3.185  1.00  0.00           H  \\nATOM   1663  HE2 TYR A 110      -3.340   0.106   4.692  1.00  0.00           H  \\nATOM   1664  HH  TYR A 110      -4.381   1.622   5.987  1.00  0.00           H  \\nATOM   1665  N   ALA A 111      -6.488   0.296   0.095  1.00  0.00           N  \\nATOM   1666  CA  ALA A 111      -6.488  -0.466   1.342  1.00  0.00           C  \\nATOM   1667  C   ALA A 111      -6.188  -1.955   1.128  1.00  0.00           C  \\nATOM   1668  O   ALA A 111      -6.524  -2.523   0.087  1.00  0.00           O  \\nATOM   1669  CB  ALA A 111      -7.832  -0.260   2.041  1.00  0.00           C  \\nATOM   1670  H   ALA A 111      -7.348   0.757  -0.167  1.00  0.00           H  \\nATOM   1671  HA  ALA A 111      -5.718  -0.063   2.000  1.00  0.00           H  \\nATOM   1672  HB1 ALA A 111      -8.638  -0.630   1.409  1.00  0.00           H  \\nATOM   1673  HB2 ALA A 111      -7.980   0.803   2.236  1.00  0.00           H  \\nATOM   1674  HB3 ALA A 111      -7.842  -0.797   2.990  1.00  0.00           H  \\nATOM   1675  N   PHE A 112      -5.561  -2.575   2.133  1.00  0.00           N  \\nATOM   1676  CA  PHE A 112      -5.115  -3.963   2.062  1.00  0.00           C  \\nATOM   1677  C   PHE A 112      -6.242  -4.893   1.621  1.00  0.00           C  \\nATOM   1678  O   PHE A 112      -7.362  -4.792   2.134  1.00  0.00           O  \\nATOM   1679  CB  PHE A 112      -4.629  -4.434   3.444  1.00  0.00           C  \\nATOM   1680  CG  PHE A 112      -3.524  -3.608   4.059  1.00  0.00           C  \\nATOM   1681  CD1 PHE A 112      -2.329  -3.412   3.347  1.00  0.00           C  \\nATOM   1682  CD2 PHE A 112      -3.677  -3.041   5.341  1.00  0.00           C  \\nATOM   1683  CE1 PHE A 112      -1.304  -2.645   3.911  1.00  0.00           C  \\nATOM   1684  CE2 PHE A 112      -2.645  -2.266   5.902  1.00  0.00           C  \\nATOM   1685  CZ  PHE A 112      -1.455  -2.064   5.180  1.00  0.00           C  \\nATOM   1686  H   PHE A 112      -5.237  -2.017   2.911  1.00  0.00           H  \\nATOM   1687  HA  PHE A 112      -4.298  -4.026   1.343  1.00  0.00           H  \\nATOM   1688  HB3 PHE A 112      -5.477  -4.458   4.128  1.00  0.00           H  \\nATOM   1689  HB2 PHE A 112      -4.257  -5.454   3.343  1.00  0.00           H  \\nATOM   1690  HD1 PHE A 112      -2.184  -3.823   2.360  1.00  0.00           H  \\nATOM   1691  HD2 PHE A 112      -4.591  -3.201   5.894  1.00  0.00           H  \\nATOM   1692  HE1 PHE A 112      -0.399  -2.486   3.342  1.00  0.00           H  \\nATOM   1693  HE2 PHE A 112      -2.761  -1.817   6.877  1.00  0.00           H  \\nATOM   1694  HZ  PHE A 112      -0.657  -1.456   5.579  1.00  0.00           H  \\nATOM   1695  N   ALA A 113      -5.941  -5.890   0.785  1.00  0.00           N  \\nATOM   1696  CA  ALA A 113      -6.917  -6.935   0.479  1.00  0.00           C  \\nATOM   1697  C   ALA A 113      -7.399  -7.672   1.734  1.00  0.00           C  \\nATOM   1698  O   ALA A 113      -8.535  -8.126   1.774  1.00  0.00           O  \\nATOM   1699  CB  ALA A 113      -6.336  -7.922  -0.530  1.00  0.00           C  \\nATOM   1700  H   ALA A 113      -5.057  -5.872   0.296  1.00  0.00           H  \\nATOM   1701  HA  ALA A 113      -7.793  -6.466   0.028  1.00  0.00           H  \\nATOM   1702  HB1 ALA A 113      -5.307  -8.184  -0.281  1.00  0.00           H  \\nATOM   1703  HB2 ALA A 113      -6.949  -8.822  -0.566  1.00  0.00           H  \\nATOM   1704  HB3 ALA A 113      -6.380  -7.458  -1.516  1.00  0.00           H  \\nATOM   1705  N   SER A 114      -6.587  -7.757   2.790  1.00  0.00           N  \\nATOM   1706  CA  SER A 114      -6.965  -8.371   4.067  1.00  0.00           C  \\nATOM   1707  C   SER A 114      -7.937  -7.533   4.908  1.00  0.00           C  \\nATOM   1708  O   SER A 114      -8.323  -7.989   5.981  1.00  0.00           O  \\nATOM   1709  CB  SER A 114      -5.713  -8.643   4.880  1.00  0.00           C  \\nATOM   1710  OG  SER A 114      -5.079  -7.447   5.241  1.00  0.00           O  \\nATOM   1711  H   SER A 114      -5.694  -7.285   2.752  1.00  0.00           H  \\nATOM   1712  HA  SER A 114      -7.477  -9.312   3.863  1.00  0.00           H  \\nATOM   1713  HB3 SER A 114      -5.977  -9.204   5.776  1.00  0.00           H  \\nATOM   1714  HB2 SER A 114      -5.029  -9.260   4.298  1.00  0.00           H  \\nATOM   1715  HG  SER A 114      -5.752  -6.764   5.187  1.00  0.00           H  \\nATOM   1716  N   SER A 115      -8.348  -6.343   4.460  1.00  0.00           N  \\nATOM   1717  CA  SER A 115      -9.326  -5.484   5.139  1.00  0.00           C  \\nATOM   1718  C   SER A 115     -10.640  -5.338   4.362  1.00  0.00           C  \\nATOM   1719  O   SER A 115     -10.665  -5.468   3.142  1.00  0.00           O  \\nATOM   1720  CB  SER A 115      -8.725  -4.112   5.370  1.00  0.00           C  \\nATOM   1721  OG  SER A 115      -8.441  -3.476   4.153  1.00  0.00           O  \\nATOM   1722  H   SER A 115      -7.996  -6.015   3.573  1.00  0.00           H  \\nATOM   1723  HA  SER A 115      -9.565  -5.941   6.099  1.00  0.00           H  \\nATOM   1724  HB3 SER A 115      -9.419  -3.502   5.948  1.00  0.00           H  \\nATOM   1725  HB2 SER A 115      -7.811  -4.209   5.954  1.00  0.00           H  \\nATOM   1726  HG  SER A 115      -8.226  -4.177   3.533  1.00  0.00           H  \\nATOM   1727  N   LEU A 116     -11.736  -5.023   5.056  1.00  0.00           N  \\nATOM   1728  CA  LEU A 116     -13.058  -4.808   4.452  1.00  0.00           C  \\nATOM   1729  C   LEU A 116     -13.015  -3.723   3.366  1.00  0.00           C  \\nATOM   1730  O   LEU A 116     -13.743  -3.793   2.382  1.00  0.00           O  \\nATOM   1731  CB  LEU A 116     -14.073  -4.416   5.534  1.00  0.00           C  \\nATOM   1732  CG  LEU A 116     -15.529  -4.293   5.066  1.00  0.00           C  \\nATOM   1733  CD1 LEU A 116     -16.027  -5.651   4.592  1.00  0.00           C  \\nATOM   1734  CD2 LEU A 116     -16.385  -3.764   6.207  1.00  0.00           C  \\nATOM   1735  H   LEU A 116     -11.650  -4.922   6.058  1.00  0.00           H  \\nATOM   1736  HA  LEU A 116     -13.384  -5.732   3.976  1.00  0.00           H  \\nATOM   1737  HB3 LEU A 116     -14.041  -5.161   6.326  1.00  0.00           H  \\nATOM   1738  HB2 LEU A 116     -13.777  -3.455   5.955  1.00  0.00           H  \\nATOM   1739  HG  LEU A 116     -15.583  -3.604   4.222  1.00  0.00           H  \\nATOM   1740 HD11 LEU A 116     -17.061  -5.564   4.260  1.00  0.00           H  \\nATOM   1741 HD12 LEU A 116     -15.408  -5.996   3.763  1.00  0.00           H  \\nATOM   1742 HD13 LEU A 116     -15.968  -6.366   5.412  1.00  0.00           H  \\nATOM   1743 HD21 LEU A 116     -17.420  -3.676   5.875  1.00  0.00           H  \\nATOM   1744 HD22 LEU A 116     -16.333  -4.452   7.051  1.00  0.00           H  \\nATOM   1745 HD23 LEU A 116     -16.018  -2.785   6.514  1.00  0.00           H  \\nATOM   1746  N   GLN A 117     -12.142  -2.735   3.551  1.00  0.00           N  \\nATOM   1747  CA  GLN A 117     -11.996  -1.549   2.714  1.00  0.00           C  \\nATOM   1748  C   GLN A 117     -11.171  -1.813   1.446  1.00  0.00           C  \\nATOM   1749  O   GLN A 117     -11.013  -0.912   0.625  1.00  0.00           O  \\nATOM   1750  CB  GLN A 117     -11.330  -0.444   3.550  1.00  0.00           C  \\nATOM   1751  CG  GLN A 117     -12.168   0.058   4.741  1.00  0.00           C  \\nATOM   1752  CD  GLN A 117     -12.309  -0.921   5.911  1.00  0.00           C  \\nATOM   1753  OE1 GLN A 117     -11.534  -1.849   6.121  1.00  0.00           O  \\nATOM   1754  NE2 GLN A 117     -13.337  -0.770   6.714  1.00  0.00           N  \\nATOM   1755  H   GLN A 117     -11.541  -2.796   4.361  1.00  0.00           H  \\nATOM   1756  HA  GLN A 117     -12.981  -1.211   2.393  1.00  0.00           H  \\nATOM   1757  HB3 GLN A 117     -10.360  -0.795   3.903  1.00  0.00           H  \\nATOM   1758  HB2 GLN A 117     -11.150   0.404   2.891  1.00  0.00           H  \\nATOM   1759  HG3 GLN A 117     -11.703   0.965   5.132  1.00  0.00           H  \\nATOM   1760  HG2 GLN A 117     -13.159   0.330   4.379  1.00  0.00           H  \\nATOM   1761 HE21 GLN A 117     -13.969   0.007   6.587  1.00  0.00           H  \\nATOM   1762 HE22 GLN A 117     -13.411  -1.389   7.510  1.00  0.00           H  \\nATOM   1763  N   GLY A 118     -10.619  -3.020   1.298  1.00  0.00           N  \\nATOM   1764  CA  GLY A 118      -9.825  -3.405   0.142  1.00  0.00           C  \\nATOM   1765  C   GLY A 118     -10.668  -3.781  -1.078  1.00  0.00           C  \\nATOM   1766  O   GLY A 118     -11.906  -3.776  -1.075  1.00  0.00           O  \\nATOM   1767  H   GLY A 118     -10.807  -3.725   1.996  1.00  0.00           H  \\nATOM   1768  HA3 GLY A 118      -9.165  -2.582  -0.131  1.00  0.00           H  \\nATOM   1769  HA2 GLY A 118      -9.202  -4.261   0.405  1.00  0.00           H  \\nATOM   1770  N   GLY A 119      -9.970  -4.189  -2.134  1.00  0.00           N  \\nATOM   1771  CA  GLY A 119     -10.571  -4.549  -3.412  1.00  0.00           C  \\nATOM   1772  C   GLY A 119     -10.892  -3.336  -4.269  1.00  0.00           C  \\nATOM   1773  O   GLY A 119      -9.984  -2.634  -4.704  1.00  0.00           O  \\nATOM   1774  H   GLY A 119      -8.962  -4.180  -2.075  1.00  0.00           H  \\nATOM   1775  HA3 GLY A 119      -9.840  -5.131  -3.972  1.00  0.00           H  \\nATOM   1776  HA2 GLY A 119     -11.464  -5.158  -3.266  1.00  0.00           H  \\nATOM   1777  N   THR A 120     -12.171  -3.115  -4.564  1.00  0.00           N  \\nATOM   1778  CA  THR A 120     -12.609  -2.071  -5.503  1.00  0.00           C  \\nATOM   1779  C   THR A 120     -12.036  -0.707  -5.110  1.00  0.00           C  \\nATOM   1780  O   THR A 120     -12.150  -0.295  -3.958  1.00  0.00           O  \\nATOM   1781  CB  THR A 120     -14.145  -1.987  -5.567  1.00  0.00           C  \\nATOM   1782  OG1 THR A 120     -14.681  -3.262  -5.945  1.00  0.00           O  \\nATOM   1783  CG2 THR A 120     -14.580  -0.938  -6.580  1.00  0.00           C  \\nATOM   1784  H   THR A 120     -12.881  -3.691  -4.134  1.00  0.00           H  \\nATOM   1785  HA  THR A 120     -12.248  -2.332  -6.498  1.00  0.00           H  \\nATOM   1786  HB  THR A 120     -14.537  -1.720  -4.586  1.00  0.00           H  \\nATOM   1787  HG1 THR A 120     -14.495  -3.424  -6.873  1.00  0.00           H  \\nATOM   1788 HG21 THR A 120     -15.669  -0.893  -6.611  1.00  0.00           H  \\nATOM   1789 HG22 THR A 120     -14.184   0.034  -6.289  1.00  0.00           H  \\nATOM   1790 HG23 THR A 120     -14.200  -1.205  -7.565  1.00  0.00           H  \\nATOM   1791  N   ASN A 121     -11.448   0.003  -6.077  1.00  0.00           N  \\nATOM   1792  CA  ASN A 121     -10.777   1.299  -5.902  1.00  0.00           C  \\nATOM   1793  C   ASN A 121      -9.494   1.281  -5.054  1.00  0.00           C  \\nATOM   1794  O   ASN A 121      -8.929   2.353  -4.814  1.00  0.00           O  \\nATOM   1795  CB  ASN A 121     -11.760   2.293  -5.312  1.00  0.00           C  \\nATOM   1796  CG  ASN A 121     -12.964   2.506  -6.187  1.00  0.00           C  \\nATOM   1797  OD1 ASN A 121     -12.861   2.487  -7.419  1.00  0.00           O  \\nATOM   1798  ND2 ASN A 121     -14.103   2.707  -5.575  1.00  0.00           N  \\nATOM   1799  H   ASN A 121     -11.416  -0.400  -7.002  1.00  0.00           H  \\nATOM   1800  HA  ASN A 121     -10.447   1.604  -6.896  1.00  0.00           H  \\nATOM   1801  HB3 ASN A 121     -12.093   1.939  -4.335  1.00  0.00           H  \\nATOM   1802  HB2 ASN A 121     -11.262   3.251  -5.160  1.00  0.00           H  \\nATOM   1803 HD21 ASN A 121     -14.937   2.855  -6.107  1.00  0.00           H  \\nATOM   1804 HD22 ASN A 121     -14.139   2.714  -4.576  1.00  0.00           H  \\nATOM   1805  N   ALA A 122      -8.991   0.113  -4.641  1.00  0.00           N  \\nATOM   1806  CA  ALA A 122      -7.664   0.041  -4.047  1.00  0.00           C  \\nATOM   1807  C   ALA A 122      -6.612   0.525  -5.056  1.00  0.00           C  \\nATOM   1808  O   ALA A 122      -6.688   0.265  -6.260  1.00  0.00           O  \\nATOM   1809  CB  ALA A 122      -7.369  -1.369  -3.531  1.00  0.00           C  \\nATOM   1810  H   ALA A 122      -9.478  -0.753  -4.826  1.00  0.00           H  \\nATOM   1811  HA  ALA A 122      -7.651   0.716  -3.192  1.00  0.00           H  \\nATOM   1812  HB1 ALA A 122      -6.387  -1.390  -3.059  1.00  0.00           H  \\nATOM   1813  HB2 ALA A 122      -8.118  -1.659  -2.794  1.00  0.00           H  \\nATOM   1814  HB3 ALA A 122      -7.378  -2.073  -4.363  1.00  0.00           H  \\nATOM   1815  N   ILE A 123      -5.644   1.270  -4.544  1.00  0.00           N  \\nATOM   1816  CA  ILE A 123      -4.539   1.851  -5.284  1.00  0.00           C  \\nATOM   1817  C   ILE A 123      -3.444   0.806  -5.375  1.00  0.00           C  \\nATOM   1818  O   ILE A 123      -3.020   0.278  -4.353  1.00  0.00           O  \\nATOM   1819  CB  ILE A 123      -4.005   3.129  -4.611  1.00  0.00           C  \\nATOM   1820  CG1 ILE A 123      -5.079   4.220  -4.610  1.00  0.00           C  \\nATOM   1821  CG2 ILE A 123      -2.748   3.615  -5.316  1.00  0.00           C  \\nATOM   1822  CD1 ILE A 123      -4.732   5.416  -3.754  1.00  0.00           C  \\nATOM   1823  H   ILE A 123      -5.583   1.308  -3.536  1.00  0.00           H  \\nATOM   1824  HA  ILE A 123      -4.864   2.113  -6.291  1.00  0.00           H  \\nATOM   1825  HB  ILE A 123      -3.768   2.919  -3.569  1.00  0.00           H  \\nATOM   1826 HG13 ILE A 123      -5.246   4.566  -5.629  1.00  0.00           H  \\nATOM   1827 HG12 ILE A 123      -6.020   3.804  -4.248  1.00  0.00           H  \\nATOM   1828 HG21 ILE A 123      -2.384   4.519  -4.828  1.00  0.00           H  \\nATOM   1829 HG22 ILE A 123      -1.981   2.843  -5.266  1.00  0.00           H  \\nATOM   1830 HG23 ILE A 123      -2.977   3.833  -6.359  1.00  0.00           H  \\nATOM   1831 HD11 ILE A 123      -5.541   6.145  -3.803  1.00  0.00           H  \\nATOM   1832 HD12 ILE A 123      -4.595   5.097  -2.720  1.00  0.00           H  \\nATOM   1833 HD13 ILE A 123      -3.812   5.870  -4.119  1.00  0.00           H  \\nATOM   1834  N   LEU A 124      -2.971   0.550  -6.589  1.00  0.00           N  \\nATOM   1835  CA  LEU A 124      -1.894  -0.381  -6.865  1.00  0.00           C  \\nATOM   1836  C   LEU A 124      -0.588   0.352  -7.132  1.00  0.00           C  \\nATOM   1837  O   LEU A 124      -0.598   1.410  -7.761  1.00  0.00           O  \\nATOM   1838  CB  LEU A 124      -2.252  -1.260  -8.069  1.00  0.00           C  \\nATOM   1839  CG  LEU A 124      -3.521  -2.109  -7.920  1.00  0.00           C  \\nATOM   1840  CD1 LEU A 124      -3.696  -2.984  -9.154  1.00  0.00           C  \\nATOM   1841  CD2 LEU A 124      -3.420  -2.955  -6.659  1.00  0.00           C  \\nATOM   1842  H   LEU A 124      -3.379   1.043  -7.370  1.00  0.00           H  \\nATOM   1843  HA  LEU A 124      -1.757  -0.987  -5.970  1.00  0.00           H  \\nATOM   1844  HB3 LEU A 124      -2.383  -0.620  -8.940  1.00  0.00           H  \\nATOM   1845  HB2 LEU A 124      -1.421  -1.938  -8.265  1.00  0.00           H  \\nATOM   1846  HG  LEU A 124      -4.390  -1.455  -7.849  1.00  0.00           H  \\nATOM   1847 HD11 LEU A 124      -4.598  -3.587  -9.047  1.00  0.00           H  \\nATOM   1848 HD12 LEU A 124      -3.785  -2.352 -10.038  1.00  0.00           H  \\nATOM   1849 HD13 LEU A 124      -2.832  -3.639  -9.260  1.00  0.00           H  \\nATOM   1850 HD21 LEU A 124      -4.322  -3.558  -6.552  1.00  0.00           H  \\nATOM   1851 HD22 LEU A 124      -2.552  -3.611  -6.730  1.00  0.00           H  \\nATOM   1852 HD23 LEU A 124      -3.313  -2.304  -5.791  1.00  0.00           H  \\nATOM   1853  N   ALA A 125       0.531  -0.255  -6.745  1.00  0.00           N  \\nATOM   1854  CA  ALA A 125       1.867   0.229  -7.077  1.00  0.00           C  \\nATOM   1855  C   ALA A 125       2.861  -0.935  -7.252  1.00  0.00           C  \\nATOM   1856  O   ALA A 125       2.704  -1.985  -6.627  1.00  0.00           O  \\nATOM   1857  CB  ALA A 125       2.329   1.219  -6.004  1.00  0.00           C  \\nATOM   1858  H   ALA A 125       0.450  -1.126  -6.239  1.00  0.00           H  \\nATOM   1859  HA  ALA A 125       1.806   0.767  -8.023  1.00  0.00           H  \\nATOM   1860  HB1 ALA A 125       3.262   1.697  -6.302  1.00  0.00           H  \\nATOM   1861  HB2 ALA A 125       2.495   0.681  -5.070  1.00  0.00           H  \\nATOM   1862  HB3 ALA A 125       1.572   1.989  -5.853  1.00  0.00           H  \\nATOM   1863  N   PRO A 126       3.888  -0.781  -8.103  1.00  0.00           N  \\nATOM   1864  CA  PRO A 126       4.891  -1.805  -8.329  1.00  0.00           C  \\nATOM   1865  C   PRO A 126       5.883  -1.867  -7.168  1.00  0.00           C  \\nATOM   1866  O   PRO A 126       6.600  -0.909  -6.876  1.00  0.00           O  \\nATOM   1867  CB  PRO A 126       5.563  -1.346  -9.627  1.00  0.00           C  \\nATOM   1868  CG  PRO A 126       5.454   0.140  -9.589  1.00  0.00           C  \\nATOM   1869  CD  PRO A 126       4.114   0.401  -8.955  1.00  0.00           C  \\nATOM   1870  HA  PRO A 126       4.413  -2.778  -8.453  1.00  0.00           H  \\nATOM   1871  HB3 PRO A 126       6.605  -1.697  -9.656  1.00  0.00           H  \\nATOM   1872  HB2 PRO A 126       5.050  -1.790 -10.493  1.00  0.00           H  \\nATOM   1873  HG3 PRO A 126       6.286   0.567  -9.011  1.00  0.00           H  \\nATOM   1874  HG2 PRO A 126       5.528   0.552 -10.606  1.00  0.00           H  \\nATOM   1875  HD3 PRO A 126       4.165   1.320  -8.353  1.00  0.00           H  \\nATOM   1876  HD2 PRO A 126       3.348   0.493  -9.739  1.00  0.00           H  \\nATOM   1877  N   VAL A 127       5.982  -3.043  -6.565  1.00  0.00           N  \\nATOM   1878  CA  VAL A 127       6.985  -3.395  -5.559  1.00  0.00           C  \\nATOM   1879  C   VAL A 127       7.608  -4.745  -5.917  1.00  0.00           C  \\nATOM   1880  O   VAL A 127       7.159  -5.427  -6.842  1.00  0.00           O  \\nATOM   1881  CB  VAL A 127       6.352  -3.468  -4.156  1.00  0.00           C  \\nATOM   1882  CG1 VAL A 127       5.677  -2.150  -3.808  1.00  0.00           C  \\nATOM   1883  CG2 VAL A 127       5.355  -4.616  -4.097  1.00  0.00           C  \\nATOM   1884  H   VAL A 127       5.340  -3.769  -6.850  1.00  0.00           H  \\nATOM   1885  HA  VAL A 127       7.779  -2.649  -5.586  1.00  0.00           H  \\nATOM   1886  HB  VAL A 127       7.139  -3.631  -3.420  1.00  0.00           H  \\nATOM   1887 HG11 VAL A 127       5.235  -2.219  -2.813  1.00  0.00           H  \\nATOM   1888 HG12 VAL A 127       6.416  -1.349  -3.821  1.00  0.00           H  \\nATOM   1889 HG13 VAL A 127       4.896  -1.937  -4.537  1.00  0.00           H  \\nATOM   1890 HG21 VAL A 127       4.912  -4.662  -3.103  1.00  0.00           H  \\nATOM   1891 HG22 VAL A 127       4.571  -4.456  -4.838  1.00  0.00           H  \\nATOM   1892 HG23 VAL A 127       5.868  -5.554  -4.309  1.00  0.00           H  \\nATOM   1893  N   ASN A 128       8.679  -5.149  -5.237  1.00  0.00           N  \\nATOM   1894  CA  ASN A 128       9.275  -6.466  -5.470  1.00  0.00           C  \\nATOM   1895  C   ASN A 128       8.638  -7.536  -4.561  1.00  0.00           C  \\nATOM   1896  O   ASN A 128       7.823  -7.239  -3.684  1.00  0.00           O  \\nATOM   1897  CB  ASN A 128      10.807  -6.368  -5.434  1.00  0.00           C  \\nATOM   1898  CG  ASN A 128      11.367  -6.050  -4.067  1.00  0.00           C  \\nATOM   1899  OD1 ASN A 128      10.881  -6.522  -3.065  1.00  0.00           O  \\nATOM   1900  ND2 ASN A 128      12.421  -5.278  -3.980  1.00  0.00           N  \\nATOM   1901  H   ASN A 128       8.972  -4.624  -4.426  1.00  0.00           H  \\nATOM   1902  HA  ASN A 128       9.043  -6.780  -6.488  1.00  0.00           H  \\nATOM   1903  HB3 ASN A 128      11.136  -5.617  -6.152  1.00  0.00           H  \\nATOM   1904  HB2 ASN A 128      11.221  -7.326  -5.747  1.00  0.00           H  \\nATOM   1905 HD21 ASN A 128      12.791  -4.833  -4.807  1.00  0.00           H  \\nATOM   1906 HD22 ASN A 128      12.637  -4.956  -3.046  1.00  0.00           H  \\nATOM   1907  N   LEU A 129       8.968  -8.806  -4.795  1.00  0.00           N  \\nATOM   1908  CA  LEU A 129       8.395  -9.920  -4.032  1.00  0.00           C  \\nATOM   1909  C   LEU A 129       8.880  -9.935  -2.577  1.00  0.00           C  \\nATOM   1910  O   LEU A 129       8.076 -10.115  -1.665  1.00  0.00           O  \\nATOM   1911  CB  LEU A 129       8.752 -11.252  -4.704  1.00  0.00           C  \\nATOM   1912  CG  LEU A 129       8.201 -12.510  -4.022  1.00  0.00           C  \\nATOM   1913  CD1 LEU A 129       6.682 -12.424  -3.948  1.00  0.00           C  \\nATOM   1914  CD2 LEU A 129       8.641 -13.743  -4.797  1.00  0.00           C  \\nATOM   1915  H   LEU A 129       9.701  -9.001  -5.463  1.00  0.00           H  \\nATOM   1916  HA  LEU A 129       7.313  -9.800  -3.999  1.00  0.00           H  \\nATOM   1917  HB3 LEU A 129       8.376 -11.237  -5.726  1.00  0.00           H  \\nATOM   1918  HB2 LEU A 129       9.838 -11.343  -4.740  1.00  0.00           H  \\nATOM   1919  HG  LEU A 129       8.581 -12.568  -3.002  1.00  0.00           H  \\nATOM   1920 HD11 LEU A 129       6.290 -13.318  -3.463  1.00  0.00           H  \\nATOM   1921 HD12 LEU A 129       6.395 -11.545  -3.372  1.00  0.00           H  \\nATOM   1922 HD13 LEU A 129       6.273 -12.349  -4.955  1.00  0.00           H  \\nATOM   1923 HD21 LEU A 129       8.250 -14.638  -4.312  1.00  0.00           H  \\nATOM   1924 HD22 LEU A 129       8.260 -13.687  -5.817  1.00  0.00           H  \\nATOM   1925 HD23 LEU A 129       9.730 -13.789  -4.819  1.00  0.00           H  \\nATOM   1926  N   ALA A 130      10.176  -9.728  -2.349  1.00  0.00           N  \\nATOM   1927  CA  ALA A 130      10.730  -9.709  -1.001  1.00  0.00           C  \\nATOM   1928  C   ALA A 130      10.058  -8.652  -0.104  1.00  0.00           C  \\nATOM   1929  O   ALA A 130       9.712  -8.948   1.039  1.00  0.00           O  \\nATOM   1930  CB  ALA A 130      12.238  -9.525  -1.124  1.00  0.00           C  \\nATOM   1931  H   ALA A 130      10.796  -9.553  -3.126  1.00  0.00           H  \\nATOM   1932  HA  ALA A 130      10.547 -10.678  -0.537  1.00  0.00           H  \\nATOM   1933  HB1 ALA A 130      12.663 -10.337  -1.713  1.00  0.00           H  \\nATOM   1934  HB2 ALA A 130      12.464  -8.572  -1.601  1.00  0.00           H  \\nATOM   1935  HB3 ALA A 130      12.680  -9.560  -0.128  1.00  0.00           H  \\nATOM   1936  N   SER A 131       9.765  -7.470  -0.648  1.00  0.00           N  \\nATOM   1937  CA  SER A 131       9.046  -6.396   0.033  1.00  0.00           C  \\nATOM   1938  C   SER A 131       7.611  -6.781   0.367  1.00  0.00           C  \\nATOM   1939  O   SER A 131       7.186  -6.569   1.500  1.00  0.00           O  \\nATOM   1940  CB  SER A 131       9.021  -5.144  -0.842  1.00  0.00           C  \\nATOM   1941  OG  SER A 131      10.311  -4.589  -0.928  1.00  0.00           O  \\nATOM   1942  H   SER A 131      10.074  -7.294  -1.593  1.00  0.00           H  \\nATOM   1943  HA  SER A 131       9.547  -6.168   0.974  1.00  0.00           H  \\nATOM   1944  HB3 SER A 131       8.652  -5.384  -1.839  1.00  0.00           H  \\nATOM   1945  HB2 SER A 131       8.349  -4.416  -0.388  1.00  0.00           H  \\nATOM   1946  HG  SER A 131      10.448  -4.121  -0.102  1.00  0.00           H  \\nATOM   1947  N   GLN A 132       6.897  -7.401  -0.578  1.00  0.00           N  \\nATOM   1948  CA  GLN A 132       5.528  -7.870  -0.346  1.00  0.00           C  \\nATOM   1949  C   GLN A 132       5.455  -8.945   0.747  1.00  0.00           C  \\nATOM   1950  O   GLN A 132       4.621  -8.890   1.650  1.00  0.00           O  \\nATOM   1951  CB  GLN A 132       4.931  -8.414  -1.646  1.00  0.00           C  \\nATOM   1952  CG  GLN A 132       3.460  -8.784  -1.549  1.00  0.00           C  \\nATOM   1953  CD  GLN A 132       2.855  -9.109  -2.901  1.00  0.00           C  \\nATOM   1954  OE1 GLN A 132       3.311 -10.019  -3.598  1.00  0.00           O  \\nATOM   1955  NE2 GLN A 132       1.822  -8.366  -3.280  1.00  0.00           N  \\nATOM   1956  H   GLN A 132       7.303  -7.527  -1.494  1.00  0.00           H  \\nATOM   1957  HA  GLN A 132       4.952  -7.022   0.024  1.00  0.00           H  \\nATOM   1958  HB3 GLN A 132       5.039  -7.671  -2.435  1.00  0.00           H  \\nATOM   1959  HB2 GLN A 132       5.481  -9.303  -1.955  1.00  0.00           H  \\nATOM   1960  HG3 GLN A 132       3.358  -9.660  -0.909  1.00  0.00           H  \\nATOM   1961  HG2 GLN A 132       2.913  -7.943  -1.122  1.00  0.00           H  \\nATOM   1962 HE21 GLN A 132       1.381  -8.534  -4.163  1.00  0.00           H  \\nATOM   1963 HE22 GLN A 132       1.484  -7.637  -2.684  1.00  0.00           H  \\nATOM   1964  N   ASN A 133       6.380  -9.907   0.719  1.00  0.00           N  \\nATOM   1965  CA  ASN A 133       6.455 -10.945   1.748  1.00  0.00           C  \\nATOM   1966  C   ASN A 133       6.823 -10.355   3.119  1.00  0.00           C  \\nATOM   1967  O   ASN A 133       6.248 -10.736   4.139  1.00  0.00           O  \\nATOM   1968  CB  ASN A 133       7.450 -12.019   1.343  1.00  0.00           C  \\nATOM   1969  CG  ASN A 133       6.916 -12.925   0.269  1.00  0.00           C  \\nATOM   1970  OD1 ASN A 133       5.702 -12.991   0.041  1.00  0.00           O  \\nATOM   1971  ND2 ASN A 133       7.799 -13.626  -0.395  1.00  0.00           N  \\nATOM   1972  H   ASN A 133       7.020  -9.936  -0.061  1.00  0.00           H  \\nATOM   1973  HA  ASN A 133       5.477 -11.415   1.853  1.00  0.00           H  \\nATOM   1974  HB3 ASN A 133       8.366 -11.549   0.984  1.00  0.00           H  \\nATOM   1975  HB2 ASN A 133       7.710 -12.620   2.214  1.00  0.00           H  \\nATOM   1976 HD21 ASN A 133       7.501 -14.247  -1.121  1.00  0.00           H  \\nATOM   1977 HD22 ASN A 133       8.771 -13.542  -0.177  1.00  0.00           H  \\nATOM   1978  N   SER A 134       7.760  -9.402   3.139  1.00  0.00           N  \\nATOM   1979  CA  SER A 134       8.184  -8.716   4.357  1.00  0.00           C  \\nATOM   1980  C   SER A 134       7.043  -7.907   4.984  1.00  0.00           C  \\nATOM   1981  O   SER A 134       6.833  -8.013   6.192  1.00  0.00           O  \\nATOM   1982  CB  SER A 134       9.354  -7.801   4.051  1.00  0.00           C  \\nATOM   1983  OG  SER A 134      10.470  -8.536   3.631  1.00  0.00           O  \\nATOM   1984  H   SER A 134       8.190  -9.136   2.265  1.00  0.00           H  \\nATOM   1985  HA  SER A 134       8.486  -9.470   5.085  1.00  0.00           H  \\nATOM   1986  HB3 SER A 134       9.068  -7.092   3.274  1.00  0.00           H  \\nATOM   1987  HB2 SER A 134       9.608  -7.226   4.941  1.00  0.00           H  \\nATOM   1988  HG  SER A 134      10.282  -8.812   2.731  1.00  0.00           H  \\nATOM   1989  N   GLN A 135       6.259  -7.170   4.185  1.00  0.00           N  \\nATOM   1990  CA  GLN A 135       5.072  -6.441   4.656  1.00  0.00           C  \\nATOM   1991  C   GLN A 135       4.049  -7.399   5.279  1.00  0.00           C  \\nATOM   1992  O   GLN A 135       3.587  -7.159   6.393  1.00  0.00           O  \\nATOM   1993  CB  GLN A 135       4.446  -5.668   3.484  1.00  0.00           C  \\nATOM   1994  CG  GLN A 135       3.220  -4.802   3.872  1.00  0.00           C  \\nATOM   1995  CD  GLN A 135       2.364  -4.415   2.663  1.00  0.00           C  \\nATOM   1996  OE1 GLN A 135       2.148  -5.242   1.798  1.00  0.00           O  \\nATOM   1997  NE2 GLN A 135       1.805  -3.227   2.572  1.00  0.00           N  \\nATOM   1998  H   GLN A 135       6.474  -7.121   3.200  1.00  0.00           H  \\nATOM   1999  HA  GLN A 135       5.372  -5.726   5.422  1.00  0.00           H  \\nATOM   2000  HB3 GLN A 135       5.203  -5.010   3.056  1.00  0.00           H  \\nATOM   2001  HB2 GLN A 135       4.176  -6.390   2.712  1.00  0.00           H  \\nATOM   2002  HG3 GLN A 135       3.567  -3.899   4.374  1.00  0.00           H  \\nATOM   2003  HG2 GLN A 135       2.575  -5.345   4.563  1.00  0.00           H  \\nATOM   2004 HE21 GLN A 135       1.799  -2.495   3.267  1.00  0.00           H  \\nATOM   2005 HE22 GLN A 135       1.363  -3.020   1.688  1.00  0.00           H  \\nATOM   2006  N   GLY A 136       3.767  -8.532   4.625  1.00  0.00           N  \\nATOM   2007  CA  GLY A 136       2.876  -9.561   5.162  1.00  0.00           C  \\nATOM   2008  C   GLY A 136       3.328 -10.098   6.525  1.00  0.00           C  \\nATOM   2009  O   GLY A 136       2.511 -10.227   7.438  1.00  0.00           O  \\nATOM   2010  H   GLY A 136       4.130  -8.643   3.689  1.00  0.00           H  \\nATOM   2011  HA3 GLY A 136       2.844 -10.393   4.457  1.00  0.00           H  \\nATOM   2012  HA2 GLY A 136       1.872  -9.148   5.264  1.00  0.00           H  \\nATOM   2013  N   GLY A 137       4.628 -10.359   6.698  1.00  0.00           N  \\nATOM   2014  CA  GLY A 137       5.210 -10.730   7.991  1.00  0.00           C  \\nATOM   2015  C   GLY A 137       4.972  -9.673   9.074  1.00  0.00           C  \\nATOM   2016  O   GLY A 137       4.449  -9.995  10.143  1.00  0.00           O  \\nATOM   2017  H   GLY A 137       5.243 -10.260   5.902  1.00  0.00           H  \\nATOM   2018  HA3 GLY A 137       4.786 -11.675   8.326  1.00  0.00           H  \\nATOM   2019  HA2 GLY A 137       6.286 -10.852   7.869  1.00  0.00           H  \\nATOM   2020  N   VAL A 138       5.286  -8.408   8.780  1.00  0.00           N  \\nATOM   2021  CA  VAL A 138       5.105  -7.287   9.718  1.00  0.00           C  \\nATOM   2022  C   VAL A 138       3.626  -7.084  10.063  1.00  0.00           C  \\nATOM   2023  O   VAL A 138       3.291  -6.953  11.238  1.00  0.00           O  \\nATOM   2024  CB  VAL A 138       5.668  -5.987   9.115  1.00  0.00           C  \\nATOM   2025  CG1 VAL A 138       5.307  -4.794   9.987  1.00  0.00           C  \\nATOM   2026  CG2 VAL A 138       7.177  -6.101   8.955  1.00  0.00           C  \\nATOM   2027  H   VAL A 138       5.673  -8.210   7.869  1.00  0.00           H  \\nATOM   2028  HA  VAL A 138       5.612  -7.531  10.652  1.00  0.00           H  \\nATOM   2029  HB  VAL A 138       5.211  -5.822   8.139  1.00  0.00           H  \\nATOM   2030 HG11 VAL A 138       5.713  -3.884   9.546  1.00  0.00           H  \\nATOM   2031 HG12 VAL A 138       4.222  -4.710  10.058  1.00  0.00           H  \\nATOM   2032 HG13 VAL A 138       5.727  -4.932  10.984  1.00  0.00           H  \\nATOM   2033 HG21 VAL A 138       7.568  -5.179   8.528  1.00  0.00           H  \\nATOM   2034 HG22 VAL A 138       7.633  -6.272   9.930  1.00  0.00           H  \\nATOM   2035 HG23 VAL A 138       7.410  -6.936   8.294  1.00  0.00           H  \\nATOM   2036  N   LEU A 139       2.724  -7.133   9.077  1.00  0.00           N  \\nATOM   2037  CA  LEU A 139       1.280  -7.011   9.299  1.00  0.00           C  \\nATOM   2038  C   LEU A 139       0.729  -8.161  10.146  1.00  0.00           C  \\nATOM   2039  O   LEU A 139      -0.066  -7.925  11.050  1.00  0.00           O  \\nATOM   2040  CB  LEU A 139       0.544  -6.968   7.954  1.00  0.00           C  \\nATOM   2041  CG  LEU A 139       0.682  -5.662   7.162  1.00  0.00           C  \\nATOM   2042  CD1 LEU A 139       0.066  -5.838   5.781  1.00  0.00           C  \\nATOM   2043  CD2 LEU A 139       0.005  -4.532   7.923  1.00  0.00           C  \\nATOM   2044  H   LEU A 139       3.054  -7.240   8.129  1.00  0.00           H  \\nATOM   2045  HA  LEU A 139       1.090  -6.098   9.863  1.00  0.00           H  \\nATOM   2046  HB3 LEU A 139       0.917  -7.776   7.328  1.00  0.00           H  \\nATOM   2047  HB2 LEU A 139      -0.518  -7.136   8.133  1.00  0.00           H  \\nATOM   2048  HG  LEU A 139       1.738  -5.427   7.028  1.00  0.00           H  \\nATOM   2049 HD11 LEU A 139       0.164  -4.909   5.217  1.00  0.00           H  \\nATOM   2050 HD12 LEU A 139       0.582  -6.639   5.251  1.00  0.00           H  \\nATOM   2051 HD13 LEU A 139      -0.989  -6.090   5.883  1.00  0.00           H  \\nATOM   2052 HD21 LEU A 139       0.104  -3.603   7.360  1.00  0.00           H  \\nATOM   2053 HD22 LEU A 139      -1.052  -4.765   8.056  1.00  0.00           H  \\nATOM   2054 HD23 LEU A 139       0.477  -4.417   8.899  1.00  0.00           H  \\nATOM   2055  N   ASN A 140       1.165  -9.401   9.906  1.00  0.00           N  \\nATOM   2056  CA  ASN A 140       0.760 -10.536  10.737  1.00  0.00           C  \\nATOM   2057  C   ASN A 140       1.237 -10.385  12.188  1.00  0.00           C  \\nATOM   2058  O   ASN A 140       0.470 -10.672  13.111  1.00  0.00           O  \\nATOM   2059  CB  ASN A 140       1.273 -11.834  10.141  1.00  0.00           C  \\nATOM   2060  CG  ASN A 140       0.507 -12.251   8.916  1.00  0.00           C  \\nATOM   2061  OD1 ASN A 140      -0.633 -11.821   8.706  1.00  0.00           O  \\nATOM   2062  ND2 ASN A 140       1.110 -13.080   8.103  1.00  0.00           N  \\nATOM   2063  H   ASN A 140       1.796  -9.553   9.133  1.00  0.00           H  \\nATOM   2064  HA  ASN A 140      -0.329 -10.569  10.780  1.00  0.00           H  \\nATOM   2065  HB3 ASN A 140       2.325 -11.722   9.876  1.00  0.00           H  \\nATOM   2066  HB2 ASN A 140       1.206 -12.628  10.885  1.00  0.00           H  \\nATOM   2067 HD21 ASN A 140       0.648 -13.391   7.273  1.00  0.00           H  \\nATOM   2068 HD22 ASN A 140       2.033 -13.402   8.313  1.00  0.00           H  \\nATOM   2069  N   GLY A 141       2.462  -9.887  12.390  1.00  0.00           N  \\nATOM   2070  CA  GLY A 141       2.977  -9.525  13.711  1.00  0.00           C  \\nATOM   2071  C   GLY A 141       2.135  -8.434  14.371  1.00  0.00           C  \\nATOM   2072  O   GLY A 141       1.652  -8.626  15.488  1.00  0.00           O  \\nATOM   2073  H   GLY A 141       3.045  -9.699  11.588  1.00  0.00           H  \\nATOM   2074  HA3 GLY A 141       3.995  -9.153  13.605  1.00  0.00           H  \\nATOM   2075  HA2 GLY A 141       2.986 -10.404  14.358  1.00  0.00           H  \\nATOM   2076  N   PHE A 142       1.864  -7.343  13.646  1.00  0.00           N  \\nATOM   2077  CA  PHE A 142       0.998  -6.256  14.101  1.00  0.00           C  \\nATOM   2078  C   PHE A 142      -0.388  -6.773  14.494  1.00  0.00           C  \\nATOM   2079  O   PHE A 142      -0.848  -6.483  15.592  1.00  0.00           O  \\nATOM   2080  CB  PHE A 142       0.895  -5.169  13.018  1.00  0.00           C  \\nATOM   2081  CG  PHE A 142      -0.046  -4.031  13.377  1.00  0.00           C  \\nATOM   2082  CD1 PHE A 142      -1.385  -4.053  12.941  1.00  0.00           C  \\nATOM   2083  CD2 PHE A 142       0.413  -2.951  14.153  1.00  0.00           C  \\nATOM   2084  CE1 PHE A 142      -2.270  -3.021  13.306  1.00  0.00           C  \\nATOM   2085  CE2 PHE A 142      -0.463  -1.909  14.498  1.00  0.00           C  \\nATOM   2086  CZ  PHE A 142      -1.807  -1.947  14.087  1.00  0.00           C  \\nATOM   2087  H   PHE A 142       2.304  -7.249  12.742  1.00  0.00           H  \\nATOM   2088  HA  PHE A 142       1.442  -5.808  14.990  1.00  0.00           H  \\nATOM   2089  HB3 PHE A 142       0.549  -5.613  12.085  1.00  0.00           H  \\nATOM   2090  HB2 PHE A 142       1.889  -4.763  12.832  1.00  0.00           H  \\nATOM   2091  HD1 PHE A 142      -1.731  -4.864  12.318  1.00  0.00           H  \\nATOM   2092  HD2 PHE A 142       1.442  -2.905  14.481  1.00  0.00           H  \\nATOM   2093  HE1 PHE A 142      -3.297  -3.041  12.972  1.00  0.00           H  \\nATOM   2094  HE2 PHE A 142      -0.096  -1.072  15.072  1.00  0.00           H  \\nATOM   2095  HZ  PHE A 142      -2.471  -1.138  14.354  1.00  0.00           H  \\nATOM   2096  N   TYR A 143      -1.056  -7.564  13.653  1.00  0.00           N  \\nATOM   2097  CA  TYR A 143      -2.395  -8.064  13.963  1.00  0.00           C  \\nATOM   2098  C   TYR A 143      -2.410  -8.978  15.189  1.00  0.00           C  \\nATOM   2099  O   TYR A 143      -3.325  -8.872  16.006  1.00  0.00           O  \\nATOM   2100  CB  TYR A 143      -2.974  -8.802  12.754  1.00  0.00           C  \\nATOM   2101  CG  TYR A 143      -3.240  -7.908  11.563  1.00  0.00           C  \\nATOM   2102  CD1 TYR A 143      -3.059  -8.394  10.276  1.00  0.00           C  \\nATOM   2103  CD2 TYR A 143      -3.666  -6.603  11.757  1.00  0.00           C  \\nATOM   2104  CE1 TYR A 143      -3.302  -7.578   9.189  1.00  0.00           C  \\nATOM   2105  CE2 TYR A 143      -3.910  -5.787  10.670  1.00  0.00           C  \\nATOM   2106  CZ  TYR A 143      -3.729  -6.270   9.390  1.00  0.00           C  \\nATOM   2107  OH  TYR A 143      -3.971  -5.457   8.307  1.00  0.00           O  \\nATOM   2108  H   TYR A 143      -0.652  -7.761  12.748  1.00  0.00           H  \\nATOM   2109  HA  TYR A 143      -3.034  -7.212  14.200  1.00  0.00           H  \\nATOM   2110  HB3 TYR A 143      -2.283  -9.588  12.442  1.00  0.00           H  \\nATOM   2111  HB2 TYR A 143      -3.910  -9.283  13.035  1.00  0.00           H  \\nATOM   2112  HD1 TYR A 143      -2.724  -9.421  10.123  1.00  0.00           H  \\nATOM   2113  HD2 TYR A 143      -3.808  -6.221  12.768  1.00  0.00           H  \\nATOM   2114  HE1 TYR A 143      -3.160  -7.960   8.178  1.00  0.00           H  \\nATOM   2115  HE2 TYR A 143      -4.244  -4.760  10.822  1.00  0.00           H  \\nATOM   2116  HH  TYR A 143      -3.682  -5.899   7.505  1.00  0.00           H  \\nATOM   2117  N   SER A 144      -1.407  -9.851  15.324  1.00  0.00           N  \\nATOM   2118  CA  SER A 144      -1.276 -10.759  16.468  1.00  0.00           C  \\nATOM   2119  C   SER A 144      -1.010  -9.986  17.758  1.00  0.00           C  \\nATOM   2120  O   SER A 144      -1.738 -10.148  18.742  1.00  0.00           O  \\nATOM   2121  CB  SER A 144      -0.156 -11.751  16.224  1.00  0.00           C  \\nATOM   2122  OG  SER A 144      -0.445 -12.575  15.128  1.00  0.00           O  \\nATOM   2123  H   SER A 144      -0.688  -9.869  14.615  1.00  0.00           H  \\nATOM   2124  HA  SER A 144      -2.207 -11.312  16.594  1.00  0.00           H  \\nATOM   2125  HB3 SER A 144       0.774 -11.213  16.043  1.00  0.00           H  \\nATOM   2126  HB2 SER A 144      -0.012 -12.363  17.113  1.00  0.00           H  \\nATOM   2127  HG  SER A 144      -0.207 -12.069  14.347  1.00  0.00           H  \\nATOM   2128  N   ALA A 145      -0.053  -9.053  17.718  1.00  0.00           N  \\nATOM   2129  CA  ALA A 145       0.334  -8.247  18.870  1.00  0.00           C  \\nATOM   2130  C   ALA A 145      -0.849  -7.419  19.379  1.00  0.00           C  \\nATOM   2131  O   ALA A 145      -1.035  -7.283  20.589  1.00  0.00           O  \\nATOM   2132  CB  ALA A 145       1.501  -7.341  18.466  1.00  0.00           C  \\nATOM   2133  H   ALA A 145       0.432  -8.883  16.849  1.00  0.00           H  \\nATOM   2134  HA  ALA A 145       0.659  -8.906  19.674  1.00  0.00           H  \\nATOM   2135  HB1 ALA A 145       2.345  -7.947  18.135  1.00  0.00           H  \\nATOM   2136  HB2 ALA A 145       1.816  -6.743  19.321  1.00  0.00           H  \\nATOM   2137  HB3 ALA A 145       1.202  -6.675  17.658  1.00  0.00           H  \\nATOM   2138  N   ASN A 146      -1.696  -6.931  18.463  1.00  0.00           N  \\nATOM   2139  CA  ASN A 146      -2.853  -6.113  18.783  1.00  0.00           C  \\nATOM   2140  C   ASN A 146      -4.203  -6.837  18.821  1.00  0.00           C  \\nATOM   2141  O   ASN A 146      -5.204  -6.171  19.058  1.00  0.00           O  \\nATOM   2142  CB  ASN A 146      -2.923  -4.957  17.801  1.00  0.00           C  \\nATOM   2143  CG  ASN A 146      -1.819  -3.957  18.007  1.00  0.00           C  \\nATOM   2144  OD1 ASN A 146      -1.897  -3.101  18.895  1.00  0.00           O  \\nATOM   2145  ND2 ASN A 146      -0.791  -4.049  17.202  1.00  0.00           N  \\nATOM   2146  H   ASN A 146      -1.428  -7.002  17.492  1.00  0.00           H  \\nATOM   2147  HA  ASN A 146      -2.690  -5.809  19.817  1.00  0.00           H  \\nATOM   2148  HB3 ASN A 146      -2.865  -5.341  16.782  1.00  0.00           H  \\nATOM   2149  HB2 ASN A 146      -3.881  -4.449  17.906  1.00  0.00           H  \\nATOM   2150 HD21 ASN A 146      -0.026  -3.410  17.292  1.00  0.00           H  \\nATOM   2151 HD22 ASN A 146      -0.769  -4.758  16.497  1.00  0.00           H  \\nATOM   2152  N   LYS A 147      -4.240  -8.176  18.770  1.00  0.00           N  \\nATOM   2153  CA  LYS A 147      -5.483  -8.976  18.806  1.00  0.00           C  \\nATOM   2154  C   LYS A 147      -6.518  -8.500  17.773  1.00  0.00           C  \\nATOM   2155  O   LYS A 147      -7.697  -8.366  18.080  1.00  0.00           O  \\nATOM   2156  CB  LYS A 147      -6.097  -8.938  20.206  1.00  0.00           C  \\nATOM   2157  CG  LYS A 147      -5.144  -9.343  21.323  1.00  0.00           C  \\nATOM   2158  CD  LYS A 147      -4.682 -10.783  21.160  1.00  0.00           C  \\nATOM   2159  CE  LYS A 147      -3.755 -11.199  22.292  1.00  0.00           C  \\nATOM   2160  NZ  LYS A 147      -3.246 -12.586  22.114  1.00  0.00           N  \\nATOM   2161  H   LYS A 147      -3.385  -8.655  18.526  1.00  0.00           H  \\nATOM   2162  HA  LYS A 147      -5.218  -9.987  18.494  1.00  0.00           H  \\nATOM   2163  HB3 LYS A 147      -6.453  -7.930  20.420  1.00  0.00           H  \\nATOM   2164  HB2 LYS A 147      -6.959  -9.604  20.244  1.00  0.00           H  \\nATOM   2165  HG3 LYS A 147      -4.272  -8.687  21.313  1.00  0.00           H  \\nATOM   2166  HG2 LYS A 147      -5.645  -9.237  22.285  1.00  0.00           H  \\nATOM   2167  HD3 LYS A 147      -5.550 -11.445  21.150  1.00  0.00           H  \\nATOM   2168  HD2 LYS A 147      -4.154 -10.891  20.213  1.00  0.00           H  \\nATOM   2169  HE3 LYS A 147      -2.907 -10.517  22.337  1.00  0.00           H  \\nATOM   2170  HE2 LYS A 147      -4.290 -11.142  23.240  1.00  0.00           H  \\nATOM   2171  HZ1 LYS A 147      -2.637 -12.824  22.884  1.00  0.00           H  \\nATOM   2172  HZ2 LYS A 147      -4.024 -13.231  22.089  1.00  0.00           H  \\nATOM   2173  HZ3 LYS A 147      -2.732 -12.647  21.247  1.00  0.00           H  \\nATOM   2174  N   VAL A 148      -6.075  -8.241  16.541  1.00  0.00           N  \\nATOM   2175  CA  VAL A 148      -6.955  -7.812  15.445  1.00  0.00           C  \\nATOM   2176  C   VAL A 148      -7.574  -9.045  14.791  1.00  0.00           C  \\nATOM   2177  O   VAL A 148      -6.926  -9.741  13.998  1.00  0.00           O  \\nATOM   2178  CB  VAL A 148      -6.212  -6.934  14.423  1.00  0.00           C  \\nATOM   2179  CG1 VAL A 148      -7.126  -6.506  13.268  1.00  0.00           C  \\nATOM   2180  CG2 VAL A 148      -5.668  -5.662  15.084  1.00  0.00           C  \\nATOM   2181  H   VAL A 148      -5.107  -8.448  16.339  1.00  0.00           H  \\nATOM   2182  HA  VAL A 148      -7.761  -7.210  15.863  1.00  0.00           H  \\nATOM   2183  HB  VAL A 148      -5.377  -7.499  14.012  1.00  0.00           H  \\nATOM   2184 HG11 VAL A 148      -6.565  -5.888  12.567  1.00  0.00           H  \\nATOM   2185 HG12 VAL A 148      -7.971  -5.932  13.649  1.00  0.00           H  \\nATOM   2186 HG13 VAL A 148      -7.502  -7.378  12.732  1.00  0.00           H  \\nATOM   2187 HG21 VAL A 148      -5.165  -5.038  14.345  1.00  0.00           H  \\nATOM   2188 HG22 VAL A 148      -6.484  -5.094  15.531  1.00  0.00           H  \\nATOM   2189 HG23 VAL A 148      -4.954  -5.925  15.864  1.00  0.00           H  \\nATOM   2190  N   ALA A 149      -8.835  -9.313  15.127  1.00  0.00           N  \\nATOM   2191  CA  ALA A 149      -9.547 -10.497  14.684  1.00  0.00           C  \\nATOM   2192  C   ALA A 149     -10.089 -10.326  13.260  1.00  0.00           C  \\nATOM   2193  O   ALA A 149     -10.545  -9.252  12.862  1.00  0.00           O  \\nATOM   2194  CB  ALA A 149     -10.658 -10.844  15.679  1.00  0.00           C  \\nATOM   2195  H   ALA A 149      -9.335  -8.670  15.726  1.00  0.00           H  \\nATOM   2196  HA  ALA A 149      -8.842 -11.330  14.683  1.00  0.00           H  \\nATOM   2197  HB1 ALA A 149     -11.114 -11.797  15.409  1.00  0.00           H  \\nATOM   2198  HB2 ALA A 149     -10.250 -10.916  16.687  1.00  0.00           H  \\nATOM   2199  HB3 ALA A 149     -11.432 -10.077  15.660  1.00  0.00           H  \\nATOM   2200  N   GLN A 150     -10.054 -11.425  12.510  1.00  0.00           N  \\nATOM   2201  CA  GLN A 150     -10.719 -11.524  11.216  1.00  0.00           C  \\nATOM   2202  C   GLN A 150     -12.223 -11.754  11.397  1.00  0.00           C  \\nATOM   2203  O   GLN A 150     -12.641 -12.365  12.380  1.00  0.00           O  \\nATOM   2204  CB  GLN A 150     -10.107 -12.655  10.384  1.00  0.00           C  \\nATOM   2205  CG  GLN A 150      -8.681 -12.392   9.931  1.00  0.00           C  \\nATOM   2206  CD  GLN A 150      -8.074 -13.581   9.210  1.00  0.00           C  \\nATOM   2207  OE1 GLN A 150      -7.768 -14.607   9.824  1.00  0.00           O  \\nATOM   2208  NE2 GLN A 150      -7.897 -13.451   7.901  1.00  0.00           N  \\nATOM   2209  H   GLN A 150      -9.756 -12.283  12.950  1.00  0.00           H  \\nATOM   2210  HA  GLN A 150     -10.576 -10.577  10.695  1.00  0.00           H  \\nATOM   2211  HB3 GLN A 150     -10.111 -13.577  10.965  1.00  0.00           H  \\nATOM   2212  HB2 GLN A 150     -10.716 -12.824   9.496  1.00  0.00           H  \\nATOM   2213  HG3 GLN A 150      -8.677 -11.541   9.250  1.00  0.00           H  \\nATOM   2214  HG2 GLN A 150      -8.067 -12.173  10.805  1.00  0.00           H  \\nATOM   2215 HE21 GLN A 150      -7.501 -14.202   7.371  1.00  0.00           H  \\nATOM   2216 HE22 GLN A 150      -8.159 -12.601   7.442  1.00  0.00           H  \\nATOM   2217  N   PHE A 151     -13.023 -11.360  10.410  1.00  0.00           N  \\nATOM   2218  CA  PHE A 151     -14.429 -11.760  10.317  1.00  0.00           C  \\nATOM   2219  C   PHE A 151     -14.853 -12.054   8.881  1.00  0.00           C  \\nATOM   2220  O   PHE A 151     -14.134 -11.750   7.928  1.00  0.00           O  \\nATOM   2221  CB  PHE A 151     -15.328 -10.667  10.898  1.00  0.00           C  \\nATOM   2222  CG  PHE A 151     -15.500  -9.480   9.995  1.00  0.00           C  \\nATOM   2223  CD1 PHE A 151     -16.709  -9.244   9.357  1.00  0.00           C  \\nATOM   2224  CD2 PHE A 151     -14.453  -8.596   9.779  1.00  0.00           C  \\nATOM   2225  CE1 PHE A 151     -16.868  -8.152   8.525  1.00  0.00           C  \\nATOM   2226  CE2 PHE A 151     -14.609  -7.503   8.950  1.00  0.00           C  \\nATOM   2227  CZ  PHE A 151     -15.819  -7.281   8.322  1.00  0.00           C  \\nATOM   2228  H   PHE A 151     -12.646 -10.778   9.676  1.00  0.00           H  \\nATOM   2229  HA  PHE A 151     -14.564 -12.690  10.871  1.00  0.00           H  \\nATOM   2230  HB3 PHE A 151     -16.314 -11.081  11.107  1.00  0.00           H  \\nATOM   2231  HB2 PHE A 151     -14.913 -10.319  11.843  1.00  0.00           H  \\nATOM   2232  HD1 PHE A 151     -17.539  -9.932   9.519  1.00  0.00           H  \\nATOM   2233  HD2 PHE A 151     -13.498  -8.772  10.275  1.00  0.00           H  \\nATOM   2234  HE1 PHE A 151     -17.823  -7.979   8.031  1.00  0.00           H  \\nATOM   2235  HE2 PHE A 151     -13.779  -6.816   8.789  1.00  0.00           H  \\nATOM   2236  HZ  PHE A 151     -15.943  -6.421   7.666  1.00  0.00           H  \\nATOM   2237  N   ASP A 152     -16.015 -12.681   8.735  1.00  0.00           N  \\nATOM   2238  CA  ASP A 152     -16.634 -12.996   7.457  1.00  0.00           C  \\nATOM   2239  C   ASP A 152     -17.738 -11.966   7.163  1.00  0.00           C  \\nATOM   2240  O   ASP A 152     -18.811 -12.027   7.773  1.00  0.00           O  \\nATOM   2241  CB  ASP A 152     -17.212 -14.413   7.466  1.00  0.00           C  \\nATOM   2242  CG  ASP A 152     -17.768 -14.835   6.113  1.00  0.00           C  \\nATOM   2243  OD1 ASP A 152     -18.158 -13.976   5.358  1.00  0.00           O  \\nATOM   2244  OD2 ASP A 152     -17.797 -16.013   5.846  1.00  0.00           O  \\nATOM   2245  H   ASP A 152     -16.610 -12.790   9.544  1.00  0.00           H  \\nATOM   2246  HA  ASP A 152     -15.884 -12.965   6.666  1.00  0.00           H  \\nATOM   2247  HB3 ASP A 152     -16.436 -15.121   7.760  1.00  0.00           H  \\nATOM   2248  HB2 ASP A 152     -18.010 -14.476   8.206  1.00  0.00           H  \\nATOM   2249  N   PRO A 153     -17.530 -11.032   6.217  1.00  0.00           N  \\nATOM   2250  CA  PRO A 153     -18.520 -10.006   5.903  1.00  0.00           C  \\nATOM   2251  C   PRO A 153     -19.868 -10.568   5.443  1.00  0.00           C  \\nATOM   2252  O   PRO A 153     -20.870  -9.868   5.528  1.00  0.00           O  \\nATOM   2253  CB  PRO A 153     -17.893  -9.139   4.808  1.00  0.00           C  \\nATOM   2254  CG  PRO A 153     -16.394  -9.313   5.019  1.00  0.00           C  \\nATOM   2255  CD  PRO A 153     -16.281 -10.775   5.430  1.00  0.00           C  \\nATOM   2256  HA  PRO A 153     -18.690  -9.400   6.793  1.00  0.00           H  \\nATOM   2257  HB3 PRO A 153     -18.159  -9.523   3.823  1.00  0.00           H  \\nATOM   2258  HB2 PRO A 153     -18.193  -8.094   4.900  1.00  0.00           H  \\nATOM   2259  HG3 PRO A 153     -16.058  -8.678   5.838  1.00  0.00           H  \\nATOM   2260  HG2 PRO A 153     -15.836  -9.096   4.108  1.00  0.00           H  \\nATOM   2261  HD3 PRO A 153     -15.370 -10.917   6.010  1.00  0.00           H  \\nATOM   2262  HD2 PRO A 153     -16.281 -11.412   4.546  1.00  0.00           H  \\nATOM   2263  N   SER A 154     -19.923 -11.826   4.987  1.00  0.00           N  \\nATOM   2264  CA  SER A 154     -21.178 -12.460   4.564  1.00  0.00           C  \\nATOM   2265  C   SER A 154     -21.995 -13.025   5.734  1.00  0.00           C  \\nATOM   2266  O   SER A 154     -23.140 -13.433   5.547  1.00  0.00           O  \\nATOM   2267  CB  SER A 154     -20.918 -13.521   3.488  1.00  0.00           C  \\nATOM   2268  OG  SER A 154     -20.261 -14.681   3.967  1.00  0.00           O  \\nATOM   2269  H   SER A 154     -19.097 -12.404   5.048  1.00  0.00           H  \\nATOM   2270  HA  SER A 154     -21.802 -11.694   4.105  1.00  0.00           H  \\nATOM   2271  HB3 SER A 154     -20.326 -13.076   2.689  1.00  0.00           H  \\nATOM   2272  HB2 SER A 154     -21.879 -13.821   3.067  1.00  0.00           H  \\nATOM   2273  HG  SER A 154     -19.464 -14.430   4.439  1.00  0.00           H  \\nATOM   2274  N   LYS A 155     -21.428 -13.041   6.950  1.00  0.00           N  \\nATOM   2275  CA  LYS A 155     -22.088 -13.453   8.196  1.00  0.00           C  \\nATOM   2276  C   LYS A 155     -21.760 -12.493   9.357  1.00  0.00           C  \\nATOM   2277  O   LYS A 155     -21.146 -12.907  10.344  1.00  0.00           O  \\nATOM   2278  CB  LYS A 155     -21.682 -14.880   8.565  1.00  0.00           C  \\nATOM   2279  CG  LYS A 155     -22.094 -15.935   7.546  1.00  0.00           C  \\nATOM   2280  CD  LYS A 155     -21.609 -17.317   7.955  1.00  0.00           C  \\nATOM   2281  CE  LYS A 155     -20.102 -17.448   7.792  1.00  0.00           C  \\nATOM   2282  NZ  LYS A 155     -19.699 -17.488   6.360  1.00  0.00           N  \\nATOM   2283  H   LYS A 155     -20.460 -12.757   7.014  1.00  0.00           H  \\nATOM   2284  HA  LYS A 155     -23.168 -13.426   8.053  1.00  0.00           H  \\nATOM   2285  HB3 LYS A 155     -20.599 -14.932   8.681  1.00  0.00           H  \\nATOM   2286  HB2 LYS A 155     -22.127 -15.149   9.523  1.00  0.00           H  \\nATOM   2287  HG3 LYS A 155     -23.181 -15.951   7.458  1.00  0.00           H  \\nATOM   2288  HG2 LYS A 155     -21.672 -15.684   6.573  1.00  0.00           H  \\nATOM   2289  HD3 LYS A 155     -21.870 -17.501   8.999  1.00  0.00           H  \\nATOM   2290  HD2 LYS A 155     -22.098 -18.072   7.339  1.00  0.00           H  \\nATOM   2291  HE3 LYS A 155     -19.610 -16.603   8.272  1.00  0.00           H  \\nATOM   2292  HE2 LYS A 155     -19.761 -18.362   8.277  1.00  0.00           H  \\nATOM   2293  HZ1 LYS A 155     -18.695 -17.575   6.295  1.00  0.00           H  \\nATOM   2294  HZ2 LYS A 155     -20.135 -18.280   5.909  1.00  0.00           H  \\nATOM   2295  HZ3 LYS A 155     -19.993 -16.637   5.903  1.00  0.00           H  \\nATOM   2296  N   PRO A 156     -22.199 -11.220   9.302  1.00  0.00           N  \\nATOM   2297  CA  PRO A 156     -21.924 -10.234  10.354  1.00  0.00           C  \\nATOM   2298  C   PRO A 156     -22.411 -10.662  11.747  1.00  0.00           C  \\nATOM   2299  O   PRO A 156     -21.840 -10.250  12.748  1.00  0.00           O  \\nATOM   2300  CB  PRO A 156     -22.688  -9.004   9.855  1.00  0.00           C  \\nATOM   2301  CG  PRO A 156     -22.684  -9.148   8.371  1.00  0.00           C  \\nATOM   2302  CD  PRO A 156     -22.884 -10.622   8.138  1.00  0.00           C  \\nATOM   2303  HA  PRO A 156     -20.851 -10.057  10.417  1.00  0.00           H  \\nATOM   2304  HB3 PRO A 156     -23.703  -8.993  10.280  1.00  0.00           H  \\nATOM   2305  HB2 PRO A 156     -22.187  -8.086  10.196  1.00  0.00           H  \\nATOM   2306  HG3 PRO A 156     -23.484  -8.537   7.928  1.00  0.00           H  \\nATOM   2307  HG2 PRO A 156     -21.735  -8.778   7.957  1.00  0.00           H  \\nATOM   2308  HD3 PRO A 156     -23.960 -10.850   8.135  1.00  0.00           H  \\nATOM   2309  HD2 PRO A 156     -22.423 -10.907   7.181  1.00  0.00           H  \\nATOM   2310  N   GLN A 157     -23.429 -11.526  11.824  1.00  0.00           N  \\nATOM   2311  CA  GLN A 157     -24.011 -11.985  13.093  1.00  0.00           C  \\nATOM   2312  C   GLN A 157     -23.277 -13.190  13.711  1.00  0.00           C  \\nATOM   2313  O   GLN A 157     -23.572 -13.573  14.836  1.00  0.00           O  \\nATOM   2314  CB  GLN A 157     -25.486 -12.341  12.886  1.00  0.00           C  \\nATOM   2315  CG  GLN A 157     -26.365 -11.158  12.516  1.00  0.00           C  \\nATOM   2316  CD  GLN A 157     -27.811 -11.560  12.290  1.00  0.00           C  \\nATOM   2317  OE1 GLN A 157     -28.105 -12.712  11.958  1.00  0.00           O  \\nATOM   2318  NE2 GLN A 157     -28.722 -10.611  12.469  1.00  0.00           N  \\nATOM   2319  H   GLN A 157     -23.822 -11.877  10.962  1.00  0.00           H  \\nATOM   2320  HA  GLN A 157     -23.937 -11.182  13.826  1.00  0.00           H  \\nATOM   2321  HB3 GLN A 157     -25.573 -13.085  12.094  1.00  0.00           H  \\nATOM   2322  HB2 GLN A 157     -25.885 -12.784  13.798  1.00  0.00           H  \\nATOM   2323  HG3 GLN A 157     -26.336 -10.429  13.325  1.00  0.00           H  \\nATOM   2324  HG2 GLN A 157     -25.987 -10.711  11.596  1.00  0.00           H  \\nATOM   2325 HE21 GLN A 157     -29.693 -10.817  12.334  1.00  0.00           H  \\nATOM   2326 HE22 GLN A 157     -28.440  -9.690  12.738  1.00  0.00           H  \\nATOM   2327  N   GLN A 158     -22.346 -13.816  12.982  1.00  0.00           N  \\nATOM   2328  CA  GLN A 158     -21.609 -15.007  13.437  1.00  0.00           C  \\nATOM   2329  C   GLN A 158     -20.145 -14.700  13.763  1.00  0.00           C  \\nATOM   2330  O   GLN A 158     -19.543 -15.370  14.598  1.00  0.00           O  \\nATOM   2331  CB  GLN A 158     -21.678 -16.109  12.377  1.00  0.00           C  \\nATOM   2332  CG  GLN A 158     -23.087 -16.570  12.048  1.00  0.00           C  \\nATOM   2333  CD  GLN A 158     -23.769 -17.237  13.228  1.00  0.00           C  \\nATOM   2334  OE1 GLN A 158     -23.254 -18.205  13.794  1.00  0.00           O  \\nATOM   2335  NE2 GLN A 158     -24.933 -16.722  13.607  1.00  0.00           N  \\nATOM   2336  H   GLN A 158     -22.119 -13.442  12.072  1.00  0.00           H  \\nATOM   2337  HA  GLN A 158     -22.061 -15.394  14.351  1.00  0.00           H  \\nATOM   2338  HB3 GLN A 158     -21.215 -15.757  11.456  1.00  0.00           H  \\nATOM   2339  HB2 GLN A 158     -21.111 -16.976  12.717  1.00  0.00           H  \\nATOM   2340  HG3 GLN A 158     -23.682 -15.705  11.755  1.00  0.00           H  \\nATOM   2341  HG2 GLN A 158     -23.042 -17.287  11.229  1.00  0.00           H  \\nATOM   2342 HE21 GLN A 158     -25.429 -17.120  14.379  1.00  0.00           H  \\nATOM   2343 HE22 GLN A 158     -25.315 -15.936  13.120  1.00  0.00           H  \\nATOM   2344  N   THR A 159     -19.557 -13.702  13.101  1.00  0.00           N  \\nATOM   2345  CA  THR A 159     -18.144 -13.347  13.260  1.00  0.00           C  \\nATOM   2346  C   THR A 159     -18.006 -11.856  13.538  1.00  0.00           C  \\nATOM   2347  O   THR A 159     -18.365 -11.043  12.687  1.00  0.00           O  \\nATOM   2348  CB  THR A 159     -17.324 -13.721  12.012  1.00  0.00           C  \\nATOM   2349  OG1 THR A 159     -17.873 -13.065  10.861  1.00  0.00           O  \\nATOM   2350  CG2 THR A 159     -17.345 -15.226  11.791  1.00  0.00           C  \\nATOM   2351  H   THR A 159     -20.104 -13.183  12.430  1.00  0.00           H  \\nATOM   2352  HA  THR A 159     -17.723 -13.887  14.108  1.00  0.00           H  \\nATOM   2353  HB  THR A 159     -16.292 -13.394  12.142  1.00  0.00           H  \\nATOM   2354  HG1 THR A 159     -18.830 -13.041  10.934  1.00  0.00           H  \\nATOM   2355 HG21 THR A 159     -16.760 -15.472  10.905  1.00  0.00           H  \\nATOM   2356 HG22 THR A 159     -16.918 -15.727  12.659  1.00  0.00           H  \\nATOM   2357 HG23 THR A 159     -18.373 -15.558  11.650  1.00  0.00           H  \\nATOM   2358  N   LYS A 160     -17.446 -11.495  14.699  1.00  0.00           N  \\nATOM   2359  CA  LYS A 160     -17.205 -10.099  15.084  1.00  0.00           C  \\nATOM   2360  C   LYS A 160     -15.801  -9.682  14.648  1.00  0.00           C  \\nATOM   2361  O   LYS A 160     -14.817 -10.269  15.088  1.00  0.00           O  \\nATOM   2362  CB  LYS A 160     -17.373  -9.909  16.593  1.00  0.00           C  \\nATOM   2363  CG  LYS A 160     -17.154  -8.481  17.075  1.00  0.00           C  \\nATOM   2364  CD  LYS A 160     -17.309  -8.379  18.585  1.00  0.00           C  \\nATOM   2365  CE  LYS A 160     -16.922  -6.996  19.089  1.00  0.00           C  \\nATOM   2366  NZ  LYS A 160     -16.961  -6.914  20.575  1.00  0.00           N  \\nATOM   2367  H   LYS A 160     -17.122 -12.218  15.325  1.00  0.00           H  \\nATOM   2368  HA  LYS A 160     -17.928  -9.466  14.571  1.00  0.00           H  \\nATOM   2369  HB3 LYS A 160     -18.378 -10.211  16.888  1.00  0.00           H  \\nATOM   2370  HB2 LYS A 160     -16.669 -10.553  17.121  1.00  0.00           H  \\nATOM   2371  HG3 LYS A 160     -16.151  -8.153  16.798  1.00  0.00           H  \\nATOM   2372  HG2 LYS A 160     -17.877  -7.821  16.598  1.00  0.00           H  \\nATOM   2373  HD3 LYS A 160     -18.345  -8.579  18.859  1.00  0.00           H  \\nATOM   2374  HD2 LYS A 160     -16.673  -9.121  19.068  1.00  0.00           H  \\nATOM   2375  HE3 LYS A 160     -15.916  -6.754  18.751  1.00  0.00           H  \\nATOM   2376  HE2 LYS A 160     -17.608  -6.254  18.679  1.00  0.00           H  \\nATOM   2377  HZ1 LYS A 160     -16.699  -5.983  20.868  1.00  0.00           H  \\nATOM   2378  HZ2 LYS A 160     -17.896  -7.119  20.899  1.00  0.00           H  \\nATOM   2379  HZ3 LYS A 160     -16.315  -7.584  20.966  1.00  0.00           H  \\nATOM   2380  N   GLY A 161     -15.716  -8.682  13.776  1.00  0.00           N  \\nATOM   2381  CA  GLY A 161     -14.441  -8.194  13.250  1.00  0.00           C  \\nATOM   2382  C   GLY A 161     -13.876  -7.063  14.089  1.00  0.00           C  \\nATOM   2383  O   GLY A 161     -14.632  -6.245  14.608  1.00  0.00           O  \\nATOM   2384  H   GLY A 161     -16.562  -8.198  13.511  1.00  0.00           H  \\nATOM   2385  HA3 GLY A 161     -13.712  -9.002  13.222  1.00  0.00           H  \\nATOM   2386  HA2 GLY A 161     -14.583  -7.820  12.237  1.00  0.00           H  \\nATOM   2387  N   THR A 162     -12.551  -6.972  14.150  1.00  0.00           N  \\nATOM   2388  CA  THR A 162     -11.861  -5.922  14.904  1.00  0.00           C  \\nATOM   2389  C   THR A 162     -11.538  -4.742  14.000  1.00  0.00           C  \\nATOM   2390  O   THR A 162     -10.920  -4.908  12.943  1.00  0.00           O  \\nATOM   2391  CB  THR A 162     -10.578  -6.465  15.534  1.00  0.00           C  \\nATOM   2392  OG1 THR A 162     -10.863  -7.546  16.387  1.00  0.00           O  \\nATOM   2393  CG2 THR A 162      -9.847  -5.443  16.388  1.00  0.00           C  \\nATOM   2394  H   THR A 162     -11.986  -7.659  13.672  1.00  0.00           H  \\nATOM   2395  HA  THR A 162     -12.501  -5.572  15.714  1.00  0.00           H  \\nATOM   2396  HB  THR A 162      -9.913  -6.813  14.744  1.00  0.00           H  \\nATOM   2397  HG1 THR A 162     -11.314  -7.193  17.157  1.00  0.00           H  \\nATOM   2398 HG21 THR A 162      -9.447  -4.643  15.764  1.00  0.00           H  \\nATOM   2399 HG22 THR A 162      -9.021  -5.939  16.896  1.00  0.00           H  \\nATOM   2400 HG23 THR A 162     -10.523  -5.024  17.133  1.00  0.00           H  \\nATOM   2401  N   TRP A 163     -11.937  -3.549  14.430  1.00  0.00           N  \\nATOM   2402  CA  TRP A 163     -11.526  -2.291  13.820  1.00  0.00           C  \\nATOM   2403  C   TRP A 163     -10.167  -1.860  14.364  1.00  0.00           C  \\nATOM   2404  O   TRP A 163      -9.920  -1.935  15.567  1.00  0.00           O  \\nATOM   2405  CB  TRP A 163     -12.565  -1.200  14.086  1.00  0.00           C  \\nATOM   2406  CG  TRP A 163     -13.835  -1.381  13.312  1.00  0.00           C  \\nATOM   2407  CD1 TRP A 163     -14.969  -2.004  13.740  1.00  0.00           C  \\nATOM   2408  CD2 TRP A 163     -14.109  -0.932  11.963  1.00  0.00           C  \\nATOM   2409  NE1 TRP A 163     -15.925  -1.975  12.755  1.00  0.00           N  \\nATOM   2410  CE2 TRP A 163     -15.416  -1.322  11.661  1.00  0.00           C  \\nATOM   2411  CE3 TRP A 163     -13.361  -0.240  11.002  1.00  0.00           C  \\nATOM   2412  CZ2 TRP A 163     -15.999  -1.046  10.434  1.00  0.00           C  \\nATOM   2413  CZ3 TRP A 163     -13.946   0.038   9.772  1.00  0.00           C  \\nATOM   2414  CH2 TRP A 163     -15.231  -0.355   9.497  1.00  0.00           C  \\nATOM   2415  H   TRP A 163     -12.431  -3.494  15.309  1.00  0.00           H  \\nATOM   2416  HA  TRP A 163     -11.442  -2.435  12.743  1.00  0.00           H  \\nATOM   2417  HB3 TRP A 163     -12.810  -1.182  15.148  1.00  0.00           H  \\nATOM   2418  HB2 TRP A 163     -12.144  -0.228  13.832  1.00  0.00           H  \\nATOM   2419  HD1 TRP A 163     -15.097  -2.459  14.720  1.00  0.00           H  \\nATOM   2420  HE1 TRP A 163     -16.851  -2.371  12.822  1.00  0.00           H  \\nATOM   2421  HE3 TRP A 163     -12.341   0.077  11.217  1.00  0.00           H  \\nATOM   2422  HZ2 TRP A 163     -17.019  -1.350  10.196  1.00  0.00           H  \\nATOM   2423  HZ3 TRP A 163     -13.356   0.576   9.030  1.00  0.00           H  \\nATOM   2424  HH2 TRP A 163     -15.660  -0.120   8.522  1.00  0.00           H  \\nATOM   2425  N   PHE A 164      -9.283  -1.395  13.489  1.00  0.00           N  \\nATOM   2426  CA  PHE A 164      -7.947  -0.942  13.856  1.00  0.00           C  \\nATOM   2427  C   PHE A 164      -7.485   0.228  12.984  1.00  0.00           C  \\nATOM   2428  O   PHE A 164      -7.979   0.443  11.879  1.00  0.00           O  \\nATOM   2429  CB  PHE A 164      -6.948  -2.094  13.739  1.00  0.00           C  \\nATOM   2430  CG  PHE A 164      -6.700  -2.541  12.326  1.00  0.00           C  \\nATOM   2431  CD1 PHE A 164      -5.633  -2.033  11.600  1.00  0.00           C  \\nATOM   2432  CD2 PHE A 164      -7.533  -3.469  11.720  1.00  0.00           C  \\nATOM   2433  CE1 PHE A 164      -5.404  -2.442  10.300  1.00  0.00           C  \\nATOM   2434  CE2 PHE A 164      -7.306  -3.881  10.421  1.00  0.00           C  \\nATOM   2435  CZ  PHE A 164      -6.240  -3.366   9.711  1.00  0.00           C  \\nATOM   2436  H   PHE A 164      -9.537  -1.355  12.512  1.00  0.00           H  \\nATOM   2437  HA  PHE A 164      -7.981  -0.582  14.884  1.00  0.00           H  \\nATOM   2438  HB3 PHE A 164      -5.995  -1.795  14.173  1.00  0.00           H  \\nATOM   2439  HB2 PHE A 164      -7.312  -2.950  14.307  1.00  0.00           H  \\nATOM   2440  HD1 PHE A 164      -4.971  -1.303  12.067  1.00  0.00           H  \\nATOM   2441  HD2 PHE A 164      -8.376  -3.874  12.281  1.00  0.00           H  \\nATOM   2442  HE1 PHE A 164      -4.562  -2.035   9.741  1.00  0.00           H  \\nATOM   2443  HE2 PHE A 164      -7.969  -4.611   9.956  1.00  0.00           H  \\nATOM   2444  HZ  PHE A 164      -6.061  -3.689   8.686  1.00  0.00           H  \\nATOM   2445  N   GLN A 165      -6.506   0.964  13.491  1.00  0.00           N  \\nATOM   2446  CA  GLN A 165      -5.859   2.088  12.838  1.00  0.00           C  \\nATOM   2447  C   GLN A 165      -4.366   2.036  13.162  1.00  0.00           C  \\nATOM   2448  O   GLN A 165      -3.983   1.911  14.326  1.00  0.00           O  \\nATOM   2449  CB  GLN A 165      -6.504   3.388  13.336  1.00  0.00           C  \\nATOM   2450  CG  GLN A 165      -5.851   4.633  12.718  1.00  0.00           C  \\nATOM   2451  CD  GLN A 165      -6.501   5.943  13.147  1.00  0.00           C  \\nATOM   2452  OE1 GLN A 165      -7.430   6.014  13.938  1.00  0.00           O  \\nATOM   2453  NE2 GLN A 165      -5.995   7.053  12.658  1.00  0.00           N  \\nATOM   2454  H   GLN A 165      -6.166   0.725  14.412  1.00  0.00           H  \\nATOM   2455  HA  GLN A 165      -5.994   2.020  11.759  1.00  0.00           H  \\nATOM   2456  HB3 GLN A 165      -7.563   3.374  13.081  1.00  0.00           H  \\nATOM   2457  HB2 GLN A 165      -6.412   3.440  14.421  1.00  0.00           H  \\nATOM   2458  HG3 GLN A 165      -5.894   4.567  11.631  1.00  0.00           H  \\nATOM   2459  HG2 GLN A 165      -4.802   4.681  13.012  1.00  0.00           H  \\nATOM   2460 HE21 GLN A 165      -5.202   7.010  12.034  1.00  0.00           H  \\nATOM   2461 HE22 GLN A 165      -6.430   7.916  12.954  1.00  0.00           H  \\nATOM   2462  N   ILE A 166      -3.529   2.174  12.135  1.00  0.00           N  \\nATOM   2463  CA  ILE A 166      -2.089   2.344  12.336  1.00  0.00           C  \\nATOM   2464  C   ILE A 166      -1.821   3.848  12.399  1.00  0.00           C  \\nATOM   2465  O   ILE A 166      -1.973   4.548  11.397  1.00  0.00           O  \\nATOM   2466  CB  ILE A 166      -1.266   1.696  11.208  1.00  0.00           C  \\nATOM   2467  CG1 ILE A 166      -1.576   0.200  11.113  1.00  0.00           C  \\nATOM   2468  CG2 ILE A 166       0.221   1.921  11.437  1.00  0.00           C  \\nATOM   2469  CD1 ILE A 166      -1.055  -0.453   9.853  1.00  0.00           C  \\nATOM   2470  H   ILE A 166      -3.907   2.383  11.222  1.00  0.00           H  \\nATOM   2471  HA  ILE A 166      -1.815   1.897  13.292  1.00  0.00           H  \\nATOM   2472  HB  ILE A 166      -1.548   2.137  10.252  1.00  0.00           H  \\nATOM   2473 HG13 ILE A 166      -1.143  -0.317  11.969  1.00  0.00           H  \\nATOM   2474 HG12 ILE A 166      -2.655   0.048  11.153  1.00  0.00           H  \\nATOM   2475 HG21 ILE A 166       0.788   1.456  10.631  1.00  0.00           H  \\nATOM   2476 HG22 ILE A 166       0.428   2.990  11.456  1.00  0.00           H  \\nATOM   2477 HG23 ILE A 166       0.513   1.478  12.389  1.00  0.00           H  \\nATOM   2478 HD11 ILE A 166      -1.314  -1.513   9.858  1.00  0.00           H  \\nATOM   2479 HD12 ILE A 166      -1.503   0.026   8.982  1.00  0.00           H  \\nATOM   2480 HD13 ILE A 166       0.028  -0.346   9.810  1.00  0.00           H  \\nATOM   2481  N   THR A 167      -1.488   4.360  13.582  1.00  0.00           N  \\nATOM   2482  CA  THR A 167      -1.393   5.807  13.839  1.00  0.00           C  \\nATOM   2483  C   THR A 167      -0.001   6.352  13.581  1.00  0.00           C  \\nATOM   2484  O   THR A 167       0.169   7.559  13.403  1.00  0.00           O  \\nATOM   2485  CB  THR A 167      -1.801   6.141  15.286  1.00  0.00           C  \\nATOM   2486  OG1 THR A 167      -0.933   5.457  16.199  1.00  0.00           O  \\nATOM   2487  CG2 THR A 167      -3.238   5.718  15.545  1.00  0.00           C  \\nATOM   2488  H   THR A 167      -1.340   3.735  14.363  1.00  0.00           H  \\nATOM   2489  HA  THR A 167      -2.055   6.334  13.152  1.00  0.00           H  \\nATOM   2490  HB  THR A 167      -1.708   7.214  15.452  1.00  0.00           H  \\nATOM   2491  HG1 THR A 167      -0.027   5.517  15.885  1.00  0.00           H  \\nATOM   2492 HG21 THR A 167      -3.509   5.961  16.572  1.00  0.00           H  \\nATOM   2493 HG22 THR A 167      -3.902   6.244  14.860  1.00  0.00           H  \\nATOM   2494 HG23 THR A 167      -3.335   4.644  15.390  1.00  0.00           H  \\nATOM   2495  N   LYS A 168       0.990   5.465  13.528  1.00  0.00           N  \\nATOM   2496  CA  LYS A 168       2.382   5.818  13.290  1.00  0.00           C  \\nATOM   2497  C   LYS A 168       3.127   4.648  12.673  1.00  0.00           C  \\nATOM   2498  O   LYS A 168       2.988   3.504  13.097  1.00  0.00           O  \\nATOM   2499  CB  LYS A 168       3.061   6.253  14.589  1.00  0.00           C  \\nATOM   2500  CG  LYS A 168       4.494   6.739  14.418  1.00  0.00           C  \\nATOM   2501  CD  LYS A 168       5.066   7.248  15.733  1.00  0.00           C  \\nATOM   2502  CE  LYS A 168       6.503   7.718  15.567  1.00  0.00           C  \\nATOM   2503  NZ  LYS A 168       7.076   8.220  16.845  1.00  0.00           N  \\nATOM   2504  H   LYS A 168       0.775   4.501  13.739  1.00  0.00           H  \\nATOM   2505  HA  LYS A 168       2.412   6.658  12.596  1.00  0.00           H  \\nATOM   2506  HB3 LYS A 168       2.487   7.058  15.048  1.00  0.00           H  \\nATOM   2507  HB2 LYS A 168       3.072   5.419  15.291  1.00  0.00           H  \\nATOM   2508  HG3 LYS A 168       5.115   5.920  14.053  1.00  0.00           H  \\nATOM   2509  HG2 LYS A 168       4.519   7.545  13.685  1.00  0.00           H  \\nATOM   2510  HD3 LYS A 168       4.460   8.079  16.096  1.00  0.00           H  \\nATOM   2511  HD2 LYS A 168       5.038   6.449  16.474  1.00  0.00           H  \\nATOM   2512  HE3 LYS A 168       7.117   6.893  15.208  1.00  0.00           H  \\nATOM   2513  HE2 LYS A 168       6.542   8.518  14.828  1.00  0.00           H  \\nATOM   2514  HZ1 LYS A 168       8.027   8.522  16.693  1.00  0.00           H  \\nATOM   2515  HZ2 LYS A 168       6.525   8.999  17.178  1.00  0.00           H  \\nATOM   2516  HZ3 LYS A 168       7.063   7.482  17.534  1.00  0.00           H  \\nATOM   2517  N   PHE A 169       3.963   4.977  11.701  1.00  0.00           N  \\nATOM   2518  CA  PHE A 169       4.900   4.057  11.078  1.00  0.00           C  \\nATOM   2519  C   PHE A 169       6.313   4.448  11.520  1.00  0.00           C  \\nATOM   2520  O   PHE A 169       6.780   5.551  11.223  1.00  0.00           O  \\nATOM   2521  CB  PHE A 169       4.782   4.097   9.553  1.00  0.00           C  \\nATOM   2522  CG  PHE A 169       3.489   3.539   9.030  1.00  0.00           C  \\nATOM   2523  CD1 PHE A 169       2.343   4.321   8.995  1.00  0.00           C  \\nATOM   2524  CD2 PHE A 169       3.414   2.232   8.574  1.00  0.00           C  \\nATOM   2525  CE1 PHE A 169       1.153   3.808   8.515  1.00  0.00           C  \\nATOM   2526  CE2 PHE A 169       2.226   1.718   8.092  1.00  0.00           C  \\nATOM   2527  CZ  PHE A 169       1.094   2.507   8.063  1.00  0.00           C  \\nATOM   2528  H   PHE A 169       4.038   5.951  11.446  1.00  0.00           H  \\nATOM   2529  HA  PHE A 169       4.693   3.036  11.395  1.00  0.00           H  \\nATOM   2530  HB3 PHE A 169       4.872   5.126   9.208  1.00  0.00           H  \\nATOM   2531  HB2 PHE A 169       5.600   3.531   9.110  1.00  0.00           H  \\nATOM   2532  HD1 PHE A 169       2.390   5.350   9.352  1.00  0.00           H  \\nATOM   2533  HD2 PHE A 169       4.309   1.608   8.597  1.00  0.00           H  \\nATOM   2534  HE1 PHE A 169       0.261   4.433   8.493  1.00  0.00           H  \\nATOM   2535  HE2 PHE A 169       2.182   0.689   7.735  1.00  0.00           H  \\nATOM   2536  HZ  PHE A 169       0.157   2.102   7.685  1.00  0.00           H  \\nATOM   2537  N   THR A 170       6.975   3.579  12.277  1.00  0.00           N  \\nATOM   2538  CA  THR A 170       8.299   3.819  12.878  1.00  0.00           C  \\nATOM   2539  C   THR A 170       9.329   2.814  12.350  1.00  0.00           C  \\nATOM   2540  O   THR A 170       9.057   2.099  11.389  1.00  0.00           O  \\nATOM   2541  CB  THR A 170       8.238   3.739  14.414  1.00  0.00           C  \\nATOM   2542  OG1 THR A 170       9.488   4.170  14.968  1.00  0.00           O  \\nATOM   2543  CG2 THR A 170       7.953   2.314  14.863  1.00  0.00           C  \\nATOM   2544  H   THR A 170       6.538   2.690  12.475  1.00  0.00           H  \\nATOM   2545  HA  THR A 170       8.657   4.806  12.585  1.00  0.00           H  \\nATOM   2546  HB  THR A 170       7.448   4.395  14.782  1.00  0.00           H  \\nATOM   2547  HG1 THR A 170       9.446   4.123  15.926  1.00  0.00           H  \\nATOM   2548 HG21 THR A 170       7.913   2.277  15.952  1.00  0.00           H  \\nATOM   2549 HG22 THR A 170       6.997   1.988  14.454  1.00  0.00           H  \\nATOM   2550 HG23 THR A 170       8.744   1.656  14.507  1.00  0.00           H  \\nATOM   2551  N   GLY A 171      10.536   2.771  12.923  1.00  0.00           N  \\nATOM   2552  CA  GLY A 171      11.647   1.982  12.376  1.00  0.00           C  \\nATOM   2553  C   GLY A 171      12.096   2.475  10.998  1.00  0.00           C  \\nATOM   2554  O   GLY A 171      12.319   3.671  10.804  1.00  0.00           O  \\nATOM   2555  H   GLY A 171      10.693   3.264  13.790  1.00  0.00           H  \\nATOM   2556  HA3 GLY A 171      11.340   0.938  12.301  1.00  0.00           H  \\nATOM   2557  HA2 GLY A 171      12.500   2.028  13.053  1.00  0.00           H  \\nATOM   2558  N   ALA A 172      12.203   1.564  10.029  1.00  0.00           N  \\nATOM   2559  CA  ALA A 172      12.658   1.838   8.661  1.00  0.00           C  \\nATOM   2560  C   ALA A 172      11.577   2.510   7.782  1.00  0.00           C  \\nATOM   2561  O   ALA A 172      11.364   2.126   6.635  1.00  0.00           O  \\nATOM   2562  CB  ALA A 172      13.211   0.538   8.059  1.00  0.00           C  \\nATOM   2563  H   ALA A 172      11.894   0.624  10.233  1.00  0.00           H  \\nATOM   2564  HA  ALA A 172      13.484   2.547   8.721  1.00  0.00           H  \\nATOM   2565  HB1 ALA A 172      12.414  -0.198   7.964  1.00  0.00           H  \\nATOM   2566  HB2 ALA A 172      13.998   0.140   8.701  1.00  0.00           H  \\nATOM   2567  HB3 ALA A 172      13.630   0.738   7.074  1.00  0.00           H  \\nATOM   2568  N   ALA A 173      10.855   3.493   8.323  1.00  0.00           N  \\nATOM   2569  CA  ALA A 173       9.763   4.166   7.636  1.00  0.00           C  \\nATOM   2570  C   ALA A 173      10.257   4.946   6.403  1.00  0.00           C  \\nATOM   2571  O   ALA A 173      11.331   5.566   6.414  1.00  0.00           O  \\nATOM   2572  CB  ALA A 173       9.023   5.065   8.633  1.00  0.00           C  \\nATOM   2573  H   ALA A 173      11.095   3.791   9.258  1.00  0.00           H  \\nATOM   2574  HA  ALA A 173       9.064   3.402   7.295  1.00  0.00           H  \\nATOM   2575  HB1 ALA A 173       8.191   5.564   8.136  1.00  0.00           H  \\nATOM   2576  HB2 ALA A 173       8.633   4.463   9.452  1.00  0.00           H  \\nATOM   2577  HB3 ALA A 173       9.702   5.819   9.033  1.00  0.00           H  \\nATOM   2578  N   GLY A 174       9.432   4.940   5.356  1.00  0.00           N  \\nATOM   2579  CA  GLY A 174       9.681   5.677   4.125  1.00  0.00           C  \\nATOM   2580  C   GLY A 174       9.313   7.168   4.224  1.00  0.00           C  \\nATOM   2581  O   GLY A 174       8.690   7.607   5.201  1.00  0.00           O  \\nATOM   2582  H   GLY A 174       8.553   4.447   5.427  1.00  0.00           H  \\nATOM   2583  HA3 GLY A 174       9.091   5.227   3.326  1.00  0.00           H  \\nATOM   2584  HA2 GLY A 174      10.734   5.587   3.859  1.00  0.00           H  \\nATOM   2585  N   PRO A 175       9.743   7.981   3.246  1.00  0.00           N  \\nATOM   2586  CA  PRO A 175       9.451   9.411   3.210  1.00  0.00           C  \\nATOM   2587  C   PRO A 175       7.964   9.763   3.384  1.00  0.00           C  \\nATOM   2588  O   PRO A 175       7.669  10.708   4.116  1.00  0.00           O  \\nATOM   2589  CB  PRO A 175       9.952   9.802   1.816  1.00  0.00           C  \\nATOM   2590  CG  PRO A 175      11.061   8.844   1.544  1.00  0.00           C  \\nATOM   2591  CD  PRO A 175      10.570   7.535   2.101  1.00  0.00           C  \\nATOM   2592  HA  PRO A 175      10.002   9.912   4.006  1.00  0.00           H  \\nATOM   2593  HB3 PRO A 175       9.132   9.725   1.086  1.00  0.00           H  \\nATOM   2594  HB2 PRO A 175      10.284  10.851   1.816  1.00  0.00           H  \\nATOM   2595  HG3 PRO A 175      11.264   8.795   0.464  1.00  0.00           H  \\nATOM   2596  HG2 PRO A 175      11.987   9.189   2.026  1.00  0.00           H  \\nATOM   2597  HD3 PRO A 175       9.973   7.012   1.339  1.00  0.00           H  \\nATOM   2598  HD2 PRO A 175      11.429   6.921   2.409  1.00  0.00           H  \\nATOM   2599  N   TYR A 176       7.021   9.025   2.788  1.00  0.00           N  \\nATOM   2600  CA  TYR A 176       5.591   9.321   2.945  1.00  0.00           C  \\nATOM   2601  C   TYR A 176       5.060   8.876   4.301  1.00  0.00           C  \\nATOM   2602  O   TYR A 176       4.217   9.557   4.880  1.00  0.00           O  \\nATOM   2603  CB  TYR A 176       4.753   8.667   1.845  1.00  0.00           C  \\nATOM   2604  CG  TYR A 176       4.939   9.293   0.485  1.00  0.00           C  \\nATOM   2605  CD1 TYR A 176       4.587  10.640   0.286  1.00  0.00           C  \\nATOM   2606  CD2 TYR A 176       5.423   8.526  -0.588  1.00  0.00           C  \\nATOM   2607  CE1 TYR A 176       4.740  11.224  -0.986  1.00  0.00           C  \\nATOM   2608  CE2 TYR A 176       5.569   9.101  -1.860  1.00  0.00           C  \\nATOM   2609  CZ  TYR A 176       5.222  10.446  -2.061  1.00  0.00           C  \\nATOM   2610  OH  TYR A 176       5.342  10.981  -3.301  1.00  0.00           O  \\nATOM   2611  H   TYR A 176       7.273   8.149   2.353  1.00  0.00           H  \\nATOM   2612  HA  TYR A 176       5.449  10.401   2.886  1.00  0.00           H  \\nATOM   2613  HB3 TYR A 176       3.700   8.769   2.107  1.00  0.00           H  \\nATOM   2614  HB2 TYR A 176       4.969   7.600   1.800  1.00  0.00           H  \\nATOM   2615  HD1 TYR A 176       4.192  11.214   1.111  1.00  0.00           H  \\nATOM   2616  HD2 TYR A 176       5.659   7.482  -0.444  1.00  0.00           H  \\nATOM   2617  HE1 TYR A 176       4.473  12.259  -1.145  1.00  0.00           H  \\nATOM   2618  HE2 TYR A 176       5.948   8.511  -2.681  1.00  0.00           H  \\nATOM   2619  HH  TYR A 176       5.776  10.364  -3.896  1.00  0.00           H  \\nATOM   2620  N   CYS A 177       5.567   7.776   4.848  1.00  0.00           N  \\nATOM   2621  CA  CYS A 177       5.217   7.353   6.200  1.00  0.00           C  \\nATOM   2622  C   CYS A 177       5.687   8.335   7.272  1.00  0.00           C  \\nATOM   2623  O   CYS A 177       4.939   8.641   8.203  1.00  0.00           O  \\nATOM   2624  CB  CYS A 177       5.817   5.977   6.491  1.00  0.00           C  \\nATOM   2625  SG  CYS A 177       4.999   4.617   5.623  1.00  0.00           S  \\nATOM   2626  H   CYS A 177       6.179   7.201   4.286  1.00  0.00           H  \\nATOM   2627  HA  CYS A 177       4.132   7.324   6.301  1.00  0.00           H  \\nATOM   2628  HB3 CYS A 177       6.870   5.973   6.210  1.00  0.00           H  \\nATOM   2629  HB2 CYS A 177       5.762   5.776   7.561  1.00  0.00           H  \\nATOM   2630  N   LYS A 178       6.895   8.891   7.115  1.00  0.00           N  \\nATOM   2631  CA  LYS A 178       7.399   9.961   7.986  1.00  0.00           C  \\nATOM   2632  C   LYS A 178       6.537  11.221   7.863  1.00  0.00           C  \\nATOM   2633  O   LYS A 178       6.149  11.781   8.885  1.00  0.00           O  \\nATOM   2634  CB  LYS A 178       8.855  10.287   7.652  1.00  0.00           C  \\nATOM   2635  CG  LYS A 178       9.849   9.204   8.052  1.00  0.00           C  \\nATOM   2636  CD  LYS A 178      11.266   9.575   7.639  1.00  0.00           C  \\nATOM   2637  CE  LYS A 178      12.266   8.515   8.077  1.00  0.00           C  \\nATOM   2638  NZ  LYS A 178      13.646   8.827   7.615  1.00  0.00           N  \\nATOM   2639  H   LYS A 178       7.453   8.594   6.327  1.00  0.00           H  \\nATOM   2640  HA  LYS A 178       7.322   9.645   9.027  1.00  0.00           H  \\nATOM   2641  HB3 LYS A 178       8.955  10.452   6.579  1.00  0.00           H  \\nATOM   2642  HB2 LYS A 178       9.146  11.210   8.154  1.00  0.00           H  \\nATOM   2643  HG3 LYS A 178       9.820   9.064   9.133  1.00  0.00           H  \\nATOM   2644  HG2 LYS A 178       9.575   8.264   7.574  1.00  0.00           H  \\nATOM   2645  HD3 LYS A 178      11.313   9.681   6.555  1.00  0.00           H  \\nATOM   2646  HD2 LYS A 178      11.539  10.528   8.092  1.00  0.00           H  \\nATOM   2647  HE3 LYS A 178      12.267   8.443   9.164  1.00  0.00           H  \\nATOM   2648  HE2 LYS A 178      11.971   7.547   7.671  1.00  0.00           H  \\nATOM   2649  HZ1 LYS A 178      14.277   8.102   7.924  1.00  0.00           H  \\nATOM   2650  HZ2 LYS A 178      13.660   8.878   6.606  1.00  0.00           H  \\nATOM   2651  HZ3 LYS A 178      13.938   9.714   8.000  1.00  0.00           H  \\nATOM   2652  N   ALA A 179       6.179  11.609   6.637  1.00  0.00           N  \\nATOM   2653  CA  ALA A 179       5.250  12.707   6.374  1.00  0.00           C  \\nATOM   2654  C   ALA A 179       3.840  12.452   6.933  1.00  0.00           C  \\nATOM   2655  O   ALA A 179       3.221  13.366   7.469  1.00  0.00           O  \\nATOM   2656  CB  ALA A 179       5.211  12.944   4.864  1.00  0.00           C  \\nATOM   2657  H   ALA A 179       6.611  11.158   5.844  1.00  0.00           H  \\nATOM   2658  HA  ALA A 179       5.632  13.610   6.850  1.00  0.00           H  \\nATOM   2659  HB1 ALA A 179       4.560  13.792   4.646  1.00  0.00           H  \\nATOM   2660  HB2 ALA A 179       6.213  13.168   4.500  1.00  0.00           H  \\nATOM   2661  HB3 ALA A 179       4.826  12.060   4.356  1.00  0.00           H  \\nATOM   2662  N   LEU A 180       3.332  11.217   6.874  1.00  0.00           N  \\nATOM   2663  CA  LEU A 180       2.035  10.859   7.448  1.00  0.00           C  \\nATOM   2664  C   LEU A 180       2.024  11.051   8.965  1.00  0.00           C  \\nATOM   2665  O   LEU A 180       1.144  11.735   9.482  1.00  0.00           O  \\nATOM   2666  CB  LEU A 180       1.644   9.434   7.015  1.00  0.00           C  \\nATOM   2667  CG  LEU A 180       0.297   8.945   7.587  1.00  0.00           C  \\nATOM   2668  CD1 LEU A 180      -0.891   9.803   7.141  1.00  0.00           C  \\nATOM   2669  CD2 LEU A 180       0.037   7.511   7.131  1.00  0.00           C  \\nATOM   2670  H   LEU A 180       3.855  10.506   6.382  1.00  0.00           H  \\nATOM   2671  HA  LEU A 180       1.293  11.558   7.065  1.00  0.00           H  \\nATOM   2672  HB3 LEU A 180       1.600   9.397   5.926  1.00  0.00           H  \\nATOM   2673  HB2 LEU A 180       2.424   8.745   7.341  1.00  0.00           H  \\nATOM   2674  HG  LEU A 180       0.344   8.946   8.676  1.00  0.00           H  \\nATOM   2675 HD11 LEU A 180      -1.808   9.393   7.564  1.00  0.00           H  \\nATOM   2676 HD12 LEU A 180      -0.966   9.802   6.053  1.00  0.00           H  \\nATOM   2677 HD13 LEU A 180      -0.773  10.822   7.510  1.00  0.00           H  \\nATOM   2678 HD21 LEU A 180      -0.066   7.476   6.046  1.00  0.00           H  \\nATOM   2679 HD22 LEU A 180       0.868   6.872   7.429  1.00  0.00           H  \\nATOM   2680 HD23 LEU A 180      -0.874   7.132   7.592  1.00  0.00           H  \\nATOM   2681  N   GLY A 181       3.035  10.523   9.659  1.00  0.00           N  \\nATOM   2682  CA  GLY A 181       3.193  10.699  11.104  1.00  0.00           C  \\nATOM   2683  C   GLY A 181       3.290  12.161  11.559  1.00  0.00           C  \\nATOM   2684  O   GLY A 181       2.835  12.477  12.655  1.00  0.00           O  \\nATOM   2685  H   GLY A 181       3.718   9.971   9.161  1.00  0.00           H  \\nATOM   2686  HA3 GLY A 181       2.343  10.244  11.614  1.00  0.00           H  \\nATOM   2687  HA2 GLY A 181       4.104  10.190  11.417  1.00  0.00           H  \\nATOM   2688  N   SER A 182       3.855  13.051  10.739  1.00  0.00           N  \\nATOM   2689  CA  SER A 182       3.949  14.496  11.010  1.00  0.00           C  \\nATOM   2690  C   SER A 182       2.818  15.323  10.384  1.00  0.00           C  \\nATOM   2691  O   SER A 182       2.805  16.543  10.534  1.00  0.00           O  \\nATOM   2692  CB  SER A 182       5.282  15.016  10.507  1.00  0.00           C  \\nATOM   2693  OG  SER A 182       6.347  14.407  11.183  1.00  0.00           O  \\nATOM   2694  H   SER A 182       4.263  12.725   9.874  1.00  0.00           H  \\nATOM   2695  HA  SER A 182       3.886  14.659  12.085  1.00  0.00           H  \\nATOM   2696  HB3 SER A 182       5.368  14.823   9.438  1.00  0.00           H  \\nATOM   2697  HB2 SER A 182       5.327  16.095  10.649  1.00  0.00           H  \\nATOM   2698  HG  SER A 182       6.536  13.597  10.704  1.00  0.00           H  \\nATOM   2699  N   ASN A 183       1.875  14.683   9.682  1.00  0.00           N  \\nATOM   2700  CA  ASN A 183       0.825  15.328   8.886  1.00  0.00           C  \\nATOM   2701  C   ASN A 183       1.363  16.344   7.846  1.00  0.00           C  \\nATOM   2702  O   ASN A 183       0.736  17.369   7.571  1.00  0.00           O  \\nATOM   2703  CB  ASN A 183      -0.172  16.002   9.810  1.00  0.00           C  \\nATOM   2704  CG  ASN A 183      -0.906  15.022  10.682  1.00  0.00           C  \\nATOM   2705  OD1 ASN A 183      -1.439  14.018  10.196  1.00  0.00           O  \\nATOM   2706  ND2 ASN A 183      -0.943  15.293  11.962  1.00  0.00           N  \\nATOM   2707  H   ASN A 183       1.971  13.683   9.587  1.00  0.00           H  \\nATOM   2708  HA  ASN A 183       0.365  14.534   8.297  1.00  0.00           H  \\nATOM   2709  HB3 ASN A 183       0.349  16.718  10.447  1.00  0.00           H  \\nATOM   2710  HB2 ASN A 183      -0.898  16.559   9.218  1.00  0.00           H  \\nATOM   2711 HD21 ASN A 183      -1.418  14.675  12.590  1.00  0.00           H  \\nATOM   2712 HD22 ASN A 183      -0.497  16.116  12.312  1.00  0.00           H  \\nATOM   2713  N   ASP A 184       2.522  16.065   7.244  1.00  0.00           N  \\nATOM   2714  CA  ASP A 184       3.174  16.963   6.292  1.00  0.00           C  \\nATOM   2715  C   ASP A 184       2.654  16.718   4.867  1.00  0.00           C  \\nATOM   2716  O   ASP A 184       3.080  15.806   4.165  1.00  0.00           O  \\nATOM   2717  CB  ASP A 184       4.693  16.777   6.332  1.00  0.00           C  \\nATOM   2718  CG  ASP A 184       5.433  17.743   5.416  1.00  0.00           C  \\nATOM   2719  OD1 ASP A 184       4.805  18.313   4.555  1.00  0.00           O  \\nATOM   2720  OD2 ASP A 184       6.618  17.901   5.586  1.00  0.00           O  \\nATOM   2721  H   ASP A 184       2.938  15.160   7.410  1.00  0.00           H  \\nATOM   2722  HA  ASP A 184       2.928  17.990   6.562  1.00  0.00           H  \\nATOM   2723  HB3 ASP A 184       5.050  16.921   7.352  1.00  0.00           H  \\nATOM   2724  HB2 ASP A 184       4.942  15.757   6.038  1.00  0.00           H  \\nATOM   2725  N   LYS A 185       1.737  17.571   4.401  1.00  0.00           N  \\nATOM   2726  CA  LYS A 185       1.097  17.428   3.081  1.00  0.00           C  \\nATOM   2727  C   LYS A 185       1.963  17.922   1.923  1.00  0.00           C  \\nATOM   2728  O   LYS A 185       1.597  17.707   0.768  1.00  0.00           O  \\nATOM   2729  CB  LYS A 185      -0.241  18.170   3.066  1.00  0.00           C  \\nATOM   2730  CG  LYS A 185      -1.256  17.658   4.078  1.00  0.00           C  \\nATOM   2731  CD  LYS A 185      -1.902  16.363   3.607  1.00  0.00           C  \\nATOM   2732  CE  LYS A 185      -2.870  16.612   2.460  1.00  0.00           C  \\nATOM   2733  NZ  LYS A 185      -3.825  15.485   2.283  1.00  0.00           N  \\nATOM   2734  H   LYS A 185       1.460  18.322   5.017  1.00  0.00           H  \\nATOM   2735  HA  LYS A 185       0.943  16.366   2.887  1.00  0.00           H  \\nATOM   2736  HB3 LYS A 185      -0.072  19.228   3.267  1.00  0.00           H  \\nATOM   2737  HB2 LYS A 185      -0.688  18.093   2.075  1.00  0.00           H  \\nATOM   2738  HG3 LYS A 185      -0.761  17.480   5.033  1.00  0.00           H  \\nATOM   2739  HG2 LYS A 185      -2.032  18.408   4.226  1.00  0.00           H  \\nATOM   2740  HD3 LYS A 185      -1.128  15.670   3.274  1.00  0.00           H  \\nATOM   2741  HD2 LYS A 185      -2.444  15.905   4.434  1.00  0.00           H  \\nATOM   2742  HE3 LYS A 185      -3.434  17.523   2.652  1.00  0.00           H  \\nATOM   2743  HE2 LYS A 185      -2.311  16.746   1.534  1.00  0.00           H  \\nATOM   2744  HZ1 LYS A 185      -4.448  15.688   1.515  1.00  0.00           H  \\nATOM   2745  HZ2 LYS A 185      -3.313  14.636   2.086  1.00  0.00           H  \\nATOM   2746  HZ3 LYS A 185      -4.363  15.363   3.129  1.00  0.00           H  \\nATOM   2747  N   SER A 186       3.100  18.567   2.182  1.00  0.00           N  \\nATOM   2748  CA  SER A 186       3.934  19.125   1.120  1.00  0.00           C  \\nATOM   2749  C   SER A 186       4.500  18.038   0.206  1.00  0.00           C  \\nATOM   2750  O   SER A 186       4.722  18.307  -0.974  1.00  0.00           O  \\nATOM   2751  CB  SER A 186       5.071  19.927   1.723  1.00  0.00           C  \\nATOM   2752  OG  SER A 186       6.016  19.084   2.325  1.00  0.00           O  \\nATOM   2753  H   SER A 186       3.444  18.613   3.131  1.00  0.00           H  \\nATOM   2754  HA  SER A 186       3.304  19.770   0.506  1.00  0.00           H  \\nATOM   2755  HB3 SER A 186       5.552  20.519   0.945  1.00  0.00           H  \\nATOM   2756  HB2 SER A 186       4.674  20.620   2.464  1.00  0.00           H  \\nATOM   2757  HG  SER A 186       5.670  18.879   3.197  1.00  0.00           H  \\nATOM   2758  N   VAL A 187       4.700  16.813   0.710  1.00  0.00           N  \\nATOM   2759  CA  VAL A 187       5.144  15.668  -0.094  1.00  0.00           C  \\nATOM   2760  C   VAL A 187       4.099  15.228  -1.116  1.00  0.00           C  \\nATOM   2761  O   VAL A 187       4.447  14.465  -2.003  1.00  0.00           O  \\nATOM   2762  CB  VAL A 187       5.476  14.476   0.823  1.00  0.00           C  \\nATOM   2763  CG1 VAL A 187       6.571  14.853   1.810  1.00  0.00           C  \\nATOM   2764  CG2 VAL A 187       4.222  14.022   1.556  1.00  0.00           C  \\nATOM   2765  H   VAL A 187       4.555  16.676   1.701  1.00  0.00           H  \\nATOM   2766  HA  VAL A 187       6.022  15.977  -0.661  1.00  0.00           H  \\nATOM   2767  HB  VAL A 187       5.859  13.657   0.215  1.00  0.00           H  \\nATOM   2768 HG11 VAL A 187       6.793  13.999   2.451  1.00  0.00           H  \\nATOM   2769 HG12 VAL A 187       7.469  15.140   1.264  1.00  0.00           H  \\nATOM   2770 HG13 VAL A 187       6.235  15.688   2.424  1.00  0.00           H  \\nATOM   2771 HG21 VAL A 187       4.464  13.178   2.202  1.00  0.00           H  \\nATOM   2772 HG22 VAL A 187       3.836  14.843   2.159  1.00  0.00           H  \\nATOM   2773 HG23 VAL A 187       3.467  13.718   0.830  1.00  0.00           H  \\nATOM   2774  N   CYS A 188       2.842  15.680  -1.046  1.00  0.00           N  \\nATOM   2775  CA  CYS A 188       1.797  15.328  -2.013  1.00  0.00           C  \\nATOM   2776  C   CYS A 188       1.844  16.144  -3.313  1.00  0.00           C  \\nATOM   2777  O   CYS A 188       1.136  15.802  -4.258  1.00  0.00           O  \\nATOM   2778  CB  CYS A 188       0.420  15.510  -1.373  1.00  0.00           C  \\nATOM   2779  SG  CYS A 188       0.155  14.505   0.107  1.00  0.00           S  \\nATOM   2780  H   CYS A 188       2.578  16.269  -0.268  1.00  0.00           H  \\nATOM   2781  HA  CYS A 188       1.935  14.286  -2.301  1.00  0.00           H  \\nATOM   2782  HB3 CYS A 188       0.281  16.557  -1.099  1.00  0.00           H  \\nATOM   2783  HB2 CYS A 188      -0.354  15.256  -2.097  1.00  0.00           H  \\nATOM   2784  N   ASP A 189       2.680  17.182  -3.386  1.00  0.00           N  \\nATOM   2785  CA  ASP A 189       2.851  17.995  -4.593  1.00  0.00           C  \\nATOM   2786  C   ASP A 189       3.349  17.135  -5.772  1.00  0.00           C  \\nATOM   2787  O   ASP A 189       4.379  16.455  -5.680  1.00  0.00           O  \\nATOM   2788  CB  ASP A 189       3.799  19.163  -4.276  1.00  0.00           C  \\nATOM   2789  CG  ASP A 189       3.934  20.204  -5.398  1.00  0.00           C  \\nATOM   2790  OD1 ASP A 189       3.777  19.850  -6.591  1.00  0.00           O  \\nATOM   2791  OD2 ASP A 189       4.236  21.363  -5.060  1.00  0.00           O  \\nATOM   2792  H   ASP A 189       3.249  17.403  -2.582  1.00  0.00           H  \\nATOM   2793  HA  ASP A 189       1.884  18.417  -4.862  1.00  0.00           H  \\nATOM   2794  HB3 ASP A 189       3.429  19.671  -3.385  1.00  0.00           H  \\nATOM   2795  HB2 ASP A 189       4.785  18.761  -4.042  1.00  0.00           H  \\nATOM   2796  N   LYS A 190       2.608  17.181  -6.885  1.00  0.00           N  \\nATOM   2797  CA  LYS A 190       2.890  16.434  -8.119  1.00  0.00           C  \\nATOM   2798  C   LYS A 190       4.251  16.763  -8.734  1.00  0.00           C  \\nATOM   2799  O   LYS A 190       4.803  15.960  -9.479  1.00  0.00           O  \\nATOM   2800  CB  LYS A 190       1.789  16.695  -9.149  1.00  0.00           C  \\nATOM   2801  CG  LYS A 190       0.428  16.126  -8.772  1.00  0.00           C  \\nATOM   2802  CD  LYS A 190      -0.596  16.367  -9.871  1.00  0.00           C  \\nATOM   2803  CE  LYS A 190      -1.944  15.758  -9.517  1.00  0.00           C  \\nATOM   2804  NZ  LYS A 190      -2.955  15.985 -10.585  1.00  0.00           N  \\nATOM   2805  H   LYS A 190       1.877  17.878  -6.910  1.00  0.00           H  \\nATOM   2806  HA  LYS A 190       2.917  15.373  -7.872  1.00  0.00           H  \\nATOM   2807  HB3 LYS A 190       1.674  17.769  -9.294  1.00  0.00           H  \\nATOM   2808  HB2 LYS A 190       2.079  16.263 -10.107  1.00  0.00           H  \\nATOM   2809  HG3 LYS A 190       0.517  15.052  -8.600  1.00  0.00           H  \\nATOM   2810  HG2 LYS A 190       0.079  16.595  -7.853  1.00  0.00           H  \\nATOM   2811  HD3 LYS A 190      -0.720  17.440 -10.023  1.00  0.00           H  \\nATOM   2812  HD2 LYS A 190      -0.241  15.925 -10.802  1.00  0.00           H  \\nATOM   2813  HE3 LYS A 190      -1.829  14.685  -9.366  1.00  0.00           H  \\nATOM   2814  HE2 LYS A 190      -2.310  16.197  -8.589  1.00  0.00           H  \\nATOM   2815  HZ1 LYS A 190      -3.833  15.567 -10.312  1.00  0.00           H  \\nATOM   2816  HZ2 LYS A 190      -3.084  16.978 -10.722  1.00  0.00           H  \\nATOM   2817  HZ3 LYS A 190      -2.636  15.566 -11.447  1.00  0.00           H  \\nATOM   2818  N   ASN A 191       4.814  17.922  -8.397  1.00  0.00           N  \\nATOM   2819  CA  ASN A 191       6.080  18.398  -8.947  1.00  0.00           C  \\nATOM   2820  C   ASN A 191       7.295  17.884  -8.160  1.00  0.00           C  \\nATOM   2821  O   ASN A 191       8.428  18.014  -8.623  1.00  0.00           O  \\nATOM   2822  CB  ASN A 191       6.086  19.915  -9.000  1.00  0.00           C  \\nATOM   2823  CG  ASN A 191       5.044  20.464  -9.934  1.00  0.00           C  \\nATOM   2824  OD1 ASN A 191       5.222  20.454 -11.158  1.00  0.00           O  \\nATOM   2825  ND2 ASN A 191       3.959  20.944  -9.382  1.00  0.00           N  \\nATOM   2826  H   ASN A 191       4.329  18.514  -7.737  1.00  0.00           H  \\nATOM   2827  HA  ASN A 191       6.183  18.034  -9.970  1.00  0.00           H  \\nATOM   2828  HB3 ASN A 191       5.909  20.315  -8.001  1.00  0.00           H  \\nATOM   2829  HB2 ASN A 191       7.067  20.263  -9.324  1.00  0.00           H  \\nATOM   2830 HD21 ASN A 191       3.232  21.323  -9.954  1.00  0.00           H  \\nATOM   2831 HD22 ASN A 191       3.858  20.933  -8.387  1.00  0.00           H  \\nATOM   2832  N   LYS A 192       7.087  17.293  -6.976  1.00  0.00           N  \\nATOM   2833  CA  LYS A 192       8.180  16.815  -6.125  1.00  0.00           C  \\nATOM   2834  C   LYS A 192       8.415  15.323  -6.315  1.00  0.00           C  \\nATOM   2835  O   LYS A 192       7.477  14.533  -6.348  1.00  0.00           O  \\nATOM   2836  CB  LYS A 192       7.887  17.116  -4.654  1.00  0.00           C  \\nATOM   2837  CG  LYS A 192       7.861  18.598  -4.308  1.00  0.00           C  \\nATOM   2838  CD  LYS A 192       7.685  18.814  -2.812  1.00  0.00           C  \\nATOM   2839  CE  LYS A 192       7.513  20.289  -2.481  1.00  0.00           C  \\nATOM   2840  NZ  LYS A 192       8.735  21.077  -2.800  1.00  0.00           N  \\nATOM   2841  H   LYS A 192       6.137  17.085  -6.705  1.00  0.00           H  \\nATOM   2842  HA  LYS A 192       9.094  17.327  -6.420  1.00  0.00           H  \\nATOM   2843  HB3 LYS A 192       6.920  16.691  -4.382  1.00  0.00           H  \\nATOM   2844  HB2 LYS A 192       8.642  16.639  -4.028  1.00  0.00           H  \\nATOM   2845  HG3 LYS A 192       8.795  19.063  -4.626  1.00  0.00           H  \\nATOM   2846  HG2 LYS A 192       7.038  19.081  -4.835  1.00  0.00           H  \\nATOM   2847  HD3 LYS A 192       6.807  18.267  -2.467  1.00  0.00           H  \\nATOM   2848  HD2 LYS A 192       8.560  18.433  -2.285  1.00  0.00           H  \\nATOM   2849  HE3 LYS A 192       6.676  20.694  -3.048  1.00  0.00           H  \\nATOM   2850  HE2 LYS A 192       7.292  20.400  -1.420  1.00  0.00           H  \\nATOM   2851  HZ1 LYS A 192       8.580  22.048  -2.566  1.00  0.00           H  \\nATOM   2852  HZ2 LYS A 192       9.515  20.722  -2.265  1.00  0.00           H  \\nATOM   2853  HZ3 LYS A 192       8.939  20.997  -3.786  1.00  0.00           H  \\nATOM   2854  N   ASN A 193       9.691  14.950  -6.369  1.00  0.00           N  \\nATOM   2855  CA  ASN A 193      10.136  13.566  -6.457  1.00  0.00           C  \\nATOM   2856  C   ASN A 193      10.526  13.058  -5.066  1.00  0.00           C  \\nATOM   2857  O   ASN A 193      11.619  13.352  -4.587  1.00  0.00           O  \\nATOM   2858  CB  ASN A 193      11.290  13.430  -7.432  1.00  0.00           C  \\nATOM   2859  CG  ASN A 193      11.725  12.003  -7.618  1.00  0.00           C  \\nATOM   2860  OD1 ASN A 193      11.574  11.172  -6.715  1.00  0.00           O  \\nATOM   2861  ND2 ASN A 193      12.263  11.702  -8.772  1.00  0.00           N  \\nATOM   2862  H   ASN A 193      10.395  15.664  -6.249  1.00  0.00           H  \\nATOM   2863  HA  ASN A 193       9.324  12.939  -6.826  1.00  0.00           H  \\nATOM   2864  HB3 ASN A 193      10.999  13.838  -8.401  1.00  0.00           H  \\nATOM   2865  HB2 ASN A 193      12.140  14.013  -7.074  1.00  0.00           H  \\nATOM   2866 HD21 ASN A 193      12.571  10.767  -8.952  1.00  0.00           H  \\nATOM   2867 HD22 ASN A 193      12.366  12.406  -9.474  1.00  0.00           H  \\nATOM   2868  N   ILE A 194       9.632  12.310  -4.430  1.00  0.00           N  \\nATOM   2869  CA  ILE A 194       9.791  11.842  -3.052  1.00  0.00           C  \\nATOM   2870  C   ILE A 194      10.238  10.378  -3.075  1.00  0.00           C  \\nATOM   2871  O   ILE A 194       9.428   9.465  -2.941  1.00  0.00           O  \\nATOM   2872  CB  ILE A 194       8.481  12.080  -2.275  1.00  0.00           C  \\nATOM   2873  CG1 ILE A 194       7.989  13.545  -2.332  1.00  0.00           C  \\nATOM   2874  CG2 ILE A 194       8.625  11.631  -0.816  1.00  0.00           C  \\nATOM   2875  CD1 ILE A 194       8.959  14.589  -1.759  1.00  0.00           C  \\nATOM   2876  H   ILE A 194       8.770  12.071  -4.900  1.00  0.00           H  \\nATOM   2877  HA  ILE A 194      10.577  12.412  -2.557  1.00  0.00           H  \\nATOM   2878  HB  ILE A 194       7.708  11.465  -2.735  1.00  0.00           H  \\nATOM   2879 HG13 ILE A 194       7.760  13.814  -3.362  1.00  0.00           H  \\nATOM   2880 HG12 ILE A 194       7.049  13.611  -1.784  1.00  0.00           H  \\nATOM   2881 HG21 ILE A 194       8.747  10.548  -0.791  1.00  0.00           H  \\nATOM   2882 HG22 ILE A 194       9.486  12.105  -0.344  1.00  0.00           H  \\nATOM   2883 HG23 ILE A 194       7.717  11.868  -0.260  1.00  0.00           H  \\nATOM   2884 HD11 ILE A 194       9.191  14.366  -0.718  1.00  0.00           H  \\nATOM   2885 HD12 ILE A 194       9.880  14.613  -2.341  1.00  0.00           H  \\nATOM   2886 HD13 ILE A 194       8.489  15.572  -1.797  1.00  0.00           H  \\nATOM   2887  N   ALA A 195      11.526  10.150  -3.326  1.00  0.00           N  \\nATOM   2888  CA  ALA A 195      12.088   8.803  -3.404  1.00  0.00           C  \\nATOM   2889  C   ALA A 195      12.666   8.351  -2.054  1.00  0.00           C  \\nATOM   2890  O   ALA A 195      13.237   9.157  -1.321  1.00  0.00           O  \\nATOM   2891  CB  ALA A 195      13.119   8.747  -4.537  1.00  0.00           C  \\nATOM   2892  H   ALA A 195      12.160  10.935  -3.361  1.00  0.00           H  \\nATOM   2893  HA  ALA A 195      11.285   8.112  -3.655  1.00  0.00           H  \\nATOM   2894  HB1 ALA A 195      13.939   9.433  -4.325  1.00  0.00           H  \\nATOM   2895  HB2 ALA A 195      13.519   7.736  -4.611  1.00  0.00           H  \\nATOM   2896  HB3 ALA A 195      12.647   9.017  -5.481  1.00  0.00           H  \\nATOM   2897  N   GLY A 196      12.531   7.059  -1.747  1.00  0.00           N  \\nATOM   2898  CA  GLY A 196      13.169   6.424  -0.590  1.00  0.00           C  \\nATOM   2899  C   GLY A 196      14.384   5.583  -0.991  1.00  0.00           C  \\nATOM   2900  O   GLY A 196      14.913   5.721  -2.096  1.00  0.00           O  \\nATOM   2901  H   GLY A 196      12.095   6.450  -2.426  1.00  0.00           H  \\nATOM   2902  HA3 GLY A 196      12.447   5.777  -0.093  1.00  0.00           H  \\nATOM   2903  HA2 GLY A 196      13.495   7.177   0.128  1.00  0.00           H  \\nATOM   2904  N   ASP A 197      14.781   4.650  -0.123  1.00  0.00           N  \\nATOM   2905  CA  ASP A 197      15.955   3.783  -0.328  1.00  0.00           C  \\nATOM   2906  C   ASP A 197      15.872   2.935  -1.608  1.00  0.00           C  \\nATOM   2907  O   ASP A 197      16.887   2.551  -2.179  1.00  0.00           O  \\nATOM   2908  CB  ASP A 197      16.136   2.855   0.876  1.00  0.00           C  \\nATOM   2909  CG  ASP A 197      16.569   3.595   2.134  1.00  0.00           C  \\nATOM   2910  OD1 ASP A 197      16.940   4.740   2.028  1.00  0.00           O  \\nATOM   2911  OD2 ASP A 197      16.525   3.009   3.189  1.00  0.00           O  \\nATOM   2912  H   ASP A 197      14.377   4.649   0.803  1.00  0.00           H  \\nATOM   2913  HA  ASP A 197      16.845   4.408  -0.405  1.00  0.00           H  \\nATOM   2914  HB3 ASP A 197      15.198   2.338   1.081  1.00  0.00           H  \\nATOM   2915  HB2 ASP A 197      16.883   2.097   0.642  1.00  0.00           H  \\nATOM   2916  N   TRP A 198      14.670   2.680  -2.124  1.00  0.00           N  \\nATOM   2917  CA  TRP A 198      14.439   1.905  -3.347  1.00  0.00           C  \\nATOM   2918  C   TRP A 198      14.746   2.671  -4.649  1.00  0.00           C  \\nATOM   2919  O   TRP A 198      14.702   2.070  -5.719  1.00  0.00           O  \\nATOM   2920  CB  TRP A 198      12.992   1.414  -3.328  1.00  0.00           C  \\nATOM   2921  CG  TRP A 198      12.035   2.421  -2.789  1.00  0.00           C  \\nATOM   2922  CD1 TRP A 198      11.393   3.358  -3.508  1.00  0.00           C  \\nATOM   2923  CD2 TRP A 198      11.660   2.658  -1.401  1.00  0.00           C  \\nATOM   2924  NE1 TRP A 198      10.629   4.135  -2.674  1.00  0.00           N  \\nATOM   2925  CE2 TRP A 198      10.743   3.750  -1.364  1.00  0.00           C  \\nATOM   2926  CE3 TRP A 198      12.018   2.064  -0.175  1.00  0.00           C  \\nATOM   2927  CZ2 TRP A 198      10.187   4.222  -0.173  1.00  0.00           C  \\nATOM   2928  CZ3 TRP A 198      11.466   2.529   1.031  1.00  0.00           C  \\nATOM   2929  CH2 TRP A 198      10.551   3.594   1.026  1.00  0.00           C  \\nATOM   2930  H   TRP A 198      13.873   3.105  -1.672  1.00  0.00           H  \\nATOM   2931  HA  TRP A 198      15.096   1.034  -3.336  1.00  0.00           H  \\nATOM   2932  HB3 TRP A 198      12.688   1.126  -4.335  1.00  0.00           H  \\nATOM   2933  HB2 TRP A 198      12.935   0.528  -2.697  1.00  0.00           H  \\nATOM   2934  HD1 TRP A 198      11.475   3.484  -4.579  1.00  0.00           H  \\nATOM   2935  HE1 TRP A 198      10.132   4.950  -3.007  1.00  0.00           H  \\nATOM   2936  HE3 TRP A 198      12.715   1.240  -0.165  1.00  0.00           H  \\nATOM   2937  HZ2 TRP A 198       9.487   5.044  -0.166  1.00  0.00           H  \\nATOM   2938  HZ3 TRP A 198      11.733   2.062   1.967  1.00  0.00           H  \\nATOM   2939  HH2 TRP A 198      10.126   3.933   1.960  1.00  0.00           H  \\nATOM   2940  N   GLY A 199      15.078   3.966  -4.585  1.00  0.00           N  \\nATOM   2941  CA  GLY A 199      15.558   4.757  -5.726  1.00  0.00           C  \\nATOM   2942  C   GLY A 199      14.497   5.422  -6.614  1.00  0.00           C  \\nATOM   2943  O   GLY A 199      14.843   5.990  -7.646  1.00  0.00           O  \\nATOM   2944  H   GLY A 199      15.173   4.380  -3.669  1.00  0.00           H  \\nATOM   2945  HA3 GLY A 199      16.172   4.123  -6.368  1.00  0.00           H  \\nATOM   2946  HA2 GLY A 199      16.199   5.549  -5.339  1.00  0.00           H  \\nATOM   2947  N   PHE A 200      13.214   5.394  -6.245  1.00  0.00           N  \\nATOM   2948  CA  PHE A 200      12.131   6.002  -7.035  1.00  0.00           C  \\nATOM   2949  C   PHE A 200      11.000   6.528  -6.144  1.00  0.00           C  \\nATOM   2950  O   PHE A 200      10.899   6.146  -4.985  1.00  0.00           O  \\nATOM   2951  CB  PHE A 200      11.606   4.996  -8.076  1.00  0.00           C  \\nATOM   2952  CG  PHE A 200      10.862   3.806  -7.500  1.00  0.00           C  \\nATOM   2953  CD1 PHE A 200      11.571   2.658  -7.101  1.00  0.00           C  \\nATOM   2954  CD2 PHE A 200       9.459   3.832  -7.371  1.00  0.00           C  \\nATOM   2955  CE1 PHE A 200      10.884   1.573  -6.532  1.00  0.00           C  \\nATOM   2956  CE2 PHE A 200       8.775   2.729  -6.830  1.00  0.00           C  \\nATOM   2957  CZ  PHE A 200       9.489   1.605  -6.390  1.00  0.00           C  \\nATOM   2958  H   PHE A 200      12.982   5.002  -5.343  1.00  0.00           H  \\nATOM   2959  HA  PHE A 200      12.535   6.859  -7.575  1.00  0.00           H  \\nATOM   2960  HB3 PHE A 200      10.942   5.520  -8.762  1.00  0.00           H  \\nATOM   2961  HB2 PHE A 200      12.445   4.631  -8.669  1.00  0.00           H  \\nATOM   2962  HD1 PHE A 200      12.645   2.613  -7.209  1.00  0.00           H  \\nATOM   2963  HD2 PHE A 200       8.900   4.696  -7.699  1.00  0.00           H  \\nATOM   2964  HE1 PHE A 200      11.433   0.709  -6.190  1.00  0.00           H  \\nATOM   2965  HE2 PHE A 200       7.697   2.721  -6.758  1.00  0.00           H  \\nATOM   2966  HZ  PHE A 200       8.965   0.762  -5.965  1.00  0.00           H  \\nATOM   2967  N   ASP A 201      10.132   7.396  -6.663  1.00  0.00           N  \\nATOM   2968  CA  ASP A 201       8.924   7.830  -5.943  1.00  0.00           C  \\nATOM   2969  C   ASP A 201       7.735   6.916  -6.297  1.00  0.00           C  \\nATOM   2970  O   ASP A 201       7.245   6.999  -7.431  1.00  0.00           O  \\nATOM   2971  CB  ASP A 201       8.637   9.302  -6.271  1.00  0.00           C  \\nATOM   2972  CG  ASP A 201       7.435   9.890  -5.519  1.00  0.00           C  \\nATOM   2973  OD1 ASP A 201       6.570   9.137  -5.018  1.00  0.00           O  \\nATOM   2974  OD2 ASP A 201       7.362  11.137  -5.456  1.00  0.00           O  \\nATOM   2975  H   ASP A 201      10.247   7.699  -7.619  1.00  0.00           H  \\nATOM   2976  HA  ASP A 201       9.114   7.777  -4.871  1.00  0.00           H  \\nATOM   2977  HB3 ASP A 201       8.469   9.407  -7.343  1.00  0.00           H  \\nATOM   2978  HB2 ASP A 201       9.520   9.886  -6.015  1.00  0.00           H  \\nATOM   2979  N   PRO A 202       7.248   6.054  -5.381  1.00  0.00           N  \\nATOM   2980  CA  PRO A 202       6.161   5.126  -5.686  1.00  0.00           C  \\nATOM   2981  C   PRO A 202       4.883   5.827  -6.154  1.00  0.00           C  \\nATOM   2982  O   PRO A 202       4.112   5.242  -6.918  1.00  0.00           O  \\nATOM   2983  CB  PRO A 202       5.948   4.416  -4.345  1.00  0.00           C  \\nATOM   2984  CG  PRO A 202       7.277   4.491  -3.676  1.00  0.00           C  \\nATOM   2985  CD  PRO A 202       7.804   5.853  -4.042  1.00  0.00           C  \\nATOM   2986  HA  PRO A 202       6.466   4.450  -6.485  1.00  0.00           H  \\nATOM   2987  HB3 PRO A 202       5.154   4.921  -3.775  1.00  0.00           H  \\nATOM   2988  HB2 PRO A 202       5.613   3.382  -4.515  1.00  0.00           H  \\nATOM   2989  HG3 PRO A 202       7.164   4.359  -2.590  1.00  0.00           H  \\nATOM   2990  HG2 PRO A 202       7.927   3.677  -4.029  1.00  0.00           H  \\nATOM   2991  HD3 PRO A 202       7.428   6.598  -3.325  1.00  0.00           H  \\nATOM   2992  HD2 PRO A 202       8.904   5.834  -4.040  1.00  0.00           H  \\nATOM   2993  N   ALA A 203       4.697   7.104  -5.808  1.00  0.00           N  \\nATOM   2994  CA  ALA A 203       3.526   7.881  -6.206  1.00  0.00           C  \\nATOM   2995  C   ALA A 203       3.440   8.094  -7.719  1.00  0.00           C  \\nATOM   2996  O   ALA A 203       2.367   8.377  -8.248  1.00  0.00           O  \\nATOM   2997  CB  ALA A 203       3.589   9.236  -5.509  1.00  0.00           C  \\nATOM   2998  H   ALA A 203       5.425   7.584  -5.298  1.00  0.00           H  \\nATOM   2999  HA  ALA A 203       2.625   7.360  -5.882  1.00  0.00           H  \\nATOM   3000  HB1 ALA A 203       2.687   9.799  -5.748  1.00  0.00           H  \\nATOM   3001  HB2 ALA A 203       3.665   9.093  -4.432  1.00  0.00           H  \\nATOM   3002  HB3 ALA A 203       4.448   9.802  -5.868  1.00  0.00           H  \\nATOM   3003  N   LYS A 204       4.552   7.945  -8.448  1.00  0.00           N  \\nATOM   3004  CA  LYS A 204       4.555   8.090  -9.903  1.00  0.00           C  \\nATOM   3005  C   LYS A 204       4.094   6.817 -10.612  1.00  0.00           C  \\nATOM   3006  O   LYS A 204       3.938   6.856 -11.821  1.00  0.00           O  \\nATOM   3007  CB  LYS A 204       5.950   8.481 -10.393  1.00  0.00           C  \\nATOM   3008  CG  LYS A 204       6.449   9.820  -9.865  1.00  0.00           C  \\nATOM   3009  CD  LYS A 204       5.583  10.968 -10.362  1.00  0.00           C  \\nATOM   3010  CE  LYS A 204       5.908  11.323 -11.805  1.00  0.00           C  \\nATOM   3011  NZ  LYS A 204       5.121  12.493 -12.281  1.00  0.00           N  \\nATOM   3012  H   LYS A 204       5.398   7.665  -7.972  1.00  0.00           H  \\nATOM   3013  HA  LYS A 204       3.825   8.852 -10.182  1.00  0.00           H  \\nATOM   3014  HB3 LYS A 204       6.668   7.716 -10.096  1.00  0.00           H  \\nATOM   3015  HB2 LYS A 204       5.954   8.528 -11.482  1.00  0.00           H  \\nATOM   3016  HG3 LYS A 204       6.435   9.810  -8.775  1.00  0.00           H  \\nATOM   3017  HG2 LYS A 204       7.475   9.981 -10.195  1.00  0.00           H  \\nATOM   3018  HD3 LYS A 204       4.531  10.686 -10.294  1.00  0.00           H  \\nATOM   3019  HD2 LYS A 204       5.747  11.845  -9.736  1.00  0.00           H  \\nATOM   3020  HE3 LYS A 204       6.969  11.554 -11.893  1.00  0.00           H  \\nATOM   3021  HE2 LYS A 204       5.690  10.470 -12.448  1.00  0.00           H  \\nATOM   3022  HZ1 LYS A 204       5.365  12.696 -13.240  1.00  0.00           H  \\nATOM   3023  HZ2 LYS A 204       4.135  12.283 -12.222  1.00  0.00           H  \\nATOM   3024  HZ3 LYS A 204       5.329  13.296 -11.704  1.00  0.00           H  \\nATOM   3025  N   TRP A 205       3.900   5.686  -9.934  1.00  0.00           N  \\nATOM   3026  CA  TRP A 205       3.454   4.422 -10.546  1.00  0.00           C  \\nATOM   3027  C   TRP A 205       2.136   3.911  -9.947  1.00  0.00           C  \\nATOM   3028  O   TRP A 205       1.871   2.713  -9.971  1.00  0.00           O  \\nATOM   3029  CB  TRP A 205       4.565   3.365 -10.495  1.00  0.00           C  \\nATOM   3030  CG  TRP A 205       5.737   3.528 -11.406  1.00  0.00           C  \\nATOM   3031  CD1 TRP A 205       5.871   2.926 -12.611  1.00  0.00           C  \\nATOM   3032  CD2 TRP A 205       6.957   4.295 -11.205  1.00  0.00           C  \\nATOM   3033  NE1 TRP A 205       7.086   3.270 -13.163  1.00  0.00           N  \\nATOM   3034  CE2 TRP A 205       7.776   4.154 -12.364  1.00  0.00           C  \\nATOM   3035  CE3 TRP A 205       7.443   5.124 -10.177  1.00  0.00           C  \\nATOM   3036  CZ2 TRP A 205       8.975   4.856 -12.529  1.00  0.00           C  \\nATOM   3037  CZ3 TRP A 205       8.667   5.805 -10.312  1.00  0.00           C  \\nATOM   3038  CH2 TRP A 205       9.428   5.683 -11.488  1.00  0.00           C  \\nATOM   3039  H   TRP A 205       4.013   5.699  -8.931  1.00  0.00           H  \\nATOM   3040  HA  TRP A 205       3.222   4.586 -11.599  1.00  0.00           H  \\nATOM   3041  HB3 TRP A 205       4.118   2.421 -10.806  1.00  0.00           H  \\nATOM   3042  HB2 TRP A 205       4.918   3.262  -9.469  1.00  0.00           H  \\nATOM   3043  HD1 TRP A 205       5.148   2.259 -13.059  1.00  0.00           H  \\nATOM   3044  HE1 TRP A 205       7.402   2.928 -14.059  1.00  0.00           H  \\nATOM   3045  HE3 TRP A 205       6.874   5.206  -9.262  1.00  0.00           H  \\nATOM   3046  HZ2 TRP A 205       9.563   4.719 -13.424  1.00  0.00           H  \\nATOM   3047  HZ3 TRP A 205       9.032   6.406  -9.491  1.00  0.00           H  \\nATOM   3048  HH2 TRP A 205      10.378   6.189 -11.574  1.00  0.00           H  \\nATOM   3049  N   ALA A 206       1.304   4.809  -9.421  1.00  0.00           N  \\nATOM   3050  CA  ALA A 206       0.039   4.433  -8.809  1.00  0.00           C  \\nATOM   3051  C   ALA A 206      -1.110   4.320  -9.822  1.00  0.00           C  \\nATOM   3052  O   ALA A 206      -1.350   5.227 -10.628  1.00  0.00           O  \\nATOM   3053  CB  ALA A 206      -0.270   5.417  -7.694  1.00  0.00           C  \\nATOM   3054  H   ALA A 206       1.570   5.783  -9.400  1.00  0.00           H  \\nATOM   3055  HA  ALA A 206       0.167   3.463  -8.329  1.00  0.00           H  \\nATOM   3056  HB1 ALA A 206      -0.324   6.429  -8.099  1.00  0.00           H  \\nATOM   3057  HB2 ALA A 206      -1.231   5.158  -7.250  1.00  0.00           H  \\nATOM   3058  HB3 ALA A 206       0.510   5.372  -6.935  1.00  0.00           H  \\nATOM   3059  N   TYR A 207      -1.863   3.226  -9.727  1.00  0.00           N  \\nATOM   3060  CA  TYR A 207      -3.013   2.914 -10.574  1.00  0.00           C  \\nATOM   3061  C   TYR A 207      -4.221   2.470  -9.748  1.00  0.00           C  \\nATOM   3062  O   TYR A 207      -4.072   1.966  -8.643  1.00  0.00           O  \\nATOM   3063  CB  TYR A 207      -2.622   1.831 -11.589  1.00  0.00           C  \\nATOM   3064  CG  TYR A 207      -1.618   2.306 -12.619  1.00  0.00           C  \\nATOM   3065  CD1 TYR A 207      -2.074   2.840 -13.837  1.00  0.00           C  \\nATOM   3066  CD2 TYR A 207      -0.238   2.254 -12.345  1.00  0.00           C  \\nATOM   3067  CE1 TYR A 207      -1.159   3.357 -14.769  1.00  0.00           C  \\nATOM   3068  CE2 TYR A 207       0.688   2.765 -13.276  1.00  0.00           C  \\nATOM   3069  CZ  TYR A 207       0.221   3.330 -14.482  1.00  0.00           C  \\nATOM   3070  OH  TYR A 207       1.079   3.876 -15.379  1.00  0.00           O  \\nATOM   3071  H   TYR A 207      -1.579   2.519  -9.065  1.00  0.00           H  \\nATOM   3072  HA  TYR A 207      -3.299   3.811 -11.124  1.00  0.00           H  \\nATOM   3073  HB3 TYR A 207      -3.522   1.500 -12.108  1.00  0.00           H  \\nATOM   3074  HB2 TYR A 207      -2.216   0.971 -11.056  1.00  0.00           H  \\nATOM   3075  HD1 TYR A 207      -3.131   2.856 -14.056  1.00  0.00           H  \\nATOM   3076  HD2 TYR A 207       0.106   1.842 -11.407  1.00  0.00           H  \\nATOM   3077  HE1 TYR A 207      -1.520   3.764 -15.702  1.00  0.00           H  \\nATOM   3078  HE2 TYR A 207       1.746   2.741 -13.064  1.00  0.00           H  \\nATOM   3079  HH  TYR A 207       0.606   4.488 -15.946  1.00  0.00           H  \\nATOM   3080  N   GLN A 208      -5.421   2.617 -10.300  1.00  0.00           N  \\nATOM   3081  CA  GLN A 208      -6.639   1.964  -9.814  1.00  0.00           C  \\nATOM   3082  C   GLN A 208      -7.211   1.079 -10.914  1.00  0.00           C  \\nATOM   3083  O   GLN A 208      -7.139   1.422 -12.097  1.00  0.00           O  \\nATOM   3084  CB  GLN A 208      -7.678   2.997  -9.368  1.00  0.00           C  \\nATOM   3085  CG  GLN A 208      -7.394   3.620  -8.012  1.00  0.00           C  \\nATOM   3086  CD  GLN A 208      -8.431   4.657  -7.623  1.00  0.00           C  \\nATOM   3087  OE1 GLN A 208      -8.782   5.533  -8.418  1.00  0.00           O  \\nATOM   3088  NE2 GLN A 208      -8.927   4.563  -6.394  1.00  0.00           N  \\nATOM   3089  H   GLN A 208      -5.480   3.107 -11.180  1.00  0.00           H  \\nATOM   3090  HA  GLN A 208      -6.408   1.322  -8.963  1.00  0.00           H  \\nATOM   3091  HB3 GLN A 208      -7.730   3.800 -10.103  1.00  0.00           H  \\nATOM   3092  HB2 GLN A 208      -8.661   2.529  -9.324  1.00  0.00           H  \\nATOM   3093  HG3 GLN A 208      -7.394   2.835  -7.256  1.00  0.00           H  \\nATOM   3094  HG2 GLN A 208      -6.419   4.107  -8.044  1.00  0.00           H  \\nATOM   3095 HE21 GLN A 208      -9.615   5.220  -6.081  1.00  0.00           H  \\nATOM   3096 HE22 GLN A 208      -8.614   3.838  -5.782  1.00  0.00           H  \\nATOM   3097  N   TYR A 209      -7.780  -0.060 -10.534  1.00  0.00           N  \\nATOM   3098  CA  TYR A 209      -8.380  -0.978 -11.491  1.00  0.00           C  \\nATOM   3099  C   TYR A 209      -9.835  -0.600 -11.778  1.00  0.00           C  \\nATOM   3100  O   TYR A 209     -10.666  -0.487 -10.879  1.00  0.00           O  \\nATOM   3101  CB  TYR A 209      -8.293  -2.417 -10.975  1.00  0.00           C  \\nATOM   3102  CG  TYR A 209      -8.865  -3.443 -11.928  1.00  0.00           C  \\nATOM   3103  CD1 TYR A 209      -8.545  -3.391 -13.277  1.00  0.00           C  \\nATOM   3104  CD2 TYR A 209      -9.710  -4.435 -11.453  1.00  0.00           C  \\nATOM   3105  CE1 TYR A 209      -9.068  -4.328 -14.147  1.00  0.00           C  \\nATOM   3106  CE2 TYR A 209     -10.233  -5.372 -12.324  1.00  0.00           C  \\nATOM   3107  CZ  TYR A 209      -9.914  -5.320 -13.665  1.00  0.00           C  \\nATOM   3108  OH  TYR A 209     -10.434  -6.254 -14.532  1.00  0.00           O  \\nATOM   3109  H   TYR A 209      -7.865  -0.275  -9.550  1.00  0.00           H  \\nATOM   3110  HA  TYR A 209      -7.826  -0.915 -12.426  1.00  0.00           H  \\nATOM   3111  HB3 TYR A 209      -7.250  -2.673 -10.785  1.00  0.00           H  \\nATOM   3112  HB2 TYR A 209      -8.827  -2.496 -10.029  1.00  0.00           H  \\nATOM   3113  HD1 TYR A 209      -7.881  -2.611 -13.650  1.00  0.00           H  \\nATOM   3114  HD2 TYR A 209      -9.963  -4.475 -10.394  1.00  0.00           H  \\nATOM   3115  HE1 TYR A 209      -8.816  -4.287 -15.207  1.00  0.00           H  \\nATOM   3116  HE2 TYR A 209     -10.897  -6.151 -11.951  1.00  0.00           H  \\nATOM   3117  HH  TYR A 209     -10.078  -6.106 -15.412  1.00  0.00           H  \\nATOM   3118  N   ASP A 210     -10.130  -0.415 -13.058  1.00  0.00           N  \\nATOM   3119  CA  ASP A 210     -11.461  -0.170 -13.588  1.00  0.00           C  \\nATOM   3120  C   ASP A 210     -12.069  -1.511 -13.993  1.00  0.00           C  \\nATOM   3121  O   ASP A 210     -11.810  -2.031 -15.080  1.00  0.00           O  \\nATOM   3122  CB  ASP A 210     -11.316   0.805 -14.754  1.00  0.00           C  \\nATOM   3123  CG  ASP A 210     -12.608   1.176 -15.476  1.00  0.00           C  \\nATOM   3124  OD1 ASP A 210     -13.623   0.450 -15.388  1.00  0.00           O  \\nATOM   3125  OD2 ASP A 210     -12.541   2.188 -16.215  1.00  0.00           O  \\nATOM   3126  H   ASP A 210      -9.402  -0.584 -13.737  1.00  0.00           H  \\nATOM   3127  HA  ASP A 210     -12.093   0.293 -12.830  1.00  0.00           H  \\nATOM   3128  HB3 ASP A 210     -10.881   1.719 -14.347  1.00  0.00           H  \\nATOM   3129  HB2 ASP A 210     -10.619   0.396 -15.485  1.00  0.00           H  \\nATOM   3130  N   GLU A 211     -12.859  -2.086 -13.086  1.00  0.00           N  \\nATOM   3131  CA  GLU A 211     -13.465  -3.406 -13.257  1.00  0.00           C  \\nATOM   3132  C   GLU A 211     -14.391  -3.469 -14.475  1.00  0.00           C  \\nATOM   3133  O   GLU A 211     -14.407  -4.474 -15.184  1.00  0.00           O  \\nATOM   3134  CB  GLU A 211     -14.245  -3.792 -11.999  1.00  0.00           C  \\nATOM   3135  CG  GLU A 211     -13.378  -4.049 -10.775  1.00  0.00           C  \\nATOM   3136  CD  GLU A 211     -14.182  -4.330  -9.536  1.00  0.00           C  \\nATOM   3137  OE1 GLU A 211     -15.025  -5.193  -9.581  1.00  0.00           O  \\nATOM   3138  OE2 GLU A 211     -13.952  -3.681  -8.543  1.00  0.00           O  \\nATOM   3139  H   GLU A 211     -12.954  -1.635 -12.187  1.00  0.00           H  \\nATOM   3140  HA  GLU A 211     -12.673  -4.135 -13.426  1.00  0.00           H  \\nATOM   3141  HB3 GLU A 211     -14.948  -2.997 -11.749  1.00  0.00           H  \\nATOM   3142  HB2 GLU A 211     -14.826  -4.693 -12.193  1.00  0.00           H  \\nATOM   3143  HG3 GLU A 211     -12.730  -4.902 -10.975  1.00  0.00           H  \\nATOM   3144  HG2 GLU A 211     -12.745  -3.180 -10.602  1.00  0.00           H  \\nATOM   3145  N   LYS A 212     -15.098  -2.369 -14.763  1.00  0.00           N  \\nATOM   3146  CA  LYS A 212     -16.096  -2.297 -15.834  1.00  0.00           C  \\nATOM   3147  C   LYS A 212     -15.441  -2.306 -17.212  1.00  0.00           C  \\nATOM   3148  O   LYS A 212     -15.948  -2.952 -18.123  1.00  0.00           O  \\nATOM   3149  CB  LYS A 212     -16.962  -1.046 -15.675  1.00  0.00           C  \\nATOM   3150  CG  LYS A 212     -17.907  -1.083 -14.481  1.00  0.00           C  \\nATOM   3151  CD  LYS A 212     -18.722   0.198 -14.382  1.00  0.00           C  \\nATOM   3152  CE  LYS A 212     -19.662   0.166 -13.187  1.00  0.00           C  \\nATOM   3153  NZ  LYS A 212     -20.437   1.430 -13.054  1.00  0.00           N  \\nATOM   3154  H   LYS A 212     -14.873  -1.528 -14.251  1.00  0.00           H  \\nATOM   3155  HA  LYS A 212     -16.727  -3.184 -15.783  1.00  0.00           H  \\nATOM   3156  HB3 LYS A 212     -16.320  -0.171 -15.568  1.00  0.00           H  \\nATOM   3157  HB2 LYS A 212     -17.563  -0.904 -16.573  1.00  0.00           H  \\nATOM   3158  HG3 LYS A 212     -18.587  -1.929 -14.581  1.00  0.00           H  \\nATOM   3159  HG2 LYS A 212     -17.330  -1.208 -13.565  1.00  0.00           H  \\nATOM   3160  HD3 LYS A 212     -18.049   1.051 -14.282  1.00  0.00           H  \\nATOM   3161  HD2 LYS A 212     -19.310   0.326 -15.291  1.00  0.00           H  \\nATOM   3162  HE3 LYS A 212     -20.360  -0.664 -13.295  1.00  0.00           H  \\nATOM   3163  HE2 LYS A 212     -19.086   0.010 -12.275  1.00  0.00           H  \\nATOM   3164  HZ1 LYS A 212     -21.047   1.368 -12.251  1.00  0.00           H  \\nATOM   3165  HZ2 LYS A 212     -19.800   2.205 -12.934  1.00  0.00           H  \\nATOM   3166  HZ3 LYS A 212     -20.990   1.575 -13.886  1.00  0.00           H  \\nATOM   3167  N   ASN A 213     -14.337  -1.578 -17.364  1.00  0.00           N  \\nATOM   3168  CA  ASN A 213     -13.628  -1.463 -18.640  1.00  0.00           C  \\nATOM   3169  C   ASN A 213     -12.374  -2.350 -18.725  1.00  0.00           C  \\nATOM   3170  O   ASN A 213     -11.661  -2.292 -19.724  1.00  0.00           O  \\nATOM   3171  CB  ASN A 213     -13.263  -0.012 -18.895  1.00  0.00           C  \\nATOM   3172  CG  ASN A 213     -14.468   0.853 -19.140  1.00  0.00           C  \\nATOM   3173  OD1 ASN A 213     -15.293   0.557 -20.012  1.00  0.00           O  \\nATOM   3174  ND2 ASN A 213     -14.585   1.917 -18.388  1.00  0.00           N  \\nATOM   3175  H   ASN A 213     -14.025  -1.030 -16.575  1.00  0.00           H  \\nATOM   3176  HA  ASN A 213     -14.282  -1.807 -19.442  1.00  0.00           H  \\nATOM   3177  HB3 ASN A 213     -12.715   0.381 -18.038  1.00  0.00           H  \\nATOM   3178  HB2 ASN A 213     -12.603   0.050 -19.761  1.00  0.00           H  \\nATOM   3179 HD21 ASN A 213     -15.368   2.529 -18.506  1.00  0.00           H  \\nATOM   3180 HD22 ASN A 213     -13.894   2.117 -17.694  1.00  0.00           H  \\nATOM   3181  N   ASN A 214     -12.091  -3.151 -17.690  1.00  0.00           N  \\nATOM   3182  CA  ASN A 214     -10.961  -4.077 -17.605  1.00  0.00           C  \\nATOM   3183  C   ASN A 214      -9.602  -3.409 -17.907  1.00  0.00           C  \\nATOM   3184  O   ASN A 214      -8.791  -3.929 -18.675  1.00  0.00           O  \\nATOM   3185  CB  ASN A 214     -11.188  -5.252 -18.539  1.00  0.00           C  \\nATOM   3186  CG  ASN A 214     -10.252  -6.397 -18.267  1.00  0.00           C  \\nATOM   3187  OD1 ASN A 214      -9.843  -6.621 -17.122  1.00  0.00           O  \\nATOM   3188  ND2 ASN A 214      -9.904  -7.124 -19.297  1.00  0.00           N  \\nATOM   3189  H   ASN A 214     -12.663  -3.057 -16.864  1.00  0.00           H  \\nATOM   3190  HA  ASN A 214     -10.906  -4.395 -16.564  1.00  0.00           H  \\nATOM   3191  HB3 ASN A 214     -12.215  -5.606 -18.436  1.00  0.00           H  \\nATOM   3192  HB2 ASN A 214     -11.056  -4.928 -19.571  1.00  0.00           H  \\nATOM   3193 HD21 ASN A 214      -9.283  -7.899 -19.176  1.00  0.00           H  \\nATOM   3194 HD22 ASN A 214     -10.259  -6.906 -20.206  1.00  0.00           H  \\nATOM   3195  N   LYS A 215      -9.351  -2.245 -17.297  1.00  0.00           N  \\nATOM   3196  CA  LYS A 215      -8.112  -1.467 -17.474  1.00  0.00           C  \\nATOM   3197  C   LYS A 215      -7.646  -0.826 -16.170  1.00  0.00           C  \\nATOM   3198  O   LYS A 215      -8.405  -0.731 -15.211  1.00  0.00           O  \\nATOM   3199  CB  LYS A 215      -8.307  -0.386 -18.538  1.00  0.00           C  \\nATOM   3200  CG  LYS A 215      -9.362   0.655 -18.192  1.00  0.00           C  \\nATOM   3201  CD  LYS A 215      -9.518   1.677 -19.308  1.00  0.00           C  \\nATOM   3202  CE  LYS A 215     -10.545   2.740 -18.947  1.00  0.00           C  \\nATOM   3203  NZ  LYS A 215     -10.654   3.788 -19.998  1.00  0.00           N  \\nATOM   3204  H   LYS A 215     -10.046  -1.901 -16.649  1.00  0.00           H  \\nATOM   3205  HA  LYS A 215      -7.321  -2.147 -17.788  1.00  0.00           H  \\nATOM   3206  HB3 LYS A 215      -7.364   0.135 -18.704  1.00  0.00           H  \\nATOM   3207  HB2 LYS A 215      -8.593  -0.852 -19.481  1.00  0.00           H  \\nATOM   3208  HG3 LYS A 215     -10.320   0.162 -18.026  1.00  0.00           H  \\nATOM   3209  HG2 LYS A 215      -9.077   1.171 -17.275  1.00  0.00           H  \\nATOM   3210  HD3 LYS A 215      -8.558   2.160 -19.496  1.00  0.00           H  \\nATOM   3211  HD2 LYS A 215      -9.836   1.173 -20.220  1.00  0.00           H  \\nATOM   3212  HE3 LYS A 215     -11.520   2.273 -18.814  1.00  0.00           H  \\nATOM   3213  HE2 LYS A 215     -10.263   3.215 -18.008  1.00  0.00           H  \\nATOM   3214  HZ1 LYS A 215     -11.344   4.472 -19.722  1.00  0.00           H  \\nATOM   3215  HZ2 LYS A 215      -9.759   4.241 -20.117  1.00  0.00           H  \\nATOM   3216  HZ3 LYS A 215     -10.934   3.362 -20.871  1.00  0.00           H  \\nATOM   3217  N   PHE A 216      -6.404  -0.350 -16.150  1.00  0.00           N  \\nATOM   3218  CA  PHE A 216      -5.794   0.267 -14.974  1.00  0.00           C  \\nATOM   3219  C   PHE A 216      -5.546   1.753 -15.240  1.00  0.00           C  \\nATOM   3220  O   PHE A 216      -4.712   2.119 -16.065  1.00  0.00           O  \\nATOM   3221  CB  PHE A 216      -4.520  -0.498 -14.599  1.00  0.00           C  \\nATOM   3222  CG  PHE A 216      -4.803  -1.917 -14.144  1.00  0.00           C  \\nATOM   3223  CD1 PHE A 216      -4.920  -2.209 -12.771  1.00  0.00           C  \\nATOM   3224  CD2 PHE A 216      -5.020  -2.933 -15.092  1.00  0.00           C  \\nATOM   3225  CE1 PHE A 216      -5.251  -3.511 -12.357  1.00  0.00           C  \\nATOM   3226  CE2 PHE A 216      -5.350  -4.234 -14.674  1.00  0.00           C  \\nATOM   3227  CZ  PHE A 216      -5.460  -4.524 -13.305  1.00  0.00           C  \\nATOM   3228  H   PHE A 216      -5.875  -0.337 -17.010  1.00  0.00           H  \\nATOM   3229  HA  PHE A 216      -6.472   0.193 -14.124  1.00  0.00           H  \\nATOM   3230  HB3 PHE A 216      -4.009   0.033 -13.796  1.00  0.00           H  \\nATOM   3231  HB2 PHE A 216      -3.847  -0.520 -15.457  1.00  0.00           H  \\nATOM   3232  HD1 PHE A 216      -4.782  -1.431 -12.035  1.00  0.00           H  \\nATOM   3233  HD2 PHE A 216      -4.957  -2.709 -16.147  1.00  0.00           H  \\nATOM   3234  HE1 PHE A 216      -5.377  -3.730 -11.306  1.00  0.00           H  \\nATOM   3235  HE2 PHE A 216      -5.525  -5.014 -15.401  1.00  0.00           H  \\nATOM   3236  HZ  PHE A 216      -5.722  -5.520 -12.978  1.00  0.00           H  \\nATOM   3237  N   ASN A 217      -6.276   2.618 -14.541  1.00  0.00           N  \\nATOM   3238  CA  ASN A 217      -6.218   4.061 -14.739  1.00  0.00           C  \\nATOM   3239  C   ASN A 217      -5.142   4.662 -13.837  1.00  0.00           C  \\nATOM   3240  O   ASN A 217      -5.069   4.337 -12.652  1.00  0.00           O  \\nATOM   3241  CB  ASN A 217      -7.596   4.675 -14.459  1.00  0.00           C  \\nATOM   3242  CG  ASN A 217      -8.644   4.293 -15.491  1.00  0.00           C  \\nATOM   3243  OD1 ASN A 217      -8.396   4.194 -16.685  1.00  0.00           O  \\nATOM   3244  ND2 ASN A 217      -9.867   4.102 -15.058  1.00  0.00           N  \\nATOM   3245  H   ASN A 217      -6.830   2.268 -13.772  1.00  0.00           H  \\nATOM   3246  HA  ASN A 217      -5.951   4.271 -15.775  1.00  0.00           H  \\nATOM   3247  HB3 ASN A 217      -7.927   4.359 -13.469  1.00  0.00           H  \\nATOM   3248  HB2 ASN A 217      -7.516   5.762 -14.455  1.00  0.00           H  \\nATOM   3249 HD21 ASN A 217     -10.566   3.765 -15.703  1.00  0.00           H  \\nATOM   3250 HD22 ASN A 217     -10.061   4.145 -14.068  1.00  0.00           H  \\nATOM   3251  N   TYR A 218      -4.317   5.550 -14.388  1.00  0.00           N  \\nATOM   3252  CA  TYR A 218      -3.272   6.217 -13.621  1.00  0.00           C  \\nATOM   3253  C   TYR A 218      -3.883   7.196 -12.613  1.00  0.00           C  \\nATOM   3254  O   TYR A 218      -4.684   8.049 -12.991  1.00  0.00           O  \\nATOM   3255  CB  TYR A 218      -2.320   6.945 -14.576  1.00  0.00           C  \\nATOM   3256  CG  TYR A 218      -1.159   7.591 -13.854  1.00  0.00           C  \\nATOM   3257  CD1 TYR A 218      -1.014   8.993 -13.788  1.00  0.00           C  \\nATOM   3258  CD2 TYR A 218      -0.241   6.759 -13.195  1.00  0.00           C  \\nATOM   3259  CE1 TYR A 218       0.077   9.561 -13.092  1.00  0.00           C  \\nATOM   3260  CE2 TYR A 218       0.833   7.322 -12.499  1.00  0.00           C  \\nATOM   3261  CZ  TYR A 218       1.016   8.716 -12.463  1.00  0.00           C  \\nATOM   3262  OH  TYR A 218       2.096   9.183 -11.790  1.00  0.00           O  \\nATOM   3263  H   TYR A 218      -4.476   5.845 -15.341  1.00  0.00           H  \\nATOM   3264  HA  TYR A 218      -2.709   5.464 -13.070  1.00  0.00           H  \\nATOM   3265  HB3 TYR A 218      -2.873   7.703 -15.129  1.00  0.00           H  \\nATOM   3266  HB2 TYR A 218      -1.920   6.230 -15.295  1.00  0.00           H  \\nATOM   3267  HD1 TYR A 218      -1.739   9.638 -14.263  1.00  0.00           H  \\nATOM   3268  HD2 TYR A 218      -0.374   5.687 -13.191  1.00  0.00           H  \\nATOM   3269  HE1 TYR A 218       0.200  10.632 -13.028  1.00  0.00           H  \\nATOM   3270  HE2 TYR A 218       1.513   6.676 -11.965  1.00  0.00           H  \\nATOM   3271  HH  TYR A 218       2.143  10.136 -11.690  1.00  0.00           H  \\nATOM   3272  N   VAL A 219      -3.497   7.087 -11.341  1.00  0.00           N  \\nATOM   3273  CA  VAL A 219      -3.994   7.952 -10.249  1.00  0.00           C  \\nATOM   3274  C   VAL A 219      -2.877   8.632  -9.461  1.00  0.00           C  \\nATOM   3275  O   VAL A 219      -3.149   9.281  -8.441  1.00  0.00           O  \\nATOM   3276  CB  VAL A 219      -4.851   7.127  -9.271  1.00  0.00           C  \\nATOM   3277  CG1 VAL A 219      -6.053   6.531  -9.988  1.00  0.00           C  \\nATOM   3278  CG2 VAL A 219      -4.005   6.033  -8.637  1.00  0.00           C  \\nATOM   3279  H   VAL A 219      -2.833   6.362 -11.107  1.00  0.00           H  \\nATOM   3280  HA  VAL A 219      -4.572   8.763 -10.689  1.00  0.00           H  \\nATOM   3281  HB  VAL A 219      -5.235   7.787  -8.493  1.00  0.00           H  \\nATOM   3282 HG11 VAL A 219      -6.648   5.951  -9.282  1.00  0.00           H  \\nATOM   3283 HG12 VAL A 219      -6.664   7.333 -10.402  1.00  0.00           H  \\nATOM   3284 HG13 VAL A 219      -5.711   5.881 -10.793  1.00  0.00           H  \\nATOM   3285 HG21 VAL A 219      -4.617   5.454  -7.946  1.00  0.00           H  \\nATOM   3286 HG22 VAL A 219      -3.616   5.377  -9.416  1.00  0.00           H  \\nATOM   3287 HG23 VAL A 219      -3.174   6.484  -8.095  1.00  0.00           H  \\nATOM   3288  N   GLY A 220      -1.641   8.440  -9.923  1.00  0.00           N  \\nATOM   3289  CA  GLY A 220      -0.431   8.981  -9.335  1.00  0.00           C  \\nATOM   3290  C   GLY A 220      -0.188  10.451  -9.660  1.00  0.00           C  \\nATOM   3291  O   GLY A 220      -1.081  11.171 -10.109  1.00  0.00           O  \\nATOM   3292  H   GLY A 220      -1.537   7.933 -10.791  1.00  0.00           H  \\nATOM   3293  HA3 GLY A 220      -0.465   8.880  -8.251  1.00  0.00           H  \\nATOM   3294  HA2 GLY A 220       0.415   8.405  -9.711  1.00  0.00           H  \\nATOM   3295  N   LYS A 221       1.038  10.884  -9.381  1.00  0.00           N  \\nATOM   3296  CA  LYS A 221       1.501  12.266  -9.552  1.00  0.00           C  \\nATOM   3297  C   LYS A 221       1.845  12.635 -10.988  1.00  0.00           C  \\nATOM   3298  O   LYS A 221       2.415  11.777 -11.697  1.00  0.00           O  \\nATOM   3299  OXT LYS A 221       1.566  13.800 -11.324  1.00  0.00           O  \\nATOM   3300  CB  LYS A 221       2.709  12.494  -8.660  1.00  0.00           C  \\nATOM   3301  CG  LYS A 221       2.321  12.547  -7.179  1.00  0.00           C  \\nATOM   3302  CD  LYS A 221       3.613  12.875  -6.452  1.00  0.00           C  \\nATOM   3303  CE  LYS A 221       3.450  13.068  -4.960  1.00  0.00           C  \\nATOM   3304  NZ  LYS A 221       4.721  13.654  -4.511  1.00  0.00           N  \\nATOM   3305  H   LYS A 221       1.715  10.187  -9.102  1.00  0.00           H  \\nATOM   3306  HA  LYS A 221       0.705  12.956  -9.273  1.00  0.00           H  \\nATOM   3307  HB3 LYS A 221       3.435  11.700  -8.835  1.00  0.00           H  \\nATOM   3308  HB2 LYS A 221       3.176  13.435  -8.950  1.00  0.00           H  \\nATOM   3309  HG3 LYS A 221       1.585  13.334  -7.012  1.00  0.00           H  \\nATOM   3310  HG2 LYS A 221       1.929  11.587  -6.845  1.00  0.00           H  \\nATOM   3311  HD3 LYS A 221       4.335  12.078  -6.627  1.00  0.00           H  \\nATOM   3312  HD2 LYS A 221       4.025  13.793  -6.872  1.00  0.00           H  \\nATOM   3313  HE3 LYS A 221       2.622  13.751  -4.769  1.00  0.00           H  \\nATOM   3314  HE2 LYS A 221       3.278  12.108  -4.474  1.00  0.00           H  \\nATOM   3315  HZ1 LYS A 221       5.480  13.005  -4.659  1.00  0.00           H  \\nATOM   3316  HZ2 LYS A 221       4.702  13.865  -3.523  1.00  0.00           H  \\nATOM   3317  HZ3 LYS A 221       4.915  14.507  -5.015  1.00  0.00           H  \\nTER    3318      LYS A 221\\nCONECT  659 2625\\nCONECT  741 2779\\nCONECT 2625  659\\nCONECT 2779  741\\nEND\\n\",\"pdb\");\n\tviewer_16745495978711402.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_16745495978711402.zoomTo();\nviewer_16745495978711402.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16745495978711402\"  style=\"position: relative; width: 640px; height: 480px\">\n",
              "        <p id=\"3dmolwarning_16745495978711402\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16745495978711402 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16745495978711402\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16745495978711402 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16745495978711402\"),{backgroundColor:\"white\"});\n",
              "viewer_16745495978711402.zoomTo();\n",
              "\tviewer_16745495978711402.addModel(\"REMARK   1 CREATED WITH OPENMM 7.7, 2023-01-24\\nATOM      1  N   VAL A   1      33.328  -6.825  50.080  1.00  0.00           N  \\nATOM      2  CA  VAL A   1      33.399  -5.965  48.871  1.00  0.00           C  \\nATOM      3  C   VAL A   1      34.509  -6.547  48.012  1.00  0.00           C  \\nATOM      4  O   VAL A   1      35.515  -6.895  48.622  1.00  0.00           O  \\nATOM      5  CB  VAL A   1      33.709  -4.497  49.220  1.00  0.00           C  \\nATOM      6  CG1 VAL A   1      33.860  -3.669  47.953  1.00  0.00           C  \\nATOM      7  CG2 VAL A   1      32.609  -3.931  50.105  1.00  0.00           C  \\nATOM      8  H   VAL A   1      32.609  -6.489  50.688  1.00  0.00           H  \\nATOM      9  H2  VAL A   1      33.121  -7.765  49.809  1.00  0.00           H  \\nATOM     10  H3  VAL A   1      34.206  -6.803  50.558  1.00  0.00           H  \\nATOM     11  HA  VAL A   1      32.459  -6.016  48.320  1.00  0.00           H  \\nATOM     12  HB  VAL A   1      34.661  -4.452  49.749  1.00  0.00           H  \\nATOM     13 HG11 VAL A   1      34.079  -2.634  48.218  1.00  0.00           H  \\nATOM     14 HG12 VAL A   1      34.676  -4.069  47.352  1.00  0.00           H  \\nATOM     15 HG13 VAL A   1      32.933  -3.707  47.380  1.00  0.00           H  \\nATOM     16 HG21 VAL A   1      32.835  -2.894  50.349  1.00  0.00           H  \\nATOM     17 HG22 VAL A   1      31.655  -3.982  49.579  1.00  0.00           H  \\nATOM     18 HG23 VAL A   1      32.546  -4.514  51.025  1.00  0.00           H  \\nATOM     19  N   PRO A   2      34.329  -6.775  46.695  1.00  0.00           N  \\nATOM     20  CA  PRO A   2      33.142  -6.395  45.911  1.00  0.00           C  \\nATOM     21  C   PRO A   2      32.058  -7.511  45.886  1.00  0.00           C  \\nATOM     22  O   PRO A   2      32.207  -8.488  46.604  1.00  0.00           O  \\nATOM     23  CB  PRO A   2      33.832  -6.172  44.568  1.00  0.00           C  \\nATOM     24  CG  PRO A   2      34.837  -7.330  44.496  1.00  0.00           C  \\nATOM     25  CD  PRO A   2      35.385  -7.389  45.922  1.00  0.00           C  \\nATOM     26  HA  PRO A   2      32.674  -5.473  46.256  1.00  0.00           H  \\nATOM     27  HB3 PRO A   2      33.124  -6.124  43.741  1.00  0.00           H  \\nATOM     28  HB2 PRO A   2      34.389  -5.234  44.583  1.00  0.00           H  \\nATOM     29  HG3 PRO A   2      35.625  -7.120  43.772  1.00  0.00           H  \\nATOM     30  HG2 PRO A   2      34.334  -8.266  44.255  1.00  0.00           H  \\nATOM     31  HD3 PRO A   2      35.554  -8.426  46.214  1.00  0.00           H  \\nATOM     32  HD2 PRO A   2      36.334  -6.853  45.964  1.00  0.00           H  \\nATOM     33  N   VAL A   3      30.929  -7.482  45.173  1.00  0.00           N  \\nATOM     34  CA  VAL A   3      30.301  -6.579  44.194  1.00  0.00           C  \\nATOM     35  C   VAL A   3      28.793  -6.693  44.482  1.00  0.00           C  \\nATOM     36  O   VAL A   3      28.309  -7.780  44.797  1.00  0.00           O  \\nATOM     37  CB  VAL A   3      30.620  -6.985  42.742  1.00  0.00           C  \\nATOM     38  CG1 VAL A   3      30.028  -8.351  42.429  1.00  0.00           C  \\nATOM     39  CG2 VAL A   3      30.087  -5.933  41.782  1.00  0.00           C  \\nATOM     40  H   VAL A   3      30.321  -8.253  45.414  1.00  0.00           H  \\nATOM     41  HA  VAL A   3      30.645  -5.555  44.333  1.00  0.00           H  \\nATOM     42  HB  VAL A   3      31.700  -7.069  42.627  1.00  0.00           H  \\nATOM     43 HG11 VAL A   3      30.263  -8.622  41.399  1.00  0.00           H  \\nATOM     44 HG12 VAL A   3      30.450  -9.094  43.105  1.00  0.00           H  \\nATOM     45 HG13 VAL A   3      28.946  -8.316  42.556  1.00  0.00           H  \\nATOM     46 HG21 VAL A   3      30.316  -6.225  40.758  1.00  0.00           H  \\nATOM     47 HG22 VAL A   3      29.007  -5.845  41.901  1.00  0.00           H  \\nATOM     48 HG23 VAL A   3      30.555  -4.972  41.998  1.00  0.00           H  \\nATOM     49  N   ASN A   4      28.060  -5.590  44.381  1.00  0.00           N  \\nATOM     50  CA  ASN A   4      26.599  -5.580  44.406  1.00  0.00           C  \\nATOM     51  C   ASN A   4      26.074  -5.843  42.976  1.00  0.00           C  \\nATOM     52  O   ASN A   4      26.362  -5.014  42.111  1.00  0.00           O  \\nATOM     53  CB  ASN A   4      26.071  -4.269  44.960  1.00  0.00           C  \\nATOM     54  CG  ASN A   4      26.430  -4.067  46.406  1.00  0.00           C  \\nATOM     55  OD1 ASN A   4      26.224  -4.956  47.240  1.00  0.00           O  \\nATOM     56  ND2 ASN A   4      26.964  -2.914  46.719  1.00  0.00           N  \\nATOM     57  H   ASN A   4      28.508  -4.754  44.033  1.00  0.00           H  \\nATOM     58  HA  ASN A   4      26.257  -6.363  45.085  1.00  0.00           H  \\nATOM     59  HB3 ASN A   4      26.475  -3.440  44.378  1.00  0.00           H  \\nATOM     60  HB2 ASN A   4      24.986  -4.242  44.860  1.00  0.00           H  \\nATOM     61 HD21 ASN A   4      27.223  -2.723  47.666  1.00  0.00           H  \\nATOM     62 HD22 ASN A   4      27.113  -2.223  46.012  1.00  0.00           H  \\nATOM     63  N   PRO A   5      25.338  -6.933  42.695  1.00  0.00           N  \\nATOM     64  CA  PRO A   5      24.864  -7.193  41.343  1.00  0.00           C  \\nATOM     65  C   PRO A   5      24.042  -5.992  40.859  1.00  0.00           C  \\nATOM     66  O   PRO A   5      23.241  -5.428  41.607  1.00  0.00           O  \\nATOM     67  CB  PRO A   5      24.008  -8.453  41.503  1.00  0.00           C  \\nATOM     68  CG  PRO A   5      24.599  -9.150  42.680  1.00  0.00           C  \\nATOM     69  CD  PRO A   5      24.984  -8.038  43.619  1.00  0.00           C  \\nATOM     70  HA  PRO A   5      25.712  -7.332  40.672  1.00  0.00           H  \\nATOM     71  HB3 PRO A   5      22.954  -8.175  41.656  1.00  0.00           H  \\nATOM     72  HB2 PRO A   5      24.050  -9.056  40.584  1.00  0.00           H  \\nATOM     73  HG3 PRO A   5      23.865  -9.841  43.120  1.00  0.00           H  \\nATOM     74  HG2 PRO A   5      25.460  -9.758  42.368  1.00  0.00           H  \\nATOM     75  HD3 PRO A   5      24.124  -7.778  44.253  1.00  0.00           H  \\nATOM     76  HD2 PRO A   5      25.838  -8.357  44.235  1.00  0.00           H  \\nATOM     77  N   GLU A   6      24.299  -5.577  39.619  1.00  0.00           N  \\nATOM     78  CA  GLU A   6      23.572  -4.490  38.968  1.00  0.00           C  \\nATOM     79  C   GLU A   6      22.172  -4.979  38.571  1.00  0.00           C  \\nATOM     80  O   GLU A   6      22.040  -6.057  37.985  1.00  0.00           O  \\nATOM     81  CB  GLU A   6      24.332  -3.989  37.738  1.00  0.00           C  \\nATOM     82  CG  GLU A   6      23.709  -2.774  37.066  1.00  0.00           C  \\nATOM     83  CD  GLU A   6      24.515  -2.274  35.899  1.00  0.00           C  \\nATOM     84  OE1 GLU A   6      25.491  -2.901  35.563  1.00  0.00           O  \\nATOM     85  OE2 GLU A   6      24.154  -1.263  35.343  1.00  0.00           O  \\nATOM     86  H   GLU A   6      24.902  -6.135  39.032  1.00  0.00           H  \\nATOM     87  HA  GLU A   6      23.484  -3.670  39.680  1.00  0.00           H  \\nATOM     88  HB3 GLU A   6      25.352  -3.728  38.022  1.00  0.00           H  \\nATOM     89  HB2 GLU A   6      24.392  -4.787  36.997  1.00  0.00           H  \\nATOM     90  HG3 GLU A   6      22.711  -3.036  36.717  1.00  0.00           H  \\nATOM     91  HG2 GLU A   6      23.610  -1.976  37.801  1.00  0.00           H  \\nATOM     92  N   PRO A   7      21.113  -4.225  38.910  1.00  0.00           N  \\nATOM     93  CA  PRO A   7      19.762  -4.617  38.567  1.00  0.00           C  \\nATOM     94  C   PRO A   7      19.674  -4.852  37.061  1.00  0.00           C  \\nATOM     95  O   PRO A   7      20.196  -4.060  36.273  1.00  0.00           O  \\nATOM     96  CB  PRO A   7      18.921  -3.416  39.014  1.00  0.00           C  \\nATOM     97  CG  PRO A   7      19.708  -2.813  40.126  1.00  0.00           C  \\nATOM     98  CD  PRO A   7      21.143  -2.961  39.694  1.00  0.00           C  \\nATOM     99  HA  PRO A   7      19.494  -5.533  39.092  1.00  0.00           H  \\nATOM    100  HB3 PRO A   7      18.775  -2.725  38.171  1.00  0.00           H  \\nATOM    101  HB2 PRO A   7      17.923  -3.753  39.330  1.00  0.00           H  \\nATOM    102  HG3 PRO A   7      19.417  -1.762  40.272  1.00  0.00           H  \\nATOM    103  HG2 PRO A   7      19.494  -3.336  41.070  1.00  0.00           H  \\nATOM    104  HD3 PRO A   7      21.430  -2.101  39.072  1.00  0.00           H  \\nATOM    105  HD2 PRO A   7      21.789  -3.031  40.582  1.00  0.00           H  \\nATOM    106  N   ASP A   8      18.976  -5.925  36.687  1.00  0.00           N  \\nATOM    107  CA  ASP A   8      18.709  -6.285  35.302  1.00  0.00           C  \\nATOM    108  C   ASP A   8      17.950  -5.140  34.630  1.00  0.00           C  \\nATOM    109  O   ASP A   8      16.859  -4.737  35.045  1.00  0.00           O  \\nATOM    110  CB  ASP A   8      17.901  -7.582  35.216  1.00  0.00           C  \\nATOM    111  CG  ASP A   8      17.617  -8.010  33.783  1.00  0.00           C  \\nATOM    112  OD1 ASP A   8      18.518  -8.493  33.138  1.00  0.00           O  \\nATOM    113  OD2 ASP A   8      16.503  -7.849  33.346  1.00  0.00           O  \\nATOM    114  H   ASP A   8      18.589  -6.524  37.401  1.00  0.00           H  \\nATOM    115  HA  ASP A   8      19.656  -6.428  34.780  1.00  0.00           H  \\nATOM    116  HB3 ASP A   8      18.445  -8.383  35.718  1.00  0.00           H  \\nATOM    117  HB2 ASP A   8      16.952  -7.455  35.737  1.00  0.00           H  \\nATOM    118  N   ALA A   9      18.553  -4.603  33.572  1.00  0.00           N  \\nATOM    119  CA  ALA A   9      17.984  -3.536  32.786  1.00  0.00           C  \\nATOM    120  C   ALA A   9      16.924  -4.166  31.877  1.00  0.00           C  \\nATOM    121  O   ALA A   9      17.080  -4.226  30.658  1.00  0.00           O  \\nATOM    122  CB  ALA A   9      19.110  -2.822  32.026  1.00  0.00           C  \\nATOM    123  H   ALA A   9      19.403  -5.042  33.248  1.00  0.00           H  \\nATOM    124  HA  ALA A   9      17.496  -2.821  33.449  1.00  0.00           H  \\nATOM    125  HB1 ALA A   9      18.690  -2.022  31.416  1.00  0.00           H  \\nATOM    126  HB2 ALA A   9      19.631  -3.527  31.379  1.00  0.00           H  \\nATOM    127  HB3 ALA A   9      19.820  -2.397  32.736  1.00  0.00           H  \\nATOM    128  N   THR A  10      15.835  -4.657  32.470  1.00  0.00           N  \\nATOM    129  CA  THR A  10      14.765  -5.293  31.713  1.00  0.00           C  \\nATOM    130  C   THR A  10      14.378  -4.315  30.623  1.00  0.00           C  \\nATOM    131  O   THR A  10      13.975  -3.183  30.904  1.00  0.00           O  \\nATOM    132  CB  THR A  10      13.548  -5.644  32.589  1.00  0.00           C  \\nATOM    133  OG1 THR A  10      13.952  -6.534  33.638  1.00  0.00           O  \\nATOM    134  CG2 THR A  10      12.464  -6.308  31.755  1.00  0.00           C  \\nATOM    135  H   THR A  10      15.706  -4.487  33.458  1.00  0.00           H  \\nATOM    136  HA  THR A  10      15.139  -6.210  31.259  1.00  0.00           H  \\nATOM    137  HB  THR A  10      13.148  -4.735  33.038  1.00  0.00           H  \\nATOM    138  HG1 THR A  10      14.822  -6.888  33.439  1.00  0.00           H  \\nATOM    139 HG21 THR A  10      11.612  -6.548  32.390  1.00  0.00           H  \\nATOM    140 HG22 THR A  10      12.146  -5.628  30.964  1.00  0.00           H  \\nATOM    141 HG23 THR A  10      12.855  -7.223  31.312  1.00  0.00           H  \\nATOM    142  N   SER A  11      14.554  -4.743  29.376  1.00  0.00           N  \\nATOM    143  CA  SER A  11      14.343  -3.933  28.194  1.00  0.00           C  \\nATOM    144  C   SER A  11      12.839  -3.829  27.990  1.00  0.00           C  \\nATOM    145  O   SER A  11      12.246  -4.581  27.218  1.00  0.00           O  \\nATOM    146  CB  SER A  11      15.014  -4.546  26.981  1.00  0.00           C  \\nATOM    147  OG  SER A  11      16.404  -4.602  27.151  1.00  0.00           O  \\nATOM    148  H   SER A  11      14.753  -5.721  29.224  1.00  0.00           H  \\nATOM    149  HA  SER A  11      14.762  -2.938  28.345  1.00  0.00           H  \\nATOM    150  HB3 SER A  11      14.624  -5.551  26.819  1.00  0.00           H  \\nATOM    151  HB2 SER A  11      14.777  -3.956  26.097  1.00  0.00           H  \\nATOM    152  HG  SER A  11      16.573  -4.310  28.050  1.00  0.00           H  \\nATOM    153  N   VAL A  12      12.211  -2.921  28.730  1.00  0.00           N  \\nATOM    154  CA  VAL A  12      10.781  -2.689  28.622  1.00  0.00           C  \\nATOM    155  C   VAL A  12      10.621  -2.056  27.250  1.00  0.00           C  \\nATOM    156  O   VAL A  12      10.745  -0.842  27.090  1.00  0.00           O  \\nATOM    157  CB  VAL A  12      10.255  -1.757  29.730  1.00  0.00           C  \\nATOM    158  CG1 VAL A  12       8.769  -1.493  29.543  1.00  0.00           C  \\nATOM    159  CG2 VAL A  12      10.527  -2.371  31.095  1.00  0.00           C  \\nATOM    160  H   VAL A  12      12.764  -2.312  29.315  1.00  0.00           H  \\nATOM    161  HA  VAL A  12      10.261  -3.647  28.639  1.00  0.00           H  \\nATOM    162  HB  VAL A  12      10.764  -0.796  29.656  1.00  0.00           H  \\nATOM    163 HG11 VAL A  12       8.414  -0.833  30.335  1.00  0.00           H  \\nATOM    164 HG12 VAL A  12       8.602  -1.021  28.576  1.00  0.00           H  \\nATOM    165 HG13 VAL A  12       8.224  -2.436  29.587  1.00  0.00           H  \\nATOM    166 HG21 VAL A  12      10.153  -1.707  31.874  1.00  0.00           H  \\nATOM    167 HG22 VAL A  12      10.022  -3.335  31.167  1.00  0.00           H  \\nATOM    168 HG23 VAL A  12      11.600  -2.513  31.224  1.00  0.00           H  \\nATOM    169  N   GLU A  13      10.465  -2.893  26.223  1.00  0.00           N  \\nATOM    170  CA  GLU A  13      10.227  -2.409  24.874  1.00  0.00           C  \\nATOM    171  C   GLU A  13       8.996  -1.512  24.980  1.00  0.00           C  \\nATOM    172  O   GLU A  13       7.963  -1.910  25.520  1.00  0.00           O  \\nATOM    173  CB  GLU A  13      10.001  -3.557  23.887  1.00  0.00           C  \\nATOM    174  CG  GLU A  13       9.890  -3.122  22.432  1.00  0.00           C  \\nATOM    175  CD  GLU A  13       9.647  -4.272  21.495  1.00  0.00           C  \\nATOM    176  OE1 GLU A  13       9.612  -5.389  21.952  1.00  0.00           O  \\nATOM    177  OE2 GLU A  13       9.497  -4.033  20.320  1.00  0.00           O  \\nATOM    178  H   GLU A  13      10.327  -3.875  26.415  1.00  0.00           H  \\nATOM    179  HA  GLU A  13      11.077  -1.818  24.531  1.00  0.00           H  \\nATOM    180  HB3 GLU A  13      10.823  -4.269  23.961  1.00  0.00           H  \\nATOM    181  HB2 GLU A  13       9.084  -4.086  24.148  1.00  0.00           H  \\nATOM    182  HG3 GLU A  13       9.068  -2.412  22.338  1.00  0.00           H  \\nATOM    183  HG2 GLU A  13      10.809  -2.613  22.145  1.00  0.00           H  \\nATOM    184  N   ASN A  14       9.139  -0.260  24.542  1.00  0.00           N  \\nATOM    185  CA  ASN A  14       8.110   0.762  24.636  1.00  0.00           C  \\nATOM    186  C   ASN A  14       7.114   0.531  23.494  1.00  0.00           C  \\nATOM    187  O   ASN A  14       6.993   1.333  22.569  1.00  0.00           O  \\nATOM    188  CB  ASN A  14       8.708   2.156  24.585  1.00  0.00           C  \\nATOM    189  CG  ASN A  14       9.522   2.480  25.807  1.00  0.00           C  \\nATOM    190  OD1 ASN A  14       9.120   2.173  26.935  1.00  0.00           O  \\nATOM    191  ND2 ASN A  14      10.658   3.097  25.605  1.00  0.00           N  \\nATOM    192  H   ASN A  14      10.027  -0.013  24.127  1.00  0.00           H  \\nATOM    193  HA  ASN A  14       7.565   0.639  25.570  1.00  0.00           H  \\nATOM    194  HB3 ASN A  14       9.345   2.247  23.704  1.00  0.00           H  \\nATOM    195  HB2 ASN A  14       7.909   2.891  24.488  1.00  0.00           H  \\nATOM    196 HD21 ASN A  14      11.241   3.338  26.381  1.00  0.00           H  \\nATOM    197 HD22 ASN A  14      10.944   3.326  24.675  1.00  0.00           H  \\nATOM    198  N   VAL A  15       6.407  -0.595  23.535  1.00  0.00           N  \\nATOM    199  CA  VAL A  15       5.349  -0.901  22.589  1.00  0.00           C  \\nATOM    200  C   VAL A  15       4.192  -0.022  23.034  1.00  0.00           C  \\nATOM    201  O   VAL A  15       3.593  -0.266  24.081  1.00  0.00           O  \\nATOM    202  CB  VAL A  15       4.956  -2.390  22.613  1.00  0.00           C  \\nATOM    203  CG1 VAL A  15       3.827  -2.660  21.630  1.00  0.00           C  \\nATOM    204  CG2 VAL A  15       6.169  -3.250  22.292  1.00  0.00           C  \\nATOM    205  H   VAL A  15       6.481  -1.161  24.370  1.00  0.00           H  \\nATOM    206  HA  VAL A  15       5.665  -0.616  21.585  1.00  0.00           H  \\nATOM    207  HB  VAL A  15       4.584  -2.642  23.607  1.00  0.00           H  \\nATOM    208 HG11 VAL A  15       3.563  -3.717  21.661  1.00  0.00           H  \\nATOM    209 HG12 VAL A  15       2.958  -2.061  21.901  1.00  0.00           H  \\nATOM    210 HG13 VAL A  15       4.151  -2.397  20.623  1.00  0.00           H  \\nATOM    211 HG21 VAL A  15       5.884  -4.302  22.312  1.00  0.00           H  \\nATOM    212 HG22 VAL A  15       6.544  -2.994  21.301  1.00  0.00           H  \\nATOM    213 HG23 VAL A  15       6.949  -3.071  23.033  1.00  0.00           H  \\nATOM    214  N   ALA A  16       3.940   1.054  22.289  1.00  0.00           N  \\nATOM    215  CA  ALA A  16       2.936   2.044  22.644  1.00  0.00           C  \\nATOM    216  C   ALA A  16       1.622   1.311  22.953  1.00  0.00           C  \\nATOM    217  O   ALA A  16       1.146   0.502  22.151  1.00  0.00           O  \\nATOM    218  CB  ALA A  16       2.794   3.044  21.491  1.00  0.00           C  \\nATOM    219  H   ALA A  16       4.450   1.171  21.424  1.00  0.00           H  \\nATOM    220  HA  ALA A  16       3.268   2.577  23.535  1.00  0.00           H  \\nATOM    221  HB1 ALA A  16       2.480   2.528  20.583  1.00  0.00           H  \\nATOM    222  HB2 ALA A  16       3.747   3.541  21.307  1.00  0.00           H  \\nATOM    223  HB3 ALA A  16       2.046   3.795  21.746  1.00  0.00           H  \\nATOM    224  N   LEU A  17       1.102   1.518  24.166  1.00  0.00           N  \\nATOM    225  CA  LEU A  17      -0.101   0.844  24.635  1.00  0.00           C  \\nATOM    226  C   LEU A  17      -1.257   1.173  23.690  1.00  0.00           C  \\nATOM    227  O   LEU A  17      -1.555   2.339  23.429  1.00  0.00           O  \\nATOM    228  CB  LEU A  17      -0.443   1.278  26.066  1.00  0.00           C  \\nATOM    229  CG  LEU A  17      -1.656   0.585  26.699  1.00  0.00           C  \\nATOM    230  CD1 LEU A  17      -1.359  -0.898  26.878  1.00  0.00           C  \\nATOM    231  CD2 LEU A  17      -1.978   1.243  28.033  1.00  0.00           C  \\nATOM    232  H   LEU A  17       1.600   2.120  24.806  1.00  0.00           H  \\nATOM    233  HA  LEU A  17       0.074  -0.232  24.603  1.00  0.00           H  \\nATOM    234  HB3 LEU A  17       0.419   1.083  26.703  1.00  0.00           H  \\nATOM    235  HB2 LEU A  17      -0.635   2.350  26.068  1.00  0.00           H  \\nATOM    236  HG  LEU A  17      -2.515   0.675  26.034  1.00  0.00           H  \\nATOM    237 HD11 LEU A  17      -2.221  -1.390  27.328  1.00  0.00           H  \\nATOM    238 HD12 LEU A  17      -1.151  -1.347  25.907  1.00  0.00           H  \\nATOM    239 HD13 LEU A  17      -0.493  -1.019  27.527  1.00  0.00           H  \\nATOM    240 HD21 LEU A  17      -2.841   0.751  28.483  1.00  0.00           H  \\nATOM    241 HD22 LEU A  17      -1.120   1.152  28.700  1.00  0.00           H  \\nATOM    242 HD23 LEU A  17      -2.204   2.297  27.873  1.00  0.00           H  \\nATOM    243  N   LYS A  18      -1.917   0.124  23.198  1.00  0.00           N  \\nATOM    244  CA  LYS A  18      -3.060   0.257  22.299  1.00  0.00           C  \\nATOM    245  C   LYS A  18      -4.194   0.983  23.001  1.00  0.00           C  \\nATOM    246  O   LYS A  18      -4.485   0.699  24.162  1.00  0.00           O  \\nATOM    247  CB  LYS A  18      -3.531  -1.112  21.806  1.00  0.00           C  \\nATOM    248  CG  LYS A  18      -4.146  -1.992  22.887  1.00  0.00           C  \\nATOM    249  CD  LYS A  18      -4.544  -3.351  22.332  1.00  0.00           C  \\nATOM    250  CE  LYS A  18      -5.170  -4.227  23.408  1.00  0.00           C  \\nATOM    251  NZ  LYS A  18      -5.504  -5.584  22.896  1.00  0.00           N  \\nATOM    252  H   LYS A  18      -1.643  -0.794  23.519  1.00  0.00           H  \\nATOM    253  HA  LYS A  18      -2.758   0.834  21.425  1.00  0.00           H  \\nATOM    254  HB3 LYS A  18      -4.274  -0.980  21.019  1.00  0.00           H  \\nATOM    255  HB2 LYS A  18      -2.688  -1.653  21.374  1.00  0.00           H  \\nATOM    256  HG3 LYS A  18      -3.426  -2.135  23.693  1.00  0.00           H  \\nATOM    257  HG2 LYS A  18      -5.029  -1.502  23.295  1.00  0.00           H  \\nATOM    258  HD3 LYS A  18      -5.261  -3.218  21.521  1.00  0.00           H  \\nATOM    259  HD2 LYS A  18      -3.663  -3.855  21.935  1.00  0.00           H  \\nATOM    260  HE3 LYS A  18      -4.478  -4.327  24.243  1.00  0.00           H  \\nATOM    261  HE2 LYS A  18      -6.082  -3.757  23.775  1.00  0.00           H  \\nATOM    262  HZ1 LYS A  18      -5.917  -6.132  23.637  1.00  0.00           H  \\nATOM    263  HZ2 LYS A  18      -6.161  -5.504  22.132  1.00  0.00           H  \\nATOM    264  HZ3 LYS A  18      -4.663  -6.038  22.570  1.00  0.00           H  \\nATOM    265  N   THR A  19      -4.886   1.837  22.262  1.00  0.00           N  \\nATOM    266  CA  THR A  19      -6.093   2.508  22.752  1.00  0.00           C  \\nATOM    267  C   THR A  19      -7.273   2.188  21.842  1.00  0.00           C  \\nATOM    268  O   THR A  19      -7.079   1.884  20.668  1.00  0.00           O  \\nATOM    269  CB  THR A  19      -5.898   4.033  22.837  1.00  0.00           C  \\nATOM    270  OG1 THR A  19      -5.649   4.559  21.526  1.00  0.00           O  \\nATOM    271  CG2 THR A  19      -4.727   4.372  23.746  1.00  0.00           C  \\nATOM    272  H   THR A  19      -4.669   1.894  21.277  1.00  0.00           H  \\nATOM    273  HA  THR A  19      -6.351   2.111  23.734  1.00  0.00           H  \\nATOM    274  HB  THR A  19      -6.803   4.495  23.232  1.00  0.00           H  \\nATOM    275  HG1 THR A  19      -4.715   4.471  21.319  1.00  0.00           H  \\nATOM    276 HG21 THR A  19      -4.604   5.453  23.794  1.00  0.00           H  \\nATOM    277 HG22 THR A  19      -4.919   3.983  24.746  1.00  0.00           H  \\nATOM    278 HG23 THR A  19      -3.818   3.921  23.350  1.00  0.00           H  \\nATOM    279  N   GLY A  20      -8.482   2.233  22.397  1.00  0.00           N  \\nATOM    280  CA  GLY A  20      -9.726   1.868  21.717  1.00  0.00           C  \\nATOM    281  C   GLY A  20     -10.095   0.386  21.827  1.00  0.00           C  \\nATOM    282  O   GLY A  20      -9.234  -0.488  21.948  1.00  0.00           O  \\nATOM    283  H   GLY A  20      -8.558   2.486  23.373  1.00  0.00           H  \\nATOM    284  HA3 GLY A  20     -10.527   2.441  22.182  1.00  0.00           H  \\nATOM    285  HA2 GLY A  20      -9.686   2.145  20.664  1.00  0.00           H  \\nATOM    286  N   SER A  21     -11.399   0.104  21.832  1.00  0.00           N  \\nATOM    287  CA  SER A  21     -11.942  -1.254  21.984  1.00  0.00           C  \\nATOM    288  C   SER A  21     -11.802  -2.109  20.723  1.00  0.00           C  \\nATOM    289  O   SER A  21     -11.679  -3.329  20.811  1.00  0.00           O  \\nATOM    290  CB  SER A  21     -13.406  -1.177  22.373  1.00  0.00           C  \\nATOM    291  OG  SER A  21     -14.173  -0.630  21.336  1.00  0.00           O  \\nATOM    292  H   SER A  21     -12.055   0.870  21.766  1.00  0.00           H  \\nATOM    293  HA  SER A  21     -11.400  -1.762  22.782  1.00  0.00           H  \\nATOM    294  HB3 SER A  21     -13.772  -2.175  22.613  1.00  0.00           H  \\nATOM    295  HB2 SER A  21     -13.512  -0.566  23.269  1.00  0.00           H  \\nATOM    296  HG  SER A  21     -14.039   0.320  21.381  1.00  0.00           H  \\nATOM    297  N   GLY A  22     -11.832  -1.484  19.544  1.00  0.00           N  \\nATOM    298  CA  GLY A  22     -11.911  -2.176  18.260  1.00  0.00           C  \\nATOM    299  C   GLY A  22     -13.315  -2.641  17.882  1.00  0.00           C  \\nATOM    300  O   GLY A  22     -13.474  -3.295  16.851  1.00  0.00           O  \\nATOM    301  H   GLY A  22     -12.009  -0.490  19.545  1.00  0.00           H  \\nATOM    302  HA3 GLY A  22     -11.275  -3.061  18.292  1.00  0.00           H  \\nATOM    303  HA2 GLY A  22     -11.556  -1.507  17.475  1.00  0.00           H  \\nATOM    304  N   ASP A  23     -14.333  -2.314  18.680  1.00  0.00           N  \\nATOM    305  CA  ASP A  23     -15.706  -2.774  18.459  1.00  0.00           C  \\nATOM    306  C   ASP A  23     -16.466  -1.925  17.438  1.00  0.00           C  \\nATOM    307  O   ASP A  23     -17.547  -2.306  16.988  1.00  0.00           O  \\nATOM    308  CB  ASP A  23     -16.458  -2.780  19.796  1.00  0.00           C  \\nATOM    309  CG  ASP A  23     -15.974  -3.859  20.764  1.00  0.00           C  \\nATOM    310  OD1 ASP A  23     -15.415  -4.882  20.307  1.00  0.00           O  \\nATOM    311  OD2 ASP A  23     -16.467  -3.873  21.909  1.00  0.00           O  \\nATOM    312  H   ASP A  23     -14.137  -1.795  19.523  1.00  0.00           H  \\nATOM    313  HA  ASP A  23     -15.682  -3.786  18.058  1.00  0.00           H  \\nATOM    314  HB3 ASP A  23     -17.509  -2.986  19.594  1.00  0.00           H  \\nATOM    315  HB2 ASP A  23     -16.409  -1.796  20.263  1.00  0.00           H  \\nATOM    316  N   SER A  24     -15.900  -0.782  17.052  1.00  0.00           N  \\nATOM    317  CA  SER A  24     -16.547   0.194  16.182  1.00  0.00           C  \\nATOM    318  C   SER A  24     -15.522   0.974  15.368  1.00  0.00           C  \\nATOM    319  O   SER A  24     -14.424   1.269  15.834  1.00  0.00           O  \\nATOM    320  CB  SER A  24     -17.385   1.153  17.005  1.00  0.00           C  \\nATOM    321  OG  SER A  24     -17.879   2.197  16.212  1.00  0.00           O  \\nATOM    322  H   SER A  24     -14.949  -0.602  17.340  1.00  0.00           H  \\nATOM    323  HA  SER A  24     -17.196  -0.330  15.480  1.00  0.00           H  \\nATOM    324  HB3 SER A  24     -18.216   0.612  17.458  1.00  0.00           H  \\nATOM    325  HB2 SER A  24     -16.781   1.561  17.814  1.00  0.00           H  \\nATOM    326  HG  SER A  24     -18.800   1.986  16.040  1.00  0.00           H  \\nATOM    327  N   GLN A  25     -15.909   1.370  14.158  1.00  0.00           N  \\nATOM    328  CA  GLN A  25     -15.119   2.291  13.345  1.00  0.00           C  \\nATOM    329  C   GLN A  25     -14.810   3.609  14.078  1.00  0.00           C  \\nATOM    330  O   GLN A  25     -13.792   4.236  13.801  1.00  0.00           O  \\nATOM    331  CB  GLN A  25     -15.852   2.587  12.033  1.00  0.00           C  \\nATOM    332  CG  GLN A  25     -15.043   3.402  11.039  1.00  0.00           C  \\nATOM    333  CD  GLN A  25     -15.736   3.530   9.696  1.00  0.00           C  \\nATOM    334  OE1 GLN A  25     -16.756   2.883   9.444  1.00  0.00           O  \\nATOM    335  NE2 GLN A  25     -15.186   4.368   8.825  1.00  0.00           N  \\nATOM    336  H   GLN A  25     -16.780   1.023  13.782  1.00  0.00           H  \\nATOM    337  HA  GLN A  25     -14.169   1.815  13.104  1.00  0.00           H  \\nATOM    338  HB3 GLN A  25     -16.129   1.649  11.552  1.00  0.00           H  \\nATOM    339  HB2 GLN A  25     -16.772   3.131  12.245  1.00  0.00           H  \\nATOM    340  HG3 GLN A  25     -14.892   4.403  11.443  1.00  0.00           H  \\nATOM    341  HG2 GLN A  25     -14.081   2.914  10.882  1.00  0.00           H  \\nATOM    342 HE21 GLN A  25     -15.599   4.494   7.922  1.00  0.00           H  \\nATOM    343 HE22 GLN A  25     -14.358   4.873   9.070  1.00  0.00           H  \\nATOM    344  N   SER A  26     -15.666   4.031  15.016  1.00  0.00           N  \\nATOM    345  CA  SER A  26     -15.465   5.237  15.835  1.00  0.00           C  \\nATOM    346  C   SER A  26     -14.514   5.027  17.017  1.00  0.00           C  \\nATOM    347  O   SER A  26     -14.084   6.006  17.621  1.00  0.00           O  \\nATOM    348  CB  SER A  26     -16.803   5.724  16.355  1.00  0.00           C  \\nATOM    349  OG  SER A  26     -17.345   4.818  17.276  1.00  0.00           O  \\nATOM    350  H   SER A  26     -16.481   3.473  15.227  1.00  0.00           H  \\nATOM    351  HA  SER A  26     -15.024   6.016  15.215  1.00  0.00           H  \\nATOM    352  HB3 SER A  26     -16.676   6.697  16.829  1.00  0.00           H  \\nATOM    353  HB2 SER A  26     -17.491   5.855  15.521  1.00  0.00           H  \\nATOM    354  HG  SER A  26     -17.442   3.987  16.804  1.00  0.00           H  \\nATOM    355  N   ASP A  27     -14.174   3.776  17.334  1.00  0.00           N  \\nATOM    356  CA  ASP A  27     -13.286   3.409  18.439  1.00  0.00           C  \\nATOM    357  C   ASP A  27     -12.361   2.237  18.047  1.00  0.00           C  \\nATOM    358  O   ASP A  27     -12.427   1.159  18.647  1.00  0.00           O  \\nATOM    359  CB  ASP A  27     -14.105   3.036  19.677  1.00  0.00           C  \\nATOM    360  CG  ASP A  27     -13.251   2.883  20.928  1.00  0.00           C  \\nATOM    361  OD1 ASP A  27     -12.333   3.652  21.094  1.00  0.00           O  \\nATOM    362  OD2 ASP A  27     -13.525   2.001  21.706  1.00  0.00           O  \\nATOM    363  H   ASP A  27     -14.502   3.022  16.748  1.00  0.00           H  \\nATOM    364  HA  ASP A  27     -12.636   4.254  18.666  1.00  0.00           H  \\nATOM    365  HB3 ASP A  27     -14.858   3.803  19.860  1.00  0.00           H  \\nATOM    366  HB2 ASP A  27     -14.630   2.097  19.496  1.00  0.00           H  \\nATOM    367  N   PRO A  28     -11.535   2.393  16.993  1.00  0.00           N  \\nATOM    368  CA  PRO A  28     -10.661   1.330  16.516  1.00  0.00           C  \\nATOM    369  C   PRO A  28      -9.482   1.117  17.469  1.00  0.00           C  \\nATOM    370  O   PRO A  28      -9.092   2.034  18.190  1.00  0.00           O  \\nATOM    371  CB  PRO A  28     -10.201   1.854  15.151  1.00  0.00           C  \\nATOM    372  CG  PRO A  28     -10.216   3.336  15.304  1.00  0.00           C  \\nATOM    373  CD  PRO A  28     -11.425   3.613  16.158  1.00  0.00           C  \\nATOM    374  HA  PRO A  28     -11.221   0.401  16.406  1.00  0.00           H  \\nATOM    375  HB3 PRO A  28      -9.202   1.459  14.914  1.00  0.00           H  \\nATOM    376  HB2 PRO A  28     -10.882   1.499  14.363  1.00  0.00           H  \\nATOM    377  HG3 PRO A  28      -9.280   3.679  15.771  1.00  0.00           H  \\nATOM    378  HG2 PRO A  28     -10.274   3.819  14.318  1.00  0.00           H  \\nATOM    379  HD3 PRO A  28     -11.244   4.508  16.770  1.00  0.00           H  \\nATOM    380  HD2 PRO A  28     -12.305   3.754  15.513  1.00  0.00           H  \\nATOM    381  N   ILE A  29      -8.833  -0.048  17.391  1.00  0.00           N  \\nATOM    382  CA  ILE A  29      -7.560  -0.269  18.088  1.00  0.00           C  \\nATOM    383  C   ILE A  29      -6.487   0.587  17.417  1.00  0.00           C  \\nATOM    384  O   ILE A  29      -6.111   0.336  16.272  1.00  0.00           O  \\nATOM    385  CB  ILE A  29      -7.146  -1.752  18.064  1.00  0.00           C  \\nATOM    386  CG1 ILE A  29      -8.176  -2.606  18.807  1.00  0.00           C  \\nATOM    387  CG2 ILE A  29      -5.764  -1.929  18.674  1.00  0.00           C  \\nATOM    388  CD1 ILE A  29      -7.875  -4.087  18.782  1.00  0.00           C  \\nATOM    389  H   ILE A  29      -9.146  -0.739  16.723  1.00  0.00           H  \\nATOM    390  HA  ILE A  29      -7.674   0.056  19.122  1.00  0.00           H  \\nATOM    391  HB  ILE A  29      -7.126  -2.108  17.034  1.00  0.00           H  \\nATOM    392 HG13 ILE A  29      -8.231  -2.287  19.848  1.00  0.00           H  \\nATOM    393 HG12 ILE A  29      -9.162  -2.453  18.367  1.00  0.00           H  \\nATOM    394 HG21 ILE A  29      -5.488  -2.982  18.650  1.00  0.00           H  \\nATOM    395 HG22 ILE A  29      -5.038  -1.350  18.104  1.00  0.00           H  \\nATOM    396 HG23 ILE A  29      -5.775  -1.580  19.707  1.00  0.00           H  \\nATOM    397 HD11 ILE A  29      -8.649  -4.626  19.329  1.00  0.00           H  \\nATOM    398 HD12 ILE A  29      -7.850  -4.436  17.749  1.00  0.00           H  \\nATOM    399 HD13 ILE A  29      -6.908  -4.269  19.250  1.00  0.00           H  \\nATOM    400  N   LYS A  30      -5.967   1.572  18.138  1.00  0.00           N  \\nATOM    401  CA  LYS A  30      -4.924   2.490  17.683  1.00  0.00           C  \\nATOM    402  C   LYS A  30      -3.577   2.010  18.187  1.00  0.00           C  \\nATOM    403  O   LYS A  30      -3.397   1.869  19.397  1.00  0.00           O  \\nATOM    404  CB  LYS A  30      -5.195   3.917  18.162  1.00  0.00           C  \\nATOM    405  CG  LYS A  30      -6.427   4.565  17.544  1.00  0.00           C  \\nATOM    406  CD  LYS A  30      -6.635   5.975  18.076  1.00  0.00           C  \\nATOM    407  CE  LYS A  30      -7.887   6.610  17.489  1.00  0.00           C  \\nATOM    408  NZ  LYS A  30      -8.126   7.974  18.032  1.00  0.00           N  \\nATOM    409  H   LYS A  30      -6.343   1.700  19.067  1.00  0.00           H  \\nATOM    410  HA  LYS A  30      -4.896   2.521  16.593  1.00  0.00           H  \\nATOM    411  HB3 LYS A  30      -5.324   3.919  19.245  1.00  0.00           H  \\nATOM    412  HB2 LYS A  30      -4.335   4.547  17.934  1.00  0.00           H  \\nATOM    413  HG3 LYS A  30      -6.310   4.608  16.460  1.00  0.00           H  \\nATOM    414  HG2 LYS A  30      -7.307   3.966  17.773  1.00  0.00           H  \\nATOM    415  HD3 LYS A  30      -6.728   5.944  19.162  1.00  0.00           H  \\nATOM    416  HD2 LYS A  30      -5.773   6.592  17.820  1.00  0.00           H  \\nATOM    417  HE3 LYS A  30      -7.789   6.675  16.406  1.00  0.00           H  \\nATOM    418  HE2 LYS A  30      -8.752   5.986  17.716  1.00  0.00           H  \\nATOM    419  HZ1 LYS A  30      -8.964   8.360  17.619  1.00  0.00           H  \\nATOM    420  HZ2 LYS A  30      -8.238   7.924  19.035  1.00  0.00           H  \\nATOM    421  HZ3 LYS A  30      -7.340   8.568  17.811  1.00  0.00           H  \\nATOM    422  N   ALA A  31      -2.657   1.752  17.267  1.00  0.00           N  \\nATOM    423  CA  ALA A  31      -1.329   1.257  17.590  1.00  0.00           C  \\nATOM    424  C   ALA A  31      -0.307   1.668  16.529  1.00  0.00           C  \\nATOM    425  O   ALA A  31      -0.647   1.893  15.365  1.00  0.00           O  \\nATOM    426  CB  ALA A  31      -1.391  -0.267  17.735  1.00  0.00           C  \\nATOM    427  H   ALA A  31      -2.868   1.932  16.296  1.00  0.00           H  \\nATOM    428  HA  ALA A  31      -1.007   1.684  18.540  1.00  0.00           H  \\nATOM    429  HB1 ALA A  31      -2.047  -0.526  18.567  1.00  0.00           H  \\nATOM    430  HB2 ALA A  31      -1.785  -0.714  16.823  1.00  0.00           H  \\nATOM    431  HB3 ALA A  31      -0.396  -0.663  17.935  1.00  0.00           H  \\nATOM    432  N   ASP A  32       0.957   1.705  16.932  1.00  0.00           N  \\nATOM    433  CA  ASP A  32       2.069   1.938  16.021  1.00  0.00           C  \\nATOM    434  C   ASP A  32       2.450   0.643  15.287  1.00  0.00           C  \\nATOM    435  O   ASP A  32       2.255  -0.469  15.786  1.00  0.00           O  \\nATOM    436  CB  ASP A  32       3.279   2.485  16.784  1.00  0.00           C  \\nATOM    437  CG  ASP A  32       2.997   3.818  17.465  1.00  0.00           C  \\nATOM    438  OD1 ASP A  32       1.995   4.419  17.158  1.00  0.00           O  \\nATOM    439  OD2 ASP A  32       3.787   4.221  18.284  1.00  0.00           O  \\nATOM    440  H   ASP A  32       1.173   1.490  17.895  1.00  0.00           H  \\nATOM    441  HA  ASP A  32       1.757   2.656  15.262  1.00  0.00           H  \\nATOM    442  HB3 ASP A  32       3.587   1.764  17.541  1.00  0.00           H  \\nATOM    443  HB2 ASP A  32       4.115   2.614  16.095  1.00  0.00           H  \\nATOM    444  N   LEU A  33       3.003   0.794  14.085  1.00  0.00           N  \\nATOM    445  CA  LEU A  33       3.597  -0.288  13.308  1.00  0.00           C  \\nATOM    446  C   LEU A  33       5.084  -0.004  13.091  1.00  0.00           C  \\nATOM    447  O   LEU A  33       5.466   1.054  12.592  1.00  0.00           O  \\nATOM    448  CB  LEU A  33       2.790  -0.471  12.011  1.00  0.00           C  \\nATOM    449  CG  LEU A  33       3.297  -1.623  11.122  1.00  0.00           C  \\nATOM    450  CD1 LEU A  33       2.152  -2.215  10.297  1.00  0.00           C  \\nATOM    451  CD2 LEU A  33       4.361  -1.153  10.129  1.00  0.00           C  \\nATOM    452  H   LEU A  33       3.058   1.727  13.703  1.00  0.00           H  \\nATOM    453  HA  LEU A  33       3.520  -1.216  13.875  1.00  0.00           H  \\nATOM    454  HB3 LEU A  33       1.760  -0.680  12.301  1.00  0.00           H  \\nATOM    455  HB2 LEU A  33       2.794   0.459  11.442  1.00  0.00           H  \\nATOM    456  HG  LEU A  33       3.707  -2.416  11.748  1.00  0.00           H  \\nATOM    457 HD11 LEU A  33       1.336  -2.521  10.953  1.00  0.00           H  \\nATOM    458 HD12 LEU A  33       2.509  -3.088   9.748  1.00  0.00           H  \\nATOM    459 HD13 LEU A  33       1.792  -1.475   9.584  1.00  0.00           H  \\nATOM    460 HD21 LEU A  33       3.959  -0.359   9.499  1.00  0.00           H  \\nATOM    461 HD22 LEU A  33       4.661  -1.981   9.485  1.00  0.00           H  \\nATOM    462 HD23 LEU A  33       5.249  -0.788  10.645  1.00  0.00           H  \\nATOM    463  N   GLU A  34       5.926  -0.973  13.437  1.00  0.00           N  \\nATOM    464  CA  GLU A  34       7.369  -0.880  13.238  1.00  0.00           C  \\nATOM    465  C   GLU A  34       7.754  -1.446  11.864  1.00  0.00           C  \\nATOM    466  O   GLU A  34       7.588  -2.636  11.586  1.00  0.00           O  \\nATOM    467  CB  GLU A  34       8.113  -1.628  14.346  1.00  0.00           C  \\nATOM    468  CG  GLU A  34       9.629  -1.502  14.281  1.00  0.00           C  \\nATOM    469  CD  GLU A  34      10.326  -2.261  15.376  1.00  0.00           C  \\nATOM    470  OE1 GLU A  34       9.796  -3.252  15.817  1.00  0.00           O  \\nATOM    471  OE2 GLU A  34      11.391  -1.849  15.772  1.00  0.00           O  \\nATOM    472  H   GLU A  34       5.557  -1.833  13.817  1.00  0.00           H  \\nATOM    473  HA  GLU A  34       7.665   0.169  13.271  1.00  0.00           H  \\nATOM    474  HB3 GLU A  34       7.788  -1.256  15.318  1.00  0.00           H  \\nATOM    475  HB2 GLU A  34       7.863  -2.688  14.301  1.00  0.00           H  \\nATOM    476  HG3 GLU A  34       9.973  -1.878  13.318  1.00  0.00           H  \\nATOM    477  HG2 GLU A  34       9.899  -0.449  14.347  1.00  0.00           H  \\nATOM    478  N   VAL A  35       8.279  -0.585  10.994  1.00  0.00           N  \\nATOM    479  CA  VAL A  35       8.665  -0.926   9.624  1.00  0.00           C  \\nATOM    480  C   VAL A  35      10.051  -1.566   9.626  1.00  0.00           C  \\nATOM    481  O   VAL A  35      11.004  -1.007  10.172  1.00  0.00           O  \\nATOM    482  CB  VAL A  35       8.637   0.329   8.736  1.00  0.00           C  \\nATOM    483  CG1 VAL A  35       9.114   0.041   7.315  1.00  0.00           C  \\nATOM    484  CG2 VAL A  35       7.224   0.920   8.654  1.00  0.00           C  \\nATOM    485  H   VAL A  35       8.475   0.355  11.307  1.00  0.00           H  \\nATOM    486  HA  VAL A  35       7.948  -1.642   9.223  1.00  0.00           H  \\nATOM    487  HB  VAL A  35       9.307   1.082   9.151  1.00  0.00           H  \\nATOM    488 HG11 VAL A  35      10.155  -0.280   7.310  1.00  0.00           H  \\nATOM    489 HG12 VAL A  35       9.054   0.970   6.749  1.00  0.00           H  \\nATOM    490 HG13 VAL A  35       8.492  -0.724   6.848  1.00  0.00           H  \\nATOM    491 HG21 VAL A  35       7.217   1.786   7.993  1.00  0.00           H  \\nATOM    492 HG22 VAL A  35       6.895   1.241   9.643  1.00  0.00           H  \\nATOM    493 HG23 VAL A  35       6.526   0.173   8.274  1.00  0.00           H  \\nATOM    494  N   LYS A  36      10.181  -2.725   8.969  1.00  0.00           N  \\nATOM    495  CA  LYS A  36      11.411  -3.537   8.972  1.00  0.00           C  \\nATOM    496  C   LYS A  36      11.876  -3.872   7.561  1.00  0.00           C  \\nATOM    497  O   LYS A  36      11.099  -4.374   6.748  1.00  0.00           O  \\nATOM    498  CB  LYS A  36      11.215  -4.817   9.812  1.00  0.00           C  \\nATOM    499  CG  LYS A  36      10.903  -4.466  11.271  1.00  0.00           C  \\nATOM    500  CD  LYS A  36      10.742  -5.669  12.216  1.00  0.00           C  \\nATOM    501  CE  LYS A  36      10.413  -4.992  13.549  1.00  0.00           C  \\nATOM    502  NZ  LYS A  36      10.201  -5.852  14.733  1.00  0.00           N  \\nATOM    503  H   LYS A  36       9.370  -3.075   8.478  1.00  0.00           H  \\nATOM    504  HA  LYS A  36      12.206  -2.961   9.443  1.00  0.00           H  \\nATOM    505  HB3 LYS A  36      10.400  -5.411   9.396  1.00  0.00           H  \\nATOM    506  HB2 LYS A  36      12.131  -5.407   9.780  1.00  0.00           H  \\nATOM    507  HG3 LYS A  36      11.707  -3.834  11.647  1.00  0.00           H  \\nATOM    508  HG2 LYS A  36       9.979  -3.888  11.297  1.00  0.00           H  \\nATOM    509  HD3 LYS A  36      11.676  -6.226  12.283  1.00  0.00           H  \\nATOM    510  HD2 LYS A  36       9.922  -6.308  11.890  1.00  0.00           H  \\nATOM    511  HE3 LYS A  36       9.536  -4.358  13.413  1.00  0.00           H  \\nATOM    512  HE2 LYS A  36      11.232  -4.315  13.785  1.00  0.00           H  \\nATOM    513  HZ1 LYS A  36       9.389  -6.446  14.658  1.00  0.00           H  \\nATOM    514  HZ2 LYS A  36      10.040  -5.195  15.483  1.00  0.00           H  \\nATOM    515  HZ3 LYS A  36      11.037  -6.359  14.988  1.00  0.00           H  \\nATOM    516  N   GLY A  37      13.161  -3.654   7.290  1.00  0.00           N  \\nATOM    517  CA  GLY A  37      13.820  -4.087   6.057  1.00  0.00           C  \\nATOM    518  C   GLY A  37      13.093  -3.641   4.784  1.00  0.00           C  \\nATOM    519  O   GLY A  37      12.685  -2.491   4.648  1.00  0.00           O  \\nATOM    520  H   GLY A  37      13.721  -3.190   7.990  1.00  0.00           H  \\nATOM    521  HA3 GLY A  37      13.898  -5.175   6.065  1.00  0.00           H  \\nATOM    522  HA2 GLY A  37      14.831  -3.677   6.025  1.00  0.00           H  \\nATOM    523  N   GLN A  38      12.902  -4.572   3.848  1.00  0.00           N  \\nATOM    524  CA  GLN A  38      12.286  -4.280   2.550  1.00  0.00           C  \\nATOM    525  C   GLN A  38      10.769  -4.035   2.599  1.00  0.00           C  \\nATOM    526  O   GLN A  38      10.204  -3.676   1.563  1.00  0.00           O  \\nATOM    527  CB  GLN A  38      12.666  -5.370   1.538  1.00  0.00           C  \\nATOM    528  CG  GLN A  38      14.165  -5.303   1.205  1.00  0.00           C  \\nATOM    529  CD  GLN A  38      14.620  -6.481   0.357  1.00  0.00           C  \\nATOM    530  OE1 GLN A  38      14.118  -7.583   0.465  1.00  0.00           O  \\nATOM    531  NE2 GLN A  38      15.593  -6.313  -0.511  1.00  0.00           N  \\nATOM    532  H   GLN A  38      13.244  -5.507   4.014  1.00  0.00           H  \\nATOM    533  HA  GLN A  38      12.708  -3.342   2.190  1.00  0.00           H  \\nATOM    534  HB3 GLN A  38      12.112  -5.210   0.613  1.00  0.00           H  \\nATOM    535  HB2 GLN A  38      12.405  -6.353   1.930  1.00  0.00           H  \\nATOM    536  HG3 GLN A  38      14.756  -5.287   2.121  1.00  0.00           H  \\nATOM    537  HG2 GLN A  38      14.365  -4.384   0.653  1.00  0.00           H  \\nATOM    538 HE21 GLN A  38      15.918  -7.145  -0.983  1.00  0.00           H  \\nATOM    539 HE22 GLN A  38      16.118  -5.451  -0.561  1.00  0.00           H  \\nATOM    540  N   SER A  39      10.114  -4.151   3.764  1.00  0.00           N  \\nATOM    541  CA  SER A  39       8.699  -3.769   3.921  1.00  0.00           C  \\nATOM    542  C   SER A  39       8.469  -2.254   3.837  1.00  0.00           C  \\nATOM    543  O   SER A  39       7.337  -1.825   3.636  1.00  0.00           O  \\nATOM    544  CB  SER A  39       8.099  -4.298   5.227  1.00  0.00           C  \\nATOM    545  OG  SER A  39       8.640  -3.655   6.366  1.00  0.00           O  \\nATOM    546  H   SER A  39      10.623  -4.437   4.588  1.00  0.00           H  \\nATOM    547  HA  SER A  39       8.134  -4.219   3.104  1.00  0.00           H  \\nATOM    548  HB3 SER A  39       7.023  -4.128   5.203  1.00  0.00           H  \\nATOM    549  HB2 SER A  39       8.268  -5.372   5.304  1.00  0.00           H  \\nATOM    550  HG  SER A  39       9.581  -3.843   6.390  1.00  0.00           H  \\nATOM    551  N   ALA A  40       9.529  -1.443   3.934  1.00  0.00           N  \\nATOM    552  CA  ALA A  40       9.429   0.012   3.909  1.00  0.00           C  \\nATOM    553  C   ALA A  40       8.778   0.559   2.636  1.00  0.00           C  \\nATOM    554  O   ALA A  40       7.953   1.459   2.728  1.00  0.00           O  \\nATOM    555  CB  ALA A  40      10.829   0.597   4.121  1.00  0.00           C  \\nATOM    556  H   ALA A  40      10.436  -1.845   4.126  1.00  0.00           H  \\nATOM    557  HA  ALA A  40       8.784   0.331   4.727  1.00  0.00           H  \\nATOM    558  HB1 ALA A  40      11.490   0.315   3.302  1.00  0.00           H  \\nATOM    559  HB2 ALA A  40      11.245   0.229   5.059  1.00  0.00           H  \\nATOM    560  HB3 ALA A  40      10.767   1.684   4.173  1.00  0.00           H  \\nATOM    561  N   LEU A  41       9.104  -0.002   1.468  1.00  0.00           N  \\nATOM    562  CA  LEU A  41       8.523   0.425   0.194  1.00  0.00           C  \\nATOM    563  C   LEU A  41       6.992   0.249   0.140  1.00  0.00           C  \\nATOM    564  O   LEU A  41       6.305   1.248  -0.062  1.00  0.00           O  \\nATOM    565  CB  LEU A  41       9.169  -0.362  -0.953  1.00  0.00           C  \\nATOM    566  CG  LEU A  41       8.475  -0.242  -2.316  1.00  0.00           C  \\nATOM    567  CD1 LEU A  41       8.431   1.221  -2.737  1.00  0.00           C  \\nATOM    568  CD2 LEU A  41       9.219  -1.084  -3.341  1.00  0.00           C  \\nATOM    569  H   LEU A  41       9.772  -0.758   1.473  1.00  0.00           H  \\nATOM    570  HA  LEU A  41       8.699   1.496   0.093  1.00  0.00           H  \\nATOM    571  HB3 LEU A  41      10.196  -0.020  -1.073  1.00  0.00           H  \\nATOM    572  HB2 LEU A  41       9.188  -1.417  -0.683  1.00  0.00           H  \\nATOM    573  HG  LEU A  41       7.447  -0.596  -2.233  1.00  0.00           H  \\nATOM    574 HD11 LEU A  41       7.937   1.306  -3.705  1.00  0.00           H  \\nATOM    575 HD12 LEU A  41       7.876   1.796  -1.996  1.00  0.00           H  \\nATOM    576 HD13 LEU A  41       9.446   1.608  -2.812  1.00  0.00           H  \\nATOM    577 HD21 LEU A  41       8.726  -0.999  -4.309  1.00  0.00           H  \\nATOM    578 HD22 LEU A  41      10.247  -0.730  -3.425  1.00  0.00           H  \\nATOM    579 HD23 LEU A  41       9.220  -2.127  -3.024  1.00  0.00           H  \\nATOM    580  N   PRO A  42       6.427  -0.964   0.302  1.00  0.00           N  \\nATOM    581  CA  PRO A  42       4.981  -1.133   0.257  1.00  0.00           C  \\nATOM    582  C   PRO A  42       4.269  -0.359   1.371  1.00  0.00           C  \\nATOM    583  O   PRO A  42       3.202   0.187   1.124  1.00  0.00           O  \\nATOM    584  CB  PRO A  42       4.745  -2.641   0.309  1.00  0.00           C  \\nATOM    585  CG  PRO A  42       5.979  -3.193   0.999  1.00  0.00           C  \\nATOM    586  CD  PRO A  42       7.076  -2.292   0.447  1.00  0.00           C  \\nATOM    587  HA  PRO A  42       4.602  -0.756  -0.693  1.00  0.00           H  \\nATOM    588  HB3 PRO A  42       4.705  -3.032  -0.708  1.00  0.00           H  \\nATOM    589  HB2 PRO A  42       3.835  -2.894   0.854  1.00  0.00           H  \\nATOM    590  HG3 PRO A  42       6.144  -4.241   0.747  1.00  0.00           H  \\nATOM    591  HG2 PRO A  42       5.879  -3.058   2.076  1.00  0.00           H  \\nATOM    592  HD3 PRO A  42       7.365  -2.630  -0.547  1.00  0.00           H  \\nATOM    593  HD2 PRO A  42       7.936  -2.292   1.118  1.00  0.00           H  \\nATOM    594  N   PHE A  43       4.896  -0.161   2.539  1.00  0.00           N  \\nATOM    595  CA  PHE A  43       4.305   0.725   3.549  1.00  0.00           C  \\nATOM    596  C   PHE A  43       4.338   2.199   3.133  1.00  0.00           C  \\nATOM    597  O   PHE A  43       3.377   2.920   3.379  1.00  0.00           O  \\nATOM    598  CB  PHE A  43       4.972   0.524   4.914  1.00  0.00           C  \\nATOM    599  CG  PHE A  43       4.549  -0.742   5.638  1.00  0.00           C  \\nATOM    600  CD1 PHE A  43       3.187  -1.006   5.872  1.00  0.00           C  \\nATOM    601  CD2 PHE A  43       5.518  -1.644   6.114  1.00  0.00           C  \\nATOM    602  CE1 PHE A  43       2.799  -2.157   6.578  1.00  0.00           C  \\nATOM    603  CE2 PHE A  43       5.132  -2.797   6.813  1.00  0.00           C  \\nATOM    604  CZ  PHE A  43       3.773  -3.053   7.051  1.00  0.00           C  \\nATOM    605  H   PHE A  43       5.788  -0.597   2.723  1.00  0.00           H  \\nATOM    606  HA  PHE A  43       3.248   0.478   3.638  1.00  0.00           H  \\nATOM    607  HB3 PHE A  43       4.701   1.362   5.555  1.00  0.00           H  \\nATOM    608  HB2 PHE A  43       6.055   0.545   4.790  1.00  0.00           H  \\nATOM    609  HD1 PHE A  43       2.429  -0.330   5.506  1.00  0.00           H  \\nATOM    610  HD2 PHE A  43       6.565  -1.446   5.942  1.00  0.00           H  \\nATOM    611  HE1 PHE A  43       1.748  -2.354   6.737  1.00  0.00           H  \\nATOM    612  HE2 PHE A  43       5.886  -3.479   7.177  1.00  0.00           H  \\nATOM    613  HZ  PHE A  43       3.481  -3.939   7.596  1.00  0.00           H  \\nATOM    614  N   ASP A  44       5.389   2.666   2.459  1.00  0.00           N  \\nATOM    615  CA  ASP A  44       5.412   4.030   1.926  1.00  0.00           C  \\nATOM    616  C   ASP A  44       4.335   4.272   0.855  1.00  0.00           C  \\nATOM    617  O   ASP A  44       3.787   5.374   0.776  1.00  0.00           O  \\nATOM    618  CB  ASP A  44       6.792   4.337   1.338  1.00  0.00           C  \\nATOM    619  CG  ASP A  44       7.127   5.822   1.357  1.00  0.00           C  \\nATOM    620  OD1 ASP A  44       7.074   6.410   2.412  1.00  0.00           O  \\nATOM    621  OD2 ASP A  44       7.432   6.355   0.317  1.00  0.00           O  \\nATOM    622  H   ASP A  44       6.198   2.077   2.320  1.00  0.00           H  \\nATOM    623  HA  ASP A  44       5.192   4.710   2.749  1.00  0.00           H  \\nATOM    624  HB3 ASP A  44       7.555   3.800   1.901  1.00  0.00           H  \\nATOM    625  HB2 ASP A  44       6.835   3.983   0.308  1.00  0.00           H  \\nATOM    626  N   VAL A  45       3.964   3.235   0.089  1.00  0.00           N  \\nATOM    627  CA  VAL A  45       2.805   3.270  -0.825  1.00  0.00           C  \\nATOM    628  C   VAL A  45       1.509   3.430  -0.025  1.00  0.00           C  \\nATOM    629  O   VAL A  45       0.703   4.305  -0.349  1.00  0.00           O  \\nATOM    630  CB  VAL A  45       2.734   1.982  -1.666  1.00  0.00           C  \\nATOM    631  CG1 VAL A  45       1.440   1.938  -2.465  1.00  0.00           C  \\nATOM    632  CG2 VAL A  45       3.941   1.898  -2.588  1.00  0.00           C  \\nATOM    633  H   VAL A  45       4.411   2.345   0.261  1.00  0.00           H  \\nATOM    634  HA  VAL A  45       2.899   4.136  -1.479  1.00  0.00           H  \\nATOM    635  HB  VAL A  45       2.727   1.121  -0.996  1.00  0.00           H  \\nATOM    636 HG11 VAL A  45       1.407   1.021  -3.054  1.00  0.00           H  \\nATOM    637 HG12 VAL A  45       0.591   1.963  -1.783  1.00  0.00           H  \\nATOM    638 HG13 VAL A  45       1.396   2.799  -3.133  1.00  0.00           H  \\nATOM    639 HG21 VAL A  45       3.885   0.985  -3.179  1.00  0.00           H  \\nATOM    640 HG22 VAL A  45       3.951   2.762  -3.254  1.00  0.00           H  \\nATOM    641 HG23 VAL A  45       4.854   1.889  -1.993  1.00  0.00           H  \\nATOM    642  N   ASP A  46       1.325   2.637   1.034  1.00  0.00           N  \\nATOM    643  CA  ASP A  46       0.176   2.736   1.946  1.00  0.00           C  \\nATOM    644  C   ASP A  46       0.038   4.158   2.502  1.00  0.00           C  \\nATOM    645  O   ASP A  46      -1.025   4.781   2.446  1.00  0.00           O  \\nATOM    646  CB  ASP A  46       0.317   1.741   3.101  1.00  0.00           C  \\nATOM    647  CG  ASP A  46       0.303   0.290   2.638  1.00  0.00           C  \\nATOM    648  OD1 ASP A  46      -0.159   0.038   1.551  1.00  0.00           O  \\nATOM    649  OD2 ASP A  46       0.755  -0.552   3.377  1.00  0.00           O  \\nATOM    650  H   ASP A  46       1.946   1.850   1.153  1.00  0.00           H  \\nATOM    651  HA  ASP A  46      -0.733   2.480   1.402  1.00  0.00           H  \\nATOM    652  HB3 ASP A  46       1.251   1.929   3.631  1.00  0.00           H  \\nATOM    653  HB2 ASP A  46      -0.498   1.889   3.810  1.00  0.00           H  \\nATOM    654  N   CYS A  47       1.146   4.713   2.990  1.00  0.00           N  \\nATOM    655  CA  CYS A  47       1.184   6.019   3.630  1.00  0.00           C  \\nATOM    656  C   CYS A  47       0.864   7.149   2.647  1.00  0.00           C  \\nATOM    657  O   CYS A  47       0.121   8.066   3.002  1.00  0.00           O  \\nATOM    658  CB  CYS A  47       2.563   6.206   4.256  1.00  0.00           C  \\nATOM    659  SG  CYS A  47       2.895   5.094   5.649  1.00  0.00           S  \\nATOM    660  H   CYS A  47       1.992   4.163   2.960  1.00  0.00           H  \\nATOM    661  HA  CYS A  47       0.434   6.040   4.421  1.00  0.00           H  \\nATOM    662  HB3 CYS A  47       3.319   6.039   3.488  1.00  0.00           H  \\nATOM    663  HB2 CYS A  47       2.648   7.236   4.604  1.00  0.00           H  \\nATOM    664  N   TRP A  48       1.356   7.082   1.405  1.00  0.00           N  \\nATOM    665  CA  TRP A  48       0.961   8.019   0.354  1.00  0.00           C  \\nATOM    666  C   TRP A  48      -0.534   7.922   0.031  1.00  0.00           C  \\nATOM    667  O   TRP A  48      -1.213   8.949  -0.044  1.00  0.00           O  \\nATOM    668  CB  TRP A  48       1.775   7.760  -0.915  1.00  0.00           C  \\nATOM    669  CG  TRP A  48       1.386   8.639  -2.065  1.00  0.00           C  \\nATOM    670  CD1 TRP A  48       1.744   9.940  -2.254  1.00  0.00           C  \\nATOM    671  CD2 TRP A  48       0.559   8.282  -3.199  1.00  0.00           C  \\nATOM    672  NE1 TRP A  48       1.199  10.417  -3.420  1.00  0.00           N  \\nATOM    673  CE2 TRP A  48       0.471   9.415  -4.012  1.00  0.00           C  \\nATOM    674  CE3 TRP A  48      -0.104   7.110  -3.583  1.00  0.00           C  \\nATOM    675  CZ2 TRP A  48      -0.256   9.417  -5.192  1.00  0.00           C  \\nATOM    676  CZ3 TRP A  48      -0.832   7.111  -4.767  1.00  0.00           C  \\nATOM    677  CH2 TRP A  48      -0.905   8.235  -5.551  1.00  0.00           C  \\nATOM    678  H   TRP A  48       1.979   6.323   1.168  1.00  0.00           H  \\nATOM    679  HA  TRP A  48       1.155   9.036   0.695  1.00  0.00           H  \\nATOM    680  HB3 TRP A  48       2.833   7.916  -0.707  1.00  0.00           H  \\nATOM    681  HB2 TRP A  48       1.651   6.721  -1.221  1.00  0.00           H  \\nATOM    682  HD1 TRP A  48       2.372  10.518  -1.577  1.00  0.00           H  \\nATOM    683  HE1 TRP A  48       1.312  11.352  -3.783  1.00  0.00           H  \\nATOM    684  HE3 TRP A  48      -0.047   6.214  -2.966  1.00  0.00           H  \\nATOM    685  HZ2 TRP A  48      -0.326  10.300  -5.828  1.00  0.00           H  \\nATOM    686  HZ3 TRP A  48      -1.345   6.195  -5.059  1.00  0.00           H  \\nATOM    687  HH2 TRP A  48      -1.484   8.202  -6.473  1.00  0.00           H  \\nATOM    688  N   ALA A  49      -1.068   6.709  -0.127  1.00  0.00           N  \\nATOM    689  CA  ALA A  49      -2.484   6.489  -0.396  1.00  0.00           C  \\nATOM    690  C   ALA A  49      -3.367   7.139   0.686  1.00  0.00           C  \\nATOM    691  O   ALA A  49      -4.376   7.779   0.373  1.00  0.00           O  \\nATOM    692  CB  ALA A  49      -2.722   4.978  -0.514  1.00  0.00           C  \\nATOM    693  H   ALA A  49      -0.464   5.902  -0.051  1.00  0.00           H  \\nATOM    694  HA  ALA A  49      -2.734   6.953  -1.351  1.00  0.00           H  \\nATOM    695  HB1 ALA A  49      -3.784   4.774  -0.648  1.00  0.00           H  \\nATOM    696  HB2 ALA A  49      -2.165   4.580  -1.362  1.00  0.00           H  \\nATOM    697  HB3 ALA A  49      -2.390   4.466   0.389  1.00  0.00           H  \\nATOM    698  N   ILE A  50      -2.958   7.058   1.955  1.00  0.00           N  \\nATOM    699  CA  ILE A  50      -3.655   7.699   3.078  1.00  0.00           C  \\nATOM    700  C   ILE A  50      -3.441   9.217   3.045  1.00  0.00           C  \\nATOM    701  O   ILE A  50      -4.397   9.996   3.066  1.00  0.00           O  \\nATOM    702  CB  ILE A  50      -3.171   7.138   4.427  1.00  0.00           C  \\nATOM    703  CG1 ILE A  50      -3.529   5.654   4.547  1.00  0.00           C  \\nATOM    704  CG2 ILE A  50      -3.772   7.929   5.579  1.00  0.00           C  \\nATOM    705  CD1 ILE A  50      -2.830   4.947   5.686  1.00  0.00           C  \\nATOM    706  H   ILE A  50      -2.146   6.490   2.151  1.00  0.00           H  \\nATOM    707  HA  ILE A  50      -4.727   7.515   2.989  1.00  0.00           H  \\nATOM    708  HB  ILE A  50      -2.085   7.205   4.480  1.00  0.00           H  \\nATOM    709 HG13 ILE A  50      -4.605   5.550   4.690  1.00  0.00           H  \\nATOM    710 HG12 ILE A  50      -3.272   5.141   3.620  1.00  0.00           H  \\nATOM    711 HG21 ILE A  50      -3.419   7.518   6.525  1.00  0.00           H  \\nATOM    712 HG22 ILE A  50      -3.468   8.972   5.501  1.00  0.00           H  \\nATOM    713 HG23 ILE A  50      -4.859   7.863   5.537  1.00  0.00           H  \\nATOM    714 HD11 ILE A  50      -3.135   3.901   5.706  1.00  0.00           H  \\nATOM    715 HD12 ILE A  50      -1.751   5.009   5.544  1.00  0.00           H  \\nATOM    716 HD13 ILE A  50      -3.101   5.421   6.628  1.00  0.00           H  \\nATOM    717  N   LEU A  51      -2.181   9.658   2.980  1.00  0.00           N  \\nATOM    718  CA  LEU A  51      -1.821  11.064   3.138  1.00  0.00           C  \\nATOM    719  C   LEU A  51      -2.303  11.923   1.963  1.00  0.00           C  \\nATOM    720  O   LEU A  51      -2.836  13.010   2.179  1.00  0.00           O  \\nATOM    721  CB  LEU A  51      -0.300  11.198   3.283  1.00  0.00           C  \\nATOM    722  CG  LEU A  51       0.225  12.623   3.496  1.00  0.00           C  \\nATOM    723  CD1 LEU A  51      -0.355  13.192   4.783  1.00  0.00           C  \\nATOM    724  CD2 LEU A  51       1.746  12.603   3.543  1.00  0.00           C  \\nATOM    725  H   LEU A  51      -1.437   8.977   2.930  1.00  0.00           H  \\nATOM    726  HA  LEU A  51      -2.315  11.422   4.041  1.00  0.00           H  \\nATOM    727  HB3 LEU A  51       0.023  10.596   4.131  1.00  0.00           H  \\nATOM    728  HB2 LEU A  51       0.171  10.802   2.383  1.00  0.00           H  \\nATOM    729  HG  LEU A  51      -0.103  13.259   2.673  1.00  0.00           H  \\nATOM    730 HD11 LEU A  51       0.018  14.205   4.935  1.00  0.00           H  \\nATOM    731 HD12 LEU A  51      -1.443  13.213   4.714  1.00  0.00           H  \\nATOM    732 HD13 LEU A  51      -0.057  12.566   5.624  1.00  0.00           H  \\nATOM    733 HD21 LEU A  51       2.119  13.616   3.694  1.00  0.00           H  \\nATOM    734 HD22 LEU A  51       2.075  11.968   4.367  1.00  0.00           H  \\nATOM    735 HD23 LEU A  51       2.134  12.209   2.604  1.00  0.00           H  \\nATOM    736  N   CYS A  52      -2.113  11.457   0.732  1.00  0.00           N  \\nATOM    737  CA  CYS A  52      -2.268  12.273  -0.469  1.00  0.00           C  \\nATOM    738  C   CYS A  52      -3.500  11.919  -1.306  1.00  0.00           C  \\nATOM    739  O   CYS A  52      -3.990  12.776  -2.038  1.00  0.00           O  \\nATOM    740  CB  CYS A  52      -1.025  12.139  -1.349  1.00  0.00           C  \\nATOM    741  SG  CYS A  52       0.516  12.608  -0.526  1.00  0.00           S  \\nATOM    742  H   CYS A  52      -1.675  10.553   0.632  1.00  0.00           H  \\nATOM    743  HA  CYS A  52      -2.387  13.318  -0.185  1.00  0.00           H  \\nATOM    744  HB3 CYS A  52      -0.927  11.107  -1.686  1.00  0.00           H  \\nATOM    745  HB2 CYS A  52      -1.138  12.763  -2.235  1.00  0.00           H  \\nATOM    746  N   LYS A  53      -4.013  10.685  -1.213  1.00  0.00           N  \\nATOM    747  CA  LYS A  53      -5.196  10.239  -1.972  1.00  0.00           C  \\nATOM    748  C   LYS A  53      -6.451  10.085  -1.116  1.00  0.00           C  \\nATOM    749  O   LYS A  53      -7.524   9.889  -1.675  1.00  0.00           O  \\nATOM    750  CB  LYS A  53      -4.898   8.914  -2.674  1.00  0.00           C  \\nATOM    751  CG  LYS A  53      -3.748   8.974  -3.670  1.00  0.00           C  \\nATOM    752  CD  LYS A  53      -4.073   9.900  -4.833  1.00  0.00           C  \\nATOM    753  CE  LYS A  53      -5.235   9.367  -5.658  1.00  0.00           C  \\nATOM    754  NZ  LYS A  53      -5.387  10.101  -6.944  1.00  0.00           N  \\nATOM    755  H   LYS A  53      -3.538  10.008  -0.633  1.00  0.00           H  \\nATOM    756  HA  LYS A  53      -5.471  11.011  -2.691  1.00  0.00           H  \\nATOM    757  HB3 LYS A  53      -4.657   8.154  -1.930  1.00  0.00           H  \\nATOM    758  HB2 LYS A  53      -5.787   8.577  -3.208  1.00  0.00           H  \\nATOM    759  HG3 LYS A  53      -2.850   9.336  -3.168  1.00  0.00           H  \\nATOM    760  HG2 LYS A  53      -3.549   7.975  -4.057  1.00  0.00           H  \\nATOM    761  HD3 LYS A  53      -4.333  10.888  -4.450  1.00  0.00           H  \\nATOM    762  HD2 LYS A  53      -3.198   9.997  -5.476  1.00  0.00           H  \\nATOM    763  HE3 LYS A  53      -5.074   8.312  -5.874  1.00  0.00           H  \\nATOM    764  HE2 LYS A  53      -6.159   9.463  -5.089  1.00  0.00           H  \\nATOM    765  HZ1 LYS A  53      -6.166   9.719  -7.460  1.00  0.00           H  \\nATOM    766  HZ2 LYS A  53      -5.555  11.080  -6.756  1.00  0.00           H  \\nATOM    767  HZ3 LYS A  53      -4.543  10.004  -7.490  1.00  0.00           H  \\nATOM    768  N   GLY A  54      -6.335  10.206   0.209  1.00  0.00           N  \\nATOM    769  CA  GLY A  54      -7.479  10.146   1.118  1.00  0.00           C  \\nATOM    770  C   GLY A  54      -7.973   8.722   1.362  1.00  0.00           C  \\nATOM    771  O   GLY A  54      -9.132   8.536   1.728  1.00  0.00           O  \\nATOM    772  H   GLY A  54      -5.409  10.247   0.613  1.00  0.00           H  \\nATOM    773  HA3 GLY A  54      -8.303  10.733   0.713  1.00  0.00           H  \\nATOM    774  HA2 GLY A  54      -7.185  10.565   2.081  1.00  0.00           H  \\nATOM    775  N   ALA A  55      -7.122   7.715   1.130  1.00  0.00           N  \\nATOM    776  CA  ALA A  55      -7.490   6.325   1.356  1.00  0.00           C  \\nATOM    777  C   ALA A  55      -7.851   6.093   2.840  1.00  0.00           C  \\nATOM    778  O   ALA A  55      -7.234   6.698   3.725  1.00  0.00           O  \\nATOM    779  CB  ALA A  55      -6.352   5.417   0.879  1.00  0.00           C  \\nATOM    780  H   ALA A  55      -6.159   7.924   0.907  1.00  0.00           H  \\nATOM    781  HA  ALA A  55      -8.370   6.119   0.747  1.00  0.00           H  \\nATOM    782  HB1 ALA A  55      -6.082   5.668  -0.147  1.00  0.00           H  \\nATOM    783  HB2 ALA A  55      -5.484   5.537   1.528  1.00  0.00           H  \\nATOM    784  HB3 ALA A  55      -6.668   4.374   0.912  1.00  0.00           H  \\nATOM    785  N   PRO A  56      -8.842   5.232   3.133  1.00  0.00           N  \\nATOM    786  CA  PRO A  56      -9.236   4.942   4.506  1.00  0.00           C  \\nATOM    787  C   PRO A  56      -8.031   4.493   5.336  1.00  0.00           C  \\nATOM    788  O   PRO A  56      -7.273   3.638   4.899  1.00  0.00           O  \\nATOM    789  CB  PRO A  56     -10.263   3.817   4.340  1.00  0.00           C  \\nATOM    790  CG  PRO A  56     -10.857   4.060   2.995  1.00  0.00           C  \\nATOM    791  CD  PRO A  56      -9.698   4.516   2.150  1.00  0.00           C  \\nATOM    792  HA  PRO A  56      -9.673   5.833   4.959  1.00  0.00           H  \\nATOM    793  HB3 PRO A  56      -9.765   2.840   4.417  1.00  0.00           H  \\nATOM    794  HB2 PRO A  56     -11.006   3.864   5.150  1.00  0.00           H  \\nATOM    795  HG3 PRO A  56     -11.323   3.140   2.614  1.00  0.00           H  \\nATOM    796  HG2 PRO A  56     -11.655   4.815   3.063  1.00  0.00           H  \\nATOM    797  HD3 PRO A  56      -9.186   3.640   1.726  1.00  0.00           H  \\nATOM    798  HD2 PRO A  56     -10.063   5.177   1.351  1.00  0.00           H  \\nATOM    799  N   ASN A  57      -7.889   5.037   6.546  1.00  0.00           N  \\nATOM    800  CA  ASN A  57      -6.860   4.645   7.518  1.00  0.00           C  \\nATOM    801  C   ASN A  57      -7.417   3.912   8.751  1.00  0.00           C  \\nATOM    802  O   ASN A  57      -6.652   3.518   9.629  1.00  0.00           O  \\nATOM    803  CB  ASN A  57      -6.062   5.894   7.922  1.00  0.00           C  \\nATOM    804  CG  ASN A  57      -6.897   6.964   8.609  1.00  0.00           C  \\nATOM    805  OD1 ASN A  57      -8.115   6.924   8.664  1.00  0.00           O  \\nATOM    806  ND2 ASN A  57      -6.262   7.984   9.134  1.00  0.00           N  \\nATOM    807  H   ASN A  57      -8.482   5.816   6.798  1.00  0.00           H  \\nATOM    808  HA  ASN A  57      -6.168   3.954   7.037  1.00  0.00           H  \\nATOM    809  HB3 ASN A  57      -5.609   6.328   7.030  1.00  0.00           H  \\nATOM    810  HB2 ASN A  57      -5.262   5.592   8.596  1.00  0.00           H  \\nATOM    811 HD21 ASN A  57      -6.842   8.689   9.565  1.00  0.00           H  \\nATOM    812 HD22 ASN A  57      -5.259   8.061   9.044  1.00  0.00           H  \\nATOM    813  N   VAL A  58      -8.737   3.726   8.816  1.00  0.00           N  \\nATOM    814  CA  VAL A  58      -9.416   2.928   9.838  1.00  0.00           C  \\nATOM    815  C   VAL A  58     -10.046   1.731   9.143  1.00  0.00           C  \\nATOM    816  O   VAL A  58     -10.973   1.879   8.343  1.00  0.00           O  \\nATOM    817  CB  VAL A  58     -10.497   3.750  10.565  1.00  0.00           C  \\nATOM    818  CG1 VAL A  58     -11.207   2.895  11.604  1.00  0.00           C  \\nATOM    819  CG2 VAL A  58      -9.869   4.975  11.211  1.00  0.00           C  \\nATOM    820  H   VAL A  58      -9.293   4.069   8.046  1.00  0.00           H  \\nATOM    821  HA  VAL A  58      -8.681   2.564  10.556  1.00  0.00           H  \\nATOM    822  HB  VAL A  58     -11.249   4.065   9.841  1.00  0.00           H  \\nATOM    823 HG11 VAL A  58     -11.968   3.492  12.108  1.00  0.00           H  \\nATOM    824 HG12 VAL A  58     -11.679   2.044  11.114  1.00  0.00           H  \\nATOM    825 HG13 VAL A  58     -10.483   2.538  12.337  1.00  0.00           H  \\nATOM    826 HG21 VAL A  58     -10.640   5.552  11.723  1.00  0.00           H  \\nATOM    827 HG22 VAL A  58      -9.114   4.660  11.931  1.00  0.00           H  \\nATOM    828 HG23 VAL A  58      -9.403   5.593  10.444  1.00  0.00           H  \\nATOM    829  N   LEU A  59      -9.510   0.550   9.424  1.00  0.00           N  \\nATOM    830  CA  LEU A  59      -9.762  -0.671   8.672  1.00  0.00           C  \\nATOM    831  C   LEU A  59     -10.378  -1.748   9.562  1.00  0.00           C  \\nATOM    832  O   LEU A  59     -10.216  -1.730  10.779  1.00  0.00           O  \\nATOM    833  CB  LEU A  59      -8.457  -1.190   8.055  1.00  0.00           C  \\nATOM    834  CG  LEU A  59      -7.689  -0.189   7.182  1.00  0.00           C  \\nATOM    835  CD1 LEU A  59      -6.380  -0.815   6.722  1.00  0.00           C  \\nATOM    836  CD2 LEU A  59      -8.550   0.214   5.994  1.00  0.00           C  \\nATOM    837  H   LEU A  59      -8.823   0.504  10.163  1.00  0.00           H  \\nATOM    838  HA  LEU A  59     -10.470  -0.461   7.871  1.00  0.00           H  \\nATOM    839  HB3 LEU A  59      -7.794  -1.505   8.859  1.00  0.00           H  \\nATOM    840  HB2 LEU A  59      -8.686  -2.060   7.438  1.00  0.00           H  \\nATOM    841  HG  LEU A  59      -7.447   0.696   7.771  1.00  0.00           H  \\nATOM    842 HD11 LEU A  59      -5.834  -0.103   6.102  1.00  0.00           H  \\nATOM    843 HD12 LEU A  59      -5.776  -1.075   7.591  1.00  0.00           H  \\nATOM    844 HD13 LEU A  59      -6.590  -1.713   6.143  1.00  0.00           H  \\nATOM    845 HD21 LEU A  59      -8.005   0.926   5.374  1.00  0.00           H  \\nATOM    846 HD22 LEU A  59      -8.792  -0.670   5.404  1.00  0.00           H  \\nATOM    847 HD23 LEU A  59      -9.471   0.674   6.351  1.00  0.00           H  \\nATOM    848  N   GLN A  60     -11.044  -2.723   8.949  1.00  0.00           N  \\nATOM    849  CA  GLN A  60     -11.561  -3.898   9.653  1.00  0.00           C  \\nATOM    850  C   GLN A  60     -11.051  -5.164   8.972  1.00  0.00           C  \\nATOM    851  O   GLN A  60     -11.257  -5.337   7.774  1.00  0.00           O  \\nATOM    852  CB  GLN A  60     -13.092  -3.844   9.690  1.00  0.00           C  \\nATOM    853  CG  GLN A  60     -13.659  -4.775  10.770  1.00  0.00           C  \\nATOM    854  CD  GLN A  60     -15.182  -4.759  10.813  1.00  0.00           C  \\nATOM    855  OE1 GLN A  60     -15.862  -4.405   9.865  1.00  0.00           O  \\nATOM    856  NE2 GLN A  60     -15.793  -5.155  11.909  1.00  0.00           N  \\nATOM    857  H   GLN A  60     -11.156  -2.667   7.948  1.00  0.00           H  \\nATOM    858  HA  GLN A  60     -11.194  -3.893  10.680  1.00  0.00           H  \\nATOM    859  HB3 GLN A  60     -13.493  -4.107   8.711  1.00  0.00           H  \\nATOM    860  HB2 GLN A  60     -13.412  -2.828   9.926  1.00  0.00           H  \\nATOM    861  HG3 GLN A  60     -13.293  -4.438  11.740  1.00  0.00           H  \\nATOM    862  HG2 GLN A  60     -13.320  -5.798  10.608  1.00  0.00           H  \\nATOM    863 HE21 GLN A  60     -16.801  -5.114  11.873  1.00  0.00           H  \\nATOM    864 HE22 GLN A  60     -15.294  -5.340  12.768  1.00  0.00           H  \\nATOM    865  N   ARG A  61     -10.363  -6.047   9.703  1.00  0.00           N  \\nATOM    866  CA  ARG A  61      -9.668  -7.196   9.100  1.00  0.00           C  \\nATOM    867  C   ARG A  61     -10.657  -8.279   8.666  1.00  0.00           C  \\nATOM    868  O   ARG A  61     -11.500  -8.682   9.455  1.00  0.00           O  \\nATOM    869  CB  ARG A  61      -8.614  -7.737  10.075  1.00  0.00           C  \\nATOM    870  CG  ARG A  61      -7.678  -8.756   9.412  1.00  0.00           C  \\nATOM    871  CD  ARG A  61      -6.748  -9.365  10.462  1.00  0.00           C  \\nATOM    872  NE  ARG A  61      -5.890 -10.404   9.869  1.00  0.00           N  \\nATOM    873  CZ  ARG A  61      -5.146 -11.265  10.540  1.00  0.00           C  \\nATOM    874  NH1 ARG A  61      -5.205 -11.357  11.841  1.00  0.00           N  \\nATOM    875  NH2 ARG A  61      -4.321 -12.053   9.911  1.00  0.00           N  \\nATOM    876  H   ARG A  61     -10.316  -5.911  10.703  1.00  0.00           H  \\nATOM    877  HA  ARG A  61      -9.154  -6.841   8.206  1.00  0.00           H  \\nATOM    878  HB3 ARG A  61      -8.016  -6.904  10.445  1.00  0.00           H  \\nATOM    879  HB2 ARG A  61      -9.118  -8.203  10.922  1.00  0.00           H  \\nATOM    880  HG3 ARG A  61      -8.258  -9.556   8.952  1.00  0.00           H  \\nATOM    881  HG2 ARG A  61      -7.082  -8.257   8.647  1.00  0.00           H  \\nATOM    882  HD3 ARG A  61      -7.356  -9.797  11.257  1.00  0.00           H  \\nATOM    883  HD2 ARG A  61      -6.126  -8.577  10.886  1.00  0.00           H  \\nATOM    884  HE  ARG A  61      -5.848 -10.441   8.861  1.00  0.00           H  \\nATOM    885 HH11 ARG A  61      -5.850 -10.767  12.346  1.00  0.00           H  \\nATOM    886 HH12 ARG A  61      -4.637 -12.026  12.340  1.00  0.00           H  \\nATOM    887 HH21 ARG A  61      -4.237 -12.000   8.906  1.00  0.00           H  \\nATOM    888 HH22 ARG A  61      -3.685 -12.645  10.427  1.00  0.00           H  \\nATOM    889  N   VAL A  62     -10.522  -8.812   7.457  1.00  0.00           N  \\nATOM    890  CA  VAL A  62     -11.429  -9.831   6.905  1.00  0.00           C  \\nATOM    891  C   VAL A  62     -10.750 -11.185   6.725  1.00  0.00           C  \\nATOM    892  O   VAL A  62      -9.524 -11.302   6.760  1.00  0.00           O  \\nATOM    893  CB  VAL A  62     -12.102  -9.368   5.602  1.00  0.00           C  \\nATOM    894  CG1 VAL A  62     -12.973  -8.140   5.868  1.00  0.00           C  \\nATOM    895  CG2 VAL A  62     -11.107  -9.066   4.477  1.00  0.00           C  \\nATOM    896  H   VAL A  62      -9.741  -8.512   6.893  1.00  0.00           H  \\nATOM    897  HA  VAL A  62     -12.242  -9.994   7.613  1.00  0.00           H  \\nATOM    898  HB  VAL A  62     -12.770 -10.163   5.271  1.00  0.00           H  \\nATOM    899 HG11 VAL A  62     -12.364  -7.320   6.247  1.00  0.00           H  \\nATOM    900 HG12 VAL A  62     -13.471  -7.833   4.948  1.00  0.00           H  \\nATOM    901 HG13 VAL A  62     -13.727  -8.384   6.616  1.00  0.00           H  \\nATOM    902 HG21 VAL A  62     -10.469  -9.930   4.286  1.00  0.00           H  \\nATOM    903 HG22 VAL A  62     -10.493  -8.207   4.744  1.00  0.00           H  \\nATOM    904 HG23 VAL A  62     -11.646  -8.820   3.562  1.00  0.00           H  \\nATOM    905  N   ASN A  63     -11.568 -12.219   6.540  1.00  0.00           N  \\nATOM    906  CA  ASN A  63     -11.124 -13.594   6.348  1.00  0.00           C  \\nATOM    907  C   ASN A  63     -10.470 -13.829   4.972  1.00  0.00           C  \\nATOM    908  O   ASN A  63     -10.441 -12.965   4.094  1.00  0.00           O  \\nATOM    909  CB  ASN A  63     -12.291 -14.543   6.550  1.00  0.00           C  \\nATOM    910  CG  ASN A  63     -13.321 -14.432   5.461  1.00  0.00           C  \\nATOM    911  OD1 ASN A  63     -13.125 -13.712   4.475  1.00  0.00           O  \\nATOM    912  ND2 ASN A  63     -14.416 -15.131   5.620  1.00  0.00           N  \\nATOM    913  H   ASN A  63     -12.562 -12.036   6.539  1.00  0.00           H  \\nATOM    914  HA  ASN A  63     -10.357 -13.794   7.096  1.00  0.00           H  \\nATOM    915  HB3 ASN A  63     -11.923 -15.570   6.583  1.00  0.00           H  \\nATOM    916  HB2 ASN A  63     -12.768 -14.335   7.508  1.00  0.00           H  \\nATOM    917 HD21 ASN A  63     -15.137 -15.096   4.927  1.00  0.00           H  \\nATOM    918 HD22 ASN A  63     -14.533 -15.700   6.433  1.00  0.00           H  \\nATOM    919  N   GLU A  64      -9.997 -15.058   4.766  1.00  0.00           N  \\nATOM    920  CA  GLU A  64      -9.261 -15.468   3.566  1.00  0.00           C  \\nATOM    921  C   GLU A  64     -10.100 -15.410   2.280  1.00  0.00           C  \\nATOM    922  O   GLU A  64      -9.631 -14.938   1.246  1.00  0.00           O  \\nATOM    923  CB  GLU A  64      -8.753 -16.900   3.806  1.00  0.00           C  \\nATOM    924  CG  GLU A  64      -7.922 -17.466   2.648  1.00  0.00           C  \\nATOM    925  CD  GLU A  64      -6.577 -16.752   2.489  1.00  0.00           C  \\nATOM    926  OE1 GLU A  64      -5.991 -16.811   1.385  1.00  0.00           O  \\nATOM    927  OE2 GLU A  64      -6.113 -16.083   3.444  1.00  0.00           O  \\nATOM    928  H   GLU A  64     -10.006 -15.704   5.543  1.00  0.00           H  \\nATOM    929  HA  GLU A  64      -8.412 -14.795   3.437  1.00  0.00           H  \\nATOM    930  HB3 GLU A  64      -8.150 -16.919   4.713  1.00  0.00           H  \\nATOM    931  HB2 GLU A  64      -9.608 -17.556   3.972  1.00  0.00           H  \\nATOM    932  HG3 GLU A  64      -8.496 -17.394   1.724  1.00  0.00           H  \\nATOM    933  HG2 GLU A  64      -7.740 -18.523   2.842  1.00  0.00           H  \\nATOM    934  N   LYS A  65     -11.364 -15.847   2.347  1.00  0.00           N  \\nATOM    935  CA  LYS A  65     -12.282 -15.819   1.201  1.00  0.00           C  \\nATOM    936  C   LYS A  65     -12.479 -14.392   0.687  1.00  0.00           C  \\nATOM    937  O   LYS A  65     -12.390 -14.152  -0.515  1.00  0.00           O  \\nATOM    938  CB  LYS A  65     -13.632 -16.431   1.577  1.00  0.00           C  \\nATOM    939  CG  LYS A  65     -14.640 -16.482   0.436  1.00  0.00           C  \\nATOM    940  CD  LYS A  65     -15.928 -17.166   0.866  1.00  0.00           C  \\nATOM    941  CE  LYS A  65     -16.947 -17.191  -0.264  1.00  0.00           C  \\nATOM    942  NZ  LYS A  65     -18.209 -17.869   0.140  1.00  0.00           N  \\nATOM    943  H   LYS A  65     -11.693 -16.195   3.237  1.00  0.00           H  \\nATOM    944  HA  LYS A  65     -11.838 -16.383   0.381  1.00  0.00           H  \\nATOM    945  HB3 LYS A  65     -13.482 -17.450   1.936  1.00  0.00           H  \\nATOM    946  HB2 LYS A  65     -14.077 -15.859   2.391  1.00  0.00           H  \\nATOM    947  HG3 LYS A  65     -14.869 -15.467   0.108  1.00  0.00           H  \\nATOM    948  HG2 LYS A  65     -14.211 -17.028  -0.404  1.00  0.00           H  \\nATOM    949  HD3 LYS A  65     -15.712 -18.190   1.173  1.00  0.00           H  \\nATOM    950  HD2 LYS A  65     -16.356 -16.634   1.716  1.00  0.00           H  \\nATOM    951  HE3 LYS A  65     -17.178 -16.172  -0.568  1.00  0.00           H  \\nATOM    952  HE2 LYS A  65     -16.526 -17.716  -1.121  1.00  0.00           H  \\nATOM    953  HZ1 LYS A  65     -18.857 -17.865  -0.636  1.00  0.00           H  \\nATOM    954  HZ2 LYS A  65     -18.009 -18.823   0.407  1.00  0.00           H  \\nATOM    955  HZ3 LYS A  65     -18.619 -17.380   0.922  1.00  0.00           H  \\nATOM    956  N   THR A  66     -12.720 -13.450   1.598  1.00  0.00           N  \\nATOM    957  CA  THR A  66     -12.908 -12.041   1.242  1.00  0.00           C  \\nATOM    958  C   THR A  66     -11.611 -11.440   0.723  1.00  0.00           C  \\nATOM    959  O   THR A  66     -11.626 -10.795  -0.317  1.00  0.00           O  \\nATOM    960  CB  THR A  66     -13.413 -11.220   2.443  1.00  0.00           C  \\nATOM    961  OG1 THR A  66     -14.621 -11.803   2.949  1.00  0.00           O  \\nATOM    962  CG2 THR A  66     -13.683  -9.781   2.031  1.00  0.00           C  \\nATOM    963  H   THR A  66     -12.751 -13.707   2.574  1.00  0.00           H  \\nATOM    964  HA  THR A  66     -13.655 -11.964   0.452  1.00  0.00           H  \\nATOM    965  HB  THR A  66     -12.662 -11.231   3.233  1.00  0.00           H  \\nATOM    966  HG1 THR A  66     -14.406 -12.576   3.477  1.00  0.00           H  \\nATOM    967 HG21 THR A  66     -14.039  -9.217   2.892  1.00  0.00           H  \\nATOM    968 HG22 THR A  66     -12.764  -9.332   1.656  1.00  0.00           H  \\nATOM    969 HG23 THR A  66     -14.441  -9.763   1.248  1.00  0.00           H  \\nATOM    970  N   LYS A  67     -10.473 -11.719   1.373  1.00  0.00           N  \\nATOM    971  CA  LYS A  67      -9.158 -11.263   0.910  1.00  0.00           C  \\nATOM    972  C   LYS A  67      -8.898 -11.650  -0.543  1.00  0.00           C  \\nATOM    973  O   LYS A  67      -8.531 -10.798  -1.344  1.00  0.00           O  \\nATOM    974  CB  LYS A  67      -8.069 -11.818   1.838  1.00  0.00           C  \\nATOM    975  CG  LYS A  67      -6.653 -11.377   1.426  1.00  0.00           C  \\nATOM    976  CD  LYS A  67      -5.570 -12.250   2.064  1.00  0.00           C  \\nATOM    977  CE  LYS A  67      -5.505 -13.583   1.317  1.00  0.00           C  \\nATOM    978  NZ  LYS A  67      -4.546 -14.528   1.923  1.00  0.00           N  \\nATOM    979  H   LYS A  67     -10.524 -12.252   2.229  1.00  0.00           H  \\nATOM    980  HA  LYS A  67      -9.148 -10.173   0.944  1.00  0.00           H  \\nATOM    981  HB3 LYS A  67      -8.257 -11.478   2.858  1.00  0.00           H  \\nATOM    982  HB2 LYS A  67      -8.124 -12.907   1.830  1.00  0.00           H  \\nATOM    983  HG3 LYS A  67      -6.530 -11.409   0.343  1.00  0.00           H  \\nATOM    984  HG2 LYS A  67      -6.501 -10.346   1.748  1.00  0.00           H  \\nATOM    985  HD3 LYS A  67      -4.610 -11.740   1.979  1.00  0.00           H  \\nATOM    986  HD2 LYS A  67      -5.798 -12.412   3.118  1.00  0.00           H  \\nATOM    987  HE3 LYS A  67      -6.491 -14.048   1.331  1.00  0.00           H  \\nATOM    988  HE2 LYS A  67      -5.260 -13.412   0.269  1.00  0.00           H  \\nATOM    989  HZ1 LYS A  67      -4.678 -15.414   1.455  1.00  0.00           H  \\nATOM    990  HZ2 LYS A  67      -3.580 -14.236   1.941  1.00  0.00           H  \\nATOM    991  HZ3 LYS A  67      -4.895 -14.798   2.831  1.00  0.00           H  \\nATOM    992  N   ASN A  68      -9.117 -12.913  -0.903  1.00  0.00           N  \\nATOM    993  CA  ASN A  68      -8.921 -13.371  -2.278  1.00  0.00           C  \\nATOM    994  C   ASN A  68      -9.882 -12.677  -3.252  1.00  0.00           C  \\nATOM    995  O   ASN A  68      -9.439 -12.218  -4.300  1.00  0.00           O  \\nATOM    996  CB  ASN A  68      -8.999 -14.905  -2.300  1.00  0.00           C  \\nATOM    997  CG  ASN A  68      -7.769 -15.528  -1.655  1.00  0.00           C  \\nATOM    998  OD1 ASN A  68      -6.697 -14.938  -1.598  1.00  0.00           O  \\nATOM    999  ND2 ASN A  68      -7.869 -16.725  -1.133  1.00  0.00           N  \\nATOM   1000  H   ASN A  68      -9.364 -13.583  -0.188  1.00  0.00           H  \\nATOM   1001  HA  ASN A  68      -7.922 -13.081  -2.604  1.00  0.00           H  \\nATOM   1002  HB3 ASN A  68      -9.897 -15.237  -1.778  1.00  0.00           H  \\nATOM   1003  HB2 ASN A  68      -9.048 -15.250  -3.333  1.00  0.00           H  \\nATOM   1004 HD21 ASN A  68      -8.742 -17.232  -1.148  1.00  0.00           H  \\nATOM   1005 HD22 ASN A  68      -7.055 -17.033  -0.621  1.00  0.00           H  \\nATOM   1006  N   SER A  69     -11.144 -12.473  -2.857  1.00  0.00           N  \\nATOM   1007  CA  SER A  69     -12.095 -11.684  -3.653  1.00  0.00           C  \\nATOM   1008  C   SER A  69     -11.657 -10.223  -3.818  1.00  0.00           C  \\nATOM   1009  O   SER A  69     -11.785  -9.657  -4.901  1.00  0.00           O  \\nATOM   1010  CB  SER A  69     -13.466 -11.731  -3.007  1.00  0.00           C  \\nATOM   1011  OG  SER A  69     -13.986 -13.032  -3.024  1.00  0.00           O  \\nATOM   1012  H   SER A  69     -11.440 -12.844  -1.966  1.00  0.00           H  \\nATOM   1013  HA  SER A  69     -12.154 -12.123  -4.650  1.00  0.00           H  \\nATOM   1014  HB3 SER A  69     -13.396 -11.378  -1.978  1.00  0.00           H  \\nATOM   1015  HB2 SER A  69     -14.141 -11.060  -3.537  1.00  0.00           H  \\nATOM   1016  HG  SER A  69     -13.490 -13.524  -2.366  1.00  0.00           H  \\nATOM   1017  N   ASN A  70     -11.092  -9.607  -2.780  1.00  0.00           N  \\nATOM   1018  CA  ASN A  70     -10.568  -8.247  -2.860  1.00  0.00           C  \\nATOM   1019  C   ASN A  70      -9.391  -8.170  -3.838  1.00  0.00           C  \\nATOM   1020  O   ASN A  70      -9.339  -7.236  -4.627  1.00  0.00           O  \\nATOM   1021  CB  ASN A  70     -10.160  -7.751  -1.485  1.00  0.00           C  \\nATOM   1022  CG  ASN A  70     -11.339  -7.369  -0.635  1.00  0.00           C  \\nATOM   1023  OD1 ASN A  70     -12.436  -7.121  -1.149  1.00  0.00           O  \\nATOM   1024  ND2 ASN A  70     -11.136  -7.318   0.657  1.00  0.00           N  \\nATOM   1025  H   ASN A  70     -11.069 -10.081  -1.888  1.00  0.00           H  \\nATOM   1026  HA  ASN A  70     -11.347  -7.601  -3.265  1.00  0.00           H  \\nATOM   1027  HB3 ASN A  70      -9.593  -8.528  -0.972  1.00  0.00           H  \\nATOM   1028  HB2 ASN A  70      -9.507  -6.885  -1.590  1.00  0.00           H  \\nATOM   1029 HD21 ASN A  70     -11.885  -7.070   1.272  1.00  0.00           H  \\nATOM   1030 HD22 ASN A  70     -10.232  -7.528   1.029  1.00  0.00           H  \\nATOM   1031  N   ARG A  71      -8.507  -9.175  -3.871  1.00  0.00           N  \\nATOM   1032  CA  ARG A  71      -7.391  -9.219  -4.837  1.00  0.00           C  \\nATOM   1033  C   ARG A  71      -7.843  -9.309  -6.297  1.00  0.00           C  \\nATOM   1034  O   ARG A  71      -7.131  -8.851  -7.192  1.00  0.00           O  \\nATOM   1035  CB  ARG A  71      -6.487 -10.406  -4.537  1.00  0.00           C  \\nATOM   1036  CG  ARG A  71      -5.660 -10.275  -3.268  1.00  0.00           C  \\nATOM   1037  CD  ARG A  71      -4.765 -11.444  -3.072  1.00  0.00           C  \\nATOM   1038  NE  ARG A  71      -3.814 -11.588  -4.162  1.00  0.00           N  \\nATOM   1039  CZ  ARG A  71      -2.635 -10.940  -4.239  1.00  0.00           C  \\nATOM   1040  NH1 ARG A  71      -2.279 -10.109  -3.284  1.00  0.00           N  \\nATOM   1041  NH2 ARG A  71      -1.837 -11.139  -5.273  1.00  0.00           N  \\nATOM   1042  H   ARG A  71      -8.617  -9.930  -3.210  1.00  0.00           H  \\nATOM   1043  HA  ARG A  71      -6.818  -8.297  -4.755  1.00  0.00           H  \\nATOM   1044  HB3 ARG A  71      -7.090 -11.308  -4.445  1.00  0.00           H  \\nATOM   1045  HB2 ARG A  71      -5.797 -10.557  -5.367  1.00  0.00           H  \\nATOM   1046  HG3 ARG A  71      -5.043  -9.378  -3.327  1.00  0.00           H  \\nATOM   1047  HG2 ARG A  71      -6.324 -10.202  -2.406  1.00  0.00           H  \\nATOM   1048  HD3 ARG A  71      -4.206 -11.323  -2.145  1.00  0.00           H  \\nATOM   1049  HD2 ARG A  71      -5.363 -12.354  -3.019  1.00  0.00           H  \\nATOM   1050  HE  ARG A  71      -4.053 -12.219  -4.915  1.00  0.00           H  \\nATOM   1051 HH11 ARG A  71      -2.888  -9.956  -2.493  1.00  0.00           H  \\nATOM   1052 HH12 ARG A  71      -1.395  -9.623  -3.342  1.00  0.00           H  \\nATOM   1053 HH21 ARG A  71      -2.112 -11.778  -6.007  1.00  0.00           H  \\nATOM   1054 HH22 ARG A  71      -0.955 -10.653  -5.331  1.00  0.00           H  \\nATOM   1055  N   ASP A  72      -9.005  -9.905  -6.550  1.00  0.00           N  \\nATOM   1056  CA  ASP A  72      -9.632  -9.877  -7.874  1.00  0.00           C  \\nATOM   1057  C   ASP A  72     -10.218  -8.492  -8.172  1.00  0.00           C  \\nATOM   1058  O   ASP A  72      -9.948  -7.919  -9.230  1.00  0.00           O  \\nATOM   1059  CB  ASP A  72     -10.731 -10.938  -7.972  1.00  0.00           C  \\nATOM   1060  CG  ASP A  72     -10.180 -12.351  -8.107  1.00  0.00           C  \\nATOM   1061  OD1 ASP A  72      -9.049 -12.491  -8.508  1.00  0.00           O  \\nATOM   1062  OD2 ASP A  72     -10.896 -13.277  -7.808  1.00  0.00           O  \\nATOM   1063  H   ASP A  72      -9.471 -10.392  -5.798  1.00  0.00           H  \\nATOM   1064  HA  ASP A  72      -8.876 -10.075  -8.633  1.00  0.00           H  \\nATOM   1065  HB3 ASP A  72     -11.361 -10.893  -7.083  1.00  0.00           H  \\nATOM   1066  HB2 ASP A  72     -11.365 -10.727  -8.833  1.00  0.00           H  \\nATOM   1067  N   ARG A  73     -10.961  -7.923  -7.212  1.00  0.00           N  \\nATOM   1068  CA  ARG A  73     -11.616  -6.608  -7.339  1.00  0.00           C  \\nATOM   1069  C   ARG A  73     -10.638  -5.446  -7.505  1.00  0.00           C  \\nATOM   1070  O   ARG A  73     -10.949  -4.490  -8.210  1.00  0.00           O  \\nATOM   1071  CB  ARG A  73     -12.484  -6.337  -6.119  1.00  0.00           C  \\nATOM   1072  CG  ARG A  73     -13.739  -7.190  -6.025  1.00  0.00           C  \\nATOM   1073  CD  ARG A  73     -14.452  -6.980  -4.739  1.00  0.00           C  \\nATOM   1074  NE  ARG A  73     -15.646  -7.804  -4.639  1.00  0.00           N  \\nATOM   1075  CZ  ARG A  73     -16.339  -8.011  -3.503  1.00  0.00           C  \\nATOM   1076  NH1 ARG A  73     -15.947  -7.449  -2.381  1.00  0.00           N  \\nATOM   1077  NH2 ARG A  73     -17.415  -8.779  -3.516  1.00  0.00           N  \\nATOM   1078  H   ARG A  73     -11.124  -8.465  -6.374  1.00  0.00           H  \\nATOM   1079  HA  ARG A  73     -12.232  -6.603  -8.238  1.00  0.00           H  \\nATOM   1080  HB3 ARG A  73     -11.903  -6.506  -5.214  1.00  0.00           H  \\nATOM   1081  HB2 ARG A  73     -12.795  -5.292  -6.118  1.00  0.00           H  \\nATOM   1082  HG3 ARG A  73     -14.417  -6.931  -6.839  1.00  0.00           H  \\nATOM   1083  HG2 ARG A  73     -13.469  -8.244  -6.099  1.00  0.00           H  \\nATOM   1084  HD3 ARG A  73     -13.791  -7.236  -3.911  1.00  0.00           H  \\nATOM   1085  HD2 ARG A  73     -14.750  -5.936  -4.655  1.00  0.00           H  \\nATOM   1086  HE  ARG A  73     -15.979  -8.253  -5.482  1.00  0.00           H  \\nATOM   1087 HH11 ARG A  73     -15.124  -6.862  -2.371  1.00  0.00           H  \\nATOM   1088 HH12 ARG A  73     -16.467  -7.604  -1.530  1.00  0.00           H  \\nATOM   1089 HH21 ARG A  73     -17.717  -9.211  -4.378  1.00  0.00           H  \\nATOM   1090 HH22 ARG A  73     -17.935  -8.934  -2.665  1.00  0.00           H  \\nATOM   1091  N   SER A  74      -9.470  -5.507  -6.875  1.00  0.00           N  \\nATOM   1092  CA  SER A  74      -8.403  -4.509  -7.029  1.00  0.00           C  \\nATOM   1093  C   SER A  74      -7.611  -4.681  -8.328  1.00  0.00           C  \\nATOM   1094  O   SER A  74      -6.836  -3.808  -8.710  1.00  0.00           O  \\nATOM   1095  CB  SER A  74      -7.437  -4.620  -5.852  1.00  0.00           C  \\nATOM   1096  OG  SER A  74      -6.713  -5.826  -5.948  1.00  0.00           O  \\nATOM   1097  H   SER A  74      -9.281  -6.299  -6.278  1.00  0.00           H  \\nATOM   1098  HA  SER A  74      -8.835  -3.508  -7.028  1.00  0.00           H  \\nATOM   1099  HB3 SER A  74      -7.984  -4.592  -4.910  1.00  0.00           H  \\nATOM   1100  HB2 SER A  74      -6.741  -3.782  -5.870  1.00  0.00           H  \\nATOM   1101  HG  SER A  74      -6.194  -5.885  -5.142  1.00  0.00           H  \\nATOM   1102  N   GLY A  75      -7.756  -5.823  -9.005  1.00  0.00           N  \\nATOM   1103  CA  GLY A  75      -6.926  -6.200 -10.141  1.00  0.00           C  \\nATOM   1104  C   GLY A  75      -5.553  -6.767  -9.762  1.00  0.00           C  \\nATOM   1105  O   GLY A  75      -4.807  -7.146 -10.665  1.00  0.00           O  \\nATOM   1106  H   GLY A  75      -8.431  -6.496  -8.669  1.00  0.00           H  \\nATOM   1107  HA3 GLY A  75      -6.773  -5.328 -10.777  1.00  0.00           H  \\nATOM   1108  HA2 GLY A  75      -7.460  -6.957 -10.714  1.00  0.00           H  \\nATOM   1109  N   ALA A  76      -5.201  -6.899  -8.478  1.00  0.00           N  \\nATOM   1110  CA  ALA A  76      -3.892  -7.411  -8.058  1.00  0.00           C  \\nATOM   1111  C   ALA A  76      -3.616  -8.827  -8.601  1.00  0.00           C  \\nATOM   1112  O   ALA A  76      -2.510  -9.108  -9.073  1.00  0.00           O  \\nATOM   1113  CB  ALA A  76      -3.809  -7.364  -6.530  1.00  0.00           C  \\nATOM   1114  H   ALA A  76      -5.864  -6.648  -7.758  1.00  0.00           H  \\nATOM   1115  HA  ALA A  76      -3.120  -6.754  -8.459  1.00  0.00           H  \\nATOM   1116  HB1 ALA A  76      -2.846  -7.745  -6.192  1.00  0.00           H  \\nATOM   1117  HB2 ALA A  76      -3.912  -6.333  -6.190  1.00  0.00           H  \\nATOM   1118  HB3 ALA A  76      -4.605  -7.961  -6.086  1.00  0.00           H  \\nATOM   1119  N   ASN A  77      -4.634  -9.695  -8.652  1.00  0.00           N  \\nATOM   1120  CA  ASN A  77      -4.500 -11.059  -9.187  1.00  0.00           C  \\nATOM   1121  C   ASN A  77      -4.252 -11.110 -10.706  1.00  0.00           C  \\nATOM   1122  O   ASN A  77      -3.873 -12.155 -11.235  1.00  0.00           O  \\nATOM   1123  CB  ASN A  77      -5.737 -11.885  -8.790  1.00  0.00           C  \\nATOM   1124  CG  ASN A  77      -5.675 -12.418  -7.368  1.00  0.00           C  \\nATOM   1125  OD1 ASN A  77      -4.635 -12.478  -6.728  1.00  0.00           O  \\nATOM   1126  ND2 ASN A  77      -6.791 -12.843  -6.838  1.00  0.00           N  \\nATOM   1127  H   ASN A  77      -5.518  -9.410  -8.256  1.00  0.00           H  \\nATOM   1128  HA  ASN A  77      -3.621 -11.520  -8.737  1.00  0.00           H  \\nATOM   1129  HB3 ASN A  77      -5.823 -12.754  -9.442  1.00  0.00           H  \\nATOM   1130  HB2 ASN A  77      -6.644 -11.294  -8.909  1.00  0.00           H  \\nATOM   1131 HD21 ASN A  77      -7.654 -12.769  -7.359  1.00  0.00           H  \\nATOM   1132 HD22 ASN A  77      -6.793 -13.193  -5.891  1.00  0.00           H  \\nATOM   1133  N   LYS A  78      -4.376  -9.981 -11.418  1.00  0.00           N  \\nATOM   1134  CA  LYS A  78      -4.076  -9.902 -12.859  1.00  0.00           C  \\nATOM   1135  C   LYS A  78      -2.590  -9.677 -13.135  1.00  0.00           C  \\nATOM   1136  O   LYS A  78      -2.168  -9.760 -14.286  1.00  0.00           O  \\nATOM   1137  CB  LYS A  78      -4.894  -8.786 -13.511  1.00  0.00           C  \\nATOM   1138  CG  LYS A  78      -6.403  -8.955 -13.384  1.00  0.00           C  \\nATOM   1139  CD  LYS A  78      -6.882 -10.201 -14.115  1.00  0.00           C  \\nATOM   1140  CE  LYS A  78      -8.399 -10.316 -14.075  1.00  0.00           C  \\nATOM   1141  NZ  LYS A  78      -8.881 -11.553 -14.749  1.00  0.00           N  \\nATOM   1142  H   LYS A  78      -4.589  -9.126 -10.923  1.00  0.00           H  \\nATOM   1143  HA  LYS A  78      -4.324 -10.857 -13.321  1.00  0.00           H  \\nATOM   1144  HB3 LYS A  78      -4.626  -7.829 -13.063  1.00  0.00           H  \\nATOM   1145  HB2 LYS A  78      -4.652  -8.730 -14.573  1.00  0.00           H  \\nATOM   1146  HG3 LYS A  78      -6.672  -9.036 -12.330  1.00  0.00           H  \\nATOM   1147  HG2 LYS A  78      -6.903  -8.083 -13.803  1.00  0.00           H  \\nATOM   1148  HD3 LYS A  78      -6.556 -10.161 -15.155  1.00  0.00           H  \\nATOM   1149  HD2 LYS A  78      -6.448 -11.085 -13.650  1.00  0.00           H  \\nATOM   1150  HE3 LYS A  78      -8.736 -10.327 -13.040  1.00  0.00           H  \\nATOM   1151  HE2 LYS A  78      -8.843  -9.453 -14.570  1.00  0.00           H  \\nATOM   1152  HZ1 LYS A  78      -9.889 -11.592 -14.701  1.00  0.00           H  \\nATOM   1153  HZ2 LYS A  78      -8.589 -11.546 -15.716  1.00  0.00           H  \\nATOM   1154  HZ3 LYS A  78      -8.491 -12.362 -14.287  1.00  0.00           H  \\nATOM   1155  N   GLY A  79      -1.802  -9.397 -12.093  1.00  0.00           N  \\nATOM   1156  CA  GLY A  79      -0.392  -9.038 -12.195  1.00  0.00           C  \\nATOM   1157  C   GLY A  79      -0.144  -7.863 -13.144  1.00  0.00           C  \\nATOM   1158  O   GLY A  79       0.587  -8.053 -14.117  1.00  0.00           O  \\nATOM   1159  H   GLY A  79      -2.216  -9.421 -11.172  1.00  0.00           H  \\nATOM   1160  HA3 GLY A  79       0.172  -9.899 -12.557  1.00  0.00           H  \\nATOM   1161  HA2 GLY A  79      -0.024  -8.768 -11.205  1.00  0.00           H  \\nATOM   1162  N   PRO A  80      -0.713  -6.664 -12.891  1.00  0.00           N  \\nATOM   1163  CA  PRO A  80      -0.565  -5.508 -13.781  1.00  0.00           C  \\nATOM   1164  C   PRO A  80       0.892  -5.181 -14.116  1.00  0.00           C  \\nATOM   1165  O   PRO A  80       1.208  -4.793 -15.234  1.00  0.00           O  \\nATOM   1166  CB  PRO A  80      -1.213  -4.382 -12.969  1.00  0.00           C  \\nATOM   1167  CG  PRO A  80      -2.269  -5.068 -12.171  1.00  0.00           C  \\nATOM   1168  CD  PRO A  80      -1.638  -6.371 -11.758  1.00  0.00           C  \\nATOM   1169  HA  PRO A  80      -1.096  -5.705 -14.713  1.00  0.00           H  \\nATOM   1170  HB3 PRO A  80      -0.457  -3.887 -12.342  1.00  0.00           H  \\nATOM   1171  HB2 PRO A  80      -1.620  -3.616 -13.646  1.00  0.00           H  \\nATOM   1172  HG3 PRO A  80      -2.558  -4.446 -11.311  1.00  0.00           H  \\nATOM   1173  HG2 PRO A  80      -3.174  -5.209 -12.780  1.00  0.00           H  \\nATOM   1174  HD3 PRO A  80      -1.090  -6.231 -10.815  1.00  0.00           H  \\nATOM   1175  HD2 PRO A  80      -2.420  -7.137 -11.643  1.00  0.00           H  \\nATOM   1176  N   PHE A  81       1.801  -5.421 -13.172  1.00  0.00           N  \\nATOM   1177  CA  PHE A  81       3.216  -5.078 -13.290  1.00  0.00           C  \\nATOM   1178  C   PHE A  81       4.100  -6.222 -13.815  1.00  0.00           C  \\nATOM   1179  O   PHE A  81       5.300  -6.026 -13.992  1.00  0.00           O  \\nATOM   1180  CB  PHE A  81       3.687  -4.551 -11.929  1.00  0.00           C  \\nATOM   1181  CG  PHE A  81       2.851  -3.405 -11.399  1.00  0.00           C  \\nATOM   1182  CD1 PHE A  81       2.887  -2.151 -12.032  1.00  0.00           C  \\nATOM   1183  CD2 PHE A  81       2.017  -3.598 -10.284  1.00  0.00           C  \\nATOM   1184  CE1 PHE A  81       2.072  -1.103 -11.571  1.00  0.00           C  \\nATOM   1185  CE2 PHE A  81       1.214  -2.545  -9.812  1.00  0.00           C  \\nATOM   1186  CZ  PHE A  81       1.235  -1.299 -10.460  1.00  0.00           C  \\nATOM   1187  H   PHE A  81       1.467  -5.726 -12.269  1.00  0.00           H  \\nATOM   1188  HA  PHE A  81       3.316  -4.267 -14.012  1.00  0.00           H  \\nATOM   1189  HB3 PHE A  81       3.655  -5.369 -11.210  1.00  0.00           H  \\nATOM   1190  HB2 PHE A  81       4.721  -4.213 -12.006  1.00  0.00           H  \\nATOM   1191  HD1 PHE A  81       3.548  -1.994 -12.871  1.00  0.00           H  \\nATOM   1192  HD2 PHE A  81       2.004  -4.555  -9.782  1.00  0.00           H  \\nATOM   1193  HE1 PHE A  81       2.096  -0.140 -12.060  1.00  0.00           H  \\nATOM   1194  HE2 PHE A  81       0.588  -2.679  -8.942  1.00  0.00           H  \\nATOM   1195  HZ  PHE A  81       0.624  -0.488 -10.095  1.00  0.00           H  \\nATOM   1196  N   LYS A  82       3.541  -7.412 -14.086  1.00  0.00           N  \\nATOM   1197  CA  LYS A  82       4.293  -8.521 -14.706  1.00  0.00           C  \\nATOM   1198  C   LYS A  82       4.459  -8.326 -16.212  1.00  0.00           C  \\nATOM   1199  O   LYS A  82       5.407  -8.857 -16.778  1.00  0.00           O  \\nATOM   1200  CB  LYS A  82       3.600  -9.857 -14.433  1.00  0.00           C  \\nATOM   1201  CG  LYS A  82       3.619 -10.288 -12.973  1.00  0.00           C  \\nATOM   1202  CD  LYS A  82       2.910 -11.621 -12.781  1.00  0.00           C  \\nATOM   1203  CE  LYS A  82       2.901 -12.038 -11.318  1.00  0.00           C  \\nATOM   1204  NZ  LYS A  82       2.174 -13.320 -11.109  1.00  0.00           N  \\nATOM   1205  H   LYS A  82       2.540  -7.498 -13.998  1.00  0.00           H  \\nATOM   1206  HA  LYS A  82       5.304  -8.546 -14.300  1.00  0.00           H  \\nATOM   1207  HB3 LYS A  82       2.559  -9.798 -14.751  1.00  0.00           H  \\nATOM   1208  HB2 LYS A  82       4.078 -10.641 -15.020  1.00  0.00           H  \\nATOM   1209  HG3 LYS A  82       4.651 -10.383 -12.634  1.00  0.00           H  \\nATOM   1210  HG2 LYS A  82       3.124  -9.532 -12.364  1.00  0.00           H  \\nATOM   1211  HD3 LYS A  82       1.881 -11.540 -13.134  1.00  0.00           H  \\nATOM   1212  HD2 LYS A  82       3.417 -12.389 -13.364  1.00  0.00           H  \\nATOM   1213  HE3 LYS A  82       3.925 -12.153 -10.966  1.00  0.00           H  \\nATOM   1214  HE2 LYS A  82       2.421 -11.262 -10.721  1.00  0.00           H  \\nATOM   1215  HZ1 LYS A  82       2.190 -13.560 -10.128  1.00  0.00           H  \\nATOM   1216  HZ2 LYS A  82       1.216 -13.218 -11.415  1.00  0.00           H  \\nATOM   1217  HZ3 LYS A  82       2.622 -14.051 -11.642  1.00  0.00           H  \\nATOM   1218  N   ASP A  83       3.545  -7.578 -16.823  1.00  0.00           N  \\nATOM   1219  CA  ASP A  83       3.576  -7.198 -18.235  1.00  0.00           C  \\nATOM   1220  C   ASP A  83       2.919  -5.816 -18.405  1.00  0.00           C  \\nATOM   1221  O   ASP A  83       1.740  -5.719 -18.760  1.00  0.00           O  \\nATOM   1222  CB  ASP A  83       2.857  -8.239 -19.096  1.00  0.00           C  \\nATOM   1223  CG  ASP A  83       2.965  -7.951 -20.587  1.00  0.00           C  \\nATOM   1224  OD1 ASP A  83       3.018  -6.799 -20.948  1.00  0.00           O  \\nATOM   1225  OD2 ASP A  83       2.994  -8.886 -21.352  1.00  0.00           O  \\nATOM   1226  H   ASP A  83       2.803  -7.183 -16.264  1.00  0.00           H  \\nATOM   1227  HA  ASP A  83       4.608  -7.128 -18.580  1.00  0.00           H  \\nATOM   1228  HB3 ASP A  83       3.276  -9.226 -18.900  1.00  0.00           H  \\nATOM   1229  HB2 ASP A  83       1.802  -8.271 -18.823  1.00  0.00           H  \\nATOM   1230  N   PRO A  84       3.659  -4.733 -18.096  1.00  0.00           N  \\nATOM   1231  CA  PRO A  84       3.128  -3.378 -18.156  1.00  0.00           C  \\nATOM   1232  C   PRO A  84       2.463  -3.019 -19.491  1.00  0.00           C  \\nATOM   1233  O   PRO A  84       1.463  -2.298 -19.526  1.00  0.00           O  \\nATOM   1234  CB  PRO A  84       4.384  -2.534 -17.919  1.00  0.00           C  \\nATOM   1235  CG  PRO A  84       5.245  -3.393 -17.057  1.00  0.00           C  \\nATOM   1236  CD  PRO A  84       5.045  -4.784 -17.596  1.00  0.00           C  \\nATOM   1237  HA  PRO A  84       2.392  -3.302 -17.356  1.00  0.00           H  \\nATOM   1238  HB3 PRO A  84       4.857  -2.285 -18.880  1.00  0.00           H  \\nATOM   1239  HB2 PRO A  84       4.113  -1.583 -17.438  1.00  0.00           H  \\nATOM   1240  HG3 PRO A  84       6.292  -3.062 -17.115  1.00  0.00           H  \\nATOM   1241  HG2 PRO A  84       4.941  -3.300 -16.004  1.00  0.00           H  \\nATOM   1242  HD3 PRO A  84       5.766  -4.971 -18.406  1.00  0.00           H  \\nATOM   1243  HD2 PRO A  84       5.176  -5.515 -16.784  1.00  0.00           H  \\nATOM   1244  N   GLN A  85       2.996  -3.558 -20.590  1.00  0.00           N  \\nATOM   1245  CA  GLN A  85       2.570  -3.218 -21.946  1.00  0.00           C  \\nATOM   1246  C   GLN A  85       1.144  -3.688 -22.220  1.00  0.00           C  \\nATOM   1247  O   GLN A  85       0.328  -2.891 -22.682  1.00  0.00           O  \\nATOM   1248  CB  GLN A  85       3.541  -3.835 -22.961  1.00  0.00           C  \\nATOM   1249  CG  GLN A  85       4.923  -3.166 -22.919  1.00  0.00           C  \\nATOM   1250  CD  GLN A  85       5.906  -3.796 -23.900  1.00  0.00           C  \\nATOM   1251  OE1 GLN A  85       5.763  -4.911 -24.364  1.00  0.00           O  \\nATOM   1252  NE2 GLN A  85       6.966  -3.106 -24.259  1.00  0.00           N  \\nATOM   1253  H   GLN A  85       3.682  -4.292 -20.480  1.00  0.00           H  \\nATOM   1254  HA  GLN A  85       2.578  -2.135 -22.062  1.00  0.00           H  \\nATOM   1255  HB3 GLN A  85       3.640  -4.902 -22.766  1.00  0.00           H  \\nATOM   1256  HB2 GLN A  85       3.129  -3.705 -23.962  1.00  0.00           H  \\nATOM   1257  HG3 GLN A  85       4.811  -2.109 -23.162  1.00  0.00           H  \\nATOM   1258  HG2 GLN A  85       5.345  -3.247 -21.916  1.00  0.00           H  \\nATOM   1259 HE21 GLN A  85       7.118  -2.164 -23.928  1.00  0.00           H  \\nATOM   1260 HE22 GLN A  85       7.571  -3.569 -24.922  1.00  0.00           H  \\nATOM   1261  N   LYS A  86       0.809  -4.934 -21.856  1.00  0.00           N  \\nATOM   1262  CA  LYS A  86      -0.526  -5.513 -22.087  1.00  0.00           C  \\nATOM   1263  C   LYS A  86      -1.643  -4.743 -21.386  1.00  0.00           C  \\nATOM   1264  O   LYS A  86      -2.784  -4.767 -21.842  1.00  0.00           O  \\nATOM   1265  CB  LYS A  86      -0.552  -6.973 -21.633  1.00  0.00           C  \\nATOM   1266  CG  LYS A  86       0.123  -7.943 -22.595  1.00  0.00           C  \\nATOM   1267  CD  LYS A  86      -0.018  -9.381 -22.119  1.00  0.00           C  \\nATOM   1268  CE  LYS A  86       0.623 -10.354 -23.098  1.00  0.00           C  \\nATOM   1269  NZ  LYS A  86       2.093 -10.151 -23.202  1.00  0.00           N  \\nATOM   1270  H   LYS A  86       1.515  -5.517 -21.428  1.00  0.00           H  \\nATOM   1271  HA  LYS A  86      -0.747  -5.484 -23.154  1.00  0.00           H  \\nATOM   1272  HB3 LYS A  86      -0.056  -7.063 -20.666  1.00  0.00           H  \\nATOM   1273  HB2 LYS A  86      -1.585  -7.296 -21.503  1.00  0.00           H  \\nATOM   1274  HG3 LYS A  86      -0.330  -7.850 -23.583  1.00  0.00           H  \\nATOM   1275  HG2 LYS A  86       1.181  -7.698 -22.675  1.00  0.00           H  \\nATOM   1276  HD3 LYS A  86       0.460  -9.490 -21.145  1.00  0.00           H  \\nATOM   1277  HD2 LYS A  86      -1.075  -9.628 -22.015  1.00  0.00           H  \\nATOM   1278  HE3 LYS A  86       0.432 -11.375 -22.772  1.00  0.00           H  \\nATOM   1279  HE2 LYS A  86       0.180 -10.222 -24.085  1.00  0.00           H  \\nATOM   1280  HZ1 LYS A  86       2.479 -10.813 -23.860  1.00  0.00           H  \\nATOM   1281  HZ2 LYS A  86       2.282  -9.211 -23.522  1.00  0.00           H  \\nATOM   1282  HZ3 LYS A  86       2.518 -10.290 -22.296  1.00  0.00           H  \\nATOM   1283  N   TRP A  87      -1.326  -4.091 -20.273  1.00  0.00           N  \\nATOM   1284  CA  TRP A  87      -2.296  -3.354 -19.465  1.00  0.00           C  \\nATOM   1285  C   TRP A  87      -2.250  -1.838 -19.670  1.00  0.00           C  \\nATOM   1286  O   TRP A  87      -2.998  -1.121 -19.005  1.00  0.00           O  \\nATOM   1287  CB  TRP A  87      -2.069  -3.662 -17.984  1.00  0.00           C  \\nATOM   1288  CG  TRP A  87      -2.309  -5.099 -17.628  1.00  0.00           C  \\nATOM   1289  CD1 TRP A  87      -1.372  -6.012 -17.249  1.00  0.00           C  \\nATOM   1290  CD2 TRP A  87      -3.578  -5.797 -17.616  1.00  0.00           C  \\nATOM   1291  NE1 TRP A  87      -1.966  -7.226 -17.005  1.00  0.00           N  \\nATOM   1292  CE2 TRP A  87      -3.315  -7.112 -17.223  1.00  0.00           C  \\nATOM   1293  CE3 TRP A  87      -4.895  -5.417 -17.903  1.00  0.00           C  \\nATOM   1294  CZ2 TRP A  87      -4.322  -8.058 -17.108  1.00  0.00           C  \\nATOM   1295  CZ3 TRP A  87      -5.904  -6.366 -17.787  1.00  0.00           C  \\nATOM   1296  CH2 TRP A  87      -5.624  -7.652 -17.399  1.00  0.00           C  \\nATOM   1297  H   TRP A  87      -0.374  -4.159 -19.944  1.00  0.00           H  \\nATOM   1298  HA  TRP A  87      -3.301  -3.650 -19.763  1.00  0.00           H  \\nATOM   1299  HB3 TRP A  87      -1.045  -3.409 -17.712  1.00  0.00           H  \\nATOM   1300  HB2 TRP A  87      -2.732  -3.044 -17.378  1.00  0.00           H  \\nATOM   1301  HD1 TRP A  87      -0.307  -5.809 -17.154  1.00  0.00           H  \\nATOM   1302  HE1 TRP A  87      -1.488  -8.066 -16.712  1.00  0.00           H  \\nATOM   1303  HE3 TRP A  87      -5.123  -4.397 -18.209  1.00  0.00           H  \\nATOM   1304  HZ2 TRP A  87      -4.119  -9.085 -16.802  1.00  0.00           H  \\nATOM   1305  HZ3 TRP A  87      -6.927  -6.062 -18.013  1.00  0.00           H  \\nATOM   1306  HH2 TRP A  87      -6.440  -8.371 -17.318  1.00  0.00           H  \\nATOM   1307  N   GLY A  88      -1.390  -1.340 -20.565  1.00  0.00           N  \\nATOM   1308  CA  GLY A  88      -1.339   0.080 -20.918  1.00  0.00           C  \\nATOM   1309  C   GLY A  88      -0.763   0.961 -19.806  1.00  0.00           C  \\nATOM   1310  O   GLY A  88      -1.060   2.153 -19.747  1.00  0.00           O  \\nATOM   1311  H   GLY A  88      -0.727  -1.964 -21.004  1.00  0.00           H  \\nATOM   1312  HA3 GLY A  88      -2.344   0.433 -21.150  1.00  0.00           H  \\nATOM   1313  HA2 GLY A  88      -0.712   0.198 -21.802  1.00  0.00           H  \\nATOM   1314  N   ILE A  89       0.038   0.385 -18.906  1.00  0.00           N  \\nATOM   1315  CA  ILE A  89       0.639   1.120 -17.797  1.00  0.00           C  \\nATOM   1316  C   ILE A  89       2.125   1.348 -18.086  1.00  0.00           C  \\nATOM   1317  O   ILE A  89       2.762   0.630 -18.861  1.00  0.00           O  \\nATOM   1318  CB  ILE A  89       0.469   0.364 -16.467  1.00  0.00           C  \\nATOM   1319  CG1 ILE A  89       1.267  -0.942 -16.490  1.00  0.00           C  \\nATOM   1320  CG2 ILE A  89      -1.002   0.089 -16.195  1.00  0.00           C  \\nATOM   1321  CD1 ILE A  89       1.383  -1.611 -15.139  1.00  0.00           C  \\nATOM   1322  H   ILE A  89       0.395  -0.537 -19.113  1.00  0.00           H  \\nATOM   1323  HA  ILE A  89       0.178   2.106 -17.755  1.00  0.00           H  \\nATOM   1324  HB  ILE A  89       0.873   0.966 -15.653  1.00  0.00           H  \\nATOM   1325 HG13 ILE A  89       0.796  -1.644 -17.178  1.00  0.00           H  \\nATOM   1326 HG12 ILE A  89       2.274  -0.748 -16.860  1.00  0.00           H  \\nATOM   1327 HG21 ILE A  89      -1.103  -0.446 -15.251  1.00  0.00           H  \\nATOM   1328 HG22 ILE A  89      -1.543   1.033 -16.137  1.00  0.00           H  \\nATOM   1329 HG23 ILE A  89      -1.414  -0.517 -17.002  1.00  0.00           H  \\nATOM   1330 HD11 ILE A  89       1.962  -2.529 -15.236  1.00  0.00           H  \\nATOM   1331 HD12 ILE A  89       1.884  -0.938 -14.442  1.00  0.00           H  \\nATOM   1332 HD13 ILE A  89       0.388  -1.847 -14.763  1.00  0.00           H  \\nATOM   1333  N   LYS A  90       2.698   2.376 -17.466  1.00  0.00           N  \\nATOM   1334  CA  LYS A  90       4.059   2.803 -17.786  1.00  0.00           C  \\nATOM   1335  C   LYS A  90       5.081   1.735 -17.381  1.00  0.00           C  \\nATOM   1336  O   LYS A  90       4.807   0.892 -16.526  1.00  0.00           O  \\nATOM   1337  CB  LYS A  90       4.379   4.130 -17.096  1.00  0.00           C  \\nATOM   1338  CG  LYS A  90       4.469   4.043 -15.578  1.00  0.00           C  \\nATOM   1339  CD  LYS A  90       4.843   5.387 -14.969  1.00  0.00           C  \\nATOM   1340  CE  LYS A  90       3.744   6.417 -15.183  1.00  0.00           C  \\nATOM   1341  NZ  LYS A  90       4.031   7.693 -14.472  1.00  0.00           N  \\nATOM   1342  H   LYS A  90       2.204   2.821 -16.705  1.00  0.00           H  \\nATOM   1343  HA  LYS A  90       4.118   2.898 -18.869  1.00  0.00           H  \\nATOM   1344  HB3 LYS A  90       5.330   4.512 -17.466  1.00  0.00           H  \\nATOM   1345  HB2 LYS A  90       3.612   4.863 -17.344  1.00  0.00           H  \\nATOM   1346  HG3 LYS A  90       3.507   3.726 -15.173  1.00  0.00           H  \\nATOM   1347  HG2 LYS A  90       5.221   3.306 -15.300  1.00  0.00           H  \\nATOM   1348  HD3 LYS A  90       5.014   5.266 -13.899  1.00  0.00           H  \\nATOM   1349  HD2 LYS A  90       5.762   5.750 -15.428  1.00  0.00           H  \\nATOM   1350  HE3 LYS A  90       3.641   6.624 -16.247  1.00  0.00           H  \\nATOM   1351  HE2 LYS A  90       2.796   6.019 -14.820  1.00  0.00           H  \\nATOM   1352  HZ1 LYS A  90       3.280   8.348 -14.639  1.00  0.00           H  \\nATOM   1353  HZ2 LYS A  90       4.110   7.516 -13.481  1.00  0.00           H  \\nATOM   1354  HZ3 LYS A  90       4.898   8.081 -14.815  1.00  0.00           H  \\nATOM   1355  N   ALA A  91       6.273   1.806 -17.973  1.00  0.00           N  \\nATOM   1356  CA  ALA A  91       7.367   0.887 -17.667  1.00  0.00           C  \\nATOM   1357  C   ALA A  91       7.739   0.925 -16.176  1.00  0.00           C  \\nATOM   1358  O   ALA A  91       7.567   1.954 -15.516  1.00  0.00           O  \\nATOM   1359  CB  ALA A  91       8.572   1.232 -18.550  1.00  0.00           C  \\nATOM   1360  H   ALA A  91       6.434   2.536 -18.652  1.00  0.00           H  \\nATOM   1361  HA  ALA A  91       7.040  -0.123 -17.909  1.00  0.00           H  \\nATOM   1362  HB1 ALA A  91       8.298   1.163 -19.602  1.00  0.00           H  \\nATOM   1363  HB2 ALA A  91       8.918   2.243 -18.330  1.00  0.00           H  \\nATOM   1364  HB3 ALA A  91       9.385   0.533 -18.351  1.00  0.00           H  \\nATOM   1365  N   LEU A  92       8.244  -0.196 -15.656  1.00  0.00           N  \\nATOM   1366  CA  LEU A  92       8.487  -0.380 -14.224  1.00  0.00           C  \\nATOM   1367  C   LEU A  92       9.609   0.539 -13.717  1.00  0.00           C  \\nATOM   1368  O   LEU A  92      10.442   0.986 -14.511  1.00  0.00           O  \\nATOM   1369  CB  LEU A  92       8.846  -1.843 -13.936  1.00  0.00           C  \\nATOM   1370  CG  LEU A  92       7.770  -2.875 -14.295  1.00  0.00           C  \\nATOM   1371  CD1 LEU A  92       8.278  -4.273 -13.969  1.00  0.00           C  \\nATOM   1372  CD2 LEU A  92       6.493  -2.566 -13.528  1.00  0.00           C  \\nATOM   1373  H   LEU A  92       8.512  -0.943 -16.281  1.00  0.00           H  \\nATOM   1374  HA  LEU A  92       7.575  -0.103 -13.694  1.00  0.00           H  \\nATOM   1375  HB3 LEU A  92       9.747  -2.095 -14.493  1.00  0.00           H  \\nATOM   1376  HB2 LEU A  92       9.062  -1.945 -12.872  1.00  0.00           H  \\nATOM   1377  HG  LEU A  92       7.569  -2.834 -15.365  1.00  0.00           H  \\nATOM   1378 HD11 LEU A  92       7.514  -5.007 -14.225  1.00  0.00           H  \\nATOM   1379 HD12 LEU A  92       9.181  -4.476 -14.546  1.00  0.00           H  \\nATOM   1380 HD13 LEU A  92       8.504  -4.340 -12.906  1.00  0.00           H  \\nATOM   1381 HD21 LEU A  92       5.728  -3.299 -13.783  1.00  0.00           H  \\nATOM   1382 HD22 LEU A  92       6.693  -2.608 -12.457  1.00  0.00           H  \\nATOM   1383 HD23 LEU A  92       6.142  -1.568 -13.793  1.00  0.00           H  \\nATOM   1384  N   PRO A  93       9.643   0.832 -12.406  1.00  0.00           N  \\nATOM   1385  CA  PRO A  93      10.688   1.679 -11.855  1.00  0.00           C  \\nATOM   1386  C   PRO A  93      12.079   1.087 -12.134  1.00  0.00           C  \\nATOM   1387  O   PRO A  93      12.230  -0.140 -12.191  1.00  0.00           O  \\nATOM   1388  CB  PRO A  93      10.361   1.690 -10.358  1.00  0.00           C  \\nATOM   1389  CG  PRO A  93       8.900   1.404 -10.300  1.00  0.00           C  \\nATOM   1390  CD  PRO A  93       8.668   0.415 -11.411  1.00  0.00           C  \\nATOM   1391  HA  PRO A  93      10.622   2.665 -12.315  1.00  0.00           H  \\nATOM   1392  HB3 PRO A  93      10.965   0.934  -9.835  1.00  0.00           H  \\nATOM   1393  HB2 PRO A  93      10.623   2.666  -9.923  1.00  0.00           H  \\nATOM   1394  HG3 PRO A  93       8.630   1.002  -9.312  1.00  0.00           H  \\nATOM   1395  HG2 PRO A  93       8.325   2.333 -10.433  1.00  0.00           H  \\nATOM   1396  HD3 PRO A  93       8.863  -0.603 -11.044  1.00  0.00           H  \\nATOM   1397  HD2 PRO A  93       7.633   0.506 -11.773  1.00  0.00           H  \\nATOM   1398  N   PRO A  94      13.105   1.941 -12.298  1.00  0.00           N  \\nATOM   1399  CA  PRO A  94      14.481   1.476 -12.379  1.00  0.00           C  \\nATOM   1400  C   PRO A  94      14.855   0.712 -11.107  1.00  0.00           C  \\nATOM   1401  O   PRO A  94      14.376   1.019 -10.015  1.00  0.00           O  \\nATOM   1402  CB  PRO A  94      15.281   2.775 -12.522  1.00  0.00           C  \\nATOM   1403  CG  PRO A  94      14.290   3.758 -13.047  1.00  0.00           C  \\nATOM   1404  CD  PRO A  94      13.002   3.396 -12.357  1.00  0.00           C  \\nATOM   1405  HA  PRO A  94      14.602   0.823 -13.243  1.00  0.00           H  \\nATOM   1406  HB3 PRO A  94      15.698   3.069 -11.548  1.00  0.00           H  \\nATOM   1407  HB2 PRO A  94      16.131   2.621 -13.203  1.00  0.00           H  \\nATOM   1408  HG3 PRO A  94      14.617   4.784 -12.822  1.00  0.00           H  \\nATOM   1409  HG2 PRO A  94      14.221   3.680 -14.142  1.00  0.00           H  \\nATOM   1410  HD3 PRO A  94      12.976   3.859 -11.360  1.00  0.00           H  \\nATOM   1411  HD2 PRO A  94      12.151   3.737 -12.966  1.00  0.00           H  \\nATOM   1412  N   LYS A  95      15.741  -0.275 -11.248  1.00  0.00           N  \\nATOM   1413  CA  LYS A  95      16.263  -1.037 -10.109  1.00  0.00           C  \\nATOM   1414  C   LYS A  95      17.185  -0.163  -9.267  1.00  0.00           C  \\nATOM   1415  O   LYS A  95      17.894   0.680  -9.809  1.00  0.00           O  \\nATOM   1416  CB  LYS A  95      17.010  -2.289 -10.576  1.00  0.00           C  \\nATOM   1417  CG  LYS A  95      16.059  -3.316 -11.194  1.00  0.00           C  \\nATOM   1418  CD  LYS A  95      16.831  -4.580 -11.577  1.00  0.00           C  \\nATOM   1419  CE  LYS A  95      15.798  -5.550 -12.122  1.00  0.00           C  \\nATOM   1420  NZ  LYS A  95      16.329  -6.873 -12.518  1.00  0.00           N  \\nATOM   1421  H   LYS A  95      16.161  -0.409 -12.157  1.00  0.00           H  \\nATOM   1422  HA  LYS A  95      15.434  -1.355  -9.477  1.00  0.00           H  \\nATOM   1423  HB3 LYS A  95      17.506  -2.744  -9.718  1.00  0.00           H  \\nATOM   1424  HB2 LYS A  95      17.771  -2.006 -11.303  1.00  0.00           H  \\nATOM   1425  HG3 LYS A  95      15.280  -3.568 -10.474  1.00  0.00           H  \\nATOM   1426  HG2 LYS A  95      15.596  -2.895 -12.087  1.00  0.00           H  \\nATOM   1427  HD3 LYS A  95      17.573  -4.354 -12.343  1.00  0.00           H  \\nATOM   1428  HD2 LYS A  95      17.318  -4.991 -10.692  1.00  0.00           H  \\nATOM   1429  HE3 LYS A  95      15.289  -5.094 -12.972  1.00  0.00           H  \\nATOM   1430  HE2 LYS A  95      15.041  -5.698 -11.352  1.00  0.00           H  \\nATOM   1431  HZ1 LYS A  95      16.932  -6.799 -13.325  1.00  0.00           H  \\nATOM   1432  HZ2 LYS A  95      15.540  -7.452 -12.764  1.00  0.00           H  \\nATOM   1433  HZ3 LYS A  95      16.819  -7.298 -11.744  1.00  0.00           H  \\nATOM   1434  N   ASN A  96      17.260  -0.464  -7.976  1.00  0.00           N  \\nATOM   1435  CA  ASN A  96      18.326   0.030  -7.113  1.00  0.00           C  \\nATOM   1436  C   ASN A  96      19.061  -1.170  -6.493  1.00  0.00           C  \\nATOM   1437  O   ASN A  96      18.666  -1.610  -5.411  1.00  0.00           O  \\nATOM   1438  CB  ASN A  96      17.760   0.990  -6.055  1.00  0.00           C  \\nATOM   1439  CG  ASN A  96      18.879   1.552  -5.192  1.00  0.00           C  \\nATOM   1440  OD1 ASN A  96      20.056   1.325  -5.426  1.00  0.00           O  \\nATOM   1441  ND2 ASN A  96      18.579   2.295  -4.164  1.00  0.00           N  \\nATOM   1442  H   ASN A  96      16.672  -1.202  -7.615  1.00  0.00           H  \\nATOM   1443  HA  ASN A  96      19.041   0.607  -7.697  1.00  0.00           H  \\nATOM   1444  HB3 ASN A  96      17.262   1.823  -6.552  1.00  0.00           H  \\nATOM   1445  HB2 ASN A  96      17.040   0.478  -5.418  1.00  0.00           H  \\nATOM   1446 HD21 ASN A  96      19.349   2.645  -3.610  1.00  0.00           H  \\nATOM   1447 HD22 ASN A  96      17.640   2.356  -3.796  1.00  0.00           H  \\nATOM   1448  N   PRO A  97      20.111  -1.712  -7.143  1.00  0.00           N  \\nATOM   1449  CA  PRO A  97      20.819  -2.902  -6.668  1.00  0.00           C  \\nATOM   1450  C   PRO A  97      21.322  -2.805  -5.223  1.00  0.00           C  \\nATOM   1451  O   PRO A  97      21.407  -3.830  -4.555  1.00  0.00           O  \\nATOM   1452  CB  PRO A  97      21.988  -2.995  -7.653  1.00  0.00           C  \\nATOM   1453  CG  PRO A  97      21.448  -2.421  -8.918  1.00  0.00           C  \\nATOM   1454  CD  PRO A  97      20.589  -1.269  -8.468  1.00  0.00           C  \\nATOM   1455  HA  PRO A  97      20.147  -3.757  -6.742  1.00  0.00           H  \\nATOM   1456  HB3 PRO A  97      22.852  -2.436  -7.265  1.00  0.00           H  \\nATOM   1457  HB2 PRO A  97      22.306  -4.043  -7.762  1.00  0.00           H  \\nATOM   1458  HG3 PRO A  97      22.272  -2.104  -9.573  1.00  0.00           H  \\nATOM   1459  HG2 PRO A  97      20.879  -3.183  -9.469  1.00  0.00           H  \\nATOM   1460  HD3 PRO A  97      21.203  -0.359  -8.394  1.00  0.00           H  \\nATOM   1461  HD2 PRO A  97      19.766  -1.124  -9.183  1.00  0.00           H  \\nATOM   1462  N   SER A  98      21.592  -1.593  -4.724  1.00  0.00           N  \\nATOM   1463  CA  SER A  98      22.043  -1.367  -3.344  1.00  0.00           C  \\nATOM   1464  C   SER A  98      20.947  -1.622  -2.302  1.00  0.00           C  \\nATOM   1465  O   SER A  98      21.248  -1.768  -1.123  1.00  0.00           O  \\nATOM   1466  CB  SER A  98      22.551   0.054  -3.200  1.00  0.00           C  \\nATOM   1467  OG  SER A  98      21.495   0.973  -3.254  1.00  0.00           O  \\nATOM   1468  H   SER A  98      21.493  -0.776  -5.311  1.00  0.00           H  \\nATOM   1469  HA  SER A  98      22.868  -2.046  -3.125  1.00  0.00           H  \\nATOM   1470  HB3 SER A  98      23.079   0.158  -2.252  1.00  0.00           H  \\nATOM   1471  HB2 SER A  98      23.263   0.268  -3.996  1.00  0.00           H  \\nATOM   1472  HG  SER A  98      21.239   1.029  -4.178  1.00  0.00           H  \\nATOM   1473  N   TRP A  99      19.681  -1.681  -2.725  1.00  0.00           N  \\nATOM   1474  CA  TRP A  99      18.532  -1.976  -1.867  1.00  0.00           C  \\nATOM   1475  C   TRP A  99      17.848  -3.305  -2.225  1.00  0.00           C  \\nATOM   1476  O   TRP A  99      17.452  -4.074  -1.345  1.00  0.00           O  \\nATOM   1477  CB  TRP A  99      17.563  -0.795  -1.955  1.00  0.00           C  \\nATOM   1478  CG  TRP A  99      16.372  -0.911  -1.064  1.00  0.00           C  \\nATOM   1479  CD1 TRP A  99      16.340  -0.600   0.249  1.00  0.00           C  \\nATOM   1480  CD2 TRP A  99      15.037  -1.385  -1.392  1.00  0.00           C  \\nATOM   1481  NE1 TRP A  99      15.076  -0.828   0.754  1.00  0.00           N  \\nATOM   1482  CE2 TRP A  99      14.229  -1.312  -0.217  1.00  0.00           C  \\nATOM   1483  CE3 TRP A  99      14.435  -1.869  -2.569  1.00  0.00           C  \\nATOM   1484  CZ2 TRP A  99      12.875  -1.672  -0.216  1.00  0.00           C  \\nATOM   1485  CZ3 TRP A  99      13.087  -2.256  -2.566  1.00  0.00           C  \\nATOM   1486  CH2 TRP A  99      12.303  -2.138  -1.409  1.00  0.00           C  \\nATOM   1487  H   TRP A  99      19.510  -1.529  -3.709  1.00  0.00           H  \\nATOM   1488  HA  TRP A  99      18.858  -2.061  -0.830  1.00  0.00           H  \\nATOM   1489  HB3 TRP A  99      17.222  -0.682  -2.985  1.00  0.00           H  \\nATOM   1490  HB2 TRP A  99      18.100   0.115  -1.686  1.00  0.00           H  \\nATOM   1491  HD1 TRP A  99      17.176  -0.207   0.808  1.00  0.00           H  \\nATOM   1492  HE1 TRP A  99      14.822  -0.600   1.704  1.00  0.00           H  \\nATOM   1493  HE3 TRP A  99      15.024  -1.933  -3.472  1.00  0.00           H  \\nATOM   1494  HZ2 TRP A  99      12.282  -1.574   0.683  1.00  0.00           H  \\nATOM   1495  HZ3 TRP A  99      12.648  -2.649  -3.471  1.00  0.00           H  \\nATOM   1496  HH2 TRP A  99      11.255  -2.397  -1.444  1.00  0.00           H  \\nATOM   1497  N   SER A 100      17.696  -3.600  -3.520  1.00  0.00           N  \\nATOM   1498  CA  SER A 100      17.189  -4.879  -4.023  1.00  0.00           C  \\nATOM   1499  C   SER A 100      17.529  -5.084  -5.498  1.00  0.00           C  \\nATOM   1500  O   SER A 100      17.288  -4.217  -6.338  1.00  0.00           O  \\nATOM   1501  CB  SER A 100      15.687  -4.953  -3.831  1.00  0.00           C  \\nATOM   1502  OG  SER A 100      15.175  -6.159  -4.328  1.00  0.00           O  \\nATOM   1503  H   SER A 100      18.028  -2.930  -4.199  1.00  0.00           H  \\nATOM   1504  HA  SER A 100      17.633  -5.684  -3.436  1.00  0.00           H  \\nATOM   1505  HB3 SER A 100      15.451  -4.864  -2.771  1.00  0.00           H  \\nATOM   1506  HB2 SER A 100      15.213  -4.115  -4.343  1.00  0.00           H  \\nATOM   1507  HG  SER A 100      14.480  -6.422  -3.721  1.00  0.00           H  \\nATOM   1508  N   ALA A 101      17.996  -6.286  -5.842  1.00  0.00           N  \\nATOM   1509  CA  ALA A 101      18.214  -6.696  -7.232  1.00  0.00           C  \\nATOM   1510  C   ALA A 101      16.934  -7.223  -7.928  1.00  0.00           C  \\nATOM   1511  O   ALA A 101      16.959  -7.568  -9.115  1.00  0.00           O  \\nATOM   1512  CB  ALA A 101      19.340  -7.735  -7.240  1.00  0.00           C  \\nATOM   1513  H   ALA A 101      18.202  -6.947  -5.108  1.00  0.00           H  \\nATOM   1514  HA  ALA A 101      18.553  -5.830  -7.802  1.00  0.00           H  \\nATOM   1515  HB1 ALA A 101      19.572  -8.018  -8.267  1.00  0.00           H  \\nATOM   1516  HB2 ALA A 101      19.040  -8.624  -6.685  1.00  0.00           H  \\nATOM   1517  HB3 ALA A 101      20.237  -7.312  -6.786  1.00  0.00           H  \\nATOM   1518  N   GLN A 102      15.813  -7.322  -7.202  1.00  0.00           N  \\nATOM   1519  CA  GLN A 102      14.568  -7.892  -7.720  1.00  0.00           C  \\nATOM   1520  C   GLN A 102      13.799  -6.910  -8.612  1.00  0.00           C  \\nATOM   1521  O   GLN A 102      13.778  -5.708  -8.368  1.00  0.00           O  \\nATOM   1522  CB  GLN A 102      13.675  -8.344  -6.562  1.00  0.00           C  \\nATOM   1523  CG  GLN A 102      14.239  -9.502  -5.757  1.00  0.00           C  \\nATOM   1524  CD  GLN A 102      13.233 -10.066  -4.771  1.00  0.00           C  \\nATOM   1525  OE1 GLN A 102      12.301  -9.375  -4.351  1.00  0.00           O  \\nATOM   1526  NE2 GLN A 102      13.416 -11.327  -4.397  1.00  0.00           N  \\nATOM   1527  H   GLN A 102      15.810  -6.935  -6.269  1.00  0.00           H  \\nATOM   1528  HA  GLN A 102      14.817  -8.761  -8.329  1.00  0.00           H  \\nATOM   1529  HB3 GLN A 102      13.512  -7.509  -5.881  1.00  0.00           H  \\nATOM   1530  HB2 GLN A 102      12.702  -8.645  -6.949  1.00  0.00           H  \\nATOM   1531  HG3 GLN A 102      14.530 -10.299  -6.442  1.00  0.00           H  \\nATOM   1532  HG2 GLN A 102      15.107  -9.154  -5.198  1.00  0.00           H  \\nATOM   1533 HE21 GLN A 102      12.784 -11.753  -3.748  1.00  0.00           H  \\nATOM   1534 HE22 GLN A 102      14.185 -11.851  -4.762  1.00  0.00           H  \\nATOM   1535  N   ASP A 103      13.107  -7.447  -9.618  1.00  0.00           N  \\nATOM   1536  CA  ASP A 103      12.131  -6.695 -10.410  1.00  0.00           C  \\nATOM   1537  C   ASP A 103      10.942  -6.221  -9.568  1.00  0.00           C  \\nATOM   1538  O   ASP A 103      10.395  -6.964  -8.746  1.00  0.00           O  \\nATOM   1539  CB  ASP A 103      11.622  -7.551 -11.573  1.00  0.00           C  \\nATOM   1540  CG  ASP A 103      12.728  -7.956 -12.539  1.00  0.00           C  \\nATOM   1541  OD1 ASP A 103      13.820  -7.457 -12.403  1.00  0.00           O  \\nATOM   1542  OD2 ASP A 103      12.469  -8.759 -13.403  1.00  0.00           O  \\nATOM   1543  H   ASP A 103      13.194  -8.435  -9.804  1.00  0.00           H  \\nATOM   1544  HA  ASP A 103      12.618  -5.813 -10.827  1.00  0.00           H  \\nATOM   1545  HB3 ASP A 103      11.152  -8.453 -11.183  1.00  0.00           H  \\nATOM   1546  HB2 ASP A 103      10.861  -6.998 -12.126  1.00  0.00           H  \\nATOM   1547  N   PHE A 104      10.480  -5.005  -9.853  1.00  0.00           N  \\nATOM   1548  CA  PHE A 104       9.309  -4.389  -9.233  1.00  0.00           C  \\nATOM   1549  C   PHE A 104       8.010  -4.803  -9.940  1.00  0.00           C  \\nATOM   1550  O   PHE A 104       7.320  -3.987 -10.538  1.00  0.00           O  \\nATOM   1551  CB  PHE A 104       9.443  -2.865  -9.250  1.00  0.00           C  \\nATOM   1552  CG  PHE A 104      10.578  -2.347  -8.413  1.00  0.00           C  \\nATOM   1553  CD1 PHE A 104      11.656  -1.704  -9.003  1.00  0.00           C  \\nATOM   1554  CD2 PHE A 104      10.570  -2.502  -7.036  1.00  0.00           C  \\nATOM   1555  CE1 PHE A 104      12.701  -1.227  -8.234  1.00  0.00           C  \\nATOM   1556  CE2 PHE A 104      11.612  -2.025  -6.264  1.00  0.00           C  \\nATOM   1557  CZ  PHE A 104      12.679  -1.387  -6.865  1.00  0.00           C  \\nATOM   1558  H   PHE A 104      10.981  -4.449 -10.530  1.00  0.00           H  \\nATOM   1559  HA  PHE A 104       9.246  -4.742  -8.204  1.00  0.00           H  \\nATOM   1560  HB3 PHE A 104       9.592  -2.525 -10.274  1.00  0.00           H  \\nATOM   1561  HB2 PHE A 104       8.520  -2.415  -8.888  1.00  0.00           H  \\nATOM   1562  HD1 PHE A 104      11.673  -1.577 -10.086  1.00  0.00           H  \\nATOM   1563  HD2 PHE A 104       9.727  -3.006  -6.561  1.00  0.00           H  \\nATOM   1564  HE1 PHE A 104      13.542  -0.724  -8.710  1.00  0.00           H  \\nATOM   1565  HE2 PHE A 104      11.593  -2.153  -5.182  1.00  0.00           H  \\nATOM   1566  HZ  PHE A 104      13.502  -1.013  -6.258  1.00  0.00           H  \\nATOM   1567  N   LYS A 105       7.708  -6.103  -9.939  1.00  0.00           N  \\nATOM   1568  CA  LYS A 105       6.587  -6.690 -10.701  1.00  0.00           C  \\nATOM   1569  C   LYS A 105       5.399  -7.144  -9.846  1.00  0.00           C  \\nATOM   1570  O   LYS A 105       4.397  -7.616 -10.385  1.00  0.00           O  \\nATOM   1571  CB  LYS A 105       7.091  -7.875 -11.525  1.00  0.00           C  \\nATOM   1572  CG  LYS A 105       7.985  -7.491 -12.698  1.00  0.00           C  \\nATOM   1573  CD  LYS A 105       8.659  -8.713 -13.302  1.00  0.00           C  \\nATOM   1574  CE  LYS A 105       9.398  -8.362 -14.585  1.00  0.00           C  \\nATOM   1575  NZ  LYS A 105      10.373  -9.416 -14.972  1.00  0.00           N  \\nATOM   1576  H   LYS A 105       8.334  -6.707  -9.428  1.00  0.00           H  \\nATOM   1577  HA  LYS A 105       6.183  -5.921 -11.360  1.00  0.00           H  \\nATOM   1578  HB3 LYS A 105       7.655  -8.552 -10.883  1.00  0.00           H  \\nATOM   1579  HB2 LYS A 105       6.241  -8.431 -11.921  1.00  0.00           H  \\nATOM   1580  HG3 LYS A 105       7.386  -7.000 -13.466  1.00  0.00           H  \\nATOM   1581  HG2 LYS A 105       8.750  -6.794 -12.358  1.00  0.00           H  \\nATOM   1582  HD3 LYS A 105       9.370  -9.129 -12.585  1.00  0.00           H  \\nATOM   1583  HD2 LYS A 105       7.908  -9.472 -13.523  1.00  0.00           H  \\nATOM   1584  HE3 LYS A 105       8.680  -8.233 -15.394  1.00  0.00           H  \\nATOM   1585  HE2 LYS A 105       9.933  -7.422 -14.450  1.00  0.00           H  \\nATOM   1586  HZ1 LYS A 105      10.841  -9.146 -15.826  1.00  0.00           H  \\nATOM   1587  HZ2 LYS A 105      11.056  -9.531 -14.236  1.00  0.00           H  \\nATOM   1588  HZ3 LYS A 105       9.886 -10.289 -15.119  1.00  0.00           H  \\nATOM   1589  N   SER A 106       5.516  -7.055  -8.525  1.00  0.00           N  \\nATOM   1590  CA  SER A 106       4.486  -7.470  -7.574  1.00  0.00           C  \\nATOM   1591  C   SER A 106       3.534  -6.302  -7.305  1.00  0.00           C  \\nATOM   1592  O   SER A 106       4.014  -5.198  -7.059  1.00  0.00           O  \\nATOM   1593  CB  SER A 106       5.116  -7.943  -6.278  1.00  0.00           C  \\nATOM   1594  OG  SER A 106       5.855  -9.117  -6.478  1.00  0.00           O  \\nATOM   1595  H   SER A 106       6.312  -6.559  -8.147  1.00  0.00           H  \\nATOM   1596  HA  SER A 106       3.937  -8.314  -7.991  1.00  0.00           H  \\nATOM   1597  HB3 SER A 106       5.768  -7.163  -5.885  1.00  0.00           H  \\nATOM   1598  HB2 SER A 106       4.337  -8.122  -5.539  1.00  0.00           H  \\nATOM   1599  HG  SER A 106       5.667  -9.679  -5.723  1.00  0.00           H  \\nATOM   1600  N   PRO A 107       2.207  -6.499  -7.346  1.00  0.00           N  \\nATOM   1601  CA  PRO A 107       1.263  -5.412  -7.114  1.00  0.00           C  \\nATOM   1602  C   PRO A 107       1.073  -5.177  -5.615  1.00  0.00           C  \\nATOM   1603  O   PRO A 107       0.433  -5.986  -4.950  1.00  0.00           O  \\nATOM   1604  CB  PRO A 107      -0.028  -5.855  -7.812  1.00  0.00           C  \\nATOM   1605  CG  PRO A 107      -0.012  -7.377  -7.672  1.00  0.00           C  \\nATOM   1606  CD  PRO A 107       1.469  -7.748  -7.735  1.00  0.00           C  \\nATOM   1607  HA  PRO A 107       1.621  -4.483  -7.557  1.00  0.00           H  \\nATOM   1608  HB3 PRO A 107       0.016  -5.591  -8.868  1.00  0.00           H  \\nATOM   1609  HB2 PRO A 107      -0.912  -5.417  -7.350  1.00  0.00           H  \\nATOM   1610  HG3 PRO A 107      -0.571  -7.851  -8.480  1.00  0.00           H  \\nATOM   1611  HG2 PRO A 107      -0.423  -7.665  -6.705  1.00  0.00           H  \\nATOM   1612  HD3 PRO A 107       1.731  -8.031  -8.755  1.00  0.00           H  \\nATOM   1613  HD2 PRO A 107       1.678  -8.566  -7.047  1.00  0.00           H  \\nATOM   1614  N   GLU A 108       1.630  -4.087  -5.090  1.00  0.00           N  \\nATOM   1615  CA  GLU A 108       1.217  -3.554  -3.786  1.00  0.00           C  \\nATOM   1616  C   GLU A 108      -0.188  -2.983  -3.901  1.00  0.00           C  \\nATOM   1617  O   GLU A 108      -0.502  -2.424  -4.950  1.00  0.00           O  \\nATOM   1618  CB  GLU A 108       2.188  -2.476  -3.300  1.00  0.00           C  \\nATOM   1619  CG  GLU A 108       1.799  -1.832  -1.976  1.00  0.00           C  \\nATOM   1620  CD  GLU A 108       1.656  -2.829  -0.860  1.00  0.00           C  \\nATOM   1621  OE1 GLU A 108       2.138  -3.926  -1.004  1.00  0.00           O  \\nATOM   1622  OE2 GLU A 108       1.063  -2.492   0.138  1.00  0.00           O  \\nATOM   1623  H   GLU A 108       2.060  -3.420  -5.714  1.00  0.00           H  \\nATOM   1624  HA  GLU A 108       1.177  -4.368  -3.063  1.00  0.00           H  \\nATOM   1625  HB3 GLU A 108       3.182  -2.907  -3.183  1.00  0.00           H  \\nATOM   1626  HB2 GLU A 108       2.259  -1.686  -4.048  1.00  0.00           H  \\nATOM   1627  HG3 GLU A 108       2.561  -1.103  -1.701  1.00  0.00           H  \\nATOM   1628  HG2 GLU A 108       0.858  -1.300  -2.106  1.00  0.00           H  \\nATOM   1629  N   GLU A 109      -1.026  -3.130  -2.872  1.00  0.00           N  \\nATOM   1630  CA  GLU A 109      -2.392  -2.611  -2.897  1.00  0.00           C  \\nATOM   1631  C   GLU A 109      -2.815  -1.983  -1.563  1.00  0.00           C  \\nATOM   1632  O   GLU A 109      -2.575  -2.541  -0.490  1.00  0.00           O  \\nATOM   1633  CB  GLU A 109      -3.366  -3.732  -3.268  1.00  0.00           C  \\nATOM   1634  CG  GLU A 109      -3.363  -4.911  -2.306  1.00  0.00           C  \\nATOM   1635  CD  GLU A 109      -4.216  -6.055  -2.779  1.00  0.00           C  \\nATOM   1636  OE1 GLU A 109      -4.982  -5.861  -3.692  1.00  0.00           O  \\nATOM   1637  OE2 GLU A 109      -4.101  -7.124  -2.227  1.00  0.00           O  \\nATOM   1638  H   GLU A 109      -0.674  -3.438  -1.977  1.00  0.00           H  \\nATOM   1639  HA  GLU A 109      -2.417  -1.818  -3.645  1.00  0.00           H  \\nATOM   1640  HB3 GLU A 109      -4.381  -3.334  -3.306  1.00  0.00           H  \\nATOM   1641  HB2 GLU A 109      -3.125  -4.110  -4.261  1.00  0.00           H  \\nATOM   1642  HG3 GLU A 109      -2.339  -5.263  -2.183  1.00  0.00           H  \\nATOM   1643  HG2 GLU A 109      -3.721  -4.575  -1.334  1.00  0.00           H  \\nATOM   1644  N   TYR A 110      -3.531  -0.856  -1.640  1.00  0.00           N  \\nATOM   1645  CA  TYR A 110      -4.091  -0.182  -0.470  1.00  0.00           C  \\nATOM   1646  C   TYR A 110      -5.464   0.456  -0.745  1.00  0.00           C  \\nATOM   1647  O   TYR A 110      -5.625   1.191  -1.713  1.00  0.00           O  \\nATOM   1648  CB  TYR A 110      -3.119   0.855   0.086  1.00  0.00           C  \\nATOM   1649  CG  TYR A 110      -3.523   1.261   1.487  1.00  0.00           C  \\nATOM   1650  CD1 TYR A 110      -4.307   2.411   1.700  1.00  0.00           C  \\nATOM   1651  CD2 TYR A 110      -3.204   0.417   2.568  1.00  0.00           C  \\nATOM   1652  CE1 TYR A 110      -4.756   2.726   2.998  1.00  0.00           C  \\nATOM   1653  CE2 TYR A 110      -3.631   0.740   3.867  1.00  0.00           C  \\nATOM   1654  CZ  TYR A 110      -4.399   1.901   4.085  1.00  0.00           C  \\nATOM   1655  OH  TYR A 110      -4.782   2.218   5.349  1.00  0.00           O  \\nATOM   1656  H   TYR A 110      -3.645  -0.423  -2.546  1.00  0.00           H  \\nATOM   1657  HA  TYR A 110      -4.186  -0.931   0.316  1.00  0.00           H  \\nATOM   1658  HB3 TYR A 110      -2.118   0.422   0.112  1.00  0.00           H  \\nATOM   1659  HB2 TYR A 110      -3.085   1.726  -0.568  1.00  0.00           H  \\nATOM   1660  HD1 TYR A 110      -4.564   3.042   0.863  1.00  0.00           H  \\nATOM   1661  HD2 TYR A 110      -2.601  -0.461   2.393  1.00  0.00           H  \\nATOM   1662  HE1 TYR A 110      -5.356   3.605   3.185  1.00  0.00           H  \\nATOM   1663  HE2 TYR A 110      -3.340   0.106   4.692  1.00  0.00           H  \\nATOM   1664  HH  TYR A 110      -4.381   1.622   5.987  1.00  0.00           H  \\nATOM   1665  N   ALA A 111      -6.488   0.296   0.095  1.00  0.00           N  \\nATOM   1666  CA  ALA A 111      -6.488  -0.466   1.342  1.00  0.00           C  \\nATOM   1667  C   ALA A 111      -6.188  -1.955   1.128  1.00  0.00           C  \\nATOM   1668  O   ALA A 111      -6.524  -2.523   0.087  1.00  0.00           O  \\nATOM   1669  CB  ALA A 111      -7.832  -0.260   2.041  1.00  0.00           C  \\nATOM   1670  H   ALA A 111      -7.348   0.757  -0.167  1.00  0.00           H  \\nATOM   1671  HA  ALA A 111      -5.718  -0.063   2.000  1.00  0.00           H  \\nATOM   1672  HB1 ALA A 111      -8.638  -0.630   1.409  1.00  0.00           H  \\nATOM   1673  HB2 ALA A 111      -7.980   0.803   2.236  1.00  0.00           H  \\nATOM   1674  HB3 ALA A 111      -7.842  -0.797   2.990  1.00  0.00           H  \\nATOM   1675  N   PHE A 112      -5.561  -2.575   2.133  1.00  0.00           N  \\nATOM   1676  CA  PHE A 112      -5.115  -3.963   2.062  1.00  0.00           C  \\nATOM   1677  C   PHE A 112      -6.242  -4.893   1.621  1.00  0.00           C  \\nATOM   1678  O   PHE A 112      -7.362  -4.792   2.134  1.00  0.00           O  \\nATOM   1679  CB  PHE A 112      -4.629  -4.434   3.444  1.00  0.00           C  \\nATOM   1680  CG  PHE A 112      -3.524  -3.608   4.059  1.00  0.00           C  \\nATOM   1681  CD1 PHE A 112      -2.329  -3.412   3.347  1.00  0.00           C  \\nATOM   1682  CD2 PHE A 112      -3.677  -3.041   5.341  1.00  0.00           C  \\nATOM   1683  CE1 PHE A 112      -1.304  -2.645   3.911  1.00  0.00           C  \\nATOM   1684  CE2 PHE A 112      -2.645  -2.266   5.902  1.00  0.00           C  \\nATOM   1685  CZ  PHE A 112      -1.455  -2.064   5.180  1.00  0.00           C  \\nATOM   1686  H   PHE A 112      -5.237  -2.017   2.911  1.00  0.00           H  \\nATOM   1687  HA  PHE A 112      -4.298  -4.026   1.343  1.00  0.00           H  \\nATOM   1688  HB3 PHE A 112      -5.477  -4.458   4.128  1.00  0.00           H  \\nATOM   1689  HB2 PHE A 112      -4.257  -5.454   3.343  1.00  0.00           H  \\nATOM   1690  HD1 PHE A 112      -2.184  -3.823   2.360  1.00  0.00           H  \\nATOM   1691  HD2 PHE A 112      -4.591  -3.201   5.894  1.00  0.00           H  \\nATOM   1692  HE1 PHE A 112      -0.399  -2.486   3.342  1.00  0.00           H  \\nATOM   1693  HE2 PHE A 112      -2.761  -1.817   6.877  1.00  0.00           H  \\nATOM   1694  HZ  PHE A 112      -0.657  -1.456   5.579  1.00  0.00           H  \\nATOM   1695  N   ALA A 113      -5.941  -5.890   0.785  1.00  0.00           N  \\nATOM   1696  CA  ALA A 113      -6.917  -6.935   0.479  1.00  0.00           C  \\nATOM   1697  C   ALA A 113      -7.399  -7.672   1.734  1.00  0.00           C  \\nATOM   1698  O   ALA A 113      -8.535  -8.126   1.774  1.00  0.00           O  \\nATOM   1699  CB  ALA A 113      -6.336  -7.922  -0.530  1.00  0.00           C  \\nATOM   1700  H   ALA A 113      -5.057  -5.872   0.296  1.00  0.00           H  \\nATOM   1701  HA  ALA A 113      -7.793  -6.466   0.028  1.00  0.00           H  \\nATOM   1702  HB1 ALA A 113      -5.307  -8.184  -0.281  1.00  0.00           H  \\nATOM   1703  HB2 ALA A 113      -6.949  -8.822  -0.566  1.00  0.00           H  \\nATOM   1704  HB3 ALA A 113      -6.380  -7.458  -1.516  1.00  0.00           H  \\nATOM   1705  N   SER A 114      -6.587  -7.757   2.790  1.00  0.00           N  \\nATOM   1706  CA  SER A 114      -6.965  -8.371   4.067  1.00  0.00           C  \\nATOM   1707  C   SER A 114      -7.937  -7.533   4.908  1.00  0.00           C  \\nATOM   1708  O   SER A 114      -8.323  -7.989   5.981  1.00  0.00           O  \\nATOM   1709  CB  SER A 114      -5.713  -8.643   4.880  1.00  0.00           C  \\nATOM   1710  OG  SER A 114      -5.079  -7.447   5.241  1.00  0.00           O  \\nATOM   1711  H   SER A 114      -5.694  -7.285   2.752  1.00  0.00           H  \\nATOM   1712  HA  SER A 114      -7.477  -9.312   3.863  1.00  0.00           H  \\nATOM   1713  HB3 SER A 114      -5.977  -9.204   5.776  1.00  0.00           H  \\nATOM   1714  HB2 SER A 114      -5.029  -9.260   4.298  1.00  0.00           H  \\nATOM   1715  HG  SER A 114      -5.752  -6.764   5.187  1.00  0.00           H  \\nATOM   1716  N   SER A 115      -8.348  -6.343   4.460  1.00  0.00           N  \\nATOM   1717  CA  SER A 115      -9.326  -5.484   5.139  1.00  0.00           C  \\nATOM   1718  C   SER A 115     -10.640  -5.338   4.362  1.00  0.00           C  \\nATOM   1719  O   SER A 115     -10.665  -5.468   3.142  1.00  0.00           O  \\nATOM   1720  CB  SER A 115      -8.725  -4.112   5.370  1.00  0.00           C  \\nATOM   1721  OG  SER A 115      -8.441  -3.476   4.153  1.00  0.00           O  \\nATOM   1722  H   SER A 115      -7.996  -6.015   3.573  1.00  0.00           H  \\nATOM   1723  HA  SER A 115      -9.565  -5.941   6.099  1.00  0.00           H  \\nATOM   1724  HB3 SER A 115      -9.419  -3.502   5.948  1.00  0.00           H  \\nATOM   1725  HB2 SER A 115      -7.811  -4.209   5.954  1.00  0.00           H  \\nATOM   1726  HG  SER A 115      -8.226  -4.177   3.533  1.00  0.00           H  \\nATOM   1727  N   LEU A 116     -11.736  -5.023   5.056  1.00  0.00           N  \\nATOM   1728  CA  LEU A 116     -13.058  -4.808   4.452  1.00  0.00           C  \\nATOM   1729  C   LEU A 116     -13.015  -3.723   3.366  1.00  0.00           C  \\nATOM   1730  O   LEU A 116     -13.743  -3.793   2.382  1.00  0.00           O  \\nATOM   1731  CB  LEU A 116     -14.073  -4.416   5.534  1.00  0.00           C  \\nATOM   1732  CG  LEU A 116     -15.529  -4.293   5.066  1.00  0.00           C  \\nATOM   1733  CD1 LEU A 116     -16.027  -5.651   4.592  1.00  0.00           C  \\nATOM   1734  CD2 LEU A 116     -16.385  -3.764   6.207  1.00  0.00           C  \\nATOM   1735  H   LEU A 116     -11.650  -4.922   6.058  1.00  0.00           H  \\nATOM   1736  HA  LEU A 116     -13.384  -5.732   3.976  1.00  0.00           H  \\nATOM   1737  HB3 LEU A 116     -14.041  -5.161   6.326  1.00  0.00           H  \\nATOM   1738  HB2 LEU A 116     -13.777  -3.455   5.955  1.00  0.00           H  \\nATOM   1739  HG  LEU A 116     -15.583  -3.604   4.222  1.00  0.00           H  \\nATOM   1740 HD11 LEU A 116     -17.061  -5.564   4.260  1.00  0.00           H  \\nATOM   1741 HD12 LEU A 116     -15.408  -5.996   3.763  1.00  0.00           H  \\nATOM   1742 HD13 LEU A 116     -15.968  -6.366   5.412  1.00  0.00           H  \\nATOM   1743 HD21 LEU A 116     -17.420  -3.676   5.875  1.00  0.00           H  \\nATOM   1744 HD22 LEU A 116     -16.333  -4.452   7.051  1.00  0.00           H  \\nATOM   1745 HD23 LEU A 116     -16.018  -2.785   6.514  1.00  0.00           H  \\nATOM   1746  N   GLN A 117     -12.142  -2.735   3.551  1.00  0.00           N  \\nATOM   1747  CA  GLN A 117     -11.996  -1.549   2.714  1.00  0.00           C  \\nATOM   1748  C   GLN A 117     -11.171  -1.813   1.446  1.00  0.00           C  \\nATOM   1749  O   GLN A 117     -11.013  -0.912   0.625  1.00  0.00           O  \\nATOM   1750  CB  GLN A 117     -11.330  -0.444   3.550  1.00  0.00           C  \\nATOM   1751  CG  GLN A 117     -12.168   0.058   4.741  1.00  0.00           C  \\nATOM   1752  CD  GLN A 117     -12.309  -0.921   5.911  1.00  0.00           C  \\nATOM   1753  OE1 GLN A 117     -11.534  -1.849   6.121  1.00  0.00           O  \\nATOM   1754  NE2 GLN A 117     -13.337  -0.770   6.714  1.00  0.00           N  \\nATOM   1755  H   GLN A 117     -11.541  -2.796   4.361  1.00  0.00           H  \\nATOM   1756  HA  GLN A 117     -12.981  -1.211   2.393  1.00  0.00           H  \\nATOM   1757  HB3 GLN A 117     -10.360  -0.795   3.903  1.00  0.00           H  \\nATOM   1758  HB2 GLN A 117     -11.150   0.404   2.891  1.00  0.00           H  \\nATOM   1759  HG3 GLN A 117     -11.703   0.965   5.132  1.00  0.00           H  \\nATOM   1760  HG2 GLN A 117     -13.159   0.330   4.379  1.00  0.00           H  \\nATOM   1761 HE21 GLN A 117     -13.969   0.007   6.587  1.00  0.00           H  \\nATOM   1762 HE22 GLN A 117     -13.411  -1.389   7.510  1.00  0.00           H  \\nATOM   1763  N   GLY A 118     -10.619  -3.020   1.298  1.00  0.00           N  \\nATOM   1764  CA  GLY A 118      -9.825  -3.405   0.142  1.00  0.00           C  \\nATOM   1765  C   GLY A 118     -10.668  -3.781  -1.078  1.00  0.00           C  \\nATOM   1766  O   GLY A 118     -11.906  -3.776  -1.075  1.00  0.00           O  \\nATOM   1767  H   GLY A 118     -10.807  -3.725   1.996  1.00  0.00           H  \\nATOM   1768  HA3 GLY A 118      -9.165  -2.582  -0.131  1.00  0.00           H  \\nATOM   1769  HA2 GLY A 118      -9.202  -4.261   0.405  1.00  0.00           H  \\nATOM   1770  N   GLY A 119      -9.970  -4.189  -2.134  1.00  0.00           N  \\nATOM   1771  CA  GLY A 119     -10.571  -4.549  -3.412  1.00  0.00           C  \\nATOM   1772  C   GLY A 119     -10.892  -3.336  -4.269  1.00  0.00           C  \\nATOM   1773  O   GLY A 119      -9.984  -2.634  -4.704  1.00  0.00           O  \\nATOM   1774  H   GLY A 119      -8.962  -4.180  -2.075  1.00  0.00           H  \\nATOM   1775  HA3 GLY A 119      -9.840  -5.131  -3.972  1.00  0.00           H  \\nATOM   1776  HA2 GLY A 119     -11.464  -5.158  -3.266  1.00  0.00           H  \\nATOM   1777  N   THR A 120     -12.171  -3.115  -4.564  1.00  0.00           N  \\nATOM   1778  CA  THR A 120     -12.609  -2.071  -5.503  1.00  0.00           C  \\nATOM   1779  C   THR A 120     -12.036  -0.707  -5.110  1.00  0.00           C  \\nATOM   1780  O   THR A 120     -12.150  -0.295  -3.958  1.00  0.00           O  \\nATOM   1781  CB  THR A 120     -14.145  -1.987  -5.567  1.00  0.00           C  \\nATOM   1782  OG1 THR A 120     -14.681  -3.262  -5.945  1.00  0.00           O  \\nATOM   1783  CG2 THR A 120     -14.580  -0.938  -6.580  1.00  0.00           C  \\nATOM   1784  H   THR A 120     -12.881  -3.691  -4.134  1.00  0.00           H  \\nATOM   1785  HA  THR A 120     -12.248  -2.332  -6.498  1.00  0.00           H  \\nATOM   1786  HB  THR A 120     -14.537  -1.720  -4.586  1.00  0.00           H  \\nATOM   1787  HG1 THR A 120     -14.495  -3.424  -6.873  1.00  0.00           H  \\nATOM   1788 HG21 THR A 120     -15.669  -0.893  -6.611  1.00  0.00           H  \\nATOM   1789 HG22 THR A 120     -14.184   0.034  -6.289  1.00  0.00           H  \\nATOM   1790 HG23 THR A 120     -14.200  -1.205  -7.565  1.00  0.00           H  \\nATOM   1791  N   ASN A 121     -11.448   0.003  -6.077  1.00  0.00           N  \\nATOM   1792  CA  ASN A 121     -10.777   1.299  -5.902  1.00  0.00           C  \\nATOM   1793  C   ASN A 121      -9.494   1.281  -5.054  1.00  0.00           C  \\nATOM   1794  O   ASN A 121      -8.929   2.353  -4.814  1.00  0.00           O  \\nATOM   1795  CB  ASN A 121     -11.760   2.293  -5.312  1.00  0.00           C  \\nATOM   1796  CG  ASN A 121     -12.964   2.506  -6.187  1.00  0.00           C  \\nATOM   1797  OD1 ASN A 121     -12.861   2.487  -7.419  1.00  0.00           O  \\nATOM   1798  ND2 ASN A 121     -14.103   2.707  -5.575  1.00  0.00           N  \\nATOM   1799  H   ASN A 121     -11.416  -0.400  -7.002  1.00  0.00           H  \\nATOM   1800  HA  ASN A 121     -10.447   1.604  -6.896  1.00  0.00           H  \\nATOM   1801  HB3 ASN A 121     -12.093   1.939  -4.335  1.00  0.00           H  \\nATOM   1802  HB2 ASN A 121     -11.262   3.251  -5.160  1.00  0.00           H  \\nATOM   1803 HD21 ASN A 121     -14.937   2.855  -6.107  1.00  0.00           H  \\nATOM   1804 HD22 ASN A 121     -14.139   2.714  -4.576  1.00  0.00           H  \\nATOM   1805  N   ALA A 122      -8.991   0.113  -4.641  1.00  0.00           N  \\nATOM   1806  CA  ALA A 122      -7.664   0.041  -4.047  1.00  0.00           C  \\nATOM   1807  C   ALA A 122      -6.612   0.525  -5.056  1.00  0.00           C  \\nATOM   1808  O   ALA A 122      -6.688   0.265  -6.260  1.00  0.00           O  \\nATOM   1809  CB  ALA A 122      -7.369  -1.369  -3.531  1.00  0.00           C  \\nATOM   1810  H   ALA A 122      -9.478  -0.753  -4.826  1.00  0.00           H  \\nATOM   1811  HA  ALA A 122      -7.651   0.716  -3.192  1.00  0.00           H  \\nATOM   1812  HB1 ALA A 122      -6.387  -1.390  -3.059  1.00  0.00           H  \\nATOM   1813  HB2 ALA A 122      -8.118  -1.659  -2.794  1.00  0.00           H  \\nATOM   1814  HB3 ALA A 122      -7.378  -2.073  -4.363  1.00  0.00           H  \\nATOM   1815  N   ILE A 123      -5.644   1.270  -4.544  1.00  0.00           N  \\nATOM   1816  CA  ILE A 123      -4.539   1.851  -5.284  1.00  0.00           C  \\nATOM   1817  C   ILE A 123      -3.444   0.806  -5.375  1.00  0.00           C  \\nATOM   1818  O   ILE A 123      -3.020   0.278  -4.353  1.00  0.00           O  \\nATOM   1819  CB  ILE A 123      -4.005   3.129  -4.611  1.00  0.00           C  \\nATOM   1820  CG1 ILE A 123      -5.079   4.220  -4.610  1.00  0.00           C  \\nATOM   1821  CG2 ILE A 123      -2.748   3.615  -5.316  1.00  0.00           C  \\nATOM   1822  CD1 ILE A 123      -4.732   5.416  -3.754  1.00  0.00           C  \\nATOM   1823  H   ILE A 123      -5.583   1.308  -3.536  1.00  0.00           H  \\nATOM   1824  HA  ILE A 123      -4.864   2.113  -6.291  1.00  0.00           H  \\nATOM   1825  HB  ILE A 123      -3.768   2.919  -3.569  1.00  0.00           H  \\nATOM   1826 HG13 ILE A 123      -5.246   4.566  -5.629  1.00  0.00           H  \\nATOM   1827 HG12 ILE A 123      -6.020   3.804  -4.248  1.00  0.00           H  \\nATOM   1828 HG21 ILE A 123      -2.384   4.519  -4.828  1.00  0.00           H  \\nATOM   1829 HG22 ILE A 123      -1.981   2.843  -5.266  1.00  0.00           H  \\nATOM   1830 HG23 ILE A 123      -2.977   3.833  -6.359  1.00  0.00           H  \\nATOM   1831 HD11 ILE A 123      -5.541   6.145  -3.803  1.00  0.00           H  \\nATOM   1832 HD12 ILE A 123      -4.595   5.097  -2.720  1.00  0.00           H  \\nATOM   1833 HD13 ILE A 123      -3.812   5.870  -4.119  1.00  0.00           H  \\nATOM   1834  N   LEU A 124      -2.971   0.550  -6.589  1.00  0.00           N  \\nATOM   1835  CA  LEU A 124      -1.894  -0.381  -6.865  1.00  0.00           C  \\nATOM   1836  C   LEU A 124      -0.588   0.352  -7.132  1.00  0.00           C  \\nATOM   1837  O   LEU A 124      -0.598   1.410  -7.761  1.00  0.00           O  \\nATOM   1838  CB  LEU A 124      -2.252  -1.260  -8.069  1.00  0.00           C  \\nATOM   1839  CG  LEU A 124      -3.521  -2.109  -7.920  1.00  0.00           C  \\nATOM   1840  CD1 LEU A 124      -3.696  -2.984  -9.154  1.00  0.00           C  \\nATOM   1841  CD2 LEU A 124      -3.420  -2.955  -6.659  1.00  0.00           C  \\nATOM   1842  H   LEU A 124      -3.379   1.043  -7.370  1.00  0.00           H  \\nATOM   1843  HA  LEU A 124      -1.757  -0.987  -5.970  1.00  0.00           H  \\nATOM   1844  HB3 LEU A 124      -2.383  -0.620  -8.940  1.00  0.00           H  \\nATOM   1845  HB2 LEU A 124      -1.421  -1.938  -8.265  1.00  0.00           H  \\nATOM   1846  HG  LEU A 124      -4.390  -1.455  -7.849  1.00  0.00           H  \\nATOM   1847 HD11 LEU A 124      -4.598  -3.587  -9.047  1.00  0.00           H  \\nATOM   1848 HD12 LEU A 124      -3.785  -2.352 -10.038  1.00  0.00           H  \\nATOM   1849 HD13 LEU A 124      -2.832  -3.639  -9.260  1.00  0.00           H  \\nATOM   1850 HD21 LEU A 124      -4.322  -3.558  -6.552  1.00  0.00           H  \\nATOM   1851 HD22 LEU A 124      -2.552  -3.611  -6.730  1.00  0.00           H  \\nATOM   1852 HD23 LEU A 124      -3.313  -2.304  -5.791  1.00  0.00           H  \\nATOM   1853  N   ALA A 125       0.531  -0.255  -6.745  1.00  0.00           N  \\nATOM   1854  CA  ALA A 125       1.867   0.229  -7.077  1.00  0.00           C  \\nATOM   1855  C   ALA A 125       2.861  -0.935  -7.252  1.00  0.00           C  \\nATOM   1856  O   ALA A 125       2.704  -1.985  -6.627  1.00  0.00           O  \\nATOM   1857  CB  ALA A 125       2.329   1.219  -6.004  1.00  0.00           C  \\nATOM   1858  H   ALA A 125       0.450  -1.126  -6.239  1.00  0.00           H  \\nATOM   1859  HA  ALA A 125       1.806   0.767  -8.023  1.00  0.00           H  \\nATOM   1860  HB1 ALA A 125       3.262   1.697  -6.302  1.00  0.00           H  \\nATOM   1861  HB2 ALA A 125       2.495   0.681  -5.070  1.00  0.00           H  \\nATOM   1862  HB3 ALA A 125       1.572   1.989  -5.853  1.00  0.00           H  \\nATOM   1863  N   PRO A 126       3.888  -0.781  -8.103  1.00  0.00           N  \\nATOM   1864  CA  PRO A 126       4.891  -1.805  -8.329  1.00  0.00           C  \\nATOM   1865  C   PRO A 126       5.883  -1.867  -7.168  1.00  0.00           C  \\nATOM   1866  O   PRO A 126       6.600  -0.909  -6.876  1.00  0.00           O  \\nATOM   1867  CB  PRO A 126       5.563  -1.346  -9.627  1.00  0.00           C  \\nATOM   1868  CG  PRO A 126       5.454   0.140  -9.589  1.00  0.00           C  \\nATOM   1869  CD  PRO A 126       4.114   0.401  -8.955  1.00  0.00           C  \\nATOM   1870  HA  PRO A 126       4.413  -2.778  -8.453  1.00  0.00           H  \\nATOM   1871  HB3 PRO A 126       6.605  -1.697  -9.656  1.00  0.00           H  \\nATOM   1872  HB2 PRO A 126       5.050  -1.790 -10.493  1.00  0.00           H  \\nATOM   1873  HG3 PRO A 126       6.286   0.567  -9.011  1.00  0.00           H  \\nATOM   1874  HG2 PRO A 126       5.528   0.552 -10.606  1.00  0.00           H  \\nATOM   1875  HD3 PRO A 126       4.165   1.320  -8.353  1.00  0.00           H  \\nATOM   1876  HD2 PRO A 126       3.348   0.493  -9.739  1.00  0.00           H  \\nATOM   1877  N   VAL A 127       5.982  -3.043  -6.565  1.00  0.00           N  \\nATOM   1878  CA  VAL A 127       6.985  -3.395  -5.559  1.00  0.00           C  \\nATOM   1879  C   VAL A 127       7.608  -4.745  -5.917  1.00  0.00           C  \\nATOM   1880  O   VAL A 127       7.159  -5.427  -6.842  1.00  0.00           O  \\nATOM   1881  CB  VAL A 127       6.352  -3.468  -4.156  1.00  0.00           C  \\nATOM   1882  CG1 VAL A 127       5.677  -2.150  -3.808  1.00  0.00           C  \\nATOM   1883  CG2 VAL A 127       5.355  -4.616  -4.097  1.00  0.00           C  \\nATOM   1884  H   VAL A 127       5.340  -3.769  -6.850  1.00  0.00           H  \\nATOM   1885  HA  VAL A 127       7.779  -2.649  -5.586  1.00  0.00           H  \\nATOM   1886  HB  VAL A 127       7.139  -3.631  -3.420  1.00  0.00           H  \\nATOM   1887 HG11 VAL A 127       5.235  -2.219  -2.813  1.00  0.00           H  \\nATOM   1888 HG12 VAL A 127       6.416  -1.349  -3.821  1.00  0.00           H  \\nATOM   1889 HG13 VAL A 127       4.896  -1.937  -4.537  1.00  0.00           H  \\nATOM   1890 HG21 VAL A 127       4.912  -4.662  -3.103  1.00  0.00           H  \\nATOM   1891 HG22 VAL A 127       4.571  -4.456  -4.838  1.00  0.00           H  \\nATOM   1892 HG23 VAL A 127       5.868  -5.554  -4.309  1.00  0.00           H  \\nATOM   1893  N   ASN A 128       8.679  -5.149  -5.237  1.00  0.00           N  \\nATOM   1894  CA  ASN A 128       9.275  -6.466  -5.470  1.00  0.00           C  \\nATOM   1895  C   ASN A 128       8.638  -7.536  -4.561  1.00  0.00           C  \\nATOM   1896  O   ASN A 128       7.823  -7.239  -3.684  1.00  0.00           O  \\nATOM   1897  CB  ASN A 128      10.807  -6.368  -5.434  1.00  0.00           C  \\nATOM   1898  CG  ASN A 128      11.367  -6.050  -4.067  1.00  0.00           C  \\nATOM   1899  OD1 ASN A 128      10.881  -6.522  -3.065  1.00  0.00           O  \\nATOM   1900  ND2 ASN A 128      12.421  -5.278  -3.980  1.00  0.00           N  \\nATOM   1901  H   ASN A 128       8.972  -4.624  -4.426  1.00  0.00           H  \\nATOM   1902  HA  ASN A 128       9.043  -6.780  -6.488  1.00  0.00           H  \\nATOM   1903  HB3 ASN A 128      11.136  -5.617  -6.152  1.00  0.00           H  \\nATOM   1904  HB2 ASN A 128      11.221  -7.326  -5.747  1.00  0.00           H  \\nATOM   1905 HD21 ASN A 128      12.791  -4.833  -4.807  1.00  0.00           H  \\nATOM   1906 HD22 ASN A 128      12.637  -4.956  -3.046  1.00  0.00           H  \\nATOM   1907  N   LEU A 129       8.968  -8.806  -4.795  1.00  0.00           N  \\nATOM   1908  CA  LEU A 129       8.395  -9.920  -4.032  1.00  0.00           C  \\nATOM   1909  C   LEU A 129       8.880  -9.935  -2.577  1.00  0.00           C  \\nATOM   1910  O   LEU A 129       8.076 -10.115  -1.665  1.00  0.00           O  \\nATOM   1911  CB  LEU A 129       8.752 -11.252  -4.704  1.00  0.00           C  \\nATOM   1912  CG  LEU A 129       8.201 -12.510  -4.022  1.00  0.00           C  \\nATOM   1913  CD1 LEU A 129       6.682 -12.424  -3.948  1.00  0.00           C  \\nATOM   1914  CD2 LEU A 129       8.641 -13.743  -4.797  1.00  0.00           C  \\nATOM   1915  H   LEU A 129       9.701  -9.001  -5.463  1.00  0.00           H  \\nATOM   1916  HA  LEU A 129       7.313  -9.800  -3.999  1.00  0.00           H  \\nATOM   1917  HB3 LEU A 129       8.376 -11.237  -5.726  1.00  0.00           H  \\nATOM   1918  HB2 LEU A 129       9.838 -11.343  -4.740  1.00  0.00           H  \\nATOM   1919  HG  LEU A 129       8.581 -12.568  -3.002  1.00  0.00           H  \\nATOM   1920 HD11 LEU A 129       6.290 -13.318  -3.463  1.00  0.00           H  \\nATOM   1921 HD12 LEU A 129       6.395 -11.545  -3.372  1.00  0.00           H  \\nATOM   1922 HD13 LEU A 129       6.273 -12.349  -4.955  1.00  0.00           H  \\nATOM   1923 HD21 LEU A 129       8.250 -14.638  -4.312  1.00  0.00           H  \\nATOM   1924 HD22 LEU A 129       8.260 -13.687  -5.817  1.00  0.00           H  \\nATOM   1925 HD23 LEU A 129       9.730 -13.789  -4.819  1.00  0.00           H  \\nATOM   1926  N   ALA A 130      10.176  -9.728  -2.349  1.00  0.00           N  \\nATOM   1927  CA  ALA A 130      10.730  -9.709  -1.001  1.00  0.00           C  \\nATOM   1928  C   ALA A 130      10.058  -8.652  -0.104  1.00  0.00           C  \\nATOM   1929  O   ALA A 130       9.712  -8.948   1.039  1.00  0.00           O  \\nATOM   1930  CB  ALA A 130      12.238  -9.525  -1.124  1.00  0.00           C  \\nATOM   1931  H   ALA A 130      10.796  -9.553  -3.126  1.00  0.00           H  \\nATOM   1932  HA  ALA A 130      10.547 -10.678  -0.537  1.00  0.00           H  \\nATOM   1933  HB1 ALA A 130      12.663 -10.337  -1.713  1.00  0.00           H  \\nATOM   1934  HB2 ALA A 130      12.464  -8.572  -1.601  1.00  0.00           H  \\nATOM   1935  HB3 ALA A 130      12.680  -9.560  -0.128  1.00  0.00           H  \\nATOM   1936  N   SER A 131       9.765  -7.470  -0.648  1.00  0.00           N  \\nATOM   1937  CA  SER A 131       9.046  -6.396   0.033  1.00  0.00           C  \\nATOM   1938  C   SER A 131       7.611  -6.781   0.367  1.00  0.00           C  \\nATOM   1939  O   SER A 131       7.186  -6.569   1.500  1.00  0.00           O  \\nATOM   1940  CB  SER A 131       9.021  -5.144  -0.842  1.00  0.00           C  \\nATOM   1941  OG  SER A 131      10.311  -4.589  -0.928  1.00  0.00           O  \\nATOM   1942  H   SER A 131      10.074  -7.294  -1.593  1.00  0.00           H  \\nATOM   1943  HA  SER A 131       9.547  -6.168   0.974  1.00  0.00           H  \\nATOM   1944  HB3 SER A 131       8.652  -5.384  -1.839  1.00  0.00           H  \\nATOM   1945  HB2 SER A 131       8.349  -4.416  -0.388  1.00  0.00           H  \\nATOM   1946  HG  SER A 131      10.448  -4.121  -0.102  1.00  0.00           H  \\nATOM   1947  N   GLN A 132       6.897  -7.401  -0.578  1.00  0.00           N  \\nATOM   1948  CA  GLN A 132       5.528  -7.870  -0.346  1.00  0.00           C  \\nATOM   1949  C   GLN A 132       5.455  -8.945   0.747  1.00  0.00           C  \\nATOM   1950  O   GLN A 132       4.621  -8.890   1.650  1.00  0.00           O  \\nATOM   1951  CB  GLN A 132       4.931  -8.414  -1.646  1.00  0.00           C  \\nATOM   1952  CG  GLN A 132       3.460  -8.784  -1.549  1.00  0.00           C  \\nATOM   1953  CD  GLN A 132       2.855  -9.109  -2.901  1.00  0.00           C  \\nATOM   1954  OE1 GLN A 132       3.311 -10.019  -3.598  1.00  0.00           O  \\nATOM   1955  NE2 GLN A 132       1.822  -8.366  -3.280  1.00  0.00           N  \\nATOM   1956  H   GLN A 132       7.303  -7.527  -1.494  1.00  0.00           H  \\nATOM   1957  HA  GLN A 132       4.952  -7.022   0.024  1.00  0.00           H  \\nATOM   1958  HB3 GLN A 132       5.039  -7.671  -2.435  1.00  0.00           H  \\nATOM   1959  HB2 GLN A 132       5.481  -9.303  -1.955  1.00  0.00           H  \\nATOM   1960  HG3 GLN A 132       3.358  -9.660  -0.909  1.00  0.00           H  \\nATOM   1961  HG2 GLN A 132       2.913  -7.943  -1.122  1.00  0.00           H  \\nATOM   1962 HE21 GLN A 132       1.381  -8.534  -4.163  1.00  0.00           H  \\nATOM   1963 HE22 GLN A 132       1.484  -7.637  -2.684  1.00  0.00           H  \\nATOM   1964  N   ASN A 133       6.380  -9.907   0.719  1.00  0.00           N  \\nATOM   1965  CA  ASN A 133       6.455 -10.945   1.748  1.00  0.00           C  \\nATOM   1966  C   ASN A 133       6.823 -10.355   3.119  1.00  0.00           C  \\nATOM   1967  O   ASN A 133       6.248 -10.736   4.139  1.00  0.00           O  \\nATOM   1968  CB  ASN A 133       7.450 -12.019   1.343  1.00  0.00           C  \\nATOM   1969  CG  ASN A 133       6.916 -12.925   0.269  1.00  0.00           C  \\nATOM   1970  OD1 ASN A 133       5.702 -12.991   0.041  1.00  0.00           O  \\nATOM   1971  ND2 ASN A 133       7.799 -13.626  -0.395  1.00  0.00           N  \\nATOM   1972  H   ASN A 133       7.020  -9.936  -0.061  1.00  0.00           H  \\nATOM   1973  HA  ASN A 133       5.477 -11.415   1.853  1.00  0.00           H  \\nATOM   1974  HB3 ASN A 133       8.366 -11.549   0.984  1.00  0.00           H  \\nATOM   1975  HB2 ASN A 133       7.710 -12.620   2.214  1.00  0.00           H  \\nATOM   1976 HD21 ASN A 133       7.501 -14.247  -1.121  1.00  0.00           H  \\nATOM   1977 HD22 ASN A 133       8.771 -13.542  -0.177  1.00  0.00           H  \\nATOM   1978  N   SER A 134       7.760  -9.402   3.139  1.00  0.00           N  \\nATOM   1979  CA  SER A 134       8.184  -8.716   4.357  1.00  0.00           C  \\nATOM   1980  C   SER A 134       7.043  -7.907   4.984  1.00  0.00           C  \\nATOM   1981  O   SER A 134       6.833  -8.013   6.192  1.00  0.00           O  \\nATOM   1982  CB  SER A 134       9.354  -7.801   4.051  1.00  0.00           C  \\nATOM   1983  OG  SER A 134      10.470  -8.536   3.631  1.00  0.00           O  \\nATOM   1984  H   SER A 134       8.190  -9.136   2.265  1.00  0.00           H  \\nATOM   1985  HA  SER A 134       8.486  -9.470   5.085  1.00  0.00           H  \\nATOM   1986  HB3 SER A 134       9.068  -7.092   3.274  1.00  0.00           H  \\nATOM   1987  HB2 SER A 134       9.608  -7.226   4.941  1.00  0.00           H  \\nATOM   1988  HG  SER A 134      10.282  -8.812   2.731  1.00  0.00           H  \\nATOM   1989  N   GLN A 135       6.259  -7.170   4.185  1.00  0.00           N  \\nATOM   1990  CA  GLN A 135       5.072  -6.441   4.656  1.00  0.00           C  \\nATOM   1991  C   GLN A 135       4.049  -7.399   5.279  1.00  0.00           C  \\nATOM   1992  O   GLN A 135       3.587  -7.159   6.393  1.00  0.00           O  \\nATOM   1993  CB  GLN A 135       4.446  -5.668   3.484  1.00  0.00           C  \\nATOM   1994  CG  GLN A 135       3.220  -4.802   3.872  1.00  0.00           C  \\nATOM   1995  CD  GLN A 135       2.364  -4.415   2.663  1.00  0.00           C  \\nATOM   1996  OE1 GLN A 135       2.148  -5.242   1.798  1.00  0.00           O  \\nATOM   1997  NE2 GLN A 135       1.805  -3.227   2.572  1.00  0.00           N  \\nATOM   1998  H   GLN A 135       6.474  -7.121   3.200  1.00  0.00           H  \\nATOM   1999  HA  GLN A 135       5.372  -5.726   5.422  1.00  0.00           H  \\nATOM   2000  HB3 GLN A 135       5.203  -5.010   3.056  1.00  0.00           H  \\nATOM   2001  HB2 GLN A 135       4.176  -6.390   2.712  1.00  0.00           H  \\nATOM   2002  HG3 GLN A 135       3.567  -3.899   4.374  1.00  0.00           H  \\nATOM   2003  HG2 GLN A 135       2.575  -5.345   4.563  1.00  0.00           H  \\nATOM   2004 HE21 GLN A 135       1.799  -2.495   3.267  1.00  0.00           H  \\nATOM   2005 HE22 GLN A 135       1.363  -3.020   1.688  1.00  0.00           H  \\nATOM   2006  N   GLY A 136       3.767  -8.532   4.625  1.00  0.00           N  \\nATOM   2007  CA  GLY A 136       2.876  -9.561   5.162  1.00  0.00           C  \\nATOM   2008  C   GLY A 136       3.328 -10.098   6.525  1.00  0.00           C  \\nATOM   2009  O   GLY A 136       2.511 -10.227   7.438  1.00  0.00           O  \\nATOM   2010  H   GLY A 136       4.130  -8.643   3.689  1.00  0.00           H  \\nATOM   2011  HA3 GLY A 136       2.844 -10.393   4.457  1.00  0.00           H  \\nATOM   2012  HA2 GLY A 136       1.872  -9.148   5.264  1.00  0.00           H  \\nATOM   2013  N   GLY A 137       4.628 -10.359   6.698  1.00  0.00           N  \\nATOM   2014  CA  GLY A 137       5.210 -10.730   7.991  1.00  0.00           C  \\nATOM   2015  C   GLY A 137       4.972  -9.673   9.074  1.00  0.00           C  \\nATOM   2016  O   GLY A 137       4.449  -9.995  10.143  1.00  0.00           O  \\nATOM   2017  H   GLY A 137       5.243 -10.260   5.902  1.00  0.00           H  \\nATOM   2018  HA3 GLY A 137       4.786 -11.675   8.326  1.00  0.00           H  \\nATOM   2019  HA2 GLY A 137       6.286 -10.852   7.869  1.00  0.00           H  \\nATOM   2020  N   VAL A 138       5.286  -8.408   8.780  1.00  0.00           N  \\nATOM   2021  CA  VAL A 138       5.105  -7.287   9.718  1.00  0.00           C  \\nATOM   2022  C   VAL A 138       3.626  -7.084  10.063  1.00  0.00           C  \\nATOM   2023  O   VAL A 138       3.291  -6.953  11.238  1.00  0.00           O  \\nATOM   2024  CB  VAL A 138       5.668  -5.987   9.115  1.00  0.00           C  \\nATOM   2025  CG1 VAL A 138       5.307  -4.794   9.987  1.00  0.00           C  \\nATOM   2026  CG2 VAL A 138       7.177  -6.101   8.955  1.00  0.00           C  \\nATOM   2027  H   VAL A 138       5.673  -8.210   7.869  1.00  0.00           H  \\nATOM   2028  HA  VAL A 138       5.612  -7.531  10.652  1.00  0.00           H  \\nATOM   2029  HB  VAL A 138       5.211  -5.822   8.139  1.00  0.00           H  \\nATOM   2030 HG11 VAL A 138       5.713  -3.884   9.546  1.00  0.00           H  \\nATOM   2031 HG12 VAL A 138       4.222  -4.710  10.058  1.00  0.00           H  \\nATOM   2032 HG13 VAL A 138       5.727  -4.932  10.984  1.00  0.00           H  \\nATOM   2033 HG21 VAL A 138       7.568  -5.179   8.528  1.00  0.00           H  \\nATOM   2034 HG22 VAL A 138       7.633  -6.272   9.930  1.00  0.00           H  \\nATOM   2035 HG23 VAL A 138       7.410  -6.936   8.294  1.00  0.00           H  \\nATOM   2036  N   LEU A 139       2.724  -7.133   9.077  1.00  0.00           N  \\nATOM   2037  CA  LEU A 139       1.280  -7.011   9.299  1.00  0.00           C  \\nATOM   2038  C   LEU A 139       0.729  -8.161  10.146  1.00  0.00           C  \\nATOM   2039  O   LEU A 139      -0.066  -7.925  11.050  1.00  0.00           O  \\nATOM   2040  CB  LEU A 139       0.544  -6.968   7.954  1.00  0.00           C  \\nATOM   2041  CG  LEU A 139       0.682  -5.662   7.162  1.00  0.00           C  \\nATOM   2042  CD1 LEU A 139       0.066  -5.838   5.781  1.00  0.00           C  \\nATOM   2043  CD2 LEU A 139       0.005  -4.532   7.923  1.00  0.00           C  \\nATOM   2044  H   LEU A 139       3.054  -7.240   8.129  1.00  0.00           H  \\nATOM   2045  HA  LEU A 139       1.090  -6.098   9.863  1.00  0.00           H  \\nATOM   2046  HB3 LEU A 139       0.917  -7.776   7.328  1.00  0.00           H  \\nATOM   2047  HB2 LEU A 139      -0.518  -7.136   8.133  1.00  0.00           H  \\nATOM   2048  HG  LEU A 139       1.738  -5.427   7.028  1.00  0.00           H  \\nATOM   2049 HD11 LEU A 139       0.164  -4.909   5.217  1.00  0.00           H  \\nATOM   2050 HD12 LEU A 139       0.582  -6.639   5.251  1.00  0.00           H  \\nATOM   2051 HD13 LEU A 139      -0.989  -6.090   5.883  1.00  0.00           H  \\nATOM   2052 HD21 LEU A 139       0.104  -3.603   7.360  1.00  0.00           H  \\nATOM   2053 HD22 LEU A 139      -1.052  -4.765   8.056  1.00  0.00           H  \\nATOM   2054 HD23 LEU A 139       0.477  -4.417   8.899  1.00  0.00           H  \\nATOM   2055  N   ASN A 140       1.165  -9.401   9.906  1.00  0.00           N  \\nATOM   2056  CA  ASN A 140       0.760 -10.536  10.737  1.00  0.00           C  \\nATOM   2057  C   ASN A 140       1.237 -10.385  12.188  1.00  0.00           C  \\nATOM   2058  O   ASN A 140       0.470 -10.672  13.111  1.00  0.00           O  \\nATOM   2059  CB  ASN A 140       1.273 -11.834  10.141  1.00  0.00           C  \\nATOM   2060  CG  ASN A 140       0.507 -12.251   8.916  1.00  0.00           C  \\nATOM   2061  OD1 ASN A 140      -0.633 -11.821   8.706  1.00  0.00           O  \\nATOM   2062  ND2 ASN A 140       1.110 -13.080   8.103  1.00  0.00           N  \\nATOM   2063  H   ASN A 140       1.796  -9.553   9.133  1.00  0.00           H  \\nATOM   2064  HA  ASN A 140      -0.329 -10.569  10.780  1.00  0.00           H  \\nATOM   2065  HB3 ASN A 140       2.325 -11.722   9.876  1.00  0.00           H  \\nATOM   2066  HB2 ASN A 140       1.206 -12.628  10.885  1.00  0.00           H  \\nATOM   2067 HD21 ASN A 140       0.648 -13.391   7.273  1.00  0.00           H  \\nATOM   2068 HD22 ASN A 140       2.033 -13.402   8.313  1.00  0.00           H  \\nATOM   2069  N   GLY A 141       2.462  -9.887  12.390  1.00  0.00           N  \\nATOM   2070  CA  GLY A 141       2.977  -9.525  13.711  1.00  0.00           C  \\nATOM   2071  C   GLY A 141       2.135  -8.434  14.371  1.00  0.00           C  \\nATOM   2072  O   GLY A 141       1.652  -8.626  15.488  1.00  0.00           O  \\nATOM   2073  H   GLY A 141       3.045  -9.699  11.588  1.00  0.00           H  \\nATOM   2074  HA3 GLY A 141       3.995  -9.153  13.605  1.00  0.00           H  \\nATOM   2075  HA2 GLY A 141       2.986 -10.404  14.358  1.00  0.00           H  \\nATOM   2076  N   PHE A 142       1.864  -7.343  13.646  1.00  0.00           N  \\nATOM   2077  CA  PHE A 142       0.998  -6.256  14.101  1.00  0.00           C  \\nATOM   2078  C   PHE A 142      -0.388  -6.773  14.494  1.00  0.00           C  \\nATOM   2079  O   PHE A 142      -0.848  -6.483  15.592  1.00  0.00           O  \\nATOM   2080  CB  PHE A 142       0.895  -5.169  13.018  1.00  0.00           C  \\nATOM   2081  CG  PHE A 142      -0.046  -4.031  13.377  1.00  0.00           C  \\nATOM   2082  CD1 PHE A 142      -1.385  -4.053  12.941  1.00  0.00           C  \\nATOM   2083  CD2 PHE A 142       0.413  -2.951  14.153  1.00  0.00           C  \\nATOM   2084  CE1 PHE A 142      -2.270  -3.021  13.306  1.00  0.00           C  \\nATOM   2085  CE2 PHE A 142      -0.463  -1.909  14.498  1.00  0.00           C  \\nATOM   2086  CZ  PHE A 142      -1.807  -1.947  14.087  1.00  0.00           C  \\nATOM   2087  H   PHE A 142       2.304  -7.249  12.742  1.00  0.00           H  \\nATOM   2088  HA  PHE A 142       1.442  -5.808  14.990  1.00  0.00           H  \\nATOM   2089  HB3 PHE A 142       0.549  -5.613  12.085  1.00  0.00           H  \\nATOM   2090  HB2 PHE A 142       1.889  -4.763  12.832  1.00  0.00           H  \\nATOM   2091  HD1 PHE A 142      -1.731  -4.864  12.318  1.00  0.00           H  \\nATOM   2092  HD2 PHE A 142       1.442  -2.905  14.481  1.00  0.00           H  \\nATOM   2093  HE1 PHE A 142      -3.297  -3.041  12.972  1.00  0.00           H  \\nATOM   2094  HE2 PHE A 142      -0.096  -1.072  15.072  1.00  0.00           H  \\nATOM   2095  HZ  PHE A 142      -2.471  -1.138  14.354  1.00  0.00           H  \\nATOM   2096  N   TYR A 143      -1.056  -7.564  13.653  1.00  0.00           N  \\nATOM   2097  CA  TYR A 143      -2.395  -8.064  13.963  1.00  0.00           C  \\nATOM   2098  C   TYR A 143      -2.410  -8.978  15.189  1.00  0.00           C  \\nATOM   2099  O   TYR A 143      -3.325  -8.872  16.006  1.00  0.00           O  \\nATOM   2100  CB  TYR A 143      -2.974  -8.802  12.754  1.00  0.00           C  \\nATOM   2101  CG  TYR A 143      -3.240  -7.908  11.563  1.00  0.00           C  \\nATOM   2102  CD1 TYR A 143      -3.059  -8.394  10.276  1.00  0.00           C  \\nATOM   2103  CD2 TYR A 143      -3.666  -6.603  11.757  1.00  0.00           C  \\nATOM   2104  CE1 TYR A 143      -3.302  -7.578   9.189  1.00  0.00           C  \\nATOM   2105  CE2 TYR A 143      -3.910  -5.787  10.670  1.00  0.00           C  \\nATOM   2106  CZ  TYR A 143      -3.729  -6.270   9.390  1.00  0.00           C  \\nATOM   2107  OH  TYR A 143      -3.971  -5.457   8.307  1.00  0.00           O  \\nATOM   2108  H   TYR A 143      -0.652  -7.761  12.748  1.00  0.00           H  \\nATOM   2109  HA  TYR A 143      -3.034  -7.212  14.200  1.00  0.00           H  \\nATOM   2110  HB3 TYR A 143      -2.283  -9.588  12.442  1.00  0.00           H  \\nATOM   2111  HB2 TYR A 143      -3.910  -9.283  13.035  1.00  0.00           H  \\nATOM   2112  HD1 TYR A 143      -2.724  -9.421  10.123  1.00  0.00           H  \\nATOM   2113  HD2 TYR A 143      -3.808  -6.221  12.768  1.00  0.00           H  \\nATOM   2114  HE1 TYR A 143      -3.160  -7.960   8.178  1.00  0.00           H  \\nATOM   2115  HE2 TYR A 143      -4.244  -4.760  10.822  1.00  0.00           H  \\nATOM   2116  HH  TYR A 143      -3.682  -5.899   7.505  1.00  0.00           H  \\nATOM   2117  N   SER A 144      -1.407  -9.851  15.324  1.00  0.00           N  \\nATOM   2118  CA  SER A 144      -1.276 -10.759  16.468  1.00  0.00           C  \\nATOM   2119  C   SER A 144      -1.010  -9.986  17.758  1.00  0.00           C  \\nATOM   2120  O   SER A 144      -1.738 -10.148  18.742  1.00  0.00           O  \\nATOM   2121  CB  SER A 144      -0.156 -11.751  16.224  1.00  0.00           C  \\nATOM   2122  OG  SER A 144      -0.445 -12.575  15.128  1.00  0.00           O  \\nATOM   2123  H   SER A 144      -0.688  -9.869  14.615  1.00  0.00           H  \\nATOM   2124  HA  SER A 144      -2.207 -11.312  16.594  1.00  0.00           H  \\nATOM   2125  HB3 SER A 144       0.774 -11.213  16.043  1.00  0.00           H  \\nATOM   2126  HB2 SER A 144      -0.012 -12.363  17.113  1.00  0.00           H  \\nATOM   2127  HG  SER A 144      -0.207 -12.069  14.347  1.00  0.00           H  \\nATOM   2128  N   ALA A 145      -0.053  -9.053  17.718  1.00  0.00           N  \\nATOM   2129  CA  ALA A 145       0.334  -8.247  18.870  1.00  0.00           C  \\nATOM   2130  C   ALA A 145      -0.849  -7.419  19.379  1.00  0.00           C  \\nATOM   2131  O   ALA A 145      -1.035  -7.283  20.589  1.00  0.00           O  \\nATOM   2132  CB  ALA A 145       1.501  -7.341  18.466  1.00  0.00           C  \\nATOM   2133  H   ALA A 145       0.432  -8.883  16.849  1.00  0.00           H  \\nATOM   2134  HA  ALA A 145       0.659  -8.906  19.674  1.00  0.00           H  \\nATOM   2135  HB1 ALA A 145       2.345  -7.947  18.135  1.00  0.00           H  \\nATOM   2136  HB2 ALA A 145       1.816  -6.743  19.321  1.00  0.00           H  \\nATOM   2137  HB3 ALA A 145       1.202  -6.675  17.658  1.00  0.00           H  \\nATOM   2138  N   ASN A 146      -1.696  -6.931  18.463  1.00  0.00           N  \\nATOM   2139  CA  ASN A 146      -2.853  -6.113  18.783  1.00  0.00           C  \\nATOM   2140  C   ASN A 146      -4.203  -6.837  18.821  1.00  0.00           C  \\nATOM   2141  O   ASN A 146      -5.204  -6.171  19.058  1.00  0.00           O  \\nATOM   2142  CB  ASN A 146      -2.923  -4.957  17.801  1.00  0.00           C  \\nATOM   2143  CG  ASN A 146      -1.819  -3.957  18.007  1.00  0.00           C  \\nATOM   2144  OD1 ASN A 146      -1.897  -3.101  18.895  1.00  0.00           O  \\nATOM   2145  ND2 ASN A 146      -0.791  -4.049  17.202  1.00  0.00           N  \\nATOM   2146  H   ASN A 146      -1.428  -7.002  17.492  1.00  0.00           H  \\nATOM   2147  HA  ASN A 146      -2.690  -5.809  19.817  1.00  0.00           H  \\nATOM   2148  HB3 ASN A 146      -2.865  -5.341  16.782  1.00  0.00           H  \\nATOM   2149  HB2 ASN A 146      -3.881  -4.449  17.906  1.00  0.00           H  \\nATOM   2150 HD21 ASN A 146      -0.026  -3.410  17.292  1.00  0.00           H  \\nATOM   2151 HD22 ASN A 146      -0.769  -4.758  16.497  1.00  0.00           H  \\nATOM   2152  N   LYS A 147      -4.240  -8.176  18.770  1.00  0.00           N  \\nATOM   2153  CA  LYS A 147      -5.483  -8.976  18.806  1.00  0.00           C  \\nATOM   2154  C   LYS A 147      -6.518  -8.500  17.773  1.00  0.00           C  \\nATOM   2155  O   LYS A 147      -7.697  -8.366  18.080  1.00  0.00           O  \\nATOM   2156  CB  LYS A 147      -6.097  -8.938  20.206  1.00  0.00           C  \\nATOM   2157  CG  LYS A 147      -5.144  -9.343  21.323  1.00  0.00           C  \\nATOM   2158  CD  LYS A 147      -4.682 -10.783  21.160  1.00  0.00           C  \\nATOM   2159  CE  LYS A 147      -3.755 -11.199  22.292  1.00  0.00           C  \\nATOM   2160  NZ  LYS A 147      -3.246 -12.586  22.114  1.00  0.00           N  \\nATOM   2161  H   LYS A 147      -3.385  -8.655  18.526  1.00  0.00           H  \\nATOM   2162  HA  LYS A 147      -5.218  -9.987  18.494  1.00  0.00           H  \\nATOM   2163  HB3 LYS A 147      -6.453  -7.930  20.420  1.00  0.00           H  \\nATOM   2164  HB2 LYS A 147      -6.959  -9.604  20.244  1.00  0.00           H  \\nATOM   2165  HG3 LYS A 147      -4.272  -8.687  21.313  1.00  0.00           H  \\nATOM   2166  HG2 LYS A 147      -5.645  -9.237  22.285  1.00  0.00           H  \\nATOM   2167  HD3 LYS A 147      -5.550 -11.445  21.150  1.00  0.00           H  \\nATOM   2168  HD2 LYS A 147      -4.154 -10.891  20.213  1.00  0.00           H  \\nATOM   2169  HE3 LYS A 147      -2.907 -10.517  22.337  1.00  0.00           H  \\nATOM   2170  HE2 LYS A 147      -4.290 -11.142  23.240  1.00  0.00           H  \\nATOM   2171  HZ1 LYS A 147      -2.637 -12.824  22.884  1.00  0.00           H  \\nATOM   2172  HZ2 LYS A 147      -4.024 -13.231  22.089  1.00  0.00           H  \\nATOM   2173  HZ3 LYS A 147      -2.732 -12.647  21.247  1.00  0.00           H  \\nATOM   2174  N   VAL A 148      -6.075  -8.241  16.541  1.00  0.00           N  \\nATOM   2175  CA  VAL A 148      -6.955  -7.812  15.445  1.00  0.00           C  \\nATOM   2176  C   VAL A 148      -7.574  -9.045  14.791  1.00  0.00           C  \\nATOM   2177  O   VAL A 148      -6.926  -9.741  13.998  1.00  0.00           O  \\nATOM   2178  CB  VAL A 148      -6.212  -6.934  14.423  1.00  0.00           C  \\nATOM   2179  CG1 VAL A 148      -7.126  -6.506  13.268  1.00  0.00           C  \\nATOM   2180  CG2 VAL A 148      -5.668  -5.662  15.084  1.00  0.00           C  \\nATOM   2181  H   VAL A 148      -5.107  -8.448  16.339  1.00  0.00           H  \\nATOM   2182  HA  VAL A 148      -7.761  -7.210  15.863  1.00  0.00           H  \\nATOM   2183  HB  VAL A 148      -5.377  -7.499  14.012  1.00  0.00           H  \\nATOM   2184 HG11 VAL A 148      -6.565  -5.888  12.567  1.00  0.00           H  \\nATOM   2185 HG12 VAL A 148      -7.971  -5.932  13.649  1.00  0.00           H  \\nATOM   2186 HG13 VAL A 148      -7.502  -7.378  12.732  1.00  0.00           H  \\nATOM   2187 HG21 VAL A 148      -5.165  -5.038  14.345  1.00  0.00           H  \\nATOM   2188 HG22 VAL A 148      -6.484  -5.094  15.531  1.00  0.00           H  \\nATOM   2189 HG23 VAL A 148      -4.954  -5.925  15.864  1.00  0.00           H  \\nATOM   2190  N   ALA A 149      -8.835  -9.313  15.127  1.00  0.00           N  \\nATOM   2191  CA  ALA A 149      -9.547 -10.497  14.684  1.00  0.00           C  \\nATOM   2192  C   ALA A 149     -10.089 -10.326  13.260  1.00  0.00           C  \\nATOM   2193  O   ALA A 149     -10.545  -9.252  12.862  1.00  0.00           O  \\nATOM   2194  CB  ALA A 149     -10.658 -10.844  15.679  1.00  0.00           C  \\nATOM   2195  H   ALA A 149      -9.335  -8.670  15.726  1.00  0.00           H  \\nATOM   2196  HA  ALA A 149      -8.842 -11.330  14.683  1.00  0.00           H  \\nATOM   2197  HB1 ALA A 149     -11.114 -11.797  15.409  1.00  0.00           H  \\nATOM   2198  HB2 ALA A 149     -10.250 -10.916  16.687  1.00  0.00           H  \\nATOM   2199  HB3 ALA A 149     -11.432 -10.077  15.660  1.00  0.00           H  \\nATOM   2200  N   GLN A 150     -10.054 -11.425  12.510  1.00  0.00           N  \\nATOM   2201  CA  GLN A 150     -10.719 -11.524  11.216  1.00  0.00           C  \\nATOM   2202  C   GLN A 150     -12.223 -11.754  11.397  1.00  0.00           C  \\nATOM   2203  O   GLN A 150     -12.641 -12.365  12.380  1.00  0.00           O  \\nATOM   2204  CB  GLN A 150     -10.107 -12.655  10.384  1.00  0.00           C  \\nATOM   2205  CG  GLN A 150      -8.681 -12.392   9.931  1.00  0.00           C  \\nATOM   2206  CD  GLN A 150      -8.074 -13.581   9.210  1.00  0.00           C  \\nATOM   2207  OE1 GLN A 150      -7.768 -14.607   9.824  1.00  0.00           O  \\nATOM   2208  NE2 GLN A 150      -7.897 -13.451   7.901  1.00  0.00           N  \\nATOM   2209  H   GLN A 150      -9.756 -12.283  12.950  1.00  0.00           H  \\nATOM   2210  HA  GLN A 150     -10.576 -10.577  10.695  1.00  0.00           H  \\nATOM   2211  HB3 GLN A 150     -10.111 -13.577  10.965  1.00  0.00           H  \\nATOM   2212  HB2 GLN A 150     -10.716 -12.824   9.496  1.00  0.00           H  \\nATOM   2213  HG3 GLN A 150      -8.677 -11.541   9.250  1.00  0.00           H  \\nATOM   2214  HG2 GLN A 150      -8.067 -12.173  10.805  1.00  0.00           H  \\nATOM   2215 HE21 GLN A 150      -7.501 -14.202   7.371  1.00  0.00           H  \\nATOM   2216 HE22 GLN A 150      -8.159 -12.601   7.442  1.00  0.00           H  \\nATOM   2217  N   PHE A 151     -13.023 -11.360  10.410  1.00  0.00           N  \\nATOM   2218  CA  PHE A 151     -14.429 -11.760  10.317  1.00  0.00           C  \\nATOM   2219  C   PHE A 151     -14.853 -12.054   8.881  1.00  0.00           C  \\nATOM   2220  O   PHE A 151     -14.134 -11.750   7.928  1.00  0.00           O  \\nATOM   2221  CB  PHE A 151     -15.328 -10.667  10.898  1.00  0.00           C  \\nATOM   2222  CG  PHE A 151     -15.500  -9.480   9.995  1.00  0.00           C  \\nATOM   2223  CD1 PHE A 151     -16.709  -9.244   9.357  1.00  0.00           C  \\nATOM   2224  CD2 PHE A 151     -14.453  -8.596   9.779  1.00  0.00           C  \\nATOM   2225  CE1 PHE A 151     -16.868  -8.152   8.525  1.00  0.00           C  \\nATOM   2226  CE2 PHE A 151     -14.609  -7.503   8.950  1.00  0.00           C  \\nATOM   2227  CZ  PHE A 151     -15.819  -7.281   8.322  1.00  0.00           C  \\nATOM   2228  H   PHE A 151     -12.646 -10.778   9.676  1.00  0.00           H  \\nATOM   2229  HA  PHE A 151     -14.564 -12.690  10.871  1.00  0.00           H  \\nATOM   2230  HB3 PHE A 151     -16.314 -11.081  11.107  1.00  0.00           H  \\nATOM   2231  HB2 PHE A 151     -14.913 -10.319  11.843  1.00  0.00           H  \\nATOM   2232  HD1 PHE A 151     -17.539  -9.932   9.519  1.00  0.00           H  \\nATOM   2233  HD2 PHE A 151     -13.498  -8.772  10.275  1.00  0.00           H  \\nATOM   2234  HE1 PHE A 151     -17.823  -7.979   8.031  1.00  0.00           H  \\nATOM   2235  HE2 PHE A 151     -13.779  -6.816   8.789  1.00  0.00           H  \\nATOM   2236  HZ  PHE A 151     -15.943  -6.421   7.666  1.00  0.00           H  \\nATOM   2237  N   ASP A 152     -16.015 -12.681   8.735  1.00  0.00           N  \\nATOM   2238  CA  ASP A 152     -16.634 -12.996   7.457  1.00  0.00           C  \\nATOM   2239  C   ASP A 152     -17.738 -11.966   7.163  1.00  0.00           C  \\nATOM   2240  O   ASP A 152     -18.811 -12.027   7.773  1.00  0.00           O  \\nATOM   2241  CB  ASP A 152     -17.212 -14.413   7.466  1.00  0.00           C  \\nATOM   2242  CG  ASP A 152     -17.768 -14.835   6.113  1.00  0.00           C  \\nATOM   2243  OD1 ASP A 152     -18.158 -13.976   5.358  1.00  0.00           O  \\nATOM   2244  OD2 ASP A 152     -17.797 -16.013   5.846  1.00  0.00           O  \\nATOM   2245  H   ASP A 152     -16.610 -12.790   9.544  1.00  0.00           H  \\nATOM   2246  HA  ASP A 152     -15.884 -12.965   6.666  1.00  0.00           H  \\nATOM   2247  HB3 ASP A 152     -16.436 -15.121   7.760  1.00  0.00           H  \\nATOM   2248  HB2 ASP A 152     -18.010 -14.476   8.206  1.00  0.00           H  \\nATOM   2249  N   PRO A 153     -17.530 -11.032   6.217  1.00  0.00           N  \\nATOM   2250  CA  PRO A 153     -18.520 -10.006   5.903  1.00  0.00           C  \\nATOM   2251  C   PRO A 153     -19.868 -10.568   5.443  1.00  0.00           C  \\nATOM   2252  O   PRO A 153     -20.870  -9.868   5.528  1.00  0.00           O  \\nATOM   2253  CB  PRO A 153     -17.893  -9.139   4.808  1.00  0.00           C  \\nATOM   2254  CG  PRO A 153     -16.394  -9.313   5.019  1.00  0.00           C  \\nATOM   2255  CD  PRO A 153     -16.281 -10.775   5.430  1.00  0.00           C  \\nATOM   2256  HA  PRO A 153     -18.690  -9.400   6.793  1.00  0.00           H  \\nATOM   2257  HB3 PRO A 153     -18.159  -9.523   3.823  1.00  0.00           H  \\nATOM   2258  HB2 PRO A 153     -18.193  -8.094   4.900  1.00  0.00           H  \\nATOM   2259  HG3 PRO A 153     -16.058  -8.678   5.838  1.00  0.00           H  \\nATOM   2260  HG2 PRO A 153     -15.836  -9.096   4.108  1.00  0.00           H  \\nATOM   2261  HD3 PRO A 153     -15.370 -10.917   6.010  1.00  0.00           H  \\nATOM   2262  HD2 PRO A 153     -16.281 -11.412   4.546  1.00  0.00           H  \\nATOM   2263  N   SER A 154     -19.923 -11.826   4.987  1.00  0.00           N  \\nATOM   2264  CA  SER A 154     -21.178 -12.460   4.564  1.00  0.00           C  \\nATOM   2265  C   SER A 154     -21.995 -13.025   5.734  1.00  0.00           C  \\nATOM   2266  O   SER A 154     -23.140 -13.433   5.547  1.00  0.00           O  \\nATOM   2267  CB  SER A 154     -20.918 -13.521   3.488  1.00  0.00           C  \\nATOM   2268  OG  SER A 154     -20.261 -14.681   3.967  1.00  0.00           O  \\nATOM   2269  H   SER A 154     -19.097 -12.404   5.048  1.00  0.00           H  \\nATOM   2270  HA  SER A 154     -21.802 -11.694   4.105  1.00  0.00           H  \\nATOM   2271  HB3 SER A 154     -20.326 -13.076   2.689  1.00  0.00           H  \\nATOM   2272  HB2 SER A 154     -21.879 -13.821   3.067  1.00  0.00           H  \\nATOM   2273  HG  SER A 154     -19.464 -14.430   4.439  1.00  0.00           H  \\nATOM   2274  N   LYS A 155     -21.428 -13.041   6.950  1.00  0.00           N  \\nATOM   2275  CA  LYS A 155     -22.088 -13.453   8.196  1.00  0.00           C  \\nATOM   2276  C   LYS A 155     -21.760 -12.493   9.357  1.00  0.00           C  \\nATOM   2277  O   LYS A 155     -21.146 -12.907  10.344  1.00  0.00           O  \\nATOM   2278  CB  LYS A 155     -21.682 -14.880   8.565  1.00  0.00           C  \\nATOM   2279  CG  LYS A 155     -22.094 -15.935   7.546  1.00  0.00           C  \\nATOM   2280  CD  LYS A 155     -21.609 -17.317   7.955  1.00  0.00           C  \\nATOM   2281  CE  LYS A 155     -20.102 -17.448   7.792  1.00  0.00           C  \\nATOM   2282  NZ  LYS A 155     -19.699 -17.488   6.360  1.00  0.00           N  \\nATOM   2283  H   LYS A 155     -20.460 -12.757   7.014  1.00  0.00           H  \\nATOM   2284  HA  LYS A 155     -23.168 -13.426   8.053  1.00  0.00           H  \\nATOM   2285  HB3 LYS A 155     -20.599 -14.932   8.681  1.00  0.00           H  \\nATOM   2286  HB2 LYS A 155     -22.127 -15.149   9.523  1.00  0.00           H  \\nATOM   2287  HG3 LYS A 155     -23.181 -15.951   7.458  1.00  0.00           H  \\nATOM   2288  HG2 LYS A 155     -21.672 -15.684   6.573  1.00  0.00           H  \\nATOM   2289  HD3 LYS A 155     -21.870 -17.501   8.999  1.00  0.00           H  \\nATOM   2290  HD2 LYS A 155     -22.098 -18.072   7.339  1.00  0.00           H  \\nATOM   2291  HE3 LYS A 155     -19.610 -16.603   8.272  1.00  0.00           H  \\nATOM   2292  HE2 LYS A 155     -19.761 -18.362   8.277  1.00  0.00           H  \\nATOM   2293  HZ1 LYS A 155     -18.695 -17.575   6.295  1.00  0.00           H  \\nATOM   2294  HZ2 LYS A 155     -20.135 -18.280   5.909  1.00  0.00           H  \\nATOM   2295  HZ3 LYS A 155     -19.993 -16.637   5.903  1.00  0.00           H  \\nATOM   2296  N   PRO A 156     -22.199 -11.220   9.302  1.00  0.00           N  \\nATOM   2297  CA  PRO A 156     -21.924 -10.234  10.354  1.00  0.00           C  \\nATOM   2298  C   PRO A 156     -22.411 -10.662  11.747  1.00  0.00           C  \\nATOM   2299  O   PRO A 156     -21.840 -10.250  12.748  1.00  0.00           O  \\nATOM   2300  CB  PRO A 156     -22.688  -9.004   9.855  1.00  0.00           C  \\nATOM   2301  CG  PRO A 156     -22.684  -9.148   8.371  1.00  0.00           C  \\nATOM   2302  CD  PRO A 156     -22.884 -10.622   8.138  1.00  0.00           C  \\nATOM   2303  HA  PRO A 156     -20.851 -10.057  10.417  1.00  0.00           H  \\nATOM   2304  HB3 PRO A 156     -23.703  -8.993  10.280  1.00  0.00           H  \\nATOM   2305  HB2 PRO A 156     -22.187  -8.086  10.196  1.00  0.00           H  \\nATOM   2306  HG3 PRO A 156     -23.484  -8.537   7.928  1.00  0.00           H  \\nATOM   2307  HG2 PRO A 156     -21.735  -8.778   7.957  1.00  0.00           H  \\nATOM   2308  HD3 PRO A 156     -23.960 -10.850   8.135  1.00  0.00           H  \\nATOM   2309  HD2 PRO A 156     -22.423 -10.907   7.181  1.00  0.00           H  \\nATOM   2310  N   GLN A 157     -23.429 -11.526  11.824  1.00  0.00           N  \\nATOM   2311  CA  GLN A 157     -24.011 -11.985  13.093  1.00  0.00           C  \\nATOM   2312  C   GLN A 157     -23.277 -13.190  13.711  1.00  0.00           C  \\nATOM   2313  O   GLN A 157     -23.572 -13.573  14.836  1.00  0.00           O  \\nATOM   2314  CB  GLN A 157     -25.486 -12.341  12.886  1.00  0.00           C  \\nATOM   2315  CG  GLN A 157     -26.365 -11.158  12.516  1.00  0.00           C  \\nATOM   2316  CD  GLN A 157     -27.811 -11.560  12.290  1.00  0.00           C  \\nATOM   2317  OE1 GLN A 157     -28.105 -12.712  11.958  1.00  0.00           O  \\nATOM   2318  NE2 GLN A 157     -28.722 -10.611  12.469  1.00  0.00           N  \\nATOM   2319  H   GLN A 157     -23.822 -11.877  10.962  1.00  0.00           H  \\nATOM   2320  HA  GLN A 157     -23.937 -11.182  13.826  1.00  0.00           H  \\nATOM   2321  HB3 GLN A 157     -25.573 -13.085  12.094  1.00  0.00           H  \\nATOM   2322  HB2 GLN A 157     -25.885 -12.784  13.798  1.00  0.00           H  \\nATOM   2323  HG3 GLN A 157     -26.336 -10.429  13.325  1.00  0.00           H  \\nATOM   2324  HG2 GLN A 157     -25.987 -10.711  11.596  1.00  0.00           H  \\nATOM   2325 HE21 GLN A 157     -29.693 -10.817  12.334  1.00  0.00           H  \\nATOM   2326 HE22 GLN A 157     -28.440  -9.690  12.738  1.00  0.00           H  \\nATOM   2327  N   GLN A 158     -22.346 -13.816  12.982  1.00  0.00           N  \\nATOM   2328  CA  GLN A 158     -21.609 -15.007  13.437  1.00  0.00           C  \\nATOM   2329  C   GLN A 158     -20.145 -14.700  13.763  1.00  0.00           C  \\nATOM   2330  O   GLN A 158     -19.543 -15.370  14.598  1.00  0.00           O  \\nATOM   2331  CB  GLN A 158     -21.678 -16.109  12.377  1.00  0.00           C  \\nATOM   2332  CG  GLN A 158     -23.087 -16.570  12.048  1.00  0.00           C  \\nATOM   2333  CD  GLN A 158     -23.769 -17.237  13.228  1.00  0.00           C  \\nATOM   2334  OE1 GLN A 158     -23.254 -18.205  13.794  1.00  0.00           O  \\nATOM   2335  NE2 GLN A 158     -24.933 -16.722  13.607  1.00  0.00           N  \\nATOM   2336  H   GLN A 158     -22.119 -13.442  12.072  1.00  0.00           H  \\nATOM   2337  HA  GLN A 158     -22.061 -15.394  14.351  1.00  0.00           H  \\nATOM   2338  HB3 GLN A 158     -21.215 -15.757  11.456  1.00  0.00           H  \\nATOM   2339  HB2 GLN A 158     -21.111 -16.976  12.717  1.00  0.00           H  \\nATOM   2340  HG3 GLN A 158     -23.682 -15.705  11.755  1.00  0.00           H  \\nATOM   2341  HG2 GLN A 158     -23.042 -17.287  11.229  1.00  0.00           H  \\nATOM   2342 HE21 GLN A 158     -25.429 -17.120  14.379  1.00  0.00           H  \\nATOM   2343 HE22 GLN A 158     -25.315 -15.936  13.120  1.00  0.00           H  \\nATOM   2344  N   THR A 159     -19.557 -13.702  13.101  1.00  0.00           N  \\nATOM   2345  CA  THR A 159     -18.144 -13.347  13.260  1.00  0.00           C  \\nATOM   2346  C   THR A 159     -18.006 -11.856  13.538  1.00  0.00           C  \\nATOM   2347  O   THR A 159     -18.365 -11.043  12.687  1.00  0.00           O  \\nATOM   2348  CB  THR A 159     -17.324 -13.721  12.012  1.00  0.00           C  \\nATOM   2349  OG1 THR A 159     -17.873 -13.065  10.861  1.00  0.00           O  \\nATOM   2350  CG2 THR A 159     -17.345 -15.226  11.791  1.00  0.00           C  \\nATOM   2351  H   THR A 159     -20.104 -13.183  12.430  1.00  0.00           H  \\nATOM   2352  HA  THR A 159     -17.723 -13.887  14.108  1.00  0.00           H  \\nATOM   2353  HB  THR A 159     -16.292 -13.394  12.142  1.00  0.00           H  \\nATOM   2354  HG1 THR A 159     -18.830 -13.041  10.934  1.00  0.00           H  \\nATOM   2355 HG21 THR A 159     -16.760 -15.472  10.905  1.00  0.00           H  \\nATOM   2356 HG22 THR A 159     -16.918 -15.727  12.659  1.00  0.00           H  \\nATOM   2357 HG23 THR A 159     -18.373 -15.558  11.650  1.00  0.00           H  \\nATOM   2358  N   LYS A 160     -17.446 -11.495  14.699  1.00  0.00           N  \\nATOM   2359  CA  LYS A 160     -17.205 -10.099  15.084  1.00  0.00           C  \\nATOM   2360  C   LYS A 160     -15.801  -9.682  14.648  1.00  0.00           C  \\nATOM   2361  O   LYS A 160     -14.817 -10.269  15.088  1.00  0.00           O  \\nATOM   2362  CB  LYS A 160     -17.373  -9.909  16.593  1.00  0.00           C  \\nATOM   2363  CG  LYS A 160     -17.154  -8.481  17.075  1.00  0.00           C  \\nATOM   2364  CD  LYS A 160     -17.309  -8.379  18.585  1.00  0.00           C  \\nATOM   2365  CE  LYS A 160     -16.922  -6.996  19.089  1.00  0.00           C  \\nATOM   2366  NZ  LYS A 160     -16.961  -6.914  20.575  1.00  0.00           N  \\nATOM   2367  H   LYS A 160     -17.122 -12.218  15.325  1.00  0.00           H  \\nATOM   2368  HA  LYS A 160     -17.928  -9.466  14.571  1.00  0.00           H  \\nATOM   2369  HB3 LYS A 160     -18.378 -10.211  16.888  1.00  0.00           H  \\nATOM   2370  HB2 LYS A 160     -16.669 -10.553  17.121  1.00  0.00           H  \\nATOM   2371  HG3 LYS A 160     -16.151  -8.153  16.798  1.00  0.00           H  \\nATOM   2372  HG2 LYS A 160     -17.877  -7.821  16.598  1.00  0.00           H  \\nATOM   2373  HD3 LYS A 160     -18.345  -8.579  18.859  1.00  0.00           H  \\nATOM   2374  HD2 LYS A 160     -16.673  -9.121  19.068  1.00  0.00           H  \\nATOM   2375  HE3 LYS A 160     -15.916  -6.754  18.751  1.00  0.00           H  \\nATOM   2376  HE2 LYS A 160     -17.608  -6.254  18.679  1.00  0.00           H  \\nATOM   2377  HZ1 LYS A 160     -16.699  -5.983  20.868  1.00  0.00           H  \\nATOM   2378  HZ2 LYS A 160     -17.896  -7.119  20.899  1.00  0.00           H  \\nATOM   2379  HZ3 LYS A 160     -16.315  -7.584  20.966  1.00  0.00           H  \\nATOM   2380  N   GLY A 161     -15.716  -8.682  13.776  1.00  0.00           N  \\nATOM   2381  CA  GLY A 161     -14.441  -8.194  13.250  1.00  0.00           C  \\nATOM   2382  C   GLY A 161     -13.876  -7.063  14.089  1.00  0.00           C  \\nATOM   2383  O   GLY A 161     -14.632  -6.245  14.608  1.00  0.00           O  \\nATOM   2384  H   GLY A 161     -16.562  -8.198  13.511  1.00  0.00           H  \\nATOM   2385  HA3 GLY A 161     -13.712  -9.002  13.222  1.00  0.00           H  \\nATOM   2386  HA2 GLY A 161     -14.583  -7.820  12.237  1.00  0.00           H  \\nATOM   2387  N   THR A 162     -12.551  -6.972  14.150  1.00  0.00           N  \\nATOM   2388  CA  THR A 162     -11.861  -5.922  14.904  1.00  0.00           C  \\nATOM   2389  C   THR A 162     -11.538  -4.742  14.000  1.00  0.00           C  \\nATOM   2390  O   THR A 162     -10.920  -4.908  12.943  1.00  0.00           O  \\nATOM   2391  CB  THR A 162     -10.578  -6.465  15.534  1.00  0.00           C  \\nATOM   2392  OG1 THR A 162     -10.863  -7.546  16.387  1.00  0.00           O  \\nATOM   2393  CG2 THR A 162      -9.847  -5.443  16.388  1.00  0.00           C  \\nATOM   2394  H   THR A 162     -11.986  -7.659  13.672  1.00  0.00           H  \\nATOM   2395  HA  THR A 162     -12.501  -5.572  15.714  1.00  0.00           H  \\nATOM   2396  HB  THR A 162      -9.913  -6.813  14.744  1.00  0.00           H  \\nATOM   2397  HG1 THR A 162     -11.314  -7.193  17.157  1.00  0.00           H  \\nATOM   2398 HG21 THR A 162      -9.447  -4.643  15.764  1.00  0.00           H  \\nATOM   2399 HG22 THR A 162      -9.021  -5.939  16.896  1.00  0.00           H  \\nATOM   2400 HG23 THR A 162     -10.523  -5.024  17.133  1.00  0.00           H  \\nATOM   2401  N   TRP A 163     -11.937  -3.549  14.430  1.00  0.00           N  \\nATOM   2402  CA  TRP A 163     -11.526  -2.291  13.820  1.00  0.00           C  \\nATOM   2403  C   TRP A 163     -10.167  -1.860  14.364  1.00  0.00           C  \\nATOM   2404  O   TRP A 163      -9.920  -1.935  15.567  1.00  0.00           O  \\nATOM   2405  CB  TRP A 163     -12.565  -1.200  14.086  1.00  0.00           C  \\nATOM   2406  CG  TRP A 163     -13.835  -1.381  13.312  1.00  0.00           C  \\nATOM   2407  CD1 TRP A 163     -14.969  -2.004  13.740  1.00  0.00           C  \\nATOM   2408  CD2 TRP A 163     -14.109  -0.932  11.963  1.00  0.00           C  \\nATOM   2409  NE1 TRP A 163     -15.925  -1.975  12.755  1.00  0.00           N  \\nATOM   2410  CE2 TRP A 163     -15.416  -1.322  11.661  1.00  0.00           C  \\nATOM   2411  CE3 TRP A 163     -13.361  -0.240  11.002  1.00  0.00           C  \\nATOM   2412  CZ2 TRP A 163     -15.999  -1.046  10.434  1.00  0.00           C  \\nATOM   2413  CZ3 TRP A 163     -13.946   0.038   9.772  1.00  0.00           C  \\nATOM   2414  CH2 TRP A 163     -15.231  -0.355   9.497  1.00  0.00           C  \\nATOM   2415  H   TRP A 163     -12.431  -3.494  15.309  1.00  0.00           H  \\nATOM   2416  HA  TRP A 163     -11.442  -2.435  12.743  1.00  0.00           H  \\nATOM   2417  HB3 TRP A 163     -12.810  -1.182  15.148  1.00  0.00           H  \\nATOM   2418  HB2 TRP A 163     -12.144  -0.228  13.832  1.00  0.00           H  \\nATOM   2419  HD1 TRP A 163     -15.097  -2.459  14.720  1.00  0.00           H  \\nATOM   2420  HE1 TRP A 163     -16.851  -2.371  12.822  1.00  0.00           H  \\nATOM   2421  HE3 TRP A 163     -12.341   0.077  11.217  1.00  0.00           H  \\nATOM   2422  HZ2 TRP A 163     -17.019  -1.350  10.196  1.00  0.00           H  \\nATOM   2423  HZ3 TRP A 163     -13.356   0.576   9.030  1.00  0.00           H  \\nATOM   2424  HH2 TRP A 163     -15.660  -0.120   8.522  1.00  0.00           H  \\nATOM   2425  N   PHE A 164      -9.283  -1.395  13.489  1.00  0.00           N  \\nATOM   2426  CA  PHE A 164      -7.947  -0.942  13.856  1.00  0.00           C  \\nATOM   2427  C   PHE A 164      -7.485   0.228  12.984  1.00  0.00           C  \\nATOM   2428  O   PHE A 164      -7.979   0.443  11.879  1.00  0.00           O  \\nATOM   2429  CB  PHE A 164      -6.948  -2.094  13.739  1.00  0.00           C  \\nATOM   2430  CG  PHE A 164      -6.700  -2.541  12.326  1.00  0.00           C  \\nATOM   2431  CD1 PHE A 164      -5.633  -2.033  11.600  1.00  0.00           C  \\nATOM   2432  CD2 PHE A 164      -7.533  -3.469  11.720  1.00  0.00           C  \\nATOM   2433  CE1 PHE A 164      -5.404  -2.442  10.300  1.00  0.00           C  \\nATOM   2434  CE2 PHE A 164      -7.306  -3.881  10.421  1.00  0.00           C  \\nATOM   2435  CZ  PHE A 164      -6.240  -3.366   9.711  1.00  0.00           C  \\nATOM   2436  H   PHE A 164      -9.537  -1.355  12.512  1.00  0.00           H  \\nATOM   2437  HA  PHE A 164      -7.981  -0.582  14.884  1.00  0.00           H  \\nATOM   2438  HB3 PHE A 164      -5.995  -1.795  14.173  1.00  0.00           H  \\nATOM   2439  HB2 PHE A 164      -7.312  -2.950  14.307  1.00  0.00           H  \\nATOM   2440  HD1 PHE A 164      -4.971  -1.303  12.067  1.00  0.00           H  \\nATOM   2441  HD2 PHE A 164      -8.376  -3.874  12.281  1.00  0.00           H  \\nATOM   2442  HE1 PHE A 164      -4.562  -2.035   9.741  1.00  0.00           H  \\nATOM   2443  HE2 PHE A 164      -7.969  -4.611   9.956  1.00  0.00           H  \\nATOM   2444  HZ  PHE A 164      -6.061  -3.689   8.686  1.00  0.00           H  \\nATOM   2445  N   GLN A 165      -6.506   0.964  13.491  1.00  0.00           N  \\nATOM   2446  CA  GLN A 165      -5.859   2.088  12.838  1.00  0.00           C  \\nATOM   2447  C   GLN A 165      -4.366   2.036  13.162  1.00  0.00           C  \\nATOM   2448  O   GLN A 165      -3.983   1.911  14.326  1.00  0.00           O  \\nATOM   2449  CB  GLN A 165      -6.504   3.388  13.336  1.00  0.00           C  \\nATOM   2450  CG  GLN A 165      -5.851   4.633  12.718  1.00  0.00           C  \\nATOM   2451  CD  GLN A 165      -6.501   5.943  13.147  1.00  0.00           C  \\nATOM   2452  OE1 GLN A 165      -7.430   6.014  13.938  1.00  0.00           O  \\nATOM   2453  NE2 GLN A 165      -5.995   7.053  12.658  1.00  0.00           N  \\nATOM   2454  H   GLN A 165      -6.166   0.725  14.412  1.00  0.00           H  \\nATOM   2455  HA  GLN A 165      -5.994   2.020  11.759  1.00  0.00           H  \\nATOM   2456  HB3 GLN A 165      -7.563   3.374  13.081  1.00  0.00           H  \\nATOM   2457  HB2 GLN A 165      -6.412   3.440  14.421  1.00  0.00           H  \\nATOM   2458  HG3 GLN A 165      -5.894   4.567  11.631  1.00  0.00           H  \\nATOM   2459  HG2 GLN A 165      -4.802   4.681  13.012  1.00  0.00           H  \\nATOM   2460 HE21 GLN A 165      -5.202   7.010  12.034  1.00  0.00           H  \\nATOM   2461 HE22 GLN A 165      -6.430   7.916  12.954  1.00  0.00           H  \\nATOM   2462  N   ILE A 166      -3.529   2.174  12.135  1.00  0.00           N  \\nATOM   2463  CA  ILE A 166      -2.089   2.344  12.336  1.00  0.00           C  \\nATOM   2464  C   ILE A 166      -1.821   3.848  12.399  1.00  0.00           C  \\nATOM   2465  O   ILE A 166      -1.973   4.548  11.397  1.00  0.00           O  \\nATOM   2466  CB  ILE A 166      -1.266   1.696  11.208  1.00  0.00           C  \\nATOM   2467  CG1 ILE A 166      -1.576   0.200  11.113  1.00  0.00           C  \\nATOM   2468  CG2 ILE A 166       0.221   1.921  11.437  1.00  0.00           C  \\nATOM   2469  CD1 ILE A 166      -1.055  -0.453   9.853  1.00  0.00           C  \\nATOM   2470  H   ILE A 166      -3.907   2.383  11.222  1.00  0.00           H  \\nATOM   2471  HA  ILE A 166      -1.815   1.897  13.292  1.00  0.00           H  \\nATOM   2472  HB  ILE A 166      -1.548   2.137  10.252  1.00  0.00           H  \\nATOM   2473 HG13 ILE A 166      -1.143  -0.317  11.969  1.00  0.00           H  \\nATOM   2474 HG12 ILE A 166      -2.655   0.048  11.153  1.00  0.00           H  \\nATOM   2475 HG21 ILE A 166       0.788   1.456  10.631  1.00  0.00           H  \\nATOM   2476 HG22 ILE A 166       0.428   2.990  11.456  1.00  0.00           H  \\nATOM   2477 HG23 ILE A 166       0.513   1.478  12.389  1.00  0.00           H  \\nATOM   2478 HD11 ILE A 166      -1.314  -1.513   9.858  1.00  0.00           H  \\nATOM   2479 HD12 ILE A 166      -1.503   0.026   8.982  1.00  0.00           H  \\nATOM   2480 HD13 ILE A 166       0.028  -0.346   9.810  1.00  0.00           H  \\nATOM   2481  N   THR A 167      -1.488   4.360  13.582  1.00  0.00           N  \\nATOM   2482  CA  THR A 167      -1.393   5.807  13.839  1.00  0.00           C  \\nATOM   2483  C   THR A 167      -0.001   6.352  13.581  1.00  0.00           C  \\nATOM   2484  O   THR A 167       0.169   7.559  13.403  1.00  0.00           O  \\nATOM   2485  CB  THR A 167      -1.801   6.141  15.286  1.00  0.00           C  \\nATOM   2486  OG1 THR A 167      -0.933   5.457  16.199  1.00  0.00           O  \\nATOM   2487  CG2 THR A 167      -3.238   5.718  15.545  1.00  0.00           C  \\nATOM   2488  H   THR A 167      -1.340   3.735  14.363  1.00  0.00           H  \\nATOM   2489  HA  THR A 167      -2.055   6.334  13.152  1.00  0.00           H  \\nATOM   2490  HB  THR A 167      -1.708   7.214  15.452  1.00  0.00           H  \\nATOM   2491  HG1 THR A 167      -0.027   5.517  15.885  1.00  0.00           H  \\nATOM   2492 HG21 THR A 167      -3.509   5.961  16.572  1.00  0.00           H  \\nATOM   2493 HG22 THR A 167      -3.902   6.244  14.860  1.00  0.00           H  \\nATOM   2494 HG23 THR A 167      -3.335   4.644  15.390  1.00  0.00           H  \\nATOM   2495  N   LYS A 168       0.990   5.465  13.528  1.00  0.00           N  \\nATOM   2496  CA  LYS A 168       2.382   5.818  13.290  1.00  0.00           C  \\nATOM   2497  C   LYS A 168       3.127   4.648  12.673  1.00  0.00           C  \\nATOM   2498  O   LYS A 168       2.988   3.504  13.097  1.00  0.00           O  \\nATOM   2499  CB  LYS A 168       3.061   6.253  14.589  1.00  0.00           C  \\nATOM   2500  CG  LYS A 168       4.494   6.739  14.418  1.00  0.00           C  \\nATOM   2501  CD  LYS A 168       5.066   7.248  15.733  1.00  0.00           C  \\nATOM   2502  CE  LYS A 168       6.503   7.718  15.567  1.00  0.00           C  \\nATOM   2503  NZ  LYS A 168       7.076   8.220  16.845  1.00  0.00           N  \\nATOM   2504  H   LYS A 168       0.775   4.501  13.739  1.00  0.00           H  \\nATOM   2505  HA  LYS A 168       2.412   6.658  12.596  1.00  0.00           H  \\nATOM   2506  HB3 LYS A 168       2.487   7.058  15.048  1.00  0.00           H  \\nATOM   2507  HB2 LYS A 168       3.072   5.419  15.291  1.00  0.00           H  \\nATOM   2508  HG3 LYS A 168       5.115   5.920  14.053  1.00  0.00           H  \\nATOM   2509  HG2 LYS A 168       4.519   7.545  13.685  1.00  0.00           H  \\nATOM   2510  HD3 LYS A 168       4.460   8.079  16.096  1.00  0.00           H  \\nATOM   2511  HD2 LYS A 168       5.038   6.449  16.474  1.00  0.00           H  \\nATOM   2512  HE3 LYS A 168       7.117   6.893  15.208  1.00  0.00           H  \\nATOM   2513  HE2 LYS A 168       6.542   8.518  14.828  1.00  0.00           H  \\nATOM   2514  HZ1 LYS A 168       8.027   8.522  16.693  1.00  0.00           H  \\nATOM   2515  HZ2 LYS A 168       6.525   8.999  17.178  1.00  0.00           H  \\nATOM   2516  HZ3 LYS A 168       7.063   7.482  17.534  1.00  0.00           H  \\nATOM   2517  N   PHE A 169       3.963   4.977  11.701  1.00  0.00           N  \\nATOM   2518  CA  PHE A 169       4.900   4.057  11.078  1.00  0.00           C  \\nATOM   2519  C   PHE A 169       6.313   4.448  11.520  1.00  0.00           C  \\nATOM   2520  O   PHE A 169       6.780   5.551  11.223  1.00  0.00           O  \\nATOM   2521  CB  PHE A 169       4.782   4.097   9.553  1.00  0.00           C  \\nATOM   2522  CG  PHE A 169       3.489   3.539   9.030  1.00  0.00           C  \\nATOM   2523  CD1 PHE A 169       2.343   4.321   8.995  1.00  0.00           C  \\nATOM   2524  CD2 PHE A 169       3.414   2.232   8.574  1.00  0.00           C  \\nATOM   2525  CE1 PHE A 169       1.153   3.808   8.515  1.00  0.00           C  \\nATOM   2526  CE2 PHE A 169       2.226   1.718   8.092  1.00  0.00           C  \\nATOM   2527  CZ  PHE A 169       1.094   2.507   8.063  1.00  0.00           C  \\nATOM   2528  H   PHE A 169       4.038   5.951  11.446  1.00  0.00           H  \\nATOM   2529  HA  PHE A 169       4.693   3.036  11.395  1.00  0.00           H  \\nATOM   2530  HB3 PHE A 169       4.872   5.126   9.208  1.00  0.00           H  \\nATOM   2531  HB2 PHE A 169       5.600   3.531   9.110  1.00  0.00           H  \\nATOM   2532  HD1 PHE A 169       2.390   5.350   9.352  1.00  0.00           H  \\nATOM   2533  HD2 PHE A 169       4.309   1.608   8.597  1.00  0.00           H  \\nATOM   2534  HE1 PHE A 169       0.261   4.433   8.493  1.00  0.00           H  \\nATOM   2535  HE2 PHE A 169       2.182   0.689   7.735  1.00  0.00           H  \\nATOM   2536  HZ  PHE A 169       0.157   2.102   7.685  1.00  0.00           H  \\nATOM   2537  N   THR A 170       6.975   3.579  12.277  1.00  0.00           N  \\nATOM   2538  CA  THR A 170       8.299   3.819  12.878  1.00  0.00           C  \\nATOM   2539  C   THR A 170       9.329   2.814  12.350  1.00  0.00           C  \\nATOM   2540  O   THR A 170       9.057   2.099  11.389  1.00  0.00           O  \\nATOM   2541  CB  THR A 170       8.238   3.739  14.414  1.00  0.00           C  \\nATOM   2542  OG1 THR A 170       9.488   4.170  14.968  1.00  0.00           O  \\nATOM   2543  CG2 THR A 170       7.953   2.314  14.863  1.00  0.00           C  \\nATOM   2544  H   THR A 170       6.538   2.690  12.475  1.00  0.00           H  \\nATOM   2545  HA  THR A 170       8.657   4.806  12.585  1.00  0.00           H  \\nATOM   2546  HB  THR A 170       7.448   4.395  14.782  1.00  0.00           H  \\nATOM   2547  HG1 THR A 170       9.446   4.123  15.926  1.00  0.00           H  \\nATOM   2548 HG21 THR A 170       7.913   2.277  15.952  1.00  0.00           H  \\nATOM   2549 HG22 THR A 170       6.997   1.988  14.454  1.00  0.00           H  \\nATOM   2550 HG23 THR A 170       8.744   1.656  14.507  1.00  0.00           H  \\nATOM   2551  N   GLY A 171      10.536   2.771  12.923  1.00  0.00           N  \\nATOM   2552  CA  GLY A 171      11.647   1.982  12.376  1.00  0.00           C  \\nATOM   2553  C   GLY A 171      12.096   2.475  10.998  1.00  0.00           C  \\nATOM   2554  O   GLY A 171      12.319   3.671  10.804  1.00  0.00           O  \\nATOM   2555  H   GLY A 171      10.693   3.264  13.790  1.00  0.00           H  \\nATOM   2556  HA3 GLY A 171      11.340   0.938  12.301  1.00  0.00           H  \\nATOM   2557  HA2 GLY A 171      12.500   2.028  13.053  1.00  0.00           H  \\nATOM   2558  N   ALA A 172      12.203   1.564  10.029  1.00  0.00           N  \\nATOM   2559  CA  ALA A 172      12.658   1.838   8.661  1.00  0.00           C  \\nATOM   2560  C   ALA A 172      11.577   2.510   7.782  1.00  0.00           C  \\nATOM   2561  O   ALA A 172      11.364   2.126   6.635  1.00  0.00           O  \\nATOM   2562  CB  ALA A 172      13.211   0.538   8.059  1.00  0.00           C  \\nATOM   2563  H   ALA A 172      11.894   0.624  10.233  1.00  0.00           H  \\nATOM   2564  HA  ALA A 172      13.484   2.547   8.721  1.00  0.00           H  \\nATOM   2565  HB1 ALA A 172      12.414  -0.198   7.964  1.00  0.00           H  \\nATOM   2566  HB2 ALA A 172      13.998   0.140   8.701  1.00  0.00           H  \\nATOM   2567  HB3 ALA A 172      13.630   0.738   7.074  1.00  0.00           H  \\nATOM   2568  N   ALA A 173      10.855   3.493   8.323  1.00  0.00           N  \\nATOM   2569  CA  ALA A 173       9.763   4.166   7.636  1.00  0.00           C  \\nATOM   2570  C   ALA A 173      10.257   4.946   6.403  1.00  0.00           C  \\nATOM   2571  O   ALA A 173      11.331   5.566   6.414  1.00  0.00           O  \\nATOM   2572  CB  ALA A 173       9.023   5.065   8.633  1.00  0.00           C  \\nATOM   2573  H   ALA A 173      11.095   3.791   9.258  1.00  0.00           H  \\nATOM   2574  HA  ALA A 173       9.064   3.402   7.295  1.00  0.00           H  \\nATOM   2575  HB1 ALA A 173       8.191   5.564   8.136  1.00  0.00           H  \\nATOM   2576  HB2 ALA A 173       8.633   4.463   9.452  1.00  0.00           H  \\nATOM   2577  HB3 ALA A 173       9.702   5.819   9.033  1.00  0.00           H  \\nATOM   2578  N   GLY A 174       9.432   4.940   5.356  1.00  0.00           N  \\nATOM   2579  CA  GLY A 174       9.681   5.677   4.125  1.00  0.00           C  \\nATOM   2580  C   GLY A 174       9.313   7.168   4.224  1.00  0.00           C  \\nATOM   2581  O   GLY A 174       8.690   7.607   5.201  1.00  0.00           O  \\nATOM   2582  H   GLY A 174       8.553   4.447   5.427  1.00  0.00           H  \\nATOM   2583  HA3 GLY A 174       9.091   5.227   3.326  1.00  0.00           H  \\nATOM   2584  HA2 GLY A 174      10.734   5.587   3.859  1.00  0.00           H  \\nATOM   2585  N   PRO A 175       9.743   7.981   3.246  1.00  0.00           N  \\nATOM   2586  CA  PRO A 175       9.451   9.411   3.210  1.00  0.00           C  \\nATOM   2587  C   PRO A 175       7.964   9.763   3.384  1.00  0.00           C  \\nATOM   2588  O   PRO A 175       7.669  10.708   4.116  1.00  0.00           O  \\nATOM   2589  CB  PRO A 175       9.952   9.802   1.816  1.00  0.00           C  \\nATOM   2590  CG  PRO A 175      11.061   8.844   1.544  1.00  0.00           C  \\nATOM   2591  CD  PRO A 175      10.570   7.535   2.101  1.00  0.00           C  \\nATOM   2592  HA  PRO A 175      10.002   9.912   4.006  1.00  0.00           H  \\nATOM   2593  HB3 PRO A 175       9.132   9.725   1.086  1.00  0.00           H  \\nATOM   2594  HB2 PRO A 175      10.284  10.851   1.816  1.00  0.00           H  \\nATOM   2595  HG3 PRO A 175      11.264   8.795   0.464  1.00  0.00           H  \\nATOM   2596  HG2 PRO A 175      11.987   9.189   2.026  1.00  0.00           H  \\nATOM   2597  HD3 PRO A 175       9.973   7.012   1.339  1.00  0.00           H  \\nATOM   2598  HD2 PRO A 175      11.429   6.921   2.409  1.00  0.00           H  \\nATOM   2599  N   TYR A 176       7.021   9.025   2.788  1.00  0.00           N  \\nATOM   2600  CA  TYR A 176       5.591   9.321   2.945  1.00  0.00           C  \\nATOM   2601  C   TYR A 176       5.060   8.876   4.301  1.00  0.00           C  \\nATOM   2602  O   TYR A 176       4.217   9.557   4.880  1.00  0.00           O  \\nATOM   2603  CB  TYR A 176       4.753   8.667   1.845  1.00  0.00           C  \\nATOM   2604  CG  TYR A 176       4.939   9.293   0.485  1.00  0.00           C  \\nATOM   2605  CD1 TYR A 176       4.587  10.640   0.286  1.00  0.00           C  \\nATOM   2606  CD2 TYR A 176       5.423   8.526  -0.588  1.00  0.00           C  \\nATOM   2607  CE1 TYR A 176       4.740  11.224  -0.986  1.00  0.00           C  \\nATOM   2608  CE2 TYR A 176       5.569   9.101  -1.860  1.00  0.00           C  \\nATOM   2609  CZ  TYR A 176       5.222  10.446  -2.061  1.00  0.00           C  \\nATOM   2610  OH  TYR A 176       5.342  10.981  -3.301  1.00  0.00           O  \\nATOM   2611  H   TYR A 176       7.273   8.149   2.353  1.00  0.00           H  \\nATOM   2612  HA  TYR A 176       5.449  10.401   2.886  1.00  0.00           H  \\nATOM   2613  HB3 TYR A 176       3.700   8.769   2.107  1.00  0.00           H  \\nATOM   2614  HB2 TYR A 176       4.969   7.600   1.800  1.00  0.00           H  \\nATOM   2615  HD1 TYR A 176       4.192  11.214   1.111  1.00  0.00           H  \\nATOM   2616  HD2 TYR A 176       5.659   7.482  -0.444  1.00  0.00           H  \\nATOM   2617  HE1 TYR A 176       4.473  12.259  -1.145  1.00  0.00           H  \\nATOM   2618  HE2 TYR A 176       5.948   8.511  -2.681  1.00  0.00           H  \\nATOM   2619  HH  TYR A 176       5.776  10.364  -3.896  1.00  0.00           H  \\nATOM   2620  N   CYS A 177       5.567   7.776   4.848  1.00  0.00           N  \\nATOM   2621  CA  CYS A 177       5.217   7.353   6.200  1.00  0.00           C  \\nATOM   2622  C   CYS A 177       5.687   8.335   7.272  1.00  0.00           C  \\nATOM   2623  O   CYS A 177       4.939   8.641   8.203  1.00  0.00           O  \\nATOM   2624  CB  CYS A 177       5.817   5.977   6.491  1.00  0.00           C  \\nATOM   2625  SG  CYS A 177       4.999   4.617   5.623  1.00  0.00           S  \\nATOM   2626  H   CYS A 177       6.179   7.201   4.286  1.00  0.00           H  \\nATOM   2627  HA  CYS A 177       4.132   7.324   6.301  1.00  0.00           H  \\nATOM   2628  HB3 CYS A 177       6.870   5.973   6.210  1.00  0.00           H  \\nATOM   2629  HB2 CYS A 177       5.762   5.776   7.561  1.00  0.00           H  \\nATOM   2630  N   LYS A 178       6.895   8.891   7.115  1.00  0.00           N  \\nATOM   2631  CA  LYS A 178       7.399   9.961   7.986  1.00  0.00           C  \\nATOM   2632  C   LYS A 178       6.537  11.221   7.863  1.00  0.00           C  \\nATOM   2633  O   LYS A 178       6.149  11.781   8.885  1.00  0.00           O  \\nATOM   2634  CB  LYS A 178       8.855  10.287   7.652  1.00  0.00           C  \\nATOM   2635  CG  LYS A 178       9.849   9.204   8.052  1.00  0.00           C  \\nATOM   2636  CD  LYS A 178      11.266   9.575   7.639  1.00  0.00           C  \\nATOM   2637  CE  LYS A 178      12.266   8.515   8.077  1.00  0.00           C  \\nATOM   2638  NZ  LYS A 178      13.646   8.827   7.615  1.00  0.00           N  \\nATOM   2639  H   LYS A 178       7.453   8.594   6.327  1.00  0.00           H  \\nATOM   2640  HA  LYS A 178       7.322   9.645   9.027  1.00  0.00           H  \\nATOM   2641  HB3 LYS A 178       8.955  10.452   6.579  1.00  0.00           H  \\nATOM   2642  HB2 LYS A 178       9.146  11.210   8.154  1.00  0.00           H  \\nATOM   2643  HG3 LYS A 178       9.820   9.064   9.133  1.00  0.00           H  \\nATOM   2644  HG2 LYS A 178       9.575   8.264   7.574  1.00  0.00           H  \\nATOM   2645  HD3 LYS A 178      11.313   9.681   6.555  1.00  0.00           H  \\nATOM   2646  HD2 LYS A 178      11.539  10.528   8.092  1.00  0.00           H  \\nATOM   2647  HE3 LYS A 178      12.267   8.443   9.164  1.00  0.00           H  \\nATOM   2648  HE2 LYS A 178      11.971   7.547   7.671  1.00  0.00           H  \\nATOM   2649  HZ1 LYS A 178      14.277   8.102   7.924  1.00  0.00           H  \\nATOM   2650  HZ2 LYS A 178      13.660   8.878   6.606  1.00  0.00           H  \\nATOM   2651  HZ3 LYS A 178      13.938   9.714   8.000  1.00  0.00           H  \\nATOM   2652  N   ALA A 179       6.179  11.609   6.637  1.00  0.00           N  \\nATOM   2653  CA  ALA A 179       5.250  12.707   6.374  1.00  0.00           C  \\nATOM   2654  C   ALA A 179       3.840  12.452   6.933  1.00  0.00           C  \\nATOM   2655  O   ALA A 179       3.221  13.366   7.469  1.00  0.00           O  \\nATOM   2656  CB  ALA A 179       5.211  12.944   4.864  1.00  0.00           C  \\nATOM   2657  H   ALA A 179       6.611  11.158   5.844  1.00  0.00           H  \\nATOM   2658  HA  ALA A 179       5.632  13.610   6.850  1.00  0.00           H  \\nATOM   2659  HB1 ALA A 179       4.560  13.792   4.646  1.00  0.00           H  \\nATOM   2660  HB2 ALA A 179       6.213  13.168   4.500  1.00  0.00           H  \\nATOM   2661  HB3 ALA A 179       4.826  12.060   4.356  1.00  0.00           H  \\nATOM   2662  N   LEU A 180       3.332  11.217   6.874  1.00  0.00           N  \\nATOM   2663  CA  LEU A 180       2.035  10.859   7.448  1.00  0.00           C  \\nATOM   2664  C   LEU A 180       2.024  11.051   8.965  1.00  0.00           C  \\nATOM   2665  O   LEU A 180       1.144  11.735   9.482  1.00  0.00           O  \\nATOM   2666  CB  LEU A 180       1.644   9.434   7.015  1.00  0.00           C  \\nATOM   2667  CG  LEU A 180       0.297   8.945   7.587  1.00  0.00           C  \\nATOM   2668  CD1 LEU A 180      -0.891   9.803   7.141  1.00  0.00           C  \\nATOM   2669  CD2 LEU A 180       0.037   7.511   7.131  1.00  0.00           C  \\nATOM   2670  H   LEU A 180       3.855  10.506   6.382  1.00  0.00           H  \\nATOM   2671  HA  LEU A 180       1.293  11.558   7.065  1.00  0.00           H  \\nATOM   2672  HB3 LEU A 180       1.600   9.397   5.926  1.00  0.00           H  \\nATOM   2673  HB2 LEU A 180       2.424   8.745   7.341  1.00  0.00           H  \\nATOM   2674  HG  LEU A 180       0.344   8.946   8.676  1.00  0.00           H  \\nATOM   2675 HD11 LEU A 180      -1.808   9.393   7.564  1.00  0.00           H  \\nATOM   2676 HD12 LEU A 180      -0.966   9.802   6.053  1.00  0.00           H  \\nATOM   2677 HD13 LEU A 180      -0.773  10.822   7.510  1.00  0.00           H  \\nATOM   2678 HD21 LEU A 180      -0.066   7.476   6.046  1.00  0.00           H  \\nATOM   2679 HD22 LEU A 180       0.868   6.872   7.429  1.00  0.00           H  \\nATOM   2680 HD23 LEU A 180      -0.874   7.132   7.592  1.00  0.00           H  \\nATOM   2681  N   GLY A 181       3.035  10.523   9.659  1.00  0.00           N  \\nATOM   2682  CA  GLY A 181       3.193  10.699  11.104  1.00  0.00           C  \\nATOM   2683  C   GLY A 181       3.290  12.161  11.559  1.00  0.00           C  \\nATOM   2684  O   GLY A 181       2.835  12.477  12.655  1.00  0.00           O  \\nATOM   2685  H   GLY A 181       3.718   9.971   9.161  1.00  0.00           H  \\nATOM   2686  HA3 GLY A 181       2.343  10.244  11.614  1.00  0.00           H  \\nATOM   2687  HA2 GLY A 181       4.104  10.190  11.417  1.00  0.00           H  \\nATOM   2688  N   SER A 182       3.855  13.051  10.739  1.00  0.00           N  \\nATOM   2689  CA  SER A 182       3.949  14.496  11.010  1.00  0.00           C  \\nATOM   2690  C   SER A 182       2.818  15.323  10.384  1.00  0.00           C  \\nATOM   2691  O   SER A 182       2.805  16.543  10.534  1.00  0.00           O  \\nATOM   2692  CB  SER A 182       5.282  15.016  10.507  1.00  0.00           C  \\nATOM   2693  OG  SER A 182       6.347  14.407  11.183  1.00  0.00           O  \\nATOM   2694  H   SER A 182       4.263  12.725   9.874  1.00  0.00           H  \\nATOM   2695  HA  SER A 182       3.886  14.659  12.085  1.00  0.00           H  \\nATOM   2696  HB3 SER A 182       5.368  14.823   9.438  1.00  0.00           H  \\nATOM   2697  HB2 SER A 182       5.327  16.095  10.649  1.00  0.00           H  \\nATOM   2698  HG  SER A 182       6.536  13.597  10.704  1.00  0.00           H  \\nATOM   2699  N   ASN A 183       1.875  14.683   9.682  1.00  0.00           N  \\nATOM   2700  CA  ASN A 183       0.825  15.328   8.886  1.00  0.00           C  \\nATOM   2701  C   ASN A 183       1.363  16.344   7.846  1.00  0.00           C  \\nATOM   2702  O   ASN A 183       0.736  17.369   7.571  1.00  0.00           O  \\nATOM   2703  CB  ASN A 183      -0.172  16.002   9.810  1.00  0.00           C  \\nATOM   2704  CG  ASN A 183      -0.906  15.022  10.682  1.00  0.00           C  \\nATOM   2705  OD1 ASN A 183      -1.439  14.018  10.196  1.00  0.00           O  \\nATOM   2706  ND2 ASN A 183      -0.943  15.293  11.962  1.00  0.00           N  \\nATOM   2707  H   ASN A 183       1.971  13.683   9.587  1.00  0.00           H  \\nATOM   2708  HA  ASN A 183       0.365  14.534   8.297  1.00  0.00           H  \\nATOM   2709  HB3 ASN A 183       0.349  16.718  10.447  1.00  0.00           H  \\nATOM   2710  HB2 ASN A 183      -0.898  16.559   9.218  1.00  0.00           H  \\nATOM   2711 HD21 ASN A 183      -1.418  14.675  12.590  1.00  0.00           H  \\nATOM   2712 HD22 ASN A 183      -0.497  16.116  12.312  1.00  0.00           H  \\nATOM   2713  N   ASP A 184       2.522  16.065   7.244  1.00  0.00           N  \\nATOM   2714  CA  ASP A 184       3.174  16.963   6.292  1.00  0.00           C  \\nATOM   2715  C   ASP A 184       2.654  16.718   4.867  1.00  0.00           C  \\nATOM   2716  O   ASP A 184       3.080  15.806   4.165  1.00  0.00           O  \\nATOM   2717  CB  ASP A 184       4.693  16.777   6.332  1.00  0.00           C  \\nATOM   2718  CG  ASP A 184       5.433  17.743   5.416  1.00  0.00           C  \\nATOM   2719  OD1 ASP A 184       4.805  18.313   4.555  1.00  0.00           O  \\nATOM   2720  OD2 ASP A 184       6.618  17.901   5.586  1.00  0.00           O  \\nATOM   2721  H   ASP A 184       2.938  15.160   7.410  1.00  0.00           H  \\nATOM   2722  HA  ASP A 184       2.928  17.990   6.562  1.00  0.00           H  \\nATOM   2723  HB3 ASP A 184       5.050  16.921   7.352  1.00  0.00           H  \\nATOM   2724  HB2 ASP A 184       4.942  15.757   6.038  1.00  0.00           H  \\nATOM   2725  N   LYS A 185       1.737  17.571   4.401  1.00  0.00           N  \\nATOM   2726  CA  LYS A 185       1.097  17.428   3.081  1.00  0.00           C  \\nATOM   2727  C   LYS A 185       1.963  17.922   1.923  1.00  0.00           C  \\nATOM   2728  O   LYS A 185       1.597  17.707   0.768  1.00  0.00           O  \\nATOM   2729  CB  LYS A 185      -0.241  18.170   3.066  1.00  0.00           C  \\nATOM   2730  CG  LYS A 185      -1.256  17.658   4.078  1.00  0.00           C  \\nATOM   2731  CD  LYS A 185      -1.902  16.363   3.607  1.00  0.00           C  \\nATOM   2732  CE  LYS A 185      -2.870  16.612   2.460  1.00  0.00           C  \\nATOM   2733  NZ  LYS A 185      -3.825  15.485   2.283  1.00  0.00           N  \\nATOM   2734  H   LYS A 185       1.460  18.322   5.017  1.00  0.00           H  \\nATOM   2735  HA  LYS A 185       0.943  16.366   2.887  1.00  0.00           H  \\nATOM   2736  HB3 LYS A 185      -0.072  19.228   3.267  1.00  0.00           H  \\nATOM   2737  HB2 LYS A 185      -0.688  18.093   2.075  1.00  0.00           H  \\nATOM   2738  HG3 LYS A 185      -0.761  17.480   5.033  1.00  0.00           H  \\nATOM   2739  HG2 LYS A 185      -2.032  18.408   4.226  1.00  0.00           H  \\nATOM   2740  HD3 LYS A 185      -1.128  15.670   3.274  1.00  0.00           H  \\nATOM   2741  HD2 LYS A 185      -2.444  15.905   4.434  1.00  0.00           H  \\nATOM   2742  HE3 LYS A 185      -3.434  17.523   2.652  1.00  0.00           H  \\nATOM   2743  HE2 LYS A 185      -2.311  16.746   1.534  1.00  0.00           H  \\nATOM   2744  HZ1 LYS A 185      -4.448  15.688   1.515  1.00  0.00           H  \\nATOM   2745  HZ2 LYS A 185      -3.313  14.636   2.086  1.00  0.00           H  \\nATOM   2746  HZ3 LYS A 185      -4.363  15.363   3.129  1.00  0.00           H  \\nATOM   2747  N   SER A 186       3.100  18.567   2.182  1.00  0.00           N  \\nATOM   2748  CA  SER A 186       3.934  19.125   1.120  1.00  0.00           C  \\nATOM   2749  C   SER A 186       4.500  18.038   0.206  1.00  0.00           C  \\nATOM   2750  O   SER A 186       4.722  18.307  -0.974  1.00  0.00           O  \\nATOM   2751  CB  SER A 186       5.071  19.927   1.723  1.00  0.00           C  \\nATOM   2752  OG  SER A 186       6.016  19.084   2.325  1.00  0.00           O  \\nATOM   2753  H   SER A 186       3.444  18.613   3.131  1.00  0.00           H  \\nATOM   2754  HA  SER A 186       3.304  19.770   0.506  1.00  0.00           H  \\nATOM   2755  HB3 SER A 186       5.552  20.519   0.945  1.00  0.00           H  \\nATOM   2756  HB2 SER A 186       4.674  20.620   2.464  1.00  0.00           H  \\nATOM   2757  HG  SER A 186       5.670  18.879   3.197  1.00  0.00           H  \\nATOM   2758  N   VAL A 187       4.700  16.813   0.710  1.00  0.00           N  \\nATOM   2759  CA  VAL A 187       5.144  15.668  -0.094  1.00  0.00           C  \\nATOM   2760  C   VAL A 187       4.099  15.228  -1.116  1.00  0.00           C  \\nATOM   2761  O   VAL A 187       4.447  14.465  -2.003  1.00  0.00           O  \\nATOM   2762  CB  VAL A 187       5.476  14.476   0.823  1.00  0.00           C  \\nATOM   2763  CG1 VAL A 187       6.571  14.853   1.810  1.00  0.00           C  \\nATOM   2764  CG2 VAL A 187       4.222  14.022   1.556  1.00  0.00           C  \\nATOM   2765  H   VAL A 187       4.555  16.676   1.701  1.00  0.00           H  \\nATOM   2766  HA  VAL A 187       6.022  15.977  -0.661  1.00  0.00           H  \\nATOM   2767  HB  VAL A 187       5.859  13.657   0.215  1.00  0.00           H  \\nATOM   2768 HG11 VAL A 187       6.793  13.999   2.451  1.00  0.00           H  \\nATOM   2769 HG12 VAL A 187       7.469  15.140   1.264  1.00  0.00           H  \\nATOM   2770 HG13 VAL A 187       6.235  15.688   2.424  1.00  0.00           H  \\nATOM   2771 HG21 VAL A 187       4.464  13.178   2.202  1.00  0.00           H  \\nATOM   2772 HG22 VAL A 187       3.836  14.843   2.159  1.00  0.00           H  \\nATOM   2773 HG23 VAL A 187       3.467  13.718   0.830  1.00  0.00           H  \\nATOM   2774  N   CYS A 188       2.842  15.680  -1.046  1.00  0.00           N  \\nATOM   2775  CA  CYS A 188       1.797  15.328  -2.013  1.00  0.00           C  \\nATOM   2776  C   CYS A 188       1.844  16.144  -3.313  1.00  0.00           C  \\nATOM   2777  O   CYS A 188       1.136  15.802  -4.258  1.00  0.00           O  \\nATOM   2778  CB  CYS A 188       0.420  15.510  -1.373  1.00  0.00           C  \\nATOM   2779  SG  CYS A 188       0.155  14.505   0.107  1.00  0.00           S  \\nATOM   2780  H   CYS A 188       2.578  16.269  -0.268  1.00  0.00           H  \\nATOM   2781  HA  CYS A 188       1.935  14.286  -2.301  1.00  0.00           H  \\nATOM   2782  HB3 CYS A 188       0.281  16.557  -1.099  1.00  0.00           H  \\nATOM   2783  HB2 CYS A 188      -0.354  15.256  -2.097  1.00  0.00           H  \\nATOM   2784  N   ASP A 189       2.680  17.182  -3.386  1.00  0.00           N  \\nATOM   2785  CA  ASP A 189       2.851  17.995  -4.593  1.00  0.00           C  \\nATOM   2786  C   ASP A 189       3.349  17.135  -5.772  1.00  0.00           C  \\nATOM   2787  O   ASP A 189       4.379  16.455  -5.680  1.00  0.00           O  \\nATOM   2788  CB  ASP A 189       3.799  19.163  -4.276  1.00  0.00           C  \\nATOM   2789  CG  ASP A 189       3.934  20.204  -5.398  1.00  0.00           C  \\nATOM   2790  OD1 ASP A 189       3.777  19.850  -6.591  1.00  0.00           O  \\nATOM   2791  OD2 ASP A 189       4.236  21.363  -5.060  1.00  0.00           O  \\nATOM   2792  H   ASP A 189       3.249  17.403  -2.582  1.00  0.00           H  \\nATOM   2793  HA  ASP A 189       1.884  18.417  -4.862  1.00  0.00           H  \\nATOM   2794  HB3 ASP A 189       3.429  19.671  -3.385  1.00  0.00           H  \\nATOM   2795  HB2 ASP A 189       4.785  18.761  -4.042  1.00  0.00           H  \\nATOM   2796  N   LYS A 190       2.608  17.181  -6.885  1.00  0.00           N  \\nATOM   2797  CA  LYS A 190       2.890  16.434  -8.119  1.00  0.00           C  \\nATOM   2798  C   LYS A 190       4.251  16.763  -8.734  1.00  0.00           C  \\nATOM   2799  O   LYS A 190       4.803  15.960  -9.479  1.00  0.00           O  \\nATOM   2800  CB  LYS A 190       1.789  16.695  -9.149  1.00  0.00           C  \\nATOM   2801  CG  LYS A 190       0.428  16.126  -8.772  1.00  0.00           C  \\nATOM   2802  CD  LYS A 190      -0.596  16.367  -9.871  1.00  0.00           C  \\nATOM   2803  CE  LYS A 190      -1.944  15.758  -9.517  1.00  0.00           C  \\nATOM   2804  NZ  LYS A 190      -2.955  15.985 -10.585  1.00  0.00           N  \\nATOM   2805  H   LYS A 190       1.877  17.878  -6.910  1.00  0.00           H  \\nATOM   2806  HA  LYS A 190       2.917  15.373  -7.872  1.00  0.00           H  \\nATOM   2807  HB3 LYS A 190       1.674  17.769  -9.294  1.00  0.00           H  \\nATOM   2808  HB2 LYS A 190       2.079  16.263 -10.107  1.00  0.00           H  \\nATOM   2809  HG3 LYS A 190       0.517  15.052  -8.600  1.00  0.00           H  \\nATOM   2810  HG2 LYS A 190       0.079  16.595  -7.853  1.00  0.00           H  \\nATOM   2811  HD3 LYS A 190      -0.720  17.440 -10.023  1.00  0.00           H  \\nATOM   2812  HD2 LYS A 190      -0.241  15.925 -10.802  1.00  0.00           H  \\nATOM   2813  HE3 LYS A 190      -1.829  14.685  -9.366  1.00  0.00           H  \\nATOM   2814  HE2 LYS A 190      -2.310  16.197  -8.589  1.00  0.00           H  \\nATOM   2815  HZ1 LYS A 190      -3.833  15.567 -10.312  1.00  0.00           H  \\nATOM   2816  HZ2 LYS A 190      -3.084  16.978 -10.722  1.00  0.00           H  \\nATOM   2817  HZ3 LYS A 190      -2.636  15.566 -11.447  1.00  0.00           H  \\nATOM   2818  N   ASN A 191       4.814  17.922  -8.397  1.00  0.00           N  \\nATOM   2819  CA  ASN A 191       6.080  18.398  -8.947  1.00  0.00           C  \\nATOM   2820  C   ASN A 191       7.295  17.884  -8.160  1.00  0.00           C  \\nATOM   2821  O   ASN A 191       8.428  18.014  -8.623  1.00  0.00           O  \\nATOM   2822  CB  ASN A 191       6.086  19.915  -9.000  1.00  0.00           C  \\nATOM   2823  CG  ASN A 191       5.044  20.464  -9.934  1.00  0.00           C  \\nATOM   2824  OD1 ASN A 191       5.222  20.454 -11.158  1.00  0.00           O  \\nATOM   2825  ND2 ASN A 191       3.959  20.944  -9.382  1.00  0.00           N  \\nATOM   2826  H   ASN A 191       4.329  18.514  -7.737  1.00  0.00           H  \\nATOM   2827  HA  ASN A 191       6.183  18.034  -9.970  1.00  0.00           H  \\nATOM   2828  HB3 ASN A 191       5.909  20.315  -8.001  1.00  0.00           H  \\nATOM   2829  HB2 ASN A 191       7.067  20.263  -9.324  1.00  0.00           H  \\nATOM   2830 HD21 ASN A 191       3.232  21.323  -9.954  1.00  0.00           H  \\nATOM   2831 HD22 ASN A 191       3.858  20.933  -8.387  1.00  0.00           H  \\nATOM   2832  N   LYS A 192       7.087  17.293  -6.976  1.00  0.00           N  \\nATOM   2833  CA  LYS A 192       8.180  16.815  -6.125  1.00  0.00           C  \\nATOM   2834  C   LYS A 192       8.415  15.323  -6.315  1.00  0.00           C  \\nATOM   2835  O   LYS A 192       7.477  14.533  -6.348  1.00  0.00           O  \\nATOM   2836  CB  LYS A 192       7.887  17.116  -4.654  1.00  0.00           C  \\nATOM   2837  CG  LYS A 192       7.861  18.598  -4.308  1.00  0.00           C  \\nATOM   2838  CD  LYS A 192       7.685  18.814  -2.812  1.00  0.00           C  \\nATOM   2839  CE  LYS A 192       7.513  20.289  -2.481  1.00  0.00           C  \\nATOM   2840  NZ  LYS A 192       8.735  21.077  -2.800  1.00  0.00           N  \\nATOM   2841  H   LYS A 192       6.137  17.085  -6.705  1.00  0.00           H  \\nATOM   2842  HA  LYS A 192       9.094  17.327  -6.420  1.00  0.00           H  \\nATOM   2843  HB3 LYS A 192       6.920  16.691  -4.382  1.00  0.00           H  \\nATOM   2844  HB2 LYS A 192       8.642  16.639  -4.028  1.00  0.00           H  \\nATOM   2845  HG3 LYS A 192       8.795  19.063  -4.626  1.00  0.00           H  \\nATOM   2846  HG2 LYS A 192       7.038  19.081  -4.835  1.00  0.00           H  \\nATOM   2847  HD3 LYS A 192       6.807  18.267  -2.467  1.00  0.00           H  \\nATOM   2848  HD2 LYS A 192       8.560  18.433  -2.285  1.00  0.00           H  \\nATOM   2849  HE3 LYS A 192       6.676  20.694  -3.048  1.00  0.00           H  \\nATOM   2850  HE2 LYS A 192       7.292  20.400  -1.420  1.00  0.00           H  \\nATOM   2851  HZ1 LYS A 192       8.580  22.048  -2.566  1.00  0.00           H  \\nATOM   2852  HZ2 LYS A 192       9.515  20.722  -2.265  1.00  0.00           H  \\nATOM   2853  HZ3 LYS A 192       8.939  20.997  -3.786  1.00  0.00           H  \\nATOM   2854  N   ASN A 193       9.691  14.950  -6.369  1.00  0.00           N  \\nATOM   2855  CA  ASN A 193      10.136  13.566  -6.457  1.00  0.00           C  \\nATOM   2856  C   ASN A 193      10.526  13.058  -5.066  1.00  0.00           C  \\nATOM   2857  O   ASN A 193      11.619  13.352  -4.587  1.00  0.00           O  \\nATOM   2858  CB  ASN A 193      11.290  13.430  -7.432  1.00  0.00           C  \\nATOM   2859  CG  ASN A 193      11.725  12.003  -7.618  1.00  0.00           C  \\nATOM   2860  OD1 ASN A 193      11.574  11.172  -6.715  1.00  0.00           O  \\nATOM   2861  ND2 ASN A 193      12.263  11.702  -8.772  1.00  0.00           N  \\nATOM   2862  H   ASN A 193      10.395  15.664  -6.249  1.00  0.00           H  \\nATOM   2863  HA  ASN A 193       9.324  12.939  -6.826  1.00  0.00           H  \\nATOM   2864  HB3 ASN A 193      10.999  13.838  -8.401  1.00  0.00           H  \\nATOM   2865  HB2 ASN A 193      12.140  14.013  -7.074  1.00  0.00           H  \\nATOM   2866 HD21 ASN A 193      12.571  10.767  -8.952  1.00  0.00           H  \\nATOM   2867 HD22 ASN A 193      12.366  12.406  -9.474  1.00  0.00           H  \\nATOM   2868  N   ILE A 194       9.632  12.310  -4.430  1.00  0.00           N  \\nATOM   2869  CA  ILE A 194       9.791  11.842  -3.052  1.00  0.00           C  \\nATOM   2870  C   ILE A 194      10.238  10.378  -3.075  1.00  0.00           C  \\nATOM   2871  O   ILE A 194       9.428   9.465  -2.941  1.00  0.00           O  \\nATOM   2872  CB  ILE A 194       8.481  12.080  -2.275  1.00  0.00           C  \\nATOM   2873  CG1 ILE A 194       7.989  13.545  -2.332  1.00  0.00           C  \\nATOM   2874  CG2 ILE A 194       8.625  11.631  -0.816  1.00  0.00           C  \\nATOM   2875  CD1 ILE A 194       8.959  14.589  -1.759  1.00  0.00           C  \\nATOM   2876  H   ILE A 194       8.770  12.071  -4.900  1.00  0.00           H  \\nATOM   2877  HA  ILE A 194      10.577  12.412  -2.557  1.00  0.00           H  \\nATOM   2878  HB  ILE A 194       7.708  11.465  -2.735  1.00  0.00           H  \\nATOM   2879 HG13 ILE A 194       7.760  13.814  -3.362  1.00  0.00           H  \\nATOM   2880 HG12 ILE A 194       7.049  13.611  -1.784  1.00  0.00           H  \\nATOM   2881 HG21 ILE A 194       8.747  10.548  -0.791  1.00  0.00           H  \\nATOM   2882 HG22 ILE A 194       9.486  12.105  -0.344  1.00  0.00           H  \\nATOM   2883 HG23 ILE A 194       7.717  11.868  -0.260  1.00  0.00           H  \\nATOM   2884 HD11 ILE A 194       9.191  14.366  -0.718  1.00  0.00           H  \\nATOM   2885 HD12 ILE A 194       9.880  14.613  -2.341  1.00  0.00           H  \\nATOM   2886 HD13 ILE A 194       8.489  15.572  -1.797  1.00  0.00           H  \\nATOM   2887  N   ALA A 195      11.526  10.150  -3.326  1.00  0.00           N  \\nATOM   2888  CA  ALA A 195      12.088   8.803  -3.404  1.00  0.00           C  \\nATOM   2889  C   ALA A 195      12.666   8.351  -2.054  1.00  0.00           C  \\nATOM   2890  O   ALA A 195      13.237   9.157  -1.321  1.00  0.00           O  \\nATOM   2891  CB  ALA A 195      13.119   8.747  -4.537  1.00  0.00           C  \\nATOM   2892  H   ALA A 195      12.160  10.935  -3.361  1.00  0.00           H  \\nATOM   2893  HA  ALA A 195      11.285   8.112  -3.655  1.00  0.00           H  \\nATOM   2894  HB1 ALA A 195      13.939   9.433  -4.325  1.00  0.00           H  \\nATOM   2895  HB2 ALA A 195      13.519   7.736  -4.611  1.00  0.00           H  \\nATOM   2896  HB3 ALA A 195      12.647   9.017  -5.481  1.00  0.00           H  \\nATOM   2897  N   GLY A 196      12.531   7.059  -1.747  1.00  0.00           N  \\nATOM   2898  CA  GLY A 196      13.169   6.424  -0.590  1.00  0.00           C  \\nATOM   2899  C   GLY A 196      14.384   5.583  -0.991  1.00  0.00           C  \\nATOM   2900  O   GLY A 196      14.913   5.721  -2.096  1.00  0.00           O  \\nATOM   2901  H   GLY A 196      12.095   6.450  -2.426  1.00  0.00           H  \\nATOM   2902  HA3 GLY A 196      12.447   5.777  -0.093  1.00  0.00           H  \\nATOM   2903  HA2 GLY A 196      13.495   7.177   0.128  1.00  0.00           H  \\nATOM   2904  N   ASP A 197      14.781   4.650  -0.123  1.00  0.00           N  \\nATOM   2905  CA  ASP A 197      15.955   3.783  -0.328  1.00  0.00           C  \\nATOM   2906  C   ASP A 197      15.872   2.935  -1.608  1.00  0.00           C  \\nATOM   2907  O   ASP A 197      16.887   2.551  -2.179  1.00  0.00           O  \\nATOM   2908  CB  ASP A 197      16.136   2.855   0.876  1.00  0.00           C  \\nATOM   2909  CG  ASP A 197      16.569   3.595   2.134  1.00  0.00           C  \\nATOM   2910  OD1 ASP A 197      16.940   4.740   2.028  1.00  0.00           O  \\nATOM   2911  OD2 ASP A 197      16.525   3.009   3.189  1.00  0.00           O  \\nATOM   2912  H   ASP A 197      14.377   4.649   0.803  1.00  0.00           H  \\nATOM   2913  HA  ASP A 197      16.845   4.408  -0.405  1.00  0.00           H  \\nATOM   2914  HB3 ASP A 197      15.198   2.338   1.081  1.00  0.00           H  \\nATOM   2915  HB2 ASP A 197      16.883   2.097   0.642  1.00  0.00           H  \\nATOM   2916  N   TRP A 198      14.670   2.680  -2.124  1.00  0.00           N  \\nATOM   2917  CA  TRP A 198      14.439   1.905  -3.347  1.00  0.00           C  \\nATOM   2918  C   TRP A 198      14.746   2.671  -4.649  1.00  0.00           C  \\nATOM   2919  O   TRP A 198      14.702   2.070  -5.719  1.00  0.00           O  \\nATOM   2920  CB  TRP A 198      12.992   1.414  -3.328  1.00  0.00           C  \\nATOM   2921  CG  TRP A 198      12.035   2.421  -2.789  1.00  0.00           C  \\nATOM   2922  CD1 TRP A 198      11.393   3.358  -3.508  1.00  0.00           C  \\nATOM   2923  CD2 TRP A 198      11.660   2.658  -1.401  1.00  0.00           C  \\nATOM   2924  NE1 TRP A 198      10.629   4.135  -2.674  1.00  0.00           N  \\nATOM   2925  CE2 TRP A 198      10.743   3.750  -1.364  1.00  0.00           C  \\nATOM   2926  CE3 TRP A 198      12.018   2.064  -0.175  1.00  0.00           C  \\nATOM   2927  CZ2 TRP A 198      10.187   4.222  -0.173  1.00  0.00           C  \\nATOM   2928  CZ3 TRP A 198      11.466   2.529   1.031  1.00  0.00           C  \\nATOM   2929  CH2 TRP A 198      10.551   3.594   1.026  1.00  0.00           C  \\nATOM   2930  H   TRP A 198      13.873   3.105  -1.672  1.00  0.00           H  \\nATOM   2931  HA  TRP A 198      15.096   1.034  -3.336  1.00  0.00           H  \\nATOM   2932  HB3 TRP A 198      12.688   1.126  -4.335  1.00  0.00           H  \\nATOM   2933  HB2 TRP A 198      12.935   0.528  -2.697  1.00  0.00           H  \\nATOM   2934  HD1 TRP A 198      11.475   3.484  -4.579  1.00  0.00           H  \\nATOM   2935  HE1 TRP A 198      10.132   4.950  -3.007  1.00  0.00           H  \\nATOM   2936  HE3 TRP A 198      12.715   1.240  -0.165  1.00  0.00           H  \\nATOM   2937  HZ2 TRP A 198       9.487   5.044  -0.166  1.00  0.00           H  \\nATOM   2938  HZ3 TRP A 198      11.733   2.062   1.967  1.00  0.00           H  \\nATOM   2939  HH2 TRP A 198      10.126   3.933   1.960  1.00  0.00           H  \\nATOM   2940  N   GLY A 199      15.078   3.966  -4.585  1.00  0.00           N  \\nATOM   2941  CA  GLY A 199      15.558   4.757  -5.726  1.00  0.00           C  \\nATOM   2942  C   GLY A 199      14.497   5.422  -6.614  1.00  0.00           C  \\nATOM   2943  O   GLY A 199      14.843   5.990  -7.646  1.00  0.00           O  \\nATOM   2944  H   GLY A 199      15.173   4.380  -3.669  1.00  0.00           H  \\nATOM   2945  HA3 GLY A 199      16.172   4.123  -6.368  1.00  0.00           H  \\nATOM   2946  HA2 GLY A 199      16.199   5.549  -5.339  1.00  0.00           H  \\nATOM   2947  N   PHE A 200      13.214   5.394  -6.245  1.00  0.00           N  \\nATOM   2948  CA  PHE A 200      12.131   6.002  -7.035  1.00  0.00           C  \\nATOM   2949  C   PHE A 200      11.000   6.528  -6.144  1.00  0.00           C  \\nATOM   2950  O   PHE A 200      10.899   6.146  -4.985  1.00  0.00           O  \\nATOM   2951  CB  PHE A 200      11.606   4.996  -8.076  1.00  0.00           C  \\nATOM   2952  CG  PHE A 200      10.862   3.806  -7.500  1.00  0.00           C  \\nATOM   2953  CD1 PHE A 200      11.571   2.658  -7.101  1.00  0.00           C  \\nATOM   2954  CD2 PHE A 200       9.459   3.832  -7.371  1.00  0.00           C  \\nATOM   2955  CE1 PHE A 200      10.884   1.573  -6.532  1.00  0.00           C  \\nATOM   2956  CE2 PHE A 200       8.775   2.729  -6.830  1.00  0.00           C  \\nATOM   2957  CZ  PHE A 200       9.489   1.605  -6.390  1.00  0.00           C  \\nATOM   2958  H   PHE A 200      12.982   5.002  -5.343  1.00  0.00           H  \\nATOM   2959  HA  PHE A 200      12.535   6.859  -7.575  1.00  0.00           H  \\nATOM   2960  HB3 PHE A 200      10.942   5.520  -8.762  1.00  0.00           H  \\nATOM   2961  HB2 PHE A 200      12.445   4.631  -8.669  1.00  0.00           H  \\nATOM   2962  HD1 PHE A 200      12.645   2.613  -7.209  1.00  0.00           H  \\nATOM   2963  HD2 PHE A 200       8.900   4.696  -7.699  1.00  0.00           H  \\nATOM   2964  HE1 PHE A 200      11.433   0.709  -6.190  1.00  0.00           H  \\nATOM   2965  HE2 PHE A 200       7.697   2.721  -6.758  1.00  0.00           H  \\nATOM   2966  HZ  PHE A 200       8.965   0.762  -5.965  1.00  0.00           H  \\nATOM   2967  N   ASP A 201      10.132   7.396  -6.663  1.00  0.00           N  \\nATOM   2968  CA  ASP A 201       8.924   7.830  -5.943  1.00  0.00           C  \\nATOM   2969  C   ASP A 201       7.735   6.916  -6.297  1.00  0.00           C  \\nATOM   2970  O   ASP A 201       7.245   6.999  -7.431  1.00  0.00           O  \\nATOM   2971  CB  ASP A 201       8.637   9.302  -6.271  1.00  0.00           C  \\nATOM   2972  CG  ASP A 201       7.435   9.890  -5.519  1.00  0.00           C  \\nATOM   2973  OD1 ASP A 201       6.570   9.137  -5.018  1.00  0.00           O  \\nATOM   2974  OD2 ASP A 201       7.362  11.137  -5.456  1.00  0.00           O  \\nATOM   2975  H   ASP A 201      10.247   7.699  -7.619  1.00  0.00           H  \\nATOM   2976  HA  ASP A 201       9.114   7.777  -4.871  1.00  0.00           H  \\nATOM   2977  HB3 ASP A 201       8.469   9.407  -7.343  1.00  0.00           H  \\nATOM   2978  HB2 ASP A 201       9.520   9.886  -6.015  1.00  0.00           H  \\nATOM   2979  N   PRO A 202       7.248   6.054  -5.381  1.00  0.00           N  \\nATOM   2980  CA  PRO A 202       6.161   5.126  -5.686  1.00  0.00           C  \\nATOM   2981  C   PRO A 202       4.883   5.827  -6.154  1.00  0.00           C  \\nATOM   2982  O   PRO A 202       4.112   5.242  -6.918  1.00  0.00           O  \\nATOM   2983  CB  PRO A 202       5.948   4.416  -4.345  1.00  0.00           C  \\nATOM   2984  CG  PRO A 202       7.277   4.491  -3.676  1.00  0.00           C  \\nATOM   2985  CD  PRO A 202       7.804   5.853  -4.042  1.00  0.00           C  \\nATOM   2986  HA  PRO A 202       6.466   4.450  -6.485  1.00  0.00           H  \\nATOM   2987  HB3 PRO A 202       5.154   4.921  -3.775  1.00  0.00           H  \\nATOM   2988  HB2 PRO A 202       5.613   3.382  -4.515  1.00  0.00           H  \\nATOM   2989  HG3 PRO A 202       7.164   4.359  -2.590  1.00  0.00           H  \\nATOM   2990  HG2 PRO A 202       7.927   3.677  -4.029  1.00  0.00           H  \\nATOM   2991  HD3 PRO A 202       7.428   6.598  -3.325  1.00  0.00           H  \\nATOM   2992  HD2 PRO A 202       8.904   5.834  -4.040  1.00  0.00           H  \\nATOM   2993  N   ALA A 203       4.697   7.104  -5.808  1.00  0.00           N  \\nATOM   2994  CA  ALA A 203       3.526   7.881  -6.206  1.00  0.00           C  \\nATOM   2995  C   ALA A 203       3.440   8.094  -7.719  1.00  0.00           C  \\nATOM   2996  O   ALA A 203       2.367   8.377  -8.248  1.00  0.00           O  \\nATOM   2997  CB  ALA A 203       3.589   9.236  -5.509  1.00  0.00           C  \\nATOM   2998  H   ALA A 203       5.425   7.584  -5.298  1.00  0.00           H  \\nATOM   2999  HA  ALA A 203       2.625   7.360  -5.882  1.00  0.00           H  \\nATOM   3000  HB1 ALA A 203       2.687   9.799  -5.748  1.00  0.00           H  \\nATOM   3001  HB2 ALA A 203       3.665   9.093  -4.432  1.00  0.00           H  \\nATOM   3002  HB3 ALA A 203       4.448   9.802  -5.868  1.00  0.00           H  \\nATOM   3003  N   LYS A 204       4.552   7.945  -8.448  1.00  0.00           N  \\nATOM   3004  CA  LYS A 204       4.555   8.090  -9.903  1.00  0.00           C  \\nATOM   3005  C   LYS A 204       4.094   6.817 -10.612  1.00  0.00           C  \\nATOM   3006  O   LYS A 204       3.938   6.856 -11.821  1.00  0.00           O  \\nATOM   3007  CB  LYS A 204       5.950   8.481 -10.393  1.00  0.00           C  \\nATOM   3008  CG  LYS A 204       6.449   9.820  -9.865  1.00  0.00           C  \\nATOM   3009  CD  LYS A 204       5.583  10.968 -10.362  1.00  0.00           C  \\nATOM   3010  CE  LYS A 204       5.908  11.323 -11.805  1.00  0.00           C  \\nATOM   3011  NZ  LYS A 204       5.121  12.493 -12.281  1.00  0.00           N  \\nATOM   3012  H   LYS A 204       5.398   7.665  -7.972  1.00  0.00           H  \\nATOM   3013  HA  LYS A 204       3.825   8.852 -10.182  1.00  0.00           H  \\nATOM   3014  HB3 LYS A 204       6.668   7.716 -10.096  1.00  0.00           H  \\nATOM   3015  HB2 LYS A 204       5.954   8.528 -11.482  1.00  0.00           H  \\nATOM   3016  HG3 LYS A 204       6.435   9.810  -8.775  1.00  0.00           H  \\nATOM   3017  HG2 LYS A 204       7.475   9.981 -10.195  1.00  0.00           H  \\nATOM   3018  HD3 LYS A 204       4.531  10.686 -10.294  1.00  0.00           H  \\nATOM   3019  HD2 LYS A 204       5.747  11.845  -9.736  1.00  0.00           H  \\nATOM   3020  HE3 LYS A 204       6.969  11.554 -11.893  1.00  0.00           H  \\nATOM   3021  HE2 LYS A 204       5.690  10.470 -12.448  1.00  0.00           H  \\nATOM   3022  HZ1 LYS A 204       5.365  12.696 -13.240  1.00  0.00           H  \\nATOM   3023  HZ2 LYS A 204       4.135  12.283 -12.222  1.00  0.00           H  \\nATOM   3024  HZ3 LYS A 204       5.329  13.296 -11.704  1.00  0.00           H  \\nATOM   3025  N   TRP A 205       3.900   5.686  -9.934  1.00  0.00           N  \\nATOM   3026  CA  TRP A 205       3.454   4.422 -10.546  1.00  0.00           C  \\nATOM   3027  C   TRP A 205       2.136   3.911  -9.947  1.00  0.00           C  \\nATOM   3028  O   TRP A 205       1.871   2.713  -9.971  1.00  0.00           O  \\nATOM   3029  CB  TRP A 205       4.565   3.365 -10.495  1.00  0.00           C  \\nATOM   3030  CG  TRP A 205       5.737   3.528 -11.406  1.00  0.00           C  \\nATOM   3031  CD1 TRP A 205       5.871   2.926 -12.611  1.00  0.00           C  \\nATOM   3032  CD2 TRP A 205       6.957   4.295 -11.205  1.00  0.00           C  \\nATOM   3033  NE1 TRP A 205       7.086   3.270 -13.163  1.00  0.00           N  \\nATOM   3034  CE2 TRP A 205       7.776   4.154 -12.364  1.00  0.00           C  \\nATOM   3035  CE3 TRP A 205       7.443   5.124 -10.177  1.00  0.00           C  \\nATOM   3036  CZ2 TRP A 205       8.975   4.856 -12.529  1.00  0.00           C  \\nATOM   3037  CZ3 TRP A 205       8.667   5.805 -10.312  1.00  0.00           C  \\nATOM   3038  CH2 TRP A 205       9.428   5.683 -11.488  1.00  0.00           C  \\nATOM   3039  H   TRP A 205       4.013   5.699  -8.931  1.00  0.00           H  \\nATOM   3040  HA  TRP A 205       3.222   4.586 -11.599  1.00  0.00           H  \\nATOM   3041  HB3 TRP A 205       4.118   2.421 -10.806  1.00  0.00           H  \\nATOM   3042  HB2 TRP A 205       4.918   3.262  -9.469  1.00  0.00           H  \\nATOM   3043  HD1 TRP A 205       5.148   2.259 -13.059  1.00  0.00           H  \\nATOM   3044  HE1 TRP A 205       7.402   2.928 -14.059  1.00  0.00           H  \\nATOM   3045  HE3 TRP A 205       6.874   5.206  -9.262  1.00  0.00           H  \\nATOM   3046  HZ2 TRP A 205       9.563   4.719 -13.424  1.00  0.00           H  \\nATOM   3047  HZ3 TRP A 205       9.032   6.406  -9.491  1.00  0.00           H  \\nATOM   3048  HH2 TRP A 205      10.378   6.189 -11.574  1.00  0.00           H  \\nATOM   3049  N   ALA A 206       1.304   4.809  -9.421  1.00  0.00           N  \\nATOM   3050  CA  ALA A 206       0.039   4.433  -8.809  1.00  0.00           C  \\nATOM   3051  C   ALA A 206      -1.110   4.320  -9.822  1.00  0.00           C  \\nATOM   3052  O   ALA A 206      -1.350   5.227 -10.628  1.00  0.00           O  \\nATOM   3053  CB  ALA A 206      -0.270   5.417  -7.694  1.00  0.00           C  \\nATOM   3054  H   ALA A 206       1.570   5.783  -9.400  1.00  0.00           H  \\nATOM   3055  HA  ALA A 206       0.167   3.463  -8.329  1.00  0.00           H  \\nATOM   3056  HB1 ALA A 206      -0.324   6.429  -8.099  1.00  0.00           H  \\nATOM   3057  HB2 ALA A 206      -1.231   5.158  -7.250  1.00  0.00           H  \\nATOM   3058  HB3 ALA A 206       0.510   5.372  -6.935  1.00  0.00           H  \\nATOM   3059  N   TYR A 207      -1.863   3.226  -9.727  1.00  0.00           N  \\nATOM   3060  CA  TYR A 207      -3.013   2.914 -10.574  1.00  0.00           C  \\nATOM   3061  C   TYR A 207      -4.221   2.470  -9.748  1.00  0.00           C  \\nATOM   3062  O   TYR A 207      -4.072   1.966  -8.643  1.00  0.00           O  \\nATOM   3063  CB  TYR A 207      -2.622   1.831 -11.589  1.00  0.00           C  \\nATOM   3064  CG  TYR A 207      -1.618   2.306 -12.619  1.00  0.00           C  \\nATOM   3065  CD1 TYR A 207      -2.074   2.840 -13.837  1.00  0.00           C  \\nATOM   3066  CD2 TYR A 207      -0.238   2.254 -12.345  1.00  0.00           C  \\nATOM   3067  CE1 TYR A 207      -1.159   3.357 -14.769  1.00  0.00           C  \\nATOM   3068  CE2 TYR A 207       0.688   2.765 -13.276  1.00  0.00           C  \\nATOM   3069  CZ  TYR A 207       0.221   3.330 -14.482  1.00  0.00           C  \\nATOM   3070  OH  TYR A 207       1.079   3.876 -15.379  1.00  0.00           O  \\nATOM   3071  H   TYR A 207      -1.579   2.519  -9.065  1.00  0.00           H  \\nATOM   3072  HA  TYR A 207      -3.299   3.811 -11.124  1.00  0.00           H  \\nATOM   3073  HB3 TYR A 207      -3.522   1.500 -12.108  1.00  0.00           H  \\nATOM   3074  HB2 TYR A 207      -2.216   0.971 -11.056  1.00  0.00           H  \\nATOM   3075  HD1 TYR A 207      -3.131   2.856 -14.056  1.00  0.00           H  \\nATOM   3076  HD2 TYR A 207       0.106   1.842 -11.407  1.00  0.00           H  \\nATOM   3077  HE1 TYR A 207      -1.520   3.764 -15.702  1.00  0.00           H  \\nATOM   3078  HE2 TYR A 207       1.746   2.741 -13.064  1.00  0.00           H  \\nATOM   3079  HH  TYR A 207       0.606   4.488 -15.946  1.00  0.00           H  \\nATOM   3080  N   GLN A 208      -5.421   2.617 -10.300  1.00  0.00           N  \\nATOM   3081  CA  GLN A 208      -6.639   1.964  -9.814  1.00  0.00           C  \\nATOM   3082  C   GLN A 208      -7.211   1.079 -10.914  1.00  0.00           C  \\nATOM   3083  O   GLN A 208      -7.139   1.422 -12.097  1.00  0.00           O  \\nATOM   3084  CB  GLN A 208      -7.678   2.997  -9.368  1.00  0.00           C  \\nATOM   3085  CG  GLN A 208      -7.394   3.620  -8.012  1.00  0.00           C  \\nATOM   3086  CD  GLN A 208      -8.431   4.657  -7.623  1.00  0.00           C  \\nATOM   3087  OE1 GLN A 208      -8.782   5.533  -8.418  1.00  0.00           O  \\nATOM   3088  NE2 GLN A 208      -8.927   4.563  -6.394  1.00  0.00           N  \\nATOM   3089  H   GLN A 208      -5.480   3.107 -11.180  1.00  0.00           H  \\nATOM   3090  HA  GLN A 208      -6.408   1.322  -8.963  1.00  0.00           H  \\nATOM   3091  HB3 GLN A 208      -7.730   3.800 -10.103  1.00  0.00           H  \\nATOM   3092  HB2 GLN A 208      -8.661   2.529  -9.324  1.00  0.00           H  \\nATOM   3093  HG3 GLN A 208      -7.394   2.835  -7.256  1.00  0.00           H  \\nATOM   3094  HG2 GLN A 208      -6.419   4.107  -8.044  1.00  0.00           H  \\nATOM   3095 HE21 GLN A 208      -9.615   5.220  -6.081  1.00  0.00           H  \\nATOM   3096 HE22 GLN A 208      -8.614   3.838  -5.782  1.00  0.00           H  \\nATOM   3097  N   TYR A 209      -7.780  -0.060 -10.534  1.00  0.00           N  \\nATOM   3098  CA  TYR A 209      -8.380  -0.978 -11.491  1.00  0.00           C  \\nATOM   3099  C   TYR A 209      -9.835  -0.600 -11.778  1.00  0.00           C  \\nATOM   3100  O   TYR A 209     -10.666  -0.487 -10.879  1.00  0.00           O  \\nATOM   3101  CB  TYR A 209      -8.293  -2.417 -10.975  1.00  0.00           C  \\nATOM   3102  CG  TYR A 209      -8.865  -3.443 -11.928  1.00  0.00           C  \\nATOM   3103  CD1 TYR A 209      -8.545  -3.391 -13.277  1.00  0.00           C  \\nATOM   3104  CD2 TYR A 209      -9.710  -4.435 -11.453  1.00  0.00           C  \\nATOM   3105  CE1 TYR A 209      -9.068  -4.328 -14.147  1.00  0.00           C  \\nATOM   3106  CE2 TYR A 209     -10.233  -5.372 -12.324  1.00  0.00           C  \\nATOM   3107  CZ  TYR A 209      -9.914  -5.320 -13.665  1.00  0.00           C  \\nATOM   3108  OH  TYR A 209     -10.434  -6.254 -14.532  1.00  0.00           O  \\nATOM   3109  H   TYR A 209      -7.865  -0.275  -9.550  1.00  0.00           H  \\nATOM   3110  HA  TYR A 209      -7.826  -0.915 -12.426  1.00  0.00           H  \\nATOM   3111  HB3 TYR A 209      -7.250  -2.673 -10.785  1.00  0.00           H  \\nATOM   3112  HB2 TYR A 209      -8.827  -2.496 -10.029  1.00  0.00           H  \\nATOM   3113  HD1 TYR A 209      -7.881  -2.611 -13.650  1.00  0.00           H  \\nATOM   3114  HD2 TYR A 209      -9.963  -4.475 -10.394  1.00  0.00           H  \\nATOM   3115  HE1 TYR A 209      -8.816  -4.287 -15.207  1.00  0.00           H  \\nATOM   3116  HE2 TYR A 209     -10.897  -6.151 -11.951  1.00  0.00           H  \\nATOM   3117  HH  TYR A 209     -10.078  -6.106 -15.412  1.00  0.00           H  \\nATOM   3118  N   ASP A 210     -10.130  -0.415 -13.058  1.00  0.00           N  \\nATOM   3119  CA  ASP A 210     -11.461  -0.170 -13.588  1.00  0.00           C  \\nATOM   3120  C   ASP A 210     -12.069  -1.511 -13.993  1.00  0.00           C  \\nATOM   3121  O   ASP A 210     -11.810  -2.031 -15.080  1.00  0.00           O  \\nATOM   3122  CB  ASP A 210     -11.316   0.805 -14.754  1.00  0.00           C  \\nATOM   3123  CG  ASP A 210     -12.608   1.176 -15.476  1.00  0.00           C  \\nATOM   3124  OD1 ASP A 210     -13.623   0.450 -15.388  1.00  0.00           O  \\nATOM   3125  OD2 ASP A 210     -12.541   2.188 -16.215  1.00  0.00           O  \\nATOM   3126  H   ASP A 210      -9.402  -0.584 -13.737  1.00  0.00           H  \\nATOM   3127  HA  ASP A 210     -12.093   0.293 -12.830  1.00  0.00           H  \\nATOM   3128  HB3 ASP A 210     -10.881   1.719 -14.347  1.00  0.00           H  \\nATOM   3129  HB2 ASP A 210     -10.619   0.396 -15.485  1.00  0.00           H  \\nATOM   3130  N   GLU A 211     -12.859  -2.086 -13.086  1.00  0.00           N  \\nATOM   3131  CA  GLU A 211     -13.465  -3.406 -13.257  1.00  0.00           C  \\nATOM   3132  C   GLU A 211     -14.391  -3.469 -14.475  1.00  0.00           C  \\nATOM   3133  O   GLU A 211     -14.407  -4.474 -15.184  1.00  0.00           O  \\nATOM   3134  CB  GLU A 211     -14.245  -3.792 -11.999  1.00  0.00           C  \\nATOM   3135  CG  GLU A 211     -13.378  -4.049 -10.775  1.00  0.00           C  \\nATOM   3136  CD  GLU A 211     -14.182  -4.330  -9.536  1.00  0.00           C  \\nATOM   3137  OE1 GLU A 211     -15.025  -5.193  -9.581  1.00  0.00           O  \\nATOM   3138  OE2 GLU A 211     -13.952  -3.681  -8.543  1.00  0.00           O  \\nATOM   3139  H   GLU A 211     -12.954  -1.635 -12.187  1.00  0.00           H  \\nATOM   3140  HA  GLU A 211     -12.673  -4.135 -13.426  1.00  0.00           H  \\nATOM   3141  HB3 GLU A 211     -14.948  -2.997 -11.749  1.00  0.00           H  \\nATOM   3142  HB2 GLU A 211     -14.826  -4.693 -12.193  1.00  0.00           H  \\nATOM   3143  HG3 GLU A 211     -12.730  -4.902 -10.975  1.00  0.00           H  \\nATOM   3144  HG2 GLU A 211     -12.745  -3.180 -10.602  1.00  0.00           H  \\nATOM   3145  N   LYS A 212     -15.098  -2.369 -14.763  1.00  0.00           N  \\nATOM   3146  CA  LYS A 212     -16.096  -2.297 -15.834  1.00  0.00           C  \\nATOM   3147  C   LYS A 212     -15.441  -2.306 -17.212  1.00  0.00           C  \\nATOM   3148  O   LYS A 212     -15.948  -2.952 -18.123  1.00  0.00           O  \\nATOM   3149  CB  LYS A 212     -16.962  -1.046 -15.675  1.00  0.00           C  \\nATOM   3150  CG  LYS A 212     -17.907  -1.083 -14.481  1.00  0.00           C  \\nATOM   3151  CD  LYS A 212     -18.722   0.198 -14.382  1.00  0.00           C  \\nATOM   3152  CE  LYS A 212     -19.662   0.166 -13.187  1.00  0.00           C  \\nATOM   3153  NZ  LYS A 212     -20.437   1.430 -13.054  1.00  0.00           N  \\nATOM   3154  H   LYS A 212     -14.873  -1.528 -14.251  1.00  0.00           H  \\nATOM   3155  HA  LYS A 212     -16.727  -3.184 -15.783  1.00  0.00           H  \\nATOM   3156  HB3 LYS A 212     -16.320  -0.171 -15.568  1.00  0.00           H  \\nATOM   3157  HB2 LYS A 212     -17.563  -0.904 -16.573  1.00  0.00           H  \\nATOM   3158  HG3 LYS A 212     -18.587  -1.929 -14.581  1.00  0.00           H  \\nATOM   3159  HG2 LYS A 212     -17.330  -1.208 -13.565  1.00  0.00           H  \\nATOM   3160  HD3 LYS A 212     -18.049   1.051 -14.282  1.00  0.00           H  \\nATOM   3161  HD2 LYS A 212     -19.310   0.326 -15.291  1.00  0.00           H  \\nATOM   3162  HE3 LYS A 212     -20.360  -0.664 -13.295  1.00  0.00           H  \\nATOM   3163  HE2 LYS A 212     -19.086   0.010 -12.275  1.00  0.00           H  \\nATOM   3164  HZ1 LYS A 212     -21.047   1.368 -12.251  1.00  0.00           H  \\nATOM   3165  HZ2 LYS A 212     -19.800   2.205 -12.934  1.00  0.00           H  \\nATOM   3166  HZ3 LYS A 212     -20.990   1.575 -13.886  1.00  0.00           H  \\nATOM   3167  N   ASN A 213     -14.337  -1.578 -17.364  1.00  0.00           N  \\nATOM   3168  CA  ASN A 213     -13.628  -1.463 -18.640  1.00  0.00           C  \\nATOM   3169  C   ASN A 213     -12.374  -2.350 -18.725  1.00  0.00           C  \\nATOM   3170  O   ASN A 213     -11.661  -2.292 -19.724  1.00  0.00           O  \\nATOM   3171  CB  ASN A 213     -13.263  -0.012 -18.895  1.00  0.00           C  \\nATOM   3172  CG  ASN A 213     -14.468   0.853 -19.140  1.00  0.00           C  \\nATOM   3173  OD1 ASN A 213     -15.293   0.557 -20.012  1.00  0.00           O  \\nATOM   3174  ND2 ASN A 213     -14.585   1.917 -18.388  1.00  0.00           N  \\nATOM   3175  H   ASN A 213     -14.025  -1.030 -16.575  1.00  0.00           H  \\nATOM   3176  HA  ASN A 213     -14.282  -1.807 -19.442  1.00  0.00           H  \\nATOM   3177  HB3 ASN A 213     -12.715   0.381 -18.038  1.00  0.00           H  \\nATOM   3178  HB2 ASN A 213     -12.603   0.050 -19.761  1.00  0.00           H  \\nATOM   3179 HD21 ASN A 213     -15.368   2.529 -18.506  1.00  0.00           H  \\nATOM   3180 HD22 ASN A 213     -13.894   2.117 -17.694  1.00  0.00           H  \\nATOM   3181  N   ASN A 214     -12.091  -3.151 -17.690  1.00  0.00           N  \\nATOM   3182  CA  ASN A 214     -10.961  -4.077 -17.605  1.00  0.00           C  \\nATOM   3183  C   ASN A 214      -9.602  -3.409 -17.907  1.00  0.00           C  \\nATOM   3184  O   ASN A 214      -8.791  -3.929 -18.675  1.00  0.00           O  \\nATOM   3185  CB  ASN A 214     -11.188  -5.252 -18.539  1.00  0.00           C  \\nATOM   3186  CG  ASN A 214     -10.252  -6.397 -18.267  1.00  0.00           C  \\nATOM   3187  OD1 ASN A 214      -9.843  -6.621 -17.122  1.00  0.00           O  \\nATOM   3188  ND2 ASN A 214      -9.904  -7.124 -19.297  1.00  0.00           N  \\nATOM   3189  H   ASN A 214     -12.663  -3.057 -16.864  1.00  0.00           H  \\nATOM   3190  HA  ASN A 214     -10.906  -4.395 -16.564  1.00  0.00           H  \\nATOM   3191  HB3 ASN A 214     -12.215  -5.606 -18.436  1.00  0.00           H  \\nATOM   3192  HB2 ASN A 214     -11.056  -4.928 -19.571  1.00  0.00           H  \\nATOM   3193 HD21 ASN A 214      -9.283  -7.899 -19.176  1.00  0.00           H  \\nATOM   3194 HD22 ASN A 214     -10.259  -6.906 -20.206  1.00  0.00           H  \\nATOM   3195  N   LYS A 215      -9.351  -2.245 -17.297  1.00  0.00           N  \\nATOM   3196  CA  LYS A 215      -8.112  -1.467 -17.474  1.00  0.00           C  \\nATOM   3197  C   LYS A 215      -7.646  -0.826 -16.170  1.00  0.00           C  \\nATOM   3198  O   LYS A 215      -8.405  -0.731 -15.211  1.00  0.00           O  \\nATOM   3199  CB  LYS A 215      -8.307  -0.386 -18.538  1.00  0.00           C  \\nATOM   3200  CG  LYS A 215      -9.362   0.655 -18.192  1.00  0.00           C  \\nATOM   3201  CD  LYS A 215      -9.518   1.677 -19.308  1.00  0.00           C  \\nATOM   3202  CE  LYS A 215     -10.545   2.740 -18.947  1.00  0.00           C  \\nATOM   3203  NZ  LYS A 215     -10.654   3.788 -19.998  1.00  0.00           N  \\nATOM   3204  H   LYS A 215     -10.046  -1.901 -16.649  1.00  0.00           H  \\nATOM   3205  HA  LYS A 215      -7.321  -2.147 -17.788  1.00  0.00           H  \\nATOM   3206  HB3 LYS A 215      -7.364   0.135 -18.704  1.00  0.00           H  \\nATOM   3207  HB2 LYS A 215      -8.593  -0.852 -19.481  1.00  0.00           H  \\nATOM   3208  HG3 LYS A 215     -10.320   0.162 -18.026  1.00  0.00           H  \\nATOM   3209  HG2 LYS A 215      -9.077   1.171 -17.275  1.00  0.00           H  \\nATOM   3210  HD3 LYS A 215      -8.558   2.160 -19.496  1.00  0.00           H  \\nATOM   3211  HD2 LYS A 215      -9.836   1.173 -20.220  1.00  0.00           H  \\nATOM   3212  HE3 LYS A 215     -11.520   2.273 -18.814  1.00  0.00           H  \\nATOM   3213  HE2 LYS A 215     -10.263   3.215 -18.008  1.00  0.00           H  \\nATOM   3214  HZ1 LYS A 215     -11.344   4.472 -19.722  1.00  0.00           H  \\nATOM   3215  HZ2 LYS A 215      -9.759   4.241 -20.117  1.00  0.00           H  \\nATOM   3216  HZ3 LYS A 215     -10.934   3.362 -20.871  1.00  0.00           H  \\nATOM   3217  N   PHE A 216      -6.404  -0.350 -16.150  1.00  0.00           N  \\nATOM   3218  CA  PHE A 216      -5.794   0.267 -14.974  1.00  0.00           C  \\nATOM   3219  C   PHE A 216      -5.546   1.753 -15.240  1.00  0.00           C  \\nATOM   3220  O   PHE A 216      -4.712   2.119 -16.065  1.00  0.00           O  \\nATOM   3221  CB  PHE A 216      -4.520  -0.498 -14.599  1.00  0.00           C  \\nATOM   3222  CG  PHE A 216      -4.803  -1.917 -14.144  1.00  0.00           C  \\nATOM   3223  CD1 PHE A 216      -4.920  -2.209 -12.771  1.00  0.00           C  \\nATOM   3224  CD2 PHE A 216      -5.020  -2.933 -15.092  1.00  0.00           C  \\nATOM   3225  CE1 PHE A 216      -5.251  -3.511 -12.357  1.00  0.00           C  \\nATOM   3226  CE2 PHE A 216      -5.350  -4.234 -14.674  1.00  0.00           C  \\nATOM   3227  CZ  PHE A 216      -5.460  -4.524 -13.305  1.00  0.00           C  \\nATOM   3228  H   PHE A 216      -5.875  -0.337 -17.010  1.00  0.00           H  \\nATOM   3229  HA  PHE A 216      -6.472   0.193 -14.124  1.00  0.00           H  \\nATOM   3230  HB3 PHE A 216      -4.009   0.033 -13.796  1.00  0.00           H  \\nATOM   3231  HB2 PHE A 216      -3.847  -0.520 -15.457  1.00  0.00           H  \\nATOM   3232  HD1 PHE A 216      -4.782  -1.431 -12.035  1.00  0.00           H  \\nATOM   3233  HD2 PHE A 216      -4.957  -2.709 -16.147  1.00  0.00           H  \\nATOM   3234  HE1 PHE A 216      -5.377  -3.730 -11.306  1.00  0.00           H  \\nATOM   3235  HE2 PHE A 216      -5.525  -5.014 -15.401  1.00  0.00           H  \\nATOM   3236  HZ  PHE A 216      -5.722  -5.520 -12.978  1.00  0.00           H  \\nATOM   3237  N   ASN A 217      -6.276   2.618 -14.541  1.00  0.00           N  \\nATOM   3238  CA  ASN A 217      -6.218   4.061 -14.739  1.00  0.00           C  \\nATOM   3239  C   ASN A 217      -5.142   4.662 -13.837  1.00  0.00           C  \\nATOM   3240  O   ASN A 217      -5.069   4.337 -12.652  1.00  0.00           O  \\nATOM   3241  CB  ASN A 217      -7.596   4.675 -14.459  1.00  0.00           C  \\nATOM   3242  CG  ASN A 217      -8.644   4.293 -15.491  1.00  0.00           C  \\nATOM   3243  OD1 ASN A 217      -8.396   4.194 -16.685  1.00  0.00           O  \\nATOM   3244  ND2 ASN A 217      -9.867   4.102 -15.058  1.00  0.00           N  \\nATOM   3245  H   ASN A 217      -6.830   2.268 -13.772  1.00  0.00           H  \\nATOM   3246  HA  ASN A 217      -5.951   4.271 -15.775  1.00  0.00           H  \\nATOM   3247  HB3 ASN A 217      -7.927   4.359 -13.469  1.00  0.00           H  \\nATOM   3248  HB2 ASN A 217      -7.516   5.762 -14.455  1.00  0.00           H  \\nATOM   3249 HD21 ASN A 217     -10.566   3.765 -15.703  1.00  0.00           H  \\nATOM   3250 HD22 ASN A 217     -10.061   4.145 -14.068  1.00  0.00           H  \\nATOM   3251  N   TYR A 218      -4.317   5.550 -14.388  1.00  0.00           N  \\nATOM   3252  CA  TYR A 218      -3.272   6.217 -13.621  1.00  0.00           C  \\nATOM   3253  C   TYR A 218      -3.883   7.196 -12.613  1.00  0.00           C  \\nATOM   3254  O   TYR A 218      -4.684   8.049 -12.991  1.00  0.00           O  \\nATOM   3255  CB  TYR A 218      -2.320   6.945 -14.576  1.00  0.00           C  \\nATOM   3256  CG  TYR A 218      -1.159   7.591 -13.854  1.00  0.00           C  \\nATOM   3257  CD1 TYR A 218      -1.014   8.993 -13.788  1.00  0.00           C  \\nATOM   3258  CD2 TYR A 218      -0.241   6.759 -13.195  1.00  0.00           C  \\nATOM   3259  CE1 TYR A 218       0.077   9.561 -13.092  1.00  0.00           C  \\nATOM   3260  CE2 TYR A 218       0.833   7.322 -12.499  1.00  0.00           C  \\nATOM   3261  CZ  TYR A 218       1.016   8.716 -12.463  1.00  0.00           C  \\nATOM   3262  OH  TYR A 218       2.096   9.183 -11.790  1.00  0.00           O  \\nATOM   3263  H   TYR A 218      -4.476   5.845 -15.341  1.00  0.00           H  \\nATOM   3264  HA  TYR A 218      -2.709   5.464 -13.070  1.00  0.00           H  \\nATOM   3265  HB3 TYR A 218      -2.873   7.703 -15.129  1.00  0.00           H  \\nATOM   3266  HB2 TYR A 218      -1.920   6.230 -15.295  1.00  0.00           H  \\nATOM   3267  HD1 TYR A 218      -1.739   9.638 -14.263  1.00  0.00           H  \\nATOM   3268  HD2 TYR A 218      -0.374   5.687 -13.191  1.00  0.00           H  \\nATOM   3269  HE1 TYR A 218       0.200  10.632 -13.028  1.00  0.00           H  \\nATOM   3270  HE2 TYR A 218       1.513   6.676 -11.965  1.00  0.00           H  \\nATOM   3271  HH  TYR A 218       2.143  10.136 -11.690  1.00  0.00           H  \\nATOM   3272  N   VAL A 219      -3.497   7.087 -11.341  1.00  0.00           N  \\nATOM   3273  CA  VAL A 219      -3.994   7.952 -10.249  1.00  0.00           C  \\nATOM   3274  C   VAL A 219      -2.877   8.632  -9.461  1.00  0.00           C  \\nATOM   3275  O   VAL A 219      -3.149   9.281  -8.441  1.00  0.00           O  \\nATOM   3276  CB  VAL A 219      -4.851   7.127  -9.271  1.00  0.00           C  \\nATOM   3277  CG1 VAL A 219      -6.053   6.531  -9.988  1.00  0.00           C  \\nATOM   3278  CG2 VAL A 219      -4.005   6.033  -8.637  1.00  0.00           C  \\nATOM   3279  H   VAL A 219      -2.833   6.362 -11.107  1.00  0.00           H  \\nATOM   3280  HA  VAL A 219      -4.572   8.763 -10.689  1.00  0.00           H  \\nATOM   3281  HB  VAL A 219      -5.235   7.787  -8.493  1.00  0.00           H  \\nATOM   3282 HG11 VAL A 219      -6.648   5.951  -9.282  1.00  0.00           H  \\nATOM   3283 HG12 VAL A 219      -6.664   7.333 -10.402  1.00  0.00           H  \\nATOM   3284 HG13 VAL A 219      -5.711   5.881 -10.793  1.00  0.00           H  \\nATOM   3285 HG21 VAL A 219      -4.617   5.454  -7.946  1.00  0.00           H  \\nATOM   3286 HG22 VAL A 219      -3.616   5.377  -9.416  1.00  0.00           H  \\nATOM   3287 HG23 VAL A 219      -3.174   6.484  -8.095  1.00  0.00           H  \\nATOM   3288  N   GLY A 220      -1.641   8.440  -9.923  1.00  0.00           N  \\nATOM   3289  CA  GLY A 220      -0.431   8.981  -9.335  1.00  0.00           C  \\nATOM   3290  C   GLY A 220      -0.188  10.451  -9.660  1.00  0.00           C  \\nATOM   3291  O   GLY A 220      -1.081  11.171 -10.109  1.00  0.00           O  \\nATOM   3292  H   GLY A 220      -1.537   7.933 -10.791  1.00  0.00           H  \\nATOM   3293  HA3 GLY A 220      -0.465   8.880  -8.251  1.00  0.00           H  \\nATOM   3294  HA2 GLY A 220       0.415   8.405  -9.711  1.00  0.00           H  \\nATOM   3295  N   LYS A 221       1.038  10.884  -9.381  1.00  0.00           N  \\nATOM   3296  CA  LYS A 221       1.501  12.266  -9.552  1.00  0.00           C  \\nATOM   3297  C   LYS A 221       1.845  12.635 -10.988  1.00  0.00           C  \\nATOM   3298  O   LYS A 221       2.415  11.777 -11.697  1.00  0.00           O  \\nATOM   3299  OXT LYS A 221       1.566  13.800 -11.324  1.00  0.00           O  \\nATOM   3300  CB  LYS A 221       2.709  12.494  -8.660  1.00  0.00           C  \\nATOM   3301  CG  LYS A 221       2.321  12.547  -7.179  1.00  0.00           C  \\nATOM   3302  CD  LYS A 221       3.613  12.875  -6.452  1.00  0.00           C  \\nATOM   3303  CE  LYS A 221       3.450  13.068  -4.960  1.00  0.00           C  \\nATOM   3304  NZ  LYS A 221       4.721  13.654  -4.511  1.00  0.00           N  \\nATOM   3305  H   LYS A 221       1.715  10.187  -9.102  1.00  0.00           H  \\nATOM   3306  HA  LYS A 221       0.705  12.956  -9.273  1.00  0.00           H  \\nATOM   3307  HB3 LYS A 221       3.435  11.700  -8.835  1.00  0.00           H  \\nATOM   3308  HB2 LYS A 221       3.176  13.435  -8.950  1.00  0.00           H  \\nATOM   3309  HG3 LYS A 221       1.585  13.334  -7.012  1.00  0.00           H  \\nATOM   3310  HG2 LYS A 221       1.929  11.587  -6.845  1.00  0.00           H  \\nATOM   3311  HD3 LYS A 221       4.335  12.078  -6.627  1.00  0.00           H  \\nATOM   3312  HD2 LYS A 221       4.025  13.793  -6.872  1.00  0.00           H  \\nATOM   3313  HE3 LYS A 221       2.622  13.751  -4.769  1.00  0.00           H  \\nATOM   3314  HE2 LYS A 221       3.278  12.108  -4.474  1.00  0.00           H  \\nATOM   3315  HZ1 LYS A 221       5.480  13.005  -4.659  1.00  0.00           H  \\nATOM   3316  HZ2 LYS A 221       4.702  13.865  -3.523  1.00  0.00           H  \\nATOM   3317  HZ3 LYS A 221       4.915  14.507  -5.015  1.00  0.00           H  \\nTER    3318      LYS A 221\\nCONECT  659 2625\\nCONECT  741 2779\\nCONECT 2625  659\\nCONECT 2779  741\\nEND\\n\",\"pdb\");\n",
              "\tviewer_16745495978711402.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
              "\tviewer_16745495978711402.zoomTo();\n",
              "viewer_16745495978711402.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<py3Dmol.view at 0x7fbf5eaad850>"
            ]
          },
          "metadata": {},
          "execution_count": 18
        }
      ],
      "source": [
        "show_pdb(pdb_preprocessed_path)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "q7W8ZdcpFI5X"
      },
      "source": [
        "# Add mutants"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Mutants to simulate\n",
        "#@markdown If you want to simulate mutants upload a file with a list of mutants\n",
        "#@markdown The file should have the following format:\n",
        "\n",
        "#@markdown Commented lines start with \\#.\n",
        "#@markdown Each line corresponds to a single mutant. Each mutation is described\n",
        "#@markdown as XNY, where X is one-letter code of the original amino acid, N\n",
        "#@markdown is the residue id of the mutated amino acid in the PDB file, \n",
        "#@markdown and Y is the mutated amino acid.\n",
        "\n",
        "simulate_mutants = True #@param {type:\"boolean\"}\n",
        "if simulate_mutants:\n",
        "    uploaded = files.upload()\n",
        "    mutations_file = list(uploaded.keys())[-1]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 74
        },
        "id": "OxHFVkKqYI9d",
        "outputId": "7a367b29-bf59-4e2b-d298-b032a7823b6d"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ],
            "text/html": [
              "\n",
              "     <input type=\"file\" id=\"files-c25b9089-0f71-47ca-aba5-1346528c3ae9\" name=\"files[]\" multiple disabled\n",
              "        style=\"border:none\" />\n",
              "     <output id=\"result-c25b9089-0f71-47ca-aba5-1346528c3ae9\">\n",
              "      Upload widget is only available when the cell has been executed in the\n",
              "      current browser session. Please rerun this cell to enable.\n",
              "      </output>\n",
              "      <script>// Copyright 2017 Google LLC\n",
              "//\n",
              "// Licensed under the Apache License, Version 2.0 (the \"License\");\n",
              "// you may not use this file except in compliance with the License.\n",
              "// You may obtain a copy of the License at\n",
              "//\n",
              "//      http://www.apache.org/licenses/LICENSE-2.0\n",
              "//\n",
              "// Unless required by applicable law or agreed to in writing, software\n",
              "// distributed under the License is distributed on an \"AS IS\" BASIS,\n",
              "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
              "// See the License for the specific language governing permissions and\n",
              "// limitations under the License.\n",
              "\n",
              "/**\n",
              " * @fileoverview Helpers for google.colab Python module.\n",
              " */\n",
              "(function(scope) {\n",
              "function span(text, styleAttributes = {}) {\n",
              "  const element = document.createElement('span');\n",
              "  element.textContent = text;\n",
              "  for (const key of Object.keys(styleAttributes)) {\n",
              "    element.style[key] = styleAttributes[key];\n",
              "  }\n",
              "  return element;\n",
              "}\n",
              "\n",
              "// Max number of bytes which will be uploaded at a time.\n",
              "const MAX_PAYLOAD_SIZE = 100 * 1024;\n",
              "\n",
              "function _uploadFiles(inputId, outputId) {\n",
              "  const steps = uploadFilesStep(inputId, outputId);\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  // Cache steps on the outputElement to make it available for the next call\n",
              "  // to uploadFilesContinue from Python.\n",
              "  outputElement.steps = steps;\n",
              "\n",
              "  return _uploadFilesContinue(outputId);\n",
              "}\n",
              "\n",
              "// This is roughly an async generator (not supported in the browser yet),\n",
              "// where there are multiple asynchronous steps and the Python side is going\n",
              "// to poll for completion of each step.\n",
              "// This uses a Promise to block the python side on completion of each step,\n",
              "// then passes the result of the previous step as the input to the next step.\n",
              "function _uploadFilesContinue(outputId) {\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  const steps = outputElement.steps;\n",
              "\n",
              "  const next = steps.next(outputElement.lastPromiseValue);\n",
              "  return Promise.resolve(next.value.promise).then((value) => {\n",
              "    // Cache the last promise value to make it available to the next\n",
              "    // step of the generator.\n",
              "    outputElement.lastPromiseValue = value;\n",
              "    return next.value.response;\n",
              "  });\n",
              "}\n",
              "\n",
              "/**\n",
              " * Generator function which is called between each async step of the upload\n",
              " * process.\n",
              " * @param {string} inputId Element ID of the input file picker element.\n",
              " * @param {string} outputId Element ID of the output display.\n",
              " * @return {!Iterable<!Object>} Iterable of next steps.\n",
              " */\n",
              "function* uploadFilesStep(inputId, outputId) {\n",
              "  const inputElement = document.getElementById(inputId);\n",
              "  inputElement.disabled = false;\n",
              "\n",
              "  const outputElement = document.getElementById(outputId);\n",
              "  outputElement.innerHTML = '';\n",
              "\n",
              "  const pickedPromise = new Promise((resolve) => {\n",
              "    inputElement.addEventListener('change', (e) => {\n",
              "      resolve(e.target.files);\n",
              "    });\n",
              "  });\n",
              "\n",
              "  const cancel = document.createElement('button');\n",
              "  inputElement.parentElement.appendChild(cancel);\n",
              "  cancel.textContent = 'Cancel upload';\n",
              "  const cancelPromise = new Promise((resolve) => {\n",
              "    cancel.onclick = () => {\n",
              "      resolve(null);\n",
              "    };\n",
              "  });\n",
              "\n",
              "  // Wait for the user to pick the files.\n",
              "  const files = yield {\n",
              "    promise: Promise.race([pickedPromise, cancelPromise]),\n",
              "    response: {\n",
              "      action: 'starting',\n",
              "    }\n",
              "  };\n",
              "\n",
              "  cancel.remove();\n",
              "\n",
              "  // Disable the input element since further picks are not allowed.\n",
              "  inputElement.disabled = true;\n",
              "\n",
              "  if (!files) {\n",
              "    return {\n",
              "      response: {\n",
              "        action: 'complete',\n",
              "      }\n",
              "    };\n",
              "  }\n",
              "\n",
              "  for (const file of files) {\n",
              "    const li = document.createElement('li');\n",
              "    li.append(span(file.name, {fontWeight: 'bold'}));\n",
              "    li.append(span(\n",
              "        `(${file.type || 'n/a'}) - ${file.size} bytes, ` +\n",
              "        `last modified: ${\n",
              "            file.lastModifiedDate ? file.lastModifiedDate.toLocaleDateString() :\n",
              "                                    'n/a'} - `));\n",
              "    const percent = span('0% done');\n",
              "    li.appendChild(percent);\n",
              "\n",
              "    outputElement.appendChild(li);\n",
              "\n",
              "    const fileDataPromise = new Promise((resolve) => {\n",
              "      const reader = new FileReader();\n",
              "      reader.onload = (e) => {\n",
              "        resolve(e.target.result);\n",
              "      };\n",
              "      reader.readAsArrayBuffer(file);\n",
              "    });\n",
              "    // Wait for the data to be ready.\n",
              "    let fileData = yield {\n",
              "      promise: fileDataPromise,\n",
              "      response: {\n",
              "        action: 'continue',\n",
              "      }\n",
              "    };\n",
              "\n",
              "    // Use a chunked sending to avoid message size limits. See b/62115660.\n",
              "    let position = 0;\n",
              "    do {\n",
              "      const length = Math.min(fileData.byteLength - position, MAX_PAYLOAD_SIZE);\n",
              "      const chunk = new Uint8Array(fileData, position, length);\n",
              "      position += length;\n",
              "\n",
              "      const base64 = btoa(String.fromCharCode.apply(null, chunk));\n",
              "      yield {\n",
              "        response: {\n",
              "          action: 'append',\n",
              "          file: file.name,\n",
              "          data: base64,\n",
              "        },\n",
              "      };\n",
              "\n",
              "      let percentDone = fileData.byteLength === 0 ?\n",
              "          100 :\n",
              "          Math.round((position / fileData.byteLength) * 100);\n",
              "      percent.textContent = `${percentDone}% done`;\n",
              "\n",
              "    } while (position < fileData.byteLength);\n",
              "  }\n",
              "\n",
              "  // All done.\n",
              "  yield {\n",
              "    response: {\n",
              "      action: 'complete',\n",
              "    }\n",
              "  };\n",
              "}\n",
              "\n",
              "scope.google = scope.google || {};\n",
              "scope.google.colab = scope.google.colab || {};\n",
              "scope.google.colab._files = {\n",
              "  _uploadFiles,\n",
              "  _uploadFilesContinue,\n",
              "};\n",
              "})(self);\n",
              "</script> "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Saving mutations.txt to mutations.txt\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 44,
      "metadata": {
        "id": "VoylKPKIFMgU",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4fb46ae8-2772-4fc6-8077-7a7a5dfbbf9c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/alphagmx/md_base/target_preprocessed_no_water\n"
          ]
        }
      ],
      "source": [
        "#@title Read mutation file and prepare alignments\n",
        "alphabet = list(Bio.SeqUtils.IUPACData.protein_letters_1to3.keys())\n",
        "alphabet.append(\"-\")\n",
        "\n",
        "parser = PDBParser()\n",
        "structure = parser.get_structure('X', pdb_preprocessed_path)\n",
        "\n",
        "residues = \"\"\n",
        "resids = []\n",
        "for model in structure:\n",
        "    for chain in model:\n",
        "        for i, residue in enumerate(chain.get_residues()):\n",
        "            residue_1l = Bio.SeqUtils.seq1(residue.resname)\n",
        "            resid = residue.id[1]\n",
        "            if residue_1l != \"X\":\n",
        "                residues += residue_1l\n",
        "                resids.append(resid)\n",
        "\n",
        "record = SeqRecord(Seq(residues), id=\"0\", name=\"template\", \n",
        "                   description=\"input protein\")\n",
        "\n",
        "mutant_n = 0\n",
        "with open(mutations_file, \"r\") as f:\n",
        "    for line in f:\n",
        "        if line.startswith(\"#\") or line.strip() == \"\":\n",
        "            continue\n",
        "\n",
        "        folder_mask = \"alphagmx/md_mutant_\"\n",
        "\n",
        "        if not os.path.exists(f\"{folder_mask}{mutant_n}\"):\n",
        "            os.mkdir(f\"{folder_mask}{mutant_n}\")\n",
        "        \n",
        "        new_sequence = residues\n",
        "        for mutation in line.split():\n",
        "            original_AA = mutation[0].upper()\n",
        "            if original_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "            \n",
        "            mutant_AA = mutation[-1].upper()\n",
        "            if mutant_AA not in alphabet:\n",
        "                raise Exception(\"The wrong amino acid code is provided for in\"+\\\n",
        "                                f\"in the mutation file for mutation {mutation}\")\n",
        "                \n",
        "            try:\n",
        "                resid = int(mutation[1:-1])\n",
        "            except:\n",
        "                print(f\"Can't get resid for mutation {mutation}. Check you input\")\n",
        "                raise\n",
        "            \n",
        "            if residues[resids.index(resid)] != original_AA:\n",
        "                raise Exception(f\"In the PDB file resid {resid} has amino acid\"+\\\n",
        "                                f\" {residues[resids.index(resid)]} while in\"+\\\n",
        "                                f\"mutation file {original_AA} is provide\")\n",
        "            new_sequence = list(new_sequence)\n",
        "            new_sequence[resids.index(resid)] = mutant_AA\n",
        "            new_sequence = \"\".join(new_sequence)\n",
        "\n",
        "        record_mutant = SeqRecord(Seq(new_sequence), id=\"1\", name=\"mutant\", \n",
        "                description=\"mutated protein\")\n",
        "        \n",
        "        handle = open(f\"{folder_mask}{mutant_n}/alignment.ali\", \"w\")\n",
        "        writer = Bio.SeqIO.PirIO.PirWriter(handle)\n",
        "        writer.write_record(record)\n",
        "        writer.write_record(record_mutant)\n",
        "        handle.close()\n",
        "        mutant_n += 1\n",
        "\n",
        "# setting environ variables for modeller \n",
        "os.environ['N_MUTANT'] = str(mutant_n)\n",
        "os.environ['N_RESID'] = str(len(residues))\n",
        "os.environ['PDB'] = os.path.basename(pdb_preprocessed_path).split('.')[0]\n",
        "if not os.path.exists(os.path.join(project_dir, 'md_base')):\n",
        "    os.mkdir(os.path.join(project_dir, 'md_base'))\n",
        "os.environ['MD_PATH'] = os.path.join(project_dir, 'md_base')\n",
        "fn_pdb_md = os.path.join(os.path.dirname(pdb_preprocessed_path), 'md_base', os.path.splitext(os.path.basename(pdb_preprocessed_path))[0]+\"_no_water.pdb\")\n",
        "shutil.copy(os.path.splitext(pdb_preprocessed_path)[0]+\"_no_water.pdb\", fn_pdb_md)\n",
        "os.environ['PDB4MD'] = fn_pdb_md.split('.')[0]\n",
        "print(fn_pdb_md.split('.')[0])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 47,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "e3QMz2A94hb4",
        "outputId": "ed8d12c6-5017-4ff9-c0ac-ea8cb3890803"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">P1;0\n",
            "structureX:/content/alphagmx/md_base/target_preprocessed_no_water:1:A:221:A::::\n",
            "VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQ\n",
            "RVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGT\n",
            "NAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKAL\n",
            "GSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK*\n",
            ">P1;1\n",
            "sequence:mutated protein:1::221:::::\n",
            "APVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQ\n",
            "RVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGT\n",
            "NAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKAL\n",
            "GSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK*\n"
          ]
        }
      ],
      "source": [
        "!head -n 20 alphagmx/md_mutant_0/alignment.ali"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 46,
      "metadata": {
        "id": "W1IGNECJ1LyS",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7d0a5ef7-738a-4451-d0c8-aff2255ea1c9"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "s/template - input protein/structureX:/content/alphagmx/md_base/target_preprocessed_no_water:1:A:221:A::::/g\n",
            "s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/target_preprocessed_no_water:1:A:221:A::::/g\n"
          ]
        }
      ],
      "source": [
        "#@title Correct naming for alignment file\n",
        "%%bash\n",
        "#echo \"s/template - input protein/structureX:${PDB4MD}:1:A:${N_RESID}:A::::/g\"\n",
        "#echo \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/${PDB}_no_water:1:A:${N_RESID}:A::::/g\"\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "sed -i 's/XX/P1/g' alphagmx/md_mutant_${i}/alignment.ali\n",
        "#sed -i \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/${PDB}_no_water:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "#sed -i \"s/template - input protein/structureX:${PDB4MD}:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/template - input protein/structureX:\\/content\\/alphagmx\\/md_base\\/target_preprocessed_no_water:1:A:${N_RESID}:A::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "sed -i \"s/mutant - mutated protein/sequence:mutated protein:1::${N_RESID}:::::/g\" alphagmx/md_mutant_${i}/alignment.ali\n",
        "done\n",
        "#TODO: A is for chain id? Is it always A here?\n",
        "#TODO: this cell will be completed even if something goes wrong"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 49,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aTlUZ6aqNLwy",
        "outputId": "1d07b252-6043-40fe-cdd2-6687c7a72638"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "                         MODELLER 10.4, 2022/10/28, r12463\n",
            "\n",
            "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
            "\n",
            "\n",
            "                     Copyright(c) 1989-2022 Andrej Sali\n",
            "                            All Rights Reserved\n",
            "\n",
            "                             Written by A. Sali\n",
            "                               with help from\n",
            "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
            "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
            "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
            "                     F. Melo, J.P. Overington, E. Feyfant\n",
            "                 University of California, San Francisco, USA\n",
            "                    Rockefeller University, New York, USA\n",
            "                      Harvard University, Cambridge, USA\n",
            "                   Imperial Cancer Research Fund, London, UK\n",
            "              Birkbeck College, University of London, London, UK\n",
            "\n",
            "\n",
            "Kind, OS, HostName, Kernel, Processor: 4, Linux 920a41d163a5 5.10.147+ x86_64\n",
            "Date and time of compilation         : 2022/10/28 20:54:52\n",
            "MODELLER executable type             : x86_64-intel8\n",
            "Job starting time (YY/MM/DD HH:MM:SS): 2023/01/24 10:44:54\n",
            "\n",
            "\n",
            "check_ali___> Checking the sequence-structure alignment. \n",
            "\n",
            "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
            "\n",
            "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
            "----------------------------------------------\n",
            "END OF TABLE\n",
            "read_to_681_> topology.submodel read from topology file:        3\n",
            "patch_s_522_> Number of disulfides patched in MODEL:        2\n",
            "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
            "              least one known structure available. MDT, not library, potential is used.\n",
            "0 atoms in HETATM/BLK residues constrained\n",
            "to protein atoms within 2.30 angstroms\n",
            "and protein CA atoms within 10.00 angstroms\n",
            "0 atoms in residues without defined topology\n",
            "constrained to be rigid bodies\n",
            "condens_443_> Restraints marked for deletion were removed.\n",
            "              Total number of restraints before, now:    22974    21366\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
            ">> Model assessment by DOPE potential\n",
            "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      221\n",
            "Number of all, selected real atoms                :     1686    1686\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    21366   21366\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   307252\n",
            "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        1    9999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :   15.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :      -20898.3105\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "DOPE score               : -20898.310547\n",
            ">> Model assessment by GA341 potential\n",
            "\n",
            "Surface library          : /usr/local/lib/modeller-10.4/modlib/surf5.de\n",
            "Pair library             : /usr/local/lib/modeller-10.4/modlib/pair9.de\n",
            "Chain identifier         : _\n",
            "% sequence identity      :    99.547997\n",
            "Sequence length          :          221\n",
            "Compactness              :     0.104846\n",
            "Native energy (pair)     :   -72.574940\n",
            "Native energy (surface)  :   -12.566317\n",
            "Native energy (combined) :    -4.529687\n",
            "Z score (pair)           :    -4.304306\n",
            "Z score (surface)        :    -5.955426\n",
            "Z score (combined)       :    -7.262771\n",
            "GA341 score              :     1.000000\n",
            "\n",
            "\n",
            "\n",
            ">> ENERGY; Differences between the model's features and restraints:\n",
            "Number of all residues in MODEL                   :      221\n",
            "Number of all, selected real atoms                :     1686    1686\n",
            "Number of all, selected pseudo atoms              :        0       0\n",
            "Number of all static, selected restraints         :    21366   21366\n",
            "COVALENT_CYS                                      :        F\n",
            "NONBONDED_SEL_ATOMS                               :        1\n",
            "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3533\n",
            "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
            "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
            "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
            "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
            "RESIDUE_SPAN_RANGE                                :        0   99999\n",
            "NLOGN_USE                                         :       15\n",
            "CONTACT_SHELL                                     :    4.000\n",
            "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
            "SPHERE_STDV                                       :    0.050\n",
            "RADII_FACTOR                                      :    0.820\n",
            "Current energy                                    :        1284.0504\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Summary of the restraint violations: \n",
            "\n",
            "   NUM     ... number of restraints.\n",
            "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
            "   RVIOL   ... relative difference from the best value.\n",
            "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
            "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
            "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
            "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
            "\n",
            " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
            "------------------------------------------------------------------------------------------------------\n",
            " 1 Bond length potential              :    1730       0      0   0.006   0.006      18.871       1.000\n",
            " 2 Bond angle potential               :    2348       2      7   2.322   2.322      241.75       1.000\n",
            " 3 Stereochemical cosine torsion poten:    1121       0     41  49.489  49.489      407.48       1.000\n",
            " 4 Stereochemical improper torsion pot:     699       0      0   1.210   1.210      22.505       1.000\n",
            " 5 Soft-sphere overlap restraints     :    3533       0      0   0.001   0.001     0.48444       1.000\n",
            " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            " 9 Distance restraints 1 (CA-CA)      :    4288       0      0   0.110   0.110      55.917       1.000\n",
            "10 Distance restraints 2 (N-O)        :    4510       0      3   0.172   0.172      128.02       1.000\n",
            "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "13 Mainchain Omega dihedral restraints:     220       0      0   3.574   3.574      31.672       1.000\n",
            "14 Sidechain Chi_1 dihedral restraints:     179       0      1  47.306  47.306      7.4784       1.000\n",
            "15 Sidechain Chi_2 dihedral restraints:     137       0      1  64.280  64.280      31.522       1.000\n",
            "16 Sidechain Chi_3 dihedral restraints:      64       0      1  53.814  53.814      32.514       1.000\n",
            "17 Sidechain Chi_4 dihedral restraints:      27       0      0  74.120  74.120      16.683       1.000\n",
            "18 Disulfide distance restraints      :       2       0      0   0.009   0.009     0.29457E-01   1.000\n",
            "19 Disulfide angle restraints         :       4       0      0   1.770   1.770     0.27678       1.000\n",
            "20 Disulfide dihedral angle restraints:       2       0      0  28.931  28.931      1.6899       1.000\n",
            "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "23 Distance restraints 3 (SDCH-MNCH)  :    3628       0      0   0.395   0.395      100.37       1.000\n",
            "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "25 Phi/Psi pair of dihedral restraints:     219       4     14  25.306  32.946      44.167       1.000\n",
            "26 Distance restraints 4 (SDCH-SDCH)  :    2188       0      0   0.673   0.673      142.62       1.000\n",
            "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
            "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
            "\n",
            "\n",
            "\n",
            "# Heavy relative violation of each residue is written to: 1.V99990001\n",
            "# The profile is NOT normalized by the number of restraints.\n",
            "# The profiles are smoothed over a window of residues:    1\n",
            "# The sum of all numbers in the file:   17775.1680\n",
            "\n",
            "\n",
            "\n",
            "List of the violated restraints:\n",
            "   A restraint is violated when the relative difference\n",
            "   from the best value (RVIOL) is larger than CUTOFF.\n",
            "\n",
            "   ICSR   ... index of a restraint in the current set.\n",
            "   RESNO  ... residue numbers of the first two atoms.\n",
            "   ATM    ... IUPAC atom names of the first two atoms.\n",
            "   FEAT   ... the value of the feature in the model.\n",
            "   restr  ... the mean of the basis restraint with the smallest\n",
            "              difference from the model (local minimum).\n",
            "   viol   ... difference from the local minimum.\n",
            "   rviol  ... relative difference from the local minimum.\n",
            "   RESTR  ... the best value (global minimum).\n",
            "   VIOL   ... difference from the best value.\n",
            "   RVIOL  ... relative difference from the best value.\n",
            "\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature  2                           : Bond angle potential                    \n",
            "List of the RVIOL violations larger than   :       4.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   2905 110Y 111A C   N     838  840  151.35  120.00   31.35    7.12  120.00   31.35    7.12\n",
            "\n",
            "-------------------------------------------------------------------------------------------------\n",
            "\n",
            "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
            "List of the RVIOL violations larger than   :       6.5000\n",
            "\n",
            "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
            "    1   5917  11S  12V C   N      75   77  -70.41  -62.40   16.69    1.83  -73.50  166.98   14.23\n",
            "    1         12V  12V N   CA     77   78  -27.75  -42.40                  139.20\n",
            "    2   6002  96N  97P C   N     720  722  -58.97  -64.50   28.29    1.84  -58.70  149.95   12.37\n",
            "    2         97P  97P N   CA    722  723  119.45  147.20                  -30.50\n",
            "    3   6003  97P  98S C   N     727  729  148.13  -64.10  152.55   20.36  -64.10  152.55   20.36\n",
            "    3         98S  98S N   CA    729  730  -72.92  -35.00                  -35.00\n",
            "    4   6016 110Y 111A C   N     838  840  125.83 -134.00  108.18    5.16  -68.20  170.53   15.66\n",
            "    4        111A 111A N   CA    840  841  106.14  147.00                  145.30\n",
            "\n",
            "\n",
            "report______> Distribution of short non-bonded contacts:\n",
            "\n",
            "\n",
            "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
            "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
            "FREQUENCY:     0    0    0    0    0    4   23   61   82  119  168  177  241  269  276\n",
            "\n",
            "\n",
            "<< end of ENERGY.\n",
            "\n",
            ">> Summary of successfully produced models:\n",
            "Filename                          molpdf     DOPE score    GA341 score\n",
            "----------------------------------------------------------------------\n",
            "1.B99990001.pdb               1284.05042   -20898.31055        1.00000\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run modeller\n",
        "for i in range(mutant_n):\n",
        "    os.chdir(f\"/content/alphagmx/md_mutant_{i}\")\n",
        "    env = Environ()\n",
        "    a = AutoModel(env, alnfile='alignment.ali',\n",
        "                  knowns='0', sequence='1',\n",
        "                  assess_methods=(assess.DOPE,\n",
        "                                  #soap_protein_od.Scorer(),\n",
        "                                  assess.GA341))\n",
        "    a.starting_model = 1\n",
        "    a.ending_model = 1\n",
        "    a.make()\n",
        "    for file in os.listdir(\".\"):\n",
        "        if file.endswith(\"pdb\"):\n",
        "            shutil.copy2(file, 'input.pdb')\n",
        "os.chdir(\"/content\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "o4PcAAuQAh5c"
      },
      "source": [
        "# Calculate most probable protonation states "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 64,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L7OhlVxhCvhe",
        "outputId": "16b189b2-c98e-41ee-94e1-9216b2a1f97d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Preparing /content/alphagmx/md_base/target_preprocessed_no_water for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for /content/alphagmx/md_base/target_preprocessed_no_water.pdb\n",
            "Preparing input for pKa calculations\n",
            "Loading pKa-ANI Models and ANI-2x...\n",
            "Finished Loading.\n",
            "Calculating pKa for input.pdb\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:288: UserWarning: Trying to unpickle estimator DecisionTreeRegressor from version 1.0.2 when using version 1.2.0. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:288: UserWarning: Trying to unpickle estimator RandomForestRegressor from version 1.0.2 when using version 1.2.0. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:288: UserWarning: Trying to unpickle estimator DecisionTreeRegressor from version 1.0.2 when using version 1.2.0. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.8/site-packages/sklearn/base.py:288: UserWarning: Trying to unpickle estimator RandomForestRegressor from version 1.0.2 when using version 1.2.0. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:\n",
            "https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations\n",
            "  warnings.warn(\n"
          ]
        }
      ],
      "source": [
        "#@ Run pkaAni\n",
        "%%bash\n",
        "cd ${MD_PATH}\n",
        "in_pdb=${PDB4MD}.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "cd ../md_mutant_${i}\n",
        "in_pdb=input.pdb\n",
        "pkaani -i $in_pdb -p True\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Select the force field\n",
        "#@markdown We currently provide access to three force fields: AMBER99-ILDN, CHARMM36m, and AMBER14SB\n",
        "ff = 'charmm36' #@param {type:\"string\"} [\"amber99\", \"amber14\", \"charmm36\"]\n",
        "os.environ['FF']=ff\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "# md_path = f\"./md_base/\"\n",
        "# os.environ['MD_PATH']=path\n",
        "# os.environ['PDB']=\"base\""
      ],
      "metadata": {
        "id": "UykZO7pon2Er"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "YLJhqZARoCKN"
      },
      "outputs": [],
      "source": [
        "#@title Get force fields for pdb2gmx\n",
        "if ff == 'amber14':\n",
        "    wget.download(\"https://github.com/pbuslaev/alphagmx/raw/main/forcefields/amber14sb.ff.tar.gz\")\n",
        "    tar = tarfile.open(\"amber14sb.ff.tar.gz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n",
        "if ff == 'charmm36':\n",
        "    wget.download(\"http://mackerell.umaryland.edu/download.php?filename=CHARMM_ff_params_files/charmm36-jul2021.ff.tgz\")\n",
        "    tar = tarfile.open(\"charmm36-jul2021.ff.tgz\")\n",
        "    tar.extractall()\n",
        "    tar.close()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        },
        "id": "6C2J-2u78nB2",
        "outputId": "a784ccd9-86a1-4dad-fa86-a92705076fc6"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'./md_1aki/'"
            ]
          },
          "execution_count": 70,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "path"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "0eqhMhZ3NzHU"
      },
      "outputs": [],
      "source": [
        "#@title Change residue names in accordance with pKa calculation\n",
        "pH = 7\n",
        "names              = [\"ASP\",  \"GLU\",  \"HIS\", \"LYS\"]\n",
        "charmmProtonated   = [\"ASPP\", \"GLUP\", \"HSP\", \"LYS\"]\n",
        "charmmDeprotonated = [\"ASP\",  \"GLU\",  \"HIS\", \"LSN\"]\n",
        "amberProtonated    = [\"ASH\",  \"GLH\",  \"HIP\", \"LYS\"]\n",
        "amberDeprotonated  = [\"ASP\",  \"GLU\",  \"HIS\", \"LYN\"]\n",
        "if ff == 'charmm36':\n",
        "    protNames   = charmmProtonated\n",
        "    deprotNames = charmmDeprotonated\n",
        "else:\n",
        "    protNames   = amberProtonated\n",
        "    deprotNames = amberDeprotonated\n",
        "for i in range(-1, mutant_n, 1):\n",
        "    parser = PDBParser()\n",
        "    if i == -1:\n",
        "        structure = parser.get_structure('X', path+\"pdb\"+pdbid+\".ent\")\n",
        "        logname = path+pdbid+\"_processed_pka.log\"\n",
        "        outpath = path+\"input_pka.pdb\"\n",
        "    else:\n",
        "        structure = parser.get_structure('X', f\"./md_mutant_{i}/input.pdb\")\n",
        "        logname = f\"./md_mutant_{i}/input_pka.log\"\n",
        "        outpath = f\"./md_mutant_{i}/input_pka.pdb\"\n",
        "    residues = []\n",
        "    pkas     = []\n",
        "    with open(logname, \"r\") as f:\n",
        "        for line in f:\n",
        "            if line.startswith(\"Residue\"):\n",
        "                continue\n",
        "            if line.strip() == \"\":\n",
        "                continue\n",
        "            resname, resid, chain, pka = line.split()[0].split(\"-\")[0], \\\n",
        "                                         int(line.split()[0].split(\"-\")[1]), \\\n",
        "                                         line.split()[1], \\\n",
        "                                         float(line.split()[2])\n",
        "            residues.append((resname, resid, chain))\n",
        "            pkas.append(pka)\n",
        "\n",
        "    for model in structure:\n",
        "        for chain in model:\n",
        "            for i, residue in enumerate(chain.get_residues()):\n",
        "                if (residue.resname, residue.id[1], chain.id) in residues:\n",
        "                    id = residues.index((residue.resname, residue.id[1], chain.id))\n",
        "                    if (residue.resname in names) and pH < pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            protNames[names.index(residue.resname)]\n",
        "                    if (residue.resname in names) and pH > pkas[id]:\n",
        "                        residue.resname = \\\n",
        "                            deprotNames[names.index(residue.resname)]\n",
        "\n",
        "    pdb_io = PDBIO()\n",
        "    pdb_io.set_structure(structure)\n",
        "    pdb_io.save(outpath)             "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "SG_Rljap7QnB"
      },
      "outputs": [],
      "source": [
        "%%bash\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "\n",
        "sed -i 's/GLUP /GLUP/g' ${value}/input_pka.pdb\n",
        "sed -i 's/ASPP /ASPP/g' ${value}/input_pka.pdb\n",
        "\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CLvZTTAcZe2-",
        "outputId": "425a448b-8a9d-4515-a17f-b2149ee8a40c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 1079 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 1 blocks of water and 207 residues with 1079 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   129   1001  \n",
            "\n",
            "  2 ' '    78     78  (only water)\n",
            "\n",
            "there were 0 atoms with zero occupancy and 74 atoms with          occupancy unequal to one (out of 1079 atoms). Check your pdb file.\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (1001 atoms, 129 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU129 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-129: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 129 residues with 1960 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1984, now 1984\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  127 cmap torsion pairs\n",
            "\n",
            "There are 5187 dihedrals,  373 impropers, 3547 angles\n",
            "          5106 pairs,     1984 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14313.255 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Processing chain 2 (78 atoms, 78 residues)\n",
            "\n",
            "Problem with chain definition, or missing terminal residues. This chain does not appear to contain a recognized chain molecule. If this is incorrect, you can edit residuetypes.dat to modify the behavior.\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 78 residues with 234 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 234, now 234\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are    0 dihedrals,    0 impropers,  234 angles\n",
            "             0 pairs,      234 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 1405.201 a.m.u.\n",
            "\n",
            "Total charge 0.000 e\n",
            "\n",
            "Including chain 1 in system: 1960 atoms 129 residues\n",
            "\n",
            "Including chain 2 in system: 234 atoms 78 residues\n",
            "\n",
            "Now there are 2194 atoms and 207 residues\n",
            "\n",
            "Total mass in system 15718.456 a.m.u.\n",
            "\n",
            "Total charge in system 8.000 e\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 2194 atoms\n",
            "Volume: 61.8999 nm^3, corresponds to roughly 27800 electrons\n",
            "No velocities found\n",
            "    system size :  4.196  4.268  3.455 (nm)\n",
            "    center      :  2.772  2.507  0.003 (nm)\n",
            "    box vectors :  4.197  4.268  3.456 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  61.90               (nm^3)\n",
            "    shift       :  1.326  1.627  3.725 (nm)\n",
            "new center      :  4.098  4.134  3.727 (nm)\n",
            "new box vectors :  8.196  8.268  7.455 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 505.18               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 15945 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -545522963\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 16023      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.443 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 46 SOD ions and 54 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 100 solute molecules in topology file (topol.top)  by 46 SOD and 54 CLA ions.\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 994 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 129 residues with 994 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   129    994  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (994 atoms, 129 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU129 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-129: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 129 residues with 1949 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1972, now 1972\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  127 cmap torsion pairs\n",
            "\n",
            "There are 5155 dihedrals,  373 impropers, 3528 angles\n",
            "          5077 pairs,     1972 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14221.157 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 1949 atoms\n",
            "Volume: 55.7625 nm^3, corresponds to roughly 25000 electrons\n",
            "No velocities found\n",
            "    system size :  4.100  4.270  3.185 (nm)\n",
            "    center      :  2.852  3.153  0.238 (nm)\n",
            "    box vectors :  4.100  4.270  3.185 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  55.76               (nm^3)\n",
            "    shift       :  1.198  0.982  3.355 (nm)\n",
            "new center      :  4.050  4.135  3.593 (nm)\n",
            "new box vectors :  8.100  8.270  7.185 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 481.30               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 15296 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -268973061\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15296      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 43 SOD ions and 51 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 94 solute molecules in topology file (topol.top)  by 43 SOD and 51 CLA ions.\n",
            "Using the Charmm36-jul2021 force field in directory ./charmm36-jul2021.ff\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/carb.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/cgenff.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/ethers.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/lipid.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/metals.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/na.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/silicates.r2b\n",
            "\n",
            "going to rename ./charmm36-jul2021.ff/solvent.r2b\n",
            "Reading input_pka.pdb...\n",
            "Read '', 989 atoms\n",
            "\n",
            "Analyzing pdb file\n",
            "Splitting chemical chains based on TER records or chain id changing.\n",
            "\n",
            "There are 1 chains and 0 blocks of water and 128 residues with 989 atoms\n",
            "\n",
            "  chain  #res #atoms\n",
            "\n",
            "  1 'A'   128    989  \n",
            "\n",
            "All occupancies are one\n",
            "\n",
            "Reading residue database... (Charmm36-jul2021)\n",
            "\n",
            "Processing chain 1 'A' (989 atoms, 128 residues)\n",
            "\n",
            "Identified residue LYS1 as a starting terminus.\n",
            "\n",
            "Identified residue LEU128 as a ending terminus.\n",
            "Start terminus LYS-1: NH3+\n",
            "End terminus LEU-128: COO-\n",
            "\n",
            "Checking for duplicate atoms....\n",
            "\n",
            "Generating any missing hydrogen atoms and/or adding termini.\n",
            "\n",
            "Now there are 128 residues with 1939 atoms\n",
            "\n",
            "Making bonds...\n",
            "\n",
            "Number of bonds was 1962, now 1962\n",
            "\n",
            "Generating angles, dihedrals and pairs...\n",
            "\n",
            "Making cmap torsions...\n",
            "\n",
            "There are  126 cmap torsion pairs\n",
            "\n",
            "There are 5130 dihedrals,  371 impropers, 3510 angles\n",
            "          5052 pairs,     1962 bonds and     0 virtual sites\n",
            "\n",
            "Total mass 14150.078 a.m.u.\n",
            "\n",
            "Total charge 8.000 e\n",
            "\n",
            "Writing topology\n",
            "\n",
            "Writing coordinate file...\n",
            "\n",
            "\t\t--------- PLEASE NOTE ------------\n",
            "\n",
            "You have successfully generated a topology from: input_pka.pdb.\n",
            "\n",
            "The Charmm36-jul2021 force field and the tip3p water model are used.\n",
            "\n",
            "\t\t--------- ETON ESAELP ------------\n",
            "Note that major changes are planned in future for editconf, to improve usability and utility.\n",
            "Read 1939 atoms\n",
            "Volume: 56.0089 nm^3, corresponds to roughly 25200 electrons\n",
            "No velocities found\n",
            "    system size :  3.619  4.216  3.671 (nm)\n",
            "    center      :  2.515  2.529  0.061 (nm)\n",
            "    box vectors :  3.619  4.216  3.671 (nm)\n",
            "    box angles  :  90.00  90.00  90.00 (degrees)\n",
            "    box volume  :  56.01               (nm^3)\n",
            "    shift       :  1.294  1.579  3.775 (nm)\n",
            "new center      :  3.810  4.108  3.835 (nm)\n",
            "new box vectors :  7.619  8.216  7.671 (nm)\n",
            "new box angles  :  90.00  90.00  90.00 (degrees)\n",
            "new box volume  : 480.19               (nm^3)\n",
            "\n",
            "WARNING: No boxtype specified - distance condition applied in each dimension.\n",
            "If the molecule rotates the actual distance will be smaller. You might want\n",
            "to use a cubic box instead, or why not try a dodecahedron today?\n",
            "\n",
            "WARNING: Masses and atomic (Van der Waals) radii will be guessed\n",
            "         based on residue and atom names, since they could not be\n",
            "         definitively assigned from the information in your input\n",
            "         files. These guessed numbers might deviate from the mass\n",
            "         and radius of the atom type. Please check the output\n",
            "         files if necessary. Note, that this functionality may\n",
            "         be removed in a future GROMACS version. Please, consider\n",
            "         using another file format for your input.\n",
            "\n",
            "NOTE: From version 5.0 gmx solvate uses the Van der Waals radii\n",
            "from the source below. This means the results may be different\n",
            "compared to previous GROMACS versions.\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "A. Bondi\n",
            "van der Waals Volumes and Radii\n",
            "J. Phys. Chem. 68 (1964) pp. 441-451\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Adding line for 14617 solvent molecules with resname (SOL) to topology file (topol.top)\n",
            "Setting the LD random seed to -3276969\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "++++ PLEASE READ AND CITE THE FOLLOWING REFERENCE ++++\n",
            "J. S. Hub, B. L. de Groot, H. Grubmueller, G. Groenhof\n",
            "Quantifying Artifacts in Ewald Simulations of Inhomogeneous Systems with a Net\n",
            "Charge\n",
            "J. Chem. Theory Comput. 10 (2014) pp. 381-393\n",
            "-------- -------- --- Thank You --- -------- --------\n",
            "\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14617      Water residues\n",
            "Analysing Protein...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Will try to add 43 SOD ions and 51 CLA ions.\n",
            "Select a continuous group of solvent molecules\n",
            "Selected 13: 'SOL'\n",
            "\n",
            "Processing topology\n",
            "Replacing 94 solute molecules in topology file (topol.top)  by 43 SOD and 51 CLA ions.\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "fatal: destination path 'alphagmx' already exists and is not an empty directory.\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "there were 0 atoms with zero occupancy and 74 atoms with          occupancy unequal to one (out of 1079 atoms). Check your pdb file.\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.24#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 213 donors and 184 acceptors were found.\n",
            "There are 255 hydrogen bonds\n",
            "Will use HISE for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS30   CYS64   CYS76   CYS80\n",
            "                    SG48    SD87  NE2118   SG238   SG513   SG601   SG630\n",
            "   MET12    SD87   1.166\n",
            "   HIS15  NE2118   1.776   1.019\n",
            "   CYS30   SG238   1.406   1.054   2.069\n",
            "   CYS64   SG513   2.835   1.794   1.789   2.241\n",
            "   CYS76   SG601   2.704   1.551   1.468   2.116   0.765\n",
            "   CYS80   SG630   2.959   1.951   1.916   2.391   0.199   0.944\n",
            "   CYS94   SG724   2.550   1.407   1.382   1.975   0.665   0.202   0.855\n",
            "  MET105   SD799   1.827   0.911   1.683   0.888   1.849   1.461   2.036\n",
            "  CYS115   SG889   1.576   1.084   2.078   0.200   2.111   1.989   2.262\n",
            "  CYS127   SG981   0.197   1.072   1.721   1.313   2.799   2.622   2.934\n",
            "                   CYS94  MET105  CYS115\n",
            "                   SG724   SD799   SG889\n",
            "  MET105   SD799   1.381\n",
            "  CYS115   SG889   1.853   0.790\n",
            "  CYS127   SG981   2.475   1.686   1.483\n",
            "Linking CYS-6 SG-48 and CYS-127 SG-981...\n",
            "Linking CYS-30 SG-238 and CYS-115 SG-889...\n",
            "Linking CYS-64 SG-513 and CYS-80 SG-630...\n",
            "Linking CYS-76 SG-601 and CYS-94 SG-724...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5142 pairs\n",
            "Before cleaning: 5187 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.8#\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"Christianity may be OK between consenting adults in private but should not be taught to young children.\" (Francis Crick)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"Christianity may be OK between consenting adults in private but should not be taught to young children.\" (Francis Crick)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x5 boxes\n",
            "Solvent box contains 56946 atoms in 18982 residues\n",
            "Removed 7026 solvent atoms due to solvent-solvent overlap\n",
            "Removed 2085 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 15945 residues\n",
            "Generated solvent containing 47835 atoms in 15945 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.8#\n",
            "\n",
            "Output configuration contains 50029 atoms in 16152 residues\n",
            "Volume                 :     505.185 (nm^3)\n",
            "Density                :     999.014 (g/l)\n",
            "Number of solvent molecules:  15945   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.25#\n",
            "\n",
            "GROMACS reminds you: \"A computer once beat me at chess, but it was no match for me at kick boxing.\" (Emo Philips)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18487]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18487]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 102015.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.8#\n",
            "\n",
            "GROMACS reminds you: \"You got one part of that wrong. This is not meth.\" (Breaking Bad)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 50029 elements\n",
            "Group     1 (        Protein) has  1960 elements\n",
            "Group     2 (      Protein-H) has  1001 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1316 elements\n",
            "Group     9 (    SideChain-H) has   486 elements\n",
            "Group    10 (    Prot-Masses) has  1960 elements\n",
            "Group    11 (    non-Protein) has 48069 elements\n",
            "Group    12 (          Water) has 48069 elements\n",
            "Group    13 (            SOL) has 48069 elements\n",
            "Group    14 (      non-Water) has  1960 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 16023\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.26#\n",
            "Using random seed -1212158546.\n",
            "Replacing solvent molecule 3120 (atom 11320) with SOD\n",
            "Replacing solvent molecule 15187 (atom 47521) with SOD\n",
            "Replacing solvent molecule 5588 (atom 18724) with SOD\n",
            "Replacing solvent molecule 15367 (atom 48061) with SOD\n",
            "Replacing solvent molecule 3087 (atom 11221) with SOD\n",
            "Replacing solvent molecule 4994 (atom 16942) with SOD\n",
            "Replacing solvent molecule 406 (atom 3178) with SOD\n",
            "Replacing solvent molecule 1538 (atom 6574) with SOD\n",
            "Replacing solvent molecule 2170 (atom 8470) with SOD\n",
            "Replacing solvent molecule 1650 (atom 6910) with SOD\n",
            "Replacing solvent molecule 10192 (atom 32536) with SOD\n",
            "Replacing solvent molecule 15719 (atom 49117) with SOD\n",
            "Replacing solvent molecule 9542 (atom 30586) with SOD\n",
            "Replacing solvent molecule 13236 (atom 41668) with SOD\n",
            "Replacing solvent molecule 828 (atom 4444) with SOD\n",
            "Replacing solvent molecule 3539 (atom 12577) with SOD\n",
            "Replacing solvent molecule 14427 (atom 45241) with SOD\n",
            "Replacing solvent molecule 993 (atom 4939) with SOD\n",
            "Replacing solvent molecule 10348 (atom 33004) with SOD\n",
            "Replacing solvent molecule 2263 (atom 8749) with SOD\n",
            "Replacing solvent molecule 1293 (atom 5839) with SOD\n",
            "Replacing solvent molecule 6767 (atom 22261) with SOD\n",
            "Replacing solvent molecule 2727 (atom 10141) with SOD\n",
            "Replacing solvent molecule 8270 (atom 26770) with SOD\n",
            "Replacing solvent molecule 15069 (atom 47167) with SOD\n",
            "Replacing solvent molecule 3560 (atom 12640) with SOD\n",
            "Replacing solvent molecule 14383 (atom 45109) with SOD\n",
            "Replacing solvent molecule 1063 (atom 5149) with SOD\n",
            "Replacing solvent molecule 5456 (atom 18328) with SOD\n",
            "Replacing solvent molecule 8062 (atom 26146) with SOD\n",
            "Replacing solvent molecule 10622 (atom 33826) with SOD\n",
            "Replacing solvent molecule 15323 (atom 47929) with SOD\n",
            "Replacing solvent molecule 7660 (atom 24940) with SOD\n",
            "Replacing solvent molecule 6755 (atom 22225) with SOD\n",
            "Replacing solvent molecule 5553 (atom 18619) with SOD\n",
            "Replacing solvent molecule 14166 (atom 44458) with SOD\n",
            "Replacing solvent molecule 6414 (atom 21202) with SOD\n",
            "Replacing solvent molecule 7880 (atom 25600) with SOD\n",
            "Replacing solvent molecule 2911 (atom 10693) with SOD\n",
            "Replacing solvent molecule 12962 (atom 40846) with SOD\n",
            "Replacing solvent molecule 13934 (atom 43762) with SOD\n",
            "Replacing solvent molecule 6193 (atom 20539) with SOD\n",
            "Replacing solvent molecule 12724 (atom 40132) with SOD\n",
            "Replacing solvent molecule 14476 (atom 45388) with SOD\n",
            "Replacing solvent molecule 538 (atom 3574) with SOD\n",
            "Replacing solvent molecule 6925 (atom 22735) with SOD\n",
            "Replacing solvent molecule 9089 (atom 29227) with CLA\n",
            "Replacing solvent molecule 4494 (atom 15442) with CLA\n",
            "Replacing solvent molecule 8672 (atom 27976) with CLA\n",
            "Replacing solvent molecule 13468 (atom 42364) with CLA\n",
            "Replacing solvent molecule 11760 (atom 37240) with CLA\n",
            "Replacing solvent molecule 15207 (atom 47581) with CLA\n",
            "Replacing solvent molecule 4502 (atom 15466) with CLA\n",
            "Replacing solvent molecule 943 (atom 4789) with CLA\n",
            "Replacing solvent molecule 1865 (atom 7555) with CLA\n",
            "Replacing solvent molecule 6237 (atom 20671) with CLA\n",
            "Replacing solvent molecule 2773 (atom 10279) with CLA\n",
            "Replacing solvent molecule 12632 (atom 39856) with CLA\n",
            "Replacing solvent molecule 5574 (atom 18682) with CLA\n",
            "Replacing solvent molecule 5601 (atom 18763) with CLA\n",
            "Replacing solvent molecule 10288 (atom 32824) with CLA\n",
            "Replacing solvent molecule 15513 (atom 48499) with CLA\n",
            "Replacing solvent molecule 5876 (atom 19588) with CLA\n",
            "Replacing solvent molecule 5209 (atom 17587) with CLA\n",
            "Replacing solvent molecule 4189 (atom 14527) with CLA\n",
            "Replacing solvent molecule 4752 (atom 16216) with CLA\n",
            "Replacing solvent molecule 5558 (atom 18634) with CLA\n",
            "Replacing solvent molecule 10725 (atom 34135) with CLA\n",
            "Replacing solvent molecule 12096 (atom 38248) with CLA\n",
            "Replacing solvent molecule 2218 (atom 8614) with CLA\n",
            "Replacing solvent molecule 13510 (atom 42490) with CLA\n",
            "Replacing solvent molecule 9932 (atom 31756) with CLA\n",
            "Replacing solvent molecule 3225 (atom 11635) with CLA\n",
            "Replacing solvent molecule 7218 (atom 23614) with CLA\n",
            "Replacing solvent molecule 13737 (atom 43171) with CLA\n",
            "Replacing solvent molecule 4712 (atom 16096) with CLA\n",
            "Replacing solvent molecule 154 (atom 2422) with CLA\n",
            "Replacing solvent molecule 12685 (atom 40015) with CLA\n",
            "Replacing solvent molecule 1505 (atom 6475) with CLA\n",
            "Replacing solvent molecule 15764 (atom 49252) with CLA\n",
            "Replacing solvent molecule 7744 (atom 25192) with CLA\n",
            "Replacing solvent molecule 9720 (atom 31120) with CLA\n",
            "Replacing solvent molecule 907 (atom 4681) with CLA\n",
            "Replacing solvent molecule 12388 (atom 39124) with CLA\n",
            "Replacing solvent molecule 4508 (atom 15484) with CLA\n",
            "Replacing solvent molecule 7423 (atom 24229) with CLA\n",
            "Replacing solvent molecule 4563 (atom 15649) with CLA\n",
            "Replacing solvent molecule 9045 (atom 29095) with CLA\n",
            "Replacing solvent molecule 9449 (atom 30307) with CLA\n",
            "Replacing solvent molecule 120 (atom 2320) with CLA\n",
            "Replacing solvent molecule 9605 (atom 30775) with CLA\n",
            "Replacing solvent molecule 11801 (atom 37363) with CLA\n",
            "Replacing solvent molecule 10387 (atom 33121) with CLA\n",
            "Replacing solvent molecule 1884 (atom 7612) with CLA\n",
            "Replacing solvent molecule 13529 (atom 42547) with CLA\n",
            "Replacing solvent molecule 8905 (atom 28675) with CLA\n",
            "Replacing solvent molecule 10997 (atom 34951) with CLA\n",
            "Replacing solvent molecule 1243 (atom 5689) with CLA\n",
            "Replacing solvent molecule 10303 (atom 32869) with CLA\n",
            "Replacing solvent molecule 7775 (atom 25285) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.8#\n",
            "\n",
            "GROMACS reminds you: \"You got one part of that wrong. This is not meth.\" (Breaking Bad)\n",
            "\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.3#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 212 donors and 183 acceptors were found.\n",
            "There are 269 hydrogen bonds\n",
            "Will use HISD for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS30   CYS64   CYS76   CYS80\n",
            "                    SG46    SD85  NE2114   SG229   SG504   SG592   SG621\n",
            "   MET12    SD85   1.157\n",
            "   HIS15  NE2114   1.595   0.933\n",
            "   CYS30   SG229   1.573   1.021   1.935\n",
            "   CYS64   SG504   2.839   1.837   1.695   2.220\n",
            "   CYS76   SG592   2.687   1.577   1.402   2.089   0.761\n",
            "   CYS80   SG621   2.966   2.000   1.822   2.379   0.204   0.942\n",
            "   CYS94   SG715   2.527   1.424   1.297   1.933   0.678   0.204   0.871\n",
            "  MET105   SD790   1.969   0.917   1.548   0.977   1.764   1.346   1.961\n",
            "  CYS115   SG880   1.731   1.048   1.933   0.203   2.091   1.953   2.254\n",
            "  CYS127   SG972   0.202   1.038   1.578   1.403   2.779   2.599   2.915\n",
            "                   CYS94  MET105  CYS115\n",
            "                   SG715   SD790   SG880\n",
            "  MET105   SD790   1.255\n",
            "  CYS115   SG880   1.803   0.850\n",
            "  CYS127   SG972   2.441   1.804   1.563\n",
            "Linking CYS-6 SG-46 and CYS-127 SG-972...\n",
            "Linking CYS-30 SG-229 and CYS-115 SG-880...\n",
            "Linking CYS-64 SG-504 and CYS-80 SG-621...\n",
            "Linking CYS-76 SG-592 and CYS-94 SG-715...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5110 pairs\n",
            "Before cleaning: 5155 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.1#\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x4 boxes\n",
            "Solvent box contains 53784 atoms in 17928 residues\n",
            "Removed 6102 solvent atoms due to solvent-solvent overlap\n",
            "Removed 1794 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 15296 residues\n",
            "Generated solvent containing 45888 atoms in 15296 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.1#\n",
            "\n",
            "Output configuration contains 47837 atoms in 15425 residues\n",
            "Volume                 :     481.302 (nm^3)\n",
            "Density                :     1003.07 (g/l)\n",
            "Number of solvent molecules:  15296   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.4#\n",
            "\n",
            "GROMACS reminds you: \"Those people who think they know everything are a great annoyance to those of us who do.\" (Isaac Asimov)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18383]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18383]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 97620.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"FORTRAN, the infantile disorder, by now nearly 20 years old, is hopelessly inadequate for whatever computer application you have in mind today: it is now too clumsy, too risky, and too expensive to use.\" (Edsger Dijkstra, 1970)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 47837 elements\n",
            "Group     1 (        Protein) has  1949 elements\n",
            "Group     2 (      Protein-H) has   994 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1305 elements\n",
            "Group     9 (    SideChain-H) has   479 elements\n",
            "Group    10 (    Prot-Masses) has  1949 elements\n",
            "Group    11 (    non-Protein) has 45888 elements\n",
            "Group    12 (          Water) has 45888 elements\n",
            "Group    13 (            SOL) has 45888 elements\n",
            "Group    14 (      non-Water) has  1949 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 15296\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.5#\n",
            "Using random seed -18945541.\n",
            "Replacing solvent molecule 4699 (atom 16046) with SOD\n",
            "Replacing solvent molecule 1242 (atom 5675) with SOD\n",
            "Replacing solvent molecule 2979 (atom 10886) with SOD\n",
            "Replacing solvent molecule 3149 (atom 11396) with SOD\n",
            "Replacing solvent molecule 3555 (atom 12614) with SOD\n",
            "Replacing solvent molecule 3072 (atom 11165) with SOD\n",
            "Replacing solvent molecule 15153 (atom 47408) with SOD\n",
            "Replacing solvent molecule 1757 (atom 7220) with SOD\n",
            "Replacing solvent molecule 8199 (atom 26546) with SOD\n",
            "Replacing solvent molecule 3001 (atom 10952) with SOD\n",
            "Replacing solvent molecule 10428 (atom 33233) with SOD\n",
            "Replacing solvent molecule 13187 (atom 41510) with SOD\n",
            "Replacing solvent molecule 11192 (atom 35525) with SOD\n",
            "Replacing solvent molecule 9601 (atom 30752) with SOD\n",
            "Replacing solvent molecule 1491 (atom 6422) with SOD\n",
            "Replacing solvent molecule 4559 (atom 15626) with SOD\n",
            "Replacing solvent molecule 3558 (atom 12623) with SOD\n",
            "Replacing solvent molecule 3574 (atom 12671) with SOD\n",
            "Replacing solvent molecule 11886 (atom 37607) with SOD\n",
            "Replacing solvent molecule 4546 (atom 15587) with SOD\n",
            "Replacing solvent molecule 8377 (atom 27080) with SOD\n",
            "Replacing solvent molecule 10126 (atom 32327) with SOD\n",
            "Replacing solvent molecule 2592 (atom 9725) with SOD\n",
            "Replacing solvent molecule 217 (atom 2600) with SOD\n",
            "Replacing solvent molecule 7275 (atom 23774) with SOD\n",
            "Replacing solvent molecule 13300 (atom 41849) with SOD\n",
            "Replacing solvent molecule 534 (atom 3551) with SOD\n",
            "Replacing solvent molecule 64 (atom 2141) with SOD\n",
            "Replacing solvent molecule 7661 (atom 24932) with SOD\n",
            "Replacing solvent molecule 4731 (atom 16142) with SOD\n",
            "Replacing solvent molecule 14161 (atom 44432) with SOD\n",
            "Replacing solvent molecule 6104 (atom 20261) with SOD\n",
            "Replacing solvent molecule 12082 (atom 38195) with SOD\n",
            "Replacing solvent molecule 12550 (atom 39599) with SOD\n",
            "Replacing solvent molecule 2564 (atom 9641) with SOD\n",
            "Replacing solvent molecule 9175 (atom 29474) with SOD\n",
            "Replacing solvent molecule 2541 (atom 9572) with SOD\n",
            "Replacing solvent molecule 14492 (atom 45425) with SOD\n",
            "Replacing solvent molecule 8720 (atom 28109) with SOD\n",
            "Replacing solvent molecule 11414 (atom 36191) with SOD\n",
            "Replacing solvent molecule 3911 (atom 13682) with SOD\n",
            "Replacing solvent molecule 6298 (atom 20843) with SOD\n",
            "Replacing solvent molecule 12840 (atom 40469) with SOD\n",
            "Replacing solvent molecule 3865 (atom 13544) with CLA\n",
            "Replacing solvent molecule 14254 (atom 44711) with CLA\n",
            "Replacing solvent molecule 10624 (atom 33821) with CLA\n",
            "Replacing solvent molecule 11130 (atom 35339) with CLA\n",
            "Replacing solvent molecule 7957 (atom 25820) with CLA\n",
            "Replacing solvent molecule 12782 (atom 40295) with CLA\n",
            "Replacing solvent molecule 5100 (atom 17249) with CLA\n",
            "Replacing solvent molecule 5103 (atom 17258) with CLA\n",
            "Replacing solvent molecule 1920 (atom 7709) with CLA\n",
            "Replacing solvent molecule 12816 (atom 40397) with CLA\n",
            "Replacing solvent molecule 10080 (atom 32189) with CLA\n",
            "Replacing solvent molecule 9507 (atom 30470) with CLA\n",
            "Replacing solvent molecule 7058 (atom 23123) with CLA\n",
            "Replacing solvent molecule 4847 (atom 16490) with CLA\n",
            "Replacing solvent molecule 6591 (atom 21722) with CLA\n",
            "Replacing solvent molecule 3464 (atom 12341) with CLA\n",
            "Replacing solvent molecule 13938 (atom 43763) with CLA\n",
            "Replacing solvent molecule 2802 (atom 10355) with CLA\n",
            "Replacing solvent molecule 6635 (atom 21854) with CLA\n",
            "Replacing solvent molecule 5393 (atom 18128) with CLA\n",
            "Replacing solvent molecule 2855 (atom 10514) with CLA\n",
            "Replacing solvent molecule 799 (atom 4346) with CLA\n",
            "Replacing solvent molecule 6675 (atom 21974) with CLA\n",
            "Replacing solvent molecule 2638 (atom 9863) with CLA\n",
            "Replacing solvent molecule 14862 (atom 46535) with CLA\n",
            "Replacing solvent molecule 11998 (atom 37943) with CLA\n",
            "Replacing solvent molecule 2005 (atom 7964) with CLA\n",
            "Replacing solvent molecule 2158 (atom 8423) with CLA\n",
            "Replacing solvent molecule 8083 (atom 26198) with CLA\n",
            "Replacing solvent molecule 14962 (atom 46835) with CLA\n",
            "Replacing solvent molecule 15110 (atom 47279) with CLA\n",
            "Replacing solvent molecule 14551 (atom 45602) with CLA\n",
            "Replacing solvent molecule 8461 (atom 27332) with CLA\n",
            "Replacing solvent molecule 7638 (atom 24863) with CLA\n",
            "Replacing solvent molecule 10798 (atom 34343) with CLA\n",
            "Replacing solvent molecule 1297 (atom 5840) with CLA\n",
            "Replacing solvent molecule 4125 (atom 14324) with CLA\n",
            "Replacing solvent molecule 12565 (atom 39644) with CLA\n",
            "Replacing solvent molecule 10199 (atom 32546) with CLA\n",
            "Replacing solvent molecule 1617 (atom 6800) with CLA\n",
            "Replacing solvent molecule 10208 (atom 32573) with CLA\n",
            "Replacing solvent molecule 14757 (atom 46220) with CLA\n",
            "Replacing solvent molecule 4606 (atom 15767) with CLA\n",
            "Replacing solvent molecule 10031 (atom 32042) with CLA\n",
            "Replacing solvent molecule 8085 (atom 26204) with CLA\n",
            "Replacing solvent molecule 14610 (atom 45779) with CLA\n",
            "Replacing solvent molecule 4347 (atom 14990) with CLA\n",
            "Replacing solvent molecule 1144 (atom 5381) with CLA\n",
            "Replacing solvent molecule 1587 (atom 6710) with CLA\n",
            "Replacing solvent molecule 8860 (atom 28529) with CLA\n",
            "Replacing solvent molecule 10000 (atom 31949) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"This is Tense !\" (Star Wars Episode I The Phantom Menace)\n",
            "\n",
            "                     :-) GROMACS - gmx pdb2gmx, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx pdb2gmx -f input_pka.pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
            "\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/na.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.r2b\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.r2b\n",
            "All occupancies are one\n",
            "Opening force field file ./charmm36-jul2021.ff/atomtypes.atp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/na.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.rtp\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.hdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.n.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/carb.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/cgenff.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/ethers.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/lipid.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/metals.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/na.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/silicates.c.tdb\n",
            "Opening force field file ./charmm36-jul2021.ff/solvent.c.tdb\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.3#\n",
            "Analysing hydrogen-bonding network for automated assignment of histidine\n",
            " protonation. 211 donors and 182 acceptors were found.\n",
            "There are 263 hydrogen bonds\n",
            "Will use HISE for residue 15\n",
            "8 out of 8 lines of specbond.dat converted successfully\n",
            "Special Atom Distance matrix:\n",
            "                    CYS6   MET12   HIS15   CYS29   CYS63   CYS75   CYS79\n",
            "                    SG46    SD85  NE2114   SG224   SG499   SG587   SG616\n",
            "   MET12    SD85   1.203\n",
            "   HIS15  NE2114   1.736   0.884\n",
            "   CYS29   SG224   1.457   1.021   1.889\n",
            "   CYS63   SG499   2.872   1.804   1.637   2.238\n",
            "   CYS75   SG587   2.744   1.559   1.343   2.095   0.760\n",
            "   CYS79   SG616   2.993   1.962   1.764   2.395   0.204   0.943\n",
            "   CYS93   SG710   2.578   1.404   1.235   1.942   0.677   0.203   0.870\n",
            "  MET104   SD785   1.857   0.903   1.552   0.831   1.894   1.505   2.086\n",
            "  CYS114   SG875   1.622   1.038   1.874   0.203   2.092   1.954   2.252\n",
            "  CYS126   SG967   0.203   1.103   1.686   1.340   2.832   2.659   2.964\n",
            "                   CYS93  MET104  CYS114\n",
            "                   SG710   SD785   SG875\n",
            "  MET104   SD785   1.407\n",
            "  CYS114   SG875   1.805   0.731\n",
            "  CYS126   SG967   2.500   1.703   1.506\n",
            "Linking CYS-6 SG-46 and CYS-126 SG-967...\n",
            "Linking CYS-29 SG-224 and CYS-114 SG-875...\n",
            "Linking CYS-63 SG-499 and CYS-79 SG-616...\n",
            "Linking CYS-75 SG-587 and CYS-93 SG-710...\n",
            "Opening force field file ./charmm36-jul2021.ff/aminoacids.arn\n",
            "Before cleaning: 5085 pairs\n",
            "Before cleaning: 5130 dihedrals\n",
            "\n",
            "Back Off! I just backed up posre.itp to ./#posre.itp.1#\n",
            "\n",
            "Back Off! I just backed up 1_prot.gro to ./#1_prot.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                     :-) GROMACS - gmx editconf, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
            "\n",
            "\n",
            "Back Off! I just backed up 2_box.gro to ./#2_box.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                     :-) GROMACS - gmx solvate, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
            "\n",
            "Reading solute configuration\n",
            "Reading solvent configuration\n",
            "\n",
            "Initialising inter-atomic distances...\n",
            "Generating solvent configuration\n",
            "Will generate new solvent configuration of 5x5x5 boxes\n",
            "Solvent box contains 54135 atoms in 18045 residues\n",
            "Removed 8490 solvent atoms due to solvent-solvent overlap\n",
            "Removed 1794 solvent atoms due to solute-solvent overlap\n",
            "Sorting configuration\n",
            "Found 1 molecule type:\n",
            "    SOL (   3 atoms): 14617 residues\n",
            "Generated solvent containing 43851 atoms in 14617 residues\n",
            "Writing generated configuration to 3_solv.gro\n",
            "\n",
            "Back Off! I just backed up 3_solv.gro to ./#3_solv.gro.1#\n",
            "\n",
            "Output configuration contains 45790 atoms in 14745 residues\n",
            "Volume                 :     480.187 (nm^3)\n",
            "Density                :     962.833 (g/l)\n",
            "Number of solvent molecules:  14617   \n",
            "\n",
            "Processing topology\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.4#\n",
            "\n",
            "GROMACS reminds you: \"Problems worthy / of attack / prove their worth / by hitting back.\" (Piet Hein)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "\n",
            "NOTE 1 [file topol.top, line 18291]:\n",
            "  System has non-zero total charge: 8.000000\n",
            "  Total charge should normally be an integer. See\n",
            "  http://www.gromacs.org/Documentation/Floating_Point_Arithmetic\n",
            "  for discussion on how close it should be to an integer.\n",
            "  \n",
            "\n",
            "\n",
            "\n",
            "WARNING 1 [file topol.top, line 18291]:\n",
            "  You are using Ewald electrostatics in a system with net charge. This can\n",
            "  lead to severe artifacts, such as ions moving into regions with low\n",
            "  dielectric, due to the uniform background charge. We suggest to\n",
            "  neutralize your system with counter ions, possibly in combination with a\n",
            "  physiological salt concentration.\n",
            "\n",
            "Number of degrees of freedom in T-Coupling group rest is 93516.00\n",
            "\n",
            "There was 1 note\n",
            "\n",
            "There was 1 warning\n",
            "\n",
            "Back Off! I just backed up 4_ions.tpr to ./#4_ions.tpr.1#\n",
            "\n",
            "GROMACS reminds you: \"There is no place like ~\" (Anonymous)\n",
            "\n",
            "                      :-) GROMACS - gmx genion, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA\n",
            "\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file 4_ions.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 45790 elements\n",
            "Group     1 (        Protein) has  1939 elements\n",
            "Group     2 (      Protein-H) has   989 elements\n",
            "Group     3 (        C-alpha) has   128 elements\n",
            "Group     4 (       Backbone) has   384 elements\n",
            "Group     5 (      MainChain) has   511 elements\n",
            "Group     6 (   MainChain+Cb) has   627 elements\n",
            "Group     7 (    MainChain+H) has   639 elements\n",
            "Group     8 (      SideChain) has  1300 elements\n",
            "Group     9 (    SideChain-H) has   478 elements\n",
            "Group    10 (    Prot-Masses) has  1939 elements\n",
            "Group    11 (    non-Protein) has 43851 elements\n",
            "Group    12 (          Water) has 43851 elements\n",
            "Group    13 (            SOL) has 43851 elements\n",
            "Group    14 (      non-Water) has  1939 elements\n",
            "Select a group: Number of (3-atomic) solvent molecules: 14617\n",
            "\n",
            "Back Off! I just backed up topol.top to ./#topol.top.5#\n",
            "Using random seed -1844226.\n",
            "Replacing solvent molecule 10305 (atom 32854) with SOD\n",
            "Replacing solvent molecule 4693 (atom 16018) with SOD\n",
            "Replacing solvent molecule 11875 (atom 37564) with SOD\n",
            "Replacing solvent molecule 7423 (atom 24208) with SOD\n",
            "Replacing solvent molecule 11092 (atom 35215) with SOD\n",
            "Replacing solvent molecule 3175 (atom 11464) with SOD\n",
            "Replacing solvent molecule 14455 (atom 45304) with SOD\n",
            "Replacing solvent molecule 10744 (atom 34171) with SOD\n",
            "Replacing solvent molecule 14153 (atom 44398) with SOD\n",
            "Replacing solvent molecule 11232 (atom 35635) with SOD\n",
            "Replacing solvent molecule 6065 (atom 20134) with SOD\n",
            "Replacing solvent molecule 2709 (atom 10066) with SOD\n",
            "Replacing solvent molecule 6948 (atom 22783) with SOD\n",
            "Replacing solvent molecule 11038 (atom 35053) with SOD\n",
            "Replacing solvent molecule 1409 (atom 6166) with SOD\n",
            "Replacing solvent molecule 2786 (atom 10297) with SOD\n",
            "Replacing solvent molecule 6377 (atom 21070) with SOD\n",
            "Replacing solvent molecule 11463 (atom 36328) with SOD\n",
            "Replacing solvent molecule 13777 (atom 43270) with SOD\n",
            "Replacing solvent molecule 14205 (atom 44554) with SOD\n",
            "Replacing solvent molecule 9768 (atom 31243) with SOD\n",
            "Replacing solvent molecule 9079 (atom 29176) with SOD\n",
            "Replacing solvent molecule 8721 (atom 28102) with SOD\n",
            "Replacing solvent molecule 13983 (atom 43888) with SOD\n",
            "Replacing solvent molecule 8627 (atom 27820) with SOD\n",
            "Replacing solvent molecule 7954 (atom 25801) with SOD\n",
            "Replacing solvent molecule 11322 (atom 35905) with SOD\n",
            "Replacing solvent molecule 5618 (atom 18793) with SOD\n",
            "Replacing solvent molecule 5388 (atom 18103) with SOD\n",
            "Replacing solvent molecule 11920 (atom 37699) with SOD\n",
            "Replacing solvent molecule 8283 (atom 26788) with SOD\n",
            "Replacing solvent molecule 13695 (atom 43024) with SOD\n",
            "Replacing solvent molecule 1413 (atom 6178) with SOD\n",
            "Replacing solvent molecule 13877 (atom 43570) with SOD\n",
            "Replacing solvent molecule 7340 (atom 23959) with SOD\n",
            "Replacing solvent molecule 11932 (atom 37735) with SOD\n",
            "Replacing solvent molecule 13059 (atom 41116) with SOD\n",
            "Replacing solvent molecule 1107 (atom 5260) with SOD\n",
            "Replacing solvent molecule 42 (atom 2065) with SOD\n",
            "Replacing solvent molecule 11703 (atom 37048) with SOD\n",
            "Replacing solvent molecule 3816 (atom 13387) with SOD\n",
            "Replacing solvent molecule 11476 (atom 36367) with SOD\n",
            "Replacing solvent molecule 10428 (atom 33223) with SOD\n",
            "Replacing solvent molecule 12052 (atom 38095) with CLA\n",
            "Replacing solvent molecule 1534 (atom 6541) with CLA\n",
            "Replacing solvent molecule 10248 (atom 32683) with CLA\n",
            "Replacing solvent molecule 3559 (atom 12616) with CLA\n",
            "Replacing solvent molecule 9675 (atom 30964) with CLA\n",
            "Replacing solvent molecule 7467 (atom 24340) with CLA\n",
            "Replacing solvent molecule 14130 (atom 44329) with CLA\n",
            "Replacing solvent molecule 7711 (atom 25072) with CLA\n",
            "Replacing solvent molecule 8317 (atom 26890) with CLA\n",
            "Replacing solvent molecule 12305 (atom 38854) with CLA\n",
            "Replacing solvent molecule 10160 (atom 32419) with CLA\n",
            "Replacing solvent molecule 2324 (atom 8911) with CLA\n",
            "Replacing solvent molecule 4139 (atom 14356) with CLA\n",
            "Replacing solvent molecule 10557 (atom 33610) with CLA\n",
            "Replacing solvent molecule 5342 (atom 17965) with CLA\n",
            "Replacing solvent molecule 13176 (atom 41467) with CLA\n",
            "Replacing solvent molecule 2464 (atom 9331) with CLA\n",
            "Replacing solvent molecule 913 (atom 4678) with CLA\n",
            "Replacing solvent molecule 9019 (atom 28996) with CLA\n",
            "Replacing solvent molecule 4260 (atom 14719) with CLA\n",
            "Replacing solvent molecule 13406 (atom 42157) with CLA\n",
            "Replacing solvent molecule 13582 (atom 42685) with CLA\n",
            "Replacing solvent molecule 7918 (atom 25693) with CLA\n",
            "Replacing solvent molecule 7732 (atom 25135) with CLA\n",
            "Replacing solvent molecule 8122 (atom 26305) with CLA\n",
            "Replacing solvent molecule 8083 (atom 26188) with CLA\n",
            "Replacing solvent molecule 3498 (atom 12433) with CLA\n",
            "Replacing solvent molecule 6610 (atom 21769) with CLA\n",
            "Replacing solvent molecule 8076 (atom 26167) with CLA\n",
            "Replacing solvent molecule 2922 (atom 10705) with CLA\n",
            "Replacing solvent molecule 8017 (atom 25990) with CLA\n",
            "Replacing solvent molecule 626 (atom 3817) with CLA\n",
            "Replacing solvent molecule 11016 (atom 34987) with CLA\n",
            "Replacing solvent molecule 3660 (atom 12919) with CLA\n",
            "Replacing solvent molecule 6582 (atom 21685) with CLA\n",
            "Replacing solvent molecule 3473 (atom 12358) with CLA\n",
            "Replacing solvent molecule 2282 (atom 8785) with CLA\n",
            "Replacing solvent molecule 10410 (atom 33169) with CLA\n",
            "Replacing solvent molecule 6564 (atom 21631) with CLA\n",
            "Replacing solvent molecule 7750 (atom 25189) with CLA\n",
            "Replacing solvent molecule 1754 (atom 7201) with CLA\n",
            "Replacing solvent molecule 3153 (atom 11398) with CLA\n",
            "Replacing solvent molecule 11838 (atom 37453) with CLA\n",
            "Replacing solvent molecule 7213 (atom 23578) with CLA\n",
            "Replacing solvent molecule 11313 (atom 35878) with CLA\n",
            "Replacing solvent molecule 3648 (atom 12883) with CLA\n",
            "Replacing solvent molecule 2010 (atom 7969) with CLA\n",
            "Replacing solvent molecule 2357 (atom 9010) with CLA\n",
            "Replacing solvent molecule 1510 (atom 6469) with CLA\n",
            "Replacing solvent molecule 12574 (atom 39661) with CLA\n",
            "Replacing solvent molecule 2142 (atom 8365) with CLA\n",
            "\n",
            "\n",
            "Back Off! I just backed up 4_ions.gro to ./#4_ions.gro.1#\n",
            "\n",
            "GROMACS reminds you: \"There is no place like ~\" (Anonymous)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run pdb2gmx, editconf, solvate and genion\n",
        "%%bash\n",
        "git clone https://github.com/pbuslaev/alphagmx.git\n",
        "\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "in_pdb=input_pka.pdb\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "cp -r amber14sb.ff $value/.\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber14sb\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "cp alphagmx/parameters/amber/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -ter -water spce -ff amber99sb-ildn\n",
        "fi\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "cp -r charmm36-jul2021.ff $value/.\n",
        "cp alphagmx/parameters/charmm/*mdp $value/.\n",
        "cd $value\n",
        "gmx pdb2gmx -f $in_pdb -o 1_prot.gro -p -ignh -water tip3p -ff charmm36-jul2021\n",
        "fi\n",
        "gmx editconf -f 1_prot.gro -o 2_box.gro -d 2\n",
        "gmx solvate -cp 2_box.gro -cs -p topol.top -o 3_solv.gro\n",
        "# Here we should apply some tricks Issue !4\n",
        "# https://github.com/pbuslaev/alphagmx/issues/4\n",
        "gmx grompp -f ions.mdp -c 3_solv.gro -p topol.top -o 4_ions.tpr -maxwarn 1\n",
        "#This is a trick to provide interactive options to gmx\n",
        "echo \"SOL\" > options\n",
        "echo \" \" >> options\n",
        "if [ ${FF} == \"charmm36\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname SOD -nname CLA < options\n",
        "fi\n",
        "if [ ${FF} == \"amber99\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "if [ ${FF} == \"amber14\" ]; then\n",
        "gmx genion -s 4_ions.tpr -p topol.top -o 4_ions.gro -neutral -conc 0.15 -pname NA -nname CL < options\n",
        "fi\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "p-TrrUBhmPVe",
        "outputId": "562691d8-5548-491d-ba8f-52680b4fccd3"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Setting the LD random seed to -1622155281\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.443 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -908066849\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.442 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -771825809\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.441 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.137 0.138 0.133\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -272629796\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.14\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -1109426757\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.439 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -67150989\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.441 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.135 0.138 0.138\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -46239765\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.449 nm\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.15\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -25428163\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.446 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.211 nm, buffer size 0.011 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n",
            "Setting the LD random seed to -268501761\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.444 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.233 nm, buffer size 0.033 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.136 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 4 Mb of data\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 101715.00\n",
            "\n",
            "GROMACS reminds you: \"Chemistry: It tends to be a messy science.\" (Gunnar von Heijne, former chair of the Nobel Committee for chemistry)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 182 steps\n",
            "Potential Energy  = -7.8605156e+05\n",
            "Maximum force     =  9.0188641e+02 on atom 1366\n",
            "Norm of force     =  4.8009664e+01\n",
            "\n",
            "GROMACS reminds you: \"I Do It All the Time\" (Magnapop)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"In a talk you have a choice: You can make one point or no points.\" (Paul Sigler)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.298\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       63.801       31.901      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       27.087        0.886\n",
            "\n",
            "GROMACS reminds you: \"I am at two with nature.\" (Woody Allen)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"I cannot think of a single one, not even intelligence.\" (Enrico Fermi, when asked what characteristics physics Nobel laureates had in common.)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.343\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       26.701       13.351      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       64.728        0.371\n",
            "\n",
            "GROMACS reminds you: \"Uh-oh\" (Tinky Winky)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 97338.00\n",
            "\n",
            "GROMACS reminds you: \"I had a polynomial once. My doctor removed it.\" (Michael Grant)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "step 15: One or more water molecules can not be settled.\n",
            "Check for bad contacts and/or reduce the timestep if appropriate.\n",
            "Wrote pdb files with previous and current coordinates\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 364 steps\n",
            "Potential Energy  = -7.6430638e+05\n",
            "Maximum force     =  8.2403162e+02 on atom 164\n",
            "Norm of force     =  3.2836407e+01\n",
            "\n",
            "GROMACS reminds you: \"We'll celebrate a woman for anything, as long as it's not her talent.\" (Colleen McCullough)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"Since the mathematicians have invaded the theory of relativity I do not understand it myself any more.\" (Albert Einstein)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.298\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       70.294       35.147      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       24.585        0.976\n",
            "\n",
            "GROMACS reminds you: \"He's using code that only you and I know\" (Kate Bush)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"The time for theory is over\" (J. Hajdu)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       29.517       14.759      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       58.554        0.410\n",
            "\n",
            "GROMACS reminds you: \"All sorts of things can happen when you're open to new ideas and playing around with things.\" (Stephanie Kwolek, inventor of Kevlar)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group rest is 93234.00\n",
            "\n",
            "GROMACS reminds you: \"Garbage Collecting...\" (GNU Emacs)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 5_min\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 5_min.tpr, VERSION 2022.3 (single precision)\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 1 GPU task in the 1 rank on this node:\n",
            "  PP:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "\n",
            "Steepest Descents:\n",
            "   Tolerance (Fmax)   =  1.00000e+03\n",
            "   Number of steps    =        10000\n",
            "\n",
            "writing lowest energy coordinates.\n",
            "\n",
            "Steepest Descents converged to Fmax < 1000 in 638 steps\n",
            "Potential Energy  = -7.4052944e+05\n",
            "Maximum force     =  8.1008386e+02 on atom 1107\n",
            "Norm of force     =  2.2840394e+01\n",
            "\n",
            "GROMACS reminds you: \"Shake Yourself\" (YES)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"You Should Sleep Late Man, It's Much Easier On Your Constitution\" (Beastie Boys)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 6_nvt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 6_nvt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.296\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "10000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       72.276       36.138      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       23.911        1.004\n",
            "\n",
            "GROMACS reminds you: \"All Beauty Must Die\" (Nick Cave)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"One Ripple At a Time\" (Bianca's Smut Shack)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm 7_npt\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file 7_npt.tpr, VERSION 2022.3 (single precision)\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.341\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "5000 steps,     10.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:       29.146       14.573      200.0\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       59.299        0.405\n",
            "\n",
            "GROMACS reminds you: \"Remember, being healthy is basically dying as slowly as possible.\" (Ricky Gervais)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title Run equilibration\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f min.mdp -c 4_ions.gro -p topol.top -o 5_min.tpr -r 4_ions.gro \n",
        "gmx mdrun -deffnm 5_min\n",
        "gmx grompp -f nvt.mdp -c 5_min.gro -p topol.top -o 6_nvt.tpr -r 5_min.gro \n",
        "gmx mdrun -deffnm 6_nvt\n",
        "gmx grompp -f npt.mdp -c 6_nvt.gro -p topol.top -o 7_npt.tpr -r 6_nvt.gro \n",
        "gmx mdrun -deffnm 7_npt\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GQqWYtC4_xvq",
        "outputId": "1209da99-439d-4e66-9d8f-19e349992a6b"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Setting the LD random seed to -889201811\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15923      Water residues\n",
            "There are:   100      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.450 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x56, spacing 0.136 0.137 0.132\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 30 Mb of data\n",
            "Setting the LD random seed to -86184001\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   129    Protein residues\n",
            "There are: 15202      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.452 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 60x60x52, spacing 0.134 0.137 0.137\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 29 Mb of data\n",
            "Setting the LD random seed to -788481\n",
            "\n",
            "Generated 167799 of the 167910 non-bonded parameter combinations\n",
            "\n",
            "Generated 117519 of the 167910 1-4 parameter combinations\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'Protein_chain_A'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 2 bonded neighbours molecule type 'SOL'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'SOD'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "\n",
            "Excluding 3 bonded neighbours molecule type 'CLA'\n",
            "\n",
            "turning H bonds into constraints...\n",
            "Analysing residue names:\n",
            "There are:   128    Protein residues\n",
            "There are: 14523      Water residues\n",
            "There are:    94      Other residues\n",
            "Analysing Protein...\n",
            "Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...\n",
            "\n",
            "The largest distance between excluded atoms is 0.450 nm\n",
            "\n",
            "Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K\n",
            "\n",
            "Calculated rlist for 1x1 atom pair-list as 1.234 nm, buffer size 0.034 nm\n",
            "\n",
            "Set rlist, assuming 4x4 atom pair-list, to 1.200 nm, buffer size 0.000 nm\n",
            "\n",
            "Note that mdrun will redetermine rlist based on the actual pair-list setup\n",
            "Calculating fourier grid dimensions for X Y Z\n",
            "Using a fourier grid of 56x60x56, spacing 0.134 0.135 0.135\n",
            "\n",
            "Estimate for the relative computational load of the PME mesh part: 0.16\n",
            "\n",
            "This run will generate roughly 27 Mb of data\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 100756.00\n",
            "\n",
            "GROMACS reminds you: \"In mathematics you don't understand things, you just get used to them\" (John von Neumann)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.345\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     5581.285     2790.643      200.0\n",
            "                         46:30\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       61.921        0.388\n",
            "\n",
            "GROMACS reminds you: \"A Pretty Village Burning Makes a Pretty Fire\" (David Sandstrom)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 96383.00\n",
            "\n",
            "GROMACS reminds you: \"We Don't Bother Anyone\" (LIVE)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.345\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     4993.828     2496.914      200.0\n",
            "                         41:36\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       69.205        0.347\n",
            "\n",
            "GROMACS reminds you: \"Ubiquitin's just a rock\" (Berk Hess)\n",
            "\n",
            "                      :-) GROMACS - gmx grompp, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
            "\n",
            "Generating 1-4 interactions: fudge = 1\n",
            "Number of degrees of freedom in T-Coupling group System is 92284.00\n",
            "\n",
            "GROMACS reminds you: \"Discovery: A couple of months in the laboratory can frequently save a couple of hours in the library.\" (Anonymous)\n",
            "\n",
            "                      :-) GROMACS - gmx mdrun, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx mdrun -deffnm run -nsteps 1000000\n",
            "\n",
            "Compiled SIMD: AVX2_256, but for this host/run AVX_512 might be better (see\n",
            "log).\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Overriding nsteps with value passed on the command line: 1000000 steps, 2e+03 ps\n",
            "Changing nstlist from 10 to 100, rlist from 1.2 to 1.344\n",
            "\n",
            "\n",
            "1 GPU selected for this run.\n",
            "Mapping of GPU IDs to the 2 GPU tasks in the 1 rank on this node:\n",
            "  PP:0,PME:0\n",
            "PP tasks will do (non-perturbed) short-ranged interactions on the GPU\n",
            "PP task will update and constrain coordinates on the CPU\n",
            "PME tasks will do all aspects on the GPU\n",
            "Using 1 MPI thread\n",
            "Using 2 OpenMP threads \n",
            "\n",
            "starting mdrun 'Protein in water'\n",
            "1000000 steps,   2000.0 ps.\n",
            "\n",
            "Writing final coordinates.\n",
            "\n",
            "NOTE: 10 % of the run time was spent in pair search,\n",
            "      you might want to increase nstlist (this has no effect on accuracy)\n",
            "\n",
            "               Core t (s)   Wall t (s)        (%)\n",
            "       Time:     4349.434     2174.717      200.0\n",
            "                         36:14\n",
            "                 (ns/day)    (hour/ns)\n",
            "Performance:       79.459        0.302\n",
            "\n",
            "GROMACS reminds you: \"California, R.I.P.\" (Red Hot Chili Peppars)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "gmx grompp -f md.mdp -c 7_npt.gro -p topol.top -o run.tpr\n",
        "gmx mdrun -deffnm run -nsteps 1000000\n",
        "done"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bzmIt5F6FLri",
        "outputId": "bedd697f-975a-46e3-d818-1f34c3f81936"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n",
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n",
            "Note that major changes are planned in future for trjconv, to improve usability and utility.\n",
            "Select group for output\n",
            "Selected 0: 'System'\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_1aki\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 49829 elements\n",
            "Group     1 (        Protein) has  1960 elements\n",
            "Group     2 (      Protein-H) has  1001 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1316 elements\n",
            "Group     9 (    SideChain-H) has   486 elements\n",
            "Group    10 (    Prot-Masses) has  1960 elements\n",
            "Group    11 (    non-Protein) has 47869 elements\n",
            "Group    12 (          Water) has 47769 elements\n",
            "Group    13 (            SOL) has 47769 elements\n",
            "Group    14 (      non-Water) has  2060 elements\n",
            "Group    15 (          Other) has   100 elements\n",
            "Group    16 (            SOD) has    46 elements\n",
            "Group    17 (            CLA) has    54 elements\n",
            "Select a group: \r",
            "Reading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\r",
            "Reading frame       1 time   10.000    ->  frame      0 time    0.000      \r",
            "\r",
            "Reading frame       2 time   20.000    ->  frame      1 time   10.000      \r",
            "\r",
            "Reading frame       3 time   30.000    ->  frame      2 time   20.000      \r",
            "\r",
            "Reading frame       4 time   40.000    ->  frame      3 time   30.000      \r",
            "\r",
            "Reading frame       5 time   50.000    ->  frame      4 time   40.000      \r",
            "\r",
            "Reading frame       6 time   60.000    ->  frame      5 time   50.000      \r",
            "\r",
            "Reading frame       7 time   70.000    ->  frame      6 time   60.000      \r",
            "\r",
            "Reading frame       8 time   80.000    ->  frame      7 time   70.000      \r",
            "\r",
            "Reading frame       9 time   90.000    ->  frame      8 time   80.000      \r",
            "\r",
            "Reading frame      10 time  100.000    ->  frame      9 time   90.000      \r",
            "\r",
            "Reading frame      11 time  110.000    ->  frame     10 time  100.000      \r",
            "\r",
            "Reading frame      12 time  120.000    ->  frame     11 time  110.000      \r",
            "\r",
            "Reading frame      13 time  130.000    ->  frame     12 time  120.000      \r",
            "\r",
            "Reading frame      14 time  140.000    ->  frame     13 time  130.000      \r",
            "\r",
            "Reading frame      15 time  150.000    ->  frame     14 time  140.000      \r",
            "\r",
            "Reading frame      16 time  160.000    ->  frame     15 time  150.000      \r",
            "\r",
            "Reading frame      17 time  170.000    ->  frame     16 time  160.000      \r",
            "\r",
            "Reading frame      18 time  180.000    ->  frame     17 time  170.000      \r",
            "\r",
            "Reading frame      19 time  190.000    ->  frame     18 time  180.000      \r",
            "\r",
            "Reading frame      20 time  200.000    ->  frame     19 time  190.000      \r",
            "\r",
            "Reading frame      30 time  300.000    ->  frame     29 time  290.000      \r",
            "\r",
            "Reading frame      40 time  400.000    ->  frame     39 time  390.000      \r",
            "\r",
            "Reading frame      50 time  500.000    ->  frame     49 time  490.000      \r",
            "\r",
            "Reading frame      60 time  600.000    ->  frame     59 time  590.000      \r",
            "\r",
            "Reading frame      70 time  700.000    ->  frame     69 time  690.000      \r",
            "\r",
            "Reading frame      80 time  800.000    ->  frame     79 time  790.000      \r",
            "\r",
            "Reading frame      90 time  900.000    ->  frame     89 time  890.000      \r",
            "\r",
            "Reading frame     100 time 1000.000    ->  frame     99 time  990.000      \r",
            "\r",
            "Reading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r",
            "\r",
            "Reading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r",
            "\r",
            "Reading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r",
            "\r",
            "Reading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r",
            "\r",
            "Reading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r",
            "\r",
            "Reading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r",
            "\r",
            "Reading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r",
            "\r",
            "Reading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r",
            "\r",
            "Reading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r",
            "\r",
            "Reading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r",
            "\r",
            "Last frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"Microsecond Here I Come\" (P.J. Van Maaren)\n",
            "\n",
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_0\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 47649 elements\n",
            "Group     1 (        Protein) has  1949 elements\n",
            "Group     2 (      Protein-H) has   994 elements\n",
            "Group     3 (        C-alpha) has   129 elements\n",
            "Group     4 (       Backbone) has   387 elements\n",
            "Group     5 (      MainChain) has   515 elements\n",
            "Group     6 (   MainChain+Cb) has   632 elements\n",
            "Group     7 (    MainChain+H) has   644 elements\n",
            "Group     8 (      SideChain) has  1305 elements\n",
            "Group     9 (    SideChain-H) has   479 elements\n",
            "Group    10 (    Prot-Masses) has  1949 elements\n",
            "Group    11 (    non-Protein) has 45700 elements\n",
            "Group    12 (          Water) has 45606 elements\n",
            "Group    13 (            SOL) has 45606 elements\n",
            "Group    14 (      non-Water) has  2043 elements\n",
            "Group    15 (          Other) has    94 elements\n",
            "Group    16 (            SOD) has    43 elements\n",
            "Group    17 (            CLA) has    51 elements\n",
            "Select a group: \r",
            "Reading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\r",
            "Reading frame       1 time   10.000    ->  frame      0 time    0.000      \r",
            "\r",
            "Reading frame       2 time   20.000    ->  frame      1 time   10.000      \r",
            "\r",
            "Reading frame       3 time   30.000    ->  frame      2 time   20.000      \r",
            "\r",
            "Reading frame       4 time   40.000    ->  frame      3 time   30.000      \r",
            "\r",
            "Reading frame       5 time   50.000    ->  frame      4 time   40.000      \r",
            "\r",
            "Reading frame       6 time   60.000    ->  frame      5 time   50.000      \r",
            "\r",
            "Reading frame       7 time   70.000    ->  frame      6 time   60.000      \r",
            "\r",
            "Reading frame       8 time   80.000    ->  frame      7 time   70.000      \r",
            "\r",
            "Reading frame       9 time   90.000    ->  frame      8 time   80.000      \r",
            "\r",
            "Reading frame      10 time  100.000    ->  frame      9 time   90.000      \r",
            "\r",
            "Reading frame      11 time  110.000    ->  frame     10 time  100.000      \r",
            "\r",
            "Reading frame      12 time  120.000    ->  frame     11 time  110.000      \r",
            "\r",
            "Reading frame      13 time  130.000    ->  frame     12 time  120.000      \r",
            "\r",
            "Reading frame      14 time  140.000    ->  frame     13 time  130.000      \r",
            "\r",
            "Reading frame      15 time  150.000    ->  frame     14 time  140.000      \r",
            "\r",
            "Reading frame      16 time  160.000    ->  frame     15 time  150.000      \r",
            "\r",
            "Reading frame      17 time  170.000    ->  frame     16 time  160.000      \r",
            "\r",
            "Reading frame      18 time  180.000    ->  frame     17 time  170.000      \r",
            "\r",
            "Reading frame      19 time  190.000    ->  frame     18 time  180.000      \r",
            "\r",
            "Reading frame      20 time  200.000    ->  frame     19 time  190.000      \r",
            "\r",
            "Reading frame      30 time  300.000    ->  frame     29 time  290.000      \r",
            "\r",
            "Reading frame      40 time  400.000    ->  frame     39 time  390.000      \r",
            "\r",
            "Reading frame      50 time  500.000    ->  frame     49 time  490.000      \r",
            "\r",
            "Reading frame      60 time  600.000    ->  frame     59 time  590.000      \r",
            "\r",
            "Reading frame      70 time  700.000    ->  frame     69 time  690.000      \r",
            "\r",
            "Reading frame      80 time  800.000    ->  frame     79 time  790.000      \r",
            "\r",
            "Reading frame      90 time  900.000    ->  frame     89 time  890.000      \r",
            "\r",
            "Reading frame     100 time 1000.000    ->  frame     99 time  990.000      \r",
            "\r",
            "Reading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r",
            "\r",
            "Reading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r",
            "\r",
            "Reading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r",
            "\r",
            "Reading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r",
            "\r",
            "Reading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r",
            "\r",
            "Reading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r",
            "\r",
            "Reading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r",
            "\r",
            "Reading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r",
            "\r",
            "Reading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r",
            "\r",
            "Reading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r",
            "\r",
            "Last frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"If at first you don't succeed, try two more times so that your failure is statistically significant.\" (Dallas Warren)\n",
            "\n",
            "                     :-) GROMACS - gmx trjconv, 2022.3 (-:\n",
            "\n",
            "Executable:   /content/gromacs-2022/bin/gmx\n",
            "Data prefix:  /content/gromacs-2022\n",
            "Working dir:  /content/md_mutant_1\n",
            "Command line:\n",
            "  gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
            "\n",
            "Will write xtc: Compressed trajectory (portable xdr format): xtc\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Reading file run.tpr, VERSION 2022.3 (single precision)\n",
            "Group     0 (         System) has 45602 elements\n",
            "Group     1 (        Protein) has  1939 elements\n",
            "Group     2 (      Protein-H) has   989 elements\n",
            "Group     3 (        C-alpha) has   128 elements\n",
            "Group     4 (       Backbone) has   384 elements\n",
            "Group     5 (      MainChain) has   511 elements\n",
            "Group     6 (   MainChain+Cb) has   627 elements\n",
            "Group     7 (    MainChain+H) has   639 elements\n",
            "Group     8 (      SideChain) has  1300 elements\n",
            "Group     9 (    SideChain-H) has   478 elements\n",
            "Group    10 (    Prot-Masses) has  1939 elements\n",
            "Group    11 (    non-Protein) has 43663 elements\n",
            "Group    12 (          Water) has 43569 elements\n",
            "Group    13 (            SOL) has 43569 elements\n",
            "Group    14 (      non-Water) has  2033 elements\n",
            "Group    15 (          Other) has    94 elements\n",
            "Group    16 (            SOD) has    43 elements\n",
            "Group    17 (            CLA) has    51 elements\n",
            "Select a group: \r",
            "Reading frame       0 time    0.000   \n",
            "Precision of run.xtc is 0.001 (nm)\n",
            "Using output precision of 0.001 (nm)\n",
            "\r",
            "Reading frame       1 time   10.000    ->  frame      0 time    0.000      \r",
            "\r",
            "Reading frame       2 time   20.000    ->  frame      1 time   10.000      \r",
            "\r",
            "Reading frame       3 time   30.000    ->  frame      2 time   20.000      \r",
            "\r",
            "Reading frame       4 time   40.000    ->  frame      3 time   30.000      \r",
            "\r",
            "Reading frame       5 time   50.000    ->  frame      4 time   40.000      \r",
            "\r",
            "Reading frame       6 time   60.000    ->  frame      5 time   50.000      \r",
            "\r",
            "Reading frame       7 time   70.000    ->  frame      6 time   60.000      \r",
            "\r",
            "Reading frame       8 time   80.000    ->  frame      7 time   70.000      \r",
            "\r",
            "Reading frame       9 time   90.000    ->  frame      8 time   80.000      \r",
            "\r",
            "Reading frame      10 time  100.000    ->  frame      9 time   90.000      \r",
            "\r",
            "Reading frame      11 time  110.000    ->  frame     10 time  100.000      \r",
            "\r",
            "Reading frame      12 time  120.000    ->  frame     11 time  110.000      \r",
            "\r",
            "Reading frame      13 time  130.000    ->  frame     12 time  120.000      \r",
            "\r",
            "Reading frame      14 time  140.000    ->  frame     13 time  130.000      \r",
            "\r",
            "Reading frame      15 time  150.000    ->  frame     14 time  140.000      \r",
            "\r",
            "Reading frame      16 time  160.000    ->  frame     15 time  150.000      \r",
            "\r",
            "Reading frame      17 time  170.000    ->  frame     16 time  160.000      \r",
            "\r",
            "Reading frame      18 time  180.000    ->  frame     17 time  170.000      \r",
            "\r",
            "Reading frame      19 time  190.000    ->  frame     18 time  180.000      \r",
            "\r",
            "Reading frame      20 time  200.000    ->  frame     19 time  190.000      \r",
            "\r",
            "Reading frame      30 time  300.000    ->  frame     29 time  290.000      \r",
            "\r",
            "Reading frame      40 time  400.000    ->  frame     39 time  390.000      \r",
            "\r",
            "Reading frame      50 time  500.000    ->  frame     49 time  490.000      \r",
            "\r",
            "Reading frame      60 time  600.000    ->  frame     59 time  590.000      \r",
            "\r",
            "Reading frame      70 time  700.000    ->  frame     69 time  690.000      \r",
            "\r",
            "Reading frame      80 time  800.000    ->  frame     79 time  790.000      \r",
            "\r",
            "Reading frame      90 time  900.000    ->  frame     89 time  890.000      \r",
            "\r",
            "Reading frame     100 time 1000.000    ->  frame     99 time  990.000      \r",
            "\r",
            "Reading frame     110 time 1100.000    ->  frame    109 time 1090.000      \r",
            "\r",
            "Reading frame     120 time 1200.000    ->  frame    119 time 1190.000      \r",
            "\r",
            "Reading frame     130 time 1300.000    ->  frame    129 time 1290.000      \r",
            "\r",
            "Reading frame     140 time 1400.000    ->  frame    139 time 1390.000      \r",
            "\r",
            "Reading frame     150 time 1500.000    ->  frame    149 time 1490.000      \r",
            "\r",
            "Reading frame     160 time 1600.000    ->  frame    159 time 1590.000      \r",
            "\r",
            "Reading frame     170 time 1700.000    ->  frame    169 time 1690.000      \r",
            "\r",
            "Reading frame     180 time 1800.000    ->  frame    179 time 1790.000      \r",
            "\r",
            "Reading frame     190 time 1900.000    ->  frame    189 time 1890.000      \r",
            "\r",
            "Reading frame     200 time 2000.000    ->  frame    199 time 1990.000      \r",
            "\r",
            "Last frame        200 time 2000.000   \n",
            " ->  frame    200 time 2000.000      \r\n",
            "Last written: frame    200 time 2000.000\n",
            "\n",
            "\n",
            "GROMACS reminds you: \"I'm Only Faking When I Get It Right\" (Soundgarden)\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#@title make trajectories whole\n",
        "%%bash\n",
        "source $GMX_LOCAL_PATH/bin/GMXRC\n",
        "# Declare a filename array\n",
        "arrVar=(${MD_PATH})\n",
        "\n",
        "# Add mutants\n",
        "for (( i=0; i<${N_MUTANT}; i++ ))\n",
        "do\n",
        "arrVar+=(\"./md_mutant_${i}\")\n",
        "done\n",
        "\n",
        "for value in \"${arrVar[@]}\"\n",
        "do\n",
        "cd /content/\n",
        "cd $value\n",
        "echo 0 | gmx trjconv -f run.xtc -s run.tpr -pbc mol -o run_nj.xtc\n",
        "done"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9WK7EtsiG_Wp"
      },
      "source": [
        "# Analysis"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 654
        },
        "id": "DBUzmhW5HCpl",
        "outputId": "c31d25b6-c628-4c7c-dc28-dd8239e95624"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "./md_1aki/\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.7/site-packages/MDAnalysis/analysis/align.py:950: DeprecationWarning: The `universe` attribute was deprecated in MDAnalysis 2.0.0 and will be removed in MDAnalysis 3.0.0. Please use `results.universe` instead.\n",
            "  warnings.warn(wmsg, DeprecationWarning)\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "./md_mutant_0/\n",
            "./md_mutant_1/\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"854c7745-8e26-48ec-8dae-86a4eb8beb72\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"854c7745-8e26-48ec-8dae-86a4eb8beb72\")) {                    Plotly.newPlot(                        \"854c7745-8e26-48ec-8dae-86a4eb8beb72\",                        [{\"hovertemplate\":\"variable=RMSD, ./md_1aki/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_1aki/\",\"line\":{\"color\":\"#636efa\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_1aki/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[1.032046167612902,1.1185287201515637,0.9053039117305199,1.096550776622478,0.8569471999476059,0.6623549406936307,0.681778306499831,0.6267686443725318,0.6098904116771962,0.5982637032141044,0.6314692802560377,0.693832312370695,0.712689293885094,0.8309735508925363,0.7160732778835688,0.6532374845747646,0.637583591452282,0.6348954905915243,0.6966604526475105,0.6968950078536992,0.5944849709091232,0.7187256625097829,0.7363351007904025,0.6212901030506489,0.6729408208304348,0.5554389212747025,0.5617077111978223,0.56571268657775,0.5545824217700807,0.5680537328321923,0.5867247243876421,0.6693554558872484,0.6446586939657045,0.6363748680586134,0.6747921231735874,0.6071540103303,0.581907182760015,0.6946948248955014,0.677356549696271,0.6035162641964112,0.6589939192242869,0.619854907814415,0.7282623814625158,0.5284372238090012,0.7449222734214063,0.7046463591874618,0.6085383831947432,0.7566785381708997,0.6637487004864056,0.6877972796228419,0.5582861043728765,0.5276558828700373,0.6174995220814532,0.6727017523836123,0.6281796913945263,0.5495968962493648,0.6351747017371681,0.627287437102995,0.6303993692380706,0.5223752449776417,0.5824735785345144,0.6297267860436583,0.6051573215984482,0.6800974643706139,0.49395036454330254,0.7096282415383902,0.6776376846124672,0.6839558851179192,0.503129719082874,0.477509853512699,0.642866351139969,0.5453779865726373,0.7122522772563246,0.5283285702648822,0.4672028176681692,0.5368303436243438,0.4943627453117062,0.49458767344570154,0.4722835768199092,0.6474300248071415,0.5727165406237953,0.5000463494759019,0.5544827918471928,0.5952259778642874,0.6239664479451075,0.5265406343592773,0.47385002909762913,0.5604109630882516,0.7567102715644147,0.6451527482447709,0.5256640719773212,0.5336940938226433,0.5134490500180161,0.5562553109110584,0.596945607466486,0.473013639605088,0.46527565681500954,0.5027812029163647,0.5389167862401805,0.5153942210911894,0.5201799395774083,0.49959258409376667,0.5397946007938622,0.49591918809466995,0.5323664873025759,0.5423094599456173,0.5667855420291555,0.49317051639364506,0.5549037716225255,0.48850835121472896,0.48057737021195485,0.5273444140210441,0.6031341673344883,0.531098828144626,0.548709306447126,0.6570930520056151,0.6112580596105617,0.5886995673336353,0.7032169544519802,0.6725294105946029,0.6416115398351864,0.5384127950433936,0.7768893319773776,0.45030929247685386,0.5318452083340797,0.5730391501522434,0.5307333166507427,0.4482125337847616,0.5766388631623679,0.6344248023151627,0.5671042237740269,0.4974590251630182,0.4703031180631319,0.47456756809704925,0.5329775907912593,0.5097068974120458,0.5360676142061581,0.5308997944201521,0.45687088195467074,0.5905022284130305,0.6829611404740132,0.5872946675500536,0.6091910058607112,0.510219196220267,0.6127036205121118,0.6075800898217022,0.5342136302294059,0.5223155829285429,0.5713120435239806,0.6491385378317993,0.6953940143876028,0.6493919449331373,0.5830900808979392,0.539874618628931,0.5089336679581667,0.5603144172749372,0.4857634968170615,0.5001399144772399,0.5375872767908646,0.5130743726167859,0.4987211097126465,0.5254609627848581,0.5609490931481168,0.5739689187496014,0.5181234023628852,0.4894535931390501,0.5311505706603818,0.6222725105780872,0.6842645585526926,0.6113872402195338,0.5239929648614972,0.5134400748468773,0.5419438074088183,0.5832612267847865,0.6521648959541768,0.5618551705459967,0.6517564364165281,0.5579349614312972,0.6257294629498745,0.6038451978992304,0.5025174422235265,0.5221754448601554,0.706579533796579,0.6945046350765325,0.5565235006334751,0.5510581667068304,0.5540912903837921,0.6119194012037941,0.5621954538931098,0.7043451551236283,0.5400545699449205,0.7175302878048971,0.5483601628092931,0.5517408301534846,0.5781762746694978,0.5911353207522784,0.6152167441704078,0.6068584123042037,0.6381491711222889,0.7469851976454915,0.6399852285441667],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"variable=RMSD, ./md_mutant_0/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_mutant_0/\",\"line\":{\"color\":\"#EF553B\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_mutant_0/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[0.8950691283213248,0.9330506702836896,0.9629492534352861,0.9014369264225996,0.7098989785086244,0.7505119550103851,0.7513122126319364,0.6530786464340292,0.6200255894062668,0.5640816637011241,0.7623003565500289,0.6764313501797952,0.5467265807133349,0.5398768259215796,0.7689001157486994,0.7149990634640426,0.8092735838233949,0.8347750123117836,1.034462286506098,0.7985911626554008,0.7874113701577683,0.719529463281206,0.7230121280539931,0.6931137170986094,0.6926764559225104,0.6381858336002099,0.7750054793586917,0.7408981270547716,0.7701567411220986,0.7347039292206463,0.7138656439356634,0.8267962279336547,0.7190578480662766,0.6813235570945719,0.7496681841796925,0.7106150422507188,0.6959149924416773,0.6080627025461992,0.7475703261295766,0.732083121373128,0.7273982151750275,0.70579346420091,0.6375994271319497,0.7507100050081401,0.5224469929122376,0.5434694670148456,0.6776691456877357,0.6181507678343403,0.62775583755458,0.6587004692333172,0.4781809558165378,0.5425834904913681,0.46823171139941655,0.6312597070779906,0.7224533084644799,0.7676083510864062,0.5876178505554986,0.7459100815976574,0.6344625557504687,0.7193169424373183,0.632230499657909,0.6217559046485365,0.5947592761297589,0.6130246775273163,0.5274825665917566,0.5974213880666223,0.6176548628621945,0.4949588600737308,0.5735533767433332,0.6286903300489566,0.6845973266064451,0.6351886884433041,0.6844237676458131,0.7560516958130707,0.6989574083739538,0.6912306466927657,0.5637417832456513,0.6374796085100722,0.5919329478702928,0.5434883786035933,0.5396041949032179,0.60220613793613,0.5598555626015238,0.5843503989583712,0.6618408944202693,0.5983442354351383,0.6851218622919669,0.9507571609721818,0.6833152511858949,0.5242076175566981,0.5763003002160035,0.6715270480047165,0.8887988878311499,0.8490826794928182,0.643381193705708,0.5888937606064665,0.6350024874655006,0.6443039464340006,0.6712712330469355,0.6504860610204671,0.7368531895112104,0.6647912928283207,0.7904681773235823,0.7802854321050882,0.8387152534052721,0.6590322634432528,0.7238031776111834,0.9432692544191965,0.713904726068527,0.6266258338322745,0.6453869909848863,0.657935857861034,0.6811815875737337,0.6396187487818671,0.6975445941928636,0.7010261141240398,0.6408211138740119,0.6660896311808716,0.5853396187533817,0.7145635150168108,0.6186255438499946,0.5969519118315705,0.6467559794342275,0.6819658827264542,0.63603311119373,0.6297738514893394,0.6225069217673148,0.6078896665089722,0.716574220280723,0.6732675347236662,0.7320402148167336,0.652992152103134,0.8146826711798022,0.7377276342591335,0.8383411670454468,0.7169322636177792,0.6372328068713391,0.6799146634177806,0.6979552894721543,0.6732937528065969,0.5147819087263966,0.7103580934997419,0.6646651577988654,0.7218888097353934,0.647117365251897,0.5866094505636923,0.5193080975838472,0.514588632757893,0.5585843395890192,0.7113798686596811,0.5892110034606793,0.7072381209051167,0.8028877901263968,0.8443576546576514,0.8824911404474756,0.7727269074241718,0.8231731201017233,0.569210109057893,0.667688395627092,0.6469625402601931,0.6121805788388087,0.5800448617207979,0.7531894694816159,0.79164443364741,0.6881048781190203,0.6709236924674382,0.7296252194581856,0.8078610477862908,0.844967898079777,0.7766364638274493,0.8521839105105198,0.6889348272752375,0.7267464809056479,0.7248996009292756,0.690364934492127,0.7574402728071652,0.7358700914273146,0.9050942775961274,0.8762343208476625,0.7971999500506398,0.6703127200060552,0.6729923052959642,0.6667534808855887,0.814564951262505,0.6395621606725868,0.7736264800214666,0.7769443120538794,0.7929281953092027,0.7315605647671329,0.6027703920405673,0.5856677975023528,0.7829390242842506,0.5948421851195214,0.6183969350968722,0.7055657580113154,0.6522341476118848,0.5936605970498846,0.6635194333352338,0.6181620469403598,0.7194649284482415,0.7399138541163629],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"variable=RMSD, ./md_mutant_1/<br>Time (ns)=%{x}<br>RMSD (A)=%{y}<extra></extra>\",\"legendgroup\":\"RMSD, ./md_mutant_1/\",\"line\":{\"color\":\"#00cc96\",\"dash\":\"solid\",\"shape\":\"linear\"},\"marker\":{\"symbol\":\"circle\"},\"mode\":\"lines\",\"name\":\"RMSD, ./md_mutant_1/\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[0.0,0.01,0.02,0.03,0.04,0.05,0.06,0.07,0.08,0.09,0.1,0.11,0.12,0.13,0.14,0.15,0.16,0.17,0.18,0.19,0.2,0.21,0.22,0.23,0.24,0.25,0.26,0.27,0.28,0.29,0.3,0.31,0.32,0.33,0.34,0.35,0.36,0.37,0.38,0.39,0.4,0.41,0.42,0.43,0.44,0.45,0.46,0.47,0.48,0.49,0.5,0.51,0.52,0.53,0.54,0.55,0.56,0.57,0.58,0.59,0.6,0.61,0.62,0.63,0.64,0.65,0.66,0.67,0.68,0.69,0.7,0.71,0.72,0.73,0.74,0.75,0.76,0.77,0.78,0.79,0.8,0.81,0.82,0.83,0.84,0.85,0.86,0.87,0.88,0.89,0.9,0.91,0.92,0.93,0.94,0.95,0.96,0.97,0.98,0.99,1.0,1.01,1.02,1.03,1.04,1.05,1.06,1.07,1.08,1.09,1.1,1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18,1.19,1.2,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,1.3,1.31,1.32,1.33,1.34,1.35,1.36,1.37,1.38,1.39,1.4,1.41,1.42,1.43,1.44,1.45,1.46,1.47,1.48,1.49,1.5,1.51,1.52,1.53,1.54,1.55,1.56,1.57,1.58,1.59,1.6,1.61,1.62,1.63,1.64,1.65,1.66,1.67,1.68,1.69,1.7,1.71,1.72,1.73,1.74,1.75,1.76,1.77,1.78,1.79,1.8,1.81,1.82,1.83,1.84,1.85,1.86,1.87,1.88,1.89,1.9,1.91,1.92,1.93,1.94,1.95,1.96,1.97,1.98,1.99,2.0],\"xaxis\":\"x\",\"y\":[0.9801477763521751,0.7880427152972244,0.855545680442817,0.9852744314907413,1.0131982797764445,0.9438787906356434,1.0493673124661576,0.9178856029428756,0.8613044180099316,0.8140677535784075,0.7914496444308067,0.8375986105687059,0.8617376472662067,0.7717351714654814,0.8606976679147849,0.7952436805567896,0.899346173760099,0.8873194459681459,0.8146739866242194,0.8847531675511334,0.7244472301373095,0.7036565629883436,0.8703620150539823,0.8292691049672817,0.9380888746001486,0.9553133334139056,0.8088296557040754,0.7892944283438644,0.8936410804253999,0.9009203833265458,0.7254004508685998,0.7987462852492363,0.8209855220426773,0.9121004782880905,0.9551512753727368,0.9245663192735264,0.9458905778377926,1.002314978443962,0.8343143755590204,0.7242336562646784,0.8701838923081673,0.771123612713634,0.7946482685502512,0.8413198178136733,0.8106790068479435,0.8046486453146658,0.8133706623098602,0.9363340308969317,0.6968704708690712,0.7198541990940941,0.8382538235233616,0.6862918216652732,0.7316491285935042,1.011422118326304,0.7159748387155725,0.7621289630216386,0.6296779195103905,0.7673039429803535,0.7618417772883694,0.7231138785189741,0.8717399289271559,0.7433296274869668,0.7164751471776556,0.6554624890758449,0.6435088802770018,0.8406378535508745,0.8370171189743946,0.7102029924398566,0.7479607095197128,0.622869490840246,0.7042779384729997,0.833552091528869,0.8297763699612104,0.8326646266419323,0.8316202391681293,0.9772765906967015,0.678053232633059,0.7187827285582736,0.7550707080191973,0.8704363246915319,0.8708078924670594,0.9281098922205859,0.8589361115064311,0.7288356515241313,0.8512855445662045,0.6627894510293071,0.7053927958389011,0.8802814891260673,0.95836367596443,0.8457308532931829,0.7297183625964304,0.7913492740640258,0.8017741500782968,0.7707714603620058,0.8558789074162402,0.7737279464051118,0.6698573052220069,0.7777856160151843,0.9182922682636672,0.8954752658166047,0.782690896457608,0.8927864738789812,0.9696228893456047,0.6137263580929448,0.7499332353206388,0.7757206616740235,0.6754659427031056,0.7308503454809736,0.714478335554908,0.749612319345148,0.7774792800581328,0.693611127375695,0.5876034826083185,0.6533038936717077,0.77711312315606,0.5827270204025208,0.6479627591875162,0.6335492112967199,0.6715950396428507,0.5165550723484915,0.6904951328684757,0.5929053820733184,0.6808357084585306,0.7447913468119552,0.739387751935155,0.5821888007887761,0.6131891933184735,0.6010149372228576,0.757973055263559,0.6586994822060873,0.7868565473909399,0.7829711050051074,0.7964145035476736,0.9043776379826093,0.8793037787018592,0.7471814769082922,0.7639049971985936,0.8374664452964428,0.758754496589933,0.8932435987403324,0.904381263997207,0.8150407213581856,0.7330954546306339,0.9831445040289231,1.0005509222225448,0.9799861045117457,0.9622165390970389,0.8895985450789385,0.987281233514066,0.9363903589063867,1.0090875656183864,0.8925997653681251,0.9154517668635428,1.022422179574082,1.0658641614477469,0.8786290449154478,0.9122093922322302,1.0286609308553567,1.0192859402892167,0.9371928032601478,0.9033347373562319,0.9159066993817582,1.0494739697877378,0.9225100581385169,0.9202461861837705,0.9913820301545369,0.8749338142921257,0.9908411158304186,0.9314751755753814,0.9618641009600561,0.942307683182546,1.0168523146851722,1.0344501103788795,1.0571932747302495,0.9753629064633156,1.1111568171910222,1.1449594056786,0.9896398506685339,1.191565211129711,1.0912233936043214,1.1411857137624648,1.1307602327357458,1.0600781635853889,1.2748604909420511,1.1219382885211775,0.9410162405313741,0.9320139592991049,1.0715946120159583,1.1264391615369993,1.0120337170762779,1.3316205061294446,1.1837662046859199,1.1489589101031874,1.17940553152056,1.1361321031309486,1.1123982978599145,1.1167852429906082,1.0219278207083773,1.205790979448861,1.111168015303999,1.046476108130852],\"yaxis\":\"y\",\"type\":\"scatter\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Time (ns)\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"RMSD (A)\"}},\"legend\":{\"title\":{\"text\":\"variable\"},\"tracegroupgap\":0},\"margin\":{\"t\":60}},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('854c7745-8e26-48ec-8dae-86a4eb8beb72');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "#@title RMSD\n",
        "paths = [path]\n",
        "for i in range(mutant_n):\n",
        "    paths.append(f\"./md_mutant_{i}/\")\n",
        "\n",
        "first = True\n",
        "rmsdList = []\n",
        "for current_path in paths:\n",
        "    mobile = mda.Universe(f\"{current_path}run.gro\", f\"{current_path}run_nj.xtc\")\n",
        "    averageBB = align.AverageStructure(mobile, \n",
        "                                       mobile, \n",
        "                                       select='protein and backbone',\n",
        "                                       ref_frame=0).run()\n",
        "    refBB = averageBB.universe\n",
        "    R_rmsd = rms.RMSD(mobile,  # universe to align\n",
        "                     refBB,  # reference universe or atomgroup\n",
        "                     select='protein and backbone')  # group to superimpose and calculate RMSD\n",
        "        \n",
        "    R_rmsd.run()\n",
        "    if first:\n",
        "        df_rmsd = pd.DataFrame(R_rmsd.results.rmsd,\n",
        "                  columns=['Frame', 'Time (ns)',f'RMSD, {current_path}'])\n",
        "        df_rmsd['Time (ns)'] /= 1000\n",
        "        rmsdList.append(f'RMSD, {current_path}')\n",
        "        first = False\n",
        "    else:\n",
        "        colname = f'RMSD, {current_path}'\n",
        "        df_rmsd[colname] = R_rmsd.results.rmsd[:,2]\n",
        "        rmsdList.append(colname)\n",
        "\n",
        "df_rmsd_melt = df_rmsd.melt(id_vars='Time (ns)', value_vars=rmsdList)\n",
        "df_rmsd_melt.rename(columns = {'value': 'RMSD (A)'}, inplace = True)\n",
        "fig = px.line(df_rmsd_melt, x=\"Time (ns)\", y=\"RMSD (A)\",\n",
        "              color='variable',\n",
        "              line_shape=\"linear\", render_mode=\"svg\", \n",
        "              labels={ \"Name CA\": \"RMSD(Å)\" })\n",
        "fig.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "QgBYxxDKo_9x"
      },
      "outputs": [],
      "source": []
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "provenance": []
    },
    "gpuClass": "standard",
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.9.14"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}